<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-01-28" />

<title>Bulk_cellComposition_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bulk_cellComposition_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-01-28</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-04-20
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.1). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree507af1a3aa2ec3771a3b2a37ce3a4db895a7773atargetblank507af1aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/507af1a3aa2ec3771a3b2a37ce3a4db895a7773a" target="_blank">507af1a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree507af1a3aa2ec3771a3b2a37ce3a4db895a7773atargetblank507af1aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/507af1a3aa2ec3771a3b2a37ce3a4db895a7773a" target="_blank">507af1a</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds

Untracked files:
    Untracked:  .RDataTmp

Unstaged changes:
    Modified:   code/jobname.332468.err
    Modified:   code/jobname.332468.out
    Modified:   data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Bulk_cellComposition_analysis.Rmd</code>) and HTML (<code>docs/Bulk_cellComposition_analysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/507af1a3aa2ec3771a3b2a37ce3a4db895a7773a/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">507af1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-20
</td>
<td>
add encode data to semisup
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/Bulk_cellComposition_analysis.html" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
<td>
Subset pseudobulk to 1000 DEG, dotplots instead of boxplots
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
Add boxplot by sample for top DEGenes; k=7 for supervised definingh early and late chondros
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/Bulk_cellComposition_analysis.html" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
Add boxplot by sample for top DEGenes; k=7 for supervised definingh early and late chondros
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/925e86dc706803cd69f455e15f93cb6879c77090/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">925e86d</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
add printout of psseudobulk correlations
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
define early/late passage chondros as a supervised cluster
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/940b53efe5b3edcac621e238213ebf8d896fd263/docs/Bulk_cellComposition_analysis.html" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
define early/late passage chondros as a supervised cluster
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6baf979f9ff594f844a14171435152ff5b0e7081/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">6baf979</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-10
</td>
<td>
Pseudobulk correlation heatmap with bulk samples
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
re-run semi-supervised without heart sample that is outlier
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/Bulk_cellComposition_analysis.html" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
re-run semi-supervised without heart sample that is outlier
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c9015d5f25785b2ffd97f0119b94015c2cc84498/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">c9015d5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Revise pseudobulk to use raw counts not normalized
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a46d6f37799b4f9c74c12be5c41c412f95f2fa3e/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">a46d6f3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Add gene analysis to semi-supervised topic model
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a46d6f37799b4f9c74c12be5c41c412f95f2fa3e/docs/Bulk_cellComposition_analysis.html" target="_blank">a46d6f3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Add gene analysis to semi-supervised topic model
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ad1e27bd3b051a731c2357fb8d8f981baa35f6af/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">ad1e27b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-02
</td>
<td>
separate up/down in de analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/68e5e33dc85bf6340efc50200e47b0766711c93c/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
<td>
add k = 8 for semi-supervised gom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/68e5e33dc85bf6340efc50200e47b0766711c93c/docs/Bulk_cellComposition_analysis.html" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
<td>
add k = 8 for semi-supervised gom
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/59b30d0b555983d416401d53765f1863f550e7ab/docs/Bulk_cellComposition_analysis.html" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
Semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/Bulk_cellComposition_analysis.html" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
Semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/Bulk_cellComposition_analysis.html" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
<td>
Separate out sample based on origin; also add pseudobulk
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/Bulk_cellComposition_analysis.html" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
<td>
Separate out sample based on origin; also add pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
check expression using RLE norm
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/Bulk_cellComposition_analysis.html" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
check expression using RLE norm
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/605da875aae021b28ea700a7c7af372b50a19dc6/docs/Bulk_cellComposition_analysis.html" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/2bbbdad15237735154e8db0270272fefd55760a9/docs/Bulk_cellComposition_analysis.html" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/26132157c6605eb09d8628dff14a004a6a374809/docs/Bulk_cellComposition_analysis.html" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="cluster-pilot-bulk-rna-data-with-subset-of-cellatlas-bulk-data-and-other-data-to-determine-where-my-samples-fall-in-comparison" class="section level1">
<h1>Cluster pilot Bulk RNA data with subset of cellAtlas bulk data and other data to determine where my samples fall in comparison</h1>
<div id="load-data" class="section level2">
<h2>Load data</h2>
<pre class="r"><code>library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)
library(&quot;cowplot&quot;)</code></pre>
<pre><code>Loading required package: ggplot2</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>library(&quot;CountClust&quot;)
library(&quot;maptpx&quot;)</code></pre>
<pre><code>Loading required package: slam</code></pre>
<pre class="r"><code>library(&quot;MAST&quot;)</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following object is masked from &#39;package:maptpx&#39;:

    normalize</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, cbind, colMeans, colnames,
    colSums, do.call, duplicated, eval, evalq, Filter, Find, get, grep,
    grepl, intersect, is.unsorted, lapply, lengths, Map, mapply, match,
    mget, order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,
    rbind, Reduce, rowMeans, rownames, rowSums, sapply, setdiff, sort,
    table, tapply, union, unique, unsplit, which, which.max, which.min</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    apply</code></pre>
<pre><code>
Attaching package: &#39;MAST&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    filter</code></pre>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;ggfortify&quot;)
library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:IRanges&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:S4Vectors&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code># load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

# Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))

#load cell atlas data matrix
load(&quot;data/cell_atlas_ref_panel&quot;)
cell.line &lt;- cell.line[,c(12:20,35,36,40,55:57)]

#load pilot data
raw_counts &lt;- as.matrix(readRDS(&quot;data/raw_counts_relabeled.rds&quot;))

# load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

#relabel pilot data genes to symbols
pilot &lt;- raw_counts
rownames(pilot) &lt;- gene_anno$external_gene_name[match(unlist(rownames(pilot)), gene_anno$ensembl_gene_id)]</code></pre>
</div>
<div id="before-combining-the-datasets-i-apply-the-same-standardizationsnormalizations-performed-on-the-cellatlas-data-to-my-pilot-data" class="section level2">
<h2>Before combining the datasets, I apply the same standardizations/normalizations performed on the cellAtlas data to my pilot data</h2>
<pre class="r"><code>#perform normalizations (taken from cellAtlas_assignment.R code)
pilot_norm &lt;- apply(pilot, 2, function(x) x/sum(x))
pilot_libsize &lt;- apply(pilot_norm, 2, function(x) x * 1e+06)
Data.use &lt;- pilot_libsize

#Only include genes that are common between the two data sets
gene.id &lt;- rownames(Data.use)
gene.ref &lt;- rownames(cell.line)
common &lt;- intersect(gene.id, gene.ref)
print(paste0(&quot;There are &quot;, length(common), &quot; genes in common between the two sets&quot;))</code></pre>
<pre><code>[1] &quot;There are 4204 genes in common between the two sets&quot;</code></pre>
<pre class="r"><code>logxx &lt;- apply(Data.use[common, ], 2, function(x) {
        log(x + 0.1)
    })
selected.cell.line &lt;- apply(cell.line[common, ], 2, function(x) {
        x - mean(x)
    })</code></pre>
</div>
<div id="combine-the-two-datasets-into-one-matrix" class="section level2">
<h2>Combine the two datasets into one matrix</h2>
<pre class="r"><code>combined &lt;- cbind(logxx, selected.cell.line)</code></pre>
</div>
<div id="clustering-pca" class="section level2">
<h2>Clustering (PCA)</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(combined), scale = T)

#Make PCA plots 

autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA%20cell%20atlas-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-cell-atlas-1">
Past versions of “PCA cell atlas-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-cell-atlas-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA cell atlas-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="clustering-corr-heatmap" class="section level2">
<h2>Clustering (corr heatmap)</h2>
<pre class="r"><code>#Corr heatmap
cors &lt;- cor(combined, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)
heatmap.2(cors, scale=&quot;none&quot;, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;,
          cexCol = 0.1 + 1/log10(15), cexRow = 0.1 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-1">
Past versions of “corr heatmap-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="outside-data-analysis" class="section level1">
<h1>Outside data analysis</h1>
<p>Compare pilot bulk data with bulk data collected from iPSCs, MSCs, chondrocytes from OA patients, and heart tissue (note: I have only used a Single read from each sample and trimmed to the first 50 bases to ensure that there are not mapping biases between samples). Here, I try both upperquartile and RLE normalization methods</p>
<pre class="r"><code># load in counts
outside_data_counts &lt;- read.table(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/bulkRNAseq_outsideData/out/counts/counts.txt&quot;, header = T)

# assign row.names
row.names(outside_data_counts) &lt;- outside_data_counts$Geneid

# exclude extra data columns
outside_data_counts &lt;- outside_data_counts[, -c(1:6)]

names(outside_data_counts) &lt;- c(&quot;MSC-Chond-d14_3&quot;, &quot;iPSC-MSC5_2&quot;, &quot;MSC-Chond-d3_3&quot;, &quot;adMSC&quot;, &quot;Heart_4&quot;,
                                &quot;Chond_enc_2&quot;, &quot;Lung_1&quot;, &quot;Chond_SCM_P4_1&quot;, &quot;bmMSC_2.2&quot;, &quot;MSC-Chond-d21_1&quot;,
                                &quot;Chond_TCP_P4_2&quot;, &quot;MSC-Chond-d3_1&quot;, &quot;Chond_SCM_P1_1&quot;, &quot;bmMSC_1.2&quot;, &quot;MSC-Chond-d0_2&quot;,
                                &quot;Chond_TCP_P1_2&quot;, &quot;MSC-Chond-d14_2&quot;, &quot;iPSC-MSC9_1&quot;, &quot;OA_dmg_2&quot;, &quot;Chond_TCP_P1_3&quot;,
                                &quot;Heart_2&quot;, &quot;MSC-Chond-d14_1&quot;, &quot;MSC-Chond-d1_3&quot;, &quot;Chond_SCM_P4_3&quot;, &quot;Kidney_2&quot;,
                                &quot;OA_dmg_3&quot;, &quot;Chond_SCM_P1_2&quot;, &quot;Chond_TCP_P1_1&quot;, &quot;MSC-Chond-d7_3&quot;, &quot;Liver_3&quot;,
                                &quot;Chond_SCM_P4_2&quot;, &quot;OA_dmg_1&quot;, &quot;MSC-Chond-d3_2&quot;, &quot;bmMSC_2.1&quot;, &quot;Kidney_4&quot;,
                                &quot;bmMSC_1.1&quot;, &quot;Lung_2&quot;, &quot;MSC-Chond-d21_3&quot;, &quot;Liver_2&quot;, &quot;Chond_TCP_P4_1&quot;, 
                                &quot;Heart_3&quot;, &quot;Lung_4&quot;, &quot;Heart_1&quot;, &quot;Chond_SCM_P1_3&quot;, &quot;Liver_4&quot;,
                                &quot;OA_intact_2&quot;, &quot;Liver_1&quot;, &quot;MSC-Chond-d1_2&quot;, &quot;Chond_enc_1&quot;, &quot;iPSC_1&quot;,
                                &quot;MSC-Chond-d1_1&quot;, &quot;MSC-Chond-d21_2&quot;, &quot;iPSC_2&quot;, &quot;iPSC-MSC5_1&quot;, &quot;Lung_3&quot;,
                                &quot;iPSC-MSC9_2&quot;, &quot;MSC-Chond-d7_1&quot;, &quot;Kidney_1&quot;, &quot;Chond_TCP_P4_3&quot;, &quot;OA_intact_3&quot;,
                                &quot;Kidney_3&quot;, &quot;OA_intact_1&quot;, &quot;bmMSC_1.3&quot;, &quot;MSC-Chond-d0_1&quot;, &quot;MSC-Chond-d0_3&quot;,
                                &quot;MSC-Chond-d7_2&quot;)

#remove MSC-chond samples and OA samples
MSC_chond_samples &lt;- which(grepl(&quot;MSC-Chond-&quot;, names(outside_data_counts)))
OA_samples &lt;- which(grepl(&quot;OA_&quot;, names(outside_data_counts)))
Heart1 &lt;- which(grepl(&quot;Heart_1&quot;, names(outside_data_counts))) #seems to cluster with liver rather than hearts!
outside_data_counts&lt;- outside_data_counts[,-c(MSC_chond_samples, OA_samples, Heart1)]

#Combine all bulk data together
merged_raw_data &lt;- merge(raw_counts, outside_data_counts, by = &quot;row.names&quot;)
row.names(merged_raw_data) &lt;- merged_raw_data$Row.names
merged_raw_data &lt;- merged_raw_data[,-1]

#Remove rows with all 0s
dim(merged_raw_data)</code></pre>
<pre><code>[1] 58395    58</code></pre>
<pre class="r"><code>merged_raw_data &lt;- merged_raw_data[apply(merged_raw_data, 1, function(x) !all(x==0)),]
dim(merged_raw_data)</code></pre>
<pre><code>[1] 49481    58</code></pre>
<pre class="r"><code>#indices for later
adMSC_index &lt;- which(grepl(&quot;adMSC&quot;, names(merged_raw_data)))
bmMSC1_index &lt;- which(grepl(&quot;bmMSC_1.&quot;, names(merged_raw_data)))
bmMSC2_index &lt;- which(grepl(&quot;bmMSC_2.&quot;, names(merged_raw_data)))
early_chond &lt;- which(grepl(&quot;Chond_SCM_P1|Chond_TCP_P1&quot;, names(merged_raw_data)))
late_chond &lt;- which(grepl(&quot;Chond_SCM_P4|Chond_TCP_P4&quot;, names(merged_raw_data)))
chond_encode &lt;- which(grepl(&quot;Chond_enc&quot;, names(merged_raw_data)))</code></pre>
<div id="raw-counts-analysis" class="section level2">
<h2>Raw counts analysis</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(merged_raw_data)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/raw%20analysis-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-raw-analysis-1">
Past versions of “raw analysis-1.png”
</button>
</p>
<div id="fig-no-spaces-raw-analysis-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Corr heatmap
cors &lt;- cor(merged_raw_data, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/raw%20analysis-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-raw-analysis-2">
Past versions of “raw analysis-2.png”
</button>
</p>
<div id="fig-no-spaces-raw-analysis-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="normalize-raw-merged-counts-upperquartile-normalization" class="section level2">
<h2>Normalize raw merged counts: upperquartile Normalization</h2>
<pre class="r"><code>library(edgeR)</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre><code>
Attaching package: &#39;limma&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:BiocGenerics&#39;:

    plotMA</code></pre>
<pre class="r"><code>merged_raw_data &lt;- DGEList(merged_raw_data, group = colnames(merged_raw_data))
upperquartile &lt;- calcNormFactors(merged_raw_data, method = &quot;upperquartile&quot;)
upperquartile &lt;- cpm(upperquartile, log=TRUE, normalized.lib.sizes = T)
head(upperquartile)</code></pre>
<pre><code>                18855_3_S 19160_3_S 18856_3_U 18856_1_U 18855_2_S 18856_2_S
ENSG00000000003  7.758588  7.712307  8.240573  8.079476  7.976949  7.028176
ENSG00000000005 -7.165676 -7.165676 -3.250188 -7.165676 -7.165676 -7.165676
ENSG00000000419  5.967959  6.118638  6.479137  6.182408  6.029238  5.966816
ENSG00000000457  3.636590  3.851257  3.927673  3.723599  3.817878  3.468489
ENSG00000000460  2.891709  3.310113  2.674831  2.215989  1.873681  2.007739
ENSG00000000938 -7.165676 -7.165676 -3.250188 -7.165676 -7.165676 -7.165676
                19160_3_U 18855_2_U 19160_2_S 18855_1_S 18856_1_S 19160_1_S
ENSG00000000003  7.616724  8.039482  7.774070  7.619857  7.824242  7.590621
ENSG00000000005 -2.608422 -7.165676 -7.165676 -7.165676 -7.165676 -3.743846
ENSG00000000419  5.943833  6.267568  6.200855  5.804785  6.449618  6.009809
ENSG00000000457  3.929533  4.086108  4.239243  3.789831  3.828575  4.069630
ENSG00000000460  3.030630  2.485574  3.210941  2.309264  2.172986  3.104689
ENSG00000000938 -3.548410 -7.165676 -7.165676 -7.165676 -7.165676 -7.165676
                19160_2_U 18855_1_U 18856_3_S 18856_2_U 18855_3_U iPSC-MSC5_2
ENSG00000000003  7.203241  7.657124  7.948115  7.535562  7.668964    7.866950
ENSG00000000005 -3.548410 -7.165676 -7.165676 -7.165676 -7.165676   -2.608422
ENSG00000000419  5.830974  6.043358  6.395248  6.205162  6.017366    5.875061
ENSG00000000457  4.064271  3.747561  4.085618  3.863324  3.754921    4.689099
ENSG00000000460  3.236917  2.520537  2.441793  2.279742  2.296791    4.940312
ENSG00000000938 -7.165676 -7.165676 -7.165676 -7.165676 -7.165676   -7.165676
                    adMSC    Heart_4 Chond_enc_2     Lung_1 Chond_SCM_P4_1
ENSG00000000003  5.111410  2.7694710    4.651789  3.3684210       4.647447
ENSG00000000005 -4.188762 -4.1470370   -5.356333 -3.5709184      -7.165676
ENSG00000000419  5.824895  4.4973503    4.499330  3.6552277       5.432481
ENSG00000000457  3.960206  1.6661308    3.623375  2.6433504       3.553510
ENSG00000000460  3.108283 -0.2376793    2.350831  0.4929255       3.077630
ENSG00000000938 -7.165676  3.3006059   -3.703714  5.9153656      -3.466214
                bmMSC_2.2 Chond_TCP_P4_2 Chond_SCM_P1_1 bmMSC_1.2
ENSG00000000003  5.151436       4.596668       6.239300  5.073434
ENSG00000000005 -7.165676      -7.165676      -4.248564 -7.165676
ENSG00000000419  5.532461       5.331569       5.002192  6.042798
ENSG00000000457  3.369717       3.334215       3.666284  3.483576
ENSG00000000460  1.682220       4.021370       1.699396  5.347347
ENSG00000000938 -7.165676      -2.190040      -2.796909 -3.437552
                Chond_TCP_P1_2 iPSC-MSC9_1 Chond_TCP_P1_3    Heart_2
ENSG00000000003       5.548612    6.889267       5.542888  2.2858076
ENSG00000000005      -7.165676   -7.165676      -7.165676 -4.9744289
ENSG00000000419       4.523060    5.736565       4.749294  4.0272041
ENSG00000000457       3.013590    3.094623       3.600084  1.6609015
ENSG00000000460       1.787657    4.360822       2.606735  0.2013634
ENSG00000000938      -2.151752   -7.165676      -2.298802  4.0979827
                Chond_SCM_P4_3   Kidney_2 Chond_SCM_P1_2 Chond_TCP_P1_1
ENSG00000000003       5.040866  5.3883170       5.765196       6.028824
ENSG00000000005      -7.165676 -1.4105059      -7.165676      -7.165676
ENSG00000000419       5.352996  3.4109469       4.767918       4.682382
ENSG00000000457       3.564781  2.2055283       3.480351       3.257322
ENSG00000000460       3.436535 -0.1506356       1.640858       1.401124
ENSG00000000938      -7.165676  1.8331276      -3.283424      -2.626714
                   Liver_3 Chond_SCM_P4_2 bmMSC_2.1   Kidney_4 bmMSC_1.1
ENSG00000000003  6.1071182       5.141389  5.413625  4.6782652  4.893251
ENSG00000000005 -7.1656761      -7.165676 -7.165676 -2.2732293 -7.165676
ENSG00000000419  4.2819116       5.519458  5.580709  3.6217574  6.100824
ENSG00000000457  3.1269190       3.523887  3.322970  2.1180908  3.490769
ENSG00000000460  0.7532304       3.024678  1.905112 -0.0988446  5.218393
ENSG00000000938  3.9615482      -7.165676 -7.165676  1.8859860 -3.743846
                    Lung_2    Liver_2 Chond_TCP_P4_1    Heart_3     Lung_4
ENSG00000000003  2.7193014  5.1913969       4.393626  2.3187862  2.8101304
ENSG00000000005 -7.1656761 -7.1656761      -7.165676 -7.1656761 -5.5711169
ENSG00000000419  3.6067712  4.1225368       5.209806  3.8199828  3.4213859
ENSG00000000457  2.5234583  2.6515953       3.448517  2.2517679  2.3604281
ENSG00000000460  0.4087398  0.5880913       4.212325 -0.1925326  0.4559521
ENSG00000000938  5.3091769  4.3360824      -3.030136  2.3828290  6.4755437
                Chond_SCM_P1_3   Liver_4   Liver_1 Chond_enc_1    iPSC_1
ENSG00000000003       5.832394  4.835584  5.363136    4.803048  7.137906
ENSG00000000005      -3.987562 -7.165676 -7.165676   -7.165676  1.909904
ENSG00000000419       4.967976  3.895757  4.568619    4.440738  6.503442
ENSG00000000457       3.444370  3.120813  3.705054    3.788320  3.036777
ENSG00000000460       1.912295  1.203255  3.254439    2.690355  5.179945
ENSG00000000938      -1.541944  5.006552  3.174881   -5.564166 -2.462560
                   iPSC_2 iPSC-MSC5_1     Lung_3 iPSC-MSC9_2 Kidney_1
ENSG00000000003  7.107160    7.269601  3.6095535    7.225360 5.067148
ENSG00000000005  2.422151   -1.225880 -7.1656761   -3.003177 2.894265
ENSG00000000419  6.530302    5.209034  3.1563869    6.444770 3.446674
ENSG00000000457  2.935361    4.519928  2.1943718    3.591889 2.316526
ENSG00000000460  5.189617    4.297315  0.2278601    4.644512 0.282989
ENSG00000000938 -2.683605   -4.836376  5.1256376   -7.165676 2.100174
                Chond_TCP_P4_3   Kidney_3 bmMSC_1.3
ENSG00000000003       4.050239  5.3112694  5.025102
ENSG00000000005      -7.165676 -1.8387818 -7.165676
ENSG00000000419       5.225750  3.7412620  6.031550
ENSG00000000457       3.331367  2.8080790  3.483576
ENSG00000000460       4.148309  0.5011702  5.313693
ENSG00000000938      -2.947510  0.8673038 -4.332596</code></pre>
<pre class="r"><code># Look at density plots
plotDensities(upperquartile, legend=&quot;topright&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/upperquartile%20normalization-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-upperquartile-normalization-1">
Past versions of “upperquartile normalization-1.png”
</button>
</p>
<div id="fig-no-spaces-upperquartile-normalization-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2upperquartile-normalized-and-unfiltered-samples" class="section level2">
<h2>Correlation heatmap of log2upperquartile normalized and unfiltered samples</h2>
<pre class="r"><code>cors &lt;- cor(upperquartile, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20unfiltered%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-unfiltered-upperquartile-1">
Past versions of “corr heatmap unfiltered upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-unfiltered-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="boxplots-of-upperquartile-across-samples" class="section level2">
<h2>Boxplots of upperquartile across samples</h2>
<pre class="r"><code>library(reshape2)
meltupperquartile &lt;- melt(upperquartile)
names(meltupperquartile) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;upperquartile&quot;)
p &lt;- ggplot(meltupperquartile, aes(factor(sampleID), upperquartile)) 
p + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-upperquartile-1">
Past versions of “boxplot upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="filtering-for-lowly-expressed-genes-avg-log2upperquartile-2-in-at-least-10-samples" class="section level2">
<h2>Filtering for lowly expressed genes (avg log2upperquartile &gt; 2 in at least 10 samples)</h2>
<pre class="r"><code>cutoff &lt;- 2

keep &lt;- rowSums( upperquartile &gt; cutoff ) &gt;= 10

counts_upperquartile &lt;- merged_raw_data[keep,]
filtered_upperquartile &lt;- upperquartile[keep,]
dim(filtered_upperquartile)</code></pre>
<pre><code>[1] 13231    58</code></pre>
</div>
<div id="boxplots-of-normalizedfiltered-counts-across-samples" class="section level2">
<h2>Boxplots of normalized+filtered counts across samples</h2>
<pre class="r"><code>melt_filt_upperquartile &lt;- melt(filtered_upperquartile)
names(melt_filt_upperquartile) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;log2upperquartile&quot;)
p1 &lt;- ggplot(melt_filt_upperquartile, aes(factor(sampleID), log2upperquartile)) 
p1 + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-upperquartile-1">
Past versions of “boxplot filt upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotDensities(filtered_upperquartile, legend = F)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20upperquartile-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-upperquartile-2">
Past versions of “boxplot filt upperquartile-2.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-upperquartile-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2upperquartile-normalized-and-filtered-samples" class="section level2">
<h2>Correlation heatmap of log2upperquartile normalized and filtered samples</h2>
<pre class="r"><code>cors &lt;- cor(filtered_upperquartile, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20filtered%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-filtered-upperquartile-1">
Past versions of “corr heatmap filtered upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-filtered-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-log2upperquartile-normalized-and-filtered-samples" class="section level2">
<h2>PCA of log2upperquartile normalized and filtered samples</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(filtered_upperquartile)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA%20quartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-quartile-1">
Past versions of “PCA quartile-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-quartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="normalize-raw-merged-counts-rle-normalization" class="section level2">
<h2>Normalize raw merged counts: RLE Normalization</h2>
<pre class="r"><code>library(edgeR)
merged_raw_data &lt;- DGEList(merged_raw_data, group = colnames(merged_raw_data))
RLE &lt;- calcNormFactors(merged_raw_data, method = &quot;RLE&quot;)
RLE &lt;- cpm(RLE, log=TRUE, normalized.lib.sizes = T)
head(RLE)</code></pre>
<pre><code>                18855_3_S 19160_3_S 18856_3_U 18856_1_U 18855_2_S 18856_2_S
ENSG00000000003  7.173398  7.215897  7.462430  7.363886  7.411258  6.412557
ENSG00000000005 -6.781401 -6.781401 -3.914580 -6.781401 -6.781401 -6.781401
ENSG00000000419  5.382879  5.622310  5.701093  5.466934  5.463654  5.351281
ENSG00000000457  3.052129  3.355392  3.150307  3.008836  3.252814  2.853701
ENSG00000000460  2.307772  2.814515  1.898594  1.502827  1.310506  1.394539
ENSG00000000938 -6.781401 -6.781401 -3.914580 -6.781401 -6.781401 -6.781401
                19160_3_U 18855_2_U 19160_2_S 18855_1_S 18856_1_S 19160_1_S
ENSG00000000003  7.154175  7.387982  7.212963  7.001472  7.014104  7.111655
ENSG00000000005 -3.022888 -6.781401 -6.781401 -6.781401 -6.781401 -4.116591
ENSG00000000419  5.481374  5.616164  5.639833  5.186529  5.639571  5.530929
ENSG00000000457  3.467469  3.435189  3.678586  3.172124  3.019291  3.591112
ENSG00000000460  2.569022  1.835917  2.650796  1.692862  1.365659  2.626637
ENSG00000000938 -3.920044 -6.781401 -6.781401 -6.781401 -6.781401 -6.781401
                19160_2_U 18855_1_U 18856_3_S 18856_2_U 18855_3_U iPSC-MSC5_2
ENSG00000000003  6.724120  7.150652  7.040733  6.884671  7.103599    7.688648
ENSG00000000005 -3.934338 -6.781401 -6.781401 -6.781401 -6.781401   -2.757812
ENSG00000000419  5.351941  5.536974  5.487980  5.554357  5.452099    5.696820
ENSG00000000457  3.585585  3.241685  3.179034  3.213101  3.190203    4.510963
ENSG00000000460  2.758612  2.015518  1.537028  1.630964  1.733284    4.762146
ENSG00000000938 -6.781401 -6.781401 -6.781401 -6.781401 -6.781401   -6.781401
                    adMSC    Heart_4 Chond_enc_2    Lung_1 Chond_SCM_P4_1
ENSG00000000003  4.525265  3.7278017    5.135950  4.743321       4.408040
ENSG00000000005 -4.609230 -3.2479205   -4.899954 -2.256126      -6.781401
ENSG00000000419  5.238641  5.4560190    4.983488  5.030214       5.192982
ENSG00000000457  3.374402  2.6239052    4.107508  4.017934       3.314349
ENSG00000000460  2.522977  0.7172383    2.834887  1.864756       2.838649
ENSG00000000938 -6.781401  4.2590857   -3.228313  7.290666      -3.646972
                bmMSC_2.2 Chond_TCP_P4_2 Chond_SCM_P1_1 bmMSC_1.2
ENSG00000000003  4.694543       4.245696       6.021634  4.452596
ENSG00000000005 -6.781401      -6.781401      -4.370636 -6.781401
ENSG00000000419  5.075515       4.980486       4.784619  5.421812
ENSG00000000457  2.913370       2.983629       3.448959  2.863342
ENSG00000000460  1.227579       3.670533       1.483262  4.726456
ENSG00000000938 -6.781401      -2.511115      -2.978930 -3.952975
                Chond_TCP_P1_2 iPSC-MSC9_1 Chond_TCP_P1_3    Heart_2
ENSG00000000003       5.544340    6.669838       5.360722  3.0213557
ENSG00000000005      -6.781401   -6.781401      -6.781401 -4.3084507
ENSG00000000419       4.518856    5.517189       4.567205  4.7630648
ENSG00000000457       3.009635    2.875761       3.418214  2.3962079
ENSG00000000460       1.784218    4.141603       2.425260  0.9354655
ENSG00000000938      -2.142353   -6.781401      -2.457484  4.8338498
                Chond_SCM_P4_3  Kidney_2 Chond_SCM_P1_2 Chond_TCP_P1_1
ENSG00000000003       4.645285  6.513148       5.666832       6.034677
ENSG00000000005      -6.781401 -0.296344      -6.781401      -6.781401
ENSG00000000419       4.957372  4.535495       4.669627       4.688307
ENSG00000000457       3.169589  3.329581       3.382271       3.263445
ENSG00000000460       3.041400  0.969806       1.543700       1.408072
ENSG00000000938      -6.781401  2.956922      -3.343499      -2.602410
                  Liver_3 Chond_SCM_P4_2 bmMSC_2.1  Kidney_4 bmMSC_1.1
ENSG00000000003  7.072768       4.762230  4.909603  5.860739  4.238144
ENSG00000000005 -6.781401      -6.781401 -6.781401 -1.111382 -6.781401
ENSG00000000419  5.247438       5.140253  5.076665  4.804051  5.445515
ENSG00000000457  4.092235       3.145139  2.819603  3.299747  2.836249
ENSG00000000460  1.716949       2.646181  1.403172  1.079215  4.563215
ENSG00000000938  4.927032      -6.781401 -6.781401  3.067471 -4.263747
                   Lung_2   Liver_2 Chond_TCP_P4_1    Heart_3    Lung_4
ENSG00000000003  3.828484  6.102465       3.945398  3.2326954  4.061140
ENSG00000000005 -6.781401 -6.781401      -6.781401 -6.7814007 -4.553282
ENSG00000000419  4.716230  5.033513       4.761416  4.7342927  4.672619
ENSG00000000457  3.632553  3.562259       3.000634  3.1656477  3.611201
ENSG00000000460  1.515531  1.497197       3.764147  0.7184601  1.704295
ENSG00000000938  6.418862  5.247083      -3.415996  3.2967651  7.727150
                Chond_SCM_P1_3   Liver_4   Liver_1 Chond_enc_1    iPSC_1
ENSG00000000003       5.624894  6.019869  6.139960    5.252806  7.210588
ENSG00000000005      -4.116493 -6.781401 -6.781401   -6.781401  1.983213
ENSG00000000419       4.760548  5.079905  5.345400    4.890491  6.576133
ENSG00000000457       3.237247  4.304756  4.481752    4.238062  3.109736
ENSG00000000460       1.706058  2.385831  4.031070    3.140060  5.252676
ENSG00000000938      -1.734770  6.190854  3.951498   -5.135649 -2.376604
                   iPSC_2 iPSC-MSC5_1    Lung_3 iPSC-MSC9_2 Kidney_1
ENSG00000000003  7.151043   7.5131081  4.874715    7.026631 6.266156
ENSG00000000005  2.466514  -0.9799725 -6.781401   -3.162366 4.092818
ENSG00000000419  6.574194   5.4525621  4.421409    6.246065 4.645413
ENSG00000000457  2.979574   4.7634737  3.458905    3.393542 3.514792
ENSG00000000460  5.233554   4.5408679  1.489470    4.445950 1.478557
ENSG00000000938 -2.622663  -4.5637477  6.391044   -6.781401 3.298299
                Chond_TCP_P4_3   Kidney_3 bmMSC_1.3
ENSG00000000003       3.651668  6.4955100  4.378976
ENSG00000000005      -6.781401 -0.6698155 -6.781401
ENSG00000000419       4.826936  4.9252905  5.385262
ENSG00000000457       2.933079  3.9918165  2.838064
ENSG00000000460       3.749710  1.6824934  4.667508
ENSG00000000938      -3.291682  2.0493055 -4.781968</code></pre>
<pre class="r"><code># Look at density plots
plotDensities(RLE, legend=&quot;topright&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/RLE%20normalization-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-RLE-normalization-1">
Past versions of “RLE normalization-1.png”
</button>
</p>
<div id="fig-no-spaces-RLE-normalization-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2rle-normalized-and-unfiltered-samples" class="section level2">
<h2>Correlation heatmap of log2RLE normalized and unfiltered samples</h2>
<pre class="r"><code>cors &lt;- cor(RLE, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20unfiltered%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-unfiltered-RLE-1">
Past versions of “corr heatmap unfiltered RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-unfiltered-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="boxplots-of-rle-across-samples" class="section level2">
<h2>Boxplots of RLE across samples</h2>
<pre class="r"><code>library(reshape2)
meltRLE &lt;- melt(RLE)
names(meltRLE) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;RLE&quot;)
p &lt;- ggplot(meltRLE, aes(factor(sampleID), RLE)) 
p + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-RLE-1">
Past versions of “boxplot RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="filtering-for-lowly-expressed-genes-avg-log2rle-2-in-at-least-10-samples" class="section level2">
<h2>Filtering for lowly expressed genes (avg log2RLE &gt; 2 in at least 10 samples)</h2>
<pre class="r"><code>cutoff &lt;- 2

keep &lt;- rowSums( RLE &gt; cutoff ) &gt;= 10

counts_RLE &lt;- merged_raw_data[keep,]
filtered_RLE &lt;- RLE[keep,]
dim(filtered_RLE)</code></pre>
<pre><code>[1] 13465    58</code></pre>
</div>
<div id="boxplots-of-normalizedfiltered-counts-across-samples-1" class="section level2">
<h2>Boxplots of normalized+filtered counts across samples</h2>
<pre class="r"><code>melt_filt_RLE &lt;- melt(filtered_RLE)
names(melt_filt_RLE) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;log2RLE&quot;)
p1 &lt;- ggplot(melt_filt_RLE, aes(factor(sampleID), log2RLE)) 
p1 + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-RLE-1">
Past versions of “boxplot filt RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotDensities(filtered_RLE, legend = F)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20RLE-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-RLE-2">
Past versions of “boxplot filt RLE-2.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-RLE-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2rle-normalized-and-filtered-samples" class="section level2">
<h2>Correlation heatmap of log2RLE normalized and filtered samples</h2>
<pre class="r"><code>cors &lt;- cor(filtered_RLE, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20filtered%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-filtered-RLE-1">
Past versions of “corr heatmap filtered RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-filtered-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-log2rle-normalized-and-filtered-samples" class="section level2">
<h2>PCA of log2RLE normalized and filtered samples</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(filtered_RLE)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-1">
Past versions of PCA-1.png
</button>
</p>
<div id="fig-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="grade-of-membership-analysis" class="section level1">
<h1>Grade of Membership analysis</h1>
<pre class="r"><code>merged_raw_data_counts &lt;- merged_raw_data$counts
Combined_counts_celltypes &lt;- sapply(strsplit(as.character(counts_upperquartile$samples$group),&quot;_&quot;), `[`, 1)

if (file.exists(&quot;data/combined_gom_bulk_4to8.rda&quot;)){
     Combined_GoM &lt;- get(load(file=&quot;data/combined_gom_bulk_4to8.rda&quot;))
} else {
     Combined_GoM &lt;- CountClust::FitGoM(t(merged_raw_data_counts), K=c(4:8), tol = 0.1, control=list(tmax=100))
     save(Combined_GoM, file=&quot;data/combined_gom_bulk_4to8.rda&quot;)
}</code></pre>
<pre class="r"><code>plot_structure_plot &lt;- function(omega, labels){
     annotation &lt;- data.frame(
          sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
          tissue_label = labels
          )
     rownames(omega) &lt;- annotation$sample_id
     StructureGGplot(omega = omega,
                     annotation = annotation,
                     order_sample = TRUE,
                     axis_tick = list(axis_ticks_length = .1,
                                      axis_ticks_lwd_y = .1,
                                      axis_ticks_lwd_x = .1,
                                      axis_label_size = 7,
                                      axis_label_face = &quot;bold&quot;)
                     )
     }


make_topic_plots &lt;- function(omega, labels){
  omega &lt;- as.data.frame(omega)
  omega &lt;- cbind(omega,labels)
  for (k in 1:(ncol(omega)-1)) {
    base_plot &lt;- ggplot(data = omega, mapping = aes(x = labels, y = omega[,k])) + labs(y = paste0(&quot;Topic &quot;, k, &quot; value&quot;), x = paste0(&quot;Cell Type&quot;)) + theme(axis.text.x = element_text(angle = -20, hjust = 0, vjust = 0))
    plot(base_plot + geom_violin(width = 1.5, fill = &quot;skyblue&quot;,color = &quot;skyblue&quot;) +
      geom_boxplot(width = 0.1,outlier.shape = NA, alpha = 0.2))
   }
}

annotate_clusters &lt;- function(theta_mat, countmatrix){
  top_features &lt;- ExtractTopFeatures(theta_mat,
                                     top_features=100,
                                     method=&quot;poisson&quot;,
                                     options=&quot;min&quot;)
  gene_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) rownames(countmatrix)[top_features[[1]][x,]]))
  scores_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) top_features[[2]][x,]))
  return(list(gene_vector, scores_vector))
}


labels &lt;- sapply(strsplit(colnames(merged_raw_data_counts),&quot;_&quot;), `[`, 1)

# k = 4
plot_structure_plot(omega = Combined_GoM$clust_4$omega, labels = labels)</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-1">
Past versions of “plotting GoM-1.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_4$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-2">
Past versions of “plotting GoM-2.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-3">
Past versions of “plotting GoM-3.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-4">
Past versions of “plotting GoM-4.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-5">
Past versions of “plotting GoM-5.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_4$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 5
plot_structure_plot(omega = Combined_GoM$clust_5$omega, labels = labels)</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-6">
Past versions of “plotting GoM-6.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_5$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-7">
Past versions of “plotting GoM-7.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-8">
Past versions of “plotting GoM-8.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-9">
Past versions of “plotting GoM-9.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-10">
Past versions of “plotting GoM-10.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-11">
Past versions of “plotting GoM-11.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_5$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 6
plot_structure_plot(omega = Combined_GoM$clust_6$omega, labels = labels)</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-12">
Past versions of “plotting GoM-12.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_6$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-13.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-13">
Past versions of “plotting GoM-13.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-14.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-14">
Past versions of “plotting GoM-14.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-15.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-15">
Past versions of “plotting GoM-15.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-16.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-16">
Past versions of “plotting GoM-16.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-16" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-17.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-17">
Past versions of “plotting GoM-17.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-17" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-18.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-18">
Past versions of “plotting GoM-18.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-18" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_6$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 7
plot_structure_plot(omega = Combined_GoM$clust_7$omega, labels = labels)</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-19.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-19">
Past versions of “plotting GoM-19.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-19" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_7$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-20.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-20">
Past versions of “plotting GoM-20.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-21.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-21">
Past versions of “plotting GoM-21.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-22.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-22">
Past versions of “plotting GoM-22.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-23.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-23">
Past versions of “plotting GoM-23.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-23" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-24.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-24">
Past versions of “plotting GoM-24.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-24" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-25.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-25">
Past versions of “plotting GoM-25.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-25" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-26.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-26">
Past versions of “plotting GoM-26.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-26" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_7$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 8
plot_structure_plot(omega = Combined_GoM$clust_8$omega, labels = labels)</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-27.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-27">
Past versions of “plotting GoM-27.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-27" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_8$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-28.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-28">
Past versions of “plotting GoM-28.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-28" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-29.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-29">
Past versions of “plotting GoM-29.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-29" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-30.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-30">
Past versions of “plotting GoM-30.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-30" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-31.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-31">
Past versions of “plotting GoM-31.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-31" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-32.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-32">
Past versions of “plotting GoM-32.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-32" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-33.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-33">
Past versions of “plotting GoM-33.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-33" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-34.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-34">
Past versions of “plotting GoM-34.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-34" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-35.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-35">
Past versions of “plotting GoM-35.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-35" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_8$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]</code></pre>
</div>
<div id="check-expression-of-chondrocyte-markers-across-data" class="section level1">
<h1>Check expression of chondrocyte markers across data</h1>
<pre class="r"><code>#Inputs: count matrix, sample information data frame, and gene of interest in the ENSG format
get_gene_long &lt;- function(counts, info, gene){
     gene_counts &lt;- as.data.frame(counts[gene,])
     names(gene_counts) &lt;- &quot;norm_expr&quot;
     gene_counts_info &lt;- cbind(gene_counts, info)
     return(gene_counts_info)
}

boxplot_gene_sample_type &lt;- function(counts_, info_, gene_, name, y_axis = &quot;log2 Normalized Expression&quot;){
     df &lt;- get_gene_long(counts = counts_, info = info_, gene = gene_)
     
     ggplot(df, aes(x = labels, y = norm_expr, color = labels)) +
          geom_boxplot() +
          labs(title=paste0(&quot;Expression of &quot;, as.character(name), &quot; by sample type&quot;), x= &quot;Sample Type&quot;, y = y_axis) +
          theme(legend.position = &quot;none&quot;)
}



labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

#SOX9
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000125398&quot;, &quot;SOX9&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-1">
Past versions of “chond markers-1.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#COL11A1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000060718&quot;, &quot;COL11A1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-2">
Past versions of “chond markers-2.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#SOX5
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000134532&quot;, &quot;SOX5&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-3">
Past versions of “chond markers-3.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#SOX6
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000110693&quot;, &quot;SOX6&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-4">
Past versions of “chond markers-4.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#TGFB3
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000119699&quot;, &quot;TGFB3&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-5">
Past versions of “chond markers-5.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#COL2A1
boxplot_gene_sample_type(merged_raw_data$counts, labels, &quot;ENSG00000139219&quot;, &quot;COL2A1&quot;, y_axis = &quot;Raw counts&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-6">
Past versions of “chond markers-6.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#ACAN
boxplot_gene_sample_type(merged_raw_data$counts, labels, &quot;ENSG00000157766&quot;, &quot;ACAN&quot;, y_axis = &quot;Raw counts&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-7">
Past versions of “chond markers-7.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#other chond markers
#NKX3-2
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000109705&quot;, &quot;NKX3-2&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-8">
Past versions of “chond markers-8.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-8.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#IGFBP5
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000115461&quot;, &quot;IGFBP5&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-9">
Past versions of “chond markers-9.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-9.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-define-some-known-clusters-prior-to-fitting-topic-model" class="section level1">
<h1>Semi-supervised topic modeling (define some known clusters prior to fitting topic model)</h1>
<pre class="r"><code>library(&quot;classtpx&quot;)</code></pre>
<pre><code>
Attaching package: &#39;classtpx&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:maptpx&#39;:

    expit, logit, rdir, stm_tfidf</code></pre>
<pre class="r"><code># labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- which(labels==&quot;Chond&quot;)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond))
                )

# Perform topic modeling k=6
if (file.exists(&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;)){
     Topic_clus_6 &lt;- get(load(file=&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_6 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=6,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_6, file=&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_6$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-1">
Past versions of supervised-1.png
</button>
</p>
<div id="fig-supervised-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;)){
     Topic_clus_7 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_7 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_7$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-2">
Past versions of supervised-2.png
</button>
</p>
<div id="fig-supervised-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;)){
     Topic_clus_8 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_8 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_8$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-3">
Past versions of supervised-3.png
</button>
</p>
<div id="fig-supervised-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/68e5e33dc85bf6340efc50200e47b0766711c93c/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-genes-analysis" class="section level1">
<h1>Semi-supervised Topic Modeling Genes Analysis</h1>
<pre class="r"><code>genes &lt;- annotate_clusters(theta_mat = Topic_clus_7$theta, countmatrix = merged_raw_data_counts)
matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)</code></pre>
<pre><code>     [,1]     [,2]     [,3]      [,4]      [,5]      [,6]       [,7]      
[1,] &quot;COL3A1&quot; &quot;SPARC&quot;  &quot;COL1A1&quot;  &quot;FN1&quot;     &quot;SFTPB&quot;   &quot;SFTPC&quot;    &quot;COL4A1&quot;  
[2,] &quot;RN7SL2&quot; &quot;IGFBP3&quot; &quot;CHI3L1&quot;  &quot;ACAN&quot;    &quot;COL12A1&quot; &quot;SULF1&quot;    &quot;POSTN&quot;   
[3,] &quot;MMP1&quot;   &quot;TERT&quot;   &quot;TFPI2&quot;   &quot;DYNC1H1&quot; &quot;DCBLD2&quot;  &quot;POLR3G&quot;   &quot;HSP90AA1&quot;
[4,] &quot;UMOD&quot;   &quot;GPX3&quot;   &quot;SLC12A1&quot; &quot;AQP2&quot;    &quot;CDH16&quot;   &quot;SLC12A3&quot;  &quot;SPP1&quot;    
[5,] &quot;ALB&quot;    &quot;HP&quot;     &quot;FGA&quot;     &quot;FGB&quot;     &quot;FGG&quot;     &quot;SERPINA1&quot; &quot;SAA1&quot;    
[6,] &quot;KRT8&quot;   &quot;CTSV&quot;   &quot;PRTG&quot;    &quot;SLC2A3&quot;  &quot;KRT19&quot;   &quot;GABRP&quot;    &quot;HAND1&quot;   
[7,] &quot;MYH7&quot;   &quot;DES&quot;    &quot;MYL2&quot;    &quot;MB&quot;      &quot;TNNT2&quot;   &quot;MT-CO1&quot;   &quot;CKM&quot;     
     [,8]     [,9]     [,10]     [,11]      [,12]     [,13]    [,14]   
[1,] &quot;COL4A2&quot; &quot;SFTPA1&quot; &quot;TIMP3&quot;   &quot;TIMP1&quot;    &quot;SFTPA2&quot;  &quot;FBLN5&quot;  &quot;SPON2&quot; 
[2,] &quot;EFEMP1&quot; &quot;CEMIP&quot;  &quot;MALAT1&quot;  &quot;BGN&quot;      &quot;TGFBI&quot;   &quot;CCDC80&quot; &quot;PENK&quot;  
[3,] &quot;RGS4&quot;   &quot;MKI67&quot;  &quot;ACTB&quot;    &quot;DNMT3B&quot;   &quot;CLDN11&quot;  &quot;IL1B&quot;   &quot;POU5F1&quot;
[4,] &quot;PAX8&quot;   &quot;ATP1B1&quot; &quot;HSD11B2&quot; &quot;PDZK1IP1&quot; &quot;KCNJ1&quot;   &quot;CLCNKB&quot; &quot;CALB1&quot; 
[5,] &quot;ORM1&quot;   &quot;TF&quot;     &quot;CRP&quot;     &quot;APOC3&quot;    &quot;AMBP&quot;    &quot;CYP2E1&quot; &quot;APOB&quot;  
[6,] &quot;MDK&quot;    &quot;DSP&quot;    &quot;LRRN1&quot;   &quot;FREM2&quot;    &quot;SLC38A1&quot; &quot;ANXA3&quot;  &quot;NPPB&quot;  
[7,] &quot;MT-ND1&quot; &quot;MYL3&quot;   &quot;MYBPC3&quot;  &quot;TNNI3&quot;    &quot;MT-ND4&quot;  &quot;TTN&quot;    &quot;TCAP&quot;  
     [,15]     [,16]    [,17]    [,18]     [,19]    [,20]     [,21]    
[1,] &quot;CILP&quot;    &quot;AGER&quot;   &quot;ACTG2&quot;  &quot;MATN2&quot;   &quot;ACTA2&quot;  &quot;MFAP4&quot;   &quot;COL5A1&quot; 
[2,] &quot;TNC&quot;     &quot;DCN&quot;    &quot;VIM&quot;    &quot;COL6A1&quot;  &quot;LUM&quot;    &quot;ADAMTS5&quot; &quot;COL6A3&quot; 
[3,] &quot;EIF4G1&quot;  &quot;CDCP1&quot;  &quot;SFRP1&quot;  &quot;FGF5&quot;    &quot;HSPA8&quot;  &quot;LDHA&quot;    &quot;ZFHX4&quot;  
[4,] &quot;TMEM213&quot; &quot;MIOX&quot;   &quot;CUBN&quot;   &quot;SLC22A8&quot; &quot;KCNJ16&quot; &quot;SLC5A12&quot; &quot;SLC34A1&quot;
[5,] &quot;APOA2&quot;   &quot;APOH&quot;   &quot;APOA1&quot;  &quot;SAA2&quot;    &quot;CYP3A4&quot; &quot;TTR&quot;     &quot;RBP4&quot;   
[6,] &quot;CDH3&quot;    &quot;AMOT&quot;   &quot;FAM84B&quot; &quot;SLC2A1&quot;  &quot;CTNNB1&quot; &quot;DSC2&quot;    &quot;UBE2J1&quot; 
[7,] &quot;MT-RNR2&quot; &quot;MT-CYB&quot; &quot;HSPB7&quot;  &quot;TNNC1&quot;   &quot;XIRP1&quot;  &quot;MT-ND2&quot;  &quot;LDB3&quot;   
     [,22]     [,23]       [,24]       [,25]        [,26]      [,27]   
[1,] &quot;WNT2&quot;    &quot;COL11A1&quot;   &quot;COL1A2&quot;    &quot;AC089987.1&quot; &quot;CDKN2B&quot;   &quot;FLT1&quot;  
[2,] &quot;CYP1B1&quot;  &quot;PDGFRA&quot;    &quot;NDUFA4L2&quot;  &quot;SEMA3C&quot;     &quot;COL6A2&quot;   &quot;PRUNE2&quot;
[3,] &quot;PLAT&quot;    &quot;LINC00678&quot; &quot;KIAA1549L&quot; &quot;IPO5&quot;       &quot;ESRG&quot;     &quot;SEPHS1&quot;
[4,] &quot;TFCP2L1&quot; &quot;DPEP1&quot;     &quot;TMEM52B&quot;   &quot;SLC13A3&quot;    &quot;RHCG&quot;     &quot;CA12&quot;  
[5,] &quot;HPX&quot;     &quot;C3&quot;        &quot;FGL1&quot;      &quot;AGXT&quot;       &quot;SERPINC1&quot; &quot;APOC1&quot; 
[6,] &quot;KRT18&quot;   &quot;BAMBI&quot;     &quot;CLDN6&quot;     &quot;ARID3A&quot;     &quot;NES&quot;      &quot;PCAT14&quot;
[7,] &quot;ACTN2&quot;   &quot;MT-ND5&quot;    &quot;MT-CO2&quot;    &quot;MT-ATP6&quot;    &quot;FABP3&quot;    &quot;NRAP&quot;  
     [,28]     [,29]      [,30]    [,31]     [,32]     [,33]        [,34]   
[1,] &quot;ITGA11&quot;  &quot;PCDH10&quot;   &quot;CLDN18&quot; &quot;IGFL2&quot;   &quot;CEACAM6&quot; &quot;SERPINE2&quot;   &quot;P4HA2&quot; 
[2,] &quot;SCARNA7&quot; &quot;ISLR&quot;     &quot;LRP1&quot;   &quot;FBN1&quot;    &quot;AEBP1&quot;   &quot;ABI3BP&quot;     &quot;LTBP2&quot; 
[3,] &quot;SLC7A1&quot;  &quot;CAV1&quot;     &quot;HHIP&quot;   &quot;PRKDC&quot;   &quot;OASL&quot;    &quot;AC064802.1&quot; &quot;COTL1&quot; 
[4,] &quot;SLC22A6&quot; &quot;SLC22A12&quot; &quot;DEFB1&quot;  &quot;SULT1C2&quot; &quot;CLCNKA&quot;  &quot;MAL&quot;        &quot;PTGER3&quot;
[5,] &quot;AHSG&quot;    &quot;MAT1A&quot;    &quot;GC&quot;     &quot;ADH4&quot;    &quot;CPS1&quot;    &quot;TAT&quot;        &quot;VTN&quot;   
[6,] &quot;HAPLN1&quot;  &quot;ITM2C&quot;    &quot;PERP&quot;   &quot;IGDCC3&quot;  &quot;DOK4&quot;    &quot;PKDCC&quot;      &quot;DACT1&quot; 
[7,] &quot;PDK4&quot;    &quot;ACTC1&quot;    &quot;EEF1A2&quot; &quot;PLN&quot;     &quot;HRC&quot;     &quot;CRYAB&quot;      &quot;MYOZ2&quot; 
     [,35]    [,36]    [,37]     [,38]    [,39]     [,40]    [,41]   
[1,] &quot;LRRC32&quot; &quot;SPTBN2&quot; &quot;CDKN1A&quot;  &quot;SRPX2&quot;  &quot;PDPN&quot;    &quot;HOPX&quot;   &quot;FIBIN&quot; 
[2,] &quot;FSTL1&quot;  &quot;AHNAK&quot;  &quot;FNDC1&quot;   &quot;DST&quot;    &quot;CD109&quot;   &quot;PCOLCE&quot; &quot;PIEZO2&quot;
[3,] &quot;RAB3B&quot;  &quot;DNER&quot;   &quot;ALDH1A3&quot; &quot;IFIT2&quot;  &quot;USP44&quot;   &quot;DUSP4&quot;  &quot;NCL&quot;   
[4,] &quot;SMIM24&quot; &quot;CYS1&quot;   &quot;TMEM72&quot;  &quot;NPHS1&quot;  &quot;PAX2&quot;    &quot;SLC4A9&quot; &quot;SLC3A1&quot;
[5,] &quot;ITIH1&quot;  &quot;CYP2C8&quot; &quot;ORM2&quot;    &quot;ITIH3&quot;  &quot;C9&quot;      &quot;APCS&quot;   &quot;F2&quot;    
[6,] &quot;DSG2&quot;   &quot;ANXA1&quot;  &quot;VTCN1&quot;   &quot;ATP2B1&quot; &quot;CDH1&quot;    &quot;SESN3&quot;  &quot;CRABP2&quot;
[7,] &quot;CKMT2&quot;  &quot;MT-CO3&quot; &quot;APOD&quot;    &quot;MYOM2&quot;  &quot;SLC25A4&quot; &quot;COX6A2&quot; &quot;CSRP3&quot; 
     [,42]      [,43]    [,44]     [,45]    [,46]     [,47]     [,48]   
[1,] &quot;TPM2&quot;     &quot;OGFRL1&quot; &quot;SCGB1A1&quot; &quot;NEDD9&quot;  &quot;TMEM100&quot; &quot;CD55&quot;    &quot;DKK2&quot;  
[2,] &quot;MXRA8&quot;    &quot;ARID5B&quot; &quot;IGFBP5&quot;  &quot;PRRX1&quot;  &quot;CLMP&quot;    &quot;IGFBP4&quot;  &quot;DDR2&quot;  
[3,] &quot;SOX2&quot;     &quot;DPPA4&quot;  &quot;PTPRZ1&quot;  &quot;DDX21&quot;  &quot;HSPD1&quot;   &quot;KPNB1&quot;   &quot;CORO1C&quot;
[4,] &quot;ITM2B&quot;    &quot;NPHS2&quot;  &quot;ATP1A1&quot;  &quot;TSPAN1&quot; &quot;ADGRG1&quot;  &quot;SLC6A19&quot; &quot;FXYD2&quot; 
[5,] &quot;CYP2C9&quot;   &quot;ITIH2&quot;  &quot;SDS&quot;     &quot;ARG1&quot;   &quot;CES1&quot;    &quot;HAMP&quot;    &quot;C4BPA&quot; 
[6,] &quot;ADAMTS18&quot; &quot;ISL1&quot;   &quot;PSTPIP2&quot; &quot;MSX2&quot;   &quot;SPINT2&quot;  &quot;ISYNA1&quot;  &quot;WLS&quot;   
[7,] &quot;ACTA1&quot;    &quot;MYL7&quot;   &quot;MT-ND4L&quot; &quot;MYH7B&quot;  &quot;MYH6&quot;    &quot;RYR2&quot;    &quot;CASQ2&quot; 
     [,49]     [,50]     [,51]      [,52]     [,53]     [,54]     [,55]   
[1,] &quot;FOSB&quot;    &quot;ADGRA2&quot;  &quot;CRISPLD2&quot; &quot;SPON1&quot;   &quot;AQP1&quot;    &quot;LZTS1&quot;   &quot;LMCD1&quot; 
[2,] &quot;TRPS1&quot;   &quot;HMOX1&quot;   &quot;EDIL3&quot;    &quot;SHISAL1&quot; &quot;SLC38A2&quot; &quot;NT5E&quot;    &quot;FAT1&quot;  
[3,] &quot;ANLN&quot;    &quot;CSE1L&quot;   &quot;VASH2&quot;    &quot;SFRP2&quot;   &quot;DOCK10&quot;  &quot;TPX2&quot;    &quot;PKM&quot;   
[4,] &quot;SLC47A2&quot; &quot;NAT8&quot;    &quot;KIF12&quot;    &quot;KCNJ15&quot;  &quot;MMP7&quot;    &quot;SLC26A7&quot; &quot;SLC4A1&quot;
[5,] &quot;TFR2&quot;    &quot;CYP2A6&quot;  &quot;ADH1B&quot;    &quot;LRG1&quot;    &quot;IGFBP1&quot;  &quot;CPB2&quot;    &quot;HRG&quot;   
[6,] &quot;ANKS1A&quot;  &quot;EZR&quot;     &quot;CDX2&quot;     &quot;RPS19&quot;   &quot;DAG1&quot;    &quot;SLC7A5&quot;  &quot;OSBPL9&quot;
[7,] &quot;SRL&quot;     &quot;MT-ATP8&quot; &quot;LMOD2&quot;    &quot;COX7A1&quot;  &quot;ADPRHL1&quot; &quot;CMYA5&quot;   &quot;TECRL&quot; 
     [,56]       [,57]      [,58]    [,59]     [,60]     [,61]      [,62]   
[1,] &quot;ATF3&quot;      &quot;CHPF&quot;     &quot;DEPP1&quot;  &quot;PGC&quot;     &quot;SLC6A17&quot; &quot;HLA-E&quot;    &quot;EGFL6&quot; 
[2,] &quot;COL14A1&quot;   &quot;TBL1XR1&quot;  &quot;REV3L&quot;  &quot;ATL3&quot;    &quot;NEAT1&quot;   &quot;WNT5A&quot;    &quot;HSPG2&quot; 
[3,] &quot;FOXG1&quot;     &quot;ETV4&quot;     &quot;PSIP1&quot;  &quot;SLC20A1&quot; &quot;ESM1&quot;    &quot;NEK7&quot;     &quot;ZBTB38&quot;
[4,] &quot;LINC01187&quot; &quot;ATP6V0A4&quot; &quot;AOC1&quot;   &quot;PIGR&quot;    &quot;EMX2OS&quot;  &quot;SCIN&quot;     &quot;POU3F3&quot;
[5,] &quot;SULT2A1&quot;   &quot;RDH16&quot;    &quot;BAAT&quot;   &quot;SLC22A1&quot; &quot;LBP&quot;     &quot;SERPINF2&quot; &quot;ASGR1&quot; 
[6,] &quot;TCEAL9&quot;    &quot;RPS11&quot;    &quot;TRIM55&quot; &quot;APLNR&quot;   &quot;ID1&quot;     &quot;ARSI&quot;     &quot;SDK1&quot;  
[7,] &quot;MT-ND3&quot;    &quot;NDRG4&quot;    &quot;NEBL&quot;   &quot;MT-TL1&quot;  &quot;HHATL&quot;   &quot;POPDC2&quot;   &quot;LPL&quot;   
     [,63]    [,64]      [,65]    [,66]     [,67]     [,68]     [,69]      
[1,] &quot;RFLNA&quot;  &quot;SLC34A2&quot;  &quot;FSTL3&quot;  &quot;JUN&quot;     &quot;FKBP10&quot;  &quot;PTGER2&quot;  &quot;DUOX1&quot;    
[2,] &quot;LMNA&quot;   &quot;ADAMTS1&quot;  &quot;AHNAK2&quot; &quot;GPC6&quot;    &quot;MEG8&quot;    &quot;TNFAIP6&quot; &quot;ELL2&quot;     
[3,] &quot;PMAIP1&quot; &quot;PAICS&quot;    &quot;SH2B3&quot;  &quot;NAA15&quot;   &quot;HMGA1&quot;   &quot;RRM2&quot;    &quot;LNCPRESS2&quot;
[4,] &quot;PTH1R&quot;  &quot;SLC22A11&quot; &quot;PROM2&quot;  &quot;SCNN1A&quot;  &quot;SLC36A2&quot; &quot;SLC22A2&quot; &quot;EMX1&quot;     
[5,] &quot;AOX1&quot;   &quot;UGT2B4&quot;   &quot;AGT&quot;    &quot;C1S&quot;     &quot;ASGR2&quot;   &quot;C3P1&quot;    &quot;CYP1A2&quot;   
[6,] &quot;LYPD6&quot;  &quot;GREB1L&quot;   &quot;IGSF3&quot;  &quot;COL18A1&quot; &quot;RPL3&quot;    &quot;FLRT3&quot;   &quot;STMN2&quot;    
[7,] &quot;RPL3L&quot;  &quot;TRIM54&quot;   &quot;TRDN&quot;   &quot;UNC45B&quot;  &quot;NMRK2&quot;   &quot;KCNIP2&quot;  &quot;KLHL31&quot;   
     [,70]      [,71]      [,72]       [,73]     [,74]     [,75]      [,76]    
[1,] &quot;SMYD3&quot;    &quot;KCTD16&quot;   &quot;FENDRR&quot;    &quot;SCEL&quot;    &quot;PAG1&quot;    &quot;PPP1R15A&quot; &quot;NALCN&quot;  
[2,] &quot;NFIX&quot;     &quot;COL8A1&quot;   &quot;NEGR1&quot;     &quot;ELN&quot;     &quot;CSPG4&quot;   &quot;PSD3&quot;     &quot;PCSK1&quot;  
[3,] &quot;NOP56&quot;    &quot;PDZD4&quot;    &quot;HRAS&quot;      &quot;LAMC1&quot;   &quot;NANOG&quot;   &quot;CEP170&quot;   &quot;ASAP1&quot;  
[4,] &quot;XPNPEP2&quot;  &quot;CLDN2&quot;    &quot;LINC00982&quot; &quot;WFDC2&quot;   &quot;EGF&quot;     &quot;DDX17&quot;    &quot;RAP1GAP&quot;
[5,] &quot;ATF5&quot;     &quot;SERPING1&quot; &quot;F9&quot;        &quot;SLC13A5&quot; &quot;CLU&quot;     &quot;AZGP1&quot;    &quot;C8A&quot;    
[6,] &quot;PLEKHG4B&quot; &quot;ADGRL2&quot;   &quot;FBN3&quot;      &quot;CPE&quot;     &quot;CAPN6&quot;   &quot;SLITRK6&quot;  &quot;RHOU&quot;   
[7,] &quot;SMYD1&quot;    &quot;ATP2A2&quot;   &quot;SLC4A3&quot;    &quot;CD36&quot;    &quot;APOBEC2&quot; &quot;MYZAP&quot;    &quot;XIRP2&quot;  
     [,77]       [,78]       [,79]    [,80]     [,81]   [,82]       [,83]     
[1,] &quot;SERPINH1&quot;  &quot;NKX2-1&quot;    &quot;STXBP6&quot; &quot;STARD4&quot;  &quot;RGCC&quot;  &quot;BMP6&quot;      &quot;SCUBE2&quot;  
[2,] &quot;LOXL3&quot;     &quot;RN7SL660P&quot; &quot;CHI3L2&quot; &quot;TRAM1&quot;   &quot;MEGF6&quot; &quot;MMP14&quot;     &quot;EMILIN1&quot; 
[3,] &quot;FOXD3-AS1&quot; &quot;GLB1L3&quot;    &quot;SAMD9&quot;  &quot;ADAR&quot;    &quot;FOSL1&quot; &quot;MDN1&quot;      &quot;FGF2&quot;    
[4,] &quot;VAV3&quot;      &quot;AQP6&quot;      &quot;CLTRN&quot;  &quot;CYFIP2&quot;  &quot;PTPRO&quot; &quot;BICDL1&quot;    &quot;SLC13A1&quot; 
[5,] &quot;PLA2G2A&quot;   &quot;MT2A&quot;      &quot;ACSL1&quot;  &quot;HAL&quot;     &quot;C8B&quot;   &quot;SERPINA10&quot; &quot;CREB3L3&quot; 
[6,] &quot;CDH10&quot;     &quot;MYH10&quot;     &quot;PLS3&quot;   &quot;LRRN4&quot;   &quot;BMP4&quot;  &quot;RPS9&quot;      &quot;PTPRF&quot;   
[7,] &quot;SMPX&quot;      &quot;SH3RF2&quot;    &quot;PTP4A3&quot; &quot;MT-RNR1&quot; &quot;MYOM1&quot; &quot;ATP1A2&quot;    &quot;MTATP6P1&quot;
     [,84]        [,85]      [,86]         [,87]    [,88]      [,89]    
[1,] &quot;IGFL3&quot;      &quot;CLEC3A&quot;   &quot;LAMP3&quot;       &quot;ZFP36&quot;  &quot;MEST&quot;     &quot;TSC22D3&quot;
[2,] &quot;HIST1H1B&quot;   &quot;ZBTB20&quot;   &quot;SPOCK1&quot;      &quot;CYBRD1&quot; &quot;GPNMB&quot;    &quot;STXBP5&quot; 
[3,] &quot;AL117378.1&quot; &quot;UTP20&quot;    &quot;HSPA9&quot;       &quot;PLAU&quot;   &quot;SHCBP1&quot;   &quot;AMD1&quot;   
[4,] &quot;AC015910.1&quot; &quot;ATP6V0D2&quot; &quot;PKHD1&quot;       &quot;FMO1&quot;   &quot;CTSH&quot;     &quot;DUSP9&quot;  
[5,] &quot;CFHR1&quot;      &quot;SLC27A5&quot;  &quot;C4BPB&quot;       &quot;EPHX1&quot;  &quot;CYP8B1&quot;   &quot;PON1&quot;   
[6,] &quot;EFNB2&quot;      &quot;HEPH&quot;     &quot;LRRC75A-AS1&quot; &quot;CHD7&quot;   &quot;MARCKSL1&quot; &quot;RPS3&quot;   
[7,] &quot;HSPB3&quot;      &quot;TRDN-AS1&quot; &quot;MT-ND6&quot;      &quot;FBXO40&quot; &quot;ASB2&quot;     &quot;PERM1&quot;  
     [,90]       [,91]     [,92]       [,93]     [,94]     [,95]      [,96]   
[1,] &quot;AQP4&quot;      &quot;ICAM1&quot;   &quot;PALLD&quot;     &quot;OTULINL&quot; &quot;FILIP1L&quot; &quot;CALD1&quot;    &quot;CCL18&quot; 
[2,] &quot;RN7SL364P&quot; &quot;STAT1&quot;   &quot;B4GALT1&quot;   &quot;NFATC4&quot;  &quot;PLEC&quot;    &quot;ADAMTSL1&quot; &quot;TRAM2&quot; 
[3,] &quot;SCRN1&quot;     &quot;YWHAG&quot;   &quot;DDX58&quot;     &quot;MICAL2&quot;  &quot;MYO10&quot;   &quot;HSPA4&quot;    &quot;PRDM14&quot;
[4,] &quot;CLDN8&quot;     &quot;AIF1L&quot;   &quot;OGDHL&quot;     &quot;USH1C&quot;   &quot;KCNJ10&quot;  &quot;ADGRF1&quot;   &quot;TINAG&quot; 
[5,] &quot;CYP2B6&quot;    &quot;C6&quot;      &quot;HMGCS2&quot;    &quot;ADH1A&quot;   &quot;UGT2B10&quot; &quot;C1R&quot;      &quot;FABP1&quot; 
[6,] &quot;RAB31&quot;     &quot;SUCO&quot;    &quot;LAPTM4A&quot;   &quot;NECTIN2&quot; &quot;SEPT8&quot;   &quot;RAD23B&quot;   &quot;MEX3A&quot; 
[7,] &quot;NKX2-5&quot;    &quot;SYNPO2L&quot; &quot;LINC00881&quot; &quot;HSPB8&quot;   &quot;RBM24&quot;   &quot;PPP1R12B&quot; &quot;GSN&quot;   
     [,97]     [,98]    [,99]     [,100]  
[1,] &quot;FBLN2&quot;   &quot;COL5A2&quot; &quot;COL15A1&quot; &quot;THBD&quot;  
[2,] &quot;LAMA4&quot;   &quot;CRLF1&quot;  &quot;LHFPL2&quot;  &quot;DPYSL2&quot;
[3,] &quot;KIF1A&quot;   &quot;TDGF1&quot;  &quot;CDH4&quot;    &quot;CCND1&quot; 
[4,] &quot;TMEM174&quot; &quot;CASR&quot;   &quot;SLC4A4&quot;  &quot;MUC1&quot;  
[5,] &quot;PLG&quot;     &quot;FMO3&quot;   &quot;HSD17B6&quot; &quot;MBL2&quot;  
[6,] &quot;GRHL2&quot;   &quot;RPLP0&quot;  &quot;F2RL1&quot;   &quot;TMEM88&quot;
[7,] &quot;ITGA7&quot;   &quot;SPHKAP&quot; &quot;SORBS1&quot;  &quot;DHRS7C&quot;</code></pre>
<pre class="r"><code>genes[[2]]</code></pre>
<pre><code>            [,1]        [,2]        [,3]        [,4]         [,5]        [,6]
[1,] 0.021825104 0.016537241 0.007709330 0.006212573 0.0052201338 0.004906711
[2,] 0.084734796 0.026676302 0.021340268 0.011791839 0.0090897219 0.009015172
[3,] 0.008554176 0.007538613 0.001321305 0.001086288 0.0008409766 0.000733597
[4,] 0.027903744 0.009986872 0.008265194 0.006687326 0.0057289514 0.003087330
[5,] 0.149622278 0.048025703 0.046162691 0.034254539 0.0307277803 0.030190394
[6,] 0.006403934 0.004974635 0.003505534 0.003401739 0.0031502482 0.002898689
[7,] 0.049764496 0.032193312 0.026725854 0.021409347 0.0178075575 0.015047113
            [,7]         [,8]        [,9]        [,10]        [,11]
[1,] 0.004582384 0.0033514339 0.002514186 0.0025116518 0.0022580675
[2,] 0.008081491 0.0069979738 0.006168069 0.0054832501 0.0046847420
[3,] 0.000578746 0.0005615462 0.000533221 0.0005128261 0.0005034014
[4,] 0.002348549 0.0023336433 0.002281399 0.0022723147 0.0020109132
[5,] 0.023582778 0.0217573572 0.016616594 0.0162949360 0.0147382608
[6,] 0.001831383 0.0013090040 0.001126257 0.0010587682 0.0009412233
[7,] 0.013528846 0.0125681714 0.011500031 0.0113016440 0.0088753684
            [,12]        [,13]        [,14]        [,15]        [,16]
[1,] 0.0021324990 0.0017901344 0.0017640148 0.0012228832 0.0011745618
[2,] 0.0042616498 0.0039809203 0.0034684899 0.0034609852 0.0032429481
[3,] 0.0004751552 0.0004259303 0.0003951467 0.0003467127 0.0003343050
[4,] 0.0019371049 0.0018588332 0.0018445313 0.0015893146 0.0015817622
[5,] 0.0143257732 0.0130639655 0.0128495848 0.0123424475 0.0121330615
[6,] 0.0008730130 0.0008722652 0.0008329767 0.0006824984 0.0006376898
[7,] 0.0084942735 0.0078863951 0.0075671804 0.0071708580 0.0064734126
            [,17]        [,18]        [,19]        [,20]        [,21]
[1,] 0.0010985629 0.0010832805 0.0010657597 0.0009262985 0.0008646501
[2,] 0.0026364631 0.0020971853 0.0017869391 0.0015634478 0.0013996726
[3,] 0.0003342128 0.0003249762 0.0003127213 0.0003091273 0.0003025950
[4,] 0.0014129627 0.0013384980 0.0012814807 0.0012544423 0.0012391852
[5,] 0.0101422830 0.0097937284 0.0096570112 0.0095836304 0.0094557221
[6,] 0.0006138530 0.0006052091 0.0005670472 0.0005590154 0.0005490160
[7,] 0.0054117628 0.0047888066 0.0046821956 0.0046703064 0.0045496679
            [,22]        [,23]        [,24]        [,25]        [,26]
[1,] 0.0008429472 0.0007733435 0.0007347303 0.0007038013 0.0005899387
[2,] 0.0013559187 0.0013467552 0.0011397532 0.0010374186 0.0010278529
[3,] 0.0003020841 0.0002940397 0.0002937638 0.0002822903 0.0002795728
[4,] 0.0012263902 0.0011927942 0.0011806676 0.0011594775 0.0011391254
[5,] 0.0094042151 0.0081654245 0.0075158651 0.0065499225 0.0061607990
[6,] 0.0005141411 0.0004734702 0.0004579324 0.0004341187 0.0004216588
[7,] 0.0042863993 0.0036848414 0.0031508005 0.0031411425 0.0027823779
            [,27]        [,28]        [,29]        [,30]        [,31]
[1,] 0.0005564933 0.0005477390 0.0005448356 0.0005405469 0.0005197751
[2,] 0.0010250386 0.0009483568 0.0008778577 0.0008606731 0.0007929356
[3,] 0.0002692509 0.0002600602 0.0002542371 0.0002402740 0.0002396066
[4,] 0.0011140978 0.0010968673 0.0010550063 0.0009659756 0.0009604997
[5,] 0.0060985708 0.0060709160 0.0057235478 0.0056480286 0.0056006693
[6,] 0.0004106549 0.0003918036 0.0003679634 0.0003672176 0.0003613290
[7,] 0.0026870539 0.0025957640 0.0024345177 0.0023783395 0.0023407002
            [,32]        [,33]        [,34]        [,35]        [,36]
[1,] 0.0005121895 0.0004658419 0.0004445176 0.0003868936 0.0003524071
[2,] 0.0007927572 0.0007666858 0.0007550613 0.0007502080 0.0007261559
[3,] 0.0002281568 0.0002205917 0.0002157365 0.0002146591 0.0002099080
[4,] 0.0009416902 0.0009144551 0.0008751122 0.0008652483 0.0008419725
[5,] 0.0053436139 0.0050037367 0.0049669693 0.0048130498 0.0045688860
[6,] 0.0003608689 0.0003345777 0.0003293307 0.0003163590 0.0003146090
[7,] 0.0022465827 0.0021887099 0.0019872710 0.0018746179 0.0017319343
            [,37]        [,38]        [,39]        [,40]        [,41]
[1,] 0.0003431629 0.0003430909 0.0003261807 0.0003255797 0.0002974177
[2,] 0.0006932400 0.0006529085 0.0006186688 0.0005967782 0.0005911342
[3,] 0.0002074280 0.0002061398 0.0002059575 0.0002043666 0.0002039065
[4,] 0.0008281892 0.0008251698 0.0008098151 0.0008091930 0.0007890196
[5,] 0.0042016702 0.0041582890 0.0040485393 0.0037981273 0.0036772031
[6,] 0.0003074534 0.0003055421 0.0002970700 0.0002925365 0.0002918306
[7,] 0.0016686313 0.0015640987 0.0015452245 0.0015138311 0.0014414590
            [,42]        [,43]        [,44]        [,45]        [,46]
[1,] 0.0002872956 0.0002777363 0.0002763680 0.0002600812 0.0002491809
[2,] 0.0005842789 0.0005271438 0.0005251579 0.0005195768 0.0005190550
[3,] 0.0001960947 0.0001892250 0.0001860221 0.0001850854 0.0001836828
[4,] 0.0007430984 0.0007281969 0.0007020758 0.0006870549 0.0006764122
[5,] 0.0033350223 0.0031542763 0.0030859431 0.0028209263 0.0028121817
[6,] 0.0002739100 0.0002730152 0.0002688237 0.0002651514 0.0002471811
[7,] 0.0014023364 0.0013808373 0.0013203828 0.0012778914 0.0012576974
            [,47]        [,48]        [,49]        [,50]        [,51]
[1,] 0.0002485871 0.0002456934 0.0002365247 0.0002325957 0.0002253652
[2,] 0.0004992131 0.0004930957 0.0004478132 0.0004164504 0.0004123575
[3,] 0.0001794416 0.0001769426 0.0001665452 0.0001625045 0.0001614018
[4,] 0.0006350275 0.0006258576 0.0006235135 0.0005780850 0.0005756445
[5,] 0.0026950202 0.0023991530 0.0023517149 0.0023238100 0.0023223054
[6,] 0.0002452541 0.0002444196 0.0002373764 0.0002302624 0.0002291655
[7,] 0.0011844099 0.0011501752 0.0011495488 0.0010556161 0.0010553138
            [,52]        [,53]        [,54]        [,55]        [,56]
[1,] 0.0002158383 0.0002156144 0.0001968928 0.0001849266 0.0001840905
[2,] 0.0004097581 0.0003952724 0.0003777601 0.0003758807 0.0003709869
[3,] 0.0001590493 0.0001521236 0.0001498290 0.0001476964 0.0001456165
[4,] 0.0005640905 0.0005463118 0.0005432124 0.0005411957 0.0005317293
[5,] 0.0022572292 0.0022267936 0.0021961468 0.0021812699 0.0020963545
[6,] 0.0002285325 0.0002218878 0.0002200332 0.0002090206 0.0002089556
[7,] 0.0010453463 0.0009583566 0.0009531616 0.0009525753 0.0009052890
            [,57]        [,58]        [,59]        [,60]        [,61]
[1,] 0.0001831597 0.0001730311 0.0001728963 0.0001719671 0.0001705543
[2,] 0.0003598870 0.0003498376 0.0003478979 0.0003325717 0.0003226229
[3,] 0.0001441405 0.0001420625 0.0001403150 0.0001365675 0.0001359717
[4,] 0.0005132940 0.0005075738 0.0004735688 0.0004490096 0.0004487926
[5,] 0.0020638170 0.0019499470 0.0019486456 0.0019322918 0.0019254956
[6,] 0.0002083463 0.0002076651 0.0001932390 0.0001923841 0.0001910734
[7,] 0.0008149208 0.0008088153 0.0007955886 0.0007843247 0.0007724917
            [,62]        [,63]        [,64]        [,65]        [,66]
[1,] 0.0001682034 0.0001659809 0.0001658239 0.0001648165 0.0001631576
[2,] 0.0003107147 0.0002812035 0.0002705157 0.0002698501 0.0002674329
[3,] 0.0001337704 0.0001306444 0.0001305300 0.0001301441 0.0001298207
[4,] 0.0004485467 0.0004326371 0.0004206634 0.0004199263 0.0004181681
[5,] 0.0018073425 0.0017856487 0.0017603199 0.0017206270 0.0017130194
[6,] 0.0001817603 0.0001803411 0.0001790988 0.0001767177 0.0001765014
[7,] 0.0007432703 0.0007006584 0.0006980464 0.0006980449 0.0006837111
            [,67]        [,68]        [,69]        [,70]        [,71]
[1,] 0.0001573175 0.0001416509 0.0001393480 0.0001326777 0.0001296361
[2,] 0.0002655971 0.0002653823 0.0002518832 0.0002494583 0.0002490892
[3,] 0.0001262849 0.0001257738 0.0001242324 0.0001223121 0.0001204680
[4,] 0.0004051586 0.0004020051 0.0003850366 0.0003800100 0.0003778727
[5,] 0.0017098304 0.0016929635 0.0016644593 0.0016016274 0.0015907226
[6,] 0.0001753560 0.0001719480 0.0001718652 0.0001714835 0.0001707253
[7,] 0.0006309924 0.0006226195 0.0005563003 0.0005455108 0.0005431937
            [,72]        [,73]        [,74]        [,75]        [,76]
[1,] 0.0001267936 0.0001245616 0.0001194320 0.0001160665 0.0001135407
[2,] 0.0002463203 0.0002456776 0.0002419061 0.0002401867 0.0002328947
[3,] 0.0001188029 0.0001183572 0.0001179844 0.0001163802 0.0001161620
[4,] 0.0003770805 0.0003708623 0.0003667669 0.0003536303 0.0003498626
[5,] 0.0015905809 0.0015860853 0.0015798087 0.0015502273 0.0014678396
[6,] 0.0001693842 0.0001684362 0.0001652907 0.0001615332 0.0001591128
[7,] 0.0005383147 0.0005378746 0.0005259845 0.0005253856 0.0005093965
            [,77]        [,78]        [,79]        [,80]        [,81]
[1,] 0.0001126467 0.0001085288 0.0001079663 0.0001068811 0.0001064648
[2,] 0.0002324271 0.0002307244 0.0002230300 0.0002222025 0.0002181444
[3,] 0.0001147019 0.0001144538 0.0001134532 0.0001130324 0.0001129265
[4,] 0.0003478309 0.0003458074 0.0003445506 0.0003378857 0.0003347802
[5,] 0.0014641625 0.0014380595 0.0014096872 0.0013804765 0.0013378899
[6,] 0.0001568157 0.0001511464 0.0001479836 0.0001478977 0.0001477228
[7,] 0.0005057191 0.0004994261 0.0004660732 0.0004640633 0.0004355087
            [,82]        [,83]        [,84]        [,85]        [,86]
[1,] 0.0001059608 0.0001046176 0.0001037930 0.0001035381 0.0001019756
[2,] 0.0002158133 0.0002141311 0.0002127803 0.0002110480 0.0002018342
[3,] 0.0001122747 0.0001121109 0.0001095385 0.0001092562 0.0001074087
[4,] 0.0003337614 0.0003308536 0.0003097516 0.0002968503 0.0002938386
[5,] 0.0013295496 0.0012817361 0.0012673107 0.0012031291 0.0011912556
[6,] 0.0001459826 0.0001459180 0.0001451234 0.0001442586 0.0001415410
[7,] 0.0004327310 0.0004294944 0.0004158267 0.0004033116 0.0004030161
            [,87]        [,88]        [,89]        [,90]        [,91]
[1,] 9.876953e-05 9.870192e-05 9.777762e-05 9.757241e-05 9.610905e-05
[2,] 1.991761e-04 1.978447e-04 1.976136e-04 1.959800e-04 1.930092e-04
[3,] 1.073900e-04 1.073406e-04 1.062564e-04 1.054404e-04 1.049324e-04
[4,] 2.900339e-04 2.881258e-04 2.877974e-04 2.864165e-04 2.779995e-04
[5,] 1.150784e-03 1.143253e-03 1.138622e-03 1.095397e-03 1.086368e-03
[6,] 1.414498e-04 1.406702e-04 1.338888e-04 1.302205e-04 1.291365e-04
[7,] 3.879292e-04 3.855449e-04 3.841038e-04 3.640621e-04 3.377762e-04
            [,92]        [,93]        [,94]        [,95]        [,96]
[1,] 9.557062e-05 9.469119e-05 9.399902e-05 9.232479e-05 9.157678e-05
[2,] 1.869430e-04 1.863738e-04 1.858462e-04 1.855943e-04 1.836730e-04
[3,] 1.043709e-04 1.030077e-04 1.020395e-04 1.006374e-04 9.893372e-05
[4,] 2.714777e-04 2.659815e-04 2.612673e-04 2.579661e-04 2.508860e-04
[5,] 1.083361e-03 1.071028e-03 1.024391e-03 1.000511e-03 9.819323e-04
[6,] 1.289537e-04 1.288331e-04 1.286600e-04 1.284899e-04 1.229693e-04
[7,] 3.370224e-04 3.326332e-04 3.197540e-04 3.035110e-04 3.002230e-04
            [,97]        [,98]        [,99]       [,100]
[1,] 8.868488e-05 8.837695e-05 8.798181e-05 8.686654e-05
[2,] 1.822866e-04 1.815244e-04 1.685941e-04 1.648436e-04
[3,] 9.810065e-05 9.742211e-05 9.729296e-05 9.550909e-05
[4,] 2.463778e-04 2.418817e-04 2.405755e-04 2.393767e-04
[5,] 9.737093e-04 9.702176e-04 9.508956e-04 9.366931e-04
[6,] 1.204797e-04 1.200348e-04 1.196755e-04 1.190108e-04
[7,] 2.976940e-04 2.949740e-04 2.938082e-04 2.897018e-04</code></pre>
<pre class="r"><code>#Gene boxplot for cluster 1
#COL11A1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000060718&quot;, &quot;COL11A1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-1">
Past versions of genes_classtpx-1.png
</button>
</p>
<div id="fig-genes_classtpx-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 2
#POU5F1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000204531&quot;, &quot;POU5F1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-2">
Past versions of genes_classtpx-2.png
</button>
</p>
<div id="fig-genes_classtpx-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 3
#
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000163631&quot;, &quot;ALB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-3">
Past versions of genes_classtpx-3.png
</button>
</p>
<div id="fig-genes_classtpx-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 4
#SFTPB
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000168878&quot;, &quot;SFTPB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-4">
Past versions of genes_classtpx-4.png
</button>
</p>
<div id="fig-genes_classtpx-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 5
#UMOD
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000169344&quot;, &quot;UMOD&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-5">
Past versions of genes_classtpx-5.png
</button>
</p>
<div id="fig-genes_classtpx-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 6
#TNNT2
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000118194&quot;, &quot;TNNT2&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-6">
Past versions of genes_classtpx-6.png
</button>
</p>
<div id="fig-genes_classtpx-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 7
#ALB
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000163631&quot;, &quot;ALB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-7">
Past versions of genes_classtpx-7.png
</button>
</p>
<div id="fig-genes_classtpx-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># #function to plot genes that distinguish topics
# plot_topic_factors &lt;- function(GoM_object){
#      theta &lt;- GoM_object$theta
#
#      for(topic in 1:ncol(theta)){
#           plot_ratio_factor_by_mean_factor(topic, theta)
#      }
# }
#
# plot_ratio_factor_by_mean_factor &lt;- function(topic_n, GoM_theta){
#      factors_topic_n &lt;- GoM_theta[,topic_n]
#      mean_factors_minus_topic_n &lt;- row_means(GoM_theta[, -topic_n])
#      ratio_factors_topic_n &lt;- factors_topic_n/mean_factors_minus_topic_n
#
#      mean_factors &lt;- row_means(GoM_theta)
#
#      #get matrix of data to be fed into plot
#      data &lt;- as.data.frame(cbind(&quot;factor ratio&quot; = ratio_factors_topic_n, &quot;mean factor&quot; = mean_factors))
#      #identify genes to highlight
#      data &lt;- data %&gt;%
#           mutate(reg = case_when(
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &gt;= 10 ~ &quot;UP&quot;,
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &lt;= -10 ~ &quot;DOWN&quot;,
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &gt; -10 &amp; log(data$`factor ratio`) &lt; 10~ &quot;no_change&quot;,
#                log(data$`mean factor`) &lt; -20 ~ &quot;no_change&quot;
#                ))%&gt;%
#           mutate(reg = factor(reg, levels = c(&quot;UP&quot;, &quot;no_change&quot;,&quot;DOWN&quot;)))
#
#      data_subset= data[which(log(data$`mean factor`) &gt;= -20),]
#      data_subset= data[which(log(data$`factor ratio`) &lt;= -10 | log(data$`factor ratio`) &gt;= 10) ,]
#
#      #plot
#      ggplot(data, aes(x = log(`mean factor`), y = log(`factor ratio`)), label = rownames(data)) +
#           geom_point(aes(color=reg)) +
#           scale_color_manual(name = &quot;Differential \n regulation&quot;,
#                      values = c(&quot;DOWN&quot; = &quot;#058983&quot;,
#                                 &quot;no_change&quot; = &quot;grey&quot;,
#                                 &quot;UP&quot; = &quot;#DEB132&quot;),
#                      labels = c(&quot;UP&quot;, &quot;No Change&quot;, &quot;DOWN&quot;))</code></pre>
</div>
<div id="semi-supervised-topic-modeling-define-mscs-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE MSCs as a cluster)</h1>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs and bmMSCs
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- which(labels==&quot;Chond&quot;)
index_MSCs &lt;- c(adMSC_index, bmMSC1_index, bmMSC2_index)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )


# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;)){
     Topic_clus_7_allMSC &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;))
} else {
Topic_clus_7_allMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;)
}

omega &lt;- Topic_clus_7_allMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-1">
Past versions of “supervised with MSC-1.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-1.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;)){
     Topic_clus_8_allMSC &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;))
} else {
Topic_clus_8_allMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-2">
Past versions of “supervised with MSC-2.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-2.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-2.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- adMSC_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )




# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;)){
     Topic_clus_7_adMSC &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;))
} else {
Topic_clus_7_adMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;)
}

omega &lt;- Topic_clus_7_adMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-3">
Past versions of “supervised with MSC-3.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-3.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-3.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;)){
     Topic_clus_8_adMSC &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;))
} else {
Topic_clus_8_adMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-4">
Past versions of “supervised with MSC-4.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-4.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-4.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC1_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )




# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;)){
     Topic_clus_7_bmMSC1 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;))
} else {
Topic_clus_7_bmMSC1 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC1$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-5">
Past versions of “supervised with MSC-5.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-5.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;)){
     Topic_clus_8_bmMSC1 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;))
} else {
Topic_clus_8_bmMSC1 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-6">
Past versions of “supervised with MSC-6.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-6.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC2_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )


# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;)){
     Topic_clus_7_bmMSC2 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;))
} else {
Topic_clus_7_bmMSC2 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC2$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-7">
Past versions of “supervised with MSC-7.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-7.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;)){
     Topic_clus_8_bmMSC2 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;))
} else {
Topic_clus_8_bmMSC2 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-8">
Past versions of “supervised with MSC-8.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-8.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-define-early-chondros-separately-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE early chondros separately as a cluster)</h1>
<pre class="r"><code>#Here I define the chond cluster based on only early passage chond
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- early_chond
index_MSCs &lt;- c(adMSC_index, bmMSC1_index, bmMSC2_index)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)){
     Topic_clus_7_allMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;))
} else {
Topic_clus_7_allMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_7_allMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-1">
Past versions of “supervised with chond early-1.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-1.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)){
     Topic_clus_8_allMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;))
} else {
Topic_clus_8_allMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-2">
Past versions of “supervised with chond early-2.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-2.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- adMSC_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)){
     Topic_clus_7_adMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;))
} else {
Topic_clus_7_adMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_7_adMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-3">
Past versions of “supervised with chond early-3.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-3.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)){
     Topic_clus_8_adMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;))
} else {
Topic_clus_8_adMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-4">
Past versions of “supervised with chond early-4.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-4.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC1_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)){
     Topic_clus_7_bmMSC1_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;))
} else {
Topic_clus_7_bmMSC1_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC1_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-5">
Past versions of “supervised with chond early-5.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-5.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)){
     Topic_clus_8_bmMSC1_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;))
} else {
Topic_clus_8_bmMSC1_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-6">
Past versions of “supervised with chond early-6.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-6.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC2_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)){
     Topic_clus_7_bmMSC2_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;))
} else {
Topic_clus_7_bmMSC2_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC2_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-7">
Past versions of “supervised with chond early-7.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-7.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)){
     Topic_clus_8_bmMSC2_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;))
} else {
Topic_clus_8_bmMSC2_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-early-8">
Past versions of “supervised with chond early-8.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-early-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond early-8.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-define-late-chondros-separately-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE late chondros separately as a cluster)</h1>
<pre class="r"><code>#Here I define the chond cluster based on only late passage chond
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- late_chond
index_MSCs &lt;- c(adMSC_index, bmMSC1_index, bmMSC2_index)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;)){
     Topic_clus_7_allMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;))
} else {
Topic_clus_7_allMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_7_allMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-1">
Past versions of “supervised with chond late-1.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-1.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;)){
     Topic_clus_8_allMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;))
} else {
Topic_clus_8_allMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-2">
Past versions of “supervised with chond late-2.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-2.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- adMSC_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;)){
     Topic_clus_7_adMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;))
} else {
Topic_clus_7_adMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_7_adMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-3">
Past versions of “supervised with chond late-3.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-3.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;)){
     Topic_clus_8_adMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;))
} else {
Topic_clus_8_adMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-4">
Past versions of “supervised with chond late-4.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-4.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC1_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)){
     Topic_clus_7_bmMSC1_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;))
} else {
Topic_clus_7_bmMSC1_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC1_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-5">
Past versions of “supervised with chond late-5.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-5.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)){
     Topic_clus_8_bmMSC1_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;))
} else {
Topic_clus_8_bmMSC1_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-6">
Past versions of “supervised with chond late-6.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-6.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC2_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)){
     Topic_clus_7_bmMSC2_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;))
} else {
Topic_clus_7_bmMSC2_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC2_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-7">
Past versions of “supervised with chond late-7.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-7.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)){
     Topic_clus_8_bmMSC2_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;))
} else {
Topic_clus_8_bmMSC2_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-chond-late-8">
Past versions of “supervised with chond late-8.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-chond-late-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with chond late-8.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-define-encode-chondros-separately-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE ENCODE chondros separately as a cluster)</h1>
<pre class="r"><code>#Here I define the chond cluster based on only early passage chond
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- chond_encode ######## for ENCODE
index_MSCs &lt;- c(adMSC_index, bmMSC1_index, bmMSC2_index)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;)){
     Topic_clus_7_allMSC_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;))
} else {
Topic_clus_7_allMSC_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC_ENCODEChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_7_allMSC_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;)){
     Topic_clus_8_allMSC_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;))
} else {
Topic_clus_8_allMSC_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC_ENCODEChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- adMSC_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;)){
     Topic_clus_7_adMSC_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;))
} else {
Topic_clus_7_adMSC_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC_ENCODEChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_7_adMSC_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;)){
     Topic_clus_8_adMSC_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;))
} else {
Topic_clus_8_adMSC_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC_ENCODEChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC1_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;)){
     Topic_clus_7_bmMSC1_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;))
} else {
Topic_clus_7_bmMSC1_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1_ENCODEChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC1_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;)){
     Topic_clus_8_bmMSC1_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;))
} else {
Topic_clus_8_bmMSC1_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1_ENCODEChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- bmMSC2_index

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;)){
     Topic_clus_7_bmMSC2_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;))
} else {
Topic_clus_7_bmMSC2_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2_ENCODEChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC2_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;)){
     Topic_clus_8_bmMSC2_ENCODEChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;))
} else {
Topic_clus_8_bmMSC2_ENCODEChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2_ENCODEChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2_ENCODEChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<pre><code>Warning: Use of `df_mlt$document` is discouraged. Use `document` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$value` is discouraged. Use `value` instead.</code></pre>
<pre><code>Warning: Use of `df_mlt$topic` is discouraged. Use `topic` instead.</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20ENCODE-8.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] classtpx_0.99.0            reshape2_1.4.4            
 [3] edgeR_3.20.9               limma_3.34.9              
 [5] gplots_3.0.3               ggfortify_0.4.9           
 [7] MAST_1.4.1                 SummarizedExperiment_1.8.1
 [9] DelayedArray_0.4.1         matrixStats_0.56.0        
[11] Biobase_2.38.0             GenomicRanges_1.30.3      
[13] GenomeInfoDb_1.14.0        IRanges_2.12.0            
[15] S4Vectors_0.16.0           BiocGenerics_0.24.0       
[17] maptpx_1.9-2               slam_0.1-47               
[19] CountClust_1.4.1           cowplot_0.9.3             
[21] ggplot2_3.3.0              scales_1.1.0              
[23] RColorBrewer_1.1-2        

loaded via a namespace (and not attached):
 [1] nlme_3.1-147           bitops_1.0-6           fs_1.4.1              
 [4] rprojroot_1.3-2        tools_3.4.3            backports_1.1.6       
 [7] R6_2.4.1               vegan_2.5-6            KernSmooth_2.23-16    
[10] mgcv_1.8-31            colorspace_1.4-1       permute_0.9-5         
[13] nnet_7.3-13            withr_2.1.2            tidyselect_1.0.0      
[16] gridExtra_2.3          compiler_3.4.3         git2r_0.26.1          
[19] cli_2.0.2              labeling_0.3           caTools_1.17.1.3      
[22] stringr_1.4.0          digest_0.6.25          rmarkdown_2.1         
[25] XVector_0.18.0         pkgconfig_2.0.3        htmltools_0.4.0       
[28] rlang_0.4.5            farver_2.0.3           gtools_3.8.2          
[31] dplyr_0.8.5            RCurl_1.98-1.2         magrittr_1.5          
[34] modeltools_0.2-23      GenomeInfoDbData_1.0.0 Matrix_1.2-18         
[37] Rcpp_1.0.4.6           munsell_0.5.0          fansi_0.4.1           
[40] ape_5.3                abind_1.4-5            lifecycle_0.2.0       
[43] stringi_1.4.6          whisker_0.4            yaml_2.2.1            
[46] MASS_7.3-51.5          zlibbioc_1.24.0        flexmix_2.3-15        
[49] plyr_1.8.6             grid_3.4.3             gdata_2.18.0          
[52] promises_1.1.0         crayon_1.3.4           lattice_0.20-41       
[55] splines_3.4.3          locfit_1.5-9.1         knitr_1.28            
[58] pillar_1.4.3           picante_1.8.1          glue_1.4.0            
[61] evaluate_0.14          data.table_1.12.8      vctrs_0.2.4           
[64] httpuv_1.5.2           gtable_0.3.0           purrr_0.3.4           
[67] tidyr_1.0.2            assertthat_0.2.1       xfun_0.13             
[70] later_1.0.0            tibble_3.0.0           workflowr_1.6.1       
[73] cluster_2.1.0          ellipsis_0.3.0        </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
