<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-02" />

<title>ANT1/ANT2_Pilot</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">ANT1/ANT2_Pilot</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-02</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-12-13
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreeeeffc602ae11f01f4d12e9ef2b1c0d7d419055ddtargetblankeeffc60a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/eeffc602ae11f01f4d12e9ef2b1c0d7d419055dd" target="_blank">eeffc60</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreeeeffc602ae11f01f4d12e9ef2b1c0d7d419055ddtargetblankeeffc60a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    output/ANT1_seurat.rds
    Ignored:    output/ANT2_seurat.rds

Untracked files:
    Untracked:  .RDataTmp
    Untracked:  analysis/ANT2QC.Rmd

Unstaged changes:
    Modified:   analysis/ANT1QC.Rmd
    Modified:   analysis/ANT1_ANT2_.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/analysis/ANT1_ANT2_.Rmd" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
<td>
Color UMAP by expression of marker genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/ANT1_ANT2_.html" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
<td>
Color UMAP by expression of marker genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/ANT1_ANT2_.html" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
<td>
publish analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/analysis/ANT1_ANT2_.Rmd" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
<td>
Lower stringency of QC for ANT1
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/ANT1_ANT2_.html" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
<td>
Lower stringency of QC for ANT1
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/df3d36c226d39f8ba977ba0ad980c8705f7e6643/analysis/ANT1_ANT2_.Rmd" target="_blank">df3d36c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
<td>
Add Seurat analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="analysis-of-processed-sc-pilot-data-ant1ant2" class="section level1">
<h1>Analysis of processed sc pilot data ANT1/ANT2</h1>
<p>ANT1_ANT2 contain multiplexed samples from replicate of the pilot. Equal numbers of cells from 18855 (OA strained), 19160 (unstrained), and 18856 (unstrained) were pooled into one collection and sequenced using the 10x chromium technology.</p>
<p>ANT1: (19160 unstrained; 18856 unstrained; 18855 strained) estimated saturation statistics from Kenneth’s Pipeline: n.reads n.unique total.size saturation 192546655 101667280 2.15403e+08 0.471986</p>
<p>ANT2: (19160 strained; 18856 strained; 18855 unstrained) estimated saturation statistics from Kenneth’s Pipeline: n.reads n.unique total.size saturation 180694884 120674466 3.63297e+08 0.332164</p>
<p>Raw FASTQ files were processed through Kenneth Barr’s pipeline, which aligns reads to hg38 using STAR solo and runs demuxlet on the output.</p>
<div id="load-libraries" class="section level2">
<h2>Load libraries</h2>
<pre class="r"><code>library(data.table)
library(tidyverse)</code></pre>
<pre><code>── Attaching packages ─────────────────────────────────────────────────────────────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>✓ ggplot2 3.2.1     ✓ purrr   0.3.3
✓ tibble  2.1.3     ✓ dplyr   0.8.3
✓ tidyr   1.0.0     ✓ stringr 1.4.0
✓ readr   1.3.1     ✓ forcats 0.4.0</code></pre>
<pre><code>── Conflicts ────────────────────────────────────────────────────────────────────────────────────────────── tidyverse_conflicts() ──
x dplyr::between()   masks data.table::between()
x dplyr::filter()    masks stats::filter()
x dplyr::first()     masks data.table::first()
x dplyr::lag()       masks stats::lag()
x dplyr::last()      masks data.table::last()
x purrr::transpose() masks data.table::transpose()</code></pre>
<pre class="r"><code>library(dplyr)
library(stringr)
library(readr)
library(Matrix)</code></pre>
<pre><code>
Attaching package: &#39;Matrix&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:tidyr&#39;:

    expand, pack, unpack</code></pre>
<pre class="r"><code>library(Seurat)
library(gridExtra)</code></pre>
<pre><code>
Attaching package: &#39;gridExtra&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    combine</code></pre>
</div>
<div id="load-in-data" class="section level2">
<h2>Load in data</h2>
<pre class="r"><code>## link to directories containing data files (count matrices)
proj_dir &lt;- &quot;~/Desktop/scPilot/&quot;
ANT1_dir &lt;- paste0(proj_dir, &quot;YG-AH-2S-ANT-1_S1_L008/&quot;)
ANT2_dir &lt;- paste0(proj_dir, &quot;YG-AH-2S-ANT-2_S2_L008/&quot;)

#read in data

# ##Gene output from STARSOLO
# #ANT1
# demuxlet1 &lt;- fread(paste0(ANT1_dir, &quot;demuxlet.best&quot;, sep = &quot;&quot;))
# count_data1 &lt;- readMM(paste0(ANT1_dir, &quot;Gene/filtered/matrix.mtx&quot;))
# genes1 &lt;- read_tsv(paste0(ANT1_dir, &quot;Gene/filtered/genes.tsv&quot;), col_names = F)
# barcodes1 &lt;- as.data.frame(read_tsv(paste0(ANT1_dir, &quot;Gene/filtered/barcodes.tsv&quot;), col_names = F))
# #ANT2
# demuxlet2 &lt;- fread(paste0(ANT2_dir, &quot;demuxlet.best&quot;, sep = &quot;&quot;))
# count_data2 &lt;- readMM(paste0(ANT2_dir, &quot;Gene/filtered/matrix.mtx&quot;))
# genes2 &lt;- read_tsv(paste0(ANT2_dir, &quot;Gene/filtered/genes.tsv&quot;), col_names = F)
# barcodes2 &lt;- as.data.frame(read_tsv(paste0(ANT2_dir, &quot;Gene/filtered/barcodes.tsv&quot;), col_names = F))


##GeneFull Output from STARSOLO
#ANT1
demuxlet1 &lt;- fread(paste0(ANT1_dir, &quot;demuxlet.best&quot;, sep = &quot;&quot;))
count_data1 &lt;- readMM(paste0(ANT1_dir, &quot;GeneFull/filtered/matrix.mtx&quot;))
genes1 &lt;- read_tsv(paste0(ANT1_dir, &quot;GeneFull/filtered/genes.tsv&quot;), col_names = F)</code></pre>
<pre><code>Parsed with column specification:
cols(
  X1 = col_character(),
  X2 = col_character()
)</code></pre>
<pre class="r"><code>barcodes1 &lt;- as.data.frame(read_tsv(paste0(ANT1_dir, &quot;GeneFull/filtered/barcodes.tsv&quot;), col_names = F))</code></pre>
<pre><code>Parsed with column specification:
cols(
  X1 = col_character()
)</code></pre>
<pre class="r"><code>UMIperCell1 &lt;- read.table(paste0(ANT1_dir, &quot;GeneFull/UMIperCellSorted.txt&quot;), header = F)
#ANT2
demuxlet2 &lt;- fread(paste0(ANT2_dir, &quot;demuxlet.best&quot;, sep = &quot;&quot;))
count_data2 &lt;- readMM(paste0(ANT2_dir, &quot;GeneFull/filtered/matrix.mtx&quot;))
genes2 &lt;- read_tsv(paste0(ANT2_dir, &quot;GeneFull/filtered/genes.tsv&quot;), col_names = F)</code></pre>
<pre><code>Parsed with column specification:
cols(
  X1 = col_character(),
  X2 = col_character()
)</code></pre>
<pre class="r"><code>barcodes2 &lt;- as.data.frame(read_tsv(paste0(ANT2_dir, &quot;GeneFull/filtered/barcodes.tsv&quot;), col_names = F))</code></pre>
<pre><code>Parsed with column specification:
cols(
  X1 = col_character()
)</code></pre>
<pre class="r"><code>UMIperCell2 &lt;- read.table(paste0(ANT2_dir, &quot;GeneFull/UMIperCellSorted.txt&quot;), header = F)</code></pre>
</div>
</div>
<div id="number-of-umis-per-cell-for-each-sample" class="section level1">
<h1>number of umis per cell for each sample</h1>
<pre class="r"><code>table(UMIperCell1$V1 &gt; 10000)</code></pre>
<pre><code>
  FALSE    TRUE 
1208716    1145 </code></pre>
<pre class="r"><code>table(UMIperCell2$V1 &gt; 10000)</code></pre>
<pre><code>
 FALSE   TRUE 
798974    383 </code></pre>
</div>
<div id="decontx-to-remove-data-from-ambient-rna-in-droplets" class="section level1">
<h1>DecontX to remove data from ambient RNA in droplets</h1>
<pre class="r"><code># library(celda)
# #read in data
# 
# #ANT1
# count_data1_raw &lt;- readMM(paste0(ANT1_dir, &quot;Gene/raw/matrix.mtx&quot;))
# genes1_raw &lt;- read_tsv(paste0(ANT1_dir, &quot;Gene/raw/genes.tsv&quot;), col_names = F)
# barcodes1_raw &lt;- as.data.frame(read_tsv(paste0(ANT1_dir, &quot;Gene/raw/barcodes.tsv&quot;), col_names = F))
# #ANT2
# count_data2_raw &lt;- readMM(paste0(ANT2_dir, &quot;Gene/raw/matrix.mtx&quot;))
# genes2_raw &lt;- read_tsv(paste0(ANT2_dir, &quot;Gene/raw/genes.tsv&quot;), col_names = F)
# barcodes2_raw &lt;- as.data.frame(read_tsv(paste0(ANT2_dir, &quot;Gene/raw/barcodes.tsv&quot;), col_names = F))
# 
# 
# #decontxModel &lt;- decontX(counts = count_data1_raw@x)</code></pre>
</div>
<div id="based-on-the-demuxlet-output-assign-label-for-barcodes-based-on-best-output-and-filter-for-sng--barcodes" class="section level1">
<h1>Based on the demuxlet output, assign label for barcodes based on “BEST” output and filter for “SNG-” barcodes</h1>
<pre class="r"><code>#returns a dataframe with two columns, one corresponding to the barcodes and one corresponding to the label given by demuxlet
return_singlet_label &lt;- function(barcodes, demuxlet.out){
  labels &lt;- demuxlet.out$BEST[match(unlist(barcodes), demuxlet.out$BARCODE)]
  return(cbind(barcodes, labels))
}

barcodes1_labeled &lt;- return_singlet_label(barcodes1, demuxlet1)
barcodes2_labeled &lt;- return_singlet_label(barcodes2, demuxlet2)

#table of singlets/multiplets in the filtered data based on demuxlet
table(barcodes1_labeled$labels)</code></pre>
<pre><code>
DBL-NA18855-NA18856-0.500 DBL-NA18856-NA18855-0.500 DBL-NA19160-NA18855-0.500 
                       12                         2                         7 
DBL-NA19160-NA18856-0.500               SNG-NA18855               SNG-NA18856 
                        7                       425                       484 
              SNG-NA19160 
                      302 </code></pre>
<pre class="r"><code>table(barcodes2_labeled$labels)</code></pre>
<pre><code>
DBL-NA18855-NA18856-0.500 DBL-NA18855-NA19160-0.500 DBL-NA18856-NA18855-0.500 
                       16                      2539                         8 
DBL-NA19160-NA18855-0.500 DBL-NA19160-NA18856-0.500               SNG-NA18855 
                     5340                        19                      4744 
              SNG-NA19160 
                     1494 </code></pre>
<pre class="r"><code>## filter for droplets that are singlets
#ANT1
demuxlet_single1 &lt;- demuxlet1 %&gt;%
     dplyr::filter(grepl(&quot;SNG-&quot;, BEST))
singlets_index1 &lt;- unlist(lapply(barcodes1_labeled$X1,&quot;%in%&quot;, table = demuxlet_single1$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets1 &lt;- barcodes1_labeled[singlets_index1,] #use index to subset matrix + barcode names
count_data_singlets1 &lt;- count_data1[,singlets_index1]

#ANT2
demuxlet_single2 &lt;- demuxlet2 %&gt;%
     dplyr::filter(grepl(&quot;SNG-&quot;, BEST))
singlets_index2 &lt;- unlist(lapply(barcodes2_labeled$X1,&quot;%in%&quot;, table = demuxlet_single2$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets2 &lt;- barcodes2_labeled[singlets_index2,] #use index to subset matrix + barcode names
count_data_singlets2 &lt;- count_data2[,singlets_index2]</code></pre>
</div>
<div id="create-seurat-object-for-each-dataset-for-singlet-barcodes-and-add-metadata-in-the-form-of-singlet-identity-for-each-barcode.-rest-of-analysis-adapted-from-httpssatijalab.orgseuratv3.0pbmc3k_tutorial.html" class="section level1">
<h1>Create Seurat object for each dataset (for singlet barcodes) and add metadata in the form of singlet identity for each barcode. Rest of analysis adapted from: <a href="https://satijalab.org/seurat/v3.0/pbmc3k_tutorial.html" class="uri">https://satijalab.org/seurat/v3.0/pbmc3k_tutorial.html</a></h1>
<pre class="r"><code>rownames(count_data_singlets1) &lt;- genes1$X2
colnames(count_data_singlets1) &lt;- barcodes_singlets1$X1

ANT1_seurat &lt;- CreateSeuratObject(counts = count_data_singlets1, project = &quot;ANT1&quot;) %&gt;% 
  AddMetaData(barcodes_singlets1$labels, col.name = &quot;labels&quot;)</code></pre>
<pre><code>Warning: Non-unique features (rownames) present in the input matrix, making
unique</code></pre>
<pre class="r"><code>rownames(count_data_singlets2) &lt;- genes2$X2
colnames(count_data_singlets2) &lt;- barcodes_singlets2$X1

ANT2_seurat &lt;- CreateSeuratObject(counts = count_data_singlets2, project = &quot;ANT2&quot;) %&gt;% 
  AddMetaData(barcodes_singlets2$labels, col.name = &quot;labels&quot;)</code></pre>
<pre><code>Warning: Non-unique features (rownames) present in the input matrix, making
unique</code></pre>
<p>Now we can perform some QC on cells in Seurat</p>
<pre class="r"><code>ANT1_seurat[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(ANT1_seurat, pattern = &quot;^MT-&quot;)
ANT2_seurat[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(ANT2_seurat, pattern = &quot;^MT-&quot;)

# Visualize QC metrics as a violin plot
VlnPlot(ANT1_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-1">
Past versions of QC-1.png
</button>
</p>
<div id="fig-QC-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/QC-1.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-2">
Past versions of QC-2.png
</button>
</p>
<div id="fig-QC-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/QC-2.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT1_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-3">
Past versions of QC-3.png
</button>
</p>
<div id="fig-QC-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-3.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/QC-3.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-4">
Past versions of QC-4.png
</button>
</p>
<div id="fig-QC-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-4.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/QC-4.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># break down nFeature_RNA histograms by individual
hist(ANT1_seurat$nFeature_RNA[unname(ANT1_seurat$labels) == &quot;SNG-NA19160&quot;], main = &quot;ANT1 SNG-NA19160 nFeatures&quot;, xlab = &quot;nFeatures&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-5">
Past versions of QC-5.png
</button>
</p>
<div id="fig-QC-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-5.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(ANT1_seurat$nFeature_RNA[unname(ANT1_seurat$labels) == &quot;SNG-NA18855&quot;], main = &quot;ANT1 SNG-NA18855 nFeatures&quot;, xlab = &quot;nFeatures&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-6">
Past versions of QC-6.png
</button>
</p>
<div id="fig-QC-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-6.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(ANT1_seurat$nFeature_RNA[unname(ANT1_seurat$labels) == &quot;SNG-NA18856&quot;], main = &quot;ANT1 SNG-NA18856 nFeatures&quot;, xlab = &quot;nFeatures&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-7">
Past versions of QC-7.png
</button>
</p>
<div id="fig-QC-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-7.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(ANT2_seurat$nFeature_RNA[unname(ANT1_seurat$labels) == &quot;SNG-NA19160&quot;], main = &quot;ANT2 SNG-NA19160 nFeatures&quot;, xlab = &quot;nFeatures&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-8">
Past versions of QC-8.png
</button>
</p>
<div id="fig-QC-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-8.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(ANT2_seurat$nFeature_RNA[unname(ANT1_seurat$labels) == &quot;SNG-NA18855&quot;], main = &quot;ANT2 SNG-NA18855 nFeatures&quot;, xlab = &quot;nFeatures&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-QC-9">
Past versions of QC-9.png
</button>
</p>
<div id="fig-QC-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/QC-9.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># FeatureScatter is typically used to visualize feature-feature relationships, but can be used
# for anything calculated by the object, i.e. columns in object metadata, PC scores etc.

plot1 &lt;- FeatureScatter(ANT1_seurat, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;percent.mt&quot;)
plot2 &lt;- FeatureScatter(ANT1_seurat, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;nFeature_RNA&quot;)
CombinePlots(plots = list(plot1, plot2))</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot1 &lt;- FeatureScatter(ANT2_seurat, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;percent.mt&quot;)
plot2 &lt;- FeatureScatter(ANT2_seurat, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;nFeature_RNA&quot;)
CombinePlots(plots = list(plot1, plot2))</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Filter barcodes based on nFeatures and %MT 
ANT1_seurat &lt;- subset(ANT1_seurat, subset = nFeature_RNA &gt; 1000 &amp; percent.mt &lt; 10)
table(ANT1_seurat$labels)</code></pre>
<pre><code>
DBL-NA18855-NA18856-0.500 DBL-NA18856-NA18855-0.500 DBL-NA19160-NA18855-0.500 
                        0                         0                         0 
DBL-NA19160-NA18856-0.500               SNG-NA18855               SNG-NA18856 
                        0                       380                       436 
              SNG-NA19160 
                      256 </code></pre>
<pre class="r"><code>ANT2_seurat &lt;- subset(ANT2_seurat, subset = nFeature_RNA &gt; 1000 &amp; nFeature_RNA &amp; percent.mt &lt; 10)
table(ANT2_seurat$labels)</code></pre>
<pre><code>
DBL-NA18855-NA18856-0.500 DBL-NA18855-NA19160-0.500 DBL-NA18856-NA18855-0.500 
                        0                         0                         0 
DBL-NA19160-NA18855-0.500 DBL-NA19160-NA18856-0.500               SNG-NA18855 
                        0                         0                      3437 
              SNG-NA19160 
                      937 </code></pre>
<pre class="r"><code>#Look at QC metrics after filtering
VlnPlot(ANT1_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;, &quot;percent.mt&quot;), ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/QC-13.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="normalizing-data" class="section level1">
<h1>Normalizing Data</h1>
<pre class="r"><code>ANT1_seurat &lt;- NormalizeData(ANT1_seurat, normalization.method = &quot;LogNormalize&quot;, scale.factor = 10000)
ANT2_seurat &lt;- NormalizeData(ANT2_seurat, normalization.method = &quot;LogNormalize&quot;, scale.factor = 10000)</code></pre>
</div>
<div id="feature-selection" class="section level1">
<h1>Feature selection</h1>
<pre class="r"><code>ANT1_seurat &lt;- FindVariableFeatures(ANT1_seurat, selection.method = &quot;vst&quot;, nfeatures = 2000)
ANT2_seurat &lt;- FindVariableFeatures(ANT2_seurat, selection.method = &quot;vst&quot;, nfeatures = 2000)


plot_variable_features &lt;- function(seurat.obj){
  # Identify the 10 most highly variable genes
  top10 &lt;- head(VariableFeatures(seurat.obj), 10)
  
  # plot variable features with and without labels
  plot1 &lt;- VariableFeaturePlot(seurat.obj) + 
    ggtitle(paste0(unname(seurat.obj$orig.ident[1]))) +
       theme(legend.position=&quot;bottom&quot;)
  plot2 &lt;- LabelPoints(plot = plot1, points = top10, repel = TRUE) +
    ggtitle(paste0(unname(seurat.obj$orig.ident[1]))) +
       theme(legend.position=&quot;bottom&quot;)
  return(CombinePlots(plots = list(plot1, plot2)))
}

plot_variable_features(ANT1_seurat)</code></pre>
<pre><code>Warning: Using `as.character()` on a quosure is deprecated as of rlang 0.3.0.
Please use `as_label()` or `as_name()` instead.
This warning is displayed once per session.</code></pre>
<pre><code>When using repel, set xnudge and ynudge to 0 for optimal results</code></pre>
<pre><code>Warning: Transformation introduced infinite values in continuous x-axis</code></pre>
<pre><code>Warning: Transformation introduced infinite values in continuous x-axis</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/feature%20selection-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-selection-1">
Past versions of “feature selection-1.png”
</button>
</p>
<div id="fig-no-spaces-feature-selection-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/feature selection-1.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/feature selection-1.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_variable_features(ANT2_seurat)</code></pre>
<pre><code>When using repel, set xnudge and ynudge to 0 for optimal results</code></pre>
<pre><code>Warning: Transformation introduced infinite values in continuous x-axis

Warning: Transformation introduced infinite values in continuous x-axis</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/feature%20selection-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-selection-2">
Past versions of “feature selection-2.png”
</button>
</p>
<div id="fig-no-spaces-feature-selection-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/feature selection-2.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/feature selection-2.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="scaling-data" class="section level1">
<h1>Scaling Data</h1>
<pre class="r"><code>scale_data &lt;- function(seurat.obj){
  all.genes &lt;- rownames(seurat.obj)
  return(ScaleData(seurat.obj, features = all.genes))
}

ANT1_seurat &lt;- scale_data(ANT1_seurat)</code></pre>
<pre><code>Centering and scaling data matrix</code></pre>
<pre class="r"><code>ANT2_seurat &lt;- scale_data(ANT2_seurat)</code></pre>
<pre><code>Centering and scaling data matrix</code></pre>
</div>
<div id="dimensionality-reduction" class="section level1">
<h1>Dimensionality Reduction</h1>
<p>Using the highly variable features selected above, perform PCA</p>
<pre class="r"><code>nPC &lt;- 100
ANT1_seurat &lt;- RunPCA(ANT1_seurat, features = VariableFeatures(object = ANT1_seurat), npcs = nPC)</code></pre>
<pre><code>PC_ 1 
Positive:  FKBP5, PRLR, LINC02388, CRISPLD2, FOXO1, OTULINL, SAA1, SPON2, AC020637.1, CDKN1C 
       PDPN, DPP7, IFITM3, TSPO, PLIN2, DCN, ZBTB16, IQGAP2, GGT5, PNMT 
       FMO5, ABI3BP, IRS2, LINC01088, CA8, ADAMTS2, MMP23B, WASF3, CLIC6, FAM213A 
Negative:  INHBA, TIMP3, LZTS1, TPM1, CCND2, RAI14, PDLIM3, NT5E, RDH10, CDKN2B 
       MICAL2, AFAP1, STK17B, SOX4, C12orf75, LINC01239, XRCC4, CPE, KCTD16, MANCR 
       TCAF1, MAP3K7CL, BBOX1-AS1, VIM, PAG1, DPF3, AC112721.2, ADAM12, MAML3, COL6A3 
PC_ 2 
Positive:  FTH1, ZFAS1, RPS4X, VIM, SH3BGRL3, TXN, LGALS1, UCHL1, SNHG8, TUBA1B 
       C9orf16, B2M, CD9, KRT8, PHLDA2, CAV1, TIMP1, TPM2, S100A10, YWHAH 
       LGALS3, HSPA5, NQO1, TRIB3, PHGDH, GSTO1, TAGLN, RPL22L1, PSAT1, MTHFD2 
Negative:  PDZRN3, MAGI1, ATRNL1, FAM155A, RERE, GREB1L, SMYD3, MLLT3, ANKRD6, KIF26B 
       WDPCP, ALCAM, RORA, FRS2, AC079380.1, LDLRAD4, UACA, CTNND2, SORBS2, SOX6 
       NOX4, DENND1B, GRIP1, GLCCI1, PARD3B, DGKI, NBEA, PBX1, TMTC2, COL11A1 
PC_ 3 
Positive:  RPS4Y1, FAM129A, PTGR1, SLC3A2, MT1E, SERPINE1, UTY, SLC22A15, DDIT3, TRIB3 
       EZR, ANKRD28, KCNG1, AUTS2, COBLL1, SEL1L3, PTPRM, MGST1, SLC7A11, AL357507.1 
       ANK2, GPR1, FAP, IRS2, GDF15, SQSTM1, TTTY14, BTG1, PKP2, IGFL3 
Negative:  CHCHD2, SLC35F1, AC105402.3, MDK, MFAP4, MEST, PITX1, C5orf17, RPS4X, HLA-A 
       B2M, PLAC9, COL15A1, SPON1, OGN, NETO2, SLC40A1, NREP, LURAP1L, BMP6 
       MXRA5, GNG11, ANKH, HSD17B2, KIF26B, CEMIP, METTL7A, RSPO2, FIBIN, ACTA2 
PC_ 4 
Positive:  CLTB, TMEM88, SMOC2, PROCR, SELENOP, S100A10, GLRX, PLA2G16, PDZRN4, MEST 
       AQP1, PLAC9, TSPAN15, CNTN1, C12orf75, IGFBP7, COL4A4, PDLIM4, GPC3, LGMN 
       ODAPH, SRPX, NSG1, SLIT3, IGFBP2, OLFML3, WNT4, GSTM3, PALMD, LEFTY2 
Negative:  MSC-AS1, PLEKHA5, MAN1A1, ABCC4, EPB41L2, RUNX1, PRKCA, MCC, CBLB, WDR27 
       TSHZ1, FOXP1, LIMS1, GPC6, SERINC5, ANGPT1, FAT4, TMTC1, KLHL4, CSMD2 
       SLC7A11, TGFBI, MBNL1, PTPRK, PRKD1, NEAT1, DLEU2, MGST1, HECTD2, PCED1B 
PC_ 5 
Positive:  NR2F1, TIMP1, IL6ST, LTBP1, MYL10, TGM2, HOPX, LTBP2, FBLN2, MT2A 
       EGFLAM, TIMP3, PDGFRA, NR2F1-AS1, SLC22A3, LOXL3, GUCY1A1, FGF1, COL6A1, GDF6 
       MME, RSPO2, PMP22, PRKCE, AC112721.2, GSTO1, FGF7, TGFBR1, RAMP1, SLC20A1 
Negative:  NRG1, DYNC1I1, ANOS1, PDZRN4, NKAIN4, DPP10, ODAPH, PLXDC1, TENM2, PROCR 
       MFAP5, CADM1, HS6ST2, SMOC2, ROR1, AL356056.1, FRAS1, LANCL2, SDC1, LEFTY2 
       ANGPTL7, PCAT1, HMCN1, COL4A4, ENAH, RASGRF2, FRMD4A, GALNT13, LAMC2, CLSTN2 </code></pre>
<pre class="r"><code>ANT2_seurat &lt;- RunPCA(ANT2_seurat, features = VariableFeatures(object = ANT2_seurat), npcs = nPC)</code></pre>
<pre><code>PC_ 1 
Positive:  MAGI2, SOX5, GPC6, GREB1L, PTPRM, AUTS2, FOXO1, ATRNL1, ARHGAP24, LDLRAD4 
       AC016831.7, CADPS2, FKBP5, PLXDC2, RORA, LINC02388, MCTP2, NAV2, SORBS2, FOXO3 
       ALCAM, ST6GALNAC5, PTPRG, SMYD3, COBLL1, GLIS3, EXT1, MYO10, GRIP1, SCMH1 
Negative:  RPS4X, TPT1, RPS8, RPS14, TAGLN, FTL, ACTB, RPL10, VIM, TPM2 
       LGALS1, TMSB4X, RPL23, RPL7A, RPL37A, RPS3, B2M, ACTA2, S100A10, TIMP1 
       MDK, FTH1, TPM1, SERPINE2, MT2A, COL1A2, TXN, TMEM59, PLAC9, C12orf75 
PC_ 2 
Positive:  BBOX1-AS1, KCTD16, COL4A1, TPM1, DKK2, CADM1, COL6A3, LINC01239, ITGA11, TIMP3 
       HMCN1, FN1, PLS3, COL5A2, ATXN1, ADAM12, LMO7, GLS, XRCC4, MAML3 
       FAM155A, ANOS1, SUGCT, THBS1, RUNX1, RASGRF2, PRICKLE1, APBB2, INHBA, UACA 
Negative:  SPON2, FTH1, TSPO, SAA1, RPS4Y1, FTL, SMOC2, DPP7, LINC02388, TMEM88 
       SELENOP, PDZRN4, PDPN, PNMT, CDKN1C, FKBP5, PRLR, NUPR1, APOE, ADAMTS2 
       LUM, LINC01088, APOD, IFITM3, SLPI, OTULINL, MGST3, FDPS, AUTS2, S100A6 
PC_ 3 
Positive:  ANKRD1, NRG1, NTM, TUBA1A, IGFBP2, MAP3K7CL, SOX4, KRT18, ODAPH, SMOC2 
       ENAH, DYNC1I1, AC097478.1, PRSS23, PDZRN4, LINC01013, MFAP5, IGFBP7, VIM, WNT5B 
       MT2A, MANCR, TSPO, FDPS, CDKN2A, INHBA, LINC00862, GPC4, AUTS2, THBS1 
Negative:  RSPO2, MFAP4, MDK, B2M, MYOCD, DCN, SOX5, COL6A1, COL15A1, SPON1 
       PLAGL1, SERPINF1, LSAMP, KIF26B, ITGA1, LUM, EBF1, COL6A2, BMPER, SLC40A1 
       VCAN, HOPX, CHSY3, COL4A1, TGFBI, OGN, MME, GALNT1, UMAD1, ADAMTS19 
PC_ 4 
Positive:  FN1, MGST1, TGFBI, ANKRD28, NR2F1, KCND2, ACTG2, ABCC4, COL6A2, NEAT1 
       RAMP1, PLIN2, EGFL6, THBS2, COMP, SLC12A8, SNCAIP, THSD7A, DLK1, RUNX1 
       ADAMTS1, CLMP, HIPK2, EPB41L2, NR2F1-AS1, COL1A2, MMP2, COL6A1, UGT2B7, RARB 
Negative:  PLAC9, PDZRN4, SLC4A8, SLIT3, MEST, ODAPH, AQP1, OGN, SMOC2, MDK 
       IGFBP7, SLC40A1, ATRNL1, CTNND2, FLT1, SELENOP, FAM155A, PDZRN3, COL4A4, S100A10 
       LURAP1L, ADAM28, GPC3, AL390957.1, C12orf75, TMEM88, RNF152, GLIPR1, TMTC2, RGS4 
PC_ 5 
Positive:  RPS3, RPL7A, RPL10, RPS14, TPT1, TIMP1, FTH1, RPS8, RPL37A, RPL23 
       KRT18, LUM, RPS4X, IFITM3, PCBP2, H2AFZ, MT2A, ACTB, MGST3, PDPN 
       FTL, TMSB4X, MEST, SERPINE2, B2M, TUBB, ERG28, CKAP4, IL6ST, MYADM 
Negative:  BBOX1-AS1, DPP10, CNTN5, ZFPM2, GRIK2, FAM155A, SMYD3, AEBP2, PCAT1, NRG1 
       LUZP2, HMCN1, CTNNA3, MCC, SLC24A2, DYNC1I1, KCTD16, MON2, CNTN6, FAM19A2 
       PLXDC1, NOX4, AC024901.1, PURPL, GLS, JAKMIP2-AS1, LINC01429, AL356108.1, LRMDA, GALNT13 </code></pre>
<pre class="r"><code># Examine and visualize PCA results a few different ways
print(ANT2_seurat[[&quot;pca&quot;]], dims = 1:5, nfeatures = 5)</code></pre>
<pre><code>PC_ 1 
Positive:  MAGI2, SOX5, GPC6, GREB1L, PTPRM 
Negative:  RPS4X, TPT1, RPS8, RPS14, TAGLN 
PC_ 2 
Positive:  BBOX1-AS1, KCTD16, COL4A1, TPM1, DKK2 
Negative:  SPON2, FTH1, TSPO, SAA1, RPS4Y1 
PC_ 3 
Positive:  ANKRD1, NRG1, NTM, TUBA1A, IGFBP2 
Negative:  RSPO2, MFAP4, MDK, B2M, MYOCD 
PC_ 4 
Positive:  FN1, MGST1, TGFBI, ANKRD28, NR2F1 
Negative:  PLAC9, PDZRN4, SLC4A8, SLIT3, MEST 
PC_ 5 
Positive:  RPS3, RPL7A, RPL10, RPS14, TPT1 
Negative:  BBOX1-AS1, DPP10, CNTN5, ZFPM2, GRIK2 </code></pre>
<pre class="r"><code>VizDimLoadings(ANT1_seurat, dims = 1:2, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-1">
Past versions of PCA-1.png
</button>
</p>
<div id="fig-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-1.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/PCA-1.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(ANT1_seurat, reduction = &quot;pca&quot;, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-2">
Past versions of PCA-2.png
</button>
</p>
<div id="fig-PCA-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-2.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/PCA-2.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Get the total variance:
pca &lt;- ANT1_seurat[[&quot;pca&quot;]]
mat &lt;- Seurat::GetAssayData(ANT1_seurat, assay = &quot;RNA&quot;, slot = &quot;scale.data&quot;)
total_variance &lt;- sum(matrixStats::rowVars(mat))


eigValues &lt;- (pca@stdev)^2  ## EigenValues
varExplained &lt;- eigValues / total_variance
head(varExplained) #var explained by first 10 PCs</code></pre>
<pre><code>[1] 0.005799424 0.003629453 0.002796576 0.002379185 0.001721852 0.001479069</code></pre>
<pre class="r"><code>DimHeatmap(ANT1_seurat, dims = 1:15, cells = 500, balanced = TRUE)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-3">
Past versions of PCA-3.png
</button>
</p>
<div id="fig-PCA-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-3.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/PCA-3.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ElbowPlot(ANT1_seurat, ndims = nPC) #42 PCs?</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-4">
Past versions of PCA-4.png
</button>
</p>
<div id="fig-PCA-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/PCA-4.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-4.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/PCA-4.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Examine and visualize PCA results a few different ways
print(ANT2_seurat[[&quot;pca&quot;]], dims = 1:5, nfeatures = 5)</code></pre>
<pre><code>PC_ 1 
Positive:  MAGI2, SOX5, GPC6, GREB1L, PTPRM 
Negative:  RPS4X, TPT1, RPS8, RPS14, TAGLN 
PC_ 2 
Positive:  BBOX1-AS1, KCTD16, COL4A1, TPM1, DKK2 
Negative:  SPON2, FTH1, TSPO, SAA1, RPS4Y1 
PC_ 3 
Positive:  ANKRD1, NRG1, NTM, TUBA1A, IGFBP2 
Negative:  RSPO2, MFAP4, MDK, B2M, MYOCD 
PC_ 4 
Positive:  FN1, MGST1, TGFBI, ANKRD28, NR2F1 
Negative:  PLAC9, PDZRN4, SLC4A8, SLIT3, MEST 
PC_ 5 
Positive:  RPS3, RPL7A, RPL10, RPS14, TPT1 
Negative:  BBOX1-AS1, DPP10, CNTN5, ZFPM2, GRIK2 </code></pre>
<pre class="r"><code>VizDimLoadings(ANT2_seurat, dims = 1:2, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-5">
Past versions of PCA-5.png
</button>
</p>
<div id="fig-PCA-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/PCA-5.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-5.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(ANT2_seurat, reduction = &quot;pca&quot;, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-6">
Past versions of PCA-6.png
</button>
</p>
<div id="fig-PCA-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/PCA-6.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-6.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Get the total variance:
pca &lt;- ANT2_seurat[[&quot;pca&quot;]]
mat &lt;- Seurat::GetAssayData(ANT2_seurat, assay = &quot;RNA&quot;, slot = &quot;scale.data&quot;)
total_variance &lt;- sum(matrixStats::rowVars(mat))


eigValues &lt;- (pca@stdev)^2  ## EigenValues
varExplained &lt;- eigValues / total_variance
head(varExplained) #var explained by first 10 PCs</code></pre>
<pre><code>[1] 0.0024593841 0.0022554025 0.0010839355 0.0008948233 0.0006893108
[6] 0.0006387141</code></pre>
<pre class="r"><code>DimHeatmap(ANT2_seurat, dims = 1:15, cells = 500, balanced = TRUE)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-7">
Past versions of PCA-7.png
</button>
</p>
<div id="fig-PCA-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/PCA-7.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-7.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ElbowPlot(ANT2_seurat, ndims = nPC) #29 PCs?</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/PCA-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-8">
Past versions of PCA-8.png
</button>
</p>
<div id="fig-PCA-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/PCA-8.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/PCA-8.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="clustering" class="section level1">
<h1>Clustering</h1>
<pre class="r"><code>num_PCs &lt;- 42

ANT1_seurat &lt;- FindNeighbors(ANT1_seurat, dims = 1:num_PCs)</code></pre>
<pre><code>Computing nearest neighbor graph</code></pre>
<pre><code>Computing SNN</code></pre>
<pre class="r"><code>ANT1_seurat &lt;- FindClusters(ANT1_seurat, resolution = 0.5)</code></pre>
<pre><code>Modularity Optimizer version 1.3.0 by Ludo Waltman and Nees Jan van Eck

Number of nodes: 1072
Number of edges: 42617

Running Louvain algorithm...
Maximum modularity in 10 random starts: 0.8415
Number of communities: 6
Elapsed time: 0 seconds</code></pre>
<pre class="r"><code>#Run UMAP
ANT1_seurat &lt;- RunUMAP(ANT1_seurat, dims = 1:num_PCs)</code></pre>
<pre><code>Warning: The default method for RunUMAP has changed from calling Python UMAP via reticulate to the R-native UWOT using the cosine metric
To use Python UMAP via reticulate, set umap.method to &#39;umap-learn&#39; and metric to &#39;correlation&#39;
This message will be shown once per session</code></pre>
<pre><code>12:44:58 UMAP embedding parameters a = 0.9922 b = 1.112</code></pre>
<pre><code>12:44:58 Read 1072 rows and found 42 numeric columns</code></pre>
<pre><code>12:44:58 Using Annoy for neighbor search, n_neighbors = 30</code></pre>
<pre><code>12:44:58 Building Annoy index with metric = cosine, n_trees = 50</code></pre>
<pre><code>0%   10   20   30   40   50   60   70   80   90   100%</code></pre>
<pre><code>[----|----|----|----|----|----|----|----|----|----|</code></pre>
<pre><code>**************************************************|
12:45:00 Writing NN index file to temp file /var/folders/td/y4r5skr139d88lvmz1yy97440000gn/T//RtmpRAo2BR/file8aa66e369e6
12:45:00 Searching Annoy index using 1 thread, search_k = 3000
12:45:00 Annoy recall = 100%
12:45:00 Commencing smooth kNN distance calibration using 1 thread
12:45:02 Initializing from normalized Laplacian + noise
12:45:02 Commencing optimization for 500 epochs, with 41976 positive edges
12:45:07 Optimization finished</code></pre>
<pre class="r"><code>p1 &lt;- DimPlot(ANT1_seurat, reduction = &quot;umap&quot;)
p2 &lt;- DimPlot(ANT1_seurat, reduction = &quot;umap&quot;, group.by = &quot;labels&quot;)
p3 &lt;- FeaturePlot(ANT1_seurat, features = &quot;COL10A1&quot;)
p4 &lt;- FeaturePlot(ANT1_seurat, features = &quot;COL11A1&quot;)
p5 &lt;- FeaturePlot(ANT1_seurat, features = &quot;SOX5&quot;)
p6 &lt;- FeaturePlot(ANT1_seurat, features = &quot;SOX6&quot;)
p7 &lt;- FeaturePlot(ANT1_seurat, features = &quot;SOX9&quot;)
p8 &lt;- FeaturePlot(ANT1_seurat, features = &quot;MMP2&quot;)
p9 &lt;- FeaturePlot(ANT1_seurat, features = &quot;MMP1&quot;)
p10 &lt;- FeaturePlot(ANT1_seurat, features = &quot;MMP13&quot;)</code></pre>
<pre><code>Warning in FeaturePlot(ANT1_seurat, features = &quot;MMP13&quot;): All cells have the same
value (0) of MMP13.</code></pre>
<pre class="r"><code>grid.arrange(p1, p2, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-1">
Past versions of clustering-1.png
</button>
</p>
<div id="fig-clustering-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-1.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-1.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/clustering-1.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/clustering-1.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>grid.arrange(p3, p4, p5, p6, p7, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-2">
Past versions of clustering-2.png
</button>
</p>
<div id="fig-clustering-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-2.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-2.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/clustering-2.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2ea66286c069d19bfc7e7c2bd11bd31015f5c49/docs/figure/ANT1_ANT2_.Rmd/clustering-2.png" target="_blank">e2ea662</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>grid.arrange(p8, p9, p10, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-3">
Past versions of clustering-3.png
</button>
</p>
<div id="fig-clustering-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-3.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-3.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/clustering-3.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>saveRDS(ANT1_seurat, file = &quot;output/ANT1_seurat.rds&quot;)





num_PCs &lt;- 29

ANT2_seurat &lt;- FindNeighbors(ANT2_seurat, dims = 1:num_PCs)</code></pre>
<pre><code>Computing nearest neighbor graph
Computing SNN</code></pre>
<pre class="r"><code>ANT2_seurat &lt;- FindClusters(ANT2_seurat, resolution = 0.5)</code></pre>
<pre><code>Modularity Optimizer version 1.3.0 by Ludo Waltman and Nees Jan van Eck

Number of nodes: 4374
Number of edges: 150324

Running Louvain algorithm...
Maximum modularity in 10 random starts: 0.7720
Number of communities: 6
Elapsed time: 1 seconds</code></pre>
<pre class="r"><code>#Run UMAP
ANT2_seurat &lt;- RunUMAP(ANT2_seurat, dims = 1:num_PCs)</code></pre>
<pre><code>12:45:44 UMAP embedding parameters a = 0.9922 b = 1.112
12:45:44 Read 4374 rows and found 29 numeric columns
12:45:44 Using Annoy for neighbor search, n_neighbors = 30
12:45:44 Building Annoy index with metric = cosine, n_trees = 50
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
12:45:45 Writing NN index file to temp file /var/folders/td/y4r5skr139d88lvmz1yy97440000gn/T//RtmpRAo2BR/file8aa5c3dafcf
12:45:46 Searching Annoy index using 1 thread, search_k = 3000
12:45:47 Annoy recall = 100%
12:45:48 Commencing smooth kNN distance calibration using 1 thread
12:45:49 Initializing from normalized Laplacian + noise
12:45:50 Commencing optimization for 500 epochs, with 181986 positive edges
12:46:05 Optimization finished</code></pre>
<pre class="r"><code>p1_ &lt;- DimPlot(ANT2_seurat, reduction = &quot;umap&quot;)
p2_ &lt;- DimPlot(ANT2_seurat, reduction = &quot;umap&quot;, group.by = &quot;labels&quot;)
p3_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;COL10A1&quot;)
p4_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;COL11A1&quot;)
p5_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;SOX5&quot;)
p6_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;SOX6&quot;)
p7_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;SOX9&quot;)
p8_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;MMP2&quot;)
p9_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;MMP1&quot;)
p10_ &lt;- FeaturePlot(ANT2_seurat, features = &quot;MMP13&quot;)</code></pre>
<pre><code>Warning in FeaturePlot(ANT2_seurat, features = &quot;MMP13&quot;): All cells have the same
value (0) of MMP13.</code></pre>
<pre class="r"><code>grid.arrange(p1_, p2_, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-4">
Past versions of clustering-4.png
</button>
</p>
<div id="fig-clustering-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-4.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-4.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5e0aa130f083744c554c62a1b3831a100a6a73db/docs/figure/ANT1_ANT2_.Rmd/clustering-4.png" target="_blank">5e0aa13</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>grid.arrange(p3_, p4_, p5_, p6_, p7_, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-5">
Past versions of clustering-5.png
</button>
</p>
<div id="fig-clustering-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-5.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-5.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>grid.arrange(p8_, p9_, p10_, nrow = 2)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/clustering-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-clustering-6">
Past versions of clustering-6.png
</button>
</p>
<div id="fig-clustering-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/clustering-6.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8cf5920aed71bff561ebc5fadb4ac0c17195dfba/docs/figure/ANT1_ANT2_.Rmd/clustering-6.png" target="_blank">8cf5920</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>saveRDS(ANT2_seurat, file = &quot;output/ANT2_seurat.rds&quot;)</code></pre>
<pre class="r"><code>#Chondrocyte markers
VlnPlot(ANT1_seurat, features = c(&quot;COL2A1&quot;, &quot;ACAN&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-1">
Past versions of “expression by individual-1.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-1.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT1_seurat, features = c(&quot;COL10A1&quot;, &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;, &quot;SOX9&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-2">
Past versions of “expression by individual-2.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-2.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Strain markers
VlnPlot(ANT1_seurat, features = c(&quot;MMP1&quot;, &quot;MMP13&quot;, &quot;IL1B&quot;, &quot;MMP3&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of MMP13.</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of MMP3.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-3">
Past versions of “expression by individual-3.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-3.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT1_seurat, features = c(&quot;MMP2&quot;, &quot;COMP&quot;, &quot;TNF&quot;, &quot;IBSP&quot;, &quot;TIMP1&quot;,  &quot;MMP9&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-4">
Past versions of “expression by individual-4.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-4.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#MSC Flow markers
VlnPlot(ANT1_seurat, features = c(&quot;ENG&quot;, &quot;THY1&quot;,  &quot;NT5E&quot;), 
     pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-5">
Past versions of “expression by individual-5.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-5.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT1_seurat, features = c(&quot;PTPRC&quot;, &quot;CD34&quot;, &quot;CD14&quot;, &quot;ITGAM&quot;, &quot;CD79A&quot;, &quot;CD19&quot;), 
     pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of CD34.</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of CD19.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-6">
Past versions of “expression by individual-6.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-6.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Chondrocyte markers
VlnPlot(ANT2_seurat, features = c(&quot;COL2A1&quot;, &quot;ACAN&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of ACAN.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-7">
Past versions of “expression by individual-7.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-7.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;COL10A1&quot;, &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;, &quot;SOX9&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-8">
Past versions of “expression by individual-8.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-8.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Strain markers
VlnPlot(ANT2_seurat, features = c(&quot;MMP1&quot;, &quot;MMP13&quot;, &quot;IL1B&quot;, &quot;MMP3&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of MMP13.</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of IL1B.</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of MMP3.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-9">
Past versions of “expression by individual-9.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-9.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;MMP2&quot;, &quot;COMP&quot;, &quot;TNF&quot;, &quot;IBSP&quot;, &quot;TIMP1&quot;,  &quot;MMP9&quot;), 
    pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of IBSP.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-10">
Past versions of “expression by individual-10.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-10.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#MSC Flow markers
VlnPlot(ANT2_seurat, features = c(&quot;ENG&quot;, &quot;THY1&quot;,  &quot;NT5E&quot;), 
     pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-11">
Past versions of “expression by individual-11.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-11.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(ANT2_seurat, features = c(&quot;PTPRC&quot;, &quot;CD34&quot;, &quot;CD14&quot;, &quot;ITGAM&quot;, &quot;CD79A&quot;, &quot;CD19&quot;), 
     pt.size = 0.2, ncol = 3, group.by = &quot;labels&quot;)</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of CD34.</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE], idents =
idents, : All cells have the same value of CD19.</code></pre>
<p><img src="figure/ANT1_ANT2_.Rmd/expression%20by%20individual-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-expression-by-individual-12">
Past versions of “expression by individual-12.png”
</button>
</p>
<div id="fig-no-spaces-expression-by-individual-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1bb7658309c6ee18f03cbc04bfd97c70b8b555dd/docs/figure/ANT1_ANT2_.Rmd/expression by individual-12.png" target="_blank">1bb7658</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] gridExtra_2.3     Seurat_3.1.1      Matrix_1.2-18     forcats_0.4.0    
 [5] stringr_1.4.0     dplyr_0.8.3       purrr_0.3.3       readr_1.3.1      
 [9] tidyr_1.0.0       tibble_2.1.3      ggplot2_3.2.1     tidyverse_1.3.0  
[13] data.table_1.12.6

loaded via a namespace (and not attached):
  [1] readxl_1.3.1        backports_1.1.5     workflowr_1.5.0    
  [4] sn_1.5-4            plyr_1.8.5          igraph_1.2.4.2     
  [7] lazyeval_0.2.2      splines_3.6.1       listenv_0.8.0      
 [10] TH.data_1.0-10      digest_0.6.23       htmltools_0.4.0    
 [13] gdata_2.18.0        fansi_0.4.0         magrittr_1.5       
 [16] cluster_2.1.0       ROCR_1.0-7          globals_0.12.5     
 [19] modelr_0.1.5        matrixStats_0.55.0  RcppParallel_4.4.4 
 [22] R.utils_2.9.2       sandwich_2.5-1      colorspace_1.4-1   
 [25] rvest_0.3.5         ggrepel_0.8.1       haven_2.2.0        
 [28] xfun_0.11           crayon_1.3.4        jsonlite_1.6       
 [31] zeallot_0.1.0       survival_3.1-8      zoo_1.8-6          
 [34] ape_5.3             glue_1.3.1          gtable_0.3.0       
 [37] leiden_0.3.1        future.apply_1.3.0  BiocGenerics_0.30.0
 [40] scales_1.1.0        mvtnorm_1.0-11      DBI_1.0.0          
 [43] bibtex_0.4.2        Rcpp_1.0.3          metap_1.2          
 [46] plotrix_3.7-7       viridisLite_0.3.0   reticulate_1.13    
 [49] rsvd_1.0.2          SDMTools_1.1-221.2  tsne_0.1-3         
 [52] stats4_3.6.1        htmlwidgets_1.5.1   httr_1.4.1         
 [55] gplots_3.0.1.1      RColorBrewer_1.1-2  TFisher_0.2.0      
 [58] ica_1.0-2           farver_2.0.1        pkgconfig_2.0.3    
 [61] R.methodsS3_1.7.1   uwot_0.1.5          dbplyr_1.4.2       
 [64] labeling_0.3        tidyselect_0.2.5    rlang_0.4.2        
 [67] reshape2_1.4.3      later_1.0.0         munsell_0.5.0      
 [70] cellranger_1.1.0    tools_3.6.1         cli_2.0.0          
 [73] generics_0.0.2      broom_0.5.2         ggridges_0.5.1     
 [76] evaluate_0.14       yaml_2.2.0          npsurv_0.4-0       
 [79] knitr_1.26          fs_1.3.1            fitdistrplus_1.0-14
 [82] caTools_1.17.1.3    RANN_2.6.1          pbapply_1.4-2      
 [85] future_1.15.1       nlme_3.1-143        whisker_0.4        
 [88] R.oo_1.23.0         xml2_1.2.2          compiler_3.6.1     
 [91] rstudioapi_0.10     plotly_4.9.1        png_0.1-7          
 [94] lsei_1.2-0          reprex_0.3.0        stringi_1.4.3      
 [97] RSpectra_0.16-0     lattice_0.20-38     multtest_2.40.0    
[100] vctrs_0.2.0         mutoss_0.1-12       pillar_1.4.2       
[103] lifecycle_0.1.0     Rdpack_0.11-0       lmtest_0.9-37      
[106] RcppAnnoy_0.0.14    cowplot_1.0.0       bitops_1.0-6       
[109] irlba_2.3.3         gbRd_0.4-11         httpuv_1.5.2       
[112] R6_2.4.1            promises_1.1.0      KernSmooth_2.23-16 
[115] codetools_0.2-16    MASS_7.3-51.4       gtools_3.8.1       
[118] assertthat_0.2.1    rprojroot_1.3-2     withr_2.1.2        
[121] sctransform_0.2.0   mnormt_1.5-5        multcomp_1.4-11    
[124] parallel_3.6.1      hms_0.5.2           grid_3.6.1         
[127] rmarkdown_1.18      Rtsne_0.15          git2r_0.26.1       
[130] numDeriv_2016.8-1.1 Biobase_2.44.0      lubridate_1.7.4    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
