<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-01-28" />

<title>Bulk_cellComposition_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bulk_cellComposition_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-01-28</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-03-23
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree925e86dc706803cd69f455e15f93cb6879c77090targetblank925e86da"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/925e86dc706803cd69f455e15f93cb6879c77090" target="_blank">925e86d</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree925e86dc706803cd69f455e15f93cb6879c77090targetblank925e86da" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds

Untracked files:
    Untracked:  analysis/coefficients_of_variation.Rmd
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda
    Untracked:  data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda

Unstaged changes:
    Modified:   analysis/Bulk_cellComposition_analysis.Rmd
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/925e86dc706803cd69f455e15f93cb6879c77090/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">925e86d</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
add printout of psseudobulk correlations
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
define early/late passage chondros as a supervised cluster
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/940b53efe5b3edcac621e238213ebf8d896fd263/docs/Bulk_cellComposition_analysis.html" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
define early/late passage chondros as a supervised cluster
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6baf979f9ff594f844a14171435152ff5b0e7081/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">6baf979</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-10
</td>
<td>
Pseudobulk correlation heatmap with bulk samples
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
re-run semi-supervised without heart sample that is outlier
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/Bulk_cellComposition_analysis.html" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
re-run semi-supervised without heart sample that is outlier
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c9015d5f25785b2ffd97f0119b94015c2cc84498/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">c9015d5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Revise pseudobulk to use raw counts not normalized
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a46d6f37799b4f9c74c12be5c41c412f95f2fa3e/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">a46d6f3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Add gene analysis to semi-supervised topic model
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a46d6f37799b4f9c74c12be5c41c412f95f2fa3e/docs/Bulk_cellComposition_analysis.html" target="_blank">a46d6f3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Add gene analysis to semi-supervised topic model
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ad1e27bd3b051a731c2357fb8d8f981baa35f6af/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">ad1e27b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-02
</td>
<td>
separate up/down in de analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/68e5e33dc85bf6340efc50200e47b0766711c93c/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
<td>
add k = 8 for semi-supervised gom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/68e5e33dc85bf6340efc50200e47b0766711c93c/docs/Bulk_cellComposition_analysis.html" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
<td>
add k = 8 for semi-supervised gom
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/59b30d0b555983d416401d53765f1863f550e7ab/docs/Bulk_cellComposition_analysis.html" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
Semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/Bulk_cellComposition_analysis.html" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
Semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/Bulk_cellComposition_analysis.html" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
<td>
Separate out sample based on origin; also add pseudobulk
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/Bulk_cellComposition_analysis.html" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
<td>
Separate out sample based on origin; also add pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
check expression using RLE norm
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/Bulk_cellComposition_analysis.html" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
check expression using RLE norm
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/605da875aae021b28ea700a7c7af372b50a19dc6/docs/Bulk_cellComposition_analysis.html" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/2bbbdad15237735154e8db0270272fefd55760a9/docs/Bulk_cellComposition_analysis.html" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/analysis/Bulk_cellComposition_analysis.Rmd" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/26132157c6605eb09d8628dff14a004a6a374809/docs/Bulk_cellComposition_analysis.html" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="cluster-pilot-bulk-rna-data-with-subset-of-cellatlas-bulk-data-and-other-data-to-determine-where-my-samples-fall-in-comparison" class="section level1">
<h1>Cluster pilot Bulk RNA data with subset of cellAtlas bulk data and other data to determine where my samples fall in comparison</h1>
<div id="load-data" class="section level2">
<h2>Load data</h2>
<pre class="r"><code>library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)
library(&quot;cowplot&quot;)</code></pre>
<pre><code>Loading required package: ggplot2</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>library(&quot;CountClust&quot;)
library(&quot;maptpx&quot;)</code></pre>
<pre><code>Loading required package: slam</code></pre>
<pre class="r"><code>library(&quot;MAST&quot;)</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following object is masked from &#39;package:maptpx&#39;:

    normalize</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, cbind, colMeans, colnames,
    colSums, do.call, duplicated, eval, evalq, Filter, Find, get, grep,
    grepl, intersect, is.unsorted, lapply, lengths, Map, mapply, match,
    mget, order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,
    rbind, Reduce, rowMeans, rownames, rowSums, sapply, setdiff, sort,
    table, tapply, union, unique, unsplit, which, which.max, which.min</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    apply</code></pre>
<pre><code>
Attaching package: &#39;MAST&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    filter</code></pre>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;ggfortify&quot;)
library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:IRanges&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:S4Vectors&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code># load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

# Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))

#load cell atlas data matrix
load(&quot;data/cell_atlas_ref_panel&quot;)
cell.line &lt;- cell.line[,c(12:20,35,36,40,55:57)]

#load pilot data
raw_counts &lt;- as.matrix(readRDS(&quot;data/raw_counts_relabeled.rds&quot;))

# load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

#relabel pilot data genes to symbols
pilot &lt;- raw_counts
rownames(pilot) &lt;- gene_anno$external_gene_name[match(unlist(rownames(pilot)), gene_anno$ensembl_gene_id)]</code></pre>
</div>
<div id="before-combining-the-datasets-i-apply-the-same-standardizationsnormalizations-performed-on-the-cellatlas-data-to-my-pilot-data" class="section level2">
<h2>Before combining the datasets, I apply the same standardizations/normalizations performed on the cellAtlas data to my pilot data</h2>
<pre class="r"><code>#perform normalizations (taken from cellAtlas_assignment.R code)
pilot_norm &lt;- apply(pilot, 2, function(x) x/sum(x))
pilot_libsize &lt;- apply(pilot_norm, 2, function(x) x * 1e+06)
Data.use &lt;- pilot_libsize

#Only include genes that are common between the two data sets
gene.id &lt;- rownames(Data.use)
gene.ref &lt;- rownames(cell.line)
common &lt;- intersect(gene.id, gene.ref)
print(paste0(&quot;There are &quot;, length(common), &quot; genes in common between the two sets&quot;))</code></pre>
<pre><code>[1] &quot;There are 4204 genes in common between the two sets&quot;</code></pre>
<pre class="r"><code>logxx &lt;- apply(Data.use[common, ], 2, function(x) {
        log(x + 0.1)
    })
selected.cell.line &lt;- apply(cell.line[common, ], 2, function(x) {
        x - mean(x)
    })</code></pre>
</div>
<div id="combine-the-two-datasets-into-one-matrix" class="section level2">
<h2>Combine the two datasets into one matrix</h2>
<pre class="r"><code>combined &lt;- cbind(logxx, selected.cell.line)</code></pre>
</div>
<div id="clustering-pca" class="section level2">
<h2>Clustering (PCA)</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(combined), scale = T)

#Make PCA plots 

autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA%20cell%20atlas-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-cell-atlas-1">
Past versions of “PCA cell atlas-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-cell-atlas-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA cell atlas-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="clustering-corr-heatmap" class="section level2">
<h2>Clustering (corr heatmap)</h2>
<pre class="r"><code>#Corr heatmap
cors &lt;- cor(combined, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)
heatmap.2(cors, scale=&quot;none&quot;, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;,
          cexCol = 0.1 + 1/log10(15), cexRow = 0.1 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-1">
Past versions of “corr heatmap-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="outside-data-analysis" class="section level1">
<h1>Outside data analysis</h1>
<p>Compare pilot bulk data with bulk data collected from iPSCs, MSCs, chondrocytes from OA patients, and heart tissue (note: I have only used a Single read from each sample and trimmed to the first 50 bases to ensure that there are not mapping biases between samples). Here, I try both upperquartile and RLE normalization methods</p>
<pre class="r"><code># load in counts
outside_data_counts &lt;- read.table(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/bulkRNAseq_outsideData/out/counts/counts.txt&quot;, header = T)

# assign row.names
row.names(outside_data_counts) &lt;- outside_data_counts$Geneid

# exclude extra data columns
outside_data_counts &lt;- outside_data_counts[, -c(1:6)]

names(outside_data_counts) &lt;- c(&quot;MSC-Chond-d3_3&quot;, &quot;Heart_2&quot;, &quot;Chond_TCP_P4_2&quot;, &quot;Chond_SCM_P4_3&quot;, &quot;MSC-Chond-d21_3&quot;, 
                                &quot;Chond_SCM_P4_1&quot;, &quot;MSC-Chond-d1_3&quot;, &quot;MSC-Chond-d0_3&quot;, &quot;MSC-Chond-d0_1&quot;, &quot;MSC-Chond-d0_2&quot;, 
                                &quot;Heart_3&quot;, &quot;Lung_3&quot;, &quot;Lung_2&quot;, &quot;Chond_TCP_P1_1&quot;, &quot;MSC-Chond-d7_2&quot;,
                                &quot;iPSC-MSC9_1&quot;, &quot;Liver_2&quot;, &quot;MSC-Chond-d3_1&quot;, &quot;Kidney_4&quot;, &quot;OA_dmg_3&quot;, 
                                &quot;Kidney_1&quot;, &quot;bmMSC_1.1&quot;, &quot;iPSC-MSC5_1&quot;, &quot;OA_intact_1&quot;, &quot;bmMSC_2.1&quot;,
                                &quot;Liver_1&quot;, &quot;iPSC-MSC5_2&quot;, &quot;Chond_SCM_P1_2&quot;, &quot;Chond_TCP_P4_3&quot;, &quot;MSC-Chond-d14_2&quot;, 
                                &quot;MSC-Chond-d3_2&quot;, &quot;iPSC_2&quot;, &quot;Chond_TCP_P1_2&quot;, &quot;MSC-Chond-d14_3&quot;, &quot;Lung_4&quot;, 
                                &quot;OA_intact_2&quot;, &quot;MSC-Chond-d21_1&quot;, &quot;OA_dmg_2&quot;, &quot;adMSC&quot;, &quot;Heart_1&quot;, 
                                &quot;Kidney_2&quot;, &quot;OA_intact_3&quot;, &quot;bmMSC_2.2&quot;, &quot;Chond_SCM_P4_2&quot;, &quot;Chond_SCM_P1_1&quot;, 
                                &quot;MSC-Chond-d1_2&quot;, &quot;MSC-Chond-d1_1&quot;, &quot;Lung_1&quot;, &quot;MSC-Chond-d7_1&quot;, &quot;iPSC-MSC9_2&quot;, 
                                &quot;Kidney_3&quot;, &quot;OA_dmg_1&quot;, &quot;Chond_TCP_P4_1&quot;, &quot;bmMSC_1.3&quot;, &quot;Chond_TCP_P1_3&quot;, 
                                &quot;Liver_3&quot;, &quot;MSC-Chond-d14_1&quot;, &quot;Heart_4&quot;, &quot;MSC-Chond-d7_3&quot;, &quot;Chond_SCM_P1_3&quot;, 
                                &quot;iPSC_1&quot;, &quot;MSC-Chond-d21_2&quot;, &quot;bmMSC_1.2&quot;, &quot;Liver_4&quot;)

#remove MSC-chond samples and OA samples
MSC_chond_samples &lt;- c(1,5,7:10,15,18,30:31,34,37,46:47,49,57,59,62)
OA_samples &lt;- c(20,24,36,38,42,52)
Heart1 &lt;- c(40) #seems to cluster with liver rather than hearts!
outside_data_counts&lt;- outside_data_counts[,-c(MSC_chond_samples, OA_samples, Heart1)]

#Combine all bulk data together
merged_raw_data &lt;- merge(raw_counts, outside_data_counts, by = &quot;row.names&quot;)
row.names(merged_raw_data) &lt;- merged_raw_data$Row.names
merged_raw_data &lt;- merged_raw_data[,-1]

#Remove rows with all 0s
dim(merged_raw_data)</code></pre>
<pre><code>[1] 58395    56</code></pre>
<pre class="r"><code>merged_raw_data &lt;- merged_raw_data[apply(merged_raw_data, 1, function(x) !all(x==0)),]
dim(merged_raw_data)</code></pre>
<pre><code>[1] 48326    56</code></pre>
<div id="raw-counts-analysis" class="section level2">
<h2>Raw counts analysis</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(merged_raw_data)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/raw%20analysis-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-raw-analysis-1">
Past versions of “raw analysis-1.png”
</button>
</p>
<div id="fig-no-spaces-raw-analysis-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Corr heatmap
cors &lt;- cor(merged_raw_data, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/raw%20analysis-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-raw-analysis-2">
Past versions of “raw analysis-2.png”
</button>
</p>
<div id="fig-no-spaces-raw-analysis-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/raw analysis-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="normalize-raw-merged-counts-upperquartile-normalization" class="section level2">
<h2>Normalize raw merged counts: upperquartile Normalization</h2>
<pre class="r"><code>library(edgeR)</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre><code>
Attaching package: &#39;limma&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:BiocGenerics&#39;:

    plotMA</code></pre>
<pre class="r"><code>merged_raw_data &lt;- DGEList(merged_raw_data, group = colnames(merged_raw_data))
upperquartile &lt;- calcNormFactors(merged_raw_data, method = &quot;upperquartile&quot;)
upperquartile &lt;- cpm(upperquartile, log=TRUE, normalized.lib.sizes = T)
head(upperquartile)</code></pre>
<pre><code>                18855_3_S 19160_3_S 18856_3_U 18856_1_U 18855_2_S 18856_2_S
ENSG00000000003  7.745718  7.720196  8.248460  8.047990  8.002046  7.053276
ENSG00000000005 -7.014883 -7.014883 -3.232383 -7.014883 -7.014883 -7.014883
ENSG00000000419  5.955103  6.126537  6.487033  6.150936  6.054346  5.991924
ENSG00000000457  3.623812  3.859213  3.935626  3.692211  3.843036  3.493665
ENSG00000000460  2.878997  3.318102  2.682878  2.184790  1.899024  2.033059
ENSG00000000938 -7.014883 -7.014883 -3.232383 -7.014883 -7.014883 -7.014883
                19160_3_U 18855_2_U 19160_2_S 18855_1_S 18856_1_S 19160_1_S
ENSG00000000003  7.563215  7.985971  7.732056  7.598055  7.728914  7.528845
ENSG00000000005 -2.652647 -7.014883 -7.014883 -7.014883 -7.014883 -3.784416
ENSG00000000419  5.890341  6.214071  6.158854  5.783000  6.354302  5.948049
ENSG00000000457  3.876116  4.032681  4.197298  3.768116  3.733370  4.007941
ENSG00000000460  2.977300  2.432330  3.169075  2.287714  2.078063  3.043090
ENSG00000000938 -3.584151 -7.014883 -7.014883 -7.014883 -7.014883 -7.014883
                19160_2_U 18855_1_U 18856_3_S 18856_2_U 18855_3_U    Heart_2
ENSG00000000003  7.149735  7.603615  7.870063  7.510621  7.556563  2.3074461
ENSG00000000005 -3.584151 -7.014883 -7.014883 -7.014883 -7.014883 -4.9236530
ENSG00000000419  5.777484  5.989864  6.317209  6.180232  5.904986  4.0487071
ENSG00000000457  4.010845  3.694158  4.007661  3.838466  3.642659  1.6826448
ENSG00000000460  3.183562  2.467287  2.364052  2.255062  2.184790  0.2236282
ENSG00000000938 -7.014883 -7.014883 -7.014883 -7.014883 -7.014883  4.1194830
                Chond_TCP_P4_2 Chond_SCM_P4_3 Chond_SCM_P4_1    Heart_3
ENSG00000000003       4.589423       4.950879       4.630244  2.3220701
ENSG00000000005      -7.014883      -7.014883      -7.014883 -7.0148827
ENSG00000000419       5.324304       5.262998       5.415256  3.8231262
ENSG00000000457       3.327037       3.474892       3.536362  2.2550622
ENSG00000000460       4.014148       3.346661       3.060524 -0.1882283
ENSG00000000938      -2.192034      -7.014883      -3.469813  2.3861035
                    Lung_3     Lung_2 Chond_TCP_P1_1 iPSC-MSC9_1    Liver_2
ENSG00000000003  3.6777638  2.7530982       5.994437    6.852635  5.2042852
ENSG00000000005 -7.0148827 -7.0148827      -7.014883   -7.014883 -7.0148827
ENSG00000000419  3.2246151  3.6405087       4.648028    5.699948  4.1354554
ENSG00000000457  2.2626630  2.5572739       3.223058    3.058143  2.6646164
ENSG00000000460  0.2965276  0.4430476       1.367237    4.324247  0.6016225
ENSG00000000938  5.1938164  5.3428661      -2.652647   -7.014883  4.3489931
                   Kidney_4  Kidney_1 bmMSC_1.1 iPSC-MSC5_1 bmMSC_2.1   Liver_1
ENSG00000000003  4.71196623 5.1136895  4.873329    7.286480  5.371637  5.395797
ENSG00000000005 -2.23545933 2.9408847 -7.014883   -1.206723 -7.014883 -7.014883
ENSG00000000419  3.65549423 3.4932619  6.080878    5.225933  5.538718  4.601295
ENSG00000000457  2.15195436 2.3631962  3.470917    4.536844  3.281093  3.737759
ENSG00000000460 -0.06426703 0.3301265  5.198463    4.314237  1.863475  3.287169
ENSG00000000938  1.91988371 2.1468688 -3.746994   -4.791846 -7.014883  3.207616
                iPSC-MSC5_2 Chond_SCM_P1_2 Chond_TCP_P4_3    iPSC_2
ENSG00000000003    7.813439       5.747326       4.011914  7.113152
ENSG00000000005   -2.652647      -7.014883      -7.014883  2.428334
ENSG00000000419    5.821570       4.750071       5.187378  6.536297
ENSG00000000457    4.635642       3.462570       3.293097  2.941484
ENSG00000000460    4.886845       1.623365       4.109978  5.195630
ENSG00000000938   -7.014883      -3.289232      -2.975098 -2.670820
                Chond_TCP_P1_2     Lung_4     adMSC   Kidney_2 bmMSC_2.2
ENSG00000000003       5.522091  2.8584727  5.093553  5.4358849  5.149481
ENSG00000000005      -7.014883 -5.4881076 -4.184103 -1.3609755 -7.014883
ENSG00000000419       4.496567  3.4696917  5.807024  3.4585672  5.530499
ENSG00000000457       2.987203  2.4088090  3.942393  2.2532401  3.367840
ENSG00000000460       1.761488  0.5047284  3.090535 -0.1022582  1.680583
ENSG00000000938      -2.172473  6.5237887 -7.014883  1.8808869 -7.014883
                Chond_SCM_P4_2 Chond_SCM_P1_1     Lung_1 iPSC-MSC9_2   Kidney_3
ENSG00000000003       5.107675       6.202676  3.4044964    7.196099  5.3449586
ENSG00000000005      -7.014883      -4.258948 -3.5250672   -3.021776 -1.8020774
ENSG00000000419       5.485734       4.965593  3.6912886    6.415516  3.7749933
ENSG00000000457       3.490253       3.629751  2.6794785    3.562730  2.8418681
ENSG00000000460       2.991093       1.663182  0.5295116    4.615296  0.5354383
ENSG00000000938      -7.014883      -2.823896  5.9513743   -7.014883  0.9014373
                Chond_TCP_P4_1 bmMSC_1.3 Chond_TCP_P1_3    Liver_3    Heart_4
ENSG00000000003       4.298378  4.976868       5.489404  6.1722549  2.8078368
ENSG00000000005      -7.014883 -7.014883      -7.014883 -7.0148827 -4.0944187
ENSG00000000419       5.114519  5.983293       4.695834  4.3470709  4.5356326
ENSG00000000457       3.353351  3.435429       3.546694  3.1921171  1.7046339
ENSG00000000460       4.117089  5.265450       2.553469  0.8187228 -0.1984721
ENSG00000000938      -3.110284 -4.351228      -2.344817  4.0267154  3.3389348
                Chond_SCM_P1_3    iPSC_1 bmMSC_1.2   Liver_4
ENSG00000000003       5.813665  7.138845  4.999209  4.875205
ENSG00000000005      -3.986572  1.911128 -7.014883 -7.014883
ENSG00000000419       4.949264  6.504385  5.968548  3.935404
ENSG00000000457       3.425733  3.037842  3.409452  3.160499
ENSG00000000460       1.893876  5.180907  5.273113  1.243197
ENSG00000000938      -1.557051 -2.455579 -3.493563  5.046170</code></pre>
<pre class="r"><code># Look at density plots
plotDensities(upperquartile, legend=&quot;topright&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/upperquartile%20normalization-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-upperquartile-normalization-1">
Past versions of “upperquartile normalization-1.png”
</button>
</p>
<div id="fig-no-spaces-upperquartile-normalization-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/upperquartile normalization-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2upperquartile-normalized-and-unfiltered-samples" class="section level2">
<h2>Correlation heatmap of log2upperquartile normalized and unfiltered samples</h2>
<pre class="r"><code>cors &lt;- cor(upperquartile, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20unfiltered%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-unfiltered-upperquartile-1">
Past versions of “corr heatmap unfiltered upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-unfiltered-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="boxplots-of-upperquartile-across-samples" class="section level2">
<h2>Boxplots of upperquartile across samples</h2>
<pre class="r"><code>library(reshape2)
meltupperquartile &lt;- melt(upperquartile)
names(meltupperquartile) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;upperquartile&quot;)
p &lt;- ggplot(meltupperquartile, aes(factor(sampleID), upperquartile)) 
p + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-upperquartile-1">
Past versions of “boxplot upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="filtering-for-lowly-expressed-genes-avg-log2upperquartile-2-in-at-least-10-samples" class="section level2">
<h2>Filtering for lowly expressed genes (avg log2upperquartile &gt; 2 in at least 10 samples)</h2>
<pre class="r"><code>cutoff &lt;- 2

keep &lt;- rowSums( upperquartile &gt; cutoff ) &gt;= 10

counts_upperquartile &lt;- merged_raw_data[keep,]
filtered_upperquartile &lt;- upperquartile[keep,]
dim(filtered_upperquartile)</code></pre>
<pre><code>[1] 13183    56</code></pre>
</div>
<div id="boxplots-of-normalizedfiltered-counts-across-samples" class="section level2">
<h2>Boxplots of normalized+filtered counts across samples</h2>
<pre class="r"><code>melt_filt_upperquartile &lt;- melt(filtered_upperquartile)
names(melt_filt_upperquartile) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;log2upperquartile&quot;)
p1 &lt;- ggplot(melt_filt_upperquartile, aes(factor(sampleID), log2upperquartile)) 
p1 + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-upperquartile-1">
Past versions of “boxplot filt upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotDensities(filtered_upperquartile, legend = F)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20upperquartile-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-upperquartile-2">
Past versions of “boxplot filt upperquartile-2.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-upperquartile-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt upperquartile-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2upperquartile-normalized-and-filtered-samples" class="section level2">
<h2>Correlation heatmap of log2upperquartile normalized and filtered samples</h2>
<pre class="r"><code>cors &lt;- cor(filtered_upperquartile, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20filtered%20upperquartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-filtered-upperquartile-1">
Past versions of “corr heatmap filtered upperquartile-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-filtered-upperquartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered upperquartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-log2upperquartile-normalized-and-filtered-samples" class="section level2">
<h2>PCA of log2upperquartile normalized and filtered samples</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(filtered_upperquartile)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA%20quartile-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-quartile-1">
Past versions of “PCA quartile-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-quartile-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA quartile-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="normalize-raw-merged-counts-rle-normalization" class="section level2">
<h2>Normalize raw merged counts: RLE Normalization</h2>
<pre class="r"><code>library(edgeR)
merged_raw_data &lt;- DGEList(merged_raw_data, group = colnames(merged_raw_data))
RLE &lt;- calcNormFactors(merged_raw_data, method = &quot;RLE&quot;)
RLE &lt;- cpm(RLE, log=TRUE, normalized.lib.sizes = T)
head(RLE)</code></pre>
<pre><code>                18855_3_S 19160_3_S 18856_3_U 18856_1_U 18855_2_S 18856_2_S
ENSG00000000003  7.176483  7.217581  7.462904  7.364979  7.415473  6.415842
ENSG00000000005 -6.652110 -6.652110 -3.895758 -6.652110 -6.652110 -6.652110
ENSG00000000419  5.385983  5.624010  5.701584  5.468047  5.467888  5.354581
ENSG00000000457  3.055350  3.357186  3.150912  3.010073  3.257146  2.857137
ENSG00000000460  2.311090  2.816363  1.899390  1.504343  1.315193  1.398265
ENSG00000000938 -6.652110 -6.652110 -3.895758 -6.652110 -6.652110 -6.652110
                19160_3_U 18855_2_U 19160_2_S 18855_1_S 18856_1_S 19160_1_S
ENSG00000000003  7.155222  7.392982  7.214869  7.003373  7.013402  7.109200
ENSG00000000005 -3.011973 -6.652110 -6.652110 -6.652110 -6.652110 -4.097463
ENSG00000000419  5.482440  5.621181  5.641754  5.188455  5.638884  5.528491
ENSG00000000457  3.468618  3.440291  3.680577  3.174150  3.018732  3.588752
ENSG00000000460  2.570266  1.841240  2.652886  1.695128  1.365427  2.624375
ENSG00000000938 -3.900666 -6.652110 -6.652110 -6.652110 -6.652110 -6.652110
                19160_2_U 18855_1_U 18856_3_S 18856_2_U 18855_3_U   Heart_2
ENSG00000000003  6.728647  7.154374  7.042938  6.888038  7.102340  3.024170
ENSG00000000005 -3.911822 -6.652110 -6.652110 -6.652110 -6.652110 -4.282140
ENSG00000000419  5.356486  5.540713  5.490203  5.557740  5.450859  4.765776
ENSG00000000457  3.590200  3.245525  3.181363  3.216587  3.189070  2.399103
ENSG00000000460  2.763303  2.019526  1.539640  1.634707  1.732389  0.938761
ENSG00000000938 -6.652110 -6.652110 -6.652110 -6.652110 -6.652110  4.836559
                Chond_TCP_P4_2 Chond_SCM_P4_3 Chond_SCM_P4_1    Heart_3
ENSG00000000003       4.244943       4.644172       4.406479  3.2348678
ENSG00000000005      -6.652110      -6.652110      -6.652110 -6.6521099
ENSG00000000419       4.979707       4.956250       5.191397  4.7363818
ENSG00000000457       2.982968       3.168565       3.312854  3.1678261
ENSG00000000460       3.669812       3.040389       2.837204  0.7212386
ENSG00000000938      -2.504894      -6.652110      -3.633071  3.2989319
                   Lung_3    Lung_2 Chond_TCP_P1_1 iPSC-MSC9_1   Liver_2
ENSG00000000003  4.875456  3.832192       6.033029    6.671840  6.103014
ENSG00000000005 -6.652110 -6.652110      -6.652110   -6.652110 -6.652110
ENSG00000000419  4.422166  4.719900       4.686688    5.519206  5.034081
ENSG00000000457  3.459716  3.636273       3.261907    2.877916  3.562893
ENSG00000000460  1.490605  1.519572       1.406874    4.143662  1.498162
ENSG00000000938  6.391758  6.422500      -2.596529   -6.652110  5.247646
                 Kidney_4 Kidney_1 bmMSC_1.1 iPSC-MSC5_1 bmMSC_2.1   Liver_1
ENSG00000000003  5.863901 6.275367  4.243392   7.5120926  4.911080  6.142660
ENSG00000000005 -1.105653 4.102082 -6.652110  -0.9785515 -6.652110 -6.652110
ENSG00000000419  4.807235 4.654655  5.450727   5.4515681  5.078138  5.348113
ENSG00000000457  3.303010 3.524089  2.841599   4.7624970  2.821211  4.484489
ENSG00000000460  1.082924 1.488163  4.568450   4.5398988  1.405068  4.033826
ENSG00000000938  3.070755 3.307612 -4.236106  -4.5357436 -6.652110  3.954259
                iPSC-MSC5_2 Chond_SCM_P1_2 Chond_TCP_P4_3    iPSC_2
ENSG00000000003    7.688291       5.666103       3.651132  7.153509
ENSG00000000005   -2.749858      -6.652110      -6.652110  2.469189
ENSG00000000419    5.696481       4.668922       4.826345  6.576664
ENSG00000000457    4.510654       3.381636       2.932606  2.982184
ENSG00000000460    4.761828       1.543371       3.749167  5.236043
ENSG00000000938   -6.652110      -3.331749      -3.280264 -2.612802
                Chond_TCP_P1_2    Lung_4     adMSC   Kidney_2 bmMSC_2.2
ENSG00000000003       5.546100  4.061620  4.525168  6.5203138  4.699615
ENSG00000000005      -6.652110 -4.524384 -4.579642 -0.2877688 -6.652110
ENSG00000000419       4.520642  4.673073  5.238523  4.5426979  5.080577
ENSG00000000457       3.011520  3.611707  3.374369  3.3368490  2.918553
ENSG00000000460       1.786304  1.705078  2.523040  0.9775505  1.233110
ENSG00000000938      -2.135277  7.727562 -6.652110  2.9642244 -6.652110
                Chond_SCM_P4_2 Chond_SCM_P1_1    Lung_1 iPSC-MSC9_2   Kidney_3
ENSG00000000003       4.761844       6.022843  4.747246    7.029789  6.4966797
ENSG00000000005      -6.652110      -4.344471 -2.246567   -3.148528 -0.6667228
ENSG00000000419       5.139855       4.785854  5.034132    6.249230  4.9264867
ENSG00000000457       3.144846       3.450261  4.021889    3.396804  3.9930492
ENSG00000000460       2.645946       1.484888  1.868964    4.449153  1.6840290
ENSG00000000938      -6.652110      -2.968170  7.294555   -6.652110  2.0507560
                Chond_TCP_P4_1 bmMSC_1.3 Chond_TCP_P1_3   Liver_3    Heart_4
ENSG00000000003       3.947265  4.378423       5.361974  7.078356  3.7297542
ENSG00000000005      -6.652110 -6.652110      -6.652110 -6.652110 -3.2345995
ENSG00000000419       4.763249  5.384679       4.568478  5.253048  5.4579077
ENSG00000000457       3.002574  2.837625       3.419550  4.097883  2.6259627
ENSG00000000460       3.766025  4.666945       2.426708  1.722886  0.7198359
ENSG00000000938      -3.401330 -4.748978      -2.449591  4.932650  4.2610100
                Chond_SCM_P1_3    iPSC_1 bmMSC_1.2   Liver_4
ENSG00000000003       5.624788  7.208825  4.455966  6.020615
ENSG00000000005      -4.095430  1.981757 -6.652110 -6.652110
ENSG00000000419       4.760463  6.574374  5.425156  5.080668
ENSG00000000457       3.237247  3.108109  2.866821  4.305545
ENSG00000000460       1.706304  5.250937  4.729817  2.386792
ENSG00000000938      -1.730810 -2.371913 -3.931244  6.191597</code></pre>
<pre class="r"><code># Look at density plots
plotDensities(RLE, legend=&quot;topright&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/RLE%20normalization-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-RLE-normalization-1">
Past versions of “RLE normalization-1.png”
</button>
</p>
<div id="fig-no-spaces-RLE-normalization-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/RLE normalization-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2rle-normalized-and-unfiltered-samples" class="section level2">
<h2>Correlation heatmap of log2RLE normalized and unfiltered samples</h2>
<pre class="r"><code>cors &lt;- cor(RLE, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20unfiltered%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-unfiltered-RLE-1">
Past versions of “corr heatmap unfiltered RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-unfiltered-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap unfiltered RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="boxplots-of-rle-across-samples" class="section level2">
<h2>Boxplots of RLE across samples</h2>
<pre class="r"><code>library(reshape2)
meltRLE &lt;- melt(RLE)
names(meltRLE) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;RLE&quot;)
p &lt;- ggplot(meltRLE, aes(factor(sampleID), RLE)) 
p + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-RLE-1">
Past versions of “boxplot RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="filtering-for-lowly-expressed-genes-avg-log2rle-2-in-at-least-10-samples" class="section level2">
<h2>Filtering for lowly expressed genes (avg log2RLE &gt; 2 in at least 10 samples)</h2>
<pre class="r"><code>cutoff &lt;- 2

keep &lt;- rowSums( RLE &gt; cutoff ) &gt;= 10

counts_RLE &lt;- merged_raw_data[keep,]
filtered_RLE &lt;- RLE[keep,]
dim(filtered_RLE)</code></pre>
<pre><code>[1] 13400    56</code></pre>
</div>
<div id="boxplots-of-normalizedfiltered-counts-across-samples-1" class="section level2">
<h2>Boxplots of normalized+filtered counts across samples</h2>
<pre class="r"><code>melt_filt_RLE &lt;- melt(filtered_RLE)
names(melt_filt_RLE) &lt;- c(&quot;gene&quot;, &quot;sampleID&quot;, &quot;log2RLE&quot;)
p1 &lt;- ggplot(melt_filt_RLE, aes(factor(sampleID), log2RLE)) 
p1 + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-RLE-1">
Past versions of “boxplot filt RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotDensities(filtered_RLE, legend = F)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/boxplot%20filt%20RLE-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-boxplot-filt-RLE-2">
Past versions of “boxplot filt RLE-2.png”
</button>
</p>
<div id="fig-no-spaces-boxplot-filt-RLE-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/boxplot filt RLE-2.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-heatmap-of-log2rle-normalized-and-filtered-samples" class="section level2">
<h2>Correlation heatmap of log2RLE normalized and filtered samples</h2>
<pre class="r"><code>cors &lt;- cor(filtered_RLE, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(labels))], RowSideColors=pal[as.integer(as.factor(labels))], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/corr%20heatmap%20filtered%20RLE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-filtered-RLE-1">
Past versions of “corr heatmap filtered RLE-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-filtered-RLE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/corr heatmap filtered RLE-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-log2rle-normalized-and-filtered-samples" class="section level2">
<h2>PCA of log2RLE normalized and filtered samples</h2>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(data.matrix(filtered_RLE)), scale = T)

#Make PCA plots 
autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-1">
Past versions of PCA-1.png
</button>
</p>
<div id="fig-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/docs/figure/Bulk_cellComposition_analysis.Rmd/PCA-1.png" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="grade-of-membership-analysis" class="section level1">
<h1>Grade of Membership analysis</h1>
<pre class="r"><code>merged_raw_data_counts &lt;- merged_raw_data$counts
Combined_counts_celltypes &lt;- sapply(strsplit(as.character(counts_upperquartile$samples$group),&quot;_&quot;), `[`, 1)

if (file.exists(&quot;data/combined_gom_bulk_4to8.rda&quot;)){
     Combined_GoM &lt;- get(load(file=&quot;data/combined_gom_bulk_4to8.rda&quot;))
} else {
     Combined_GoM &lt;- CountClust::FitGoM(t(merged_raw_data_counts), K=c(4:8), tol = 0.1, control=list(tmax=100))
     save(Combined_GoM, file=&quot;data/combined_gom_bulk_4to8.rda&quot;)
}</code></pre>
<pre class="r"><code>plot_structure_plot &lt;- function(omega, labels){
     annotation &lt;- data.frame(
          sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
          tissue_label = labels
          )
     rownames(omega) &lt;- annotation$sample_id
     StructureGGplot(omega = omega,
                     annotation = annotation,
                     order_sample = TRUE,
                     axis_tick = list(axis_ticks_length = .1,
                                      axis_ticks_lwd_y = .1,
                                      axis_ticks_lwd_x = .1,
                                      axis_label_size = 7,
                                      axis_label_face = &quot;bold&quot;)
                     )
     }


make_topic_plots &lt;- function(omega, labels){
  omega &lt;- as.data.frame(omega)
  omega &lt;- cbind(omega,labels)
  for (k in 1:(ncol(omega)-1)) {
    base_plot &lt;- ggplot(data = omega, mapping = aes(x = labels, y = omega[,k])) + labs(y = paste0(&quot;Topic &quot;, k, &quot; value&quot;), x = paste0(&quot;Cell Type&quot;)) + theme(axis.text.x = element_text(angle = -20, hjust = 0, vjust = 0))
    plot(base_plot + geom_violin(width = 1.5, fill = &quot;skyblue&quot;,color = &quot;skyblue&quot;) +
      geom_boxplot(width = 0.1,outlier.shape = NA, alpha = 0.2))
   }
}

annotate_clusters &lt;- function(theta_mat, countmatrix){
  top_features &lt;- ExtractTopFeatures(theta_mat,
                                     top_features=100,
                                     method=&quot;poisson&quot;,
                                     options=&quot;min&quot;)
  gene_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) rownames(countmatrix)[top_features[[1]][x,]]))
  scores_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) top_features[[2]][x,]))
  return(list(gene_vector, scores_vector))
}


labels &lt;- sapply(strsplit(colnames(merged_raw_data_counts),&quot;_&quot;), `[`, 1)

# k = 4
plot_structure_plot(omega = Combined_GoM$clust_4$omega, labels = labels)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-1">
Past versions of “plotting GoM-1.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-1.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_4$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-2">
Past versions of “plotting GoM-2.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-2.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-3">
Past versions of “plotting GoM-3.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-3.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-4">
Past versions of “plotting GoM-4.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-4.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-5">
Past versions of “plotting GoM-5.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-5.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_4$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 5
plot_structure_plot(omega = Combined_GoM$clust_5$omega, labels = labels)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-6">
Past versions of “plotting GoM-6.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-6.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_5$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-7">
Past versions of “plotting GoM-7.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-7.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-8">
Past versions of “plotting GoM-8.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-8.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-9">
Past versions of “plotting GoM-9.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-9.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-10">
Past versions of “plotting GoM-10.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-10.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-11">
Past versions of “plotting GoM-11.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-11.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_5$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 6
plot_structure_plot(omega = Combined_GoM$clust_6$omega, labels = labels)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-12">
Past versions of “plotting GoM-12.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-12.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_6$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-13.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-13">
Past versions of “plotting GoM-13.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-13.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-14.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-14">
Past versions of “plotting GoM-14.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-14.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-15.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-15">
Past versions of “plotting GoM-15.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-15.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-16.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-16">
Past versions of “plotting GoM-16.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-16" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-16.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-17.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-17">
Past versions of “plotting GoM-17.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-17" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-17.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-18.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-18">
Past versions of “plotting GoM-18.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-18" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-18.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_6$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 7
plot_structure_plot(omega = Combined_GoM$clust_7$omega, labels = labels)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-19.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-19">
Past versions of “plotting GoM-19.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-19" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-19.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_7$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-20.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-20">
Past versions of “plotting GoM-20.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-20.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-21.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-21">
Past versions of “plotting GoM-21.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-21.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-22.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-22">
Past versions of “plotting GoM-22.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-22.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-23.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-23">
Past versions of “plotting GoM-23.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-23" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-23.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-24.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-24">
Past versions of “plotting GoM-24.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-24" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-24.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-25.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-25">
Past versions of “plotting GoM-25.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-25" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-25.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-26.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-26">
Past versions of “plotting GoM-26.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-26" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-26.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_7$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]

# k = 8
plot_structure_plot(omega = Combined_GoM$clust_8$omega, labels = labels)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-27.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-27">
Past versions of “plotting GoM-27.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-27" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-27.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_topic_plots(omega = Combined_GoM$clust_8$omega, labels = labels)</code></pre>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-28.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-28">
Past versions of “plotting GoM-28.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-28" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-28.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-29.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-29">
Past versions of “plotting GoM-29.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-29" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-29.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-30.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-30">
Past versions of “plotting GoM-30.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-30" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-30.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-31.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-31">
Past versions of “plotting GoM-31.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-31" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-31.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-32.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-32">
Past versions of “plotting GoM-32.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-32" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-32.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-33.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-33">
Past versions of “plotting GoM-33.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-33" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-33.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-34.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-34">
Past versions of “plotting GoM-34.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-34" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-34.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>Warning: position_dodge requires non-overlapping x intervals</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/plotting%20GoM-35.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plotting-GoM-35">
Past versions of “plotting GoM-35.png”
</button>
</p>
<div id="fig-no-spaces-plotting-GoM-35" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Bulk_cellComposition_analysis.Rmd/plotting GoM-35.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># genes &lt;- annotate_clusters(theta_mat = Combined_GoM$clust_8$theta, countmatrix = merged_raw_data_counts)
# matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)
# genes[[2]]</code></pre>
</div>
<div id="check-expression-of-chondrocyte-markers-across-data" class="section level1">
<h1>Check expression of chondrocyte markers across data</h1>
<pre class="r"><code>#Inputs: count matrix, sample information data frame, and gene of interest in the ENSG format
get_gene_long &lt;- function(counts, info, gene){
     gene_counts &lt;- as.data.frame(counts[gene,])
     names(gene_counts) &lt;- &quot;norm_expr&quot;
     gene_counts_info &lt;- cbind(gene_counts, info)
     return(gene_counts_info)
}

boxplot_gene_sample_type &lt;- function(counts_, info_, gene_, name, y_axis = &quot;log2 Normalized Expression&quot;){
     df &lt;- get_gene_long(counts = counts_, info = info_, gene = gene_)
     
     ggplot(df, aes(x = labels, y = norm_expr, color = labels)) +
          geom_boxplot() +
          labs(title=paste0(&quot;Expression of &quot;, as.character(name), &quot; by sample type&quot;), x= &quot;Sample Type&quot;, y = y_axis) +
          theme(legend.position = &quot;none&quot;)
}



labels &lt;- sapply(strsplit(colnames(merged_raw_data),&quot;_&quot;), `[`, 1)

#SOX9
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000125398&quot;, &quot;SOX9&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-1">
Past versions of “chond markers-1.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-1.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#COL11A1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000060718&quot;, &quot;COL11A1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-2">
Past versions of “chond markers-2.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-2.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#SOX5
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000134532&quot;, &quot;SOX5&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-3">
Past versions of “chond markers-3.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-3.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#SOX6
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000110693&quot;, &quot;SOX6&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-4">
Past versions of “chond markers-4.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-4.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#TGFB3
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000119699&quot;, &quot;TGFB3&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-5">
Past versions of “chond markers-5.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-5.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#COL2A1
boxplot_gene_sample_type(merged_raw_data$counts, labels, &quot;ENSG00000139219&quot;, &quot;COL2A1&quot;, y_axis = &quot;Raw counts&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-6">
Past versions of “chond markers-6.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/7fe1d0c84437c41023cfa69b6c56ce46235b4f65/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">7fe1d0c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-6.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#ACAN
boxplot_gene_sample_type(merged_raw_data$counts, labels, &quot;ENSG00000157766&quot;, &quot;ACAN&quot;, y_axis = &quot;Raw counts&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-7">
Past versions of “chond markers-7.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ce14abda48e6fcb102f2e629bcd136c8067354d8/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">ce14abd</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-7.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#other chond markers
#NKX3-2
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000109705&quot;, &quot;NKX3-2&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-8">
Past versions of “chond markers-8.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-8.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#IGFBP5
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000115461&quot;, &quot;IGFBP5&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/chond%20markers-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-chond-markers-9">
Past versions of “chond markers-9.png”
</button>
</p>
<div id="fig-no-spaces-chond-markers-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/chond markers-9.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-define-some-known-clusters-prior-to-fitting-topic-model" class="section level1">
<h1>Semi-supervised topic modeling (define some known clusters prior to fitting topic model)</h1>
<pre class="r"><code>library(&quot;classtpx&quot;)</code></pre>
<pre><code>
Attaching package: &#39;classtpx&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:maptpx&#39;:

    expit, logit, rdir, stm_tfidf</code></pre>
<pre class="r"><code># labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- which(labels==&quot;Chond&quot;)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond))
                )

# Perform topic modeling k=6
if (file.exists(&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;)){
     Topic_clus_6 &lt;- get(load(file=&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_6 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=6,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_6, file=&quot;data/merged_topic_fit_6_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_6$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-1">
Past versions of supervised-1.png
</button>
</p>
<div id="fig-supervised-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-1.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;)){
     Topic_clus_7 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_7 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_7$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-2">
Past versions of supervised-2.png
</button>
</p>
<div id="fig-supervised-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-2.png" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;)){
     Topic_clus_8 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;))
} else {
Topic_clus_8 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix.rda&quot;)
}

omega &lt;- Topic_clus_8$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-supervised-3">
Past versions of supervised-3.png
</button>
</p>
<div id="fig-supervised-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/68e5e33dc85bf6340efc50200e47b0766711c93c/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised-3.png" target="_blank">68e5e33</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-topic-modeling-genes-analysis" class="section level1">
<h1>Semi-supervised Topic Modeling Genes Analysis</h1>
<pre class="r"><code>genes &lt;- annotate_clusters(theta_mat = Topic_clus_7$theta, countmatrix = merged_raw_data_counts)
matrix(gene_anno$external_gene_name[match(genes[[1]], gene_anno$ensembl_gene_id)], ncol = 100)</code></pre>
<pre><code>     [,1]     [,2]      [,3]      [,4]     [,5]     [,6]       [,7]    
[1,] &quot;COL3A1&quot; &quot;COL1A2&quot;  &quot;SPARC&quot;   &quot;COL1A1&quot; &quot;FN1&quot;    &quot;IGFBP3&quot;   &quot;CHI3L1&quot;
[2,] &quot;LIN28A&quot; &quot;KRT8&quot;    &quot;CLDN6&quot;   &quot;CTSV&quot;   &quot;L1TD1&quot;  &quot;DNMT3B&quot;   &quot;PRTG&quot;  
[3,] &quot;MMP1&quot;   &quot;TERT&quot;    &quot;DYNC1H1&quot; &quot;TFPI2&quot;  &quot;ACTB&quot;   &quot;THBS1&quot;    &quot;DCBLD2&quot;
[4,] &quot;SFTPB&quot;  &quot;SFTPC&quot;   &quot;SFTPA1&quot;  &quot;SFTPA2&quot; &quot;AGER&quot;   &quot;A2M&quot;      &quot;CLDN18&quot;
[5,] &quot;UMOD&quot;   &quot;SLC12A1&quot; &quot;GPX3&quot;    &quot;AQP2&quot;   &quot;CDH16&quot;  &quot;SPP1&quot;     &quot;ATP1B1&quot;
[6,] &quot;ALB&quot;    &quot;HP&quot;      &quot;FGA&quot;     &quot;FGB&quot;    &quot;FGG&quot;    &quot;SERPINA1&quot; &quot;SAA1&quot;  
[7,] &quot;MYH7&quot;   &quot;DES&quot;     &quot;MYL2&quot;    &quot;TNNT2&quot;  &quot;MT-CO1&quot; &quot;MB&quot;       &quot;MT-ND1&quot;
     [,8]      [,9]      [,10]      [,11]    [,12]    [,13]    [,14]    
[1,] &quot;COL12A1&quot; &quot;COL5A1&quot;  &quot;ITGA11&quot;   &quot;SULF1&quot;  &quot;ACAN&quot;   &quot;POSTN&quot;  &quot;COL11A1&quot;
[2,] &quot;LRRN1&quot;   &quot;GABRP&quot;   &quot;DPPA4&quot;    &quot;MDK&quot;    &quot;ESRG&quot;   &quot;TRIML2&quot; &quot;IGDCC3&quot; 
[3,] &quot;RGS4&quot;    &quot;MKI67&quot;   &quot;SERPINE1&quot; &quot;CLDN11&quot; &quot;LDHA&quot;   &quot;SFRP1&quot;  &quot;PRKDC&quot;  
[4,] &quot;CEACAM6&quot; &quot;B2M&quot;     &quot;HLA-E&quot;    &quot;FOS&quot;    &quot;JUN&quot;    &quot;ZFP36&quot;  &quot;DEPP1&quot;  
[5,] &quot;SLC12A3&quot; &quot;HSD11B2&quot; &quot;PAX8&quot;     &quot;CA12&quot;   &quot;CALB1&quot;  &quot;KCNJ1&quot;  &quot;CLCNKB&quot; 
[6,] &quot;ORM1&quot;    &quot;CRP&quot;     &quot;TF&quot;       &quot;APOC3&quot;  &quot;AMBP&quot;   &quot;CYP2E1&quot; &quot;APOB&quot;   
[7,] &quot;CKM&quot;     &quot;MT-RNR2&quot; &quot;MYL3&quot;     &quot;MT-ND4&quot; &quot;MYBPC3&quot; &quot;TTN&quot;    &quot;TNNI3&quot;  
     [,15]      [,16]        [,17]     [,18]    [,19]     [,20]     [,21]    
[1,] &quot;COL4A1&quot;   &quot;PENK&quot;       &quot;BGN&quot;     &quot;TIMP1&quot;  &quot;CEMIP&quot;   &quot;COL4A2&quot;  &quot;MXRA5&quot;  
[2,] &quot;POU5F1&quot;   &quot;AC064802.1&quot; &quot;VRTN&quot;    &quot;HAND1&quot;  &quot;TRIM71&quot;  &quot;RPS19&quot;   &quot;CDH3&quot;   
[3,] &quot;PKM&quot;      &quot;ACTG1&quot;      &quot;NOTCH2&quot;  &quot;MYH9&quot;   &quot;PLAT&quot;    &quot;ZFHX4&quot;   &quot;FGF5&quot;   
[4,] &quot;EPAS1&quot;    &quot;ATF3&quot;       &quot;TMEM100&quot; &quot;FOSB&quot;   &quot;PECAM1&quot;  &quot;DUSP1&quot;   &quot;SCGB1A1&quot;
[5,] &quot;PDZK1IP1&quot; &quot;KCNJ16&quot;     &quot;SLC5A12&quot; &quot;MIOX&quot;   &quot;TMEM213&quot; &quot;SLC13A3&quot; &quot;CUBN&quot;   
[6,] &quot;APOA2&quot;    &quot;APOH&quot;       &quot;APOA1&quot;   &quot;SAA2&quot;   &quot;RBP4&quot;    &quot;HPX&quot;     &quot;TTR&quot;    
[7,] &quot;MT-CYB&quot;   &quot;TCAP&quot;       &quot;HSPB7&quot;   &quot;MT-ND2&quot; &quot;XIRP1&quot;   &quot;MT-ND5&quot;  &quot;LDB3&quot;   
     [,22]   [,23]     [,24]     [,25]    [,26]     [,27]     [,28]     
[1,] &quot;LOX&quot;   &quot;LRP1&quot;    &quot;COL6A2&quot;  &quot;COL5A2&quot; &quot;LUM&quot;     &quot;MRC2&quot;    &quot;MMP2&quot;    
[2,] &quot;RPS18&quot; &quot;KRT19&quot;   &quot;SLC16A1&quot; &quot;RPS24&quot;  &quot;SLC38A1&quot; &quot;ZSCAN10&quot; &quot;LIN28B&quot;  
[3,] &quot;SAMD9&quot; &quot;CORO1C&quot;  &quot;CDCP1&quot;   &quot;HIF1A&quot;  &quot;MICAL2&quot;  &quot;COTL1&quot;   &quot;AXL&quot;     
[4,] &quot;EGR1&quot;  &quot;SLC34A2&quot; &quot;HLA-B&quot;   &quot;DNAJB1&quot; &quot;HLA-DRA&quot; &quot;CD74&quot;    &quot;PPP1R15A&quot;
[5,] &quot;DPEP1&quot; &quot;SLC34A1&quot; &quot;SLC22A8&quot; &quot;DEFB1&quot;  &quot;SULT1C2&quot; &quot;ATP1A1&quot;  &quot;TMEM52B&quot; 
[6,] &quot;FGL1&quot;  &quot;CYP3A4&quot;  &quot;C3&quot;      &quot;AGXT&quot;   &quot;AHSG&quot;    &quot;MAT1A&quot;   &quot;CPS1&quot;    
[7,] &quot;ACTC1&quot; &quot;MT-RNR1&quot; &quot;MT-CO2&quot;  &quot;TNNC1&quot;  &quot;MT-ATP6&quot; &quot;FABP3&quot;   &quot;ACTN2&quot;   
     [,29]       [,30]      [,31]      [,32]       [,33]    [,34]    [,35]    
[1,] &quot;COMP&quot;      &quot;SPON2&quot;    &quot;EFEMP1&quot;   &quot;FKBP10&quot;    &quot;TGFBI&quot;  &quot;COL6A1&quot; &quot;LTBP2&quot;  
[2,] &quot;LINC00678&quot; &quot;SEPHS1&quot;   &quot;RPL5&quot;     &quot;MARCKSL1&quot;  &quot;RPL23&quot;  &quot;RPL3&quot;   &quot;SLC2A3&quot; 
[3,] &quot;EIF4G1&quot;    &quot;FGF2&quot;     &quot;LAMC1&quot;    &quot;KIAA1549L&quot; &quot;TLN1&quot;   &quot;ACTN4&quot;  &quot;IL1B&quot;   
[4,] &quot;UBC&quot;       &quot;RGCC&quot;     &quot;EMP2&quot;     &quot;PGC&quot;       &quot;DUOX1&quot;  &quot;CX3CL1&quot; &quot;ICAM1&quot;  
[5,] &quot;RHCG&quot;      &quot;SLC22A6&quot;  &quot;SLC22A12&quot; &quot;SLC3A1&quot;    &quot;CLCNKA&quot; &quot;PAX2&quot;   &quot;SLC47A2&quot;
[6,] &quot;VTN&quot;       &quot;SERPINC1&quot; &quot;GC&quot;       &quot;TAT&quot;       &quot;ITIH1&quot;  &quot;ADH4&quot;   &quot;ORM2&quot;   
[7,] &quot;CRYAB&quot;     &quot;EEF1A2&quot;   &quot;NRAP&quot;     &quot;CKMT2&quot;     &quot;MT-CO3&quot; &quot;MYL7&quot;   &quot;PLN&quot;    
     [,36]    [,37]     [,38]      [,39]      [,40]     [,41]    [,42]   
[1,] &quot;CILP&quot;   &quot;FSTL1&quot;   &quot;NDUFA4L2&quot; &quot;SERPINE2&quot; &quot;TNC&quot;     &quot;MATN2&quot;  &quot;MFGE8&quot; 
[2,] &quot;PTPRZ1&quot; &quot;STK26&quot;   &quot;RPS3&quot;     &quot;RPL8&quot;     &quot;CTNNB1&quot;  &quot;RPS6&quot;   &quot;FREM2&quot; 
[3,] &quot;MET&quot;    &quot;ALDH1A3&quot; &quot;CCND1&quot;    &quot;PSMD2&quot;    &quot;SLC7A1&quot;  &quot;NEK7&quot;   &quot;DNER&quot;  
[4,] &quot;TIMP3&quot;  &quot;RHOB&quot;    &quot;GLUL&quot;     &quot;HLA-DRB1&quot; &quot;SLC11A1&quot; &quot;HSPA1B&quot; &quot;AQP4&quot;  
[5,] &quot;SMIM24&quot; &quot;TFCP2L1&quot; &quot;EMX2OS&quot;   &quot;CYS1&quot;     &quot;SLC6A19&quot; &quot;TMEM72&quot; &quot;FXYD2&quot; 
[6,] &quot;ITIH3&quot;  &quot;APOC1&quot;   &quot;C9&quot;       &quot;CYP2C8&quot;   &quot;APCS&quot;    &quot;F2&quot;     &quot;CYP2C9&quot;
[7,] &quot;HRC&quot;    &quot;SLC25A4&quot; &quot;COX6A2&quot;   &quot;MYOZ2&quot;    &quot;MT-ND4L&quot; &quot;TPM1&quot;   &quot;MYH7B&quot; 
     [,43]     [,44]    [,45]        [,46]    [,47]    [,48]      [,49]   
[1,] &quot;HSPG2&quot;   &quot;PDPN&quot;   &quot;AC089987.1&quot; &quot;HTRA1&quot;  &quot;CDH11&quot;  &quot;PCDH10&quot;   &quot;CYP1B1&quot;
[2,] &quot;DSP&quot;     &quot;RPS5&quot;   &quot;HNRNPC&quot;     &quot;RPS14&quot;  &quot;NKX1-2&quot; &quot;PRDX1&quot;    &quot;RPL37&quot; 
[3,] &quot;TUBB6&quot;   &quot;DUSP4&quot;  &quot;ITGA2&quot;      &quot;FLNA&quot;   &quot;SMURF2&quot; &quot;ITGA5&quot;    &quot;RAB3B&quot; 
[4,] &quot;CALCRL&quot;  &quot;AOC3&quot;   &quot;SCEL&quot;       &quot;CYP4B1&quot; &quot;CD93&quot;   &quot;HLA-DPA1&quot; &quot;SOCS3&quot; 
[5,] &quot;NPHS1&quot;   &quot;PTGER3&quot; &quot;SLC4A9&quot;     &quot;NAT8&quot;   &quot;KIF12&quot;  &quot;MAL&quot;      &quot;NPHS2&quot; 
[6,] &quot;SDS&quot;     &quot;HAMP&quot;   &quot;ITIH2&quot;      &quot;ARG1&quot;   &quot;TFR2&quot;   &quot;IGFBP1&quot;   &quot;HRG&quot;   
[7,] &quot;MT-ATP8&quot; &quot;PDK4&quot;   &quot;COX7A1&quot;     &quot;CMYA5&quot;  &quot;APOD&quot;   &quot;MYOM2&quot;    &quot;CSRP3&quot; 
     [,50]      [,51]      [,52]     [,53]     [,54]      [,55]     [,56]    
[1,] &quot;CCDC80&quot;   &quot;DCN&quot;      &quot;IGFL2&quot;   &quot;THY1&quot;    &quot;FBLN5&quot;    &quot;EDIL3&quot;   &quot;FNDC1&quot;  
[2,] &quot;RPL31&quot;    &quot;KIF1A&quot;    &quot;NPPB&quot;    &quot;RPL10A&quot;  &quot;RPS12&quot;    &quot;DSC2&quot;    &quot;TUBB2B&quot; 
[3,] &quot;HUWE1&quot;    &quot;EGFR&quot;     &quot;SLC20A1&quot; &quot;UHMK1&quot;   &quot;PRPF8&quot;    &quot;NAV1&quot;    &quot;FLNB&quot;   
[4,] &quot;MTCO1P53&quot; &quot;HOPX&quot;     &quot;NKX2-1&quot;  &quot;JUNB&quot;    &quot;THBD&quot;     &quot;LAMP3&quot;   &quot;MCL1&quot;   
[5,] &quot;SLC4A1&quot;   &quot;ATP6V0A4&quot; &quot;AOC1&quot;    &quot;EGF&quot;     &quot;SLC22A2&quot;  &quot;PTH1R&quot;   &quot;MMP7&quot;   
[6,] &quot;LRG1&quot;     &quot;RDH16&quot;    &quot;BAAT&quot;    &quot;LBP&quot;     &quot;SERPINF2&quot; &quot;SLC22A1&quot; &quot;SULT2A1&quot;
[7,] &quot;MT-ND3&quot;   &quot;TECRL&quot;    &quot;MT-TL1&quot;  &quot;ADPRHL1&quot; &quot;SRL&quot;      &quot;ACTA1&quot;   &quot;RYR2&quot;   
     [,57]       [,58]     [,59]    [,60]       [,61]      [,62]    
[1,] &quot;P4HA2&quot;     &quot;CERCAM&quot;  &quot;SEMA3C&quot; &quot;CHPF&quot;      &quot;ADAM12&quot;   &quot;CDKN2B&quot; 
[2,] &quot;SINHCAF&quot;   &quot;PLS3&quot;    &quot;RPL13A&quot; &quot;LAPTM4B&quot;   &quot;ARID3A&quot;   &quot;EIF3E&quot;  
[3,] &quot;SACS&quot;      &quot;ADAM19&quot;  &quot;TRIO&quot;   &quot;PTPN14&quot;    &quot;HHIP&quot;     &quot;ANLN&quot;   
[4,] &quot;IGKC&quot;      &quot;FENDRR&quot;  &quot;HLA-C&quot;  &quot;LIMCH1&quot;    &quot;NFKBIA&quot;   &quot;SFTPD&quot;  
[5,] &quot;LINC01187&quot; &quot;SLC26A7&quot; &quot;CLDN2&quot;  &quot;LINC00982&quot; &quot;SLC22A11&quot; &quot;SLC36A2&quot;
[6,] &quot;AOX1&quot;      &quot;UGT2B4&quot;  &quot;CYP2A6&quot; &quot;CPB2&quot;      &quot;AGT&quot;      &quot;ASGR1&quot;  
[7,] &quot;MYH6&quot;      &quot;LMOD2&quot;   &quot;TRIM54&quot; &quot;CASQ2&quot;     &quot;POPDC2&quot;   &quot;NEBL&quot;   
     [,63]        [,64]    [,65]     [,66]   [,67]     [,68]     [,69]    
[1,] &quot;VIM&quot;        &quot;ISLR&quot;   &quot;ADAMTS5&quot; &quot;MXRA8&quot; &quot;PCOLCE&quot;  &quot;CTGF&quot;    &quot;SCRG1&quot;  
[2,] &quot;AP000459.2&quot; &quot;HDAC2&quot;  &quot;SNHG14&quot;  &quot;RPS21&quot; &quot;RPS20&quot;   &quot;FAM84B&quot;  &quot;VASH2&quot;  
[3,] &quot;GAPDH&quot;      &quot;PTX3&quot;   &quot;FOSL1&quot;   &quot;FRMD6&quot; &quot;CRIM1&quot;   &quot;ACTR2&quot;   &quot;ZBTB38&quot; 
[4,] &quot;TNS1&quot;       &quot;SAT1&quot;   &quot;CCL18&quot;   &quot;LAMA3&quot; &quot;VIPR1&quot;   &quot;MTCO1P2&quot; &quot;CD55&quot;   
[5,] &quot;TSPAN1&quot;     &quot;POU3F3&quot; &quot;LRP2&quot;    &quot;PROM2&quot; &quot;RAP1GAP&quot; &quot;CLTRN&quot;   &quot;XPNPEP2&quot;
[6,] &quot;C3P1&quot;       &quot;ASGR2&quot;  &quot;ATF5&quot;    &quot;F9&quot;    &quot;SLC13A5&quot; &quot;C4BPA&quot;   &quot;AZGP1&quot;  
[7,] &quot;KCNIP2&quot;     &quot;TRDN&quot;   &quot;SLC4A3&quot;  &quot;HHATL&quot; &quot;NMRK2&quot;   &quot;RPL3L&quot;   &quot;SMPX&quot;   
     [,70]     [,71]     [,72]    [,73]      [,74]      [,75]    [,76]      
[1,] &quot;CALD1&quot;   &quot;PLOD1&quot;   &quot;COL8A2&quot; &quot;ANTXR1&quot;   &quot;MEST&quot;     &quot;ACTA2&quot;  &quot;CD99&quot;     
[2,] &quot;RPL41&quot;   &quot;NASP&quot;    &quot;GSTP1&quot;  &quot;HMGB2&quot;    &quot;EEF1A1&quot;   &quot;MSH2&quot;   &quot;CRABP2&quot;   
[3,] &quot;PRRC2C&quot;  &quot;ASAP1&quot;   &quot;ANXA2&quot;  &quot;MARCH4&quot;   &quot;HRAS&quot;     &quot;PLAU&quot;   &quot;TPX2&quot;     
[4,] &quot;KLF6&quot;    &quot;TSC22D3&quot; &quot;BTG2&quot;   &quot;MSLN&quot;     &quot;CTSE&quot;     &quot;LAPTM5&quot; &quot;FMO2&quot;     
[5,] &quot;ADGRG1&quot;  &quot;SCIN&quot;    &quot;EMX1&quot;   &quot;ATP6V0D2&quot; &quot;KCNJ15&quot;   &quot;PKHD1&quot;  &quot;CYFIP2&quot;   
[6,] &quot;PLA2G2A&quot; &quot;C8A&quot;     &quot;MT2A&quot;   &quot;CES1&quot;     &quot;ACSL1&quot;    &quot;HAL&quot;    &quot;SERPINA10&quot;
[7,] &quot;SMYD1&quot;   &quot;SH3RF2&quot;  &quot;UNC45B&quot; &quot;MYOM1&quot;    &quot;MTATP6P1&quot; &quot;MT-ND6&quot; &quot;NKX2-5&quot;   
     [,77]    [,78]     [,79]    [,80]     [,81]      [,82]     [,83]   
[1,] &quot;FBN1&quot;   &quot;SRPX2&quot;   &quot;PITX1&quot;  &quot;ADGRA2&quot;  &quot;P4HB&quot;     &quot;ACTG2&quot;   &quot;PTGER2&quot;
[2,] &quot;RPL19&quot;  &quot;SALL4&quot;   &quot;SLC2A1&quot; &quot;AMOT&quot;    &quot;CCNG1&quot;    &quot;CBX2&quot;    &quot;MEX3A&quot; 
[3,] &quot;NAV3&quot;   &quot;TRIP12&quot;  &quot;IFIT2&quot;  &quot;VGLL3&quot;   &quot;ADAM9&quot;    &quot;FOXG1&quot;   &quot;CEP170&quot;
[4,] &quot;IGHA1&quot;  &quot;MS4A15&quot;  &quot;RNASE1&quot; &quot;MGP&quot;     &quot;MTCO1P40&quot; &quot;LYZ&quot;     &quot;CDH5&quot;  
[5,] &quot;OGDHL&quot;  &quot;EMX2&quot;    &quot;USH1C&quot;  &quot;PIGR&quot;    &quot;KCNJ10&quot;   &quot;SLC13A1&quot; &quot;AQP6&quot;  
[6,] &quot;CYP1A2&quot; &quot;CREB3L3&quot; &quot;CFHR1&quot;  &quot;SLC27A5&quot; &quot;CLU&quot;      &quot;CYP8B1&quot;  &quot;C4BPB&quot; 
[7,] &quot;KLHL31&quot; &quot;NDRG4&quot;   &quot;ANKRD1&quot; &quot;APOBEC2&quot; &quot;ATP2A2&quot;   &quot;RBM24&quot;   &quot;XIRP2&quot; 
     [,84]     [,85]         [,86]    [,87]     [,88]      [,89]    [,90]   
[1,] &quot;P4HA3&quot;   &quot;COL2A1&quot;      &quot;FAP&quot;    &quot;SPTBN2&quot;  &quot;LOXL1&quot;    &quot;CD63&quot;   &quot;SSC5D&quot; 
[2,] &quot;CASP3&quot;   &quot;RPS11&quot;       &quot;RPL27&quot;  &quot;AP1S2&quot;   &quot;RCN2&quot;     &quot;TERF1&quot;  &quot;RPL11&quot; 
[3,] &quot;EIF2AK2&quot; &quot;NCEH1&quot;       &quot;ELK3&quot;   &quot;POLR2A&quot;  &quot;AHNAK&quot;    &quot;ESM1&quot;   &quot;DOCK10&quot;
[4,] &quot;HSPA1A&quot;  &quot;WIF1&quot;        &quot;IGHM&quot;   &quot;ZFP36L2&quot; &quot;HLA-DPB1&quot; &quot;PRX&quot;    &quot;C1QB&quot;  
[5,] &quot;ITGB8&quot;   &quot;ADAMTS9-AS1&quot; &quot;BICDL1&quot; &quot;CASR&quot;    &quot;SLC4A4&quot;   &quot;ITM2B&quot;  &quot;AIF1L&quot; 
[6,] &quot;CYP2B6&quot;  &quot;C6&quot;          &quot;HMGCS2&quot; &quot;C8B&quot;     &quot;C1S&quot;      &quot;C5&quot;     &quot;PON1&quot;  
[7,] &quot;HSPB3&quot;   &quot;TRDN-AS1&quot;    &quot;ATP1A2&quot; &quot;PERM1&quot;   &quot;ITGA7&quot;    &quot;DHRS7C&quot; &quot;MYZAP&quot; 
     [,91]       [,92]        [,93]    [,94]      [,95]      [,96]     [,97]   
[1,] &quot;PTTG1IP&quot;   &quot;TIMP2&quot;      &quot;CPA4&quot;   &quot;ITGB5&quot;    &quot;P3H1&quot;     &quot;MMP14&quot;   &quot;AEBP1&quot; 
[2,] &quot;USP44&quot;     &quot;BAMBI&quot;      &quot;XRCC5&quot;  &quot;TKT&quot;      &quot;SLC39A10&quot; &quot;DSG2&quot;    &quot;RPS4X&quot; 
[3,] &quot;ARHGAP29&quot;  &quot;OASL&quot;       &quot;RAI14&quot;  &quot;SHCBP1&quot;   &quot;MYO10&quot;    &quot;PAPPA&quot;   &quot;AFAP1&quot; 
[4,] &quot;PTGS2&quot;     &quot;CHRDL1&quot;     &quot;S100A4&quot; &quot;ABCA3&quot;    &quot;DDX5&quot;     &quot;MSR1&quot;    &quot;SLC6A4&quot;
[5,] &quot;FMO1&quot;      &quot;AC015910.1&quot; &quot;DUSP9&quot;  &quot;BBOX1&quot;    &quot;ENPEP&quot;    &quot;PTPRO&quot;   &quot;VAV3&quot;  
[6,] &quot;FABP1&quot;     &quot;PLG&quot;        &quot;C1R&quot;    &quot;CP&quot;       &quot;SLC38A4&quot;  &quot;CFHR3&quot;   &quot;ADH1A&quot; 
[7,] &quot;LINC00881&quot; &quot;TRIM63&quot;     &quot;MT-TM&quot;  &quot;MTND2P28&quot; &quot;FBXO40&quot;   &quot;SYNPO2L&quot; &quot;MYLK3&quot; 
     [,98]    [,99]    [,100]   
[1,] &quot;WNT2&quot;   &quot;PRRX1&quot;  &quot;SHISAL1&quot;
[2,] &quot;HAPLN1&quot; &quot;RPS8&quot;   &quot;RPL35&quot;  
[3,] &quot;SH3BP4&quot; &quot;INHBA&quot;  &quot;CLTC&quot;   
[4,] &quot;SUSD2&quot;  &quot;RTKN2&quot;  &quot;SFTA3&quot;  
[5,] &quot;TINAG&quot;  &quot;CLDN16&quot; &quot;CCDC198&quot;
[6,] &quot;EPHX1&quot;  &quot;MBL2&quot;   &quot;HPD&quot;    
[7,] &quot;ASB2&quot;   &quot;PTP4A3&quot; &quot;PTGDS&quot;  </code></pre>
<pre class="r"><code>genes[[2]]</code></pre>
<pre><code>            [,1]        [,2]        [,3]        [,4]        [,5]        [,6]
[1,] 0.069713411 0.038211474 0.032937472 0.031671786 0.018433702 0.015700254
[2,] 0.009177174 0.005294249 0.004612971 0.004389133 0.004187272 0.002323472
[3,] 0.012801712 0.005242384 0.002449337 0.001646465 0.001569825 0.001448502
[4,] 0.023698163 0.021790473 0.011178605 0.009558630 0.006038909 0.004427344
[5,] 0.021883016 0.006557708 0.005891701 0.005093153 0.004360327 0.003047806
[6,] 0.148447224 0.048067834 0.046203563 0.034286559 0.030617302 0.027152265
[7,] 0.034896410 0.022063635 0.018775006 0.016761211 0.015114078 0.014692060
            [,7]        [,8]        [,9]       [,10]        [,11]        [,12]
[1,] 0.011339840 0.006894910 0.005791885 0.004206575 0.0036900881 0.0036197124
[2,] 0.001972678 0.001928346 0.001620891 0.001401284 0.0012380891 0.0011621645
[3,] 0.001288872 0.001249977 0.001122633 0.001115923 0.0008661454 0.0008168399
[4,] 0.002505981 0.002497642 0.002453896 0.002149606 0.0020257592 0.0019851321
[5,] 0.002577579 0.002332925 0.002050978 0.002005850 0.0017429954 0.0016743428
[6,] 0.023595683 0.021776356 0.016311338 0.015524831 0.0147500230 0.0135540367
[7,] 0.012584546 0.010016972 0.008893278 0.008544886 0.0085012993 0.0077212829
            [,13]       [,14]        [,15]        [,16]        [,17]
[1,] 0.0034144016 0.003175118 0.0030603820 0.0022559356 0.0022454362
[2,] 0.0011177694 0.001099179 0.0010872644 0.0010715626 0.0010264297
[3,] 0.0007808734 0.000740572 0.0007201052 0.0006821768 0.0006496101
[4,] 0.0018354751 0.001802604 0.0017757531 0.0014311637 0.0014256483
[5,] 0.0015020619 0.001439442 0.0013533705 0.0012848601 0.0012323480
[6,] 0.0130611662 0.012862881 0.0123584332 0.0108816774 0.0101526704
[7,] 0.0066650500 0.006554995 0.0064964757 0.0051895367 0.0049243229
            [,18]        [,19]        [,20]        [,21]        [,22]
[1,] 0.0021087057 0.0020524494 0.0019078029 0.0016609335 0.0015001775
[2,] 0.0010117446 0.0010102768 0.0008917705 0.0008860663 0.0008641577
[3,] 0.0006224498 0.0005871834 0.0005704622 0.0005223891 0.0005139016
[4,] 0.0013933258 0.0013900686 0.0013730385 0.0013228828 0.0012801850
[5,] 0.0012205006 0.0011960443 0.0011615900 0.0011086784 0.0011051556
[6,] 0.0097874233 0.0094514632 0.0094019451 0.0093240607 0.0075225090
[7,] 0.0046526828 0.0037604349 0.0036975664 0.0035045832 0.0034281050
            [,23]        [,24]        [,25]        [,26]        [,27]
[1,] 0.0013556502 0.0012472609 0.0012022021 0.0011830393 0.0011815877
[2,] 0.0008009056 0.0007883469 0.0007176063 0.0007075906 0.0007021197
[3,] 0.0004659865 0.0004627636 0.0004607030 0.0004606655 0.0004604999
[4,] 0.0012686578 0.0012029915 0.0011714354 0.0011360044 0.0011284101
[5,] 0.0010775916 0.0010092985 0.0009686768 0.0009632688 0.0009000655
[6,] 0.0069471215 0.0066219925 0.0065463462 0.0060774472 0.0057293043
[7,] 0.0033389578 0.0031652696 0.0031642534 0.0031565442 0.0027677069
            [,28]        [,29]        [,30]        [,31]        [,32]
[1,] 0.0011165531 0.0011006474 0.0010622825 0.0010523695 0.0009456679
[2,] 0.0006636473 0.0006588961 0.0006554331 0.0006429366 0.0006242472
[3,] 0.0004561391 0.0004489252 0.0004172413 0.0004053238 0.0004042397
[4,] 0.0010343514 0.0009701514 0.0008690974 0.0008101721 0.0007828231
[5,] 0.0008894434 0.0008635821 0.0008293117 0.0007986950 0.0007910918
[6,] 0.0055567122 0.0054902856 0.0053304610 0.0051052367 0.0050064130
[7,] 0.0027616864 0.0021762290 0.0021249211 0.0018927551 0.0018659915
            [,33]        [,34]        [,35]        [,36]        [,37]
[1,] 0.0008753123 0.0008597538 0.0008487463 0.0008280522 0.0007549572
[2,] 0.0005961593 0.0005913374 0.0005653215 0.0005559381 0.0005350089
[3,] 0.0003864693 0.0003619283 0.0003603799 0.0003507765 0.0003506068
[4,] 0.0007618811 0.0007494897 0.0006956023 0.0006806127 0.0006752140
[5,] 0.0007387927 0.0006916185 0.0006856925 0.0006665444 0.0006661484
[6,] 0.0048166293 0.0045481162 0.0042053509 0.0041592106 0.0040066004
[7,] 0.0017473386 0.0016727517 0.0016028688 0.0015670885 0.0015463182
            [,38]        [,39]        [,40]        [,41]        [,42]
[1,] 0.0007442724 0.0007379651 0.0007355034 0.0007269730 0.0007192042
[2,] 0.0005159227 0.0005147261 0.0005053842 0.0004992195 0.0004930988
[3,] 0.0003449601 0.0003254838 0.0003142367 0.0003102818 0.0003076091
[4,] 0.0006688015 0.0006637070 0.0006592509 0.0006526643 0.0006290464
[5,] 0.0006650458 0.0006504149 0.0006367450 0.0006364200 0.0006275101
[6,] 0.0039390475 0.0039193584 0.0037708621 0.0036793347 0.0031213249
[7,] 0.0014656962 0.0013518941 0.0013299418 0.0011144301 0.0010790902
            [,43]        [,44]        [,45]        [,46]        [,47]
[1,] 0.0007097017 0.0006477561 0.0006095124 0.0005450105 0.0005258729
[2,] 0.0004865231 0.0004777867 0.0004682761 0.0004572844 0.0004463287
[3,] 0.0003009907 0.0002996948 0.0002925200 0.0002917879 0.0002916229
[4,] 0.0006233153 0.0006211177 0.0006060241 0.0005886619 0.0005785410
[5,] 0.0006191067 0.0006171086 0.0006112515 0.0005798483 0.0005765911
[6,] 0.0030865977 0.0026982294 0.0026561911 0.0024171809 0.0023536922
[7,] 0.0010628604 0.0010253823 0.0009969585 0.0009498408 0.0009447167
            [,48]        [,49]        [,50]        [,51]        [,52]
[1,] 0.0005151098 0.0005124243 0.0005024416 0.0004899473 0.0004783175
[2,] 0.0004435016 0.0004414384 0.0004389996 0.0004306038 0.0004140753
[3,] 0.0002879835 0.0002864195 0.0002856813 0.0002670194 0.0002501955
[4,] 0.0005717940 0.0005414527 0.0005339166 0.0005048775 0.0004963301
[5,] 0.0005633005 0.0005600040 0.0005427514 0.0005319866 0.0005089659
[6,] 0.0022288441 0.0021795133 0.0021535709 0.0020642359 0.0019502216
[7,] 0.0009366916 0.0009194068 0.0009057109 0.0008275400 0.0007959406
            [,53]        [,54]        [,55]        [,56]        [,57]
[1,] 0.0004485633 0.0004445311 0.0004392448 0.0004275195 0.0004204849
[2,] 0.0004097753 0.0004061820 0.0003977774 0.0003855601 0.0003810501
[3,] 0.0002432185 0.0002403277 0.0002337336 0.0002324820 0.0002283282
[4,] 0.0004930956 0.0004798857 0.0004776960 0.0004690544 0.0004643000
[5,] 0.0004707198 0.0004516943 0.0004322972 0.0004184293 0.0003984152
[6,] 0.0019348857 0.0019255806 0.0018413578 0.0018328574 0.0017838950
[7,] 0.0007728881 0.0007585278 0.0007574093 0.0007543255 0.0007339165
            [,58]        [,59]        [,60]        [,61]        [,62]
[1,] 0.0004151371 0.0003992637 0.0003802660 0.0003798322 0.0003353041
[2,] 0.0003767234 0.0003730908 0.0003639893 0.0003633209 0.0003608578
[3,] 0.0002263737 0.0002224719 0.0002215734 0.0002213604 0.0002192209
[4,] 0.0004640261 0.0004580403 0.0004525627 0.0004470451 0.0004369860
[5,] 0.0003939800 0.0003789263 0.0003784977 0.0003672536 0.0003671114
[6,] 0.0017591771 0.0017336122 0.0017326901 0.0017240411 0.0017170435
[7,] 0.0006865357 0.0006798649 0.0006695755 0.0006513104 0.0006330947
            [,63]        [,64]        [,65]        [,66]        [,67]
[1,] 0.0003278933 0.0003266543 0.0003178639 0.0003009795 0.0002960860
[2,] 0.0003579323 0.0003556223 0.0003518969 0.0003494443 0.0003481428
[3,] 0.0002159781 0.0002148534 0.0002126592 0.0002084895 0.0002058100
[4,] 0.0004359895 0.0004352903 0.0004351875 0.0004281397 0.0004261515
[5,] 0.0003598262 0.0003586856 0.0003521275 0.0003503733 0.0003497528
[6,] 0.0016941527 0.0016903191 0.0016024452 0.0015918565 0.0015872628
[7,] 0.0006206428 0.0006046328 0.0005993457 0.0005539770 0.0005443099
            [,68]        [,69]        [,70]        [,71]        [,72]
[1,] 0.0002938375 0.0002927558 0.0002894287 0.0002869840 0.0002813389
[2,] 0.0003461658 0.0003315799 0.0003255897 0.0003248274 0.0003245211
[3,] 0.0002036553 0.0002022503 0.0002001835 0.0001984603 0.0001981455
[4,] 0.0004223428 0.0004156843 0.0004137916 0.0004122321 0.0004090318
[5,] 0.0003446871 0.0003437866 0.0003424529 0.0003323437 0.0003216108
[6,] 0.0015626140 0.0015480549 0.0014684801 0.0014684347 0.0014363263
[7,] 0.0005230442 0.0005117819 0.0005026036 0.0005024660 0.0004645358
            [,73]        [,74]        [,75]        [,76]        [,77]
[1,] 0.0002774910 0.0002734493 0.0002667435 0.0002622799 0.0002524311
[2,] 0.0003168535 0.0003150565 0.0003112721 0.0003094456 0.0003092670
[3,] 0.0001932116 0.0001870540 0.0001842719 0.0001820540 0.0001812711
[4,] 0.0003938060 0.0003910837 0.0003880417 0.0003832493 0.0003764504
[5,] 0.0003212446 0.0003120485 0.0002946743 0.0002849778 0.0002715837
[6,] 0.0014260017 0.0014079982 0.0013805089 0.0013310613 0.0013154306
[7,] 0.0004354753 0.0004311740 0.0004064202 0.0004054469 0.0003889443
            [,78]        [,79]        [,80]        [,81]        [,82]
[1,] 0.0002413255 0.0002326412 0.0002225653 0.0002212995 0.0002074727
[2,] 0.0003087145 0.0003075062 0.0003070520 0.0003044523 0.0003006607
[3,] 0.0001792876 0.0001783666 0.0001765943 0.0001761176 0.0001757241
[4,] 0.0003696264 0.0003696190 0.0003448919 0.0003419880 0.0003277888
[5,] 0.0002679189 0.0002665366 0.0002651089 0.0002615965 0.0002565494
[6,] 0.0012838464 0.0012209629 0.0012033985 0.0011683682 0.0011424909
[7,] 0.0003885023 0.0003792343 0.0003785644 0.0003739041 0.0003698281
            [,83]        [,84]        [,85]        [,86]        [,87]
[1,] 0.0002072950 0.0002022817 0.0001929161 0.0001881849 0.0001869558
[2,] 0.0002972648 0.0002971524 0.0002883777 0.0002872070 0.0002826859
[3,] 0.0001756392 0.0001744796 0.0001739879 0.0001692100 0.0001685677
[4,] 0.0003260276 0.0003221277 0.0003137154 0.0003125536 0.0003089933
[5,] 0.0002554760 0.0002530984 0.0002429079 0.0002426172 0.0002421829
[6,] 0.0011126494 0.0010946810 0.0010875248 0.0010819311 0.0010590555
[7,] 0.0003430547 0.0003411770 0.0003155315 0.0003094391 0.0003076962
            [,88]        [,89]        [,90]        [,91]        [,92]
[1,] 0.0001860524 0.0001832022 0.0001821626 0.0001817027 0.0001809147
[2,] 0.0002795534 0.0002788643 0.0002776843 0.0002752815 0.0002730303
[3,] 0.0001658799 0.0001637776 0.0001625544 0.0001613500 0.0001597848
[4,] 0.0003066466 0.0003058548 0.0003049919 0.0002894214 0.0002878581
[5,] 0.0002413045 0.0002410343 0.0002395200 0.0002380244 0.0002320285
[6,] 0.0010181949 0.0010099912 0.0010047406 0.0009802772 0.0009720860
[7,] 0.0003026312 0.0002888936 0.0002873506 0.0002788392 0.0002782299
            [,93]        [,94]        [,95]        [,96]        [,97]
[1,] 0.0001807100 0.0001738561 0.0001723012 0.0001719198 0.0001663196
[2,] 0.0002711692 0.0002691644 0.0002651189 0.0002635874 0.0002631856
[3,] 0.0001595225 0.0001532319 0.0001530438 0.0001529704 0.0001524310
[4,] 0.0002819307 0.0002682035 0.0002667161 0.0002650547 0.0002627607
[5,] 0.0002293668 0.0002283033 0.0002229724 0.0002182785 0.0002070131
[6,] 0.0009318479 0.0009274390 0.0008589545 0.0008473271 0.0008431582
[7,] 0.0002629582 0.0002611733 0.0002541220 0.0002505643 0.0002435198
            [,98]        [,99]       [,100]
[1,] 0.0001657887 0.0001616257 0.0001611425
[2,] 0.0002575803 0.0002556975 0.0002555558
[3,] 0.0001506129 0.0001490815 0.0001486766
[4,] 0.0002597099 0.0002571737 0.0002555783
[5,] 0.0001999261 0.0001986548 0.0001967817
[6,] 0.0007955332 0.0007910709 0.0007900110
[7,] 0.0002385616 0.0002333254 0.0002318979</code></pre>
<pre class="r"><code>#Gene boxplot for cluster 1
#COL11A1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000060718&quot;, &quot;COL11A1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-1">
Past versions of genes_classtpx-1.png
</button>
</p>
<div id="fig-genes_classtpx-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-1.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 2
#POU5F1
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000204531&quot;, &quot;POU5F1&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-2">
Past versions of genes_classtpx-2.png
</button>
</p>
<div id="fig-genes_classtpx-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-2.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 3
#
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000163631&quot;, &quot;ALB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-3">
Past versions of genes_classtpx-3.png
</button>
</p>
<div id="fig-genes_classtpx-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-3.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 4
#SFTPB
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000168878&quot;, &quot;SFTPB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-4">
Past versions of genes_classtpx-4.png
</button>
</p>
<div id="fig-genes_classtpx-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-4.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 5
#UMOD
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000169344&quot;, &quot;UMOD&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-5">
Past versions of genes_classtpx-5.png
</button>
</p>
<div id="fig-genes_classtpx-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-5.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 6
#TNNT2
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000118194&quot;, &quot;TNNT2&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-6">
Past versions of genes_classtpx-6.png
</button>
</p>
<div id="fig-genes_classtpx-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-6.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Gene boxplot for cluster 7
#ALB
boxplot_gene_sample_type(filtered_RLE, labels, &quot;ENSG00000163631&quot;, &quot;ALB&quot;)</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-genes_classtpx-7">
Past versions of genes_classtpx-7.png
</button>
</p>
<div id="fig-genes_classtpx-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/420df284ae0245bac41830c68d8c9fbc9a311a74/docs/figure/Bulk_cellComposition_analysis.Rmd/genes_classtpx-7.png" target="_blank">420df28</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># #function to plot genes that distinguish topics
# plot_topic_factors &lt;- function(GoM_object){
#      theta &lt;- GoM_object$theta
#           
#      for(topic in 1:ncol(theta)){
#           plot_ratio_factor_by_mean_factor(topic, theta)
#      }
# }
# 
# plot_ratio_factor_by_mean_factor &lt;- function(topic_n, GoM_theta){
#      factors_topic_n &lt;- GoM_theta[,topic_n]
#      mean_factors_minus_topic_n &lt;- row_means(GoM_theta[, -topic_n])
#      ratio_factors_topic_n &lt;- factors_topic_n/mean_factors_minus_topic_n
#      
#      mean_factors &lt;- row_means(GoM_theta)
#      
#      #get matrix of data to be fed into plot
#      data &lt;- as.data.frame(cbind(&quot;factor ratio&quot; = ratio_factors_topic_n, &quot;mean factor&quot; = mean_factors))
#      #identify genes to highlight
#      data &lt;- data %&gt;%
#           mutate(reg = case_when(
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &gt;= 10 ~ &quot;UP&quot;,
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &lt;= -10 ~ &quot;DOWN&quot;,
#                log(data$`mean factor`) &gt;= -20 &amp; log(data$`factor ratio`) &gt; -10 &amp; log(data$`factor ratio`) &lt; 10~ &quot;no_change&quot;,
#                log(data$`mean factor`) &lt; -20 ~ &quot;no_change&quot;
#                ))%&gt;%
#           mutate(reg = factor(reg, levels = c(&quot;UP&quot;, &quot;no_change&quot;,&quot;DOWN&quot;)))
#      
#      data_subset= data[which(log(data$`mean factor`) &gt;= -20),]
#      data_subset= data[which(log(data$`factor ratio`) &lt;= -10 | log(data$`factor ratio`) &gt;= 10) ,]
#      
#      #plot
#      ggplot(data, aes(x = log(`mean factor`), y = log(`factor ratio`)), label = rownames(data)) +
#           geom_point(aes(color=reg)) +
#           scale_color_manual(name = &quot;Differential \n regulation&quot;,
#                      values = c(&quot;DOWN&quot; = &quot;#058983&quot;,
#                                 &quot;no_change&quot; = &quot;grey&quot;,
#                                 &quot;UP&quot; = &quot;#DEB132&quot;),
#                      labels = c(&quot;UP&quot;, &quot;No Change&quot;, &quot;DOWN&quot;))</code></pre>
</div>
<div id="semi-supervised-topic-modeling-define-mscs-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE MSCs as a cluster)</h1>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs and bmMSCs
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- which(labels==&quot;Chond&quot;)
index_MSCs &lt;- c(40, 32, 42, 30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )


# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;)){
     Topic_clus_7_allMSC &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;))
} else {
Topic_clus_7_allMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC.rda&quot;)
}

omega &lt;- Topic_clus_7_allMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-1">
Past versions of “supervised with MSC-1.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-1.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;)){
     Topic_clus_8_allMSC &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;))
} else {
Topic_clus_8_allMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-2">
Past versions of “supervised with MSC-2.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-2.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(40)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )




# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;)){
     Topic_clus_7_adMSC &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;))
} else {
Topic_clus_7_adMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC.rda&quot;)
}

omega &lt;- Topic_clus_7_adMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-3">
Past versions of “supervised with MSC-3.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-3.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;)){
     Topic_clus_8_adMSC &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;))
} else {
Topic_clus_8_adMSC &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-supervised-with-MSC-4">
Past versions of “supervised with MSC-4.png”
</button>
</p>
<div id="fig-no-spaces-supervised-with-MSC-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/docs/figure/Bulk_cellComposition_analysis.Rmd/supervised with MSC-4.png" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )




# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;)){
     Topic_clus_7_bmMSC1 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;))
} else {
Topic_clus_7_bmMSC1 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC1$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;)){
     Topic_clus_8_bmMSC1 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;))
} else {
Topic_clus_8_bmMSC1 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(32,42)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )


# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;)){
     Topic_clus_7_bmMSC2 &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;))
} else {
Topic_clus_7_bmMSC2 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2.rda&quot;)
}

omega &lt;- Topic_clus_7_bmMSC2$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;)){
     Topic_clus_8_bmMSC2 &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;))
} else {
Topic_clus_8_bmMSC2 &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20MSC-8.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="semi-supervised-topic-modeling-define-early-chondros-separately-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE early chondros separately as a cluster)</h1>
<pre class="r"><code>#Here I define the chond cluster based on only early passage chond
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- c(25,35,38,44,50,53)
index_MSCs &lt;- c(40, 32, 42, 30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)){
     Topic_clus_7_allMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;))
} else {
Topic_clus_7_allMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 1) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 17126222.14, 14501.38, 112.44, 7.22, 42.96, 8613.22, 15771.47, 139600.22, 1105573.27, 830560.59, 455578.13, 324993.89, 100927.6, 152647.69, 110688.89, 47012.89, 28529.4, 19465.79, 12344.01, 8443.2, 6179.72, 4821.3, 4042.27, 3967.07, 4189.49, 4108.66, 3390.01, 2580.66, 2188.25, 2099.85, 2009.7, 1772.43, 1404.04, 1049.29, 793.13, 622.61, 504.11, 413.9, 339.87, 279.42, 230.54, 192.32, 163.49, 142.23, 126.91, 116, 108.13, 102.23, 97.44, 93.22, 89.28, 85.47, 81.69, 77.92, 74.22, 70.62, 67.17, 63.95, 61.03, 58.47, 56.32, 54.6, 53.28, 52.32, 51.66, 51.25, 51.04, 50.97, 50.97, 50.99, 50.98, 50.87, 50.62, 50.12, 49.37, 48.34, 47.03, 45.47, 43.68, 41.74, 39.75, 37.77, 35.9, 34.14, 32.35, 30.43, 28.51, 26.62, 24.75, 23.13, 21.78, 19.83, 17.87, 15.98, 14.4, 12.46, 10.67, 9.07, 7.71, 6.83, p 48326 iter 1000 diff 6
5.88, 4.87, 4.18, 3.6, 3.12, 2.7, 2.35, 2.05, 1.79, 1.57, 1.38, 1.21, 1.08, 0.96, 0.86, 0.78, 0.71, 0.61, 0.54, 0.47, 0.42, 0.38, 0.34, 0.3, 0.27, 0.24, 0.22, 0.2, 0.18, 0.17, 0.15, 0.14, 0.12, 0.11, 0.1, 0.09, 0.08, 0.07, 0.07, 0.06, 0.06, 0.05, 0.05, 0.05, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_allMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)){
     Topic_clus_8_allMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;))
} else {
Topic_clus_8_allMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(40)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)){
     Topic_clus_7_adMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;))
} else {
Topic_clus_7_adMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 24678473.82, 630168.38, 13466.86, 1390.63, 434.2, 29.86, 8.47, 16.52, 1030.78, 58390.05, 238862.22, 158619.46, 168054.22, 224846.2, 184933.74, 76400.44, 75261.85, 93454.31, 15389.59, 9911.88, 12745.04, 9654.19, 6851.75, 5089.49, 3822.82, 2936.73, 2773.32, 3223.93, 3620.08, 3310.8, 2523.74, 1970.51, 1783.33, 1690.5, 1498.42, 1170.88, 815.79, 553.66, 391.76, 294.76, 232.87, 189.91, 159.84, 139.03, 124.49, 114.28, 106.54, 100.23, 94.79, 89.88, 85.28, 80.89, 76.6, 72.4, 68.33, 64.39, 60.56, 56.91, 53.5, 50.37, 47.6, 45.24, 43.34, 41.92, 40.99, 40.53, 40.44, 40.65, 41.1, 41.72, 42.46, 43.24, 44, 44.7, 45.26, 45.65, 45.78, 45.6, 45.1, 44.26, 43.11, 41.7, 40.06, 38.3, 36.5, 34.75, 33.16, 31.49, 29.8, 28.02, 26.26, 24.53, 22.88, 21.72, 20.07, 18.2, 16.38, 14.74, 13.15, 11.28, p 48326 iter 1000 diff 10
9.68, 8.26, 7.2, 6.51, 5.37, 4.63, 4.03, 3.52, 3.09, 2.73, 2.41, 2.14, 1.9, 1.69, 1.51, 1.36, 1.22, 1.11, 1.02, 0.94, 0.84, 0.74, 0.67, 0.6, 0.54, 0.49, 0.45, 0.41, 0.37, 0.34, 0.31, 0.29, 0.26, 0.24, 0.22, 0.2, 0.18, 0.16, 0.15, 0.14, 0.13, 0.12, 0.11, 0.1, 0.09, 0.09, 0.08, 0.07, 0.07, 0.06, 0.06, 0.05, 0.05, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_adMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)){
     Topic_clus_8_adMSC_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;))
} else {
Topic_clus_8_adMSC_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)){
     Topic_clus_7_bmMSC1_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;))
} else {
Topic_clus_7_bmMSC1_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 16769345.34, 277546.76, 27810.91, 9524.75, 8085.33, 372.16, 120.67, 39.84, 265.33, 43621.82, 7513.89, 51299.26, 248900.93, 695528.71, 1568239.79, 816074.63, 463802.12, 328939.58, 128576.72, 57347.06, 36238.4, 18903.75, 13009.29, 9821.57, 8641.01, 10619.87, 12703.8, 11326.45, 9708.03, 7796.7, 7017.63, 6102.75, 4344.03, 3362.32, 2451, 1843.55, 1406.23, 1180.76, 1086.66, 1036.32, 967.68, 862.68, 738.73, 653.44, 584.7, 496.6, 391.38, 298.33, 226.45, 183.42, 162.82, 158, 160.35, 166.89, 176.58, 187.12, 197.13, 205.27, 210.5, 210.71, 205.53, 194.4, 178.99, 161.08, 143.01, 126.44, 112.23, 100.48, 90.87, 82.93, 76.3, 70.7, 66, 62.1, 58.91, 56.33, 54.23, 52.46, 50.92, 49.47, 48.05, 46.59, 45.07, 43.51, 41.95, 40.4, 38.97, 37.49, 35.96, 34.23, 32.47, 30.68, 28.86, 27.42, 25.73, 23.6, 21.47, 19.42, 17.62, 15.35, p 48326 iter 1000 diff 14
13.39, 11.61, 10.14, 9.19, 7.83, 6.79, 5.96, 5.26, 4.64, 4.12, 3.66, 3.26, 2.91, 2.61, 2.34, 2.1, 1.89, 1.71, 1.55, 1.42, 1.28, 1.14, 1.02, 0.91, 0.82, 0.74, 0.67, 0.6, 0.54, 0.49, 0.44, 0.4, 0.37, 0.34, 0.3, 0.27, 0.25, 0.22, 0.2, 0.18, 0.17, 0.15, 0.14, 0.13, 0.12, 0.11, 0.11, 0.1, 0.09, 0.08, 0.08, 0.07, 0.06, 0.06, 0.06, 0.05, 0.05, 0.05, 0.04, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_bmMSC1_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)){
     Topic_clus_8_bmMSC1_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;))
} else {
Topic_clus_8_bmMSC1_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(32,42)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)){
     Topic_clus_7_bmMSC2_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;))
} else {
Topic_clus_7_bmMSC2_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2_earlyChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 29478045.98, 84300.09, 2980.49, 1425.27, 270.84, 35.4, 63.14, 14771.44, 22691.72, 211273.42, 187464.55, 200774.93, 185516.29, 152530.52, 49421.55, 107155.47, 43948.26, 8948.9, 11948.21, 11517.71, 7972.84, 5887.83, 4403.56, 3320.69, 2747.75, 2954.26, 3478.59, 3574.71, 2935.42, 2202.33, 1848.44, 1744.63, 1612.62, 1345.83, 981.69, 661.84, 450.59, 323.58, 245.39, 193.9, 157.82, 133.36, 116.49, 105.01, 96.85, 90.61, 85.54, 81.21, 77.34, 73.75, 70.3, 66.88, 63.51, 60.19, 56.89, 53.63, 50.51, 47.58, 44.92, 42.6, 40.69, 39.23, 38.25, 37.75, 37.65, 37.88, 38.38, 39.07, 39.9, 40.79, 41.7, 42.56, 43.32, 43.92, 44.3, 44.41, 44.19, 43.63, 42.75, 41.57, 40.13, 38.5, 36.78, 35.07, 33.44, 31.89, 30.3, 28.56, 26.83, 25.11, 23.42, 22.01, 20.75, 18.93, 17.09, 15.33, 13.88, 11.99, 10.31, 8.79, p 48326 iter 1000 diff 8
7.52, 6.75, 5.77, 4.85, 4.19, 3.65, 3.18, 2.79, 2.45, 2.16, 1.91, 1.7, 1.51, 1.35, 1.21, 1.08, 0.98, 0.9, 0.82, 0.72, 0.64, 0.57, 0.52, 0.47, 0.42, 0.38, 0.35, 0.31, 0.29, 0.26, 0.24, 0.22, 0.2, 0.18, 0.16, 0.15, 0.14, 0.12, 0.11, 0.1, 0.1, 0.09, 0.08, 0.08, 0.07, 0.07, 0.06, 0.06, 0.05, 0.05, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_bmMSC2_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)){
     Topic_clus_8_bmMSC2_earlyChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;))
} else {
Topic_clus_8_bmMSC2_earlyChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2_earlyChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_earlyChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2_earlyChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20early-8.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="semi-supervised-topic-modeling-define-late-chondros-separately-as-a-cluster" class="section level1">
<h1>Semi-supervised topic modeling (DEFINE late chondros separately as a cluster)</h1>
<pre class="r"><code>#Here I define the chond cluster based on only early passage chond
# labels for &quot;known&quot; clusters
index_heart &lt;- which(labels==&quot;Heart&quot;)
index_lung &lt;- which(labels==&quot;Lung&quot;)
index_liver &lt;- which(labels==&quot;Liver&quot;)
index_kidney &lt;- which(labels==&quot;Kidney&quot;)
index_iPSC &lt;- which(labels==&quot;iPSC&quot;)
index_chond &lt;- c(19,20,21,36,43,48)
index_MSCs &lt;- c(40, 32, 42, 30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;)){
     Topic_clus_7_allMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;))
} else {
Topic_clus_7_allMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_allMSC_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_allMSC_lateChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 1) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 31531347.13, 11446.07, 105.76, 4.79, 64.14, 14584.72, 14507.54, 181992.71, 1140301.29, 917478.43, 435961.93, 317470.01, 100646.05, 155464.39, 99835.31, 46924.96, 31116.52, 22245.64, 14996.87, 10553.55, 7531.89, 5540.95, 4498.11, 4440.51, 4776.87, 4774.26, 4061.29, 3207.8, 2711.4, 2468.16, 2209.24, 1826.96, 1365.63, 971.58, 707.55, 544.28, 442.33, 375.28, 329.13, 298.02, 276.51, 259.73, 243.88, 227.02, 209.41, 192.42, 177.35, 164.74, 154.28, 145.2, 136.76, 128.42, 119.9, 111.25, 102.74, 94.68, 87.37, 81.03, 75.74, 71.49, 68.2, 65.74, 63.93, 62.57, 61.54, 60.69, 59.96, 59.28, 58.58, 57.86, 57.08, 56.23, 55.26, 54.14, 52.82, 51.29, 49.55, 47.62, 45.55, 43.38, 41.22, 39.11, 37.17, 35.34, 33.51, 31.56, 29.64, 27.75, 25.9, 24.36, 22.97, 21.03, 19.08, 17.22, 15.66, 13.68, 11.91, 10.31, 8.96, 8.11, p 48326 iter 1000 diff 7
7.05, 6.05, 5.32, 4.7, 4.16, 3.7, 3.29, 2.94, 2.62, 2.35, 2.1, 1.88, 1.7, 1.53, 1.39, 1.27, 1.15, 1.02, 0.91, 0.82, 0.74, 0.67, 0.6, 0.54, 0.49, 0.45, 0.41, 0.37, 0.34, 0.31, 0.28, 0.26, 0.23, 0.21, 0.19, 0.18, 0.16, 0.15, 0.14, 0.13, 0.12, 0.11, 0.1, 0.09, 0.09, 0.08, 0.07, 0.07, 0.06, 0.06, 0.05, 0.05, 0.05, 0.04, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_allMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;)){
     Topic_clus_8_allMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;))
} else {
Topic_clus_8_allMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_allMSC_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_allMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_allMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on adMSCs
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(40)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;)){
     Topic_clus_7_adMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;))
} else {
Topic_clus_7_adMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_adMSC_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_adMSC_lateChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 27152802.76, 221021.85, 10308.92, 1461.48, 422.53, 25.21, 7.28, 51.72, 9604.41, 65115.04, 267403.08, 144833.59, 197193.03, 196510.3, 168343.73, 50807.48, 103186.58, 55489.63, 11213.96, 14211.74, 15065.83, 11169.47, 8191.69, 5891.5, 4273.98, 3462.75, 3645, 4260.62, 4467.86, 3818.05, 2903.93, 2329.85, 2066.29, 1842.89, 1531.09, 1143.61, 808.45, 590.53, 463.91, 389.68, 342.31, 308.56, 283.14, 261.35, 240.23, 218.4, 196.19, 175.23, 157.09, 142.45, 131.07, 122.08, 114.47, 107.46, 100.57, 93.61, 86.68, 80.06, 74, 68.72, 64.32, 60.81, 58.13, 56.19, 54.86, 54, 53.44, 53.08, 52.85, 52.69, 52.56, 52.42, 52.25, 52.03, 51.72, 51.28, 50.65, 49.81, 48.69, 47.31, 45.69, 43.86, 41.9, 39.87, 37.87, 36, 34.28, 32.51, 30.63, 28.77, 26.95, 25.15, 23.58, 22.31, 20.43, 18.51, 16.66, 15.09, 13.2, 11.4, p 48326 iter 1000 diff 10
9.79, 8.41, 7.49, 6.54, 5.48, 4.75, 4.13, 3.61, 3.16, 2.77, 2.43, 2.14, 1.88, 1.66, 1.47, 1.3, 1.16, 1.04, 0.94, 0.85, 0.74, 0.65, 0.57, 0.51, 0.46, 0.41, 0.36, 0.33, 0.29, 0.27, 0.24, 0.22, 0.2, 0.18, 0.16, 0.15, 0.13, 0.12, 0.11, 0.1, 0.09, 0.08, 0.08, 0.07, 0.07, 0.06, 0.06, 0.06, 0.05, 0.05, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_adMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;)){
     Topic_clus_8_adMSC_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;))
} else {
Topic_clus_8_adMSC_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_adMSC_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_adMSC_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_adMSC_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Aref et al 2020)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(30, 49, 55)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)){
     Topic_clus_7_bmMSC1_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;))
} else {
Topic_clus_7_bmMSC1_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC1_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 20184057.92, 68202.85, 2231.27, 86.04, 4.36, 1.89, 0.12, 0.03, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_bmMSC1_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)){
     Topic_clus_8_bmMSC1_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;))
} else {
Topic_clus_8_bmMSC1_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC1_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC1_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Here I define a MSC cluster based on bmMSCs (from Carrow et al 2018)
# labels for &quot;known&quot; clusters
index_MSCs &lt;- c(32,42)

known_samples &lt;- c(index_heart, index_lung, index_liver, index_kidney, index_iPSC, index_chond, index_MSCs)

class_labs &lt;- c(rep(&quot;Heart&quot;, length(index_heart)),
                rep(&quot;Lung&quot;, length(index_lung)),
                rep(&quot;Liver&quot;, length(index_liver)),
                rep(&quot;Kidney&quot;, length(index_kidney)),
                rep(&quot;iPSC&quot;, length(index_iPSC)),
                rep(&quot;Chond&quot;, length(index_chond)),
                rep(&quot;MSC&quot;, length(index_MSCs))
                )

# Perform topic modeling k=7
if (file.exists(&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)){
     Topic_clus_7_bmMSC2_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;))
} else {
Topic_clus_7_bmMSC2_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=7,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_7_bmMSC2_lateChond, file=&quot;data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)
}</code></pre>
<pre><code>Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used

Warning in if (shrink &amp; shrink.method == 2) {: the condition has length &gt; 1 and
only the first element will be used</code></pre>
<pre><code>
Estimating on a 56 document collection.
start the fit 
log posterior increase: 39969843.16, 67141.77, 2531.44, 1470.68, 264.4, 30.71, 61.4, 14736.21, 20180.67, 193031.04, 199341.35, 201880.25, 185821.62, 154786.19, 49594.08, 107290.49, 45906.34, 10781.96, 14569.56, 14699.82, 10805.48, 7973.64, 5755.5, 4183.91, 3446.86, 3694.22, 4309.99, 4446.19, 3735.55, 2833.84, 2291.37, 2036.72, 1805.37, 1481.17, 1090.57, 764.19, 555.31, 434.12, 362.83, 317.6, 286.09, 263.02, 243.46, 224.42, 204.34, 183.57, 163.81, 146.71, 133.06, 122.65, 114.65, 108, 101.88, 95.77, 89.44, 82.97, 76.66, 70.8, 65.63, 61.31, 57.85, 55.24, 53.37, 52.14, 51.39, 50.95, 50.72, 50.62, 50.6, 50.6, 50.58, 50.54, 50.44, 50.25, 49.92, 49.4, 48.65, 47.62, 46.32, 44.77, 43.02, 41.12, 39.16, 37.23, 35.42, 33.73, 32, 30.15, 28.31, 26.51, 24.73, 23.22, 21.91, 20.02, 18.1, 16.27, 14.74, 12.8, 11.03, 9.44, p 48326 iter 1000 diff 8
8.1, 7.24, 6.23, 5.22, 4.51, 3.91, 3.41, 2.97, 2.59, 2.27, 1.99, 1.74, 1.53, 1.35, 1.2, 1.06, 0.95, 0.86, 0.77, 0.67, 0.58, 0.51, 0.46, 0.41, 0.36, 0.32, 0.29, 0.26, 0.23, 0.21, 0.19, 0.18, 0.16, 0.14, 0.13, 0.11, 0.1, 0.1, 0.09, 0.08, 0.07, 0.07, 0.06, 0.06, 0.06, 0.05, 0.05, 0.05, 0.04, 0.04, 0.04, 0.03, 0.03, 0.03, 0.03, 0.03, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, done.</code></pre>
<pre class="r"><code>omega &lt;- Topic_clus_7_bmMSC2_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(7, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Perform topic modeling k=8
if (file.exists(&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)){
     Topic_clus_8_bmMSC2_lateChond &lt;- get(load(file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;))
} else {
Topic_clus_8_bmMSC2_lateChond &lt;- classtpx::class_topics(
    t(merged_raw_data_counts),
    K=8,
    known_samples = known_samples,
    class_labs = class_labs,
    method=&quot;omega.fix&quot;,
    tol=0.01)
save(Topic_clus_8_bmMSC2_lateChond, file=&quot;data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_lateChond.rda&quot;)
}

omega &lt;- Topic_clus_8_bmMSC2_lateChond$omega
annotation &lt;- data.frame(
  sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
  tissue_label = as.factor(labels))
rownames(omega) &lt;- annotation$sample_id
CountClust::StructureGGplot(omega = omega,
                annotation = annotation,
                palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
                yaxis_label = &quot;Tissue/Cell Type&quot;,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = &quot;bold&quot;)
                )</code></pre>
<p><img src="figure/Bulk_cellComposition_analysis.Rmd/supervised%20with%20chond%20late-8.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] classtpx_0.99.0            reshape2_1.4.3            
 [3] edgeR_3.20.9               limma_3.34.9              
 [5] gplots_3.0.1.1             ggfortify_0.4.8           
 [7] MAST_1.4.1                 SummarizedExperiment_1.8.1
 [9] DelayedArray_0.4.1         matrixStats_0.55.0        
[11] Biobase_2.38.0             GenomicRanges_1.30.3      
[13] GenomeInfoDb_1.14.0        IRanges_2.12.0            
[15] S4Vectors_0.16.0           BiocGenerics_0.24.0       
[17] maptpx_1.9-2               slam_0.1-47               
[19] CountClust_1.4.1           cowplot_0.9.3             
[21] ggplot2_3.2.1              scales_1.1.0              
[23] RColorBrewer_1.1-2        

loaded via a namespace (and not attached):
 [1] nlme_3.1-131           bitops_1.0-6           fs_1.3.1              
 [4] rprojroot_1.3-2        tools_3.4.3            backports_1.1.5       
 [7] irlba_2.3.3            R6_2.4.1               vegan_2.5-6           
[10] KernSmooth_2.23-15     lazyeval_0.2.2         mgcv_1.8-22           
[13] colorspace_1.4-1       permute_0.9-5          nnet_7.3-12           
[16] withr_2.1.2            tidyselect_0.2.5       gridExtra_2.3         
[19] compiler_3.4.3         git2r_0.26.1           labeling_0.3          
[22] caTools_1.17.1.3       SQUAREM_2020.2         mixsqp_0.3-17         
[25] stringr_1.4.0          digest_0.6.23          rmarkdown_1.18        
[28] XVector_0.18.0         pkgconfig_2.0.3        htmltools_0.4.0       
[31] invgamma_1.1           rlang_0.4.4            farver_2.0.3          
[34] gtools_3.8.1           dplyr_0.8.3            RCurl_1.95-4.12       
[37] magrittr_1.5           modeltools_0.2-22      GenomeInfoDbData_1.0.0
[40] Matrix_1.2-18          Rcpp_1.0.3             munsell_0.5.0         
[43] ape_5.3                abind_1.4-5            lifecycle_0.1.0       
[46] stringi_1.4.3          whisker_0.4            yaml_2.2.0            
[49] MASS_7.3-51.4          zlibbioc_1.24.0        flexmix_2.3-15        
[52] plyr_1.8.4             grid_3.4.3             gdata_2.18.0          
[55] promises_1.1.0         crayon_1.3.4           lattice_0.20-35       
[58] locfit_1.5-9.1         zeallot_0.1.0          knitr_1.26            
[61] pillar_1.4.2           picante_1.8            glue_1.3.1            
[64] evaluate_0.14          data.table_1.12.6      vctrs_0.2.0           
[67] httpuv_1.5.2           gtable_0.3.0           purrr_0.3.3           
[70] tidyr_1.0.0            assertthat_0.2.1       ashr_2.2-47           
[73] xfun_0.12              later_1.0.0            truncnorm_1.0-8       
[76] tibble_2.1.3           workflowr_1.5.0        cluster_2.0.6         </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
