<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-01-24" />

<title>cell Atlas or Human Cell Landscape Assignments</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">cell Atlas or Human Cell Landscape Assignments</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-01-24</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-09-03
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreee3ee7ee16d345b8aa6e8f87d418a6933955590aatargetblanke3ee7eea"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/e3ee7ee16d345b8aa6e8f87d418a6933955590aa" target="_blank">e3ee7ee</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreee3ee7ee16d345b8aa6e8f87d418a6933955590aatargetblanke3ee7eea" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/e3ee7ee16d345b8aa6e8f87d418a6933955590aa" target="_blank">e3ee7ee</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds
    Ignored:    output/SNPs.csv
    Ignored:    output/bhat.rds
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash_500k.rds
    Ignored:    output/mash_500k_INT.rds
    Ignored:    output/mash_all_refit_500k_1.rds
    Ignored:    output/mash_all_refit_500k_2.rds
    Ignored:    output/mash_all_refit_500k_3.rds
    Ignored:    output/mash_all_refit_500k_4.rds
    Ignored:    output/mash_all_refit_500k_5.rds
    Ignored:    output/mash_all_refit_500k_6.rds
    Ignored:    output/mash_all_refit_500k_7.rds
    Ignored:    output/mash_all_refit_500k_8.rds
    Ignored:    output/mash_all_refit_500k_INT_1.rds
    Ignored:    output/mash_all_refit_500k_INT_2.rds
    Ignored:    output/mash_all_refit_500k_INT_3.rds
    Ignored:    output/mash_all_refit_500k_INT_4.rds
    Ignored:    output/mash_all_refit_500k_INT_5.rds
    Ignored:    output/mash_all_refit_500k_INT_6.rds
    Ignored:    output/mash_all_refit_500k_INT_7.rds
    Ignored:    output/mash_all_refit_500k_INT_8.rds
    Ignored:    output/mash_full_data.rds
    Ignored:    output/mash_full_data_INT.rds
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/shat.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Unstaged changes:
    Modified:   analysis/cellAtlasAssignments.Rmd
    Modified:   analysis/powerAnalysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/cellAtlasAssignments.Rmd</code>) and HTML (<code>docs/cellAtlasAssignments.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/analysis/cellAtlasAssignments.Rmd" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-01
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/26132157c6605eb09d8628dff14a004a6a374809/docs/cellAtlasAssignments.html" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-01
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/fe389f20fba1596c42e7a7c15b253b4e8090ecac/analysis/cellAtlasAssignments.Rmd" target="_blank">fe389f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
make png for corr heatmap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/fe389f20fba1596c42e7a7c15b253b4e8090ecac/docs/cellAtlasAssignments.html" target="_blank">fe389f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
make png for corr heatmap
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/analysis/cellAtlasAssignments.Rmd" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
Add clustering for cellatlas sampls
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/cellAtlasAssignments.html" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
Add clustering for cellatlas sampls
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/230347d31eea2744e2e4274fa8e31c0d1c1e991b/analysis/cellAtlasAssignments.Rmd" target="_blank">230347d</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
Add crude analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/230347d31eea2744e2e4274fa8e31c0d1c1e991b/docs/cellAtlasAssignments.html" target="_blank">230347d</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
<td>
Add crude analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/4b7b862fcc613e73a5ac464c4079a642dab2d37d/analysis/cellAtlasAssignments.Rmd" target="_blank">4b7b862</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell line analysis after removing some catgories
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/4b7b862fcc613e73a5ac464c4079a642dab2d37d/docs/cellAtlasAssignments.html" target="_blank">4b7b862</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell line analysis after removing some catgories
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b967c3ceb4e6c6da17db3be8f12c4009730dde44/analysis/cellAtlasAssignments.Rmd" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/b967c3ceb4e6c6da17db3be8f12c4009730dde44/docs/cellAtlasAssignments.html" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
</div>
<div id="cell-atlas-from-mengie-chens-lab-uchicago" class="section level1">
<h1>Cell atlas from Mengie Chen’s lab (UChicago)</h1>
<p>Compute correlation between reference set of samples and individual cells (input is seurat object.) Load correlation functions</p>
<pre class="r"><code>library(Seurat)

#load correlation functions
source(&quot;code/cellAtlas_assignment.R&quot;)

#load single-cell data from pilot
#ANT1.2 &lt;- readRDS(&quot;data/ANT1_2.rds&quot;)
SCT_integrated &lt;- readRDS(&quot;data/SCT_ANT12_integrated.rds&quot;)</code></pre>
<div id="apply-functions-to-single-cell-data-to-label-them-with-assigned-cell-type" class="section level2">
<h2>Apply functions to single cell data to label them with assigned cell type</h2>
<pre class="r"><code>#Compute the correlation matrix use a seurat object (COUNTS) and cell line matrix as input
DefaultAssay(SCT_integrated) &lt;- &quot;RNA&quot;
cor.mat &lt;- ComputeCorMat(SCT_integrated,cell.line)</code></pre>
<pre><code>[1] &quot;There are 4134genes in common between the two sets&quot;</code></pre>
<pre class="r"><code>head(cor.mat)</code></pre>
<pre><code>                    L1_ESC    L2_ESC    L3_ESC    L4_ESC    L5_ESC
AAACCCAAGTAGGAAG 0.2127661 0.2280576 0.2576776 0.2665538 0.2852950
AAACCCATCTCGGGAC 0.2325975 0.2522491 0.2847764 0.2895341 0.2969894
AAACGAAGTAACTGCT 0.2709670 0.2806276 0.3119574 0.3056133 0.3225022
AAAGAACTCGTGCAGC 0.2213311 0.2332478 0.2543673 0.2498685 0.2729095
AAAGGATGTCTCAAGT 0.2161640 0.2314025 0.2501461 0.2512007 0.2635432
AAAGGGCTCTTACGGA 0.2438220 0.2615535 0.2826236 0.2885744 0.3113105
                    L6_ESC L7_Neuroepithelial L8_Astrocyte L9_Astrocyte
AAACCCAAGTAGGAAG 0.3651657          0.3221773    0.2644856    0.2366720
AAACCCATCTCGGGAC 0.3833672          0.3366648    0.3086319    0.2840851
AAACGAAGTAACTGCT 0.3966474          0.3408116    0.2992926    0.2704224
AAAGAACTCGTGCAGC 0.3557359          0.2994689    0.2493521    0.2303603
AAAGGATGTCTCAAGT 0.3349858          0.2928124    0.2579703    0.2347415
AAAGGGCTCTTACGGA 0.3906065          0.3448182    0.2922450    0.2787903
                 L10_Neural_Precursor L11_Neural_Precursor  L12_iPSC
AAACCCAAGTAGGAAG            0.3205696            0.3106295 0.4412213
AAACCCATCTCGGGAC            0.3273081            0.3169374 0.4730965
AAACGAAGTAACTGCT            0.3450115            0.3253898 0.4985778
AAAGAACTCGTGCAGC            0.3033935            0.2949636 0.4431311
AAAGGATGTCTCAAGT            0.2860495            0.2814570 0.4368565
AAAGGGCTCTTACGGA            0.3341364            0.3190007 0.4895454
                  L13_iPSC  L14_iPSC  L15_iPSC   L16_MSC   L17_MSC
AAACCCAAGTAGGAAG 0.4737592 0.2642885 0.4579575 0.4450518 0.4089045
AAACCCATCTCGGGAC 0.4902393 0.2971788 0.4871561 0.4796679 0.4370790
AAACGAAGTAACTGCT 0.5268932 0.3121624 0.5209353 0.5080193 0.4723240
AAAGAACTCGTGCAGC 0.4666426 0.2597496 0.4718939 0.4534033 0.4225432
AAAGGATGTCTCAAGT 0.4700325 0.2556880 0.4615746 0.4403018 0.3929940
AAAGGGCTCTTACGGA 0.5047001 0.2994724 0.4998121 0.4898813 0.4491311
                   L18_MSC   L19_MSC   L20_MSC L21_Mesodermal_Stem.Cell
AAACCCAAGTAGGAAG 0.3815813 0.4471752 0.4518252                0.3542508
AAACCCATCTCGGGAC 0.4067208 0.4691416 0.4789260                0.3522645
AAACGAAGTAACTGCT 0.4382410 0.5069705 0.5174162                0.3676749
AAAGAACTCGTGCAGC 0.3891178 0.4598642 0.4638207                0.3463360
AAAGGATGTCTCAAGT 0.3671689 0.4459976 0.4522972                0.3298743
AAAGGGCTCTTACGGA 0.4128713 0.4875941 0.4988265                0.3754376
                 L22_Epithelial_Bladder L23_Epithelial_Bladder
AAACCCAAGTAGGAAG              0.2316352              0.2671622
AAACCCATCTCGGGAC              0.2877353              0.3286498
AAACGAAGTAACTGCT              0.3321162              0.3706654
AAAGAACTCGTGCAGC              0.2577582              0.2948993
AAAGGATGTCTCAAGT              0.2289472              0.2640708
AAAGGGCTCTTACGGA              0.2983314              0.3367303
                 L24_Epithelial_Bronchial L25_Epithelial_Bronchial
AAACCCAAGTAGGAAG                0.1838736                0.1822692
AAACCCATCTCGGGAC                0.2257487                0.2268387
AAACGAAGTAACTGCT                0.2664858                0.2642978
AAAGAACTCGTGCAGC                0.1983332                0.1966023
AAAGGATGTCTCAAGT                0.1843270                0.1892689
AAAGGGCTCTTACGGA                0.2417999                0.2411382
                 L26_Fibroblast_Foreskin L27_Fibroblast_Breast.Tissue
AAACCCAAGTAGGAAG               0.4329908                    0.4624092
AAACCCATCTCGGGAC               0.4558441                    0.4903512
AAACGAAGTAACTGCT               0.4969899                    0.5200754
AAAGAACTCGTGCAGC               0.4373219                    0.4678355
AAAGGATGTCTCAAGT               0.4314800                    0.4636499
AAAGGGCTCTTACGGA               0.4751569                    0.5122947
                 L28_Endothelial_Humbilical.Vein
AAACCCAAGTAGGAAG                       0.3100539
AAACCCATCTCGGGAC                       0.3271866
AAACGAAGTAACTGCT                       0.3552835
AAAGAACTCGTGCAGC                       0.3174467
AAAGGATGTCTCAAGT                       0.2728890
AAAGGGCTCTTACGGA                       0.3599641
                 L29_Endothelial_Humbilical.Vein
AAACCCAAGTAGGAAG                       0.3335253
AAACCCATCTCGGGAC                       0.3454137
AAACGAAGTAACTGCT                       0.3719869
AAAGAACTCGTGCAGC                       0.3315706
AAAGGATGTCTCAAGT                       0.2877973
AAAGGGCTCTTACGGA                       0.3801740
                 L30_Endothelial_Humbilical.Vein
AAACCCAAGTAGGAAG                       0.3590146
AAACCCATCTCGGGAC                       0.3700965
AAACGAAGTAACTGCT                       0.3985975
AAAGAACTCGTGCAGC                       0.3585557
AAAGGATGTCTCAAGT                       0.3317444
AAAGGGCTCTTACGGA                       0.4076300
                 L31_Endothelial_Lymph.Vessel L32_Endothelial_Blood.Vessel
AAACCCAAGTAGGAAG                    0.2891486                    0.3014286
AAACCCATCTCGGGAC                    0.3072357                    0.3244661
AAACGAAGTAACTGCT                    0.3366871                    0.3583441
AAAGAACTCGTGCAGC                    0.3044000                    0.3177784
AAAGGATGTCTCAAGT                    0.2612912                    0.2794534
AAAGGGCTCTTACGGA                    0.3444807                    0.3602949
                 L33_Endothelial_Humbilical.Vein L34_Endothelial_Lymphatic
AAACCCAAGTAGGAAG                       0.3255503                 0.3063615
AAACCCATCTCGGGAC                       0.3429656                 0.3336868
AAACGAAGTAACTGCT                       0.3606642                 0.3535497
AAAGAACTCGTGCAGC                       0.3299209                 0.3094668
AAAGGATGTCTCAAGT                       0.2894251                 0.2863140
AAAGGGCTCTTACGGA                       0.3736863                 0.3626130
                 L35_Chondrocyte L36_Chondrocyte L37_Osteoblast
AAACCCAAGTAGGAAG       0.4443542       0.4128319      0.4554819
AAACCCATCTCGGGAC       0.4768039       0.4508199      0.4799630
AAACGAAGTAACTGCT       0.4665734       0.4602453      0.4885063
AAAGAACTCGTGCAGC       0.4454958       0.4078837      0.4461702
AAAGGATGTCTCAAGT       0.4532327       0.4133727      0.4623004
AAAGGGCTCTTACGGA       0.4673211       0.4490568      0.4743739
                 L38_Osteoblast L39_Adipocyte L40_Hepatocyte
AAACCCAAGTAGGAAG      0.4288885     0.3973992      0.1404015
AAACCCATCTCGGGAC      0.4539705     0.4265490      0.1625652
AAACGAAGTAACTGCT      0.4958842     0.4328301      0.1856851
AAAGAACTCGTGCAGC      0.4394513     0.3863947      0.1492740
AAAGGATGTCTCAAGT      0.4247691     0.4101849      0.1573037
AAAGGGCTCTTACGGA      0.4682938     0.4195391      0.1552834
                 L41_Epithelial_Keratinocyte L42_Schwann.cell
AAACCCAAGTAGGAAG                   0.1317285        0.3399465
AAACCCATCTCGGGAC                   0.1700799        0.3926095
AAACGAAGTAACTGCT                   0.2037107        0.4079090
AAAGAACTCGTGCAGC                   0.1499551        0.3589414
AAAGGATGTCTCAAGT                   0.1470710        0.3493402
AAAGGGCTCTTACGGA                   0.1794205        0.3866526
                 L43_Smooth.muscle.cell_Bronchial L44_HSC_Bone.Marrow
AAACCCAAGTAGGAAG                        0.4112326          0.09177966
AAACCCATCTCGGGAC                        0.4450729          0.13141009
AAACGAAGTAACTGCT                        0.4437330          0.13800687
AAAGAACTCGTGCAGC                        0.4110590          0.09927104
AAAGGATGTCTCAAGT                        0.4106680          0.09298633
AAAGGGCTCTTACGGA                        0.4434619          0.10338167
                 L45_CMP_Bone.Marrow L46_GMP_Bone.Marrow
AAACCCAAGTAGGAAG           0.1020825          0.08289062
AAACCCATCTCGGGAC           0.1269503          0.11389655
AAACGAAGTAACTGCT           0.1485444          0.12938490
AAAGAACTCGTGCAGC           0.1161569          0.09975551
AAAGGATGTCTCAAGT           0.1090988          0.09403200
AAAGGGCTCTTACGGA           0.1111755          0.09226560
                 L47_MEP_Bone.Marrow L48_Myelocyte_Bone.Marrow
AAACCCAAGTAGGAAG           0.1098459                0.02468094
AAACCCATCTCGGGAC           0.1295484                0.05701652
AAACGAAGTAACTGCT           0.1640709                0.07299742
AAAGAACTCGTGCAGC           0.1148460                0.04778671
AAAGGATGTCTCAAGT           0.1142347                0.03988963
AAAGGGCTCTTACGGA           0.1219098                0.04417893
                 L49_Myelocyte_Bone.Marrow L50_CFU_Bone.Marrow
AAACCCAAGTAGGAAG                0.05765211           0.1090757
AAACCCATCTCGGGAC                0.08256149           0.1234394
AAACGAAGTAACTGCT                0.10396944           0.1554019
AAAGAACTCGTGCAGC                0.06178092           0.1054661
AAAGGATGTCTCAAGT                0.06919351           0.1113522
AAAGGGCTCTTACGGA                0.06902595           0.1174252
                 L51_B.Cell_Bone.Marrow L52_Platelet L53_Erythroblast_Late
AAACCCAAGTAGGAAG             0.03920281   0.06453785            0.10064393
AAACCCATCTCGGGAC             0.06544178   0.09299309            0.11959534
AAACGAAGTAACTGCT             0.08329355   0.11096482            0.15198434
AAAGAACTCGTGCAGC             0.05354244   0.08810920            0.10117482
AAAGGATGTCTCAAGT             0.03904266   0.07599502            0.12153049
AAAGGGCTCTTACGGA             0.04963729   0.08176423            0.09548154
                 L54_Erythroblast_Pro L55_MSC_Fetal.Blood
AAACCCAAGTAGGAAG            0.1053904           0.4619908
AAACCCATCTCGGGAC            0.1212415           0.4939485
AAACGAAGTAACTGCT            0.1548096           0.5266282
AAAGAACTCGTGCAGC            0.1062466           0.4661060
AAAGGATGTCTCAAGT            0.1151713           0.4476693
AAAGGGCTCTTACGGA            0.1149825           0.5058889
                 L56_MSC_Fetal.Bone.Marrow L57_MSC_Fetal.Liver
AAACCCAAGTAGGAAG                 0.4605339           0.4497178
AAACCCATCTCGGGAC                 0.4961010           0.4840106
AAACGAAGTAACTGCT                 0.5186203           0.5203435
AAAGAACTCGTGCAGC                 0.4564124           0.4617801
AAAGGATGTCTCAAGT                 0.4434417           0.4402316
AAAGGGCTCTTACGGA                 0.5090683           0.5010826
                 L58_Monocyte L59_Monocyte_CD14 L60_Monocyte_CD16
AAACCCAAGTAGGAAG   0.05173794        0.03593747        0.02697876
AAACCCATCTCGGGAC   0.09946028        0.07746175        0.06824645
AAACGAAGTAACTGCT   0.11546377        0.08737047        0.06796966
AAAGAACTCGTGCAGC   0.09408212        0.06078943        0.05822294
AAAGGATGTCTCAAGT   0.07123376        0.05466978        0.04865763
AAAGGGCTCTTACGGA   0.08037198        0.05014720        0.04145187
                 L61_Monocyte_CD16 L62_Monocyte
AAACCCAAGTAGGAAG        0.04296360   0.05542529
AAACCCATCTCGGGAC        0.08259408   0.08359650
AAACGAAGTAACTGCT        0.09048492   0.10304389
AAAGAACTCGTGCAGC        0.07334547   0.07906865
AAAGGATGTCTCAAGT        0.05826452   0.07186702
AAAGGGCTCTTACGGA        0.05875617   0.06609767
                 L63_Macrophage_Monocyte.derived
AAACCCAAGTAGGAAG                       0.0824440
AAACCCATCTCGGGAC                       0.1234566
AAACGAAGTAACTGCT                       0.1572033
AAAGAACTCGTGCAGC                       0.1045270
AAAGGATGTCTCAAGT                       0.1019106
AAAGGGCTCTTACGGA                       0.1036298
                 L64_Macrophage_Monocyte.derived
AAACCCAAGTAGGAAG                      0.05901435
AAACCCATCTCGGGAC                      0.11278058
AAACGAAGTAACTGCT                      0.12115702
AAAGAACTCGTGCAGC                      0.09090508
AAAGGATGTCTCAAGT                      0.08200592
AAAGGGCTCTTACGGA                      0.07713344
                 L65_Macrophage_Monocyte.derived
AAACCCAAGTAGGAAG                       0.0811060
AAACCCATCTCGGGAC                       0.1148498
AAACGAAGTAACTGCT                       0.1538604
AAAGAACTCGTGCAGC                       0.1044641
AAAGGATGTCTCAAGT                       0.1016540
AAAGGGCTCTTACGGA                       0.1082500
                 L66_Macrophage_Monocyte.derived L67_Macrophage_Alveolar
AAACCCAAGTAGGAAG                      0.09301521              0.08299623
AAACCCATCTCGGGAC                      0.12786528              0.12110815
AAACGAAGTAACTGCT                      0.16068865              0.15484259
AAAGAACTCGTGCAGC                      0.11696518              0.11135814
AAAGGATGTCTCAAGT                      0.10897967              0.10349510
AAAGGGCTCTTACGGA                      0.11753404              0.11445564
                 L68_Dendritic.Cell_Monocyte.derived
AAACCCAAGTAGGAAG                          0.05573460
AAACCCATCTCGGGAC                          0.09897222
AAACGAAGTAACTGCT                          0.12846718
AAAGAACTCGTGCAGC                          0.07941838
AAAGGATGTCTCAAGT                          0.07212239
AAAGGGCTCTTACGGA                          0.08306194
                 L69_Dendritic.Cell_Monocyte.derived
AAACCCAAGTAGGAAG                          0.08524724
AAACCCATCTCGGGAC                          0.13063159
AAACGAAGTAACTGCT                          0.16164048
AAAGAACTCGTGCAGC                          0.09841075
AAAGGATGTCTCAAGT                          0.09033391
AAAGGGCTCTTACGGA                          0.10643921
                 L70_Dendritic.Cell_CD16 L71_Dendritic.Cell_Plasmacytoid
AAACCCAAGTAGGAAG              0.04651195                      0.08093629
AAACCCATCTCGGGAC              0.07966604                      0.10395546
AAACGAAGTAACTGCT              0.08988058                      0.13094716
AAAGAACTCGTGCAGC              0.07183364                      0.09787005
AAAGGATGTCTCAAGT              0.06182741                      0.07553312
AAAGGGCTCTTACGGA              0.05652148                      0.08458535
                 L72_Dendritic.Cell_Plasmacytoid L73_T.Cell_CD4.Naive
AAACCCAAGTAGGAAG                      0.07599686           0.06216039
AAACCCATCTCGGGAC                      0.09844140           0.08698866
AAACGAAGTAACTGCT                      0.13177897           0.09857947
AAAGAACTCGTGCAGC                      0.10203538           0.06888272
AAAGGATGTCTCAAGT                      0.07928447           0.06215743
AAAGGGCTCTTACGGA                      0.08288808           0.06061962
                 L74_T.Cell_CD4.Centr..Memory L75_T.Cell_CD4.Centr..Memory
AAACCCAAGTAGGAAG                   0.04953319                   0.05494155
AAACCCATCTCGGGAC                   0.07249161                   0.07883242
AAACGAAGTAACTGCT                   0.08588006                   0.09324688
AAAGAACTCGTGCAGC                   0.06405441                   0.06554876
AAAGGATGTCTCAAGT                   0.04726033                   0.05453517
AAAGGGCTCTTACGGA                   0.05015015                   0.05516487
                 L76_T.Cell_CD4.Eff..Memory L77_T.Cell_CD8.Centr..Memory
AAACCCAAGTAGGAAG                 0.05390077                   0.06432035
AAACCCATCTCGGGAC                 0.07878613                   0.09232980
AAACGAAGTAACTGCT                 0.09427484                   0.11002129
AAAGAACTCGTGCAGC                 0.06983914                   0.09340385
AAAGGATGTCTCAAGT                 0.05298160                   0.06210662
AAAGGGCTCTTACGGA                 0.05456320                   0.08069680
                 L78_T.Cell_CD8 L79_T.Cell_CD4 L80_T.Cell_CD8.Eff..Memory
AAACCCAAGTAGGAAG     0.01880585     0.02762185                 0.06512476
AAACCCATCTCGGGAC     0.03308793     0.04201755                 0.09030937
AAACGAAGTAACTGCT     0.04606601     0.05656105                 0.10417355
AAAGAACTCGTGCAGC     0.03554590     0.04852084                 0.08439920
AAAGGATGTCTCAAGT     0.02201421     0.02836940                 0.06822748
AAAGGGCTCTTACGGA     0.02050169     0.02546175                 0.07278221
                 L81_T.Cell_CD8.Naive L82_T.Cell_CD8 L83_T.Cell_CD4
AAACCCAAGTAGGAAG           0.05691919     0.04676661     0.06706488
AAACCCATCTCGGGAC           0.08847197     0.06096056     0.09516055
AAACGAAGTAACTGCT           0.10229245     0.08465174     0.10939437
AAAGAACTCGTGCAGC           0.08549343     0.06475892     0.07809166
AAAGGATGTCTCAAGT           0.06414804     0.03902209     0.05558493
AAAGGGCTCTTACGGA           0.07319977     0.05327473     0.07406548
                 L84_T.Cell_CD4.Naive L85_NK.Cell_CD56Hi
AAACCCAAGTAGGAAG           0.08209967         0.08125768
AAACCCATCTCGGGAC           0.11352514         0.10851064
AAACGAAGTAACTGCT           0.12374505         0.13441205
AAAGAACTCGTGCAGC           0.08855340         0.10816652
AAAGGATGTCTCAAGT           0.07454126         0.08271360
AAAGGGCTCTTACGGA           0.08607345         0.09443548
                 L86_NK.Cell_CD56Lo L87_Neutrophil L88_Neutrophil
AAACCCAAGTAGGAAG         0.07899103     0.01575484     0.02818251
AAACCCATCTCGGGAC         0.10779893     0.05660629     0.07162125
AAACGAAGTAACTGCT         0.13621195     0.05845380     0.07950457
AAAGAACTCGTGCAGC         0.11036089     0.04358736     0.05844867
AAAGGATGTCTCAAGT         0.08532550     0.03330101     0.04488609
AAAGGGCTCTTACGGA         0.09151158     0.01757654     0.04095760
                 L89_B.Cell L90_B.Cell_Naive L91_B.Cell L92_B.Cell_Memory
AAACCCAAGTAGGAAG 0.01348786       0.05039954 0.04384330        0.04289718
AAACCCATCTCGGGAC 0.03213743       0.07595001 0.06207980        0.06974403
AAACGAAGTAACTGCT 0.04723777       0.09253198 0.08399226        0.08845356
AAAGAACTCGTGCAGC 0.03218967       0.06448139 0.04903087        0.06132907
AAAGGATGTCTCAAGT 0.01986525       0.04389510 0.03810033        0.04565663
AAAGGGCTCTTACGGA 0.02222625       0.06396337 0.05294153        0.05423905
                 L93_B.Cell_Plasma.Cell L94_Oocyte L95_Spermatocyte
AAACCCAAGTAGGAAG             0.08167834 0.06857847     -0.004409334
AAACCCATCTCGGGAC             0.08942042 0.08437433      0.019012178
AAACGAAGTAACTGCT             0.12011594 0.11193538      0.052030198
AAAGAACTCGTGCAGC             0.09265843 0.09064305      0.017412677
AAAGGATGTCTCAAGT             0.08814670 0.07783627      0.020392316
AAAGGGCTCTTACGGA             0.08164909 0.08967909      0.024580861</code></pre>
<pre class="r"><code>#Add a metadata column &quot;clusteredCellAtlas&quot; to the object
SCT_integrated &lt;- AssignClusteredLabel(SCT_integrated,cor.mat, K=20, topCorrelation=5)

#Table of assigned cell labels
table(SCT_integrated$clusteredCellAtlas)</code></pre>
<pre><code>
                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 </code></pre>
</div>
<div id="exploring-different-values-of-k-clusters-of-cellatlas-populations" class="section level2">
<h2>Exploring different values of K (clusters of cellatlas populations)</h2>
<p>Out of interest, I look at how trying different resolutions of the cellatlas data affect assignments of cell types</p>
<pre class="r"><code>for(kval in 11:95){
     print(kval)
     print(table(AssignClusteredLabel(SCT_integrated,cor.mat, K=kval, topCorrelation=5)$clusteredCellAtlas))
}</code></pre>
<pre><code>[1] 11

MSC_iPSC_Fibroblast_Osteoblast_Chondrocyte 
                                      1815 
[1] 12

MSC_iPSC_Fibroblast_Osteoblast_Chondrocyte 
                                      1815 
[1] 13

                       MSC_iPSC_Fibroblast 
                                      1813 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         2 
[1] 14

                       MSC_iPSC_Fibroblast 
                                      1813 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         2 
[1] 15

                       MSC_iPSC_Fibroblast 
                                      1813 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         2 
[1] 16

                       MSC_iPSC_Fibroblast 
                                      1813 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         2 
[1] 17

                       MSC_iPSC_Fibroblast 
                                      1813 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         2 
[1] 18

                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 
[1] 19

                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 
[1] 20

                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 
[1] 21

                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 
[1] 22

                       iPSC_MSC_Fibroblast 
                                      1543 
                                       MSC 
                                       263 
Osteoblast_Chondrocyte_Fibroblast_iPSC_MSC 
                                         9 
[1] 23

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 24

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 25

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 26

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 27

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 28

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 29

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 30

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 31

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 32

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 33

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 34

Chondrocyte_Osteoblast    iPSC_MSC_Fibroblast                    MSC 
                     8                   1545                    262 
[1] 35

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 36

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 37

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 38

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 39

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 40

        Chondrocyte_Osteoblast Fibroblast_iPSC_MSC_Osteoblast 
                            46                              1 
               iPSC_Fibroblast                            MSC 
                          1079                            689 
[1] 41

Chondrocyte_Osteoblast    Fibroblast_iPSC_MSC        iPSC_Fibroblast 
                    46                      1                   1079 
                   MSC 
                   689 
[1] 42

Chondrocyte_Osteoblast        iPSC_Fibroblast                    MSC 
                     4                   1503                    308 
[1] 43

Chondrocyte_Osteoblast        iPSC_Fibroblast                    MSC 
                     4                   1503                    308 
[1] 44

Chondrocyte_Osteoblast        iPSC_Fibroblast                    MSC 
                     4                   1503                    308 
[1] 45

Chondrocyte_Osteoblast        iPSC_Fibroblast                    MSC 
                     4                   1503                    308 
[1] 46

Chondrocyte_Osteoblast    Fibroblast_iPSC_MSC        iPSC_Fibroblast 
                     4                      4                   1501 
                   MSC 
                   306 
[1] 47

Chondrocyte_Osteoblast    Fibroblast_iPSC_MSC        iPSC_Fibroblast 
                     4                      4                   1501 
                   MSC 
                   306 
[1] 48

Chondrocyte_Osteoblast    Fibroblast_iPSC_MSC        iPSC_Fibroblast 
                     4                      4                   1501 
                   MSC 
                   306 
[1] 49

Fibroblast_iPSC_MSC     iPSC_Fibroblast                 MSC 
                  5                1503                 307 
[1] 50

iPSC_Fibroblast             MSC 
           1506             309 
[1] 51

iPSC_Fibroblast             MSC 
           1506             309 
[1] 52

iPSC_Fibroblast             MSC 
           1506             309 
[1] 53

iPSC_Fibroblast             MSC 
           1506             309 
[1] 54

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 55

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 56

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 57

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 58

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 59

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 60

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 61

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 62

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 63

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 64

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 65

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 66

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 67

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 68

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 69

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 70

Fibroblast_iPSC            iPSC             MSC 
           1110              64             641 
[1] 71

Fibroblast_iPSC            iPSC             MSC 
           1117              77             621 
[1] 72

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 73

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 74

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 75

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 76

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 77

Fibroblast_iPSC            iPSC             MSC 
           1113              80             622 
[1] 78

iPSC  MSC 
 722 1093 
[1] 79

iPSC  MSC 
 722 1093 
[1] 80

iPSC  MSC 
 722 1093 
[1] 81

iPSC  MSC 
 722 1093 
[1] 82

iPSC  MSC 
1258  557 
[1] 83

iPSC  MSC 
1258  557 
[1] 84

iPSC  MSC 
1258  557 
[1] 85

iPSC  MSC 
1258  557 
[1] 86

iPSC  MSC 
1794   21 
[1] 87

iPSC  MSC 
1794   21 
[1] 88

iPSC  MSC 
1794   21 
[1] 89

iPSC  MSC 
1794   21 
[1] 90

iPSC  MSC 
1794   21 
[1] 91

iPSC  MSC 
1794   21 
[1] 92

iPSC  MSC 
1794   21 
[1] 93

iPSC  MSC 
1794   21 
[1] 94

iPSC  MSC 
1794   21 
[1] 95

iPSC  MSC 
1794   21 </code></pre>
</div>
<div id="crude-analyses-of-correlation-matrix" class="section level2">
<h2>Crude analyses of correlation matrix</h2>
<pre class="r"><code># see which cell lines are highest correlated (sum over all cells) in the data with the summed values
colSums(cor.mat)[colSums(cor.mat) &gt; 800]</code></pre>
<pre><code>named numeric(0)</code></pre>
</div>
<div id="clustering-analysis-of-cell-atlas-data-pca" class="section level2">
<h2>clustering Analysis of cell atlas data (PCA)</h2>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;ggfortify&quot;)

# Perform PCA
pca_genes &lt;- prcomp(t(cell.line), scale = T)

#relabel cell.line with cell types
cell.line.relabeled &lt;- as.data.frame(t(cell.line))
cell.line.relabeled$celltype &lt;- sapply(strsplit(as.character(rownames(cell.line.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 2)
rownames(cell.line.relabeled) &lt;- paste0(sapply(strsplit(as.character(rownames(cell.line.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 1), &quot; &quot;, sapply(strsplit(as.character(rownames(cell.line.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 2))

#Make PCA plots

autoplot(pca_genes, data = cell.line.relabeled, colour = &#39;celltype&#39;)</code></pre>
<pre><code>Warning: `select_()` is deprecated as of dplyr 0.7.0.
Please use `select()` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_warnings()` to see where this warning was generated.</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-1">
Past versions of “clustering of cell atlas alone-1.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-1.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>autoplot(pca_genes, data = cell.line.relabeled, colour = &#39;celltype&#39;, label=T)</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-2">
Past versions of “clustering of cell atlas alone-2.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-2.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Trying PCA after removing spermatocyte
cell.line.sperm &lt;- cell.line[, -95]
# Perform PCA
pca_genes.sperm &lt;- prcomp(t(cell.line.sperm), scale = T)

#relabel cell.line with cell types
cell.line.sperm.relabeled &lt;- as.data.frame(t(cell.line.sperm))
cell.line.sperm.relabeled$celltype &lt;- sapply(strsplit(as.character(rownames(cell.line.sperm.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 2)
rownames(cell.line.sperm.relabeled) &lt;- paste0(sapply(strsplit(as.character(rownames(cell.line.sperm.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 1), &quot; &quot;, sapply(strsplit(as.character(rownames(cell.line.sperm.relabeled)), &quot;\\_&quot;), &quot;[[&quot;, 2))

#Make PCA plots

autoplot(pca_genes.sperm, data = cell.line.sperm.relabeled, colour = &#39;celltype&#39;)</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-3">
Past versions of “clustering of cell atlas alone-3.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-3.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>autoplot(pca_genes.sperm, data = cell.line.sperm.relabeled, colour = &#39;celltype&#39;, label=T) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-4">
Past versions of “clustering of cell atlas alone-4.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-4.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>autoplot(pca_genes.sperm, data = cell.line.sperm.relabeled, colour = &#39;celltype&#39;, label=T, x=2, y=3) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-5">
Past versions of “clustering of cell atlas alone-5.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-5.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>autoplot(pca_genes.sperm, data = cell.line.sperm.relabeled, colour = &#39;celltype&#39;, label=T, x=3, y=4) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/clustering%20of%20cell%20atlas%20alone-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-of-cell-atlas-alone-6">
Past versions of “clustering of cell atlas alone-6.png”
</button>
</p>
<div id="fig-no-spaces-clustering-of-cell-atlas-alone-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/clustering of cell atlas alone-6.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="clustering-analysis-of-cell-atlas-data-corr-heatmap" class="section level2">
<h2>clustering Analysis of cell atlas data (Corr Heatmap)</h2>
<pre class="r"><code>library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)
# Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))


#Corr heatmap
cors &lt;- cor(cell.line, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)
heatmap.2(cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=cell.line.relabeled$celltype,
          ColSideColors=pal[as.integer(as.factor(cell.line.relabeled$celltype))],
          RowSideColors=pal[as.integer(as.factor(cell.line.relabeled$celltype))],
          cexCol = 0.1 + 1/log10(15), cexRow = 0.1 + 1/log10(15))</code></pre>
<p><img src="figure/cellAtlasAssignments.Rmd/corr%20heatmap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-1">
Past versions of “corr heatmap-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/56119a80c2a1a204bfcc311a3c21b6f93bad488a/docs/figure/cellAtlasAssignments.Rmd/corr heatmap-1.png" target="_blank">56119a8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="human-cell-landscape-httpbis.zju.edu.cnhclblast.html" class="section level1">
<h1>Human cell Landscape (<a href="http://bis.zju.edu.cn/HCL/blast.html" class="uri">http://bis.zju.edu.cn/HCL/blast.html</a>)</h1>
<p>This is a similar method, but uses DGE objects and a different reference set (the reference set comes from single cell rather than bulk data). Computes correlations between cells in a test sample and the reference set samples and assigns best correlation identity.</p>
<pre class="r"><code>#This require devtools  
#install.packages(&#39;devtools&#39;)
#library(devtools)
# scHCL requires ggplot2/reshape2/plotly/shiny/shinythemes/shiny
#install_github(&quot;ggjlab/scHCL&quot;)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] scales_1.1.1       RColorBrewer_1.1-2 gplots_3.0.1.1    
[4] ggfortify_0.4.10   ggplot2_3.3.2      Seurat_3.1.3      

loaded via a namespace (and not attached):
  [1] tsne_0.1-3          nlme_3.1-140        bitops_1.0-6       
  [4] fs_1.3.1            RcppAnnoy_0.0.12    httr_1.4.2         
  [7] rprojroot_1.3-2     sctransform_0.2.0   tools_3.6.1        
 [10] backports_1.1.4     R6_2.4.1            irlba_2.3.3        
 [13] KernSmooth_2.23-15  uwot_0.1.5          lazyeval_0.2.2     
 [16] colorspace_1.4-1    withr_2.2.0         npsurv_0.4-0       
 [19] gridExtra_2.3       tidyselect_1.1.0    compiler_3.6.1     
 [22] git2r_0.26.1        plotly_4.9.2.1      labeling_0.3       
 [25] caTools_1.17.1.2    lmtest_0.9-37       ggridges_0.5.1     
 [28] pbapply_1.4-0       rappdirs_0.3.1      stringr_1.4.0      
 [31] digest_0.6.25       rmarkdown_1.13      pkgconfig_2.0.3    
 [34] htmltools_0.5.0     bibtex_0.4.2        htmlwidgets_1.5.1  
 [37] rlang_0.4.7         farver_2.0.3        generics_0.0.2     
 [40] zoo_1.8-8           jsonlite_1.7.0      ica_1.0-2          
 [43] gtools_3.8.1        dplyr_1.0.1         magrittr_1.5       
 [46] Matrix_1.2-18       Rcpp_1.0.5          munsell_0.5.0      
 [49] ape_5.3             reticulate_1.16     lifecycle_0.2.0    
 [52] stringi_1.4.6       whisker_0.3-2       yaml_2.2.1         
 [55] gbRd_0.4-11         MASS_7.3-51.6       Rtsne_0.15         
 [58] plyr_1.8.6          grid_3.6.1          parallel_3.6.1     
 [61] gdata_2.18.0        listenv_0.8.0       promises_1.1.1     
 [64] ggrepel_0.8.1       crayon_1.3.4        lattice_0.20-41    
 [67] cowplot_1.0.0       splines_3.6.1       knitr_1.23         
 [70] pillar_1.4.6        igraph_1.2.4.1      reshape2_1.4.3     
 [73] future.apply_1.3.0  codetools_0.2-16    leiden_0.3.1       
 [76] glue_1.4.1          evaluate_0.14       lsei_1.2-0         
 [79] metap_1.1           RcppParallel_4.4.3  data.table_1.13.0  
 [82] png_0.1-7           vctrs_0.3.2         httpuv_1.5.1       
 [85] Rdpack_0.11-0       gtable_0.3.0        RANN_2.6.1         
 [88] purrr_0.3.4         tidyr_1.0.3         future_1.18.0      
 [91] xfun_0.8            rsvd_1.0.1          later_1.1.0.1      
 [94] viridisLite_0.3.0   survival_2.44-1.1   tibble_3.0.3       
 [97] workflowr_1.6.2     cluster_2.1.0       globals_0.12.5     
[100] fitdistrplus_1.0-14 ellipsis_0.3.1      ROCR_1.0-7         </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
