<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>DE_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">DE_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-03-02
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree68e5e33dc85bf6340efc50200e47b0766711c93ctargetblank68e5e33a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/68e5e33dc85bf6340efc50200e47b0766711c93c" target="_blank">68e5e33</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree68e5e33dc85bf6340efc50200e47b0766711c93ctargetblank68e5e33a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds

Unstaged changes:
    Modified:   analysis/Bulk_cellComposition_analysis.Rmd
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/Pilot_GoMAnalysis.Rmd
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/DE_analysis.html" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
add additional chondro markers
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b967c3ceb4e6c6da17db3be8f12c4009730dde44/analysis/DE_analysis.Rmd" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/b967c3ceb4e6c6da17db3be8f12c4009730dde44/docs/DE_analysis.html" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/analysis/DE_analysis.Rmd" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
<td>
include Individual as a FE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/DE_analysis.html" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
<td>
include Individual as a FE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a3565148512e79e1482c774b845a50b8c9c06e74/docs/DE_analysis.html" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/3c163de1c62d5f63d88254fa028dda84186718b7/analysis/DE_analysis.Rmd" target="_blank">3c163de</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
reorder by pvalue
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/analysis/DE_analysis.Rmd" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/DE_analysis.html" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/analysis/DE_analysis.Rmd" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/158fdf3e906a4a488e486cb96684b4e23b567895/docs/DE_analysis.html" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/analysis/DE_analysis.Rmd" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/docs/DE_analysis.html" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/analysis/DE_analysis.Rmd" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/DE_analysis.html" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/analysis/DE_analysis.Rmd" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/DE_analysis.html" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/analysis/DE_analysis.Rmd" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/03b499e0b710c5a48bba76aef6472cc713c58301/docs/DE_analysis.html" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5eb29e88b6057837b2c5c0719c225364eb484690/docs/DE_analysis.html" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/DE_analysis.html" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/DE_analysis.html" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/analysis/DE_analysis.Rmd" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/DE_analysis.html" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/analysis/DE_analysis.Rmd" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/DE_analysis.html" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/analysis/DE_analysis.Rmd" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/DE_analysis.html" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/d80f643520bd77f37ac3f3862b7b747bccdbba30/analysis/DE_analysis.Rmd" target="_blank">d80f643</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Run Limma:
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/DE_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/93017062e918d7a60fcad8ee8ca0983de6e9aef7/analysis/DE_analysis.Rmd" target="_blank">9301706</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-17
</td>
<td>
build analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/DE_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/DE_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<pre class="r"><code>library(&quot;limma&quot;)
library(&quot;plyr&quot;)
library(&quot;edgeR&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;ashr&quot;)
library(&quot;ggplot2&quot;)
library(&quot;cowplot&quot;)
library(&quot;VennDiagram&quot;)
library(&quot;RUVSeq&quot;)
library(&quot;topGO&quot;)

#load in filtered count data, RUVs output
filt_counts &lt;- readRDS(&quot;data/filtered_counts.rds&quot;)
filt_counts &lt;- filt_counts$counts
RUVsOut &lt;- readRDS(&quot;data/RUVsOut.rds&quot;)

# load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

# load in reordered sample information
sampleinfo &lt;- readRDS(&quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="limma-doesnt-deal-with-technical-replicates-so-here-i-randomly-sample-1-replicate-for-each-of-the-6-individualxcondition" class="section level1">
<h1>Limma doesn’t deal with technical replicates, so here I randomly sample 1 replicate for each of the 6 individualxcondition</h1>
<pre class="r"><code>#randomly sample 1 replicate from each set of 3 replicates
individuals &lt;- c(&quot;18855&quot;, &quot;18856&quot;, &quot;19160&quot;)
treatments &lt;- c(&quot;S&quot;, &quot;U&quot;)

set.seed(1234)
selected_samples &lt;- c()
for(i in c(&quot;18856&quot;, &quot;18855&quot;)){
     for(t in treatments){
          sample_num &lt;- base::sample(c(1:3), 1)
          selected_samples &lt;- c(selected_samples, paste0(i, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, t))
     }
}

sample_num &lt;- base::sample(c(2:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;U&quot;))
sample_num &lt;- base::sample(c(1:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;S&quot;))

selected_samples</code></pre>
<pre><code>[1] &quot;18856_1_S&quot; &quot;18856_2_U&quot; &quot;18855_2_S&quot; &quot;18855_2_U&quot; &quot;19160_3_U&quot; &quot;19160_2_S&quot;</code></pre>
<pre class="r"><code>randomly_sampled_indices &lt;- match(selected_samples, colnames(filt_counts))

single_filt_counts &lt;- filt_counts[,randomly_sampled_indices]
anno &lt;- pData(RUVsOut)[randomly_sampled_indices,]</code></pre>
<pre class="r"><code>single_filt_counts &lt;- filt_counts
anno &lt;- pData(RUVsOut)
x &lt;- paste0(anno$Individual, anno$treatment)

anno$LibraryPrepBatch &lt;- factor(anno$LibraryPrepBatch, levels = c(&quot;1&quot;, &quot;2&quot;))
anno$Replicate &lt;- factor(anno$Replicate, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;))

design &lt;- model.matrix(~treatment + Individual + W_1 + W_2 + RIN,
                       data = anno)
colnames(design) &lt;- gsub(&quot;treatment&quot;, &quot;&quot;, colnames(design))
colnames(design)</code></pre>
<pre><code>[1] &quot;(Intercept)&quot;        &quot;Unstrain&quot;           &quot;IndividualNA18856 &quot;
[4] &quot;IndividualNA19160 &quot; &quot;W_1&quot;                &quot;W_2&quot;               
[7] &quot;RIN&quot;               </code></pre>
<pre class="r"><code># Model individual as a random effect (use duplicate correlation function); treatment as a fixed effect, covariates = RUV factors
     # Because of issues with duplicateCorrelation with the number of covariates I include in the design matrix, I elect intead to include all samples (all technical replicates) and use duplicateCorrelation for technical replicates and instead include Individual as a fixed effect in the linear model
# Recommended to run both voom and duplicateCorrelation twice.
# https://support.bioconductor.org/p/59700/#67620

#TMM Normalization
y &lt;- DGEList(single_filt_counts)
y &lt;- calcNormFactors(y, method = &quot;TMM&quot;)
#Voom for differential expression
v1 &lt;- voom(y, design)
corfit1 &lt;- duplicateCorrelation(v1, design, block = x)</code></pre>
<pre><code>Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable</code></pre>
<pre class="r"><code>corfit1$consensus.correlation</code></pre>
<pre><code>[1] 0.354764</code></pre>
<pre class="r"><code>v2 &lt;- voom(y, design, block = x, correlation = corfit1$consensus.correlation)
corfit2 &lt;- duplicateCorrelation(v2, design, block = x)
corfit2$consensus.correlation</code></pre>
<pre><code>[1] 0.3549247</code></pre>
<pre class="r"><code>fit &lt;- lmFit(v2, design, block = x,
             correlation = corfit2$consensus.correlation)
fit &lt;- eBayes(fit)</code></pre>
<div id="assess-model-results" class="section level2">
<h2>Assess model results</h2>
<pre class="r"><code>get_results &lt;- function(x, number = nrow(x$coefficients), sort.by = &quot;none&quot;,
                        ...) {
  # x - object MArrayLM from eBayes output
  # ... - additional arguments passed to topTable
  stopifnot(class(x) == &quot;MArrayLM&quot;)
  results &lt;- topTable(x, number = number, sort.by = sort.by, ...)
  return(results)
}

top_treatment &lt;- get_results(fit, coef = &quot;Unstrain&quot;, sort.by = &quot;B&quot;)
head(top_treatment)</code></pre>
<pre><code>                     logFC  AveExpr         t      P.Value    adj.P.Val
ENSG00000102317  0.9646069 6.618528  9.788653 1.959478e-08 0.0002047067
ENSG00000104368 -3.0887783 3.753098 -7.935283 3.851911e-07 0.0020120458
ENSG00000100292 -1.1068651 5.399316 -7.090906 1.727530e-06 0.0058401006
ENSG00000080824 -0.5373642 8.888436 -6.533334 4.905709e-06 0.0058401006
ENSG00000133142  0.5627867 7.056389  6.522848 5.004984e-06 0.0058401006
ENSG00000181019 -0.7843440 5.824401 -6.517648 5.054991e-06 0.0058401006
                       B
ENSG00000102317 9.570798
ENSG00000104368 6.229870
ENSG00000100292 5.330764
ENSG00000080824 4.350280
ENSG00000133142 4.328594
ENSG00000181019 4.322443</code></pre>
<pre class="r"><code>results_treatment &lt;- get_results(fit, coef = &quot;Unstrain&quot;,
                              number = nrow(filt_counts), sort = &quot;none&quot;)
ma_treatment &lt;- ggplot(data.frame(Amean = fit$Amean, logFC = fit$coef[, &quot;Unstrain&quot;]),
                       aes(x = Amean, y = logFC)) +
  geom_point() +
  labs(x = &quot;Average expression level&quot;, y = &quot;Log fold change&quot;,
       title = &quot;Treatment effect&quot;)
ma_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-model-results-1">
Past versions of model-results-1.png
</button>
</p>
<div id="fig-model-results-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist_treatment &lt;- ggplot(results_treatment, aes(x = P.Value)) +
  geom_histogram(binwidth = 0.01) +
  labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;, title = &quot;Treatment effect&quot;)
hist_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-model-results-2">
Past versions of model-results-2.png
</button>
</p>
<div id="fig-model-results-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="explore-top-hits" class="section level2">
<h2>Explore top hits</h2>
<p>Boxplot function.</p>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An EList object containing log2 counts per million
  # g - character vector of a single gene
  stopifnot(class(v) == &quot;EList&quot;,
            is.character(g), length(g) == 1)
  library(&quot;tidyr&quot;)
  single_gene &lt;- v$E[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))

  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
    geom_boxplot() +
    labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;))
}</code></pre>
<p>Treatment</p>
<pre class="r"><code>#rename top_treatment genes to gene symbols for easier interpretation
top_treatment_top4 &lt;- gene_anno$external_gene_name[match(head(rownames(top_treatment), n=4), gene_anno$ensembl_gene_id)]
top_treatment_top4</code></pre>
<pre><code>[1] RBM3     PLAT     HMOX1    HSP90AA1
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(v2, rownames(top_treatment)[1])
p_treatment_2 &lt;- plot_gene(v2, rownames(top_treatment)[2]) 
p_treatment_3 &lt;- plot_gene(v2, rownames(top_treatment)[3]) 
p_treatment_4 &lt;- plot_gene(v2, rownames(top_treatment)[4]) 
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)
plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<p><img src="figure/DE_analysis.Rmd/treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-treatment-1">
Past versions of treatment-1.png
</button>
</p>
<div id="fig-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="use-ash-for-mutliple-testing-correction" class="section level2">
<h2>Use ash for mutliple testing correction</h2>
<p>Treatment effect.</p>
<pre class="r"><code>run_ash &lt;- function(x, coef) {
  # Perform multiple testing correction with adaptive shrinkage (ASH)
  #
  # x - object MArrayLM from eBayes output
  # coef - coefficient tested by eBayes
  stopifnot(class(x) == &quot;MArrayLM&quot;, coef %in% colnames(x$coefficients))
  result &lt;- ash(betahat = x$coefficients[, coef],
                sebetahat = x$stdev.unscaled[, coef] * sqrt(x$s2.post),
                df = x$df.total[1])
  return(result)
}

ash_treatment &lt;- run_ash(fit, &quot;Unstrain&quot;)
class(ash_treatment)</code></pre>
<pre><code>[1] &quot;ash&quot;</code></pre>
<pre class="r"><code>names(ash_treatment)</code></pre>
<pre><code>[1] &quot;fitted_g&quot; &quot;loglik&quot;   &quot;logLR&quot;    &quot;data&quot;     &quot;result&quot;   &quot;call&quot;    </code></pre>
<pre class="r"><code>sum(ash_treatment$result$svalue &lt; .05)</code></pre>
<pre><code>[1] 784</code></pre>
<pre class="r"><code>hist(ash_treatment$result$svalue)</code></pre>
<p><img src="figure/DE_analysis.Rmd/ash-treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ash-treatment-1">
Past versions of ash-treatment-1.png
</button>
</p>
<div id="fig-ash-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="plots" class="section level1">
<h1>Plots</h1>
<pre class="r"><code>plot_ma &lt;- function(x, qval) {
  # Create MA plot.
  #
  # x - data frame with topTable and ASH output
  #     (columns logFC, AveExpr, and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;AveExpr&quot;, &quot;qvalue&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = AveExpr, y = logFC, color = highlight, shape = highlight)) +
    geom_point() +
    labs(x = &quot;Average expression level&quot;, y = &quot;Log fold change&quot;) +
    scale_color_identity(drop = FALSE) +
    scale_shape_manual(values = c(16, 1), drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
#   scale_color_gradient(low = &quot;red&quot;, high = &quot;white&quot;, limits = c(0, 0.25))
}

plot_volcano &lt;- function(x, qval) {
  # Create volcano plot.
  #
  # x - data frame with topTable and ASH output
  #     (columns logFC, P.Value, and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;P.Value&quot;, &quot;qvalue&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(P.Value), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, qval) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;P.Value&quot;, &quot;qvalue&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = P.Value, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}</code></pre>
<pre class="r"><code>tests &lt;- colnames(fit$coefficients)
results &lt;- vector(length = length(tests), mode = &quot;list&quot;)
names(results) &lt;- tests

for (test in tests) {
  # Extract limma results
  results[[test]] &lt;- get_results(fit, coef = test)
  # Add mutliple testing correction with ASH
  output_ash &lt;- run_ash(fit, coef = test)$result
  results[[test]] &lt;- cbind(results[[test]], lfsr = output_ash$lfsr,
                           lfdr = output_ash$lfdr, qvalue = output_ash$qvalue,
                           svalue = output_ash$svalue)
}</code></pre>
<pre><code>Warning in estimate_mixprop(data, g, prior, optmethod = optmethod, control
= control, : Optimization failed to converge. Results may be unreliable. Try
increasing maxiter and rerunning.</code></pre>
<pre class="r"><code>#FDR 0.1
plot_ma(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-1">
Past versions of “use Functions-1.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-2">
Past versions of “use Functions-2.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-3">
Past versions of “use Functions-3.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.1)</code></pre>
<pre><code>
FALSE  TRUE 
 8154  2293 </code></pre>
<pre class="r"><code>significant_genes_10 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.1]
significant_symbols_10 &lt;- gene_anno$external_gene_name[match(significant_genes_10, gene_anno$ensembl_gene_id)]
head(significant_symbols_10)</code></pre>
<pre><code>[1] LINC01128  DVL1       MXRA8      AURKAIP1   AL391244.1 MRPL20    
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_10 &lt;- gene_anno[match(significant_genes_10, gene_anno$ensembl_gene_id),]


#FDR 0.05
plot_ma(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-4">
Past versions of “use Functions-4.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-5">
Past versions of “use Functions-5.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-6">
Past versions of “use Functions-6.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05)</code></pre>
<pre><code>
FALSE  TRUE 
 9438  1009 </code></pre>
<pre class="r"><code>significant_genes_05 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05]
significant_symbols_05 &lt;- gene_anno$external_gene_name[match(significant_genes_05, gene_anno$ensembl_gene_id)]
head(significant_symbols_05)</code></pre>
<pre><code>[1] DVL1   MRPL20 MMP23B PEX10  CEP104 ACOT7 
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_05 &lt;- gene_anno[match(significant_genes_05, gene_anno$ensembl_gene_id),]


#FDR 0.01
plot_ma(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-7">
Past versions of “use Functions-7.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-8">
Past versions of “use Functions-8.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-9">
Past versions of “use Functions-9.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.01)</code></pre>
<pre><code>
FALSE  TRUE 
10317   130 </code></pre>
<pre class="r"><code>significant_genes_01 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.01]
significant_symbols_01 &lt;- gene_anno$external_gene_name[match(significant_genes_01, gene_anno$ensembl_gene_id)]
head(significant_symbols_01)</code></pre>
<pre><code>[1] SPSB1      TENT5B     RCC1       SLC6A9     PLK3       ST6GALNAC5
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_01 &lt;- gene_anno[match(significant_genes_01, gene_anno$ensembl_gene_id),]</code></pre>
<div id="gene-ontology-analysis-with-topgo" class="section level2">
<h2>Gene ontology analysis with topGO</h2>
<p>Use topGO for GO analysis. It accounts for the nested graph structure of GO terms to prune the number of GO categories tested (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">Alexa et al. 2006</a>). Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe. This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.05
gene_universe &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold)
gene_universe &lt;- factor(gene_universe)
names(gene_universe) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 ENSG00000198744 
              0               0               0               0               0 
ENSG00000228794 
              0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>Loading required package: org.Hs.eg.db</code></pre>
<pre><code></code></pre>
<pre><code>    ( 9640 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13704 GO terms and 31801 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9322 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 5706 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  3 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  10 nodes to be scored   (10 eliminated genes)</code></pre>
<pre><code>
     Level 16:  17 nodes to be scored   (16 eliminated genes)</code></pre>
<pre><code>
     Level 15:  47 nodes to be scored   (58 eliminated genes)</code></pre>
<pre><code>
     Level 14:  106 nodes to be scored  (176 eliminated genes)</code></pre>
<pre><code>
     Level 13:  170 nodes to be scored  (597 eliminated genes)</code></pre>
<pre><code>
     Level 12:  287 nodes to be scored  (1446 eliminated genes)</code></pre>
<pre><code>
     Level 11:  484 nodes to be scored  (2969 eliminated genes)</code></pre>
<pre><code>
     Level 10:  637 nodes to be scored  (4066 eliminated genes)</code></pre>
<pre><code>
     Level 9:   768 nodes to be scored  (5624 eliminated genes)</code></pre>
<pre><code>
     Level 8:   797 nodes to be scored  (6882 eliminated genes)</code></pre>
<pre><code>
     Level 7:   833 nodes to be scored  (7755 eliminated genes)</code></pre>
<pre><code>
     Level 6:   713 nodes to be scored  (8386 eliminated genes)</code></pre>
<pre><code>
     Level 5:   457 nodes to be scored  (8751 eliminated genes)</code></pre>
<pre><code>
     Level 4:   248 nodes to be scored  (8966 eliminated genes)</code></pre>
<pre><code>
     Level 3:   105 nodes to be scored  (9083 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9153 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9213 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated Significant
1  GO:0055114                 oxidation-reduction process       619          94
2  GO:0045454                      cell redox homeostasis        53          15
3  GO:0002576                      platelet degranulation        71          18
4  GO:0006027         glycosaminoglycan catabolic process        40          10
5  GO:0001568                    blood vessel development       393          60
6  GO:0030574                  collagen catabolic process        40          12
7  GO:0048842 positive regulation of axon extension in...         5           4
8  GO:0046364         monosaccharide biosynthetic process        58           9
9  GO:0006695            cholesterol biosynthetic process        56          14
10 GO:0030198           extracellular matrix organization       206          47
11 GO:1904950 negative regulation of establishment of ...       129          17
12 GO:0051639            actin filament network formation         6           4
13 GO:0006986                response to unfolded protein       147          30
14 GO:0042493                            response to drug       217          37
15 GO:0002063                     chondrocyte development        20           7
16 GO:1901700      response to oxygen-containing compound       873         108
17 GO:0030199                collagen fibril organization        31           9
18 GO:0050731 positive regulation of peptidyl-tyrosine...        78          19
19 GO:0036499     PERK-mediated unfolded protein response        16           5
20 GO:0017185               peptidyl-lysine hydroxylation         7           4
21 GO:0032964               collagen biosynthetic process        24           5
22 GO:0042060                               wound healing       311          47
23 GO:0035987             endodermal cell differentiation        32           9
24 GO:0007566                         embryo implantation        33          10
25 GO:0034329                      cell junction assembly       141          17
26 GO:0022617            extracellular matrix disassembly        53          12
27 GO:0070527                        platelet aggregation        33           8
28 GO:0006024      glycosaminoglycan biosynthetic process        74          15
29 GO:0006516              glycoprotein catabolic process        13           5
30 GO:0010332                 response to gamma radiation        43           7
31 GO:0009617                       response to bacterium       204          21
32 GO:0051898 negative regulation of protein kinase B ...        36           9
33 GO:0042267 natural killer cell mediated cytotoxicit...        19           6
34 GO:0019321                   pentose metabolic process         9           4
35 GO:0031643          positive regulation of myelination         9           4
36 GO:0043619 regulation of transcription from RNA pol...         9           4
37 GO:0021781                  glial cell fate commitment         5           3
38 GO:0050665      hydrogen peroxide biosynthetic process         5           3
39 GO:1900016 negative regulation of cytokine producti...         5           3
40 GO:0048875        chemical homeostasis within a tissue         5           3
41 GO:2001256 regulation of store-operated calcium ent...         5           3
42 GO:0071578            zinc II ion transmembrane import         5           3
43 GO:0071500     cellular response to nitrosative stress         5           3
44 GO:0060638       mesenchymal-epithelial cell signaling         5           3
45 GO:0098780    response to mitochondrial depolarisation        18           4
46 GO:0000423                                   mitophagy        19           4
47 GO:0009725                         response to hormone       523          51
48 GO:0006098                     pentose-phosphate shunt        14           5
49 GO:0045471                         response to ethanol        57          12
50 GO:0016266                         O-glycan processing        25           7
   Expected weightFisher
1     62.15      3.2e-07
2      5.32      0.00015
3      7.13      0.00016
4      4.02      0.00018
5     39.46      0.00025
6      4.02      0.00038
7      0.50      0.00046
8      5.82      0.00047
9      5.62      0.00078
10    20.68      0.00094
11    12.95      0.00100
12     0.60      0.00128
13    14.76      0.00146
14    21.79      0.00164
15     2.01      0.00241
16    87.66      0.00255
17     3.11      0.00262
18     7.83      0.00266
19     1.61      0.00276
20     0.70      0.00276
21     2.41      0.00276
22    31.23      0.00299
23     3.21      0.00333
24     3.31      0.00388
25    14.16      0.00440
26     5.32      0.00508
27     3.31      0.00584
28     7.43      0.00621
29     1.31      0.00653
30     4.32      0.00654
31    20.48      0.00663
32     3.61      0.00776
33     1.91      0.00839
34     0.90      0.00842
35     0.90      0.00842
36     0.90      0.00842
37     0.50      0.00864
38     0.50      0.00864
39     0.50      0.00864
40     0.50      0.00864
41     0.50      0.00864
42     0.50      0.00864
43     0.50      0.00864
44     0.50      0.00864
45     1.81      0.00865
46     1.91      0.00865
47    52.51      0.00896
48     1.41      0.00933
49     5.72      0.00942
50     2.51      0.00959</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant Expected
1                  oxidation-reduction process       619          94    62.15
2                       cell redox homeostasis        53          15     5.32
3                       platelet degranulation        71          18     7.13
4          glycosaminoglycan catabolic process        40          10     4.02
5                     blood vessel development       393          60    39.46
6                   collagen catabolic process        40          12     4.02
7  positive regulation of axon extension in...         5           4     0.50
8          monosaccharide biosynthetic process        58           9     5.82
9             cholesterol biosynthetic process        56          14     5.62
10           extracellular matrix organization       206          47    20.68
11 negative regulation of establishment of ...       129          17    12.95
12            actin filament network formation         6           4     0.60
13                response to unfolded protein       147          30    14.76
14                            response to drug       217          37    21.79
15                     chondrocyte development        20           7     2.01
16      response to oxygen-containing compound       873         108    87.66
17                collagen fibril organization        31           9     3.11
18 positive regulation of peptidyl-tyrosine...        78          19     7.83
19     PERK-mediated unfolded protein response        16           5     1.61
20               peptidyl-lysine hydroxylation         7           4     0.70
21               collagen biosynthetic process        24           5     2.41
22                               wound healing       311          47    31.23
23             endodermal cell differentiation        32           9     3.21
24                         embryo implantation        33          10     3.31
25                      cell junction assembly       141          17    14.16
26            extracellular matrix disassembly        53          12     5.32
27                        platelet aggregation        33           8     3.31
28      glycosaminoglycan biosynthetic process        74          15     7.43
29              glycoprotein catabolic process        13           5     1.31
30                 response to gamma radiation        43           7     4.32
31                       response to bacterium       204          21    20.48
32 negative regulation of protein kinase B ...        36           9     3.61
33 natural killer cell mediated cytotoxicit...        19           6     1.91
34                   pentose metabolic process         9           4     0.90
35          positive regulation of myelination         9           4     0.90
36 regulation of transcription from RNA pol...         9           4     0.90
37                  glial cell fate commitment         5           3     0.50
38      hydrogen peroxide biosynthetic process         5           3     0.50
39 negative regulation of cytokine producti...         5           3     0.50
40        chemical homeostasis within a tissue         5           3     0.50
41 regulation of store-operated calcium ent...         5           3     0.50
42            zinc II ion transmembrane import         5           3     0.50
43     cellular response to nitrosative stress         5           3     0.50
44       mesenchymal-epithelial cell signaling         5           3     0.50
45    response to mitochondrial depolarisation        18           4     1.81
46                                   mitophagy        19           4     1.91
47                         response to hormone       523          51    52.51
48                     pentose-phosphate shunt        14           5     1.41
49                         response to ethanol        57          12     5.72
50                         O-glycan processing        25           7     2.51
   weightFisher
1       3.2e-07
2       0.00015
3       0.00016
4       0.00018
5       0.00025
6       0.00038
7       0.00046
8       0.00047
9       0.00078
10      0.00094
11      0.00100
12      0.00128
13      0.00146
14      0.00164
15      0.00241
16      0.00255
17      0.00262
18      0.00266
19      0.00276
20      0.00276
21      0.00276
22      0.00299
23      0.00333
24      0.00388
25      0.00440
26      0.00508
27      0.00584
28      0.00621
29      0.00653
30      0.00654
31      0.00663
32      0.00776
33      0.00839
34      0.00842
35      0.00842
36      0.00842
37      0.00864
38      0.00864
39      0.00864
40      0.00864
41      0.00864
42      0.00864
43      0.00864
44      0.00864
45      0.00865
46      0.00865
47      0.00896
48      0.00933
49      0.00942
50      0.00959</code></pre>
</div>
<div id="gene-ontology-analysis-with-topgo-separate-up-and-downregulated-genes" class="section level2">
<h2>Gene ontology analysis with topGO (separate up and downregulated genes)</h2>
<p>Use topGO for GO analysis. It accounts for the nested graph structure of GO terms to prune the number of GO categories tested (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">Alexa et al. 2006</a>). Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe for upregulated genes This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.05
gene_universe_up &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold &amp; results[[&quot;Unstrain&quot;]]$logFC &gt; 0)
gene_universe_up &lt;- factor(gene_universe_up)
names(gene_universe_up) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe_up)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 ENSG00000198744 
              0               0               0               0               0 
ENSG00000228794 
              0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_up,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9640 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13704 GO terms and 31801 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9322 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 4449 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  2 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  5 nodes to be scored    (10 eliminated genes)</code></pre>
<pre><code>
     Level 16:  11 nodes to be scored   (15 eliminated genes)</code></pre>
<pre><code>
     Level 15:  30 nodes to be scored   (47 eliminated genes)</code></pre>
<pre><code>
     Level 14:  59 nodes to be scored   (143 eliminated genes)</code></pre>
<pre><code>
     Level 13:  113 nodes to be scored  (512 eliminated genes)</code></pre>
<pre><code>
     Level 12:  194 nodes to be scored  (1158 eliminated genes)</code></pre>
<pre><code>
     Level 11:  336 nodes to be scored  (2764 eliminated genes)</code></pre>
<pre><code>
     Level 10:  480 nodes to be scored  (3837 eliminated genes)</code></pre>
<pre><code>
     Level 9:   600 nodes to be scored  (5443 eliminated genes)</code></pre>
<pre><code>
     Level 8:   621 nodes to be scored  (6759 eliminated genes)</code></pre>
<pre><code>
     Level 7:   663 nodes to be scored  (7625 eliminated genes)</code></pre>
<pre><code>
     Level 6:   604 nodes to be scored  (8336 eliminated genes)</code></pre>
<pre><code>
     Level 5:   400 nodes to be scored  (8720 eliminated genes)</code></pre>
<pre><code>
     Level 4:   211 nodes to be scored  (8951 eliminated genes)</code></pre>
<pre><code>
     Level 3:   96 nodes to be scored   (9073 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9151 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9213 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated Significant
1  GO:0030574                  collagen catabolic process        40          12
2  GO:0030198           extracellular matrix organization       206          41
3  GO:0030199                collagen fibril organization        31           9
4  GO:0051639            actin filament network formation         6           4
5  GO:0006027         glycosaminoglycan catabolic process        40           9
6  GO:0032964               collagen biosynthetic process        24           5
7  GO:0006024      glycosaminoglycan biosynthetic process        74          13
8  GO:0031643          positive regulation of myelination         9           4
9  GO:0001503                                ossification       238          25
10 GO:0022617            extracellular matrix disassembly        53          10
11 GO:0035987             endodermal cell differentiation        32           7
12 GO:0006768                    biotin metabolic process        10           4
13 GO:0034341                response to interferon-gamma        68           6
14 GO:0060638       mesenchymal-epithelial cell signaling         5           3
15 GO:1900016 negative regulation of cytokine producti...         5           3
16 GO:0016266                         O-glycan processing        25           6
17 GO:0070208                 protein heterotrimerization        11           4
18 GO:0035455                response to interferon-alpha        11           4
19 GO:0042362    fat-soluble vitamin biosynthetic process         7           3
20 GO:0051897 positive regulation of protein kinase B ...        69          10
21 GO:0003009                 skeletal muscle contraction        13           4
22 GO:0031032           actomyosin structure organization       116          12
23 GO:0033692 cellular polysaccharide biosynthetic pro...        41           5
24 GO:0017185               peptidyl-lysine hydroxylation         7           3
25 GO:0007588                                   excretion        15           4
26 GO:0035413 positive regulation of catenin import in...         8           3
27 GO:0007566                         embryo implantation        33           6
28 GO:0034446 substrate adhesion-dependent cell spread...        62           7
29 GO:0048592                           eye morphogenesis        81           9
30 GO:0051450                      myoblast proliferation        10           3
31 GO:0001938 positive regulation of endothelial cell ...        44           7
32 GO:0005975              carbohydrate metabolic process       372          35
   Expected weightFisher
1      2.15      7.3e-07
2     11.09      7.4e-06
3      1.67      2.5e-05
4      0.32      0.00011
5      2.15      0.00019
6      1.29      0.00026
7      3.98      0.00067
8      0.48      0.00084
9     12.82      0.00098
10     2.85      0.00110
11     1.72      0.00131
12     0.54      0.00135
13     3.66      0.00136
14     0.27      0.00143
15     0.27      0.00143
16     1.35      0.00174
17     0.59      0.00202
18     0.59      0.00274
19     0.38      0.00289
20     3.72      0.00363
21     0.70      0.00402
22     6.25      0.00433
23     2.21      0.00461
24     0.38      0.00462
25     0.81      0.00705
26     0.43      0.00709
27     1.78      0.00756
28     3.34      0.00805
29     4.36      0.00828
30     0.54      0.00835
31     2.37      0.00852
32    20.03      0.00955</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant Expected
1                   collagen catabolic process        40          12     2.15
2            extracellular matrix organization       206          41    11.09
3                 collagen fibril organization        31           9     1.67
4             actin filament network formation         6           4     0.32
5          glycosaminoglycan catabolic process        40           9     2.15
6                collagen biosynthetic process        24           5     1.29
7       glycosaminoglycan biosynthetic process        74          13     3.98
8           positive regulation of myelination         9           4     0.48
9                                 ossification       238          25    12.82
10            extracellular matrix disassembly        53          10     2.85
11             endodermal cell differentiation        32           7     1.72
12                    biotin metabolic process        10           4     0.54
13                response to interferon-gamma        68           6     3.66
14       mesenchymal-epithelial cell signaling         5           3     0.27
15 negative regulation of cytokine producti...         5           3     0.27
16                         O-glycan processing        25           6     1.35
17                 protein heterotrimerization        11           4     0.59
18                response to interferon-alpha        11           4     0.59
19    fat-soluble vitamin biosynthetic process         7           3     0.38
20 positive regulation of protein kinase B ...        69          10     3.72
21                 skeletal muscle contraction        13           4     0.70
22           actomyosin structure organization       116          12     6.25
23 cellular polysaccharide biosynthetic pro...        41           5     2.21
24               peptidyl-lysine hydroxylation         7           3     0.38
25                                   excretion        15           4     0.81
26 positive regulation of catenin import in...         8           3     0.43
27                         embryo implantation        33           6     1.78
28 substrate adhesion-dependent cell spread...        62           7     3.34
29                           eye morphogenesis        81           9     4.36
30                      myoblast proliferation        10           3     0.54
31 positive regulation of endothelial cell ...        44           7     2.37
32              carbohydrate metabolic process       372          35    20.03
   weightFisher
1       7.3e-07
2       7.4e-06
3       2.5e-05
4       0.00011
5       0.00019
6       0.00026
7       0.00067
8       0.00084
9       0.00098
10      0.00110
11      0.00131
12      0.00135
13      0.00136
14      0.00143
15      0.00143
16      0.00174
17      0.00202
18      0.00274
19      0.00289
20      0.00363
21      0.00402
22      0.00433
23      0.00461
24      0.00462
25      0.00705
26      0.00709
27      0.00756
28      0.00805
29      0.00828
30      0.00835
31      0.00852
32      0.00955</code></pre>
<p>First create the gene universe for downregulated genes This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.05
gene_universe_down &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold &amp; results[[&quot;Unstrain&quot;]]$logFC &lt; 0)
gene_universe_down &lt;- factor(gene_universe_down)
names(gene_universe_down) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe_down)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 ENSG00000198744 
              0               0               0               0               0 
ENSG00000228794 
              0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_down,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9640 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13704 GO terms and 31801 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9322 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 4805 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  3 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  10 nodes to be scored   (10 eliminated genes)</code></pre>
<pre><code>
     Level 16:  16 nodes to be scored   (16 eliminated genes)</code></pre>
<pre><code>
     Level 15:  40 nodes to be scored   (58 eliminated genes)</code></pre>
<pre><code>
     Level 14:  90 nodes to be scored   (176 eliminated genes)</code></pre>
<pre><code>
     Level 13:  141 nodes to be scored  (464 eliminated genes)</code></pre>
<pre><code>
     Level 12:  225 nodes to be scored  (1333 eliminated genes)</code></pre>
<pre><code>
     Level 11:  378 nodes to be scored  (2802 eliminated genes)</code></pre>
<pre><code>
     Level 10:  522 nodes to be scored  (3840 eliminated genes)</code></pre>
<pre><code>
     Level 9:   639 nodes to be scored  (5366 eliminated genes)</code></pre>
<pre><code>
     Level 8:   644 nodes to be scored  (6708 eliminated genes)</code></pre>
<pre><code>
     Level 7:   716 nodes to be scored  (7647 eliminated genes)</code></pre>
<pre><code>
     Level 6:   626 nodes to be scored  (8320 eliminated genes)</code></pre>
<pre><code>
     Level 5:   405 nodes to be scored  (8720 eliminated genes)</code></pre>
<pre><code>
     Level 4:   229 nodes to be scored  (8954 eliminated genes)</code></pre>
<pre><code>
     Level 3:   98 nodes to be scored   (9081 eliminated genes)</code></pre>
<pre><code>
     Level 2:   21 nodes to be scored   (9149 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9213 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated Significant
1  GO:0055114                 oxidation-reduction process       619          47
2  GO:0006695            cholesterol biosynthetic process        56          14
3  GO:0046364         monosaccharide biosynthetic process        58           6
4  GO:0036499     PERK-mediated unfolded protein response        16           5
5  GO:0006986                response to unfolded protein       147          19
6  GO:1900034     regulation of cellular response to heat        72          11
7  GO:0019321                   pentose metabolic process         9           4
8  GO:0071578            zinc II ion transmembrane import         5           3
9  GO:0071500     cellular response to nitrosative stress         5           3
10 GO:0043066    negative regulation of apoptotic process       520          47
11 GO:2001032 regulation of double-strand break repair...        11           4
12 GO:0006974    cellular response to DNA damage stimulus       608          42
13 GO:1905323      telomerase holoenzyme complex assembly         6           3
14 GO:1903747 regulation of establishment of protein l...        83           6
15 GO:0045454                      cell redox homeostasis        53           8
16 GO:0030511 positive regulation of transforming grow...        22           5
17 GO:0090084 negative regulation of inclusion body as...         7           3
18 GO:0070389 chaperone cofactor-dependent protein ref...         7           3
19 GO:0006098                     pentose-phosphate shunt        14           4
20 GO:0051131 chaperone-mediated protein complex assem...        15           4
21 GO:0045540 regulation of cholesterol biosynthetic p...        39           7
22 GO:0019430              removal of superoxide radicals        14           4
23 GO:0061179 negative regulation of insulin secretion...         8           3
24 GO:0001946                           lymphangiogenesis         8           3
25 GO:0002576                      platelet degranulation        71           9
26 GO:0051898 negative regulation of protein kinase B ...        36           6
27 GO:1904950 negative regulation of establishment of ...       129          10
28 GO:0050857 positive regulation of antigen receptor-...        10           3
29 GO:0050731 positive regulation of peptidyl-tyrosine...        78          12
30 GO:1901214                  regulation of neuron death       174          18
31 GO:0032874 positive regulation of stress-activated ...        92          12
32 GO:0061098 positive regulation of protein tyrosine ...        22           5
33 GO:0043619 regulation of transcription from RNA pol...         9           3
34 GO:0001934 positive regulation of protein phosphory...       528          48
35 GO:0071158    positive regulation of cell cycle arrest        71           6
36 GO:0008631 intrinsic apoptotic signaling pathway in...        35           6
37 GO:0046717                              acid secretion        36           6
38 GO:0008299             isoprenoid biosynthetic process        15           4
39 GO:0006950                          response to stress      2182         145
   Expected weightFisher
1     28.82      1.8e-06
2      2.61      5.4e-06
3      2.70      2.2e-05
4      0.74      0.00014
5      6.84      0.00026
6      3.35      0.00045
7      0.42      0.00048
8      0.23      0.00093
9      0.23      0.00093
10    24.21      0.00113
11     0.51      0.00118
12    28.31      0.00136
13     0.28      0.00180
14     3.86      0.00216
15     2.47      0.00290
16     1.02      0.00291
17     0.33      0.00305
18     0.33      0.00305
19     0.65      0.00319
20     0.70      0.00420
21     1.82      0.00432
22     0.65      0.00469
23     0.37      0.00471
24     0.37      0.00471
25     3.31      0.00542
26     1.68      0.00583
27     6.01      0.00623
28     0.47      0.00627
29     3.63      0.00640
30     8.10      0.00650
31     4.28      0.00660
32     1.02      0.00676
33     0.42      0.00682
34    24.58      0.00692
35     3.31      0.00846
36     1.63      0.00930
37     1.68      0.00931
38     0.70      0.00937
39   101.59      0.00947</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant Expected
1                  oxidation-reduction process       619          47    28.82
2             cholesterol biosynthetic process        56          14     2.61
3          monosaccharide biosynthetic process        58           6     2.70
4      PERK-mediated unfolded protein response        16           5     0.74
5                 response to unfolded protein       147          19     6.84
6      regulation of cellular response to heat        72          11     3.35
7                    pentose metabolic process         9           4     0.42
8             zinc II ion transmembrane import         5           3     0.23
9      cellular response to nitrosative stress         5           3     0.23
10    negative regulation of apoptotic process       520          47    24.21
11 regulation of double-strand break repair...        11           4     0.51
12    cellular response to DNA damage stimulus       608          42    28.31
13      telomerase holoenzyme complex assembly         6           3     0.28
14 regulation of establishment of protein l...        83           6     3.86
15                      cell redox homeostasis        53           8     2.47
16 positive regulation of transforming grow...        22           5     1.02
17 negative regulation of inclusion body as...         7           3     0.33
18 chaperone cofactor-dependent protein ref...         7           3     0.33
19                     pentose-phosphate shunt        14           4     0.65
20 chaperone-mediated protein complex assem...        15           4     0.70
21 regulation of cholesterol biosynthetic p...        39           7     1.82
22              removal of superoxide radicals        14           4     0.65
23 negative regulation of insulin secretion...         8           3     0.37
24                           lymphangiogenesis         8           3     0.37
25                      platelet degranulation        71           9     3.31
26 negative regulation of protein kinase B ...        36           6     1.68
27 negative regulation of establishment of ...       129          10     6.01
28 positive regulation of antigen receptor-...        10           3     0.47
29 positive regulation of peptidyl-tyrosine...        78          12     3.63
30                  regulation of neuron death       174          18     8.10
31 positive regulation of stress-activated ...        92          12     4.28
32 positive regulation of protein tyrosine ...        22           5     1.02
33 regulation of transcription from RNA pol...         9           3     0.42
34 positive regulation of protein phosphory...       528          48    24.58
35    positive regulation of cell cycle arrest        71           6     3.31
36 intrinsic apoptotic signaling pathway in...        35           6     1.63
37                              acid secretion        36           6     1.68
38             isoprenoid biosynthetic process        15           4     0.70
39                          response to stress      2182         145   101.59
   weightFisher
1       1.8e-06
2       5.4e-06
3       2.2e-05
4       0.00014
5       0.00026
6       0.00045
7       0.00048
8       0.00093
9       0.00093
10      0.00113
11      0.00118
12      0.00136
13      0.00180
14      0.00216
15      0.00290
16      0.00291
17      0.00305
18      0.00305
19      0.00319
20      0.00420
21      0.00432
22      0.00469
23      0.00471
24      0.00471
25      0.00542
26      0.00583
27      0.00623
28      0.00627
29      0.00640
30      0.00650
31      0.00660
32      0.00676
33      0.00682
34      0.00692
35      0.00846
36      0.00930
37      0.00931
38      0.00937
39      0.00947</code></pre>
</div>
</div>
<div id="perform-de-analysis-using-edger" class="section level1">
<h1>Perform DE analysis using edgeR</h1>
<pre class="r"><code>#Subset RUVsOut for only one of each of the samples (no technical replicates)
RUVsSubset &lt;- RUVsOut[, RUVsOut$Sample_ID %in% selected_samples]


design &lt;- model.matrix(~treatment + W_1 + W_2 + RIN, data=pData(RUVsSubset))
y &lt;- DGEList(counts=counts(RUVsSubset), group=pData(RUVsSubset)$Individual)
y &lt;- calcNormFactors(y, method=&quot;upperquartile&quot;)
y &lt;- estimateGLMCommonDisp(y, design)
y &lt;- estimateGLMTagwiseDisp(y, design)
fit &lt;- glmFit(y, design)
lrt &lt;- glmLRT(fit, coef=2)
gene_values &lt;- topTags(lrt, n = nrow(lrt$coefficients), adjust.method=&quot;BH&quot;, sort.by = &quot;PValue&quot;)</code></pre>
<div id="explore-top-hits-1" class="section level2">
<h2>Explore top hits</h2>
<p>Boxplot function.</p>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An df containing log2 counts per million
  # g - character vector of a single gene
  library(&quot;tidyr&quot;)
  single_gene &lt;- v[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))

  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
    geom_boxplot() +
    labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;))
}

plot_volcano &lt;- function(x, qval) {
  # Create volcano plot.
  #
  # x - data frame with topTable output
  #     (columns logFC, PValue, and FDR)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(PValue), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, qval) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = PValue, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}</code></pre>
</div>
<div id="box-plots-of-top-genes" class="section level2">
<h2>Box plots of top genes</h2>
<pre class="r"><code>cpm &lt;- cpm(filt_counts, log = T)

# top genes by p-value

#rename top_treatment genes to gene symbols for easier interpretation
top_treatment_top10 &lt;- gene_anno$external_gene_name[match(head(rownames(gene_values@.Data[[1]]), n=10), gene_anno$ensembl_gene_id)]
top_treatment_top10</code></pre>
<pre><code> [1] LINC01013 APLNR     TBX1      CTSF      ACTC1     EGR1      WNT10A   
 [8] NQO1      HMOX1     EXTL1    
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[1])
p_treatment_2 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[2]) 
p_treatment_3 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[3]) 
p_treatment_4 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[4]) 
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)
plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<p><img src="figure/DE_analysis.Rmd/boxplots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-boxplots-1">
Past versions of boxplots-1.png
</button>
</p>
<div id="fig-boxplots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a3565148512e79e1482c774b845a50b8c9c06e74/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plots-with-significant-genes-highlighted" class="section level2">
<h2>Plots with significant genes highlighted</h2>
<pre class="r"><code>thresh &lt;- 0.01
plot_volcano(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-plotting-functions-1">
Past versions of “use plotting functions-1.png”
</button>
</p>
<div id="fig-no-spaces-use-plotting-functions-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a3565148512e79e1482c774b845a50b8c9c06e74/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-plotting-functions-2">
Past versions of “use plotting functions-2.png”
</button>
</p>
<div id="fig-no-spaces-use-plotting-functions-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use plotting functions-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use plotting functions-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(gene_values@.Data[[1]]$FDR &lt; thresh)</code></pre>
<pre><code>
FALSE  TRUE 
10241   206 </code></pre>
</div>
<div id="gene-ontology-analysis-with-topgo-1" class="section level2">
<h2>Gene ontology analysis with topGO</h2>
<p>Use topGO for GO analysis. It accounts for the nested graph structure of GO terms to prune the number of GO categories tested (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">Alexa et al. 2006</a>). Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe. This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.01
gene_universe &lt;- as.numeric(gene_values@.Data[[1]]$FDR &lt; threshold)
gene_universe &lt;- factor(gene_universe)
names(gene_universe) &lt;- rownames(gene_values@.Data[[1]])
head(gene_universe)</code></pre>
<pre><code>ENSG00000228495 ENSG00000134817 ENSG00000184058 ENSG00000174080 ENSG00000159251 
              1               1               1               1               1 
ENSG00000120738 
              1 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9640 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13704 GO terms and 31801 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9322 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 3289 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  3 nodes to be scored    (10 eliminated genes)</code></pre>
<pre><code>
     Level 16:  5 nodes to be scored    (14 eliminated genes)</code></pre>
<pre><code>
     Level 15:  11 nodes to be scored   (40 eliminated genes)</code></pre>
<pre><code>
     Level 14:  32 nodes to be scored   (88 eliminated genes)</code></pre>
<pre><code>
     Level 13:  64 nodes to be scored   (243 eliminated genes)</code></pre>
<pre><code>
     Level 12:  107 nodes to be scored  (775 eliminated genes)</code></pre>
<pre><code>
     Level 11:  204 nodes to be scored  (2265 eliminated genes)</code></pre>
<pre><code>
     Level 10:  309 nodes to be scored  (3221 eliminated genes)</code></pre>
<pre><code>
     Level 9:   402 nodes to be scored  (4625 eliminated genes)</code></pre>
<pre><code>
     Level 8:   466 nodes to be scored  (6168 eliminated genes)</code></pre>
<pre><code>
     Level 7:   530 nodes to be scored  (7174 eliminated genes)</code></pre>
<pre><code>
     Level 6:   489 nodes to be scored  (8139 eliminated genes)</code></pre>
<pre><code>
     Level 5:   353 nodes to be scored  (8660 eliminated genes)</code></pre>
<pre><code>
     Level 4:   194 nodes to be scored  (8938 eliminated genes)</code></pre>
<pre><code>
     Level 3:   95 nodes to be scored   (9078 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9142 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9213 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated Significant
1  GO:0045540 regulation of cholesterol biosynthetic p...        39          10
2  GO:0048662 negative regulation of smooth muscle cel...        22           6
3  GO:0007588                                   excretion        15           5
4  GO:0000305                  response to oxygen radical        21           4
5  GO:0008299             isoprenoid biosynthetic process        15           6
6  GO:0036066               protein O-linked fucosylation        11           4
7  GO:0007586                                   digestion        31           5
8  GO:0021520 spinal cord motor neuron cell fate speci...         6           3
9  GO:0055114                 oxidation-reduction process       619          27
10 GO:0006695            cholesterol biosynthetic process        56          14
11 GO:0007171 activation of transmembrane receptor pro...         7           3
12 GO:0035094                        response to nicotine        18           4
13 GO:0071636 positive regulation of transforming grow...        10           3
14 GO:0007157 heterophilic cell-cell adhesion via plas...        23           4
15 GO:0098869             cellular oxidant detoxification        54           6
16 GO:0061082       myeloid leukocyte cytokine production        14           4
17 GO:0009611                        response to wounding       365          13
18 GO:0042474                    middle ear morphogenesis        12           3
19 GO:0010628      positive regulation of gene expression      1166          32
20 GO:0002686 negative regulation of leukocyte migrati...        13           3
21 GO:0009954           proximal/distal pattern formation        13           3
22 GO:0051897 positive regulation of protein kinase B ...        69           6
23 GO:0090130                            tissue migration       172           6
24 GO:0030325                   adrenal gland development        14           3
25 GO:0050922           negative regulation of chemotaxis        14           3
26 GO:0001649                  osteoblast differentiation       146           8
27 GO:0008543 fibroblast growth factor receptor signal...        71           7
28 GO:2001223     negative regulation of neuron migration         5           2
29 GO:0048265                            response to pain         5           2
30 GO:0021781                  glial cell fate commitment         5           2
31 GO:0045080 positive regulation of chemokine biosynt...         5           2
32 GO:0016114              terpenoid biosynthetic process         5           2
33 GO:0061081 positive regulation of myeloid leukocyte...         5           2
34 GO:0070661                     leukocyte proliferation       113           4
35 GO:0030154                        cell differentiation      2089          68
36 GO:0045471                         response to ethanol        57           5
37 GO:0030198           extracellular matrix organization       206          11
38 GO:0072132                    mesenchyme morphogenesis        33           5
39 GO:0048791 calcium ion-regulated exocytosis of neur...         6           2
40 GO:0006069                           ethanol oxidation         6           2
41 GO:0097384         cellular lipid biosynthetic process         6           2
42 GO:2001053 regulation of mesenchymal cell apoptotic...         6           2
43 GO:0007584                        response to nutrient       100           8
44 GO:0060021                          palate development        63           5
45 GO:0051048            negative regulation of secretion        91           7
46 GO:0001658 branching involved in ureteric bud morph...        33           4
47 GO:0010463              mesenchymal cell proliferation        32           4
48 GO:0043586                          tongue development        12           3
49 GO:0019432           triglyceride biosynthetic process        16           3
50 GO:0043030         regulation of macrophage activation        17           3
51 GO:0007271          synaptic transmission, cholinergic         7           2
52 GO:0043568 positive regulation of insulin-like grow...         7           2
53 GO:0023019 signal transduction involved in regulati...         7           2
54 GO:0045766         positive regulation of angiogenesis        77           6
55 GO:0001558                   regulation of cell growth       244          12
56 GO:0051781        positive regulation of cell division        43           4
57 GO:0030326                embryonic limb morphogenesis        91           5
58 GO:0060415                 muscle tissue morphogenesis        46           4
59 GO:0006084                acetyl-CoA metabolic process        26           3
60 GO:0010984 regulation of lipoprotein particle clear...         8           2
61 GO:0010935 regulation of macrophage cytokine produc...         8           2
62 GO:0043931    ossification involved in bone maturation         8           2
   Expected weightFisher
1      0.74      1.9e-09
2      0.42      4.0e-06
3      0.28      6.0e-06
4      0.40      6.7e-06
5      0.28      2.3e-05
6      0.21      3.7e-05
7      0.59      0.00013
8      0.11      0.00013
9     11.75      0.00017
10     1.06      0.00020
11     0.13      0.00022
12     0.34      0.00031
13     0.19      0.00073
14     0.44      0.00084
15     1.03      0.00088
16     0.27      0.00104
17     6.93      0.00110
18     0.23      0.00131
19    22.14      0.00146
20     0.25      0.00167
21     0.25      0.00167
22     1.31      0.00192
23     3.27      0.00208
24     0.27      0.00210
25     0.27      0.00210
26     2.77      0.00218
27     1.35      0.00308
28     0.09      0.00345
29     0.09      0.00345
30     0.09      0.00345
31     0.09      0.00345
32     0.09      0.00345
33     0.09      0.00345
34     2.15      0.00345
35    39.66      0.00350
36     1.08      0.00440
37     3.91      0.00446
38     0.63      0.00497
39     0.11      0.00511
40     0.11      0.00511
41     0.11      0.00511
42     0.11      0.00511
43     1.90      0.00581
44     1.20      0.00674
45     1.73      0.00681
46     0.63      0.00688
47     0.61      0.00695
48     0.23      0.00700
49     0.30      0.00701
50     0.32      0.00701
51     0.13      0.00707
52     0.13      0.00707
53     0.13      0.00707
54     1.46      0.00708
55     4.63      0.00867
56     0.82      0.00871
57     1.73      0.00891
58     0.87      0.00918
59     0.49      0.00924
60     0.15      0.00931
61     0.15      0.00931
62     0.15      0.00931</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant Expected
1  regulation of cholesterol biosynthetic p...        39          10     0.74
2  negative regulation of smooth muscle cel...        22           6     0.42
3                                    excretion        15           5     0.28
4                   response to oxygen radical        21           4     0.40
5              isoprenoid biosynthetic process        15           6     0.28
6                protein O-linked fucosylation        11           4     0.21
7                                    digestion        31           5     0.59
8  spinal cord motor neuron cell fate speci...         6           3     0.11
9                  oxidation-reduction process       619          27    11.75
10            cholesterol biosynthetic process        56          14     1.06
11 activation of transmembrane receptor pro...         7           3     0.13
12                        response to nicotine        18           4     0.34
13 positive regulation of transforming grow...        10           3     0.19
14 heterophilic cell-cell adhesion via plas...        23           4     0.44
15             cellular oxidant detoxification        54           6     1.03
16       myeloid leukocyte cytokine production        14           4     0.27
17                        response to wounding       365          13     6.93
18                    middle ear morphogenesis        12           3     0.23
19      positive regulation of gene expression      1166          32    22.14
20 negative regulation of leukocyte migrati...        13           3     0.25
21           proximal/distal pattern formation        13           3     0.25
22 positive regulation of protein kinase B ...        69           6     1.31
23                            tissue migration       172           6     3.27
24                   adrenal gland development        14           3     0.27
25           negative regulation of chemotaxis        14           3     0.27
26                  osteoblast differentiation       146           8     2.77
27 fibroblast growth factor receptor signal...        71           7     1.35
28     negative regulation of neuron migration         5           2     0.09
29                            response to pain         5           2     0.09
30                  glial cell fate commitment         5           2     0.09
31 positive regulation of chemokine biosynt...         5           2     0.09
32              terpenoid biosynthetic process         5           2     0.09
33 positive regulation of myeloid leukocyte...         5           2     0.09
34                     leukocyte proliferation       113           4     2.15
35                        cell differentiation      2089          68    39.66
36                         response to ethanol        57           5     1.08
37           extracellular matrix organization       206          11     3.91
38                    mesenchyme morphogenesis        33           5     0.63
39 calcium ion-regulated exocytosis of neur...         6           2     0.11
40                           ethanol oxidation         6           2     0.11
41         cellular lipid biosynthetic process         6           2     0.11
42 regulation of mesenchymal cell apoptotic...         6           2     0.11
43                        response to nutrient       100           8     1.90
44                          palate development        63           5     1.20
45            negative regulation of secretion        91           7     1.73
46 branching involved in ureteric bud morph...        33           4     0.63
47              mesenchymal cell proliferation        32           4     0.61
48                          tongue development        12           3     0.23
49           triglyceride biosynthetic process        16           3     0.30
50         regulation of macrophage activation        17           3     0.32
51          synaptic transmission, cholinergic         7           2     0.13
52 positive regulation of insulin-like grow...         7           2     0.13
53 signal transduction involved in regulati...         7           2     0.13
54         positive regulation of angiogenesis        77           6     1.46
55                   regulation of cell growth       244          12     4.63
56        positive regulation of cell division        43           4     0.82
57                embryonic limb morphogenesis        91           5     1.73
58                 muscle tissue morphogenesis        46           4     0.87
59                acetyl-CoA metabolic process        26           3     0.49
60 regulation of lipoprotein particle clear...         8           2     0.15
61 regulation of macrophage cytokine produc...         8           2     0.15
62    ossification involved in bone maturation         8           2     0.15
   weightFisher
1       1.9e-09
2       4.0e-06
3       6.0e-06
4       6.7e-06
5       2.3e-05
6       3.7e-05
7       0.00013
8       0.00013
9       0.00017
10      0.00020
11      0.00022
12      0.00031
13      0.00073
14      0.00084
15      0.00088
16      0.00104
17      0.00110
18      0.00131
19      0.00146
20      0.00167
21      0.00167
22      0.00192
23      0.00208
24      0.00210
25      0.00210
26      0.00218
27      0.00308
28      0.00345
29      0.00345
30      0.00345
31      0.00345
32      0.00345
33      0.00345
34      0.00345
35      0.00350
36      0.00440
37      0.00446
38      0.00497
39      0.00511
40      0.00511
41      0.00511
42      0.00511
43      0.00581
44      0.00674
45      0.00681
46      0.00688
47      0.00695
48      0.00700
49      0.00701
50      0.00701
51      0.00707
52      0.00707
53      0.00707
54      0.00708
55      0.00867
56      0.00871
57      0.00891
58      0.00918
59      0.00924
60      0.00931
61      0.00931
62      0.00931</code></pre>
<p>There are 62 significant results.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] stats4    parallel  grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] org.Hs.eg.db_3.5.0         topGO_2.30.1              
 [3] SparseM_1.77               GO.db_3.5.0               
 [5] AnnotationDbi_1.40.0       graph_1.56.0              
 [7] RUVSeq_1.12.0              EDASeq_2.12.0             
 [9] ShortRead_1.36.1           GenomicAlignments_1.14.2  
[11] SummarizedExperiment_1.8.1 DelayedArray_0.4.1        
[13] matrixStats_0.55.0         Rsamtools_1.30.0          
[15] GenomicRanges_1.30.3       GenomeInfoDb_1.14.0       
[17] Biostrings_2.46.0          XVector_0.18.0            
[19] IRanges_2.12.0             S4Vectors_0.16.0          
[21] BiocParallel_1.12.0        Biobase_2.38.0            
[23] BiocGenerics_0.24.0        VennDiagram_1.6.20        
[25] futile.logger_1.4.3        cowplot_0.9.3             
[27] ggplot2_3.2.1              ashr_2.2-47               
[29] tidyr_1.0.0                dplyr_0.8.3               
[31] edgeR_3.20.9               plyr_1.8.4                
[33] limma_3.34.9              

loaded via a namespace (and not attached):
 [1] colorspace_1.4-1       hwriter_1.3.2          ellipsis_0.3.0        
 [4] rprojroot_1.3-2        fs_1.3.1               farver_2.0.3          
 [7] bit64_0.9-7            splines_3.4.3          R.methodsS3_1.7.1     
[10] DESeq_1.30.0           geneplotter_1.56.0     knitr_1.26            
[13] zeallot_0.1.0          workflowr_1.5.0        annotate_1.56.2       
[16] R.oo_1.23.0            compiler_3.4.3         httr_1.4.1            
[19] backports_1.1.5        assertthat_0.2.1       Matrix_1.2-18         
[22] lazyeval_0.2.2         later_1.0.0            formatR_1.7           
[25] htmltools_0.4.0        prettyunits_1.0.2      tools_3.4.3           
[28] gtable_0.3.0           glue_1.3.1             GenomeInfoDbData_1.0.0
[31] Rcpp_1.0.3             vctrs_0.2.0            rtracklayer_1.38.3    
[34] xfun_0.12              stringr_1.4.0          lifecycle_0.1.0       
[37] irlba_2.3.3            statmod_1.4.32         XML_3.98-1.20         
[40] zlibbioc_1.24.0        MASS_7.3-51.4          scales_1.1.0          
[43] aroma.light_3.8.0      hms_0.5.2              promises_1.1.0        
[46] lambda.r_1.2.4         RColorBrewer_1.1-2     yaml_2.2.0            
[49] memoise_1.1.0          biomaRt_2.34.2         latticeExtra_0.6-28   
[52] stringi_1.4.3          RSQLite_2.1.4          SQUAREM_2020.2        
[55] genefilter_1.60.0      RMySQL_0.10.17         GenomicFeatures_1.30.3
[58] truncnorm_1.0-8        rlang_0.4.4            pkgconfig_2.0.3       
[61] bitops_1.0-6           evaluate_0.14          lattice_0.20-35       
[64] invgamma_1.1           purrr_0.3.3            labeling_0.3          
[67] bit_1.1-14             tidyselect_0.2.5       magrittr_1.5          
[70] R6_2.4.1               DBI_1.0.0              pillar_1.4.2          
[73] whisker_0.4            withr_2.1.2            survival_2.41-3       
[76] RCurl_1.95-4.12        mixsqp_0.3-17          tibble_2.1.3          
[79] crayon_1.3.4           futile.options_1.0.1   rmarkdown_1.18        
[82] progress_1.2.2         locfit_1.5-9.1         blob_1.2.0            
[85] git2r_0.26.1           etrunct_0.1            digest_0.6.23         
[88] xtable_1.8-4           httpuv_1.5.2           R.utils_2.9.2         
[91] munsell_0.5.0         </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
