<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>DE_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">DE_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-01-06
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreed80f643520bd77f37ac3f3862b7b747bccdbba30targetblankd80f643a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/d80f643520bd77f37ac3f3862b7b747bccdbba30" target="_blank">d80f643</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreed80f643520bd77f37ac3f3862b7b747bccdbba30targetblankd80f643a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/d80f643520bd77f37ac3f3862b7b747bccdbba30/analysis/DE_analysis.Rmd" target="_blank">d80f643</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Run Limma:
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/DE_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/93017062e918d7a60fcad8ee8ca0983de6e9aef7/analysis/DE_analysis.Rmd" target="_blank">9301706</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-17
</td>
<td>
build analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/DE_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/DE_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<pre class="r"><code>library(&quot;limma&quot;)
library(&quot;plyr&quot;)
library(&quot;edgeR&quot;)
library(&quot;dplyr&quot;)</code></pre>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:plyr&#39;:

    arrange, count, desc, failwith, id, mutate, rename, summarise,
    summarize</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code>library(&quot;tidyr&quot;)
library(&quot;ashr&quot;)
library(&quot;ggplot2&quot;)
library(&quot;cowplot&quot;)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>library(&quot;VennDiagram&quot;)</code></pre>
<pre><code>Loading required package: grid</code></pre>
<pre><code>Loading required package: futile.logger</code></pre>
<pre class="r"><code>library(&quot;RUVSeq&quot;)</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    combine, intersect, setdiff, union</code></pre>
<pre><code>The following object is masked from &#39;package:limma&#39;:

    plotMA</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, cbind, colMeans, colnames,
    colSums, do.call, duplicated, eval, evalq, Filter, Find, get, grep,
    grepl, intersect, is.unsorted, lapply, lengths, Map, mapply, match,
    mget, order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,
    rbind, Reduce, rowMeans, rownames, rowSums, sapply, setdiff, sort,
    table, tapply, union, unique, unsplit, which, which.max, which.min</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: EDASeq</code></pre>
<pre><code>Loading required package: ShortRead</code></pre>
<pre><code>Loading required package: BiocParallel</code></pre>
<pre><code>Loading required package: Biostrings</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    expand</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    first, rename</code></pre>
<pre><code>The following object is masked from &#39;package:plyr&#39;:

    rename</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>
Attaching package: &#39;IRanges&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    collapse, desc, slice</code></pre>
<pre><code>The following object is masked from &#39;package:plyr&#39;:

    desc</code></pre>
<pre><code>Loading required package: XVector</code></pre>
<pre><code>
Attaching package: &#39;XVector&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:plyr&#39;:

    compact</code></pre>
<pre><code>
Attaching package: &#39;Biostrings&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    strsplit</code></pre>
<pre><code>Loading required package: Rsamtools</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: GenomicAlignments</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    count</code></pre>
<pre><code>The following object is masked from &#39;package:plyr&#39;:

    count</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following object is masked from &#39;package:Biostrings&#39;:

    type</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    apply</code></pre>
<pre><code>
Attaching package: &#39;GenomicAlignments&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    last</code></pre>
<pre><code>
Attaching package: &#39;ShortRead&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    id</code></pre>
<pre><code>The following object is masked from &#39;package:plyr&#39;:

    id</code></pre>
<pre class="r"><code>#load in filtered count data, RUVs output
filt_counts &lt;- readRDS(&quot;data/filtered_counts.rds&quot;)
RUVsCPM &lt;- readRDS(&quot;data/RUVsCPM.rds&quot;)</code></pre>
</div>
<div id="limma-doesnt-deal-with-technical-replicates-so-here-i-randomly-sample-1-replicate-for-each-of-the-6-individualxcondition" class="section level1">
<h1>Limma doesn’t deal with technical replicates, so here I randomly sample 1 replicate for each of the 6 individualxcondition</h1>
<pre class="r"><code>#randomly sample 1 replicate from each set of 3 replicates
individuals &lt;- c(&quot;18855&quot;, &quot;18856&quot;, &quot;19160&quot;)
treatments &lt;- c(&quot;S&quot;, &quot;U&quot;)

selected_samples &lt;- c()
for(i in individuals){
     for(t in treatments){
          sample_num &lt;- base::sample(c(1:3), 1)
          selected_samples &lt;- c(selected_samples, paste0(i, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, t))
     }
}
selected_samples</code></pre>
<pre><code>[1] &quot;18855_2_S&quot; &quot;18855_1_U&quot; &quot;18856_2_S&quot; &quot;18856_1_U&quot; &quot;19160_2_S&quot; &quot;19160_3_U&quot;</code></pre>
<pre class="r"><code>randomly_sampled_indices &lt;- match(selected_samples, names(filt_counts))

single_filt_counts &lt;- filt_counts[,randomly_sampled_indices]
anno &lt;- pData(RUVsCPM)[randomly_sampled_indices,]</code></pre>
</div>
<div id="model-with-limma" class="section level1">
<h1>Model with limma</h1>
<pre class="r"><code>#sample details
#anno &lt;- pData(RUVsCPM)

anno$LibraryPrepBatch &lt;- factor(anno$LibraryPrepBatch, levels = c(&quot;1&quot;, &quot;2&quot;))
anno$Replicate &lt;- factor(anno$Replicate, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;))

design &lt;- model.matrix(~treatment + W_1,
                       data = anno)
colnames(design)[1] &lt;- &quot;Intercept&quot;
colnames(design) &lt;- gsub(&quot;treatment&quot;, &quot;&quot;, colnames(design))
colnames(design)</code></pre>
<pre><code>[1] &quot;Intercept&quot; &quot;Unstrain&quot;  &quot;W_1&quot;      </code></pre>
<pre class="r"><code>#TMM Normalization
y &lt;- DGEList(single_filt_counts)
y &lt;- calcNormFactors(y)
#Voom for differential expression
v &lt;- voom(y, design)


# Model individual as a random effect (use duplicate correlation function); treatment as a fixed effect, covariates = RUV factors

corfit &lt;- duplicateCorrelation(v, design, block = anno$Individual)</code></pre>
<pre><code>Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable

Warning in glmgam.fit(dx, dy, coef.start = start, tol = tol, maxit = maxit, :
Too much damping - convergence tolerance not achievable</code></pre>
<pre><code>Warning in atanh(pmax(-1, rho)): NaNs produced</code></pre>
<pre class="r"><code>corfit$consensus</code></pre>
<pre><code>[1] -0.7703515</code></pre>
<pre class="r"><code>v &lt;- voom(y, design, block = anno$Individual, correlation = corfit$consensus)
fit &lt;- lmFit(v, design, block = anno$Individual, correlation = corfit$consensus)
fit &lt;- eBayes(fit)</code></pre>
<div id="assess-model-results" class="section level2">
<h2>Assess model results</h2>
<pre class="r"><code>top_treatment &lt;- topTable(fit, coef = &quot;Unstrain&quot;)
top_treatment</code></pre>
<pre><code>                    logFC  AveExpr          t      P.Value    adj.P.Val
ENSG00000130396 -3.190048 7.463676  -71.22179 3.568918e-07 0.0006526195
ENSG00000198793  2.328920 7.364266  128.55867 3.641173e-08 0.0002781795
ENSG00000065559  2.328920 7.364266  128.55867 3.641173e-08 0.0002781795
ENSG00000182718 -2.315307 7.364266 -111.52436 6.307558e-08 0.0002781795
ENSG00000169750 -2.315307 7.364266 -111.52436 6.307558e-08 0.0002781795
ENSG00000198160  1.592533 6.995783   92.74844 1.286214e-07 0.0002781795
ENSG00000068001  1.592533 6.995783   92.74844 1.286214e-07 0.0002781795
ENSG00000114126  1.592533 6.995783   92.74844 1.286214e-07 0.0002781795
ENSG00000114209  1.592533 6.995783   92.74844 1.286214e-07 0.0002781795
ENSG00000138772  1.592533 6.995783   92.74844 1.286214e-07 0.0002781795
                       B
ENSG00000130396 5.367684
ENSG00000198793 5.129181
ENSG00000065559 5.129181
ENSG00000182718 5.060726
ENSG00000169750 5.060726
ENSG00000198160 4.664514
ENSG00000068001 4.664514
ENSG00000114126 4.664514
ENSG00000114209 4.664514
ENSG00000138772 4.664514</code></pre>
<pre class="r"><code>results_treatment &lt;- topTable(fit, coef = &quot;Unstrain&quot;,
                              number = nrow(filt_counts), sort = &quot;none&quot;)
ma_treatment &lt;- ggplot(data.frame(Amean = fit$Amean, logFC = fit$coef[, &quot;Unstrain&quot;]),
                       aes(x = Amean, y = logFC)) +
  geom_point() +
  labs(x = &quot;Average expression level&quot;, y = &quot;Log fold change&quot;,
       title = &quot;Treatment effect&quot;)
ma_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>hist_treatment &lt;- ggplot(results_treatment, aes(x = P.Value)) +
  geom_histogram(binwidth = 0.01) +
  labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;, title = &quot;Treatment effect&quot;)
hist_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="explore-top-hits" class="section level2">
<h2>Explore top hits</h2>
<p>Boxplot function.</p>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An EList object containing log2 counts per million
  # g - character vector of a single gene
  stopifnot(class(v) == &quot;EList&quot;,
            is.character(g), length(g) == 1)
  library(&quot;tidyr&quot;)
  single_gene &lt;- v$E[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))

  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
    geom_boxplot() +
    labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;))
}</code></pre>
<p>Treatment</p>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(v, rownames(top_treatment)[1]) +
  ylim(3, 11) + theme(legend.position = &quot;bottom&quot;)
p_treatment_2 &lt;- plot_gene(v, rownames(top_treatment)[2]) +
  ylim(3, 11) + theme(legend.position = &quot;bottom&quot;)
p_treatment_3 &lt;- plot_gene(v, rownames(top_treatment)[3]) +
  ylim(3, 11) + theme(legend.position = &quot;bottom&quot;)
p_treatment_4 &lt;- plot_gene(v, rownames(top_treatment)[4]) +
  ylim(3, 11) + theme(legend.position = &quot;bottom&quot;)
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)
plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="use-ash-for-mutliple-testing-correction" class="section level2">
<h2>Use ash for mutliple testing correction</h2>
<p>Treatment effect.</p>
<pre class="r"><code># ash_treatment &lt;- ash(betahat = fit$coefficients[, &quot;Unstrain&quot;],
#                      sebetahat = fit$stdev.unscaled[, &quot;Unstrain&quot;] * sqrt(fit$s2.post),
#                      df = fit$df.total)
# class(ash_treatment)
# names(ash_treatment)
# sum(ash_treatment$svalue &lt; .05)
# hist(ash_treatment$svalue)</code></pre>
<!-- Venn diagram -->
<!-- ```{r venn} -->
<!-- ash_treatment_de <- ash_treatment$svalue < .05 -->
<!-- ash_status_de <- ash_status$svalue < .05 -->
<!-- ash_interaction_de <- ash_interaction$svalue < .05 -->
<!-- grid.newpage() -->
<!-- venn_rv <- draw.triple.venn( -->
<!--   area1 = sum(ash_treatment_de), -->
<!--   area2 = sum(ash_status_de), -->
<!--   area3 = sum(ash_interaction_de), -->
<!--   n12 = sum(ash_treatment_de & ash_status_de), -->
<!--   n23 = sum(ash_status_de & ash_interaction_de), -->
<!--   n13 = sum(ash_treatment_de & ash_interaction_de), -->
<!--   n123 = sum(ash_treatment_de & ash_status_de & ash_interaction_de), -->
<!--   category = c("Treatment", "Status", "Interaction"), -->
<!-- #   fill = c("darkolivegreen3", "cadetblue3", "darkorchid"), -->
<!--   alpha = c(0.5, 0.5, 0.5), col = "black", cex = 2, cat.cex = 1.5, -->
<!--   euler.d = FALSE, scaled = FALSE, ind = TRUE) -->
<!-- grid.draw(venn_rv) -->
<!-- ``` -->
<!-- ## Save results of differential expression analysis -->
<!-- Save results from limma and ash. -->
<!-- ```{r save-limma} -->
<!-- saveRDS(v, "../data/results-limma-voom.rds") -->
<!-- saveRDS(fit, "../data/results-limma-fit.rds") -->
<!-- saveRDS(ash_treatment, "../data/results-ash-treatment.rds") -->
<!-- saveRDS(ash_status, "../data/results-ash-status.rds") -->
<!-- saveRDS(ash_interaction, "../data/results-ash-interaction.rds") -->
<!-- ``` -->
<!-- ## Create figure -->
<!-- ```{r fig-limma} -->
<!-- theme_set(theme_bw(base_size = 8)) -->
<!-- theme_update(panel.grid.minor.x = element_blank(), -->
<!--              panel.grid.minor.y = element_blank(), -->
<!--              panel.grid.major.x = element_blank(), -->
<!--              panel.grid.major.y = element_blank()) -->
<!-- fig_limma <- ggdraw() + -->
<!--   draw_plot(ma_treatment,     0.00, 0.67, 0.33, 0.33) + -->
<!--   draw_plot(ma_status,        0.00, 0.33, 0.33, 0.33) + -->
<!--   draw_plot(ma_interaction,   0.00, 0.00, 0.33, 0.33) + -->
<!--   draw_plot(hist_treatment,   0.33, 0.67, 0.33, 0.33) + -->
<!--   draw_plot(hist_status,      0.33, 0.33, 0.33, 0.33) + -->
<!--   draw_plot(hist_interaction, 0.33, 0.00, 0.33, 0.33) + -->
<!--   draw_plot(p_treatment_1,    0.67, 0.82, 0.15, 0.15) + -->
<!--   draw_plot(p_treatment_2,    0.67, 0.67, 0.15, 0.15) + -->
<!--   draw_plot(p_treatment_3,    0.82, 0.67, 0.15, 0.15) + -->
<!--   draw_plot(p_treatment_4,    0.82, 0.82, 0.15, 0.15) + -->
<!--   draw_plot_label(c("A", "B", "C"), c(0.00, 0.00, 0.00), -->
<!--                                     c(1.00, 0.67, 0.33), size = 15) -->
<!-- fig_limma -->
<!-- # Using cowplot ggsave, which by default does not use Dingbats font -->
<!-- ggsave("fig-limma.eps", plot = fig_limma, width = 8, height = 5, units = "in") -->
<!-- ``` -->
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] stats4    parallel  grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] RUVSeq_1.12.0              EDASeq_2.12.0             
 [3] ShortRead_1.36.1           GenomicAlignments_1.14.2  
 [5] SummarizedExperiment_1.8.1 DelayedArray_0.4.1        
 [7] matrixStats_0.55.0         Rsamtools_1.30.0          
 [9] GenomicRanges_1.30.3       GenomeInfoDb_1.14.0       
[11] Biostrings_2.46.0          XVector_0.18.0            
[13] IRanges_2.12.0             S4Vectors_0.16.0          
[15] BiocParallel_1.12.0        Biobase_2.38.0            
[17] BiocGenerics_0.24.0        VennDiagram_1.6.20        
[19] futile.logger_1.4.3        cowplot_0.9.3             
[21] ggplot2_3.2.1              ashr_2.2-39               
[23] tidyr_1.0.0                dplyr_0.8.3               
[25] edgeR_3.20.9               plyr_1.8.4                
[27] limma_3.34.9              

loaded via a namespace (and not attached):
 [1] colorspace_1.4-1       hwriter_1.3.2          ellipsis_0.3.0        
 [4] rprojroot_1.3-2        fs_1.3.1               farver_2.0.1          
 [7] bit64_0.9-7            AnnotationDbi_1.40.0   codetools_0.2-15      
[10] splines_3.4.3          R.methodsS3_1.7.1      pscl_1.5.2            
[13] doParallel_1.0.15      DESeq_1.30.0           geneplotter_1.56.0    
[16] knitr_1.26             zeallot_0.1.0          workflowr_1.5.0       
[19] annotate_1.56.2        R.oo_1.23.0            compiler_3.4.3        
[22] httr_1.4.1             backports_1.1.5        assertthat_0.2.1      
[25] Matrix_1.2-18          lazyeval_0.2.2         later_1.0.0           
[28] formatR_1.7            htmltools_0.4.0        prettyunits_1.0.2     
[31] tools_3.4.3            gtable_0.3.0           glue_1.3.1            
[34] GenomeInfoDbData_1.0.0 Rcpp_1.0.3             vctrs_0.2.0           
[37] rtracklayer_1.38.3     iterators_1.0.12       xfun_0.11             
[40] stringr_1.4.0          lifecycle_0.1.0        statmod_1.4.32        
[43] XML_3.98-1.20          zlibbioc_1.24.0        MASS_7.3-51.4         
[46] scales_1.1.0           aroma.light_3.8.0      hms_0.5.2             
[49] promises_1.1.0         lambda.r_1.2.4         RColorBrewer_1.1-2    
[52] yaml_2.2.0             memoise_1.1.0          biomaRt_2.34.2        
[55] latticeExtra_0.6-28    stringi_1.4.3          RSQLite_2.1.4         
[58] SQUAREM_2017.10-1      genefilter_1.60.0      foreach_1.4.7         
[61] RMySQL_0.10.17         GenomicFeatures_1.30.3 truncnorm_1.0-8       
[64] rlang_0.4.2            pkgconfig_2.0.3        bitops_1.0-6          
[67] evaluate_0.14          lattice_0.20-35        purrr_0.3.3           
[70] labeling_0.3           bit_1.1-14             tidyselect_0.2.5      
[73] magrittr_1.5           R6_2.4.1               DBI_1.0.0             
[76] pillar_1.4.2           whisker_0.4            withr_2.1.2           
[79] survival_2.41-3        RCurl_1.95-4.12        mixsqp_0.2-2          
[82] tibble_2.1.3           crayon_1.3.4           futile.options_1.0.1  
[85] rmarkdown_1.18         progress_1.2.2         locfit_1.5-9.1        
[88] blob_1.2.0             git2r_0.26.1           digest_0.6.23         
[91] xtable_1.8-4           httpuv_1.5.2           R.utils_2.9.2         
[94] munsell_0.5.0         </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
