<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>DE_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">DE_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-01-18
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree629408f29313ac35a6601656ac80732937de98e1targetblank629408fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/629408f29313ac35a6601656ac80732937de98e1" target="_blank">629408f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree629408f29313ac35a6601656ac80732937de98e1targetblank629408fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/Norm_filter.Rmd
    Modified:   analysis/check_expression.Rmd
    Modified:   analysis/technical_factors_analysis.Rmd
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/analysis/DE_analysis.Rmd" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/158fdf3e906a4a488e486cb96684b4e23b567895/docs/DE_analysis.html" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/analysis/DE_analysis.Rmd" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/docs/DE_analysis.html" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/analysis/DE_analysis.Rmd" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/DE_analysis.html" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/analysis/DE_analysis.Rmd" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/DE_analysis.html" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/analysis/DE_analysis.Rmd" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/03b499e0b710c5a48bba76aef6472cc713c58301/docs/DE_analysis.html" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5eb29e88b6057837b2c5c0719c225364eb484690/docs/DE_analysis.html" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/DE_analysis.html" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/DE_analysis.html" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/analysis/DE_analysis.Rmd" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/DE_analysis.html" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/analysis/DE_analysis.Rmd" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/DE_analysis.html" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/analysis/DE_analysis.Rmd" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/DE_analysis.html" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/d80f643520bd77f37ac3f3862b7b747bccdbba30/analysis/DE_analysis.Rmd" target="_blank">d80f643</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Run Limma:
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/DE_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/93017062e918d7a60fcad8ee8ca0983de6e9aef7/analysis/DE_analysis.Rmd" target="_blank">9301706</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-17
</td>
<td>
build analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/DE_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/DE_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<pre class="r"><code>library(&quot;limma&quot;)
library(&quot;plyr&quot;)
library(&quot;edgeR&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;ashr&quot;)
library(&quot;ggplot2&quot;)
library(&quot;cowplot&quot;)
library(&quot;VennDiagram&quot;)
library(&quot;RUVSeq&quot;)
library(&quot;topGO&quot;)

#load in filtered count data, RUVs output
filt_counts &lt;- readRDS(&quot;data/filtered_counts.rds&quot;)
filt_counts &lt;- filt_counts$counts
RUVsOut &lt;- readRDS(&quot;data/RUVsOut.rds&quot;)

# load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

# load in reordered sample information
sampleinfo &lt;- readRDS(&quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="limma-doesnt-deal-with-technical-replicates-so-here-i-randomly-sample-1-replicate-for-each-of-the-6-individualxcondition" class="section level1">
<h1>Limma doesn’t deal with technical replicates, so here I randomly sample 1 replicate for each of the 6 individualxcondition</h1>
<pre class="r"><code>#randomly sample 1 replicate from each set of 3 replicates
individuals &lt;- c(&quot;18855&quot;, &quot;18856&quot;, &quot;19160&quot;)
treatments &lt;- c(&quot;S&quot;, &quot;U&quot;)

set.seed(1)
selected_samples &lt;- c()
for(i in c(&quot;18856&quot;, &quot;18855&quot;)){
     for(t in treatments){
          sample_num &lt;- base::sample(c(1:3), 1)
          selected_samples &lt;- c(selected_samples, paste0(i, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, t))
     }
}

sample_num &lt;- base::sample(c(2:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;U&quot;))
sample_num &lt;- base::sample(c(1:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;S&quot;))

selected_samples</code></pre>
<pre><code>[1] &quot;18856_1_S&quot; &quot;18856_2_U&quot; &quot;18855_2_S&quot; &quot;18855_3_U&quot; &quot;19160_2_U&quot; &quot;19160_3_S&quot;</code></pre>
<pre class="r"><code>randomly_sampled_indices &lt;- match(selected_samples, colnames(filt_counts))

single_filt_counts &lt;- filt_counts[,randomly_sampled_indices]
anno &lt;- pData(RUVsOut)[randomly_sampled_indices,]</code></pre>
<!-- # Model with limma -->
<!-- ```{r design-matrix} -->
<!-- #sample details -->
<!-- #anno <- pData(RUVsOut) -->
<!-- anno$LibraryPrepBatch <- factor(anno$LibraryPrepBatch, levels = c("1", "2")) -->
<!-- anno$Replicate <- factor(anno$Replicate, levels = c("1", "2", "3")) -->
<!-- anno$LibSize <- as.numeric(anno$LibSize) -->
<!-- design <- model.matrix(~treatment + W_1 + W_2 + RIN + LibSize, -->
<!--                        data = anno) -->
<!-- colnames(design)[1] <- "Intercept" -->
<!-- colnames(design) <- gsub("treatment", "", colnames(design)) -->
<!-- colnames(design) -->
<!-- ``` -->
<!-- ```{r fit-model, fig.width=8} -->
<!-- # Model individual as a random effect. -->
<!-- # Model individual as a random effect (use duplicate correlation function); treatment as a fixed effect, covariates = RUV factors -->
<!-- # Recommended to run both voom and duplicateCorrelation twice. -->
<!-- # https://support.bioconductor.org/p/59700/#67620 -->
<!-- #TMM Normalization -->
<!-- y <- DGEList(single_filt_counts) -->
<!-- y <- calcNormFactors(y, method = "TMM") -->
<!-- #Voom for differential expression -->
<!-- v1 <- voom(y, design) -->
<!-- corfit1 <- duplicateCorrelation(v1, design, block = anno$Individual) -->
<!-- corfit1$consensus -->
<!-- v2 <- voom(y, design, block = anno$Individual, correlation = corfit1$consensus) -->
<!-- corfit2 <- duplicateCorrelation(v2, design, block = anno$Individual) -->
<!-- # corfit2$consensus -->
<!-- fit <- lmFit(v2, design, block = anno$Individual, -->
<!--              correlation = corfit2$consensus) -->
<!-- fit <- eBayes(fit) -->
<!-- ``` -->
<!-- ## Assess model results -->
<!-- ```{r model-results} -->
<!-- get_results <- function(x, number = nrow(x$coefficients), sort.by = "none", -->
<!--                         ...) { -->
<!--   # x - object MArrayLM from eBayes output -->
<!--   # ... - additional arguments passed to topTable -->
<!--   stopifnot(class(x) == "MArrayLM") -->
<!--   results <- topTable(x, number = number, sort.by = sort.by, ...) -->
<!--   return(results) -->
<!-- } -->
<!-- top_treatment <- get_results(fit, coef = "Unstrain", sort.by = "B") -->
<!-- top_treatment -->
<!-- results_treatment <- get_results(fit, coef = "Unstrain", -->
<!--                               number = nrow(filt_counts), sort = "none") -->
<!-- ma_treatment <- ggplot(data.frame(Amean = fit$Amean, logFC = fit$coef[, "Unstrain"]), -->
<!--                        aes(x = Amean, y = logFC)) + -->
<!--   geom_point() + -->
<!--   labs(x = "Average expression level", y = "Log fold change", -->
<!--        title = "Treatment effect") -->
<!-- ma_treatment -->
<!-- hist_treatment <- ggplot(results_treatment, aes(x = P.Value)) + -->
<!--   geom_histogram(binwidth = 0.01) + -->
<!--   labs(x = "p-value", y = "Number of genes", title = "Treatment effect") -->
<!-- hist_treatment -->
<!-- ``` -->
<!-- ## Explore top hits -->
<!-- Boxplot function. -->
<!-- ```{r boxplot-function} -->
<!-- plot_gene <- function(v, g) { -->
<!--   # v - An EList object containing log2 counts per million -->
<!--   # g - character vector of a single gene -->
<!--   stopifnot(class(v) == "EList", -->
<!--             is.character(g), length(g) == 1) -->
<!--   library("tidyr") -->
<!--   single_gene <- v$E[g, ] -->
<!--   single_gene_long <- as.data.frame(single_gene) -->
<!--   colnames(single_gene_long) <- "log2cpm" -->
<!--   single_gene_long$sample <- rownames(single_gene_long) -->
<!--   single_gene_long <- separate(single_gene_long, col = "sample", sep = "_", -->
<!--                                 into = c("Individual", "Replicate", "Treatment")) -->
<!--   single_gene_long$Treatment <- factor(single_gene_long$Treatment, levels = c("S", "U")) -->
<!--   single_gene_long$Treatment <- revalue(single_gene_long$Treatment, c("S"="Strain", "U"="No Strain")) -->
<!--   ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) + -->
<!--     geom_boxplot() + -->
<!--     labs(title = g, x = "Treatment", y = expression("Expression level (" * log[2] * " cpm)")) -->
<!-- } -->
<!-- ``` -->
<!-- Treatment -->
<!-- ```{r treatment} -->
<!-- #rename top_treatment genes to gene symbols for easier interpretation -->
<!-- top_treatment_top4 <- gene_anno$external_gene_name[match(head(rownames(top_treatment), n=4), gene_anno$ensembl_gene_id)] -->
<!-- top_treatment_top4 -->
<!-- p_treatment_1 <- plot_gene(v2, rownames(top_treatment)[1]) + -->
<!--   ylim(3, 11) + theme(legend.position = "bottom") -->
<!-- p_treatment_2 <- plot_gene(v2, rownames(top_treatment)[2]) + -->
<!--   ylim(3, 11) + theme(legend.position = "bottom") -->
<!-- p_treatment_3 <- plot_gene(v2, rownames(top_treatment)[3]) + -->
<!--   ylim(3, 11) + theme(legend.position = "bottom") -->
<!-- p_treatment_4 <- plot_gene(v2, rownames(top_treatment)[4]) + -->
<!--   ylim(3, 11) + theme(legend.position = "bottom") -->
<!-- ggsave("fig-limma-treatment-hit-1.eps", plot = p_treatment_1, -->
<!--        width = 7, height = 7, units = "in") -->
<!-- plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4) -->
<!-- ``` -->
<!-- ## Use ash for mutliple testing correction -->
<!-- Treatment effect. -->
<!-- ```{r ash-treatment} -->
<!-- run_ash <- function(x, coef) { -->
<!--   # Perform multiple testing correction with adaptive shrinkage (ASH) -->
<!--   # -->
<!--   # x - object MArrayLM from eBayes output -->
<!--   # coef - coefficient tested by eBayes -->
<!--   stopifnot(class(x) == "MArrayLM", coef %in% colnames(x$coefficients)) -->
<!--   result <- ash(betahat = x$coefficients[, coef], -->
<!--                 sebetahat = x$stdev.unscaled[, coef] * sqrt(x$s2.post), -->
<!--                 df = x$df.total[1]) -->
<!--   return(result) -->
<!-- } -->
<!-- ash_treatment <- run_ash(fit, "Unstrain") -->
<!-- class(ash_treatment) -->
<!-- names(ash_treatment) -->
<!-- sum(ash_treatment$result$svalue < .05) -->
<!-- hist(ash_treatment$result$svalue) -->
<!-- ``` -->
<!-- # Plots -->
<!-- ```{r Plot Functions} -->
<!-- plot_ma <- function(x, qval) { -->
<!--   # Create MA plot. -->
<!--   # -->
<!--   # x - data frame with topTable and ASH output -->
<!--   #     (columns logFC, AveExpr, and qvalue) -->
<!--   # qval - qvalue cutoff for calling a gene DE -->
<!--   # -->
<!--   stopifnot(is.data.frame(x), c("logFC", "AveExpr", "qvalue") %in% colnames(x), -->
<!--             is.numeric(qval), qval <= 1, qval >= 0) -->
<!--   x$highlight <- ifelse(x$qvalue < qval, "darkred", "gray75") -->
<!--   x$highlight <- factor(x$highlight, levels = c("darkred", "gray75")) -->
<!--   ggplot(x, aes(x = AveExpr, y = logFC, color = highlight, shape = highlight)) + -->
<!--     geom_point() + -->
<!--     labs(x = "Average expression level", y = "Log fold change") + -->
<!--     scale_color_identity(drop = FALSE) + -->
<!--     scale_shape_manual(values = c(16, 1), drop = FALSE) + -->
<!--     theme(legend.position = "none") -->
<!-- #   scale_color_gradient(low = "red", high = "white", limits = c(0, 0.25)) -->
<!-- } -->
<!-- plot_volcano <- function(x, qval) { -->
<!--   # Create volcano plot. -->
<!--   # -->
<!--   # x - data frame with topTable and ASH output -->
<!--   #     (columns logFC, P.Value, and qvalue) -->
<!--   # qval - qvalue cutoff for calling a gene DE -->
<!--   # -->
<!--   stopifnot(is.data.frame(x), c("logFC", "P.Value", "qvalue") %in% colnames(x), -->
<!--             is.numeric(qval), qval <= 1, qval >= 0) -->
<!--   x$highlight <- ifelse(x$qvalue < qval, "darkred", "gray75") -->
<!--   x$highlight <- factor(x$highlight, levels = c("darkred", "gray75")) -->
<!--   ggplot(x, aes(x = logFC, y = -log10(P.Value), color = highlight)) + -->
<!--     geom_point(shape = 1) + -->
<!--     labs(x = "Log fold change", -->
<!--          y = expression(-log[10] * " p-value")) + -->
<!--     scale_color_identity(drop = FALSE) + -->
<!--     theme(legend.position = "none") -->
<!-- } -->
<!-- plot_pval_hist <- function(x, qval) { -->
<!--   # Create histogram of p-values. -->
<!--   # -->
<!--   # x - data frame with topTable and ash output (columns P.Value and qvalue) -->
<!--   # qval - qvalue cutoff for calling a gene DE -->
<!--   # -->
<!--   stopifnot(is.data.frame(x), c("P.Value", "qvalue") %in% colnames(x)) -->
<!--   x$highlight <- ifelse(x$qvalue < qval, "darkred", "gray75") -->
<!--   x$highlight <- factor(x$highlight, levels = c("darkred", "gray75")) -->
<!--   ggplot(x, aes(x = P.Value, fill = highlight)) + -->
<!--     geom_histogram(position = "stack", binwidth = 0.01) + -->
<!--     scale_fill_identity(drop = FALSE) + -->
<!--     labs(x = "p-value", y = "Number of genes") -->
<!-- } -->
<!-- ``` -->
<!-- ```{r use Functions} -->
<!-- tests <- colnames(fit$coefficients) -->
<!-- results <- vector(length = length(tests), mode = "list") -->
<!-- names(results) <- tests -->
<!-- for (test in tests) { -->
<!--   # Extract limma results -->
<!--   results[[test]] <- get_results(fit, coef = test) -->
<!--   # Add mutliple testing correction with ASH -->
<!--   output_ash <- run_ash(fit, coef = test)$result -->
<!--   results[[test]] <- cbind(results[[test]], lfsr = output_ash$lfsr, -->
<!--                            lfdr = output_ash$lfdr, qvalue = output_ash$qvalue, -->
<!--                            svalue = output_ash$svalue) -->
<!-- } -->
<!-- #FDR 0.1 -->
<!-- plot_ma(results[["Unstrain"]], 0.1) -->
<!-- plot_volcano(results[["Unstrain"]], 0.1) -->
<!-- plot_pval_hist(results[["Unstrain"]], 0.1) -->
<!-- table(results[["Unstrain"]]$qvalue < 0.1) -->
<!-- significant_genes_10 <- row.names(results[["Unstrain"]])[results[["Unstrain"]]$qvalue < 0.1] -->
<!-- significant_symbols_10 <- gene_anno$external_gene_name[match(significant_genes_10, gene_anno$ensembl_gene_id)] -->
<!-- significant_symbols_10 -->
<!-- significant_anno_10 <- gene_anno[match(significant_genes_10, gene_anno$ensembl_gene_id),] -->
<!-- #FDR 0.05 -->
<!-- plot_ma(results[["Unstrain"]], 0.05) -->
<!-- plot_volcano(results[["Unstrain"]], 0.05) -->
<!-- plot_pval_hist(results[["Unstrain"]], 0.05) -->
<!-- table(results[["Unstrain"]]$qvalue < 0.05) -->
<!-- significant_genes_05 <- row.names(results[["Unstrain"]])[results[["Unstrain"]]$qvalue < 0.05] -->
<!-- significant_symbols_05 <- gene_anno$external_gene_name[match(significant_genes_05, gene_anno$ensembl_gene_id)] -->
<!-- significant_symbols_05 -->
<!-- significant_anno_05 <- gene_anno[match(significant_genes_05, gene_anno$ensembl_gene_id),] -->
<!-- #FDR 0.01 -->
<!-- plot_ma(results[["Unstrain"]], 0.01) -->
<!-- plot_volcano(results[["Unstrain"]], 0.01) -->
<!-- plot_pval_hist(results[["Unstrain"]], 0.01) -->
<!-- table(results[["Unstrain"]]$qvalue < 0.01) -->
<!-- significant_genes_01 <- row.names(results[["Unstrain"]])[results[["Unstrain"]]$qvalue < 0.01] -->
<!-- significant_symbols_01 <- gene_anno$external_gene_name[match(significant_genes_01, gene_anno$ensembl_gene_id)] -->
<!-- significant_symbols_01 -->
<!-- significant_anno_01 <- gene_anno[match(significant_genes_01, gene_anno$ensembl_gene_id),] -->
<!-- ``` -->
<!-- ## Gene ontology analysis with topGO -->
<!-- Use topGO for GO analysis. -->
<!-- It accounts for the nested graph structure of GO terms to prune the number of GO categories tested ([Alexa et al. 2006][Alexa2006]). -->
<!-- Essentially, it decreases the redundancy of the results. -->
<!-- [Alexa2006]: http://www.ncbi.nlm.nih.gov/pubmed/16606683 -->
<!-- First create the gene universe. -->
<!-- This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not. -->
<!-- ```{r gene-universe} -->
<!-- threshold <- 0.1 -->
<!-- gene_universe <- as.numeric(results[["Unstrain"]]$qvalue < threshold) -->
<!-- gene_universe <- factor(gene_universe) -->
<!-- names(gene_universe) <- rownames(results[["Unstrain"]]) -->
<!-- head(gene_universe) -->
<!-- ``` -->
<!-- Create the topGO data object. -->
<!-- Only consider "Biological Process" categories and use the Mouse Ensembl database for annotation. -->
<!-- ```{r go-data} -->
<!-- go_data <- new("topGOdata", -->
<!--                ontology = "BP", -->
<!--                allGenes = gene_universe, -->
<!--                nodeSize = 5, -->
<!--                annotationFun = annFUN.org, -->
<!--                mapping = "org.Hs.eg.db", -->
<!--                ID = "ensembl") -->
<!-- ``` -->
<!-- Use the weight01 algorithm and score the tests with Fisher's exact test. -->
<!-- ```{r go-test} -->
<!-- go_test <- runTest(go_data, algorithm = "weight01", statistic = "fisher") -->
<!-- ``` -->
<!-- Keep the results with a Fisher's exact test p-value < 0.01. -->
<!-- ```{r go-table} -->
<!-- go_table <- GenTable(go_data, weightFisher = go_test, -->
<!--                      orderBy = "weightFisher", ranksOf = "weightFisher", -->
<!--                      topNodes = sum(score(go_test) < .01)) -->
<!-- go_table -->
<!-- ``` -->
</div>
<div id="perform-de-analysis-using-edger" class="section level1">
<h1>Perform DE analysis using edgeR</h1>
<pre class="r"><code>#Subset RUVsOut for only one of each of the samples (no technical replicates)
RUVsSubset &lt;- RUVsOut[, RUVsOut$Sample_ID %in% selected_samples]


design &lt;- model.matrix(~treatment + W_1 + W_2 + RIN, data=pData(RUVsSubset))
y &lt;- DGEList(counts=counts(RUVsSubset), group=pData(RUVsSubset)$Individual)
y &lt;- calcNormFactors(y, method=&quot;upperquartile&quot;)
y &lt;- estimateGLMCommonDisp(y, design)
y &lt;- estimateGLMTagwiseDisp(y, design)
fit &lt;- glmFit(y, design)
lrt &lt;- glmLRT(fit, coef=2)
gene_values &lt;- topTags(lrt, n = nrow(lrt$coefficients), adjust.method=&quot;BH&quot;, sort.by = &quot;logFC&quot;)</code></pre>
<div id="explore-top-hits" class="section level2">
<h2>Explore top hits</h2>
<p>Boxplot function.</p>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An df containing log2 counts per million
  # g - character vector of a single gene
  library(&quot;tidyr&quot;)
  single_gene &lt;- v[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))

  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
    geom_boxplot() +
    labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;))
}

plot_volcano &lt;- function(x, qval) {
  # Create volcano plot.
  #
  # x - data frame with topTable output
  #     (columns logFC, PValue, and FDR)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(PValue), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, qval) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = PValue, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}</code></pre>
</div>
<div id="box-plots-of-top-genes" class="section level2">
<h2>Box plots of top genes</h2>
<pre class="r"><code>cpm &lt;- cpm(filt_counts, log = T)

# top genes by p-value

#rename top_treatment genes to gene symbols for easier interpretation
top_treatment_top10 &lt;- gene_anno$external_gene_name[match(head(rownames(gene_values@.Data[[1]]), n=10), gene_anno$ensembl_gene_id)]
top_treatment_top10</code></pre>
<pre><code> [1] USP9Y  UTY    TTTY15 PRKY   UGT2B7 NLGN4Y ZFY    KDM5D  IGFL3  RPS4Y1
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 A2ML1-AS2 ... ZZEF1</code></pre>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[1])
p_treatment_2 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[2]) 
p_treatment_3 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[3]) 
p_treatment_4 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[4]) 
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)
plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<p><img src="figure/DE_analysis.Rmd/boxplots-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="plots-with-significant-genes-highlighted" class="section level2">
<h2>Plots with significant genes highlighted</h2>
<pre class="r"><code>thresh &lt;- 0.01
plot_volcano(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot_pval_hist(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>table(gene_values@.Data[[1]]$FDR &lt; thresh)</code></pre>
<pre><code>
FALSE  TRUE 
 9569   878 </code></pre>
</div>
<div id="gene-ontology-analysis-with-topgo" class="section level2">
<h2>Gene ontology analysis with topGO</h2>
<p>Use topGO for GO analysis. It accounts for the nested graph structure of GO terms to prune the number of GO categories tested (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">Alexa et al. 2006</a>). Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe. This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.01
gene_universe &lt;- as.numeric(gene_values@.Data[[1]]$FDR &lt; threshold)
gene_universe &lt;- factor(gene_universe)
names(gene_universe) &lt;- rownames(gene_values@.Data[[1]])
head(gene_universe)</code></pre>
<pre><code>ENSG00000114374 ENSG00000183878 ENSG00000233864 ENSG00000099725 ENSG00000171234 
              1               1               1               1               1 
ENSG00000165246 
              1 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>Loading required package: org.Hs.eg.db</code></pre>
<pre><code></code></pre>
<pre><code>    ( 9640 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13704 GO terms and 31801 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9322 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 5453 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  11 nodes to be scored   (10 eliminated genes)</code></pre>
<pre><code>
     Level 16:  21 nodes to be scored   (14 eliminated genes)</code></pre>
<pre><code>
     Level 15:  43 nodes to be scored   (68 eliminated genes)</code></pre>
<pre><code>
     Level 14:  91 nodes to be scored   (201 eliminated genes)</code></pre>
<pre><code>
     Level 13:  161 nodes to be scored  (555 eliminated genes)</code></pre>
<pre><code>
     Level 12:  262 nodes to be scored  (1267 eliminated genes)</code></pre>
<pre><code>
     Level 11:  409 nodes to be scored  (2861 eliminated genes)</code></pre>
<pre><code>
     Level 10:  584 nodes to be scored  (3916 eliminated genes)</code></pre>
<pre><code>
     Level 9:   757 nodes to be scored  (5411 eliminated genes)</code></pre>
<pre><code>
     Level 8:   791 nodes to be scored  (6736 eliminated genes)</code></pre>
<pre><code>
     Level 7:   809 nodes to be scored  (7691 eliminated genes)</code></pre>
<pre><code>
     Level 6:   696 nodes to be scored  (8356 eliminated genes)</code></pre>
<pre><code>
     Level 5:   442 nodes to be scored  (8735 eliminated genes)</code></pre>
<pre><code>
     Level 4:   248 nodes to be scored  (8956 eliminated genes)</code></pre>
<pre><code>
     Level 3:   104 nodes to be scored  (9083 eliminated genes)</code></pre>
<pre><code>
     Level 2:   21 nodes to be scored   (9153 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9213 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>         GO.ID                                        Term Annotated
1   GO:0030198           extracellular matrix organization       206
2   GO:0045540 regulation of cholesterol biosynthetic p...        39
3   GO:0010469             regulation of receptor activity       176
4   GO:0055114                 oxidation-reduction process       619
5   GO:0098869             cellular oxidant detoxification        54
6   GO:0007267                         cell-cell signaling       777
7   GO:0007155                               cell adhesion       685
8   GO:0071281               cellular response to iron ion         8
9   GO:0050919                         negative chemotaxis        21
10  GO:0010951 negative regulation of endopeptidase act...       110
11  GO:0006695            cholesterol biosynthetic process        56
12  GO:0051897 positive regulation of protein kinase B ...        69
13  GO:0048662 negative regulation of smooth muscle cel...        22
14  GO:0007271          synaptic transmission, cholinergic         7
15  GO:0007588                                   excretion        15
16  GO:0051781        positive regulation of cell division        43
17  GO:0060021                          palate development        63
18  GO:0060600 dichotomous subdivision of an epithelial...         8
19  GO:0042474                    middle ear morphogenesis        12
20  GO:0048265                            response to pain         5
21  GO:0008217                regulation of blood pressure        70
22  GO:0007413                        axonal fasciculation        13
23  GO:0032956 regulation of actin cytoskeleton organiz...       191
24  GO:0035094                        response to nicotine        18
25  GO:0006805                xenobiotic metabolic process        35
26  GO:0000305                  response to oxygen radical        21
27  GO:1902106 negative regulation of leukocyte differe...        46
28  GO:0003223 ventricular compact myocardium morphogen...         6
29  GO:0007601                           visual perception        64
30  GO:0008299             isoprenoid biosynthetic process        15
31  GO:0031290         retinal ganglion cell axon guidance        10
32  GO:0071636 positive regulation of transforming grow...        10
33  GO:0002063                     chondrocyte development        20
34  GO:0042493                            response to drug       217
35  GO:0001658 branching involved in ureteric bud morph...        33
36  GO:0007156 homophilic cell adhesion via plasma memb...        53
37  GO:0006954                       inflammatory response       266
38  GO:0010463              mesenchymal cell proliferation        32
39  GO:0032836    glomerular basement membrane development         7
40  GO:0002480 antigen processing and presentation of e...         7
41  GO:0090331 negative regulation of platelet aggregat...         7
42  GO:0008544                       epidermis development       158
43  GO:0051965     positive regulation of synapse assembly        28
44  GO:0009952    anterior/posterior pattern specification       117
45  GO:0030890 positive regulation of B cell proliferat...        12
46  GO:0045766         positive regulation of angiogenesis        77
47  GO:0006501               C-terminal protein lipidation        23
48  GO:0000768 syncytium formation by plasma membrane f...        23
49  GO:0001764                            neuron migration        77
50  GO:1901616 organic hydroxy compound catabolic proce...        23
51  GO:0007186 G-protein coupled receptor signaling pat...       288
52  GO:0015850          organic hydroxy compound transport        83
53  GO:0000302         response to reactive oxygen species       150
54  GO:0007166     cell surface receptor signaling pathway      1522
55  GO:0060415                 muscle tissue morphogenesis        46
56  GO:0019433              triglyceride catabolic process         8
57  GO:0032026                   response to magnesium ion         8
58  GO:0015909             long-chain fatty acid transport        28
59  GO:0009611                        response to wounding       365
60  GO:0045987 positive regulation of smooth muscle con...        13
61  GO:0008347                        glial cell migration        29
62  GO:0001525                                angiogenesis       271
63  GO:0051384                  response to glucocorticoid        74
64  GO:0030199                collagen fibril organization        31
65  GO:0001501                 skeletal system development       306
66  GO:0043410         positive regulation of MAPK cascade       272
67  GO:0042127            regulation of cell proliferation       867
68  GO:0030154                        cell differentiation      2089
69  GO:0007411                               axon guidance       135
70  GO:0060349                          bone morphogenesis        51
71  GO:0033280                       response to vitamin D        21
72  GO:2000353 positive regulation of endothelial cell ...         9
73  GO:0042744         hydrogen peroxide catabolic process         9
74  GO:0045880 positive regulation of smoothened signal...        20
75  GO:0050776               regulation of immune response       449
76  GO:0030574                  collagen catabolic process        40
77  GO:0006809           nitric oxide biosynthetic process        36
78  GO:0030828 positive regulation of cGMP biosynthetic...         5
79  GO:0016114              terpenoid biosynthetic process         5
80  GO:0060100 positive regulation of phagocytosis, eng...         5
81  GO:0060539                       diaphragm development         5
82  GO:0045080 positive regulation of chemokine biosynt...         5
83  GO:0035930            corticosteroid hormone secretion         5
84  GO:0046851      negative regulation of bone remodeling         5
85  GO:2000192 negative regulation of fatty acid transp...         5
86  GO:0061302          smooth muscle cell-matrix adhesion         5
87  GO:0097151 positive regulation of inhibitory postsy...         5
88  GO:0090193 positive regulation of glomerulus develo...         5
89  GO:0002544               chronic inflammatory response         5
90  GO:0090270 regulation of fibroblast growth factor p...         5
91  GO:0014820             tonic smooth muscle contraction         5
92  GO:0042472                     inner ear morphogenesis        38
93  GO:0001942                   hair follicle development        58
94  GO:0048663                      neuron fate commitment        21
95  GO:0010575 positive regulation of vascular endothel...        15
96  GO:0016486                  peptide hormone processing        15
97  GO:0042572                   retinol metabolic process        10
98  GO:0036342                post-anal tail morphogenesis        10
99  GO:0030195    negative regulation of blood coagulation        24
100 GO:0044259 multicellular organismal macromolecule m...        71
101 GO:0046456              icosanoid biosynthetic process        21
102 GO:0006833                             water transport         7
103 GO:0042632                     cholesterol homeostasis        29
104 GO:0001937 negative regulation of endothelial cell ...        22
105 GO:0055088                           lipid homeostasis        54
106 GO:0034599       cellular response to oxidative stress       203
107 GO:0007568                                       aging       189
108 GO:0008285 negative regulation of cell proliferatio...       411
109 GO:0030326                embryonic limb morphogenesis        91
110 GO:0003094                       glomerular filtration        13
111 GO:0031638                          zymogen activation        27
112 GO:0021602                 cranial nerve morphogenesis        12
113 GO:0007586                                   digestion        31
114 GO:0001913                T cell mediated cytotoxicity        14
115 GO:2001053 regulation of mesenchymal cell apoptotic...         6
116 GO:0019371                      cyclooxygenase pathway         6
117 GO:0060004                                      reflex         6
118 GO:0021520 spinal cord motor neuron cell fate speci...         6
119 GO:0072124 regulation of glomerular mesangial cell ...         6
120 GO:0048791 calcium ion-regulated exocytosis of neur...         6
121 GO:0006069                           ethanol oxidation         6
122 GO:0035810         positive regulation of urine volume         6
123 GO:0071625                       vocalization behavior         6
124 GO:0006703               estrogen biosynthetic process         6
125 GO:0009247             glycolipid biosynthetic process        43
126 GO:0071548                   response to dexamethasone        27
127 GO:0030258                          lipid modification       165
128 GO:0008037                            cell recognition        48
129 GO:0007157 heterophilic cell-cell adhesion via plas...        23
130 GO:0048844                        artery morphogenesis        38
131 GO:0007422       peripheral nervous system development        37
132 GO:0036066               protein O-linked fucosylation        11
    Significant Expected weightFisher
1            56    17.10      1.4e-12
2            17     3.24      2.3e-09
3            37    14.61      7.4e-09
4            78    51.40      6.4e-08
5            19     4.48      1.9e-06
6            96    64.51      4.7e-06
7           100    56.88      7.8e-06
8             6     0.66      7.8e-06
9             9     1.74      2.1e-05
10           19     9.13      2.2e-05
11           25     4.65      2.4e-05
12           17     5.73      3.4e-05
13            9     1.83      5.3e-05
14            5     0.58      7.1e-05
15            7     1.25      9.4e-05
16           12     3.57      0.00011
17           15     5.23      0.00015
18            5     0.66      0.00018
19            6     1.00      0.00019
20            4     0.42      0.00022
21           19     5.81      0.00032
22            6     1.08      0.00033
23           20    15.86      0.00033
24            7     1.49      0.00037
25           10     2.91      0.00052
26            9     1.74      0.00056
27           11     3.82      0.00056
28            4     0.50      0.00062
29           14     5.31      0.00062
30            8     1.25      0.00068
31            5     0.83      0.00069
32            5     0.83      0.00069
33            7     1.66      0.00078
34           29    18.02      0.00100
35            9     2.74      0.00108
36           12     4.40      0.00109
37           42    22.09      0.00112
38            9     2.66      0.00133
39            4     0.58      0.00135
40            4     0.58      0.00135
41            4     0.58      0.00135
42           27    13.12      0.00139
43            8     2.32      0.00151
44           18     9.71      0.00179
45            5     1.00      0.00188
46           16     6.39      0.00194
47            7     1.91      0.00199
48            8     1.91      0.00211
49           14     6.39      0.00212
50            6     1.91      0.00213
51           44    23.91      0.00221
52           18     6.89      0.00239
53           23    12.45      0.00243
54          164   126.37      0.00246
55           10     3.82      0.00248
56            4     0.66      0.00252
57            4     0.66      0.00252
58            8     2.32      0.00282
59           57    30.31      0.00285
60            5     1.08      0.00285
61            6     2.41      0.00286
62           48    22.50      0.00296
63           13     6.14      0.00303
64            8     2.57      0.00307
65           48    25.41      0.00324
66           37    22.58      0.00352
67          133    71.99      0.00354
68          256   173.45      0.00354
69           29    11.21      0.00366
70           10     4.23      0.00418
71            6     1.74      0.00421
72            4     0.75      0.00423
73            4     0.75      0.00423
74            6     1.66      0.00452
75           39    37.28      0.00460
76            9     3.32      0.00468
77            8     2.99      0.00497
78            3     0.42      0.00502
79            3     0.42      0.00502
80            3     0.42      0.00502
81            3     0.42      0.00502
82            3     0.42      0.00502
83            3     0.42      0.00502
84            3     0.42      0.00502
85            3     0.42      0.00502
86            3     0.42      0.00502
87            3     0.42      0.00502
88            3     0.42      0.00502
89            3     0.42      0.00502
90            3     0.42      0.00502
91            3     0.42      0.00502
92            9     3.16      0.00534
93           11     4.82      0.00561
94            8     1.74      0.00570
95            5     1.25      0.00578
96            5     1.25      0.00658
97            4     0.83      0.00659
98            4     0.83      0.00659
99           13     1.99      0.00672
100          16     5.90      0.00674
101           8     1.74      0.00681
102           3     0.58      0.00688
103           7     2.41      0.00752
104           6     1.83      0.00754
105          12     4.48      0.00768
106          26    16.85      0.00797
107          20    15.69      0.00826
108          66    34.13      0.00834
109          15     7.56      0.00931
110           6     1.08      0.00934
111           7     2.24      0.00934
112           5     1.00      0.00937
113           6     2.57      0.00939
114           4     1.16      0.00941
115           3     0.50      0.00942
116           3     0.50      0.00942
117           3     0.50      0.00942
118           3     0.50      0.00942
119           3     0.50      0.00942
120           3     0.50      0.00942
121           3     0.50      0.00942
122           3     0.50      0.00942
123           3     0.50      0.00942
124           3     0.50      0.00942
125           6     3.57      0.00942
126           4     2.24      0.00945
127          15    13.70      0.00946
128          12     3.99      0.00947
129           6     1.91      0.00950
130           9     3.16      0.00957
131           7     3.07      0.00965
132           4     0.91      0.00969</code></pre>
<p>There are 132 significant results.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] stats4    parallel  grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] org.Hs.eg.db_3.5.0         topGO_2.30.1              
 [3] SparseM_1.77               GO.db_3.5.0               
 [5] AnnotationDbi_1.40.0       graph_1.56.0              
 [7] RUVSeq_1.12.0              EDASeq_2.12.0             
 [9] ShortRead_1.36.1           GenomicAlignments_1.14.2  
[11] SummarizedExperiment_1.8.1 DelayedArray_0.4.1        
[13] matrixStats_0.55.0         Rsamtools_1.30.0          
[15] GenomicRanges_1.30.3       GenomeInfoDb_1.14.0       
[17] Biostrings_2.46.0          XVector_0.18.0            
[19] IRanges_2.12.0             S4Vectors_0.16.0          
[21] BiocParallel_1.12.0        Biobase_2.38.0            
[23] BiocGenerics_0.24.0        VennDiagram_1.6.20        
[25] futile.logger_1.4.3        cowplot_0.9.3             
[27] ggplot2_3.2.1              ashr_2.2-39               
[29] tidyr_1.0.0                dplyr_0.8.3               
[31] edgeR_3.20.9               plyr_1.8.4                
[33] limma_3.34.9              

loaded via a namespace (and not attached):
 [1] colorspace_1.4-1       hwriter_1.3.2          ellipsis_0.3.0        
 [4] rprojroot_1.3-2        fs_1.3.1               farver_2.0.1          
 [7] bit64_0.9-7            codetools_0.2-15       splines_3.4.3         
[10] R.methodsS3_1.7.1      pscl_1.5.2             doParallel_1.0.15     
[13] DESeq_1.30.0           geneplotter_1.56.0     knitr_1.26            
[16] zeallot_0.1.0          workflowr_1.5.0        annotate_1.56.2       
[19] R.oo_1.23.0            compiler_3.4.3         httr_1.4.1            
[22] backports_1.1.5        assertthat_0.2.1       Matrix_1.2-18         
[25] lazyeval_0.2.2         later_1.0.0            formatR_1.7           
[28] htmltools_0.4.0        prettyunits_1.0.2      tools_3.4.3           
[31] gtable_0.3.0           glue_1.3.1             GenomeInfoDbData_1.0.0
[34] Rcpp_1.0.3             vctrs_0.2.0            rtracklayer_1.38.3    
[37] iterators_1.0.12       xfun_0.11              stringr_1.4.0         
[40] lifecycle_0.1.0        XML_3.98-1.20          zlibbioc_1.24.0       
[43] MASS_7.3-51.4          scales_1.1.0           aroma.light_3.8.0     
[46] hms_0.5.2              promises_1.1.0         lambda.r_1.2.4        
[49] RColorBrewer_1.1-2     yaml_2.2.0             memoise_1.1.0         
[52] biomaRt_2.34.2         latticeExtra_0.6-28    stringi_1.4.3         
[55] RSQLite_2.1.4          SQUAREM_2017.10-1      genefilter_1.60.0     
[58] foreach_1.4.7          RMySQL_0.10.17         GenomicFeatures_1.30.3
[61] truncnorm_1.0-8        rlang_0.4.2            pkgconfig_2.0.3       
[64] bitops_1.0-6           evaluate_0.14          lattice_0.20-35       
[67] purrr_0.3.3            labeling_0.3           bit_1.1-14            
[70] tidyselect_0.2.5       magrittr_1.5           R6_2.4.1              
[73] DBI_1.0.0              pillar_1.4.2           whisker_0.4           
[76] withr_2.1.2            survival_2.41-3        RCurl_1.95-4.12       
[79] mixsqp_0.2-2           tibble_2.1.3           crayon_1.3.4          
[82] futile.options_1.0.1   rmarkdown_1.18         progress_1.2.2        
[85] locfit_1.5-9.1         blob_1.2.0             git2r_0.26.1          
[88] digest_0.6.23          xtable_1.8-4           httpuv_1.5.2          
[91] R.utils_2.9.2          munsell_0.5.0         </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
