<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>DE_analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">DE_analysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-05-31
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreedd6a48bfb866db0663b341f0c7a8c300ec83780ftargetblankdd6a48ba"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/dd6a48bfb866db0663b341f0c7a8c300ec83780f" target="_blank">dd6a48b</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreedd6a48bfb866db0663b341f0c7a8c300ec83780ftargetblankdd6a48ba" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/dd6a48bfb866db0663b341f0c7a8c300ec83780f" target="_blank">dd6a48b</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/voom_results.rds

Untracked files:
    Untracked:  .RDataTmp
    Untracked:  analysis/examples_DE_genes_overlap_GWAS.Rmd

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/enrichment_DE.Rmd
    Modified:   data/GWAS_genes_within_500kb.csv
    Modified:   fig-limma-treatment-hit-1.eps
    Modified:   output/GWAS_descriptions.csv

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/DE_analysis.Rmd</code>) and HTML (<code>docs/DE_analysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6cfbae9c91c4f6cb94fb8432695184bb3cfb502e/analysis/DE_analysis.Rmd" target="_blank">6cfbae9</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-18
</td>
<td>
Add enrichment analysis file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5b1c7c6ff5eec4ebeb600f8cabda6eb39926a36/analysis/DE_analysis.Rmd" target="_blank">a5b1c7c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
<td>
separate into up/downregulated in DE analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a5b1c7c6ff5eec4ebeb600f8cabda6eb39926a36/docs/DE_analysis.html" target="_blank">a5b1c7c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
<td>
separate into up/downregulated in DE analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/analysis/DE_analysis.Rmd" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
<td>
Subset pseudobulk to 1000 DEG, dotplots instead of boxplots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/DE_analysis.html" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
<td>
Subset pseudobulk to 1000 DEG, dotplots instead of boxplots
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/analysis/DE_analysis.Rmd" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
Add boxplot by sample for top DEGenes; k=7 for supervised definingh early and late chondros
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/DE_analysis.html" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
Add boxplot by sample for top DEGenes; k=7 for supervised definingh early and late chondros
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/940b53efe5b3edcac621e238213ebf8d896fd263/analysis/DE_analysis.Rmd" target="_blank">940b53e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
<td>
define early/late passage chondros as a supervised cluster
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ad1e27bd3b051a731c2357fb8d8f981baa35f6af/analysis/DE_analysis.Rmd" target="_blank">ad1e27b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-02
</td>
<td>
separate up/down in de analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ad1e27bd3b051a731c2357fb8d8f981baa35f6af/docs/DE_analysis.html" target="_blank">ad1e27b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-02
</td>
<td>
separate up/down in de analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/DE_analysis.html" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
add additional chondro markers
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b967c3ceb4e6c6da17db3be8f12c4009730dde44/analysis/DE_analysis.Rmd" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/b967c3ceb4e6c6da17db3be8f12c4009730dde44/docs/DE_analysis.html" target="_blank">b967c3c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-27
</td>
<td>
Add cell atlas assignment
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/analysis/DE_analysis.Rmd" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
include Individual as a FE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/DE_analysis.html" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
include Individual as a FE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a3565148512e79e1482c774b845a50b8c9c06e74/docs/DE_analysis.html" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/3c163de1c62d5f63d88254fa028dda84186718b7/analysis/DE_analysis.Rmd" target="_blank">3c163de</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
reorder by pvalue
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/analysis/DE_analysis.Rmd" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/DE_analysis.html" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/analysis/DE_analysis.Rmd" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/158fdf3e906a4a488e486cb96684b4e23b567895/docs/DE_analysis.html" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/analysis/DE_analysis.Rmd" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/4e7c93ca3c08f7a4f341d13fb1e729f48d99707d/docs/DE_analysis.html" target="_blank">4e7c93c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Remove 18856 strain
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/analysis/DE_analysis.Rmd" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/DE_analysis.html" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/analysis/DE_analysis.Rmd" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/DE_analysis.html" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add GO analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/analysis/DE_analysis.Rmd" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/03b499e0b710c5a48bba76aef6472cc713c58301/docs/DE_analysis.html" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Add counts of significant DE Genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5eb29e88b6057837b2c5c0719c225364eb484690/docs/DE_analysis.html" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/DE_analysis.html" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/DE_analysis.html" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/analysis/DE_analysis.Rmd" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/DE_analysis.html" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/analysis/DE_analysis.Rmd" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/DE_analysis.html" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
<td>
Add functions to plot Limma results
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/analysis/DE_analysis.Rmd" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/DE_analysis.html" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Revise limma commands
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/d80f643520bd77f37ac3f3862b7b747bccdbba30/analysis/DE_analysis.Rmd" target="_blank">d80f643</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
Run Limma:
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/DE_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/93017062e918d7a60fcad8ee8ca0983de6e9aef7/analysis/DE_analysis.Rmd" target="_blank">9301706</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
build analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/DE_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/DE_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<pre class="r"><code>library(&quot;limma&quot;)
library(&quot;plyr&quot;)
library(&quot;edgeR&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;ashr&quot;)
library(&quot;ggplot2&quot;)
library(&quot;cowplot&quot;)
library(&quot;VennDiagram&quot;)
library(&quot;RUVSeq&quot;)
library(&quot;topGO&quot;)

#load in filtered count data, RUVs output
filt_counts &lt;- readRDS(&quot;data/filtered_counts.rds&quot;)
filt_counts &lt;- filt_counts$counts
RUVsOut &lt;- readRDS(&quot;data/RUVsOut.rds&quot;)

# load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

# load in reordered sample information
sampleinfo &lt;- readRDS(&quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="limma-doesnt-deal-with-technical-replicates-so-here-i-randomly-sample-1-replicate-for-each-of-the-6-individualxcondition" class="section level1">
<h1>Limma doesn’t deal with technical replicates, so here I randomly sample 1 replicate for each of the 6 individualxcondition</h1>
<pre class="r"><code>#randomly sample 1 replicate from each set of 3 replicates
individuals &lt;- c(&quot;18855&quot;, &quot;18856&quot;, &quot;19160&quot;)
treatments &lt;- c(&quot;S&quot;, &quot;U&quot;)

set.seed(1234)
selected_samples &lt;- c()
for(i in c(&quot;18856&quot;, &quot;18855&quot;)){
     for(t in treatments){
          sample_num &lt;- base::sample(c(1:3), 1)
          selected_samples &lt;- c(selected_samples, paste0(i, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, t))
     }
}

sample_num &lt;- base::sample(c(2:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;U&quot;))
sample_num &lt;- base::sample(c(1:3), 1)
selected_samples &lt;- c(selected_samples, paste0(&quot;19160&quot;, sep = &quot;_&quot;, sample_num, sep = &quot;_&quot;, &quot;S&quot;))

selected_samples</code></pre>
<pre><code>[1] &quot;18856_2_S&quot; &quot;18856_2_U&quot; &quot;18855_1_S&quot; &quot;18855_3_U&quot; &quot;19160_2_U&quot; &quot;19160_1_S&quot;</code></pre>
<pre class="r"><code>randomly_sampled_indices &lt;- match(selected_samples, colnames(filt_counts))

single_filt_counts &lt;- filt_counts[,randomly_sampled_indices]
anno &lt;- pData(RUVsOut)[randomly_sampled_indices,]</code></pre>
<pre class="r"><code>single_filt_counts &lt;- filt_counts
anno &lt;- pData(RUVsOut)
x &lt;- paste0(anno$Individual, anno$treatment)

anno$LibraryPrepBatch &lt;- factor(anno$LibraryPrepBatch, levels = c(&quot;1&quot;, &quot;2&quot;))
anno$Replicate &lt;- factor(anno$Replicate, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;))

design &lt;- model.matrix(~treatment + Individual + W_1 + W_2 + RIN,
                       data = anno)
colnames(design) &lt;- gsub(&quot;treatment&quot;, &quot;&quot;, colnames(design))
colnames(design)</code></pre>
<pre><code>[1] &quot;(Intercept)&quot;        &quot;Unstrain&quot;           &quot;IndividualNA18856 &quot;
[4] &quot;IndividualNA19160 &quot; &quot;W_1&quot;                &quot;W_2&quot;               
[7] &quot;RIN&quot;               </code></pre>
<pre class="r"><code># Model individual as a random effect (use duplicate correlation function); treatment as a fixed effect, covariates = RUV factors
     # Because of issues with duplicateCorrelation with the number of covariates I include in the design matrix, I elect intead to include all samples (all technical replicates) and use duplicateCorrelation for technical replicates and instead include Individual as a fixed effect in the linear model
# Recommended to run both voom and duplicateCorrelation twice.
# https://support.bioconductor.org/p/59700/#67620

#TMM Normalization
y &lt;- DGEList(single_filt_counts)
y &lt;- calcNormFactors(y, method = &quot;TMM&quot;)
#Voom for differential expression
v1 &lt;- voom(y, design)
corfit1 &lt;- duplicateCorrelation(v1, design, block = x)
corfit1$consensus.correlation</code></pre>
<pre><code>[1] 0.354764</code></pre>
<pre class="r"><code>v2 &lt;- voom(y, design, block = x, correlation = corfit1$consensus.correlation)
corfit2 &lt;- duplicateCorrelation(v2, design, block = x)
corfit2$consensus.correlation</code></pre>
<pre><code>[1] 0.3549247</code></pre>
<pre class="r"><code>fit &lt;- lmFit(v2, design, block = x,
             correlation = corfit2$consensus.correlation)
fit &lt;- eBayes(fit)

saveRDS(v2, &quot;output/voom_results.rds&quot;)</code></pre>
<div id="assess-model-results" class="section level2">
<h2>Assess model results</h2>
<pre class="r"><code>get_results &lt;- function(x, number = nrow(x$coefficients), sort.by = &quot;none&quot;,
                        ...) {
  # x - object MArrayLM from eBayes output
  # ... - additional arguments passed to topTable
  stopifnot(class(x) == &quot;MArrayLM&quot;)
  results &lt;- topTable(x, number = number, sort.by = sort.by, ...)
  return(results)
}

top_treatment &lt;- get_results(fit, coef = &quot;Unstrain&quot;, sort.by = &quot;B&quot;)
head(top_treatment)</code></pre>
<pre><code>                     logFC  AveExpr         t      P.Value    adj.P.Val
ENSG00000102317  0.9646069 6.618528  9.788653 1.959478e-08 0.0002047067
ENSG00000104368 -3.0887783 3.753098 -7.935283 3.851911e-07 0.0020120458
ENSG00000100292 -1.1068651 5.399316 -7.090906 1.727530e-06 0.0058401006
ENSG00000080824 -0.5373642 8.888436 -6.533334 4.905709e-06 0.0058401006
ENSG00000133142  0.5627867 7.056389  6.522848 5.004984e-06 0.0058401006
ENSG00000181019 -0.7843440 5.824401 -6.517648 5.054991e-06 0.0058401006
                       B
ENSG00000102317 9.570798
ENSG00000104368 6.229870
ENSG00000100292 5.330764
ENSG00000080824 4.350280
ENSG00000133142 4.328594
ENSG00000181019 4.322443</code></pre>
<pre class="r"><code>results_treatment &lt;- get_results(fit, coef = &quot;Unstrain&quot;,
                              number = nrow(filt_counts), sort = &quot;none&quot;)
ma_treatment &lt;- ggplot(data.frame(Amean = fit$Amean, logFC = fit$coef[, &quot;Unstrain&quot;]),
                       aes(x = Amean, y = logFC)) +
  geom_point() +
  labs(x = &quot;Average expression level&quot;, y = &quot;Log fold change&quot;,
       title = &quot;Treatment effect&quot;)
ma_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-model-results-1">
Past versions of model-results-1.png
</button>
</p>
<div id="fig-model-results-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/figure/DE_analysis.Rmd/model-results-1.png" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist_treatment &lt;- ggplot(results_treatment, aes(x = P.Value)) +
  geom_histogram(binwidth = 0.01) +
  labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;, title = &quot;Treatment effect&quot;)
hist_treatment</code></pre>
<p><img src="figure/DE_analysis.Rmd/model-results-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-model-results-2">
Past versions of model-results-2.png
</button>
</p>
<div id="fig-model-results-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/figure/DE_analysis.Rmd/model-results-2.png" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="explore-top-hits" class="section level2">
<h2>Explore top hits</h2>
<p>Boxplot function.</p>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An EList object containing log2 counts per million
  # g - character vector of a single gene
  stopifnot(class(v) == &quot;EList&quot;,
            is.character(g), length(g) == 1)
  library(&quot;tidyr&quot;)
  single_gene &lt;- v$E[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))
  
  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
       labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;)) + 
       geom_dotplot(binaxis = &quot;y&quot;, stackdir = &quot;center&quot;) + 
       stat_summary(fun.data = &quot;mean_sdl&quot;, fun.args = list(mult=1))
  
}</code></pre>
<p>Treatment</p>
<pre class="r"><code>#rename top_treatment genes to gene symbols for easier interpretation
top_treatment_top4 &lt;- gene_anno$external_gene_name[match(head(rownames(top_treatment), n=4), gene_anno$ensembl_gene_id)]
top_treatment_top4</code></pre>
<pre><code>[1] RBM3     PLAT     HMOX1    HSP90AA1
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 ... ZZEF1</code></pre>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(v2, rownames(top_treatment)[1])
p_treatment_2 &lt;- plot_gene(v2, rownames(top_treatment)[2]) 
p_treatment_3 &lt;- plot_gene(v2, rownames(top_treatment)[3]) 
p_treatment_4 &lt;- plot_gene(v2, rownames(top_treatment)[4]) 
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<pre class="r"><code>plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-treatment-1">
Past versions of treatment-1.png
</button>
</p>
<div id="fig-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/treatment-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="use-ash-for-mutliple-testing-correction" class="section level2">
<h2>Use ash for mutliple testing correction</h2>
<p>Treatment effect.</p>
<pre class="r"><code>run_ash &lt;- function(x, coef) {
  # Perform multiple testing correction with adaptive shrinkage (ASH)
  #
  # x - object MArrayLM from eBayes output
  # coef - coefficient tested by eBayes
  stopifnot(class(x) == &quot;MArrayLM&quot;, coef %in% colnames(x$coefficients))
  result &lt;- ash(betahat = x$coefficients[, coef],
                sebetahat = x$stdev.unscaled[, coef] * sqrt(x$s2.post),
                df = x$df.total[1])
  return(result)
}

ash_treatment &lt;- run_ash(fit, &quot;Unstrain&quot;)
class(ash_treatment)</code></pre>
<pre><code>[1] &quot;ash&quot;</code></pre>
<pre class="r"><code>names(ash_treatment)</code></pre>
<pre><code>[1] &quot;fitted_g&quot; &quot;loglik&quot;   &quot;logLR&quot;    &quot;data&quot;     &quot;result&quot;   &quot;call&quot;    </code></pre>
<pre class="r"><code>sum(ash_treatment$result$svalue &lt; .05)</code></pre>
<pre><code>[1] 784</code></pre>
<pre class="r"><code>hist(ash_treatment$result$svalue)</code></pre>
<p><img src="figure/DE_analysis.Rmd/ash-treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ash-treatment-1">
Past versions of ash-treatment-1.png
</button>
</p>
<div id="fig-ash-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/ash-treatment-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="plots" class="section level1">
<h1>Plots</h1>
<pre class="r"><code>plot_ma &lt;- function(x, qval) {
  # Create MA plot.
  #
  # x - data frame with topTable and ASH output
  #     (columns logFC, AveExpr, and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;AveExpr&quot;, &quot;qvalue&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = AveExpr, y = logFC, color = highlight, shape = highlight)) +
    geom_point() +
    labs(x = &quot;Average expression level&quot;, y = &quot;Log fold change&quot;) +
    scale_color_identity(drop = FALSE) +
    scale_shape_manual(values = c(16, 1), drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
#   scale_color_gradient(low = &quot;red&quot;, high = &quot;white&quot;, limits = c(0, 0.25))
}

plot_volcano &lt;- function(x, qval) {
  # Create volcano plot.
  #
  # x - data frame with topTable and ASH output
  #     (columns logFC, P.Value, and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;P.Value&quot;, &quot;qvalue&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(P.Value), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, qval) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;P.Value&quot;, &quot;qvalue&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$qvalue &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = P.Value, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}</code></pre>
<pre class="r"><code>tests &lt;- colnames(fit$coefficients)
results &lt;- vector(length = length(tests), mode = &quot;list&quot;)
names(results) &lt;- tests

for (test in tests) {
  # Extract limma results
  results[[test]] &lt;- get_results(fit, coef = test)
  # Add mutliple testing correction with ASH
  output_ash &lt;- run_ash(fit, coef = test)$result
  results[[test]] &lt;- cbind(results[[test]], lfsr = output_ash$lfsr,
                           lfdr = output_ash$lfdr, qvalue = output_ash$qvalue,
                           svalue = output_ash$svalue)
}

#FDR 0.1
plot_ma(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-1">
Past versions of “use Functions-1.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-1.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-2">
Past versions of “use Functions-2.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-2.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.1)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-3">
Past versions of “use Functions-3.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5c249f2f98d1b83fb038052c768a3570dbdb04c5/docs/figure/DE_analysis.Rmd/use Functions-3.png" target="_blank">5c249f2</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.1)</code></pre>
<pre><code>
FALSE  TRUE 
 8154  2293 </code></pre>
<pre class="r"><code>significant_genes_10 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.1]
significant_symbols_10 &lt;- gene_anno$external_gene_name[match(significant_genes_10, gene_anno$ensembl_gene_id)]
head(significant_symbols_10)</code></pre>
<pre><code>[1] LINC01128  DVL1       MXRA8      AURKAIP1   AL391244.1 MRPL20    
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_10 &lt;- gene_anno[match(significant_genes_10, gene_anno$ensembl_gene_id),]


#FDR 0.05
plot_ma(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-4">
Past versions of “use Functions-4.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-4.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-5">
Past versions of “use Functions-5.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-5.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.05)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-6">
Past versions of “use Functions-6.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-6.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05)</code></pre>
<pre><code>
FALSE  TRUE 
 9438  1009 </code></pre>
<pre class="r"><code>significant_genes_05 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05]
significant_symbols_05 &lt;- gene_anno$external_gene_name[match(significant_genes_05, gene_anno$ensembl_gene_id)]
head(significant_symbols_05)</code></pre>
<pre><code>[1] DVL1   MRPL20 MMP23B PEX10  CEP104 ACOT7 
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_05 &lt;- gene_anno[match(significant_genes_05, gene_anno$ensembl_gene_id),]
save(significant_genes_05, file=&quot;data/DEgenes_FDR05.rda&quot;)

#FDR 0.01
plot_ma(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-7">
Past versions of “use Functions-7.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-7.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_volcano(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-8">
Past versions of “use Functions-8.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-8.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(results[[&quot;Unstrain&quot;]], 0.01)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20Functions-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-Functions-9">
Past versions of “use Functions-9.png”
</button>
</p>
<div id="fig-no-spaces-use-Functions-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a04a1ac95f43c8a9cb54d14c3c67349dada7e8f9/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">a04a1ac</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/03b499e0b710c5a48bba76aef6472cc713c58301/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">03b499e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/figure/DE_analysis.Rmd/use Functions-9.png" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.01)</code></pre>
<pre><code>
FALSE  TRUE 
10317   130 </code></pre>
<pre class="r"><code>significant_genes_01 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.01]
significant_symbols_01 &lt;- gene_anno$external_gene_name[match(significant_genes_01, gene_anno$ensembl_gene_id)]
head(significant_symbols_01)</code></pre>
<pre><code>[1] SPSB1      TENT5B     RCC1       SLC6A9     PLK3       ST6GALNAC5
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 ... ZZEF1</code></pre>
<pre class="r"><code>significant_anno_01 &lt;- gene_anno[match(significant_genes_01, gene_anno$ensembl_gene_id),]</code></pre>
<div id="examining-top-differentially-expressed-genes-fdr-0.05" class="section level2">
<h2>Examining top Differentially expressed genes (FDR 0.05)</h2>
<pre class="r"><code>#abs(logFC) &gt; 1
significant_genes_05_FC1 &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05 &amp; abs(results[[&quot;Unstrain&quot;]]$logFC) &gt; 1]
significant_symbols_05_FC1 &lt;- gene_anno$external_gene_name[match(significant_genes_05_FC1, gene_anno$ensembl_gene_id)]
significant_anno_05_FC1 &lt;- gene_anno[match(significant_genes_05_FC1, gene_anno$ensembl_gene_id),]

significant_anno_05_FC1</code></pre>
<pre><code>      ensembl_gene_id external_gene_name chromosome_name
31021 ENSG00000232679          LINC01705               1
4381  ENSG00000115221              ITGB6               2
7662  ENSG00000138411              HECW2               2
28538 ENSG00000229056         AC020571.1               2
5175  ENSG00000121577             POPDC2               3
934   ENSG00000065534               MYLK               3
13727 ENSG00000174640            SLCO2A1               3
14924 ENSG00000180801               ARSJ               4
13297 ENSG00000172403             SYNPO2               4
9313  ENSG00000150630              VEGFC               4
13332 ENSG00000172548             NIPAL4               5
28168 ENSG00000228495          LINC01013               6
3341  ENSG00000106631               MYL7               7
10067 ENSG00000157168               NRG1               8
2910  ENSG00000104368               PLAT               8
457   ENSG00000026508               CD44              11
46915 ENSG00000261578         AP003119.3              11
3449  ENSG00000107796              ACTA2              10
7759  ENSG00000139044           B4GALNT3              12
11706 ENSG00000165891               E2F7              12
10325 ENSG00000159251              ACTC1              15
81    ENSG00000005102              MEOX1              17
5642  ENSG00000125398               SOX9              17
2562  ENSG00000101670               LIPG              18
2248  ENSG00000100292              HMOX1              22
                                                                                              description
31021                     long intergenic non-protein coding RNA 1705 [Source:HGNC Symbol;Acc:HGNC:52493]
4381                                           integrin subunit beta 6 [Source:HGNC Symbol;Acc:HGNC:6161]
7662  HECT, C2 and WW domain containing E3 ubiquitin protein ligase 2 [Source:HGNC Symbol;Acc:HGNC:29853]
28538                                                                                                    
5175                                       popeye domain containing 2 [Source:HGNC Symbol;Acc:HGNC:17648]
934                                          myosin light chain kinase [Source:HGNC Symbol;Acc:HGNC:7590]
13727      solute carrier organic anion transporter family member 2A1 [Source:HGNC Symbol;Acc:HGNC:10955]
14924                                   arylsulfatase family member J [Source:HGNC Symbol;Acc:HGNC:26286]
13297                                                  synaptopodin 2 [Source:HGNC Symbol;Acc:HGNC:17732]
9313                             vascular endothelial growth factor C [Source:HGNC Symbol;Acc:HGNC:12682]
13332                                   NIPA like domain containing 4 [Source:HGNC Symbol;Acc:HGNC:28018]
28168                     long intergenic non-protein coding RNA 1013 [Source:HGNC Symbol;Acc:HGNC:48987]
3341                                             myosin light chain 7 [Source:HGNC Symbol;Acc:HGNC:21719]
10067                                                     neuregulin 1 [Source:HGNC Symbol;Acc:HGNC:7997]
2910                                plasminogen activator, tissue type [Source:HGNC Symbol;Acc:HGNC:9051]
457                                 CD44 molecule (Indian blood group) [Source:HGNC Symbol;Acc:HGNC:1681]
46915                                                                                                    
3449                               actin, alpha 2, smooth muscle, aorta [Source:HGNC Symbol;Acc:HGNC:130]
7759                    beta-1,4-N-acetyl-galactosaminyltransferase 3 [Source:HGNC Symbol;Acc:HGNC:24137]
11706                                      E2F transcription factor 7 [Source:HGNC Symbol;Acc:HGNC:23820]
10325                                    actin, alpha, cardiac muscle 1 [Source:HGNC Symbol;Acc:HGNC:143]
81                                               mesenchyme homeobox 1 [Source:HGNC Symbol;Acc:HGNC:7013]
5642                                                        SRY-box 9 [Source:HGNC Symbol;Acc:HGNC:11204]
2562                                        lipase G, endothelial type [Source:HGNC Symbol;Acc:HGNC:6623]
2248                                                  heme oxygenase 1 [Source:HGNC Symbol;Acc:HGNC:5013]
                                                                                                                                                                                                                                                       phenotype
31021                                                                                                                                                                                                                                                           
4381                                                                                                         Alopecia-intellectual disability syndrome;Hypocalcified amelogenesis imperfecta;Hypoplastic amelogenesis imperfecta;AMELOGENESIS IMPERFECTA TYPE IH
7662                                                                                                                                                                               HECW2;NEURODEVELOPMENTAL DISORDER WITH HYPOTONIA SEIZURES AND ABSENT LANGUAGE
28538                                                                                                                                                                                                                                                           
5175                                                                                                                                                                                                                                                            
934                                                                                                        Megacystis-microcolon-intestinal hypoperistalsis syndrome;Familial thoracic aortic aneurysm and aortic dissection;AORTIC ANEURYSM FAMILIAL THORACIC 7
13727                                                                                                                         Chronic enteropathy associated with SLCO2A1 gene;Pachydermoperiostosis;HYPERTROPHIC OSTEOARTHROPATHY PRIMARY AUTOSOMAL RECESSIVE 2
14924                                                                                                                                                                                                                                                           
13297                                                                                                                                                                                                                                                           
9313                                                                                                                                                                                                                     Milroy Disease;Lymphedema hereditary id
13332                                                                                                                                           Congenital nonbullous ichthyosiform erythroderma;Lamellar ichthyosis;ICHTHYOSIS CONGENITAL AUTOSOMAL RECESSIVE 6
28168                                                                                                                                                                                                                                                           
3341                                                                                                                                                                                                                                                            
10067                                                                                                                                                                                                                                            SCHIZOPHRENIA 6
2910                                                                                                                         Lethal hydranencephaly-diaphragmatic hernia syndrome;THROMBOPHILIA FAMILIAL DUE TO DECREASED RELEASE OF TISSUE PLASMINOGENACTIVATOR
457                                                                                                                                                                                                                                    BLOOD GROUP INDIAN SYSTEM
46915                                                                                                                                                                                                                                                           
3449                                                                            AORTIC ANEURYSM FAMILIAL THORACIC 6;Moyamoya disease 5;Familial thoracic aortic aneurysm and aortic dissection;Moyamoya disease;MULTISYSTEMIC SMOOTH MUSCLE DYSFUNCTION SYNDROME
7759                                                                                                                                                                                                                                                            
11706                                                                                                                                                                                                                                                           
10325                                                 Atrial septal defect ostium secundum type;Familial isolated dilated cardiomyopathy;Left ventricular noncompaction;ATRIAL SEPTAL DEFECT 5;Cardiomyopathy dilated 1R;CARDIOMYOPATHY FAMILIAL HYPERTROPHIC 11
81                                                                                                                                                               Klippel-Feil Anomaly;Isolated Klippel-Feil syndrome;KLIPPEL-FEIL SYNDROME 2 AUTOSOMAL RECESSIVE
5642  CAMPOMELIC DYSPLASIA;PIERRE ROBIN SEQUENCE;46XX ovotesticular disorder of sex development;46XX testicular disorder of sex development;46XY complete gonadal dysgenesis;46XY partial gonadal dysgenesis;Isolated Pierre Robin syndrome;46XY SEX REVERSAL 10
2562                                                                                                                                                                                                                                                            
2248                                                                                                                                                                                           HEME OXYGENASE 1 DEFICIENCY;PULMONARY DISEASE CHRONIC OBSTRUCTIVE</code></pre>
<pre class="r"><code>#separate into up and downregulated
# upregulated
#logFC &gt; 1
significant_genes_05_FC1_up &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05 &amp; results[[&quot;Unstrain&quot;]]$logFC &gt; 1]
significant_symbols_05_FC1_up &lt;- gene_anno$external_gene_name[match(significant_genes_05_FC1_up, gene_anno$ensembl_gene_id)]
significant_anno_05_FC1_up &lt;- gene_anno[match(significant_genes_05_FC1_up, gene_anno$ensembl_gene_id),]

significant_anno_05_FC1_up</code></pre>
<pre><code>      ensembl_gene_id external_gene_name chromosome_name
5175  ENSG00000121577             POPDC2               3
934   ENSG00000065534               MYLK               3
14924 ENSG00000180801               ARSJ               4
13297 ENSG00000172403             SYNPO2               4
13332 ENSG00000172548             NIPAL4               5
28168 ENSG00000228495          LINC01013               6
3341  ENSG00000106631               MYL7               7
10067 ENSG00000157168               NRG1               8
3449  ENSG00000107796              ACTA2              10
10325 ENSG00000159251              ACTC1              15
81    ENSG00000005102              MEOX1              17
                                                                          description
5175                   popeye domain containing 2 [Source:HGNC Symbol;Acc:HGNC:17648]
934                      myosin light chain kinase [Source:HGNC Symbol;Acc:HGNC:7590]
14924               arylsulfatase family member J [Source:HGNC Symbol;Acc:HGNC:26286]
13297                              synaptopodin 2 [Source:HGNC Symbol;Acc:HGNC:17732]
13332               NIPA like domain containing 4 [Source:HGNC Symbol;Acc:HGNC:28018]
28168 long intergenic non-protein coding RNA 1013 [Source:HGNC Symbol;Acc:HGNC:48987]
3341                         myosin light chain 7 [Source:HGNC Symbol;Acc:HGNC:21719]
10067                                 neuregulin 1 [Source:HGNC Symbol;Acc:HGNC:7997]
3449           actin, alpha 2, smooth muscle, aorta [Source:HGNC Symbol;Acc:HGNC:130]
10325                actin, alpha, cardiac muscle 1 [Source:HGNC Symbol;Acc:HGNC:143]
81                           mesenchyme homeobox 1 [Source:HGNC Symbol;Acc:HGNC:7013]
                                                                                                                                                                                                       phenotype
5175                                                                                                                                                                                                            
934                                                        Megacystis-microcolon-intestinal hypoperistalsis syndrome;Familial thoracic aortic aneurysm and aortic dissection;AORTIC ANEURYSM FAMILIAL THORACIC 7
14924                                                                                                                                                                                                           
13297                                                                                                                                                                                                           
13332                                                                                           Congenital nonbullous ichthyosiform erythroderma;Lamellar ichthyosis;ICHTHYOSIS CONGENITAL AUTOSOMAL RECESSIVE 6
28168                                                                                                                                                                                                           
3341                                                                                                                                                                                                            
10067                                                                                                                                                                                            SCHIZOPHRENIA 6
3449                            AORTIC ANEURYSM FAMILIAL THORACIC 6;Moyamoya disease 5;Familial thoracic aortic aneurysm and aortic dissection;Moyamoya disease;MULTISYSTEMIC SMOOTH MUSCLE DYSFUNCTION SYNDROME
10325 Atrial septal defect ostium secundum type;Familial isolated dilated cardiomyopathy;Left ventricular noncompaction;ATRIAL SEPTAL DEFECT 5;Cardiomyopathy dilated 1R;CARDIOMYOPATHY FAMILIAL HYPERTROPHIC 11
81                                                                                                               Klippel-Feil Anomaly;Isolated Klippel-Feil syndrome;KLIPPEL-FEIL SYNDROME 2 AUTOSOMAL RECESSIVE</code></pre>
<pre class="r"><code># downregulated
#logFC &lt; -1
significant_genes_05_FC1_down &lt;- row.names(results[[&quot;Unstrain&quot;]])[results[[&quot;Unstrain&quot;]]$qvalue &lt; 0.05 &amp; results[[&quot;Unstrain&quot;]]$logFC &lt; -1]
significant_symbols_05_FC1_down &lt;- gene_anno$external_gene_name[match(significant_genes_05_FC1_down, gene_anno$ensembl_gene_id)]
significant_anno_05_FC1_down &lt;- gene_anno[match(significant_genes_05_FC1_down, gene_anno$ensembl_gene_id),]

significant_anno_05_FC1_down</code></pre>
<pre><code>      ensembl_gene_id external_gene_name chromosome_name
31021 ENSG00000232679          LINC01705               1
4381  ENSG00000115221              ITGB6               2
7662  ENSG00000138411              HECW2               2
28538 ENSG00000229056         AC020571.1               2
13727 ENSG00000174640            SLCO2A1               3
9313  ENSG00000150630              VEGFC               4
2910  ENSG00000104368               PLAT               8
457   ENSG00000026508               CD44              11
46915 ENSG00000261578         AP003119.3              11
7759  ENSG00000139044           B4GALNT3              12
11706 ENSG00000165891               E2F7              12
5642  ENSG00000125398               SOX9              17
2562  ENSG00000101670               LIPG              18
2248  ENSG00000100292              HMOX1              22
                                                                                              description
31021                     long intergenic non-protein coding RNA 1705 [Source:HGNC Symbol;Acc:HGNC:52493]
4381                                           integrin subunit beta 6 [Source:HGNC Symbol;Acc:HGNC:6161]
7662  HECT, C2 and WW domain containing E3 ubiquitin protein ligase 2 [Source:HGNC Symbol;Acc:HGNC:29853]
28538                                                                                                    
13727      solute carrier organic anion transporter family member 2A1 [Source:HGNC Symbol;Acc:HGNC:10955]
9313                             vascular endothelial growth factor C [Source:HGNC Symbol;Acc:HGNC:12682]
2910                                plasminogen activator, tissue type [Source:HGNC Symbol;Acc:HGNC:9051]
457                                 CD44 molecule (Indian blood group) [Source:HGNC Symbol;Acc:HGNC:1681]
46915                                                                                                    
7759                    beta-1,4-N-acetyl-galactosaminyltransferase 3 [Source:HGNC Symbol;Acc:HGNC:24137]
11706                                      E2F transcription factor 7 [Source:HGNC Symbol;Acc:HGNC:23820]
5642                                                        SRY-box 9 [Source:HGNC Symbol;Acc:HGNC:11204]
2562                                        lipase G, endothelial type [Source:HGNC Symbol;Acc:HGNC:6623]
2248                                                  heme oxygenase 1 [Source:HGNC Symbol;Acc:HGNC:5013]
                                                                                                                                                                                                                                                       phenotype
31021                                                                                                                                                                                                                                                           
4381                                                                                                         Alopecia-intellectual disability syndrome;Hypocalcified amelogenesis imperfecta;Hypoplastic amelogenesis imperfecta;AMELOGENESIS IMPERFECTA TYPE IH
7662                                                                                                                                                                               HECW2;NEURODEVELOPMENTAL DISORDER WITH HYPOTONIA SEIZURES AND ABSENT LANGUAGE
28538                                                                                                                                                                                                                                                           
13727                                                                                                                         Chronic enteropathy associated with SLCO2A1 gene;Pachydermoperiostosis;HYPERTROPHIC OSTEOARTHROPATHY PRIMARY AUTOSOMAL RECESSIVE 2
9313                                                                                                                                                                                                                     Milroy Disease;Lymphedema hereditary id
2910                                                                                                                         Lethal hydranencephaly-diaphragmatic hernia syndrome;THROMBOPHILIA FAMILIAL DUE TO DECREASED RELEASE OF TISSUE PLASMINOGENACTIVATOR
457                                                                                                                                                                                                                                    BLOOD GROUP INDIAN SYSTEM
46915                                                                                                                                                                                                                                                           
7759                                                                                                                                                                                                                                                            
11706                                                                                                                                                                                                                                                           
5642  CAMPOMELIC DYSPLASIA;PIERRE ROBIN SEQUENCE;46XX ovotesticular disorder of sex development;46XX testicular disorder of sex development;46XY complete gonadal dysgenesis;46XY partial gonadal dysgenesis;Isolated Pierre Robin syndrome;46XY SEX REVERSAL 10
2562                                                                                                                                                                                                                                                            
2248                                                                                                                                                                                           HEME OXYGENASE 1 DEFICIENCY;PULMONARY DISEASE CHRONIC OBSTRUCTIVE</code></pre>
</div>
</div>
<div id="boxplot-of-expression-levels-of-the-top-de-genes-separated-by-individual-x-treatment" class="section level1">
<h1>boxplot of expression levels of the top DE genes separated by individual x treatment</h1>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An EList object containing log2 counts per million
  # g - character vector of a single gene
  stopifnot(class(v) == &quot;EList&quot;,
            is.character(g), length(g) == 1)
  library(&quot;tidyr&quot;)
  single_gene &lt;- v$E[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))

  single_gene_long$Treatment &lt;- gsub(&#39;S&#39;, &#39;Strain&#39;, single_gene_long$Treatment)
  single_gene_long$Treatment &lt;- gsub(&#39;U&#39;, &#39;Control&#39;, single_gene_long$Treatment)
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels(factor(single_gene_long$Treatment))[c(1,2)])
  
  return(ggplot(single_gene_long, aes(x = Individual, y = log2cpm, fill = Individual)) +
              labs(title = g, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;)) + 
              geom_dotplot(binaxis = &quot;y&quot;, stackdir = &quot;center&quot;, key_glyph = &#39;rect&#39;, dotsize = 0.75) + 
              facet_wrap(~Treatment, strip.position = &#39;bottom&#39;) + 
              stat_summary(fun.data = &quot;mean_sdl&quot;, fun.args = list(mult=1)) +
              theme(axis.title.x=element_blank())
              )
}

#all top genes
for (gene in 1:length(significant_anno_05_FC1[,1])) {
     print(plot_gene(v2, as.character(significant_anno_05_FC1[gene,1])))
}</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-1">
Past versions of unnamed-chunk-1-1.png
</button>
</p>
<div id="fig-unnamed-chunk-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-1.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-1.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ea5b72e6347079882535a1b59ae8b850e48123ac/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-1.png" target="_blank">ea5b72e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-2">
Past versions of unnamed-chunk-1-2.png
</button>
</p>
<div id="fig-unnamed-chunk-1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-2.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-2.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-3">
Past versions of unnamed-chunk-1-3.png
</button>
</p>
<div id="fig-unnamed-chunk-1-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-3.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-3.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-4">
Past versions of unnamed-chunk-1-4.png
</button>
</p>
<div id="fig-unnamed-chunk-1-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-4.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-4.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-5">
Past versions of unnamed-chunk-1-5.png
</button>
</p>
<div id="fig-unnamed-chunk-1-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-5.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-5.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-6">
Past versions of unnamed-chunk-1-6.png
</button>
</p>
<div id="fig-unnamed-chunk-1-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-6.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-6.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-7">
Past versions of unnamed-chunk-1-7.png
</button>
</p>
<div id="fig-unnamed-chunk-1-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-7.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-7.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-8">
Past versions of unnamed-chunk-1-8.png
</button>
</p>
<div id="fig-unnamed-chunk-1-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-8.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-8.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-9">
Past versions of unnamed-chunk-1-9.png
</button>
</p>
<div id="fig-unnamed-chunk-1-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-9.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-9.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-10">
Past versions of unnamed-chunk-1-10.png
</button>
</p>
<div id="fig-unnamed-chunk-1-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-10.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-10.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-11">
Past versions of unnamed-chunk-1-11.png
</button>
</p>
<div id="fig-unnamed-chunk-1-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-11.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-11.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-12">
Past versions of unnamed-chunk-1-12.png
</button>
</p>
<div id="fig-unnamed-chunk-1-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-12.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-12.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-13.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-13">
Past versions of unnamed-chunk-1-13.png
</button>
</p>
<div id="fig-unnamed-chunk-1-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-13.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-13.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-14.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-14">
Past versions of unnamed-chunk-1-14.png
</button>
</p>
<div id="fig-unnamed-chunk-1-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-14.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-14.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-15.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-15">
Past versions of unnamed-chunk-1-15.png
</button>
</p>
<div id="fig-unnamed-chunk-1-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-15.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-15.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-16.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-16">
Past versions of unnamed-chunk-1-16.png
</button>
</p>
<div id="fig-unnamed-chunk-1-16" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-16.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-16.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-17.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-17">
Past versions of unnamed-chunk-1-17.png
</button>
</p>
<div id="fig-unnamed-chunk-1-17" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-17.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-17.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-18.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-18">
Past versions of unnamed-chunk-1-18.png
</button>
</p>
<div id="fig-unnamed-chunk-1-18" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-18.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-18.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-19.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-19">
Past versions of unnamed-chunk-1-19.png
</button>
</p>
<div id="fig-unnamed-chunk-1-19" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-19.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-19.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-20.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-20">
Past versions of unnamed-chunk-1-20.png
</button>
</p>
<div id="fig-unnamed-chunk-1-20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-20.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-20.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-21.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-21">
Past versions of unnamed-chunk-1-21.png
</button>
</p>
<div id="fig-unnamed-chunk-1-21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-21.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-21.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-22.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-22">
Past versions of unnamed-chunk-1-22.png
</button>
</p>
<div id="fig-unnamed-chunk-1-22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-22.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-22.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-23.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-23">
Past versions of unnamed-chunk-1-23.png
</button>
</p>
<div id="fig-unnamed-chunk-1-23" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-23.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-23.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-24.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-24">
Past versions of unnamed-chunk-1-24.png
</button>
</p>
<div id="fig-unnamed-chunk-1-24" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-24.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-24.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-25.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-25">
Past versions of unnamed-chunk-1-25.png
</button>
</p>
<div id="fig-unnamed-chunk-1-25" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-25.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c0e7c4535eba85795281a274263a8ddfbe3d3822/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-25.png" target="_blank">c0e7c45</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#SOX9
plot_gene(v2, &quot;ENSG00000125398&quot;)</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-26.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-26">
Past versions of unnamed-chunk-1-26.png
</button>
</p>
<div id="fig-unnamed-chunk-1-26" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-26.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#all top upregulated

for (gene in 1:length(significant_anno_05_FC1_up[,1])) {
     print(plot_gene(v2, as.character(significant_anno_05_FC1_up[gene,1])))
}</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-27.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-27">
Past versions of unnamed-chunk-1-27.png
</button>
</p>
<div id="fig-unnamed-chunk-1-27" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-27.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-28.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-28">
Past versions of unnamed-chunk-1-28.png
</button>
</p>
<div id="fig-unnamed-chunk-1-28" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-28.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-29.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-29">
Past versions of unnamed-chunk-1-29.png
</button>
</p>
<div id="fig-unnamed-chunk-1-29" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-29.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-30.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-30">
Past versions of unnamed-chunk-1-30.png
</button>
</p>
<div id="fig-unnamed-chunk-1-30" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-30.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-31.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-31">
Past versions of unnamed-chunk-1-31.png
</button>
</p>
<div id="fig-unnamed-chunk-1-31" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-31.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-32.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-32">
Past versions of unnamed-chunk-1-32.png
</button>
</p>
<div id="fig-unnamed-chunk-1-32" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-32.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-33.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-33">
Past versions of unnamed-chunk-1-33.png
</button>
</p>
<div id="fig-unnamed-chunk-1-33" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-33.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-34.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-34">
Past versions of unnamed-chunk-1-34.png
</button>
</p>
<div id="fig-unnamed-chunk-1-34" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-34.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-35.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-35">
Past versions of unnamed-chunk-1-35.png
</button>
</p>
<div id="fig-unnamed-chunk-1-35" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-35.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-36.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-36">
Past versions of unnamed-chunk-1-36.png
</button>
</p>
<div id="fig-unnamed-chunk-1-36" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-36.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-37.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-37">
Past versions of unnamed-chunk-1-37.png
</button>
</p>
<div id="fig-unnamed-chunk-1-37" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-37.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#all top downregulated

for (gene in 1:length(significant_anno_05_FC1_down[,1])) {
     print(plot_gene(v2, as.character(significant_anno_05_FC1_down[gene,1])))
}</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-38.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-38">
Past versions of unnamed-chunk-1-38.png
</button>
</p>
<div id="fig-unnamed-chunk-1-38" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-38.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-39.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-39">
Past versions of unnamed-chunk-1-39.png
</button>
</p>
<div id="fig-unnamed-chunk-1-39" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-39.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-40.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-40">
Past versions of unnamed-chunk-1-40.png
</button>
</p>
<div id="fig-unnamed-chunk-1-40" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-40.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-41.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-41">
Past versions of unnamed-chunk-1-41.png
</button>
</p>
<div id="fig-unnamed-chunk-1-41" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-41.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-42.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-42">
Past versions of unnamed-chunk-1-42.png
</button>
</p>
<div id="fig-unnamed-chunk-1-42" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-42.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-43.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-43">
Past versions of unnamed-chunk-1-43.png
</button>
</p>
<div id="fig-unnamed-chunk-1-43" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-43.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-44.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-44">
Past versions of unnamed-chunk-1-44.png
</button>
</p>
<div id="fig-unnamed-chunk-1-44" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-44.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-45.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-45">
Past versions of unnamed-chunk-1-45.png
</button>
</p>
<div id="fig-unnamed-chunk-1-45" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-45.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-46.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-46">
Past versions of unnamed-chunk-1-46.png
</button>
</p>
<div id="fig-unnamed-chunk-1-46" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-46.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-47.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-47">
Past versions of unnamed-chunk-1-47.png
</button>
</p>
<div id="fig-unnamed-chunk-1-47" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-47.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-48.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-48">
Past versions of unnamed-chunk-1-48.png
</button>
</p>
<div id="fig-unnamed-chunk-1-48" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-48.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-49.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-49">
Past versions of unnamed-chunk-1-49.png
</button>
</p>
<div id="fig-unnamed-chunk-1-49" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-49.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-50.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-50">
Past versions of unnamed-chunk-1-50.png
</button>
</p>
<div id="fig-unnamed-chunk-1-50" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-50.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/unnamed-chunk-1-51.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-51">
Past versions of unnamed-chunk-1-51.png
</button>
</p>
<div id="fig-unnamed-chunk-1-51" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/23d052f6efdb0dd53ad5d30d1da9517159694796/docs/figure/DE_analysis.Rmd/unnamed-chunk-1-51.png" target="_blank">23d052f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="gene-ontology-analysis-with-topgo" class="section level2">
<h2>Gene ontology analysis with topGO</h2>
<p>Use topGO for GO analysis. It accounts for the nested graph structure of GO terms to prune the number of GO categories tested (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16606683">Alexa et al. 2006</a>). Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe. This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>saveRDS(results, &quot;data/DE_results.rds&quot;)
threshold &lt;- 0.05
gene_universe &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold)
gene_universe &lt;- factor(gene_universe)
names(gene_universe) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 
              0               0               0               0 
ENSG00000198744 ENSG00000228794 
              0               0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>Loading required package: org.Hs.eg.db</code></pre>
<pre><code></code></pre>
<pre><code>    ( 9770 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13850 GO terms and 32624 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9172 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 5828 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  3 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  11 nodes to be scored   (11 eliminated genes)</code></pre>
<pre><code>
     Level 16:  19 nodes to be scored   (17 eliminated genes)</code></pre>
<pre><code>
     Level 15:  48 nodes to be scored   (66 eliminated genes)</code></pre>
<pre><code>
     Level 14:  103 nodes to be scored  (188 eliminated genes)</code></pre>
<pre><code>
     Level 13:  166 nodes to be scored  (543 eliminated genes)</code></pre>
<pre><code>
     Level 12:  282 nodes to be scored  (1395 eliminated genes)</code></pre>
<pre><code>
     Level 11:  479 nodes to be scored  (2889 eliminated genes)</code></pre>
<pre><code>
     Level 10:  656 nodes to be scored  (4272 eliminated genes)</code></pre>
<pre><code>
     Level 9:   774 nodes to be scored  (5382 eliminated genes)</code></pre>
<pre><code>
     Level 8:   798 nodes to be scored  (6693 eliminated genes)</code></pre>
<pre><code>
     Level 7:   874 nodes to be scored  (7528 eliminated genes)</code></pre>
<pre><code>
     Level 6:   761 nodes to be scored  (8153 eliminated genes)</code></pre>
<pre><code>
     Level 5:   471 nodes to be scored  (8537 eliminated genes)</code></pre>
<pre><code>
     Level 4:   247 nodes to be scored  (8781 eliminated genes)</code></pre>
<pre><code>
     Level 3:   112 nodes to be scored  (8931 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (8980 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9041 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated
1  GO:0055114                 oxidation-reduction process       603
2  GO:0046364         monosaccharide biosynthetic process        63
3  GO:0045454                      cell redox homeostasis        47
4  GO:0002576                      platelet degranulation        71
5  GO:0006027         glycosaminoglycan catabolic process        40
6  GO:0030198           extracellular matrix organization       206
7  GO:0001568                    blood vessel development       430
8  GO:0048842 positive regulation of axon extension in...         5
9  GO:0002063                     chondrocyte development        21
10 GO:0006986                response to unfolded protein       141
11 GO:0006695            cholesterol biosynthetic process        56
12 GO:0051897 positive regulation of protein kinase B ...        77
13 GO:0006516              glycoprotein catabolic process        14
14 GO:0050927 positive regulation of positive chemotax...        10
15 GO:0035987             endodermal cell differentiation        31
16 GO:0030199                collagen fibril organization        31
17 GO:0036499     PERK-mediated unfolded protein response        16
18 GO:0019321                   pentose metabolic process         7
19 GO:0034135 regulation of toll-like receptor 2 signa...         7
20 GO:0017185               peptidyl-lysine hydroxylation         7
21 GO:0060047                           heart contraction       112
22 GO:0050702                interleukin-1 beta secretion        18
23 GO:0034329                      cell junction assembly       150
24 GO:0006098                     pentose-phosphate shunt        12
25 GO:0042060                               wound healing       310
26 GO:1901700      response to oxygen-containing compound       898
27 GO:0006486                       protein glycosylation       151
28 GO:0022010          central nervous system myelination         8
29 GO:0033692 cellular polysaccharide biosynthetic pro...        39
30 GO:0032964               collagen biosynthetic process        29
31 GO:0006814                        sodium ion transport        86
32 GO:0061098 positive regulation of protein tyrosine ...        29
33 GO:0010332                 response to gamma radiation        45
34 GO:0006536                 glutamate metabolic process        13
35 GO:0070527                        platelet aggregation        35
36 GO:0033627          cell adhesion mediated by integrin        38
37 GO:0044267          cellular protein metabolic process      3266
38 GO:0006768                    biotin metabolic process         9
39 GO:0031643          positive regulation of myelination         9
40 GO:0043619 regulation of transcription from RNA pol...         9
41 GO:0021955 central nervous system neuron axonogenes...        25
42 GO:0010640 regulation of platelet-derived growth fa...        19
43 GO:2001256 regulation of store-operated calcium ent...         5
44 GO:0071500     cellular response to nitrosative stress         5
45 GO:0060638       mesenchymal-epithelial cell signaling         5
46 GO:0098712 L-glutamate import across plasma membran...         5
47 GO:0098780    response to mitochondrial depolarisation        18
48 GO:0030182                      neuron differentiation       735
49 GO:0060402          calcium ion transport into cytosol        66
50 GO:2000779    regulation of double-strand break repair        52
51 GO:0035270                endocrine system development        63
52 GO:0007187 G protein-coupled receptor signaling pat...        56
53 GO:0050665      hydrogen peroxide biosynthetic process         7
54 GO:0042362    fat-soluble vitamin biosynthetic process         8
55 GO:0022414                        reproductive process       663
56 GO:0014706          striated muscle tissue development       218
   Significant Expected weightFisher
1           90    60.29      1.7e-06
2            9     6.30      1.0e-04
3           14     4.70      0.00013
4           18     7.10      0.00016
5           10     4.00      0.00018
6           47    20.60      0.00034
7           64    42.99      0.00036
8            4     0.50      0.00046
9            8     2.10      0.00060
10          30    14.10      0.00073
11          14     5.60      0.00076
12          17     7.70      0.00130
13           6     1.40      0.00146
14           5     1.00      0.00162
15           9     3.10      0.00254
16           9     3.10      0.00254
17           5     1.60      0.00271
18           4     0.70      0.00271
19           4     0.70      0.00271
20           4     0.70      0.00271
21          18    11.20      0.00364
22           4     1.80      0.00369
23          22    15.00      0.00423
24           5     1.20      0.00430
25          49    30.99      0.00449
26         101    89.78      0.00482
27          26    15.10      0.00484
28           4     0.80      0.00500
29           6     3.90      0.00502
30           5     2.90      0.00502
31          15     8.60      0.00564
32           8     2.90      0.00615
33           7     4.50      0.00641
34           5     1.30      0.00641
35           8     3.50      0.00737
36          10     3.80      0.00819
37         323   326.53      0.00823
38           4     0.90      0.00829
39           4     0.90      0.00829
40           4     0.90      0.00829
41           5     2.50      0.00831
42           5     1.90      0.00852
43           3     0.50      0.00853
44           3     0.50      0.00853
45           3     0.50      0.00853
46           3     0.50      0.00853
47           4     1.80      0.00854
48          79    73.48      0.00940
49          14     6.60      0.00986
50           9     5.20      0.00994
51           9     6.30      0.00997
52           8     5.60      0.00997
53           3     0.70      0.00997
54           3     0.80      0.00998
55          68    66.29      0.00999
56          23    21.80      0.01000</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant
1                  oxidation-reduction process       603          90
2          monosaccharide biosynthetic process        63           9
3                       cell redox homeostasis        47          14
4                       platelet degranulation        71          18
5          glycosaminoglycan catabolic process        40          10
6            extracellular matrix organization       206          47
7                     blood vessel development       430          64
8  positive regulation of axon extension in...         5           4
9                      chondrocyte development        21           8
10                response to unfolded protein       141          30
11            cholesterol biosynthetic process        56          14
12 positive regulation of protein kinase B ...        77          17
13              glycoprotein catabolic process        14           6
14 positive regulation of positive chemotax...        10           5
15             endodermal cell differentiation        31           9
16                collagen fibril organization        31           9
17     PERK-mediated unfolded protein response        16           5
18                   pentose metabolic process         7           4
19 regulation of toll-like receptor 2 signa...         7           4
20               peptidyl-lysine hydroxylation         7           4
21                           heart contraction       112          18
22                interleukin-1 beta secretion        18           4
23                      cell junction assembly       150          22
24                     pentose-phosphate shunt        12           5
25                               wound healing       310          49
26      response to oxygen-containing compound       898         101
27                       protein glycosylation       151          26
28          central nervous system myelination         8           4
29 cellular polysaccharide biosynthetic pro...        39           6
30               collagen biosynthetic process        29           5
31                        sodium ion transport        86          15
32 positive regulation of protein tyrosine ...        29           8
33                 response to gamma radiation        45           7
34                 glutamate metabolic process        13           5
35                        platelet aggregation        35           8
36          cell adhesion mediated by integrin        38          10
37          cellular protein metabolic process      3266         323
38                    biotin metabolic process         9           4
39          positive regulation of myelination         9           4
40 regulation of transcription from RNA pol...         9           4
41 central nervous system neuron axonogenes...        25           5
42 regulation of platelet-derived growth fa...        19           5
43 regulation of store-operated calcium ent...         5           3
44     cellular response to nitrosative stress         5           3
45       mesenchymal-epithelial cell signaling         5           3
46 L-glutamate import across plasma membran...         5           3
47    response to mitochondrial depolarisation        18           4
48                      neuron differentiation       735          79
49          calcium ion transport into cytosol        66          14
50    regulation of double-strand break repair        52           9
51                endocrine system development        63           9
52 G protein-coupled receptor signaling pat...        56           8
53      hydrogen peroxide biosynthetic process         7           3
54    fat-soluble vitamin biosynthetic process         8           3
55                        reproductive process       663          68
56          striated muscle tissue development       218          23
   Expected weightFisher
1     60.29      1.7e-06
2      6.30      1.0e-04
3      4.70      0.00013
4      7.10      0.00016
5      4.00      0.00018
6     20.60      0.00034
7     42.99      0.00036
8      0.50      0.00046
9      2.10      0.00060
10    14.10      0.00073
11     5.60      0.00076
12     7.70      0.00130
13     1.40      0.00146
14     1.00      0.00162
15     3.10      0.00254
16     3.10      0.00254
17     1.60      0.00271
18     0.70      0.00271
19     0.70      0.00271
20     0.70      0.00271
21    11.20      0.00364
22     1.80      0.00369
23    15.00      0.00423
24     1.20      0.00430
25    30.99      0.00449
26    89.78      0.00482
27    15.10      0.00484
28     0.80      0.00500
29     3.90      0.00502
30     2.90      0.00502
31     8.60      0.00564
32     2.90      0.00615
33     4.50      0.00641
34     1.30      0.00641
35     3.50      0.00737
36     3.80      0.00819
37   326.53      0.00823
38     0.90      0.00829
39     0.90      0.00829
40     0.90      0.00829
41     2.50      0.00831
42     1.90      0.00852
43     0.50      0.00853
44     0.50      0.00853
45     0.50      0.00853
46     0.50      0.00853
47     1.80      0.00854
48    73.48      0.00940
49     6.60      0.00986
50     5.20      0.00994
51     6.30      0.00997
52     5.60      0.00997
53     0.70      0.00997
54     0.80      0.00998
55    66.29      0.00999
56    21.80      0.01000</code></pre>
</div>
<div id="gene-ontology-analysis-with-topgo-separate-up-and-downregulated-genes" class="section level2">
<h2>Gene ontology analysis with topGO (separate up and downregulated genes)</h2>
<p>Use topGO for GO analysis. Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe for upregulated genes This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.05
gene_universe_up &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold &amp; results[[&quot;Unstrain&quot;]]$logFC &gt; 0)
gene_universe_up &lt;- factor(gene_universe_up)
names(gene_universe_up) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe_up)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 
              0               0               0               0 
ENSG00000198744 ENSG00000228794 
              0               0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_up,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9770 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13850 GO terms and 32624 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9172 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 4586 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  2 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  6 nodes to be scored    (11 eliminated genes)</code></pre>
<pre><code>
     Level 16:  14 nodes to be scored   (16 eliminated genes)</code></pre>
<pre><code>
     Level 15:  27 nodes to be scored   (51 eliminated genes)</code></pre>
<pre><code>
     Level 14:  61 nodes to be scored   (166 eliminated genes)</code></pre>
<pre><code>
     Level 13:  112 nodes to be scored  (439 eliminated genes)</code></pre>
<pre><code>
     Level 12:  196 nodes to be scored  (1193 eliminated genes)</code></pre>
<pre><code>
     Level 11:  337 nodes to be scored  (2684 eliminated genes)</code></pre>
<pre><code>
     Level 10:  502 nodes to be scored  (4074 eliminated genes)</code></pre>
<pre><code>
     Level 9:   617 nodes to be scored  (5231 eliminated genes)</code></pre>
<pre><code>
     Level 8:   623 nodes to be scored  (6494 eliminated genes)</code></pre>
<pre><code>
     Level 7:   700 nodes to be scored  (7397 eliminated genes)</code></pre>
<pre><code>
     Level 6:   636 nodes to be scored  (8092 eliminated genes)</code></pre>
<pre><code>
     Level 5:   412 nodes to be scored  (8496 eliminated genes)</code></pre>
<pre><code>
     Level 4:   216 nodes to be scored  (8766 eliminated genes)</code></pre>
<pre><code>
     Level 3:   101 nodes to be scored  (8925 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (8977 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9041 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated
1  GO:0030198           extracellular matrix organization       206
2  GO:0030199                collagen fibril organization        31
3  GO:0006027         glycosaminoglycan catabolic process        40
4  GO:0032964               collagen biosynthetic process        29
5  GO:0001503                                ossification       246
6  GO:0006024      glycosaminoglycan biosynthetic process        75
7  GO:0006768                    biotin metabolic process         9
8  GO:0031643          positive regulation of myelination         9
9  GO:0035987             endodermal cell differentiation        31
10 GO:0034341                response to interferon-gamma        94
11 GO:0060638       mesenchymal-epithelial cell signaling         5
12 GO:0070208                 protein heterotrimerization        11
13 GO:0016266                         O-glycan processing        26
14 GO:0051897 positive regulation of protein kinase B ...        77
15 GO:0035455                response to interferon-alpha        12
16 GO:0010838 positive regulation of keratinocyte prol...         6
17 GO:0048709             oligodendrocyte differentiation        54
18 GO:0042362    fat-soluble vitamin biosynthetic process         8
19 GO:0045454                      cell redox homeostasis        47
20 GO:0003009                 skeletal muscle contraction        13
21 GO:0044267          cellular protein metabolic process      3266
22 GO:0017185               peptidyl-lysine hydroxylation         7
23 GO:0042476                               odontogenesis        77
24 GO:0033692 cellular polysaccharide biosynthetic pro...        39
25 GO:0022010          central nervous system myelination         8
26 GO:0034446 substrate adhesion-dependent cell spread...        62
27 GO:0042060                               wound healing       310
28 GO:0006486                       protein glycosylation       151
29 GO:0048592                           eye morphogenesis        84
30 GO:0085029               extracellular matrix assembly        16
31 GO:2000345      regulation of hepatocyte proliferation         8
32 GO:1900016 negative regulation of cytokine producti...         9
33 GO:0051004 regulation of lipoprotein lipase activit...         9
   Significant Expected weightFisher
1           41    11.01      1.5e-06
2            9     1.66      2.3e-05
3            9     2.14      0.00018
4            5     1.55      0.00047
5           27    13.14      0.00074
6           13     4.01      0.00079
7            4     0.48      0.00082
8            4     0.48      0.00082
9            7     1.66      0.00102
10           7     5.02      0.00132
11           3     0.27      0.00140
12           4     0.59      0.00197
13           6     1.39      0.00208
14          11     4.11      0.00246
15           4     0.64      0.00267
16           3     0.32      0.00268
17          10     2.88      0.00275
18           3     0.43      0.00284
19           8     2.51      0.00312
20           4     0.69      0.00391
21         152   174.48      0.00405
22           3     0.37      0.00451
23           9     4.11      0.00637
24           5     2.08      0.00692
25           3     0.43      0.00694
26           9     3.31      0.00765
27          24    16.56      0.00791
28          18     8.07      0.00798
29           9     4.49      0.00815
30           5     0.85      0.00817
31           3     0.43      0.00822
32           3     0.48      0.00999
33           3     0.48      0.00999</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant
1            extracellular matrix organization       206          41
2                 collagen fibril organization        31           9
3          glycosaminoglycan catabolic process        40           9
4                collagen biosynthetic process        29           5
5                                 ossification       246          27
6       glycosaminoglycan biosynthetic process        75          13
7                     biotin metabolic process         9           4
8           positive regulation of myelination         9           4
9              endodermal cell differentiation        31           7
10                response to interferon-gamma        94           7
11       mesenchymal-epithelial cell signaling         5           3
12                 protein heterotrimerization        11           4
13                         O-glycan processing        26           6
14 positive regulation of protein kinase B ...        77          11
15                response to interferon-alpha        12           4
16 positive regulation of keratinocyte prol...         6           3
17             oligodendrocyte differentiation        54          10
18    fat-soluble vitamin biosynthetic process         8           3
19                      cell redox homeostasis        47           8
20                 skeletal muscle contraction        13           4
21          cellular protein metabolic process      3266         152
22               peptidyl-lysine hydroxylation         7           3
23                               odontogenesis        77           9
24 cellular polysaccharide biosynthetic pro...        39           5
25          central nervous system myelination         8           3
26 substrate adhesion-dependent cell spread...        62           9
27                               wound healing       310          24
28                       protein glycosylation       151          18
29                           eye morphogenesis        84           9
30               extracellular matrix assembly        16           5
31      regulation of hepatocyte proliferation         8           3
32 negative regulation of cytokine producti...         9           3
33 regulation of lipoprotein lipase activit...         9           3
   Expected weightFisher
1     11.01      1.5e-06
2      1.66      2.3e-05
3      2.14      0.00018
4      1.55      0.00047
5     13.14      0.00074
6      4.01      0.00079
7      0.48      0.00082
8      0.48      0.00082
9      1.66      0.00102
10     5.02      0.00132
11     0.27      0.00140
12     0.59      0.00197
13     1.39      0.00208
14     4.11      0.00246
15     0.64      0.00267
16     0.32      0.00268
17     2.88      0.00275
18     0.43      0.00284
19     2.51      0.00312
20     0.69      0.00391
21   174.48      0.00405
22     0.37      0.00451
23     4.11      0.00637
24     2.08      0.00692
25     0.43      0.00694
26     3.31      0.00765
27    16.56      0.00791
28     8.07      0.00798
29     4.49      0.00815
30     0.85      0.00817
31     0.43      0.00822
32     0.48      0.00999
33     0.48      0.00999</code></pre>
<p>First create the gene universe for downregulated genes This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.05
gene_universe_down &lt;- as.numeric(results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold &amp; results[[&quot;Unstrain&quot;]]$logFC &lt; 0)
gene_universe_down &lt;- factor(gene_universe_down)
names(gene_universe_down) &lt;- rownames(results[[&quot;Unstrain&quot;]])
head(gene_universe_down)</code></pre>
<pre><code>ENSG00000225972 ENSG00000225630 ENSG00000237973 ENSG00000248527 
              0               0               0               0 
ENSG00000198744 ENSG00000228794 
              0               0 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_down,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9770 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13850 GO terms and 32624 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9172 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 4882 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  3 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  10 nodes to be scored   (11 eliminated genes)</code></pre>
<pre><code>
     Level 16:  16 nodes to be scored   (17 eliminated genes)</code></pre>
<pre><code>
     Level 15:  43 nodes to be scored   (63 eliminated genes)</code></pre>
<pre><code>
     Level 14:  88 nodes to be scored   (185 eliminated genes)</code></pre>
<pre><code>
     Level 13:  138 nodes to be scored  (481 eliminated genes)</code></pre>
<pre><code>
     Level 12:  220 nodes to be scored  (1285 eliminated genes)</code></pre>
<pre><code>
     Level 11:  370 nodes to be scored  (2750 eliminated genes)</code></pre>
<pre><code>
     Level 10:  521 nodes to be scored  (4087 eliminated genes)</code></pre>
<pre><code>
     Level 9:   644 nodes to be scored  (5185 eliminated genes)</code></pre>
<pre><code>
     Level 8:   642 nodes to be scored  (6570 eliminated genes)</code></pre>
<pre><code>
     Level 7:   746 nodes to be scored  (7429 eliminated genes)</code></pre>
<pre><code>
     Level 6:   661 nodes to be scored  (8061 eliminated genes)</code></pre>
<pre><code>
     Level 5:   423 nodes to be scored  (8489 eliminated genes)</code></pre>
<pre><code>
     Level 4:   229 nodes to be scored  (8761 eliminated genes)</code></pre>
<pre><code>
     Level 3:   105 nodes to be scored  (8930 eliminated genes)</code></pre>
<pre><code>
     Level 2:   21 nodes to be scored   (8978 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9041 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>        GO.ID                                        Term Annotated
1  GO:0046364         monosaccharide biosynthetic process        63
2  GO:0006695            cholesterol biosynthetic process        56
3  GO:0055114                 oxidation-reduction process       603
4  GO:0036499     PERK-mediated unfolded protein response        16
5  GO:0019321                   pentose metabolic process         7
6  GO:0006986                response to unfolded protein       141
7  GO:1900034     regulation of cellular response to heat        73
8  GO:0043066    negative regulation of apoptotic process       533
9  GO:0098712 L-glutamate import across plasma membran...         5
10 GO:0071500     cellular response to nitrosative stress         5
11 GO:0051085 chaperone cofactor-dependent protein ref...        11
12 GO:0006098                     pentose-phosphate shunt        12
13 GO:0030223                  neutrophil differentiation         6
14 GO:1905323      telomerase holoenzyme complex assembly         6
15 GO:0060402          calcium ion transport into cytosol        66
16 GO:0002063                     chondrocyte development        21
17 GO:0034135 regulation of toll-like receptor 2 signa...         7
18 GO:0090084 negative regulation of inclusion body as...         7
19 GO:0036289         peptidyl-serine autophosphorylation         7
20 GO:0046578 regulation of Ras protein signal transdu...       143
21 GO:0051131 chaperone-mediated protein complex assem...        14
22 GO:0045540 regulation of cholesterol biosynthetic p...        39
23 GO:0030511 positive regulation of transforming grow...        24
24 GO:0002446                neutrophil mediated immunity       307
25 GO:0035973                                  aggrephagy         8
26 GO:0090650 cellular response to oxygen-glucose depr...         8
27 GO:0001946                           lymphangiogenesis         8
28 GO:0002576                      platelet degranulation        71
29 GO:2000106 regulation of leukocyte apoptotic proces...        35
30 GO:1901214                  regulation of neuron death       186
31 GO:0032874 positive regulation of stress-activated ...        95
32 GO:0090050 positive regulation of cell migration in...        17
33 GO:0021955 central nervous system neuron axonogenes...        25
34 GO:0061179 negative regulation of insulin secretion...         9
35 GO:0043619 regulation of transcription from RNA pol...         9
36 GO:0051898 negative regulation of protein kinase B ...        38
37 GO:0071158    positive regulation of cell cycle arrest        65
38 GO:0046330          positive regulation of JNK cascade        76
39 GO:0050927 positive regulation of positive chemotax...        10
40 GO:0006793                phosphorus metabolic process      2030
41 GO:0001934 positive regulation of protein phosphory...       520
   Significant Expected weightFisher
1            6     2.93      4.7e-06
2           14     2.61      5.4e-06
3           45    28.07      6.2e-06
4            5     0.74      0.00014
5            4     0.33      0.00014
6           20     6.56      0.00049
7           11     3.40      0.00051
8           45    24.81      0.00056
9            3     0.23      0.00093
10           3     0.23      0.00093
11           4     0.51      0.00118
12           4     0.56      0.00170
13           3     0.28      0.00180
14           3     0.28      0.00180
15           7     3.07      0.00214
16           5     0.98      0.00234
17           3     0.33      0.00305
18           3     0.33      0.00305
19           3     0.33      0.00305
20          12     6.66      0.00314
21           4     0.65      0.00319
22           7     1.82      0.00432
23           5     1.12      0.00436
24          17    14.29      0.00471
25           3     0.37      0.00471
26           3     0.37      0.00471
27           3     0.37      0.00471
28           9     3.31      0.00542
29           5     1.63      0.00624
30          19     8.66      0.00647
31          13     4.42      0.00655
32           4     0.79      0.00680
33           4     1.16      0.00681
34           3     0.42      0.00682
35           3     0.42      0.00682
36           6     1.77      0.00764
37           6     3.03      0.00844
38           8     3.54      0.00858
39           3     0.47      0.00941
40         120    94.51      0.00973
41          47    24.21      0.00997</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                          Term Annotated Significant
1          monosaccharide biosynthetic process        63           6
2             cholesterol biosynthetic process        56          14
3                  oxidation-reduction process       603          45
4      PERK-mediated unfolded protein response        16           5
5                    pentose metabolic process         7           4
6                 response to unfolded protein       141          20
7      regulation of cellular response to heat        73          11
8     negative regulation of apoptotic process       533          45
9  L-glutamate import across plasma membran...         5           3
10     cellular response to nitrosative stress         5           3
11 chaperone cofactor-dependent protein ref...        11           4
12                     pentose-phosphate shunt        12           4
13                  neutrophil differentiation         6           3
14      telomerase holoenzyme complex assembly         6           3
15          calcium ion transport into cytosol        66           7
16                     chondrocyte development        21           5
17 regulation of toll-like receptor 2 signa...         7           3
18 negative regulation of inclusion body as...         7           3
19         peptidyl-serine autophosphorylation         7           3
20 regulation of Ras protein signal transdu...       143          12
21 chaperone-mediated protein complex assem...        14           4
22 regulation of cholesterol biosynthetic p...        39           7
23 positive regulation of transforming grow...        24           5
24                neutrophil mediated immunity       307          17
25                                  aggrephagy         8           3
26 cellular response to oxygen-glucose depr...         8           3
27                           lymphangiogenesis         8           3
28                      platelet degranulation        71           9
29 regulation of leukocyte apoptotic proces...        35           5
30                  regulation of neuron death       186          19
31 positive regulation of stress-activated ...        95          13
32 positive regulation of cell migration in...        17           4
33 central nervous system neuron axonogenes...        25           4
34 negative regulation of insulin secretion...         9           3
35 regulation of transcription from RNA pol...         9           3
36 negative regulation of protein kinase B ...        38           6
37    positive regulation of cell cycle arrest        65           6
38          positive regulation of JNK cascade        76           8
39 positive regulation of positive chemotax...        10           3
40                phosphorus metabolic process      2030         120
41 positive regulation of protein phosphory...       520          47
   Expected weightFisher
1      2.93      4.7e-06
2      2.61      5.4e-06
3     28.07      6.2e-06
4      0.74      0.00014
5      0.33      0.00014
6      6.56      0.00049
7      3.40      0.00051
8     24.81      0.00056
9      0.23      0.00093
10     0.23      0.00093
11     0.51      0.00118
12     0.56      0.00170
13     0.28      0.00180
14     0.28      0.00180
15     3.07      0.00214
16     0.98      0.00234
17     0.33      0.00305
18     0.33      0.00305
19     0.33      0.00305
20     6.66      0.00314
21     0.65      0.00319
22     1.82      0.00432
23     1.12      0.00436
24    14.29      0.00471
25     0.37      0.00471
26     0.37      0.00471
27     0.37      0.00471
28     3.31      0.00542
29     1.63      0.00624
30     8.66      0.00647
31     4.42      0.00655
32     0.79      0.00680
33     1.16      0.00681
34     0.42      0.00682
35     0.42      0.00682
36     1.77      0.00764
37     3.03      0.00844
38     3.54      0.00858
39     0.47      0.00941
40    94.51      0.00973
41    24.21      0.00997</code></pre>
</div>
</div>
<div id="perform-de-analysis-using-edger" class="section level1">
<h1>Perform DE analysis using edgeR</h1>
<pre class="r"><code>#Subset RUVsOut for only one of each of the samples (no technical replicates)
RUVsSubset &lt;- RUVsOut[, RUVsOut$Sample_ID %in% selected_samples]


design &lt;- model.matrix(~treatment + W_1 + W_2 + RIN, data=pData(RUVsSubset))
y &lt;- DGEList(counts=counts(RUVsSubset), group=pData(RUVsSubset)$Individual)
y &lt;- calcNormFactors(y, method=&quot;upperquartile&quot;)
y &lt;- estimateGLMCommonDisp(y, design)
y &lt;- estimateGLMTagwiseDisp(y, design)
fit &lt;- glmFit(y, design)
lrt &lt;- glmLRT(fit, coef=2)
gene_values &lt;- topTags(lrt, n = nrow(lrt$coefficients), adjust.method=&quot;BH&quot;, sort.by = &quot;PValue&quot;)</code></pre>
<div id="explore-top-hits-1" class="section level2">
<h2>Explore top hits</h2>
<pre class="r"><code>plot_gene &lt;- function(v, g) {
  # v - An df containing log2 counts per million
  # g - character vector of a single gene
  library(&quot;tidyr&quot;)
  single_gene &lt;- v[g, ]
  single_gene_long &lt;- as.data.frame(single_gene)
  colnames(single_gene_long) &lt;- &quot;log2cpm&quot;
  single_gene_long$sample &lt;- rownames(single_gene_long)
  single_gene_long &lt;- separate(single_gene_long, col = &quot;sample&quot;, sep = &quot;_&quot;,
                                into = c(&quot;Individual&quot;, &quot;Replicate&quot;, &quot;Treatment&quot;))
  single_gene_long$Treatment &lt;- factor(single_gene_long$Treatment, levels = c(&quot;S&quot;, &quot;U&quot;))
  single_gene_long$Treatment &lt;- revalue(single_gene_long$Treatment, c(&quot;S&quot;=&quot;Strain&quot;, &quot;U&quot;=&quot;No Strain&quot;))

  ggplot(single_gene_long, aes(x = Treatment, y = log2cpm, fill = Treatment)) +
       labs(title = g, x = &quot;Treatment&quot;, y = expression(&quot;Expression level (&quot; * log[2] * &quot; cpm)&quot;)) + 
       geom_dotplot(binaxis = &quot;y&quot;, stackdir = &quot;center&quot;) + 
       stat_summary(fun.data = &quot;mean_sdl&quot;, fun.args = list(mult=1))

}

plot_volcano &lt;- function(x, qval) {
  # Create volcano plot.
  #
  # x - data frame with topTable output
  #     (columns logFC, PValue, and FDR)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x),
            is.numeric(qval), qval &lt;= 1, qval &gt;= 0)
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(PValue), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, qval) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and qvalue)
  # qval - qvalue cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;PValue&quot;, &quot;FDR&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$FDR &lt; qval, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = PValue, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}</code></pre>
</div>
<div id="box-plots-of-top-genes" class="section level2">
<h2>Box plots of top genes</h2>
<pre class="r"><code>cpm &lt;- cpm(filt_counts, log = T)

# top genes by p-value

#rename top_treatment genes to gene symbols for easier interpretation
top_treatment_top10 &lt;- gene_anno$external_gene_name[match(head(rownames(gene_values@.Data[[1]]), n=10), gene_anno$ensembl_gene_id)]
top_treatment_top10</code></pre>
<pre><code> [1] PLAT     PAX8-AS1 CHST11   PDZRN4   ZNF667   LRRC8C   RPS4Y1  
 [8] PLN      ANGPTL4  C3orf70 
56858 Levels: A1BG A1BG-AS1 A1CF A2M A2M-AS1 A2ML1 A2ML1-AS1 ... ZZEF1</code></pre>
<pre class="r"><code>p_treatment_1 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[1])
p_treatment_2 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[2]) 
p_treatment_3 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[3]) 
p_treatment_4 &lt;- plot_gene(cpm, rownames(gene_values@.Data[[1]])[4]) 
ggsave(&quot;fig-limma-treatment-hit-1.eps&quot;, plot = p_treatment_1,
       width = 7, height = 7, units = &quot;in&quot;)</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<pre class="r"><code>plot_grid(p_treatment_1, p_treatment_2, p_treatment_3, p_treatment_4)</code></pre>
<pre><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/DE_analysis.Rmd/boxplots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-boxplots-1">
Past versions of boxplots-1.png
</button>
</p>
<div id="fig-boxplots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e6d3fad441ed6c8a115b37f517bc0b3c8a5691f1/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">e6d3fad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a3565148512e79e1482c774b845a50b8c9c06e74/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/boxplots-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plots-with-significant-genes-highlighted" class="section level2">
<h2>Plots with significant genes highlighted</h2>
<pre class="r"><code>thresh &lt;- 0.01
plot_volcano(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-plotting-functions-1">
Past versions of “use plotting functions-1.png”
</button>
</p>
<div id="fig-no-spaces-use-plotting-functions-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a3565148512e79e1482c774b845a50b8c9c06e74/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use plotting functions-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_pval_hist(x=data.frame(gene_values@.Data[[1]]), thresh)</code></pre>
<p><img src="figure/DE_analysis.Rmd/use%20plotting%20functions-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-use-plotting-functions-2">
Past versions of “use plotting functions-2.png”
</button>
</p>
<div id="fig-no-spaces-use-plotting-functions-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/figure/DE_analysis.Rmd/use plotting functions-2.png" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/DE_analysis.Rmd/use plotting functions-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>table(gene_values@.Data[[1]]$FDR &lt; thresh)</code></pre>
<pre><code>
FALSE  TRUE 
 9683   764 </code></pre>
</div>
<div id="gene-ontology-analysis-with-topgo-1" class="section level2">
<h2>Gene ontology analysis with topGO</h2>
<p>Use topGO for GO analysis. Essentially, it decreases the redundancy of the results.</p>
<p>First create the gene universe. This is all the genes tested for differential expression assigned a 1 for differentially expressed and 0 if not.</p>
<pre class="r"><code>threshold &lt;- 0.01
gene_universe &lt;- as.numeric(gene_values@.Data[[1]]$FDR &lt; threshold)
gene_universe &lt;- factor(gene_universe)
names(gene_universe) &lt;- rownames(gene_values@.Data[[1]])
head(gene_universe)</code></pre>
<pre><code>ENSG00000104368 ENSG00000189223 ENSG00000171310 ENSG00000165966 
              1               1               1               1 
ENSG00000198046 ENSG00000171488 
              1               1 
Levels: 0 1</code></pre>
<p>Create the topGO data object. Only consider “Biological Process” categories and use the Mouse Ensembl database for annotation.</p>
<pre class="r"><code>go_data &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;ensembl&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 9770 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 13850 GO terms and 32624 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 9172 genes annotated to the GO terms. )</code></pre>
<p>Use the weight01 algorithm and score the tests with Fisher’s exact test.</p>
<pre class="r"><code>go_test &lt;- runTest(go_data, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 5540 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  4 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  9 nodes to be scored    (11 eliminated genes)</code></pre>
<pre><code>
     Level 16:  18 nodes to be scored   (23 eliminated genes)</code></pre>
<pre><code>
     Level 15:  51 nodes to be scored   (64 eliminated genes)</code></pre>
<pre><code>
     Level 14:  99 nodes to be scored   (187 eliminated genes)</code></pre>
<pre><code>
     Level 13:  164 nodes to be scored  (543 eliminated genes)</code></pre>
<pre><code>
     Level 12:  235 nodes to be scored  (1323 eliminated genes)</code></pre>
<pre><code>
     Level 11:  429 nodes to be scored  (2759 eliminated genes)</code></pre>
<pre><code>
     Level 10:  615 nodes to be scored  (4088 eliminated genes)</code></pre>
<pre><code>
     Level 9:   737 nodes to be scored  (5241 eliminated genes)</code></pre>
<pre><code>
     Level 8:   781 nodes to be scored  (6452 eliminated genes)</code></pre>
<pre><code>
     Level 7:   832 nodes to be scored  (7345 eliminated genes)</code></pre>
<pre><code>
     Level 6:   726 nodes to be scored  (8113 eliminated genes)</code></pre>
<pre><code>
     Level 5:   457 nodes to be scored  (8509 eliminated genes)</code></pre>
<pre><code>
     Level 4:   245 nodes to be scored  (8774 eliminated genes)</code></pre>
<pre><code>
     Level 3:   114 nodes to be scored  (8927 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (8981 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (9041 eliminated genes)</code></pre>
<p>Keep the results with a Fisher’s exact test p-value &lt; 0.01.</p>
<pre class="r"><code>go_table &lt;- GenTable(go_data, weightFisher = go_test,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test) &lt; .01))
go_table</code></pre>
<pre><code>         GO.ID                                        Term Annotated
1   GO:0030198           extracellular matrix organization       206
2   GO:0010469 regulation of signaling receptor activit...       164
3   GO:0001837        epithelial to mesenchymal transition       100
4   GO:0007155                               cell adhesion       684
5   GO:0060028 convergent extension involved in axis el...         6
6   GO:0014068 positive regulation of phosphatidylinosi...        46
7   GO:0071294               cellular response to zinc ion         8
8   GO:0090179 planar cell polarity pathway involved in...         8
9   GO:0071300          cellular response to retinoic acid        34
10  GO:0003188                       heart valve formation        10
11  GO:0003402 planar cell polarity pathway involved in...         5
12  GO:1903589 positive regulation of blood vessel endo...         5
13  GO:0035989                          tendon development         5
14  GO:0033627          cell adhesion mediated by integrin        38
15  GO:0055008         cardiac muscle tissue morphogenesis        37
16  GO:0032755 positive regulation of interleukin-6 pro...        37
17  GO:0006935                                  chemotaxis       272
18  GO:0110111 negative regulation of animal organ morp...        19
19  GO:0060441 epithelial tube branching involved in lu...        20
20  GO:0002573           myeloid leukocyte differentiation       110
21  GO:0060449 bud elongation involved in lung branchin...         6
22  GO:0038065        collagen-activated signaling pathway         6
23  GO:0090037 positive regulation of protein kinase C ...         6
24  GO:0042487 regulation of odontogenesis of dentin-co...         6
25  GO:0055091                    phospholipid homeostasis         6
26  GO:0050927 positive regulation of positive chemotax...        10
27  GO:0003149             membranous septum morphogenesis        10
28  GO:0060601        lateral sprouting from an epithelium        10
29  GO:0050731 positive regulation of peptidyl-tyrosine...        80
30  GO:0071560 cellular response to transforming growth...       164
31  GO:0070374 positive regulation of ERK1 and ERK2 cas...        80
32  GO:0042472                     inner ear morphogenesis        39
33  GO:2000242 negative regulation of reproductive proc...        22
34  GO:0033138 positive regulation of peptidyl-serine p...        57
35  GO:0019852           L-ascorbic acid metabolic process         7
36  GO:0051965     positive regulation of synapse assembly        24
37  GO:0086091 regulation of heart rate by cardiac cond...        18
38  GO:0090184 positive regulation of kidney developmen...        25
39  GO:0060572          morphogenesis of an epithelial bud        11
40  GO:0048846    axon extension involved in axon guidance        13
41  GO:1901616 organic hydroxy compound catabolic proce...        21
42  GO:0016331       morphogenesis of embryonic epithelium       104
43  GO:0060415                 muscle tissue morphogenesis        45
44  GO:0006940     regulation of smooth muscle contraction        30
45  GO:0042462          eye photoreceptor cell development        13
46  GO:0060027 convergent extension involved in gastrul...         8
47  GO:0060397 JAK-STAT cascade involved in growth horm...         8
48  GO:0006929          substrate-dependent cell migration        21
49  GO:0001709                     cell fate determination        13
50  GO:0003009                 skeletal muscle contraction        13
51  GO:0051781        positive regulation of cell division        43
52  GO:0043043                peptide biosynthetic process       513
53  GO:0002053 positive regulation of mesenchymal cell ...        19
54  GO:0030336       negative regulation of cell migration       153
55  GO:0007565                            female pregnancy       107
56  GO:0048706       embryonic skeletal system development        73
57  GO:0061053                          somite development        47
58  GO:0048662 negative regulation of smooth muscle cel...        27
59  GO:0031668 cellular response to extracellular stimu...       178
60  GO:0045987 positive regulation of smooth muscle con...        14
61  GO:0045669 positive regulation of osteoblast differ...        41
62  GO:0003180                  aortic valve morphogenesis        20
63  GO:0060070             canonical Wnt signaling pathway       231
64  GO:0071229          cellular response to acid chemical       126
65  GO:0051004 regulation of lipoprotein lipase activit...         9
66  GO:0086013 membrane repolarization during cardiac m...         9
67  GO:0000187                 activation of MAPK activity        91
68  GO:0051496 positive regulation of stress fiber asse...        42
69  GO:0030335       positive regulation of cell migration       300
70  GO:0070373 negative regulation of ERK1 and ERK2 cas...        50
71  GO:0002063                     chondrocyte development        21
72  GO:0001666                         response to hypoxia       236
73  GO:0033631     cell-cell adhesion mediated by integrin        10
74  GO:1902961 positive regulation of aspartic-type end...         5
75  GO:0050910 detection of mechanical stimulus involve...         5
76  GO:0048703      embryonic viscerocranium morphogenesis         5
77  GO:0038030 non-canonical Wnt signaling pathway via ...         5
78  GO:0043129                      surfactant homeostasis         5
79  GO:0060685       regulation of prostatic bud formation         5
80  GO:0060687 regulation of branching involved in pros...         5
81  GO:0060677                     ureteric bud elongation         5
82  GO:0060907 positive regulation of macrophage cytoki...         5
83  GO:0061302          smooth muscle cell-matrix adhesion         5
84  GO:2000794 regulation of epithelial cell proliferat...         5
85  GO:0002551                        mast cell chemotaxis         5
86  GO:0072033                     renal vesicle formation         5
87  GO:0001658 branching involved in ureteric bud morph...        34
88  GO:0061098 positive regulation of protein tyrosine ...        29
89  GO:0045766         positive regulation of angiogenesis        97
90  GO:0060037               pharyngeal system development        16
91  GO:0006706                   steroid catabolic process        10
92  GO:0036342                post-anal tail morphogenesis        10
93  GO:0014912 negative regulation of smooth muscle cel...        16
94  GO:0021983                 pituitary gland development        16
95  GO:0007156 homophilic cell adhesion via plasma memb...        45
96  GO:0050918                         positive chemotaxis        33
97  GO:0006814                        sodium ion transport        86
98  GO:0034332              adherens junction organization       103
99  GO:0050900                         leukocyte migration       172
100 GO:0002448                 mast cell mediated immunity        21
101 GO:0014902                     myotube differentiation        50
102 GO:0060502 epithelial cell proliferation involved i...         7
103 GO:0099625 ventricular cardiac muscle cell membrane...        13
104 GO:0050891 multicellular organismal water homeostas...        28
105 GO:0006833                             water transport         7
106 GO:0045606 positive regulation of epidermal cell di...        11
107 GO:0043065    positive regulation of apoptotic process       424
108 GO:0043491                  protein kinase B signaling       139
109 GO:0010043                        response to zinc ion        19
110 GO:0035987             endodermal cell differentiation        31
111 GO:0030500           regulation of bone mineralization        42
112 GO:0045742 positive regulation of epidermal growth ...        17
113 GO:0010881 regulation of cardiac muscle contraction...        11
114 GO:0030049                     muscle filament sliding        11
115 GO:0090103                       cochlea morphogenesis        11
116 GO:0043583                             ear development       100
117 GO:0042060                               wound healing       310
118 GO:0017157                    regulation of exocytosis        90
119 GO:0070661                     leukocyte proliferation       112
120 GO:0032651 regulation of interleukin-1 beta product...        28
121 GO:0048048                 embryonic eye morphogenesis        23
122 GO:1901361 organic cyclic compound catabolic proces...       511
123 GO:0032682 negative regulation of chemokine product...        11
124 GO:0050680 negative regulation of epithelial cell p...        84
125 GO:0072189                          ureter development         6
126 GO:0072201 negative regulation of mesenchymal cell ...         6
127 GO:0099550 trans-synaptic signaling, modulating syn...         6
128 GO:0007171 activation of transmembrane receptor pro...         6
129 GO:0003344                   pericardium morphogenesis         6
130 GO:0060219 camera-type eye photoreceptor cell diffe...         6
131 GO:0045080 positive regulation of chemokine biosynt...         6
132 GO:0097084     vascular smooth muscle cell development         6
133 GO:0051901 positive regulation of mitochondrial dep...         6
134 GO:0042573             retinoic acid metabolic process         6
135 GO:0030855             epithelial cell differentiation       312
136 GO:0019439         aromatic compound catabolic process       499
137 GO:0006954                       inflammatory response       290
138 GO:0007160                        cell-matrix adhesion       134
139 GO:0001938 positive regulation of endothelial cell ...        57
140 GO:0050772         positive regulation of axonogenesis        50
141 GO:0050673               epithelial cell proliferation       229
142 GO:0010811 positive regulation of cell-substrate ad...        75
143 GO:0045926               negative regulation of growth       142
144 GO:0043627                        response to estrogen        43
145 GO:1905332 positive regulation of morphogenesis of ...        23
146 GO:0050919                         negative chemotaxis        17
147 GO:0030449         regulation of complement activation        17
148 GO:2001014 regulation of skeletal muscle cell diffe...        12
149 GO:0002052 positive regulation of neuroblast prolif...        12
150 GO:1905331 negative regulation of morphogenesis of ...        12
151 GO:0010453          regulation of cell fate commitment        12
152 GO:0032868                         response to insulin       174
153 GO:0045765                  regulation of angiogenesis       170
154 GO:0051897 positive regulation of protein kinase B ...        77
    Significant Expected weightFisher
1            48    15.05      4.8e-12
2            30    11.98      1.7e-08
3            18     7.30      3.7e-07
4           104    49.97      2.9e-06
5             5     0.44      1.2e-05
6            12     3.36      8.1e-05
7             5     0.58      9.5e-05
8             5     0.58      9.5e-05
9            10     2.48      0.00011
10            5     0.73      0.00013
11            4     0.37      0.00013
12            4     0.37      0.00013
13            4     0.37      0.00013
14           15     2.78      0.00019
15            9     2.70      0.00020
16            9     2.70      0.00020
17           47    19.87      0.00020
18            7     1.39      0.00025
19           10     1.46      0.00026
20           22     8.04      0.00037
21            4     0.44      0.00038
22            4     0.44      0.00038
23            4     0.44      0.00038
24            4     0.44      0.00038
25            4     0.44      0.00038
26            5     0.73      0.00038
27            5     0.73      0.00038
28            6     0.73      0.00038
29           21     5.84      0.00040
30           23    11.98      0.00042
31           15     5.84      0.00059
32           12     2.85      0.00063
33            7     1.61      0.00065
34           12     4.16      0.00070
35            4     0.51      0.00083
36            7     1.75      0.00124
37            6     1.31      0.00129
38            8     1.83      0.00145
39            6     0.80      0.00145
40            5     0.95      0.00146
41            5     1.53      0.00146
42           19     7.60      0.00149
43           13     3.29      0.00150
44           10     2.19      0.00152
45            5     0.95      0.00155
46            4     0.58      0.00156
47            4     0.58      0.00156
48            6     1.53      0.00160
49            5     0.95      0.00161
50            5     0.95      0.00161
51            8     3.14      0.00162
52           16    37.47      0.00164
53            6     1.39      0.00177
54           24    11.18      0.00188
55           18     7.82      0.00188
56           14     5.33      0.00230
57           11     3.43      0.00230
58            9     1.97      0.00231
59           21    13.00      0.00231
60            5     1.02      0.00235
61            9     2.99      0.00236
62            6     1.46      0.00237
63           29    16.87      0.00246
64           18     9.20      0.00256
65            4     0.66      0.00264
66            4     0.66      0.00264
67           13     6.65      0.00268
68            9     3.07      0.00281
69           48    21.91      0.00284
70           10     3.65      0.00287
71            6     1.53      0.00312
72           26    17.24      0.00314
73            4     0.73      0.00346
74            3     0.37      0.00347
75            3     0.37      0.00347
76            3     0.37      0.00347
77            3     0.37      0.00347
78            3     0.37      0.00347
79            3     0.37      0.00347
80            3     0.37      0.00347
81            3     0.37      0.00347
82            3     0.37      0.00347
83            3     0.37      0.00347
84            3     0.37      0.00347
85            3     0.37      0.00347
86            3     0.37      0.00347
87           10     2.48      0.00393
88            7     2.12      0.00406
89           20     7.09      0.00408
90            5     1.17      0.00414
91            4     0.73      0.00415
92            4     0.73      0.00415
93            5     1.17      0.00454
94            5     1.17      0.00454
95            9     3.29      0.00459
96           11     2.41      0.00496
97           14     6.28      0.00499
98           15     7.52      0.00500
99           30    12.56      0.00520
100           8     1.53      0.00526
101           9     3.65      0.00527
102           5     0.51      0.00529
103           4     0.95      0.00531
104           5     2.05      0.00531
105           3     0.51      0.00532
106           3     0.80      0.00532
107          46    30.97      0.00540
108          21    10.15      0.00578
109           9     1.39      0.00600
110           7     2.26      0.00602
111          10     3.07      0.00603
112           5     1.24      0.00605
113           4     0.80      0.00615
114           4     0.80      0.00615
115           4     0.80      0.00615
116          21     7.30      0.00626
117          44    22.65      0.00635
118          13     6.57      0.00646
119          14     8.18      0.00647
120           7     2.05      0.00650
121           6     1.68      0.00652
122          30    37.33      0.00652
123           4     0.80      0.00655
124          14     6.14      0.00656
125           3     0.44      0.00657
126           3     0.44      0.00657
127           3     0.44      0.00657
128           3     0.44      0.00657
129           3     0.44      0.00657
130           3     0.44      0.00657
131           3     0.44      0.00657
132           3     0.44      0.00657
133           3     0.44      0.00657
134           3     0.44      0.00657
135          40    22.79      0.00660
136          26    36.45      0.00663
137          40    21.18      0.00685
138          20     9.79      0.00705
139          13     4.16      0.00714
140          10     3.65      0.00775
141          37    16.73      0.00792
142          14     5.48      0.00834
143          20    10.37      0.00840
144          10     3.14      0.00846
145           7     1.68      0.00860
146           5     1.24      0.00867
147           5     1.24      0.00867
148           4     0.88      0.00870
149           4     0.88      0.00870
150           4     0.88      0.00870
151           4     0.88      0.00870
152          11    12.71      0.00906
153          30    12.42      0.00921
154          12     5.62      0.00952</code></pre>
<pre class="r"><code>go_table[,-1]</code></pre>
<pre><code>                                           Term Annotated Significant
1             extracellular matrix organization       206          48
2   regulation of signaling receptor activit...       164          30
3          epithelial to mesenchymal transition       100          18
4                                 cell adhesion       684         104
5   convergent extension involved in axis el...         6           5
6   positive regulation of phosphatidylinosi...        46          12
7                 cellular response to zinc ion         8           5
8   planar cell polarity pathway involved in...         8           5
9            cellular response to retinoic acid        34          10
10                        heart valve formation        10           5
11  planar cell polarity pathway involved in...         5           4
12  positive regulation of blood vessel endo...         5           4
13                           tendon development         5           4
14           cell adhesion mediated by integrin        38          15
15          cardiac muscle tissue morphogenesis        37           9
16  positive regulation of interleukin-6 pro...        37           9
17                                   chemotaxis       272          47
18  negative regulation of animal organ morp...        19           7
19  epithelial tube branching involved in lu...        20          10
20            myeloid leukocyte differentiation       110          22
21  bud elongation involved in lung branchin...         6           4
22         collagen-activated signaling pathway         6           4
23  positive regulation of protein kinase C ...         6           4
24  regulation of odontogenesis of dentin-co...         6           4
25                     phospholipid homeostasis         6           4
26  positive regulation of positive chemotax...        10           5
27              membranous septum morphogenesis        10           5
28         lateral sprouting from an epithelium        10           6
29  positive regulation of peptidyl-tyrosine...        80          21
30  cellular response to transforming growth...       164          23
31  positive regulation of ERK1 and ERK2 cas...        80          15
32                      inner ear morphogenesis        39          12
33  negative regulation of reproductive proc...        22           7
34  positive regulation of peptidyl-serine p...        57          12
35            L-ascorbic acid metabolic process         7           4
36      positive regulation of synapse assembly        24           7
37  regulation of heart rate by cardiac cond...        18           6
38  positive regulation of kidney developmen...        25           8
39           morphogenesis of an epithelial bud        11           6
40     axon extension involved in axon guidance        13           5
41  organic hydroxy compound catabolic proce...        21           5
42        morphogenesis of embryonic epithelium       104          19
43                  muscle tissue morphogenesis        45          13
44      regulation of smooth muscle contraction        30          10
45           eye photoreceptor cell development        13           5
46  convergent extension involved in gastrul...         8           4
47  JAK-STAT cascade involved in growth horm...         8           4
48           substrate-dependent cell migration        21           6
49                      cell fate determination        13           5
50                  skeletal muscle contraction        13           5
51         positive regulation of cell division        43           8
52                 peptide biosynthetic process       513          16
53  positive regulation of mesenchymal cell ...        19           6
54        negative regulation of cell migration       153          24
55                             female pregnancy       107          18
56        embryonic skeletal system development        73          14
57                           somite development        47          11
58  negative regulation of smooth muscle cel...        27           9
59  cellular response to extracellular stimu...       178          21
60  positive regulation of smooth muscle con...        14           5
61  positive regulation of osteoblast differ...        41           9
62                   aortic valve morphogenesis        20           6
63              canonical Wnt signaling pathway       231          29
64           cellular response to acid chemical       126          18
65  regulation of lipoprotein lipase activit...         9           4
66  membrane repolarization during cardiac m...         9           4
67                  activation of MAPK activity        91          13
68  positive regulation of stress fiber asse...        42           9
69        positive regulation of cell migration       300          48
70  negative regulation of ERK1 and ERK2 cas...        50          10
71                      chondrocyte development        21           6
72                          response to hypoxia       236          26
73      cell-cell adhesion mediated by integrin        10           4
74  positive regulation of aspartic-type end...         5           3
75  detection of mechanical stimulus involve...         5           3
76       embryonic viscerocranium morphogenesis         5           3
77  non-canonical Wnt signaling pathway via ...         5           3
78                       surfactant homeostasis         5           3
79        regulation of prostatic bud formation         5           3
80  regulation of branching involved in pros...         5           3
81                      ureteric bud elongation         5           3
82  positive regulation of macrophage cytoki...         5           3
83           smooth muscle cell-matrix adhesion         5           3
84  regulation of epithelial cell proliferat...         5           3
85                         mast cell chemotaxis         5           3
86                      renal vesicle formation         5           3
87  branching involved in ureteric bud morph...        34          10
88  positive regulation of protein tyrosine ...        29           7
89          positive regulation of angiogenesis        97          20
90                pharyngeal system development        16           5
91                    steroid catabolic process        10           4
92                 post-anal tail morphogenesis        10           4
93  negative regulation of smooth muscle cel...        16           5
94                  pituitary gland development        16           5
95  homophilic cell adhesion via plasma memb...        45           9
96                          positive chemotaxis        33          11
97                         sodium ion transport        86          14
98               adherens junction organization       103          15
99                          leukocyte migration       172          30
100                 mast cell mediated immunity        21           8
101                     myotube differentiation        50           9
102 epithelial cell proliferation involved i...         7           5
103 ventricular cardiac muscle cell membrane...        13           4
104 multicellular organismal water homeostas...        28           5
105                             water transport         7           3
106 positive regulation of epidermal cell di...        11           3
107    positive regulation of apoptotic process       424          46
108                  protein kinase B signaling       139          21
109                        response to zinc ion        19           9
110             endodermal cell differentiation        31           7
111           regulation of bone mineralization        42          10
112 positive regulation of epidermal growth ...        17           5
113 regulation of cardiac muscle contraction...        11           4
114                     muscle filament sliding        11           4
115                       cochlea morphogenesis        11           4
116                             ear development       100          21
117                               wound healing       310          44
118                    regulation of exocytosis        90          13
119                     leukocyte proliferation       112          14
120 regulation of interleukin-1 beta product...        28           7
121                 embryonic eye morphogenesis        23           6
122 organic cyclic compound catabolic proces...       511          30
123 negative regulation of chemokine product...        11           4
124 negative regulation of epithelial cell p...        84          14
125                          ureter development         6           3
126 negative regulation of mesenchymal cell ...         6           3
127 trans-synaptic signaling, modulating syn...         6           3
128 activation of transmembrane receptor pro...         6           3
129                   pericardium morphogenesis         6           3
130 camera-type eye photoreceptor cell diffe...         6           3
131 positive regulation of chemokine biosynt...         6           3
132     vascular smooth muscle cell development         6           3
133 positive regulation of mitochondrial dep...         6           3
134             retinoic acid metabolic process         6           3
135             epithelial cell differentiation       312          40
136         aromatic compound catabolic process       499          26
137                       inflammatory response       290          40
138                        cell-matrix adhesion       134          20
139 positive regulation of endothelial cell ...        57          13
140         positive regulation of axonogenesis        50          10
141               epithelial cell proliferation       229          37
142 positive regulation of cell-substrate ad...        75          14
143               negative regulation of growth       142          20
144                        response to estrogen        43          10
145 positive regulation of morphogenesis of ...        23           7
146                         negative chemotaxis        17           5
147         regulation of complement activation        17           5
148 regulation of skeletal muscle cell diffe...        12           4
149 positive regulation of neuroblast prolif...        12           4
150 negative regulation of morphogenesis of ...        12           4
151          regulation of cell fate commitment        12           4
152                         response to insulin       174          11
153                  regulation of angiogenesis       170          30
154 positive regulation of protein kinase B ...        77          12
    Expected weightFisher
1      15.05      4.8e-12
2      11.98      1.7e-08
3       7.30      3.7e-07
4      49.97      2.9e-06
5       0.44      1.2e-05
6       3.36      8.1e-05
7       0.58      9.5e-05
8       0.58      9.5e-05
9       2.48      0.00011
10      0.73      0.00013
11      0.37      0.00013
12      0.37      0.00013
13      0.37      0.00013
14      2.78      0.00019
15      2.70      0.00020
16      2.70      0.00020
17     19.87      0.00020
18      1.39      0.00025
19      1.46      0.00026
20      8.04      0.00037
21      0.44      0.00038
22      0.44      0.00038
23      0.44      0.00038
24      0.44      0.00038
25      0.44      0.00038
26      0.73      0.00038
27      0.73      0.00038
28      0.73      0.00038
29      5.84      0.00040
30     11.98      0.00042
31      5.84      0.00059
32      2.85      0.00063
33      1.61      0.00065
34      4.16      0.00070
35      0.51      0.00083
36      1.75      0.00124
37      1.31      0.00129
38      1.83      0.00145
39      0.80      0.00145
40      0.95      0.00146
41      1.53      0.00146
42      7.60      0.00149
43      3.29      0.00150
44      2.19      0.00152
45      0.95      0.00155
46      0.58      0.00156
47      0.58      0.00156
48      1.53      0.00160
49      0.95      0.00161
50      0.95      0.00161
51      3.14      0.00162
52     37.47      0.00164
53      1.39      0.00177
54     11.18      0.00188
55      7.82      0.00188
56      5.33      0.00230
57      3.43      0.00230
58      1.97      0.00231
59     13.00      0.00231
60      1.02      0.00235
61      2.99      0.00236
62      1.46      0.00237
63     16.87      0.00246
64      9.20      0.00256
65      0.66      0.00264
66      0.66      0.00264
67      6.65      0.00268
68      3.07      0.00281
69     21.91      0.00284
70      3.65      0.00287
71      1.53      0.00312
72     17.24      0.00314
73      0.73      0.00346
74      0.37      0.00347
75      0.37      0.00347
76      0.37      0.00347
77      0.37      0.00347
78      0.37      0.00347
79      0.37      0.00347
80      0.37      0.00347
81      0.37      0.00347
82      0.37      0.00347
83      0.37      0.00347
84      0.37      0.00347
85      0.37      0.00347
86      0.37      0.00347
87      2.48      0.00393
88      2.12      0.00406
89      7.09      0.00408
90      1.17      0.00414
91      0.73      0.00415
92      0.73      0.00415
93      1.17      0.00454
94      1.17      0.00454
95      3.29      0.00459
96      2.41      0.00496
97      6.28      0.00499
98      7.52      0.00500
99     12.56      0.00520
100     1.53      0.00526
101     3.65      0.00527
102     0.51      0.00529
103     0.95      0.00531
104     2.05      0.00531
105     0.51      0.00532
106     0.80      0.00532
107    30.97      0.00540
108    10.15      0.00578
109     1.39      0.00600
110     2.26      0.00602
111     3.07      0.00603
112     1.24      0.00605
113     0.80      0.00615
114     0.80      0.00615
115     0.80      0.00615
116     7.30      0.00626
117    22.65      0.00635
118     6.57      0.00646
119     8.18      0.00647
120     2.05      0.00650
121     1.68      0.00652
122    37.33      0.00652
123     0.80      0.00655
124     6.14      0.00656
125     0.44      0.00657
126     0.44      0.00657
127     0.44      0.00657
128     0.44      0.00657
129     0.44      0.00657
130     0.44      0.00657
131     0.44      0.00657
132     0.44      0.00657
133     0.44      0.00657
134     0.44      0.00657
135    22.79      0.00660
136    36.45      0.00663
137    21.18      0.00685
138     9.79      0.00705
139     4.16      0.00714
140     3.65      0.00775
141    16.73      0.00792
142     5.48      0.00834
143    10.37      0.00840
144     3.14      0.00846
145     1.68      0.00860
146     1.24      0.00867
147     1.24      0.00867
148     0.88      0.00870
149     0.88      0.00870
150     0.88      0.00870
151     0.88      0.00870
152    12.71      0.00906
153    12.42      0.00921
154     5.62      0.00952</code></pre>
<p>There are 154 significant results.</p>
<!-- # Looking at treatment x individual interactions -->
<!-- ```{r design-matrix_interaction} -->
<!-- design2 <- model.matrix(~treatment + Individual + treatment:Individual + W_1 + W_2 + RIN, -->
<!--                        data = anno) -->
<!-- colnames(design2) -->
<!-- ``` -->
<!-- ```{r fit-model, fig.width=8} -->
<!-- #TMM Normalization -->
<!-- y <- DGEList(single_filt_counts) -->
<!-- y <- calcNormFactors(y, method = "TMM") -->
<!-- anno <- pData(RUVsOut) -->
<!-- x <- paste0(anno$Individual, anno$treatment) -->
<!-- #Voom for differential expression -->
<!-- v1_interaction <- voom(y, design2) -->
<!-- corfit1_interaction <- duplicateCorrelation(v1_interaction, design2, block = x) -->
<!-- corfit1_interaction$consensus.correlation -->
<!-- v2_interaction <- voom(y, design2, block = x, correlation = corfit1_interaction$consensus.correlation) -->
<!-- corfit2_interaction <- duplicateCorrelation(v2_interaction, design2, block = x) -->
<!-- corfit2_interaction$consensus.correlation -->
<!-- fit_interaction <- lmFit(v2_interaction, design2, block = x, -->
<!--              correlation = corfit2_interaction$consensus.correlation) -->
<!-- fit_interaction <- eBayes(fit_interaction) -->
<!-- ``` -->
<!-- ## Assess model results -->
<!-- ```{r model-results-interaction} -->
<!-- tests_interaction <- colnames(fit_interaction$coefficients) -->
<!-- results_interaction <- vector(length = length(tests_interaction), mode = "list") -->
<!-- names(results_interaction) <- tests_interaction -->
<!-- for (test in tests_interaction) { -->
<!--   # Extract limma results -->
<!--   results_interaction[[test]] <- get_results(fit_interaction, coef = test) -->
<!--   # Add mutliple testing correction with ASH -->
<!--   output_ash <- run_ash(fit_interaction, coef = test)$result -->
<!--   results_interaction[[test]] <- cbind(results_interaction[[test]], lfsr = output_ash$lfsr, -->
<!--                            lfdr = output_ash$lfdr, qvalue = output_ash$qvalue, -->
<!--                            svalue = output_ash$svalue) -->
<!-- } -->
<!-- #FDR 0.05 -->
<!-- #interaction term 1 -->
<!-- plot_ma(results_interaction[["treatmentUnstrain:IndividualNA18856 "]], 0.05) -->
<!-- plot_volcano(results_interaction[["treatmentUnstrain:IndividualNA18856 "]], 0.05) -->
<!-- plot_pval_hist(results_interaction[["treatmentUnstrain:IndividualNA18856 "]], 0.05) -->
<!-- table(results_interaction[["treatmentUnstrain:IndividualNA18856 "]]$qvalue < 0.05) -->
<!-- significant_genes_05 <- row.names(results_interaction[["treatmentUnstrain:IndividualNA18856 "]])[results_interaction[["treatmentUnstrain:IndividualNA18856 "]]$qvalue < 0.05] -->
<!-- significant_symbols_05 <- gene_anno$external_gene_name[match(significant_genes_05, gene_anno$ensembl_gene_id)] -->
<!-- head(significant_symbols_05) -->
<!-- significant_anno_05 <- gene_anno[match(significant_genes_05, gene_anno$ensembl_gene_id),] -->
<!-- #interaction term 2 -->
<!-- plot_ma(results_interaction[["treatmentUnstrain:IndividualNA19160 "]], 0.05) -->
<!-- plot_volcano(results_interaction[["treatmentUnstrain:IndividualNA19160 "]], 0.05) -->
<!-- plot_pval_hist(results_interaction[["treatmentUnstrain:IndividualNA19160 "]], 0.05) -->
<!-- table(results_interaction[["treatmentUnstrain:IndividualNA19160 "]]$qvalue < 0.05) -->
<!-- significant_genes_05 <- row.names(results_interaction[["treatmentUnstrain:IndividualNA19160 "]])[results_interaction[["treatmentUnstrain:IndividualNA19160 "]]$qvalue < 0.05] -->
<!-- significant_symbols_05 <- gene_anno$external_gene_name[match(significant_genes_05, gene_anno$ensembl_gene_id)] -->
<!-- head(significant_symbols_05) -->
<!-- significant_anno_05 <- gene_anno[match(significant_genes_05, gene_anno$ensembl_gene_id),] -->
<!-- ``` -->
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] stats4    parallel  grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] org.Hs.eg.db_3.8.2          topGO_2.36.0               
 [3] SparseM_1.77                GO.db_3.8.2                
 [5] AnnotationDbi_1.46.0        graph_1.62.0               
 [7] RUVSeq_1.18.0               EDASeq_2.18.0              
 [9] ShortRead_1.42.0            GenomicAlignments_1.20.1   
[11] SummarizedExperiment_1.14.1 DelayedArray_0.10.0        
[13] matrixStats_0.55.0          Rsamtools_2.0.0            
[15] GenomicRanges_1.36.1        GenomeInfoDb_1.20.0        
[17] Biostrings_2.52.0           XVector_0.24.0             
[19] IRanges_2.18.3              S4Vectors_0.22.1           
[21] BiocParallel_1.18.1         Biobase_2.44.0             
[23] BiocGenerics_0.30.0         VennDiagram_1.6.20         
[25] futile.logger_1.4.3         cowplot_1.0.0              
[27] ggplot2_3.3.0               ashr_2.2-47                
[29] tidyr_1.0.3                 dplyr_0.8.5                
[31] edgeR_3.26.8                plyr_1.8.6                 
[33] limma_3.40.6               

loaded via a namespace (and not attached):
  [1] colorspace_1.4-1       hwriter_1.3.2          ellipsis_0.3.1        
  [4] rprojroot_1.3-2        htmlTable_1.13.1       base64enc_0.1-3       
  [7] fs_1.3.1               rstudioapi_0.11        farver_2.0.3          
 [10] bit64_0.9-7            splines_3.6.1          R.methodsS3_1.7.1     
 [13] DESeq_1.36.0           geneplotter_1.62.0     knitr_1.23            
 [16] Formula_1.2-3          workflowr_1.6.2        annotate_1.62.0       
 [19] cluster_2.1.0          R.oo_1.23.0            compiler_3.6.1        
 [22] httr_1.4.1             backports_1.1.7        assertthat_0.2.1      
 [25] Matrix_1.2-18          later_1.0.0            formatR_1.7           
 [28] acepack_1.4.1          htmltools_0.4.0        prettyunits_1.1.1     
 [31] tools_3.6.1            gtable_0.3.0           glue_1.4.1            
 [34] GenomeInfoDbData_1.2.1 Rcpp_1.0.4.6           vctrs_0.3.0           
 [37] rtracklayer_1.44.0     xfun_0.8               stringr_1.4.0         
 [40] lifecycle_0.2.0        irlba_2.3.3            statmod_1.4.32        
 [43] XML_3.98-1.20          zlibbioc_1.30.0        MASS_7.3-51.6         
 [46] scales_1.1.1           aroma.light_3.14.0     hms_0.5.3             
 [49] promises_1.1.0         lambda.r_1.2.4         RColorBrewer_1.1-2    
 [52] yaml_2.2.1             gridExtra_2.3          memoise_1.1.0         
 [55] rpart_4.1-15           biomaRt_2.40.1         latticeExtra_0.6-28   
 [58] stringi_1.4.6          RSQLite_2.1.1          SQUAREM_2020.2        
 [61] genefilter_1.66.0      checkmate_1.9.4        GenomicFeatures_1.36.3
 [64] truncnorm_1.0-8        rlang_0.4.6            pkgconfig_2.0.3       
 [67] bitops_1.0-6           evaluate_0.14          lattice_0.20-41       
 [70] invgamma_1.1           purrr_0.3.4            htmlwidgets_1.5.1     
 [73] labeling_0.3           bit_1.1-14             tidyselect_1.1.0      
 [76] magrittr_1.5           R6_2.4.1               Hmisc_4.2-0           
 [79] DBI_1.0.0              foreign_0.8-71         pillar_1.4.4          
 [82] whisker_0.3-2          withr_2.2.0            nnet_7.3-12           
 [85] survival_2.44-1.1      RCurl_1.98-1.1         mixsqp_0.3-43         
 [88] tibble_3.0.1           crayon_1.3.4           futile.options_1.0.1  
 [91] rmarkdown_1.13         progress_1.2.2         locfit_1.5-9.1        
 [94] data.table_1.12.8      blob_1.2.0             git2r_0.26.1          
 [97] etrunct_0.1            digest_0.6.25          xtable_1.8-4          
[100] httpuv_1.5.1           R.utils_2.9.2          munsell_0.5.0         </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
