<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-05-18" />

<title>Enrichment of genes of interest amongst DE genes</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment of genes of interest amongst DE genes</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-05-18</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-11-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreec2ccaf4d1c46d07c64bf037235cef6718583f2aatargetblankc2ccaf4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/c2ccaf4d1c46d07c64bf037235cef6718583f2aa" target="_blank">c2ccaf4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreec2ccaf4d1c46d07c64bf037235cef6718583f2aatargetblankc2ccaf4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/c2ccaf4d1c46d07c64bf037235cef6718583f2aa" target="_blank">c2ccaf4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/SCT_integrated_by_individual.rds
    Ignored:    data/Sample.info.full.RNAseq.reordered.csv
    Ignored:    data/Tachmazidou_30664745_ALLOA.txt.gz
    Ignored:    data/alasoo_etal/
    Ignored:    data/caliskan_etal/
    Ignored:    data/external_scRNA/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/filtered_counts_full.rds
    Ignored:    data/gene-annotation_hg37.txt
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/norm_filtered_counts_full.rds
    Ignored:    data/raw_counts_full_relabeled.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds
    Ignored:    output/GO_terms_enriched.csv
    Ignored:    output/LDSC-SEG/
    Ignored:    output/SNPs.csv
    Ignored:    output/ash_power_analysis/
    Ignored:    output/classtpx/
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/fasttopics/
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash/
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/pairwise_ttest_results_SCTdata.rds
    Ignored:    output/pairwise_ttest_results_integrateddata.rds
    Ignored:    output/permuted_results_DE.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Unstaged changes:
    Modified:   analysis/Norm_filter.Rmd
    Modified:   data/DEgenes_FDR05.rda
    Modified:   fig-limma-treatment-hit-1.eps
    Modified:   output/GWAS_descriptions.csv

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/enrichment_DE.Rmd</code>) and HTML (<code>docs/enrichment_DE.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/c2ccaf4d1c46d07c64bf037235cef6718583f2aa/docs/enrichment_DE.html" target="_blank">c2ccaf4</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-11-29
</td>
<td>
run analyses without excluding sample 2
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6d6d1b59562440476e915e9c670365f12f31b7ba/analysis/enrichment_DE.Rmd" target="_blank">6d6d1b5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
<td>
knit permutation analyses of strain labels
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/6d6d1b59562440476e915e9c670365f12f31b7ba/docs/enrichment_DE.html" target="_blank">6d6d1b5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
<td>
knit permutation analyses of strain labels
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/analysis/enrichment_DE.Rmd" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
<td>
knit permutation analyses of strain labels
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/09ab774d153d2b21b6cc4c63627ed0b448acda1f/docs/enrichment_DE.html" target="_blank">09ab774</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-27
</td>
<td>
add html files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e855314ea98fd1fef1af178e7dc9ca1541f8deec/docs/enrichment_DE.html" target="_blank">e855314</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-27
</td>
<td>
delete html files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e47502762d7d1672f4ecdfb0ec1c7b9a2a39e7b0/analysis/enrichment_DE.Rmd" target="_blank">e475027</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-06-01
</td>
<td>
add interaction plots
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2cb3b3fb0a677b2709df71fa7da9d0e7969409fb/analysis/enrichment_DE.Rmd" target="_blank">2cb3b3f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-06-01
</td>
<td>
add interaction term genes and plotting
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/78b9cc0d4cf785e7da03f2db950df97b103f8104/analysis/enrichment_DE.Rmd" target="_blank">78b9cc0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-31
</td>
<td>
add dotplots of 50 overlap genes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/dd6a48bfb866db0663b341f0c7a8c300ec83780f/analysis/enrichment_DE.Rmd" target="_blank">dd6a48b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-28
</td>
<td>
subset GWAS descriptions
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/dd6a48bfb866db0663b341f0c7a8c300ec83780f/docs/enrichment_DE.html" target="_blank">dd6a48b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-28
</td>
<td>
subset GWAS descriptions
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/022f905186e6ab8f784b7266671f2729d9200c6c/analysis/enrichment_DE.Rmd" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
<td>
change RUVg to use least variable genes found through raw unnormalized data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/022f905186e6ab8f784b7266671f2729d9200c6c/docs/enrichment_DE.html" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
<td>
change RUVg to use least variable genes found through raw unnormalized data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6cfbae9c91c4f6cb94fb8432695184bb3cfb502e/analysis/enrichment_DE.Rmd" target="_blank">6cfbae9</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-18
</td>
<td>
Add enrichment analysis file
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/6cfbae9c91c4f6cb94fb8432695184bb3cfb502e/docs/enrichment_DE.html" target="_blank">6cfbae9</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-18
</td>
<td>
Add enrichment analysis file
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-packagesdata" class="section level1">
<h1>Load packages/data</h1>
<pre class="r"><code>library(dplyr)</code></pre>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code># load gene annotations
gene_anno &lt;- read.delim(&quot;data/gene-annotation.txt&quot;,
                        sep = &quot;\t&quot;)

DE_results &lt;- readRDS(&quot;data/DE_results.rds&quot;)
threshold &lt;- 0.05
DE_genes_ensg &lt;- rownames(DE_results[[&quot;Unstrain&quot;]])[DE_results[[&quot;Unstrain&quot;]]$qvalue &lt; threshold]
DE_genes &lt;- gene_anno$external_gene_name[match(DE_genes_ensg, gene_anno$ensembl_gene_id)]
other_genes_ensg &lt;- rownames(DE_results[[&quot;Unstrain&quot;]])[DE_results[[&quot;Unstrain&quot;]]$qvalue &gt;= threshold]
other_genes &lt;- gene_anno$external_gene_name[match(other_genes_ensg, gene_anno$ensembl_gene_id)]

interaction_genes &lt;- readRDS(&quot;output/interaction_genes.rds&quot;)



#Genes from https://www.nature.com/articles/s41588-018-0327-1#Sec3 (supp table 6)
#Genes for which OAGWAS signals share an association eQTL signal in at least one tissue (coloc/metaxcan).
genes_GWAS_share_eQTL &lt;- c(&quot;SLC39A8&quot;, &quot;TGFA&quot;, &quot;CCDC97&quot;, &quot;KIF12&quot;, &quot;COL27A1&quot;, &quot;AC011475.1&quot;, &quot;LTBP1&quot;, &quot;CSK&quot;, &quot;RP11-10A14.8&quot;, &quot;RBM6&quot;, &quot;DPEP1&quot;,
                           &quot;DYNC1I1&quot;, &quot;RP11-21L23.2&quot;, &quot;CRADD&quot;, &quot;RAPH1&quot;,  &quot;ANAPC4&quot;, &quot;HIST1H2AC&quot;, &quot;RABGAP1L&quot;, &quot;GLIS3&quot;, &quot;RP5-1024C24.1&quot;, &quot;DCDC1&quot;, &quot;MAPT&quot;,
                           &quot;CPSF6&quot;, &quot;KIF26B&quot;, &quot;SCUBE1&quot;, &quot;TMEM241&quot;, &quot;RNU2-17P&quot;, &quot;CDC5L-RP3-449H6.1&quot;, &quot;RN7SKP15-RP11-993B23.1&quot;, &quot;RP11-415D17.4&quot;, &quot;COL11A1&quot;, &quot;ITIH1&quot;,
                           &quot;ASTN2&quot;, &quot;IL11&quot;, &quot;RUNX2&quot;, &quot;LMX1B&quot;, &quot;MAPT&quot;, &quot;RP11-22A3.1-RNU6-962P&quot;, &quot;ERG-LINC00114&quot;, &quot;SMAD3&quot;, &quot;TBX4&quot;, &quot;NACA2&quot;, &quot;TNC&quot;,
                           &quot;MLXIP&quot;, &quot;COLGALT2&quot;, &quot;LRIG3&quot;, &quot;BMP5&quot;, &quot;LTBP3&quot;, &quot;GSDMC&quot;, &quot;SPTBN2&quot;, &quot;GDF5&quot;, &quot;NFAT5&quot;, &quot;SBNO1&quot;, 
                           &quot;POLR2M-ALDH1A2&quot;, &quot;SMG6&quot;, &quot;LOC124685-AC005152.3&quot;, &quot;AP3B1&quot;, &quot;AC098872.3&quot;, &quot;USP8&quot;, &quot;RP11-95P13.2-ZC3H11B&quot;, &quot;RP11-115J23.1&quot;, &quot;HLA-DPB1&quot;, &quot;NF1&quot;,
                           &quot;SLBP&quot;, &quot;FTO&quot;, &quot;FGF18&quot;, &quot;RP11-654D12.2&quot;
                           )

#nearest gene symbol(s) to established osteoarthritis loci (Fig S3)
OA_GWAS_genes &lt;- c(&quot;SLC30A10&quot;, &quot;LYPLAL1&quot;, &quot;TGFA&quot;, &quot;COL6A4P1&quot;, &quot;GLT8D1&quot;, &quot;TP63&quot;, &quot;ANXA3&quot;, &quot;BTNL2&quot;, &quot;HLA-DQB1&quot;, &quot;SUPT3H&quot;,
                   &quot;FILIP1&quot;, &quot;SENP6&quot;, &quot;COG5&quot;, &quot;DUS4L&quot;, &quot;LINC01006&quot;, &quot;PLEC&quot;, &quot;GLIS3&quot;, &quot;MOB3B&quot;, &quot;EQTN&quot;, &quot;ASTN2&quot;,
                   &quot;MGP&quot;, &quot;ERP27&quot;, &quot;KLHL42&quot;, &quot;PTHLH&quot;, &quot;CHST11&quot;, &quot;MCF2L&quot;, &quot;ALDH1A2&quot;, &quot;ALDH1A2&quot;, &quot;SMAD3&quot;, &quot;FTO&quot;,
                   &quot;JPH3&quot;, &quot;MAP2K6&quot;, &quot;DOT1L&quot;, &quot;COMP&quot;, &quot;ZNF345&quot;, &quot;GDF5&quot;, &quot;ZMYND8&quot;, &quot;NCOA3&quot;, &quot;RWDD2B&quot;, &quot;USP16&quot;, 
                   &quot;LTN1&quot;, &quot;CHADL&quot;
)

#Table S4: Functional genomics annotations for all genes within 500kb of the osteoarthritis-associated variants. (subset to those with any evidence)
fungenomic &lt;- c(&quot;ENSG00000060718&quot;, &quot;ENSG00000270962&quot;, &quot;ENSG00000150337&quot;, &quot;ENSG00000264522&quot;, &quot;ENSG00000136631&quot;, &quot;ENSG00000023902&quot;, &quot;ENSG00000117362&quot;, &quot;ENSG00000159208&quot;, &quot;ENSG00000163125&quot;, &quot;ENSG00000143374&quot;, &quot;ENSG00000143369&quot;, &quot;ENSG00000143420&quot;, &quot;ENSG00000163131&quot;, &quot;ENSG00000143387&quot;, &quot;ENSG00000143412&quot;, &quot;ENSG00000143409&quot;, &quot;ENSG00000213190&quot;, &quot;ENSG00000120334&quot;, &quot;ENSG00000117593&quot;, &quot;ENSG00000135870&quot;, &quot;ENSG00000227373&quot;, &quot;ENSG00000152061&quot;, &quot;ENSG00000135862&quot;, &quot;ENSG00000058085&quot;, &quot;ENSG00000116701&quot;, &quot;ENSG00000198756&quot;, &quot;ENSG00000198860&quot;, &quot;ENSG00000143353&quot;, &quot;ENSG00000228536&quot;, &quot;ENSG00000116141&quot;, &quot;ENSG00000203667&quot;, &quot;ENSG00000185420&quot;, &quot;ENSG00000018699&quot;, &quot;ENSG00000049323&quot;, &quot;ENSG00000119812&quot;, &quot;ENSG00000059728&quot;, &quot;ENSG00000244617&quot;, &quot;ENSG00000116001&quot;, &quot;ENSG00000116005&quot;, &quot;ENSG00000143977&quot;, &quot;ENSG00000035141&quot;, &quot;ENSG00000163235&quot;, &quot;ENSG00000116039&quot;, &quot;ENSG00000124357&quot;, &quot;ENSG00000135636&quot;, &quot;ENSG00000115419&quot;, &quot;ENSG00000115415&quot;, &quot;ENSG00000138378&quot;, &quot;ENSG00000204217&quot;, &quot;ENSG00000138439&quot;, &quot;ENSG00000163596&quot;, &quot;ENSG00000138380&quot;, &quot;ENSG00000138442&quot;, &quot;ENSG00000173166&quot;, &quot;ENSG00000163599&quot;, &quot;ENSG00000178467&quot;, &quot;ENSG00000178252&quot;, &quot;ENSG00000178035&quot;, &quot;ENSG00000172037&quot;, &quot;ENSG00000145020&quot;, &quot;ENSG00000145029&quot;, &quot;ENSG00000173402&quot;, &quot;ENSG00000164068&quot;, &quot;ENSG00000173540&quot;, &quot;ENSG00000176095&quot;, &quot;ENSG00000185614&quot;, &quot;ENSG00000182179&quot;, &quot;ENSG00000164076&quot;, &quot;ENSG00000234667&quot;, &quot;ENSG00000164078&quot;, &quot;ENSG00000228008&quot;, &quot;ENSG00000004534&quot;, &quot;ENSG00000003756&quot;, &quot;ENSG00000001617&quot;, &quot;ENSG00000114353&quot;, &quot;ENSG00000214706&quot;, &quot;ENSG00000114378&quot;, &quot;ENSG00000068001&quot;, &quot;ENSG00000126062&quot;, &quot;ENSG00000007402&quot;, &quot;ENSG00000114737&quot;, &quot;ENSG00000114767&quot;, &quot;ENSG00000090097&quot;, &quot;ENSG00000248487&quot;, &quot;ENSG00000114779&quot;, &quot;ENSG00000114786&quot;, &quot;ENSG00000162244&quot;, &quot;ENSG00000164087&quot;, &quot;ENSG00000247596&quot;, &quot;ENSG00000164088&quot;, &quot;ENSG00000164091&quot;, &quot;ENSG00000168237&quot;, &quot;ENSG00000239557&quot;, &quot;ENSG00000010318&quot;, &quot;ENSG00000010322&quot;, &quot;ENSG00000168268&quot;, &quot;ENSG00000163939&quot;, &quot;ENSG00000163938&quot;, &quot;ENSG00000016864&quot;, &quot;ENSG00000114902&quot;, &quot;ENSG00000270941&quot;, &quot;ENSG00000114904&quot;, &quot;ENSG00000055957&quot;, &quot;ENSG00000162267&quot;, &quot;ENSG00000055955&quot;, &quot;ENSG00000272573&quot;, &quot;ENSG00000163935&quot;, &quot;ENSG00000163932&quot;, &quot;ENSG00000163931&quot;, &quot;ENSG00000127415&quot;, &quot;ENSG00000127418&quot;, &quot;ENSG00000178222&quot;, &quot;ENSG00000159692&quot;, &quot;ENSG00000163945&quot;, &quot;ENSG00000244459&quot;, &quot;ENSG00000174137&quot;, &quot;ENSG00000163950&quot;, &quot;ENSG00000270195&quot;, &quot;ENSG00000168936&quot;, &quot;ENSG00000013810&quot;, &quot;ENSG00000068078&quot;, &quot;ENSG00000168924&quot;, &quot;ENSG00000109685&quot;, &quot;ENSG00000130997&quot;, &quot;ENSG00000159733&quot;, &quot;ENSG00000157869&quot;, &quot;ENSG00000109705&quot;, &quot;ENSG00000109606&quot;, &quot;ENSG00000248444&quot;, &quot;ENSG00000109610&quot;, &quot;ENSG00000181982&quot;, &quot;ENSG00000168228&quot;, &quot;ENSG00000053900&quot;, &quot;ENSG00000091490&quot;, &quot;ENSG00000138814&quot;, &quot;ENSG00000138821&quot;, &quot;ENSG00000109320&quot;, &quot;ENSG00000109332&quot;, &quot;ENSG00000145354&quot;, &quot;ENSG00000164039&quot;, &quot;ENSG00000253558&quot;, &quot;ENSG00000132842&quot;, &quot;ENSG00000113273&quot;, &quot;ENSG00000132840&quot;, &quot;ENSG00000145692&quot;, &quot;ENSG00000132837&quot;, &quot;ENSG00000156427&quot;, &quot;ENSG00000174705&quot;, &quot;ENSG00000079691&quot;, &quot;ENSG00000272462&quot;, &quot;ENSG00000272810&quot;, &quot;ENSG00000187837&quot;, &quot;ENSG00000010704&quot;, &quot;ENSG00000218281&quot;, &quot;ENSG00000124508&quot;, &quot;ENSG00000111801&quot;, &quot;ENSG00000112763&quot;, &quot;ENSG00000124635&quot;, &quot;ENSG00000196787&quot;, &quot;ENSG00000197903&quot;, &quot;ENSG00000204305&quot;, &quot;ENSG00000204304&quot;, &quot;ENSG00000213654&quot;, &quot;ENSG00000198502&quot;, &quot;ENSG00000224796&quot;, &quot;ENSG00000223865&quot;, &quot;ENSG00000204248&quot;, &quot;ENSG00000204227&quot;, &quot;ENSG00000030110&quot;, &quot;ENSG00000096433&quot;, &quot;ENSG00000161896&quot;, &quot;ENSG00000161904&quot;, &quot;ENSG00000137207&quot;, &quot;ENSG00000171453&quot;, &quot;ENSG00000112715&quot;, &quot;ENSG00000181577&quot;, &quot;ENSG00000137216&quot;, &quot;ENSG00000112759&quot;, &quot;ENSG00000146232&quot;, &quot;ENSG00000196284&quot;, &quot;ENSG00000271857&quot;, &quot;ENSG00000124813&quot;, &quot;ENSG00000196284&quot;)
fungenomic1 &lt;- c(&quot;ENSG00000124813&quot;, &quot;ENSG00000271857&quot;, &quot;ENSG00000112782&quot;, &quot;ENSG00000172348&quot;, &quot;ENSG00000112175&quot;, &quot;ENSG00000111799&quot;, &quot;ENSG00000112697&quot;, &quot;ENSG00000118407&quot;, &quot;ENSG00000196586&quot;, &quot;ENSG00000105852&quot;, &quot;ENSG00000004799&quot;, &quot;ENSG00000158560&quot;, &quot;ENSG00000004864&quot;, &quot;ENSG00000006377&quot;, &quot;ENSG00000105880&quot;, &quot;ENSG00000253981&quot;, &quot;ENSG00000173295&quot;, &quot;ENSG00000104626&quot;, &quot;ENSG00000173281&quot;, &quot;ENSG00000248538&quot;, &quot;ENSG00000175806&quot;, &quot;ENSG00000147697&quot;, &quot;ENSG00000153310&quot;, &quot;ENSG00000106688&quot;, &quot;ENSG00000096968&quot;, &quot;ENSG00000136867&quot;, &quot;ENSG00000119321&quot;, &quot;ENSG00000176386&quot;, &quot;ENSG00000148229&quot;, &quot;ENSG00000157653&quot;, &quot;ENSG00000157657&quot;, &quot;ENSG00000196739&quot;, &quot;ENSG00000229314&quot;, &quot;ENSG00000228278&quot;, &quot;ENSG00000181634&quot;, &quot;ENSG00000106952&quot;, &quot;ENSG00000041982&quot;, &quot;ENSG00000196739&quot;, &quot;ENSG00000229314&quot;, &quot;ENSG00000228278&quot;, &quot;ENSG00000181634&quot;, &quot;ENSG00000106952&quot;, &quot;ENSG00000041982&quot;, &quot;ENSG00000182752&quot;, &quot;ENSG00000136869&quot;, &quot;ENSG00000167081&quot;, &quot;ENSG00000136944&quot;, &quot;ENSG00000169155&quot;, &quot;ENSG00000136859&quot;, &quot;ENSG00000136830&quot;, &quot;ENSG00000066382&quot;, &quot;ENSG00000110047&quot;, &quot;ENSG00000068971&quot;, &quot;ENSG00000110025&quot;, &quot;ENSG00000168060&quot;, &quot;ENSG00000149809&quot;, &quot;ENSG00000162298&quot;, &quot;ENSG00000014216&quot;, &quot;ENSG00000162241&quot;, &quot;ENSG00000126391&quot;, &quot;ENSG00000245532&quot;, &quot;ENSG00000251562&quot;, &quot;ENSG00000142186&quot;, &quot;ENSG00000168056&quot;, &quot;ENSG00000260233&quot;, &quot;ENSG00000176973&quot;, &quot;ENSG00000173338&quot;, &quot;ENSG00000173327&quot;, &quot;ENSG00000173039&quot;, &quot;ENSG00000172922&quot;, &quot;ENSG00000172638&quot;, &quot;ENSG00000175592&quot;, &quot;ENSG00000175550&quot;, &quot;ENSG00000175334&quot;, &quot;ENSG00000179292&quot;, &quot;ENSG00000174684&quot;, &quot;ENSG00000174576&quot;, &quot;ENSG00000174547&quot;, &quot;ENSG00000174516&quot;, &quot;ENSG00000174165&quot;, &quot;ENSG00000172638&quot;, &quot;ENSG00000175592&quot;, &quot;ENSG00000175550&quot;, &quot;ENSG00000175334&quot;, &quot;ENSG00000179292&quot;, &quot;ENSG00000174684&quot;, &quot;ENSG00000174576&quot;, &quot;ENSG00000174547&quot;, &quot;ENSG00000174516&quot;, &quot;ENSG00000174165&quot;, &quot;ENSG00000173914&quot;, &quot;ENSG00000173898&quot;, &quot;ENSG00000173599&quot;, &quot;ENSG00000173237&quot;, &quot;ENSG00000173156&quot;, &quot;ENSG00000172932&quot;, &quot;ENSG00000172613&quot;, &quot;ENSG00000175514&quot;, &quot;ENSG00000167800&quot;, &quot;ENSG00000137492&quot;, &quot;ENSG00000204529&quot;, &quot;ENSG00000254632&quot;, &quot;ENSG00000255100&quot;, &quot;ENSG00000182704&quot;, &quot;ENSG00000261578&quot;, &quot;ENSG00000078124&quot;, &quot;ENSG00000149269&quot;, &quot;ENSG00000227376&quot;, &quot;ENSG00000048649&quot;, &quot;ENSG00000064102&quot;, &quot;ENSG00000111790&quot;, &quot;ENSG00000110841&quot;, &quot;ENSG00000174236&quot;, &quot;ENSG00000230519&quot;, &quot;ENSG00000061794&quot;, &quot;ENSG00000087494&quot;, &quot;ENSG00000139263&quot;, &quot;ENSG00000118596&quot;, &quot;ENSG00000237176&quot;, &quot;ENSG00000111581&quot;, &quot;ENSG00000175782&quot;, &quot;ENSG00000135679&quot;, &quot;ENSG00000111605&quot;, &quot;ENSG00000090382&quot;, &quot;ENSG00000166225&quot;, &quot;ENSG00000166226&quot;, &quot;ENSG00000127328&quot;, &quot;ENSG00000166268&quot;, &quot;ENSG00000139318&quot;, &quot;ENSG00000070961&quot;, &quot;ENSG00000177889&quot;, &quot;ENSG00000120833&quot;, &quot;ENSG00000258274&quot;, &quot;ENSG00000169372&quot;, &quot;ENSG00000173588&quot;, &quot;ENSG00000089041&quot;, &quot;ENSG00000135124&quot;, &quot;ENSG00000188735&quot;, &quot;ENSG00000256546&quot;, &quot;ENSG00000184047&quot;, &quot;ENSG00000284934&quot;, &quot;ENSG00000256304&quot;, &quot;ENSG00000139726&quot;, &quot;ENSG00000130787&quot;, &quot;ENSG00000139722&quot;, &quot;ENSG00000090975&quot;, &quot;ENSG00000139726&quot;, &quot;ENSG00000130787&quot;, &quot;ENSG00000139722&quot;, &quot;ENSG00000150967&quot;, &quot;ENSG00000090975&quot;, &quot;ENSG00000051825&quot;, &quot;ENSG00000130921&quot;, &quot;ENSG00000235423&quot;, &quot;ENSG00000111328&quot;, &quot;ENSG00000139697&quot;, &quot;ENSG00000150977&quot;)
fungenomic2 &lt;- c(&quot;ENSG00000184209&quot;, &quot;ENSG00000111364&quot;, &quot;ENSG00000250091&quot;, &quot;ENSG00000178882&quot;, &quot;ENSG00000196498&quot;, &quot;ENSG00000140285&quot;, &quot;ENSG00000138592&quot;, &quot;ENSG00000242737&quot;, &quot;ENSG00000259618&quot;, &quot;ENSG00000081014&quot;, &quot;ENSG00000183578&quot;, &quot;ENSG00000137869&quot;, &quot;ENSG00000128849&quot;, &quot;ENSG00000128918&quot;, &quot;ENSG00000137845&quot;, &quot;ENSG00000169032&quot;, &quot;ENSG00000188501&quot;, &quot;ENSG00000166949&quot;, &quot;ENSG00000033800&quot;, &quot;ENSG00000129038&quot;, &quot;ENSG00000167178&quot;, &quot;ENSG00000129009&quot;, &quot;ENSG00000137868&quot;, &quot;ENSG00000140481&quot;, &quot;ENSG00000138623&quot;, &quot;ENSG00000179151&quot;, &quot;ENSG00000140465&quot;, &quot;ENSG00000103653&quot;, &quot;ENSG00000140506&quot;, &quot;ENSG00000140474&quot;, &quot;ENSG00000140497&quot;, &quot;ENSG00000178802&quot;, &quot;ENSG00000178741&quot;, &quot;ENSG00000178718&quot;, &quot;ENSG00000140398&quot;, &quot;ENSG00000140400&quot;, &quot;ENSG00000169410&quot;, &quot;ENSG00000177971&quot;, &quot;ENSG00000173548&quot;, &quot;ENSG00000173546&quot;, &quot;ENSG00000277639&quot;, &quot;ENSG00000103479&quot;, &quot;ENSG00000166971&quot;, &quot;ENSG00000103494&quot;, &quot;ENSG00000140718&quot;, &quot;ENSG00000177508&quot;, &quot;ENSG00000062038&quot;, &quot;ENSG00000103047&quot;, &quot;ENSG00000258429&quot;, &quot;ENSG00000102908&quot;, &quot;ENSG00000198373&quot;, &quot;ENSG00000157322&quot;, &quot;ENSG00000255185&quot;, &quot;ENSG00000157353&quot;, &quot;ENSG00000103051&quot;, &quot;ENSG00000189091&quot;, &quot;ENSG00000157368&quot;, &quot;ENSG00000051523&quot;, &quot;ENSG00000167513&quot;, &quot;ENSG00000141012&quot;, &quot;ENSG00000167522&quot;, &quot;ENSG00000197912&quot;, &quot;ENSG00000015413&quot;, &quot;ENSG00000131165&quot;, &quot;ENSG00000158792&quot;, &quot;ENSG00000158805&quot;, &quot;ENSG00000187741&quot;, &quot;ENSG00000258839&quot;, &quot;ENSG00000258947&quot;, &quot;ENSG00000223959&quot;, &quot;ENSG00000197879&quot;, &quot;ENSG00000132386&quot;, &quot;ENSG00000132383&quot;, &quot;ENSG00000108963&quot;, &quot;ENSG00000177374&quot;, &quot;ENSG00000070366&quot;, &quot;ENSG00000262333&quot;, &quot;ENSG00000167721&quot;, &quot;ENSG00000070444&quot;, &quot;ENSG00000127804&quot;, &quot;ENSG00000108578&quot;, &quot;ENSG00000263603&quot;, &quot;ENSG00000266490&quot;, &quot;ENSG00000181481&quot;, &quot;ENSG00000265798&quot;, &quot;ENSG00000196712&quot;, &quot;ENSG00000126861&quot;, &quot;ENSG00000185862&quot;, &quot;ENSG00000126860&quot;, &quot;ENSG00000131242&quot;, &quot;ENSG00000108651&quot;, &quot;ENSG00000178691&quot;, &quot;ENSG00000126858&quot;, &quot;ENSG00000172992&quot;, &quot;ENSG00000184922&quot;, &quot;ENSG00000159314&quot;, &quot;ENSG00000225190&quot;, &quot;ENSG00000266918&quot;, &quot;ENSG00000214425&quot;, &quot;ENSG00000264070&quot;, &quot;ENSG00000204652&quot;, &quot;ENSG00000120088&quot;, &quot;ENSG00000185294&quot;, &quot;ENSG00000186868&quot;, &quot;ENSG00000262539&quot;, &quot;ENSG00000261575&quot;, &quot;ENSG00000266497&quot;, &quot;ENSG00000238083&quot;, &quot;ENSG00000185829&quot;, &quot;ENSG00000073969&quot;, &quot;ENSG00000108379&quot;, &quot;ENSG00000108433&quot;, &quot;ENSG00000172992&quot;, &quot;ENSG00000136448&quot;, &quot;ENSG00000184922&quot;, &quot;ENSG00000225190&quot;, &quot;ENSG00000108433&quot;, &quot;ENSG00000141376&quot;, &quot;ENSG00000121068&quot;, &quot;ENSG00000121075&quot;, &quot;ENSG00000256124&quot;, &quot;ENSG00000125398&quot;, &quot;ENSG00000134490&quot;, &quot;ENSG00000101782&quot;, &quot;ENSG00000141458&quot;, &quot;ENSG00000053747&quot;, &quot;ENSG00000141447&quot;, &quot;ENSG00000105088&quot;, &quot;ENSG00000080573&quot;, &quot;ENSG00000130812&quot;, &quot;ENSG00000130810&quot;, &quot;ENSG00000244165&quot;, &quot;ENSG00000130816&quot;, &quot;ENSG00000267534&quot;, &quot;ENSG00000090339&quot;, &quot;ENSG00000105376&quot;, &quot;ENSG00000220201&quot;, &quot;ENSG00000267303&quot;, &quot;ENSG00000105397&quot;, &quot;ENSG00000065989&quot;, &quot;ENSG00000129353&quot;, &quot;ENSG00000267100&quot;, &quot;ENSG00000129351&quot;, &quot;ENSG00000079805&quot;, &quot;ENSG00000099203&quot;, &quot;ENSG00000127616&quot;, &quot;ENSG00000197256&quot;, &quot;ENSG00000105518&quot;, &quot;ENSG00000205517&quot;, &quot;ENSG00000198003&quot;, &quot;ENSG00000130176&quot;, &quot;ENSG00000102575&quot;, &quot;ENSG00000105223&quot;, &quot;ENSG00000268366&quot;, &quot;ENSG00000090013&quot;, &quot;ENSG00000090006&quot;, &quot;ENSG00000105245&quot;, &quot;ENSG00000086544&quot;, &quot;ENSG00000077312&quot;, &quot;ENSG00000261857&quot;, &quot;ENSG00000233622&quot;, &quot;ENSG00000167601&quot;, &quot;ENSG00000105329&quot;, &quot;ENSG00000105388&quot;, &quot;ENSG00000160570&quot;, &quot;ENSG00000105723&quot;, &quot;ENSG00000105722&quot;, &quot;ENSG00000079462&quot;, &quot;ENSG00000167619&quot;, &quot;ENSG00000105429&quot;, &quot;ENSG00000167646&quot;, &quot;ENSG00000080031&quot;, &quot;ENSG00000160471&quot;, &quot;ENSG00000095752&quot;, &quot;ENSG00000187902&quot;, &quot;ENSG00000090971&quot;, &quot;ENSG00000179954&quot;, &quot;ENSG00000171443&quot;, &quot;ENSG00000131848&quot;, &quot;ENSG00000078747&quot;, &quot;ENSG00000101460&quot;, &quot;ENSG00000101464&quot;, &quot;ENSG00000131069&quot;, &quot;ENSG00000078814&quot;, &quot;ENSG00000100991&quot;, &quot;ENSG00000088298&quot;, &quot;ENSG00000101000&quot;, &quot;ENSG00000242372&quot;, &quot;ENSG00000101019&quot;, &quot;ENSG00000125965&quot;, &quot;ENSG00000126001&quot;, &quot;ENSG00000125991&quot;, &quot;ENSG00000088340&quot;, &quot;ENSG00000244462&quot;, &quot;ENSG00000025293&quot;, &quot;ENSG00000171222&quot;, &quot;ENSG00000231123&quot;, &quot;ENSG00000157551&quot;, &quot;ENSG00000157554&quot;, &quot;ENSG00000183527&quot;, &quot;ENSG00000205581&quot;, &quot;ENSG00000189306&quot;, &quot;ENSG00000182841&quot;, &quot;ENSG00000100243&quot;, &quot;ENSG00000128274&quot;, &quot;ENSG00000274717&quot;, &quot;ENSG00000100266&quot;, &quot;ENSG00000100294&quot;, &quot;ENSG00000100300&quot;, &quot;ENSG00000100304&quot;, &quot;ENSG00000159307&quot;, &quot;ENSG00000100347&quot;, &quot;ENSG00000188677&quot;)
fungenomic &lt;- c(fungenomic, fungenomic1, fungenomic2)



#doi: https://doi.org/10.1101/835850
#Supplementary Table 2. Genes with significant cross-omics differences between high-grade and low-grade cartilage. (transcriptomic, proteomic both were significant)
s2 &lt;- c(&quot;ENSG00000164163&quot;, &quot;ENSG00000100997&quot;, &quot;ENSG00000154175&quot;, &quot;ENSG00000131473&quot;, &quot;ENSG00000068366&quot;, &quot;ENSG00000133627&quot;, &quot;ENSG00000148700&quot;, &quot;ENSG00000144843&quot;, &quot;ENSG00000035687&quot;, &quot;ENSG00000106624&quot;, &quot;ENSG00000173744&quot;, &quot;ENSG00000177674&quot;, &quot;ENSG00000101444&quot;, &quot;ENSG00000004455&quot;, &quot;ENSG00000085662&quot;, &quot;ENSG00000187134&quot;, &quot;ENSG00000151632&quot;, &quot;ENSG00000059573&quot;, &quot;ENSG00000128918&quot;, &quot;ENSG00000184254&quot;, &quot;ENSG00000116337&quot;, &quot;ENSG00000196510&quot;, &quot;ENSG00000151150&quot;, &quot;ENSG00000166825&quot;, &quot;ENSG00000189058&quot;, &quot;ENSG00000142192&quot;, &quot;ENSG00000140750&quot;, &quot;ENSG00000135931&quot;, &quot;ENSG00000100299&quot;, &quot;ENSG00000104763&quot;, &quot;ENSG00000160072&quot;, &quot;ENSG00000071553&quot;, &quot;ENSG00000006453&quot;, &quot;ENSG00000107949&quot;, &quot;ENSG00000099968&quot;, &quot;ENSG00000104164&quot;, &quot;ENSG00000183336&quot;, &quot;ENSG00000113460&quot;, &quot;ENSG00000186665&quot;, &quot;ENSG00000106392&quot;, &quot;ENSG00000223953&quot;, &quot;ENSG00000159403&quot;, &quot;ENSG00000123838&quot;, &quot;ENSG00000169239&quot;, &quot;ENSG00000042493&quot;, &quot;ENSG00000105519&quot;, &quot;ENSG00000153048&quot;, &quot;ENSG00000164305&quot;, &quot;ENSG00000196954&quot;, &quot;ENSG00000007080&quot;, &quot;ENSG00000152492&quot;, &quot;ENSG00000055813&quot;, &quot;ENSG00000110104&quot;, &quot;ENSG00000115009&quot;, &quot;ENSG00000260916&quot;, &quot;ENSG00000000971&quot;, &quot;ENSG00000016391&quot;, &quot;ENSG00000255112&quot;, &quot;ENSG00000130724&quot;, &quot;ENSG00000054938&quot;, &quot;ENSG00000163815&quot;, &quot;ENSG00000169583&quot;, &quot;ENSG00000159212&quot;, &quot;ENSG00000111726&quot;, &quot;ENSG00000164237&quot;, &quot;ENSG00000064666&quot;, &quot;ENSG00000117519&quot;, &quot;ENSG00000168434&quot;, &quot;ENSG00000204291&quot;, &quot;ENSG00000182871&quot;, &quot;ENSG00000164692&quot;, &quot;ENSG00000168542&quot;, &quot;ENSG00000187498&quot;, &quot;ENSG00000134871&quot;, &quot;ENSG00000130635&quot;, &quot;ENSG00000112280&quot;, &quot;ENSG00000184432&quot;, &quot;ENSG00000181789&quot;, &quot;ENSG00000160917&quot;, &quot;ENSG00000110090&quot;, &quot;ENSG00000157184&quot;, &quot;ENSG00000121005&quot;, &quot;ENSG00000006016&quot;, &quot;ENSG00000095713&quot;, &quot;ENSG00000160213&quot;, &quot;ENSG00000159692&quot;, &quot;ENSG00000117151&quot;, &quot;ENSG00000064601&quot;, &quot;ENSG00000117984&quot;, &quot;ENSG00000083799&quot;, &quot;ENSG00000138061&quot;, &quot;ENSG00000135929&quot;, &quot;ENSG00000110063&quot;, &quot;ENSG00000153904&quot;, &quot;ENSG00000162733&quot;, &quot;ENSG00000198171&quot;, &quot;ENSG00000168872&quot;, &quot;ENSG00000165732&quot;, &quot;ENSG00000145833&quot;, &quot;ENSG00000213782&quot;, &quot;ENSG00000023697&quot;, &quot;ENSG00000105928&quot;, &quot;ENSG00000065357&quot;, &quot;ENSG00000102967&quot;, &quot;ENSG00000105821&quot;, &quot;ENSG00000197635&quot;) 
s2.1&lt;- c(&quot;ENSG00000176978&quot;, &quot;ENSG00000188641&quot;, &quot;ENSG00000134755&quot;, &quot;ENSG00000103356&quot;, &quot;ENSG00000115380&quot;, &quot;ENSG00000113790&quot;, &quot;ENSG00000115211&quot;, &quot;ENSG00000145191&quot;, &quot;ENSG00000155849&quot;, &quot;ENSG00000074800&quot;, &quot;ENSG00000159023&quot;, &quot;ENSG00000082397&quot;, &quot;ENSG00000177106&quot;, &quot;ENSG00000157554&quot;, &quot;ENSG00000023318&quot;, &quot;ENSG00000081177&quot;, &quot;ENSG00000178896&quot;, &quot;ENSG00000120699&quot;, &quot;ENSG00000164687&quot;, &quot;ENSG00000119812&quot;, &quot;ENSG00000078098&quot;, &quot;ENSG00000116120&quot;, &quot;ENSG00000077942&quot;, &quot;ENSG00000079459&quot;, &quot;ENSG00000160752&quot;, &quot;ENSG00000154783&quot;, &quot;ENSG00000113578&quot;, &quot;ENSG00000132589&quot;, &quot;ENSG00000115414&quot;, &quot;ENSG00000162998&quot;, &quot;ENSG00000134363&quot;, &quot;ENSG00000163430&quot;, &quot;ENSG00000087086&quot;, &quot;ENSG00000179271&quot;, &quot;ENSG00000117308&quot;, &quot;ENSG00000164574&quot;, &quot;ENSG00000131386&quot;, &quot;ENSG00000100626&quot;, &quot;ENSG00000023909&quot;, &quot;ENSG00000131459&quot;, &quot;ENSG00000006625&quot;, &quot;ENSG00000137563&quot;, &quot;ENSG00000170266&quot;, &quot;ENSG00000124767&quot;, &quot;ENSG00000173221&quot;, &quot;ENSG00000108010&quot;, &quot;ENSG00000182512&quot;, &quot;ENSG00000163938&quot;, &quot;ENSG00000113552&quot;, &quot;ENSG00000100522&quot;, &quot;ENSG00000135677&quot;, &quot;ENSG00000171723&quot;, &quot;ENSG00000166123&quot;, &quot;ENSG00000164294&quot;, &quot;ENSG00000030582&quot;, &quot;ENSG00000105447&quot;, &quot;ENSG00000082701&quot;, &quot;ENSG00000100577&quot;, &quot;ENSG00000188342&quot;, &quot;ENSG00000144366&quot;, &quot;ENSG00000206172&quot;, &quot;ENSG00000244734&quot;, &quot;ENSG00000111911&quot;, &quot;ENSG00000170144&quot;, &quot;ENSG00000138668&quot;, &quot;ENSG00000164070&quot;, &quot;ENSG00000152137&quot;, &quot;ENSG00000166033&quot;, &quot;ENSG00000170801&quot;, &quot;ENSG00000196305&quot;, &quot;ENSG00000163453&quot;, &quot;ENSG00000172349&quot;, &quot;ENSG00000172458&quot;, &quot;ENSG00000106348&quot;, &quot;ENSG00000104613&quot;, &quot;ENSG00000198001&quot;, &quot;ENSG00000169047&quot;, &quot;ENSG00000172183&quot;, &quot;ENSG00000136156&quot;, &quot;ENSG00000205726&quot;, &quot;ENSG00000197256&quot;, &quot;ENSG00000102781&quot;, &quot;ENSG00000156113&quot;, &quot;ENSG00000116685&quot;, &quot;ENSG00000102753&quot;, &quot;ENSG00000135862&quot;, &quot;ENSG00000185896&quot;, &quot;ENSG00000111716&quot;, &quot;ENSG00000166816&quot;, &quot;ENSG00000116977&quot;, &quot;ENSG00000005156&quot;, &quot;ENSG00000050405&quot;, &quot;ENSG00000176619&quot;, &quot;ENSG00000161654&quot;, &quot;ENSG00000130332&quot;, &quot;ENSG00000119681&quot;, &quot;ENSG00000168056&quot;, &quot;ENSG00000139329&quot;, &quot;ENSG00000011009&quot;, &quot;ENSG00000166963&quot;, &quot;ENSG00000131711&quot;, &quot;ENSG00000108984&quot;, &quot;ENSG00000277443&quot;, &quot;ENSG00000124159&quot;, &quot;ENSG00000185432&quot;, &quot;ENSG00000166482&quot;, &quot;ENSG00000131446&quot;, &quot;ENSG00000008394&quot;, &quot;ENSG00000123342&quot;, &quot;ENSG00000107186&quot;, &quot;ENSG00000105926&quot;, &quot;ENSG00000133030&quot;, &quot;ENSG00000011028&quot;, &quot;ENSG00000125901&quot;, &quot;ENSG00000113048&quot;, &quot;ENSG00000053372&quot;, &quot;ENSG00000099810&quot;, &quot;ENSG00000150712&quot;, &quot;ENSG00000132382&quot;, &quot;ENSG00000105357&quot;, &quot;ENSG00000196586&quot;, &quot;ENSG00000138119&quot;, &quot;ENSG00000105835&quot;, &quot;ENSG00000095380&quot;, &quot;ENSG00000124479&quot;, &quot;ENSG00000104419&quot;, &quot;ENSG00000166136&quot;, &quot;ENSG00000049759&quot;, &quot;ENSG00000162599&quot;, &quot;ENSG00000239672&quot;)
s2.2 &lt;- c(&quot;ENSG00000243678&quot;, &quot;ENSG00000123609&quot;, &quot;ENSG00000152465&quot;, &quot;ENSG00000055044&quot;, &quot;ENSG00000139910&quot;, &quot;ENSG00000119655&quot;, &quot;ENSG00000215440&quot;, &quot;ENSG00000182446&quot;, &quot;ENSG00000159899&quot;, &quot;ENSG00000181019&quot;, &quot;ENSG00000184162&quot;, &quot;ENSG00000148572&quot;, &quot;ENSG00000065320&quot;, &quot;ENSG00000090273&quot;, &quot;ENSG00000136159&quot;, &quot;ENSG00000132661&quot;, &quot;ENSG00000136811&quot;, &quot;ENSG00000127083&quot;, &quot;ENSG00000178814&quot;, &quot;ENSG00000141447&quot;, &quot;ENSG00000070882&quot;, &quot;ENSG00000117859&quot;, &quot;ENSG00000149380&quot;, &quot;ENSG00000170515&quot;, &quot;ENSG00000111845&quot;, &quot;ENSG00000163346&quot;, &quot;ENSG00000183570&quot;, &quot;ENSG00000148843&quot;, &quot;ENSG00000067992&quot;, &quot;ENSG00000107438&quot;, &quot;ENSG00000131435&quot;, &quot;ENSG00000139197&quot;, &quot;ENSG00000178921&quot;, &quot;ENSG00000141959&quot;, &quot;ENSG00000247077&quot;, &quot;ENSG00000101856&quot;, &quot;ENSG00000067177&quot;, &quot;ENSG00000139289&quot;, &quot;ENSG00000173868&quot;, &quot;ENSG00000153823&quot;, &quot;ENSG00000087842&quot;, &quot;ENSG00000123143&quot;, &quot;ENSG00000188257&quot;, &quot;ENSG00000124181&quot;, &quot;ENSG00000198805&quot;, &quot;ENSG00000102978&quot;, &quot;ENSG00000105854&quot;, &quot;ENSG00000133110&quot;, &quot;ENSG00000131626&quot;, &quot;ENSG00000196262&quot;, &quot;ENSG00000066027&quot;, &quot;ENSG00000188783&quot;, &quot;ENSG00000126457&quot;, &quot;ENSG00000100462&quot;, &quot;ENSG00000136875&quot;, &quot;ENSG00000147224&quot;, &quot;ENSG00000157778&quot;, &quot;ENSG00000117569&quot;, &quot;ENSG00000148344&quot;, &quot;ENSG00000134247&quot;, &quot;ENSG00000124212&quot;, &quot;ENSG00000196396&quot;, &quot;ENSG00000132670&quot;, &quot;ENSG00000173482&quot;, &quot;ENSG00000105426&quot;, &quot;ENSG00000183255&quot;, &quot;ENSG00000177192&quot;, &quot;ENSG00000089159&quot;, &quot;ENSG00000183010&quot;, &quot;ENSG00000100504&quot;, &quot;ENSG00000116260&quot;, &quot;ENSG00000112210&quot;, &quot;ENSG00000132341&quot;, &quot;ENSG00000099901&quot;, &quot;ENSG00000106538&quot;, &quot;ENSG00000113643&quot;, &quot;ENSG00000159200&quot;, &quot;ENSG00000133119&quot;, &quot;ENSG00000155366&quot;, &quot;ENSG00000177189&quot;, &quot;ENSG00000116954&quot;, &quot;ENSG00000133818&quot;, &quot;ENSG00000114767&quot;, &quot;ENSG00000147655&quot;, &quot;ENSG00000133318&quot;, &quot;ENSG00000196154&quot;, &quot;ENSG00000177409&quot;, &quot;ENSG00000184178&quot;, &quot;ENSG00000006747&quot;, &quot;ENSG00000132581&quot;, &quot;ENSG00000117118&quot;, &quot;ENSG00000100445&quot;, &quot;ENSG00000157020&quot;, &quot;ENSG00000138674&quot;, &quot;ENSG00000075223&quot;, &quot;ENSG00000138758&quot;, &quot;ENSG00000166401&quot;, &quot;ENSG00000135919&quot;, &quot;ENSG00000109686&quot;, &quot;ENSG00000147010&quot;, &quot;ENSG00000174705&quot;, &quot;ENSG00000104635&quot;, &quot;ENSG00000102172&quot;, &quot;ENSG00000135632&quot;, &quot;ENSG00000145335&quot;, &quot;ENSG00000077312&quot;, &quot;ENSG00000172164&quot;, &quot;ENSG00000169371&quot;, &quot;ENSG00000124104&quot;, &quot;ENSG00000089006&quot;, &quot;ENSG00000095637&quot;, &quot;ENSG00000128487&quot;, &quot;ENSG00000115306&quot;, &quot;ENSG00000075142&quot;, &quot;ENSG00000100883&quot;, &quot;ENSG00000102359&quot;, &quot;ENSG00000111786&quot;, &quot;ENSG00000114850&quot;, &quot;ENSG00000118804&quot;, &quot;ENSG00000101109&quot;, &quot;ENSG00000165283&quot;, &quot;ENSG00000023734&quot;, &quot;ENSG00000125755&quot;, &quot;ENSG00000159082&quot;, &quot;ENSG00000136111&quot;, &quot;ENSG00000106638&quot;, &quot;ENSG00000184905&quot;, &quot;ENSG00000204065&quot;, &quot;ENSG00000204071&quot;, &quot;ENSG00000120708&quot;, &quot;ENSG00000169231&quot;, &quot;ENSG00000051596&quot;, &quot;ENSG00000100296&quot;, &quot;ENSG00000134809&quot;, &quot;ENSG00000105197&quot;, &quot;ENSG00000035862&quot;, &quot;ENSG00000142910&quot;, &quot;ENSG00000137462&quot;, &quot;ENSG00000125304&quot;, &quot;ENSG00000135048&quot;, &quot;ENSG00000136842&quot;, &quot;ENSG00000138594&quot;, &quot;ENSG00000123610&quot;, &quot;ENSG00000078902&quot;, &quot;ENSG00000025772&quot;, &quot;ENSG00000067369&quot;, &quot;ENSG00000136527&quot;, &quot;ENSG00000135148&quot;, &quot;ENSG00000126602&quot;, &quot;ENSG00000132481&quot;, &quot;ENSG00000162222&quot;, &quot;ENSG00000123416&quot;, &quot;ENSG00000183785&quot;, &quot;ENSG00000025708&quot;, &quot;ENSG00000154277&quot;, &quot;ENSG00000118939&quot;, &quot;ENSG00000132478&quot;, &quot;ENSG00000183696&quot;, &quot;ENSG00000090686&quot;, &quot;ENSG00000160293&quot;, &quot;ENSG00000162692&quot;, &quot;ENSG00000213585&quot;, &quot;ENSG00000136100&quot;, &quot;ENSG00000139722&quot;, &quot;ENSG00000138442&quot;, &quot;ENSG00000065183&quot;, &quot;ENSG00000196363&quot;, &quot;ENSG00000196419&quot;, &quot;ENSG00000139131&quot;, &quot;ENSG00000065978&quot;, &quot;ENSG00000140836&quot;, &quot;ENSG00000100711&quot;, &quot;ENSG00000109917&quot;)
cross_omic &lt;- c(s2, s2.1, s2.2)</code></pre>
</div>
<div id="enrichment-function" class="section level1">
<h1>Enrichment Function</h1>
<p>Use fisher’s exact test to compute significant enrichment.</p>
<pre class="r"><code>enrich &lt;- function(genes_of_interest, DEgenes, backgroundgenes){
     #fill contingency table
     DE_interest &lt;- length(intersect(genes_of_interest, DEgenes))
     background_interest &lt;- length(intersect(genes_of_interest, backgroundgenes))
     DE_non_interest &lt;- length(DEgenes) - DE_interest
     background_non_interest &lt;- length(backgroundgenes) - background_interest
     contingency_table &lt;- matrix(c(DE_interest, background_interest, DE_non_interest, background_non_interest), nrow = 2, ncol = 2, byrow = TRUE)
     
     #perform test
     print(contingency_table)
     return(fisher.test(contingency_table, alternative=&quot;greater&quot;))
}




enrich(fungenomic, DE_genes_ensg, other_genes_ensg)</code></pre>
<pre><code>     [,1] [,2]
[1,]   49  289
[2,] 1058 8747</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 0.02278
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
 1.061807      Inf
sample estimates:
odds ratio 
  1.401696 </code></pre>
<pre class="r"><code>enrich(cross_omic, DE_genes_ensg, other_genes_ensg)</code></pre>
<pre><code>     [,1] [,2]
[1,]   44  291
[2,] 1063 8745</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 0.11
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
 0.9294225       Inf
sample estimates:
odds ratio 
  1.243872 </code></pre>
<pre class="r"><code>#enrich(genes_GWAS_share_eQTL, DE_genes, other_genes)
#enrich(OA_GWAS_genes, DE_genes, other_genes)</code></pre>
</div>
<div id="store-the-overlapping-genes-for-futher-analysis" class="section level1">
<h1>Store the overlapping genes for futher analysis</h1>
<pre class="r"><code>overlap_fungenomic &lt;- intersect(fungenomic, DE_genes_ensg)
GWAS_descriptions &lt;- read.csv(&quot;data/GWAS_genes_within_500kb.csv&quot;, stringsAsFactors = FALSE)
GWAS_descriptions &lt;- GWAS_descriptions[match(overlap_fungenomic, GWAS_descriptions$Ensembl.ID), ] %&gt;% 
     dplyr::select(-c(Biotype, MostSevereConsequencef, PPC_Gene_MODERATEandHIGHg, Prot_logFCn, Prot_FDRo, RNASeq_logFCp, RNASeq_FDRq))
write.csv(GWAS_descriptions, &quot;output/GWAS_descriptions.csv&quot;)
saveRDS(overlap_fungenomic, &quot;output/overlap_fungenomic.rds&quot;)


overlap_crossomic &lt;- intersect(cross_omic, DE_genes_ensg)
saveRDS(overlap_crossomic, &quot;output/overlap_crossomic.rds&quot;)</code></pre>
</div>
<div id="look-at-enrichments-from-permuted-de-analysis-scrambled-strain-labels-to-make-sure-the-fet-results-arent-just-because-some-genes-are-more-likely-to-be-detected-as-de-in-these-cells" class="section level1">
<h1>Look at enrichments from permuted DE analysis (scrambled strain labels to make sure the FET results aren’t just because some genes are more likely to be detected as DE in these cells)</h1>
<pre class="r"><code>permuted_results &lt;- readRDS(&quot;output/permuted_results_DE.rds&quot;)

threshold &lt;- 0.05
for(permutation in 7:12){
     print(permutation)
     DE_results &lt;- permuted_results[[permutation]]
     DE_genes_ensg &lt;- rownames(DE_results)[DE_results$qvalue &lt; threshold]
     DE_genes &lt;- gene_anno$external_gene_name[match(DE_genes_ensg, gene_anno$ensembl_gene_id)]
     other_genes_ensg &lt;- rownames(DE_results)[DE_results$qvalue &gt;= threshold]
     other_genes &lt;- gene_anno$external_gene_name[match(other_genes_ensg, gene_anno$ensembl_gene_id)]

     print(enrich(fungenomic, DE_genes_ensg, other_genes_ensg))
     print(enrich(cross_omic, DE_genes_ensg, other_genes_ensg))
}</code></pre>
<pre><code>[1] 7
     [,1]  [,2]
[1,]    0   340
[2,]    8 10099

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

     [,1]  [,2]
[1,]    0   337
[2,]    8 10102

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

[1] 8
     [,1]  [,2]
[1,]    0   340
[2,]    0 10107

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

     [,1]  [,2]
[1,]    0   337
[2,]    0 10110

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

[1] 9
     [,1]  [,2]
[1,]    0   340
[2,]    0 10107

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

     [,1]  [,2]
[1,]    0   337
[2,]    0 10110

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

[1] 10
     [,1]  [,2]
[1,]    0   340
[2,]    0 10107

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

     [,1]  [,2]
[1,]    0   337
[2,]    0 10110

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

[1] 11
     [,1]  [,2]
[1,]    0   340
[2,]    0 10107

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

     [,1]  [,2]
[1,]    0   337
[2,]    0 10110

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 1
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
   0 Inf
sample estimates:
odds ratio 
         0 

[1] 12
     [,1]  [,2]
[1,]    2   338
[2,]   77 10030

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 0.7332
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
 0.1341083       Inf
sample estimates:
odds ratio 
  0.770783 

     [,1]  [,2]
[1,]    5   332
[2,]   74 10036

    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value = 0.1113
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
 0.7787804       Inf
sample estimates:
odds ratio 
  2.042324 </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_1.0.2

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.5       knitr_1.23       whisker_0.3-2    magrittr_1.5    
 [5] workflowr_1.6.2  tidyselect_1.1.0 R6_2.4.1         rlang_0.4.8     
 [9] stringr_1.4.0    tools_3.6.1      xfun_0.8         git2r_0.26.1    
[13] ellipsis_0.3.1   htmltools_0.5.0  yaml_2.2.1       digest_0.6.25   
[17] rprojroot_1.3-2  tibble_3.0.4     lifecycle_0.2.0  crayon_1.3.4    
[21] purrr_0.3.4      later_1.1.0.1    vctrs_0.3.4      promises_1.1.1  
[25] fs_1.3.1         glue_1.4.2       evaluate_0.14    rmarkdown_1.13  
[29] stringi_1.4.6    pillar_1.4.6     compiler_3.6.1   generics_0.0.2  
[33] backports_1.1.10 httpuv_1.5.1     pkgconfig_2.0.3 </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
