<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-09-30" />

<title>fasttopics_on_unintegrated_data</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">fasttopics_on_unintegrated_data</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-09-30</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-11-11
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree0e96722e64b52010d030697ac49fe0a119f369dctargetblank0e96722a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/0e96722e64b52010d030697ac49fe0a119f369dc" target="_blank">0e96722</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree0e96722e64b52010d030697ac49fe0a119f369dctargetblank0e96722a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/0e96722e64b52010d030697ac49fe0a119f369dc" target="_blank">0e96722</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/SCT_integrated_by_individual.rds
    Ignored:    data/Tachmazidou_30664745_ALLOA.txt.gz
    Ignored:    data/alasoo_etal/
    Ignored:    data/caliskan_etal/
    Ignored:    data/external_scRNA/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gene-annotation_hg37.txt
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds
    Ignored:    output/GO_terms_enriched.csv
    Ignored:    output/LDSC-SEG/
    Ignored:    output/SNPs.csv
    Ignored:    output/ash_power_analysis/
    Ignored:    output/classtpx/
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/fasttopics/
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash/
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/pairwise_ttest_results_SCTdata.rds
    Ignored:    output/pairwise_ttest_results_integrateddata.rds
    Ignored:    output/permuted_results_DE.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Unstaged changes:
    Modified:   analysis/Pilot_GoMAnalysis.Rmd
    Modified:   analysis/fasttopics_on_unintegrated_data.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/fasttopics_on_unintegrated_data.Rmd</code>) and HTML (<code>docs/fasttopics_on_unintegrated_data.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/0e96722e64b52010d030697ac49fe0a119f369dc/analysis/fasttopics_on_unintegrated_data.Rmd" target="_blank">0e96722</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-29
</td>
<td>
Add fasttopics on external data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/245b7f58e186a644aa4a916303219200cb592139/analysis/fasttopics_on_unintegrated_data.Rmd" target="_blank">245b7f5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-20
</td>
<td>
fasttopics on external scRNA data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/analysis/fasttopics_on_unintegrated_data.Rmd" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
<td>
knit fasttopics
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/fasttopics_on_unintegrated_data.html" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
<td>
knit fasttopics
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/analysis/fasttopics_on_unintegrated_data.Rmd" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
<td>
knit permutation analyses of strain labels
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/fasttopics_on_unintegrated_data.html" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
<td>
knit permutation analyses of strain labels
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/analysis/fasttopics_on_unintegrated_data.Rmd" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
<td>
add fasttopics analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/fasttopics_on_unintegrated_data.html" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
<td>
add fasttopics analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-data-merged-but-unintegrated-data-from-single-cell-pilot-experiment" class="section level1">
<h1>load data (merged but unintegrated data from single cell pilot experiment)</h1>
<pre class="r"><code>library(Seurat)
library(dplyr)</code></pre>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code>ANT1.2 &lt;- readRDS(&quot;data/ANT1_2.rds&quot;)</code></pre>
</div>
<div id="fasttopics-on-unintegrated-data" class="section level1">
<h1>Fasttopics on unintegrated data</h1>
<div id="separately-on-each-10x-lane" class="section level2">
<h2>separately on each 10x lane</h2>
<div id="ant1" class="section level3">
<h3>ANT1</h3>
<pre class="r"><code>library(fastTopics)
#need to fit the model to the count data (unintegrated)
raw_counts &lt;- ANT1.2@assays$RNA@counts[ANT1.2$orig.ident == &quot;ANT1&quot;,]
raw_counts &lt;- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#fit model
# fit &lt;- fit_poisson_nmf(raw_counts,k = 7,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/ANT1_k=7.rds&quot;)
fit &lt;- readRDS(&quot;output/fasttopics/ANT1_k=7.rds&quot;)

#compute weights and topics (need to add up to 1)
l &lt;- fit$L
f &lt;- fit$F
weights &lt;- sweep(l, MARGIN = 2, colSums(f), `*`)
scale &lt;- rowSums(weights)
weights &lt;- weights / scale
topics &lt;- f / colSums(f) # add up to 1</code></pre>
<pre class="r"><code>markers &lt;- c(&quot;THY1&quot;, &quot;NT5E&quot;, &quot;ENG&quot;, &quot;CD44&quot;,
             &quot;COL2A1&quot;, &quot;SOX9&quot;, &quot;COL10A1&quot;,
             &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;, &quot;TIMP2&quot;, &quot;TIMP3&quot;
             )

markers_description &lt;- data.frame(marker_type = c(rep(&quot;MSC&quot;, 4), rep(&quot;Chondrocyte&quot;, 8)))
rownames(markers_description) &lt;- markers</code></pre>
<pre class="r"><code>library(pheatmap)
library(RColorBrewer)
library(dummies)</code></pre>
<pre><code>dummies-1.5.6 provided by Decision Patterns</code></pre>
<pre class="r"><code>library(tidyr)
library(ggplot2)
library(cowplot)
theme_set(theme_cowplot())

#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT1&quot;])
rownames(sample) &lt;- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == &quot;ANT1&quot;,])
sample_labels &lt;- sample %&gt;% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT1&quot;]`) 
sample &lt;- sample_labels %&gt;% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT1&quot;]`, c(&quot;Individual&quot;, &quot;condition&quot;))
#reorder weights df to match sample order
weights &lt;- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = &quot;Heatmap of topic weights grouped by cell labels&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT1-1">
Past versions of “plot correlations ANT1-1.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT1-1.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT1&quot;])
rownames(sample_dummy) &lt;- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == &quot;ANT1&quot;,])
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>names(sample_dummies) &lt;- c(&quot;NA18855_Strain&quot;, &quot;NA18856_Unstrain&quot;, &quot;NA19160_Unstrain&quot;)
cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT1-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT1-2">
Past versions of “plot correlations ANT1-2.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT1-2.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same treatment together
sample_ind_treat_dummies &lt;- dummy.data.frame(sample)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored

Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_treat &lt;- cor(sample_ind_treat_dummies %&gt;% dplyr::select(contains(&quot;condition&quot;)), weights, method = &quot;pearson&quot;)
pheatmap(cor_by_treat, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by Treatment&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT1-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT1-3">
Past versions of “plot correlations ANT1-3.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT1-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT1-3.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#plot heatmap of relative expression of marker genes in each topic
topics_markers &lt;- topics[markers[-c(4,8,10:12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT1-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT1-4">
Past versions of “plot correlations ANT1-4.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT1-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT1-4.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>topics_markers &lt;- topics[markers[-c(1,2,4,8,10:12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT1-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT1-5">
Past versions of “plot correlations ANT1-5.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT1-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT1-5.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="ant2" class="section level3">
<h3>ANT2</h3>
<pre class="r"><code>#need to fit the model to the count data (unintegrated)
raw_counts &lt;- ANT1.2@assays$RNA@counts[ANT1.2$orig.ident == &quot;ANT2&quot;,]
raw_counts &lt;- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#fit model
# fit &lt;- fit_poisson_nmf(raw_counts,k = 7,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/ANT2_k=7.rds&quot;)
fit &lt;- readRDS(&quot;output/fasttopics/ANT2_k=7.rds&quot;)

#compute weights and topics (need to add up to 1)
l &lt;- fit$L
f &lt;- fit$F
weights &lt;- sweep(l, MARGIN = 2, colSums(f), `*`)
scale &lt;- rowSums(weights)
weights &lt;- weights / scale
topics &lt;- f / colSums(f) # add up to 1</code></pre>
<pre class="r"><code>library(pheatmap)
library(RColorBrewer)
library(dummies)
library(tidyr)

#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT2&quot;])
rownames(sample) &lt;- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == &quot;ANT2&quot;,])
sample_labels &lt;- sample %&gt;% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == \&quot;ANT2\&quot;]`)
sample &lt;- sample_labels %&gt;% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == \&quot;ANT2\&quot;]`, c(&quot;Individual&quot;, &quot;condition&quot;))
#reorder weights df to match sample order
weights &lt;- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = &quot;Heatmap of topic weights grouped by cell labels&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT2-1">
Past versions of “plot correlations ANT2-1.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT2-1.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == &quot;ANT2&quot;])
rownames(sample_dummy) &lt;- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == &quot;ANT2&quot;,])
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>names(sample_dummies) &lt;- c(&quot;NA18855_Untrain&quot;, &quot;NA19160_Strain&quot;)
cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT2-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT2-2">
Past versions of “plot correlations ANT2-2.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT2-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT2-2.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#plot heatmap of relative expression of marker genes in each topic
topics_markers &lt;- topics[markers[c(4,8,10,11,12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT2-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT2-3">
Past versions of “plot correlations ANT2-3.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT2-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT2-3.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>topics_markers &lt;- topics[markers[c(4,8,10,11)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20ANT2-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-ANT2-4">
Past versions of “plot correlations ANT2-4.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-ANT2-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations ANT2-4.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="jointly-on-both-10x-lanes" class="section level2">
<h2>jointly on both 10x lanes</h2>
<pre class="r"><code>#need to fit the model to the count data (unintegrated)
raw_counts &lt;- ANT1.2@assays$RNA@counts
raw_counts &lt;- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#fit model
# fit &lt;- fit_poisson_nmf(raw_counts,k = 11,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/unintegrated_k=11.rds&quot;)
fit &lt;- readRDS(&quot;output/fasttopics/unintegrated_k=11.rds&quot;)


#compute weights and topics (need to add up to 1)
l &lt;- fit$L
f &lt;- fit$F
weights &lt;- sweep(l, MARGIN = 2, colSums(f), `*`)
scale &lt;- rowSums(weights)
weights &lt;- weights / scale #rescaled l
topics &lt;- f / colSums(f) # add up to 1; rescaled f</code></pre>
<pre class="r"><code>#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample &lt;- as.data.frame(ANT1.2@meta.data$labels)
rownames(sample) &lt;- rownames(ANT1.2@meta.data)
sample_labels &lt;- sample %&gt;% 
     arrange(`ANT1.2@meta.data$labels`) 
sample &lt;- sample_labels %&gt;% 
     separate(`ANT1.2@meta.data$labels`, c(&quot;Individual&quot;, &quot;condition&quot;))
#reorder weights df to match sample order
weights &lt;- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = &quot;Heatmap of topic weights grouped by cell labels&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-1">
Past versions of “plot correlations joint-1.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-1.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy &lt;- as.data.frame(ANT1.2@meta.data$labels)
rownames(sample_dummy) &lt;- rownames(ANT1.2@meta.data)
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-2">
Past versions of “plot correlations joint-2.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-2.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same individual or treatment together
sample_ind_treat_dummies &lt;- dummy.data.frame(sample)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored

Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_ind &lt;- cor(sample_ind_treat_dummies %&gt;% dplyr::select(contains(&quot;Individual&quot;)), weights, method = &quot;pearson&quot;)
pheatmap(cor_by_ind, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by Individual&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-3">
Past versions of “plot correlations joint-3.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-3.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cor_by_treat &lt;- cor(sample_ind_treat_dummies %&gt;% dplyr::select(contains(&quot;condition&quot;)), weights, method = &quot;pearson&quot;)
pheatmap(cor_by_treat, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by Treatment&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-4">
Past versions of “plot correlations joint-4.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-4.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same 10x lane together 
collection_dummy &lt;- as.data.frame(ANT1.2@meta.data$orig.ident)
rownames(collection_dummy) &lt;- rownames(ANT1.2@meta.data)
collection_dummies &lt;- dummy.data.frame(collection_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_collection &lt;- cor(collection_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_collection, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by collection&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-5">
Past versions of “plot correlations joint-5.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-5.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#plot heatmap of relative expression of marker genes in each topic
topics_markers &lt;- topics[markers,]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-6">
Past versions of “plot correlations joint-6.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-6.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>topics_markers &lt;- topics[markers[-12],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-7">
Past versions of “plot correlations joint-7.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/1a4fcd8cf5b623bd412ab06158827f8ecafbd391/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint-7.png" target="_blank">1a4fcd8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="identify-clusters-from-pca-of-topic-proportions" class="section level2">
<h2>Identify clusters from PCA of topic proportions</h2>
<pre class="r"><code>#k=11, 10 pcs
p1 &lt;- pca_plot(poisson2multinom(fit),pcs = 1:2,fill = &quot;none&quot;)
p2 &lt;- pca_plot(poisson2multinom(fit),pcs = 3:4,fill = &quot;none&quot;)
p3 &lt;- pca_plot(poisson2multinom(fit),pcs = 5:6,fill = &quot;none&quot;)
p4 &lt;- pca_plot(poisson2multinom(fit),pcs = 7:8,fill = &quot;none&quot;)
p5 &lt;- pca_plot(poisson2multinom(fit),pcs = 9:10,fill = &quot;none&quot;)
plot_grid(p1,p2,p3,p4,p5,nrow = 2,ncol = 3)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/PCA%20topics-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-topics-1">
Past versions of “PCA topics-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-topics-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/PCA topics-1.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#k=11, 10 pcs
p1 &lt;- pca_plot(poisson2multinom(fit),pcs = 1:2,fill = &quot;none&quot;) + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p2 &lt;- pca_plot(poisson2multinom(fit),pcs = 3:4,fill = &quot;none&quot;) + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p3 &lt;- pca_plot(poisson2multinom(fit),pcs = 5:6,fill = &quot;none&quot;) + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels))) 
p4 &lt;- pca_plot(poisson2multinom(fit),pcs = 7:8,fill = &quot;none&quot;) + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p5 &lt;- pca_plot(poisson2multinom(fit),pcs = 9:10,fill = &quot;none&quot;) + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
plot_grid(p1,p2,p3,p4,p5,nrow = 2,ncol = 3)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/PCA%20topics-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-topics-2">
Past versions of “PCA topics-2.png”
</button>
</p>
<div id="fig-no-spaces-PCA-topics-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/PCA topics-2.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>p6  &lt;- pca_hexbin_plot(poisson2multinom(fit),pcs = 1:2) + guides(fill = &quot;none&quot;)
p7  &lt;- pca_hexbin_plot(poisson2multinom(fit),pcs = 3:4) + guides(fill = &quot;none&quot;)
p8  &lt;- pca_hexbin_plot(poisson2multinom(fit),pcs = 5:6) + guides(fill = &quot;none&quot;)
p9  &lt;- pca_hexbin_plot(poisson2multinom(fit),pcs = 7:8) + guides(fill = &quot;none&quot;)
p10 &lt;- pca_hexbin_plot(poisson2multinom(fit),pcs = 9:10)+ guides(fill = &quot;none&quot;)
plot_grid(p6,p7,p8,p9,p10,nrow = 2,ncol = 3)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/PCA%20topics-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-topics-3">
Past versions of “PCA topics-3.png”
</button>
</p>
<div id="fig-no-spaces-PCA-topics-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/PCA topics-3.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="look-at-expression-of-specific-markers-between-samples" class="section level1">
<h1>Look at expression of specific markers between samples</h1>
<pre class="r"><code>#by individual
for (marker in markers){
     query &lt;- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot &lt;- cbind(query, sample)
     plot &lt;- ggplot(query_plot, aes(x=query, color = as.factor(Individual), group=as.factor(Individual))) +
             stat_density(geom=&#39;line&#39;,kernel=&#39;gaussian&#39;,bw=&#39;nrd0&#39; 
                                                            #nrd0=&#39;Silverman&#39;
                                                            ,size=1,position=&#39;identity&#39;) +
                          labs(title = paste0(marker), x = &quot;Latent gene expression&quot;, color = &quot;Individual&quot;)
     print(plot)
}</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-1">
Past versions of “plot markers-1.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-1.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-2">
Past versions of “plot markers-2.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-2.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-3">
Past versions of “plot markers-3.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-3.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-4">
Past versions of “plot markers-4.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-4.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-5">
Past versions of “plot markers-5.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-5.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-6">
Past versions of “plot markers-6.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-6.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-7">
Past versions of “plot markers-7.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-7.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-8">
Past versions of “plot markers-8.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-8.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-9">
Past versions of “plot markers-9.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-9.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-10">
Past versions of “plot markers-10.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-10.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-11">
Past versions of “plot markers-11.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-11.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-12">
Past versions of “plot markers-12.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-12.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#by condition
for (marker in markers){
     query &lt;- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot &lt;- cbind(query, sample)
     plot &lt;- ggplot(query_plot, aes(x=query, color = as.factor(condition), group=as.factor(condition))) +
             stat_density(geom=&#39;line&#39;,kernel=&#39;gaussian&#39;,bw=&#39;nrd0&#39; 
                                                            #nrd0=&#39;Silverman&#39;
                                                            ,size=1,position=&#39;identity&#39;) +
                          labs(title = paste0(marker), x = &quot;Latent gene expression&quot;, color = &quot;Condition&quot;)
     print(plot)
}</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-13.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-13">
Past versions of “plot markers-13.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-13.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-14.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-14">
Past versions of “plot markers-14.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-14.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-15.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-15">
Past versions of “plot markers-15.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-15.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-16.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-16">
Past versions of “plot markers-16.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-16" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-16.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-17.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-17">
Past versions of “plot markers-17.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-17" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-17.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-18.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-18">
Past versions of “plot markers-18.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-18" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-18.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-19.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-19">
Past versions of “plot markers-19.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-19" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-19.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-20.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-20">
Past versions of “plot markers-20.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-20.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-21.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-21">
Past versions of “plot markers-21.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-21.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-22.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-22">
Past versions of “plot markers-22.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-22.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-23.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-23">
Past versions of “plot markers-23.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-23" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-23.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-24.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-24">
Past versions of “plot markers-24.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-24" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-24.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># by sample
for (marker in markers){
     query &lt;- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot &lt;- cbind(query, sample_labels)
     plot &lt;- ggplot(query_plot, aes(x=query, color = as.factor(ANT1.2@meta.data$labels), group=as.factor(ANT1.2@meta.data$labels))) +
             stat_density(geom=&#39;line&#39;,kernel=&#39;gaussian&#39;,bw=&#39;nrd0&#39; 
                                                            #nrd0=&#39;Silverman&#39;
                                                            ,size=1,position=&#39;identity&#39;) +
                          labs(title = paste0(marker), x = &quot;Latent gene expression&quot;, color = &quot;Label&quot;)
     print(plot)
}</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-25.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-25">
Past versions of “plot markers-25.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-25" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-25.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-26.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-26">
Past versions of “plot markers-26.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-26" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-26.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-27.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-27">
Past versions of “plot markers-27.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-27" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-27.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-28.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-28">
Past versions of “plot markers-28.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-28" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-28.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-29.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-29">
Past versions of “plot markers-29.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-29" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-29.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-30.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-30">
Past versions of “plot markers-30.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-30" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-30.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-31.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-31">
Past versions of “plot markers-31.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-31" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-31.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-32.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-32">
Past versions of “plot markers-32.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-32" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-32.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-33.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-33">
Past versions of “plot markers-33.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-33" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-33.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-34.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-34">
Past versions of “plot markers-34.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-34" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-34.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-35.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-35">
Past versions of “plot markers-35.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-35" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-35.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20markers-36.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-markers-36">
Past versions of “plot markers-36.png”
</button>
</p>
<div id="fig-no-spaces-plot-markers-36" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/757b4e7c8a57cc73b5beafaf340c8e28823ec8c8/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot markers-36.png" target="_blank">757b4e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="jointly-on-both-10x-lanes-only-unstrain-samples" class="section level2">
<h2>jointly on both 10x lanes (only unstrain samples)</h2>
<pre class="r"><code>#need to fit the model to the count data (unintegrated)
raw_counts &lt;- ANT1.2@assays$RNA@counts
raw_counts &lt;- as.matrix(raw_counts)
raw_counts &lt;- raw_counts[,ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;)]
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#fit model
# fit &lt;- fit_poisson_nmf(raw_counts,k = 11,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/unintegrated_unstrain_k=11.rds&quot;)
fit &lt;- readRDS(&quot;output/fasttopics/unintegrated_unstrain_k=11.rds&quot;)


#compute weights and topics (need to add up to 1)
l &lt;- fit$L
f &lt;- fit$F
weights &lt;- sweep(l, MARGIN = 2, colSums(f), `*`)
scale &lt;- rowSums(weights)
weights &lt;- weights / scale
topics &lt;- f / colSums(f) # add up to 1</code></pre>
<pre class="r"><code>#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;)])
rownames(sample) &lt;- rownames(ANT1.2@meta.data)[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;)]
sample_labels &lt;- sample %&gt;% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$labels %in% c(\&quot;NA19160_Unstrain\&quot;, \&quot;NA18855_Unstrain\&quot;, \&quot;NA18856_Unstrain\&quot;)]`) 
sample &lt;- sample_labels %&gt;% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$labels %in% c(\&quot;NA19160_Unstrain\&quot;, \&quot;NA18855_Unstrain\&quot;, \&quot;NA18856_Unstrain\&quot;)]`, c(&quot;Individual&quot;, &quot;condition&quot;))
#reorder weights df to match sample order
weights &lt;- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = &quot;Heatmap of topic weights grouped by cell labels&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-1">
Past versions of “plot correlations joint only unstrain-1.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-1.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy &lt;- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;)])
rownames(sample_dummy) &lt;- rownames(ANT1.2@meta.data[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;),])
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-2">
Past versions of “plot correlations joint only unstrain-2.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-2.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#same heatmap but group all the cells that are of the same individual or treatment together
sample_ind_treat_dummies &lt;- dummy.data.frame(sample)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_ind &lt;- cor(sample_ind_treat_dummies %&gt;% dplyr::select(contains(&quot;Individual&quot;)), weights, method = &quot;pearson&quot;)
pheatmap(cor_by_ind, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by Individual&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-3">
Past versions of “plot correlations joint only unstrain-3.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-3.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cor_by_treat &lt;- cor(sample_ind_treat_dummies %&gt;% dplyr::select(contains(&quot;condition&quot;)), weights, method = &quot;pearson&quot;)
#same heatmap but group all the cells that are of the same 10x lane together 
collection_dummy &lt;- as.data.frame(ANT1.2@meta.data$orig.ident[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;)])
rownames(collection_dummy) &lt;- rownames(ANT1.2@meta.data[ANT1.2$labels %in% c(&quot;NA19160_Unstrain&quot;, &quot;NA18855_Unstrain&quot;, &quot;NA18856_Unstrain&quot;),])
collection_dummies &lt;- dummy.data.frame(collection_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_collection &lt;- cor(collection_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_collection, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by collection&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-4">
Past versions of “plot correlations joint only unstrain-4.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-4.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#plot heatmap of relative expression of marker genes in each topic
topics_markers &lt;- topics[markers,]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-5">
Past versions of “plot correlations joint only unstrain-5.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-5.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>topics_markers &lt;- topics[markers[-12],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20joint%20only%20unstrain-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-plot-correlations-joint-only-unstrain-6">
Past versions of “plot correlations joint only unstrain-6.png”
</button>
</p>
<div id="fig-no-spaces-plot-correlations-joint-only-unstrain-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/96f2bbcbf8c0a731e6c0738b105dde1981518f5c/docs/figure/fasttopics_on_unintegrated_data.Rmd/plot correlations joint only unstrain-6.png" target="_blank">96f2bbc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-10-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="fasttopics-with-external-single-cell-datasets-as-well" class="section level1">
<h1>fasttopics with external single cell datasets as well</h1>
<pre class="r"><code>library(fastTopics)
library(Seurat)
library(Matrix)</code></pre>
<pre><code>
Attaching package: &#39;Matrix&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:tidyr&#39;:

    expand, pack, unpack</code></pre>
<pre class="r"><code>Combined.common &lt;- readRDS(&quot;data//external_scRNA/Combined_singlecell_data_first.rds&quot;)
#Combine.common.noliver &lt;- readRDS(&quot;data/Combined_singlecell_data_allGenes_noliver.rds&quot;)

#need to fit the model to the count data (unintegrated)
raw_counts &lt;- Combined.common@assays$RNA@counts
raw_counts &lt;- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#fit model
# fit &lt;- fit_poisson_nmf(raw_counts,k = 4,numiter = 70)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=4.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 5,numiter = 150)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=5.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 6,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=6.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 7,numiter = 100)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=7.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 8,numiter = 150)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=8.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 9,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=9.rds&quot;)
# fit &lt;- fit_poisson_nmf(raw_counts,k = 10,numiter = 200)
# saveRDS(fit, &quot;output/fasttopics/Combined_external_data_k=10.rds&quot;)
fit &lt;- readRDS(&quot;output/fasttopics/Combined_external_data_k=7.rds&quot;) #6 to 8 are pretty good

#compute weights and topics (need to add up to 1)
l &lt;- fit$L
f &lt;- fit$F
weights &lt;- sweep(l, MARGIN = 2, colSums(f), `*`)
scale &lt;- rowSums(weights)
weights &lt;- weights / scale
topics &lt;- f / colSums(f) # add up to 1</code></pre>
<pre class="r"><code>library(stringr)
library(pheatmap)
library(dummies)
library(tidyverse)</code></pre>
<pre><code>── Attaching packages ────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>✓ tibble  3.0.4     ✓ purrr   0.3.4
✓ readr   1.3.1     ✓ forcats 0.4.0</code></pre>
<pre><code>── Conflicts ───────────────────────────────────── tidyverse_conflicts() ──
x Matrix::expand() masks tidyr::expand()
x dplyr::filter()  masks stats::filter()
x dplyr::lag()     masks stats::lag()
x Matrix::pack()   masks tidyr::pack()
x Matrix::unpack() masks tidyr::unpack()</code></pre>
<pre class="r"><code>#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample &lt;- as.data.frame(Combined.common@meta.data$Cell.Type)
sample_labels &lt;- sample %&gt;% 
     dplyr::transmute(Cell.Type = stringr::word(`Combined.common@meta.data$Cell.Type`, start = 1))
rownames(sample_labels) &lt;- rownames(Combined.common@meta.data)
sample &lt;- sample_labels %&gt;% 
     dplyr::arrange(Cell.Type)
#reorder weights df to match sample order
weights_reordered &lt;- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights_reordered, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = &quot;Heatmap of topic weights grouped by cell labels&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy &lt;- as.data.frame(Combined.common@meta.data$Cell.Type)
sample_dummy &lt;- sample_dummy %&gt;% 
     dplyr::transmute(Cell.Type = stringr::word(`Combined.common@meta.data$Cell.Type`, start = 1))
rownames(sample_dummy) &lt;- rownames(Combined.common@meta.data)
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#same heatmap but group all the cells that are of the same label and strain condition together
sample_dummy &lt;- as.data.frame(Combined.common@meta.data$Cell.Type)
sample_dummy$`Combined.common@meta.data$Cell.Type` &lt;- as.character(sample_dummy$`Combined.common@meta.data$Cell.Type`)
rownames(sample_dummy) &lt;- rownames(Combined.common@meta.data)
metadata &lt;- as.data.frame(Combined.common@meta.data)
metadata &lt;- metadata %&gt;% filter(orig.ident == &quot;ANT1&quot; | orig.ident == &quot;ANT2&quot;)
sample_dummy$`Combined.common@meta.data$Cell.Type`[match(rownames(metadata), rownames(sample_dummy))] &lt;- metadata$labels
sample_dummy$`Combined.common@meta.data$Cell.Type` &lt;- gsub(&quot;NA18855_Strain|NA19160_Strain&quot;, &quot;iPSC-Chondrocyte_Strain&quot;, sample_dummy$`Combined.common@meta.data$Cell.Type`)
sample_dummy$`Combined.common@meta.data$Cell.Type` &lt;- gsub(&quot;NA18855_Unstrain|NA19160_Unstrain|NA18856_Unstrain&quot;, &quot;iPSC-Chondrocyte_Control&quot;, sample_dummy$`Combined.common@meta.data$Cell.Type`)
sample_dummy &lt;- sample_dummy %&gt;% 
     dplyr::transmute(Cell.Type = stringr::word(`Combined.common@meta.data$Cell.Type`, start = 1))
rownames(sample_dummy) &lt;- rownames(Combined.common@meta.data)
sample_dummies &lt;- dummy.data.frame(sample_dummy)</code></pre>
<pre><code>Warning in model.matrix.default(~x - 1, model.frame(~x - 1), contrasts =
FALSE): non-list contrasts argument ignored</code></pre>
<pre class="r"><code>cor_by_sample &lt;- cor(sample_dummies, weights, method = &quot;pearson&quot;)
cor_by_sample</code></pre>
<pre><code>                                          k1          k2          k3
Cell.TypeChondro                  -0.1043322 -0.11564622  0.80402893
Cell.TypeChondrocyte_             -0.2417376 -0.34787126 -0.05688802
Cell.TypeHepatocyte                0.9975943 -0.24772577 -0.10060723
Cell.TypeiPSC                     -0.1441412 -0.21161346 -0.08734820
Cell.TypeiPSC-Chondro_GAH         -0.1824084  0.64235952 -0.10966844
Cell.TypeiPSC-Chondrocyte_Control -0.0925125  0.48800563 -0.05401013
Cell.TypeiPSC-Chondrocyte_Strain  -0.0646449  0.34488087 -0.03873094
Cell.TypeiPSC-MSC                 -0.1415749 -0.19261427 -0.08467159
Cell.TypeiPSC-Osteo               -0.1233675 -0.02348919 -0.07242052
                                           k4          k5          k6
Cell.TypeChondro                  -0.16431017  0.57907761  0.02675445
Cell.TypeChondrocyte_             -0.11976722  0.21110517  0.94935346
Cell.TypeHepatocyte               -0.24457973 -0.14131601 -0.25413241
Cell.TypeiPSC                      0.94667486 -0.12068561 -0.21691099
Cell.TypeiPSC-Chondro_GAH         -0.13853237 -0.14540915 -0.27414519
Cell.TypeiPSC-Chondrocyte_Control -0.06844219 -0.07509628 -0.14055556
Cell.TypeiPSC-Chondrocyte_Strain  -0.04178083 -0.05171856 -0.09871285
Cell.TypeiPSC-MSC                 -0.01050536 -0.11849233 -0.21301402
Cell.TypeiPSC-Osteo               -0.15175968 -0.10291539 -0.18498492
                                           k7
Cell.TypeChondro                  -0.17811828
Cell.TypeChondrocyte_             -0.41910977
Cell.TypeHepatocyte               -0.29410774
Cell.TypeiPSC                     -0.24131517
Cell.TypeiPSC-Chondro_GAH          0.19996074
Cell.TypeiPSC-Chondrocyte_Control -0.02681642
Cell.TypeiPSC-Chondrocyte_Strain  -0.02672757
Cell.TypeiPSC-MSC                  0.61387798
Cell.TypeiPSC-Osteo                0.55060984</code></pre>
<pre class="r"><code>pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = &quot;Correlation by sample label&quot;)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>markers &lt;- c(&quot;THY1&quot;, &quot;NT5E&quot;, &quot;ENG&quot;, &quot;CD44&quot;,
             &quot;COL2A1&quot;, &quot;SOX9&quot;, &quot;COL10A1&quot;, &quot;ACAN&quot;,
             &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;, &quot;TIMP2&quot;, &quot;TIMP3&quot;,
             &quot;ALB&quot;, &quot;POU5F1&quot;
             )

markers_description &lt;- data.frame(marker_type = c(rep(&quot;MSC&quot;, 4), rep(&quot;Chondrocyte&quot;, 9), &quot;Hepatocyte&quot;, &quot;iPSC&quot;))
rownames(markers_description) &lt;- markers

#plot heatmap of relative expression of marker genes in each topic
topics_markers &lt;- topics[markers[-c(5,7,8,9)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>topics_markers &lt;- topics[markers[-c(5,7,8,9,14)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>topics_markers &lt;- topics[markers[-c(4,5,7,8,9,13)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/plot%20correlations%20external-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>Look at structure plot</p>
<pre class="r"><code># library(tidyverse)
# library(dplyr)
# library(ggplot2)
# weights_for_structure_plot &lt;- as.data.frame(weights)
# weights_for_structure_plot$Cell.Type &lt;- sample_labels$Cell.Type
# weights_for_structure_plot &lt;- weights_for_structure_plot %&gt;% 
#      tibble::rownames_to_column(var = &quot;Barcode&quot;) %&gt;% 
#      dplyr::mutate(Cell.Type = stringr::word(Cell.Type, start = 1))
# weights_for_structure_plot &lt;- weights_for_structure_plot %&gt;% 
#      group_by(Cell.Type) %&gt;% 
#      sample_n(800) %&gt;% 
#      ungroup() 
# #this is messy, but makes the plot look more clean by allowing each facet to be sorted by a different k
# weights_for_structure_plot &lt;- weights_for_structure_plot %&gt;% 
#      arrange(desc(k1)) %&gt;% 
#      mutate(num_1 = rank(k1, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k2) %&gt;% 
#      mutate(num_2 = rank(k2, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k3) %&gt;% 
#      mutate(num_3 = rank(k3, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k4) %&gt;% 
#      mutate(num_4 = rank(k4, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k5) %&gt;% 
#      mutate(num_5 = rank(k5, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k6) %&gt;% 
#      mutate(num_6 = rank(k6, ties.method = &quot;first&quot;)) %&gt;% 
#      arrange(k7) %&gt;% 
#      mutate(num_7 = rank(k7, ties.method = &quot;first&quot;))
# weights_for_structure_plot$num_in_facet &lt;- 0
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;Hepatocyte&quot;] &lt;- paste0(weights_for_structure_plot$num_1[weights_for_structure_plot$Cell.Type == &quot;Hepatocyte&quot;], &quot;Hepatocyte&quot;) #rank hepatocytes by k1
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;iPSC&quot;] &lt;- paste0(weights_for_structure_plot$num_4[weights_for_structure_plot$Cell.Type == &quot;iPSC&quot;], &quot;iPSC&quot;) #rank iPSC by k4
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;iPSC-Chondrocyte&quot; | weights_for_structure_plot$Cell.Type == &quot;iPSC-Chondro_GAH&quot; ] &lt;- paste0(weights_for_structure_plot$num_2[weights_for_structure_plot$Cell.Type == &quot;iPSC-Chondrocyte&quot; | weights_for_structure_plot$Cell.Type == &quot;iPSC-Chondro_GAH&quot;], &quot;iPSC-Chondrocyte&quot;) #rank iPSC-chondrocyte by k2
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;iPSC-MSC&quot; | weights_for_structure_plot$Cell.Type == &quot;iPSC-Osteo&quot; ] &lt;- paste0(weights_for_structure_plot$num_7[weights_for_structure_plot$Cell.Type == &quot;iPSC-MSC&quot; | weights_for_structure_plot$Cell.Type == &quot;iPSC-Osteo&quot;], &quot;iPSC-MSC-Osteo&quot;) #rank iPSC-MSCs/osteo by k7
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;Chondrocyte_&quot;] &lt;- paste0(weights_for_structure_plot$num_6[weights_for_structure_plot$Cell.Type == &quot;Chondrocyte_&quot;], &quot;Chondrocyte&quot;) #rank Chondrocyte_ by k6
# weights_for_structure_plot$num_in_facet[weights_for_structure_plot$Cell.Type == &quot;Chondro&quot;] &lt;- paste0(weights_for_structure_plot$num_3[weights_for_structure_plot$Cell.Type == &quot;Chondro&quot;], &quot;Chondro&quot;) #rank Chondro by k3
# 
#      
# topic_memberships &lt;- pivot_longer(weights_for_structure_plot,
#                                   cols = contains(&quot;k&quot;),
#                                   values_to = &quot;Proportion of Membership&quot;,
#                                   names_to = &quot;Topic&quot;)
# topic_memberships$Cell.Type_f &lt;- factor(topic_memberships$Cell.Type, levels = c(&quot;Hepatocyte&quot;, &quot;iPSC&quot;, &quot;iPSC-MSC&quot;, &quot;iPSC-Osteo&quot;, &quot;iPSC-Chondro_GAH&quot;, &quot;iPSC-Chondrocyte&quot;,  &quot;Chondrocyte_&quot;, &quot;Chondro&quot;))
# topic_colors &lt;- c(&quot;gold&quot;,&quot;royalblue&quot;,&quot;salmon&quot;,&quot;turquoise&quot;,&quot;purple&quot;,
#                   &quot;firebrick&quot;,&quot;olivedrab&quot;,
#                   &quot;red&quot;, &quot;black&quot;, &quot;grey&quot;, &quot;purple&quot;, &quot;green&quot;, &quot;pink&quot;, &quot;orange&quot;, &quot;yellow&quot;)
# 
# 
# ggplot(topic_memberships, aes(fill = Topic, y = `Proportion of Membership`, x = num_in_facet)) + 
#      geom_bar(position = &quot;fill&quot;, stat = &quot;identity&quot;, width = 1) + 
#      scale_fill_manual(values = topic_colors) + 
#      facet_grid(. ~ Cell.Type_f, scales = &quot;free_x&quot;, space = &quot;free_x&quot;) +
#      theme(axis.title.x=element_blank(),
#         axis.text.x=element_blank(),
#         axis.ticks.x=element_blank())




set.seed(1)
topic_colors &lt;- c(&quot;gold&quot;,&quot;royalblue&quot;,&quot;salmon&quot;,&quot;turquoise&quot;,&quot;olivedrab&quot;,
                  &quot;firebrick&quot;,&quot;forestgreen&quot;)
topics &lt;- c(1, 4, 7, 2, 6, 5, 3)
rows &lt;- sort(c(sample(which(sample_labels$Cell.Type == &quot;Hepatocyte&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;iPSC&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;iPSC-MSC&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;iPSC-Chondrocyte&quot;), 1815),
               sample(which(sample_labels$Cell.Type == &quot;iPSC-Chondro_GAH&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;iPSC-Osteo&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;Chondrocyte_&quot;), 800),
               sample(which(sample_labels$Cell.Type == &quot;Chondro&quot;), 800)))
structure_plot &lt;- structure_plot(select(poisson2multinom(fit),loadings = rows),
                      grouping = factor(sample_labels[rows,&quot;Cell.Type&quot;], 
                                        c(&quot;Hepatocyte&quot;, &quot;iPSC&quot;, &quot;iPSC-MSC&quot;, &quot;iPSC-Osteo&quot;, &quot;iPSC-Chondro_GAH&quot;, &quot;iPSC-Chondrocyte&quot;, &quot;Chondrocyte_&quot;, &quot;Chondro&quot;)),
                      topics = topics,
                      colors = topic_colors[topics],
                      perplexity = c(70,70,30,30,50,30,15,13),
                      n = Inf,gap = 50,num_threads = 4,verbose = FALSE)
print(structure_plot)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/structure%20plot%20external-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#separate by strain vs nonstrain
metadata &lt;- Combined.common@meta.data
metadata &lt;- metadata %&gt;% filter(orig.ident == &quot;ANT1&quot; | orig.ident == &quot;ANT2&quot;)
sample_labels$Cell.Type[match(rownames(metadata), rownames(sample_labels))] &lt;- metadata$labels
sample_labels$Cell.Type &lt;- gsub(&quot;NA18855_Strain|NA19160_Strain&quot;, &quot;iPSC-Chondrocyte_Strain&quot;, sample_labels$Cell.Type)
sample_labels$Cell.Type &lt;- gsub(&quot;NA18855_Unstrain|NA19160_Unstrain|NA18856_Unstrain&quot;, &quot;iPSC-Chondrocyte_Control&quot;, sample_labels$Cell.Type)

structure_plot_split &lt;- structure_plot(select(poisson2multinom(fit),loadings = rows),
                                       grouping = factor(sample_labels[rows,&quot;Cell.Type&quot;], 
                                                         c(&quot;Hepatocyte&quot;, &quot;iPSC&quot;, &quot;iPSC-MSC&quot;, &quot;iPSC-Osteo&quot;, &quot;iPSC-Chondro_GAH&quot;, &quot;iPSC-Chondrocyte_Control&quot;, &quot;iPSC-Chondrocyte_Strain&quot;, &quot;Chondrocyte_&quot;, &quot;Chondro&quot;)),
                                       topics = topics,
                                       colors = topic_colors[topics],
                                       perplexity = 60,
                                       n = Inf,gap = 50,num_threads = 4,verbose = FALSE)
print(structure_plot_split)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/structure%20plot%20external-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="differential-expression-analysis" class="section level1">
<h1>Differential Expression analysis</h1>
<pre class="r"><code>diff_count_topics &lt;- diff_count_analysis(fit, raw_counts)</code></pre>
<pre><code>Fitting 14451 x 7 = 101157 univariate Poisson models.
Computing log-fold change statistics.</code></pre>
<pre class="r"><code>#3,5,6 all seem to be chondro
fit_chondro &lt;- merge_topics(poisson2multinom(fit), c(&quot;k3&quot;, &quot;k5&quot;, &quot;k6&quot;))
diff_count_chondro &lt;- diff_count_analysis(fit_chondro, raw_counts)</code></pre>
<pre><code>Fitting 14451 x 5 = 72255 univariate Poisson models.
Computing log-fold change statistics.</code></pre>
<pre class="r"><code>library(ggrepel)
library(cowplot)
volcano_plot_with_highlighted_genes &lt;- function (diff_count_res, k, 
                                                 genes, ...) {
  dat &lt;- data.frame(beta  = diff_count_res$beta[genes,k],
                    y     = abs(diff_count_res$Z[genes,k]),
                    label = genes) 
  rows &lt;- match(genes,rownames(diff_count_res$beta))
  rownames(diff_count_res$beta)[rows] &lt;- &quot;&quot;
  rownames(diff_count_res$Z)[rows] &lt;- &quot;&quot;
  return(volcano_plot(diff_count_res,k = k,
                      ggplot_call = ggplot_call_for_volcano_plot,...) +
         geom_text_repel(data = dat,
                         mapping = aes(x = beta,y = y,label = label),
                         inherit.aes = FALSE,color = &quot;black&quot;,size = 5,
                         fontface = &quot;italic&quot;,segment.color = &quot;black&quot;,
                         segment.size = 0.25,
                         na.rm = TRUE))
}


# This is used by volcano_plot_with_highlighted_genes to create the
# volcano plot.
ggplot_call_for_volcano_plot &lt;- function (dat, y.label, topic.label) {
  ggplot(dat,aes_string(x = &quot;beta&quot;,y = &quot;y&quot;,fill = &quot;mean&quot;,label = &quot;label&quot;)) +
    geom_point(color = &quot;white&quot;,stroke = 0.3,shape = 21,na.rm = TRUE) +
    scale_y_continuous(trans = &quot;sqrt&quot;,
      breaks = c(0,1,2,5,10,20,50,100,200,500,1e3,2e3,5e3,1e4,2e4,5e4)) +
    scale_fill_gradient2(low = &quot;deepskyblue&quot;,mid = &quot;gold&quot;,high = &quot;orangered&quot;,
                         midpoint = mean(range(dat$mean))) +
    geom_text_repel(color = &quot;gray&quot;,size = 5,fontface = &quot;italic&quot;,
                    segment.color = &quot;gray&quot;,segment.size = 0.25,na.rm = TRUE) +
    labs(x = &quot;log-fold change (\u03b2)&quot;,y = y.label,fill = &quot;log10 mean&quot;) +
    theme_cowplot(font_size = 18) +
    theme(plot.title = element_text(size = 18,face = &quot;plain&quot;))}</code></pre>
<pre class="r"><code>MSC_markers &lt;- c(&quot;THY1&quot;, &quot;NT5E&quot;, &quot;ENG&quot;, &quot;CD44&quot;)
Chondrocyte_markers &lt;- c(&quot;SOX9&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;, &quot;TIMP2&quot;, &quot;TIMP3&quot;)
Hepatocyte_markers &lt;- c(&quot;AFP&quot;, &quot;ALB&quot;)
iPSC_markers &lt;- c(&quot;POU5F1&quot;, &quot;SOX2&quot;, &quot;NANOG&quot;)

#plots
p1 &lt;- volcano_plot_with_highlighted_genes(diff_count_topics,
                                          &quot;k1&quot;,
                                          genes = Hepatocyte_markers,
                                          label_above_quantile = 0.999) +
     ylim(c(0,225)) +
     guides(fill = &quot;none&quot;) +
     ggtitle(&quot;topic 1 (Hepatocytes)&quot;)</code></pre>
<pre><code>Scale for &#39;y&#39; is already present. Adding another scale for &#39;y&#39;, which
will replace the existing scale.</code></pre>
<pre class="r"><code>p2 &lt;- volcano_plot_with_highlighted_genes(diff_count_topics,
                                          &quot;k2&quot;,
                                          Chondrocyte_markers,
                                          label_above_quantile = 0.999) +
     ylim(c(0,225)) +
     guides(fill = &quot;none&quot;) +
     ggtitle(&quot;topic 2 (iPSC-Chondrocytes)&quot;)</code></pre>
<pre><code>Scale for &#39;y&#39; is already present. Adding another scale for &#39;y&#39;, which
will replace the existing scale.</code></pre>
<pre class="r"><code>p3 &lt;- volcano_plot_with_highlighted_genes(diff_count_chondro,
                                          &quot;k3+k5+k6&quot;,
                                          Chondrocyte_markers,
                                          label_above_quantile = 0.999) +
     ylim(c(0,225)) +
     guides(fill = &quot;none&quot;) +
     ggtitle(&quot;topic 3,5,6 (Chondrocyte)&quot;)</code></pre>
<pre><code>Scale for &#39;y&#39; is already present. Adding another scale for &#39;y&#39;, which
will replace the existing scale.</code></pre>
<pre class="r"><code>p4 &lt;- volcano_plot_with_highlighted_genes(diff_count_topics,
                                          &quot;k4&quot;,
                                          iPSC_markers,
                                          label_above_quantile = 0.999) +
     ylim(c(0,225)) +
     guides(fill = &quot;none&quot;) +
     ggtitle(&quot;topic 4 (iPSC)&quot;)</code></pre>
<pre><code>Scale for &#39;y&#39; is already present. Adding another scale for &#39;y&#39;, which
will replace the existing scale.</code></pre>
<pre class="r"><code>p7 &lt;- volcano_plot_with_highlighted_genes(diff_count_topics,
                                          &quot;k7&quot;,
                                          MSC_markers,
                                          label_above_quantile = 0.999) +
     ylim(c(0,225)) +
     guides(fill = &quot;none&quot;) +
     ggtitle(&quot;topic 7 (MSC)&quot;)</code></pre>
<pre><code>Scale for &#39;y&#39; is already present. Adding another scale for &#39;y&#39;, which
will replace the existing scale.</code></pre>
<pre class="r"><code>plot_grid(p1,p4,p7,p2,p3,nrow = 2,ncol = 3)</code></pre>
<p><img src="figure/fasttopics_on_unintegrated_data.Rmd/DE%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="semisupervised-fasttopics" class="section level1">
<h1>Semisupervised fasttopics</h1>
<p>Constrain it so that k = 4, and chondrocyte, iPSC, iPSC-MSC, and hepatocytes are the four existing topics/clusters. Then find out where iPSC-Chondrocytes fall in terms of membership in these four clusters.</p>
<pre class="r"><code># library(fastTopics)
# library(Seurat)
# library(Matrix)
# Combined.common &lt;- readRDS(&quot;data//external_scRNA/Combined_singlecell_data_first.rds&quot;)
# #Combine.common.noliver &lt;- readRDS(&quot;data/Combined_singlecell_data_allGenes_noliver.rds&quot;)
# 
# #randomly subset 1/4 of the cells to hopefully avoid segfault
# # Get the number of cells from the original Seurat object
# n.cells &lt;- length(rownames(Combined.common@meta.data))
# 
# # Define number of cells for the first object (subset 1)
# # &#39;ceiling&#39; is used to get integer values if n.cells is odd  
# n.cells.subset.1 &lt;- ceiling(n.cells/10)
# 
# # Set a seed for reproducible subsampling of cells
# set.seed(seed = 1)
# 
# # Sample randomly half (or half + 1 if n.cells is odd) of the cells 
# # for the first object (subset 1)
# cells.subset.1 &lt;- base::sample(x = rownames(Combined.common@meta.data), size = n.cells.subset.1, replace = F)
# 
# 
# # Create 2 subsetted objects, each formed by the corresponding cells
# # Set &#39;do.clean = T&#39; if you want to keep only the &#39;data&#39; and &#39;raw.data&#39; slots
# object.subset.1 &lt;- subset(x = Combined.common, cells = cells.subset.1)
# object.subset.1 &lt;- Combined.common
# 
# 
# #need to fit the model to the count data (unintegrated)
# raw_counts &lt;- object.subset.1@assays$RNA@counts
# raw_counts &lt;- as.matrix(raw_counts)
# #remove genes without any counts in droplets
# raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
# #get into correct orientation (barcodes x features)
# raw_counts &lt;- t(as.matrix(raw_counts))
# 
# n_topics &lt;- 4
# loadings &lt;- matrix(0, ncol = n_topics, nrow = nrow(raw_counts))
# rownames(loadings) &lt;- rownames(raw_counts)
# colnames(loadings) &lt;- c(&quot;k1&quot;, &quot;k2&quot;, &quot;k3&quot;, &quot;k4&quot;)
# celltypes &lt;- object.subset.1@meta.data$Cell.Type
# #chondrocyte: k1
# loadings[celltypes == &quot;Chondrocyte_&quot;, ] &lt;- c(1, 0, 0, 0)
# #iPSC: k2
# loadings[celltypes == &quot;iPSC&quot;, ] &lt;- c(0, 1, 0, 0)
# #hepatocyte: k3
# loadings[celltypes == &quot;Hepatocyte&quot;, ] &lt;- c(0, 0, 1, 0)
# #iPSC-MSC: k4
# loadings[celltypes == &quot;iPSC-MSC&quot;, ] &lt;- c(0, 0, 0, 1)
# update_loadings &lt;- 1 - rowSums(loadings)
# 
# initial_fit &lt;- init_poisson_nmf(X = raw_counts,
#                  F = matrix(0.1, ncol = n_topics, nrow = ncol(raw_counts)),
#                  L = loadings)
# 
# refit_semisupervised_k4 &lt;- fit_poisson_nmf(X = raw_counts,
#                                            fit0 = initial_fit,
#                                            update.loadings = update_loadings,
#                                            method = &quot;scd&quot;,
#                                            numiter = 200)
# saveRDS(refit_semisupervised_k4, &quot;output/fasttopics/Combined_external_data_semisupervised_k=4.rds&quot;)</code></pre>
<pre class="r"><code>library(Seurat)
library(Matrix)
library(classtpx)</code></pre>
<pre><code>Loading required package: slam</code></pre>
<pre class="r"><code>library(stringr)
Combined.common &lt;- readRDS(&quot;data//external_scRNA/Combined_singlecell_data_first.rds&quot;)
#need to fit the model to the count data (unintegrated)
raw_counts &lt;- Combined.common@assays$RNA@counts
raw_counts &lt;- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts &lt;- raw_counts[rowSums(raw_counts &gt; 0) &gt; 0,] 
#get into correct orientation (barcodes x features)
raw_counts &lt;- t(as.matrix(raw_counts))

#Semi-supervised topic modeling (define some known clusters prior to fitting topic model)
labels &lt;- Combined.common@meta.data$Cell.Type
#labels for &quot;known&quot; clusters
idx.ips &lt;- which(labels==&quot;iPSC&quot;)
idx.msc &lt;- which(labels==&quot;iPSC-MSC&quot;)
idx.chondro &lt;- which(labels==&quot;Chondrocyte_&quot;)
idx.hepato &lt;- which(labels==&quot;Hepatocyte&quot;)
kwn.smp &lt;- c(idx.ips,idx.msc,idx.chondro,idx.hepato)

class.labs &lt;- c(rep(&quot;iPSC&quot;, length(idx.ips)),
                rep(&quot;MSC&quot;, length(idx.msc)),
                rep(&quot;Chondrocyte&quot;, length(idx.chondro)),
                rep(&quot;Hepatocyte&quot;, length(idx.hepato))
                )

#Save GoM data
# for(k in c(4:6)) {
#   print(k)
#   tpx.clust &lt;- classtpx::class_topics(
#     raw_counts,
#     K=k,
#     known_samples=kwn.smp,
#     class_labs=class.labs,
#     method=&quot;omega.fix&quot;,
#     tol=0.1)
#   save(tpx.clust, file=paste0(&quot;output/classtpx/gom.sup.&quot;,k,&quot;.rds&quot;))
#   print(paste0(&quot;Finished k = &quot;,k))
# }




#remove MSC as a defined group
kwn.smp &lt;- c(idx.ips,idx.chondro,idx.hepato)

class.labs &lt;- c(rep(&quot;iPSC&quot;, length(idx.ips)),
                rep(&quot;Chondrocyte&quot;, length(idx.chondro)),
                rep(&quot;Hepatocyte&quot;, length(idx.hepato))
                )

#Save GoM data
# for(k in c(3)) {
#   print(k)
#   tpx.clust &lt;- classtpx::class_topics(
#     raw_counts,
#     K=k,
#     known_samples=kwn.smp,
#     class_labs=class.labs,
#     method=&quot;omega.fix&quot;,
#     tol=0.1)
#   save(tpx.clust, file=paste0(&quot;output/classtpx/gom.sup.noMSC.&quot;,k,&quot;.rds&quot;))
#   print(paste0(&quot;Finished k = &quot;,k))
# }</code></pre>
<!-- #look at results from all cells -->
<!-- ```{r analyses on all cells} -->
<!-- #Read GoM file -->
<!-- tpx.clust <- get(load(file="output/classtpx/gom.sup.4.rds")) -->
<!-- make_topic_plots <- function(omega, labels) { -->
<!--   omega <- as.data.frame(omega) -->
<!--   omega <- cbind(omega,labels) -->
<!--   col <- c("pink","mediumorchid","dodgerblue") -->
<!--   for (k in 1:(ncol(omega)-1)) { -->
<!--     p <- ggplot(data=omega, mapping=aes(x=labels, y=omega[,k])) + -->
<!--             labs(y=paste0("Topic ",k," Value"), x=paste0("Cell Type")) + -->
<!--             theme(axis.text.x=element_text(angle=-90, hjust=0, vjust=0)) -->
<!--     plot(p + geom_boxplot(width=0.5, outlier.shape=NA, outlier.colour=NA, fill=col[k])) -->
<!--   } -->
<!-- } -->
<!-- annotate_clusters <- function(theta_mat, top_features, countmatrix){ -->
<!--   top_features <- ExtractTopFeatures(theta_mat, top_features=top_features, method="poisson", options="min", shared=FALSE) -->
<!--   gene_vector <- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) rownames(countmatrix)[top_features[[1]][x,]])) -->
<!--   scores_vector <- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) top_features[[2]][x,])) -->
<!--   return(list(gene_vector, scores_vector)) -->
<!-- } -->
<!-- omega <- tpx.clust$omega -->
<!-- annotation <- data.frame(sample_id=paste0("X", c(1:NROW(omega))), tissue_label=as.factor(labels)) -->
<!-- rownames(omega) <- annotation$sample_id -->
<!-- CountClust::StructureGGplot(omega=omega, -->
<!--                 annotation=annotation, -->
<!--                 palette=c("pink","mediumorchid","blue", "yellow", "red", "brown"), -->
<!--                 yaxis_label="Tissue/Cell Type", -->
<!--                 order_sample=TRUE, -->
<!--                 axis_tick=list(axis_ticks_length=0.1, -->
<!--                                axis_ticks_lwd_y=0.1, -->
<!--                                axis_ticks_lwd_x=0.1, -->
<!--                                axis_label_size=7, -->
<!--                                axis_label_face="bold")) -->
<!-- #remove OA Chondrocytes -->
<!-- CountClust::StructureGGplot(omega=omega[-which(grepl("Chondro ", labels)),], -->
<!--                 annotation=annotation[-which(grepl("Chondro ", labels)),], -->
<!--                 palette=c("pink","mediumorchid","blue", "yellow", "red", "brown"), -->
<!--                 yaxis_label="Tissue/Cell Type", -->
<!--                 order_sample=TRUE, -->
<!--                 axis_tick=list(axis_ticks_length=0.1, -->
<!--                                axis_ticks_lwd_y=0.1, -->
<!--                                axis_ticks_lwd_x=0.1, -->
<!--                                axis_label_size=7, -->
<!--                                axis_label_face="bold")) -->
<!-- make_topic_plots(omega=omega[-which(grepl("Chondro ", labels)),], labels=labels[-which(grepl("Chondro ", labels))]) -->
<!-- genes <- annotate_clusters(theta_mat=tpx.clust$theta, top_features=10, countmatrix=raw_counts) -->
<!-- genes[[1]] -->
<!-- #assign cell type labels based on grade of membership cutoffs -->
<!-- gom.assign <- as.data.frame(omega) -->
<!-- gom.assign[,1] <- ifelse(gom.assign[,1] > 0.75, 1, 0) -->
<!-- gom.assign[,2] <- ifelse(gom.assign[,2] > 0.75, 1, 0) -->
<!-- gom.assign[,3] <- ifelse(gom.assign[,3] > 0.75, 1, 0) -->
<!-- gom.assign[,4] <- ifelse(gom.assign[,4] > 0.75, 1, 0) -->
<!-- upset(gom.assign, nintersects=NA, empty.intersections="on") -->
<!-- gom.assign$assign <- "other" -->
<!-- i=1 -->
<!-- while(i <= length(rownames(gom.assign))) { -->
<!--   if(gom.assign[i,1] == 1) {gom.assign$assign[i] <- "iPSC-MSC"} -->
<!--   if(gom.assign[i,2] == 1) {gom.assign$assign[i] <- "iPSC"} -->
<!--   if(gom.assign[i,3] == 1) {gom.assign$assign[i] <- "Chondrocyte"} -->
<!--   if(gom.assign[i,4] == 1) {gom.assign$assign[i] <- "Hepatocyte"} -->
<!--   i=i+1 -->
<!-- } -->
<!-- Combined.common@meta.data$GoM.Assign <- gom.assign$assign -->
<!-- Combined.common <- NormalizeData(Combined.common, normalization.method = "LogNormalize", scale.factor = 10000) -->
<!-- Combined.common <- FindVariableFeatures(Combined.common, selection.method = "vst", nfeatures = 2000) -->
<!-- all.genes <- rownames(Combined.common) -->
<!-- Combined.common <- ScaleData(Combined.common, features = all.genes) -->
<!-- Combined.common <- RunPCA(Combined.common, features = VariableFeatures(object = Combined.common)) -->
<!-- ElbowPlot(Combined.common, ndims = 100) -->
<!-- Combined.common <- RunUMAP(Combined.common, dims = 1:30) -->
<!-- DimPlot(Combined.common, group.by = c("GoM.Assign"), reduction = "umap") -->
<!-- DimPlot(Combined.common, group.by = c("Cell.Type"), reduction = "umap") -->
<!-- ``` -->
<!-- #look at results without iPSC-MSCs -->
<!-- ```{r no iPSC-MSCs} -->
<!-- #Read GoM file -->
<!-- tpx.clust <- get(load(file="output/classtpx/gom.sup.noMSC.3.rds")) -->
<!-- omega <- tpx.clust$omega -->
<!-- annotation <- data.frame(sample_id=paste0("X", c(1:NROW(omega))), tissue_label=as.factor(labels)) -->
<!-- rownames(omega) <- annotation$sample_id -->
<!-- CountClust::StructureGGplot(omega=omega, -->
<!--                 annotation=annotation, -->
<!--                 palette=c("pink","mediumorchid","blue", "yellow", "red", "brown"), -->
<!--                 yaxis_label="Tissue/Cell Type", -->
<!--                 order_sample=TRUE, -->
<!--                 axis_tick=list(axis_ticks_length=0.1, -->
<!--                                axis_ticks_lwd_y=0.1, -->
<!--                                axis_ticks_lwd_x=0.1, -->
<!--                                axis_label_size=12, -->
<!--                                axis_label_face="bold")) -->
<!-- #remove OA Chondrocytes -->
<!-- CountClust::StructureGGplot(omega=omega[-which(grepl("Chondro ", labels)),], -->
<!--                 annotation=annotation[-which(grepl("Chondro ", labels)),], -->
<!--                 palette=c("pink","mediumorchid","blue", "yellow", "red", "brown"), -->
<!--                 yaxis_label="Tissue/Cell Type", -->
<!--                 order_sample=TRUE, -->
<!--                 axis_tick=list(axis_ticks_length=0.1, -->
<!--                                axis_ticks_lwd_y=0.1, -->
<!--                                axis_ticks_lwd_x=0.1, -->
<!--                                axis_label_size=12, -->
<!--                                axis_label_face="bold")) -->
<!-- make_topic_plots(omega=omega[-which(grepl("Chondro ", labels)),], labels=labels[-which(grepl("Chondro ", labels))]) -->
<!-- genes <- annotate_clusters(theta_mat=tpx.clust$theta, top_features=10, countmatrix=raw_counts) -->
<!-- genes[[1]] -->
<!-- #assign cell type labels based on grade of membership cutoffs -->
<!-- gom.assign <- as.data.frame(omega) -->
<!-- gom.assign[,1] <- ifelse(gom.assign[,1] > 0.75, 1, 0) -->
<!-- gom.assign[,2] <- ifelse(gom.assign[,2] > 0.75, 1, 0) -->
<!-- gom.assign[,3] <- ifelse(gom.assign[,3] > 0.75, 1, 0) -->
<!-- gom.assign[,4] <- ifelse(gom.assign[,4] > 0.75, 1, 0) -->
<!-- upset(gom.assign, nintersects=NA, empty.intersections="on") -->
<!-- gom.assign$assign <- "other" -->
<!-- i=1 -->
<!-- while(i <= length(rownames(gom.assign))) { -->
<!--   if(gom.assign[i,1] == 1) {gom.assign$assign[i] <- "iPSC-MSC"} -->
<!--   if(gom.assign[i,2] == 1) {gom.assign$assign[i] <- "iPSC"} -->
<!--   if(gom.assign[i,3] == 1) {gom.assign$assign[i] <- "Chondrocyte"} -->
<!--   if(gom.assign[i,4] == 1) {gom.assign$assign[i] <- "Hepatocyte"} -->
<!--   i=i+1 -->
<!-- } -->
<!-- Combined.common@meta.data$GoM.Assign_noMSCs <- gom.assign$assign -->
<!-- Combined.common <- NormalizeData(Combined.common, normalization.method = "LogNormalize", scale.factor = 10000) -->
<!-- Combined.common <- FindVariableFeatures(Combined.common, selection.method = "vst", nfeatures = 2000) -->
<!-- all.genes <- rownames(Combined.common) -->
<!-- Combined.common <- ScaleData(Combined.common, features = all.genes) -->
<!-- Combined.common <- RunPCA(Combined.common, features = VariableFeatures(object = Combined.common)) -->
<!-- ElbowPlot(Combined.common, ndims = 100) -->
<!-- Combined.common <- RunUMAP(Combined.common, dims = 1:30) -->
<!-- DimPlot(Combined.common, group.by = c("GoM.Assign_noMSCs"), reduction = "umap") -->
<!-- DimPlot(Combined.common, group.by = c("Cell.Type"), reduction = "umap") -->
<!-- ``` -->
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] classtpx_0.99.0    slam_0.1-47        ggrepel_0.8.2     
 [4] forcats_0.4.0      purrr_0.3.4        readr_1.3.1       
 [7] tibble_3.0.4       tidyverse_1.3.0    stringr_1.4.0     
[10] Matrix_1.2-18      cowplot_1.1.0      ggplot2_3.3.2     
[13] tidyr_1.1.2        dummies_1.5.6      RColorBrewer_1.1-2
[16] pheatmap_1.0.12    fastTopics_0.3-181 dplyr_1.0.2       
[19] Seurat_3.1.3      

loaded via a namespace (and not attached):
  [1] readxl_1.3.1        backports_1.1.10    workflowr_1.6.2    
  [4] plyr_1.8.6          igraph_1.2.4.1      lazyeval_0.2.2     
  [7] splines_3.6.1       listenv_0.8.0       digest_0.6.25      
 [10] htmltools_0.5.0     fansi_0.4.1         gdata_2.18.0       
 [13] magrittr_1.5        cluster_2.1.0       ROCR_1.0-7         
 [16] maptpx_1.9-7        globals_0.12.5      modelr_0.1.8       
 [19] RcppParallel_5.0.2  matrixStats_0.57.0  MCMCpack_1.4-9     
 [22] prettyunits_1.1.1   colorspace_1.4-1    rvest_0.3.6        
 [25] rappdirs_0.3.1      haven_2.3.1         xfun_0.8           
 [28] crayon_1.3.4        jsonlite_1.7.1      hexbin_1.28.1      
 [31] survival_2.44-1.1   zoo_1.8-8           ape_5.4-1          
 [34] glue_1.4.2          gtable_0.3.0        MatrixModels_0.4-1 
 [37] leiden_0.3.1        future.apply_1.3.0  SparseM_1.78       
 [40] scales_1.1.1        DBI_1.1.0           bibtex_0.4.2       
 [43] Rcpp_1.0.5          metap_1.1           viridisLite_0.3.0  
 [46] progress_1.2.2      reticulate_1.16     rsvd_1.0.1         
 [49] tsne_0.1-3          htmlwidgets_1.5.2   httr_1.4.2         
 [52] gplots_3.0.1.1      ellipsis_0.3.1      ica_1.0-2          
 [55] pkgconfig_2.0.3     farver_2.0.3        uwot_0.1.5         
 [58] dbplyr_1.4.2        tidyselect_1.1.0    labeling_0.3       
 [61] rlang_0.4.8         reshape2_1.4.3      later_1.1.0.1      
 [64] munsell_0.5.0       cellranger_1.1.0    tools_3.6.1        
 [67] cli_2.1.0           generics_0.0.2      broom_0.7.0        
 [70] ggridges_0.5.1      evaluate_0.14       yaml_2.2.1         
 [73] mcmc_0.9-7          npsurv_0.4-0        knitr_1.23         
 [76] fs_1.3.1            fitdistrplus_1.0-14 caTools_1.17.1.2   
 [79] RANN_2.6.1          pbapply_1.4-0       future_1.18.0      
 [82] nlme_3.1-140        whisker_0.3-2       quantreg_5.73      
 [85] xml2_1.3.2          rstudioapi_0.11     compiler_3.6.1     
 [88] plotly_4.9.2.1      png_0.1-7           lsei_1.2-0         
 [91] reprex_0.3.0        stringi_1.4.6       lattice_0.20-41    
 [94] vctrs_0.3.4         pillar_1.4.6        lifecycle_0.2.0    
 [97] Rdpack_0.11-0       lmtest_0.9-37       RcppAnnoy_0.0.12   
[100] data.table_1.13.0   bitops_1.0-6        irlba_2.3.3        
[103] conquer_1.0.2       gbRd_0.4-11         httpuv_1.5.1       
[106] R6_2.4.1            promises_1.1.1      KernSmooth_2.23-15 
[109] gridExtra_2.3       codetools_0.2-16    MASS_7.3-52        
[112] gtools_3.8.1        assertthat_0.2.1    rprojroot_1.3-2    
[115] withr_2.3.0         sctransform_0.2.0   parallel_3.6.1     
[118] hms_0.5.3           quadprog_1.5-8      grid_3.6.1         
[121] coda_0.19-4         rmarkdown_1.13      Rtsne_0.15         
[124] git2r_0.26.1        lubridate_1.7.9    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
