<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-09-02
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 1 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree3ab1948e4a06fdf9971d62be9e58fe2154eef52btargetblank3ab1948a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/3ab1948e4a06fdf9971d62be9e58fe2154eef52b" target="_blank">3ab1948</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree3ab1948e4a06fdf9971d62be9e58fe2154eef52btargetblank3ab1948a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/3ab1948e4a06fdf9971d62be9e58fe2154eef52b" target="_blank">3ab1948</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds
    Ignored:    output/SNPs.csv
    Ignored:    output/bhat.rds
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash_500k.rds
    Ignored:    output/mash_500k_INT.rds
    Ignored:    output/mash_all_refit_500k_1.rds
    Ignored:    output/mash_all_refit_500k_2.rds
    Ignored:    output/mash_all_refit_500k_3.rds
    Ignored:    output/mash_all_refit_500k_4.rds
    Ignored:    output/mash_all_refit_500k_5.rds
    Ignored:    output/mash_all_refit_500k_6.rds
    Ignored:    output/mash_all_refit_500k_7.rds
    Ignored:    output/mash_all_refit_500k_8.rds
    Ignored:    output/mash_all_refit_500k_INT_1.rds
    Ignored:    output/mash_all_refit_500k_INT_2.rds
    Ignored:    output/mash_all_refit_500k_INT_3.rds
    Ignored:    output/mash_all_refit_500k_INT_4.rds
    Ignored:    output/mash_all_refit_500k_INT_5.rds
    Ignored:    output/mash_all_refit_500k_INT_6.rds
    Ignored:    output/mash_all_refit_500k_INT_7.rds
    Ignored:    output/mash_all_refit_500k_INT_8.rds
    Ignored:    output/mash_full_data.rds
    Ignored:    output/mash_full_data_INT.rds
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/shat.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Untracked files:
    Untracked:  code/read_pickle.py

Unstaged changes:
    Modified:   analysis/ANT1_ANT2_.Rmd
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/partition_variance.Rmd
    Modified:   analysis/powerAnalysis.Rmd
    Modified:   analysis/technical_factors_analysis.Rmd
    Deleted:    code/jobname.372680.err
    Deleted:    code/jobname.372680.out
    Deleted:    code/jobname.426937.err
    Deleted:    code/jobname.426937.out
    Deleted:    code/jobname.426966.err
    Deleted:    code/jobname.426966.out
    Modified:   code/mash.R
    Modified:   code/submit_mash.sh
    Modified:   data/MatrixEQTL/GE_strain.txt
    Modified:   data/MatrixEQTL/GE_unstrain.txt
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/63f57609e4da02fa5677e35c7caa4672b85a3c0f/analysis/index.Rmd" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/63f57609e4da02fa5677e35c7caa4672b85a3c0f/docs/index.html" target="_blank">63f5760</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f98f498bbd9ef5e2069900b48e940b7273549dee/analysis/index.Rmd" target="_blank">f98f498</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
make gene annotation analysis script
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f98f498bbd9ef5e2069900b48e940b7273549dee/docs/index.html" target="_blank">f98f498</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
make gene annotation analysis script
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/3e14c8a3a54b42da0ad190f2cbcd611e847b6e35/analysis/index.Rmd" target="_blank">3e14c8a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-09
</td>
<td>
Add notes for checking expression of certain markers
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/analysis/index.Rmd" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
run through with dummy data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/index.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a8cef1a7772685bd04a25c93dd5d7920caa1e492/analysis/index.Rmd" target="_blank">a8cef1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Create analysis files for analysis of BulkRNA pilot data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a8cef1a7772685bd04a25c93dd5d7920caa1e492/docs/index.html" target="_blank">a8cef1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Create analysis files for analysis of BulkRNA pilot data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/0cd6d9c044e13d0bbe8a9b4909eb96b1edd5c544/analysis/index.Rmd" target="_blank">0cd6d9c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-11-27
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="analysis-of-bulkrnascrna-sequencing-data-for-osteoarthritis-strain-project." class="section level1">
<h1>Analysis of bulkRNA/scRNA sequencing data for Osteoarthritis Strain project.</h1>
<p>LCL-derived iPSCs from 3 Yoruba individuals were differentiated into mesenchymal stem cells and subsequently into chondrocytes. iPSC-derived chondrocytes were either placed under control conditions or treated with cyclic tensile strain for 24 hours (Total of 6 individual x treatment pairs). 3 technical replicates of the MSC-chond differentiation + strain/control treatment were conducted. Bulk RNA sequencing was conducted for all 3 technical replicates. Single cell RNAseq using the 10x platform was conducted for one replicate of the experiment (collection of one sample failed, meaning only 5 samples were single-cell sequenced).</p>
<div id="pilot-n3-na18856-na18855-na19160" class="section level2">
<h2>PILOT (n=3; NA18856, NA18855, NA19160):</h2>
<p><strong>single-cell data</strong></p>
<ul>
<li><a href="ANT1_ANT2_.html">Analysis of single-cell pilot experiment data</a></li>
</ul>
<p><strong>Bulk Data</strong></p>
<ul>
<li><p><a href="raw_counts_analysis.html">Analysis of Raw bulkRNAseq pilot data</a></p></li>
<li><p><a href="Norm_filter.html">Normalization and filtering for lowly expressed genes</a></p></li>
<li><p><a href="download_gene_annotation.html">Download Gene Annotations</a></p></li>
<li><p><a href="technical_factors_analysis.html">Measuring impact of and accounting for technical factors</a></p></li>
<li><p><a href="check_expression.html">Checking expression of individual genes of interest</a></p></li>
<li><p><a href="partition_variance.html">Variance partitioning</a></p></li>
<li><p><a href="DE_analysis.html">Differential expression using Limma/Voom</a></p></li>
<li><p><a href="enrichment_DE.html">Looking for enrichment of DE genes amongst OA-related gene sets</a></p></li>
<li><p><a href="finding_interaction_example.html">Finding visual example of interaction between individual and treatment</a></p></li>
<li><p><a href="powerAnalysis.html">Conducting power analysis for eQTL mapping</a></p></li>
</ul>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
