<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-09-08" />

<title>Integrate single cell data across individuals</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Integrate single cell data across individuals</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-09-08</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-09-22
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreea0e5243cbddb54a33f538ee5c272e3885030900ctargetblanka0e5243a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/a0e5243cbddb54a33f538ee5c272e3885030900c" target="_blank">a0e5243</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreea0e5243cbddb54a33f538ee5c272e3885030900ctargetblanka0e5243a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/a0e5243cbddb54a33f538ee5c272e3885030900c" target="_blank">a0e5243</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/SCT_integrated_by_individual.rds
    Ignored:    data/alasoo_etal/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds
    Ignored:    output/GO_terms_enriched.csv
    Ignored:    output/SNPs.csv
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/fasttopics/
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash/
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Untracked files:
    Untracked:  data/caliskan_etal/

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/Pilot_GoMAnalysis.Rmd
    Modified:   analysis/integration_across_individuals.Rmd
    Modified:   analysis/powerAnalysis.Rmd
    Modified:   data/DEgenes_FDR05.rda
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/integration_across_individuals.Rmd</code>) and HTML (<code>docs/integration_across_individuals.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/0aa15d4ab7bf4f8337f7c635ed74f9224fa54c49/analysis/integration_across_individuals.Rmd" target="_blank">0aa15d4</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-10
</td>
<td>
normalize the topic memberships to add up to 1 for each cell
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>I would like to analyze all the samples together in order to determine cell type heterogeneity between samples jointly. However, they cluster by individual currently. Treat each individual as a separate sample and perform integration to solve this problem for this analysis.</p>
<p>Per tips from Kenneth Barr for integration, first detect variable features within each sample you are integrating separately, then integrate across the individuals using the union of both sets of variable features.</p>
<pre class="r"><code>library(Seurat)
library(gridExtra)
#split data into each individual/sample

#load single-cell data from pilot
#ANT1.2 &lt;- readRDS(&quot;data/ANT1_2.rds&quot;)
SCT_integrated &lt;- readRDS(&quot;data/SCT_ANT12_integrated.rds&quot;)

NA18855_Unstrain &lt;- subset(SCT_integrated, labels == &quot;NA18855_Unstrain&quot;)
NA18855_Strain &lt;- subset(SCT_integrated, labels == &quot;NA18855_Strain&quot;)
NA18856_Unstrain &lt;- subset(SCT_integrated, labels == &quot;NA18856_Unstrain&quot;)
NA19160_Unstrain &lt;- subset(SCT_integrated, labels == &quot;NA19160_Unstrain&quot;)
NA19160_Strain &lt;- subset(SCT_integrated, labels == &quot;NA19160_Strain&quot;)


seurat.list &lt;- list(NA18855_Unstrain, NA18856_Unstrain, NA19160_Unstrain)
for (i in 1:length(seurat.list)) {
    seurat.list[[i]] &lt;- SCTransform(seurat.list[[i]], verbose = FALSE)
}</code></pre>
<pre><code>Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached</code></pre>
<pre><code>Warning in sqrt(1/i): NaNs produced</code></pre>
<pre><code>Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached

Warning in theta.ml(y = y, mu = fit$fitted): iteration limit reached</code></pre>
<pre class="r"><code>SCT.features &lt;- SelectIntegrationFeatures(object.list = seurat.list, nfeatures = 5000)
seurat.list &lt;- PrepSCTIntegration(object.list = seurat.list, anchor.features = SCT.features, 
    verbose = FALSE)

#find anchors
seurat.anchors &lt;- FindIntegrationAnchors(object.list = seurat.list, normalization.method = &quot;SCT&quot;, 
    anchor.features = SCT.features, verbose = FALSE)
SCT.integrated &lt;- IntegrateData(anchorset = seurat.anchors, normalization.method = &quot;SCT&quot;,
    verbose = FALSE)</code></pre>
<pre><code>Warning: Adding a command log without an assay associated with it</code></pre>
<pre class="r"><code>#visualized integrated
SCT.integrated &lt;- RunPCA(SCT.integrated, verbose = FALSE, npcs = 100)
DimPlot(SCT.integrated, reduction = &quot;pca&quot;, group.by = c(&quot;labels&quot;))</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/split%20single%20cell%20data-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ElbowPlot(SCT.integrated, ndims = 100) #38 PCs?</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/split%20single%20cell%20data-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>SCT.integrated &lt;- FindNeighbors(SCT.integrated, dims = 1:38)</code></pre>
<pre><code>Computing nearest neighbor graph</code></pre>
<pre><code>Computing SNN</code></pre>
<pre class="r"><code>SCT.integrated &lt;- FindClusters(SCT.integrated, resolution = 0.4)</code></pre>
<pre><code>Modularity Optimizer version 1.3.0 by Ludo Waltman and Nees Jan van Eck

Number of nodes: 1203
Number of edges: 67982

Running Louvain algorithm...
Maximum modularity in 10 random starts: 0.6616
Number of communities: 3
Elapsed time: 0 seconds</code></pre>
<pre class="r"><code>DimPlot(SCT.integrated, group.by = c(&quot;labels&quot;), reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/split%20single%20cell%20data-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(SCT.integrated, group.by = c(&quot;seurat_clusters&quot;), reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/split%20single%20cell%20data-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>SCT.integrated &lt;- RunUMAP(SCT.integrated, dims = 1:38)</code></pre>
<pre><code>Warning: The default method for RunUMAP has changed from calling Python UMAP via reticulate to the R-native UWOT using the cosine metric
To use Python UMAP via reticulate, set umap.method to &#39;umap-learn&#39; and metric to &#39;correlation&#39;
This message will be shown once per session</code></pre>
<pre><code>11:42:07 UMAP embedding parameters a = 0.9922 b = 1.112</code></pre>
<pre><code>11:42:07 Read 1203 rows and found 38 numeric columns</code></pre>
<pre><code>11:42:07 Using Annoy for neighbor search, n_neighbors = 30</code></pre>
<pre><code>11:42:07 Building Annoy index with metric = cosine, n_trees = 50</code></pre>
<pre><code>0%   10   20   30   40   50   60   70   80   90   100%</code></pre>
<pre><code>[----|----|----|----|----|----|----|----|----|----|</code></pre>
<pre><code>**************************************************|
11:42:07 Writing NN index file to temp file /tmp/Rtmp6tESOe/file15ab46304a0f
11:42:07 Searching Annoy index using 1 thread, search_k = 3000
11:42:08 Annoy recall = 100%
11:42:08 Commencing smooth kNN distance calibration using 1 thread
11:42:09 Initializing from normalized Laplacian + noise
11:42:09 Commencing optimization for 500 epochs, with 45072 positive edges
11:42:12 Optimization finished</code></pre>
<pre class="r"><code>p1_SCT &lt;- DimPlot(SCT.integrated, group.by = c(&quot;orig.ident&quot;))
p2_SCT &lt;- DimPlot(SCT.integrated, group.by = c(&quot;labels&quot;))
p3_SCT &lt;- DimPlot(SCT.integrated, group.by = c(&quot;seurat_clusters&quot;))
grid.arrange(p1_SCT, p2_SCT, p3_SCT, nrow = 3)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/split%20single%20cell%20data-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>saveRDS(SCT.integrated, &quot;data/SCT_integrated_by_individual.rds&quot;)

# The above seems to do a good job integrating across individuals</code></pre>
<p>Examine clusters</p>
<pre class="r"><code>cluster_memberships &lt;- as.matrix(table(SCT.integrated@meta.data$integrated_snn_res.0.4, SCT.integrated@meta.data$labels))
cluster_memberships</code></pre>
<pre><code>   
    NA18855_Unstrain NA18856_Unstrain NA19160_Unstrain
  0              416              279              148
  1              141               85               59
  2               29               31               15</code></pre>
<pre class="r"><code>cluster_memberships_proportions &lt;- prop.table(cluster_memberships, margin = 2)
cluster_memberships_proportions</code></pre>
<pre><code>   
    NA18855_Unstrain NA18856_Unstrain NA19160_Unstrain
  0       0.70989761       0.70632911       0.66666667
  1       0.24061433       0.21518987       0.26576577
  2       0.04948805       0.07848101       0.06756757</code></pre>
<p>DE between clusters</p>
<pre class="r"><code>all.markers &lt;- FindAllMarkers(SCT.integrated, min.pct = 0, return.thresh = 1)</code></pre>
<pre><code>Calculating cluster 0</code></pre>
<pre><code>Calculating cluster 1</code></pre>
<pre><code>Calculating cluster 2</code></pre>
<p>Explore some cluster marker genes</p>
<pre class="r"><code>all.markers</code></pre>
<pre><code>                       p_val  avg_logFC pct.1 pct.2    p_val_adj cluster
TPM1            3.912479e-62  1.8579750 0.581 0.161 1.777439e-58       0
TAGLN           1.127590e-58  2.0272582 0.580 0.181 5.122641e-55       0
CALD1           6.781939e-51  1.4092165 0.542 0.161 3.081035e-47       0
TPM2            1.491684e-40  1.4439780 0.540 0.197 6.776721e-37       0
IGFBP7          1.498524e-36  1.1151099 0.565 0.206 6.807795e-33       0
TSC22D3         2.636282e-36 -1.7857600 0.289 0.644 1.197663e-32       0
C12orf75        3.565993e-35  0.8087089 0.505 0.189 1.620031e-31       0
CSRP2           1.257801e-33  0.9106273 0.515 0.172 5.714190e-30       0
VIM             3.116589e-33  1.2184551 0.520 0.222 1.415866e-29       0
ACTA2           2.999183e-31  1.1792399 0.562 0.267 1.362529e-27       0
GLIPR1          1.896953e-29  1.1140491 0.472 0.183 8.617859e-26       0
OTULINL         1.828324e-28 -1.6689732 0.231 0.539 8.306075e-25       0
FTH1            4.944837e-28 -1.4216012 0.359 0.658 2.246439e-24       0
MYL9            3.287405e-27  1.0568555 0.501 0.233 1.493468e-23       0
FKBP5           5.545913e-27 -1.4630875 0.270 0.544 2.519508e-23       0
SPINT2          2.331423e-26  1.0037012 0.507 0.236 1.059166e-22       0
MFGE8           2.471709e-26 -1.0870833 0.359 0.642 1.122897e-22       0
CEBPD           2.705765e-25 -1.1107933 0.355 0.642 1.229229e-21       0
C9orf16         3.869209e-25  0.7441207 0.533 0.250 1.757782e-21       0
SAA1            4.347634e-24 -1.2140268 0.240 0.542 1.975130e-20       0
MYL12A          5.013753e-24  0.9173690 0.515 0.253 2.277748e-20       0
ANXA2           7.140715e-24  0.7755254 0.515 0.244 3.244027e-20       0
CNN3            8.553264e-24  0.7261659 0.536 0.261 3.885748e-20       0
CRISPLD2        1.422210e-23 -1.0679964 0.308 0.592 6.461101e-20       0
IGF2.1          2.511689e-23 -1.4314218 0.250 0.528 1.141060e-19       0
XRCC4           5.134669e-23  1.1344994 0.498 0.208 2.332680e-19       0
PDPN            7.339452e-23 -1.3976403 0.344 0.619 3.334313e-19       0
IL1R1           9.622575e-23 -1.4821688 0.190 0.453 4.371536e-19       0
PLS3            1.027623e-22  0.9283558 0.490 0.256 4.668491e-19       0
CCND2           2.934742e-22  0.3696652 0.503 0.253 1.333253e-18       0
HOXD11          3.650764e-22 -0.8715019 0.234 0.431 1.658542e-18       0
ACTB            4.013708e-22  0.7341222 0.542 0.314 1.823427e-18       0
L2HGDH          7.841196e-22 -0.4926329 0.117 0.325 3.562255e-18       0
LMO4            1.891398e-21  0.5723928 0.502 0.258 8.592620e-18       0
GYPC            2.888049e-21 -1.3108445 0.191 0.436 1.312041e-17       0
PLAC9           3.658906e-21  0.8165066 0.459 0.236 1.662241e-17       0
MYL6            1.201701e-20  0.6057707 0.523 0.258 5.459326e-17       0
CSF1            1.806921e-20 -0.8103787 0.127 0.292 8.208843e-17       0
CEP128          3.770214e-20 -0.4960753 0.104 0.189 1.712808e-16       0
CDKN2A          4.195684e-20  1.4492038 0.496 0.222 1.906099e-16       0
RAB11FIP1       8.839149e-20 -0.8880746 0.361 0.583 4.015625e-16       0
FALEC           2.090909e-19 -0.3238915 0.108 0.272 9.498998e-16       0
PRLR            3.460912e-19 -1.4690121 0.157 0.383 1.572292e-15       0
KCTD16          6.097307e-19  1.0335995 0.466 0.217 2.770006e-15       0
S100A10         7.481027e-19  0.4709996 0.547 0.314 3.398631e-15       0
ACSL1           1.101741e-18 -0.7258456 0.333 0.547 5.005208e-15       0
ACTN1           1.806975e-18  0.4560904 0.521 0.300 8.209087e-15       0
MARCKSL1        2.978108e-18  1.3696164 0.477 0.200 1.352954e-14       0
DPYSL2          4.217337e-18 -1.0630800 0.269 0.528 1.915936e-14       0
GGT5            8.901694e-18 -1.1314890 0.147 0.369 4.044040e-14       0
IRS2            9.830908e-18 -0.9343413 0.244 0.475 4.466182e-14       0
FOXO1           1.397750e-17 -0.8918910 0.304 0.517 6.349980e-14       0
SOX4            2.399358e-17  1.0338191 0.492 0.278 1.090028e-13       0
PDCD4           2.847035e-17 -0.6623527 0.351 0.581 1.293408e-13       0
CDCA8           3.358345e-17 -0.4009570 0.066 0.169 1.525696e-13       0
MARCKS          3.447981e-17  0.4538011 0.495 0.311 1.566418e-13       0
TPM4            6.301888e-17  0.5276551 0.491 0.292 2.862948e-13       0
INHBA           6.952673e-17  1.0004416 0.463 0.239 3.158599e-13       0
TXNDC16         7.817282e-17 -0.8603224 0.297 0.525 3.551391e-13       0
PHF19           1.362135e-16 -0.8460551 0.214 0.367 6.188179e-13       0
ZIK1            1.428874e-16 -0.5252008 0.115 0.267 6.491376e-13       0
TUBA1A          1.815343e-16  1.1709972 0.476 0.206 8.247101e-13       0
CCDC69          1.911504e-16 -0.9039682 0.172 0.350 8.683962e-13       0
NFIL3           2.749537e-16 -0.9334044 0.237 0.417 1.249115e-12       0
BTG1            5.990738e-16 -1.1273106 0.288 0.497 2.721592e-12       0
TIMP3           9.132393e-16  0.8850557 0.461 0.267 4.148846e-12       0
AC242426.2      1.362172e-15 -0.6738351 0.108 0.256 6.188347e-12       0
MFAP5           2.920679e-15  1.5310798 0.450 0.189 1.326864e-11       0
PRDX2           6.262160e-15  0.4336772 0.502 0.292 2.844899e-11       0
CEBPB           7.295461e-15 -0.9651408 0.355 0.597 3.314328e-11       0
AC067838.1      8.233037e-15 -0.3676537 0.152 0.292 3.740269e-11       0
SLC7A8          1.489149e-14 -0.7957097 0.266 0.453 6.765202e-11       0
CALR            1.913779e-14  0.7150770 0.483 0.286 8.694297e-11       0
ATF4            2.072421e-14 -0.6596365 0.408 0.625 9.415010e-11       0
CDKN1C          2.179994e-14 -1.0736493 0.365 0.533 9.903712e-11       0
ZNF573          2.266169e-14 -0.4041112 0.085 0.131 1.029521e-10       0
ITGA1           2.309615e-14 -0.6247842 0.372 0.589 1.049258e-10       0
CCND1           2.975339e-14  0.2979354 0.501 0.314 1.351696e-10       0
VKORC1          3.677110e-14  0.2845901 0.470 0.267 1.670511e-10       0
CCNO            4.310200e-14 -0.2888878 0.113 0.206 1.958124e-10       0
MME             4.444862e-14 -0.8312369 0.330 0.533 2.019301e-10       0
LIMA1           4.827368e-14  0.8711659 0.484 0.308 2.193073e-10       0
COL12A1         5.147799e-14  0.4321312 0.447 0.236 2.338645e-10       0
PLA2G5          6.529560e-14 -1.0244874 0.163 0.336 2.966379e-10       0
AL357055.3      6.991407e-14 -0.4010612 0.103 0.172 3.176196e-10       0
GNL1            7.202213e-14 -0.5895639 0.312 0.450 3.271965e-10       0
IQGAP2          8.496513e-14 -1.0435670 0.198 0.428 3.859966e-10       0
RPS6KA1         1.359539e-13 -0.3623020 0.117 0.200 6.176385e-10       0
BOC             1.650828e-13 -1.3168897 0.219 0.422 7.499710e-10       0
PMEPA1          1.707947e-13  0.7610978 0.497 0.314 7.759201e-10       0
MSANTD2         1.792348e-13 -0.7274652 0.179 0.294 8.142635e-10       0
PCSK5           1.842922e-13 -0.6283466 0.141 0.294 8.372394e-10       0
TPRG1           1.933888e-13  0.4150794 0.094 0.081 8.785652e-10       0
CLDN11          1.969559e-13 -0.6584772 0.108 0.172 8.947706e-10       0
RAB31           2.440870e-13 -0.8926607 0.294 0.492 1.108887e-09       0
DDIT4           2.713620e-13 -0.7248512 0.257 0.439 1.232797e-09       0
DUSP10          3.993228e-13 -0.5835196 0.224 0.375 1.814123e-09       0
PPP1CB          4.252469e-13 -0.6071033 0.386 0.597 1.931897e-09       0
TXNRD2          4.382236e-13  0.5965714 0.418 0.211 1.990850e-09       0
NR3C1           5.564145e-13 -1.1590477 0.344 0.533 2.527791e-09       0
ABCA1           5.642337e-13 -0.8191590 0.203 0.369 2.563314e-09       0
FBXO32          5.879577e-13 -0.6294394 0.145 0.286 2.671092e-09       0
COL3A1          6.115116e-13 -1.1445896 0.376 0.603 2.778097e-09       0
TMEM141         7.237348e-13 -0.9851456 0.359 0.525 3.287927e-09       0
ABCC4           7.778764e-13 -0.5552029 0.221 0.400 3.533893e-09       0
TYMS            8.629085e-13 -0.7559489 0.171 0.247 3.920193e-09       0
PDGFRA          9.589446e-13 -0.9328437 0.257 0.472 4.356485e-09       0
THBS1           1.031778e-12  0.9477776 0.437 0.261 4.687367e-09       0
S100A16         1.078816e-12  0.7103398 0.456 0.250 4.901062e-09       0
MIGA2           1.120677e-12 -0.6181242 0.154 0.244 5.091238e-09       0
POLE            1.239939e-12 -0.2751559 0.134 0.200 5.633043e-09       0
NAV2            1.281278e-12 -0.3713964 0.330 0.531 5.820847e-09       0
SNHG7           1.390564e-12 -0.7243167 0.391 0.597 6.317334e-09       0
KDR             1.484404e-12 -0.9536113 0.151 0.306 6.743647e-09       0
SH3BGRL3        1.639401e-12  0.7728648 0.469 0.261 7.447798e-09       0
NFKBIA          1.880968e-12 -0.5512987 0.233 0.433 8.545238e-09       0
PYGB            2.864239e-12 -0.5769356 0.190 0.319 1.301224e-08       0
LRRC2           3.064919e-12 -0.4206241 0.113 0.236 1.392393e-08       0
ITPR2           3.750917e-12 -0.6953687 0.386 0.567 1.704042e-08       0
FILNC1          4.765962e-12 -0.7549709 0.119 0.192 2.165177e-08       0
CADM1           5.473900e-12  0.6621883 0.464 0.281 2.486793e-08       0
CLTB            1.102108e-11  0.7039239 0.489 0.281 5.006877e-08       0
TMEM120A        1.344455e-11 -0.6705323 0.346 0.483 6.107857e-08       0
PRKAR2B         1.375053e-11 -0.7597573 0.088 0.244 6.246867e-08       0
TTF2            1.461942e-11 -0.4489417 0.206 0.294 6.641602e-08       0
RTL9            1.594427e-11 -0.5807579 0.151 0.333 7.243481e-08       0
CALCOCO2        1.719280e-11 -0.8813686 0.351 0.500 7.810691e-08       0
C1R             1.733200e-11 -0.8411062 0.159 0.333 7.873928e-08       0
SAT1            1.843052e-11 -0.9995171 0.364 0.533 8.372986e-08       0
AC012467.2      1.869835e-11 -0.3870671 0.155 0.303 8.494660e-08       0
CD99            2.015809e-11  0.3670844 0.485 0.325 9.157818e-08       0
C1QL1           2.018283e-11 -0.5912173 0.132 0.231 9.169061e-08       0
S1PR1           2.360589e-11 -0.9341134 0.106 0.294 1.072416e-07       0
EBI3            2.387108e-11 -0.4338492 0.136 0.292 1.084463e-07       0
UACA            2.704318e-11  0.6053690 0.485 0.275 1.228571e-07       0
DYNLL1          2.714354e-11  0.4580695 0.527 0.347 1.233131e-07       0
ZFP36L2         3.015846e-11 -0.9814486 0.268 0.439 1.370099e-07       0
MKS1            4.115381e-11 -0.2629323 0.127 0.175 1.869618e-07       0
SFMBT2          4.406578e-11 -0.4489579 0.139 0.292 2.001909e-07       0
AKAP12          4.433815e-11 -1.0089383 0.140 0.319 2.014282e-07       0
HACD1           4.450201e-11  0.6742918 0.515 0.306 2.021726e-07       0
ACTBL2          4.491084e-11 -0.3207246 0.134 0.172 2.040299e-07       0
TFRC            4.647117e-11 -0.8053123 0.293 0.431 2.111185e-07       0
TGFBI           4.784234e-11 -0.5719154 0.369 0.556 2.173478e-07       0
CCDC171         5.128709e-11 -0.4741535 0.110 0.231 2.329972e-07       0
ITM2B           5.182237e-11  0.4754652 0.489 0.325 2.354290e-07       0
MERTK           5.329139e-11 -0.7558221 0.160 0.250 2.421028e-07       0
TIMP2           5.457000e-11 -0.2641930 0.492 0.358 2.479115e-07       0
ADGRG6          5.649029e-11 -0.7162125 0.107 0.278 2.566354e-07       0
YBX3            5.951743e-11 -0.8640813 0.415 0.550 2.703877e-07       0
OSR1            6.042600e-11 -0.6998598 0.108 0.278 2.745153e-07       0
LIG1            7.131771e-11 -0.4523395 0.146 0.206 3.239963e-07       0
RAB40A          9.223370e-11 -0.4657628 0.144 0.256 4.190177e-07       0
EPS8            1.003152e-10 -0.5041656 0.391 0.558 4.557322e-07       0
HEBP2           1.123310e-10 -0.6137289 0.286 0.403 5.103197e-07       0
PTGER2          1.185362e-10 -0.5475688 0.383 0.547 5.385100e-07       0
OGN             1.312328e-10  0.8202465 0.423 0.231 5.961906e-07       0
TGFBR1          1.366039e-10 -0.6926119 0.393 0.567 6.205913e-07       0
KIAA1024        1.383237e-10 -0.5352395 0.165 0.211 6.284044e-07       0
LRRC38          1.575386e-10 -0.8860265 0.171 0.297 7.156979e-07       0
PPME1           1.616027e-10  1.2547868 0.420 0.242 7.341610e-07       0
B2M             1.680901e-10 -0.8483342 0.369 0.558 7.636334e-07       0
NOS1AP          1.733140e-10 -0.5098933 0.193 0.328 7.873653e-07       0
CST3            1.764235e-10 -0.7000408 0.406 0.553 8.014919e-07       0
ADRA2C          1.816029e-10 -0.4301545 0.149 0.264 8.250222e-07       0
PLAGL1          1.936918e-10 -0.8008389 0.368 0.533 8.799418e-07       0
FAM111A         1.985553e-10 -0.5966476 0.186 0.333 9.020365e-07       0
ATP1A1          2.079040e-10 -0.8958024 0.399 0.547 9.445080e-07       0
NREP            2.540741e-10  0.6016349 0.497 0.311 1.154259e-06       0
ACTG2           2.626350e-10  1.0717353 0.453 0.250 1.193151e-06       0
Z97832.2        3.011016e-10 -0.3335727 0.090 0.136 1.367905e-06       0
PPFIA3          3.615951e-10 -0.4377852 0.136 0.231 1.642727e-06       0
PNRC1           3.671210e-10 -0.6008635 0.391 0.575 1.667831e-06       0
HOOK3           5.155261e-10 -0.2993254 0.376 0.525 2.342035e-06       0
HMGN3-AS1       5.445527e-10 -0.6128240 0.093 0.219 2.473903e-06       0
SELENOW         5.569268e-10  0.5007190 0.480 0.339 2.530118e-06       0
NAV1            5.611110e-10 -0.6888707 0.330 0.478 2.549127e-06       0
WDR90           5.801530e-10 -0.3946189 0.123 0.261 2.635635e-06       0
JPT1            6.141384e-10  0.7435202 0.495 0.314 2.790031e-06       0
CALB2           6.240931e-10 -0.8791527 0.158 0.306 2.835255e-06       0
PNPLA3          6.437496e-10 -0.4124828 0.192 0.256 2.924555e-06       0
C1GALT1         6.979516e-10 -0.6764951 0.302 0.433 3.170794e-06       0
LAMC1           7.252436e-10 -0.4448315 0.370 0.517 3.294782e-06       0
NCK1            7.424551e-10 -0.7335102 0.376 0.514 3.372973e-06       0
ADM             8.341598e-10 -0.7008541 0.120 0.258 3.789588e-06       0
SERPINF1        8.825738e-10 -0.9305893 0.291 0.436 4.009533e-06       0
WASF3           1.021439e-09 -0.7102933 0.186 0.314 4.640397e-06       0
MSX1            1.156904e-09  0.9602933 0.298 0.114 5.255816e-06       0
TNFAIP8         1.205090e-09 -0.6288550 0.358 0.481 5.474723e-06       0
CENPU           1.216718e-09 -0.3834921 0.093 0.189 5.527552e-06       0
KLF9            1.346014e-09 -1.0830930 0.210 0.303 6.114941e-06       0
SCAMP1-AS1      1.428805e-09 -0.3768051 0.192 0.294 6.491059e-06       0
AC098487.1      1.493681e-09 -0.4815557 0.153 0.269 6.785794e-06       0
TOX2            1.655281e-09 -0.8939297 0.070 0.228 7.519942e-06       0
PDGFRB          1.872177e-09 -0.3450073 0.287 0.394 8.505301e-06       0
GPX4            1.958812e-09 -0.7184297 0.388 0.564 8.898883e-06       0
DKK2            1.993731e-09  1.0009803 0.412 0.186 9.057520e-06       0
SPATS2L         1.995216e-09  0.5273888 0.493 0.319 9.064266e-06       0
ARL4D           1.998549e-09 -0.6669751 0.228 0.319 9.079410e-06       0
PRSS23          2.008149e-09  0.6788233 0.446 0.278 9.123022e-06       0
COL6A2          2.030947e-09 -0.7373474 0.365 0.492 9.226591e-06       0
DACH1           2.071156e-09 -0.3129933 0.157 0.242 9.409262e-06       0
FLNB            2.113112e-09 -0.5595948 0.267 0.386 9.599869e-06       0
EIF4EBP1        2.220460e-09 -0.8797782 0.377 0.531 1.008755e-05       0
AL031714.1      2.286872e-09  0.2684718 0.146 0.206 1.038926e-05       0
AQP1            2.304667e-09  0.8164119 0.464 0.275 1.047010e-05       0
LIX1            2.317593e-09  0.3553407 0.491 0.319 1.052882e-05       0
MMP2            2.380154e-09  0.3809057 0.458 0.339 1.081304e-05       0
DDIT3           2.402762e-09 -0.9400270 0.294 0.425 1.091575e-05       0
ARID5B          2.456725e-09 -0.4342875 0.332 0.489 1.116090e-05       0
SESN2           2.467662e-09 -0.5016335 0.238 0.342 1.121059e-05       0
LAMTOR5         2.542686e-09 -0.5702428 0.426 0.547 1.155142e-05       0
TMEM190         2.679952e-09 -0.2707873 0.115 0.169 1.217502e-05       0
EMP3            2.746156e-09 -0.4255311 0.420 0.567 1.247579e-05       0
CES4A           2.926399e-09 -0.3059346 0.184 0.294 1.329463e-05       0
HLA-C           2.981862e-09 -0.8535044 0.368 0.536 1.354660e-05       0
TESK2           3.145755e-09 -0.5216922 0.112 0.219 1.429116e-05       0
MYL12B          3.439292e-09  0.7238519 0.486 0.314 1.562470e-05       0
ZNF320          3.977806e-09 -0.3002362 0.091 0.142 1.807117e-05       0
AC124016.1      3.984329e-09 -0.3260433 0.107 0.211 1.810081e-05       0
BRI3            4.059254e-09 -0.5796861 0.383 0.575 1.844119e-05       0
ABCD3           4.808390e-09 -0.6561097 0.355 0.497 2.184452e-05       0
SLIT2           5.135915e-09 -0.3551857 0.165 0.242 2.333246e-05       0
ANOS1           5.208452e-09  1.2739107 0.420 0.217 2.366200e-05       0
SUGCT           6.116724e-09  0.9745086 0.440 0.231 2.778828e-05       0
ACTG1           6.555515e-09  0.3255797 0.504 0.350 2.978171e-05       0
AL669970.3      7.058831e-09  0.3330876 0.113 0.089 3.206827e-05       0
SCNN1A          7.819258e-09 -0.5674530 0.153 0.294 3.552289e-05       0
LMNB1           7.985083e-09 -0.5281083 0.121 0.303 3.627623e-05       0
CPM             8.067241e-09  0.2772784 0.128 0.158 3.664948e-05       0
CDC42EP1        8.145683e-09 -0.4475832 0.304 0.381 3.700584e-05       0
PDLIM7          8.313660e-09  0.3526906 0.469 0.314 3.776896e-05       0
LY6E            9.136587e-09  0.2982213 0.478 0.303 4.150751e-05       0
MBLAC1          9.632867e-09 -0.5314159 0.145 0.269 4.376212e-05       0
JAK1            1.045418e-08 -0.7713739 0.376 0.531 4.749332e-05       0
NFIA            1.047091e-08 -0.7346199 0.190 0.297 4.756935e-05       0
DPP4            1.110922e-08 -0.6043070 0.312 0.431 5.046921e-05       0
MT-CO1          1.143329e-08 -0.3721280 0.388 0.528 5.194143e-05       0
LGALS1          1.191743e-08  0.4955609 0.510 0.342 5.414089e-05       0
EBF1            1.278943e-08 -0.7883831 0.164 0.297 5.810236e-05       0
LMO7            1.285787e-08  0.4129265 0.478 0.272 5.841331e-05       0
NAMPT           1.305019e-08 -1.1872905 0.187 0.325 5.928700e-05       0
MNDA            1.323101e-08  0.4531313 0.122 0.089 6.010847e-05       0
FLRT3           1.448473e-08 -0.4358347 0.166 0.261 6.580413e-05       0
FGFR1           1.463886e-08 -0.2935736 0.415 0.575 6.650434e-05       0
MGST1           1.506961e-08 -0.6121987 0.295 0.456 6.846124e-05       0
STC2            1.530281e-08 -0.7466909 0.369 0.503 6.952068e-05       0
AL928654.1      1.562028e-08 -0.2614580 0.153 0.172 7.096291e-05       0
GAS1            1.602156e-08  0.6925827 0.447 0.264 7.278593e-05       0
CTSK            1.794084e-08  0.3735743 0.098 0.136 8.150523e-05       0
PNMT            1.889756e-08 -1.2266427 0.190 0.358 8.585159e-05       0
IFITM1          1.927817e-08 -1.1550507 0.091 0.203 8.758073e-05       0
UBAP1L          2.034734e-08 -0.3063812 0.103 0.192 9.243797e-05       0
MRPS2           2.125047e-08 -0.6659437 0.289 0.406 9.654090e-05       0
BMPER           2.286232e-08 -0.7269650 0.333 0.447 1.038635e-04       0
SMPDL3A         2.347877e-08 -0.3414639 0.244 0.308 1.066641e-04       0
CNTROB          2.408527e-08 -0.6517606 0.151 0.250 1.094194e-04       0
HAPLN3          2.438787e-08 -0.3127433 0.140 0.211 1.107941e-04       0
C1orf112        2.464425e-08 -0.5535324 0.208 0.300 1.119588e-04       0
CD68            2.507116e-08 -0.5159130 0.189 0.275 1.138983e-04       0
CPZ             2.534593e-08  0.4412564 0.149 0.139 1.151466e-04       0
FRY-AS1         2.612247e-08 -0.2543212 0.119 0.153 1.186744e-04       0
PCDHB14         3.031415e-08 -0.2657385 0.127 0.186 1.377172e-04       0
PLIN2           3.074045e-08 -0.9377860 0.166 0.342 1.396538e-04       0
ACKR2           3.075644e-08 -0.3327500 0.196 0.253 1.397265e-04       0
MYLIP           3.207072e-08 -0.4207275 0.100 0.225 1.456973e-04       0
PRPS2           3.349045e-08 -0.5955048 0.172 0.275 1.521471e-04       0
ANGPT2          3.373441e-08 -0.3149752 0.157 0.194 1.532554e-04       0
MCM3AP-AS1      3.375243e-08 -0.3899240 0.083 0.178 1.533373e-04       0
DGCR8           3.735629e-08 -0.4526302 0.187 0.300 1.697096e-04       0
PTMA            3.782004e-08  0.2631154 0.476 0.322 1.718165e-04       0
SELENBP1        3.940503e-08 -0.5228113 0.135 0.214 1.790171e-04       0
ANGPTL4         3.999722e-08 -0.7011634 0.174 0.308 1.817074e-04       0
LAMB1           4.049281e-08 -0.6118455 0.390 0.519 1.839589e-04       0
P3H2            4.319315e-08 -0.4236335 0.106 0.197 1.962265e-04       0
CUBN            4.356976e-08  0.7388330 0.147 0.072 1.979374e-04       0
CERS6           4.463107e-08 -0.3345630 0.255 0.342 2.027589e-04       0
RAD51AP1        4.550634e-08 -0.4675383 0.094 0.150 2.067353e-04       0
PDLIM3          4.862932e-08  0.9354622 0.427 0.217 2.209230e-04       0
AL138966.2      4.937906e-08 -0.4469946 0.109 0.136 2.243291e-04       0
EIF1            4.945476e-08 -0.9259953 0.400 0.556 2.246730e-04       0
SEPT9           4.955566e-08 -0.7797336 0.282 0.425 2.251314e-04       0
RNF207          4.958063e-08 -0.4066246 0.132 0.161 2.252448e-04       0
YTHDF3-AS1      5.145946e-08 -0.3428140 0.155 0.283 2.337803e-04       0
KIF26B          5.169588e-08 -0.4559697 0.256 0.325 2.348544e-04       0
PKP2            5.395300e-08 -0.5886255 0.304 0.411 2.451085e-04       0
ABHD14A-ACY1    5.863798e-08  0.2902197 0.081 0.056 2.663923e-04       0
UBASH3B         6.234522e-08 -0.4652506 0.293 0.408 2.832343e-04       0
PHACTR2         6.253487e-08 -0.3964184 0.418 0.542 2.840959e-04       0
P4HA2           6.381328e-08  0.2743874 0.469 0.356 2.899037e-04       0
PIM1            6.420098e-08 -0.7379580 0.209 0.361 2.916651e-04       0
GADD45A         6.433098e-08  0.7370293 0.458 0.358 2.922557e-04       0
RASSF4          6.698564e-08 -0.5419967 0.191 0.297 3.043158e-04       0
PLPP3           6.862861e-08 -0.7263130 0.163 0.306 3.117798e-04       0
MMP11           7.034501e-08 -0.2506026 0.129 0.192 3.195774e-04       0
DUSP1           7.364664e-08 -0.6543327 0.285 0.422 3.345767e-04       0
PRDX3           7.383233e-08 -0.5854424 0.432 0.553 3.354203e-04       0
TCEAL4          7.401822e-08 -0.6659459 0.378 0.508 3.362648e-04       0
RGS20           7.709438e-08 -0.3938233 0.122 0.208 3.502398e-04       0
FAH             7.925718e-08 -0.7762159 0.280 0.422 3.600654e-04       0
NEK5            8.238167e-08 -0.3266412 0.216 0.208 3.742599e-04       0
DIO3OS          8.387862e-08 -0.6335237 0.112 0.250 3.810606e-04       0
SPECC1L         8.748879e-08 -0.6603392 0.163 0.269 3.974616e-04       0
APOC1           8.814556e-08 -0.7705191 0.165 0.336 4.004453e-04       0
HOXB-AS3        8.832515e-08 -0.3748209 0.172 0.239 4.012612e-04       0
AC097359.2      9.015379e-08 -0.5806774 0.151 0.253 4.095687e-04       0
UGP2            9.416395e-08  0.6057209 0.459 0.339 4.277868e-04       0
IFITM3          1.003754e-07 -0.7620120 0.378 0.561 4.560055e-04       0
TMEM178A        1.013291e-07  0.2809106 0.200 0.239 4.603381e-04       0
GNAI2           1.034744e-07  0.5714521 0.452 0.306 4.700842e-04       0
ZNF653          1.130531e-07 -0.3784168 0.078 0.114 5.136002e-04       0
MANSC1          1.132777e-07 -0.2779752 0.127 0.217 5.146206e-04       0
AL117379.1      1.172830e-07 -0.3781983 0.107 0.181 5.328167e-04       0
LENG8           1.194551e-07 -0.6512156 0.412 0.558 5.426846e-04       0
FADS3           1.202272e-07 -0.4843384 0.338 0.486 5.461920e-04       0
AL135905.1      1.217222e-07 -0.2519967 0.101 0.161 5.529839e-04       0
HGH1            1.220887e-07 -0.3911556 0.189 0.289 5.546490e-04       0
AC011978.2      1.225733e-07 -0.2732018 0.123 0.192 5.568507e-04       0
ODAPH           1.229402e-07  1.0009642 0.377 0.175 5.585172e-04       0
RBMS3           1.247168e-07 -0.4970710 0.416 0.556 5.665882e-04       0
SLC3A2          1.272732e-07 -0.6463026 0.396 0.528 5.782021e-04       0
CAV1            1.296846e-07  0.5922713 0.448 0.289 5.891573e-04       0
SLPI            1.307137e-07 -0.9190358 0.192 0.336 5.938325e-04       0
BATF3           1.324068e-07 -0.3487195 0.145 0.156 6.015243e-04       0
CD55            1.466402e-07  1.1821371 0.448 0.275 6.661865e-04       0
SDHAF3          1.474404e-07 -0.7961837 0.299 0.394 6.698215e-04       0
BCO2            1.498534e-07 -0.9554325 0.149 0.275 6.807838e-04       0
ANKRD28         1.510397e-07 -0.3074713 0.253 0.367 6.861733e-04       0
C9orf47         1.586505e-07  0.3871547 0.095 0.100 7.207493e-04       0
TM6SF2          1.648421e-07 -0.2788887 0.167 0.211 7.488776e-04       0
MATN2           1.686871e-07 -0.3065727 0.390 0.536 7.663456e-04       0
KCNC4           1.760317e-07 -0.4197616 0.274 0.311 7.997120e-04       0
FAM117B         1.805682e-07 -0.6312868 0.329 0.481 8.203212e-04       0
SOCS1           1.836849e-07 -0.3807981 0.237 0.317 8.344805e-04       0
NEDD4           1.856707e-07 -0.5789064 0.318 0.447 8.435018e-04       0
FAM171A2        1.913729e-07  0.3267338 0.153 0.131 8.694070e-04       0
SLC22A3         1.935370e-07 -0.8825259 0.302 0.433 8.792387e-04       0
CTSZ            2.056996e-07  0.6636752 0.461 0.328 9.344934e-04       0
CCDC183         2.323857e-07 -0.2804348 0.127 0.233 1.055728e-03       0
MAN1A1          2.378572e-07 -0.2571286 0.389 0.536 1.080585e-03       0
PRDX6           2.419252e-07  0.5191756 0.498 0.328 1.099066e-03       0
TANGO6          2.555205e-07 -0.3230868 0.129 0.186 1.160829e-03       0
ASNS            2.590010e-07 -0.6078699 0.225 0.269 1.176641e-03       0
CENPE           2.650770e-07 -0.3384188 0.101 0.139 1.204245e-03       0
FEZ1            2.799243e-07 -0.3018051 0.158 0.222 1.271696e-03       0
MGP             2.904081e-07 -0.4269847 0.285 0.394 1.319324e-03       0
CCNA2           2.924603e-07 -0.5433717 0.136 0.175 1.328647e-03       0
BOD1L1          2.978636e-07 -0.5944460 0.400 0.517 1.353194e-03       0
HLA-B           2.979955e-07 -0.6868116 0.352 0.486 1.353793e-03       0
RGS2            3.003082e-07 -0.5166421 0.314 0.392 1.364300e-03       0
PPP3CA          3.055495e-07  0.6898709 0.466 0.353 1.388111e-03       0
CTGF            3.111740e-07  0.5292712 0.496 0.361 1.413664e-03       0
TRIM65          3.216526e-07 -0.5800115 0.164 0.256 1.461268e-03       0
TTLL7           3.316756e-07 -0.5272296 0.174 0.286 1.506802e-03       0
SERINC5         3.358324e-07 -0.4718714 0.405 0.511 1.525687e-03       0
SRPX2           3.376057e-07  0.3546540 0.454 0.308 1.533742e-03       0
ZC3H12C         3.408522e-07 -0.9150006 0.089 0.225 1.548492e-03       0
ZFAS1           3.487719e-07 -1.0419055 0.395 0.503 1.584471e-03       0
NAP1L1          3.718314e-07 -0.8596157 0.388 0.497 1.689230e-03       0
ZNF136          3.748593e-07 -0.4708551 0.149 0.225 1.702986e-03       0
CANX            3.916664e-07 -0.5993736 0.418 0.564 1.779341e-03       0
TMED9           4.014849e-07  0.2883782 0.476 0.333 1.823946e-03       0
CSPG4           4.057046e-07 -0.3300437 0.181 0.214 1.843116e-03       0
HAUS4           4.148751e-07 -0.7106828 0.127 0.183 1.884778e-03       0
UBC             4.616251e-07 -0.3884187 0.401 0.550 2.097163e-03       0
F3              4.680074e-07 -0.6109515 0.400 0.514 2.126158e-03       0
NKAIN4          4.713402e-07  0.5285050 0.479 0.269 2.141299e-03       0
XPOT            4.855989e-07 -0.4333847 0.396 0.506 2.206076e-03       0
SPSB1           4.913710e-07 -0.4234208 0.396 0.533 2.232299e-03       0
PRAC1           4.918354e-07 -0.3352443 0.198 0.311 2.234408e-03       0
MSX2            4.979141e-07 -0.6488109 0.224 0.300 2.262024e-03       0
ABTB2           5.045413e-07 -0.3593482 0.171 0.231 2.292131e-03       0
REXO5           5.182955e-07 -0.2804934 0.078 0.156 2.354617e-03       0
TNFAIP1         5.251850e-07  0.2618370 0.488 0.336 2.385915e-03       0
MAML3           5.341711e-07 -0.2989349 0.219 0.239 2.426739e-03       0
BLOC1S5         5.549367e-07 -0.4337233 0.195 0.256 2.521078e-03       0
MBNL2           5.662743e-07 -0.6458547 0.439 0.558 2.572584e-03       0
NDRG2           5.675897e-07 -0.4684380 0.132 0.164 2.578560e-03       0
UBOX5           5.702722e-07 -0.3705556 0.116 0.200 2.590746e-03       0
TMED3           5.724300e-07  0.3757048 0.480 0.333 2.600550e-03       0
HAND2-AS1       5.940626e-07 -0.7993319 0.403 0.522 2.698826e-03       0
RPP38           6.058907e-07 -0.5799551 0.307 0.406 2.752561e-03       0
BEX3            6.115852e-07  0.5900242 0.447 0.311 2.778432e-03       0
ISPD            6.499879e-07 -0.4474056 0.148 0.208 2.952895e-03       0
ECE1            6.557966e-07 -0.6513306 0.304 0.394 2.979284e-03       0
FILIP1L         6.659923e-07  0.6282044 0.459 0.322 3.025603e-03       0
ISYNA1          6.965329e-07 -0.4187121 0.229 0.319 3.164349e-03       0
ADAM23          7.033913e-07 -0.4358833 0.168 0.264 3.195507e-03       0
BTBD3           7.521996e-07 -0.8442402 0.203 0.303 3.417243e-03       0
PTPRJ           7.528987e-07 -0.3743212 0.139 0.150 3.420419e-03       0
CPEB3           7.627741e-07 -0.2986632 0.154 0.194 3.465283e-03       0
AL031777.3      7.968109e-07 -0.3120137 0.144 0.189 3.619912e-03       0
SLC26A2         8.117669e-07 -0.4690451 0.406 0.550 3.687857e-03       0
DERL3           8.177766e-07  0.2698492 0.180 0.206 3.715159e-03       0
ARSE            8.443866e-07  0.4421646 0.172 0.061 3.836048e-03       0
ZNF610          8.581970e-07  0.2989508 0.104 0.083 3.898789e-03       0
GAS7            8.609915e-07 -0.4219237 0.327 0.425 3.911484e-03       0
CHD1L           9.088731e-07 -0.2854382 0.216 0.292 4.129011e-03       0
BRCC3           9.139257e-07 -0.4522762 0.203 0.297 4.151965e-03       0
HIF3A           9.421962e-07 -0.8428513 0.197 0.311 4.280397e-03       0
EGFR            1.058141e-06 -0.3801921 0.329 0.442 4.807136e-03       0
LRRC49          1.068880e-06 -0.5806466 0.109 0.189 4.855921e-03       0
FZD2            1.094213e-06 -0.4008569 0.278 0.394 4.971009e-03       0
OCIAD2          1.133018e-06  0.9659995 0.269 0.094 5.147299e-03       0
TUT4            1.155565e-06 -0.5147517 0.330 0.464 5.249732e-03       0
NUP133          1.171009e-06 -0.2825103 0.253 0.319 5.319894e-03       0
C5orf38         1.172620e-06 -0.3794662 0.261 0.372 5.327212e-03       0
TIMP4           1.218525e-06 -1.2899856 0.139 0.272 5.535759e-03       0
CBR3            1.242716e-06 -0.4657677 0.121 0.217 5.645657e-03       0
FGD4            1.295406e-06 -0.8751279 0.148 0.250 5.885029e-03       0
GNPDA1          1.297191e-06 -0.6191342 0.302 0.378 5.893140e-03       0
ZNF345          1.330086e-06 -0.4431780 0.117 0.164 6.042582e-03       0
PPL             1.341668e-06 -0.3575244 0.165 0.233 6.095199e-03       0
RPS27           1.355185e-06 -0.6645521 0.377 0.569 6.156604e-03       0
AP000695.2      1.362551e-06 -0.2755160 0.167 0.183 6.190067e-03       0
GPBAR1          1.368185e-06 -0.3216206 0.141 0.244 6.215666e-03       0
SPOUT1          1.418828e-06 -0.4288324 0.184 0.219 6.445738e-03       0
PRICKLE1        1.433788e-06  0.6859440 0.464 0.289 6.513698e-03       0
METTL7A         1.505126e-06 -0.5485745 0.356 0.478 6.837786e-03       0
SNHG12          1.527110e-06 -0.8021168 0.236 0.356 6.937662e-03       0
UTP23           1.541713e-06 -0.3114476 0.247 0.344 7.004003e-03       0
PTMS            1.566342e-06  0.2559813 0.489 0.367 7.115893e-03       0
SACS            1.567761e-06 -0.3681035 0.279 0.361 7.122336e-03       0
PMP22           1.598558e-06 -0.8484324 0.386 0.536 7.262247e-03       0
KCTD2           1.613797e-06 -0.3793762 0.158 0.225 7.331481e-03       0
C1QTNF6         1.633588e-06 -0.4259244 0.164 0.228 7.421391e-03       0
AMMECR1         1.635112e-06 -0.3388597 0.410 0.531 7.428312e-03       0
DEPP1           1.648462e-06 -0.3711451 0.135 0.250 7.488961e-03       0
IFFO1           1.680750e-06 -0.3175618 0.190 0.208 7.635646e-03       0
GPC4            1.681368e-06  0.3822430 0.438 0.264 7.638456e-03       0
PBX1            1.751217e-06 -0.3013629 0.306 0.369 7.955780e-03       0
FST             1.803822e-06  0.4101324 0.459 0.344 8.194762e-03       0
TSLP            2.022049e-06 -0.3927709 0.122 0.217 9.186171e-03       0
SLC37A3         2.109029e-06 -0.3927906 0.249 0.308 9.581319e-03       0
APBB3           2.141343e-06 -0.2965987 0.158 0.181 9.728120e-03       0
SARDH           2.143259e-06  0.3353065 0.186 0.211 9.736824e-03       0
ZNF496          2.156692e-06 -0.4853035 0.176 0.225 9.797852e-03       0
ZDHHC14         2.161530e-06 -0.6097303 0.144 0.217 9.819832e-03       0
ZWINT           2.389776e-06 -0.3289395 0.186 0.217 1.085675e-02       0
TRIB3           2.467569e-06 -0.8041461 0.185 0.319 1.121016e-02       0
FOXF1           2.554597e-06 -0.4927675 0.337 0.428 1.160553e-02       0
MYO10           2.689482e-06 -0.6062852 0.370 0.456 1.221832e-02       0
ZNF43           2.729086e-06 -0.2779168 0.180 0.228 1.239824e-02       0
ITPKA           2.732711e-06 -0.5110587 0.093 0.189 1.241471e-02       0
RNF6            2.775386e-06 -0.3562961 0.295 0.386 1.260858e-02       0
NORAD           2.782766e-06 -0.4259992 0.454 0.558 1.264211e-02       0
FAM129A         2.817431e-06 -0.6933312 0.168 0.292 1.279959e-02       0
PER1            2.894454e-06 -0.3368560 0.173 0.272 1.314951e-02       0
ALKAL2          2.983983e-06 -0.5428433 0.167 0.256 1.355623e-02       0
GPSM2           2.994465e-06 -0.5202944 0.138 0.242 1.360385e-02       0
ERRFI1          3.057246e-06 -0.5025828 0.269 0.361 1.388907e-02       0
ZBTB16          3.059859e-06 -1.1469530 0.157 0.303 1.390094e-02       0
S1PR3           3.085688e-06 -0.4175755 0.235 0.278 1.401828e-02       0
HSPB1           3.123905e-06 -0.5169272 0.390 0.514 1.419190e-02       0
ID3             3.150095e-06  0.7887416 0.445 0.275 1.431088e-02       0
ADAMTS9         3.378327e-06 -0.4401060 0.265 0.339 1.534774e-02       0
SLC24A1         3.549966e-06 -0.5864396 0.101 0.186 1.612749e-02       0
TMEM91          3.573380e-06 -0.7733510 0.212 0.331 1.623387e-02       0
HIF1A           3.603237e-06 -0.6055190 0.408 0.553 1.636950e-02       0
AHCTF1          3.614298e-06 -0.5443895 0.236 0.322 1.641975e-02       0
CPD             3.759436e-06 -0.7665827 0.425 0.533 1.707912e-02       0
DSC3            3.777551e-06 -0.3571023 0.256 0.347 1.716141e-02       0
LHFPL6          3.814035e-06 -0.3234842 0.312 0.411 1.732716e-02       0
CRYAB           3.921988e-06 -0.4217507 0.295 0.444 1.781759e-02       0
KIF22           3.970304e-06 -0.6421704 0.187 0.236 1.803709e-02       0
TTC21B          3.991120e-06 -0.4700966 0.114 0.186 1.813166e-02       0
PGAP1           4.017219e-06 -0.3556382 0.122 0.217 1.825023e-02       0
SLC46A3         4.116538e-06 -0.7702292 0.409 0.533 1.870143e-02       0
PRNP            4.147125e-06 -0.3274468 0.335 0.458 1.884039e-02       0
TMEM98          4.170658e-06  0.6445943 0.463 0.344 1.894730e-02       0
GNG11           4.338981e-06  0.6063122 0.419 0.306 1.971199e-02       0
LZTS1           4.367374e-06  0.3109636 0.444 0.242 1.984098e-02       0
PLEKHG3         4.382483e-06  0.3935518 0.147 0.164 1.990962e-02       0
SEPSECS         4.388242e-06 -0.2759886 0.128 0.183 1.993578e-02       0
FOXO3           4.568995e-06 -0.6434726 0.286 0.403 2.075694e-02       0
SOX11           4.723586e-06  1.2789348 0.375 0.181 2.145925e-02       0
C15orf39        4.736278e-06 -0.5071150 0.173 0.300 2.151691e-02       0
PKIA            4.875575e-06 -0.8411467 0.103 0.164 2.214974e-02       0
HAND2           4.903059e-06 -0.5547893 0.408 0.525 2.227460e-02       0
STAG1           4.997167e-06 -0.4841040 0.284 0.356 2.270213e-02       0
KAT6B           5.031806e-06 -0.8056451 0.285 0.347 2.285949e-02       0
B3GNT5          5.060082e-06 -0.4003586 0.153 0.281 2.298795e-02       0
IKZF2           5.084204e-06 -0.5225467 0.098 0.158 2.309754e-02       0
ALG9            5.095203e-06 -0.2624364 0.190 0.253 2.314751e-02       0
YARS            5.117239e-06 -0.4110244 0.359 0.472 2.324761e-02       0
FGFRL1          5.157108e-06  0.4797629 0.304 0.183 2.342874e-02       0
GCH1            5.233108e-06 -0.2799178 0.198 0.272 2.377401e-02       0
HIGD1A          5.280878e-06 -0.4387142 0.395 0.525 2.399103e-02       0
FAM89A          5.405137e-06 -0.6393502 0.406 0.511 2.455554e-02       0
KIF11           5.684014e-06 -0.5408489 0.117 0.208 2.582248e-02       0
PHTF2           5.760215e-06  0.3243310 0.470 0.353 2.616866e-02       0
HS6ST2          5.920578e-06  0.3237253 0.472 0.297 2.689719e-02       0
GJA1            5.961312e-06 -0.3175427 0.397 0.494 2.708224e-02       0
ANAPC16         5.966451e-06 -0.7010462 0.416 0.550 2.710558e-02       0
IFNGR1          6.195629e-06 -0.3612567 0.291 0.400 2.814674e-02       0
PCK2            6.397568e-06 -0.5134992 0.203 0.328 2.906415e-02       0
BASP1           6.446980e-06  0.3025979 0.516 0.369 2.928863e-02       0
CLSPN           6.756840e-06 -0.4101739 0.135 0.156 3.069632e-02       0
RASAL2-AS1      6.803110e-06 -0.3644172 0.069 0.131 3.090653e-02       0
UCHL1           6.814671e-06  0.3044069 0.469 0.350 3.095905e-02       0
CILP2           6.958305e-06 -0.7129629 0.199 0.275 3.161158e-02       0
AC132192.2      7.083974e-06 -0.5379889 0.087 0.178 3.218249e-02       0
OSGIN1          7.105094e-06 -0.7019619 0.083 0.158 3.227844e-02       0
EXOSC9          7.135410e-06 -0.4184948 0.216 0.242 3.241617e-02       0
GFPT1           7.165830e-06 -0.7868970 0.434 0.558 3.255437e-02       0
TMEM160         7.254728e-06  0.2945733 0.485 0.364 3.295823e-02       0
ZNF772          7.413558e-06 -0.5745296 0.132 0.158 3.367979e-02       0
AC069544.1      7.451460e-06 -0.4549696 0.104 0.194 3.385198e-02       0
CNOT6L          7.913886e-06 -0.6878923 0.223 0.361 3.595278e-02       0
GATAD2B         7.934050e-06 -0.3636008 0.254 0.306 3.604439e-02       0
ZNF816          8.297781e-06  0.2747087 0.120 0.139 3.769682e-02       0
TUBB6           8.297870e-06  0.5271684 0.442 0.339 3.769722e-02       0
CARD8-AS1       8.378682e-06 -0.3873769 0.149 0.228 3.806435e-02       0
CYCS            8.514406e-06  0.4880011 0.477 0.375 3.868095e-02       0
MRAS            8.557629e-06  0.4040865 0.467 0.333 3.887731e-02       0
FIBIN           8.636811e-06  0.6689029 0.465 0.353 3.923703e-02       0
MT1X            9.031559e-06 -0.6921064 0.346 0.461 4.103037e-02       0
DNAJB2          9.073897e-06 -0.2651734 0.219 0.289 4.122271e-02       0
ZFPM2           9.142885e-06 -0.4242968 0.285 0.333 4.153613e-02       0
RNF112          9.173714e-06 -0.4199622 0.165 0.214 4.167618e-02       0
PHF3            9.181438e-06 -0.4746350 0.344 0.450 4.171127e-02       0
AC012615.1      9.587360e-06  0.3934721 0.196 0.117 4.355538e-02       0
ODC1            9.848547e-06  0.4587116 0.476 0.364 4.474195e-02       0
SPOCK1          1.030863e-05  0.5997898 0.418 0.311 4.683210e-02       0
CLU             1.049339e-05  0.7372506 0.409 0.308 4.767147e-02       0
KIAA0232        1.064600e-05 -0.3181304 0.254 0.311 4.836478e-02       0
APOE            1.107660e-05 -0.6870964 0.229 0.283 5.032101e-02       0
PTPRA           1.111840e-05 -0.5406674 0.321 0.400 5.051090e-02       0
CDC42EP3        1.115551e-05  0.5866122 0.440 0.311 5.067949e-02       0
MAN2B1          1.153844e-05 -0.4783180 0.116 0.203 5.241913e-02       0
SLC5A6          1.166889e-05 -0.3176667 0.335 0.406 5.301178e-02       0
ICE1            1.200359e-05  0.8497553 0.402 0.267 5.453229e-02       0
SPINT1          1.218448e-05  0.6315005 0.329 0.119 5.535408e-02       0
CLK1            1.242970e-05 -0.3023399 0.287 0.383 5.646812e-02       0
SCRG1           1.263238e-05 -0.5449227 0.203 0.286 5.738892e-02       0
KREMEN1         1.332002e-05 -0.2768261 0.174 0.186 6.051285e-02       0
KCNG1           1.343058e-05 -0.7377681 0.172 0.272 6.101514e-02       0
DYNLL2          1.346967e-05 -0.5089627 0.280 0.378 6.119272e-02       0
PDLIM2          1.354226e-05  0.2629076 0.460 0.344 6.152248e-02       0
DCBLD1          1.360395e-05  0.7463835 0.415 0.267 6.180274e-02       0
KLHDC10         1.373384e-05 -0.5428580 0.382 0.461 6.239282e-02       0
ARHGAP26        1.410087e-05 -0.5875217 0.158 0.211 6.406025e-02       0
FLCN            1.425354e-05 -0.5667331 0.202 0.275 6.475381e-02       0
FRZB            1.510554e-05 -0.3903612 0.236 0.319 6.862445e-02       0
ADARB1          1.521157e-05 -0.5784835 0.311 0.408 6.910618e-02       0
MAP3K7CL        1.548818e-05  1.4410923 0.336 0.136 7.036282e-02       0
LAMC3           1.562966e-05 -0.3136067 0.141 0.231 7.100555e-02       0
DEK             1.580386e-05 -0.4457460 0.351 0.444 7.179695e-02       0
RASA3           1.594715e-05 -0.4721423 0.206 0.275 7.244791e-02       0
AC103740.2      1.601254e-05  0.8940852 0.120 0.050 7.274498e-02       0
CTNNAL1         1.614455e-05 -0.2908562 0.323 0.406 7.334467e-02       0
ARPC5           1.616442e-05  0.3647737 0.498 0.386 7.343498e-02       0
PYCARD          1.619092e-05 -0.3913225 0.104 0.175 7.355537e-02       0
UBXN4           1.647874e-05 -0.2942552 0.397 0.525 7.486290e-02       0
DACT3           1.651226e-05  0.6499998 0.133 0.100 7.501518e-02       0
POLR2J3.1       1.678457e-05 -0.3486038 0.225 0.294 7.625231e-02       0
THRA            1.748553e-05 -0.7556765 0.216 0.297 7.943678e-02       0
C1RL            1.781695e-05 -0.7701545 0.122 0.211 8.094239e-02       0
MLEC            1.813226e-05  0.4186173 0.469 0.356 8.237484e-02       0
NAA25           1.834784e-05 -0.2673623 0.227 0.300 8.335425e-02       0
SURF4           1.855835e-05 -0.3686822 0.414 0.528 8.431057e-02       0
AC104986.2      1.870225e-05 -0.3814447 0.122 0.189 8.496434e-02       0
SLC19A2         1.873296e-05 -0.5740254 0.342 0.414 8.510385e-02       0
DDR1            1.941562e-05 -0.4198787 0.199 0.222 8.820515e-02       0
MT-CYB          2.010562e-05 -0.5789217 0.383 0.517 9.133984e-02       0
NQO2            2.012186e-05 -0.5572435 0.274 0.361 9.141360e-02       0
TNFRSF1A        2.032702e-05 -0.3296558 0.422 0.550 9.234565e-02       0
PRKD1           2.057554e-05 -0.4424612 0.148 0.225 9.347469e-02       0
ENDOD1          2.064255e-05 -0.3405122 0.428 0.503 9.377912e-02       0
SIX4            2.097973e-05  0.2606523 0.183 0.206 9.531090e-02       0
AC096733.2      2.142565e-05  0.3213809 0.114 0.117 9.733671e-02       0
CDC37L1         2.188110e-05 -0.2583380 0.342 0.431 9.940585e-02       0
METTL27         2.215681e-05 -0.2689355 0.104 0.150 1.006584e-01       0
PAPPA           2.223750e-05 -0.4394923 0.397 0.450 1.010250e-01       0
KCNS3           2.254474e-05  0.2753078 0.179 0.181 1.024207e-01       0
TFPI            2.257207e-05 -0.6243367 0.218 0.339 1.025449e-01       0
FHL1            2.270868e-05  0.5326585 0.437 0.292 1.031655e-01       0
DDAH1           2.383390e-05  0.6024002 0.426 0.250 1.082774e-01       0
SOCS2           2.413296e-05 -1.0702312 0.103 0.242 1.096360e-01       0
RTN3            2.446520e-05 -0.3027312 0.356 0.436 1.111454e-01       0
ALDH3A2         2.455381e-05 -0.3900553 0.186 0.231 1.115480e-01       0
SPTBN1          2.467252e-05 -0.4629930 0.419 0.542 1.120872e-01       0
XPC             2.468242e-05 -0.5205656 0.330 0.422 1.121323e-01       0
PLEKHA4         2.473203e-05 -0.6106589 0.084 0.142 1.123576e-01       0
MYDGF           2.477182e-05  0.5538885 0.454 0.342 1.125384e-01       0
EIF1B           2.518293e-05 -1.0352638 0.420 0.519 1.144061e-01       0
SSR4            2.552885e-05  0.2524702 0.483 0.369 1.159776e-01       0
TNKS            2.575504e-05 -0.4229964 0.363 0.400 1.170051e-01       0
COTL1           2.582743e-05  0.2515376 0.458 0.361 1.173340e-01       0
C1S             2.596232e-05 -0.7929206 0.240 0.378 1.179468e-01       0
ANK3            2.652997e-05 -0.2748875 0.229 0.306 1.205257e-01       0
SIPA1L1         2.661508e-05  0.4208553 0.476 0.342 1.209123e-01       0
AP002884.1      2.700106e-05 -0.7524793 0.218 0.314 1.226658e-01       0
SLCO4A1         2.795647e-05 -0.5619595 0.208 0.275 1.270063e-01       0
FAM53B          2.814671e-05 -0.3084172 0.106 0.181 1.278705e-01       0
UHMK1           2.831587e-05 -0.4346006 0.421 0.508 1.286390e-01       0
TMEM158         2.849658e-05  0.2763604 0.184 0.250 1.294600e-01       0
PIK3R1          2.859968e-05 -0.5760110 0.206 0.314 1.299283e-01       0
SNAI2           2.870252e-05 -0.3580043 0.402 0.511 1.303956e-01       0
NPTXR           2.969122e-05 -0.5248905 0.238 0.317 1.348872e-01       0
GDF6            2.999991e-05  0.4687976 0.381 0.219 1.362896e-01       0
TPGS2           3.007178e-05  0.4033784 0.485 0.364 1.366161e-01       0
ANXA5           3.114232e-05  0.3827957 0.488 0.364 1.414795e-01       0
HLA-DPB1        3.212135e-05 -0.3303921 0.187 0.292 1.459273e-01       0
HSPB7           3.251854e-05 -0.4199052 0.165 0.272 1.477317e-01       0
STXBP5L         3.290678e-05  0.3234230 0.168 0.144 1.494955e-01       0
MT-ATP6         3.353862e-05 -0.3924900 0.388 0.492 1.523660e-01       0
ALCAM           3.368494e-05 -0.6309682 0.429 0.536 1.530307e-01       0
FMO5            3.393855e-05 -0.4347695 0.136 0.236 1.541828e-01       0
CNPY2           3.464380e-05  0.3452226 0.447 0.344 1.573868e-01       0
PTPN11          3.476718e-05 -0.6037537 0.390 0.458 1.579473e-01       0
CYB561D1        3.546008e-05 -0.4107246 0.119 0.144 1.610951e-01       0
SCG2            3.564771e-05 -0.5142794 0.288 0.386 1.619476e-01       0
ARHGAP29        3.857767e-05 -0.5024650 0.263 0.333 1.752584e-01       0
TMX4            3.865354e-05 -0.5194239 0.318 0.406 1.756030e-01       0
KIF9            3.979292e-05 -0.3244807 0.254 0.294 1.807792e-01       0
SDCBP           4.128623e-05 -0.3060622 0.450 0.533 1.875634e-01       0
DDX60           4.148062e-05 -0.3995878 0.070 0.183 1.884465e-01       0
KCNK6           4.288288e-05 -0.5521164 0.394 0.514 1.948169e-01       0
ADGRL4          4.291628e-05 -0.3040731 0.141 0.172 1.949687e-01       0
CHCHD10         4.391664e-05 -0.2551502 0.300 0.389 1.995133e-01       0
SMARCE1         4.419180e-05 -0.7196531 0.282 0.342 2.007633e-01       0
COL14A1         4.427736e-05 -0.3627533 0.159 0.208 2.011521e-01       0
MATN3           4.576903e-05 -0.6675567 0.332 0.367 2.079287e-01       0
ATP2B4          4.708879e-05 -0.4438928 0.321 0.400 2.139244e-01       0
OSER1-DT        4.776987e-05 -0.5402128 0.185 0.267 2.170185e-01       0
CDCA3           4.819972e-05 -0.5317665 0.102 0.128 2.189713e-01       0
TACC1           4.833105e-05 -0.3997883 0.305 0.417 2.195679e-01       0
TBC1D12         4.981641e-05 -0.2628555 0.218 0.247 2.263160e-01       0
OCA2            5.020415e-05 -0.7490534 0.178 0.306 2.280775e-01       0
INMT            5.096833e-05 -0.3660929 0.229 0.275 2.315491e-01       0
SLC38A4         5.158397e-05 -0.3068882 0.216 0.214 2.343460e-01       0
MICALL2         5.291681e-05 -0.4387929 0.159 0.231 2.404011e-01       0
GPC6            5.306122e-05 -0.6644724 0.343 0.433 2.410571e-01       0
FHL2            5.363840e-05  0.3687579 0.459 0.331 2.436792e-01       0
ATOX1           5.380444e-05  0.3485610 0.480 0.392 2.444336e-01       0
CLEC2D          5.397088e-05 -0.4271917 0.206 0.256 2.451897e-01       0
AC090204.1      5.466281e-05  0.3344009 0.438 0.272 2.483332e-01       0
CRIP2           5.470455e-05  0.2915474 0.094 0.075 2.485228e-01       0
NT5E            5.628841e-05  0.7381823 0.488 0.336 2.557182e-01       0
UBE2B           5.685499e-05 -0.4592973 0.412 0.528 2.582922e-01       0
DCAF16          5.700839e-05 -0.2734983 0.230 0.256 2.589891e-01       0
CEP295          5.729445e-05  0.3650244 0.236 0.178 2.602887e-01       0
KCTD1           5.742689e-05 -0.6090577 0.189 0.242 2.608904e-01       0
LIPE-AS1        5.849671e-05  0.3031173 0.145 0.133 2.657505e-01       0
KRT10           5.940352e-05 -0.5800103 0.418 0.536 2.698702e-01       0
NT5DC1          5.954055e-05 -0.5048422 0.247 0.300 2.704927e-01       0
CENPS           5.983857e-05 -0.2702186 0.202 0.244 2.718466e-01       0
TWSG1           6.006887e-05 -0.3679636 0.372 0.478 2.728929e-01       0
GSDMD           6.090364e-05 -0.4367191 0.121 0.203 2.766853e-01       0
CDKN1A          6.313911e-05 -0.5830519 0.428 0.528 2.868410e-01       0
SLC9A7          6.326019e-05 -0.2755096 0.216 0.281 2.873910e-01       0
NID1            6.362454e-05 -0.3149046 0.288 0.378 2.890463e-01       0
NR2F1           6.570182e-05 -0.4624549 0.380 0.453 2.984833e-01       0
KIF20B          6.625605e-05 -0.3732384 0.153 0.186 3.010012e-01       0
SLC26A11        6.625626e-05 -0.2828939 0.174 0.269 3.010022e-01       0
RNF128          6.653423e-05 -0.5374089 0.151 0.208 3.022650e-01       0
ANP32B          7.115232e-05 -0.4781022 0.441 0.553 3.232450e-01       0
TRAPPC5         7.232493e-05 -0.2807643 0.135 0.181 3.285722e-01       0
C19orf33        7.282146e-05 -0.3176219 0.154 0.178 3.308279e-01       0
HSPA1A          7.293282e-05 -0.2808466 0.227 0.264 3.313338e-01       0
HIST1H4C        7.329385e-05 -0.3592581 0.337 0.383 3.329740e-01       0
TRHDE           7.357283e-05 -0.4026986 0.218 0.286 3.342414e-01       0
AGL             7.467035e-05 -0.3224635 0.276 0.322 3.392274e-01       0
NFKBIZ          7.558243e-05 -0.2651692 0.289 0.367 3.433710e-01       0
MIATNB          7.761260e-05 -0.4380619 0.145 0.186 3.525941e-01       0
NEAT1           7.851611e-05 -0.5843862 0.425 0.536 3.566987e-01       0
CYTOR           7.954732e-05  0.6738056 0.461 0.347 3.613835e-01       0
CARHSP1         8.115712e-05 -0.3327281 0.434 0.506 3.686968e-01       0
C6orf48         8.282884e-05 -0.5683769 0.435 0.533 3.762914e-01       0
LGALS3          8.501217e-05 -0.6488907 0.358 0.436 3.862103e-01       0
CD82            8.546177e-05 -0.4803184 0.248 0.369 3.882528e-01       0
H2AFJ           8.705331e-05 -0.5764862 0.384 0.511 3.954832e-01       0
CYP27A1         8.714997e-05 -0.4854310 0.110 0.225 3.959223e-01       0
DUSP16          9.220432e-05 -0.7575129 0.066 0.164 4.188842e-01       0
SLC7A11         9.227293e-05 -0.6146918 0.218 0.322 4.191959e-01       0
FGF1            9.286358e-05  1.1696749 0.325 0.117 4.218792e-01       0
PSME1           9.738730e-05 -0.5643393 0.280 0.381 4.424305e-01       0
TMEM229B        9.951935e-05 -0.5743509 0.191 0.231 4.521164e-01       0
BLVRB           1.005675e-04 -0.3826326 0.333 0.417 4.568783e-01       0
ZNF565          1.023086e-04 -0.4127763 0.107 0.164 4.647878e-01       0
SMIM3           1.033055e-04 -0.4509273 0.405 0.531 4.693169e-01       0
TEX2            1.037299e-04 -0.4397919 0.348 0.408 4.712448e-01       0
CFAP97          1.053656e-04 -0.5107327 0.331 0.422 4.786758e-01       0
ADGRV1          1.071446e-04 -0.4289114 0.135 0.197 4.867579e-01       0
HIST3H2A        1.073445e-04 -0.6002434 0.184 0.258 4.876663e-01       0
BNIP3           1.075844e-04 -0.4932936 0.413 0.478 4.887558e-01       0
GMPR            1.077043e-04 -0.5182115 0.185 0.239 4.893007e-01       0
KCND3           1.150183e-04  0.3065804 0.238 0.228 5.225282e-01       0
SLC26A6         1.151435e-04 -0.6919726 0.129 0.222 5.230967e-01       0
POPDC2          1.154443e-04 -0.6932590 0.152 0.236 5.244636e-01       0
ALDH2           1.172971e-04 -0.4188445 0.165 0.269 5.328806e-01       0
AC005476.2      1.173839e-04  0.4429849 0.138 0.103 5.332752e-01       0
ANKRD1          1.188517e-04  0.9307965 0.323 0.175 5.399431e-01       0
BLM             1.191346e-04 -0.2713357 0.154 0.189 5.412283e-01       0
HMMR            1.193536e-04 -0.4715249 0.135 0.225 5.422233e-01       0
HIP1            1.213118e-04 -0.5990341 0.186 0.278 5.511194e-01       0
PLD3            1.217610e-04 -0.5337100 0.428 0.525 5.531601e-01       0
RNF115          1.241205e-04 -0.4962074 0.380 0.456 5.638796e-01       0
ISG15           1.266164e-04  0.4189846 0.401 0.275 5.752184e-01       0
LINC01503       1.267562e-04 -0.4517177 0.326 0.386 5.758535e-01       0
FBLN5           1.267565e-04 -0.8518692 0.420 0.517 5.758548e-01       0
TGIF1           1.287798e-04 -0.3999190 0.365 0.436 5.850468e-01       0
CLIC6           1.289696e-04 -0.5175626 0.225 0.278 5.859087e-01       0
CTH             1.294451e-04 -0.3092450 0.219 0.244 5.880693e-01       0
KLHL5           1.305452e-04 -0.4805652 0.265 0.325 5.930668e-01       0
C3orf58         1.311707e-04 -0.3165666 0.246 0.303 5.959086e-01       0
AVPI1           1.323819e-04 -0.4506481 0.198 0.244 6.014111e-01       0
COMMD7          1.367295e-04 -0.4276440 0.343 0.408 6.211620e-01       0
KRT18           1.371299e-04  0.8458952 0.420 0.308 6.229811e-01       0
RAB29           1.377866e-04 -0.2680512 0.260 0.344 6.259647e-01       0
PLEKHA2         1.449806e-04 -0.4311340 0.204 0.300 6.586468e-01       0
NME4            1.460979e-04 -0.7984842 0.405 0.497 6.637226e-01       0
GTF2B           1.462047e-04 -0.3715262 0.320 0.356 6.642079e-01       0
AC016831.1      1.465790e-04  0.5910715 0.254 0.122 6.659083e-01       0
SOAT1           1.477612e-04 -0.5759637 0.187 0.283 6.712790e-01       0
TNKS1BP1        1.479778e-04 -0.4235141 0.278 0.367 6.722630e-01       0
MT1E            1.489483e-04 -0.3749156 0.286 0.406 6.766720e-01       0
STX12           1.490079e-04 -0.4133699 0.370 0.447 6.769427e-01       0
FMO4            1.516399e-04  0.5287689 0.103 0.089 6.889001e-01       0
PPIP5K1         1.549355e-04 -0.3266178 0.108 0.122 7.038721e-01       0
EGLN1           1.562957e-04 -0.2788879 0.240 0.308 7.100512e-01       0
AC007014.1      1.574369e-04  0.3989638 0.116 0.139 7.152358e-01       0
IGF2BP3         1.624369e-04 -0.4771410 0.274 0.356 7.379507e-01       0
PTPN9           1.652318e-04 -0.4407457 0.218 0.317 7.506481e-01       0
SLC38A10        1.675247e-04 -0.4689437 0.428 0.544 7.610649e-01       0
SNCAIP          1.676458e-04 -0.2782097 0.119 0.172 7.616151e-01       0
HAS2            1.689266e-04 -0.6038969 0.227 0.319 7.674336e-01       0
HEYL            1.718883e-04 -0.2719329 0.165 0.183 7.808888e-01       0
SLC25A53        1.750894e-04 -0.4088049 0.109 0.161 7.954313e-01       0
SLC39A2         1.783434e-04  0.5374706 0.120 0.089 8.102140e-01       0
TBX3            1.797039e-04 -0.4810243 0.343 0.422 8.163947e-01       0
C8orf33         1.809406e-04  0.6068750 0.448 0.347 8.220130e-01       0
RBPMS           1.891413e-04 -0.5409864 0.305 0.389 8.592689e-01       0
FAM111A-DT      1.946579e-04  0.6133076 0.204 0.058 8.843307e-01       0
Z93930.2        1.961310e-04 -0.3568417 0.152 0.211 8.910233e-01       0
LNX2            1.984011e-04 -0.3252869 0.174 0.228 9.013364e-01       0
WDR81           1.989692e-04 -0.6591322 0.180 0.258 9.039172e-01       0
DSEL            1.989714e-04 -0.7695132 0.312 0.383 9.039272e-01       0
ZSWIM3          1.990421e-04 -0.7417957 0.112 0.200 9.042482e-01       0
CCDC97          2.151171e-04 -0.4690538 0.180 0.222 9.772771e-01       0
CCDC9B          2.155809e-04 -0.3222708 0.202 0.250 9.793843e-01       0
CENPW           2.163508e-04  0.2957720 0.355 0.244 9.828817e-01       0
ITGB6           2.197709e-04  0.3997790 0.138 0.131 9.984191e-01       0
SEPT4           2.202461e-04 -0.5127631 0.256 0.361 1.000000e+00       0
CAP2            2.204819e-04  0.5243716 0.450 0.286 1.000000e+00       0
SLC5A12         2.214269e-04 -0.2775350 0.133 0.136 1.000000e+00       0
AC005332.6      2.215056e-04 -0.3361053 0.126 0.169 1.000000e+00       0
RASSF2          2.243641e-04  1.1108495 0.243 0.061 1.000000e+00       0
ABCD1           2.251659e-04 -0.2580989 0.193 0.239 1.000000e+00       0
GLB1L           2.266118e-04 -0.3673517 0.135 0.136 1.000000e+00       0
TES             2.269367e-04  0.2978909 0.467 0.369 1.000000e+00       0
AC124798.1      2.270174e-04 -0.5647415 0.184 0.236 1.000000e+00       0
WARS            2.416638e-04 -0.7439198 0.310 0.408 1.000000e+00       0
STRIP1          2.473819e-04 -0.2513546 0.218 0.256 1.000000e+00       0
FBN1            2.506425e-04 -0.7238926 0.441 0.511 1.000000e+00       0
CYFIP1          2.558349e-04 -0.2746510 0.350 0.397 1.000000e+00       0
CLDN12          2.588336e-04 -0.4575280 0.215 0.275 1.000000e+00       0
MAP3K4          2.614967e-04 -0.4501526 0.227 0.294 1.000000e+00       0
DOK5            2.641852e-04 -0.3040143 0.225 0.289 1.000000e+00       0
NUAK1           2.651184e-04  0.5231581 0.491 0.319 1.000000e+00       0
SAXO2           2.663357e-04 -0.4897860 0.093 0.172 1.000000e+00       0
WDR24           2.737522e-04 -0.3309396 0.181 0.222 1.000000e+00       0
SERPINE1        2.768308e-04 -0.4956264 0.372 0.464 1.000000e+00       0
MT-ND5          2.768505e-04 -0.4502116 0.397 0.492 1.000000e+00       0
BTG3            2.770451e-04 -0.4560638 0.369 0.439 1.000000e+00       0
BAG3            2.821533e-04 -0.6433301 0.437 0.544 1.000000e+00       0
CDKL5           2.858368e-04 -0.4448282 0.163 0.275 1.000000e+00       0
YWHAG           2.959334e-04  0.3111184 0.479 0.394 1.000000e+00       0
PGRMC1          2.991614e-04  0.2958232 0.515 0.386 1.000000e+00       0
CXCL14          3.019315e-04  0.6298917 0.367 0.233 1.000000e+00       0
STMN1           3.343495e-04  0.2649474 0.447 0.300 1.000000e+00       0
SELENOP         3.346945e-04 -0.5064018 0.228 0.297 1.000000e+00       0
TTC21A          3.366798e-04  0.4036018 0.128 0.083 1.000000e+00       0
OSBPL10         3.367977e-04 -0.4472740 0.400 0.478 1.000000e+00       0
FIGNL1          3.407933e-04 -0.3331799 0.138 0.192 1.000000e+00       0
HSD17B6         3.449575e-04  0.5592991 0.422 0.256 1.000000e+00       0
SQSTM1          3.550176e-04 -0.7332433 0.422 0.497 1.000000e+00       0
MIR4435-2HG     3.557558e-04  0.4912630 0.463 0.381 1.000000e+00       0
COL5A1          3.640924e-04 -1.0656805 0.391 0.494 1.000000e+00       0
ATL3            3.663631e-04 -0.4442278 0.353 0.433 1.000000e+00       0
AC110597.1      3.809226e-04 -0.2636433 0.135 0.097 1.000000e+00       0
MINCR           3.839469e-04 -0.5998127 0.180 0.236 1.000000e+00       0
BCAR3           3.860722e-04 -0.5329764 0.208 0.275 1.000000e+00       0
TRIM16L         3.863376e-04 -0.6375062 0.216 0.300 1.000000e+00       0
TKT             3.895412e-04 -0.6236066 0.428 0.533 1.000000e+00       0
MIR22HG         3.915557e-04 -0.5024081 0.374 0.456 1.000000e+00       0
BIN3            3.935712e-04 -0.3819703 0.170 0.225 1.000000e+00       0
FAM177A1        3.939861e-04 -0.6401492 0.414 0.489 1.000000e+00       0
LTBR            4.014996e-04 -0.2642938 0.167 0.233 1.000000e+00       0
TLCD2           4.037104e-04 -0.3684607 0.340 0.408 1.000000e+00       0
HECTD3          4.076024e-04 -0.4808800 0.198 0.253 1.000000e+00       0
RAD21           4.108334e-04 -0.6214888 0.384 0.469 1.000000e+00       0
IQGAP1          4.126678e-04 -0.3862247 0.413 0.517 1.000000e+00       0
GSTZ1           4.220980e-04 -0.3708753 0.299 0.317 1.000000e+00       0
CEP126          4.274737e-04 -0.5845536 0.189 0.250 1.000000e+00       0
MSC             4.302584e-04 -0.4483971 0.199 0.311 1.000000e+00       0
AIMP1           4.312892e-04 -0.4145630 0.401 0.489 1.000000e+00       0
ERICH5          4.326164e-04  0.3115554 0.209 0.175 1.000000e+00       0
GOLGA7          4.354334e-04  0.3784432 0.458 0.394 1.000000e+00       0
TTLL12          4.411171e-04 -0.3296203 0.218 0.289 1.000000e+00       0
AATF            4.497726e-04 -0.2574665 0.299 0.342 1.000000e+00       0
DET1            4.500693e-04 -0.2931740 0.157 0.192 1.000000e+00       0
RARB            4.502279e-04  0.4835187 0.158 0.142 1.000000e+00       0
CXCL2           4.618732e-04 -0.8636982 0.084 0.181 1.000000e+00       0
BIN1            4.625027e-04 -0.4433512 0.218 0.264 1.000000e+00       0
TBCE.1          4.749160e-04 -0.3217802 0.138 0.183 1.000000e+00       0
SMAD6           4.763763e-04 -0.3231686 0.229 0.250 1.000000e+00       0
TNFRSF12A       4.765381e-04  0.3562871 0.489 0.389 1.000000e+00       0
MAF             4.966530e-04  0.3231105 0.448 0.344 1.000000e+00       0
FADS1           5.034197e-04 -0.4922894 0.428 0.542 1.000000e+00       0
PRSS35          5.056111e-04  0.5769759 0.407 0.275 1.000000e+00       0
ANKRD6          5.111337e-04 -0.5423457 0.253 0.242 1.000000e+00       0
FAAH            5.113041e-04 -0.2854628 0.140 0.119 1.000000e+00       0
PHLDB1          5.145977e-04  0.3053640 0.419 0.292 1.000000e+00       0
BDH2            5.147716e-04  0.3797566 0.444 0.247 1.000000e+00       0
MANF            5.240597e-04  0.3885464 0.435 0.331 1.000000e+00       0
MBOAT2          5.326060e-04  0.6159436 0.414 0.297 1.000000e+00       0
MON2            5.380134e-04 -0.4294824 0.422 0.494 1.000000e+00       0
SBNO1           5.442034e-04 -0.5089409 0.320 0.403 1.000000e+00       0
CCL20           5.465850e-04  0.3350080 0.133 0.097 1.000000e+00       0
ECI1            5.499051e-04 -0.3421229 0.363 0.444 1.000000e+00       0
MAFG            5.526839e-04 -0.6394986 0.313 0.392 1.000000e+00       0
E2F6            5.773205e-04 -0.2719197 0.227 0.281 1.000000e+00       0
PDS5B           5.788696e-04  0.3579314 0.447 0.342 1.000000e+00       0
AC009159.3      5.796419e-04  0.3638941 0.104 0.086 1.000000e+00       0
CTNNB1          5.813923e-04  0.7693563 0.452 0.358 1.000000e+00       0
SNRPN           5.876552e-04  0.3084126 0.482 0.364 1.000000e+00       0
MAP3K7          5.959667e-04 -0.3535098 0.336 0.417 1.000000e+00       0
CD320           5.985616e-04  0.5114529 0.474 0.361 1.000000e+00       0
EDN1            6.027664e-04  0.3751840 0.343 0.139 1.000000e+00       0
IGDCC4          6.049934e-04 -0.2599034 0.380 0.458 1.000000e+00       0
ZNF468          6.088427e-04 -0.4319586 0.217 0.292 1.000000e+00       0
TAF1D           6.294810e-04 -0.5793408 0.429 0.531 1.000000e+00       0
FOXF2           6.301099e-04 -0.3867103 0.181 0.233 1.000000e+00       0
CEP290          6.385531e-04 -0.3446232 0.262 0.311 1.000000e+00       0
PODXL2          6.529327e-04  0.3363097 0.119 0.072 1.000000e+00       0
COL6A1          6.585531e-04 -0.4710994 0.399 0.464 1.000000e+00       0
RPL23           6.665080e-04  0.2903464 0.439 0.378 1.000000e+00       0
TMCO4           6.676131e-04 -0.4620556 0.129 0.211 1.000000e+00       0
GK5             6.713870e-04 -0.2843637 0.272 0.342 1.000000e+00       0
SLC35A3         6.783187e-04  0.4704678 0.399 0.264 1.000000e+00       0
UBXN2A          7.104608e-04 -0.5047163 0.285 0.322 1.000000e+00       0
SAT2            7.104614e-04 -0.3969415 0.429 0.506 1.000000e+00       0
INA             7.292171e-04  0.4340523 0.216 0.175 1.000000e+00       0
AC105942.1      7.340385e-04  0.2993781 0.231 0.139 1.000000e+00       0
EIF5A           7.362177e-04  0.2574325 0.463 0.383 1.000000e+00       0
PWAR6           7.367026e-04  0.5300924 0.381 0.233 1.000000e+00       0
DNAJC1          7.484331e-04 -0.5974597 0.393 0.506 1.000000e+00       0
ZSWIM6          7.585899e-04  0.6469664 0.420 0.331 1.000000e+00       0
USP1            7.625869e-04 -0.5680659 0.257 0.317 1.000000e+00       0
DBN1            7.630894e-04  0.4747378 0.526 0.392 1.000000e+00       0
AC064836.3      7.744452e-04  0.3138728 0.108 0.078 1.000000e+00       0
GSTP1           7.764821e-04 -0.2800622 0.476 0.386 1.000000e+00       0
SPTBN2          7.955465e-04 -0.5525513 0.383 0.464 1.000000e+00       0
AL355338.1      7.992017e-04  0.6300289 0.320 0.217 1.000000e+00       0
GALNT18         8.047139e-04  0.2748017 0.447 0.294 1.000000e+00       0
ENC1            8.092006e-04  0.3321192 0.426 0.264 1.000000e+00       0
DDX17           8.268473e-04 -0.3726971 0.453 0.511 1.000000e+00       0
DDX5            8.467793e-04 -0.3407725 0.427 0.522 1.000000e+00       0
SBF2-AS1        8.503673e-04 -0.3124865 0.363 0.428 1.000000e+00       0
LDLR            8.590068e-04 -0.6583812 0.343 0.419 1.000000e+00       0
AL590705.1      8.626393e-04 -0.3224093 0.168 0.222 1.000000e+00       0
STK35           8.981201e-04 -0.2890456 0.130 0.203 1.000000e+00       0
TXNRD1          9.086910e-04 -0.5846082 0.428 0.506 1.000000e+00       0
AP002761.3      9.125294e-04 -0.4451495 0.158 0.219 1.000000e+00       0
PLAU            9.322428e-04  0.4653121 0.160 0.161 1.000000e+00       0
PELI3           9.428505e-04 -0.5001399 0.120 0.128 1.000000e+00       0
NRG1            9.497959e-04  0.9184837 0.269 0.111 1.000000e+00       0
CDK5RAP2        9.678787e-04 -0.4304718 0.231 0.269 1.000000e+00       0
AHNAK           9.750904e-04 -0.4479344 0.405 0.508 1.000000e+00       0
CMSS1           9.874327e-04 -0.5731349 0.185 0.303 1.000000e+00       0
MSI2            9.941385e-04  0.4888719 0.367 0.264 1.000000e+00       0
ADAMTS1         9.960508e-04 -0.6126161 0.199 0.294 1.000000e+00       0
HMOX1           1.005084e-03 -0.4152105 0.197 0.256 1.000000e+00       0
LINC01504       1.010922e-03 -0.2686480 0.119 0.181 1.000000e+00       0
MC1R            1.012213e-03 -0.3387713 0.142 0.194 1.000000e+00       0
MIOS            1.016795e-03 -0.5638716 0.209 0.292 1.000000e+00       0
CHEK1           1.018430e-03 -0.4395315 0.198 0.211 1.000000e+00       0
XPO1            1.029957e-03 -0.3262056 0.380 0.453 1.000000e+00       0
MCUB            1.055737e-03  0.3618278 0.335 0.219 1.000000e+00       0
HTRA1           1.058936e-03  0.2871157 0.454 0.364 1.000000e+00       0
TTC39B          1.059469e-03 -0.2700387 0.189 0.242 1.000000e+00       0
RHOB            1.064235e-03 -0.3462387 0.412 0.486 1.000000e+00       0
AC128688.2      1.075892e-03  0.3656787 0.106 0.100 1.000000e+00       0
CPNE7           1.085223e-03 -0.5955824 0.134 0.228 1.000000e+00       0
SEPT11          1.112647e-03  0.2991583 0.490 0.397 1.000000e+00       0
NOL3            1.122284e-03 -0.3946472 0.316 0.389 1.000000e+00       0
FANCF           1.124064e-03 -0.4681377 0.153 0.214 1.000000e+00       0
ZNRF3           1.129467e-03 -0.4039807 0.100 0.178 1.000000e+00       0
STRADB          1.130185e-03 -0.5155728 0.200 0.275 1.000000e+00       0
HECW1           1.160127e-03  0.3726074 0.129 0.072 1.000000e+00       0
IMPA2           1.174244e-03 -0.4109893 0.186 0.261 1.000000e+00       0
GTF2A2          1.199151e-03  0.2937614 0.467 0.367 1.000000e+00       0
TMEM92-AS1      1.201817e-03 -0.3013218 0.133 0.200 1.000000e+00       0
FAM227A         1.219870e-03 -0.5317080 0.164 0.197 1.000000e+00       0
EPRS            1.221436e-03 -0.3877163 0.420 0.522 1.000000e+00       0
PCNX1           1.223763e-03 -0.3674106 0.358 0.419 1.000000e+00       0
FAM129B         1.230768e-03 -0.4261359 0.259 0.333 1.000000e+00       0
SEMA5A          1.236633e-03  0.4115805 0.412 0.350 1.000000e+00       0
AC116667.1      1.247649e-03  0.2917137 0.147 0.131 1.000000e+00       0
PDGFRL          1.282882e-03  0.3133424 0.319 0.203 1.000000e+00       0
VAMP3           1.283693e-03 -0.4850151 0.361 0.433 1.000000e+00       0
AC004803.1      1.312378e-03 -0.6467150 0.088 0.183 1.000000e+00       0
FAM83G          1.316519e-03 -0.5063382 0.157 0.172 1.000000e+00       0
FGF2            1.319021e-03 -0.4954456 0.275 0.350 1.000000e+00       0
MINDY2          1.344204e-03  0.3393477 0.472 0.392 1.000000e+00       0
SMOX            1.353972e-03 -0.3690587 0.243 0.311 1.000000e+00       0
TBCK            1.362949e-03 -0.3094189 0.195 0.256 1.000000e+00       0
TARSL2          1.383239e-03  0.4757001 0.465 0.361 1.000000e+00       0
AHCYL1          1.406454e-03 -0.2555384 0.419 0.508 1.000000e+00       0
MAP2K3          1.406895e-03  0.4024294 0.329 0.233 1.000000e+00       0
CAPG            1.433167e-03 -0.2716753 0.217 0.275 1.000000e+00       0
ARHGAP10        1.459433e-03 -0.3204008 0.306 0.356 1.000000e+00       0
ARSI            1.460789e-03  0.4177778 0.135 0.161 1.000000e+00       0
SGPP1           1.489397e-03 -0.4823700 0.227 0.289 1.000000e+00       0
MRPS30          1.500603e-03 -0.2956064 0.266 0.342 1.000000e+00       0
ADAM19          1.502475e-03  0.6153928 0.394 0.231 1.000000e+00       0
RBP1            1.513220e-03 -0.3130738 0.330 0.361 1.000000e+00       0
ZBTB1           1.526577e-03 -0.3234829 0.312 0.353 1.000000e+00       0
SMIM15          1.526578e-03  0.2854185 0.493 0.417 1.000000e+00       0
CLIP1           1.533255e-03 -0.3228355 0.340 0.411 1.000000e+00       0
MXRA5           1.540436e-03  0.2601685 0.444 0.342 1.000000e+00       0
UTP6            1.550066e-03 -0.4577160 0.307 0.381 1.000000e+00       0
ARHGEF26        1.563637e-03 -0.6394095 0.177 0.261 1.000000e+00       0
C1orf216        1.587649e-03 -0.3199950 0.243 0.272 1.000000e+00       0
NOP53           1.613004e-03 -0.4842621 0.367 0.447 1.000000e+00       0
RIOK2           1.705493e-03 -0.3234557 0.335 0.369 1.000000e+00       0
SLC23A2         1.716065e-03  0.7672872 0.206 0.111 1.000000e+00       0
TRIM16          1.763303e-03 -0.5918439 0.234 0.325 1.000000e+00       0
ARID3A          1.767658e-03 -0.3361584 0.421 0.494 1.000000e+00       0
RING1           1.768747e-03 -0.4020866 0.327 0.389 1.000000e+00       0
IER5L           1.772020e-03 -0.3480932 0.289 0.331 1.000000e+00       0
SGTA            1.780774e-03 -0.3879667 0.278 0.331 1.000000e+00       0
ZNF385A         1.787354e-03 -0.4565047 0.158 0.192 1.000000e+00       0
CACNA2D1        1.795088e-03 -0.4616784 0.311 0.358 1.000000e+00       0
LARP7           1.821231e-03 -0.3632146 0.310 0.383 1.000000e+00       0
PSIP1           1.825715e-03 -0.2801902 0.214 0.286 1.000000e+00       0
SKP2            1.844321e-03  0.2586745 0.383 0.244 1.000000e+00       0
THBD            1.878017e-03  0.4043651 0.233 0.206 1.000000e+00       0
LDLRAD4         1.889570e-03 -0.2877724 0.307 0.364 1.000000e+00       0
CDCA7L          1.900590e-03 -0.4252785 0.177 0.200 1.000000e+00       0
IGFBP5          1.909518e-03  0.3702351 0.378 0.278 1.000000e+00       0
FMOD            1.946455e-03 -0.3871719 0.238 0.292 1.000000e+00       0
WIPF1           1.966857e-03 -0.3314217 0.282 0.342 1.000000e+00       0
FAM133A         1.967456e-03  0.4970044 0.218 0.100 1.000000e+00       0
LRIG3           1.977099e-03 -0.3599808 0.247 0.294 1.000000e+00       0
ZFYVE27         1.995898e-03 -0.3246266 0.200 0.222 1.000000e+00       0
RPP25           2.029655e-03 -0.4337516 0.376 0.442 1.000000e+00       0
KCNK12          2.040799e-03 -1.0667401 0.211 0.331 1.000000e+00       0
SMC4            2.040815e-03 -0.5359390 0.259 0.319 1.000000e+00       0
COL11A1         2.055768e-03 -0.5677242 0.431 0.478 1.000000e+00       0
KAT8            2.082855e-03 -0.2798015 0.191 0.228 1.000000e+00       0
GTF2E1          2.105770e-03 -0.4895816 0.280 0.300 1.000000e+00       0
PRPS1           2.124398e-03 -0.2531629 0.291 0.319 1.000000e+00       0
FOXN2           2.146404e-03 -0.5051074 0.168 0.256 1.000000e+00       0
RNF227          2.196431e-03 -0.2837448 0.178 0.219 1.000000e+00       0
CBX4            2.223182e-03 -0.2858832 0.326 0.389 1.000000e+00       0
NUDT8           2.257034e-03 -0.5204688 0.408 0.461 1.000000e+00       0
ALDH1L2         2.259079e-03 -0.3510755 0.353 0.411 1.000000e+00       0
EGLN3           2.269333e-03  0.7300775 0.433 0.275 1.000000e+00       0
AL356056.1      2.294789e-03  0.8096991 0.333 0.139 1.000000e+00       0
OR8S1           2.312866e-03 -0.5508881 0.310 0.303 1.000000e+00       0
PITX1           2.326853e-03  0.3726855 0.442 0.356 1.000000e+00       0
SPPL2B          2.333176e-03 -0.2806828 0.286 0.347 1.000000e+00       0
ARHGEF11        2.338099e-03 -0.6381212 0.185 0.236 1.000000e+00       0
GADD45B         2.352931e-03 -0.4872390 0.418 0.503 1.000000e+00       0
LRRC73          2.405857e-03  0.3834025 0.176 0.144 1.000000e+00       0
HSBP1L1         2.413839e-03 -0.7495972 0.205 0.286 1.000000e+00       0
AL606491.1      2.418189e-03  0.3549613 0.108 0.097 1.000000e+00       0
KCNK1           2.431296e-03 -0.2727912 0.291 0.322 1.000000e+00       0
DNAJB1          2.435679e-03 -0.3487779 0.348 0.389 1.000000e+00       0
SP110           2.439317e-03  0.3557959 0.173 0.106 1.000000e+00       0
WNT2            2.441514e-03  0.3419068 0.451 0.328 1.000000e+00       0
DOCK5           2.442999e-03  0.4750843 0.466 0.356 1.000000e+00       0
PIM3            2.477673e-03 -0.4123413 0.210 0.303 1.000000e+00       0
FERMT2          2.596787e-03  0.2792251 0.442 0.372 1.000000e+00       0
MMP23B          2.601434e-03 -0.3559500 0.377 0.475 1.000000e+00       0
SPATA7          2.611529e-03 -0.4618382 0.155 0.219 1.000000e+00       0
SERPINB1        2.623248e-03 -0.3887088 0.154 0.211 1.000000e+00       0
C18orf25        2.665822e-03 -0.4116317 0.246 0.275 1.000000e+00       0
MBNL1-AS1       2.672975e-03  0.3270902 0.209 0.167 1.000000e+00       0
BMP4            2.712233e-03  0.4276451 0.437 0.308 1.000000e+00       0
GALNT6          2.717885e-03 -0.2698597 0.321 0.392 1.000000e+00       0
ETV5            2.721928e-03 -0.3912629 0.316 0.353 1.000000e+00       0
CCDC28B         2.756107e-03  0.4407078 0.393 0.275 1.000000e+00       0
RSPO2           2.844119e-03 -0.4771844 0.393 0.400 1.000000e+00       0
PPM1F           2.930356e-03 -0.2891395 0.221 0.214 1.000000e+00       0
OTUB1           3.032351e-03 -0.2669402 0.359 0.425 1.000000e+00       0
NDFIP2          3.043980e-03  0.4770207 0.427 0.325 1.000000e+00       0
PLXNA1          3.073684e-03 -0.4980956 0.325 0.369 1.000000e+00       0
RGS4            3.107198e-03  0.8360346 0.273 0.119 1.000000e+00       0
WWC2            3.151415e-03  0.5322259 0.448 0.322 1.000000e+00       0
ANTXR1          3.206418e-03  0.3695911 0.446 0.367 1.000000e+00       0
CAST            3.233773e-03 -0.2683037 0.458 0.492 1.000000e+00       0
IRAK1           3.248001e-03 -0.3402565 0.238 0.308 1.000000e+00       0
HSPA9           3.313244e-03 -0.4596805 0.442 0.519 1.000000e+00       0
FBLN1           3.331683e-03 -0.4927590 0.415 0.497 1.000000e+00       0
GREB1           3.357039e-03  0.3362317 0.148 0.117 1.000000e+00       0
MORC3           3.378686e-03 -0.6360003 0.257 0.311 1.000000e+00       0
TCEAL9          3.387568e-03  0.4457986 0.440 0.342 1.000000e+00       0
SLC12A7         3.389542e-03 -0.3418287 0.108 0.169 1.000000e+00       0
CITED2          3.466430e-03 -0.3216709 0.393 0.469 1.000000e+00       0
ASB13           3.521292e-03 -0.4924741 0.113 0.169 1.000000e+00       0
AMACR           3.533591e-03  0.2751419 0.170 0.058 1.000000e+00       0
FAM229B         3.592553e-03  0.2940620 0.456 0.378 1.000000e+00       0
VPS9D1          3.655576e-03 -0.2674597 0.210 0.244 1.000000e+00       0
PDIA4           3.664055e-03  0.2598264 0.461 0.389 1.000000e+00       0
BACE1-AS        3.731435e-03 -0.3088311 0.120 0.175 1.000000e+00       0
PCDHGB6         3.758460e-03  0.3371171 0.174 0.106 1.000000e+00       0
NT5C            3.803266e-03 -0.5007144 0.363 0.419 1.000000e+00       0
NENF            3.804363e-03 -0.2686031 0.374 0.425 1.000000e+00       0
AGPAT2          3.829690e-03 -0.2913952 0.307 0.364 1.000000e+00       0
ADAMTS2         3.888625e-03 -0.8360670 0.167 0.272 1.000000e+00       0
BBX             3.924623e-03 -0.3533518 0.439 0.533 1.000000e+00       0
SLC39A14        3.933666e-03 -0.5154375 0.302 0.392 1.000000e+00       0
TCEA1           4.032091e-03 -0.5864693 0.389 0.447 1.000000e+00       0
AXL             4.070448e-03 -0.3405396 0.427 0.511 1.000000e+00       0
EPB41L4A-AS1    4.096203e-03 -0.5305572 0.408 0.508 1.000000e+00       0
VGLL3           4.163622e-03  0.2762522 0.470 0.411 1.000000e+00       0
WDR4            4.193484e-03  0.2620638 0.128 0.164 1.000000e+00       0
LIFR            4.219966e-03 -0.2867829 0.263 0.308 1.000000e+00       0
CAVIN3          4.229628e-03 -0.3543000 0.437 0.492 1.000000e+00       0
SOD2            4.258725e-03 -0.8052313 0.270 0.331 1.000000e+00       0
FSTL3           4.300261e-03  0.2819847 0.461 0.369 1.000000e+00       0
FOXQ1           4.306394e-03 -0.4396501 0.179 0.225 1.000000e+00       0
ACOT7           4.319933e-03 -0.3075424 0.265 0.267 1.000000e+00       0
MYC             4.467553e-03  0.5363318 0.349 0.208 1.000000e+00       0
VAMP5           4.525006e-03 -0.3963306 0.127 0.142 1.000000e+00       0
TOM1            4.531471e-03 -0.4782351 0.297 0.375 1.000000e+00       0
PRSS12          4.537902e-03  0.8117898 0.206 0.147 1.000000e+00       0
KRT8            4.537908e-03  0.4244803 0.426 0.372 1.000000e+00       0
SLC1A1          4.540480e-03 -0.5077613 0.339 0.389 1.000000e+00       0
SH3BP5          4.571518e-03  0.4283631 0.363 0.269 1.000000e+00       0
WWC3            4.668407e-03 -0.4457035 0.230 0.256 1.000000e+00       0
PISD            4.685617e-03 -0.3565397 0.238 0.289 1.000000e+00       0
PLA2G4C         4.700226e-03 -0.3194758 0.093 0.150 1.000000e+00       0
PLEKHA7         4.717520e-03  0.6155170 0.135 0.097 1.000000e+00       0
ABCA5           4.750991e-03 -0.2823482 0.176 0.200 1.000000e+00       0
MEF2A           4.763112e-03 -0.3546684 0.339 0.378 1.000000e+00       0
CKB             4.863626e-03  0.5966805 0.452 0.331 1.000000e+00       0
HDAC4           4.943790e-03 -0.7651528 0.157 0.228 1.000000e+00       0
AL731569.1      4.956360e-03 -0.3188994 0.098 0.128 1.000000e+00       0
TPT1            5.133582e-03 -0.5136522 0.437 0.519 1.000000e+00       0
FBLN2           5.152324e-03 -0.5254403 0.394 0.458 1.000000e+00       0
ATAD2B          5.165356e-03 -0.4167769 0.200 0.194 1.000000e+00       0
CAMTA2          5.166802e-03 -0.4031622 0.272 0.286 1.000000e+00       0
CMTR2           5.201683e-03 -0.4464487 0.215 0.258 1.000000e+00       0
IRF1            5.277963e-03 -0.4723405 0.171 0.217 1.000000e+00       0
KANK4           5.315018e-03 -0.3941204 0.152 0.225 1.000000e+00       0
PITPNM1         5.364272e-03 -0.2791831 0.199 0.231 1.000000e+00       0
WDR26           5.388300e-03 -0.4906777 0.402 0.497 1.000000e+00       0
SLC4A8          5.397283e-03 -0.5141459 0.307 0.294 1.000000e+00       0
RAB13           5.441202e-03 -0.3473804 0.394 0.481 1.000000e+00       0
ENPP2           5.445740e-03 -0.4586540 0.145 0.244 1.000000e+00       0
LDHB            5.450318e-03  0.3567598 0.439 0.383 1.000000e+00       0
TXNIP           5.453352e-03 -0.4430292 0.391 0.486 1.000000e+00       0
GAS6-DT         5.497625e-03 -0.2781075 0.133 0.158 1.000000e+00       0
GARS            5.506836e-03 -0.4052350 0.429 0.519 1.000000e+00       0
SMIM10          5.531409e-03 -0.4148454 0.138 0.186 1.000000e+00       0
STK26           5.562326e-03 -0.3101486 0.274 0.311 1.000000e+00       0
EXPH5           5.602718e-03 -0.2909273 0.161 0.194 1.000000e+00       0
RAB11FIP5       5.616772e-03  0.3628542 0.275 0.231 1.000000e+00       0
AC073389.1      5.695374e-03 -0.2884813 0.101 0.122 1.000000e+00       0
UBE2E2          5.754181e-03  0.5112201 0.448 0.344 1.000000e+00       0
EFEMP2          5.818370e-03 -0.3728989 0.428 0.358 1.000000e+00       0
LTBP3           5.871810e-03 -0.3313431 0.406 0.444 1.000000e+00       0
AGPAT4          5.901138e-03 -0.2655679 0.229 0.283 1.000000e+00       0
GRAMD2B         5.937174e-03  0.2971981 0.235 0.217 1.000000e+00       0
SPATA13         5.951962e-03  0.2687785 0.268 0.297 1.000000e+00       0
AC092807.3      6.056496e-03  0.8075264 0.391 0.233 1.000000e+00       0
FENDRR          6.123714e-03 -0.4548103 0.159 0.253 1.000000e+00       0
ZNF251          6.125378e-03  0.2904770 0.142 0.131 1.000000e+00       0
H1F0            6.150788e-03 -0.2871406 0.221 0.283 1.000000e+00       0
TMEM88          6.160956e-03 -0.5365616 0.374 0.439 1.000000e+00       0
UBE3D           6.248071e-03  0.3764773 0.170 0.192 1.000000e+00       0
AKR1C1          6.272233e-03  0.4109704 0.167 0.097 1.000000e+00       0
LRRC37A3        6.294667e-03 -0.4931168 0.123 0.197 1.000000e+00       0
MELTF           6.392297e-03 -0.9774937 0.129 0.219 1.000000e+00       0
SERPING1        6.516267e-03 -0.7212562 0.186 0.289 1.000000e+00       0
WHAMM           6.570046e-03 -0.2796701 0.244 0.311 1.000000e+00       0
OLFM2           6.589824e-03  0.5171225 0.202 0.128 1.000000e+00       0
SESN1           6.646007e-03 -0.4056553 0.236 0.283 1.000000e+00       0
FLVCR1          6.669677e-03 -0.4306456 0.195 0.242 1.000000e+00       0
CISD1           6.678802e-03  0.4418194 0.473 0.372 1.000000e+00       0
DUSP6           6.872996e-03  0.7133660 0.265 0.142 1.000000e+00       0
CPNE2           6.882383e-03 -0.3494270 0.174 0.194 1.000000e+00       0
TMBIM1          6.936985e-03 -0.5765846 0.270 0.292 1.000000e+00       0
SEC16A          6.940766e-03 -0.4916333 0.361 0.411 1.000000e+00       0
SMURF2          7.049259e-03  0.3799744 0.408 0.311 1.000000e+00       0
CADPS2          7.062686e-03 -0.3063645 0.260 0.331 1.000000e+00       0
OTUD1           7.085761e-03 -0.7912817 0.158 0.256 1.000000e+00       0
ZADH2           7.118563e-03  0.4701756 0.471 0.372 1.000000e+00       0
MT-ND4          7.126304e-03 -0.4328865 0.399 0.483 1.000000e+00       0
PPP6R1          7.128238e-03 -0.5111638 0.297 0.347 1.000000e+00       0
BOD1            7.168998e-03 -0.5112153 0.434 0.483 1.000000e+00       0
LPCAT2          7.237394e-03  0.5904737 0.384 0.286 1.000000e+00       0
PEG10           7.247145e-03 -0.3193135 0.121 0.189 1.000000e+00       0
LINC00862       7.253074e-03  0.9688627 0.259 0.119 1.000000e+00       0
PSMC6           7.322228e-03 -0.3354460 0.408 0.442 1.000000e+00       0
SCD5            7.361889e-03  0.4166826 0.161 0.139 1.000000e+00       0
TET1            7.399902e-03 -0.3829471 0.130 0.181 1.000000e+00       0
STK24           7.405947e-03 -0.3166238 0.308 0.336 1.000000e+00       0
PROSER2         7.533134e-03 -0.3871228 0.251 0.300 1.000000e+00       0
RSPO3           7.582057e-03 -0.6177809 0.270 0.322 1.000000e+00       0
TTLL11          7.586174e-03 -0.3661621 0.109 0.131 1.000000e+00       0
MYH10           7.709923e-03  0.3461406 0.391 0.322 1.000000e+00       0
LHFPL2          7.728589e-03 -0.3826768 0.151 0.208 1.000000e+00       0
MICAL3          7.943593e-03  0.5069298 0.276 0.158 1.000000e+00       0
TMEM106A        8.005669e-03 -0.3078098 0.257 0.250 1.000000e+00       0
LMO7-AS1        8.007826e-03  0.3615034 0.448 0.364 1.000000e+00       0
STK38L          8.083352e-03  0.6433558 0.470 0.361 1.000000e+00       0
CENPV           8.115914e-03  0.5846868 0.370 0.214 1.000000e+00       0
TCEAL1          8.152959e-03 -0.4595363 0.393 0.472 1.000000e+00       0
KIAA0040        8.267210e-03  0.7151259 0.166 0.108 1.000000e+00       0
CPE             8.331439e-03  0.5433323 0.427 0.319 1.000000e+00       0
SPTAN1          8.351626e-03  0.3462125 0.448 0.389 1.000000e+00       0
ZNF711          8.486385e-03  0.9421010 0.214 0.100 1.000000e+00       0
SYT8            8.488597e-03  0.4158502 0.160 0.142 1.000000e+00       0
EGR1            8.502169e-03  0.4161517 0.312 0.228 1.000000e+00       0
DYM             8.520473e-03 -0.3452938 0.311 0.383 1.000000e+00       0
AZI2            8.666966e-03 -0.4216501 0.413 0.483 1.000000e+00       0
C16orf86        8.736402e-03 -0.3780112 0.059 0.100 1.000000e+00       0
LCOR            8.771311e-03 -0.3491849 0.350 0.381 1.000000e+00       0
CEP44           8.848541e-03 -0.2504300 0.237 0.258 1.000000e+00       0
NDRG1           8.860297e-03 -0.2831118 0.270 0.328 1.000000e+00       0
NQO1            9.161112e-03 -0.3652405 0.298 0.325 1.000000e+00       0
CNTRL           9.195126e-03 -0.5558774 0.112 0.147 1.000000e+00       0
DNAJC15         9.364284e-03  0.3349323 0.429 0.367 1.000000e+00       0
REC8            9.496092e-03 -0.3794416 0.110 0.153 1.000000e+00       0
RIMS1           9.538700e-03  0.4849387 0.183 0.144 1.000000e+00       0
GSTO1           9.586558e-03  0.7150308 0.461 0.397 1.000000e+00       0
AC010255.1      9.736488e-03 -0.5117992 0.121 0.183 1.000000e+00       0
MAK16           9.754414e-03  0.2642027 0.254 0.275 1.000000e+00       0
FIP1L1          9.795504e-03 -0.3255136 0.308 0.364 1.000000e+00       0
RAPGEF2         9.831578e-03  0.2536898 0.279 0.214 1.000000e+00       0
SVEP1           9.901485e-03 -0.3030921 0.242 0.264 1.000000e+00       0
APH1B           9.917077e-03 -0.2709604 0.281 0.300 1.000000e+00       0
TFPT            9.943127e-03  0.5044721 0.388 0.253 1.000000e+00       0
NTM             9.945700e-03  0.7892858 0.256 0.117 1.000000e+00       0
TNFRSF10B       1.014304e-02 -0.5052500 0.378 0.417 1.000000e+00       0
VTI1A           1.019624e-02 -0.3142509 0.204 0.239 1.000000e+00       0
ANKFN1          1.048233e-02  0.8730968 0.313 0.142 1.000000e+00       0
MORC4           1.061703e-02 -0.3632565 0.426 0.483 1.000000e+00       0
KIT             1.068632e-02 -0.4881154 0.227 0.219 1.000000e+00       0
CARD16          1.074473e-02 -0.4253807 0.141 0.172 1.000000e+00       0
SART1           1.075042e-02 -0.4030977 0.243 0.306 1.000000e+00       0
PARP9           1.075305e-02 -0.9839077 0.153 0.258 1.000000e+00       0
IGFBP3          1.081672e-02  0.2754505 0.423 0.353 1.000000e+00       0
PCMTD2          1.084864e-02 -0.6479475 0.278 0.311 1.000000e+00       0
MVD             1.087685e-02 -0.2639598 0.286 0.297 1.000000e+00       0
MMADHC          1.088533e-02  0.4381605 0.470 0.422 1.000000e+00       0
ID4             1.098744e-02  0.5264699 0.138 0.075 1.000000e+00       0
BARD1           1.100733e-02 -0.2929713 0.158 0.181 1.000000e+00       0
TSPAN6          1.101601e-02  0.2813506 0.442 0.394 1.000000e+00       0
ARIH2           1.107039e-02 -0.2733032 0.342 0.389 1.000000e+00       0
ZNF358          1.117119e-02 -0.2673083 0.343 0.381 1.000000e+00       0
MBNL1           1.125534e-02  0.3186664 0.489 0.425 1.000000e+00       0
AKAP17A         1.135469e-02 -0.4438057 0.246 0.289 1.000000e+00       0
VSIG10L         1.139582e-02 -0.5094202 0.095 0.117 1.000000e+00       0
ZNF324          1.146662e-02  0.5807482 0.148 0.133 1.000000e+00       0
PCSK1N          1.153966e-02  0.2758586 0.316 0.289 1.000000e+00       0
KIAA0355        1.157365e-02 -0.2548093 0.289 0.319 1.000000e+00       0
PBLD            1.167247e-02  0.3468464 0.183 0.192 1.000000e+00       0
PGD             1.176301e-02 -0.4724830 0.302 0.383 1.000000e+00       0
GBP1            1.192442e-02  0.7045203 0.289 0.206 1.000000e+00       0
CLEC1B          1.205678e-02  0.6678237 0.153 0.081 1.000000e+00       0
CCDC144NL-AS1   1.209711e-02  0.5749521 0.251 0.139 1.000000e+00       0
TRAF2           1.214685e-02 -0.3129311 0.109 0.158 1.000000e+00       0
CMIP            1.220304e-02 -0.3377556 0.450 0.531 1.000000e+00       0
HAND1           1.226887e-02 -0.3295912 0.191 0.233 1.000000e+00       0
ASS1            1.247145e-02 -0.4163379 0.416 0.467 1.000000e+00       0
SEMA3D          1.265756e-02  0.7722908 0.159 0.069 1.000000e+00       0
AL035413.1      1.277060e-02 -0.4218475 0.342 0.389 1.000000e+00       0
LINC01311       1.285589e-02  0.5520420 0.160 0.108 1.000000e+00       0
SMC6            1.309375e-02 -0.3175886 0.267 0.319 1.000000e+00       0
TYRP1           1.309706e-02 -0.3484432 0.176 0.228 1.000000e+00       0
NEK8            1.332184e-02  0.2989994 0.179 0.222 1.000000e+00       0
FXYD5           1.346447e-02  0.5842086 0.472 0.389 1.000000e+00       0
HOXA7           1.363257e-02 -0.3246019 0.289 0.286 1.000000e+00       0
IFITM2          1.365669e-02 -0.6478525 0.228 0.278 1.000000e+00       0
HMGB3           1.384442e-02  0.4695318 0.332 0.233 1.000000e+00       0
TRAF4           1.394265e-02  0.2962565 0.388 0.258 1.000000e+00       0
OTUD3           1.395673e-02  0.4376454 0.237 0.253 1.000000e+00       0
VASP            1.434530e-02  0.8199933 0.386 0.264 1.000000e+00       0
CRTAP           1.443939e-02 -0.4822032 0.413 0.492 1.000000e+00       0
SYNGR2          1.474708e-02  0.3685487 0.439 0.289 1.000000e+00       0
FAM133B         1.475818e-02 -0.4562848 0.305 0.361 1.000000e+00       0
HSPB3           1.500780e-02  0.5217892 0.166 0.153 1.000000e+00       0
VARS2           1.517376e-02 -0.2887539 0.179 0.178 1.000000e+00       0
SCUBE2          1.523084e-02 -0.5183825 0.260 0.319 1.000000e+00       0
TOB1            1.532240e-02 -0.3858543 0.342 0.383 1.000000e+00       0
OLFML2B         1.537212e-02  0.6889496 0.185 0.100 1.000000e+00       0
GPR68           1.556496e-02  0.2883949 0.151 0.089 1.000000e+00       0
KIF23           1.567776e-02 -0.4006216 0.161 0.222 1.000000e+00       0
YWHAH           1.588184e-02  0.3600135 0.461 0.375 1.000000e+00       0
CNTLN           1.616021e-02 -0.6029096 0.211 0.258 1.000000e+00       0
LETM2           1.616422e-02  0.4912755 0.098 0.064 1.000000e+00       0
AC103706.1      1.629289e-02  0.5321922 0.168 0.106 1.000000e+00       0
ZNF394          1.653260e-02 -0.2785260 0.254 0.269 1.000000e+00       0
RNF152          1.660222e-02  0.2955837 0.403 0.306 1.000000e+00       0
CNKSR3          1.664739e-02 -0.3590913 0.158 0.200 1.000000e+00       0
PARM1           1.728296e-02 -0.6335886 0.211 0.228 1.000000e+00       0
REL             1.791788e-02 -0.4639709 0.192 0.242 1.000000e+00       0
TMPO            1.795301e-02 -0.2775580 0.346 0.364 1.000000e+00       0
CAP1            1.796180e-02  0.4297491 0.473 0.411 1.000000e+00       0
MRC2            1.797499e-02 -0.2906757 0.409 0.469 1.000000e+00       0
WEE1            1.820949e-02 -0.5956928 0.206 0.242 1.000000e+00       0
PLXNB2          1.820950e-02 -0.3706473 0.383 0.406 1.000000e+00       0
SDC1            1.833893e-02  0.2762352 0.293 0.231 1.000000e+00       0
ING2            1.835238e-02 -0.2745598 0.318 0.336 1.000000e+00       0
C1QTNF3         1.842849e-02  0.4477965 0.216 0.125 1.000000e+00       0
SPTLC3          1.863653e-02 -0.3722456 0.223 0.261 1.000000e+00       0
GBE1            1.873212e-02 -0.4187717 0.313 0.378 1.000000e+00       0
ZKSCAN4         1.895215e-02 -0.2756721 0.136 0.175 1.000000e+00       0
IER3            1.899368e-02  0.4228501 0.445 0.358 1.000000e+00       0
JUP             1.903528e-02  0.8026074 0.268 0.136 1.000000e+00       0
SRRD            1.929120e-02  0.4052708 0.386 0.269 1.000000e+00       0
PPFIBP1         1.956906e-02  0.3268387 0.452 0.339 1.000000e+00       0
AC139795.3      1.960696e-02  0.2594149 0.140 0.072 1.000000e+00       0
HSD17B7         1.967804e-02 -0.3373851 0.191 0.219 1.000000e+00       0
LAMB2           1.982643e-02 -0.3471018 0.441 0.489 1.000000e+00       0
LINC01759       2.028150e-02 -0.2801081 0.237 0.211 1.000000e+00       0
TCTN3           2.036967e-02  0.4578742 0.425 0.356 1.000000e+00       0
PIDD1           2.041381e-02  0.3571946 0.141 0.117 1.000000e+00       0
GLUL            2.067097e-02 -0.3819340 0.235 0.322 1.000000e+00       0
PHYHD1          2.091580e-02 -0.2858142 0.255 0.297 1.000000e+00       0
CPQ             2.126485e-02 -0.3007145 0.270 0.308 1.000000e+00       0
LSAMP           2.133100e-02 -0.5661837 0.219 0.225 1.000000e+00       0
ANKAR           2.151053e-02 -0.3958701 0.119 0.169 1.000000e+00       0
HUNK            2.154149e-02  0.4752183 0.197 0.089 1.000000e+00       0
MTSS1           2.154152e-02  0.3196957 0.163 0.186 1.000000e+00       0
CNN1            2.159829e-02  0.7063552 0.233 0.128 1.000000e+00       0
CELF2           2.175347e-02  0.3191411 0.178 0.156 1.000000e+00       0
FLOT1           2.177453e-02 -0.5041005 0.423 0.439 1.000000e+00       0
CTSV            2.208853e-02  0.6513014 0.257 0.122 1.000000e+00       0
PHIP            2.256164e-02 -0.2891647 0.267 0.292 1.000000e+00       0
MAMSTR          2.264739e-02  0.3948780 0.174 0.122 1.000000e+00       0
SLC2A5          2.271768e-02 -0.2622263 0.129 0.142 1.000000e+00       0
ETHE1           2.293445e-02  0.4341727 0.460 0.375 1.000000e+00       0
C6orf62         2.296167e-02 -0.5218792 0.425 0.478 1.000000e+00       0
HADH            2.314191e-02 -0.3932286 0.178 0.228 1.000000e+00       0
GRIP2           2.343957e-02  0.3253321 0.153 0.106 1.000000e+00       0
IGFBP2          2.344604e-02  0.7574046 0.386 0.292 1.000000e+00       0
TAB3            2.351742e-02 -0.4670873 0.161 0.175 1.000000e+00       0
ME1             2.378585e-02 -0.2772214 0.342 0.397 1.000000e+00       0
GPR87           2.379140e-02  0.5615841 0.129 0.111 1.000000e+00       0
ZBTB38          2.379709e-02 -0.4176862 0.362 0.403 1.000000e+00       0
ST6GAL1         2.380828e-02  0.7260257 0.192 0.083 1.000000e+00       0
SZT2            2.389841e-02 -0.4327997 0.216 0.236 1.000000e+00       0
LIMK2           2.451425e-02 -0.3237610 0.433 0.436 1.000000e+00       0
PPM1H           2.456040e-02  0.3562272 0.121 0.078 1.000000e+00       0
RBMX            2.456618e-02 -0.4408707 0.359 0.394 1.000000e+00       0
EPHB6           2.474566e-02 -0.3113730 0.115 0.150 1.000000e+00       0
NUDCD1          2.493240e-02 -0.3715672 0.276 0.314 1.000000e+00       0
ZBTB7B          2.503790e-02 -0.4333070 0.219 0.228 1.000000e+00       0
ESYT2           2.510845e-02 -0.4284697 0.320 0.336 1.000000e+00       0
PACSIN3         2.527965e-02  0.3079397 0.269 0.272 1.000000e+00       0
ST7-AS1         2.555299e-02 -0.2726685 0.149 0.133 1.000000e+00       0
HIST1H2BJ       2.578724e-02  0.3797330 0.235 0.203 1.000000e+00       0
NEBL            2.597453e-02 -0.3652372 0.119 0.189 1.000000e+00       0
EFNA1           2.659294e-02  0.7141338 0.144 0.083 1.000000e+00       0
GNAS-AS1        2.682896e-02  0.2758057 0.160 0.067 1.000000e+00       0
GSTM3           2.721128e-02  0.3822738 0.407 0.367 1.000000e+00       0
AL121827.2      2.742685e-02  0.4468478 0.276 0.139 1.000000e+00       0
ASAP1           2.782962e-02 -0.3973033 0.423 0.458 1.000000e+00       0
NRTN            2.816584e-02 -0.4084178 0.146 0.181 1.000000e+00       0
ANTXR2          2.837456e-02 -0.2723440 0.240 0.306 1.000000e+00       0
COL18A1         2.861091e-02 -0.2584076 0.420 0.367 1.000000e+00       0
GAB2            2.903525e-02  0.3305194 0.191 0.128 1.000000e+00       0
SH3D19          2.909537e-02 -0.4751906 0.457 0.489 1.000000e+00       0
CDYL2           2.912199e-02 -0.4604875 0.138 0.150 1.000000e+00       0
TACC2           2.915536e-02 -0.4596508 0.146 0.203 1.000000e+00       0
PHLDA3          2.972959e-02 -0.4204634 0.418 0.467 1.000000e+00       0
BACH1           3.054254e-02 -0.5192057 0.287 0.333 1.000000e+00       0
TMEM8B          3.055643e-02  0.4311920 0.177 0.097 1.000000e+00       0
MID1IP1         3.059136e-02 -0.4804056 0.222 0.258 1.000000e+00       0
WLS             3.061929e-02 -0.2667401 0.442 0.503 1.000000e+00       0
NKTR            3.099140e-02 -0.5876816 0.433 0.436 1.000000e+00       0
TLL2            3.109031e-02  0.2983584 0.176 0.131 1.000000e+00       0
DYRK1B          3.113990e-02 -0.4061493 0.204 0.219 1.000000e+00       0
SLC29A4         3.167527e-02 -0.2843615 0.115 0.125 1.000000e+00       0
ZNF678          3.200005e-02  0.6359410 0.189 0.131 1.000000e+00       0
USP13           3.212347e-02 -0.2670722 0.161 0.206 1.000000e+00       0
MRPS6           3.217479e-02  0.3913153 0.470 0.383 1.000000e+00       0
CCDC18          3.237201e-02  0.4713380 0.112 0.075 1.000000e+00       0
ARHGAP5-AS1     3.249683e-02  0.2503857 0.206 0.181 1.000000e+00       0
ST3GAL1         3.256303e-02 -0.3275458 0.204 0.242 1.000000e+00       0
LRIF1           3.278457e-02 -0.4096218 0.335 0.350 1.000000e+00       0
KATNBL1         3.320156e-02  0.3515214 0.371 0.289 1.000000e+00       0
CYB5D1          3.330651e-02 -0.3419684 0.224 0.258 1.000000e+00       0
TNFAIP6         3.331402e-02 -0.5774394 0.120 0.161 1.000000e+00       0
KTN1            3.332905e-02 -0.3099396 0.433 0.489 1.000000e+00       0
AURKA           3.358505e-02 -0.3718430 0.163 0.153 1.000000e+00       0
CASP6           3.378232e-02  0.4499507 0.265 0.189 1.000000e+00       0
H3F3B           3.407979e-02 -0.2894224 0.473 0.408 1.000000e+00       0
PECR            3.449531e-02  0.3651055 0.157 0.089 1.000000e+00       0
BRIX1           3.454952e-02 -0.3972400 0.331 0.361 1.000000e+00       0
KLHL28          3.457276e-02 -0.2868491 0.250 0.267 1.000000e+00       0
G6PC3           3.458826e-02  0.2643016 0.429 0.358 1.000000e+00       0
SPINK2          3.461146e-02  0.3108477 0.206 0.172 1.000000e+00       0
ST3GAL6         3.491521e-02 -0.4222299 0.223 0.281 1.000000e+00       0
CMTM7           3.535523e-02 -0.4401283 0.223 0.261 1.000000e+00       0
SHROOM3         3.542638e-02  0.5807800 0.259 0.144 1.000000e+00       0
GALNT5          3.546595e-02  0.2516224 0.163 0.136 1.000000e+00       0
TIGD1           3.561673e-02 -0.3292894 0.164 0.183 1.000000e+00       0
HOXB3           3.566444e-02 -0.2798118 0.257 0.261 1.000000e+00       0
NCAM2           3.577603e-02  0.4202770 0.112 0.078 1.000000e+00       0
MED28           3.600006e-02 -0.3109405 0.397 0.419 1.000000e+00       0
LINC01750       3.628985e-02  0.7282835 0.198 0.086 1.000000e+00       0
SPRYD3          3.650037e-02  0.5061582 0.259 0.261 1.000000e+00       0
BSDC1           3.674460e-02 -0.5973768 0.335 0.356 1.000000e+00       0
TP53I3          3.687540e-02  0.6029636 0.453 0.361 1.000000e+00       0
DUSP14          3.737780e-02  0.3535445 0.407 0.319 1.000000e+00       0
DDX10           3.743581e-02 -0.5042003 0.253 0.300 1.000000e+00       0
HELLS           3.759362e-02 -0.6287375 0.138 0.197 1.000000e+00       0
LINC00342       3.829818e-02  0.3450357 0.157 0.128 1.000000e+00       0
SNAI1           3.862931e-02 -0.4478812 0.167 0.169 1.000000e+00       0
TNNC1           3.894565e-02  0.8364246 0.243 0.136 1.000000e+00       0
TP53INP1        3.922106e-02 -0.4525093 0.386 0.425 1.000000e+00       0
CFAP20          3.935068e-02  0.3284199 0.365 0.267 1.000000e+00       0
PCGF5           3.990780e-02 -0.6145249 0.215 0.267 1.000000e+00       0
DHCR7           4.012726e-02  0.2893218 0.463 0.375 1.000000e+00       0
KANSL1L         4.022413e-02 -0.4620250 0.219 0.253 1.000000e+00       0
JPH1            4.036540e-02  0.2749008 0.218 0.189 1.000000e+00       0
PER2            4.074702e-02  0.2621284 0.193 0.183 1.000000e+00       0
GSTA4           4.107807e-02 -0.4422153 0.321 0.281 1.000000e+00       0
HIST1H2BG       4.229554e-02  0.5461841 0.157 0.122 1.000000e+00       0
RAD50           4.234163e-02 -0.5274900 0.170 0.225 1.000000e+00       0
PSMB8           4.270207e-02 -0.8102628 0.134 0.242 1.000000e+00       0
ICAM3           4.285068e-02  0.4169978 0.405 0.336 1.000000e+00       0
GRIA3           4.316780e-02  0.3152665 0.168 0.125 1.000000e+00       0
AL662884.4      4.376095e-02 -0.2931204 0.112 0.103 1.000000e+00       0
RCAN1           4.394087e-02  0.5196739 0.355 0.289 1.000000e+00       0
MAP3K2          4.418877e-02 -0.3130959 0.365 0.397 1.000000e+00       0
FRMD6-AS1       4.441837e-02  0.6237710 0.263 0.139 1.000000e+00       0
DAAM1           4.462969e-02  0.3927563 0.442 0.347 1.000000e+00       0
MYLK            4.468330e-02  0.7383030 0.351 0.250 1.000000e+00       0
BICC1           4.468727e-02  0.6734728 0.158 0.114 1.000000e+00       0
HMGN3           4.539570e-02 -0.3306914 0.458 0.478 1.000000e+00       0
SSTR1           4.558156e-02  0.3432035 0.214 0.144 1.000000e+00       0
DSE             4.565020e-02  0.3720160 0.460 0.361 1.000000e+00       0
GREM1           4.605380e-02  0.4386386 0.191 0.128 1.000000e+00       0
NCS1            4.670024e-02  0.3271215 0.339 0.236 1.000000e+00       0
CYP26A1         4.710153e-02  0.6878651 0.179 0.078 1.000000e+00       0
TANGO2          4.745543e-02 -0.3179036 0.205 0.211 1.000000e+00       0
HMGN2           4.759753e-02 -0.5187264 0.384 0.392 1.000000e+00       0
RBM6            4.779095e-02 -0.2693682 0.367 0.397 1.000000e+00       0
SEPHS2          4.834428e-02 -0.5246421 0.348 0.394 1.000000e+00       0
ZC3H7B          4.843701e-02 -0.2664785 0.235 0.244 1.000000e+00       0
LINC01355       4.852990e-02  0.2756595 0.229 0.178 1.000000e+00       0
GINS1           4.875757e-02  0.4014788 0.136 0.128 1.000000e+00       0
LEF1            4.982500e-02  0.3312138 0.253 0.239 1.000000e+00       0
ASAP2           5.001546e-02 -0.3496338 0.344 0.403 1.000000e+00       0
ZFP36           5.025962e-02 -0.2681579 0.223 0.297 1.000000e+00       0
CRYGS           5.078307e-02  0.6021051 0.163 0.094 1.000000e+00       0
CYP1B1          5.105967e-02  0.2666980 0.377 0.300 1.000000e+00       0
CASTOR1         5.109512e-02 -0.3354613 0.209 0.200 1.000000e+00       0
SHKBP1          5.151720e-02 -0.2526764 0.257 0.278 1.000000e+00       0
RTN2            5.176748e-02  0.3623536 0.262 0.239 1.000000e+00       0
RUNX2           5.215033e-02 -0.4002286 0.225 0.267 1.000000e+00       0
ANO4            5.216102e-02  0.5362664 0.142 0.078 1.000000e+00       0
B4GALT1         5.223817e-02  0.2524260 0.461 0.394 1.000000e+00       0
TRPM7           5.260179e-02 -0.3797395 0.289 0.342 1.000000e+00       0
PTTG1IP         5.275670e-02  0.2593685 0.472 0.414 1.000000e+00       0
TEX22           5.300087e-02  0.4379577 0.214 0.114 1.000000e+00       0
TRAF3IP1        5.334657e-02 -0.4480794 0.241 0.272 1.000000e+00       0
SORT1           5.373914e-02 -0.2832836 0.370 0.361 1.000000e+00       0
LITAF           5.390808e-02 -0.4076969 0.221 0.292 1.000000e+00       0
TGM2            5.416672e-02 -0.7072140 0.349 0.342 1.000000e+00       0
CREBRF          5.474823e-02 -0.4344016 0.332 0.383 1.000000e+00       0
ACAA1           5.554139e-02  0.2605907 0.405 0.331 1.000000e+00       0
MYOCD           5.616879e-02  0.2999651 0.344 0.342 1.000000e+00       0
ADCY7           5.621550e-02  0.4530508 0.231 0.125 1.000000e+00       0
G6PD            5.632071e-02 -0.4067792 0.338 0.375 1.000000e+00       0
MAPRE2          5.648469e-02  0.2961085 0.374 0.275 1.000000e+00       0
NCOA1           5.650814e-02 -0.3311739 0.251 0.286 1.000000e+00       0
INAFM2          5.651985e-02 -0.2657642 0.227 0.244 1.000000e+00       0
CTSO            5.675491e-02  0.3656421 0.237 0.181 1.000000e+00       0
PLSCR1          5.686101e-02 -0.3369369 0.249 0.292 1.000000e+00       0
NSFL1C          5.796655e-02 -0.2557171 0.320 0.356 1.000000e+00       0
TSTD1           5.815824e-02  0.5113108 0.203 0.128 1.000000e+00       0
HIPK2           5.871341e-02 -0.3144320 0.408 0.489 1.000000e+00       0
TMEM11          5.939492e-02 -0.4704043 0.375 0.389 1.000000e+00       0
TIFA            5.970121e-02 -0.5572240 0.117 0.144 1.000000e+00       0
PLAA            5.995960e-02 -0.2770498 0.227 0.236 1.000000e+00       0
ITGB1BP2        6.002115e-02  0.3452900 0.134 0.078 1.000000e+00       0
THBS3           6.028073e-02 -0.4390619 0.196 0.236 1.000000e+00       0
RNASET2         6.047906e-02  0.2686282 0.262 0.278 1.000000e+00       0
GALNT2          6.057843e-02 -0.3833850 0.456 0.500 1.000000e+00       0
ZNF883          6.122724e-02  0.3707262 0.161 0.139 1.000000e+00       0
HSPH1           6.151658e-02 -0.3143779 0.421 0.450 1.000000e+00       0
COL1A1          6.181929e-02 -0.3417714 0.450 0.408 1.000000e+00       0
CSGALNACT2      6.190785e-02  0.2907164 0.437 0.375 1.000000e+00       0
KLF4            6.230116e-02 -0.7449783 0.110 0.181 1.000000e+00       0
KIAA0825        6.251737e-02  0.3957624 0.163 0.100 1.000000e+00       0
NASP            6.300409e-02 -0.4617387 0.306 0.339 1.000000e+00       0
NSRP1           6.308117e-02 -0.2817699 0.415 0.436 1.000000e+00       0
CLIC1           6.358408e-02  0.2594477 0.446 0.433 1.000000e+00       0
SYPL1           6.358408e-02 -0.3369614 0.418 0.494 1.000000e+00       0
C2CD3           6.380432e-02  0.4334377 0.243 0.158 1.000000e+00       0
MFAP4           6.401086e-02  0.2611639 0.451 0.372 1.000000e+00       0
COL25A1         6.503355e-02  0.4086971 0.218 0.139 1.000000e+00       0
AEBP1           6.564195e-02  0.6517679 0.375 0.308 1.000000e+00       0
ENOSF1          6.584168e-02 -0.3038092 0.216 0.272 1.000000e+00       0
ZNF350          6.630876e-02  0.4752082 0.163 0.136 1.000000e+00       0
BEX1            6.640286e-02  0.4974984 0.189 0.092 1.000000e+00       0
AC010336.1      6.698138e-02  0.8367877 0.149 0.081 1.000000e+00       0
MBP             6.730582e-02  0.5990068 0.229 0.147 1.000000e+00       0
INTS13          6.746911e-02  0.3410267 0.259 0.200 1.000000e+00       0
CENPF           6.775462e-02 -0.3942309 0.139 0.147 1.000000e+00       0
ID1             6.826078e-02  0.4732245 0.308 0.167 1.000000e+00       0
VLDLR           6.837033e-02  0.3809997 0.208 0.169 1.000000e+00       0
CNRIP1          6.892166e-02 -0.6237824 0.148 0.222 1.000000e+00       0
ATP2A2          6.922665e-02 -0.4345449 0.371 0.417 1.000000e+00       0
AIFM2           6.943498e-02 -0.4020398 0.244 0.272 1.000000e+00       0
CALM1           6.943505e-02 -0.3454471 0.453 0.483 1.000000e+00       0
HSPA12A         6.968579e-02  0.5227246 0.265 0.169 1.000000e+00       0
AC112220.3      7.071011e-02  0.6149405 0.166 0.128 1.000000e+00       0
PKDCC           7.134807e-02 -0.3082035 0.282 0.286 1.000000e+00       0
IFNGR2          7.177587e-02  0.2572709 0.454 0.372 1.000000e+00       0
ZNF613          7.196170e-02 -0.2520503 0.133 0.147 1.000000e+00       0
RABL2A          7.246443e-02  0.3400379 0.172 0.117 1.000000e+00       0
FAM69B          7.270984e-02  0.4088240 0.183 0.106 1.000000e+00       0
TSHZ1           7.307175e-02  0.2798894 0.425 0.350 1.000000e+00       0
DOCK1           7.336234e-02  0.4150829 0.299 0.267 1.000000e+00       0
ACTC1           7.337602e-02  0.8262581 0.288 0.144 1.000000e+00       0
TXLNA           7.349341e-02 -0.2592459 0.431 0.464 1.000000e+00       0
DUSP2           7.366834e-02  0.3500137 0.132 0.042 1.000000e+00       0
PRKG1           7.412221e-02 -0.4410838 0.209 0.197 1.000000e+00       0
GATM            7.448975e-02  0.7932491 0.276 0.131 1.000000e+00       0
RFWD3           7.490317e-02 -0.3359403 0.141 0.181 1.000000e+00       0
FBXO4           7.598999e-02 -0.3161376 0.127 0.125 1.000000e+00       0
CARMN           7.620012e-02  0.3800886 0.361 0.225 1.000000e+00       0
SESN3           7.635035e-02 -0.3080081 0.408 0.428 1.000000e+00       0
YAP1            7.656105e-02 -0.4163493 0.445 0.486 1.000000e+00       0
MANCR           7.660604e-02  0.5995511 0.251 0.194 1.000000e+00       0
AFAP1           7.705959e-02  0.6104217 0.246 0.181 1.000000e+00       0
GUSB            7.707476e-02 -0.2810317 0.233 0.272 1.000000e+00       0
ARID4A          7.719605e-02 -0.2522369 0.324 0.328 1.000000e+00       0
VWA8            7.795708e-02 -0.2993316 0.155 0.172 1.000000e+00       0
NDN             7.830987e-02 -0.3428675 0.383 0.397 1.000000e+00       0
LY6G5C          7.911104e-02  0.5882705 0.165 0.097 1.000000e+00       0
MTHFD2          7.988818e-02 -0.3096258 0.434 0.478 1.000000e+00       0
PPP1R14A        8.005958e-02  0.5807838 0.345 0.275 1.000000e+00       0
SPATA5L1        8.023185e-02  0.4013432 0.191 0.122 1.000000e+00       0
SSPN            8.093888e-02  0.3622604 0.391 0.303 1.000000e+00       0
SLC40A1         8.147375e-02 -0.6536012 0.372 0.361 1.000000e+00       0
C4BPB           8.192109e-02  0.6195568 0.206 0.122 1.000000e+00       0
FSTL1           8.196889e-02 -0.8783324 0.438 0.444 1.000000e+00       0
NRBP2           8.203264e-02 -0.3026837 0.319 0.297 1.000000e+00       0
HAPLN4          8.321852e-02  0.3693360 0.259 0.186 1.000000e+00       0
PTHLH           8.328338e-02  0.6197398 0.294 0.167 1.000000e+00       0
RBM47           8.448432e-02 -0.3048284 0.113 0.186 1.000000e+00       0
ZCCHC10         8.459871e-02 -0.3954802 0.312 0.344 1.000000e+00       0
RACGAP1         8.482729e-02 -0.3634713 0.135 0.175 1.000000e+00       0
BCL7A           8.571573e-02  0.5378064 0.240 0.153 1.000000e+00       0
NPNT            8.622892e-02 -0.3652479 0.177 0.228 1.000000e+00       0
PIGQ            8.627878e-02 -0.3447191 0.189 0.244 1.000000e+00       0
SLC22A15        8.646095e-02 -0.3711449 0.178 0.206 1.000000e+00       0
ATP10D          8.722931e-02 -0.2734364 0.356 0.314 1.000000e+00       0
SORBS3          8.774983e-02 -0.6252122 0.324 0.372 1.000000e+00       0
UNKL            8.778348e-02 -0.6034483 0.193 0.250 1.000000e+00       0
HNRNPUL2        8.828974e-02 -0.2964378 0.310 0.344 1.000000e+00       0
SEPT10          8.866248e-02 -0.2541555 0.345 0.353 1.000000e+00       0
MSMO1           8.886629e-02  0.3609512 0.465 0.400 1.000000e+00       0
KIAA0895L       8.888289e-02 -0.3001493 0.126 0.144 1.000000e+00       0
ALDH1B1         8.898541e-02  0.3718399 0.352 0.244 1.000000e+00       0
TMEM176B        8.965122e-02  0.5277186 0.126 0.072 1.000000e+00       0
ENAH            8.975398e-02  0.4808107 0.435 0.403 1.000000e+00       0
FDFT1           8.977118e-02  0.5501409 0.454 0.394 1.000000e+00       0
LINC00545       9.011443e-02 -0.4918580 0.128 0.131 1.000000e+00       0
ODF2L           9.020056e-02  0.5149465 0.351 0.275 1.000000e+00       0
LINC02001       9.025220e-02 -0.3327276 0.241 0.214 1.000000e+00       0
EPHB2           9.038997e-02  0.2525346 0.165 0.119 1.000000e+00       0
TXNDC5          9.142901e-02  0.5354821 0.262 0.153 1.000000e+00       0
ELL2            9.146383e-02 -0.2531233 0.295 0.339 1.000000e+00       0
MPLKIP          9.169028e-02 -0.3730909 0.439 0.481 1.000000e+00       0
WNT4            9.184568e-02  1.1741307 0.280 0.100 1.000000e+00       0
PROS1           9.196896e-02 -0.3031020 0.185 0.242 1.000000e+00       0
TSC22D1-AS1     9.268824e-02  0.3817517 0.196 0.131 1.000000e+00       0
ZNF438          9.297019e-02 -0.3552202 0.147 0.167 1.000000e+00       0
RASSF7          9.307609e-02  0.3122862 0.363 0.294 1.000000e+00       0
HMGA2           9.319969e-02  0.3079267 0.317 0.219 1.000000e+00       0
FOXP1           9.325271e-02  0.3267359 0.390 0.325 1.000000e+00       0
EMILIN1         9.406843e-02  0.4564719 0.269 0.189 1.000000e+00       0
CLIC4           9.408640e-02 -0.3855047 0.420 0.453 1.000000e+00       0
KANK2           9.437156e-02 -0.3816368 0.299 0.336 1.000000e+00       0
COQ8A           9.478244e-02  0.3710684 0.152 0.133 1.000000e+00       0
DNAJC12         9.489014e-02  0.2745619 0.238 0.189 1.000000e+00       0
ASPN            9.490736e-02  0.6614929 0.233 0.172 1.000000e+00       0
DCTN4           9.548313e-02 -0.5097304 0.343 0.369 1.000000e+00       0
PCAT1           9.568141e-02  0.4406875 0.256 0.097 1.000000e+00       0
RPRML           9.597040e-02  0.3859990 0.224 0.103 1.000000e+00       0
PATJ            9.635087e-02  0.3923521 0.168 0.086 1.000000e+00       0
FLYWCH1         9.702407e-02 -0.4040000 0.225 0.278 1.000000e+00       0
PLXDC1          9.762749e-02  0.3947066 0.236 0.150 1.000000e+00       0
IGF2-AS         9.779272e-02 -0.3914856 0.085 0.139 1.000000e+00       0
LYPD1           9.823391e-02  0.7010374 0.153 0.106 1.000000e+00       0
TRAF3IP2-AS1    9.882483e-02 -0.4802090 0.158 0.181 1.000000e+00       0
INTS6           9.995988e-02 -0.3121422 0.384 0.414 1.000000e+00       0
PHLDA2          1.002215e-01  0.3600369 0.416 0.356 1.000000e+00       0
MSRA            1.004279e-01 -0.3216221 0.257 0.244 1.000000e+00       0
ADGRF4          1.011044e-01 -0.3809315 0.164 0.200 1.000000e+00       0
FAM96B          1.023173e-01 -0.2745190 0.452 0.475 1.000000e+00       0
HNRNPH1         1.024887e-01  0.3583510 0.432 0.394 1.000000e+00       0
ADGRA2          1.035605e-01 -0.3959671 0.429 0.425 1.000000e+00       0
MYO9B           1.039840e-01 -0.3877527 0.308 0.361 1.000000e+00       0
MRPL52          1.040612e-01 -0.3875264 0.451 0.494 1.000000e+00       0
GXYLT2          1.054771e-01  0.3012746 0.447 0.381 1.000000e+00       0
MET             1.054963e-01 -0.7354282 0.144 0.186 1.000000e+00       0
LUM             1.056854e-01 -0.2740892 0.454 0.500 1.000000e+00       0
ECM1            1.070252e-01  0.2730086 0.357 0.217 1.000000e+00       0
PSAT1           1.070659e-01 -0.2924479 0.363 0.386 1.000000e+00       0
PPM1K           1.075800e-01 -0.3153156 0.335 0.331 1.000000e+00       0
HOXC8           1.076197e-01  0.2591027 0.457 0.411 1.000000e+00       0
CEP170          1.081358e-01  0.2792814 0.370 0.306 1.000000e+00       0
PTPN13          1.086339e-01 -0.3991124 0.286 0.339 1.000000e+00       0
HSPB11          1.090137e-01 -0.2680682 0.453 0.472 1.000000e+00       0
GCLM            1.091940e-01 -0.4127846 0.357 0.425 1.000000e+00       0
NOP14           1.093143e-01  0.4180871 0.325 0.256 1.000000e+00       0
PLP2            1.096356e-01 -0.2967742 0.427 0.478 1.000000e+00       0
AC025171.4      1.100786e-01  0.4051259 0.109 0.069 1.000000e+00       0
SOCS3           1.101392e-01 -0.4604608 0.168 0.189 1.000000e+00       0
ANGPTL7         1.108827e-01 -0.2882933 0.242 0.233 1.000000e+00       0
CHST3           1.109283e-01 -0.5897034 0.172 0.183 1.000000e+00       0
AP002387.2      1.109690e-01 -0.3205660 0.231 0.233 1.000000e+00       0
GPC3            1.130335e-01  0.8333581 0.329 0.181 1.000000e+00       0
RERE            1.149835e-01  0.2824456 0.406 0.322 1.000000e+00       0
NR2F1-AS1       1.164099e-01  0.6134806 0.326 0.294 1.000000e+00       0
CIRBP           1.166421e-01 -0.3027858 0.429 0.494 1.000000e+00       0
EPHA4           1.168528e-01  0.3596324 0.228 0.194 1.000000e+00       0
TPCN2           1.176583e-01 -0.3742921 0.179 0.183 1.000000e+00       0
DPP10-AS1       1.179138e-01  0.6870260 0.173 0.094 1.000000e+00       0
DTX3L           1.185967e-01 -0.2682055 0.203 0.181 1.000000e+00       0
ATP6V0A2        1.186185e-01 -0.2970601 0.356 0.369 1.000000e+00       0
MAPK8IP3        1.187896e-01  0.2607839 0.205 0.183 1.000000e+00       0
MANBA           1.188112e-01 -0.2862478 0.331 0.344 1.000000e+00       0
MFSD1           1.201883e-01  0.3293052 0.448 0.372 1.000000e+00       0
LINC01082       1.208815e-01  0.2834720 0.108 0.089 1.000000e+00       0
TEDC1           1.211852e-01  0.3318345 0.186 0.158 1.000000e+00       0
ST6GAL2         1.213381e-01  0.8722220 0.180 0.081 1.000000e+00       0
MYOZ1           1.219272e-01  0.5644263 0.177 0.081 1.000000e+00       0
TPP2            1.227820e-01 -0.3798486 0.233 0.239 1.000000e+00       0
ULK4            1.232220e-01  0.4633244 0.189 0.169 1.000000e+00       0
C1orf226        1.235530e-01 -0.2732631 0.109 0.122 1.000000e+00       0
ISCU            1.236636e-01 -0.4134243 0.439 0.478 1.000000e+00       0
CAPN10          1.245279e-01 -0.2894296 0.216 0.225 1.000000e+00       0
SCN1B           1.251959e-01  0.3565672 0.241 0.203 1.000000e+00       0
KLHL4           1.254639e-01  0.4011343 0.329 0.239 1.000000e+00       0
PDE4B           1.260013e-01 -0.3074500 0.126 0.144 1.000000e+00       0
STK17B          1.266304e-01  0.4769331 0.410 0.322 1.000000e+00       0
RAD54B          1.273067e-01  0.3342599 0.167 0.103 1.000000e+00       0
C2orf49         1.288061e-01 -0.4000388 0.344 0.344 1.000000e+00       0
PLEKHO1         1.293546e-01  0.4428100 0.408 0.339 1.000000e+00       0
AP1G1           1.298819e-01  0.2505800 0.386 0.392 1.000000e+00       0
DHRS3           1.299965e-01  0.5081172 0.224 0.200 1.000000e+00       0
AC124283.1      1.301341e-01 -0.2932402 0.165 0.189 1.000000e+00       0
ATF3            1.301806e-01 -0.4267726 0.362 0.403 1.000000e+00       0
TMEM50B         1.303878e-01 -0.2614323 0.446 0.456 1.000000e+00       0
SNTA1           1.316593e-01 -0.2771553 0.206 0.231 1.000000e+00       0
DKK1            1.319841e-01 -0.2955107 0.356 0.383 1.000000e+00       0
FIG4            1.320537e-01  0.4487162 0.139 0.114 1.000000e+00       0
NCL             1.323103e-01  0.3121846 0.448 0.428 1.000000e+00       0
RPL10           1.333149e-01 -0.2876443 0.399 0.481 1.000000e+00       0
EIF5            1.345848e-01 -0.5432989 0.457 0.475 1.000000e+00       0
TNFRSF21        1.364594e-01 -0.3186062 0.259 0.272 1.000000e+00       0
EPHX2           1.387401e-01  0.5996449 0.112 0.069 1.000000e+00       0
LRRC75A         1.400985e-01  0.2578657 0.399 0.347 1.000000e+00       0
LRCH4           1.403179e-01  0.3962076 0.225 0.175 1.000000e+00       0
MAP1B           1.405862e-01 -0.2728146 0.454 0.461 1.000000e+00       0
AC021242.3      1.409037e-01  0.4088893 0.140 0.131 1.000000e+00       0
AL080317.1      1.418108e-01 -0.2580893 0.185 0.189 1.000000e+00       0
HMGB2           1.418599e-01 -0.4392954 0.233 0.272 1.000000e+00       0
C1orf56         1.420567e-01  0.5178485 0.396 0.319 1.000000e+00       0
DNAJC28         1.425495e-01  0.3012330 0.172 0.108 1.000000e+00       0
TMTC1           1.437375e-01 -0.4608148 0.195 0.261 1.000000e+00       0
ARHGAP20        1.448327e-01 -0.4373019 0.158 0.231 1.000000e+00       0
ARG2            1.452833e-01 -0.2836581 0.123 0.150 1.000000e+00       0
WFDC3           1.461364e-01 -0.5185050 0.094 0.161 1.000000e+00       0
DIXDC1          1.468166e-01 -0.5040110 0.451 0.481 1.000000e+00       0
ADAT2           1.468417e-01  0.4227125 0.116 0.069 1.000000e+00       0
BAIAP2-DT       1.474484e-01  0.6381055 0.139 0.083 1.000000e+00       0
FOS             1.477272e-01 -0.4127775 0.241 0.297 1.000000e+00       0
FAM84A          1.480317e-01  0.6846492 0.168 0.103 1.000000e+00       0
EXTL1           1.484383e-01  0.4275242 0.179 0.125 1.000000e+00       0
SLC4A2          1.504851e-01 -0.4751627 0.395 0.408 1.000000e+00       0
KIF9-AS1        1.510517e-01 -0.2820288 0.130 0.133 1.000000e+00       0
CALY            1.512065e-01  0.4986773 0.199 0.122 1.000000e+00       0
CRACR2B         1.513609e-01  0.3592735 0.142 0.114 1.000000e+00       0
TSPO            1.523703e-01  0.5155333 0.260 0.200 1.000000e+00       0
TNFSF15         1.526831e-01  0.3395100 0.193 0.106 1.000000e+00       0
ZHX1            1.528652e-01 -0.2817057 0.279 0.292 1.000000e+00       0
GNG2            1.534119e-01  0.6725187 0.120 0.083 1.000000e+00       0
TM4SF1          1.540135e-01 -0.3997296 0.208 0.217 1.000000e+00       0
PSPN            1.561141e-01  0.9045274 0.273 0.139 1.000000e+00       0
TPD52           1.565417e-01 -0.2578061 0.168 0.175 1.000000e+00       0
HOXB4           1.569658e-01 -0.3126005 0.199 0.183 1.000000e+00       0
CBR1            1.595850e-01  0.2616999 0.314 0.264 1.000000e+00       0
AC137767.1      1.602039e-01  0.2632666 0.165 0.203 1.000000e+00       0
HELZ2           1.614484e-01 -0.3924150 0.134 0.144 1.000000e+00       0
RAB3B           1.616384e-01  0.7650146 0.332 0.194 1.000000e+00       0
AC073611.1      1.624545e-01  0.4080653 0.319 0.247 1.000000e+00       0
TMEM86B         1.626181e-01 -0.5213077 0.085 0.108 1.000000e+00       0
MRPS18B         1.626181e-01 -0.3042939 0.304 0.292 1.000000e+00       0
WDR1            1.628364e-01  0.3873924 0.480 0.461 1.000000e+00       0
TMLHE           1.628907e-01  0.3662659 0.136 0.128 1.000000e+00       0
LINC02334       1.644253e-01 -0.2713664 0.114 0.144 1.000000e+00       0
CDC37L1-DT      1.655286e-01  0.3820880 0.157 0.069 1.000000e+00       0
DYNC1I1         1.660821e-01  0.4660631 0.227 0.147 1.000000e+00       0
TRIM47          1.675552e-01 -0.3430913 0.184 0.214 1.000000e+00       0
FN1             1.681365e-01 -0.4716904 0.464 0.431 1.000000e+00       0
CDC42EP2        1.681977e-01 -0.2737218 0.180 0.208 1.000000e+00       0
KIAA1328        1.687583e-01  0.4490683 0.187 0.125 1.000000e+00       0
MDFI            1.689267e-01 -0.3872537 0.419 0.442 1.000000e+00       0
PIR             1.691793e-01 -0.5797599 0.121 0.206 1.000000e+00       0
PSD             1.697700e-01  0.3667368 0.142 0.086 1.000000e+00       0
NTF3            1.704187e-01  0.6391589 0.200 0.131 1.000000e+00       0
NSMF            1.706735e-01 -0.2677188 0.186 0.181 1.000000e+00       0
LINC01638       1.714952e-01 -0.2855187 0.183 0.225 1.000000e+00       0
AL050341.2      1.715230e-01  0.5053571 0.225 0.164 1.000000e+00       0
SFRP1           1.723460e-01  0.2685107 0.292 0.275 1.000000e+00       0
AMMECR1L        1.723769e-01  0.2968094 0.246 0.211 1.000000e+00       0
SRRM3           1.724909e-01 -0.7239083 0.185 0.256 1.000000e+00       0
ACAT2           1.753594e-01  0.3485812 0.355 0.283 1.000000e+00       0
SPARC           1.753594e-01 -0.7536084 0.426 0.486 1.000000e+00       0
BBOX1-AS1       1.753882e-01  0.4617727 0.375 0.217 1.000000e+00       0
SLC4A1AP        1.759373e-01  0.4493343 0.291 0.303 1.000000e+00       0
BICRA           1.765742e-01  0.4430811 0.136 0.144 1.000000e+00       0
VDR             1.767198e-01  0.2989211 0.238 0.200 1.000000e+00       0
KRT19           1.774738e-01  0.3251793 0.342 0.311 1.000000e+00       0
SNX25           1.784095e-01  0.2934802 0.228 0.186 1.000000e+00       0
SLC9A3R2        1.789949e-01  0.2640283 0.370 0.283 1.000000e+00       0
CORO2B          1.816468e-01  0.4635862 0.160 0.097 1.000000e+00       0
DMRTA1          1.817655e-01  0.4688119 0.255 0.172 1.000000e+00       0
HOXB8           1.831329e-01  0.3466182 0.292 0.217 1.000000e+00       0
MIR503HG        1.833117e-01  0.3489858 0.196 0.181 1.000000e+00       0
KLF2            1.850179e-01  0.4758451 0.230 0.197 1.000000e+00       0
FRMD4A          1.852281e-01  0.6947537 0.234 0.217 1.000000e+00       0
GTF2F1          1.860714e-01 -0.2586242 0.359 0.367 1.000000e+00       0
MEST            1.872491e-01 -0.2818384 0.437 0.408 1.000000e+00       0
KMT5C           1.878569e-01  0.6037648 0.172 0.139 1.000000e+00       0
FASTKD3         1.892271e-01  0.2543509 0.176 0.119 1.000000e+00       0
ZNF563          1.901452e-01  0.5081756 0.171 0.083 1.000000e+00       0
MAP3K8          1.912504e-01 -0.3215391 0.153 0.197 1.000000e+00       0
SMCO4           1.925471e-01  0.3639452 0.355 0.356 1.000000e+00       0
LIN54           1.950957e-01 -0.4070794 0.187 0.217 1.000000e+00       0
KLHL42          1.951270e-01  0.3121175 0.374 0.314 1.000000e+00       0
AC073323.1      2.014795e-01 -0.3914415 0.210 0.236 1.000000e+00       0
FKBP10          2.022802e-01 -0.2946378 0.450 0.419 1.000000e+00       0
THBS2           2.023439e-01  0.3274437 0.338 0.228 1.000000e+00       0
TGFBR3          2.026328e-01 -0.9176075 0.126 0.203 1.000000e+00       0
CMTM3           2.048251e-01  0.3104336 0.352 0.317 1.000000e+00       0
TMCO6           2.049869e-01  0.4628778 0.167 0.092 1.000000e+00       0
MICA            2.101166e-01  0.2993032 0.458 0.433 1.000000e+00       0
PDGFC           2.102815e-01  0.2824081 0.434 0.356 1.000000e+00       0
GULP1           2.139992e-01 -0.5169156 0.237 0.317 1.000000e+00       0
GLA             2.147016e-01 -0.3581645 0.331 0.336 1.000000e+00       0
CDC42EP4        2.184754e-01 -0.3578082 0.114 0.153 1.000000e+00       0
MPC2            2.186111e-01 -0.3276057 0.429 0.458 1.000000e+00       0
MPRIP           2.187808e-01  0.3430889 0.396 0.333 1.000000e+00       0
WNT5B           2.201755e-01  0.4464995 0.305 0.242 1.000000e+00       0
NDUFB2-AS1      2.204137e-01 -0.3178728 0.127 0.133 1.000000e+00       0
NBL1            2.229153e-01 -0.2680774 0.388 0.378 1.000000e+00       0
P4HB            2.235696e-01 -0.3614757 0.435 0.481 1.000000e+00       0
VEGFC           2.240522e-01  0.8274592 0.203 0.117 1.000000e+00       0
SLC2A8          2.247438e-01  0.2702495 0.214 0.181 1.000000e+00       0
PRKAB2          2.259225e-01 -0.2843807 0.367 0.397 1.000000e+00       0
GPS1            2.260961e-01  0.2676304 0.348 0.297 1.000000e+00       0
SGK494          2.266177e-01  0.5795455 0.158 0.103 1.000000e+00       0
LINC01013       2.271399e-01  1.0097919 0.233 0.056 1.000000e+00       0
CD248           2.271749e-01 -0.7835759 0.170 0.189 1.000000e+00       0
ALOX5AP         2.272450e-01 -0.5940122 0.167 0.228 1.000000e+00       0
SULF1           2.293090e-01  0.5344653 0.172 0.119 1.000000e+00       0
FABP3           2.336911e-01 -0.2812820 0.123 0.169 1.000000e+00       0
NKX3-1          2.348314e-01  0.4149346 0.214 0.167 1.000000e+00       0
SELENOK         2.380600e-01 -0.2970906 0.429 0.472 1.000000e+00       0
AC009414.2      2.390346e-01  0.3290237 0.266 0.222 1.000000e+00       0
AC112721.1      2.396134e-01  0.4542583 0.227 0.164 1.000000e+00       0
TMEM116         2.401204e-01  0.3446832 0.159 0.133 1.000000e+00       0
ZNF850          2.401208e-01  0.3236128 0.211 0.181 1.000000e+00       0
GALNT10         2.415017e-01 -0.2904977 0.458 0.411 1.000000e+00       0
ZNF561          2.451621e-01  0.3382847 0.312 0.256 1.000000e+00       0
PROCR           2.463043e-01  0.3511795 0.405 0.306 1.000000e+00       0
LINC02482       2.473030e-01  0.3834362 0.227 0.200 1.000000e+00       0
PUS7L           2.476364e-01  0.2790135 0.427 0.344 1.000000e+00       0
AL133338.1      2.497545e-01 -0.2930443 0.140 0.153 1.000000e+00       0
ZNF223          2.503519e-01  0.4289310 0.151 0.092 1.000000e+00       0
AC110285.6      2.515103e-01  0.3377320 0.157 0.147 1.000000e+00       0
ADAMTS12        2.522990e-01  0.3870801 0.272 0.178 1.000000e+00       0
PCBD1           2.526370e-01  0.2919260 0.497 0.419 1.000000e+00       0
COPZ2           2.527122e-01  0.2604668 0.451 0.406 1.000000e+00       0
FJX1            2.532012e-01  0.3472903 0.339 0.275 1.000000e+00       0
ULBP2           2.544075e-01  0.3446531 0.231 0.164 1.000000e+00       0
TMEM30A         2.553150e-01 -0.2516190 0.453 0.414 1.000000e+00       0
LIMS2           2.554665e-01 -0.4700909 0.414 0.431 1.000000e+00       0
MYL4            2.570602e-01  0.2909378 0.242 0.150 1.000000e+00       0
AC012645.1      2.572127e-01  0.4552959 0.117 0.122 1.000000e+00       0
ZSCAN31         2.572888e-01 -0.2871611 0.070 0.106 1.000000e+00       0
PTGFR           2.577457e-01 -0.2749078 0.164 0.200 1.000000e+00       0
HS3ST3A1        2.606530e-01  0.5362413 0.154 0.108 1.000000e+00       0
ANKRD37         2.610367e-01  0.8462447 0.217 0.125 1.000000e+00       0
TRIM24          2.638541e-01  0.4215272 0.298 0.233 1.000000e+00       0
AL590426.2      2.649398e-01  0.4825687 0.173 0.083 1.000000e+00       0
RAB9A           2.654453e-01 -0.2937616 0.333 0.322 1.000000e+00       0
MCTP1           2.678634e-01  0.5165322 0.167 0.136 1.000000e+00       0
ERG28           2.750911e-01  0.5265930 0.452 0.414 1.000000e+00       0
ZCCHC14         2.783321e-01  0.3194619 0.212 0.217 1.000000e+00       0
EPB41L2         2.815586e-01 -0.3187481 0.310 0.278 1.000000e+00       0
TEX29           2.839542e-01 -0.3221533 0.179 0.214 1.000000e+00       0
SCUBE3          2.855454e-01  0.4266785 0.356 0.242 1.000000e+00       0
LINC01252       2.859128e-01  0.8494154 0.204 0.150 1.000000e+00       0
SLC6A8          2.864456e-01 -0.4221929 0.352 0.336 1.000000e+00       0
AC062029.1      2.868555e-01  0.4230057 0.181 0.092 1.000000e+00       0
H2AFY           2.871836e-01 -0.5400591 0.446 0.447 1.000000e+00       0
PNPLA2          2.873478e-01 -0.2534124 0.338 0.339 1.000000e+00       0
KLRG1           2.890341e-01 -0.2541531 0.134 0.103 1.000000e+00       0
ZNF8            2.896108e-01  0.3021119 0.181 0.114 1.000000e+00       0
PCSK6           2.905203e-01 -0.4213401 0.204 0.217 1.000000e+00       0
TNFSF13B        2.905617e-01  0.5008732 0.163 0.125 1.000000e+00       0
OPTN            2.907272e-01  0.4221957 0.460 0.408 1.000000e+00       0
TM7SF3          2.913482e-01  0.2935815 0.399 0.336 1.000000e+00       0
HGD             2.948844e-01  0.6677442 0.210 0.111 1.000000e+00       0
NSG1            2.984490e-01  0.4743498 0.269 0.181 1.000000e+00       0
ZNF781          3.040840e-01  0.3553768 0.136 0.125 1.000000e+00       0
ARAP2           3.053228e-01  0.3397254 0.208 0.125 1.000000e+00       0
AGAP9           3.062214e-01 -0.3625755 0.178 0.169 1.000000e+00       0
MAP3K20         3.096608e-01  0.4052980 0.375 0.308 1.000000e+00       0
C19orf73        3.109570e-01  0.2516421 0.141 0.114 1.000000e+00       0
CHORDC1         3.114768e-01  0.3400641 0.387 0.303 1.000000e+00       0
TNRC6C          3.128650e-01  0.2658465 0.295 0.175 1.000000e+00       0
IL27RA          3.176681e-01  0.2775105 0.127 0.075 1.000000e+00       0
BZW2            3.208380e-01  0.3208964 0.420 0.356 1.000000e+00       0
FKRP            3.211472e-01 -0.3033969 0.190 0.203 1.000000e+00       0
KLF6            3.223419e-01 -0.2571610 0.386 0.403 1.000000e+00       0
DPP10           3.236284e-01  0.2833884 0.211 0.150 1.000000e+00       0
MYADM           3.236285e-01  0.3563859 0.437 0.372 1.000000e+00       0
LEFTY2          3.278213e-01  0.5432828 0.266 0.147 1.000000e+00       0
AL136084.3      3.303355e-01  0.5669666 0.172 0.081 1.000000e+00       0
TSPAN18         3.326366e-01  0.2913079 0.145 0.128 1.000000e+00       0
UGDH            3.327727e-01  0.3202143 0.447 0.433 1.000000e+00       0
BTN3A2          3.329535e-01 -0.6021307 0.191 0.278 1.000000e+00       0
RDH10           3.392808e-01  0.3523937 0.423 0.358 1.000000e+00       0
GPR161          3.394636e-01  0.7443577 0.155 0.122 1.000000e+00       0
KLF5            3.406567e-01  0.3505477 0.288 0.225 1.000000e+00       0
ATP6V0E2        3.411620e-01  0.5344972 0.210 0.136 1.000000e+00       0
CCDC81          3.412079e-01  0.8199486 0.184 0.075 1.000000e+00       0
SUPV3L1         3.430958e-01 -0.2738535 0.248 0.278 1.000000e+00       0
GASAL1          3.442962e-01  0.3921893 0.190 0.114 1.000000e+00       0
LPAR6           3.457309e-01  0.3539508 0.168 0.097 1.000000e+00       0
CTXN1           3.478678e-01  0.5782506 0.257 0.153 1.000000e+00       0
AL132656.2      3.487060e-01  0.6619804 0.157 0.064 1.000000e+00       0
TMEM154         3.495921e-01  0.3868375 0.154 0.131 1.000000e+00       0
FLRT2           3.497323e-01 -0.3252408 0.422 0.436 1.000000e+00       0
AFAP1L2         3.501994e-01  0.5357484 0.266 0.211 1.000000e+00       0
UBE2L6          3.512280e-01 -0.3356557 0.149 0.197 1.000000e+00       0
HOXB-AS1        3.515092e-01  0.5593848 0.176 0.094 1.000000e+00       0
LINC01705       3.517436e-01  0.3746317 0.171 0.106 1.000000e+00       0
HOPX            3.518614e-01  0.2802300 0.406 0.344 1.000000e+00       0
AC013652.1      3.539032e-01  0.9256999 0.176 0.122 1.000000e+00       0
IP6K3           3.545150e-01 -0.7547779 0.191 0.236 1.000000e+00       0
SQLE            3.553639e-01 -0.3845280 0.420 0.431 1.000000e+00       0
SYTL2           3.563551e-01  0.3902907 0.327 0.192 1.000000e+00       0
TTC7A           3.573483e-01  0.3887331 0.251 0.211 1.000000e+00       0
AC007938.3      3.579639e-01  0.3069171 0.218 0.153 1.000000e+00       0
PRAG1           3.600051e-01  0.5373100 0.311 0.236 1.000000e+00       0
CPPED1          3.601953e-01  0.4362022 0.320 0.275 1.000000e+00       0
LINC00626       3.611951e-01  0.2976677 0.203 0.158 1.000000e+00       0
PLPPR3          3.613854e-01  0.5012897 0.206 0.108 1.000000e+00       0
AC007878.1      3.621012e-01  0.7050165 0.149 0.064 1.000000e+00       0
SLC1A3          3.625309e-01 -0.5647816 0.117 0.172 1.000000e+00       0
TNK2            3.627689e-01 -0.2805488 0.122 0.144 1.000000e+00       0
GPT2            3.633913e-01 -0.2860605 0.293 0.311 1.000000e+00       0
APOBEC3C        3.633913e-01 -0.2706514 0.312 0.264 1.000000e+00       0
ST5             3.637741e-01  0.3187826 0.438 0.378 1.000000e+00       0
WNT11           3.670862e-01  0.7239439 0.195 0.122 1.000000e+00       0
DAAM2           3.672306e-01  0.3563774 0.248 0.164 1.000000e+00       0
PLN             3.695448e-01  0.7996471 0.228 0.072 1.000000e+00       0
GLIPR2          3.703201e-01  0.2663893 0.358 0.286 1.000000e+00       0
MIS18BP1        3.712402e-01  0.5110605 0.173 0.119 1.000000e+00       0
CWC25           3.727450e-01 -0.3344574 0.262 0.269 1.000000e+00       0
PIGU            3.728901e-01  0.3342067 0.186 0.189 1.000000e+00       0
DNASE2          3.756678e-01  0.3663534 0.344 0.297 1.000000e+00       0
ATL1            3.765963e-01  0.2702721 0.129 0.103 1.000000e+00       0
SH3BP4          3.766452e-01  0.2609666 0.256 0.217 1.000000e+00       0
UAP1            3.775262e-01 -0.3567188 0.421 0.408 1.000000e+00       0
IQCH-AS1        3.790460e-01  0.3487490 0.145 0.111 1.000000e+00       0
SLC22A2         3.792429e-01  0.6033733 0.208 0.100 1.000000e+00       0
SSH3            3.816537e-01  0.2614013 0.193 0.158 1.000000e+00       0
ASMTL           3.816544e-01  0.2847881 0.253 0.186 1.000000e+00       0
SDC4            3.828389e-01 -0.3878277 0.214 0.272 1.000000e+00       0
FNDC3B          3.840257e-01 -0.2944345 0.480 0.444 1.000000e+00       0
TMEM56          3.842729e-01  0.4435059 0.181 0.125 1.000000e+00       0
HES6            3.885457e-01  0.2662629 0.146 0.144 1.000000e+00       0
ADAM10          3.897436e-01 -0.3615668 0.456 0.425 1.000000e+00       0
ZNF839          3.901431e-01 -0.3344620 0.134 0.139 1.000000e+00       0
RB1CC1          3.902432e-01 -0.2544669 0.407 0.406 1.000000e+00       0
PLCB4           3.915941e-01  0.6440358 0.211 0.094 1.000000e+00       0
SSC5D           3.922958e-01  0.2552869 0.311 0.264 1.000000e+00       0
UBE2S           3.945562e-01 -0.4979632 0.405 0.358 1.000000e+00       0
RNF217          3.962693e-01 -0.2861579 0.464 0.483 1.000000e+00       0
ATP6V0C         3.969758e-01  0.2661819 0.147 0.144 1.000000e+00       0
TM7SF2          3.971779e-01  0.4332628 0.331 0.214 1.000000e+00       0
MYL7            3.996578e-01  0.3808387 0.210 0.103 1.000000e+00       0
SLC10A3         3.998102e-01  0.2594561 0.397 0.353 1.000000e+00       0
IL11            4.009270e-01  0.3412007 0.312 0.183 1.000000e+00       0
AAED1           4.017404e-01 -0.2875370 0.311 0.303 1.000000e+00       0
AC010976.2      4.027070e-01  0.2555707 0.172 0.133 1.000000e+00       0
AC034236.2      4.044928e-01  0.2856218 0.176 0.122 1.000000e+00       0
KIAA1217        4.079749e-01 -0.2605103 0.358 0.328 1.000000e+00       0
NKRF            4.093107e-01  0.3992004 0.158 0.131 1.000000e+00       0
GPAM            4.131798e-01 -0.2811721 0.222 0.208 1.000000e+00       0
AC112721.2      4.157190e-01  0.3755100 0.192 0.147 1.000000e+00       0
NHSL1           4.184237e-01  0.2757640 0.284 0.217 1.000000e+00       0
FHOD3           4.204590e-01  0.7067837 0.253 0.069 1.000000e+00       0
FAM43A          4.223954e-01  0.5590172 0.305 0.197 1.000000e+00       0
AC092069.1      4.239167e-01  0.4399289 0.238 0.189 1.000000e+00       0
WFDC1           4.242316e-01  0.4548154 0.185 0.144 1.000000e+00       0
SRFBP1          4.255464e-01 -0.3763289 0.285 0.267 1.000000e+00       0
MMP24OS         4.269161e-01  0.2570886 0.442 0.389 1.000000e+00       0
RASSF1          4.277603e-01 -0.3020476 0.326 0.325 1.000000e+00       0
RAB7B           4.283940e-01  0.2818239 0.181 0.122 1.000000e+00       0
SPOCD1          4.314118e-01  0.3739859 0.206 0.156 1.000000e+00       0
RCN3            4.332171e-01  0.2735697 0.249 0.200 1.000000e+00       0
LAMC2           4.382869e-01  0.9683467 0.228 0.086 1.000000e+00       0
SGCD            4.395728e-01  0.8621137 0.185 0.056 1.000000e+00       0
APOD            4.406993e-01  0.5206216 0.205 0.142 1.000000e+00       0
RPP14           4.422592e-01  0.3046291 0.229 0.211 1.000000e+00       0
TGFBR3L         4.434979e-01  0.2634244 0.289 0.233 1.000000e+00       0
SLC2A1-AS1      4.447926e-01  0.5956795 0.113 0.100 1.000000e+00       0
BLMH            4.449547e-01 -0.2544470 0.260 0.244 1.000000e+00       0
SERPINI1        4.491776e-01  0.6878585 0.217 0.125 1.000000e+00       0
NOTCH3          4.491780e-01  0.4521728 0.186 0.117 1.000000e+00       0
MYBL1           4.500475e-01  0.2687520 0.164 0.086 1.000000e+00       0
DCUN1D2         4.526055e-01 -0.3091377 0.141 0.133 1.000000e+00       0
RPN1            4.546811e-01 -0.3758543 0.416 0.433 1.000000e+00       0
B4GALT5         4.552279e-01 -0.3045316 0.357 0.364 1.000000e+00       0
FBXL22          4.594508e-01  0.4477512 0.113 0.142 1.000000e+00       0
SHB             4.606069e-01  0.2664406 0.302 0.239 1.000000e+00       0
AC020571.1      4.636963e-01  0.2744493 0.165 0.111 1.000000e+00       0
RNASEH2C        4.726431e-01 -0.3626885 0.437 0.408 1.000000e+00       0
PCM1            4.726990e-01 -0.3346023 0.378 0.353 1.000000e+00       0
TMC6            4.735371e-01  0.3220636 0.129 0.097 1.000000e+00       0
JUNB            4.737052e-01 -0.5213744 0.368 0.375 1.000000e+00       0
EEA1            4.760575e-01  0.3104168 0.349 0.306 1.000000e+00       0
COL4A4          4.766741e-01  0.3371991 0.157 0.125 1.000000e+00       0
LOXL1-AS1       4.776854e-01  0.2544649 0.421 0.406 1.000000e+00       0
AQP3            4.784720e-01  0.4244691 0.164 0.133 1.000000e+00       0
PHKG1           4.788661e-01  0.2779182 0.278 0.236 1.000000e+00       0
STARD4-AS1      4.789787e-01 -0.3081728 0.397 0.328 1.000000e+00       0
KCNAB3          4.811196e-01 -0.2897165 0.159 0.161 1.000000e+00       0
PLAUR           4.824180e-01  0.4223972 0.335 0.311 1.000000e+00       0
PCNA            4.827570e-01 -0.2768836 0.369 0.361 1.000000e+00       0
DNMBP           4.847372e-01  0.3478820 0.212 0.161 1.000000e+00       0
LIPG            4.848504e-01  0.5594677 0.160 0.097 1.000000e+00       0
AL451050.2      4.850204e-01  0.2854534 0.186 0.108 1.000000e+00       0
FABP5           4.853037e-01  0.5249679 0.313 0.244 1.000000e+00       0
PANK1           4.868921e-01 -0.4567779 0.219 0.197 1.000000e+00       0
CEP350          4.871760e-01 -0.3519546 0.344 0.328 1.000000e+00       0
ZFYVE21         4.883694e-01  0.3073245 0.425 0.361 1.000000e+00       0
SYNE2           4.886535e-01 -0.2949954 0.158 0.144 1.000000e+00       0
KIFC2           4.921302e-01  0.7439505 0.132 0.114 1.000000e+00       0
COMT            4.933296e-01  0.5665373 0.294 0.200 1.000000e+00       0
LINC01534       4.935590e-01  0.2886924 0.174 0.103 1.000000e+00       0
TRIM58          4.945884e-01  0.6011215 0.206 0.114 1.000000e+00       0
ITGA10          4.986056e-01 -0.3038012 0.136 0.186 1.000000e+00       0
NUDT6           4.990654e-01  0.2697786 0.183 0.197 1.000000e+00       0
AC069185.1      4.998716e-01  0.4733160 0.147 0.136 1.000000e+00       0
RBM24           5.085481e-01  0.3251418 0.267 0.114 1.000000e+00       0
C8orf88         5.110498e-01 -0.4302266 0.134 0.142 1.000000e+00       0
PPP5D1          5.112828e-01 -0.7139129 0.095 0.108 1.000000e+00       0
RASGRF2         5.123324e-01  0.4819456 0.331 0.247 1.000000e+00       0
CDC42SE1        5.142592e-01  0.3898454 0.446 0.414 1.000000e+00       0
KIAA1755        5.153117e-01  0.5752340 0.147 0.069 1.000000e+00       0
PSPH            5.156043e-01  0.4513499 0.243 0.172 1.000000e+00       0
PTGFRN          5.174203e-01  0.5013901 0.202 0.125 1.000000e+00       0
MMP24           5.178886e-01  0.5159517 0.173 0.122 1.000000e+00       0
DMKN            5.255439e-01  0.2606580 0.180 0.153 1.000000e+00       0
PLK2            5.269634e-01  0.4163229 0.260 0.189 1.000000e+00       0
MYL10           5.279700e-01 -0.2632979 0.327 0.333 1.000000e+00       0
PLEKHH2         5.335535e-01  0.3227264 0.170 0.139 1.000000e+00       0
KCNJ2           5.349246e-01  0.3842916 0.157 0.089 1.000000e+00       0
LTBP1           5.371930e-01  0.3993022 0.406 0.408 1.000000e+00       0
AC132872.1      5.383290e-01 -0.4942471 0.140 0.161 1.000000e+00       0
SVIP            5.399459e-01  0.3846991 0.247 0.178 1.000000e+00       0
RAP1B           5.430661e-01  0.2889467 0.393 0.331 1.000000e+00       0
AC044849.1      5.441482e-01 -0.3262317 0.114 0.144 1.000000e+00       0
PNP             5.443287e-01 -0.3278650 0.263 0.269 1.000000e+00       0
ZGLP1           5.448702e-01  0.6782762 0.148 0.100 1.000000e+00       0
TUSC3           5.470392e-01  0.2810630 0.448 0.431 1.000000e+00       0
APBA2           5.488496e-01  0.3807861 0.142 0.111 1.000000e+00       0
TTC27           5.493328e-01  0.4397876 0.187 0.178 1.000000e+00       0
TMEM200A        5.507846e-01  0.4257004 0.298 0.247 1.000000e+00       0
RASSF8          5.521166e-01  0.2639239 0.428 0.442 1.000000e+00       0
PCDHA10         5.527226e-01  0.3548558 0.154 0.156 1.000000e+00       0
THAP7-AS1       5.539961e-01 -0.3857707 0.160 0.167 1.000000e+00       0
AP000487.1      5.543604e-01  0.5272028 0.153 0.089 1.000000e+00       0
ID2             5.552106e-01  0.6170855 0.208 0.125 1.000000e+00       0
SHMT2           5.602640e-01 -0.2931023 0.412 0.406 1.000000e+00       0
AL441883.1      5.607521e-01  0.4785410 0.141 0.064 1.000000e+00       0
ARHGEF3         5.615460e-01  0.4138432 0.238 0.183 1.000000e+00       0
GFOD1           5.701292e-01  0.4837452 0.160 0.122 1.000000e+00       0
SSC4D           5.721615e-01 -0.4625374 0.085 0.131 1.000000e+00       0
NTRK2           5.737648e-01  0.5702884 0.236 0.139 1.000000e+00       0
CTSC            5.748149e-01  0.2815465 0.237 0.200 1.000000e+00       0
ZNF117          5.758036e-01  0.4825928 0.205 0.156 1.000000e+00       0
FAM122A         5.790846e-01  0.3830801 0.313 0.239 1.000000e+00       0
ANXA3           5.833064e-01  0.3688320 0.181 0.169 1.000000e+00       0
ZSCAN25         5.873557e-01  0.7368328 0.174 0.125 1.000000e+00       0
KCNQ5           5.875433e-01  0.2757537 0.208 0.103 1.000000e+00       0
TRABD2B         5.919821e-01  0.3646814 0.217 0.197 1.000000e+00       0
AXIN1           5.928586e-01  0.3904214 0.138 0.131 1.000000e+00       0
FAM69A          5.931724e-01 -0.3546860 0.136 0.136 1.000000e+00       0
ZNF121          5.967504e-01  0.4765644 0.247 0.183 1.000000e+00       0
LRRC45          5.970016e-01  0.4920938 0.121 0.114 1.000000e+00       0
GBP2            5.979449e-01  0.4129112 0.154 0.128 1.000000e+00       0
SPA17           5.980707e-01  0.4064131 0.163 0.119 1.000000e+00       0
ZNF329          6.000223e-01 -0.3683885 0.155 0.142 1.000000e+00       0
HES1            6.025454e-01  0.4025164 0.321 0.242 1.000000e+00       0
SEMA4F          6.054528e-01 -0.4413923 0.210 0.183 1.000000e+00       0
FGD6            6.057692e-01  0.3149622 0.261 0.225 1.000000e+00       0
HOXC6           6.065921e-01  0.3664895 0.388 0.336 1.000000e+00       0
FANCB           6.070352e-01  0.4993232 0.180 0.089 1.000000e+00       0
HINT1           6.080493e-01 -0.4313287 0.472 0.472 1.000000e+00       0
AC078850.1      6.084930e-01  0.5124039 0.208 0.117 1.000000e+00       0
GPX7            6.093810e-01  0.3331306 0.279 0.236 1.000000e+00       0
RCAN2           6.098252e-01  0.3317738 0.189 0.147 1.000000e+00       0
AP000866.2      6.131297e-01 -0.4911183 0.136 0.144 1.000000e+00       0
FAM19A2         6.164417e-01  0.6236871 0.158 0.103 1.000000e+00       0
RALGPS2         6.166334e-01  0.3947273 0.384 0.333 1.000000e+00       0
MOB3B           6.222573e-01  0.2922271 0.280 0.242 1.000000e+00       0
JAG1            6.238591e-01  0.5548905 0.208 0.131 1.000000e+00       0
MRPL35          6.243079e-01  0.4378073 0.316 0.250 1.000000e+00       0
AHNAK2          6.262330e-01 -0.2840813 0.295 0.278 1.000000e+00       0
RASL11B         6.284176e-01  0.7378570 0.211 0.094 1.000000e+00       0
RTL5            6.309273e-01 -0.3086910 0.179 0.169 1.000000e+00       0
CSNK1G1         6.317009e-01  0.2957652 0.263 0.236 1.000000e+00       0
SLC38A2         6.324103e-01 -0.3442455 0.453 0.492 1.000000e+00       0
CPA4            6.331149e-01  0.4402281 0.336 0.228 1.000000e+00       0
DDAH2           6.379661e-01  0.2599462 0.374 0.306 1.000000e+00       0
AC097478.1      6.400385e-01  0.9165343 0.183 0.092 1.000000e+00       0
POLE4           6.415947e-01 -0.3556930 0.442 0.422 1.000000e+00       0
ACOX1           6.433473e-01  0.4286206 0.435 0.417 1.000000e+00       0
TGIF2           6.439320e-01  0.4511626 0.236 0.158 1.000000e+00       0
ITGA4           6.473793e-01  0.4248146 0.170 0.072 1.000000e+00       0
TTC30B          6.476398e-01  0.4959303 0.114 0.058 1.000000e+00       0
ZCWPW1          6.477049e-01 -0.3811382 0.159 0.172 1.000000e+00       0
ARID1B          6.481610e-01 -0.2652458 0.452 0.422 1.000000e+00       0
PLD5            6.497902e-01  0.4291223 0.199 0.144 1.000000e+00       0
ZNF699          6.516173e-01  0.2868814 0.185 0.128 1.000000e+00       0
BCL10           6.518788e-01  0.4130658 0.352 0.311 1.000000e+00       0
NFYA            6.520745e-01  0.6658042 0.180 0.133 1.000000e+00       0
BAG2            6.563909e-01  0.2551358 0.412 0.347 1.000000e+00       0
SRSF3           6.579634e-01 -0.3971260 0.427 0.428 1.000000e+00       0
TMTC2           6.594060e-01  0.2638138 0.223 0.167 1.000000e+00       0
RNF150          6.607187e-01  0.5099790 0.134 0.044 1.000000e+00       0
PDCL3           6.685504e-01  0.4450588 0.281 0.236 1.000000e+00       0
SLC25A25-AS1    6.700681e-01  0.2974757 0.115 0.136 1.000000e+00       0
ZNF639          6.702663e-01 -0.4425341 0.338 0.319 1.000000e+00       0
JOSD2           6.715214e-01  0.2583692 0.464 0.492 1.000000e+00       0
TUBB4B          6.754908e-01 -0.2756975 0.464 0.469 1.000000e+00       0
NCAPD3          6.760207e-01  0.4228106 0.250 0.225 1.000000e+00       0
GINS4           6.829243e-01  0.4771867 0.183 0.119 1.000000e+00       0
TGFBR2          6.835231e-01  0.2902454 0.445 0.417 1.000000e+00       0
C2orf69         6.843883e-01  0.2620795 0.231 0.178 1.000000e+00       0
SPRY2           6.845879e-01  0.5536472 0.308 0.281 1.000000e+00       0
GPNMB           6.875862e-01 -0.3800380 0.287 0.286 1.000000e+00       0
ARL6IP1         6.895877e-01 -0.4808206 0.445 0.431 1.000000e+00       0
PEX3            6.899883e-01 -0.3017149 0.218 0.203 1.000000e+00       0
LTC4S           6.911903e-01  0.3848120 0.173 0.119 1.000000e+00       0
HMCN1           6.974814e-01  0.3432204 0.286 0.167 1.000000e+00       0
CCDC138         6.978166e-01  0.3296586 0.141 0.097 1.000000e+00       0
RAD51B          7.005672e-01  0.5157354 0.236 0.147 1.000000e+00       0
URAD            7.037931e-01  0.2504979 0.183 0.125 1.000000e+00       0
HES4            7.070916e-01  0.2631096 0.325 0.311 1.000000e+00       0
MVK             7.096532e-01  0.3965058 0.295 0.225 1.000000e+00       0
KPNA2           7.122857e-01 -0.4529407 0.377 0.367 1.000000e+00       0
EPS8L2          7.177642e-01  0.3521490 0.255 0.219 1.000000e+00       0
NOG             7.195258e-01  0.4872516 0.353 0.233 1.000000e+00       0
AL078639.1      7.208141e-01 -0.4144212 0.344 0.350 1.000000e+00       0
SLC27A5         7.228498e-01  0.2779029 0.300 0.261 1.000000e+00       0
PCDH18          7.230534e-01  0.3201270 0.317 0.206 1.000000e+00       0
VPS33B          7.234609e-01  0.4191773 0.135 0.100 1.000000e+00       0
FDPS            7.252273e-01  0.3617028 0.421 0.433 1.000000e+00       0
NET1            7.275395e-01  0.3279045 0.308 0.250 1.000000e+00       0
PLCE1           7.294454e-01  0.4137703 0.270 0.225 1.000000e+00       0
IFNAR2          7.324439e-01  0.5694366 0.265 0.183 1.000000e+00       0
LIF             7.340129e-01  0.3137829 0.230 0.161 1.000000e+00       0
TFPI2           7.382476e-01  0.2565284 0.248 0.128 1.000000e+00       0
ANO7            7.391369e-01  0.2726669 0.202 0.103 1.000000e+00       0
AL163051.1      7.429703e-01  0.3391807 0.114 0.069 1.000000e+00       0
C5orf30         7.477024e-01  0.3772545 0.145 0.108 1.000000e+00       0
DOHH            7.497631e-01  0.3643095 0.251 0.183 1.000000e+00       0
ABL2            7.513440e-01  0.4446599 0.371 0.356 1.000000e+00       0
B3GALNT2        7.514127e-01  0.5255380 0.260 0.192 1.000000e+00       0
HIST1H4E        7.569886e-01  0.3043885 0.152 0.136 1.000000e+00       0
MRAP2           7.639591e-01  0.8777495 0.298 0.131 1.000000e+00       0
NAV3            7.645119e-01  0.3960079 0.203 0.144 1.000000e+00       0
SLC39A6         7.674862e-01 -0.3535839 0.469 0.442 1.000000e+00       0
GPRC5A          7.693553e-01  0.7597358 0.282 0.200 1.000000e+00       0
KLHL21          7.697017e-01 -0.3809772 0.297 0.292 1.000000e+00       0
POLR1B          7.702557e-01  0.2797374 0.168 0.144 1.000000e+00       0
PDGFA           7.752487e-01  0.3135472 0.357 0.306 1.000000e+00       0
PITHD1          7.762206e-01 -0.2869614 0.395 0.386 1.000000e+00       0
ABLIM1          7.768455e-01  0.4240263 0.224 0.164 1.000000e+00       0
FAM107B         7.802508e-01  0.3766180 0.273 0.250 1.000000e+00       0
RAB3IP          7.890263e-01  0.2571133 0.266 0.194 1.000000e+00       0
SYN1            7.892355e-01 -0.2658142 0.172 0.158 1.000000e+00       0
BMPR2           7.900031e-01  0.3539019 0.478 0.461 1.000000e+00       0
THAP2           7.907707e-01  0.2701690 0.247 0.203 1.000000e+00       0
IDH3B           7.958704e-01 -0.3205089 0.330 0.311 1.000000e+00       0
MRS2            7.963598e-01  0.2828044 0.345 0.275 1.000000e+00       0
IPMK            7.966392e-01  0.3853867 0.173 0.122 1.000000e+00       0
ARNTL           7.970589e-01  0.3745359 0.163 0.122 1.000000e+00       0
ZNF628          7.973390e-01  0.6877786 0.121 0.097 1.000000e+00       0
MLPH            8.009085e-01  0.5159283 0.165 0.142 1.000000e+00       0
RECK            8.028002e-01 -0.2520129 0.403 0.361 1.000000e+00       0
TRIM22          8.037815e-01  0.3908399 0.344 0.272 1.000000e+00       0
TMEM147-AS1     8.056045e-01  0.3086357 0.116 0.106 1.000000e+00       0
SRGAP2C         8.060956e-01  0.2622342 0.306 0.289 1.000000e+00       0
SVIL            8.091135e-01 -0.3952347 0.270 0.275 1.000000e+00       0
ATRNL1          8.137512e-01  0.4940283 0.281 0.236 1.000000e+00       0
AL118516.1      8.220592e-01  0.3296707 0.284 0.239 1.000000e+00       0
EGR2            8.224821e-01  0.2834142 0.159 0.136 1.000000e+00       0
HSPB8           8.255150e-01  0.2701099 0.278 0.256 1.000000e+00       0
AADAT           8.257972e-01  0.3645712 0.096 0.114 1.000000e+00       0
NCKAP1          8.265736e-01 -0.3399516 0.497 0.478 1.000000e+00       0
ADAM33          8.274207e-01 -0.2917635 0.204 0.192 1.000000e+00       0
EHD3            8.315885e-01  0.2836953 0.199 0.161 1.000000e+00       0
NEGR1           8.316591e-01  0.3101416 0.127 0.103 1.000000e+00       0
DYNC1H1         8.344875e-01 -0.2709156 0.456 0.472 1.000000e+00       0
ZEB2            8.371057e-01  0.2733780 0.453 0.439 1.000000e+00       0
FIRRE           8.388757e-01  0.2950846 0.206 0.139 1.000000e+00       0
SUMF1           8.433398e-01  0.2613471 0.362 0.308 1.000000e+00       0
PDZRN4          8.467444e-01  0.6004046 0.190 0.119 1.000000e+00       0
LONRF1          8.474539e-01 -0.3539423 0.147 0.186 1.000000e+00       0
TNFSF4          8.497968e-01  0.5319570 0.209 0.111 1.000000e+00       0
KCNMB4          8.561929e-01  0.2721100 0.311 0.211 1.000000e+00       0
CAVIN4          8.579000e-01  1.0719427 0.244 0.144 1.000000e+00       0
DHRS2           8.594656e-01  0.7880589 0.268 0.133 1.000000e+00       0
TRIO            8.613165e-01  0.2884835 0.340 0.308 1.000000e+00       0
RPS4Y1          8.651629e-01  0.7167079 0.282 0.244 1.000000e+00       0
AC018647.1      8.730797e-01  0.3482380 0.186 0.114 1.000000e+00       0
HMCES           8.731512e-01  0.3244965 0.254 0.236 1.000000e+00       0
KCNN2           8.735080e-01  0.6056511 0.243 0.114 1.000000e+00       0
HIST1H2BD       8.777930e-01  0.2930681 0.165 0.111 1.000000e+00       0
TUBA4A          8.797225e-01 -0.3266155 0.210 0.225 1.000000e+00       0
CSRNP3          8.815097e-01  0.8684156 0.206 0.106 1.000000e+00       0
TAS2R14         8.824393e-01  0.4071474 0.165 0.097 1.000000e+00       0
MCAM            8.828684e-01  0.5148529 0.195 0.131 1.000000e+00       0
BX255923.1      8.829399e-01  0.4164939 0.204 0.131 1.000000e+00       0
PRRC2A          8.879485e-01 -0.3886249 0.362 0.375 1.000000e+00       0
CKS2            8.883780e-01 -0.2597309 0.351 0.317 1.000000e+00       0
LRP1            8.884496e-01 -0.3149710 0.429 0.444 1.000000e+00       0
NLRP1           8.956130e-01  0.3999722 0.340 0.286 1.000000e+00       0
CLEC3A          8.969033e-01  0.3876148 0.367 0.267 1.000000e+00       0
SEC14L2         8.976202e-01  0.4458312 0.121 0.106 1.000000e+00       0
ANKRD29         8.990545e-01  0.3714146 0.133 0.089 1.000000e+00       0
OLR1            9.002021e-01  0.5259480 0.204 0.183 1.000000e+00       0
CPVL            9.012782e-01  0.2608807 0.238 0.183 1.000000e+00       0
CCDC51          9.019239e-01  0.2959737 0.311 0.258 1.000000e+00       0
FLT1            9.024979e-01 -0.3769830 0.454 0.456 1.000000e+00       0
TGFB2           9.047945e-01  0.3512401 0.253 0.183 1.000000e+00       0
FYN             9.050099e-01  0.3503609 0.229 0.186 1.000000e+00       0
TRAPPC10        9.055842e-01  0.2725939 0.266 0.239 1.000000e+00       0
ZNF574          9.087437e-01  0.2874582 0.205 0.142 1.000000e+00       0
OPHN1           9.088155e-01  0.2613804 0.172 0.114 1.000000e+00       0
CXADR           9.106112e-01  0.8780648 0.202 0.083 1.000000e+00       0
LYSMD2          9.129825e-01  0.9456572 0.197 0.128 1.000000e+00       0
NXPH4           9.155703e-01  0.3531486 0.251 0.150 1.000000e+00       0
PRUNE2          9.157859e-01 -0.4358781 0.164 0.189 1.000000e+00       0
MPP5            9.167206e-01  0.2776065 0.304 0.250 1.000000e+00       0
TMEM65          9.187342e-01  0.2519556 0.267 0.261 1.000000e+00       0
RTKN2           9.195973e-01  0.7749476 0.181 0.075 1.000000e+00       0
MDM2            9.244900e-01 -0.3262176 0.483 0.447 1.000000e+00       0
IRF2BP2         9.252098e-01 -0.2807289 0.437 0.378 1.000000e+00       0
RAB40C          9.263615e-01  0.2985306 0.181 0.144 1.000000e+00       0
KMO             9.269375e-01  0.6702885 0.171 0.075 1.000000e+00       0
HIST1H1E        9.300338e-01  0.3142719 0.128 0.094 1.000000e+00       0
HOXA10          9.301058e-01  0.2823818 0.473 0.461 1.000000e+00       0
CAMK4           9.306819e-01  0.4699024 0.178 0.100 1.000000e+00       0
SRPX            9.347161e-01  0.3854421 0.377 0.325 1.000000e+00       0
TSPAN12         9.361574e-01  0.5445723 0.128 0.092 1.000000e+00       0
PKN1            9.406986e-01 -0.2621015 0.284 0.283 1.000000e+00       0
SPCS3           9.412754e-01 -0.2870091 0.461 0.458 1.000000e+00       0
CCDC74A         9.455302e-01  0.3455699 0.149 0.106 1.000000e+00       0
SUSD6           9.482714e-01  0.3561859 0.285 0.219 1.000000e+00       0
CYSTM1          9.510854e-01  0.2799079 0.387 0.317 1.000000e+00       0
C21orf2         9.525287e-01 -0.2503969 0.285 0.264 1.000000e+00       0
TALDO1          9.548383e-01 -0.3219585 0.432 0.428 1.000000e+00       0
SYNGR1          9.580146e-01 -0.2732650 0.253 0.256 1.000000e+00       0
FANCI           9.593142e-01 -0.6001984 0.109 0.139 1.000000e+00       0
AC006480.2      9.599641e-01  0.2876421 0.139 0.100 1.000000e+00       0
NKX2-5          9.607584e-01  0.3457451 0.191 0.158 1.000000e+00       0
CDC6            9.615527e-01  0.3901790 0.185 0.128 1.000000e+00       0
CDH6            9.650193e-01  0.4884077 0.289 0.203 1.000000e+00       0
ATP6AP1L        9.650193e-01  0.3145814 0.211 0.139 1.000000e+00       0
AC008467.1      9.657416e-01  0.7008067 0.171 0.069 1.000000e+00       0
MACROD2         9.661027e-01  0.3197299 0.181 0.106 1.000000e+00       0
ZFHX4           9.680531e-01  0.3729442 0.257 0.197 1.000000e+00       0
ANKRD34A        9.689922e-01  0.3303077 0.186 0.083 1.000000e+00       0
IL17RE          9.695702e-01 -0.3075509 0.123 0.133 1.000000e+00       0
ZNF689          9.696424e-01  0.4042504 0.168 0.169 1.000000e+00       0
DGKI            9.718821e-01  0.4634006 0.219 0.139 1.000000e+00       0
IGFL2           9.749143e-01  0.3046545 0.355 0.339 1.000000e+00       0
MBNL3           9.754950e-01 -0.2730879 0.117 0.142 1.000000e+00       0
SMCR8           9.785303e-01  0.2815855 0.219 0.206 1.000000e+00       0
BMP6            9.799757e-01  0.3499698 0.330 0.214 1.000000e+00       0
URB2            9.809153e-01  0.4093974 0.181 0.122 1.000000e+00       0
EZH2            9.814212e-01  0.4464587 0.172 0.119 1.000000e+00       0
CRK             9.823609e-01 -0.2845620 0.432 0.431 1.000000e+00       0
CTSD            9.873485e-01 -0.3148800 0.438 0.472 1.000000e+00       0
KRTCAP2         9.931318e-01 -0.2597638 0.434 0.458 1.000000e+00       0
GGH             9.962406e-01 -0.4306276 0.319 0.292 1.000000e+00       0
ZNF473          9.976142e-01 -0.2705643 0.134 0.161 1.000000e+00       0
DPH6            9.987709e-01  0.4231482 0.195 0.111 1.000000e+00       0
TPM1.1          5.494333e-68 -1.9807393 0.109 0.563 2.496076e-64       1
FTH1.1          1.567303e-61  1.7118215 0.807 0.338 7.120258e-58       1
CALD1.1         1.654852e-53 -1.3838049 0.105 0.528 7.517995e-50       1
IGFBP7.1        5.239046e-52 -2.0159223 0.095 0.570 2.380098e-48       1
TAGLN.1         7.411685e-51 -2.0216352 0.161 0.553 3.367128e-47       1
CCND2.1         5.212724e-49 -1.5264324 0.109 0.527 2.368140e-45       1
TSC22D3.1       4.467123e-46  1.9297941 0.716 0.296 2.029414e-42       1
GLIPR1.1        2.722396e-45 -2.2225187 0.091 0.477 1.236785e-41       1
TPM2.1          2.291751e-44 -1.4852248 0.147 0.527 1.041142e-40       1
PLS3.1          1.236038e-43 -1.5432550 0.137 0.508 5.615320e-40       1
VIM.1           2.352770e-38 -1.3675103 0.182 0.508 1.068863e-34       1
ITGA11          7.224677e-38 -0.8874381 0.186 0.521 3.282171e-34       1
SAA1.1          4.677765e-36  1.5013914 0.642 0.233 2.125109e-32       1
FKBP5.1         2.779219e-35  1.6824437 0.639 0.264 1.262599e-31       1
CEBPD.1         4.653991e-34  1.3342733 0.709 0.357 2.114308e-30       1
SPINT2.1        1.009929e-33 -1.5117828 0.179 0.502 4.588109e-30       1
TIMP3.1         1.406196e-33 -1.8548933 0.144 0.484 6.388350e-30       1
INHBA.1         1.482010e-33 -2.1063593 0.116 0.483 6.732774e-30       1
GLS             5.099803e-33 -1.2040314 0.182 0.512 2.316840e-29       1
ACTN1.1         1.000196e-32 -0.9084404 0.204 0.533 4.543889e-29       1
PDPN.1          1.816278e-32  1.6315720 0.691 0.344 8.251350e-29       1
TIMP2.1         6.659041e-32 -0.8653180 0.228 0.522 3.025202e-28       1
ACTA2.1         4.169612e-31 -1.1705348 0.218 0.553 1.894255e-27       1
OTULINL.1       5.753112e-31  1.9017704 0.604 0.236 2.613639e-27       1
COL12A1.1       7.984929e-31 -1.8184121 0.123 0.465 3.627553e-27       1
ACTB.1          8.399203e-31 -0.8518642 0.235 0.548 3.815758e-27       1
COL5A2          8.437559e-31 -1.1311348 0.228 0.519 3.833183e-27       1
C12orf75.1      2.682225e-30 -0.6666285 0.168 0.486 1.218535e-26       1
XRCC4.1         9.407675e-30 -1.6991279 0.140 0.496 4.273907e-26       1
IGF2.1.1        2.559635e-29  1.6725326 0.600 0.251 1.162842e-25       1
GYPC.1          4.193431e-28  1.5534581 0.530 0.182 1.905076e-24       1
MFGE8.1         6.313599e-28  1.2292488 0.688 0.368 2.868268e-24       1
KCTD16.1        6.478114e-28 -1.5068745 0.126 0.474 2.943007e-24       1
PLAC9.1         1.245017e-27 -1.0225225 0.182 0.458 5.656111e-24       1
GPX4.1          7.993167e-27  0.9411543 0.684 0.365 3.631296e-23       1
CRIM1           6.153340e-26 -0.9013108 0.172 0.464 2.795462e-22       1
MYL9.1          1.406244e-25 -1.0541380 0.211 0.486 6.388567e-22       1
COL4A1          2.571680e-25 -0.7382457 0.270 0.538 1.168314e-21       1
RPS27.1         6.529834e-25  0.8989280 0.698 0.353 2.966504e-21       1
CRISPLD2.1      1.294195e-24  1.2055261 0.632 0.319 5.879529e-21       1
IVNS1ABP        1.716006e-24 -0.9410137 0.186 0.488 7.795814e-21       1
CSRP2.1         4.161010e-24 -0.7530827 0.186 0.483 1.890347e-20       1
BRI3.1          1.265924e-23  0.8005905 0.684 0.365 5.751094e-20       1
PDCD4.1         3.667293e-23  0.8353518 0.639 0.352 1.666051e-19       1
CCND1.1         6.063854e-23 -0.7691728 0.242 0.508 2.754809e-19       1
EIF1.1          2.141184e-22  1.1446389 0.663 0.379 9.727398e-19       1
CSF1.1          4.218193e-22  0.9160781 0.323 0.131 1.916325e-18       1
CADM1.1         5.316712e-22 -1.0033557 0.207 0.472 2.415382e-18       1
TMEM141.1       7.156459e-22  1.1854896 0.604 0.349 3.251179e-18       1
IL1R1.1         1.243833e-21  1.5837101 0.491 0.199 5.650732e-18       1
THBS1.1         2.417554e-21 -1.4119691 0.179 0.448 1.098295e-17       1
FTL             4.897161e-21  0.4903887 0.670 0.386 2.224780e-17       1
MARCKSL1.1      7.507159e-21 -1.4546158 0.168 0.464 3.410502e-17       1
CNN3.1          1.481257e-20 -0.6994275 0.256 0.515 6.729350e-17       1
RAB11FIP1.1     3.155620e-20  0.9873551 0.618 0.368 1.433598e-16       1
SOX4.1          5.530424e-20 -1.1972331 0.232 0.489 2.512471e-16       1
B2M.1           6.237002e-20  1.1174890 0.642 0.358 2.833470e-16       1
BTG1.1          1.060418e-19  1.3324339 0.561 0.285 4.817477e-16       1
CDKN1C.1        1.389547e-19  1.2553869 0.607 0.356 6.312712e-16       1
EIF4EBP1.1      1.527766e-19  1.1140280 0.625 0.361 6.940642e-16       1
GPC4.1          1.848417e-19 -1.6370018 0.144 0.461 8.397358e-16       1
PPP1CB.1        2.376404e-19  0.7901530 0.660 0.383 1.079600e-15       1
AC092171.3      1.107734e-18  0.3783731 0.242 0.103 5.032437e-15       1
NFIL3.1         1.206485e-18  1.0185807 0.474 0.234 5.481062e-15       1
PRLR.1          1.240443e-18  1.6554687 0.425 0.162 5.635331e-15       1
MARCKS.1        1.408630e-18 -0.8082815 0.288 0.487 6.399407e-15       1
UACA.1          1.495663e-18 -1.1716644 0.221 0.485 6.794797e-15       1
ANOS1.1         1.716522e-18 -1.4859463 0.130 0.430 7.798158e-15       1
MFAP5.1         2.199051e-18 -1.9540692 0.130 0.447 9.990289e-15       1
DPYSL2.1        2.208939e-18  1.2017113 0.568 0.278 1.003521e-14       1
ANXA2.1         2.591584e-18 -0.6912368 0.249 0.491 1.177356e-14       1
OGN.1           3.064396e-18 -1.0047252 0.158 0.430 1.392155e-14       1
TUBA1A.1        5.463973e-18 -1.3332470 0.175 0.463 2.482283e-14       1
CDKN2A.1        6.163965e-18 -1.4922175 0.204 0.479 2.800289e-14       1
MYL12A.1        6.217005e-18 -0.8128549 0.256 0.492 2.824385e-14       1
CEBPB.1         7.595143e-18  1.1332615 0.639 0.362 3.450473e-14       1
LMO4.1          8.061061e-18 -0.4114143 0.253 0.484 3.662140e-14       1
LIMA1.1         8.150653e-18 -1.0405444 0.256 0.486 3.702841e-14       1
AQP1.1          1.102103e-17 -1.3287351 0.179 0.478 5.006852e-14       1
CST3.1          1.143337e-17  0.8612491 0.632 0.393 5.194182e-14       1
MMP2.1          2.674854e-17 -0.9534375 0.270 0.469 1.215186e-13       1
HOXD11.1        5.511205e-17  0.9439958 0.449 0.244 2.503741e-13       1
COL4A2          6.056869e-17 -0.3715326 0.302 0.545 2.751636e-13       1
ADGRG6.1        7.282467e-17  0.8762669 0.323 0.107 3.308425e-13       1
HLA-C.1         7.397899e-17  1.0643912 0.611 0.358 3.360865e-13       1
IQGAP2.1        7.727957e-17  1.2453148 0.474 0.203 3.510811e-13       1
FALEC.1         8.167389e-17  0.3104675 0.270 0.122 3.710445e-13       1
SERPINF1.1      8.383428e-17  1.1574372 0.495 0.284 3.808591e-13       1
MYL6.1          1.702846e-16 -0.5397658 0.249 0.504 7.736031e-13       1
C9orf16.1       1.763624e-16 -0.7041349 0.267 0.504 8.012145e-13       1
CCNO.1          2.321998e-16  0.4276381 0.242 0.109 1.054883e-12       1
CEP128.1        3.798126e-16  0.6491713 0.200 0.108 1.725489e-12       1
ODAPH.1         4.101043e-16 -1.6764782 0.088 0.388 1.863104e-12       1
TXNDC16.1       5.206672e-16  0.9928651 0.544 0.309 2.365391e-12       1
SAT1.1          5.577379e-16  1.2306748 0.582 0.363 2.533803e-12       1
ADAM12          6.527105e-16 -0.5782921 0.274 0.481 2.965264e-12       1
LZTS1.1         6.685651e-16 -0.8827046 0.137 0.460 3.037291e-12       1
MKS1.1          7.411206e-16  0.3309958 0.200 0.123 3.366911e-12       1
ZFAS1.1         7.812730e-16  1.0636675 0.575 0.381 3.549323e-12       1
AKAP12.1        8.241399e-16  1.2330356 0.382 0.135 3.744067e-12       1
L2HGDH.1        9.082831e-16  0.5170314 0.340 0.130 4.126330e-12       1
YBX3.1          1.413282e-15  1.0286945 0.611 0.407 6.420538e-12       1
MSX1.1          1.657555e-15 -1.4652162 0.074 0.295 7.530274e-12       1
PPIB            1.895601e-15 -0.5966204 0.270 0.471 8.611716e-12       1
CDCA8.1         2.491863e-15  0.5543984 0.196 0.066 1.132053e-11       1
FN1.1           2.513303e-15 -0.6474139 0.288 0.505 1.141794e-11       1
CDH2            2.591509e-15 -0.7770565 0.281 0.517 1.177323e-11       1
PTGER2.1        2.729031e-15  0.6215482 0.607 0.378 1.239799e-11       1
ZIK1.1          3.007090e-15  0.6612547 0.298 0.118 1.366121e-11       1
CCDC171.1       3.141548e-15  0.6790048 0.270 0.108 1.427205e-11       1
DDIT4.1         4.489356e-15  0.8602619 0.498 0.254 2.039514e-11       1
FOXO1.1         4.577447e-15  1.0119273 0.530 0.317 2.079534e-11       1
CAVIN1          5.442231e-15 -0.6691218 0.298 0.503 2.472406e-11       1
FLNA            5.803567e-15 -0.2539839 0.284 0.485 2.636561e-11       1
CALR.1          7.057013e-15 -0.9412584 0.274 0.471 3.206001e-11       1
CKAP4           1.872028e-14 -0.6220794 0.302 0.520 8.504622e-11       1
LMO7.1          1.963777e-14 -0.7665294 0.189 0.487 8.921440e-11       1
IRS2.1          2.005954e-14  1.0578623 0.509 0.253 9.113051e-11       1
SRPX2.1         2.115357e-14 -0.8641929 0.235 0.465 9.610065e-11       1
ITPR2.1         2.125010e-14  0.8423060 0.607 0.388 9.653919e-11       1
TMEM120A.1      2.152572e-14  0.8353628 0.530 0.343 9.779134e-11       1
GGT5.1          2.656548e-14  1.2917297 0.407 0.154 1.206870e-10       1
LAMTOR5.1       2.670680e-14  0.6922620 0.604 0.418 1.213290e-10       1
PDGFRA.1        3.409565e-14  0.9794239 0.509 0.264 1.548966e-10       1
TXNRD2.1        4.160376e-14 -1.0676279 0.182 0.410 1.890059e-10       1
KRT10.1         4.329122e-14  0.7912601 0.625 0.400 1.966720e-10       1
PRSS23.1        4.601662e-14 -0.8464362 0.242 0.443 2.090535e-10       1
AC007920.2      5.487317e-14  0.2709241 0.186 0.092 2.492888e-10       1
ABCA1.1         5.661890e-14  0.9969721 0.400 0.207 2.572197e-10       1
S100A16.1       5.890165e-14 -0.7333226 0.232 0.444 2.675902e-10       1
BOC.1           6.620831e-14  1.3604318 0.453 0.227 3.007843e-10       1
FBXO32.1        6.779925e-14  0.8251374 0.319 0.146 3.080120e-10       1
SCAMP1-AS1.1    7.222713e-14  0.5861475 0.337 0.187 3.281279e-10       1
CLDN11.1        7.779121e-14  0.8222819 0.200 0.105 3.534054e-10       1
COL1A1.1        8.322470e-14 -0.6321553 0.291 0.483 3.780898e-10       1
RPL10.1         8.949102e-14  0.5384150 0.596 0.369 4.065577e-10       1
S1PR1.1         1.012439e-13  1.1170666 0.340 0.107 4.599510e-10       1
SH3BGRL3.1      1.045195e-13 -0.7766562 0.232 0.461 4.748322e-10       1
P4HA2.1         1.116968e-13 -0.8278498 0.302 0.476 5.074387e-10       1
IFITM3.1        1.176095e-13  0.9791914 0.625 0.374 5.343001e-10       1
FAH.1           1.194430e-13  1.0241131 0.488 0.271 5.426297e-10       1
PRDX3.1         1.262258e-13  0.7681293 0.625 0.419 5.734440e-10       1
POLE.1          1.617807e-13  0.4340285 0.221 0.133 7.349697e-10       1
PLA2G5.1        1.683100e-13  1.1892301 0.368 0.167 7.646322e-10       1
PRSS35.1        1.845243e-13 -1.3758425 0.158 0.432 8.382941e-10       1
CTGF.1          1.978724e-13 -0.8088466 0.295 0.505 8.989341e-10       1
S100A10.1       2.300593e-13 -0.3416047 0.323 0.525 1.045159e-09       1
LOX             2.417119e-13 -0.8161439 0.267 0.493 1.098097e-09       1
NR3C1.1         3.478893e-13  1.2201340 0.568 0.349 1.580461e-09       1
ACTG1.1         4.048378e-13 -0.4390191 0.295 0.509 1.839178e-09       1
PMEPA1.1        5.188619e-13 -0.7546098 0.284 0.491 2.357190e-09       1
ANAPC16.1       5.214748e-13  0.9062582 0.621 0.405 2.369060e-09       1
COL18A1.1       5.499007e-13 -0.8677086 0.249 0.452 2.498199e-09       1
CALCOCO2.1      5.748835e-13  1.0042038 0.547 0.349 2.611696e-09       1
EMP3.1          6.404857e-13  0.5310994 0.614 0.417 2.909727e-09       1
AC073111.5      6.675854e-13  0.2778403 0.235 0.120 3.032840e-09       1
ACTG2.1         6.831994e-13 -1.0143763 0.200 0.452 3.103775e-09       1
CALB2.1         6.923452e-13  1.0682908 0.354 0.155 3.145324e-09       1
SEMA3C          7.569215e-13 -0.8834478 0.288 0.501 3.438694e-09       1
DKK2.1          7.643097e-13 -1.4674718 0.119 0.414 3.472259e-09       1
SPATS2L.1       7.650568e-13 -0.6574153 0.277 0.492 3.475653e-09       1
SLC7A8.1        8.758162e-13  0.8392828 0.474 0.275 3.978833e-09       1
CEMIP2          9.701302e-13 -0.4372817 0.281 0.489 4.407302e-09       1
DDIT3.1         1.091993e-12  1.1635719 0.477 0.289 4.960922e-09       1
PAPSS1          1.106017e-12 -0.6201912 0.253 0.490 5.024634e-09       1
TPT1.1          1.318770e-12  0.6998965 0.621 0.412 5.991170e-09       1
FSCN1           1.386467e-12 -0.3003014 0.263 0.464 6.298718e-09       1
TYMS.1          1.413088e-12  0.9266995 0.288 0.164 6.419657e-09       1
PPME1.1         2.103539e-12 -1.3988290 0.211 0.415 9.556377e-09       1
ANKH            2.119986e-12 -0.5718243 0.239 0.461 9.631098e-09       1
TNFAIP1.1       2.303740e-12 -0.5318810 0.277 0.493 1.046589e-08       1
HSPB1.1         2.373771e-12  0.6816981 0.579 0.380 1.078404e-08       1
TPM4.1          2.413980e-12 -0.4168048 0.295 0.474 1.096671e-08       1
LY6E.1          2.757425e-12 -0.3842683 0.256 0.478 1.252698e-08       1
HLA-B.1         2.894734e-12  0.8142043 0.558 0.341 1.315078e-08       1
CHPF            2.945660e-12 -0.4311052 0.281 0.490 1.338213e-08       1
SNHG7.1         4.164082e-12  0.8459011 0.614 0.403 1.891742e-08       1
TCEAL4.1        4.611286e-12  0.8744541 0.554 0.375 2.094907e-08       1
ITM2B.1         4.620843e-12 -0.6859918 0.305 0.481 2.099249e-08       1
APP             4.727020e-12 -0.5233044 0.309 0.519 2.147485e-08       1
ARSE.1          6.601753e-12 -0.5477414 0.067 0.161 2.999176e-08       1
NUAK1.1         6.665497e-12 -0.8663500 0.225 0.507 3.028135e-08       1
SPOCK1.1        6.822127e-12 -1.0655400 0.249 0.428 3.099292e-08       1
ABCC4.1         6.982485e-12  0.6009770 0.414 0.231 3.172143e-08       1
GAS1.1          7.190513e-12 -1.0659360 0.211 0.449 3.266650e-08       1
PHF19.1         7.289313e-12  0.9524325 0.361 0.228 3.311535e-08       1
HEBP2.1         8.041189e-12  0.7971862 0.467 0.276 3.653112e-08       1
ACTR3           8.648563e-12 -0.4198060 0.288 0.466 3.929042e-08       1
BCL2L11         9.051308e-12  0.3687301 0.256 0.122 4.112009e-08       1
PLAGL1.1        1.004863e-11  0.9706752 0.558 0.374 4.565091e-08       1
FAM111A.1       1.236828e-11  0.7359081 0.375 0.185 5.618912e-08       1
MME.1           1.417603e-11  0.8389157 0.540 0.344 6.440171e-08       1
SLC25A6         1.499675e-11  0.3822534 0.600 0.392 6.813026e-08       1
FAM177A1.1      2.320257e-11  0.8689375 0.572 0.394 1.054093e-07       1
NAV2.1          2.430990e-11  0.4544536 0.540 0.343 1.104399e-07       1
KIF11.1         2.562152e-11  0.7109553 0.249 0.112 1.163986e-07       1
MGP.1           3.484653e-11  0.7006539 0.453 0.276 1.583078e-07       1
PPP3CA.1        3.862538e-11 -0.7729755 0.323 0.466 1.754751e-07       1
ITGB5           3.862541e-11 -0.5880882 0.305 0.498 1.754752e-07       1
SOCS1.1         4.020932e-11  0.5534850 0.368 0.228 1.826709e-07       1
AL357055.3.1    4.044727e-11  0.6000423 0.207 0.098 1.837520e-07       1
APOC1.1         4.610086e-11  0.9895332 0.386 0.163 2.094362e-07       1
C9orf47.1       4.894007e-11 -0.3736277 0.109 0.093 2.223347e-07       1
CLTB.1          5.047162e-11 -0.8284170 0.267 0.476 2.292926e-07       1
KCTD12          5.329087e-11 -0.3989266 0.232 0.450 2.421004e-07       1
RTL9.1          5.417033e-11  0.7768080 0.361 0.157 2.460958e-07       1
PYGB.1          5.700502e-11  0.6973889 0.319 0.200 2.589738e-07       1
SIPA1L1.1       6.503898e-11 -0.5726260 0.277 0.485 2.954721e-07       1
DPP4.1          8.170947e-11  0.6718893 0.460 0.313 3.712061e-07       1
DUSP10.1        8.385599e-11  0.5885667 0.396 0.230 3.809577e-07       1
C6orf48.1       9.699757e-11  0.7535630 0.604 0.422 4.406599e-07       1
YTHDF3-AS1.1    9.762583e-11  0.5039600 0.319 0.155 4.435142e-07       1
NOS1AP.1        1.049412e-10  0.6228390 0.361 0.194 4.767477e-07       1
COL1A2          1.057382e-10 -0.4699418 0.319 0.484 4.803685e-07       1
NREP.1          1.147745e-10 -0.7234613 0.288 0.489 5.214206e-07       1
NME4.1          1.158879e-10  1.0077812 0.568 0.390 5.264788e-07       1
NAP1L1.1        1.205247e-10  1.0370546 0.533 0.386 5.475437e-07       1
APBB3.1         1.237418e-10  0.4705649 0.193 0.156 5.621590e-07       1
PFN2            1.380048e-10 -0.4010137 0.326 0.512 6.269559e-07       1
NCK1.1          1.451677e-10  0.8521942 0.544 0.378 6.594967e-07       1
JAK1.1          1.472262e-10  0.8951283 0.554 0.381 6.688486e-07       1
MSANTD2.1       1.665513e-10  0.6226135 0.291 0.190 7.566427e-07       1
HS6ST2.1        1.700950e-10 -0.9821671 0.232 0.478 7.727414e-07       1
FAM89A.1        1.718397e-10  0.8362990 0.568 0.397 7.806677e-07       1
SEMA5A.1        1.948051e-10 -0.8912194 0.267 0.432 8.849998e-07       1
TGFBR1.1        2.074497e-10  0.7490293 0.579 0.403 9.424441e-07       1
ACSL1.1         2.122488e-10  0.5173395 0.512 0.362 9.642465e-07       1
JPT1.1          2.188105e-10 -0.6611214 0.281 0.490 9.940563e-07       1
TTF2.1          2.200635e-10  0.5325025 0.323 0.205 9.997487e-07       1
TCAF1           2.563267e-10 -0.8536840 0.225 0.464 1.164492e-06       1
CCDC69.1        2.605708e-10  0.8578003 0.330 0.193 1.183773e-06       1
C1GALT1.1       3.218008e-10  0.7623473 0.463 0.304 1.461941e-06       1
ZFP36L2.1       3.511445e-10  1.1400077 0.467 0.273 1.595249e-06       1
RAB31.1         3.659760e-10  0.9327100 0.505 0.306 1.662629e-06       1
EIF1B.1         3.797513e-10  1.2263728 0.575 0.411 1.725210e-06       1
LOXL2           3.847761e-10 -0.5565032 0.288 0.488 1.748038e-06       1
SUGCT.1         4.142261e-10 -0.9584946 0.207 0.430 1.881829e-06       1
RPS6KA1.1       4.381786e-10  0.2539994 0.196 0.125 1.990646e-06       1
HIGD1A.1        4.484086e-10  0.5759408 0.582 0.388 2.037120e-06       1
PCSK5.1         4.486845e-10  0.7226209 0.309 0.149 2.038374e-06       1
CBLN3           4.817114e-10  0.2562158 0.189 0.111 2.188415e-06       1
SPSB1.1         4.950836e-10  0.5375951 0.586 0.391 2.249165e-06       1
ANGPT2.1        5.570830e-10  0.4067613 0.218 0.153 2.530828e-06       1
CD59            6.182047e-10 -0.4999426 0.302 0.488 2.808504e-06       1
MRAS.1          6.824516e-10 -0.6043023 0.274 0.475 3.100378e-06       1
BTBD3.1         7.073106e-10  1.0088921 0.326 0.204 3.213312e-06       1
WDR90.1         7.094908e-10  0.5501558 0.281 0.129 3.223217e-06       1
PDGFRB.1        7.321566e-10  0.3059855 0.396 0.295 3.326188e-06       1
KDR.1           7.536129e-10  1.1518159 0.323 0.158 3.423664e-06       1
FLNB.1          7.597120e-10  0.6858574 0.400 0.272 3.451372e-06       1
ZNF573.1        7.719701e-10  0.5181748 0.123 0.092 3.507060e-06       1
ZNF653.1        8.059327e-10  0.5109485 0.123 0.078 3.661352e-06       1
CES4A.1         8.382555e-10  0.4003671 0.323 0.184 3.808195e-06       1
PRPS2.1         8.896007e-10  0.7096689 0.316 0.168 4.041456e-06       1
ITGAV           9.332291e-10 -0.6650634 0.316 0.458 4.239660e-06       1
TFRC.1          9.794631e-10  0.8213652 0.439 0.302 4.449701e-06       1
CCDC183.1       1.001265e-09  0.4416125 0.270 0.124 4.548749e-06       1
GADD45A.1       1.056565e-09 -0.6645503 0.333 0.458 4.799973e-06       1
NDST1           1.124422e-09 -0.4655417 0.291 0.504 5.108248e-06       1
PDLIM5          1.137520e-09 -0.4941864 0.263 0.467 5.167754e-06       1
CENPU.1         1.203806e-09  0.5195072 0.211 0.094 5.468893e-06       1
AC242426.2.1    1.220022e-09  0.4839500 0.246 0.123 5.542562e-06       1
TPRG1.1         1.265366e-09 -0.4368791 0.063 0.098 5.748559e-06       1
CD99.1          1.319540e-09 -0.3125700 0.323 0.473 5.994672e-06       1
CD55.1          1.405432e-09 -1.4219826 0.249 0.442 6.384879e-06       1
MXRA5.1         1.526130e-09 -1.0789352 0.256 0.462 6.933206e-06       1
TGFBI.1         1.697107e-09  0.5839233 0.558 0.383 7.709956e-06       1
TMED3.1         1.774810e-09 -0.4995179 0.298 0.479 8.062961e-06       1
ISPD.1          1.944826e-09  0.6219530 0.246 0.142 8.835346e-06       1
MGST1.1         1.969440e-09  0.8383624 0.505 0.293 8.947165e-06       1
PLIN2.1         2.270478e-09  1.1230845 0.400 0.162 1.031478e-05       1
OCIAD2.1        2.335255e-09 -0.9524956 0.088 0.257 1.060907e-05       1
NFKBIA.1        2.480645e-09  0.6414659 0.456 0.242 1.126957e-05       1
STC2.1          2.530003e-09  0.8032075 0.540 0.368 1.149380e-05       1
SPARC.1         2.536079e-09 -0.4525378 0.351 0.473 1.152141e-05       1
PIK3R1.1        2.903517e-09  0.7175959 0.358 0.202 1.319068e-05       1
HACD1.1         2.920844e-09 -0.6795741 0.295 0.501 1.326940e-05       1
PNPLA3.1        2.938271e-09  0.6058616 0.270 0.193 1.334856e-05       1
LGALS3.1        3.110793e-09  0.8738579 0.505 0.343 1.413233e-05       1
DPP7            3.181764e-09  0.2525285 0.344 0.230 1.445475e-05       1
TNFAIP8.1       3.196932e-09  0.7397715 0.516 0.357 1.452366e-05       1
ARL4D.1         3.271758e-09  0.8380928 0.358 0.223 1.486359e-05       1
AC124016.1.1    3.455103e-09  0.4456736 0.235 0.108 1.569653e-05       1
CD47            3.535843e-09 -0.3733769 0.295 0.467 1.606333e-05       1
MBOAT2.1        3.546321e-09 -0.7655970 0.239 0.423 1.611094e-05       1
P3H1            3.738001e-09 -0.3933947 0.319 0.481 1.698174e-05       1
PRKAR2B.1       3.831869e-09  0.9427932 0.274 0.092 1.740818e-05       1
MERTK.1         3.963028e-09  0.8434646 0.263 0.163 1.800404e-05       1
AMOTL2          4.293876e-09 -0.4514893 0.288 0.476 1.950708e-05       1
SPINT1.1        4.482364e-09 -1.1058916 0.070 0.327 2.036338e-05       1
PIM1.1          4.554046e-09  0.5751343 0.393 0.211 2.068903e-05       1
MYH9            4.575576e-09 -0.4090010 0.316 0.473 2.078684e-05       1
FAM84B          5.298409e-09 -0.7646696 0.239 0.458 2.407067e-05       1
OGFRL1          5.872701e-09 -0.4854995 0.291 0.488 2.667968e-05       1
UGP2.1          6.018444e-09 -0.6315956 0.312 0.458 2.734179e-05       1
AC012615.1.1    6.461826e-09 -0.4317463 0.091 0.197 2.935608e-05       1
PDLIM7.1        6.649082e-09 -0.4083607 0.288 0.464 3.020678e-05       1
CDC42EP3.1      6.687835e-09 -0.9044463 0.267 0.443 3.038283e-05       1
ATP1B1          6.805287e-09  0.4052308 0.551 0.393 3.091642e-05       1
UBXN4.1         7.059456e-09  0.4285739 0.572 0.393 3.207111e-05       1
CDKN2B          7.429490e-09 -0.3650907 0.302 0.492 3.375217e-05       1
CDC42EP1.1      7.714853e-09  0.4790102 0.411 0.301 3.504858e-05       1
KIF22.1         8.108818e-09  0.7885739 0.270 0.181 3.683836e-05       1
CNTROB.1        8.245776e-09  0.7285127 0.274 0.151 3.746056e-05       1
FGFRL1.1        8.308230e-09 -0.8570212 0.137 0.308 3.774429e-05       1
EDIL3           8.817829e-09 -0.5114984 0.312 0.471 4.005940e-05       1
LRRC32          9.952529e-09 -0.3337539 0.274 0.449 4.521434e-05       1
NFIA.1          1.000420e-08  0.9237730 0.319 0.192 4.544906e-05       1
HAPLN3.1        1.062008e-08  0.3893693 0.214 0.145 4.824703e-05       1
CDC37L1.1       1.076799e-08  0.4277201 0.488 0.331 4.891896e-05       1
CDH11           1.226413e-08 -0.8051028 0.291 0.466 5.571596e-05       1
IGFBP3.1        1.235061e-08 -0.5978512 0.267 0.444 5.610880e-05       1
ITGB1           1.248352e-08 -0.3364872 0.330 0.520 5.671265e-05       1
NQO2.1          1.309722e-08  0.7871977 0.407 0.267 5.950068e-05       1
LAMC3.1         1.330073e-08  0.4230926 0.260 0.139 6.042522e-05       1
PLOD2           1.379459e-08 -1.0960248 0.323 0.490 6.266882e-05       1
PRICKLE1.1      1.400131e-08 -0.7053560 0.242 0.464 6.360795e-05       1
ARID5B.1        1.461235e-08  0.3483723 0.512 0.338 6.638388e-05       1
PKM             1.470419e-08 -0.4041460 0.323 0.499 6.680114e-05       1
FIBIN.1         1.516979e-08 -1.0197263 0.316 0.467 6.891636e-05       1
NDRG2.1         1.755278e-08  0.3845577 0.165 0.134 7.974228e-05       1
HSPB7.1         1.773324e-08  0.4145958 0.295 0.167 8.056212e-05       1
C5orf38.1       1.775334e-08  0.5156160 0.400 0.261 8.065343e-05       1
SOX11.1         1.999889e-08 -1.5989844 0.137 0.373 9.085497e-05       1
ADAMTS3         2.081829e-08  0.2640899 0.189 0.095 9.457747e-05       1
PNRC1.1         2.223556e-08  0.6387967 0.575 0.406 1.010161e-04       1
PHACTR2.1       2.292084e-08  0.5029751 0.554 0.424 1.041294e-04       1
PLEC            2.582936e-08 -0.4642582 0.295 0.488 1.173428e-04       1
CD68.1          2.837309e-08  0.7204106 0.298 0.188 1.288990e-04       1
BLOC1S5.1       2.930649e-08  0.5724139 0.277 0.193 1.331394e-04       1
ITPKA.1         3.045568e-08  0.6682028 0.207 0.095 1.383602e-04       1
ICE1.1          3.264999e-08 -0.8172007 0.239 0.400 1.483289e-04       1
DACH1.1         3.345823e-08  0.4012024 0.249 0.161 1.520007e-04       1
AMMECR1.1       3.422898e-08  0.4848894 0.565 0.410 1.555023e-04       1
ATF4.1          3.572306e-08  0.5174002 0.589 0.437 1.622899e-04       1
IER3-AS1        3.638183e-08  0.3365145 0.239 0.142 1.652826e-04       1
IGFBP5.1        3.852126e-08 -0.6478915 0.207 0.392 1.750021e-04       1
MAP3K7CL.1      4.382616e-08 -1.5439458 0.091 0.333 1.991022e-04       1
STAG1.1         4.438923e-08  0.6503518 0.396 0.277 2.016603e-04       1
ZNF320.1        4.552787e-08  0.4285763 0.154 0.092 2.068331e-04       1
TIMP4.1         4.636114e-08  1.4929203 0.319 0.135 2.106187e-04       1
PDS5B.1         4.760232e-08 -0.6716102 0.277 0.459 2.162573e-04       1
AC012467.2.1    5.148918e-08  0.4008809 0.305 0.167 2.339153e-04       1
FBN2            5.297827e-08 -0.7212471 0.284 0.466 2.406803e-04       1
AC067750.1      5.632804e-08 -0.2806524 0.235 0.132 2.558983e-04       1
FLRT3.1         6.017670e-08  0.5618497 0.284 0.167 2.733828e-04       1
PLPP3.1         6.251925e-08  0.8524047 0.340 0.163 2.840249e-04       1
SMPDL3A.1       6.594674e-08  0.4904405 0.344 0.239 2.995960e-04       1
TMEM143         6.695704e-08  0.4206190 0.182 0.122 3.041858e-04       1
SLC3A2.1        6.997269e-08  0.8025008 0.554 0.399 3.178859e-04       1
GNL1.1          7.127660e-08  0.4223255 0.428 0.330 3.238096e-04       1
HOOK3.1         7.127663e-08  0.3546486 0.537 0.385 3.238097e-04       1
EXOSC9.1        7.484250e-08  0.5230795 0.270 0.209 3.400095e-04       1
KLF9.1          7.578091e-08  1.0638932 0.309 0.216 3.442727e-04       1
PLEKHA4.1       7.735701e-08  0.7653941 0.175 0.078 3.514329e-04       1
GALNT10.1       7.828391e-08 -0.6414335 0.319 0.483 3.556438e-04       1
ISCU.1          7.879400e-08  0.6353153 0.561 0.416 3.579611e-04       1
MNDA.1          8.068993e-08 -0.9846448 0.053 0.131 3.665743e-04       1
AL135905.1.1    8.090744e-08  0.4053758 0.158 0.107 3.675625e-04       1
GPSM2.1         8.156620e-08  0.6393436 0.270 0.137 3.705553e-04       1
CLK1.1          8.384783e-08  0.4547241 0.421 0.283 3.809207e-04       1
TOX2.1          9.813136e-08  1.0599719 0.260 0.073 4.458108e-04       1
PITX2           9.844872e-08 -0.5221890 0.340 0.495 4.472525e-04       1
CRYAB.1         9.924117e-08  0.6394524 0.495 0.292 4.508526e-04       1
EPOR            1.004248e-07  0.2625847 0.393 0.278 4.562301e-04       1
PNMT.1          1.053904e-07  1.3808921 0.393 0.193 4.787886e-04       1
ZNF345.1        1.090089e-07  0.6437571 0.182 0.115 4.952276e-04       1
NAV1.1          1.091288e-07  0.6877528 0.484 0.340 4.957721e-04       1
CHAC1           1.140243e-07  0.3637131 0.267 0.187 5.180123e-04       1
LRRC38.1        1.153696e-07  1.1080736 0.319 0.174 5.241243e-04       1
SLPI.1          1.232019e-07  1.0844011 0.372 0.193 5.597063e-04       1
BMPER.1         1.260628e-07  0.6493704 0.446 0.343 5.727032e-04       1
RGS2.1          1.307142e-07  0.6308234 0.407 0.316 5.938348e-04       1
LMNB1.1         1.388085e-07  0.5660154 0.330 0.127 6.306069e-04       1
TMEM178A.1      1.437601e-07 -0.2917939 0.235 0.205 6.531019e-04       1
GNAI2.1         1.511117e-07 -0.6263100 0.295 0.443 6.865005e-04       1
ITGA1.1         1.587385e-07  0.4711506 0.558 0.400 7.211492e-04       1
SLC46A3.1       1.640285e-07  0.9387117 0.568 0.408 7.451816e-04       1
WASF3.1         1.666492e-07  0.6862776 0.333 0.191 7.570874e-04       1
LRP1.1          1.695786e-07 -0.5361166 0.340 0.463 7.703958e-04       1
SLC9A9          1.760501e-07  0.3446003 0.204 0.145 7.997954e-04       1
TMEM98.1        1.789511e-07 -0.7190288 0.312 0.463 8.129746e-04       1
DUSP1.1         1.845988e-07  0.7904191 0.449 0.288 8.386325e-04       1
C1orf112.1      1.877338e-07  0.7721420 0.347 0.200 8.528745e-04       1
FST.1           1.930092e-07 -0.3881679 0.323 0.456 8.768408e-04       1
FAM129A.1       1.938515e-07  0.8964404 0.316 0.171 8.806672e-04       1
ENC1.1          1.949760e-07 -0.6063915 0.182 0.438 8.857761e-04       1
LRRC2.1         2.068911e-07  0.5887738 0.249 0.119 9.399064e-04       1
PMP22.1         2.101706e-07  0.9974480 0.551 0.393 9.548048e-04       1
HECW1.1         2.158614e-07 -0.2699391 0.084 0.121 9.806585e-04       1
C1R.1           2.314089e-07  0.8424321 0.337 0.172 1.051291e-03       1
CHD9            2.436854e-07 -0.2618926 0.340 0.492 1.107063e-03       1
NKAIN4.1        3.018558e-07 -0.6537894 0.256 0.466 1.371331e-03       1
RPP38.1         3.072155e-07  0.7278096 0.439 0.305 1.395680e-03       1
CLASRP          3.210168e-07  0.3111822 0.200 0.168 1.458379e-03       1
GPBAR1.1        3.250177e-07  0.4737803 0.281 0.138 1.476555e-03       1
ZNF10           3.314509e-07 -0.2588635 0.239 0.166 1.505781e-03       1
CENPE.1         3.314582e-07  0.4684002 0.130 0.107 1.505815e-03       1
TMED9.1         4.013489e-07 -0.5362555 0.316 0.469 1.823328e-03       1
NAMPT.1         4.187787e-07  1.1956422 0.358 0.188 1.902511e-03       1
KCTD2.1         4.333754e-07  0.5603426 0.232 0.161 1.968824e-03       1
PLOD1           4.540015e-07 -0.5684051 0.312 0.472 2.062529e-03       1
TMEM91.1        4.553935e-07  0.9359478 0.354 0.215 2.068853e-03       1
ANP32B.1        4.579571e-07  0.6153314 0.593 0.438 2.080499e-03       1
DCBLD1.1        4.719445e-07 -0.7733193 0.256 0.406 2.144044e-03       1
KIAA1024.1      4.721837e-07  0.4512581 0.179 0.179 2.145130e-03       1
EPS8.1          4.748401e-07  0.5832849 0.537 0.412 2.157198e-03       1
AC245060.5      5.402837e-07 -0.4061336 0.105 0.200 2.454509e-03       1
ATP1A1.1        5.480187e-07  0.9240819 0.526 0.417 2.489649e-03       1
MATN3.1         5.566831e-07  0.8122609 0.400 0.325 2.529011e-03       1
RAD9A           5.623454e-07 -0.4718564 0.084 0.190 2.554735e-03       1
HIF3A.1         5.652141e-07  0.8996306 0.333 0.199 2.567768e-03       1
PHTF2.1         5.800024e-07 -0.2806313 0.330 0.467 2.634951e-03       1
BOD1L1.1        5.939537e-07  0.7202202 0.533 0.404 2.698332e-03       1
AVIL            6.240894e-07  0.4097041 0.281 0.151 2.835238e-03       1
GADD45B.1       6.613380e-07  0.6511092 0.554 0.408 3.004459e-03       1
ACTC1.1         6.947066e-07 -1.5381860 0.063 0.302 3.156052e-03       1
KIRREL1         7.074421e-07 -0.3081538 0.182 0.315 3.213909e-03       1
YARS.1          7.081538e-07  0.4737713 0.495 0.362 3.217143e-03       1
SLC26A11.1      7.102892e-07  0.4857844 0.298 0.173 3.226844e-03       1
PGM2L1          7.390181e-07 -0.2961388 0.256 0.489 3.357359e-03       1
BNIP3.1         7.715630e-07  0.6544198 0.523 0.404 3.505211e-03       1
EBI3.1          7.793115e-07  0.5284207 0.298 0.147 3.540412e-03       1
HAND2.1         8.275160e-07  0.5673536 0.561 0.406 3.759405e-03       1
FILIP1L.1       8.304121e-07 -0.5748801 0.298 0.455 3.772562e-03       1
GATAD2B.1       8.341538e-07  0.4357522 0.305 0.258 3.789561e-03       1
ATXN1           8.860412e-07 -0.5497866 0.319 0.492 4.025285e-03       1
FHL1.1          8.908880e-07 -0.5803837 0.260 0.435 4.047304e-03       1
MDH1            9.372949e-07  0.3655187 0.547 0.400 4.258131e-03       1
EGLN3.1         9.580104e-07 -1.0456180 0.214 0.439 4.352241e-03       1
P3H2.1          9.589623e-07  0.5220909 0.214 0.108 4.356566e-03       1
ACLY            9.637407e-07 -0.5784175 0.337 0.495 4.378274e-03       1
ANXA5.1         9.757778e-07 -0.4042984 0.340 0.485 4.432959e-03       1
TPX2            9.786755e-07  0.4131204 0.200 0.181 4.446123e-03       1
DDR1.1          9.830684e-07  0.3887397 0.239 0.196 4.466080e-03       1
CPEB3.1         1.043318e-06  0.4077670 0.211 0.153 4.739796e-03       1
BATF3.1         1.069974e-06  0.4613689 0.158 0.145 4.860893e-03       1
SPECC1L.1       1.075290e-06  0.5617908 0.270 0.171 4.885044e-03       1
ZC3H12C.1       1.107656e-06  0.9003529 0.235 0.097 5.032081e-03       1
GDF6.1          1.120862e-06 -0.7828869 0.189 0.377 5.092076e-03       1
PRDX2.1         1.123087e-06 -0.3089054 0.330 0.473 5.102182e-03       1
TTLL7.1         1.174076e-06  0.6084351 0.316 0.174 5.333829e-03       1
SEPT9.1         1.183372e-06  0.7048197 0.432 0.292 5.376057e-03       1
EBF1.1          1.206298e-06  0.9340556 0.312 0.170 5.480213e-03       1
CD151           1.206921e-06 -0.5569687 0.319 0.478 5.483043e-03       1
MT1E.1          1.226015e-06  0.5044819 0.442 0.284 5.569787e-03       1
SBF2-AS1.1      1.273378e-06  0.5581166 0.488 0.350 5.784956e-03       1
CYB561D1.1      1.309547e-06  0.5618604 0.151 0.119 5.949272e-03       1
ACKR2.1         1.324422e-06  0.2954980 0.260 0.198 6.016851e-03       1
ARHGEF26.1      1.406721e-06  0.8341241 0.309 0.169 6.390733e-03       1
RAI14           1.408099e-06 -0.3453336 0.305 0.478 6.396992e-03       1
DAPK1           1.417084e-06  0.2668252 0.214 0.194 6.437813e-03       1
PRDX6.1         1.477956e-06 -0.6145942 0.316 0.488 6.714354e-03       1
NEK5.1          1.490281e-06  0.2632048 0.211 0.215 6.770348e-03       1
BLVRB.1         1.496113e-06  0.4029326 0.463 0.326 6.796841e-03       1
MYLIP.1         1.506357e-06  0.5787576 0.256 0.100 6.843379e-03       1
ATOX1.1         1.560243e-06 -0.3530175 0.351 0.486 7.088182e-03       1
DYNLL1.1        1.585544e-06 -0.3414666 0.372 0.504 7.203125e-03       1
PGAP1.1         1.595573e-06  0.4503950 0.242 0.122 7.248688e-03       1
RHOU            1.613593e-06  0.3183893 0.337 0.255 7.330555e-03       1
PHF3.1          1.653346e-06  0.6252396 0.495 0.339 7.511151e-03       1
SULF2           1.677647e-06 -0.3211517 0.288 0.423 7.621549e-03       1
OSBPL3          1.687416e-06  0.3672203 0.214 0.144 7.665932e-03       1
GNA13           1.734849e-06  0.3619976 0.389 0.310 7.881420e-03       1
H2AFJ.1         1.747531e-06  0.7489008 0.554 0.381 7.939034e-03       1
AC007014.1.1    1.756026e-06 -0.3080756 0.154 0.113 7.977624e-03       1
RPP25.1         1.829051e-06  0.6266016 0.505 0.362 8.309380e-03       1
DEK.1           1.859427e-06  0.6037075 0.477 0.349 8.447377e-03       1
FGFR1.1         1.888459e-06  0.3128362 0.572 0.429 8.579269e-03       1
ADM.1           1.943087e-06  0.7059823 0.270 0.127 8.827444e-03       1
ANTXR1.1        1.981060e-06 -0.5090107 0.312 0.456 8.999954e-03       1
BCO2.1          2.039263e-06  1.1529871 0.309 0.149 9.264373e-03       1
SELENOW.1       2.079016e-06 -0.3837558 0.351 0.465 9.444969e-03       1
MAP2K3.1        2.112350e-06 -0.4898078 0.193 0.333 9.596405e-03       1
SDHAF3.1        2.142062e-06  0.6989525 0.407 0.303 9.731388e-03       1
FSTL3.1         2.300088e-06 -0.4270202 0.326 0.467 1.044930e-02       1
BACE1-AS.1      2.410791e-06  0.5003644 0.214 0.112 1.095222e-02       1
MCUB.1          2.469269e-06 -0.5712134 0.179 0.338 1.121789e-02       1
PDIA4.1         2.490643e-06 -0.3448947 0.326 0.475 1.131499e-02       1
GUCY1A2         2.641513e-06  0.3701714 0.196 0.160 1.200039e-02       1
MYDGF.1         2.647831e-06 -0.7601427 0.312 0.454 1.202910e-02       1
ZMAT3           2.649097e-06 -0.4896192 0.326 0.471 1.203485e-02       1
DIO3OS.1        2.771820e-06  0.8463304 0.281 0.113 1.259238e-02       1
PDGFRL.1        2.798472e-06 -0.3609574 0.168 0.320 1.271346e-02       1
ANKRD28.1       2.838747e-06  0.4001269 0.386 0.256 1.289643e-02       1
CDKN1A.1        2.856371e-06  0.7598070 0.547 0.430 1.297650e-02       1
ROGDI           2.908502e-06  0.3304106 0.337 0.275 1.321332e-02       1
GMPR.1          2.989820e-06  0.7419547 0.284 0.175 1.358275e-02       1
FLCN.1          3.041395e-06  0.6583597 0.295 0.202 1.381706e-02       1
TMEM160.1       3.089417e-06 -0.3748998 0.354 0.478 1.403522e-02       1
NT5E.1          3.099696e-06 -0.8458585 0.312 0.483 1.408192e-02       1
FSTL1.1         3.112955e-06 -0.2798505 0.316 0.478 1.414216e-02       1
PCK2.1          3.130725e-06  0.6208028 0.337 0.210 1.422288e-02       1
ZNF816.1        3.187614e-06 -0.4436813 0.144 0.120 1.448133e-02       1
CDK5RAP2.1      3.302915e-06  0.5788412 0.291 0.228 1.500514e-02       1
PHLDB1.1        3.306042e-06 -0.5356393 0.256 0.419 1.501935e-02       1
MANSC1.1        3.385159e-06  0.4105071 0.246 0.125 1.537878e-02       1
HSD17B6.1       3.422170e-06 -0.8403726 0.207 0.424 1.554692e-02       1
UGCG            3.471017e-06 -0.5783989 0.239 0.331 1.576883e-02       1
SGK1            3.637012e-06  0.3183525 0.512 0.392 1.652295e-02       1
ZNF23           3.716695e-06 -0.5818311 0.021 0.120 1.688495e-02       1
CCNA2.1         3.808724e-06  0.7257911 0.193 0.134 1.730303e-02       1
RASAL2-AS1.1    4.223246e-06  0.4451585 0.137 0.072 1.918620e-02       1
SCNN1A.1        4.309041e-06  0.6303689 0.312 0.159 1.957597e-02       1
ADAM10.1        4.311162e-06 -0.6838555 0.347 0.477 1.958561e-02       1
EEF1A1          4.549324e-06  0.3550085 0.516 0.405 2.066758e-02       1
REXO5.1         4.579155e-06  0.3815158 0.175 0.078 2.080310e-02       1
PYCARD.1        4.637232e-06  0.4831419 0.193 0.105 2.106694e-02       1
PKIA.1          4.698206e-06  0.9786390 0.186 0.101 2.134395e-02       1
EPG5            5.078126e-06 -0.5755155 0.211 0.351 2.306993e-02       1
LIG1.1          5.311987e-06  0.3589824 0.186 0.157 2.413236e-02       1
ANKRD1.1        5.328213e-06 -0.9470937 0.154 0.317 2.420607e-02       1
PRAC1.1         5.451875e-06  0.4733832 0.326 0.203 2.476787e-02       1
UBASH3B.1       5.610708e-06  0.3488258 0.389 0.308 2.548945e-02       1
DACT3.1         5.696949e-06 -0.5498388 0.109 0.127 2.588124e-02       1
MT1X.1          5.873262e-06  0.7349210 0.477 0.351 2.668223e-02       1
CTSZ.1          5.889846e-06 -0.5928514 0.344 0.446 2.675757e-02       1
GNG11.1         5.983021e-06 -0.7045992 0.295 0.413 2.718086e-02       1
CAP2.1          6.074866e-06 -0.5469727 0.267 0.442 2.759812e-02       1
ARHGAP26.1      6.130863e-06  0.5828719 0.232 0.156 2.785251e-02       1
S100A13         6.153824e-06  0.3047760 0.505 0.370 2.795682e-02       1
FAM111A-DT.1    6.294207e-06 -0.4666912 0.053 0.194 2.859458e-02       1
BEX1.1          6.303059e-06 -0.4388389 0.060 0.191 2.863480e-02       1
FAXDC2          6.590752e-06  0.2769210 0.186 0.092 2.994179e-02       1
HSD17B4         6.612006e-06  0.2834540 0.340 0.266 3.003834e-02       1
AL138966.2.1    6.660802e-06  0.5609948 0.140 0.110 3.026002e-02       1
UBE2B.1         6.685341e-06  0.4452623 0.554 0.413 3.037151e-02       1
APOE.1          6.913092e-06  0.8396009 0.302 0.228 3.140617e-02       1
DDAH1.1         6.951320e-06 -0.6321172 0.221 0.420 3.157984e-02       1
CCNI            7.008956e-06  0.3940453 0.526 0.423 3.184169e-02       1
POLR2J3.1.1     7.105743e-06  0.4132047 0.302 0.229 3.228139e-02       1
PER1.1          7.145167e-06  0.4570172 0.288 0.176 3.246049e-02       1
CILP2.1         7.280401e-06  0.9144174 0.284 0.203 3.307486e-02       1
HAND2-AS1.1     7.455796e-06  0.9305771 0.526 0.412 3.387168e-02       1
CPZ.1           7.520771e-06 -0.3209402 0.133 0.150 3.416686e-02       1
FKBP10.1        7.569171e-06 -0.3546018 0.358 0.466 3.438674e-02       1
ADGRL4.1        7.582991e-06  0.4444323 0.175 0.143 3.444953e-02       1
ABHD14A-ACY1.1  7.642117e-06 -0.2695444 0.039 0.084 3.471814e-02       1
IGFBP2.1        8.215181e-06 -1.0272124 0.228 0.398 3.732157e-02       1
OSGIN1.1        8.348725e-06  0.8775101 0.182 0.082 3.792826e-02       1
DSE.1           8.363969e-06 -0.5384226 0.291 0.474 3.799751e-02       1
FLT1.1          8.448075e-06 -0.5508978 0.361 0.484 3.837960e-02       1
MSX2.1          8.595274e-06  0.8065822 0.316 0.225 3.904833e-02       1
CPE.1           8.784353e-06 -0.6621154 0.270 0.434 3.990732e-02       1
TMEM39B         9.051698e-06  0.3125178 0.249 0.170 4.112186e-02       1
KAT6B.1         9.309551e-06  0.8045961 0.354 0.288 4.229329e-02       1
DSC3.1          9.415516e-06  0.4227145 0.361 0.259 4.277469e-02       1
CDKN3           9.578649e-06  0.2854269 0.200 0.161 4.351580e-02       1
FILNC1.1        9.922256e-06  0.8168163 0.168 0.132 4.507681e-02       1
AC011978.2.1    9.994295e-06  0.3715147 0.207 0.124 4.540408e-02       1
SLC4A7          1.012138e-05 -0.6704578 0.218 0.367 4.598141e-02       1
PLEKHG3.1       1.014403e-05 -0.3269938 0.140 0.156 4.608432e-02       1
FOXF1.1         1.028239e-05  0.5584926 0.432 0.343 4.671288e-02       1
SLC22A3.1       1.044103e-05  0.9083353 0.428 0.315 4.743358e-02       1
COL8A2          1.050243e-05 -0.7865142 0.239 0.385 4.771254e-02       1
RPN1.1          1.096106e-05 -0.5620958 0.326 0.451 4.979608e-02       1
TMEM38B         1.127060e-05  0.2940350 0.372 0.246 5.120234e-02       1
CLSPN.1         1.147467e-05  0.5712095 0.168 0.133 5.212943e-02       1
PABPC1L         1.167171e-05  0.2941697 0.218 0.138 5.302458e-02       1
UBC.1           1.181396e-05  0.4427978 0.558 0.411 5.367082e-02       1
LTBR.1          1.182984e-05  0.4307350 0.242 0.170 5.374297e-02       1
NRG1.1          1.205191e-05 -1.0742570 0.091 0.263 5.475181e-02       1
TMEM190.1       1.210812e-05  0.4629118 0.168 0.120 5.500718e-02       1
RAB40A.1        1.265629e-05  0.6050242 0.235 0.159 5.749751e-02       1
TSPAN13         1.320420e-05 -0.4312022 0.316 0.443 5.998669e-02       1
NDFIP2.1        1.402808e-05 -0.5546623 0.281 0.432 6.372956e-02       1
CARD8-AS1.1     1.437467e-05  0.5915429 0.253 0.148 6.530411e-02       1
BOD1.1          1.455475e-05  0.6722552 0.540 0.420 6.612221e-02       1
LINC00513       1.458705e-05  0.2809257 0.218 0.110 6.626898e-02       1
DGCR8.1         1.471051e-05  0.4366905 0.295 0.198 6.682983e-02       1
AL121827.2.1    1.478242e-05 -0.6837550 0.095 0.279 6.715655e-02       1
MSRB3           1.507350e-05 -0.3624699 0.312 0.462 6.847890e-02       1
SAT2.1          1.508686e-05  0.5389375 0.540 0.425 6.853959e-02       1
UBE2D1          1.578333e-05  0.3684025 0.414 0.327 7.170369e-02       1
TLE4            1.688579e-05 -0.7145954 0.253 0.401 7.671214e-02       1
GCH1.1          1.696727e-05  0.4754855 0.291 0.198 7.708232e-02       1
C15orf39.1      1.718589e-05  0.6034790 0.319 0.178 7.807549e-02       1
MCM3AP-AS1.1    1.720854e-05  0.5408005 0.165 0.095 7.817840e-02       1
MSN             1.735312e-05 -0.4142128 0.354 0.465 7.883524e-02       1
CLU.1           1.772027e-05 -0.8209407 0.302 0.403 8.050320e-02       1
LRP5            1.782491e-05 -0.3404577 0.144 0.243 8.097856e-02       1
ZDHHC14.1       1.787970e-05  0.4285883 0.207 0.153 8.122746e-02       1
ID3.1           1.799737e-05 -0.7988078 0.256 0.437 8.176205e-02       1
CENPL           1.834103e-05  0.3026520 0.256 0.190 8.332331e-02       1
ASNS.1          1.847021e-05  0.6412381 0.284 0.224 8.391017e-02       1
TUFT1           1.931466e-05 -0.6302884 0.168 0.308 8.774649e-02       1
MACF1           1.994152e-05 -0.3572821 0.340 0.473 9.059432e-02       1
ISYNA1.1        2.031085e-05  0.3504033 0.316 0.237 9.227217e-02       1
SKP2.1          2.033748e-05 -0.4453247 0.211 0.382 9.239319e-02       1
MLEC.1          2.048007e-05 -0.4057011 0.347 0.462 9.304098e-02       1
AHCTF1.1        2.059661e-05  0.6579963 0.323 0.243 9.357040e-02       1
SLC26A2.1       2.060558e-05  0.4965932 0.544 0.419 9.361114e-02       1
KCTD1.1         2.092266e-05  0.7646542 0.253 0.190 9.505166e-02       1
LINC01355.1     2.099578e-05 -0.7626017 0.140 0.236 9.538381e-02       1
RGS20.1         2.198745e-05  0.5984967 0.225 0.124 9.988898e-02       1
TBC1D9          2.211264e-05 -0.5000750 0.316 0.449 1.004577e-01       1
C1S.1           2.218970e-05  0.9218307 0.404 0.243 1.008078e-01       1
FOXO3.1         2.361131e-05  0.7925501 0.414 0.292 1.072662e-01       1
C3orf58.1       2.424374e-05  0.2818786 0.323 0.244 1.101393e-01       1
ALKAL2.1        2.583464e-05  0.7661401 0.302 0.160 1.173668e-01       1
EDN1.1          2.585646e-05 -0.4130073 0.130 0.329 1.174659e-01       1
LBH             2.592384e-05  0.2662020 0.432 0.298 1.177720e-01       1
PTPRJ.1         2.655785e-05  0.5410921 0.147 0.141 1.206523e-01       1
CHIC2           2.705466e-05  0.3068808 0.540 0.422 1.229093e-01       1
CAVIN3.1        2.802645e-05  0.5219739 0.523 0.431 1.273242e-01       1
CFAP97.1        2.807468e-05  0.6474637 0.449 0.330 1.275433e-01       1
LGALS1.1        2.835337e-05 -0.3475688 0.354 0.492 1.288094e-01       1
IFITM2.1        2.870882e-05  0.8845158 0.326 0.217 1.304242e-01       1
MIOS.1          2.913115e-05  0.7532153 0.337 0.202 1.323428e-01       1
GNPDA1.1        2.969899e-05  0.5692382 0.386 0.306 1.349225e-01       1
KCNG1.1         2.974963e-05  0.9244902 0.295 0.173 1.351526e-01       1
MGAT5           3.036815e-05 -0.3201117 0.172 0.331 1.379625e-01       1
MANF.1          3.113153e-05 -0.4717239 0.323 0.429 1.414305e-01       1
CAV1.1          3.142526e-05 -0.4642751 0.305 0.430 1.427649e-01       1
HMMR.1          3.189886e-05  0.6729303 0.249 0.135 1.449165e-01       1
EGFR.1          3.296538e-05  0.2505900 0.446 0.337 1.497617e-01       1
RND3            3.337554e-05 -0.3620260 0.337 0.452 1.516251e-01       1
CSNK1E          3.344676e-05 -0.3775936 0.281 0.443 1.519486e-01       1
ALDH1B1.1       3.465006e-05 -0.8125915 0.179 0.364 1.574152e-01       1
FRY-AS1.1       3.600125e-05  0.3352158 0.147 0.123 1.635537e-01       1
BEX3.1          3.603127e-05 -0.4637046 0.333 0.429 1.636901e-01       1
SLC27A4         3.869059e-05  0.3370633 0.204 0.157 1.757713e-01       1
FGD4.1          3.960136e-05  1.0583893 0.253 0.156 1.799090e-01       1
C1QTNF6.1       3.966780e-05  0.5103082 0.225 0.170 1.802108e-01       1
DOCK5.1         3.987027e-05 -0.6375940 0.333 0.464 1.811306e-01       1
FEZ1.1          4.007252e-05  0.2705400 0.246 0.156 1.820494e-01       1
RDH10.1         4.041228e-05 -0.7904819 0.263 0.448 1.835930e-01       1
TRAPPC5.1       4.091001e-05  0.4414524 0.204 0.132 1.858542e-01       1
FMO5.1          4.186961e-05  0.6485858 0.267 0.135 1.902137e-01       1
DNAJC1.1        4.242057e-05  0.5757112 0.540 0.391 1.927167e-01       1
MYH10.1         4.258163e-05 -0.6830637 0.284 0.398 1.934483e-01       1
COL6A2.1        4.261552e-05  0.4928721 0.456 0.387 1.936023e-01       1
HOXB-AS3.1      4.277923e-05  0.5114751 0.246 0.175 1.943460e-01       1
CHCHD10.1       4.357849e-05  0.3096429 0.418 0.298 1.979771e-01       1
SPOUT1.1        4.450280e-05  0.4969936 0.211 0.190 2.021762e-01       1
CNOT6L.1        4.508709e-05  0.7194577 0.361 0.234 2.048307e-01       1
TCEAL1.1        4.525788e-05  0.6359111 0.523 0.383 2.056065e-01       1
SLC5A6.1        4.550547e-05  0.4157537 0.414 0.338 2.067314e-01       1
C21orf91        4.552409e-05  0.3590017 0.256 0.184 2.068159e-01       1
IKZF2.1         4.670392e-05  0.4680191 0.144 0.108 2.121759e-01       1
SLC25A53.1      4.695900e-05  0.5099457 0.165 0.112 2.133347e-01       1
CENPS.1         4.719569e-05  0.3937268 0.277 0.195 2.144100e-01       1
TRMT61A         4.835546e-05  0.3926900 0.305 0.218 2.196789e-01       1
HIF1A.1         4.954215e-05  0.7580163 0.558 0.418 2.250700e-01       1
MC1R.1          5.024959e-05  0.5098355 0.204 0.144 2.282839e-01       1
F3.1            5.103247e-05  0.7115641 0.498 0.414 2.318405e-01       1
MAN1A1.1        5.154573e-05  0.2734178 0.533 0.402 2.341722e-01       1
P4HA3           5.195559e-05 -0.5937435 0.305 0.456 2.360342e-01       1
AC069544.1.1    5.269641e-05  0.4132349 0.186 0.114 2.393998e-01       1
CERCAM          5.414144e-05 -0.3516467 0.368 0.472 2.459646e-01       1
ALDOC           5.511699e-05 -0.5369798 0.123 0.159 2.503965e-01       1
ADRA2C.1        5.559945e-05  0.4648362 0.239 0.167 2.525883e-01       1
STRADB.1        5.693079e-05  0.7404539 0.326 0.191 2.586366e-01       1
ALDH3A2.1       5.759609e-05  0.6007926 0.263 0.180 2.616590e-01       1
HLA-A           5.824409e-05  0.3677806 0.544 0.406 2.646029e-01       1
DDX60.1         5.829234e-05  0.4866519 0.189 0.077 2.648221e-01       1
AC090204.1.1    6.047919e-05 -0.2881835 0.277 0.423 2.747569e-01       1
TNNC1.1         6.090512e-05 -0.8981617 0.102 0.245 2.766920e-01       1
NUDT8.1         6.171607e-05  0.6902940 0.502 0.400 2.803761e-01       1
EGLN1.1         6.274351e-05  0.3486395 0.326 0.240 2.850438e-01       1
OSBPL10.1       6.297691e-05  0.5723787 0.498 0.400 2.861041e-01       1
TRHDE.1         6.334154e-05  0.5304586 0.309 0.217 2.877606e-01       1
USP1.1          6.349835e-05  0.7342847 0.340 0.255 2.884730e-01       1
MYC.1           6.471368e-05 -0.6763172 0.204 0.339 2.939943e-01       1
TCF4            6.641383e-05 -0.4814990 0.302 0.439 3.017180e-01       1
C8orf33.1       6.765398e-05 -0.6466339 0.330 0.446 3.073520e-01       1
TUT4.1          6.770960e-05  0.5671722 0.460 0.342 3.076047e-01       1
SOAT1.1         6.854883e-05  0.6430009 0.284 0.195 3.114173e-01       1
AC104986.2.1    6.888700e-05  0.5599642 0.204 0.123 3.129536e-01       1
FAT1            6.902924e-05 -0.6863768 0.344 0.476 3.135998e-01       1
ALDH3B1         7.118561e-05  0.4058633 0.281 0.178 3.233962e-01       1
GSDMD.1         7.141821e-05  0.6309869 0.232 0.119 3.244529e-01       1
CXCL14.1        7.174899e-05 -0.5566720 0.221 0.359 3.259557e-01       1
NEAT1.1         7.315203e-05  0.6428536 0.544 0.431 3.323296e-01       1
MAF.1           7.385722e-05 -0.4105025 0.319 0.448 3.355333e-01       1
HACD2           7.409935e-05 -0.6962757 0.305 0.442 3.366333e-01       1
CAST.1          7.443343e-05  0.4023152 0.530 0.449 3.381511e-01       1
ANGPTL1         7.513614e-05  0.2505233 0.253 0.148 3.413435e-01       1
GSTZ1.1         7.593839e-05  0.5451423 0.337 0.294 3.449881e-01       1
PAPPA.1         7.684301e-05  0.4595697 0.460 0.399 3.490978e-01       1
HSBP1L1.1       7.709425e-05  0.9190937 0.319 0.202 3.502392e-01       1
PDLIM3.1        7.712556e-05 -0.8044277 0.214 0.411 3.503814e-01       1
RBMS3.1         7.884289e-05  0.3535241 0.540 0.432 3.581832e-01       1
OSR1.1          7.926148e-05  0.6736679 0.267 0.125 3.600849e-01       1
GOPC            7.929393e-05 -0.2550847 0.379 0.476 3.602323e-01       1
PTTG1IP.1       7.961761e-05 -0.3700654 0.365 0.483 3.617028e-01       1
HECTD3.1        7.997403e-05  0.4392578 0.260 0.200 3.633220e-01       1
PPA1            8.248669e-05  0.3560299 0.540 0.412 3.747371e-01       1
LY6G5C.1        8.472678e-05 -0.4488301 0.105 0.157 3.849137e-01       1
ERRFI1.1        8.801973e-05  0.5390869 0.365 0.276 3.998737e-01       1
LINC01503.1     8.887948e-05  0.4957892 0.411 0.324 4.037795e-01       1
MBNL1.1         9.014758e-05 -0.5234728 0.393 0.493 4.095404e-01       1
SMIM3.1         9.040320e-05  0.5597307 0.554 0.407 4.107017e-01       1
RAB13.1         9.047636e-05  0.4543395 0.523 0.388 4.110341e-01       1
HGH1.1          9.132169e-05  0.4226549 0.281 0.199 4.148744e-01       1
HSPG2           9.352451e-05 -0.3913811 0.368 0.468 4.248818e-01       1
MBLAC1.1        9.462146e-05  0.7370050 0.260 0.158 4.298653e-01       1
LPCAT2.1        9.508466e-05 -0.6328019 0.253 0.387 4.319696e-01       1
ACVR2A          9.589330e-05 -0.5172271 0.253 0.422 4.356432e-01       1
CEP295.1        9.612551e-05 -0.5509933 0.165 0.235 4.366982e-01       1
ZNF565.1        9.666920e-05  0.4994885 0.165 0.111 4.391682e-01       1
PRSS12.1        9.839754e-05 -0.9557911 0.123 0.209 4.470200e-01       1
SLC24A1.1       9.931271e-05  0.7599197 0.207 0.101 4.511776e-01       1
ARSJ            9.931271e-05 -1.0444418 0.161 0.307 4.511776e-01       1
ZMIZ1           9.979351e-05  0.2730987 0.442 0.375 4.533619e-01       1
AC013652.1.1    9.983244e-05 -1.1160648 0.088 0.182 4.535388e-01       1
HIST1H4C.1      1.006802e-04  0.5418646 0.421 0.329 4.573903e-01       1
CYR61           1.016153e-04 -0.2923962 0.337 0.437 4.616385e-01       1
PPFIA3.1        1.024337e-04  0.4335148 0.207 0.151 4.653562e-01       1
MRPS2.1         1.025587e-04  0.7259218 0.389 0.304 4.659242e-01       1
COL7A1          1.035933e-04  0.3243483 0.284 0.157 4.706242e-01       1
KRT18.1         1.038027e-04 -0.8849975 0.291 0.416 4.715758e-01       1
SAXO2.1         1.040935e-04  0.6417570 0.196 0.092 4.728966e-01       1
PPFIBP1.1       1.049743e-04 -0.5830525 0.295 0.456 4.768984e-01       1
CNN2            1.053539e-04 -0.4920581 0.361 0.469 4.786229e-01       1
XPC.1           1.061596e-04  0.6054294 0.421 0.338 4.822829e-01       1
ANKFN1.1        1.062865e-04 -0.9034470 0.119 0.306 4.828594e-01       1
AC025171.4.1    1.067138e-04 -0.3188180 0.049 0.112 4.848007e-01       1
TKT.1           1.087420e-04  0.7454402 0.554 0.430 4.940151e-01       1
JAGN1           1.116512e-04  0.2849535 0.491 0.365 5.072313e-01       1
TSLP.1          1.128606e-04  0.5869966 0.239 0.123 5.127257e-01       1
VGLL3.1         1.129525e-04 -0.3640746 0.375 0.476 5.131434e-01       1
MMP23B.1        1.134945e-04  0.4765781 0.505 0.376 5.156056e-01       1
SELENBP1.1      1.135384e-04  0.7223195 0.207 0.144 5.158050e-01       1
MSI2.1          1.139948e-04 -0.5923718 0.239 0.366 5.178783e-01       1
REC8.1          1.164311e-04  0.4839125 0.175 0.107 5.289463e-01       1
WWC2.1          1.185382e-04 -0.6765227 0.291 0.448 5.385191e-01       1
COL15A1         1.233256e-04 -0.4550972 0.267 0.414 5.602683e-01       1
DPT             1.259225e-04  0.3731546 0.228 0.174 5.720658e-01       1
HSPB6           1.261722e-04  0.3727106 0.218 0.176 5.732004e-01       1
DLC1            1.294186e-04 -0.4809983 0.260 0.410 5.879485e-01       1
CORO2B.1        1.340070e-04 -0.7148380 0.035 0.174 6.087937e-01       1
SPTAN1.1        1.345387e-04 -0.5271467 0.340 0.459 6.112092e-01       1
SIX4.1          1.384213e-04 -0.2602476 0.207 0.184 6.288481e-01       1
HMGN3.1         1.400701e-04  0.4696337 0.526 0.444 6.363384e-01       1
AC132192.2.1    1.403463e-04  0.7044618 0.193 0.089 6.375933e-01       1
TXN             1.488828e-04  0.2619306 0.326 0.425 6.763746e-01       1
TACC1.1         1.492975e-04  0.3890838 0.428 0.310 6.782584e-01       1
RNF152.1        1.569898e-04 -0.3799587 0.260 0.410 7.132045e-01       1
PRIM1           1.597191e-04  0.3467000 0.165 0.106 7.256038e-01       1
DBN1.1          1.612268e-04 -0.4201647 0.368 0.522 7.324534e-01       1
DYNLL2.1        1.637050e-04  0.3192623 0.375 0.289 7.437120e-01       1
TWSG1.1         1.647965e-04  0.4359601 0.481 0.380 7.486707e-01       1
SCD5.1          1.654376e-04 -0.5139051 0.123 0.164 7.515829e-01       1
AP002884.1.1    1.655057e-04  0.9580301 0.358 0.212 7.518923e-01       1
PGRMC1.1        1.658301e-04 -0.2766362 0.368 0.510 7.533659e-01       1
PWAR6.1         1.679153e-04 -0.5801259 0.204 0.378 7.628394e-01       1
XPOT.1          1.729672e-04  0.4631055 0.498 0.407 7.857902e-01       1
AGRN            1.748642e-04 -0.9204870 0.158 0.367 7.944078e-01       1
LHFPL6.1        1.793436e-04  0.3746947 0.428 0.315 8.147579e-01       1
CYTOR.1         1.794133e-04 -0.5995794 0.340 0.454 8.150746e-01       1
RGS4.1          1.795437e-04 -0.7820550 0.084 0.271 8.156668e-01       1
CTNNB1.1        1.828616e-04 -0.6645507 0.358 0.444 8.307403e-01       1
ODC1.1          1.830047e-04 -0.3270786 0.365 0.466 8.313904e-01       1
CENPK           1.839276e-04  0.2740723 0.126 0.145 8.355832e-01       1
SCG2.1          1.901423e-04  0.7181993 0.418 0.286 8.638165e-01       1
CD320.1         1.906862e-04 -0.5227438 0.347 0.469 8.662873e-01       1
TMEM229B.1      1.981290e-04  0.7741910 0.253 0.187 9.001002e-01       1
S1PR3.1         1.989772e-04  0.4547787 0.260 0.244 9.039535e-01       1
BMP4.1          2.023875e-04 -0.4518946 0.281 0.435 9.194462e-01       1
ZSWIM6.1        2.038771e-04 -0.6926330 0.309 0.419 9.262137e-01       1
SYT8.1          2.069610e-04 -0.3045458 0.137 0.160 9.402236e-01       1
TARSL2.1        2.084078e-04 -0.4153757 0.344 0.462 9.467966e-01       1
TRIB3.1         2.206168e-04  0.9522557 0.319 0.196 1.000000e+00       1
RAB11FIP5.1     2.226591e-04 -0.3761543 0.214 0.277 1.000000e+00       1
EPB41L4A-AS1.1  2.248052e-04  0.7060953 0.537 0.407 1.000000e+00       1
SLC35A3.1       2.250638e-04 -0.6897624 0.242 0.394 1.000000e+00       1
IFITM1.1        2.311811e-04  1.3316290 0.225 0.094 1.000000e+00       1
CEP44.1         2.313602e-04  0.3643778 0.281 0.232 1.000000e+00       1
IFNGR1.1        2.318923e-04  0.4954876 0.411 0.296 1.000000e+00       1
SDCBP.1         2.343906e-04  0.3760767 0.547 0.452 1.000000e+00       1
AC016831.1.1    2.346596e-04 -0.4552802 0.123 0.243 1.000000e+00       1
TSPAN6.1        2.347496e-04 -0.4025724 0.365 0.448 1.000000e+00       1
TRAF4.1         2.369141e-04 -0.6115947 0.218 0.390 1.000000e+00       1
PLEKHA2.1       2.380033e-04  0.4138094 0.319 0.206 1.000000e+00       1
ECM1.1          2.391752e-04 -0.7810648 0.161 0.363 1.000000e+00       1
MOV10L1         2.400110e-04  0.3262473 0.207 0.139 1.000000e+00       1
KCNC4.1         2.414840e-04  0.3378404 0.281 0.286 1.000000e+00       1
RAD51AP1.1      2.458541e-04  0.3921765 0.126 0.106 1.000000e+00       1
AL451085.2      2.514463e-04  0.2540922 0.165 0.114 1.000000e+00       1
ORAI3           2.537555e-04  0.2700120 0.446 0.356 1.000000e+00       1
PAM             2.537555e-04 -0.4344597 0.354 0.465 1.000000e+00       1
AVPI1.1         2.563764e-04  0.4597464 0.256 0.198 1.000000e+00       1
FADS1.1         2.593184e-04  0.5566725 0.554 0.434 1.000000e+00       1
SFMBT2.1        2.706789e-04  0.3145365 0.249 0.164 1.000000e+00       1
ENTPD4          2.714058e-04 -0.4530132 0.344 0.467 1.000000e+00       1
ZBTB16.1        2.731548e-04  1.3356950 0.312 0.166 1.000000e+00       1
SP4             2.745085e-04 -0.3150576 0.126 0.123 1.000000e+00       1
PTPRA.1         2.810282e-04  0.4908613 0.389 0.331 1.000000e+00       1
FAM96B.1        2.824135e-04  0.4278454 0.523 0.439 1.000000e+00       1
PAPSS2          2.824135e-04 -0.3014041 0.361 0.480 1.000000e+00       1
OCA2.1          2.896544e-04  0.7785904 0.319 0.184 1.000000e+00       1
SSR4.1          2.996554e-04 -0.2648160 0.382 0.469 1.000000e+00       1
PLPPR3.1        3.019179e-04 -1.2446748 0.042 0.219 1.000000e+00       1
FZD2.1          3.020349e-04  0.4275284 0.404 0.284 1.000000e+00       1
PKP2.1          3.052353e-04  0.3283563 0.404 0.315 1.000000e+00       1
SOCS2.1         3.063850e-04  1.2818341 0.284 0.101 1.000000e+00       1
SERPING1.1      3.071934e-04  0.8774564 0.333 0.181 1.000000e+00       1
GALNT18.1       3.093967e-04 -0.2865281 0.277 0.440 1.000000e+00       1
TCTN3.1         3.106788e-04 -0.6085966 0.309 0.434 1.000000e+00       1
SLC19A2.1       3.213571e-04  0.5014729 0.418 0.346 1.000000e+00       1
GTF2B.1         3.290352e-04  0.4500617 0.382 0.315 1.000000e+00       1
BEND6           3.315079e-04  0.3144796 0.295 0.253 1.000000e+00       1
HINT1.1         3.326270e-04  0.6343739 0.547 0.449 1.000000e+00       1
TMEM87B         3.331253e-04 -0.3786778 0.326 0.464 1.000000e+00       1
DYNLT3          3.410660e-04 -0.4730755 0.319 0.475 1.000000e+00       1
BRCC3.1         3.458116e-04  0.4269900 0.298 0.210 1.000000e+00       1
TMEM88.1        3.516656e-04  0.7580865 0.467 0.370 1.000000e+00       1
PSME1.1         3.536392e-04  0.7394626 0.421 0.276 1.000000e+00       1
DCAF16.1        3.597519e-04  0.3807861 0.274 0.227 1.000000e+00       1
THY1            3.684177e-04  0.3376408 0.375 0.448 1.000000e+00       1
RASSF4.1        3.700686e-04  0.6061996 0.274 0.207 1.000000e+00       1
PDCD4-AS1       3.739387e-04  0.2988261 0.260 0.222 1.000000e+00       1
CTNNAL1.1       3.757482e-04  0.3715912 0.432 0.321 1.000000e+00       1
KLHL5.1         3.795330e-04  0.3835630 0.316 0.272 1.000000e+00       1
TTC21B.1        3.844910e-04  0.4246786 0.172 0.124 1.000000e+00       1
SPRED1          3.896560e-04 -0.4871654 0.196 0.377 1.000000e+00       1
MCM5            3.932786e-04  0.3139558 0.112 0.146 1.000000e+00       1
HSPB3.1         3.956109e-04 -0.3621393 0.182 0.156 1.000000e+00       1
RTN3.1          3.959089e-04  0.4259471 0.449 0.358 1.000000e+00       1
RNF115.1        3.963487e-04  0.6414009 0.467 0.382 1.000000e+00       1
POPDC2.1        4.028473e-04  0.9120956 0.242 0.157 1.000000e+00       1
RBPMS.1         4.062891e-04  0.6201072 0.400 0.308 1.000000e+00       1
MINCR.1         4.120245e-04  0.7919266 0.260 0.178 1.000000e+00       1
SEPSECS.1       4.137030e-04  0.4347896 0.207 0.125 1.000000e+00       1
PTHLH.1         4.137042e-04 -0.8202937 0.126 0.296 1.000000e+00       1
SPATA7.1        4.138531e-04  0.6323667 0.235 0.156 1.000000e+00       1
LASP1           4.138585e-04 -0.5184842 0.372 0.484 1.000000e+00       1
EPHB3           4.150724e-04 -0.5204516 0.063 0.149 1.000000e+00       1
HEYL.1          4.231139e-04  0.3314313 0.154 0.175 1.000000e+00       1
YWHAG.1         4.256167e-04 -0.3104431 0.379 0.477 1.000000e+00       1
SLC7A11.1       4.386495e-04  0.5895955 0.333 0.223 1.000000e+00       1
ADARB1.1        4.399439e-04  0.6064961 0.396 0.322 1.000000e+00       1
SMIM10.1        4.405874e-04  0.6192762 0.218 0.132 1.000000e+00       1
PLEKHA7.1       4.446519e-04 -0.5420133 0.088 0.135 1.000000e+00       1
C19orf48        4.515629e-04  0.3408660 0.312 0.254 1.000000e+00       1
DHCR7.1         4.597482e-04 -0.4964545 0.330 0.469 1.000000e+00       1
AC092807.3.1    4.626198e-04 -0.8432310 0.214 0.385 1.000000e+00       1
CAV2            4.641471e-04 -0.3083569 0.393 0.478 1.000000e+00       1
MYL12B.1        4.753175e-04 -0.5888538 0.344 0.463 1.000000e+00       1
TXNRD1.1        4.788045e-04  0.7114449 0.519 0.430 1.000000e+00       1
SNHG12.1        4.840791e-04  0.9505426 0.354 0.246 1.000000e+00       1
NEDD4.1         4.870933e-04  0.5935036 0.418 0.338 1.000000e+00       1
NOP58           4.876271e-04  0.3035889 0.368 0.281 1.000000e+00       1
MAGI1           4.894097e-04 -0.4075062 0.228 0.382 1.000000e+00       1
THRA.1          4.915568e-04  0.8521965 0.309 0.219 1.000000e+00       1
GOLGA7.1        4.940729e-04 -0.6866929 0.379 0.458 1.000000e+00       1
TBX3.1          5.004154e-04  0.4459524 0.435 0.345 1.000000e+00       1
EMILIN1.1       5.122076e-04 -0.3953448 0.147 0.276 1.000000e+00       1
SDC1.1          5.202804e-04 -0.4847082 0.189 0.301 1.000000e+00       1
VAMP3.1         5.214162e-04  0.4594163 0.453 0.361 1.000000e+00       1
RNF112.1        5.233136e-04  0.3613906 0.204 0.172 1.000000e+00       1
SEMA3D.1        5.358023e-04 -0.7379957 0.053 0.157 1.000000e+00       1
AATF.1          5.420560e-04  0.2991251 0.361 0.296 1.000000e+00       1
SLC2A1          5.465909e-04 -0.3741953 0.214 0.349 1.000000e+00       1
HMG20A          5.481765e-04 -0.4561299 0.200 0.305 1.000000e+00       1
RASSF7.1        5.616251e-04 -0.4502363 0.260 0.368 1.000000e+00       1
MICALL2.1       5.628360e-04  0.2915662 0.221 0.168 1.000000e+00       1
RBMS1           5.638624e-04  0.2972946 0.540 0.432 1.000000e+00       1
ANGPTL4.1       5.652896e-04  0.7337776 0.284 0.193 1.000000e+00       1
DUSP16.1        5.724800e-04  0.6466650 0.165 0.074 1.000000e+00       1
NTM.1           5.728906e-04 -1.2225134 0.074 0.258 1.000000e+00       1
DDR2            5.785014e-04  0.2739130 0.347 0.254 1.000000e+00       1
SYNPO           5.812216e-04 -0.3567086 0.323 0.446 1.000000e+00       1
FARP1           5.829006e-04 -0.2739955 0.368 0.500 1.000000e+00       1
RALBP1          5.831109e-04  0.3979174 0.523 0.427 1.000000e+00       1
BCAT1           5.877529e-04 -0.2703497 0.382 0.485 1.000000e+00       1
CSGALNACT2.1    5.973553e-04 -0.3576029 0.347 0.440 1.000000e+00       1
ARHGAP29.1      6.051419e-04  0.4720466 0.344 0.266 1.000000e+00       1
ACOX1.1         6.103854e-04 -0.6067063 0.340 0.458 1.000000e+00       1
SEPT11.1        6.108246e-04 -0.2915077 0.382 0.487 1.000000e+00       1
HOXA10-AS       6.174427e-04  0.3276458 0.284 0.187 1.000000e+00       1
ATP2B4.1        6.361026e-04  0.4191127 0.386 0.332 1.000000e+00       1
FADS3.1         6.438965e-04  0.5846222 0.463 0.357 1.000000e+00       1
BIN1.1          6.494522e-04  0.6441850 0.295 0.212 1.000000e+00       1
SPATA5L1.1      6.545807e-04 -0.3350809 0.098 0.193 1.000000e+00       1
CCDC28B.1       6.595170e-04 -0.3718951 0.267 0.386 1.000000e+00       1
OTUD3.1         6.825128e-04 -0.3629559 0.267 0.234 1.000000e+00       1
MINDY2.1        6.842186e-04 -0.4973345 0.382 0.468 1.000000e+00       1
CALM1.1         6.856837e-04  0.5164624 0.526 0.442 1.000000e+00       1
TFPI.1          6.994987e-04  0.5985595 0.354 0.223 1.000000e+00       1
COL6A3          6.999160e-04 -0.3424191 0.319 0.419 1.000000e+00       1
RTN4            7.196848e-04  0.3275982 0.547 0.430 1.000000e+00       1
AK6             7.204520e-04  0.3038096 0.481 0.382 1.000000e+00       1
TEX22.1         7.227582e-04 -0.4136421 0.084 0.215 1.000000e+00       1
IGF2BP3.1       7.235291e-04  0.6394048 0.382 0.272 1.000000e+00       1
MICAL3.1        7.255873e-04 -0.7210512 0.119 0.279 1.000000e+00       1
AC097103.2      7.315341e-04  0.3506490 0.225 0.170 1.000000e+00       1
HLA-DPB1.1      7.317944e-04  0.4919978 0.295 0.195 1.000000e+00       1
SLC23A2.1       7.630374e-04 -0.6494502 0.105 0.200 1.000000e+00       1
CNPY2.1         7.829816e-04 -0.3630125 0.361 0.434 1.000000e+00       1
PATJ.1          7.899218e-04 -0.3433560 0.056 0.171 1.000000e+00       1
TAF1D.1         8.133764e-04  0.5974249 0.547 0.432 1.000000e+00       1
CEP290.1        8.139484e-04  0.3868242 0.323 0.263 1.000000e+00       1
BRSK2           8.147932e-04  0.2681885 0.260 0.168 1.000000e+00       1
CDK6            8.243287e-04 -0.3307529 0.372 0.485 1.000000e+00       1
KAT8.1          8.351236e-04  0.4051439 0.235 0.192 1.000000e+00       1
HIP1.1          8.484288e-04  0.5925679 0.288 0.191 1.000000e+00       1
HEPH            8.592144e-04 -0.3086277 0.053 0.163 1.000000e+00       1
COL5A1.1        8.704347e-04 -0.5111727 0.368 0.439 1.000000e+00       1
BIN3.1          8.731666e-04  0.4854004 0.232 0.172 1.000000e+00       1
RASA3.1         8.765551e-04  0.5122036 0.253 0.219 1.000000e+00       1
FJX1.1          8.790143e-04 -0.5259738 0.221 0.351 1.000000e+00       1
TARBP1          8.793220e-04 -0.2583818 0.133 0.210 1.000000e+00       1
RAB29.1         8.886047e-04  0.3568779 0.358 0.263 1.000000e+00       1
IFNGR2.1        9.055425e-04 -0.3221492 0.319 0.464 1.000000e+00       1
GBP1.1          9.122094e-04 -0.6472315 0.200 0.284 1.000000e+00       1
ZNF358.1        9.131659e-04  0.4256641 0.404 0.339 1.000000e+00       1
SEPT4.1         9.386819e-04  0.5825351 0.404 0.252 1.000000e+00       1
ZFYVE27.1       9.393364e-04  0.2946875 0.232 0.199 1.000000e+00       1
GPC6.1          9.488735e-04  0.6958897 0.435 0.350 1.000000e+00       1
HEATR5A         9.618415e-04 -0.3895108 0.270 0.381 1.000000e+00       1
TM4SF1.1        9.648555e-04  0.3973835 0.207 0.211 1.000000e+00       1
FERMT2.1        9.756499e-04 -0.4097060 0.365 0.439 1.000000e+00       1
PITPNM3         9.892969e-04 -0.5834123 0.168 0.205 1.000000e+00       1
CNTLN.1         1.006261e-03  0.7103274 0.274 0.210 1.000000e+00       1
AHNAK.1         1.007658e-03  0.5867543 0.526 0.407 1.000000e+00       1
SESN2.1         1.020660e-03  0.3869121 0.309 0.257 1.000000e+00       1
RARB.1          1.025966e-03 -0.6394497 0.116 0.164 1.000000e+00       1
SHPK            1.047490e-03 -0.4055715 0.102 0.109 1.000000e+00       1
DNAJB2.1        1.060238e-03  0.3195101 0.281 0.228 1.000000e+00       1
ZNF563.1        1.071638e-03 -0.5847723 0.074 0.167 1.000000e+00       1
CCDC97.1        1.095924e-03  0.4137246 0.221 0.184 1.000000e+00       1
BNC1            1.100090e-03  0.3578580 0.221 0.155 1.000000e+00       1
AP002761.3.1    1.117266e-03  0.3548868 0.221 0.162 1.000000e+00       1
CTSV.1          1.133903e-03 -0.8709618 0.098 0.254 1.000000e+00       1
FOXF2.1         1.153544e-03  0.5153430 0.246 0.182 1.000000e+00       1
GUCY1A1         1.167041e-03 -0.5849207 0.133 0.324 1.000000e+00       1
TNFRSF1A.1      1.193765e-03  0.3778414 0.540 0.436 1.000000e+00       1
EPRS.1          1.197857e-03  0.4836484 0.540 0.423 1.000000e+00       1
NQO1.1          1.213104e-03  0.3283345 0.340 0.295 1.000000e+00       1
NENF.1          1.249693e-03  0.4411868 0.470 0.364 1.000000e+00       1
CRY1            1.263402e-03  0.2532897 0.305 0.257 1.000000e+00       1
CYP27A1.1       1.265109e-03  0.6847278 0.225 0.120 1.000000e+00       1
AC020928.1      1.267274e-03  0.2920497 0.196 0.127 1.000000e+00       1
AC098487.1.1    1.272900e-03  0.3071949 0.232 0.174 1.000000e+00       1
WNT5B.1         1.285100e-03 -0.6976157 0.193 0.315 1.000000e+00       1
FLOT1.1         1.285976e-03  0.6610847 0.474 0.414 1.000000e+00       1
AL355338.1.1    1.292117e-03 -0.5593414 0.211 0.314 1.000000e+00       1
C1QL1.1         1.328627e-03  0.5796695 0.168 0.159 1.000000e+00       1
ASS1.1          1.340862e-03  0.5036623 0.495 0.412 1.000000e+00       1
UTP23.1         1.349530e-03  0.2735409 0.333 0.258 1.000000e+00       1
HMOX1.1         1.375379e-03  0.6322695 0.288 0.192 1.000000e+00       1
MSC.1           1.375379e-03  0.5883756 0.347 0.197 1.000000e+00       1
TLCD2.1         1.436676e-03  0.5177642 0.425 0.341 1.000000e+00       1
PLXNA3          1.441046e-03 -0.3118727 0.298 0.411 1.000000e+00       1
TMEM44          1.456182e-03  0.3081536 0.158 0.130 1.000000e+00       1
TMEM11.1        1.457662e-03  0.6317702 0.421 0.366 1.000000e+00       1
CBR3.1          1.468997e-03  0.4473078 0.214 0.130 1.000000e+00       1
FAM133B.1       1.469992e-03  0.6148743 0.389 0.301 1.000000e+00       1
CDCA7L.1        1.471961e-03  0.5089804 0.207 0.176 1.000000e+00       1
BBX.1           1.487416e-03  0.3928384 0.551 0.441 1.000000e+00       1
PIM2            1.501996e-03 -0.3946471 0.130 0.136 1.000000e+00       1
AP000695.2.1    1.530498e-03  0.3309846 0.193 0.166 1.000000e+00       1
FRZB.1          1.555893e-03  0.4877560 0.316 0.244 1.000000e+00       1
DET1.1          1.584808e-03  0.5040327 0.221 0.150 1.000000e+00       1
COMMD7.1        1.609958e-03  0.4481741 0.432 0.341 1.000000e+00       1
DUSP28          1.623470e-03  0.2778266 0.232 0.200 1.000000e+00       1
DNAJB1.1        1.662968e-03  0.4975799 0.404 0.346 1.000000e+00       1
ADAM19.1        1.666296e-03 -0.5665435 0.211 0.387 1.000000e+00       1
DSEL.1          1.671864e-03  0.6997951 0.400 0.313 1.000000e+00       1
NINJ1           1.735953e-03  0.3650536 0.337 0.258 1.000000e+00       1
IRF2BPL         1.745790e-03  0.2546823 0.365 0.272 1.000000e+00       1
MPLKIP.1        1.745790e-03  0.4352239 0.537 0.425 1.000000e+00       1
ZNF324.1        1.746358e-03 -0.4995472 0.133 0.147 1.000000e+00       1
SNAI2.1         1.782095e-03  0.3666517 0.498 0.415 1.000000e+00       1
EIF4A3          1.810677e-03  0.3381265 0.481 0.400 1.000000e+00       1
HMGA2.1         1.814880e-03 -0.5423795 0.182 0.320 1.000000e+00       1
REL.1           1.815481e-03  0.6082710 0.267 0.188 1.000000e+00       1
USP20           1.816657e-03 -0.4360945 0.084 0.158 1.000000e+00       1
SPATA6          1.822709e-03  0.2623678 0.168 0.122 1.000000e+00       1
KREMEN1.1       1.872810e-03  0.4357844 0.168 0.181 1.000000e+00       1
SLC25A45        1.878393e-03  0.3699864 0.253 0.181 1.000000e+00       1
SLC39A2.1       1.911517e-03 -0.3902374 0.095 0.115 1.000000e+00       1
TPGS2.1         1.933724e-03 -0.2971884 0.375 0.472 1.000000e+00       1
KIF20B.1        1.999721e-03  0.4935260 0.204 0.150 1.000000e+00       1
NUPR1           2.025501e-03  0.2559684 0.519 0.404 1.000000e+00       1
TRIM16L.1       2.027499e-03  0.7825697 0.326 0.215 1.000000e+00       1
IFI16           2.045542e-03 -0.6395697 0.337 0.459 1.000000e+00       1
EFNB2           2.079339e-03 -0.2928064 0.260 0.424 1.000000e+00       1
LITAF.1         2.118492e-03  0.6011756 0.323 0.217 1.000000e+00       1
TNFAIP3         2.123325e-03  0.2665761 0.151 0.160 1.000000e+00       1
GALNT7          2.192397e-03 -0.3683908 0.312 0.410 1.000000e+00       1
SBNO1.1         2.283455e-03  0.5444192 0.411 0.325 1.000000e+00       1
RCAN1.1         2.300549e-03 -0.5403402 0.277 0.353 1.000000e+00       1
FAM117B.1       2.332161e-03  0.6377358 0.460 0.347 1.000000e+00       1
TRAF2.1         2.355722e-03  0.4348270 0.168 0.110 1.000000e+00       1
YDJC            2.366435e-03  0.4317446 0.375 0.312 1.000000e+00       1
STX12.1         2.390316e-03  0.5268860 0.449 0.376 1.000000e+00       1
PBX1.1          2.390316e-03  0.3321427 0.333 0.322 1.000000e+00       1
EIF5.1          2.399619e-03  0.7436261 0.502 0.450 1.000000e+00       1
TESK2.1         2.407391e-03  0.4629362 0.161 0.138 1.000000e+00       1
MIATNB.1        2.426907e-03  0.6464203 0.211 0.141 1.000000e+00       1
WWC3.1          2.441892e-03  0.5371156 0.263 0.230 1.000000e+00       1
DDX5.1          2.473645e-03  0.3782405 0.526 0.434 1.000000e+00       1
C18orf25.1      2.487254e-03  0.3440060 0.274 0.248 1.000000e+00       1
MSMO1.1         2.500927e-03 -0.6043161 0.354 0.474 1.000000e+00       1
SELENOP.1       2.502520e-03  0.5426700 0.295 0.234 1.000000e+00       1
NOP53.1         2.513056e-03  0.4811001 0.460 0.369 1.000000e+00       1
MT-CO1.1        2.519545e-03  0.2835156 0.477 0.415 1.000000e+00       1
SALL4           2.523566e-03  0.2679688 0.179 0.143 1.000000e+00       1
PLAU.1          2.557952e-03 -0.5982491 0.165 0.159 1.000000e+00       1
CRTAP.1         2.612011e-03  0.6024061 0.505 0.415 1.000000e+00       1
RSPO2.1         2.616209e-03  0.5909578 0.411 0.390 1.000000e+00       1
SLC37A3.1       2.626313e-03  0.3377212 0.277 0.264 1.000000e+00       1
PPP1R14A.1      2.644051e-03 -0.6723148 0.260 0.344 1.000000e+00       1
TUSC1           2.657667e-03  0.4419682 0.225 0.169 1.000000e+00       1
MIR22HG.1       2.701504e-03  0.6184073 0.456 0.380 1.000000e+00       1
TNKS1BP1.1      2.708436e-03  0.5063175 0.365 0.285 1.000000e+00       1
FOXN2.1         2.717964e-03  0.5751576 0.267 0.172 1.000000e+00       1
AC103740.2.1    2.721465e-03 -0.8385570 0.053 0.113 1.000000e+00       1
ABCD3.1         2.732827e-03  0.4429316 0.449 0.381 1.000000e+00       1
P4HA1           2.746855e-03 -0.2941053 0.400 0.464 1.000000e+00       1
VTI1A.1         2.756536e-03  0.3246604 0.242 0.206 1.000000e+00       1
TNFRSF12A.1     2.758301e-03 -0.2653344 0.386 0.481 1.000000e+00       1
FAM84A.1        2.760942e-03 -0.6673342 0.112 0.160 1.000000e+00       1
PCCB            2.770665e-03 -0.5242573 0.137 0.242 1.000000e+00       1
CPA4.1          2.771167e-03 -1.1226614 0.151 0.351 1.000000e+00       1
DUSP6.1         2.825178e-03 -0.7099264 0.119 0.261 1.000000e+00       1
C4BPB.1         2.825185e-03 -0.9007476 0.084 0.211 1.000000e+00       1
CDCA3.1         2.855082e-03  0.6877787 0.137 0.101 1.000000e+00       1
HSPA12A.1       2.876094e-03 -0.6970388 0.165 0.258 1.000000e+00       1
FOXQ1.1         2.916673e-03  0.5962876 0.242 0.178 1.000000e+00       1
GLUL.1          2.923181e-03  0.5252498 0.344 0.235 1.000000e+00       1
NPW             2.937163e-03 -0.4374919 0.112 0.195 1.000000e+00       1
NR2F1.1         2.945583e-03  0.4701989 0.453 0.386 1.000000e+00       1
PLXNA4          2.946492e-03  0.4026067 0.186 0.187 1.000000e+00       1
DUSP8           2.950247e-03 -0.5288120 0.137 0.141 1.000000e+00       1
GLIS3           2.974749e-03 -0.3079023 0.218 0.313 1.000000e+00       1
WDR1.1          3.039653e-03 -0.4238186 0.418 0.492 1.000000e+00       1
FAM122C         3.057047e-03  0.2614337 0.161 0.122 1.000000e+00       1
SYNM            3.091131e-03  0.3221458 0.488 0.403 1.000000e+00       1
INTS13.1        3.151362e-03 -0.4436169 0.179 0.260 1.000000e+00       1
MDFI.1          3.215711e-03  0.5275828 0.498 0.403 1.000000e+00       1
PSIP1.1         3.238103e-03  0.3467970 0.305 0.214 1.000000e+00       1
GCNT1           3.258540e-03  0.2851480 0.221 0.164 1.000000e+00       1
TYRP1.1         3.294703e-03  0.4349382 0.263 0.169 1.000000e+00       1
ASPHD1          3.316560e-03  0.4110712 0.407 0.344 1.000000e+00       1
HSPB11.1        3.325964e-03  0.4443956 0.523 0.439 1.000000e+00       1
CMSS1.1         3.336442e-03  0.6466827 0.330 0.186 1.000000e+00       1
TRIM65.1        3.352209e-03  0.3346245 0.228 0.180 1.000000e+00       1
SQSTM1.1        3.369113e-03  0.7291988 0.491 0.430 1.000000e+00       1
FBXO5           3.386077e-03  0.3568959 0.144 0.115 1.000000e+00       1
AC010336.1.1    3.418120e-03 -0.7945246 0.081 0.144 1.000000e+00       1
NPTXR.1         3.418135e-03  0.4290202 0.281 0.256 1.000000e+00       1
EXPH5.1         3.500543e-03  0.4436949 0.204 0.161 1.000000e+00       1
PRKD1.1         3.571333e-03  0.5358270 0.246 0.148 1.000000e+00       1
PPP2R3C         3.620776e-03  0.2897676 0.382 0.325 1.000000e+00       1
LAGE3           3.674261e-03  0.3020341 0.505 0.403 1.000000e+00       1
SLC16A4         3.714555e-03  0.3569643 0.319 0.285 1.000000e+00       1
PTPN9.1         3.849793e-03  0.3925299 0.312 0.228 1.000000e+00       1
SNAPC1          3.852185e-03  0.2648667 0.326 0.255 1.000000e+00       1
NCS1.1          3.860563e-03 -0.2685237 0.225 0.334 1.000000e+00       1
AKR1C1.1        3.867757e-03 -0.4001918 0.077 0.168 1.000000e+00       1
FRMD6           3.874963e-03 -0.3373738 0.337 0.436 1.000000e+00       1
IMPA2.1         3.880970e-03  0.6431310 0.305 0.179 1.000000e+00       1
ZNF385A.1       3.946468e-03  0.3001816 0.175 0.166 1.000000e+00       1
AL590705.1.1    4.014210e-03  0.3742573 0.228 0.171 1.000000e+00       1
ZSWIM3.1        4.093150e-03  0.8894350 0.196 0.120 1.000000e+00       1
LENG8.1         4.103281e-03  0.4714829 0.523 0.435 1.000000e+00       1
JUP.1           4.107081e-03 -0.8934410 0.098 0.269 1.000000e+00       1
FENDRR.1        4.110886e-03  0.4118566 0.256 0.166 1.000000e+00       1
PSMC6.1         4.112155e-03  0.4179273 0.467 0.403 1.000000e+00       1
SNRPN.1         4.119776e-03 -0.2567011 0.368 0.471 1.000000e+00       1
YPEL3           4.123590e-03  0.2546004 0.386 0.326 1.000000e+00       1
HSD17B7.1       4.142626e-03  0.2975742 0.228 0.191 1.000000e+00       1
KLHDC10.1       4.178621e-03  0.5227954 0.439 0.395 1.000000e+00       1
IRS1            4.223831e-03  0.2862147 0.242 0.187 1.000000e+00       1
VAMP8           4.270934e-03  0.3326242 0.277 0.223 1.000000e+00       1
AKAP17A.1       4.280148e-03  0.5857740 0.309 0.243 1.000000e+00       1
SYNGR2.1        4.319792e-03 -0.4693559 0.260 0.436 1.000000e+00       1
C1orf216.1      4.329087e-03  0.3199287 0.281 0.243 1.000000e+00       1
ISG15.1         4.339737e-03 -0.4144013 0.288 0.387 1.000000e+00       1
CARHSP1.1       4.343737e-03  0.3950003 0.481 0.448 1.000000e+00       1
SNCAIP.1        4.366455e-03  0.3458804 0.168 0.124 1.000000e+00       1
SELENOK.1       4.369142e-03  0.3910430 0.516 0.419 1.000000e+00       1
COL14A1.1       4.427097e-03  0.3263123 0.189 0.169 1.000000e+00       1
METTL27.1       4.480334e-03  0.3714316 0.147 0.109 1.000000e+00       1
KATNBL1.1       4.489947e-03 -0.3172919 0.274 0.369 1.000000e+00       1
CENPV.1         4.491320e-03 -0.4672563 0.221 0.355 1.000000e+00       1
SMARCE1.1       4.516124e-03  0.6284417 0.340 0.288 1.000000e+00       1
GPRC5B          4.529960e-03 -0.3571867 0.351 0.461 1.000000e+00       1
ZNF700          4.592681e-03  0.3097126 0.239 0.122 1.000000e+00       1
MBNL1-AS1.1     4.605325e-03 -0.2690287 0.179 0.202 1.000000e+00       1
ZNF468.1        4.609546e-03  0.4271433 0.281 0.227 1.000000e+00       1
BAG3.1          4.622228e-03  0.6890222 0.540 0.447 1.000000e+00       1
PLP2.1          4.622229e-03  0.4377283 0.519 0.418 1.000000e+00       1
ECI1.1          4.646269e-03  0.2831501 0.453 0.367 1.000000e+00       1
HSP90B1         4.709018e-03 -0.2968509 0.372 0.453 1.000000e+00       1
GLB1L.1         4.714734e-03  0.5314201 0.123 0.139 1.000000e+00       1
ZNF711.1        4.752190e-03 -0.9445259 0.098 0.205 1.000000e+00       1
MBP.1           4.795750e-03 -0.4757097 0.133 0.227 1.000000e+00       1
HMGB3.1         4.845645e-03 -0.3572397 0.239 0.322 1.000000e+00       1
ALDH1L2.1       4.867779e-03  0.2697264 0.414 0.357 1.000000e+00       1
ACAA1.1         4.931732e-03 -0.2916813 0.312 0.404 1.000000e+00       1
LINC01504.1     4.966238e-03  0.2518389 0.193 0.120 1.000000e+00       1
MED28.1         4.997957e-03  0.4813605 0.449 0.390 1.000000e+00       1
ARID3A.1        5.091100e-03  0.4342268 0.491 0.428 1.000000e+00       1
IGDCC4.1        5.095720e-03  0.3130707 0.460 0.386 1.000000e+00       1
SLC26A6.1       5.107987e-03  0.8943191 0.225 0.136 1.000000e+00       1
LINC00862.1     5.132797e-03 -0.9132307 0.112 0.249 1.000000e+00       1
ZBTB1.1         5.157696e-03  0.3315941 0.358 0.314 1.000000e+00       1
MFSD1.1         5.280514e-03 -0.3895048 0.340 0.452 1.000000e+00       1
PPL.1           5.318818e-03  0.3257290 0.200 0.181 1.000000e+00       1
SMURF2.1        5.347734e-03 -0.3080513 0.291 0.406 1.000000e+00       1
GAS6-DT.1       5.362229e-03  0.4289424 0.175 0.130 1.000000e+00       1
SLC2A13         5.466410e-03 -0.4876647 0.235 0.343 1.000000e+00       1
ALDH4A1         5.514167e-03 -0.5370630 0.084 0.184 1.000000e+00       1
LDLR.1          5.632839e-03  0.7801685 0.432 0.345 1.000000e+00       1
BACH1.1         5.656502e-03  0.6538028 0.347 0.286 1.000000e+00       1
LRRC49.1        5.675138e-03  0.4963469 0.151 0.127 1.000000e+00       1
HAUS4.1         5.688734e-03  0.6019360 0.165 0.137 1.000000e+00       1
POLR1B.1        5.717721e-03 -0.3917521 0.081 0.186 1.000000e+00       1
NT5C.1          5.836840e-03  0.6522141 0.439 0.362 1.000000e+00       1
BDH2.1          5.847301e-03 -0.3124927 0.249 0.427 1.000000e+00       1
LDLRAD4.1       5.990170e-03  0.3297868 0.361 0.313 1.000000e+00       1
COPG2           6.031307e-03  0.2588189 0.242 0.207 1.000000e+00       1
GXYLT2.1        6.070898e-03 -0.3748024 0.354 0.450 1.000000e+00       1
AC245041.2      6.101645e-03  0.3029919 0.288 0.233 1.000000e+00       1
DNAJA1          6.261225e-03  0.2670015 0.537 0.428 1.000000e+00       1
RAD21.1         6.434049e-03  0.5704904 0.463 0.393 1.000000e+00       1
PARP9.1         6.445347e-03  1.2085274 0.284 0.154 1.000000e+00       1
SLC1A1.1        6.502826e-03  0.6227904 0.389 0.343 1.000000e+00       1
C5orf30.1       6.529724e-03 -0.3128579 0.088 0.148 1.000000e+00       1
AGL.1           6.539406e-03  0.3936555 0.309 0.284 1.000000e+00       1
CCDC144NL-AS1.1 6.566467e-03 -0.6038774 0.137 0.243 1.000000e+00       1
CUBN.1          6.583882e-03 -1.0083605 0.021 0.157 1.000000e+00       1
COL4A5          6.620905e-03 -0.3987961 0.126 0.230 1.000000e+00       1
LUM.1           6.645315e-03  0.4660067 0.526 0.450 1.000000e+00       1
PECR.1          6.663949e-03 -0.2575676 0.088 0.151 1.000000e+00       1
FIP1L1.1        6.740895e-03  0.4855507 0.382 0.307 1.000000e+00       1
MORC3.1         6.764727e-03  0.6480642 0.316 0.260 1.000000e+00       1
GALNT1          6.774681e-03  0.3785038 0.488 0.406 1.000000e+00       1
RSRC2           6.780662e-03  0.2533644 0.533 0.462 1.000000e+00       1
RFXANK          6.784649e-03  0.3612927 0.393 0.324 1.000000e+00       1
NID2            6.844723e-03 -0.7441172 0.221 0.333 1.000000e+00       1
SPRY2.1         6.996985e-03 -0.8460004 0.225 0.324 1.000000e+00       1
AC092069.1.1    7.038125e-03 -0.5291551 0.151 0.246 1.000000e+00       1
TNKS.1          7.108525e-03  0.3813851 0.368 0.376 1.000000e+00       1
STK38L.1        7.230082e-03 -0.7122636 0.344 0.466 1.000000e+00       1
SNX16           7.253345e-03  0.3805734 0.235 0.217 1.000000e+00       1
MYO10.1         7.306466e-03  0.4684898 0.414 0.390 1.000000e+00       1
ST13            7.323539e-03  0.3236977 0.509 0.435 1.000000e+00       1
ADAMTS1.1       7.340589e-03  0.7800126 0.312 0.202 1.000000e+00       1
PARD6B          7.377111e-03 -0.3079191 0.137 0.232 1.000000e+00       1
BARD1.1         7.461325e-03  0.4951876 0.196 0.155 1.000000e+00       1
STK17B.1        7.544366e-03 -0.5984149 0.288 0.414 1.000000e+00       1
THBS2.1         7.610396e-03 -0.5961557 0.179 0.344 1.000000e+00       1
CEP135          7.721723e-03  0.2762311 0.235 0.210 1.000000e+00       1
SCRG1.1         7.873116e-03  0.6624866 0.253 0.220 1.000000e+00       1
MPC2.1          7.898236e-03  0.4794759 0.491 0.422 1.000000e+00       1
NOL3.1          7.955569e-03  0.4590618 0.396 0.319 1.000000e+00       1
ID1.1           7.962443e-03 -0.8060398 0.140 0.305 1.000000e+00       1
EID2B           8.034141e-03  0.2897718 0.277 0.217 1.000000e+00       1
SH3BP5.1        8.165069e-03 -0.3126470 0.263 0.357 1.000000e+00       1
SLCO4A1.1       8.174493e-03  0.6103587 0.256 0.219 1.000000e+00       1
ODF2L.1         8.221764e-03 -0.5790720 0.256 0.351 1.000000e+00       1
CYP2U1          8.252620e-03 -0.2961738 0.123 0.199 1.000000e+00       1
VASP.1          8.285965e-03 -0.8233120 0.253 0.379 1.000000e+00       1
KCNK12.1        8.398746e-03  1.2477138 0.347 0.216 1.000000e+00       1
AC064836.3.1    8.459362e-03 -0.2581355 0.077 0.106 1.000000e+00       1
WNT5A           8.500743e-03 -0.6652781 0.186 0.231 1.000000e+00       1
RERE.1          8.559560e-03 -0.5127040 0.295 0.407 1.000000e+00       1
KCNK1.1         8.628588e-03  0.4385062 0.326 0.292 1.000000e+00       1
LARP7.1         8.628588e-03  0.4400653 0.389 0.314 1.000000e+00       1
OTUD1.1         8.633538e-03  0.7293706 0.267 0.162 1.000000e+00       1
TGIF1.1         8.658325e-03  0.4762266 0.435 0.371 1.000000e+00       1
USP15           8.673227e-03 -0.3770181 0.351 0.459 1.000000e+00       1
TCEA1.1         8.828565e-03  0.6886887 0.463 0.389 1.000000e+00       1
HIST1H4E.1      8.978664e-03 -0.4537206 0.077 0.169 1.000000e+00       1
SMOX.1          9.006928e-03  0.4196414 0.312 0.248 1.000000e+00       1
TNC             9.063553e-03 -0.5204712 0.291 0.406 1.000000e+00       1
PISD.1          9.131087e-03  0.4512263 0.288 0.243 1.000000e+00       1
TUBB6.1         9.144113e-03 -0.4707892 0.361 0.427 1.000000e+00       1
MBNL2.1         9.157149e-03  0.6276235 0.516 0.462 1.000000e+00       1
NCAM2.1         9.214708e-03 -0.3905509 0.081 0.108 1.000000e+00       1
CHEK1.1         9.225191e-03  0.5139945 0.225 0.195 1.000000e+00       1
TRIM16.1        9.277870e-03  0.5131841 0.354 0.232 1.000000e+00       1
ITGB6.1         9.304206e-03 -0.5110200 0.074 0.155 1.000000e+00       1
PHF20           9.426759e-03  0.3374618 0.382 0.331 1.000000e+00       1
HIST3H2A.1      9.437474e-03  0.7093182 0.267 0.187 1.000000e+00       1
BRD1            9.453559e-03 -0.2823533 0.305 0.286 1.000000e+00       1
AADAT.1         9.485810e-03 -0.5354174 0.081 0.108 1.000000e+00       1
PTGR1           9.670486e-03  0.2875312 0.333 0.289 1.000000e+00       1
GUSB.1          9.827768e-03  0.4817523 0.305 0.225 1.000000e+00       1
MPRIP.1         1.001836e-02 -0.3080809 0.291 0.404 1.000000e+00       1
DYM.1           1.030462e-02  0.3683723 0.393 0.314 1.000000e+00       1
POLE4.1         1.046809e-02  0.5648168 0.467 0.427 1.000000e+00       1
ADGRV1.1        1.047101e-02  0.5665394 0.175 0.147 1.000000e+00       1
AZI2.1          1.060112e-02  0.4873628 0.502 0.413 1.000000e+00       1
KCNS3.1         1.061896e-02 -0.5332748 0.151 0.188 1.000000e+00       1
CAMTA2.1        1.062194e-02  0.5183860 0.291 0.271 1.000000e+00       1
PTPN11.1        1.071462e-02  0.2803626 0.439 0.402 1.000000e+00       1
CLIP1.1         1.075069e-02  0.3411605 0.411 0.346 1.000000e+00       1
AC004803.1.1    1.108632e-02  0.6089828 0.175 0.098 1.000000e+00       1
IFFO2           1.109564e-02 -0.4306630 0.179 0.229 1.000000e+00       1
SMIM15.1        1.127021e-02 -0.2658602 0.435 0.481 1.000000e+00       1
TMEM158.1       1.145348e-02 -0.3581956 0.225 0.197 1.000000e+00       1
NSMF.1          1.146631e-02  0.3610634 0.179 0.186 1.000000e+00       1
GPR161.1        1.152690e-02 -0.6509645 0.102 0.159 1.000000e+00       1
CISD1.1         1.159766e-02 -0.3661058 0.382 0.462 1.000000e+00       1
ZNRF3.1         1.160085e-02  0.5411574 0.207 0.097 1.000000e+00       1
FGD6.1          1.161698e-02 -0.2761353 0.186 0.270 1.000000e+00       1
LPAR6.1         1.162657e-02 -0.4116171 0.067 0.172 1.000000e+00       1
B4GALT1.1       1.173034e-02 -0.2644125 0.372 0.463 1.000000e+00       1
KRT19.1         1.175275e-02 -0.3461243 0.302 0.342 1.000000e+00       1
TRIM22.1        1.179228e-02 -0.6918192 0.232 0.351 1.000000e+00       1
KRT8.1          1.179554e-02 -0.4320423 0.361 0.425 1.000000e+00       1
BBOX1-AS1.1     1.199310e-02 -0.6095245 0.196 0.368 1.000000e+00       1
RASSF2.1        1.248265e-02 -1.1345528 0.046 0.233 1.000000e+00       1
SMC4.1          1.253089e-02  0.5885996 0.323 0.263 1.000000e+00       1
RASSF8-AS1      1.297211e-02 -0.2908663 0.347 0.430 1.000000e+00       1
GOLIM4          1.302554e-02 -0.3407002 0.347 0.465 1.000000e+00       1
TBC1D12.1       1.308633e-02  0.2775589 0.232 0.225 1.000000e+00       1
ZSCAN25.1       1.320851e-02 -0.9571512 0.074 0.186 1.000000e+00       1
JUND            1.325571e-02  0.3089953 0.519 0.441 1.000000e+00       1
WNT2.1          1.329192e-02 -0.2753487 0.323 0.442 1.000000e+00       1
AC020571.1.1    1.329921e-02 -0.6751922 0.067 0.174 1.000000e+00       1
PSPN.1          1.334563e-02 -1.1407902 0.088 0.278 1.000000e+00       1
LYSMD2.1        1.337565e-02 -0.8809587 0.109 0.197 1.000000e+00       1
CARMN.1         1.341968e-02 -0.4287658 0.193 0.359 1.000000e+00       1
ATP13A3         1.387331e-02 -0.3742030 0.330 0.437 1.000000e+00       1
ADNP-AS1        1.390352e-02 -0.4232837 0.063 0.136 1.000000e+00       1
HENMT1          1.391485e-02  0.2830818 0.095 0.093 1.000000e+00       1
ENDOD1.1        1.399451e-02  0.2567620 0.477 0.442 1.000000e+00       1
FAM133A.1       1.404781e-02 -0.4002016 0.102 0.208 1.000000e+00       1
DTX3L.1         1.412420e-02  0.4454106 0.175 0.203 1.000000e+00       1
C2CD3.1         1.430937e-02 -0.4185910 0.130 0.245 1.000000e+00       1
GRAMD2B.1       1.434818e-02 -0.2814790 0.221 0.232 1.000000e+00       1
CXCL2.1         1.439489e-02  1.0214988 0.186 0.090 1.000000e+00       1
TOM1.1          1.440659e-02  0.3378209 0.375 0.303 1.000000e+00       1
LINC00842       1.447305e-02  0.2543249 0.340 0.317 1.000000e+00       1
B3GNT5.1        1.451614e-02  0.4624567 0.239 0.176 1.000000e+00       1
DNASE2.1        1.452404e-02 -0.4412842 0.256 0.353 1.000000e+00       1
KIF23.1         1.453188e-02  0.6102033 0.242 0.160 1.000000e+00       1
MMP14           1.455943e-02 -0.6156266 0.225 0.340 1.000000e+00       1
ATP8B4          1.475734e-02 -0.4495146 0.112 0.176 1.000000e+00       1
SLC7A5          1.476544e-02 -0.4882114 0.168 0.303 1.000000e+00       1
RIOK2.1         1.481335e-02  0.2997580 0.372 0.337 1.000000e+00       1
CLTCL1          1.492149e-02  0.2802177 0.228 0.171 1.000000e+00       1
SLC22A15.1      1.494962e-02  0.4805318 0.214 0.178 1.000000e+00       1
CFAP20.1        1.499825e-02 -0.3367999 0.267 0.357 1.000000e+00       1
FGFR2           1.509959e-02 -0.4320604 0.270 0.406 1.000000e+00       1
FAM83G.1        1.531223e-02  0.3228986 0.133 0.170 1.000000e+00       1
UTP6.1          1.537417e-02  0.4427443 0.379 0.314 1.000000e+00       1
PEG10.1         1.559016e-02  0.3991944 0.218 0.118 1.000000e+00       1
IER5L.1         1.574990e-02  0.4782051 0.326 0.294 1.000000e+00       1
ZWINT.1         1.579638e-02  0.2927858 0.204 0.193 1.000000e+00       1
ZNF136.1        1.632430e-02  0.3150229 0.207 0.161 1.000000e+00       1
INMT.1          1.640720e-02  0.4801175 0.256 0.239 1.000000e+00       1
PCGF5.1         1.640722e-02  0.7624428 0.284 0.214 1.000000e+00       1
SORBS2          1.651676e-02 -0.5438970 0.179 0.309 1.000000e+00       1
MUM1L1          1.662276e-02 -0.2861938 0.189 0.210 1.000000e+00       1
TXNDC5.1        1.677379e-02 -0.6509209 0.098 0.270 1.000000e+00       1
CYCS.1          1.691254e-02 -0.3598021 0.418 0.455 1.000000e+00       1
CAP1.1          1.694851e-02 -0.4390974 0.407 0.469 1.000000e+00       1
OPTN.1          1.706587e-02 -0.4293407 0.375 0.466 1.000000e+00       1
MAFF            1.710666e-02 -0.2847520 0.151 0.121 1.000000e+00       1
ASTE1           1.714278e-02  0.4010993 0.189 0.125 1.000000e+00       1
MRPS30.1        1.715663e-02  0.3460074 0.340 0.272 1.000000e+00       1
WARS.1          1.717027e-02  0.7079930 0.404 0.319 1.000000e+00       1
DAAM1.1         1.738998e-02 -0.4637590 0.333 0.439 1.000000e+00       1
RNF227.1        1.741761e-02  0.4729934 0.225 0.180 1.000000e+00       1
MEX3A           1.744069e-02 -0.3301471 0.112 0.197 1.000000e+00       1
DDX24           1.752393e-02 -0.3928359 0.361 0.453 1.000000e+00       1
AL356056.1.1    1.754235e-02 -0.6740054 0.130 0.320 1.000000e+00       1
TMEM51          1.766791e-02 -0.4706106 0.098 0.171 1.000000e+00       1
FGF1.1          1.769143e-02 -1.0930292 0.109 0.310 1.000000e+00       1
UBXN2A.1        1.771015e-02  0.3386358 0.333 0.284 1.000000e+00       1
KLF4.1          1.775698e-02  0.8688693 0.193 0.112 1.000000e+00       1
CLTA            1.786038e-02  0.3157709 0.467 0.436 1.000000e+00       1
FXYD5.1         1.803546e-02 -0.4928538 0.382 0.467 1.000000e+00       1
PHLDA3.1        1.816896e-02  0.5092874 0.488 0.415 1.000000e+00       1
TUBA4A.1        1.822158e-02  0.4281772 0.214 0.215 1.000000e+00       1
ADAMTS9.1       1.842885e-02  0.3322532 0.281 0.289 1.000000e+00       1
BCAR3.1         1.855515e-02  0.3426441 0.253 0.220 1.000000e+00       1
GPC3.1          1.856463e-02 -0.9388557 0.151 0.326 1.000000e+00       1
TFPT.1          1.856490e-02 -0.3907067 0.267 0.373 1.000000e+00       1
RBP1.1          1.880942e-02  0.5200056 0.368 0.330 1.000000e+00       1
TRIM24.1        1.887420e-02 -0.3660875 0.214 0.298 1.000000e+00       1
CMIP.1          1.888410e-02  0.4611953 0.523 0.459 1.000000e+00       1
CEBPZOS         1.891380e-02  0.3777935 0.425 0.392 1.000000e+00       1
RIMS1.1         1.903300e-02 -0.3139982 0.154 0.176 1.000000e+00       1
GNAI1           1.930875e-02  0.2613006 0.411 0.359 1.000000e+00       1
ATP9A           1.947584e-02 -0.3503543 0.168 0.242 1.000000e+00       1
CKB.1           1.951134e-02 -0.4569617 0.340 0.439 1.000000e+00       1
XYLT1           1.954712e-02 -0.3692527 0.368 0.466 1.000000e+00       1
DNAJC15.1       1.964420e-02 -0.4298096 0.372 0.423 1.000000e+00       1
CTSO.1          1.965443e-02 -0.4926609 0.186 0.231 1.000000e+00       1
ULK4.1          1.994834e-02 -0.3241343 0.179 0.184 1.000000e+00       1
ASAP1.1         2.028825e-02  0.4749320 0.474 0.422 1.000000e+00       1
UHMK1.1         2.037268e-02  0.3288210 0.484 0.436 1.000000e+00       1
TSPO.1          2.087458e-02 -0.8494841 0.172 0.264 1.000000e+00       1
ECHDC2          2.088570e-02  0.2566400 0.123 0.119 1.000000e+00       1
GSDME           2.091816e-02 -0.2942001 0.189 0.223 1.000000e+00       1
BLM.1           2.102657e-02  0.4104192 0.165 0.164 1.000000e+00       1
UBE2E2.1        2.112461e-02 -0.4100686 0.358 0.436 1.000000e+00       1
CPPED1.1        2.118469e-02 -0.4027540 0.256 0.322 1.000000e+00       1
CITED2.1        2.139894e-02  0.4335533 0.481 0.395 1.000000e+00       1
PUS7L.1         2.147079e-02 -0.3145038 0.319 0.428 1.000000e+00       1
HUNK.1          2.153722e-02 -0.4512579 0.070 0.194 1.000000e+00       1
KLF5.1          2.163736e-02 -0.3930522 0.193 0.293 1.000000e+00       1
MMS22L          2.164851e-02 -0.3017776 0.144 0.217 1.000000e+00       1
AC124798.1.1    2.186684e-02  0.6855621 0.225 0.192 1.000000e+00       1
TRAPPC6A        2.217799e-02  0.3829072 0.382 0.353 1.000000e+00       1
AC018647.1.1    2.218923e-02 -0.6416170 0.067 0.195 1.000000e+00       1
APBA2.1         2.224629e-02 -0.4985123 0.084 0.148 1.000000e+00       1
BTG3.1          2.236653e-02  0.3824603 0.432 0.377 1.000000e+00       1
MAFG.1          2.242969e-02  0.6925564 0.389 0.320 1.000000e+00       1
C1QTNF1         2.245269e-02  0.2618252 0.249 0.194 1.000000e+00       1
LYPD1.1         2.246983e-02 -0.9143541 0.028 0.173 1.000000e+00       1
C3AR1           2.255644e-02  0.3431938 0.161 0.130 1.000000e+00       1
ST20            2.305803e-02  0.2632315 0.351 0.301 1.000000e+00       1
ZNF608          2.314055e-02  0.4317518 0.214 0.178 1.000000e+00       1
ICAM3.1         2.322933e-02 -0.3787520 0.337 0.399 1.000000e+00       1
PRKCI           2.330053e-02 -0.2776496 0.326 0.400 1.000000e+00       1
AFAP1.1         2.351528e-02 -0.6259141 0.158 0.247 1.000000e+00       1
PELI3.1         2.354498e-02  0.7003190 0.119 0.123 1.000000e+00       1
KCNE4           2.406579e-02  0.2861624 0.133 0.162 1.000000e+00       1
ADAMTS12.1      2.424359e-02 -0.5213279 0.133 0.278 1.000000e+00       1
SYPL1.1         2.425590e-02  0.4041616 0.519 0.416 1.000000e+00       1
SLC39A10        2.455900e-02 -0.3287088 0.295 0.394 1.000000e+00       1
GPR68.1         2.477748e-02 -0.2716474 0.081 0.148 1.000000e+00       1
AGPAT2.1        2.487795e-02  0.3467576 0.368 0.310 1.000000e+00       1
NT5DC1.1        2.493461e-02  0.5100372 0.281 0.257 1.000000e+00       1
TNFAIP6.1       2.504193e-02  0.7123632 0.172 0.120 1.000000e+00       1
ZER1            2.524504e-02 -0.3437340 0.214 0.246 1.000000e+00       1
IL6ST           2.534709e-02 -0.5032658 0.361 0.441 1.000000e+00       1
AKAP13          2.626890e-02  0.2566186 0.523 0.443 1.000000e+00       1
SSC4D.1         2.643409e-02  0.4598744 0.130 0.089 1.000000e+00       1
MORC4.1         2.658697e-02  0.4465320 0.481 0.431 1.000000e+00       1
SMIM1           2.665364e-02  0.3932687 0.221 0.181 1.000000e+00       1
ZNF678.1        2.672026e-02 -0.6026552 0.130 0.184 1.000000e+00       1
TRAF3IP1.1      2.708383e-02  0.4114936 0.281 0.241 1.000000e+00       1
AL731563.3      2.719231e-02 -0.5045594 0.060 0.173 1.000000e+00       1
PKDCC.1         2.807372e-02  0.4807837 0.288 0.282 1.000000e+00       1
SRRM3.1         2.808072e-02  0.9080198 0.284 0.182 1.000000e+00       1
SLC5A12.1       2.827681e-02  0.4141335 0.133 0.134 1.000000e+00       1
TMEM131L        2.831892e-02 -0.4391894 0.151 0.131 1.000000e+00       1
NORAD.1         2.859446e-02  0.2684925 0.516 0.476 1.000000e+00       1
ZFP36.1         2.872237e-02  0.3845755 0.323 0.221 1.000000e+00       1
HIST1H2BD.1     2.880056e-02 -0.3224082 0.088 0.168 1.000000e+00       1
LANCL2          2.890800e-02 -0.3950931 0.309 0.318 1.000000e+00       1
AL035413.1.1    2.895725e-02  0.5098289 0.407 0.340 1.000000e+00       1
CTSK.1          2.926786e-02 -0.6330049 0.095 0.114 1.000000e+00       1
IRF1.1          2.936929e-02  0.5625077 0.228 0.171 1.000000e+00       1
ST6GALNAC5      2.936934e-02 -0.3136511 0.151 0.266 1.000000e+00       1
FOS.1           2.966089e-02  0.5166735 0.340 0.232 1.000000e+00       1
JPH2            2.978532e-02 -0.3159764 0.147 0.206 1.000000e+00       1
BCL7A.1         2.986642e-02 -0.4919649 0.147 0.234 1.000000e+00       1
MANCR.1         2.999911e-02 -0.4348836 0.211 0.242 1.000000e+00       1
AC009126.1      3.008063e-02  0.3837649 0.196 0.142 1.000000e+00       1
GPN3            3.030360e-02  0.2714520 0.382 0.322 1.000000e+00       1
PBLD.1          3.033338e-02 -0.5390846 0.186 0.185 1.000000e+00       1
FLVCR1.1        3.055802e-02  0.5771814 0.249 0.196 1.000000e+00       1
CPD.1           3.069345e-02  0.5505990 0.502 0.443 1.000000e+00       1
NCSTN           3.074625e-02 -0.3321963 0.382 0.454 1.000000e+00       1
SMYD4           3.077646e-02  0.3624047 0.263 0.219 1.000000e+00       1
CMTM7.1         3.087481e-02  0.5762479 0.284 0.219 1.000000e+00       1
JUN             3.120971e-02  0.3444661 0.484 0.418 1.000000e+00       1
ADAM23.1        3.144755e-02  0.3228724 0.204 0.195 1.000000e+00       1
ADAMTS2.1       3.171788e-02  0.9108221 0.263 0.179 1.000000e+00       1
LHFPL2.1        3.174882e-02  0.6036782 0.221 0.151 1.000000e+00       1
SLC9A8          3.180311e-02 -0.2578035 0.098 0.190 1.000000e+00       1
FASN            3.182663e-02 -0.4181365 0.302 0.358 1.000000e+00       1
CHST7           3.184219e-02  0.2657707 0.267 0.256 1.000000e+00       1
HADH.1          3.187315e-02  0.4249940 0.249 0.175 1.000000e+00       1
HMGB2.1         3.190445e-02  0.6353562 0.298 0.228 1.000000e+00       1
LETM2.1         3.206847e-02 -0.4970621 0.049 0.100 1.000000e+00       1
GREM1.1         3.207617e-02 -0.6008195 0.105 0.193 1.000000e+00       1
HAS2.1          3.234340e-02  0.3876862 0.295 0.242 1.000000e+00       1
SYTL2.1         3.241440e-02 -0.8327339 0.154 0.328 1.000000e+00       1
WNK1            3.285940e-02 -0.2608620 0.396 0.489 1.000000e+00       1
VWA8.1          3.298719e-02  0.3850912 0.179 0.155 1.000000e+00       1
COL11A1.1       3.321253e-02 -0.2675633 0.379 0.465 1.000000e+00       1
CTH.1           3.325301e-02  0.3039232 0.235 0.224 1.000000e+00       1
TRIL            3.375661e-02 -0.7122100 0.042 0.154 1.000000e+00       1
TTLL11.1        3.376493e-02  0.5360413 0.130 0.111 1.000000e+00       1
CCND3           3.417613e-02  0.3177653 0.344 0.283 1.000000e+00       1
SNAI1.1         3.499421e-02  0.6513265 0.186 0.162 1.000000e+00       1
ALOX5AP.1       3.501122e-02  0.8142747 0.274 0.158 1.000000e+00       1
DMRTA1.1        3.513798e-02 -0.4566780 0.147 0.256 1.000000e+00       1
TALDO1.1        3.529910e-02  0.4577134 0.474 0.417 1.000000e+00       1
KIAA0825.1      3.543501e-02 -0.4377148 0.112 0.154 1.000000e+00       1
ING2.1          3.550352e-02  0.3611450 0.351 0.315 1.000000e+00       1
PHYHD1.1        3.574327e-02  0.4679184 0.323 0.251 1.000000e+00       1
MKNK2           3.577756e-02  0.3417703 0.214 0.186 1.000000e+00       1
UNC5B-AS1       3.587208e-02  0.2968899 0.179 0.125 1.000000e+00       1
TM7SF3.1        3.634004e-02 -0.2835918 0.302 0.404 1.000000e+00       1
LINC02001.1     3.638361e-02  0.4709551 0.242 0.230 1.000000e+00       1
DIXDC1.1        3.654083e-02  0.4589163 0.505 0.446 1.000000e+00       1
PLEKHO1.1       3.668106e-02 -0.4088178 0.333 0.404 1.000000e+00       1
TMEM8B.1        3.686578e-02 -0.4447730 0.095 0.171 1.000000e+00       1
DDX10.1         3.699829e-02  0.6212493 0.312 0.253 1.000000e+00       1
CEP126.1        3.711340e-02  0.5738264 0.246 0.195 1.000000e+00       1
SHROOM3.1       3.714001e-02 -0.6122213 0.119 0.257 1.000000e+00       1
SGTA.1          3.741593e-02  0.3550198 0.305 0.290 1.000000e+00       1
RAB9A.1         3.758593e-02  0.4707138 0.361 0.320 1.000000e+00       1
CADPS2.1        3.833617e-02  0.4468813 0.344 0.261 1.000000e+00       1
ZNF496.1        3.845471e-02  0.4469358 0.196 0.188 1.000000e+00       1
PTPA            3.851883e-02 -0.3794695 0.242 0.329 1.000000e+00       1
MYADM.1         3.872065e-02 -0.4861751 0.340 0.441 1.000000e+00       1
RHOB.1          3.904354e-02  0.2976383 0.484 0.418 1.000000e+00       1
ZCCHC10.1       3.930350e-02  0.5646906 0.375 0.305 1.000000e+00       1
CBR1.1          3.938735e-02 -0.3317031 0.253 0.314 1.000000e+00       1
SLC29A4.1       3.940602e-02  0.4062998 0.123 0.117 1.000000e+00       1
ZWILCH          3.949939e-02  0.2650398 0.246 0.229 1.000000e+00       1
KIT.1           3.994095e-02  0.4810099 0.179 0.239 1.000000e+00       1
SLC10A3.1       4.003543e-02 -0.2713794 0.319 0.404 1.000000e+00       1
SERPINE1.1      4.006281e-02  0.5004340 0.439 0.388 1.000000e+00       1
ASMTL.1         4.025344e-02 -0.4112919 0.168 0.253 1.000000e+00       1
AC044849.1.1    4.062539e-02  0.2988185 0.116 0.125 1.000000e+00       1
NCOA1.1         4.069250e-02  0.4241558 0.309 0.247 1.000000e+00       1
TENT5C          4.092316e-02 -0.4116082 0.151 0.204 1.000000e+00       1
TGM2.1          4.097990e-02  0.8603105 0.344 0.347 1.000000e+00       1
CDKL5.1         4.119349e-02  0.6222077 0.249 0.180 1.000000e+00       1
WNT4.1          4.156227e-02 -1.1640200 0.081 0.271 1.000000e+00       1
NID1.1          4.169037e-02  0.3577462 0.351 0.304 1.000000e+00       1
LINC00626.1     4.188648e-02 -0.5046159 0.098 0.218 1.000000e+00       1
IL13RA1         4.247962e-02 -0.5020434 0.253 0.339 1.000000e+00       1
TSTD1.1         4.292886e-02 -0.5757569 0.119 0.199 1.000000e+00       1
TPCN2.1         4.342276e-02  0.5611016 0.196 0.175 1.000000e+00       1
SPTLC3.1        4.344335e-02  0.4065239 0.295 0.216 1.000000e+00       1
TLL2.1          4.421936e-02 -0.5626951 0.109 0.179 1.000000e+00       1
KIAA0040.1      4.438420e-02 -0.7503071 0.088 0.168 1.000000e+00       1
HSPA9.1         4.463250e-02  0.5494004 0.516 0.450 1.000000e+00       1
SPTBN1.1        4.476746e-02  0.4368098 0.498 0.442 1.000000e+00       1
LINC01750.1     4.578535e-02 -0.8376752 0.074 0.193 1.000000e+00       1
GSTM3.1         4.642427e-02 -0.2886039 0.382 0.399 1.000000e+00       1
MIR4435-2HG.1   4.645686e-02 -0.3794631 0.411 0.447 1.000000e+00       1
HMGB1           4.676874e-02  0.4002559 0.477 0.427 1.000000e+00       1
SOCS3.1         4.687668e-02  0.5811137 0.189 0.170 1.000000e+00       1
APH1B.1         4.759442e-02  0.3027893 0.309 0.280 1.000000e+00       1
MRPL52.1        4.761635e-02  0.4657967 0.516 0.448 1.000000e+00       1
TTC39B.1        4.779180e-02  0.2607795 0.235 0.195 1.000000e+00       1
NCAPD3.1        4.793473e-02 -0.5094199 0.186 0.260 1.000000e+00       1
CREBRF.1        4.808910e-02  0.5663511 0.393 0.333 1.000000e+00       1
HMCN1.1         4.845458e-02 -0.8849628 0.095 0.298 1.000000e+00       1
GLA.1           4.859922e-02  0.4269088 0.361 0.324 1.000000e+00       1
GSTO1.1         4.886714e-02 -0.7381674 0.400 0.455 1.000000e+00       1
PGD.1           4.887835e-02  0.5800020 0.375 0.312 1.000000e+00       1
TTYH3           4.895675e-02 -0.4411149 0.288 0.397 1.000000e+00       1
FASTKD3.1       4.929362e-02 -0.2663742 0.116 0.172 1.000000e+00       1
LCMT2           4.937267e-02 -0.3694416 0.095 0.193 1.000000e+00       1
ATP6AP1L.1      4.941801e-02 -0.8833409 0.102 0.217 1.000000e+00       1
SH3BP4.1        4.955377e-02 -0.4020506 0.182 0.264 1.000000e+00       1
LMBRD2          4.986023e-02 -0.3205701 0.218 0.319 1.000000e+00       1
MT-ATP6.1       5.017975e-02  0.4011068 0.435 0.414 1.000000e+00       1
ZNF431          5.052399e-02 -0.5163245 0.225 0.278 1.000000e+00       1
TCEAL3          5.136990e-02  0.3668916 0.488 0.408 1.000000e+00       1
SEMA4F.1        5.187365e-02  0.3638385 0.144 0.220 1.000000e+00       1
MT-CYB.1        5.202683e-02  0.4441691 0.474 0.407 1.000000e+00       1
LGR4            5.205042e-02 -0.4105785 0.256 0.392 1.000000e+00       1
PSME2           5.213306e-02  0.2875510 0.407 0.346 1.000000e+00       1
DEPP1.1         5.309730e-02  0.2880149 0.214 0.156 1.000000e+00       1
TXNIP.1         5.348230e-02  0.3571452 0.495 0.397 1.000000e+00       1
CLDN12.1        5.363939e-02  0.3769348 0.246 0.229 1.000000e+00       1
NET1.1          5.372415e-02 -0.4414526 0.214 0.315 1.000000e+00       1
LDHA            5.440613e-02  0.3601381 0.491 0.405 1.000000e+00       1
HMGN2.1         5.444286e-02  0.6825117 0.407 0.380 1.000000e+00       1
SRRD.1          5.462685e-02 -0.3164470 0.270 0.376 1.000000e+00       1
TNFRSF21.1      5.487296e-02  0.4579339 0.270 0.260 1.000000e+00       1
IP6K3.1         5.549213e-02  0.9738307 0.263 0.186 1.000000e+00       1
BRIX1.1         5.554216e-02  0.4191466 0.361 0.333 1.000000e+00       1
KCNQ5.1         5.561678e-02 -0.3224069 0.067 0.210 1.000000e+00       1
WNT11.1         5.574174e-02 -0.7031446 0.123 0.188 1.000000e+00       1
SH3PXD2B        5.596696e-02  0.3680812 0.309 0.294 1.000000e+00       1
FGF2.1          5.606728e-02  0.4687092 0.337 0.285 1.000000e+00       1
MMADHC.1        5.623071e-02 -0.3965451 0.453 0.456 1.000000e+00       1
TMEM106A.1      5.629362e-02  0.3553041 0.218 0.267 1.000000e+00       1
SLC16A10        5.678653e-02 -0.2634303 0.123 0.216 1.000000e+00       1
SOX6            5.713002e-02 -0.6164662 0.218 0.303 1.000000e+00       1
FBLN2.1         5.747512e-02  0.4881592 0.439 0.405 1.000000e+00       1
PEA15           5.757778e-02 -0.2551445 0.386 0.455 1.000000e+00       1
PDXK            5.773196e-02  0.3176778 0.442 0.467 1.000000e+00       1
TANGO2.1        5.786070e-02  0.3939694 0.218 0.204 1.000000e+00       1
SERP1           5.837806e-02  0.3351476 0.505 0.437 1.000000e+00       1
RPS4X           5.862514e-02  0.2521110 0.505 0.431 1.000000e+00       1
TWIST1          5.892540e-02  0.2522717 0.442 0.411 1.000000e+00       1
SOD2.1          5.939794e-02  0.7637354 0.319 0.279 1.000000e+00       1
TRIO.1          5.987363e-02 -0.5020180 0.274 0.349 1.000000e+00       1
DNAJC28.1       5.992667e-02 -0.2929648 0.088 0.173 1.000000e+00       1
NAV3.1          6.021889e-02 -0.8802309 0.102 0.211 1.000000e+00       1
MYOF            6.059308e-02 -0.2545667 0.319 0.425 1.000000e+00       1
MON2.1          6.142793e-02  0.2887582 0.463 0.438 1.000000e+00       1
ZNF394.1        6.188977e-02  0.3240139 0.256 0.259 1.000000e+00       1
C1orf198        6.197158e-02 -0.4636599 0.267 0.376 1.000000e+00       1
H1F0.1          6.219014e-02  0.3109932 0.277 0.228 1.000000e+00       1
ABCA5.1         6.245040e-02  0.4725497 0.211 0.174 1.000000e+00       1
ZBED3-AS1       6.316774e-02  0.2600037 0.109 0.133 1.000000e+00       1
COL4A4.1        6.337562e-02 -0.4090810 0.077 0.169 1.000000e+00       1
TRIM56          6.368224e-02  0.2532092 0.365 0.292 1.000000e+00       1
GTF2F1.1        6.372411e-02  0.4008008 0.382 0.355 1.000000e+00       1
DLG1            6.393378e-02 -0.4680649 0.309 0.399 1.000000e+00       1
SLC2A1-AS1.1    6.401777e-02 -0.5320115 0.070 0.121 1.000000e+00       1
CYB5A           6.500487e-02  0.3532094 0.382 0.363 1.000000e+00       1
AMMECR1L.1      6.507586e-02 -0.4361730 0.182 0.252 1.000000e+00       1
CDC42SE1.1      6.586091e-02 -0.3896776 0.386 0.452 1.000000e+00       1
ZNF561.1        6.591831e-02 -0.4069561 0.225 0.317 1.000000e+00       1
GDNF            6.640782e-02 -0.5755038 0.179 0.266 1.000000e+00       1
GCNA            6.649444e-02  0.3191882 0.123 0.092 1.000000e+00       1
C1orf226.1      6.746897e-02  0.4326285 0.133 0.107 1.000000e+00       1
MRPS18B.1       6.757153e-02  0.4514905 0.312 0.296 1.000000e+00       1
TTC27.1         6.765916e-02 -0.5681305 0.154 0.194 1.000000e+00       1
PITX1.1         6.786499e-02 -0.3155093 0.368 0.431 1.000000e+00       1
SVIP.1          6.838125e-02 -0.3674360 0.168 0.244 1.000000e+00       1
PCDH19          6.844044e-02 -0.3517714 0.074 0.130 1.000000e+00       1
WEE1.1          6.882627e-02  0.6374629 0.246 0.208 1.000000e+00       1
TET1.1          6.921370e-02  0.5664548 0.196 0.130 1.000000e+00       1
DGKI.1          6.924375e-02 -0.8745465 0.077 0.232 1.000000e+00       1
TRAF3IP2-AS1.1  6.993377e-02  0.6809144 0.196 0.155 1.000000e+00       1
SPRYD3.1        6.999443e-02 -0.3616097 0.274 0.255 1.000000e+00       1
CMTM3.1         7.043293e-02 -0.2501910 0.316 0.350 1.000000e+00       1
SCML1           7.063020e-02  0.3388381 0.319 0.245 1.000000e+00       1
YWHAH.1         7.171567e-02 -0.3203081 0.379 0.453 1.000000e+00       1
RNF150.1        7.309558e-02 -0.4676413 0.028 0.132 1.000000e+00       1
CNTN5           7.489478e-02 -0.3777557 0.081 0.148 1.000000e+00       1
HDDC2           7.489479e-02  0.3096128 0.481 0.415 1.000000e+00       1
ZADH2.1         7.595285e-02 -0.3495268 0.379 0.461 1.000000e+00       1
KANK4.1         7.627585e-02  0.2572468 0.221 0.159 1.000000e+00       1
USP13.1         7.782498e-02  0.4497515 0.196 0.168 1.000000e+00       1
WDR25           7.946769e-02  0.2582509 0.246 0.225 1.000000e+00       1
AC128688.2.1    7.953458e-02 -0.6694785 0.056 0.119 1.000000e+00       1
TMEM154.1       8.115480e-02 -0.6118239 0.116 0.157 1.000000e+00       1
EGR1.1          8.152925e-02 -0.4391840 0.232 0.304 1.000000e+00       1
TRIM59          8.297452e-02 -0.2569665 0.123 0.188 1.000000e+00       1
CARD16.1        8.335638e-02  0.3560147 0.172 0.144 1.000000e+00       1
PTBP3           8.393228e-02 -0.3832660 0.379 0.452 1.000000e+00       1
ANXA3.1         8.480987e-02 -0.4113325 0.154 0.185 1.000000e+00       1
FABP3.1         8.484557e-02  0.3974457 0.175 0.125 1.000000e+00       1
CPQ.1           8.484557e-02  0.3337950 0.295 0.278 1.000000e+00       1
C1orf56.1       8.564242e-02 -0.4296609 0.319 0.390 1.000000e+00       1
ST5.1           8.592720e-02 -0.3772091 0.361 0.438 1.000000e+00       1
LINC01836       8.617699e-02 -0.2517413 0.063 0.113 1.000000e+00       1
WISP1           8.676806e-02 -0.4383170 0.228 0.259 1.000000e+00       1
SRPX.1          8.707323e-02 -0.6137925 0.295 0.382 1.000000e+00       1
ZNF121.1        8.727229e-02 -0.4327109 0.165 0.247 1.000000e+00       1
CENPN           8.743482e-02  0.2666549 0.228 0.212 1.000000e+00       1
PLAA.1          8.810565e-02  0.3712488 0.225 0.231 1.000000e+00       1
STOM            8.852461e-02  0.2922617 0.228 0.236 1.000000e+00       1
FABP5.1         8.865253e-02 -0.4093828 0.242 0.308 1.000000e+00       1
THBS3.1         8.951497e-02  0.4957679 0.246 0.196 1.000000e+00       1
BICC1.1         8.962517e-02 -0.5708457 0.112 0.155 1.000000e+00       1
RBM43           8.995784e-02  0.3563105 0.179 0.186 1.000000e+00       1
SCUBE2.1        9.034689e-02  0.5169127 0.323 0.264 1.000000e+00       1
PCDHA4          9.047692e-02  0.3279805 0.105 0.092 1.000000e+00       1
MYL4.1          9.137210e-02 -0.4534963 0.077 0.257 1.000000e+00       1
DPP10.1         9.140973e-02 -1.0285466 0.063 0.233 1.000000e+00       1
AIFM2.1         9.150340e-02  0.5654918 0.298 0.239 1.000000e+00       1
CCDC18.1        9.297515e-02 -0.3430605 0.081 0.107 1.000000e+00       1
SUPV3L1.1       9.320338e-02  0.3195570 0.284 0.248 1.000000e+00       1
H2AFY.1         9.442757e-02  0.6790757 0.456 0.443 1.000000e+00       1
TRIM5           9.467779e-02  0.3433806 0.267 0.255 1.000000e+00       1
GATM.1          9.492843e-02 -0.6992367 0.105 0.272 1.000000e+00       1
AC132872.1.1    9.550900e-02  0.3378012 0.133 0.150 1.000000e+00       1
CDC6.1          9.552857e-02 -0.2991960 0.119 0.183 1.000000e+00       1
CLEC1B.1        9.556728e-02 -0.6173743 0.070 0.150 1.000000e+00       1
SPECC1          9.568403e-02  0.2780218 0.470 0.438 1.000000e+00       1
CNN1.1          9.607328e-02 -0.5873862 0.126 0.224 1.000000e+00       1
AKIRIN2         9.709139e-02  0.2611000 0.386 0.381 1.000000e+00       1
ARIH2.1         9.750497e-02  0.2981157 0.382 0.347 1.000000e+00       1
ANKRD6.1        9.817756e-02  0.4882772 0.214 0.260 1.000000e+00       1
TUSC3.1         9.825694e-02 -0.4345616 0.411 0.453 1.000000e+00       1
TCEAL9.1        9.829665e-02 -0.3397386 0.365 0.425 1.000000e+00       1
SCUBE3.1        9.933348e-02 -0.5563182 0.214 0.355 1.000000e+00       1
NKX3-1.1        1.000160e-01 -0.3711941 0.158 0.212 1.000000e+00       1
TFPI2.1         1.002979e-01 -0.6239642 0.091 0.249 1.000000e+00       1
HIST1H2BG.1     1.003990e-01 -0.4751197 0.109 0.158 1.000000e+00       1
SEPT10.1        1.004800e-01  0.3910152 0.351 0.346 1.000000e+00       1
PLXNA2          1.007632e-01 -0.5024249 0.193 0.261 1.000000e+00       1
RAB3B.1         1.010064e-01 -1.0057830 0.168 0.329 1.000000e+00       1
IER3.1          1.010471e-01 -0.4409686 0.361 0.437 1.000000e+00       1
PIDD1.1         1.014535e-01 -0.2864420 0.109 0.142 1.000000e+00       1
RAB33A          1.019433e-01 -0.2553341 0.077 0.144 1.000000e+00       1
NDN.1           1.020864e-01  0.4784462 0.414 0.379 1.000000e+00       1
CDC37L1-DT.1    1.032583e-01 -0.3853191 0.067 0.150 1.000000e+00       1
ENAH.1          1.036097e-01 -0.3955706 0.407 0.431 1.000000e+00       1
PYGO1           1.047954e-01  0.3562326 0.309 0.277 1.000000e+00       1
THAP1           1.048999e-01 -0.2938708 0.228 0.288 1.000000e+00       1
DENND1B         1.056548e-01 -0.4510717 0.168 0.282 1.000000e+00       1
OSER1-DT.1      1.058017e-01  0.3555723 0.235 0.202 1.000000e+00       1
HOXB8.1         1.060339e-01 -0.2970266 0.207 0.289 1.000000e+00       1
LIF.1           1.062661e-01 -0.5291317 0.112 0.240 1.000000e+00       1
CIRBP.1         1.066682e-01  0.3640972 0.498 0.434 1.000000e+00       1
PYGL            1.079884e-01 -0.3558369 0.302 0.364 1.000000e+00       1
AC008467.1.1    1.080522e-01 -0.5771941 0.074 0.161 1.000000e+00       1
GK5.1           1.080525e-01  0.3343929 0.312 0.286 1.000000e+00       1
BMPR2.1         1.086963e-01 -0.5117243 0.414 0.491 1.000000e+00       1
HGD.1           1.088254e-01 -0.6358233 0.098 0.206 1.000000e+00       1
AC103706.1.1    1.094726e-01 -0.4631592 0.095 0.167 1.000000e+00       1
PIM3.1          1.102972e-01  0.4914524 0.305 0.217 1.000000e+00       1
MEF2A.1         1.104931e-01  0.4513532 0.368 0.345 1.000000e+00       1
BHLHE40         1.123133e-01 -0.2993838 0.263 0.355 1.000000e+00       1
CHERP           1.124016e-01 -0.2551620 0.186 0.260 1.000000e+00       1
RAD51B.1        1.127549e-01 -0.3690045 0.165 0.223 1.000000e+00       1
KLF2.1          1.132878e-01 -0.4064195 0.196 0.228 1.000000e+00       1
NPEPL1          1.133321e-01 -0.5305237 0.161 0.180 1.000000e+00       1
RAB15           1.133322e-01 -0.3969930 0.095 0.239 1.000000e+00       1
ST6GAL1.1       1.136437e-01 -0.6108363 0.077 0.185 1.000000e+00       1
GRIA3.1         1.140452e-01 -0.2525220 0.109 0.170 1.000000e+00       1
OLFM2.1         1.155947e-01 -0.6029539 0.109 0.202 1.000000e+00       1
CEP170.1        1.160246e-01 -0.4463618 0.291 0.369 1.000000e+00       1
LIPG.1          1.165917e-01 -0.6411804 0.084 0.159 1.000000e+00       1
NOP14.1         1.168192e-01 -0.3402001 0.246 0.322 1.000000e+00       1
RACGAP1.1       1.173204e-01  0.5664399 0.186 0.135 1.000000e+00       1
ADRA2A          1.175949e-01 -0.7457808 0.105 0.202 1.000000e+00       1
DCTN4.1         1.183064e-01  0.3155866 0.372 0.344 1.000000e+00       1
CCDC125         1.183983e-01  0.2521148 0.119 0.141 1.000000e+00       1
PLA2G4C.1       1.183984e-01  0.4407965 0.151 0.097 1.000000e+00       1
SORBS3.1        1.199245e-01  0.7276065 0.386 0.324 1.000000e+00       1
STK26.1         1.205765e-01  0.3493841 0.298 0.281 1.000000e+00       1
FRMD6-AS1.1     1.216069e-01 -0.4984492 0.140 0.253 1.000000e+00       1
AC005476.2.1    1.217476e-01 -0.3054418 0.102 0.135 1.000000e+00       1
FAM69A.1        1.225729e-01  0.3191332 0.119 0.142 1.000000e+00       1
TMEM176B.1      1.233788e-01 -0.5180383 0.074 0.121 1.000000e+00       1
ME1.1           1.236881e-01  0.3004992 0.411 0.342 1.000000e+00       1
CENPF.1         1.242123e-01  0.3460799 0.137 0.143 1.000000e+00       1
LINC01013.1     1.243079e-01 -1.2335456 0.021 0.229 1.000000e+00       1
SAMD15          1.244756e-01 -0.4057861 0.119 0.111 1.000000e+00       1
MAP3K20.1       1.248349e-01 -0.3144244 0.312 0.368 1.000000e+00       1
SEMA7A          1.251229e-01 -0.4425861 0.193 0.266 1.000000e+00       1
PKIB            1.255796e-01  0.2506619 0.207 0.187 1.000000e+00       1
CCL20.1         1.269589e-01 -0.5049565 0.056 0.143 1.000000e+00       1
CCDC138.1       1.280804e-01 -0.5249317 0.074 0.145 1.000000e+00       1
DBI             1.284234e-01  0.2930680 0.467 0.408 1.000000e+00       1
COL3A1.1        1.288652e-01  0.3337635 0.509 0.424 1.000000e+00       1
PPP6R1.1        1.295796e-01  0.5043732 0.330 0.306 1.000000e+00       1
ANKZF1          1.309926e-01  0.2777710 0.221 0.207 1.000000e+00       1
ZNF350.1        1.313410e-01 -0.3412919 0.137 0.160 1.000000e+00       1
TTC30B.1        1.315911e-01 -0.5095307 0.042 0.114 1.000000e+00       1
LEFTY2.1        1.316160e-01 -0.4528782 0.144 0.257 1.000000e+00       1
WDYHV1          1.317404e-01  0.2916660 0.179 0.163 1.000000e+00       1
ZNF223.1        1.320413e-01 -0.3489224 0.095 0.145 1.000000e+00       1
SIPA1L2         1.331468e-01 -0.5289452 0.165 0.184 1.000000e+00       1
LRIF1.1         1.331723e-01  0.3566520 0.358 0.333 1.000000e+00       1
KIF13B          1.331972e-01  0.2583231 0.221 0.154 1.000000e+00       1
AC010976.2.1    1.333231e-01 -0.3008868 0.102 0.179 1.000000e+00       1
CMTR2.1         1.333489e-01  0.4630808 0.239 0.224 1.000000e+00       1
PRRC2A.1        1.345646e-01 -0.3614776 0.351 0.370 1.000000e+00       1
MYLK.1          1.357557e-01 -0.5840775 0.260 0.340 1.000000e+00       1
GALNT5.1        1.369188e-01 -0.9075412 0.039 0.191 1.000000e+00       1
ITGA10.1        1.391481e-01  0.4325473 0.204 0.135 1.000000e+00       1
KANK2.1         1.393831e-01  0.3862778 0.344 0.300 1.000000e+00       1
THBD.1          1.396444e-01 -0.6257241 0.154 0.246 1.000000e+00       1
KLHL4.1         1.398014e-01 -0.2994433 0.249 0.318 1.000000e+00       1
SLC2A8.1        1.416960e-01 -0.2730434 0.154 0.219 1.000000e+00       1
NCLN            1.432098e-01 -0.4523375 0.319 0.357 1.000000e+00       1
SUSD6.1         1.433165e-01 -0.5376589 0.175 0.293 1.000000e+00       1
CHORDC1.1       1.434233e-01 -0.3156838 0.295 0.382 1.000000e+00       1
MED31           1.434233e-01  0.3442041 0.481 0.447 1.000000e+00       1
PRAG1.1         1.438777e-01 -0.5467875 0.211 0.313 1.000000e+00       1
TSPAN2          1.443329e-01 -0.4250633 0.130 0.175 1.000000e+00       1
WDR77           1.449781e-01 -0.2740967 0.302 0.349 1.000000e+00       1
SEC14L2.1       1.450046e-01 -0.6195450 0.081 0.127 1.000000e+00       1
ZC4H2           1.460036e-01 -0.4463573 0.112 0.135 1.000000e+00       1
TMEM50B.1       1.460850e-01  0.3515541 0.474 0.441 1.000000e+00       1
ADAMTS6         1.463289e-01 -0.2933035 0.361 0.448 1.000000e+00       1
ENTPD1          1.469806e-01 -0.4556256 0.126 0.211 1.000000e+00       1
PRKAB2.1        1.473346e-01  0.3627746 0.400 0.368 1.000000e+00       1
AEBP1.1         1.474978e-01 -0.6536431 0.305 0.370 1.000000e+00       1
CLIC6.1         1.476073e-01  0.5343834 0.221 0.247 1.000000e+00       1
SERPINI1.1      1.477158e-01 -0.8342926 0.095 0.219 1.000000e+00       1
AC007114.1      1.481813e-01  0.2915829 0.105 0.098 1.000000e+00       1
ANKRD11         1.496651e-01  0.3049040 0.470 0.439 1.000000e+00       1
KCNN2.1         1.499132e-01 -0.8098230 0.074 0.245 1.000000e+00       1
CASTOR1.1       1.500792e-01  0.5014319 0.204 0.207 1.000000e+00       1
PAMR1           1.501068e-01 -0.5032409 0.102 0.251 1.000000e+00       1
DYRK1B.1        1.502175e-01  0.3871230 0.196 0.212 1.000000e+00       1
ZBTB38.1        1.514104e-01  0.4608438 0.400 0.366 1.000000e+00       1
GUCY1B1         1.528626e-01 -0.3832815 0.098 0.217 1.000000e+00       1
AURKA.1         1.533673e-01  0.2640776 0.151 0.162 1.000000e+00       1
TNFRSF10B.1     1.533678e-01  0.5571143 0.393 0.389 1.000000e+00       1
PLSCR1.1        1.543819e-01  0.3556872 0.284 0.255 1.000000e+00       1
KMO.1           1.546362e-01 -0.6318872 0.056 0.169 1.000000e+00       1
ARMCX1          1.547211e-01 -0.2882692 0.235 0.315 1.000000e+00       1
TFAP2A          1.548343e-01 -0.2528275 0.182 0.324 1.000000e+00       1
TUBB4B.1        1.553444e-01  0.4443659 0.523 0.448 1.000000e+00       1
SLC5A3          1.554863e-01 -0.3384395 0.312 0.411 1.000000e+00       1
CPNE7.1         1.565963e-01  0.5536197 0.221 0.144 1.000000e+00       1
FMO4.1          1.565967e-01 -0.4469843 0.088 0.102 1.000000e+00       1
PHIP.1          1.570540e-01  0.4091825 0.298 0.267 1.000000e+00       1
FNBP1L          1.575983e-01  0.3502974 0.319 0.321 1.000000e+00       1
DYNC1H1.1       1.577418e-01  0.2544645 0.421 0.473 1.000000e+00       1
PSMB8.1         1.586336e-01  1.0289571 0.291 0.127 1.000000e+00       1
SESN3.1         1.594424e-01  0.2926752 0.428 0.410 1.000000e+00       1
ZFHX2           1.613318e-01 -0.3777672 0.088 0.206 1.000000e+00       1
FZD3            1.624149e-01 -0.3256283 0.256 0.276 1.000000e+00       1
ZNF34           1.624437e-01  0.3376008 0.154 0.126 1.000000e+00       1
HMCES.1         1.625911e-01 -0.4840179 0.196 0.265 1.000000e+00       1
GAB2.1          1.644198e-01 -0.5515382 0.098 0.195 1.000000e+00       1
CWC25.1         1.658760e-01  0.4645318 0.284 0.258 1.000000e+00       1
TMTC1.1         1.659356e-01  0.6323808 0.274 0.196 1.000000e+00       1
B3GNT2          1.660251e-01 -0.2781239 0.267 0.368 1.000000e+00       1
TEX29.1         1.664133e-01  0.4505007 0.228 0.178 1.000000e+00       1
HOXC8.1         1.665030e-01 -0.3522078 0.418 0.451 1.000000e+00       1
ABAT            1.668921e-01 -0.2566507 0.168 0.261 1.000000e+00       1
GTF2E1.1        1.670719e-01  0.4085093 0.263 0.293 1.000000e+00       1
RBM24.1         1.675820e-01 -0.4058739 0.091 0.261 1.000000e+00       1
PIR.1           1.683947e-01  0.8005224 0.232 0.120 1.000000e+00       1
PTPRU           1.684250e-01 -0.3445328 0.161 0.230 1.000000e+00       1
RSPO3.1         1.696950e-01  0.7614275 0.316 0.277 1.000000e+00       1
GPS1.1          1.705461e-01 -0.3814962 0.298 0.343 1.000000e+00       1
CTSH            1.709419e-01 -0.3017677 0.119 0.185 1.000000e+00       1
TMBIM1.1        1.722881e-01  0.4302085 0.274 0.278 1.000000e+00       1
ZNF251.1        1.724720e-01 -0.4680705 0.105 0.149 1.000000e+00       1
SMCHD1          1.728102e-01  0.2966857 0.344 0.305 1.000000e+00       1
NR1H3           1.730870e-01 -0.3365036 0.077 0.124 1.000000e+00       1
RCHY1           1.756563e-01  0.2526667 0.351 0.343 1.000000e+00       1
UNKL.1          1.784426e-01  0.5111081 0.204 0.212 1.000000e+00       1
RCN3.1          1.787570e-01 -0.4499139 0.175 0.253 1.000000e+00       1
COL25A1.1       1.788839e-01 -0.5313629 0.119 0.218 1.000000e+00       1
MRAP2.1         1.789785e-01 -1.0288705 0.074 0.302 1.000000e+00       1
SGCD.1          1.798636e-01 -0.8601519 0.046 0.178 1.000000e+00       1
L3HYPDH         1.804665e-01  0.3265019 0.365 0.330 1.000000e+00       1
CRYGS.1         1.819955e-01 -0.4599990 0.105 0.154 1.000000e+00       1
FANCI.1         1.838238e-01  0.6155283 0.144 0.110 1.000000e+00       1
RAP1B.1         1.841460e-01 -0.3055966 0.323 0.390 1.000000e+00       1
KLHL24          1.851797e-01 -0.2516451 0.309 0.378 1.000000e+00       1
TMEM241         1.852443e-01 -0.4042285 0.119 0.217 1.000000e+00       1
MIS18BP1.1      1.856980e-01 -0.3582288 0.116 0.170 1.000000e+00       1
CDH6.1          1.892910e-01 -0.4950458 0.200 0.283 1.000000e+00       1
GINS4.1         1.912727e-01 -0.7923186 0.088 0.187 1.000000e+00       1
HS3ST3A1.1      1.921362e-01 -0.4402766 0.102 0.153 1.000000e+00       1
EXTL1.1         1.937375e-01 -1.1871569 0.039 0.202 1.000000e+00       1
NPNT.1          1.940725e-01  0.5640821 0.242 0.176 1.000000e+00       1
ZNF486          1.944750e-01  0.2537659 0.095 0.098 1.000000e+00       1
Z93930.2.1      1.950119e-01  0.2793852 0.179 0.167 1.000000e+00       1
ARHGEF11.1      1.953826e-01  0.6345877 0.232 0.191 1.000000e+00       1
HOXA7.1         1.969361e-01  0.4386760 0.284 0.290 1.000000e+00       1
FCGRT           1.973429e-01  0.2790310 0.439 0.406 1.000000e+00       1
TM7SF2.1        1.982261e-01 -0.3403629 0.193 0.328 1.000000e+00       1
TMCO6.1         1.986689e-01 -0.4355687 0.074 0.167 1.000000e+00       1
SERPINB1.1      2.001041e-01  0.4196093 0.204 0.161 1.000000e+00       1
AC112721.1.1    2.002415e-01 -0.4733930 0.151 0.225 1.000000e+00       1
FANCC           2.006526e-01  0.2511053 0.193 0.173 1.000000e+00       1
RPRML.1         2.010655e-01 -0.2572855 0.102 0.215 1.000000e+00       1
MYOCD.1         2.014785e-01 -0.2534606 0.319 0.351 1.000000e+00       1
RNF217.1        2.017197e-01  0.3016428 0.435 0.480 1.000000e+00       1
CTXN1.1         2.018231e-01 -0.7084595 0.140 0.253 1.000000e+00       1
BAIAP2-DT.1     2.021680e-01 -0.6721924 0.067 0.139 1.000000e+00       1
SIGIRR          2.022367e-01  0.2988581 0.172 0.167 1.000000e+00       1
GAN             2.024445e-01 -0.2945566 0.277 0.359 1.000000e+00       1
GLP2R           2.027553e-01 -0.3102218 0.060 0.142 1.000000e+00       1
HAND1.1         2.036566e-01  0.5157880 0.249 0.190 1.000000e+00       1
SGCA            2.038300e-01 -0.2883755 0.140 0.217 1.000000e+00       1
ATP23           2.050834e-01  0.2612320 0.172 0.183 1.000000e+00       1
GARS.1          2.057120e-01  0.3899087 0.495 0.444 1.000000e+00       1
ANO4.1          2.064817e-01 -0.4381345 0.063 0.142 1.000000e+00       1
PARM1.1         2.070786e-01  0.6310629 0.221 0.215 1.000000e+00       1
LIMCH1          2.079232e-01  0.2871202 0.200 0.205 1.000000e+00       1
GLIPR2.1        2.079585e-01 -0.2743619 0.274 0.356 1.000000e+00       1
DUSP23          2.107199e-01  0.4176117 0.189 0.159 1.000000e+00       1
MT-ND5.1        2.116827e-01  0.4264965 0.460 0.415 1.000000e+00       1
ERG28.1         2.121471e-01 -0.4967792 0.400 0.453 1.000000e+00       1
NBL1.1          2.149129e-01  0.3955616 0.404 0.379 1.000000e+00       1
C1RL.1          2.150934e-01  0.5279727 0.168 0.143 1.000000e+00       1
CCDC74A.1       2.151656e-01 -0.2619184 0.091 0.150 1.000000e+00       1
AC009113.1      2.159613e-01  0.2993031 0.081 0.112 1.000000e+00       1
RALA            2.169407e-01 -0.3226460 0.347 0.370 1.000000e+00       1
SRFBP1.1        2.173043e-01  0.5689147 0.284 0.278 1.000000e+00       1
ZFHX4.1         2.179962e-01 -0.4462376 0.175 0.259 1.000000e+00       1
AC021016.3      2.184706e-01  0.3065606 0.123 0.122 1.000000e+00       1
SLC4A8.1        2.215899e-01  0.4929994 0.263 0.316 1.000000e+00       1
KCND3.1         2.216273e-01 -0.6875956 0.168 0.256 1.000000e+00       1
TRPC1           2.225133e-01 -0.2644494 0.154 0.258 1.000000e+00       1
KPNA5           2.226990e-01 -0.3540651 0.200 0.194 1.000000e+00       1
FIRRE.1         2.234033e-01 -0.5918255 0.098 0.214 1.000000e+00       1
MELTF.1         2.240717e-01  0.9566835 0.218 0.137 1.000000e+00       1
NFKBID          2.245556e-01  0.2796195 0.154 0.115 1.000000e+00       1
EVA1A           2.245930e-01 -0.4576872 0.105 0.258 1.000000e+00       1
MAP3K8.1        2.259360e-01  0.5476149 0.232 0.146 1.000000e+00       1
DHRS3.1         2.263858e-01 -0.5685958 0.207 0.220 1.000000e+00       1
DPP9            2.267608e-01  0.3008141 0.225 0.245 1.000000e+00       1
SBF1            2.268733e-01 -0.4092818 0.175 0.229 1.000000e+00       1
B3GALNT2.1      2.278126e-01 -0.6069753 0.158 0.265 1.000000e+00       1
KIAA1841        2.300775e-01  0.2591822 0.165 0.161 1.000000e+00       1
ZNF469          2.325889e-01 -0.7165600 0.151 0.285 1.000000e+00       1
KMT5C.1         2.331620e-01 -0.6672880 0.123 0.174 1.000000e+00       1
VSIG10L.1       2.338899e-01  0.2711605 0.067 0.112 1.000000e+00       1
TUBA1B          2.349267e-01  0.3199496 0.414 0.425 1.000000e+00       1
STK24.1         2.352730e-01  0.3026218 0.305 0.320 1.000000e+00       1
EFNA1.1         2.361593e-01 -0.6064042 0.084 0.138 1.000000e+00       1
ANKRD34A.1      2.367005e-01 -0.5616202 0.060 0.185 1.000000e+00       1
ADAT2.1         2.368164e-01 -0.3096661 0.077 0.110 1.000000e+00       1
SYNE2.1         2.369322e-01  0.4603399 0.140 0.158 1.000000e+00       1
DGKH            2.374746e-01 -0.2541621 0.263 0.290 1.000000e+00       1
IRAK1.1         2.383671e-01  0.3245834 0.284 0.252 1.000000e+00       1
ZNF689.1        2.386781e-01 -0.4338323 0.168 0.169 1.000000e+00       1
AL441883.1.1    2.390282e-01 -0.3747236 0.070 0.133 1.000000e+00       1
TP53I3.1        2.407457e-01 -0.5312697 0.379 0.440 1.000000e+00       1
MMP16           2.421573e-01 -0.6381384 0.161 0.284 1.000000e+00       1
SRSF3.1         2.438115e-01  0.5686933 0.460 0.417 1.000000e+00       1
FBXL22.1        2.440479e-01 -0.3051296 0.154 0.111 1.000000e+00       1
ANKRD29.1       2.447602e-01 -0.2766315 0.091 0.129 1.000000e+00       1
GOLGA8A         2.449581e-01 -0.2819861 0.112 0.132 1.000000e+00       1
IFT88           2.469043e-01  0.2742394 0.207 0.186 1.000000e+00       1
AL356599.1      2.470238e-01  0.3968926 0.130 0.121 1.000000e+00       1
C8orf88.1       2.479814e-01  0.6093272 0.158 0.130 1.000000e+00       1
ALCAM.1         2.485413e-01  0.5380466 0.481 0.455 1.000000e+00       1
MICA.1          2.499044e-01 -0.4813488 0.439 0.454 1.000000e+00       1
AC078850.1.1    2.511920e-01 -0.5236238 0.091 0.208 1.000000e+00       1
RRM1            2.515550e-01  0.3778639 0.175 0.241 1.000000e+00       1
GK              2.528486e-01  0.3103639 0.130 0.139 1.000000e+00       1
DKK1.1          2.530102e-01  0.4689218 0.389 0.356 1.000000e+00       1
MYOZ1.1         2.537813e-01 -0.5876556 0.060 0.175 1.000000e+00       1
ASB13.1         2.547162e-01  0.3791399 0.126 0.131 1.000000e+00       1
FRMD4A.1        2.554088e-01 -0.7649724 0.196 0.239 1.000000e+00       1
SPATA17         2.555721e-01  0.3982112 0.144 0.088 1.000000e+00       1
AC073389.1.1    2.563890e-01  0.3784707 0.102 0.109 1.000000e+00       1
PPP1R3B         2.571666e-01  0.2552898 0.225 0.179 1.000000e+00       1
MOB3B.1         2.578640e-01 -0.4358023 0.232 0.280 1.000000e+00       1
MMP24.1         2.579042e-01 -0.5664754 0.077 0.183 1.000000e+00       1
SSPN.1          2.605008e-01 -0.2902598 0.323 0.378 1.000000e+00       1
ZNF692          2.607484e-01 -0.6566422 0.070 0.192 1.000000e+00       1
VEGFC.1         2.607899e-01 -0.7420792 0.105 0.199 1.000000e+00       1
GFPT1.1         2.608317e-01  0.5931969 0.502 0.462 1.000000e+00       1
TYW5            2.611629e-01  0.3056390 0.239 0.218 1.000000e+00       1
MSRA.1          2.624074e-01  0.2509168 0.253 0.254 1.000000e+00       1
PKD1            2.626152e-01 -0.2691351 0.207 0.282 1.000000e+00       1
FBLN1.1         2.631145e-01  0.4709509 0.460 0.434 1.000000e+00       1
RNF128.1        2.643650e-01  0.4161545 0.158 0.171 1.000000e+00       1
PTGFR.1         2.664594e-01  0.3907243 0.207 0.164 1.000000e+00       1
PLCE1.1         2.665434e-01 -0.6720985 0.165 0.285 1.000000e+00       1
BICRA.1         2.673418e-01 -0.5490335 0.109 0.148 1.000000e+00       1
C3orf52         2.689028e-01 -0.4511763 0.116 0.214 1.000000e+00       1
RHPN2           2.689873e-01  0.3117649 0.151 0.150 1.000000e+00       1
ZBTB7B.1        2.711064e-01  0.3031624 0.193 0.231 1.000000e+00       1
ATP10A          2.718721e-01 -0.4500188 0.207 0.333 1.000000e+00       1
KCTD5           2.735789e-01 -0.2629767 0.228 0.242 1.000000e+00       1
OR8S1.1         2.764970e-01  0.5417211 0.253 0.325 1.000000e+00       1
NR2F1-AS1.1     2.769712e-01 -0.5494967 0.295 0.324 1.000000e+00       1
PER2.1          2.771003e-01 -0.6553544 0.126 0.210 1.000000e+00       1
PTTG1           2.776186e-01  0.2725663 0.270 0.259 1.000000e+00       1
INA.1           2.783102e-01 -0.6209719 0.126 0.228 1.000000e+00       1
ZNF185          2.799576e-01  0.2868511 0.200 0.197 1.000000e+00       1
PNPLA2.1        2.809143e-01  0.3228371 0.358 0.332 1.000000e+00       1
GGACT           2.826591e-01 -0.3076032 0.119 0.176 1.000000e+00       1
ZSWIM9          2.829216e-01 -0.2859263 0.123 0.164 1.000000e+00       1
AL118516.1.1    2.843243e-01 -0.3765206 0.218 0.286 1.000000e+00       1
GPAM.1          2.847195e-01  0.4245334 0.214 0.219 1.000000e+00       1
SNTA1.1         2.848513e-01  0.4847669 0.249 0.203 1.000000e+00       1
CCDC81.1        2.861278e-01 -0.6959765 0.070 0.176 1.000000e+00       1
LINC00545.1     2.861718e-01  0.6669608 0.147 0.123 1.000000e+00       1
PLD5.1          2.863920e-01 -0.4794835 0.123 0.202 1.000000e+00       1
PCAT1.1         2.866130e-01 -0.3312559 0.088 0.246 1.000000e+00       1
MARCH3          2.871428e-01  0.2576957 0.204 0.261 1.000000e+00       1
SLC22A2.1       2.871430e-01 -0.5205584 0.084 0.204 1.000000e+00       1
ST6GAL2.1       2.878063e-01 -0.8234104 0.074 0.174 1.000000e+00       1
FMOD.1          2.882475e-01  0.2862624 0.232 0.261 1.000000e+00       1
HES1.1          2.882488e-01 -0.5361044 0.225 0.320 1.000000e+00       1
GALNT16         2.896692e-01 -0.3505140 0.105 0.219 1.000000e+00       1
PPOX            2.900695e-01  0.2779252 0.239 0.261 1.000000e+00       1
GINS2           2.902469e-01  0.4598768 0.116 0.144 1.000000e+00       1
C3orf70         2.904256e-01 -0.3951472 0.144 0.142 1.000000e+00       1
SLC40A1.1       2.908225e-01  0.6845276 0.354 0.374 1.000000e+00       1
ZNF669          2.909154e-01 -0.4744733 0.119 0.160 1.000000e+00       1
BX255923.1.1    2.912725e-01 -0.4042415 0.126 0.199 1.000000e+00       1
SGK494.1        2.922998e-01 -0.5490905 0.095 0.156 1.000000e+00       1
NFYA.1          2.931950e-01 -0.5287436 0.126 0.179 1.000000e+00       1
SSC5D.1         2.936885e-01 -0.7054188 0.211 0.324 1.000000e+00       1
FHL3            2.939578e-01  0.2756429 0.319 0.310 1.000000e+00       1
SORT1.1         2.948566e-01  0.3393909 0.340 0.376 1.000000e+00       1
AP1G2           2.977445e-01 -0.4481293 0.137 0.150 1.000000e+00       1
HOXC9           2.982434e-01  0.2850602 0.319 0.322 1.000000e+00       1
CHST6           2.992415e-01 -0.4105081 0.095 0.118 1.000000e+00       1
TAF1A           3.002878e-01  0.3098565 0.133 0.108 1.000000e+00       1
CBL             3.003334e-01 -0.2555168 0.253 0.271 1.000000e+00       1
MAP1LC3B        3.024790e-01  0.2591642 0.428 0.454 1.000000e+00       1
FAM19A2.1       3.062923e-01 -0.5285063 0.088 0.158 1.000000e+00       1
GAMT            3.063389e-01 -0.3486966 0.298 0.353 1.000000e+00       1
ZGLP1.1         3.097196e-01 -0.6294005 0.098 0.145 1.000000e+00       1
RBM3            3.097197e-01  0.2568863 0.481 0.427 1.000000e+00       1
ADD3            3.131251e-01 -0.2643678 0.256 0.328 1.000000e+00       1
AC110285.6.1    3.133581e-01 -0.5866454 0.126 0.162 1.000000e+00       1
TOB1.1          3.141562e-01  0.2523135 0.365 0.351 1.000000e+00       1
CSNK1G1.1       3.144374e-01 -0.4345473 0.225 0.265 1.000000e+00       1
NRTN.1          3.150484e-01  0.5723203 0.126 0.166 1.000000e+00       1
SNHG8           3.164608e-01  0.2516637 0.474 0.420 1.000000e+00       1
HIST1H2BN       3.170261e-01  0.2532959 0.165 0.212 1.000000e+00       1
JPH1.1          3.183502e-01 -0.3188687 0.168 0.222 1.000000e+00       1
G6PD.1          3.192503e-01  0.3320198 0.354 0.347 1.000000e+00       1
RGS3            3.197721e-01 -0.5603504 0.158 0.215 1.000000e+00       1
F2RL1           3.202942e-01 -0.4546704 0.235 0.258 1.000000e+00       1
PGM5            3.208651e-01 -0.2910325 0.112 0.222 1.000000e+00       1
CLUAP1          3.218652e-01  0.2535901 0.274 0.242 1.000000e+00       1
HSPA1B          3.220559e-01  0.3424420 0.246 0.228 1.000000e+00       1
PTAR1           3.242544e-01 -0.2653862 0.326 0.349 1.000000e+00       1
AC062029.1.1    3.253094e-01 -0.4128782 0.095 0.173 1.000000e+00       1
PLEKHH2.1       3.253570e-01 -0.6254587 0.112 0.175 1.000000e+00       1
TMEM64          3.262223e-01  0.3090663 0.365 0.373 1.000000e+00       1
UAP1.1          3.272331e-01  0.4819499 0.421 0.416 1.000000e+00       1
FZD8            3.279082e-01 -0.4822828 0.172 0.306 1.000000e+00       1
MPP5.1          3.282461e-01 -0.2629149 0.214 0.310 1.000000e+00       1
AGPAT4.1        3.284875e-01  0.2813521 0.256 0.242 1.000000e+00       1
ABLIM1.1        3.286324e-01 -0.6521090 0.137 0.228 1.000000e+00       1
RAB3IP.1        3.294546e-01 -0.3117045 0.175 0.266 1.000000e+00       1
RALGPS2.1       3.307145e-01 -0.3613849 0.319 0.385 1.000000e+00       1
EGR2.1          3.310538e-01 -0.7985885 0.081 0.174 1.000000e+00       1
AC116667.1.1    3.344150e-01 -0.4995197 0.070 0.164 1.000000e+00       1
FDFT1.1         3.346595e-01 -0.4382372 0.404 0.447 1.000000e+00       1
TGIF2.1         3.348551e-01 -0.6718085 0.105 0.246 1.000000e+00       1
DDX17.1         3.355893e-01  0.2594905 0.474 0.469 1.000000e+00       1
CD1D            3.363735e-01 -0.3666983 0.077 0.102 1.000000e+00       1
PBXIP1          3.379947e-01 -0.3011838 0.249 0.325 1.000000e+00       1
H2AFZ           3.388811e-01  0.4447965 0.453 0.410 1.000000e+00       1
ERMARD          3.403617e-01  0.3708058 0.165 0.184 1.000000e+00       1
PEX3.1          3.406090e-01  0.4794356 0.211 0.215 1.000000e+00       1
CDC14B          3.408563e-01  0.2837048 0.207 0.206 1.000000e+00       1
OGA             3.409553e-01 -0.2555524 0.312 0.381 1.000000e+00       1
LSAMP.1         3.411030e-01  0.7305906 0.221 0.221 1.000000e+00       1
GPRC5A.1        3.417968e-01 -0.9401007 0.165 0.286 1.000000e+00       1
LINC01082.1     3.437830e-01 -0.3270762 0.081 0.109 1.000000e+00       1
S100A11         3.454769e-01  0.2884874 0.456 0.429 1.000000e+00       1
LPP             3.457764e-01 -0.3214037 0.456 0.452 1.000000e+00       1
SHKBP1.1        3.461260e-01  0.3033445 0.263 0.264 1.000000e+00       1
LTBP1.1         3.465756e-01 -0.5672172 0.375 0.416 1.000000e+00       1
LINC01311.1     3.467752e-01 -0.6630926 0.095 0.160 1.000000e+00       1
LRRC37A3.1      3.477266e-01  0.3393917 0.161 0.141 1.000000e+00       1
AL050341.2.1    3.489301e-01 -0.3652723 0.175 0.217 1.000000e+00       1
ANKRD37.1       3.515980e-01 -0.7217460 0.130 0.208 1.000000e+00       1
ZNF574.1        3.524570e-01 -0.3058757 0.137 0.202 1.000000e+00       1
MARC2           3.528109e-01  0.2950484 0.084 0.102 1.000000e+00       1
GASAL1.1        3.537723e-01 -0.2591114 0.116 0.183 1.000000e+00       1
PPIC            3.539246e-01  0.4035671 0.467 0.432 1.000000e+00       1
TMEM45A         3.564128e-01  0.3312498 0.421 0.459 1.000000e+00       1
AL365203.2      3.568197e-01 -0.3724546 0.060 0.151 1.000000e+00       1
HIST1H2AC       3.573297e-01  0.3454269 0.404 0.394 1.000000e+00       1
AC009159.3.1    3.598327e-01 -0.5033157 0.046 0.115 1.000000e+00       1
ANGPTL2         3.600376e-01  0.3126413 0.288 0.272 1.000000e+00       1
JUNB.1          3.600890e-01  0.6319476 0.389 0.364 1.000000e+00       1
PCDH18.1        3.606525e-01 -0.4447345 0.182 0.315 1.000000e+00       1
NEDD4L          3.630161e-01  0.3249024 0.154 0.159 1.000000e+00       1
GSEC            3.654401e-01 -0.3343094 0.095 0.186 1.000000e+00       1
UBP1            3.683952e-01 -0.4056994 0.330 0.350 1.000000e+00       1
BTN3A2.1        3.686030e-01  0.6412462 0.263 0.203 1.000000e+00       1
IL17RE.1        3.691748e-01  0.4914267 0.147 0.120 1.000000e+00       1
LINC01759.1     3.709988e-01  0.3360484 0.207 0.236 1.000000e+00       1
HOXB-AS1.1      3.735087e-01 -0.4747801 0.098 0.168 1.000000e+00       1
MAD2L1          3.735612e-01  0.3781246 0.239 0.237 1.000000e+00       1
SMAGP           3.751878e-01 -0.4019363 0.116 0.142 1.000000e+00       1
KIFC2.1         3.753457e-01 -0.7046884 0.112 0.131 1.000000e+00       1
AL163051.1.1    3.786664e-01 -0.5956293 0.053 0.115 1.000000e+00       1
ZNF107          3.789307e-01 -0.3270810 0.032 0.139 1.000000e+00       1
ARSI.1          3.791419e-01 -0.8790830 0.116 0.151 1.000000e+00       1
PCDHA10.1       3.793009e-01 -0.2757919 0.158 0.154 1.000000e+00       1
NUDT16          3.805721e-01  0.2879040 0.347 0.308 1.000000e+00       1
MDM2.1          3.819519e-01  0.4074571 0.425 0.487 1.000000e+00       1
IER2            3.819520e-01  0.2644505 0.396 0.377 1.000000e+00       1
NNMT            3.851480e-01  0.3396636 0.491 0.466 1.000000e+00       1
ZNF117.1        3.870735e-01 -0.3546491 0.147 0.204 1.000000e+00       1
KIAA1755.1      3.872877e-01 -0.5234719 0.070 0.141 1.000000e+00       1
GSTP1.1         3.881457e-01  0.4435261 0.432 0.454 1.000000e+00       1
NKTR.1          3.920747e-01  0.5001031 0.411 0.441 1.000000e+00       1
KIF9-AS1.1      3.923447e-01  0.4156647 0.140 0.129 1.000000e+00       1
USP12           3.933717e-01 -0.3762126 0.225 0.269 1.000000e+00       1
ELL2.1          3.943461e-01  0.3484685 0.333 0.301 1.000000e+00       1
SLIT3           3.954846e-01 -0.4048552 0.151 0.240 1.000000e+00       1
PTPRF           3.965167e-01 -0.3494047 0.232 0.338 1.000000e+00       1
PCMTD2.1        4.014817e-01  0.3170121 0.277 0.291 1.000000e+00       1
NDUFA12         4.016460e-01  0.3197328 0.368 0.430 1.000000e+00       1
CDC42EP4.1      4.021939e-01  0.5199760 0.165 0.113 1.000000e+00       1
PSPH.1          4.028521e-01 -0.3215722 0.182 0.234 1.000000e+00       1
EPHX2.1         4.035107e-01 -0.4711644 0.074 0.107 1.000000e+00       1
OLFML2B.1       4.052157e-01 -1.0844961 0.077 0.185 1.000000e+00       1
ITGA3           4.067045e-01 -0.6088178 0.246 0.346 1.000000e+00       1
CLEC3A.1        4.069804e-01 -0.3218707 0.270 0.357 1.000000e+00       1
AC073323.1.1    4.115784e-01  0.4985654 0.274 0.200 1.000000e+00       1
NSFL1C.1        4.115785e-01  0.3155130 0.354 0.324 1.000000e+00       1
ANKAR.1         4.131361e-01  0.4281571 0.151 0.129 1.000000e+00       1
LPAR2           4.135821e-01 -0.4607741 0.091 0.203 1.000000e+00       1
XPR1            4.144189e-01 -0.3493896 0.235 0.277 1.000000e+00       1
AGAP9.1         4.146422e-01  0.2517553 0.161 0.180 1.000000e+00       1
AL136084.3.1    4.148092e-01 -0.4158319 0.091 0.161 1.000000e+00       1
TPD52.1         4.160949e-01  0.2671472 0.168 0.171 1.000000e+00       1
AC109322.1      4.168222e-01  0.2531822 0.105 0.143 1.000000e+00       1
LINC02482.1     4.169908e-01 -0.3460837 0.186 0.229 1.000000e+00       1
PCM1.1          4.186171e-01  0.3165425 0.368 0.371 1.000000e+00       1
CD248.1         4.208096e-01  0.8338504 0.189 0.171 1.000000e+00       1
PLAUR.1         4.208663e-01 -0.4955101 0.288 0.340 1.000000e+00       1
CNIH2           4.224448e-01 -0.5339564 0.060 0.144 1.000000e+00       1
UBE2L6.1        4.233482e-01  0.3872349 0.207 0.150 1.000000e+00       1
SLC2A5.1        4.235182e-01  0.3705908 0.133 0.133 1.000000e+00       1
MAPK8IP3.1      4.236311e-01 -0.4509564 0.158 0.211 1.000000e+00       1
AC145285.6      4.245362e-01 -0.3647002 0.098 0.159 1.000000e+00       1
ATRNL1.1        4.249326e-01 -0.5058173 0.211 0.285 1.000000e+00       1
HIPK2.1         4.260663e-01  0.3178625 0.463 0.423 1.000000e+00       1
TGFBR3.1        4.262930e-01  0.9732076 0.218 0.127 1.000000e+00       1
ALG10B          4.282820e-01 -0.3640385 0.207 0.237 1.000000e+00       1
MTCH1           4.285096e-01  0.2855792 0.474 0.439 1.000000e+00       1
OLR1.1          4.297626e-01 -0.5478850 0.137 0.217 1.000000e+00       1
LAMA5           4.310752e-01 -0.3170370 0.228 0.221 1.000000e+00       1
FAAH.1          4.314749e-01  0.2665199 0.095 0.146 1.000000e+00       1
FBXW8           4.325044e-01 -0.3605759 0.193 0.275 1.000000e+00       1
C16orf86.1      4.333632e-01  0.5143950 0.105 0.061 1.000000e+00       1
TRIM47.1        4.349685e-01  0.3241868 0.214 0.186 1.000000e+00       1
AC137767.1.1    4.353704e-01 -0.3478294 0.172 0.178 1.000000e+00       1
TGFB2.1         4.369810e-01 -0.3733844 0.158 0.255 1.000000e+00       1
ETV5.1          4.370386e-01  0.2861577 0.316 0.330 1.000000e+00       1
GBE1.1          4.375569e-01  0.4147415 0.347 0.328 1.000000e+00       1
EZH2.1          4.377870e-01 -0.4589110 0.123 0.167 1.000000e+00       1
FHOD3.1         4.396913e-01 -0.7129311 0.039 0.247 1.000000e+00       1
SLC1A3.1        4.410789e-01  0.6922824 0.196 0.114 1.000000e+00       1
ARAP2.1         4.424112e-01 -0.5835282 0.095 0.210 1.000000e+00       1
PCNA.1          4.447336e-01  0.3106206 0.337 0.376 1.000000e+00       1
CPNE2.1         4.463050e-01  0.3150601 0.193 0.176 1.000000e+00       1
ID2.1           4.492236e-01 -0.4933369 0.133 0.198 1.000000e+00       1
KIAA0895L.1     4.495155e-01  0.3168613 0.137 0.130 1.000000e+00       1
CEP350.1        4.500428e-01  0.4239381 0.319 0.345 1.000000e+00       1
ZCWPW1.1        4.506284e-01  0.5683500 0.186 0.156 1.000000e+00       1
BTN3A3          4.521530e-01  0.2730910 0.168 0.151 1.000000e+00       1
RAB40C.1        4.539745e-01 -0.3303584 0.130 0.183 1.000000e+00       1
ANKRD36C        4.563897e-01 -0.2715589 0.158 0.192 1.000000e+00       1
LIMK2.1         4.570395e-01  0.2636497 0.411 0.441 1.000000e+00       1
NLRP1.1         4.573939e-01 -0.3155462 0.295 0.333 1.000000e+00       1
LAMA1           4.579846e-01 -0.5329306 0.119 0.172 1.000000e+00       1
RPS4Y1.1        4.599376e-01 -0.5569696 0.249 0.278 1.000000e+00       1
TRABD2B.1       4.601160e-01 -0.5134387 0.179 0.221 1.000000e+00       1
TMEM56.1        4.603529e-01 -0.6037033 0.084 0.190 1.000000e+00       1
AC073611.1.1    4.604124e-01 -0.3179300 0.249 0.313 1.000000e+00       1
H3F3B.1         4.623715e-01  0.4681906 0.453 0.454 1.000000e+00       1
AP1G1.1         4.627877e-01 -0.3396272 0.337 0.403 1.000000e+00       1
SMCO4.1         4.646928e-01 -0.3439352 0.358 0.354 1.000000e+00       1
TTC7A.1         4.649908e-01 -0.4096507 0.193 0.254 1.000000e+00       1
GBP2.1          4.663032e-01 -0.2688976 0.147 0.146 1.000000e+00       1
AC147067.1      4.669606e-01  0.3115341 0.232 0.206 1.000000e+00       1
SESN1.1         4.709144e-01  0.4197082 0.260 0.247 1.000000e+00       1
CASP6.1         4.712146e-01 -0.3704122 0.196 0.256 1.000000e+00       1
ANO8            4.713347e-01 -0.3887259 0.067 0.151 1.000000e+00       1
HAPLN4.1        4.713942e-01 -0.5776014 0.137 0.268 1.000000e+00       1
ZNF628.1        4.724166e-01 -0.6398430 0.105 0.117 1.000000e+00       1
SLC30A1         4.736808e-01 -0.2982604 0.256 0.302 1.000000e+00       1
HEG1            4.739218e-01 -0.3558769 0.372 0.394 1.000000e+00       1
CNTRL.1         4.739820e-01  0.4897454 0.144 0.115 1.000000e+00       1
DIP2C           4.755502e-01 -0.4136279 0.305 0.331 1.000000e+00       1
B4GALT5.1       4.758521e-01  0.3398066 0.344 0.364 1.000000e+00       1
SEPHS2.1        4.785738e-01  0.5585930 0.382 0.355 1.000000e+00       1
ZNF883.1        4.790580e-01 -0.4254569 0.116 0.167 1.000000e+00       1
PPM1H.1         4.794223e-01 -0.3796523 0.039 0.130 1.000000e+00       1
RASGRF2.1       4.825202e-01 -0.4349516 0.228 0.330 1.000000e+00       1
CARD19          4.829463e-01  0.2888492 0.323 0.317 1.000000e+00       1
BMP6.1          4.837990e-01 -0.4423609 0.168 0.334 1.000000e+00       1
WHAMM.1         4.843475e-01  0.3627241 0.295 0.255 1.000000e+00       1
UBE2S.1         4.859343e-01  0.6442371 0.372 0.397 1.000000e+00       1
ZNF839.1        4.868508e-01  0.3066547 0.116 0.142 1.000000e+00       1
ID4.1           4.880745e-01 -0.4045459 0.070 0.134 1.000000e+00       1
GINS1.1         4.910183e-01 -0.3348828 0.123 0.137 1.000000e+00       1
EPB41L2.1       4.927400e-01  0.2638303 0.277 0.307 1.000000e+00       1
FAM122A.1       4.939716e-01 -0.3193411 0.242 0.306 1.000000e+00       1
SLC39A14.1      4.940949e-01  0.3619486 0.344 0.325 1.000000e+00       1
ZSCAN18         4.944647e-01 -0.3509211 0.189 0.273 1.000000e+00       1
LINC01252.1     4.956363e-01 -0.7657993 0.154 0.198 1.000000e+00       1
VAMP5.1         4.966248e-01  0.4073264 0.130 0.132 1.000000e+00       1
TGFBR2.1        4.968725e-01 -0.2973247 0.407 0.446 1.000000e+00       1
UNC5B           4.971197e-01 -0.4873332 0.214 0.261 1.000000e+00       1
AHR             4.982953e-01 -0.3570920 0.200 0.248 1.000000e+00       1
ADGRF4.1        4.991619e-01  0.4410275 0.186 0.171 1.000000e+00       1
KLHL28.1        5.022038e-01  0.2859326 0.246 0.258 1.000000e+00       1
AL132656.2.1    5.025146e-01 -0.7321965 0.039 0.157 1.000000e+00       1
ADAM15          5.029501e-01 -0.2563605 0.200 0.222 1.000000e+00       1
RAMP1           5.033233e-01 -0.3296678 0.196 0.248 1.000000e+00       1
ASAP2.1         5.066905e-01  0.3297151 0.389 0.353 1.000000e+00       1
OBSCN           5.083158e-01 -0.6069440 0.109 0.193 1.000000e+00       1
APOBEC3C.1      5.086287e-01  0.4230685 0.284 0.302 1.000000e+00       1
MRS2.1          5.111353e-01 -0.2963655 0.256 0.345 1.000000e+00       1
DPH6.1          5.144663e-01 -0.2534483 0.123 0.184 1.000000e+00       1
STARD10         5.162940e-01  0.2864404 0.288 0.282 1.000000e+00       1
AL590426.2.1    5.202755e-01 -1.1881185 0.018 0.186 1.000000e+00       1
CENPQ           5.216059e-01  0.4062482 0.193 0.170 1.000000e+00       1
NRP2            5.224946e-01  0.2746272 0.456 0.441 1.000000e+00       1
DNAJC14         5.228752e-01 -0.4247015 0.172 0.232 1.000000e+00       1
ASPN.1          5.242722e-01 -0.9318238 0.137 0.239 1.000000e+00       1
ENTPD6          5.243365e-01 -0.4380464 0.151 0.159 1.000000e+00       1
RASL11B.1       5.255452e-01 -1.1752257 0.053 0.215 1.000000e+00       1
CNKSR3.1        5.267556e-01  0.4704426 0.182 0.167 1.000000e+00       1
KCNJ2.1         5.280949e-01 -0.3510657 0.077 0.155 1.000000e+00       1
LRCH4.1         5.293719e-01 -0.2582557 0.172 0.222 1.000000e+00       1
RAB7B.1         5.302028e-01 -0.4669404 0.098 0.184 1.000000e+00       1
GPSM1           5.338539e-01 -0.3387957 0.154 0.220 1.000000e+00       1
ZNF577          5.360372e-01 -0.3254374 0.102 0.170 1.000000e+00       1
PRUNE2.1        5.362945e-01  0.5889885 0.193 0.164 1.000000e+00       1
FAF2            5.363593e-01  0.2638284 0.404 0.374 1.000000e+00       1
LINC01806       5.381613e-01 -0.2818175 0.102 0.164 1.000000e+00       1
GNG2.1          5.382899e-01 -1.1743626 0.039 0.131 1.000000e+00       1
CKS2.1          5.408709e-01  0.4506358 0.354 0.337 1.000000e+00       1
NSRP1.1         5.461800e-01  0.3296580 0.418 0.423 1.000000e+00       1
PLXDC1.1        5.465693e-01 -1.0386319 0.091 0.247 1.000000e+00       1
SOD1            5.480637e-01  0.3370476 0.432 0.440 1.000000e+00       1
LINC01705.1     5.504717e-01 -0.3410774 0.095 0.169 1.000000e+00       1
ADCY7.1         5.507324e-01 -0.5933110 0.116 0.225 1.000000e+00       1
NASP.1          5.529504e-01  0.5896614 0.337 0.309 1.000000e+00       1
FAM43A.1        5.532116e-01 -0.4880092 0.186 0.300 1.000000e+00       1
PPP1R13L        5.532769e-01 -0.2510320 0.147 0.281 1.000000e+00       1
NUDCD1.1        5.532769e-01  0.4130233 0.291 0.286 1.000000e+00       1
PSMA3           5.533422e-01  0.2611033 0.481 0.437 1.000000e+00       1
THAP7-AS1.1     5.544530e-01  0.4883880 0.175 0.158 1.000000e+00       1
MYL7.1          5.549762e-01 -0.6369036 0.056 0.216 1.000000e+00       1
TC2N            5.549763e-01 -0.4677249 0.102 0.244 1.000000e+00       1
NMB             5.555653e-01 -0.2514154 0.175 0.219 1.000000e+00       1
SLC6A17         5.556962e-01 -0.3251957 0.161 0.308 1.000000e+00       1
GPR87.1         5.560233e-01 -0.7507594 0.042 0.149 1.000000e+00       1
PNP.1           5.577271e-01  0.3396512 0.274 0.263 1.000000e+00       1
NDUFB2-AS1.1    5.595641e-01  0.3475155 0.123 0.131 1.000000e+00       1
RPAP1           5.605499e-01  0.3806836 0.211 0.210 1.000000e+00       1
IGF2-AS.1       5.607473e-01  0.5558281 0.133 0.092 1.000000e+00       1
SLC6A8.1        5.616682e-01  0.4339200 0.295 0.364 1.000000e+00       1
ZCCHC14.1       5.627214e-01 -0.3294791 0.193 0.220 1.000000e+00       1
CEP131          5.629848e-01 -0.4603223 0.039 0.135 1.000000e+00       1
MID1IP1.1       5.629849e-01  0.5790080 0.256 0.225 1.000000e+00       1
COMT.1          5.630500e-01 -0.5914455 0.207 0.284 1.000000e+00       1
XPO6            5.634461e-01  0.3649023 0.330 0.328 1.000000e+00       1
PIGU.1          5.646982e-01 -0.3803047 0.175 0.191 1.000000e+00       1
FBXO4.1         5.704516e-01  0.2937804 0.116 0.130 1.000000e+00       1
FBLN5.1         5.735715e-01  0.3953570 0.439 0.452 1.000000e+00       1
LAMB2.1         5.779659e-01  0.2869256 0.456 0.455 1.000000e+00       1
TADA1           5.783659e-01  0.4378820 0.175 0.147 1.000000e+00       1
TMEM200A.1      5.814392e-01 -0.4863449 0.228 0.300 1.000000e+00       1
GRIP2.1         5.819067e-01 -0.5822781 0.067 0.161 1.000000e+00       1
SULF1.1         5.819073e-01 -0.5002653 0.084 0.179 1.000000e+00       1
DOCK1.1         5.821751e-01 -0.2797995 0.284 0.291 1.000000e+00       1
RNASEH2C.1      5.829786e-01  0.5065457 0.439 0.425 1.000000e+00       1
MST1            5.834470e-01 -0.4220333 0.060 0.155 1.000000e+00       1
GREB1.1         5.834471e-01 -0.6322223 0.074 0.159 1.000000e+00       1
ZFAND2A         5.853918e-01  0.3198073 0.319 0.303 1.000000e+00       1
GALM            5.880787e-01  0.3370820 0.232 0.220 1.000000e+00       1
MVK.1           5.896262e-01 -0.3485204 0.218 0.292 1.000000e+00       1
RAD50.1         5.902321e-01  0.3518558 0.179 0.188 1.000000e+00       1
LOXL3           5.915126e-01 -0.3984072 0.119 0.233 1.000000e+00       1
AP000487.1.1    5.948205e-01 -0.4468344 0.084 0.149 1.000000e+00       1
PDCL3.1         5.961055e-01 -0.3446531 0.242 0.276 1.000000e+00       1
F2R             5.963757e-01 -0.3075559 0.196 0.248 1.000000e+00       1
CSRNP3.1        5.998999e-01 -0.9241749 0.063 0.211 1.000000e+00       1
FBXO16          6.005786e-01 -0.2624957 0.158 0.187 1.000000e+00       1
BLMH.1          6.049310e-01  0.3893704 0.267 0.252 1.000000e+00       1
NTRK2.1         6.049987e-01 -0.3924830 0.158 0.222 1.000000e+00       1
ASZ1            6.050672e-01 -0.2877996 0.074 0.174 1.000000e+00       1
MCAM.1          6.059530e-01 -0.3662727 0.105 0.197 1.000000e+00       1
ARL6IP1.1       6.061576e-01  0.6087554 0.425 0.446 1.000000e+00       1
RNASET2.1       6.067031e-01 -0.2568638 0.274 0.265 1.000000e+00       1
TNFSF4.1        6.097072e-01 -0.3675163 0.123 0.197 1.000000e+00       1
TTC21A.1        6.097756e-01 -0.6690365 0.039 0.138 1.000000e+00       1
RASAL2          6.110066e-01 -0.4079961 0.281 0.304 1.000000e+00       1
ZNF639.1        6.116910e-01  0.4033049 0.323 0.336 1.000000e+00       1
SDC4.1          6.151864e-01  0.4835586 0.277 0.217 1.000000e+00       1
CXADR.1         6.152544e-01 -0.7467654 0.095 0.188 1.000000e+00       1
TIFA.1          6.153922e-01  0.3165102 0.140 0.121 1.000000e+00       1
NKX2-5.1        6.173153e-01 -0.4075822 0.147 0.192 1.000000e+00       1
SNX18           6.191722e-01  0.2752409 0.312 0.307 1.000000e+00       1
RBMX.1          6.210315e-01  0.4378572 0.361 0.373 1.000000e+00       1
ULBP2.1         6.234452e-01 -0.5168541 0.144 0.232 1.000000e+00       1
TMC6.1          6.239974e-01 -0.7858876 0.056 0.139 1.000000e+00       1
IDE             6.249646e-01  0.2823764 0.263 0.273 1.000000e+00       1
MIR193BHG       6.252410e-01  0.3049461 0.281 0.319 1.000000e+00       1
KANSL1L.1       6.262089e-01  0.4299409 0.221 0.232 1.000000e+00       1
AC007938.3.1    6.264163e-01 -0.2740697 0.147 0.215 1.000000e+00       1
MCTP1.1         6.278692e-01 -0.3952960 0.123 0.169 1.000000e+00       1
ABL2.1          6.288392e-01 -0.4717720 0.347 0.373 1.000000e+00       1
SVIL.1          6.319600e-01  0.5257766 0.277 0.270 1.000000e+00       1
ZNF850.1        6.323767e-01 -0.6275078 0.130 0.224 1.000000e+00       1
DOK1            6.336968e-01 -0.2647653 0.221 0.278 1.000000e+00       1
PPP1R15A        6.336968e-01  0.2936588 0.372 0.370 1.000000e+00       1
ETHE1.1         6.337663e-01 -0.3067541 0.418 0.440 1.000000e+00       1
ANGPTL7.1       6.361266e-01  0.4366109 0.200 0.252 1.000000e+00       1
EFCAB7          6.366189e-01 -0.3239220 0.123 0.133 1.000000e+00       1
TRPM7.1         6.368279e-01  0.3297912 0.305 0.305 1.000000e+00       1
TNFSF13B.1      6.381518e-01 -0.5175902 0.095 0.169 1.000000e+00       1
AP002387.2.1    6.390584e-01  0.3414311 0.225 0.234 1.000000e+00       1
TP53TG1         6.396863e-01  0.3315894 0.277 0.310 1.000000e+00       1
MLPH.1          6.415015e-01 -0.7853751 0.119 0.170 1.000000e+00       1
AC012645.1.1    6.428997e-01 -0.3893436 0.112 0.121 1.000000e+00       1
DPP10-AS1.1     6.452788e-01 -1.1376283 0.060 0.178 1.000000e+00       1
TRIM58.1        6.476617e-01 -0.4549793 0.130 0.194 1.000000e+00       1
ATP6V0E2.1      6.488551e-01 -0.3764509 0.130 0.206 1.000000e+00       1
LIMS2.1         6.491360e-01  0.3721211 0.411 0.422 1.000000e+00       1
GULP1.1         6.494168e-01  0.5073188 0.305 0.247 1.000000e+00       1
WFS1            6.507516e-01 -0.5207078 0.084 0.211 1.000000e+00       1
HOXB7           6.511734e-01  0.2793970 0.396 0.437 1.000000e+00       1
ITGA4.1         6.521578e-01 -0.3902248 0.049 0.169 1.000000e+00       1
COQ8A.1         6.522983e-01 -0.2677833 0.126 0.153 1.000000e+00       1
APOD.1          6.536352e-01 -0.4028131 0.151 0.197 1.000000e+00       1
IFI35           6.557489e-01  0.2686574 0.218 0.159 1.000000e+00       1
NEO1            6.569483e-01 -0.4721107 0.151 0.208 1.000000e+00       1
STARD4-AS1.1    6.580070e-01  0.3033412 0.323 0.393 1.000000e+00       1
TPP2.1          6.592076e-01  0.3227491 0.221 0.239 1.000000e+00       1
TBC1D1          6.595609e-01 -0.3615088 0.281 0.308 1.000000e+00       1
CALY.1          6.617529e-01 -0.5109930 0.095 0.202 1.000000e+00       1
CAVIN4.1        6.618943e-01 -1.0191004 0.140 0.237 1.000000e+00       1
NMI             6.649398e-01  0.3332403 0.214 0.169 1.000000e+00       1
ZNF331          6.655783e-01  0.3022295 0.200 0.217 1.000000e+00       1
TIGD1.1         6.697678e-01  0.3111994 0.161 0.172 1.000000e+00       1
DTNB            6.700523e-01 -0.2613225 0.112 0.146 1.000000e+00       1
WFDC3.1         6.706212e-01  0.4439009 0.154 0.101 1.000000e+00       1
GPX7.1          6.731126e-01 -0.3073696 0.256 0.269 1.000000e+00       1
ANKRD44         6.760358e-01 -0.3476347 0.193 0.247 1.000000e+00       1
AC069185.1.1    6.792497e-01 -0.4776887 0.119 0.151 1.000000e+00       1
TUBB2B          6.806082e-01  0.3800047 0.253 0.211 1.000000e+00       1
FKRP.1          6.818966e-01  0.4320519 0.218 0.186 1.000000e+00       1
PPM1F.1         6.831857e-01 -0.2851213 0.168 0.234 1.000000e+00       1
ITGB1BP2.1      6.839022e-01 -0.4971855 0.042 0.141 1.000000e+00       1
PTPRK           6.844041e-01 -0.5904704 0.165 0.245 1.000000e+00       1
SZT2.1          6.845475e-01  0.4177608 0.204 0.228 1.000000e+00       1
EHD3.1          6.869150e-01 -0.3651066 0.144 0.202 1.000000e+00       1
TSPAN18.1       6.888542e-01 -0.3067582 0.112 0.148 1.000000e+00       1
AC010255.1.1    6.888544e-01  0.6580446 0.165 0.132 1.000000e+00       1
AC097478.1.1    6.892137e-01 -0.7989413 0.081 0.179 1.000000e+00       1
SMCR8.1         6.904359e-01 -0.3324632 0.182 0.225 1.000000e+00       1
TSPAN12.1       6.904361e-01 -0.4490389 0.109 0.120 1.000000e+00       1
PLD3.1          6.915874e-01  0.3459098 0.456 0.458 1.000000e+00       1
AP000866.2.1    6.920192e-01  0.6172910 0.147 0.136 1.000000e+00       1
AL138724.1      6.954058e-01  0.3655270 0.112 0.121 1.000000e+00       1
YJU2            6.955498e-01 -0.4449688 0.168 0.193 1.000000e+00       1
TOR1B           6.980762e-01 -0.2801022 0.186 0.259 1.000000e+00       1
AFAP1L2.1       6.989430e-01 -0.6686906 0.154 0.279 1.000000e+00       1
AL133338.1.1    7.015457e-01  0.2907787 0.137 0.146 1.000000e+00       1
MAPK12          7.028487e-01 -0.3364995 0.281 0.278 1.000000e+00       1
NEK8.1          7.032832e-01 -0.4633993 0.196 0.191 1.000000e+00       1
ITGA2           7.048044e-01 -0.5738752 0.256 0.300 1.000000e+00       1
AL731569.1.1    7.053119e-01  0.4213175 0.123 0.102 1.000000e+00       1
FAM196A         7.055294e-01 -0.2883163 0.123 0.224 1.000000e+00       1
RWDD3           7.106107e-01  0.3606671 0.126 0.121 1.000000e+00       1
LAMA2           7.117015e-01 -0.5133110 0.154 0.255 1.000000e+00       1
DNAJB4          7.130110e-01 -0.3038912 0.488 0.450 1.000000e+00       1
LIG3            7.131565e-01  0.3503146 0.232 0.247 1.000000e+00       1
CHST3.1         7.132292e-01  0.4939913 0.151 0.183 1.000000e+00       1
PANX2           7.184762e-01 -0.2616583 0.161 0.208 1.000000e+00       1
HDAC4.1         7.212506e-01  0.4242332 0.179 0.178 1.000000e+00       1
MET.1           7.213967e-01  0.6233145 0.154 0.157 1.000000e+00       1
DOHH.1          7.227857e-01 -0.2936941 0.165 0.252 1.000000e+00       1
ENPP2.1         7.262981e-01  0.6181957 0.225 0.159 1.000000e+00       1
WDR4.1          7.275434e-01 -0.4371231 0.130 0.142 1.000000e+00       1
IPMK.1          7.278363e-01 -0.4382906 0.116 0.171 1.000000e+00       1
C6orf62.1       7.282032e-01  0.4245156 0.432 0.443 1.000000e+00       1
IGFL2.1         7.289085e-01 -0.3705134 0.330 0.356 1.000000e+00       1
ARG2.1          7.345899e-01  0.4116923 0.147 0.126 1.000000e+00       1
RBM47.1         7.381951e-01  0.4645587 0.182 0.120 1.000000e+00       1
KPNA2.1         7.386369e-01  0.4508725 0.347 0.382 1.000000e+00       1
SLC48A1         7.425437e-01 -0.4174056 0.182 0.271 1.000000e+00       1
NSG1.1          7.438720e-01 -0.4063031 0.179 0.263 1.000000e+00       1
AC112220.3.1    7.446840e-01 -0.5351932 0.116 0.167 1.000000e+00       1
PEMT            7.513396e-01  0.3705933 0.312 0.313 1.000000e+00       1
URB2.1          7.520799e-01 -0.2756227 0.105 0.182 1.000000e+00       1
CAMK4.1         7.525244e-01 -0.3588620 0.102 0.171 1.000000e+00       1
ANKRD33B        7.531174e-01  0.4036857 0.133 0.160 1.000000e+00       1
APOBEC3F        7.566770e-01  0.2550817 0.147 0.174 1.000000e+00       1
MTHFD2.1        7.572711e-01  0.2807719 0.432 0.452 1.000000e+00       1
HELLS.1         7.573452e-01  0.4367211 0.158 0.155 1.000000e+00       1
C2orf49.1       7.573453e-01  0.3359130 0.309 0.355 1.000000e+00       1
TDO2            7.577104e-01 -0.6724758 0.225 0.342 1.000000e+00       1
NOTCH3.1        7.581619e-01 -0.3573174 0.095 0.187 1.000000e+00       1
EPS8L2.1        7.594991e-01 -0.3407949 0.200 0.258 1.000000e+00       1
PTPRN           7.608369e-01 -0.3138984 0.186 0.232 1.000000e+00       1
SFRP1.1         7.620253e-01 -0.4403252 0.249 0.298 1.000000e+00       1
SNX14           7.625473e-01 -0.2806270 0.316 0.356 1.000000e+00       1
PRKG1.1         7.626960e-01  0.3800634 0.158 0.220 1.000000e+00       1
TWNK            7.647055e-01 -0.3080206 0.070 0.151 1.000000e+00       1
DMKN.1          7.707434e-01 -0.2599518 0.154 0.178 1.000000e+00       1
HSPA8           7.719378e-01  0.3135898 0.446 0.466 1.000000e+00       1
FAM107B.1       7.728338e-01 -0.3283165 0.263 0.267 1.000000e+00       1
SLMAP           7.746270e-01 -0.3782498 0.270 0.313 1.000000e+00       1
LAMC2.1         7.767204e-01 -1.0777647 0.056 0.225 1.000000e+00       1
ARHGEF3.1       7.790401e-01 -0.6210594 0.147 0.245 1.000000e+00       1
MYO18A          7.842102e-01  0.3947641 0.172 0.221 1.000000e+00       1
ATXN1L          7.844353e-01 -0.2936504 0.165 0.244 1.000000e+00       1
PPP5D1.1        7.858607e-01  0.5467582 0.098 0.099 1.000000e+00       1
KDM1B           7.917199e-01 -0.2599342 0.161 0.158 1.000000e+00       1
TMSB10          7.923218e-01  0.2888804 0.467 0.458 1.000000e+00       1
GALNT12         7.928484e-01 -0.2574558 0.119 0.183 1.000000e+00       1
KIAA1328.1      7.929988e-01 -0.2911248 0.130 0.181 1.000000e+00       1
SLC22A23        7.943530e-01  0.2696349 0.119 0.130 1.000000e+00       1
DYNC1I1.1       7.951060e-01 -0.4419667 0.140 0.222 1.000000e+00       1
ZNF703          7.955577e-01 -0.5070903 0.218 0.328 1.000000e+00       1
AHNAK2.1        7.958589e-01  0.3054736 0.256 0.301 1.000000e+00       1
CCNB2           7.991739e-01 -0.2928007 0.109 0.159 1.000000e+00       1
FLYWCH1.1       8.002297e-01  0.3500165 0.263 0.234 1.000000e+00       1
MOB1B           8.011347e-01 -0.2761092 0.200 0.273 1.000000e+00       1
SSTR1.1         8.024929e-01 -0.9033271 0.077 0.229 1.000000e+00       1
TNFSF15.1       8.048332e-01 -0.3132654 0.077 0.195 1.000000e+00       1
ATAD2B.1        8.072509e-01  0.2759646 0.161 0.210 1.000000e+00       1
JAZF1           8.086117e-01 -0.3403655 0.291 0.314 1.000000e+00       1
TAS2R14.1       8.111078e-01 -0.2839644 0.098 0.159 1.000000e+00       1
DPF2            8.120161e-01  0.3247055 0.337 0.367 1.000000e+00       1
SRSF4           8.184564e-01  0.3237164 0.407 0.416 1.000000e+00       1
IDH3B.1         8.222505e-01  0.4727375 0.316 0.327 1.000000e+00       1
ATF3.1          8.241492e-01  0.4898093 0.379 0.373 1.000000e+00       1
LONRF1.1        8.252886e-01  0.4047030 0.186 0.150 1.000000e+00       1
LEF1.1          8.275693e-01 -0.5321103 0.204 0.263 1.000000e+00       1
WDSUB1          8.277974e-01  0.3049826 0.158 0.119 1.000000e+00       1
ABHD15          8.298511e-01  0.3223983 0.200 0.206 1.000000e+00       1
PLK2.1          8.322107e-01 -0.4196808 0.175 0.258 1.000000e+00       1
AC124283.1.1    8.327435e-01  0.4985442 0.211 0.160 1.000000e+00       1
PKN1.1          8.335815e-01  0.4172902 0.298 0.279 1.000000e+00       1
TACC2.1         8.344953e-01  0.4002421 0.189 0.155 1.000000e+00       1
PM20D2          8.366293e-01  0.3356127 0.193 0.181 1.000000e+00       1
GPNMB.1         8.367818e-01  0.3726531 0.284 0.288 1.000000e+00       1
SAMD4A          8.380779e-01 -0.3407365 0.347 0.385 1.000000e+00       1
HOXC6.1         8.395271e-01 -0.3029482 0.351 0.379 1.000000e+00       1
ATP6V0A1        8.399848e-01 -0.3638892 0.281 0.319 1.000000e+00       1
DAAM2.1         8.412057e-01 -0.6055865 0.133 0.251 1.000000e+00       1
NTF3.1          8.417398e-01 -0.7696009 0.098 0.205 1.000000e+00       1
SYNGR1.1        8.421980e-01  0.4329210 0.274 0.247 1.000000e+00       1
PITHD1.1        8.439541e-01  0.4582524 0.400 0.390 1.000000e+00       1
PHF13           8.443360e-01  0.2800069 0.232 0.199 1.000000e+00       1
GCLM.1          8.449471e-01  0.3752274 0.404 0.369 1.000000e+00       1
ACSS2           8.466280e-01 -0.3267168 0.179 0.224 1.000000e+00       1
LINC01638.1     8.481567e-01  0.3452337 0.218 0.188 1.000000e+00       1
SETD4           8.498390e-01 -0.3745117 0.182 0.197 1.000000e+00       1
FANCB.1         8.543538e-01 -0.3542126 0.088 0.173 1.000000e+00       1
SHMT2.1         8.561152e-01  0.3152258 0.396 0.414 1.000000e+00       1
FIG4.1          8.581070e-01 -0.5214591 0.095 0.143 1.000000e+00       1
KIAA1217.1      8.581071e-01  0.3779874 0.340 0.352 1.000000e+00       1
PLCB4.1         8.585670e-01 -0.5654564 0.102 0.199 1.000000e+00       1
FANCF.1         8.597166e-01  0.2567119 0.168 0.172 1.000000e+00       1
MMP7            8.617869e-01  0.3122876 0.144 0.108 1.000000e+00       1
EFNA5           8.618637e-01  0.3239851 0.105 0.125 1.000000e+00       1
NXPH4.1         8.633978e-01 -0.3726866 0.144 0.245 1.000000e+00       1
MYO9B.1         8.643186e-01  0.3644241 0.344 0.318 1.000000e+00       1
MTMR3           8.676962e-01 -0.3522511 0.235 0.272 1.000000e+00       1
FAM69B.1        8.692323e-01 -0.6551649 0.091 0.181 1.000000e+00       1
TINAGL1         8.709224e-01 -0.2727977 0.042 0.160 1.000000e+00       1
ST3GAL6.1       8.728441e-01  0.2961431 0.246 0.239 1.000000e+00       1
CYP26A1.1       8.809222e-01 -0.9612945 0.039 0.183 1.000000e+00       1
AC021242.3.1    8.824625e-01 -0.3196865 0.095 0.150 1.000000e+00       1
TAOK1           8.840802e-01 -0.2819317 0.418 0.454 1.000000e+00       1
EPHA2           8.845424e-01 -0.2668672 0.179 0.240 1.000000e+00       1
VPS33B.1        8.850818e-01 -0.6184107 0.105 0.131 1.000000e+00       1
AL358472.4      8.892444e-01 -0.4714592 0.084 0.153 1.000000e+00       1
LINC01943       8.931014e-01  0.2552151 0.088 0.110 1.000000e+00       1
ARHGEF2         8.941046e-01  0.2796613 0.196 0.183 1.000000e+00       1
TCTA            8.941818e-01 -0.3356808 0.242 0.295 1.000000e+00       1
RASSF8.1        8.968837e-01 -0.5015085 0.432 0.432 1.000000e+00       1
SGF29           8.992777e-01 -0.4587856 0.126 0.135 1.000000e+00       1
SOD3            8.998958e-01 -0.4722689 0.168 0.258 1.000000e+00       1
PCSK6.1         8.999730e-01  0.5385178 0.193 0.212 1.000000e+00       1
CALM3           9.005138e-01  0.3696113 0.358 0.398 1.000000e+00       1
TVP23B          9.012092e-01  0.3747687 0.316 0.330 1.000000e+00       1
JAG1.1          9.017501e-01 -0.6487895 0.102 0.210 1.000000e+00       1
RPL39L          9.043006e-01  0.2512053 0.225 0.207 1.000000e+00       1
PLXDC2          9.066202e-01  0.3049848 0.319 0.369 1.000000e+00       1
GCHFR           9.080123e-01  0.3667087 0.309 0.297 1.000000e+00       1
IL32            9.085537e-01  0.3454767 0.179 0.157 1.000000e+00       1
NOG.1           9.144350e-01 -0.3886900 0.242 0.341 1.000000e+00       1
SNX30           9.189266e-01 -0.3626354 0.204 0.284 1.000000e+00       1
OLFML2A         9.211733e-01 -0.3795130 0.112 0.134 1.000000e+00       1
C21orf2.1       9.212508e-01  0.4423705 0.284 0.277 1.000000e+00       1
MT-ND4.1        9.217157e-01  0.3405151 0.414 0.427 1.000000e+00       1
MYL10.1         9.227231e-01  0.2987102 0.316 0.333 1.000000e+00       1
SYN1.1          9.232656e-01  0.4444916 0.186 0.162 1.000000e+00       1
IFNAR2.1        9.241958e-01 -0.5335232 0.168 0.263 1.000000e+00       1
AC112721.2.1    9.285376e-01 -0.4442481 0.130 0.194 1.000000e+00       1
CCNDBP1         9.303991e-01  0.3637732 0.361 0.411 1.000000e+00       1
SLC4A1AP.1      9.335024e-01 -0.3978771 0.295 0.294 1.000000e+00       1
PANK1.1         9.379263e-01  0.2981535 0.175 0.224 1.000000e+00       1
BCL10.1         9.397119e-01 -0.3199132 0.323 0.345 1.000000e+00       1
NKRF.1          9.401777e-01 -0.4489662 0.105 0.163 1.000000e+00       1
NPIPA1          9.448375e-01 -0.2604389 0.161 0.182 1.000000e+00       1
SPOCD1.1        9.457697e-01 -0.4225287 0.109 0.217 1.000000e+00       1
AC007878.1.1    9.463912e-01 -0.6851797 0.060 0.144 1.000000e+00       1
SSPO            9.482559e-01 -0.3341117 0.081 0.154 1.000000e+00       1
DHRS2.1         9.518307e-01 -0.7625380 0.116 0.263 1.000000e+00       1
C1QTNF3.1       9.533075e-01 -0.5834805 0.088 0.220 1.000000e+00       1
AQP3.1          9.571169e-01 -0.4950616 0.144 0.158 1.000000e+00       1
INTS6.1         9.582833e-01  0.3155587 0.386 0.395 1.000000e+00       1
CNRIP1.1        9.585165e-01  0.5495820 0.207 0.159 1.000000e+00       1
PROSER2.1       9.594497e-01  0.3927173 0.263 0.267 1.000000e+00       1
RTKN2.1         9.599940e-01 -0.8869488 0.063 0.176 1.000000e+00       1
GFOD1.1         9.631050e-01 -0.4860187 0.119 0.158 1.000000e+00       1
HOXA3           9.640384e-01 -0.2594962 0.165 0.222 1.000000e+00       1
WFDC1.1         9.647385e-01 -0.4432212 0.112 0.192 1.000000e+00       1
GGH.1           9.664500e-01  0.5269668 0.302 0.314 1.000000e+00       1
LPXN            9.665278e-01  0.2631786 0.137 0.122 1.000000e+00       1
PLN.1           9.703401e-01 -0.6169590 0.070 0.216 1.000000e+00       1
PDZRN4.1        9.799129e-01 -0.6381938 0.123 0.183 1.000000e+00       1
FAM160B2        9.812362e-01 -0.5401064 0.242 0.285 1.000000e+00       1
CDYL2.1         9.818589e-01  0.4880646 0.112 0.150 1.000000e+00       1
MRPL35.1        9.820925e-01 -0.3616534 0.253 0.309 1.000000e+00       1
PFDN1           9.841165e-01  0.3039344 0.421 0.463 1.000000e+00       1
PTGFRN.1        9.841944e-01 -0.5818110 0.098 0.204 1.000000e+00       1
AAED1.1         9.854400e-01  0.3445152 0.305 0.309 1.000000e+00       1
ESD             9.859850e-01  0.2897277 0.456 0.435 1.000000e+00       1
LRRC45.1        9.895663e-01 -0.4018885 0.102 0.124 1.000000e+00       1
RFC3            9.918243e-01 -0.3408157 0.084 0.142 1.000000e+00       1
CMBL            9.922915e-01  0.2958798 0.267 0.267 1.000000e+00       1
FAM227A.1       9.930701e-01  0.2596989 0.144 0.183 1.000000e+00       1
DSP             9.948609e-01  0.2732634 0.439 0.437 1.000000e+00       1
CYBRD1          9.993771e-01 -0.3336253 0.411 0.441 1.000000e+00       1
SPARC.2         6.663816e-39  1.9143789 1.000 0.407 3.027371e-35       2
COL5A1.2        1.935304e-36  2.2907267 0.973 0.386 8.792088e-33       2
FSTL1.2         1.395713e-30  1.9421845 0.933 0.407 6.340726e-27       2
FN1.2           1.240166e-29  1.6985001 0.973 0.419 5.634076e-26       2
COL3A1.2        5.814391e-28  1.7187206 0.960 0.410 2.641478e-24       2
COL5A2.1        1.811731e-27  1.4822892 0.947 0.417 8.230694e-24       2
RPL10.2         9.850685e-25 -1.4349796 0.040 0.449 4.475166e-21       2
RPL7A           2.848586e-24 -1.1560196 0.040 0.441 1.294112e-20       2
RPS4X.1         1.170720e-23 -1.3775564 0.027 0.477 5.318579e-20       2
COL4A1.1        1.373469e-23  1.3679219 0.933 0.444 6.239669e-20       2
FBN1.1          1.446822e-23  1.5008571 0.893 0.434 6.572913e-20       2
CANX.1          1.738456e-23  1.2931400 0.920 0.431 7.897806e-20       2
RPS3A           2.094951e-22 -1.2621333 0.013 0.465 9.517362e-19       2
COL1A1.2        8.764590e-22  1.5236364 0.853 0.410 3.981753e-18       2
COL11A1.2       9.476400e-22  1.5454720 0.853 0.418 4.305129e-18       2
RPS8            2.159540e-20 -1.0500773 0.107 0.482 9.810790e-17       2
FTL.1           2.431442e-20 -1.7722378 0.080 0.478 1.104604e-16       2
RPL10A          3.904916e-20 -1.0360627 0.040 0.457 1.774003e-16       2
RPN1.2          5.740245e-20  1.5196818 0.840 0.394 2.607793e-16       2
RPS27.2         2.831114e-19 -1.1390588 0.080 0.458 1.286175e-15       2
LRP1.2          4.209138e-19  1.4182592 0.840 0.407 1.912211e-15       2
SURF4.1         5.385199e-18  0.9145342 0.813 0.424 2.446496e-14       2
COL18A1.2       1.369911e-17  1.5620956 0.813 0.377 6.223505e-14       2
FTH1.2          9.066243e-17 -2.2988424 0.093 0.473 4.118794e-13       2
GLS.1           1.192886e-16  1.1856514 0.853 0.406 5.419283e-13       2
TIMP2.2         1.570242e-16  1.5680141 0.853 0.426 7.133608e-13       2
FBLN5.2         1.597537e-16  1.2259761 0.813 0.425 7.257612e-13       2
SH2B2           2.136871e-16  0.3538185 0.493 0.117 9.707803e-13       2
IL6ST.1         5.954989e-16  0.9506630 0.853 0.394 2.705351e-12       2
MT-ND3          1.097729e-15  0.8263147 0.827 0.399 4.986983e-12       2
AC133550.2      1.692328e-15  0.8659275 0.533 0.090 7.688246e-12       2
RPS3            3.260333e-15 -0.9329874 0.133 0.454 1.481169e-11       2
PLOD2.1         3.510837e-15  1.4205897 0.827 0.426 1.594973e-11       2
RPL3            3.569485e-15 -1.0685732 0.067 0.420 1.621617e-11       2
IVNS1ABP.1      4.306237e-15  1.2165002 0.800 0.391 1.956323e-11       2
RPL23.1         4.419583e-15 -1.0156393 0.067 0.444 2.007816e-11       2
MRC2.1          1.314228e-14  0.8678823 0.733 0.407 5.970536e-11       2
PLXNA1.1        1.879506e-14  1.2120268 0.747 0.311 8.538594e-11       2
ITGA11.1        2.094124e-14  1.3705044 0.800 0.418 9.513607e-11       2
LOX.1           2.277328e-14  1.1177935 0.827 0.414 1.034590e-10       2
FLT1.2          2.970988e-14  1.5131288 0.813 0.431 1.349720e-10       2
AL358472.4.1    3.149387e-14  0.8187458 0.427 0.117 1.430766e-10       2
MATN2.1         5.373004e-14  0.6282175 0.840 0.407 2.440956e-10       2
ITGAV.1         5.828395e-14  0.8439224 0.733 0.403 2.647840e-10       2
CCND2.2         6.430062e-14  1.0866267 0.800 0.403 2.921177e-10       2
GPX4.2          6.909951e-14 -1.0019335 0.107 0.463 3.139191e-10       2
LAMC1.1         1.346239e-13  0.7226975 0.747 0.392 6.115963e-10       2
RPL18A          1.525871e-13 -0.6878450 0.147 0.448 6.932031e-10       2
CKAP4.1         4.125928e-13  0.7078631 0.773 0.448 1.874409e-09       2
GALNT10.2       4.944218e-13  1.4622299 0.760 0.423 2.246158e-09       2
TPT1.2          7.405365e-13 -0.7044027 0.133 0.483 3.364258e-09       2
RPL13A          1.391492e-12 -0.7609759 0.107 0.447 6.321548e-09       2
ATP6V0E1        1.604316e-12 -0.9124353 0.133 0.484 7.288406e-09       2
C1QL1.2         1.738382e-12  0.2522913 0.467 0.141 7.897469e-09       2
FGFR2.1         3.354341e-12  1.1316669 0.760 0.348 1.523877e-08       2
YAP1.1          3.760938e-12  1.2961165 0.813 0.434 1.708594e-08       2
SEC16A.1        4.639344e-12  1.3912399 0.720 0.353 2.107654e-08       2
VSIG10L.2       5.738632e-12  0.7749136 0.307 0.088 2.607060e-08       2
GALNT2.1        5.928508e-12  1.1263783 0.747 0.450 2.693321e-08       2
HACD2.1         6.791991e-12  1.3724161 0.773 0.386 3.085601e-08       2
CTSK.2          8.428096e-12  0.3425827 0.293 0.098 3.828884e-08       2
UBOX5.1         8.590791e-12  0.6286964 0.467 0.120 3.902796e-08       2
SLC38A10.1      1.031268e-11  1.1171800 0.747 0.444 4.685049e-08       2
GPC4.2          1.073662e-11  1.1156708 0.720 0.363 4.877646e-08       2
EIF1.2          1.115577e-11 -0.9511568 0.147 0.466 5.068065e-08       2
MTBP            1.605187e-11  0.8311125 0.440 0.130 7.292365e-08       2
COL6A3.1        1.684048e-11  1.1246164 0.760 0.371 7.650631e-08       2
TNFRSF19        1.866657e-11  0.7021118 0.440 0.117 8.480225e-08       2
CRIM1.1         3.758705e-11  1.1199977 0.720 0.373 1.707580e-07       2
FLNA.1          3.955397e-11  0.8660360 0.747 0.417 1.796937e-07       2
GALNT5.2        4.142889e-11  0.8782637 0.507 0.131 1.882114e-07       2
PLD3.2          4.704605e-11  0.6838562 0.787 0.435 2.137302e-07       2
COL1A2.1        5.372908e-11  1.0062156 0.733 0.426 2.440912e-07       2
FBN2.1          5.936340e-11  1.2404081 0.733 0.402 2.696879e-07       2
BCAT1.1         5.984350e-11  0.7284132 0.733 0.442 2.718690e-07       2
ISCU.2          6.703833e-11 -1.0445918 0.160 0.470 3.045551e-07       2
GFPT1.2         6.843443e-11  0.7581876 0.773 0.451 3.108976e-07       2
AL731563.3.1    7.139386e-11  0.8730169 0.467 0.125 3.243423e-07       2
HSF4            7.447756e-11  0.6954781 0.413 0.109 3.383516e-07       2
EDIL3.1         8.948900e-11  0.9506742 0.693 0.416 4.065485e-07       2
BRI3.2          9.312390e-11 -1.0005180 0.160 0.459 4.230619e-07       2
ADAM10.2        1.097819e-10  1.5831139 0.720 0.428 4.987391e-07       2
RPS23           1.121704e-10 -0.5181933 0.160 0.453 5.095902e-07       2
XYLT1.1         1.212773e-10  0.7292025 0.773 0.421 5.509628e-07       2
WDR26.1         1.557578e-10  0.9470255 0.747 0.410 7.076077e-07       2
ADGRA2.1        2.004288e-10  1.1073743 0.707 0.410 9.105479e-07       2
PRSS35.2        2.081528e-10  0.7249865 0.720 0.344 9.456380e-07       2
VCAN            2.133499e-10  0.7596750 0.787 0.418 9.692487e-07       2
MAP3K7.1        3.175431e-10  1.0772783 0.667 0.340 1.442598e-06       2
P4HB.1          3.289732e-10  1.2108751 0.733 0.430 1.494525e-06       2
SERPINB2        4.672551e-10 -0.2518608 0.320 0.099 2.122740e-06       2
PPFIBP2         4.703366e-10  0.3173543 0.373 0.167 2.136739e-06       2
SCAMP5          4.791717e-10  0.4159569 0.373 0.108 2.176877e-06       2
PRDX2.2         5.357034e-10 -0.7268654 0.147 0.458 2.433701e-06       2
IRAK4           5.415811e-10  0.2883359 0.480 0.248 2.460403e-06       2
ZNF473.1        5.681912e-10  0.9192146 0.440 0.122 2.581293e-06       2
GTF2A2.1        5.961190e-10 -0.8433786 0.107 0.459 2.708168e-06       2
DSTN            6.799355e-10 -0.7847739 0.187 0.465 3.088947e-06       2
ALG10B.1        8.379556e-10  0.9238220 0.507 0.212 3.806832e-06       2
SLC39A6.1       8.720997e-10  1.2175901 0.720 0.443 3.961949e-06       2
FAM227A.2       9.955997e-10  0.8436663 0.400 0.159 4.523009e-06       2
APP.1           1.014010e-09  0.6392775 0.787 0.448 4.606646e-06       2
PLOD1.1         1.201273e-09  1.1080653 0.720 0.415 5.457385e-06       2
SFMBT2.2        1.229795e-09  0.5410242 0.453 0.167 5.586958e-06       2
CDH11.1         1.272696e-09  0.8668768 0.693 0.407 5.781856e-06       2
SLC7A7          1.496690e-09  0.3555563 0.427 0.146 6.799465e-06       2
COL12A1.2       1.530226e-09  1.1185200 0.667 0.365 6.951817e-06       2
MAP3K4.1        1.757079e-09  0.7468297 0.453 0.233 7.982408e-06       2
IQGAP1.1        1.809996e-09  0.8216157 0.747 0.424 8.222811e-06       2
TIMP3.2         1.819485e-09  0.5518442 0.733 0.381 8.265922e-06       2
GRIK2           1.838958e-09  0.6212652 0.440 0.213 8.354385e-06       2
ZNF692.1        1.922470e-09  1.1357211 0.480 0.142 8.733783e-06       2
SERPINH1        2.307563e-09  0.7414559 0.653 0.407 1.048326e-05       2
MAN2B1.1        2.642525e-09  0.7817400 0.413 0.124 1.200499e-05       2
HDAC4.2         2.664750e-09  1.0283101 0.413 0.162 1.210596e-05       2
C1RL.2          2.744174e-09  0.8530831 0.373 0.134 1.246678e-05       2
LMAN1           2.808169e-09  0.6784598 0.760 0.455 1.275751e-05       2
EFCAB7.1        2.909901e-09  0.3773418 0.400 0.113 1.321968e-05       2
SVEP1.1         2.937429e-09  1.0999252 0.533 0.230 1.334474e-05       2
NRTN.2          3.137325e-09 -0.5706250 0.387 0.141 1.425287e-05       2
TEAD1           3.453292e-09  0.6826567 0.760 0.464 1.568831e-05       2
AL133215.2      3.607601e-09  0.2872029 0.373 0.128 1.638933e-05       2
TTC21A.2        3.796139e-09  0.3309434 0.253 0.105 1.724586e-05       2
TMSB4X          3.915935e-09 -1.6669161 0.133 0.473 1.779009e-05       2
TESK2.2         4.035916e-09  0.3740279 0.440 0.124 1.833516e-05       2
FANCF.2         4.665255e-09  0.7095676 0.387 0.157 2.119425e-05       2
SLC9A7.1        5.639570e-09  0.7502731 0.507 0.217 2.562057e-05       2
ALCAM.2         5.709561e-09  0.4923430 0.747 0.442 2.593854e-05       2
EFNB2.1         5.744856e-09  0.8484025 0.653 0.367 2.609888e-05       2
ABCD3.2         5.973476e-09  0.7699364 0.680 0.379 2.713750e-05       2
COL4A2.1        5.979614e-09  0.5065263 0.787 0.467 2.716539e-05       2
AXIN2           6.249247e-09  0.4066419 0.507 0.237 2.839033e-05       2
PLEC.1          6.564212e-09  0.9175378 0.733 0.423 2.982122e-05       2
FLNC            6.768848e-09  0.3362450 0.547 0.256 3.075088e-05       2
TTYH3.1         7.160248e-09  0.8032834 0.680 0.350 3.252900e-05       2
PER2.2          7.174731e-09  0.6044147 0.400 0.176 3.259480e-05       2
AC098487.1.2    7.337732e-09  0.6391733 0.413 0.173 3.333532e-05       2
PLS3.2          7.428241e-09  0.2977154 0.707 0.401 3.374650e-05       2
LEPR            7.435789e-09  0.4041929 0.400 0.135 3.378079e-05       2
EME2            7.504420e-09  0.8675500 0.387 0.168 3.409258e-05       2
SMC6.1          8.689086e-09  0.4927661 0.507 0.268 3.947452e-05       2
AC020571.1.2    9.376045e-09  0.6051212 0.280 0.140 4.259537e-05       2
RUNX2.1         1.044865e-08  1.1892691 0.520 0.219 4.746823e-05       2
PTMA.1          1.163996e-08 -1.1284901 0.160 0.448 5.288033e-05       2
FURIN           1.174597e-08  0.4535027 0.600 0.280 5.336194e-05       2
AC005332.6.1    1.273043e-08  0.4798504 0.413 0.121 5.783436e-05       2
DEPP1.2         1.385042e-08  0.3908569 0.387 0.155 6.292245e-05       2
KRT10.2         1.399000e-08 -0.9048695 0.200 0.470 6.355655e-05       2
AL662884.4.1    1.491589e-08  0.3704188 0.213 0.102 6.776289e-05       2
TRIM59.1        1.535412e-08  0.4901248 0.400 0.158 6.975377e-05       2
LRRC32.1        1.623816e-08  1.0421589 0.653 0.391 7.376997e-05       2
WNT3            1.630299e-08  0.2733322 0.240 0.125 7.406451e-05       2
CPM.1           1.683181e-08 -0.3343658 0.227 0.131 7.646691e-05       2
INTS6-AS1       1.874154e-08  0.5154719 0.453 0.144 8.514280e-05       2
H2AFZ.1         1.891061e-08 -1.1093862 0.147 0.438 8.591088e-05       2
EIF3E           1.902371e-08 -0.8942051 0.120 0.436 8.642472e-05       2
CHCHD2          1.957952e-08 -0.5198141 0.133 0.410 8.894977e-05       2
ADAM23.2        2.005139e-08  0.4867789 0.493 0.177 9.109344e-05       2
AC010255.1.2    2.138544e-08 -0.4356883 0.253 0.132 9.715407e-05       2
UNKL.2          2.144910e-08  0.4843260 0.427 0.196 9.744325e-05       2
UBAP1L.1        2.198626e-08  0.8779317 0.293 0.119 9.988358e-05       2
RNF128.2        2.258064e-08  0.5379637 0.400 0.152 1.025839e-04       2
CLIC4.1         2.532007e-08  1.1206703 0.680 0.413 1.150291e-04       2
ADNP-AS1.1      2.536995e-08  0.3618685 0.280 0.108 1.152557e-04       2
GGACT.1         2.541925e-08  0.8209524 0.373 0.149 1.154796e-04       2
ADA             2.620789e-08  0.4950928 0.347 0.185 1.190625e-04       2
CERCAM.1        2.628540e-08  0.7609849 0.707 0.430 1.194146e-04       2
MCC             2.758298e-08  0.6253640 0.613 0.324 1.253095e-04       2
ACSL1.2         2.986543e-08  0.7769115 0.680 0.379 1.356786e-04       2
LOXL3.1         3.121152e-08  0.8554599 0.507 0.186 1.417939e-04       2
ITGB5.1         3.158039e-08  0.5985501 0.733 0.434 1.434697e-04       2
NDRG1.1         3.182859e-08  0.9104899 0.547 0.270 1.445973e-04       2
NDST1.1         3.368558e-08  0.8684210 0.707 0.437 1.530336e-04       2
SESN2.2         3.421603e-08  0.5118401 0.467 0.256 1.554434e-04       2
PCCB.1          3.458506e-08  0.9816878 0.467 0.200 1.571199e-04       2
LZTS1.2         3.735450e-08  0.7604693 0.640 0.366 1.697015e-04       2
MT-CO1.2        3.838664e-08  0.4050466 0.720 0.410 1.743905e-04       2
SEMA3C.1        3.948473e-08  1.1090275 0.680 0.435 1.793791e-04       2
WNT5A.1         4.719506e-08  0.7873285 0.453 0.205 2.144071e-04       2
FKBP10.2        4.816380e-08  1.2318132 0.653 0.426 2.188081e-04       2
CLCN5           4.915080e-08  0.7240465 0.587 0.300 2.232921e-04       2
OSR1.2          5.001226e-08  0.3195515 0.320 0.148 2.272057e-04       2
INHBA.2         6.002901e-08  0.5058923 0.707 0.375 2.727118e-04       2
PIGQ.1          6.027866e-08  1.0195587 0.453 0.189 2.738459e-04       2
DOK5.1          6.080146e-08  0.5905941 0.467 0.230 2.762210e-04       2
RDH10.2         6.335935e-08  0.5936554 0.720 0.383 2.878415e-04       2
AC015813.1      6.348206e-08  0.6487867 0.413 0.183 2.883990e-04       2
CHIC2.1         6.464794e-08 -0.9634572 0.187 0.467 2.936956e-04       2
CEP131.1        6.621338e-08  0.5405400 0.373 0.095 3.008074e-04       2
ZNF136.2        6.646739e-08  0.5959906 0.293 0.164 3.019614e-04       2
TTC3            6.899289e-08  0.7936032 0.707 0.428 3.134347e-04       2
FAM129B.1       7.039089e-08  0.7989208 0.520 0.265 3.197858e-04       2
ANKRD33B.1      7.553809e-08 -0.7221230 0.280 0.145 3.431695e-04       2
HINT1.2         9.344611e-08 -0.8604211 0.187 0.491 4.245257e-04       2
ACTC1.2         9.495288e-08  0.4562822 0.453 0.231 4.313709e-04       2
ZNF496.2        9.622537e-08  0.3062328 0.333 0.181 4.371519e-04       2
LAMB1.1         1.028049e-07  1.0543118 0.720 0.410 4.670424e-04       2
CDH2.1          1.081745e-07  1.0822299 0.720 0.444 4.914366e-04       2
LINC00626.2     1.110640e-07  0.3072510 0.387 0.176 5.045639e-04       2
H3F3B.2         1.119039e-07 -0.7072625 0.240 0.468 5.083796e-04       2
USP20.1         1.146697e-07  0.4939564 0.387 0.124 5.209446e-04       2
PCNX4           1.147823e-07  0.4635149 0.640 0.396 5.214561e-04       2
ECE1.1          1.224675e-07  1.1868601 0.613 0.312 5.563700e-04       2
ZFPM2.1         1.225824e-07  0.9855564 0.573 0.281 5.568920e-04       2
MT-ND4.2        1.367787e-07  0.4353659 0.747 0.402 6.213858e-04       2
AL117379.1.1    1.410619e-07  1.0091444 0.400 0.111 6.408444e-04       2
NUAK2           1.457462e-07  0.3970560 0.253 0.079 6.621252e-04       2
PIK3R3          1.482144e-07  0.7295015 0.400 0.189 6.733380e-04       2
DIS3L           1.522747e-07  0.6489433 0.400 0.195 6.917840e-04       2
PLXNB2.1        1.529861e-07  1.0011076 0.640 0.373 6.950158e-04       2
NPC2            1.532714e-07 -0.8780910 0.227 0.453 6.963119e-04       2
STIL            1.634336e-07  0.4052222 0.453 0.181 7.424787e-04       2
ITGA1.2         1.637380e-07  0.6360886 0.707 0.419 7.438617e-04       2
RAN             1.668092e-07 -0.7458351 0.187 0.491 7.578143e-04       2
OGA.1           1.696207e-07  0.9633474 0.627 0.348 7.705868e-04       2
ARMCX6          1.922167e-07  0.4373147 0.387 0.111 8.732407e-04       2
LOXL1           1.961652e-07  0.4313547 0.693 0.445 8.911783e-04       2
HSPA5           1.985344e-07  0.8848524 0.693 0.439 9.019418e-04       2
CEMIP           2.013021e-07  0.3456023 0.680 0.309 9.145154e-04       2
IGFBP3.2        2.130747e-07  0.5022828 0.680 0.384 9.679981e-04       2
PPP1R13L.1      2.149217e-07  0.7867777 0.480 0.234 9.763894e-04       2
SLC2A1.1        2.159139e-07  0.8290799 0.507 0.304 9.808968e-04       2
NEBL.1          2.213448e-07  0.6941078 0.387 0.123 1.005569e-03       2
ATP2A2.1        2.281674e-07  1.0514289 0.600 0.371 1.036565e-03       2
PLA2G6          2.315458e-07  0.3383135 0.267 0.142 1.051913e-03       2
LRRN2           2.395477e-07  0.4784403 0.347 0.125 1.088265e-03       2
NUAK1.2         2.417559e-07  0.3536027 0.680 0.424 1.098297e-03       2
AGRN.1          2.462307e-07  0.9638952 0.613 0.298 1.118626e-03       2
CEMIP2.1        2.487262e-07  0.9749822 0.640 0.426 1.129963e-03       2
KCND3.2         2.540216e-07  0.5583923 0.453 0.221 1.154020e-03       2
PDLIM2.1        2.558904e-07 -0.6572037 0.173 0.442 1.162510e-03       2
MMP14.1         2.568287e-07  1.1882996 0.587 0.294 1.166773e-03       2
ADAM12.1        2.594280e-07  1.0181513 0.667 0.417 1.178582e-03       2
PTPA.1          2.599022e-07  0.9296167 0.560 0.292 1.180736e-03       2
HEG1.1          2.785991e-07  1.0779792 0.600 0.375 1.265676e-03       2
KIRREL1.1       2.839896e-07  0.5956501 0.493 0.270 1.290165e-03       2
C9orf16.2       2.876397e-07 -0.5888487 0.187 0.465 1.306747e-03       2
KCTD9           2.985905e-07  0.3733825 0.427 0.223 1.356496e-03       2
RING1.1         3.018711e-07  0.7487105 0.560 0.332 1.371400e-03       2
FNDC3B.1        3.046316e-07  0.5798277 0.720 0.453 1.383941e-03       2
NCKAP1.1        3.261254e-07  0.8477168 0.733 0.475 1.481588e-03       2
FAM78A          3.456134e-07 -0.5763001 0.360 0.132 1.570122e-03       2
ZNF23.1         3.622785e-07  0.8545740 0.293 0.083 1.645831e-03       2
MT-CO3          3.692258e-07  0.4502702 0.667 0.424 1.677393e-03       2
CSRP2.2         3.780038e-07 -1.2305734 0.120 0.432 1.717271e-03       2
AL683813.1      3.848899e-07  0.2645350 0.240 0.124 1.748555e-03       2
SEMA5A.2        3.862838e-07  0.5952268 0.667 0.375 1.754887e-03       2
SMAD9           3.933177e-07  0.2805147 0.347 0.148 1.786842e-03       2
BSDC1.1         4.081290e-07  1.0866340 0.533 0.328 1.854130e-03       2
EEF1G           4.088645e-07  0.4491489 0.227 0.133 1.857471e-03       2
NAA25.1         4.331056e-07  0.4903618 0.427 0.237 1.967599e-03       2
MYL12B.2        4.493576e-07 -0.9704402 0.200 0.450 2.041432e-03       2
AC067750.1.1    4.505678e-07  0.3048067 0.173 0.155 2.046929e-03       2
FAM177A1.2      4.603793e-07 -1.0769655 0.173 0.454 2.091503e-03       2
TDO2.1          4.815829e-07  1.1099420 0.587 0.296 2.187831e-03       2
NORAD.2         5.008226e-07  0.5842664 0.720 0.470 2.275237e-03       2
EXTL1.2         5.098505e-07  0.8212288 0.453 0.144 2.316251e-03       2
LTN1            5.340959e-07  0.6558442 0.573 0.371 2.426398e-03       2
LTBP2           5.369590e-07  0.2712783 0.707 0.423 2.439405e-03       2
SLC38A2.1       5.584460e-07  0.5919229 0.693 0.449 2.537020e-03       2
TMX4.1          5.689890e-07  0.8491763 0.587 0.328 2.584917e-03       2
PPP2R1B         5.948697e-07  0.5569693 0.453 0.209 2.702493e-03       2
ACAP3           6.087724e-07  0.4809737 0.493 0.216 2.765653e-03       2
ZKSCAN7         6.180214e-07  0.3737298 0.387 0.160 2.807671e-03       2
SLC4A2.1        6.218832e-07  1.1268734 0.653 0.382 2.825215e-03       2
LENG8.2         6.358297e-07  0.6996620 0.693 0.440 2.888574e-03       2
CPNE8           6.587713e-07  0.3109514 0.293 0.125 2.992798e-03       2
FAM84B.1        6.801025e-07  0.9160225 0.667 0.388 3.089706e-03       2
SMAD6.1         6.904046e-07  0.9334067 0.467 0.220 3.136508e-03       2
CPD.2           7.127198e-07  0.8005128 0.653 0.444 3.237886e-03       2
TMEM144         7.305517e-07  0.2960688 0.347 0.115 3.318896e-03       2
PTPN13.1        7.422454e-07  1.1050792 0.520 0.287 3.372021e-03       2
MIGA2.1         8.112376e-07  1.1545441 0.413 0.166 3.685452e-03       2
AC002456.1      8.329293e-07 -0.5197999 0.293 0.124 3.783998e-03       2
RFC3.1          8.521514e-07  0.2585302 0.387 0.111 3.871324e-03       2
C6orf62.2       8.725751e-07  0.4753351 0.653 0.426 3.964109e-03       2
HNRNPU          8.787150e-07  0.4682241 0.680 0.457 3.992002e-03       2
PPM1F.2         8.802515e-07  1.1524040 0.387 0.207 3.998983e-03       2
TUFT1.1         8.989524e-07  0.7201353 0.520 0.259 4.083941e-03       2
SPTBN2.1        9.326080e-07  1.0844080 0.667 0.390 4.236838e-03       2
MEST.1          9.741772e-07  0.7504945 0.667 0.412 4.425687e-03       2
MBNL2.2         1.001836e-06  0.2848808 0.720 0.458 4.551339e-03       2
SPPL2B.1        1.078807e-06  0.7225528 0.520 0.290 4.901020e-03       2
ATAD2B.2        1.130629e-06  0.5443626 0.320 0.191 5.136449e-03       2
CYBRD1.1        1.180744e-06  0.9737090 0.667 0.418 5.364120e-03       2
OSER1-DT.2      1.190979e-06  0.6793216 0.387 0.198 5.410618e-03       2
TEX2.1          1.198264e-06  0.8158529 0.560 0.353 5.443715e-03       2
GSTP1.2         1.246936e-06 -0.6055043 0.213 0.465 5.664832e-03       2
DPP10.2         1.311017e-06  0.9239506 0.480 0.174 5.955948e-03       2
ADI1            1.429018e-06 -0.6733762 0.200 0.444 6.492029e-03       2
FSCN1.1         1.432711e-06  0.5122029 0.667 0.400 6.508806e-03       2
WDR4.2          1.508507e-06  0.2723899 0.293 0.129 6.853148e-03       2
TMSB10.1        1.509826e-06 -0.8214322 0.227 0.475 6.859142e-03       2
PDZRN3          1.563883e-06  0.7964214 0.707 0.411 7.104722e-03       2
PTK7            1.604612e-06  0.2947180 0.667 0.435 7.289752e-03       2
ATP6V1B2        1.643543e-06  0.5149123 0.587 0.337 7.466616e-03       2
RAD50.2         1.649167e-06  0.6565684 0.400 0.172 7.492166e-03       2
IL13RA1.1       1.650590e-06  1.0586976 0.587 0.301 7.498629e-03       2
B3GNT2.1        1.663348e-06  0.6590832 0.547 0.331 7.556588e-03       2
ATF4.2          1.664772e-06  0.6186326 0.760 0.454 7.563057e-03       2
E2F6.1          1.686262e-06  0.4597012 0.413 0.231 7.660688e-03       2
L1CAM           1.722672e-06  0.5842052 0.400 0.202 7.826100e-03       2
MICAL1          1.735967e-06  0.7522656 0.387 0.177 7.886497e-03       2
MAMDC2-AS1      1.750851e-06  0.3469317 0.253 0.161 7.954117e-03       2
SYNPO.1         1.790131e-06  0.6292251 0.627 0.402 8.132563e-03       2
MPRIP-AS1       1.802369e-06  0.2629254 0.240 0.156 8.188162e-03       2
Z93930.2.2      1.805401e-06  0.3712078 0.333 0.159 8.201938e-03       2
ZBED3-AS1.1     1.884025e-06 -0.4500584 0.267 0.118 8.559125e-03       2
RAB33A.1        2.013182e-06  0.3894726 0.293 0.117 9.145884e-03       2
MST1.1          2.077338e-06  0.7877374 0.307 0.121 9.437347e-03       2
SERPINE2        2.152601e-06  0.8613721 0.680 0.460 9.779266e-03       2
ENTPD1.1        2.154475e-06  0.9060336 0.440 0.175 9.787780e-03       2
LNX2.1          2.270484e-06  0.8407455 0.427 0.175 1.031481e-02       2
ZNF254          2.276239e-06  0.3038472 0.333 0.189 1.034095e-02       2
FAM196A.1       2.278163e-06  0.4981525 0.400 0.187 1.034969e-02       2
HAUS4.2         2.307191e-06  0.5492105 0.253 0.137 1.048157e-02       2
ZMYND8          2.350464e-06  0.5483418 0.533 0.308 1.067816e-02       2
GRK5            2.360367e-06  0.3117000 0.347 0.137 1.072315e-02       2
CNIH2.1         2.408634e-06  0.6607529 0.280 0.113 1.094243e-02       2
PAPSS2.1        2.431098e-06  0.2549019 0.653 0.439 1.104448e-02       2
UBA6            2.449602e-06  0.8696101 0.600 0.381 1.112854e-02       2
AC137630.3      2.451600e-06  0.4165624 0.160 0.139 1.113762e-02       2
ARFGAP1         2.476570e-06  0.4343832 0.600 0.391 1.125106e-02       2
SLC2A1-AS1.2    2.546304e-06 -0.5684358 0.213 0.102 1.156786e-02       2
GNL1.2          2.604773e-06  0.6567501 0.533 0.341 1.183349e-02       2
TMEM30A.1       2.628960e-06  0.7068010 0.680 0.426 1.194336e-02       2
PTPRK.1         2.684728e-06  1.0225414 0.427 0.213 1.219672e-02       2
GALNT13         2.727869e-06  0.2794655 0.320 0.186 1.239271e-02       2
MT-ND5.2        2.730162e-06  0.2526877 0.613 0.413 1.240313e-02       2
TNFRSF25        2.792659e-06  0.6141644 0.320 0.131 1.268705e-02       2
FMO4.2          2.816152e-06 -0.5940947 0.093 0.099 1.279378e-02       2
AC105760.2      2.832716e-06  0.4026620 0.253 0.160 1.286903e-02       2
AHR.1           2.837469e-06  0.7600234 0.387 0.227 1.289062e-02       2
AC009133.1      2.892622e-06 -0.2880423 0.147 0.093 1.314118e-02       2
ALDH4A1.1       3.128889e-06  0.4843316 0.360 0.147 1.421454e-02       2
MT-CYB.2        3.184231e-06  0.5811474 0.680 0.406 1.446596e-02       2
AL731569.1.2    3.213551e-06 -0.2574082 0.147 0.105 1.459916e-02       2
ADAMTS9.2       3.235039e-06  0.4758091 0.560 0.269 1.469678e-02       2
FAM234B         3.283895e-06 -0.4346678 0.200 0.097 1.491873e-02       2
ENPP2.2         3.344545e-06 -0.5297224 0.320 0.165 1.519427e-02       2
TBCK.1          3.378121e-06  0.8053479 0.427 0.199 1.534680e-02       2
DDX17.2         3.495191e-06  0.4651175 0.653 0.458 1.587865e-02       2
TBC1D9.1        3.512646e-06  1.0273788 0.587 0.406 1.595795e-02       2
SLC2A10         3.580334e-06  0.7977355 0.547 0.344 1.626546e-02       2
MXRA5.2         3.725632e-06  0.9933579 0.667 0.396 1.692555e-02       2
PLPPR3.2        3.728669e-06  0.7504282 0.360 0.165 1.693934e-02       2
GOLIM4.1        3.831984e-06  0.7563605 0.680 0.421 1.740870e-02       2
ABCD1.1         3.902322e-06  0.6063138 0.427 0.192 1.772825e-02       2
JAM2            3.908785e-06  0.7307016 0.453 0.185 1.775761e-02       2
CSPG4.1         4.013446e-06  0.5538737 0.400 0.177 1.823308e-02       2
CYFIP1.1        4.050099e-06  0.3549818 0.600 0.348 1.839960e-02       2
PSMA7           4.158427e-06 -0.8267094 0.253 0.484 1.889173e-02       2
KIF9.1          4.308369e-06  0.4972743 0.373 0.259 1.957292e-02       2
MMP16.1         4.326134e-06  1.0062507 0.493 0.239 1.965362e-02       2
KIAA0930        4.329700e-06  0.6359607 0.560 0.315 1.966983e-02       2
RAB40A.2        4.358286e-06 -0.4057555 0.333 0.167 1.979970e-02       2
HEPH.1          4.365453e-06  0.3270060 0.333 0.124 1.983225e-02       2
GREB1.2         4.419573e-06  0.4232040 0.280 0.129 2.007812e-02       2
NCSTN.1         4.467157e-06  0.7199254 0.640 0.424 2.029429e-02       2
FLVCR2          4.489200e-06  0.2966136 0.227 0.106 2.039444e-02       2
C19orf18        4.489228e-06  0.3695738 0.333 0.145 2.039456e-02       2
SEMA7A.1        4.526272e-06  0.6539663 0.507 0.231 2.056286e-02       2
SH3D19.1        4.559858e-06  1.0617661 0.680 0.452 2.071544e-02       2
PCNX1.1         4.673532e-06  0.4995689 0.573 0.363 2.123186e-02       2
HSP90B1.1       4.692742e-06  0.3484161 0.627 0.421 2.131913e-02       2
EIF4EBP1.2      4.774242e-06 -1.1242781 0.173 0.440 2.168938e-02       2
AC005476.2.2    4.801644e-06 -0.7988574 0.107 0.129 2.181387e-02       2
CA2             4.888972e-06 -0.6799690 0.240 0.158 2.221060e-02       2
CDIP1           4.917118e-06  0.5653867 0.427 0.217 2.233847e-02       2
ACOT7.1         4.969720e-06  0.5153494 0.440 0.254 2.257744e-02       2
TM6SF2.1        4.994145e-06  0.3580148 0.307 0.172 2.268840e-02       2
LINC00960       5.134922e-06  0.5399359 0.307 0.244 2.332795e-02       2
LYSMD2.2        5.185382e-06 -0.8258288 0.200 0.175 2.355719e-02       2
GAS7.1          5.472411e-06  0.7931623 0.573 0.342 2.486116e-02       2
HMGN3-AS1.1     5.494748e-06  1.0272467 0.307 0.119 2.496264e-02       2
XPR1.1          5.621484e-06  0.6687057 0.427 0.256 2.553840e-02       2
EVA1A.1         5.704427e-06  0.4500900 0.413 0.209 2.591521e-02       2
KCNK6.1         5.831045e-06  0.9372001 0.667 0.414 2.649044e-02       2
AC007114.1.1    5.859534e-06 -0.4000567 0.173 0.095 2.661986e-02       2
RAB15.1         5.931371e-06  0.5658816 0.373 0.193 2.694622e-02       2
SULF2.1         5.960342e-06  0.7447104 0.600 0.377 2.707784e-02       2
HNRNPUL2.1      6.092385e-06  0.7892571 0.533 0.306 2.767771e-02       2
WDR81.1         6.107127e-06  1.1535048 0.440 0.188 2.774468e-02       2
TRIM65.2        6.181938e-06  0.8108757 0.360 0.180 2.808454e-02       2
GNG5            6.308477e-06 -0.5835033 0.213 0.452 2.865941e-02       2
SLC35G1         6.468773e-06  0.6271211 0.333 0.190 2.938764e-02       2
ARAP3           6.724527e-06  0.7525824 0.320 0.158 3.054953e-02       2
ABAT.1          6.906266e-06  0.4168426 0.453 0.225 3.137517e-02       2
CDK6.1          6.984705e-06  0.4365931 0.640 0.446 3.173151e-02       2
ARSJ.1          7.115381e-06  1.0828405 0.427 0.262 3.232518e-02       2
TBC1D10A        7.348287e-06  0.3270707 0.493 0.316 3.338327e-02       2
HELLS.2         7.407588e-06  0.7197621 0.347 0.143 3.365267e-02       2
ARSI.2          7.443327e-06  0.5712022 0.333 0.130 3.381504e-02       2
TMCO4.1         7.449451e-06  0.8501429 0.387 0.138 3.384286e-02       2
IGF2R           7.527731e-06  0.4264264 0.640 0.407 3.419848e-02       2
LDHB.1          7.668130e-06 -0.7961654 0.213 0.436 3.483631e-02       2
CCDC69.2        7.911912e-06  0.4046748 0.427 0.212 3.594382e-02       2
BEX1.2          8.020494e-06 -0.4844544 0.213 0.156 3.643711e-02       2
SPTBN1.2        8.228762e-06  0.2638899 0.707 0.439 3.738326e-02       2
PTPRF.1         8.374921e-06  1.0257103 0.507 0.300 3.804727e-02       2
ACVR2A.1        8.408486e-06  0.4896331 0.573 0.369 3.819975e-02       2
DLG1.1          8.598825e-06  0.5914076 0.573 0.364 3.906446e-02       2
AC242426.2.2    8.877864e-06  0.7265987 0.293 0.143 4.033214e-02       2
MT-ND2          9.305390e-06  0.2696511 0.627 0.429 4.227439e-02       2
TUBGCP6         9.386960e-06  0.5165685 0.280 0.159 4.264496e-02       2
PAPSS1.1        9.791316e-06  0.9063456 0.667 0.418 4.448195e-02       2
AC116667.1.2    1.006713e-05  0.3126495 0.360 0.128 4.573498e-02       2
C8orf49         1.014738e-05  0.4820559 0.333 0.132 4.609957e-02       2
PLXNA2.1        1.017963e-05  0.5146138 0.400 0.235 4.624606e-02       2
SEMA4F.2        1.094876e-05  0.4003429 0.333 0.193 4.974023e-02       2
QSOX1           1.106187e-05  0.4861780 0.600 0.425 5.025409e-02       2
LRRC49.2        1.211167e-05  0.4579269 0.333 0.120 5.502333e-02       2
ATP2C1          1.244974e-05  0.7176801 0.640 0.369 5.655915e-02       2
ANOS1.2         1.249788e-05 -0.4398060 0.547 0.347 5.677785e-02       2
AL049597.2      1.262702e-05  0.4817875 0.253 0.133 5.736454e-02       2
CLTA.1          1.278666e-05 -0.7193863 0.253 0.456 5.808979e-02       2
XPO1.1          1.281680e-05  0.4734022 0.640 0.386 5.822670e-02       2
ADCK1           1.287703e-05  0.4819119 0.253 0.149 5.850036e-02       2
ECM2            1.302969e-05  0.4611438 0.253 0.140 5.919386e-02       2
DTNB.1          1.313225e-05  0.4960734 0.267 0.129 5.965981e-02       2
CPA4.2          1.315937e-05  0.7548122 0.520 0.289 5.978303e-02       2
ZNF703.1        1.317350e-05  0.7603409 0.520 0.287 5.984719e-02       2
STARD9          1.385031e-05 -0.5911426 0.173 0.130 6.292197e-02       2
DDB2            1.386123e-05  0.4580315 0.627 0.403 6.297156e-02       2
ACTN1.2         1.402473e-05  0.5328738 0.667 0.441 6.371436e-02       2
LIG1.2          1.427867e-05  0.4441620 0.280 0.156 6.486802e-02       2
UNC5B.1         1.432361e-05  0.7904717 0.480 0.235 6.507218e-02       2
CYB5D1.1        1.434604e-05  0.9399140 0.427 0.222 6.517406e-02       2
ADAMTSL4        1.498651e-05  0.6722466 0.320 0.198 6.808370e-02       2
SEC61B          1.525741e-05 -0.3352656 0.307 0.485 6.931443e-02       2
ZNF510          1.529299e-05  0.6884073 0.280 0.135 6.947608e-02       2
PSMD8           1.532884e-05 -0.5905944 0.240 0.467 6.963892e-02       2
ETV5.2          1.629952e-05  0.4524393 0.493 0.316 7.404873e-02       2
ZNF486.1        1.632483e-05 -0.4493216 0.160 0.093 7.416372e-02       2
MGAT2           1.685222e-05  0.7189132 0.627 0.460 7.655965e-02       2
CCDC71L         1.722165e-05  0.4200982 0.520 0.295 7.823794e-02       2
AL035563.1      1.769430e-05 -0.4340117 0.147 0.093 8.038521e-02       2
ODAPH.2         1.793052e-05  0.2789137 0.507 0.304 8.145833e-02       2
HSPBAP1         1.819311e-05 -0.2596371 0.187 0.190 8.265132e-02       2
CHSY3           1.820744e-05  0.8025634 0.480 0.299 8.271641e-02       2
ZNF610.1        1.836257e-05 -0.5839503 0.147 0.095 8.342114e-02       2
NIN             1.840526e-05  0.4585213 0.533 0.289 8.361508e-02       2
CTSD.1          1.898181e-05  0.5723452 0.693 0.432 8.623438e-02       2
SLC39A14.2      2.057821e-05  0.6051059 0.573 0.313 9.348682e-02       2
EIF5A.1         2.059402e-05 -0.7065978 0.187 0.456 9.355863e-02       2
COL6A1.1        2.202859e-05  0.7762935 0.627 0.404 1.000759e-01       2
GLP2R.1         2.225051e-05  0.8417156 0.267 0.113 1.010841e-01       2
PCDH10          2.249027e-05  0.7391562 0.613 0.426 1.021733e-01       2
CHPF.1          2.301163e-05  0.8783437 0.653 0.426 1.045418e-01       2
CYCS.2          2.325884e-05 -0.7781215 0.213 0.462 1.056649e-01       2
TMPO.1          2.529110e-05  0.4447652 0.480 0.343 1.148975e-01       2
TAGLN.2         2.538294e-05 -1.5619377 0.253 0.474 1.153147e-01       2
HMCN1.2         2.591328e-05  0.7082848 0.440 0.238 1.177240e-01       2
PKM.1           2.620998e-05  0.3408909 0.613 0.447 1.190719e-01       2
FADS2           2.640954e-05  0.5041766 0.587 0.344 1.199785e-01       2
PLXDC1.2        2.695535e-05  0.7598933 0.373 0.199 1.224581e-01       2
NID2.1          2.829314e-05  0.9116941 0.520 0.293 1.285357e-01       2
DNAJC6          2.837879e-05 -0.3830865 0.227 0.155 1.289248e-01       2
CORO2B.2        2.840011e-05  0.2666767 0.333 0.129 1.290217e-01       2
RNF6.1          2.842173e-05  0.6804591 0.493 0.311 1.291199e-01       2
SPHK1           2.931530e-05  0.6226543 0.320 0.199 1.331794e-01       2
EEF1A1.1        2.953733e-05 -0.4684253 0.253 0.443 1.341881e-01       2
RNF32           2.969338e-05  0.6095536 0.267 0.140 1.348970e-01       2
SART1.1         2.973835e-05  0.7623051 0.440 0.250 1.351013e-01       2
DYNLL1.2        2.980581e-05 -0.7019900 0.253 0.488 1.354078e-01       2
C5orf30.2       2.996304e-05 -0.4296933 0.187 0.130 1.361221e-01       2
MYO10.2         3.028121e-05  0.5953903 0.613 0.381 1.375676e-01       2
COL8A1          3.071678e-05  0.2725963 0.480 0.273 1.395463e-01       2
IFITM1.2        3.106590e-05 -0.6038794 0.120 0.125 1.411324e-01       2
DTX4            3.208683e-05  0.8321306 0.440 0.203 1.457705e-01       2
COL6A2.2        3.218183e-05  0.8505476 0.627 0.388 1.462020e-01       2
ATXN1.1         3.249903e-05  0.7623797 0.707 0.434 1.476431e-01       2
CCDC71          3.279244e-05  0.4624519 0.307 0.163 1.489761e-01       2
CLEC2D.1        3.381664e-05  0.7815102 0.413 0.208 1.536290e-01       2
NME4.2          3.466253e-05 -0.8624057 0.227 0.446 1.574719e-01       2
PRDX1           3.536953e-05 -0.8981523 0.200 0.467 1.606838e-01       2
ATP6V0A2.1      3.539597e-05  0.7859792 0.560 0.347 1.608039e-01       2
LINC01806.1     3.619761e-05  0.2549506 0.160 0.149 1.644458e-01       2
SRGAP2          3.625208e-05  0.8497173 0.493 0.275 1.646932e-01       2
INAFM2.1        3.748955e-05  0.5089504 0.373 0.223 1.703150e-01       2
OGFRL1.1        3.774185e-05  1.1321970 0.693 0.425 1.714612e-01       2
ANAPC16.2       3.777003e-05 -0.8327734 0.280 0.468 1.715893e-01       2
SEC11C          3.791097e-05 -0.7543001 0.187 0.442 1.722295e-01       2
PLCE1-AS1       3.894050e-05 -0.3924071 0.320 0.198 1.769067e-01       2
LINC01004       3.899847e-05 -0.3997599 0.320 0.181 1.771701e-01       2
FOSB            3.964152e-05  0.3288373 0.267 0.176 1.800914e-01       2
SLC25A21-AS1    4.169352e-05 -0.3885862 0.067 0.098 1.894136e-01       2
AC132872.1.2    4.181677e-05  0.6049407 0.267 0.138 1.899736e-01       2
ZNF76           4.244108e-05  0.7697554 0.360 0.154 1.928098e-01       2
ATP13A3.1       4.410611e-05  0.8185671 0.600 0.399 2.003741e-01       2
SLC39A10.1      4.443322e-05  0.6087355 0.493 0.363 2.018601e-01       2
LRRC2.2         4.549548e-05 -0.5957045 0.187 0.147 2.066860e-01       2
FLRT2.1         4.586627e-05  0.9841631 0.640 0.412 2.083705e-01       2
ACTBL2.1        4.685725e-05  0.8852483 0.307 0.135 2.128725e-01       2
PAM.1           4.751770e-05  0.8021598 0.600 0.428 2.158729e-01       2
ALDH1B1.2       4.908086e-05  0.5825265 0.493 0.309 2.229744e-01       2
AMOTL2.1        4.918922e-05  0.3673080 0.600 0.420 2.234666e-01       2
AL669970.3.1    4.969770e-05 -0.4855324 0.107 0.105 2.257767e-01       2
AC006213.1      5.050688e-05 -0.6295972 0.187 0.160 2.294528e-01       2
IFNAR1          5.571252e-05  0.6101458 0.587 0.422 2.531020e-01       2
ALDH2.1         5.686198e-05  0.8809344 0.333 0.187 2.583240e-01       2
TRAM2           6.178137e-05  0.5825287 0.573 0.417 2.806728e-01       2
AL162258.2      6.232179e-05 -0.4041428 0.280 0.138 2.831279e-01       2
LPAR2.1         6.300357e-05  0.7308570 0.333 0.166 2.862252e-01       2
FMOD.2          6.322915e-05  0.4408091 0.520 0.237 2.872500e-01       2
MBLAC1.2        6.323004e-05 -0.8613650 0.307 0.174 2.872541e-01       2
NOTCH2          6.585101e-05  0.3875773 0.573 0.413 2.991612e-01       2
ARRDC4          6.666581e-05  0.4698785 0.400 0.207 3.028628e-01       2
RAB2B           6.676227e-05  0.4625490 0.413 0.264 3.033010e-01       2
RNF150.2        6.797966e-05 -0.4345716 0.107 0.107 3.088316e-01       2
SOD3.1          6.842299e-05  0.5714218 0.373 0.228 3.108457e-01       2
MAST3           6.862096e-05  0.5226010 0.400 0.149 3.117450e-01       2
KIF26B.1        6.862103e-05  1.0068117 0.427 0.267 3.117453e-01       2
B2M.2           7.051646e-05 -1.6652630 0.240 0.438 3.203563e-01       2
MYL12A.2        7.083418e-05 -0.9645450 0.240 0.449 3.217997e-01       2
SPRY2.2         7.088495e-05  0.2620716 0.493 0.287 3.220303e-01       2
LDLRAD2         7.412072e-05 -0.5978156 0.147 0.133 3.367304e-01       2
CMTM6           7.487097e-05  0.2690907 0.600 0.437 3.401388e-01       2
TRIM22.2        7.562785e-05  0.3923637 0.427 0.316 3.435773e-01       2
ZNF799          7.589986e-05  0.5226356 0.133 0.091 3.448130e-01       2
IFFO1.1         7.683071e-05  0.6850315 0.413 0.181 3.490419e-01       2
DBI.1           7.727429e-05 -0.7200699 0.227 0.435 3.510571e-01       2
HEATR5A.1       7.827832e-05  0.7436007 0.493 0.346 3.556184e-01       2
GALNT7.1        7.844679e-05  0.7850744 0.533 0.377 3.563838e-01       2
PKP2.2          7.844686e-05  0.8409029 0.440 0.329 3.563841e-01       2
AC110285.6.2    7.980453e-05  0.3505676 0.227 0.149 3.625520e-01       2
GJA1.1          8.130653e-05  0.4591357 0.600 0.415 3.693756e-01       2
ZNF438.1        8.189020e-05  0.7764662 0.347 0.140 3.720272e-01       2
NISCH           8.523302e-05  0.3743833 0.520 0.335 3.872136e-01       2
WIPF1.1         8.633432e-05  0.4288595 0.427 0.292 3.922168e-01       2
C6orf48.2       8.651941e-05 -0.6857920 0.267 0.478 3.930577e-01       2
JCAD            8.952888e-05  0.5656622 0.613 0.395 4.067297e-01       2
ZNF107.1        9.074704e-05  0.8795597 0.253 0.105 4.122638e-01       2
SLC7A5.1        9.119998e-05  0.8167690 0.507 0.255 4.143215e-01       2
SLC7A1          9.224275e-05  0.6448939 0.520 0.283 4.190588e-01       2
RAD51AP1.2      9.290034e-05  0.4028424 0.240 0.102 4.220462e-01       2
S100A10.2       9.442984e-05 -0.7735170 0.280 0.490 4.289948e-01       2
DLC1.1          9.537245e-05  0.3862677 0.573 0.361 4.332770e-01       2
AQP10           9.611903e-05 -0.3588546 0.227 0.119 4.366688e-01       2
TCAF1.1         1.044847e-04  0.9843528 0.587 0.395 4.746739e-01       2
IRF7            1.096126e-04  0.4540684 0.347 0.212 4.979702e-01       2
MICB            1.103867e-04  0.4579618 0.400 0.176 5.014870e-01       2
TUBA4A.2        1.109307e-04 -0.2509077 0.267 0.211 5.039581e-01       2
NR1H3.1         1.117924e-04  0.2515001 0.253 0.104 5.078727e-01       2
RAD9A.1         1.136113e-04  0.6930806 0.373 0.151 5.161362e-01       2
SGCE            1.138509e-04  0.4219768 0.360 0.200 5.172246e-01       2
TUBB6.2         1.142538e-04 -0.4956229 0.253 0.422 5.190552e-01       2
UBALD2          1.148972e-04  0.3041129 0.293 0.235 5.219778e-01       2
RBM47.2         1.154625e-04 -0.5735964 0.200 0.130 5.245462e-01       2
ANGPTL7.2       1.173135e-04 -0.5088072 0.360 0.231 5.329552e-01       2
UHMK1.2         1.179174e-04  0.4689640 0.600 0.437 5.356988e-01       2
LOXL2.1         1.192464e-04  0.6837463 0.613 0.429 5.417365e-01       2
PDE4B.1         1.212666e-04  0.4525205 0.280 0.121 5.509140e-01       2
LPP.1           1.229748e-04  0.3889372 0.613 0.442 5.586745e-01       2
GSDME.1         1.232329e-04  0.6615870 0.387 0.204 5.598470e-01       2
PROSER1         1.254912e-04  0.3017882 0.347 0.189 5.701067e-01       2
ARRDC3          1.267240e-04  0.4404827 0.333 0.224 5.757069e-01       2
TSPAN2.1        1.314927e-04  0.3320318 0.320 0.154 5.973713e-01       2
CACNA2D1.1      1.352052e-04  1.1589682 0.520 0.312 6.142372e-01       2
PGRMC2          1.368123e-04  0.3148576 0.627 0.432 6.215382e-01       2
PPM1E           1.393980e-04  0.4319448 0.440 0.147 6.332850e-01       2
TALDO1.2        1.394984e-04 -0.4260945 0.253 0.442 6.337411e-01       2
LBR             1.442189e-04  0.4660870 0.400 0.268 6.551865e-01       2
SNX30.1         1.455230e-04  0.8540681 0.453 0.253 6.611110e-01       2
GOLGB1          1.485753e-04  0.4663573 0.680 0.447 6.749775e-01       2
CALM2           1.499164e-04 -1.0126709 0.200 0.454 6.810701e-01       2
ARHGAP23        1.509558e-04  0.3157140 0.333 0.211 6.857921e-01       2
IGFBP5.2        1.514664e-04  0.3726734 0.547 0.335 6.881118e-01       2
AC025171.4.2    1.521066e-04 -0.5338670 0.147 0.094 6.910205e-01       2
GNG2.2          1.529463e-04  0.4143655 0.253 0.099 6.948351e-01       2
SRPX2.2         1.570114e-04  0.6681505 0.587 0.399 7.133028e-01       2
CPNE2.2         1.574446e-04  0.2511854 0.200 0.179 7.152707e-01       2
STAT2           1.609531e-04  0.8072655 0.627 0.425 7.312101e-01       2
JADE3           1.613927e-04  0.5299714 0.333 0.133 7.332070e-01       2
DPP10-AS1.2     1.625081e-04  0.3513602 0.227 0.145 7.382742e-01       2
NFKBIZ.1        1.644217e-04  0.2797665 0.440 0.304 7.469676e-01       2
ZNF10.1         1.660087e-04  0.3176350 0.240 0.179 7.541774e-01       2
MOB1B.1         1.663536e-04  0.4334437 0.387 0.247 7.557442e-01       2
TACC2.2         1.672665e-04  0.3577551 0.253 0.157 7.598917e-01       2
FEM1C           1.677300e-04  0.5461948 0.467 0.213 7.619975e-01       2
CEP85           1.746547e-04  0.6698987 0.333 0.157 7.934564e-01       2
FCMR            1.752541e-04 -0.3080157 0.347 0.199 7.961793e-01       2
TMEM87B.1       1.790118e-04  0.5276660 0.560 0.423 8.132504e-01       2
LINC01355.2     1.807332e-04  0.7021649 0.320 0.207 8.210711e-01       2
COL8A2.1        1.811042e-04  0.9576001 0.440 0.344 8.227564e-01       2
ENDOD1.2        1.820971e-04  0.3812740 0.600 0.441 8.272670e-01       2
AQP1.2          1.826793e-04  0.3060853 0.640 0.392 8.299119e-01       2
COL15A1.1       1.871596e-04  0.6330526 0.573 0.366 8.502662e-01       2
ANXA2.2         1.894487e-04 -0.7746078 0.227 0.448 8.606653e-01       2
SLIT3.1         1.929644e-04  1.0635466 0.467 0.202 8.766373e-01       2
EDEM3           1.968106e-04  0.7853470 0.587 0.387 8.941107e-01       2
AC018647.1.2    1.997722e-04  0.4111262 0.293 0.156 9.075653e-01       2
CBX4.1          2.036102e-04  0.2632325 0.467 0.337 9.250010e-01       2
ST3GAL6.2       2.044408e-04  0.5128196 0.413 0.229 9.287744e-01       2
SPOCK1.2        2.054133e-04  0.4363859 0.547 0.375 9.331926e-01       2
ARHGAP5         2.072326e-04  0.8658809 0.653 0.474 9.414577e-01       2
SLC9A8.1        2.075101e-04  0.4004446 0.293 0.160 9.427185e-01       2
ZNF691          2.089188e-04  0.4492070 0.333 0.135 9.491179e-01       2
DUSP19          2.097721e-04 -0.7692860 0.280 0.191 9.529948e-01       2
NDUFA12.1       2.099179e-04 -1.0161300 0.240 0.427 9.536571e-01       2
SBF2            2.171406e-04 -0.3446533 0.360 0.230 9.864699e-01       2
CPNE7.2         2.180208e-04  0.3453537 0.253 0.156 9.904685e-01       2
MET.2           2.214349e-04  0.5622170 0.307 0.146 1.000000e+00       2
SLC37A3.2       2.224862e-04  0.3243453 0.427 0.256 1.000000e+00       2
SPON1           2.229370e-04  0.7080489 0.573 0.388 1.000000e+00       2
FAM69B.2        2.253561e-04  0.2975925 0.160 0.160 1.000000e+00       2
SOX6.1          2.258127e-04  0.7362072 0.493 0.269 1.000000e+00       2
TAF15           2.298050e-04  0.9741629 0.493 0.333 1.000000e+00       2
MRPS6.1         2.315138e-04 -0.8240780 0.240 0.457 1.000000e+00       2
KLHL29          2.326070e-04  0.4027946 0.253 0.095 1.000000e+00       2
EVA1B           2.386290e-04 -0.5818557 0.253 0.460 1.000000e+00       2
CNTRL.2         2.387887e-04  0.4033401 0.160 0.120 1.000000e+00       2
AC092171.3.1    2.387887e-04 -0.5054899 0.080 0.140 1.000000e+00       2
CDV3            2.392713e-04  0.2944778 0.680 0.482 1.000000e+00       2
ANKRD6.2        2.397540e-04  0.3665988 0.347 0.243 1.000000e+00       2
CHST3.2         2.467742e-04  0.4894903 0.307 0.167 1.000000e+00       2
SGPP1.1         2.575831e-04  0.8946923 0.427 0.233 1.000000e+00       2
ADAM15.1        2.594849e-04  0.4078903 0.320 0.210 1.000000e+00       2
REPS1           2.624494e-04  0.3461513 0.293 0.171 1.000000e+00       2
TXN.1           2.652621e-04 -1.3656734 0.227 0.413 1.000000e+00       2
ARHGAP12        2.674026e-04  0.3528163 0.240 0.256 1.000000e+00       2
ZNF573.2        2.690129e-04 -0.2878840 0.160 0.095 1.000000e+00       2
MAD2L1.1        2.729865e-04 -0.4509129 0.280 0.235 1.000000e+00       2
EPAS1           2.744438e-04  0.4004968 0.587 0.392 1.000000e+00       2
CUL2            2.755466e-04  0.6655916 0.387 0.290 1.000000e+00       2
S100A11.1       2.790545e-04 -0.7915355 0.293 0.445 1.000000e+00       2
GUCD1           2.843002e-04  0.3981140 0.400 0.262 1.000000e+00       2
JAZF1.1         2.925363e-04  0.3689267 0.467 0.298 1.000000e+00       2
SACS.1          2.937033e-04  0.7447389 0.467 0.293 1.000000e+00       2
POLE4.2         2.944835e-04 -0.9345060 0.253 0.449 1.000000e+00       2
MESP1           2.952656e-04 -0.4548930 0.213 0.172 1.000000e+00       2
LINC02482.2     2.960500e-04 -0.3362712 0.253 0.216 1.000000e+00       2
SELENBP1.2      2.980143e-04 -0.8098432 0.240 0.153 1.000000e+00       2
TRIM21          3.001960e-04  0.4965090 0.307 0.199 1.000000e+00       2
PCDHA4.1        3.052094e-04 -0.4922229 0.053 0.098 1.000000e+00       2
SLC20A1         3.056140e-04  0.4731157 0.507 0.333 1.000000e+00       2
TMEM241.1       3.076410e-04  0.3335292 0.373 0.182 1.000000e+00       2
JMY             3.076425e-04  0.6377976 0.453 0.293 1.000000e+00       2
ATP6V1G2        3.105033e-04 -0.5372963 0.067 0.119 1.000000e+00       2
ANKH.1          3.140115e-04  0.6260771 0.573 0.397 1.000000e+00       2
ID4.2           3.217726e-04 -0.7713500 0.093 0.121 1.000000e+00       2
SCRG1.2         3.226241e-04 -0.2709550 0.413 0.215 1.000000e+00       2
FBLN1.2         3.288498e-04  0.2592392 0.640 0.426 1.000000e+00       2
C16orf86.2      3.325511e-04 -0.4127856 0.080 0.071 1.000000e+00       2
FLYWCH1.2       3.329892e-04  0.3251187 0.333 0.235 1.000000e+00       2
LY6G5C.2        3.351830e-04 -0.9018749 0.067 0.150 1.000000e+00       2
CBL.1           3.391783e-04  0.5717075 0.387 0.259 1.000000e+00       2
AGMAT           3.423129e-04 -0.3137991 0.187 0.113 1.000000e+00       2
LRIG3.1         3.454747e-04  1.0611259 0.453 0.248 1.000000e+00       2
KIAA0232.1      3.468381e-04  0.4297962 0.373 0.264 1.000000e+00       2
TMEM190.2       3.509520e-04 -0.8143269 0.173 0.129 1.000000e+00       2
SMAD1           3.523423e-04  0.5601481 0.267 0.194 1.000000e+00       2
MMP2.2          3.555870e-04  0.7088228 0.600 0.410 1.000000e+00       2
GDNF-AS1        3.579279e-04  0.4771310 0.200 0.093 1.000000e+00       2
APLP2           3.720137e-04  0.2828507 0.573 0.441 1.000000e+00       2
AL078639.1.1    3.881307e-04  0.6675488 0.560 0.332 1.000000e+00       2
KCTD12.1        3.901579e-04  0.3820957 0.533 0.389 1.000000e+00       2
MAP4K3          3.945016e-04  0.2859663 0.587 0.385 1.000000e+00       2
NCOA7           3.981117e-04  0.3374659 0.507 0.307 1.000000e+00       2
ATP6AP1L.2      4.020130e-04  0.7550283 0.280 0.184 1.000000e+00       2
NCLN.1          4.072728e-04  0.9156781 0.453 0.341 1.000000e+00       2
C3orf52.1       4.083322e-04  0.7731135 0.347 0.180 1.000000e+00       2
KCNS3.2         4.099256e-04  0.4067156 0.293 0.172 1.000000e+00       2
SCOC-AS1        4.174406e-04 -0.5316785 0.107 0.108 1.000000e+00       2
CLDN12.2        4.223405e-04  0.4133463 0.387 0.223 1.000000e+00       2
LASP1.1         4.250860e-04  0.8701879 0.600 0.448 1.000000e+00       2
ASB13.2         4.253605e-04  0.5057201 0.333 0.116 1.000000e+00       2
LRRC37A3.2      4.267364e-04  0.5986217 0.333 0.133 1.000000e+00       2
AMOTL1          4.325807e-04  0.6330893 0.400 0.225 1.000000e+00       2
LGR4.1          4.367976e-04  0.3065595 0.560 0.347 1.000000e+00       2
SERINC5.1       4.419085e-04  0.8693378 0.533 0.430 1.000000e+00       2
AL162231.2      4.499686e-04 -0.2574666 0.227 0.126 1.000000e+00       2
TPM4.2          4.511327e-04 -0.7192494 0.280 0.441 1.000000e+00       2
LAMP2           4.517150e-04  0.2755306 0.653 0.441 1.000000e+00       2
ARHGAP20.1      4.560972e-04  0.7175009 0.387 0.166 1.000000e+00       2
METTL7A.1       4.578730e-04  1.0138906 0.573 0.380 1.000000e+00       2
LGMN            4.584637e-04  0.5775826 0.613 0.428 1.000000e+00       2
SLC38A4.1       4.647071e-04  1.0878427 0.333 0.207 1.000000e+00       2
HSPB1.2         4.695107e-04 -0.5506458 0.267 0.438 1.000000e+00       2
GLIPR1.2        4.721542e-04  0.4019181 0.533 0.376 1.000000e+00       2
DYNLT3.1        4.808202e-04  0.5070046 0.613 0.426 1.000000e+00       2
TBC1D1.1        4.873474e-04  0.4669157 0.427 0.293 1.000000e+00       2
LAMA1.1         4.885961e-04  0.6165367 0.333 0.148 1.000000e+00       2
PCOLCE          4.895366e-04  0.2783391 0.400 0.224 1.000000e+00       2
ENTPD4.1        4.923761e-04  0.6838636 0.613 0.426 1.000000e+00       2
BAX             4.930081e-04 -0.4222326 0.293 0.465 1.000000e+00       2
LIF.2           4.936400e-04  0.3111669 0.347 0.200 1.000000e+00       2
ZSCAN5A         4.974478e-04  0.5007513 0.293 0.151 1.000000e+00       2
ETHE1.2         5.205759e-04 -0.7405262 0.213 0.449 1.000000e+00       2
SSC5D.2         5.262585e-04  0.6763287 0.467 0.285 1.000000e+00       2
PXDN            5.279416e-04  0.2788722 0.600 0.449 1.000000e+00       2
AEN             5.316608e-04  0.3396298 0.400 0.254 1.000000e+00       2
FGD6.2          5.398592e-04 -0.2994607 0.373 0.242 1.000000e+00       2
TRIL.1          5.412301e-04  0.8657868 0.253 0.119 1.000000e+00       2
ALDH3B1.1       5.419284e-04 -0.9777260 0.093 0.209 1.000000e+00       2
FRMD6.1         5.474788e-04  0.6396419 0.533 0.404 1.000000e+00       2
MDH1.1          5.492255e-04 -0.5455649 0.267 0.446 1.000000e+00       2
INMT.2          5.551974e-04 -0.2987035 0.347 0.236 1.000000e+00       2
VKORC1.1        5.605209e-04 -0.2914519 0.213 0.422 1.000000e+00       2
ABTB2.1         5.612338e-04  0.7733705 0.360 0.177 1.000000e+00       2
HOXB4.1         5.669589e-04  0.5166997 0.253 0.191 1.000000e+00       2
LGALS3BP        5.709437e-04  0.8894472 0.613 0.435 1.000000e+00       2
PPIC.1          5.720323e-04 -0.5837638 0.280 0.451 1.000000e+00       2
TACC3           5.745773e-04 -0.4357408 0.187 0.098 1.000000e+00       2
MAP3K9          5.786034e-04 -0.3812217 0.267 0.080 1.000000e+00       2
AL121827.2.2    5.800733e-04  0.2558518 0.307 0.230 1.000000e+00       2
KCNC4.2         5.897126e-04  0.4045417 0.427 0.276 1.000000e+00       2
FRY             5.900862e-04  0.7583267 0.467 0.305 1.000000e+00       2
LGALS1.2        5.915798e-04 -0.8894490 0.293 0.471 1.000000e+00       2
LAMB2.2         5.953413e-04  0.3222378 0.613 0.445 1.000000e+00       2
IBTK            5.960956e-04  0.3532486 0.547 0.379 1.000000e+00       2
HLA-C.2         6.017807e-04 -0.8751548 0.253 0.429 1.000000e+00       2
AC067838.1.1    6.133009e-04  0.6729162 0.333 0.184 1.000000e+00       2
CHD1L.1         6.140785e-04  0.2798493 0.360 0.230 1.000000e+00       2
NCOA4           6.199234e-04  0.5423139 0.560 0.427 1.000000e+00       2
CNKSR3.2        6.222733e-04 -0.2876198 0.267 0.164 1.000000e+00       2
C2orf49.2       6.258192e-04  0.3513152 0.480 0.335 1.000000e+00       2
CCDC138.2       6.458517e-04  0.2655342 0.187 0.124 1.000000e+00       2
STXBP6          6.523893e-04 -0.6403760 0.227 0.466 1.000000e+00       2
ECM1.2          6.556476e-04  0.7260704 0.427 0.308 1.000000e+00       2
CD59.1          6.669011e-04  0.4373213 0.613 0.433 1.000000e+00       2
SELENOW.2       6.706810e-04 -0.7307207 0.293 0.448 1.000000e+00       2
GTF2E1.2        6.890164e-04  0.4244069 0.440 0.276 1.000000e+00       2
PDLIM3.2        7.194141e-04 -1.1035284 0.227 0.373 1.000000e+00       2
IRF2BP2.1       7.207649e-04  0.5697995 0.533 0.411 1.000000e+00       2
GSN             7.221169e-04  0.5056632 0.613 0.430 1.000000e+00       2
SLC25A25        7.230149e-04  0.3609540 0.227 0.156 1.000000e+00       2
NUP133.1        7.266395e-04  0.4796396 0.453 0.261 1.000000e+00       2
MAP1B.1         7.280018e-04  0.3879427 0.613 0.446 1.000000e+00       2
STK35.1         7.422018e-04  0.4033788 0.147 0.152 1.000000e+00       2
LCOR.1          7.426797e-04  0.8411823 0.493 0.350 1.000000e+00       2
TDG             7.562183e-04  0.3677905 0.573 0.413 1.000000e+00       2
TP53INP1.1      7.576318e-04  0.7904338 0.560 0.387 1.000000e+00       2
FAM234A         7.628358e-04  0.6416708 0.547 0.355 1.000000e+00       2
HAS2.2          7.781626e-04  0.7619307 0.413 0.244 1.000000e+00       2
ITGA2.1         7.932826e-04  0.6804207 0.480 0.277 1.000000e+00       2
ATL3.1          7.942703e-04  0.7116484 0.507 0.369 1.000000e+00       2
CNRIP1.2        7.982117e-04  0.4416744 0.280 0.163 1.000000e+00       2
PTPN11.2        7.997074e-04  0.9546993 0.533 0.402 1.000000e+00       2
SLC4A7.1        8.021901e-04  0.9909183 0.453 0.324 1.000000e+00       2
PCMTD2.2        8.096811e-04  0.9806298 0.440 0.277 1.000000e+00       2
MACF1.1         8.111879e-04  0.5377042 0.613 0.430 1.000000e+00       2
CLIP4           8.177440e-04  0.2892973 0.480 0.324 1.000000e+00       2
SSTR1.2         8.182504e-04  0.7257152 0.400 0.179 1.000000e+00       2
WNT5B.2         8.408198e-04  0.2786152 0.427 0.277 1.000000e+00       2
WDFY3           8.575967e-04  0.6224524 0.347 0.239 1.000000e+00       2
LEF1.2          8.628996e-04  0.2744795 0.373 0.240 1.000000e+00       2
FRA10AC1        8.806283e-04  0.8156243 0.440 0.344 1.000000e+00       2
SH3RF3          8.822561e-04  0.6269499 0.440 0.251 1.000000e+00       2
GRIP2.2         8.882292e-04  0.3716645 0.253 0.131 1.000000e+00       2
C12orf75.2      8.915270e-04 -1.0683280 0.267 0.420 1.000000e+00       2
TMEM86B.1       9.148462e-04  1.1966046 0.307 0.078 1.000000e+00       2
STRIP1.1        9.199163e-04  0.5672104 0.360 0.221 1.000000e+00       2
TMEM141.2       9.233096e-04 -0.7800897 0.227 0.421 1.000000e+00       2
FAM168A         9.358518e-04  0.4846162 0.547 0.415 1.000000e+00       2
IL17B           9.508775e-04 -0.4177478 0.080 0.116 1.000000e+00       2
NUBP1           9.555454e-04  0.6046264 0.333 0.258 1.000000e+00       2
KIAA1024.2      9.786071e-04  0.4450668 0.333 0.168 1.000000e+00       2
TUBB4B.2        9.954835e-04 -0.6413357 0.267 0.479 1.000000e+00       2
PAMR1.1         9.960902e-04  0.7656242 0.413 0.202 1.000000e+00       2
FAM96B.2        9.991344e-04 -0.5434560 0.293 0.470 1.000000e+00       2
THBD.2          1.000352e-03  0.2600281 0.400 0.213 1.000000e+00       2
NPIPA1.1        1.003390e-03  0.7500224 0.387 0.163 1.000000e+00       2
CDKL5.2         1.035686e-03 -0.6539334 0.373 0.184 1.000000e+00       2
GPSM1.1         1.045191e-03  0.3613710 0.320 0.197 1.000000e+00       2
LIX1.1          1.045828e-03 -0.6610145 0.293 0.449 1.000000e+00       2
GALNT12.1       1.058616e-03  0.3873775 0.320 0.158 1.000000e+00       2
NDUFB6          1.059903e-03 -0.7606396 0.333 0.456 1.000000e+00       2
DPYSL3          1.067006e-03  0.3819694 0.587 0.465 1.000000e+00       2
CD82.1          1.085942e-03  0.8132805 0.453 0.273 1.000000e+00       2
MON2.2          1.091222e-03  0.5486085 0.613 0.433 1.000000e+00       2
PTAR1.1         1.094533e-03  0.3230330 0.440 0.337 1.000000e+00       2
ERCC3           1.106531e-03  0.6441277 0.373 0.217 1.000000e+00       2
PIAS4           1.107201e-03  0.5318739 0.333 0.182 1.000000e+00       2
MIR4435-2HG.2   1.108542e-03 -0.7002468 0.267 0.449 1.000000e+00       2
SP4.1           1.121358e-03  0.8423850 0.253 0.115 1.000000e+00       2
P4HA3.1         1.122038e-03  0.6166358 0.573 0.410 1.000000e+00       2
PLXNA4.1        1.155037e-03 -0.6097132 0.320 0.178 1.000000e+00       2
SULF1.2         1.173297e-03 -0.4232730 0.253 0.150 1.000000e+00       2
ADRA2A.1        1.182514e-03  0.7784081 0.307 0.170 1.000000e+00       2
AL590426.2.2    1.186090e-03  0.7384559 0.333 0.134 1.000000e+00       2
TESC            1.194696e-03 -0.4714209 0.133 0.098 1.000000e+00       2
TNKS.2          1.210614e-03  0.2966563 0.520 0.364 1.000000e+00       2
CTGF.2          1.216687e-03  0.2617512 0.613 0.445 1.000000e+00       2
FGF1.2          1.232631e-03 -1.0469044 0.147 0.270 1.000000e+00       2
BLM.2           1.238566e-03 -0.4610892 0.280 0.157 1.000000e+00       2
CSNK1E.1        1.256530e-03  0.3378670 0.520 0.397 1.000000e+00       2
NDUFAF8         1.286232e-03 -0.4637836 0.240 0.435 1.000000e+00       2
TNC.1           1.287752e-03  0.4526750 0.520 0.370 1.000000e+00       2
ISLR            1.307166e-03  0.3461633 0.280 0.164 1.000000e+00       2
ADAMTS10        1.325254e-03  0.4815851 0.333 0.200 1.000000e+00       2
ARPC5.1         1.364562e-03 -0.6448958 0.320 0.474 1.000000e+00       2
MYOF.1          1.375987e-03  0.7108100 0.493 0.394 1.000000e+00       2
KLHDC10.2       1.384202e-03  0.2672275 0.547 0.396 1.000000e+00       2
EPG5.1          1.394949e-03  0.7623441 0.440 0.309 1.000000e+00       2
GPRC5B.1        1.394949e-03  0.3858048 0.587 0.425 1.000000e+00       2
ZNF577.1        1.436130e-03  0.4515624 0.253 0.147 1.000000e+00       2
DGKA            1.442984e-03  0.3322159 0.360 0.216 1.000000e+00       2
AL928654.1.1    1.443815e-03  0.6826195 0.267 0.152 1.000000e+00       2
CDK2AP2         1.468875e-03 -0.7484479 0.187 0.408 1.000000e+00       2
ATE1            1.474104e-03  0.3564764 0.387 0.268 1.000000e+00       2
LRIG1           1.491659e-03  0.3075481 0.320 0.194 1.000000e+00       2
PAG1            1.506732e-03  0.7947122 0.573 0.426 1.000000e+00       2
ZNF14           1.517451e-03  0.4075897 0.173 0.110 1.000000e+00       2
CERS6.1         1.541841e-03  0.5067373 0.400 0.273 1.000000e+00       2
HSP90AB1        1.552795e-03 -0.7550241 0.307 0.478 1.000000e+00       2
NCL.1           1.558299e-03 -0.6185086 0.333 0.449 1.000000e+00       2
SLC29A3         1.567511e-03 -0.3251706 0.133 0.088 1.000000e+00       2
WFS1.1          1.587956e-03  0.8115542 0.413 0.166 1.000000e+00       2
KLF16           1.619085e-03  0.3881715 0.400 0.223 1.000000e+00       2
MYL6.2          1.619085e-03 -0.5840639 0.293 0.454 1.000000e+00       2
FXYD6           1.627665e-03 -0.5081670 0.253 0.167 1.000000e+00       2
AC004816.2      1.635332e-03 -0.6951841 0.173 0.150 1.000000e+00       2
LRRC75A.1       1.652707e-03 -1.0429778 0.187 0.396 1.000000e+00       2
HIST1H1E.1      1.656593e-03 -0.6046349 0.040 0.123 1.000000e+00       2
TMEM51.1        1.662408e-03  0.4357695 0.293 0.145 1.000000e+00       2
PSMA3.1         1.682046e-03 -0.7536856 0.280 0.458 1.000000e+00       2
SAMD4A.1        1.685993e-03  0.5616137 0.493 0.368 1.000000e+00       2
TNNC1.2         1.690930e-03 -0.3708605 0.267 0.207 1.000000e+00       2
F11R            1.691930e-03  0.6128670 0.387 0.200 1.000000e+00       2
CDC42EP2.1      1.695881e-03  0.2949417 0.320 0.180 1.000000e+00       2
SLC5A12.2       1.703860e-03 -0.4451347 0.147 0.133 1.000000e+00       2
CKAP2           1.713861e-03  0.2992279 0.413 0.300 1.000000e+00       2
KIF11.2         1.721899e-03 -0.5817801 0.053 0.151 1.000000e+00       2
BASP1.1         1.753388e-03 -0.6523671 0.293 0.484 1.000000e+00       2
DGKI.2          1.759541e-03  0.4812999 0.373 0.184 1.000000e+00       2
VDR.1           1.762626e-03 -0.8614707 0.093 0.236 1.000000e+00       2
OLFML2B.2       1.770865e-03  0.2895773 0.187 0.158 1.000000e+00       2
SLC4A8.2        1.808422e-03  0.2630682 0.413 0.296 1.000000e+00       2
SFRP1.2         1.824204e-03  0.2621477 0.373 0.281 1.000000e+00       2
MT-ND4L         1.829631e-03  0.2780209 0.573 0.396 1.000000e+00       2
ZER1.1          1.844596e-03  0.4802417 0.387 0.229 1.000000e+00       2
CDKN2B-AS1      1.858592e-03  0.3398355 0.307 0.184 1.000000e+00       2
AC128688.2.2    1.873773e-03  0.4140313 0.267 0.093 1.000000e+00       2
GUCY1A1.1       1.878083e-03  0.5426236 0.440 0.268 1.000000e+00       2
STK10           1.878135e-03  0.3206266 0.387 0.207 1.000000e+00       2
CCNI.1          1.904485e-03 -0.5236053 0.307 0.457 1.000000e+00       2
TENT2           1.912233e-03  0.7806371 0.467 0.280 1.000000e+00       2
C3orf70.1       1.925583e-03  0.4887229 0.253 0.135 1.000000e+00       2
COL7A1.1        1.968428e-03 -0.3156278 0.280 0.181 1.000000e+00       2
GUCY1B1.1       1.987879e-03  0.2679203 0.347 0.178 1.000000e+00       2
CEP128.2        1.995937e-03 -0.4805663 0.147 0.129 1.000000e+00       2
P3H1.1          2.034343e-03  1.0089343 0.547 0.436 1.000000e+00       2
PDS5B.2         2.043754e-03  0.4333956 0.587 0.404 1.000000e+00       2
PPIL4           2.080600e-03  0.5848201 0.427 0.363 1.000000e+00       2
TBC1D17         2.092613e-03  0.4450736 0.360 0.265 1.000000e+00       2
RPS27L          2.096233e-03 -0.4581441 0.280 0.457 1.000000e+00       2
IL17RA          2.104664e-03  0.4584635 0.267 0.146 1.000000e+00       2
ANTXR2.1        2.124154e-03  0.6612273 0.427 0.248 1.000000e+00       2
HDAC6           2.133944e-03  0.4000517 0.320 0.221 1.000000e+00       2
RBMS3.2         2.142547e-03  0.5772738 0.613 0.448 1.000000e+00       2
MIR222HG        2.163568e-03  0.5110535 0.267 0.191 1.000000e+00       2
ZNF850.2        2.167297e-03  0.4432841 0.373 0.191 1.000000e+00       2
DNAH14          2.213742e-03  0.4272483 0.280 0.180 1.000000e+00       2
HELZ2.1         2.241842e-03  0.6604589 0.280 0.128 1.000000e+00       2
TMEM147-AS1.1   2.252122e-03 -0.6330684 0.173 0.109 1.000000e+00       2
ZNF469.1        2.266349e-03  0.7931144 0.453 0.240 1.000000e+00       2
NPW.1           2.285870e-03  0.4210221 0.307 0.167 1.000000e+00       2
RBM6.1          2.309495e-03  0.3889272 0.480 0.369 1.000000e+00       2
ANO8.1          2.322710e-03  0.2846013 0.240 0.124 1.000000e+00       2
RPS6KA1.2       2.344014e-03  0.4496519 0.213 0.137 1.000000e+00       2
KCTD5.1         2.366840e-03  0.8437567 0.413 0.227 1.000000e+00       2
AL161669.3      2.396669e-03  0.6441364 0.240 0.123 1.000000e+00       2
NFAT5           2.396681e-03  0.3724652 0.573 0.473 1.000000e+00       2
METTL27.2       2.398044e-03 -0.2745751 0.160 0.115 1.000000e+00       2
LIN54.1         2.403507e-03  0.7578960 0.373 0.184 1.000000e+00       2
EIF1B.2         2.425484e-03 -0.7050104 0.307 0.459 1.000000e+00       2
SERPINF1.2      2.427862e-03 -1.0387799 0.213 0.342 1.000000e+00       2
UBE2G2          2.429627e-03  0.6751478 0.467 0.367 1.000000e+00       2
KANSL1L.2       2.456002e-03  0.2810979 0.373 0.220 1.000000e+00       2
TTC23           2.460189e-03  0.5619753 0.280 0.248 1.000000e+00       2
TENM3           2.462986e-03  0.8654385 0.507 0.344 1.000000e+00       2
MELTF.2         2.472780e-03  0.3302049 0.227 0.152 1.000000e+00       2
CDC6.2          2.503848e-03 -0.5443477 0.160 0.168 1.000000e+00       2
PDLIM5.1        2.528101e-03  0.7499504 0.547 0.410 1.000000e+00       2
RAB6B           2.546782e-03 -0.2692540 0.160 0.122 1.000000e+00       2
PSMD3           2.552561e-03  0.6309680 0.587 0.419 1.000000e+00       2
RELT            2.569956e-03  0.3802240 0.333 0.197 1.000000e+00       2
ESYT2.1         2.572865e-03  0.7415828 0.453 0.316 1.000000e+00       2
FZD6            2.609494e-03  0.3524980 0.560 0.443 1.000000e+00       2
SLC25A6.1       2.615399e-03 -0.4786220 0.320 0.449 1.000000e+00       2
C7orf50         2.621315e-03 -0.6676412 0.280 0.453 1.000000e+00       2
BRCC3.2         2.652578e-03  0.2580213 0.293 0.227 1.000000e+00       2
SLC22A23.1      2.697795e-03 -0.9598500 0.160 0.125 1.000000e+00       2
RND3.1          2.713073e-03  0.5904090 0.520 0.418 1.000000e+00       2
ABHD3           2.720722e-03  0.2821553 0.267 0.191 1.000000e+00       2
HSPH1.1         2.767038e-03  0.4797696 0.547 0.422 1.000000e+00       2
SCRN2           2.784198e-03  0.8279698 0.413 0.261 1.000000e+00       2
HMGA1           2.818814e-03 -0.3197115 0.400 0.288 1.000000e+00       2
ORC6            2.842630e-03 -0.5914191 0.133 0.200 1.000000e+00       2
CRTC1           2.845810e-03  0.2567230 0.280 0.184 1.000000e+00       2
GADD45B.2       2.852223e-03 -0.5511561 0.307 0.452 1.000000e+00       2
TRIO.2          2.900575e-03  0.3247096 0.440 0.324 1.000000e+00       2
LGALS3.2        2.907076e-03 -1.0335877 0.173 0.395 1.000000e+00       2
ZSCAN31.1       2.923393e-03  0.9795026 0.267 0.068 1.000000e+00       2
SLC2A13.1       2.938151e-03  0.5775283 0.467 0.308 1.000000e+00       2
SPRED1.1        2.971187e-03  0.9069199 0.467 0.325 1.000000e+00       2
SNHG12.2        2.986159e-03 -0.4068393 0.360 0.266 1.000000e+00       2
ACTR3.1         2.991171e-03  0.5655801 0.533 0.417 1.000000e+00       2
ZMAT3.1         3.002884e-03  0.4041020 0.600 0.426 1.000000e+00       2
RASAL2.1        3.016321e-03  0.4730029 0.467 0.287 1.000000e+00       2
CST3.2          3.019691e-03 -0.4970427 0.253 0.463 1.000000e+00       2
HECW1.2         3.021371e-03 -0.5847737 0.027 0.118 1.000000e+00       2
SLC2A3          3.023057e-03  0.5018748 0.427 0.229 1.000000e+00       2
TRNP1           3.065496e-03  0.2759852 0.307 0.285 1.000000e+00       2
IFITM3.2        3.077467e-03 -0.9411178 0.320 0.441 1.000000e+00       2
TSPAN15         3.091216e-03 -0.8711144 0.227 0.414 1.000000e+00       2
GKAP1           3.113657e-03 -0.7326949 0.107 0.115 1.000000e+00       2
HSPG2.1         3.146748e-03  0.6896064 0.600 0.434 1.000000e+00       2
LIMK2.2         3.162529e-03  0.3135074 0.533 0.427 1.000000e+00       2
AC124798.1.2    3.167804e-03 -0.2836733 0.280 0.194 1.000000e+00       2
CCND1.2         3.176617e-03  0.6694466 0.587 0.435 1.000000e+00       2
UGCG.1          3.196080e-03  0.7457554 0.373 0.305 1.000000e+00       2
MAP3K2.1        3.258738e-03  0.8043052 0.480 0.368 1.000000e+00       2
EMILIN1.2       3.298679e-03 -0.4670698 0.347 0.238 1.000000e+00       2
HOXB-AS3.2      3.331714e-03 -0.4152095 0.213 0.191 1.000000e+00       2
PBXIP1.1        3.383682e-03  0.8045348 0.440 0.298 1.000000e+00       2
NUFIP2          3.415593e-03  0.2672743 0.600 0.456 1.000000e+00       2
GPT2.1          3.432596e-03  0.6898389 0.440 0.289 1.000000e+00       2
ARL4A           3.455389e-03 -0.7437348 0.307 0.441 1.000000e+00       2
AC245041.2.1    3.489834e-03 -0.5255701 0.160 0.252 1.000000e+00       2
ACSL3           3.597035e-03  0.2825526 0.600 0.438 1.000000e+00       2
TMEM132A        3.622806e-03  0.2633786 0.453 0.324 1.000000e+00       2
PLEKHH2.2       3.626759e-03  0.4422228 0.240 0.155 1.000000e+00       2
SLC2A5.2        3.646742e-03 -0.3047224 0.173 0.130 1.000000e+00       2
HMGN3.2         3.652752e-03 -0.4411886 0.293 0.475 1.000000e+00       2
ATOH8           3.670819e-03  0.2910076 0.173 0.111 1.000000e+00       2
AC103740.2.2    3.735736e-03 -0.7535048 0.040 0.103 1.000000e+00       2
PRRC2A.2        3.750094e-03  1.3775390 0.467 0.359 1.000000e+00       2
AREL1           3.756256e-03  0.4348601 0.333 0.190 1.000000e+00       2
MCM3AP-AS1.2    3.764478e-03 -0.4935696 0.227 0.104 1.000000e+00       2
NOD1            3.797555e-03 -0.5448354 0.187 0.110 1.000000e+00       2
PIFO            3.818372e-03 -0.5446123 0.187 0.118 1.000000e+00       2
CSNK1G2         3.824643e-03  0.5457143 0.453 0.378 1.000000e+00       2
LMO4.2          3.828822e-03 -1.0156018 0.280 0.439 1.000000e+00       2
RASSF1.1        3.841391e-03  0.5891477 0.413 0.320 1.000000e+00       2
MYH9.1          3.894159e-03  0.2557392 0.560 0.427 1.000000e+00       2
ZBTB20          3.900534e-03  0.3821268 0.480 0.404 1.000000e+00       2
CHST6.1         3.917567e-03  0.6193903 0.280 0.101 1.000000e+00       2
MTMR3.1         4.021310e-03  0.5800739 0.387 0.255 1.000000e+00       2
FAH.2           4.036634e-03 -1.3066828 0.173 0.332 1.000000e+00       2
GGT5.2          4.047619e-03 -0.4875238 0.227 0.213 1.000000e+00       2
SAA1.2          4.061829e-03 -2.1563393 0.160 0.341 1.000000e+00       2
LIFR.1          4.102960e-03  0.5541438 0.427 0.267 1.000000e+00       2
BTG3.2          4.111878e-03  0.3946661 0.467 0.385 1.000000e+00       2
APLP1           4.154470e-03  0.4197784 0.280 0.140 1.000000e+00       2
ABCC5           4.199742e-03  0.2833879 0.493 0.384 1.000000e+00       2
LMCD1           4.277727e-03 -0.2786692 0.253 0.441 1.000000e+00       2
NFKBID.1        4.298573e-03 -0.5701455 0.093 0.127 1.000000e+00       2
HAGH            4.314878e-03 -0.5284888 0.267 0.472 1.000000e+00       2
C19orf48.1      4.317208e-03 -0.3445451 0.333 0.263 1.000000e+00       2
SRPX.2          4.321787e-03  0.2846727 0.440 0.356 1.000000e+00       2
CRAMP1          4.331158e-03  0.4118827 0.360 0.184 1.000000e+00       2
PRKG1.2         4.354667e-03  0.3563571 0.347 0.196 1.000000e+00       2
AC112220.3.2    4.368757e-03 -0.6435720 0.173 0.153 1.000000e+00       2
FTX             4.492953e-03  0.4392319 0.547 0.418 1.000000e+00       2
WLS.1           4.497792e-03  0.4359181 0.587 0.452 1.000000e+00       2
TRPM7.2         4.505057e-03  0.3057912 0.480 0.293 1.000000e+00       2
UACA.2          4.509905e-03  0.5277445 0.480 0.418 1.000000e+00       2
PRDX3.2         4.536670e-03 -0.6638904 0.280 0.480 1.000000e+00       2
AFP             4.541539e-03 -0.6246883 0.080 0.120 1.000000e+00       2
EPHB3.1         4.561055e-03  0.7461525 0.227 0.122 1.000000e+00       2
ADGRV1.2        4.580760e-03 -0.4058096 0.280 0.145 1.000000e+00       2
TNFAIP1.2       4.600497e-03  0.4344744 0.560 0.434 1.000000e+00       2
DYRK1A          4.695246e-03  0.3087868 0.453 0.277 1.000000e+00       2
TOB1.2          4.771293e-03  0.5169205 0.453 0.348 1.000000e+00       2
PILRB           4.802022e-03 -0.6651762 0.147 0.219 1.000000e+00       2
CCDC9B.1        4.830347e-03  0.5967497 0.387 0.205 1.000000e+00       2
P4HA2.2         4.845862e-03  0.7700877 0.560 0.426 1.000000e+00       2
DNA2            4.879614e-03 -0.4458567 0.227 0.130 1.000000e+00       2
CD1D.1          4.884843e-03  0.4718294 0.107 0.096 1.000000e+00       2
ZFP36L1         4.979562e-03  0.3626903 0.640 0.457 1.000000e+00       2
ENC1.2          4.995503e-03  0.3937896 0.573 0.364 1.000000e+00       2
TBC1D31         5.032877e-03 -0.7863245 0.067 0.137 1.000000e+00       2
CAMSAP1         5.100293e-03  0.3949964 0.373 0.251 1.000000e+00       2
OPHN1.1         5.108408e-03 -0.6696356 0.120 0.157 1.000000e+00       2
BACE1-AS.2      5.113849e-03 -0.7977896 0.027 0.144 1.000000e+00       2
SULT1B1         5.149271e-03 -0.7448784 0.240 0.165 1.000000e+00       2
LRRC38.2        5.176693e-03 -0.9864557 0.213 0.208 1.000000e+00       2
ZNF251.2        5.237516e-03  0.2589118 0.227 0.133 1.000000e+00       2
KIAA1324L       5.248633e-03 -0.5610654 0.240 0.132 1.000000e+00       2
ARID5A          5.251432e-03 -0.6955913 0.147 0.153 1.000000e+00       2
IRS2.2          5.284954e-03 -0.2594994 0.347 0.311 1.000000e+00       2
CCDC151         5.324293e-03 -0.4529086 0.227 0.137 1.000000e+00       2
PRKRIP1         5.361060e-03  0.3476208 0.413 0.279 1.000000e+00       2
KREMEN1.2       5.449649e-03 -0.5776777 0.253 0.173 1.000000e+00       2
CFL2            5.487215e-03 -0.7652762 0.253 0.420 1.000000e+00       2
ATP10D.1        5.501716e-03  0.3128943 0.440 0.337 1.000000e+00       2
P4HA1.1         5.563012e-03  0.5739306 0.560 0.441 1.000000e+00       2
IGFBP7.2        5.620421e-03  0.3099176 0.627 0.446 1.000000e+00       2
AC048341.1      5.633814e-03  0.3779468 0.387 0.292 1.000000e+00       2
WARS.2          5.747578e-03  0.3562375 0.427 0.333 1.000000e+00       2
WDR24.1         5.762707e-03  0.4237297 0.333 0.184 1.000000e+00       2
NPTXR.2         5.765735e-03  0.4724561 0.453 0.249 1.000000e+00       2
NRBP2.1         5.768766e-03  0.8510432 0.427 0.305 1.000000e+00       2
FN3K            5.829673e-03 -0.3827625 0.107 0.143 1.000000e+00       2
AC034236.2.1    5.835787e-03 -0.4104047 0.187 0.158 1.000000e+00       2
HSPD1           5.872656e-03 -0.5465514 0.267 0.462 1.000000e+00       2
CASP6.2         5.903528e-03 -0.5341929 0.160 0.247 1.000000e+00       2
PATJ.2          5.909721e-03 -0.3824116 0.200 0.140 1.000000e+00       2
C3AR1.1         6.009595e-03 -0.3685167 0.227 0.131 1.000000e+00       2
AC008467.1.2    6.060061e-03 -0.8966544 0.053 0.146 1.000000e+00       2
SMARCE1.2       6.107807e-03  0.5062626 0.347 0.297 1.000000e+00       2
SLC25A17        6.133382e-03  0.6316445 0.373 0.285 1.000000e+00       2
S1PR2           6.171814e-03  0.6350408 0.213 0.139 1.000000e+00       2
NAV3.2          6.197701e-03  0.6116075 0.307 0.177 1.000000e+00       2
TCF4.1          6.217178e-03  0.4580535 0.547 0.397 1.000000e+00       2
COL14A1.2       6.334850e-03  0.2618122 0.280 0.167 1.000000e+00       2
KLHL21.1        6.348103e-03  0.5221143 0.400 0.288 1.000000e+00       2
SV2A            6.371258e-03 -0.3714942 0.107 0.139 1.000000e+00       2
SEL1L3          6.424497e-03  0.8017916 0.227 0.134 1.000000e+00       2
FGF2.2          6.464683e-03  0.2748511 0.400 0.291 1.000000e+00       2
ARHGAP44        6.471406e-03  0.3286246 0.147 0.141 1.000000e+00       2
PRPF8           6.488240e-03  0.5818729 0.560 0.395 1.000000e+00       2
LINC02086       6.525367e-03  0.2728454 0.293 0.172 1.000000e+00       2
RGS20.2         6.627656e-03 -0.8841230 0.147 0.148 1.000000e+00       2
ORAI3.1         6.658696e-03 -0.7502899 0.187 0.390 1.000000e+00       2
TMBIM1.2        6.665597e-03  0.6070732 0.360 0.271 1.000000e+00       2
AC097103.2.1    6.689796e-03 -0.5543634 0.133 0.186 1.000000e+00       2
MSANTD2.2       6.727996e-03  0.5426026 0.307 0.207 1.000000e+00       2
GAN.1           6.731479e-03  0.5234871 0.413 0.335 1.000000e+00       2
ZNF804A         6.857911e-03 -0.3335628 0.373 0.229 1.000000e+00       2
CNN3.2          6.868550e-03 -0.5324061 0.280 0.465 1.000000e+00       2
GLIS3.1         6.893414e-03  0.4573899 0.440 0.280 1.000000e+00       2
TCEAL3.1        6.943388e-03 -0.5807073 0.293 0.436 1.000000e+00       2
CCDC80          6.957684e-03  0.6309651 0.520 0.414 1.000000e+00       2
GCLM.2          7.150231e-03  0.2817637 0.507 0.369 1.000000e+00       2
CLCN3           7.212986e-03  0.4807911 0.507 0.359 1.000000e+00       2
LSAMP.2         7.235170e-03 -0.5384839 0.240 0.220 1.000000e+00       2
IFITM2.2        7.235224e-03 -1.1689458 0.093 0.253 1.000000e+00       2
LMO7.2          7.250131e-03  0.4480235 0.587 0.405 1.000000e+00       2
TIFA.2          7.302408e-03  0.7950935 0.160 0.123 1.000000e+00       2
SPATA5L1.2      7.370124e-03 -0.4516686 0.213 0.168 1.000000e+00       2
FZD3.1          7.373913e-03  0.4704277 0.333 0.267 1.000000e+00       2
ASIC1           7.377684e-03 -0.7121322 0.120 0.160 1.000000e+00       2
CAV1.2          7.404164e-03 -0.8447359 0.227 0.412 1.000000e+00       2
KMO.2           7.430808e-03 -0.5307445 0.147 0.142 1.000000e+00       2
CPZ.2           7.461278e-03 -0.7094939 0.160 0.145 1.000000e+00       2
YJU2.1          7.468911e-03  0.3377840 0.253 0.183 1.000000e+00       2
BOLA2B          7.538070e-03 -0.3394859 0.227 0.106 1.000000e+00       2
FOSL2           7.565108e-03  0.7242629 0.573 0.409 1.000000e+00       2
LAMA5.1         7.623321e-03  0.7751352 0.347 0.215 1.000000e+00       2
SARS            7.635016e-03 -0.5144698 0.360 0.508 1.000000e+00       2
WBP1L           7.952767e-03  0.6260122 0.507 0.392 1.000000e+00       2
MGAT5.1         7.968955e-03  0.4410486 0.440 0.284 1.000000e+00       2
HS6ST2.2        8.001436e-03  0.8261931 0.547 0.411 1.000000e+00       2
SCD             8.124283e-03  0.3618143 0.547 0.414 1.000000e+00       2
WNK1.1          8.178040e-03  0.4712603 0.587 0.459 1.000000e+00       2
TCEAL9.2        8.232113e-03 -0.6450600 0.253 0.421 1.000000e+00       2
RSPO3.2         8.236235e-03 -0.4000015 0.347 0.282 1.000000e+00       2
SLIT2.1         8.417509e-03  0.6929219 0.267 0.183 1.000000e+00       2
ZNF431.1        8.430288e-03  0.6330928 0.293 0.263 1.000000e+00       2
MYL7.2          8.451579e-03  0.3323252 0.280 0.171 1.000000e+00       2
GPR161.2        8.455800e-03 -0.7898051 0.200 0.142 1.000000e+00       2
FUCA1           8.515903e-03 -0.5570793 0.187 0.249 1.000000e+00       2
MED13L          8.571972e-03  0.5876888 0.533 0.453 1.000000e+00       2
IER2.1          8.720171e-03 -0.8013891 0.213 0.393 1.000000e+00       2
HYI             8.746561e-03 -0.3808464 0.280 0.295 1.000000e+00       2
S100A3          8.750959e-03  0.3598766 0.120 0.101 1.000000e+00       2
FBXO6           8.755363e-03  0.3506812 0.120 0.112 1.000000e+00       2
ATP6V0B         8.799555e-03 -0.4460553 0.360 0.464 1.000000e+00       2
PLEKHF1         8.870647e-03  0.3649298 0.240 0.136 1.000000e+00       2
ZNF430          8.910878e-03  0.6137967 0.373 0.238 1.000000e+00       2
FAM89A.2        8.978274e-03 -0.7685957 0.293 0.447 1.000000e+00       2
SORBS2.1        9.000760e-03  0.6115645 0.400 0.270 1.000000e+00       2
HOPX.1          9.011118e-03 -0.7128101 0.253 0.396 1.000000e+00       2
PLCE1.2         9.077940e-03  0.3128999 0.453 0.244 1.000000e+00       2
SLC25A45.1      9.100735e-03 -0.7823182 0.107 0.204 1.000000e+00       2
ANO7.1          9.109862e-03 -0.4560479 0.253 0.167 1.000000e+00       2
PLA2G4C.2       9.109864e-03 -0.3499382 0.147 0.107 1.000000e+00       2
NEDD9           9.183202e-03  0.5395649 0.547 0.411 1.000000e+00       2
AGAP3           9.243178e-03  0.2984522 0.413 0.317 1.000000e+00       2
AL121603.2      9.257047e-03 -0.7050924 0.000 0.159 1.000000e+00       2
LAD1            9.420493e-03 -0.6144525 0.133 0.222 1.000000e+00       2
RASSF2.2        9.453453e-03 -0.6824070 0.120 0.193 1.000000e+00       2
BRD1.1          9.519773e-03  0.8473164 0.400 0.284 1.000000e+00       2
LINC02334.1     9.624729e-03  0.4814770 0.147 0.121 1.000000e+00       2
KIAA0895        9.692159e-03 -0.3970718 0.200 0.137 1.000000e+00       2
IFI16.1         9.701821e-03  0.6264058 0.507 0.425 1.000000e+00       2
TC2N.1          9.798856e-03  0.7607379 0.427 0.196 1.000000e+00       2
BCAR3.2         9.818384e-03  0.6888330 0.360 0.219 1.000000e+00       2
PCDH19.1        9.857513e-03  0.5411719 0.227 0.109 1.000000e+00       2
NKTR.2          9.867321e-03  0.4681924 0.533 0.427 1.000000e+00       2
KDR.2           9.916349e-03 -0.7606716 0.240 0.194 1.000000e+00       2
STON1           9.970786e-03  0.3293411 0.440 0.288 1.000000e+00       2
HSPA13          9.995569e-03  0.7581451 0.427 0.355 1.000000e+00       2
MDFI.2          1.000550e-02 -0.4323942 0.227 0.439 1.000000e+00       2
DYRK3           1.004026e-02  0.3546475 0.280 0.198 1.000000e+00       2
HLA-B.2         1.007513e-02 -0.2980730 0.213 0.404 1.000000e+00       2
MINPP1          1.008023e-02  0.3744932 0.440 0.340 1.000000e+00       2
AL357055.3.2    1.013028e-02 -0.8329900 0.040 0.129 1.000000e+00       2
RHOB.2          1.018064e-02  0.2903206 0.493 0.430 1.000000e+00       2
AC139795.3.1    1.027683e-02 -0.4508223 0.133 0.119 1.000000e+00       2
PDIA6           1.032272e-02  0.5188405 0.573 0.455 1.000000e+00       2
FZD8.1          1.037901e-02  0.4437401 0.427 0.264 1.000000e+00       2
HES4.1          1.043043e-02 -0.7864180 0.227 0.327 1.000000e+00       2
PAN2            1.047687e-02  0.2759269 0.293 0.161 1.000000e+00       2
CNTN5.1         1.049242e-02  0.6367152 0.293 0.121 1.000000e+00       2
LRP5.1          1.054434e-02  0.2866504 0.333 0.212 1.000000e+00       2
PITX1.2         1.058601e-02 -0.3998910 0.307 0.424 1.000000e+00       2
ZNF670          1.075420e-02 -0.7667902 0.133 0.121 1.000000e+00       2
RPP25.2         1.089274e-02 -0.7742538 0.200 0.409 1.000000e+00       2
PIP5K1A         1.120732e-02  0.5269916 0.373 0.269 1.000000e+00       2
RALBP1.1        1.122381e-02 -0.5524918 0.320 0.458 1.000000e+00       2
F2R.1           1.127329e-02  0.5777779 0.360 0.228 1.000000e+00       2
UBR7            1.128439e-02 -0.8577686 0.133 0.177 1.000000e+00       2
CALM1.2         1.134539e-02 -0.6357762 0.320 0.472 1.000000e+00       2
BOD1.2          1.137318e-02 -0.5274193 0.267 0.461 1.000000e+00       2
AC021242.3.2    1.145131e-02 -0.5483114 0.267 0.129 1.000000e+00       2
TBX18           1.156929e-02  0.3252213 0.200 0.144 1.000000e+00       2
ZNF93           1.159770e-02 -1.0872447 0.067 0.185 1.000000e+00       2
FAM219B         1.159771e-02  0.3541119 0.427 0.298 1.000000e+00       2
FAM84A.2        1.160903e-02 -0.4728973 0.067 0.154 1.000000e+00       2
RRAS            1.162038e-02 -0.5716290 0.293 0.417 1.000000e+00       2
BNC2            1.168292e-02  0.6131368 0.573 0.424 1.000000e+00       2
PRELP           1.168856e-02 -0.9418814 0.160 0.149 1.000000e+00       2
GINS1.2         1.169431e-02 -0.4530273 0.147 0.133 1.000000e+00       2
FAM241A         1.171143e-02 -0.3070762 0.213 0.172 1.000000e+00       2
HOXB9           1.171716e-02 -0.2503927 0.453 0.330 1.000000e+00       2
ADAMTS3.1       1.189548e-02 -0.3247097 0.147 0.115 1.000000e+00       2
PIK3R1.2        1.189548e-02 -0.3953541 0.147 0.245 1.000000e+00       2
PCF11           1.206447e-02  0.2567622 0.400 0.302 1.000000e+00       2
ST7-AS1.1       1.212307e-02  0.3740785 0.213 0.140 1.000000e+00       2
HAND1.2         1.215861e-02 -0.7500921 0.173 0.206 1.000000e+00       2
HSD17B14        1.217634e-02  0.3551808 0.440 0.323 1.000000e+00       2
EIF5.2          1.226531e-02 -0.8129219 0.373 0.468 1.000000e+00       2
FNDC5           1.239081e-02 -0.7705061 0.120 0.224 1.000000e+00       2
PTPN7           1.240282e-02 -0.5960190 0.107 0.119 1.000000e+00       2
AMPD2           1.244496e-02  0.2774639 0.373 0.300 1.000000e+00       2
SAMD5           1.252352e-02 -0.4081766 0.187 0.110 1.000000e+00       2
AC124283.1.2    1.259025e-02 -0.9180871 0.107 0.176 1.000000e+00       2
ARHGEF26.2      1.265751e-02 -0.7500263 0.080 0.210 1.000000e+00       2
TTC21B.2        1.274341e-02  0.3219918 0.240 0.129 1.000000e+00       2
ANKRD36C.1      1.275564e-02  0.6718783 0.307 0.176 1.000000e+00       2
SEPSECS.2       1.286702e-02 -0.5769579 0.093 0.148 1.000000e+00       2
PRRT2           1.297921e-02 -0.2737906 0.200 0.143 1.000000e+00       2
NRM             1.302932e-02 -0.8481355 0.107 0.140 1.000000e+00       2
RAB32           1.304192e-02 -0.5056507 0.227 0.449 1.000000e+00       2
TAB3.1          1.309847e-02  0.8318291 0.320 0.155 1.000000e+00       2
PURPL           1.311749e-02 -0.3820890 0.280 0.262 1.000000e+00       2
PRPS1.1         1.319983e-02  0.5215566 0.347 0.296 1.000000e+00       2
NASP.2          1.322525e-02 -0.3441883 0.347 0.314 1.000000e+00       2
HRAS            1.323797e-02 -0.6865157 0.293 0.477 1.000000e+00       2
SRM             1.331456e-02 -0.5477265 0.373 0.463 1.000000e+00       2
VASH2           1.368372e-02 -0.3646762 0.120 0.099 1.000000e+00       2
USP51           1.369683e-02 -0.2697039 0.160 0.101 1.000000e+00       2
RCL1            1.372303e-02  0.4037546 0.307 0.200 1.000000e+00       2
NUPR1.1         1.375598e-02 -0.5509634 0.253 0.443 1.000000e+00       2
DENND1B.1       1.377574e-02  0.4607599 0.387 0.246 1.000000e+00       2
NFATC2          1.396144e-02 -0.7218963 0.040 0.171 1.000000e+00       2
C9orf3          1.396816e-02 -0.3265047 0.280 0.469 1.000000e+00       2
CYP4V2          1.397483e-02 -0.4698366 0.080 0.108 1.000000e+00       2
CDCA3.2         1.421707e-02 -0.4910513 0.093 0.111 1.000000e+00       2
TPGS2.2         1.423065e-02 -0.6194128 0.320 0.457 1.000000e+00       2
ERBIN           1.429184e-02  0.4715944 0.507 0.380 1.000000e+00       2
ID2.2           1.435324e-02 -0.8400823 0.093 0.189 1.000000e+00       2
TAOK1.1         1.439433e-02  0.6111730 0.520 0.441 1.000000e+00       2
C19orf33.1      1.444921e-02  0.6140266 0.200 0.159 1.000000e+00       2
PRKAR2B.2       1.452505e-02 -0.6444676 0.133 0.135 1.000000e+00       2
AC010531.6      1.455966e-02 -0.3238828 0.160 0.087 1.000000e+00       2
MANBA.1         1.464993e-02  0.3577725 0.427 0.329 1.000000e+00       2
ZBTB16.2        1.475454e-02 -0.6960508 0.267 0.196 1.000000e+00       2
PLXNB1          1.478273e-02  0.4522017 0.307 0.209 1.000000e+00       2
UCP2            1.483893e-02 -0.5079205 0.413 0.309 1.000000e+00       2
HMG20A.1        1.503014e-02  0.4458277 0.333 0.277 1.000000e+00       2
CRABP2          1.510857e-02 -0.4309369 0.147 0.357 1.000000e+00       2
SELENOM         1.517316e-02 -0.3844491 0.320 0.492 1.000000e+00       2
PTBP3.1         1.518752e-02  0.4320354 0.547 0.427 1.000000e+00       2
TRIB3.2         1.528119e-02 -0.4047763 0.320 0.219 1.000000e+00       2
DOCK1.2         1.536811e-02 -0.7696997 0.200 0.295 1.000000e+00       2
PRKAR1B         1.548468e-02  0.4150468 0.293 0.123 1.000000e+00       2
SIPA1L2.1       1.555788e-02  0.6383482 0.280 0.173 1.000000e+00       2
CCT6B           1.561664e-02 -0.8948362 0.080 0.173 1.000000e+00       2
ZNF669.1        1.564617e-02  0.5808739 0.227 0.145 1.000000e+00       2
SLC4A1AP.2      1.567576e-02 -0.4271808 0.333 0.292 1.000000e+00       2
SPCS3.1         1.577210e-02  0.7263846 0.587 0.452 1.000000e+00       2
ADAM33.1        1.584655e-02  0.3252986 0.333 0.191 1.000000e+00       2
PLXDC2.1        1.598883e-02 -0.4999502 0.467 0.350 1.000000e+00       2
AL138966.2.2    1.604909e-02 -0.2759827 0.120 0.117 1.000000e+00       2
PRTFDC1         1.606418e-02  0.3596369 0.160 0.098 1.000000e+00       2
HOXA7.2         1.626918e-02 -0.3117548 0.293 0.288 1.000000e+00       2
SRI             1.629975e-02 -0.6133111 0.253 0.461 1.000000e+00       2
PPFIBP1.2       1.642251e-02  0.3694957 0.507 0.412 1.000000e+00       2
RAI14.1         1.653059e-02  0.2575593 0.560 0.429 1.000000e+00       2
ASRGL1          1.656934e-02 -0.3875466 0.160 0.129 1.000000e+00       2
TBCE.1.1        1.660028e-02  0.7304175 0.267 0.144 1.000000e+00       2
F2RL1.1         1.670161e-02  0.4363877 0.360 0.246 1.000000e+00       2
AC097359.2.1    1.677204e-02  1.1605845 0.333 0.171 1.000000e+00       2
ARMCX2          1.692962e-02  0.4534603 0.560 0.437 1.000000e+00       2
MRPL44          1.719229e-02 -0.2621362 0.333 0.350 1.000000e+00       2
ENTPD6.1        1.720024e-02  0.6224541 0.240 0.152 1.000000e+00       2
SLC26A6.2       1.734496e-02 -0.8082943 0.213 0.153 1.000000e+00       2
PACSIN3.1       1.738557e-02 -0.4563438 0.253 0.271 1.000000e+00       2
GREB1L          1.739366e-02  0.3232550 0.360 0.267 1.000000e+00       2
CAVIN1.1        1.740987e-02  0.6965286 0.533 0.449 1.000000e+00       2
TIA1            1.762165e-02  0.5043410 0.453 0.294 1.000000e+00       2
LRRC8A          1.762984e-02  0.3393253 0.387 0.283 1.000000e+00       2
COQ8A.2         1.770362e-02 -0.5866239 0.160 0.145 1.000000e+00       2
HTRA1.1         1.789191e-02 -1.4494507 0.240 0.440 1.000000e+00       2
CCPG1           1.816945e-02  0.2574031 0.613 0.443 1.000000e+00       2
WASF3.2         1.818628e-02  0.3146448 0.240 0.223 1.000000e+00       2
ERAP2           1.825362e-02 -0.7862038 0.120 0.196 1.000000e+00       2
GRPEL2          1.827909e-02  0.3221324 0.280 0.232 1.000000e+00       2
BLMH.2          1.844037e-02 -0.4437183 0.160 0.262 1.000000e+00       2
ZCRB1           1.848302e-02 -0.6840961 0.253 0.449 1.000000e+00       2
FEZ2            1.852575e-02  0.3155613 0.413 0.375 1.000000e+00       2
ZBED9           1.881861e-02 -0.4830699 0.107 0.112 1.000000e+00       2
TPX2.1          1.894038e-02 -0.9362542 0.080 0.192 1.000000e+00       2
PRKCI.1         1.906291e-02  0.5679055 0.427 0.379 1.000000e+00       2
HIGD1A.2        1.915081e-02 -0.4010137 0.307 0.442 1.000000e+00       2
MPC2.2          1.922139e-02 -0.5177236 0.333 0.445 1.000000e+00       2
SOD1.1          1.929218e-02 -0.6647431 0.333 0.445 1.000000e+00       2
APBB3.2         1.934542e-02 -0.6709508 0.133 0.167 1.000000e+00       2
CIP2A           1.950585e-02 -0.7226462 0.133 0.153 1.000000e+00       2
PIK3C3          1.980328e-02  0.3404177 0.293 0.276 1.000000e+00       2
CXCL2.2         1.986684e-02 -0.4623147 0.160 0.110 1.000000e+00       2
MT-ND6          1.991237e-02  0.5813169 0.533 0.412 1.000000e+00       2
RNASEH2C.2      2.002200e-02 -0.4597071 0.293 0.437 1.000000e+00       2
DST             2.020586e-02  0.4850753 0.587 0.449 1.000000e+00       2
PHKA2           2.032609e-02  0.4333473 0.173 0.126 1.000000e+00       2
CPPED1.2        2.035401e-02 -0.3564009 0.347 0.304 1.000000e+00       2
ZNF101          2.050309e-02 -0.6044877 0.067 0.155 1.000000e+00       2
RBM3.1          2.059679e-02 -0.4706362 0.293 0.449 1.000000e+00       2
GM2A            2.063436e-02  0.5395083 0.347 0.268 1.000000e+00       2
SSX2IP          2.063436e-02 -0.3726411 0.240 0.213 1.000000e+00       2
ZNF43.1         2.084208e-02  0.5415360 0.227 0.192 1.000000e+00       2
FAM160B2.1      2.086104e-02  0.6711748 0.387 0.268 1.000000e+00       2
LDHA.1          2.088954e-02 -0.5462245 0.293 0.434 1.000000e+00       2
CCDC74A.2       2.092757e-02 -0.4866506 0.160 0.135 1.000000e+00       2
TMEM65.1        2.095613e-02 -0.5044373 0.293 0.263 1.000000e+00       2
ZBTB7B.2        2.096567e-02  0.5259619 0.360 0.213 1.000000e+00       2
ERV3-1          2.098457e-02  0.4283233 0.200 0.179 1.000000e+00       2
MED26           2.106100e-02  0.4239799 0.267 0.143 1.000000e+00       2
HMGA2.2         2.115707e-02  0.3471557 0.360 0.283 1.000000e+00       2
RNF112.2        2.135968e-02  0.3429243 0.253 0.175 1.000000e+00       2
CBX2            2.150541e-02 -0.3843809 0.093 0.095 1.000000e+00       2
TADA1.1         2.166178e-02 -1.0121380 0.067 0.160 1.000000e+00       2
CHSY1           2.175027e-02  0.7338532 0.520 0.372 1.000000e+00       2
PIM2.1          2.191814e-02  0.6043949 0.240 0.128 1.000000e+00       2
LMO7-AS1.1      2.193796e-02 -0.6478664 0.307 0.431 1.000000e+00       2
KCNJ2.2         2.231759e-02 -0.3228378 0.133 0.137 1.000000e+00       2
STRBP           2.234758e-02  0.3681142 0.173 0.165 1.000000e+00       2
ADAM28          2.243865e-02 -0.3072138 0.160 0.122 1.000000e+00       2
PDHA1           2.252982e-02 -0.5817537 0.293 0.382 1.000000e+00       2
PELI3.2         2.253971e-02 -0.8201662 0.160 0.120 1.000000e+00       2
HCFC1R1         2.257045e-02 -0.4838504 0.307 0.422 1.000000e+00       2
TEX22.2         2.258061e-02 -0.3274563 0.227 0.181 1.000000e+00       2
THBS2.2         2.269260e-02  0.3884932 0.413 0.298 1.000000e+00       2
AIMP1.1         2.287716e-02  0.6294177 0.520 0.421 1.000000e+00       2
TNFSF15.2       2.296987e-02 -0.2719872 0.213 0.164 1.000000e+00       2
DHCR7.2         2.302153e-02  0.3130361 0.547 0.429 1.000000e+00       2
ECHDC2.1        2.319787e-02 -0.4751810 0.173 0.116 1.000000e+00       2
KRBA2           2.327070e-02 -0.5928984 0.093 0.144 1.000000e+00       2
COL4A5.1        2.332310e-02  0.4021633 0.240 0.203 1.000000e+00       2
ZFYVE21.1       2.336496e-02 -0.6654508 0.227 0.418 1.000000e+00       2
UBASH3B.2       2.350147e-02  0.5045413 0.480 0.317 1.000000e+00       2
DKK3            2.357526e-02  0.3384889 0.507 0.433 1.000000e+00       2
USP15.1         2.370222e-02  0.5389422 0.560 0.425 1.000000e+00       2
ADGRG6.2        2.385101e-02 -0.4876711 0.107 0.161 1.000000e+00       2
SGTA.2          2.395792e-02  0.2602770 0.427 0.285 1.000000e+00       2
TFPI2.2         2.401134e-02  0.5756417 0.267 0.208 1.000000e+00       2
TCF21           2.418363e-02  0.2862175 0.253 0.215 1.000000e+00       2
AC011978.2.2    2.443295e-02 -0.2537177 0.133 0.145 1.000000e+00       2
CBFB            2.443295e-02  0.2724980 0.400 0.328 1.000000e+00       2
QPRT            2.448745e-02 -0.6951878 0.320 0.420 1.000000e+00       2
GLRX            2.458571e-02 -0.8176687 0.347 0.435 1.000000e+00       2
HEXIM1          2.462962e-02  0.4052436 0.387 0.304 1.000000e+00       2
SNHG8.1         2.467351e-02 -0.5259556 0.267 0.444 1.000000e+00       2
ST3GAL1.1       2.483869e-02  0.5306072 0.333 0.207 1.000000e+00       2
ISG15.2         2.487185e-02 -0.2534987 0.227 0.372 1.000000e+00       2
CCDC107         2.494936e-02 -0.6721590 0.267 0.457 1.000000e+00       2
UBXN2A.2        2.500484e-02  0.6291204 0.280 0.297 1.000000e+00       2
EBF1.2          2.526133e-02 -0.3912302 0.240 0.201 1.000000e+00       2
ZNF791          2.544134e-02  0.4852464 0.387 0.336 1.000000e+00       2
RCN1            2.559963e-02  0.3501812 0.547 0.426 1.000000e+00       2
SEPT6           2.587296e-02 -0.3059161 0.240 0.411 1.000000e+00       2
CDH6.2          2.606792e-02 -0.2656505 0.213 0.267 1.000000e+00       2
PCSK6.2         2.620658e-02 -0.2987625 0.307 0.201 1.000000e+00       2
HOXC5           2.634567e-02  0.6171906 0.187 0.105 1.000000e+00       2
PDPN.2          2.635568e-02 -1.1846135 0.347 0.432 1.000000e+00       2
AVEN            2.647386e-02 -0.7631936 0.187 0.378 1.000000e+00       2
CDKN2A.2        2.663759e-02 -0.9181577 0.293 0.422 1.000000e+00       2
PPP1R3C         2.679024e-02 -0.6622340 0.187 0.154 1.000000e+00       2
HIST1H2BK       2.681417e-02 -0.4987653 0.107 0.135 1.000000e+00       2
ZFP62           2.697968e-02 -0.9816330 0.133 0.181 1.000000e+00       2
PLPP4           2.702731e-02 -0.7191433 0.240 0.424 1.000000e+00       2
ANK3.1          2.712250e-02  0.2903763 0.307 0.248 1.000000e+00       2
TSLP.2          2.720589e-02 -0.7957231 0.133 0.152 1.000000e+00       2
TPCN1           2.721799e-02  0.3550914 0.387 0.274 1.000000e+00       2
ARHGEF2.1       2.724183e-02 -0.9848832 0.093 0.192 1.000000e+00       2
CNN2.1          2.753031e-02  0.4978343 0.560 0.436 1.000000e+00       2
CHML            2.755443e-02 -0.3396031 0.360 0.283 1.000000e+00       2
ST13.1          2.761492e-02 -0.2850405 0.347 0.459 1.000000e+00       2
PGM5-AS1        2.762695e-02 -0.6158013 0.080 0.150 1.000000e+00       2
SDC2            2.763914e-02  0.5240042 0.373 0.355 1.000000e+00       2
LTBP3.1         2.768763e-02  0.5371859 0.533 0.410 1.000000e+00       2
SBF2-AS1.2      2.780905e-02 -1.3545438 0.200 0.395 1.000000e+00       2
SEC61G          2.780917e-02 -0.3382882 0.347 0.440 1.000000e+00       2
CYR61.1         2.783352e-02  0.5396777 0.533 0.405 1.000000e+00       2
MPRIP.2         2.785790e-02 -0.3034231 0.493 0.370 1.000000e+00       2
CREB3L2         2.795562e-02  0.2582167 0.560 0.444 1.000000e+00       2
LMBRD2.1        2.802909e-02  0.6570700 0.360 0.291 1.000000e+00       2
AKAP11          2.807817e-02  0.5459781 0.440 0.366 1.000000e+00       2
KSR1            2.877307e-02  0.4915528 0.400 0.318 1.000000e+00       2
PRPF38B         2.881071e-02  0.5350621 0.520 0.437 1.000000e+00       2
ARHGAP10.1      2.887354e-02  0.4029080 0.333 0.320 1.000000e+00       2
ZNF793          2.889870e-02  0.6191609 0.320 0.222 1.000000e+00       2
SGF29.1         2.896151e-02  0.4551607 0.107 0.135 1.000000e+00       2
SRP68           2.903744e-02  0.4984605 0.400 0.318 1.000000e+00       2
BEX3.2          2.921470e-02 -0.8347994 0.227 0.418 1.000000e+00       2
TMEM102         2.959814e-02 -0.9312905 0.107 0.175 1.000000e+00       2
TMEM80          2.968822e-02 -0.5552162 0.253 0.271 1.000000e+00       2
CERS1           2.970110e-02 -0.3719404 0.253 0.167 1.000000e+00       2
ENOSF1.1        2.983021e-02  0.5082713 0.267 0.230 1.000000e+00       2
AC110597.1.1    2.994682e-02  0.4449140 0.253 0.115 1.000000e+00       2
ADAMTS19        2.998576e-02 -0.2611440 0.253 0.188 1.000000e+00       2
AL163051.1.2    3.005076e-02  0.3614963 0.133 0.098 1.000000e+00       2
TMEM92-AS1.1    3.022041e-02  0.5107128 0.280 0.145 1.000000e+00       2
DYNC1I1.2       3.029901e-02 -0.3449413 0.173 0.205 1.000000e+00       2
ARG2.2          3.061504e-02 -0.3966421 0.160 0.129 1.000000e+00       2
TCEAL1.2        3.062828e-02 -0.6432295 0.280 0.426 1.000000e+00       2
KANK4.2         3.065475e-02  0.5274378 0.240 0.169 1.000000e+00       2
TPD52L1         3.065475e-02 -1.0080435 0.200 0.213 1.000000e+00       2
TMEM222         3.076085e-02 -0.4046263 0.280 0.316 1.000000e+00       2
MSN.1           3.097399e-02  0.3039604 0.547 0.432 1.000000e+00       2
ASZ1.1          3.136348e-02  0.2945300 0.307 0.140 1.000000e+00       2
MIATNB.2        3.141733e-02 -0.9064214 0.093 0.161 1.000000e+00       2
MYH10.2         3.147171e-02  0.4735321 0.467 0.364 1.000000e+00       2
ANKRD45         3.166181e-02 -0.3645464 0.053 0.121 1.000000e+00       2
JMJD6           3.174359e-02 -0.9410240 0.147 0.319 1.000000e+00       2
TPP2.2          3.178455e-02  0.3254718 0.307 0.230 1.000000e+00       2
KIAA1328.2      3.196254e-02 -0.9190714 0.107 0.173 1.000000e+00       2
NEO1.1          3.232111e-02  0.9174166 0.333 0.185 1.000000e+00       2
FRS2            3.234884e-02  0.3532149 0.493 0.419 1.000000e+00       2
CCDC184         3.262720e-02 -0.8007100 0.160 0.284 1.000000e+00       2
BX255923.1.2    3.276715e-02 -0.2754258 0.147 0.184 1.000000e+00       2
EGR2.2          3.280910e-02  0.7258437 0.347 0.139 1.000000e+00       2
HOXA3.1         3.309097e-02  0.4399064 0.307 0.202 1.000000e+00       2
RTL5.1          3.310478e-02  0.4350203 0.240 0.172 1.000000e+00       2
APLF            3.313342e-02 -0.3548698 0.133 0.138 1.000000e+00       2
ACLY.1          3.326103e-02  0.9486194 0.533 0.452 1.000000e+00       2
HSPB11.2        3.338905e-02 -0.6966674 0.280 0.471 1.000000e+00       2
SRSF8           3.370377e-02 -0.5262865 0.307 0.487 1.000000e+00       2
KIF20B.2        3.390531e-02 -0.3281825 0.120 0.166 1.000000e+00       2
EVC             3.407896e-02 -0.2858156 0.120 0.130 1.000000e+00       2
RIMS2           3.413690e-02  0.2648587 0.373 0.342 1.000000e+00       2
TUBB2B.1        3.419500e-02 -0.8323237 0.240 0.220 1.000000e+00       2
EFNA1.2         3.431150e-02 -0.8310508 0.080 0.129 1.000000e+00       2
IGF2-AS.2       3.463373e-02 -0.5780423 0.160 0.098 1.000000e+00       2
WSB1            3.472208e-02  0.4853255 0.547 0.455 1.000000e+00       2
TEDC2           3.491404e-02 -0.4564533 0.213 0.154 1.000000e+00       2
LOXL1-AS1.1     3.516647e-02 -0.6508530 0.280 0.426 1.000000e+00       2
HSP90AA1        3.524100e-02 -0.7828604 0.293 0.448 1.000000e+00       2
PAWR            3.527086e-02 -0.6251902 0.347 0.447 1.000000e+00       2
FCGRT.1         3.537551e-02 -0.5659988 0.267 0.424 1.000000e+00       2
C1orf198.1      3.548043e-02  0.4610281 0.453 0.343 1.000000e+00       2
LAMTOR5.2       3.549544e-02 -0.2888324 0.333 0.471 1.000000e+00       2
OTUB1.1         3.563077e-02  0.5090400 0.467 0.373 1.000000e+00       2
SOD2.2          3.588759e-02  0.3812979 0.373 0.283 1.000000e+00       2
RGS1            3.594822e-02  0.2792512 0.240 0.178 1.000000e+00       2
CLSTN2          3.625256e-02 -0.8808330 0.280 0.265 1.000000e+00       2
MSMO1.2         3.626813e-02  0.3258939 0.573 0.437 1.000000e+00       2
PPP1CB.2        3.655968e-02 -0.6630404 0.360 0.455 1.000000e+00       2
IFNGR1.2        3.699292e-02 -0.4057335 0.360 0.321 1.000000e+00       2
PCSK1N.1        3.703006e-02 -0.8610334 0.213 0.314 1.000000e+00       2
MBNL1.2         3.705517e-02  0.2896708 0.547 0.465 1.000000e+00       2
AP003068.2      3.727373e-02 -0.3917802 0.093 0.143 1.000000e+00       2
EGFL7           3.743051e-02 -0.3350567 0.307 0.279 1.000000e+00       2
MCUB.2          3.747768e-02  0.2687656 0.373 0.295 1.000000e+00       2
MYL9.2          3.760365e-02 -0.7115150 0.320 0.427 1.000000e+00       2
AK6.1           3.760365e-02 -0.5849323 0.253 0.416 1.000000e+00       2
MAPRE2.1        3.779328e-02 -0.8138250 0.227 0.352 1.000000e+00       2
TWIST1.1        3.790428e-02 -0.5619485 0.293 0.426 1.000000e+00       2
TK1             3.812698e-02 -0.9677492 0.067 0.163 1.000000e+00       2
ABCA5.2         3.835094e-02 -0.7954602 0.160 0.184 1.000000e+00       2
CMTM8           3.854386e-02 -0.4845470 0.120 0.121 1.000000e+00       2
TFPI.2          3.891576e-02  0.3036750 0.280 0.253 1.000000e+00       2
HSD17B4.1       3.917627e-02 -0.3616722 0.307 0.282 1.000000e+00       2
RAD51B.2        3.935596e-02 -0.8950766 0.080 0.218 1.000000e+00       2
DTWD1           3.945469e-02 -0.7857529 0.187 0.373 1.000000e+00       2
CDCA8.2         3.985058e-02 -0.5061989 0.067 0.099 1.000000e+00       2
TRIAP1          3.988374e-02 -0.5965831 0.293 0.471 1.000000e+00       2
FRMD8           3.990032e-02  0.2573708 0.400 0.242 1.000000e+00       2
PTPN9.2         4.018308e-02  0.3209375 0.333 0.242 1.000000e+00       2
CCNDBP1.1       4.024987e-02 -0.5212861 0.280 0.407 1.000000e+00       2
RRM1.1          4.045076e-02 -0.4852688 0.160 0.230 1.000000e+00       2
RBM12           4.088898e-02  0.4285478 0.387 0.251 1.000000e+00       2
TOX2.2          4.092284e-02 -0.5160307 0.107 0.118 1.000000e+00       2
DNAJC25         4.092287e-02  0.4171534 0.320 0.268 1.000000e+00       2
TGFB1I1         4.092287e-02 -0.6065595 0.347 0.427 1.000000e+00       2
SVIP.2          4.095676e-02 -0.2710054 0.213 0.227 1.000000e+00       2
ZNF699.1        4.099049e-02 -0.9179357 0.107 0.172 1.000000e+00       2
CMBL.1          4.104160e-02 -0.5190169 0.227 0.270 1.000000e+00       2
PRAC2           4.112661e-02 -0.8651331 0.067 0.268 1.000000e+00       2
PPM1N           4.119471e-02 -0.4977507 0.067 0.126 1.000000e+00       2
TENT5A          4.127998e-02  0.3950441 0.493 0.410 1.000000e+00       2
HACD1.2         4.143385e-02 -0.3980670 0.347 0.459 1.000000e+00       2
HNRNPDL         4.165695e-02 -0.4979300 0.320 0.447 1.000000e+00       2
CCDC125.1       4.184644e-02 -0.5513009 0.147 0.135 1.000000e+00       2
TPM3            4.186379e-02 -0.6344649 0.347 0.452 1.000000e+00       2
HSPA1A.1        4.198484e-02  0.3571388 0.267 0.236 1.000000e+00       2
RASL11B.2       4.203668e-02  0.2964936 0.253 0.171 1.000000e+00       2
FAT1.1          4.203681e-02  0.7682065 0.507 0.441 1.000000e+00       2
CD99.2          4.203682e-02 -0.3866941 0.333 0.444 1.000000e+00       2
CCNT2-AS1       4.215827e-02 -0.4552287 0.213 0.136 1.000000e+00       2
SPINT1.2        4.240214e-02  0.4201397 0.307 0.263 1.000000e+00       2
TMEM11.2        4.269985e-02 -0.5386748 0.267 0.387 1.000000e+00       2
IAH1            4.271740e-02 -0.6742804 0.320 0.300 1.000000e+00       2
KLF9.2          4.284053e-02  0.3244615 0.280 0.235 1.000000e+00       2
UBXN4.2         4.319405e-02 -0.4270392 0.347 0.441 1.000000e+00       2
EIF2AK3         4.333613e-02  0.2876450 0.267 0.240 1.000000e+00       2
GALNT6.1        4.340732e-02  0.5907731 0.427 0.337 1.000000e+00       2
AMACR.1         4.342510e-02 -0.5694210 0.053 0.142 1.000000e+00       2
ZNF160          4.342512e-02  0.5207942 0.253 0.176 1.000000e+00       2
SLC5A3.1        4.358570e-02  0.3400663 0.507 0.379 1.000000e+00       2
BCLAF1          4.369307e-02  0.4007094 0.453 0.420 1.000000e+00       2
MAN1B1-DT       4.371096e-02 -0.8846918 0.080 0.215 1.000000e+00       2
PM20D2.1        4.374682e-02 -0.2951291 0.240 0.180 1.000000e+00       2
CDC37L1.2       4.380063e-02 -0.6518494 0.213 0.379 1.000000e+00       2
SYTL2.2         4.390836e-02  0.5709162 0.333 0.284 1.000000e+00       2
EEA1.1          4.392640e-02 -0.5878531 0.200 0.345 1.000000e+00       2
HAPLN4.2        4.398010e-02  0.2617527 0.373 0.228 1.000000e+00       2
GLB1L.2         4.399828e-02 -0.5829356 0.187 0.132 1.000000e+00       2
MYLIP.2         4.403432e-02 -0.5295807 0.107 0.139 1.000000e+00       2
TRIM24.2        4.414272e-02 -0.4239189 0.307 0.277 1.000000e+00       2
BNC1.1          4.425114e-02 -0.4724567 0.120 0.174 1.000000e+00       2
BEX4            4.441439e-02 -0.4704249 0.320 0.451 1.000000e+00       2
SMIM1.1         4.446888e-02 -0.9882910 0.080 0.198 1.000000e+00       2
PANX2.1         4.454164e-02  0.6713426 0.253 0.193 1.000000e+00       2
ANKRD34A.2      4.474224e-02  0.3270122 0.173 0.154 1.000000e+00       2
AL121944.1      4.476032e-02 -0.8147542 0.173 0.165 1.000000e+00       2
TSPAN4          4.490695e-02 -0.5904356 0.333 0.434 1.000000e+00       2
CTSC.1          4.498030e-02 -0.9564016 0.133 0.232 1.000000e+00       2
ZNF121.2        4.498030e-02 -0.4179207 0.253 0.226 1.000000e+00       2
MED31.1         4.498031e-02 -0.5257635 0.360 0.461 1.000000e+00       2
WFDC3.2         4.501699e-02  0.4158774 0.187 0.109 1.000000e+00       2
BEND6.1         4.501700e-02 -0.8365593 0.067 0.276 1.000000e+00       2
ZSWIM3.2        4.579391e-02 -0.4072242 0.213 0.133 1.000000e+00       2
DERL3.1         4.635539e-02 -0.9602116 0.133 0.191 1.000000e+00       2
BCL9            4.677139e-02  0.2919885 0.187 0.184 1.000000e+00       2
MCTP2           4.679050e-02  0.4806488 0.373 0.314 1.000000e+00       2
AC006480.2.1    4.688540e-02 -0.6279780 0.120 0.128 1.000000e+00       2
ABHD15.1        4.707561e-02 -0.6283759 0.240 0.202 1.000000e+00       2
ACKR2.2         4.742015e-02  0.2535268 0.227 0.212 1.000000e+00       2
PTPRJ.2         4.749690e-02 -0.5988466 0.160 0.141 1.000000e+00       2
UCN             4.757381e-02 -0.5628758 0.147 0.151 1.000000e+00       2
AC144831.1      4.801797e-02  0.2525659 0.267 0.149 1.000000e+00       2
ZADH2.2         4.813457e-02 -0.7291909 0.347 0.448 1.000000e+00       2
NEK7            4.817345e-02  0.7917429 0.427 0.385 1.000000e+00       2
JOSD2.1         4.825127e-02 -0.5327089 0.373 0.479 1.000000e+00       2
TCEAL4.2        4.860279e-02 -0.8668545 0.333 0.423 1.000000e+00       2
SLTM            4.887770e-02  0.5545346 0.547 0.452 1.000000e+00       2
GSTA4.1         4.893677e-02  0.7237380 0.373 0.305 1.000000e+00       2
AXL.1           4.903537e-02  0.5829613 0.507 0.449 1.000000e+00       2
ANKFN1.2        4.907474e-02 -0.4770630 0.227 0.264 1.000000e+00       2
LYRM1           5.019127e-02 -0.9885585 0.120 0.230 1.000000e+00       2
FBLN2.2         5.031209e-02  0.3142022 0.533 0.405 1.000000e+00       2
RAB42           5.057465e-02 -0.8674721 0.040 0.119 1.000000e+00       2
ABHD4           5.092017e-02  0.5955263 0.360 0.199 1.000000e+00       2
VPS9D1.1        5.100170e-02  0.3722474 0.240 0.219 1.000000e+00       2
PLP2.2          5.112421e-02 -0.4632882 0.320 0.450 1.000000e+00       2
EPHB6.1         5.122625e-02  0.4405157 0.227 0.119 1.000000e+00       2
CALY.2          5.130840e-02 -0.2562063 0.227 0.173 1.000000e+00       2
CAVIN3.2        5.134943e-02 -0.6105632 0.373 0.458 1.000000e+00       2
KDM1B.1         5.139008e-02  0.2877301 0.200 0.156 1.000000e+00       2
STS             5.159607e-02 -0.5089232 0.107 0.195 1.000000e+00       2
EGFR.2          5.213389e-02  0.5059089 0.427 0.358 1.000000e+00       2
IL32.1          5.230029e-02 -0.5123671 0.173 0.161 1.000000e+00       2
FGD4.2          5.234190e-02 -0.6401474 0.240 0.175 1.000000e+00       2
SLC9A9.1        5.234195e-02 -0.3700795 0.267 0.152 1.000000e+00       2
TMTC2.1         5.273925e-02 -0.6096384 0.147 0.210 1.000000e+00       2
PLCB4.2         5.284422e-02 -0.6588785 0.067 0.184 1.000000e+00       2
SLC24A1.2       5.288625e-02 -0.5968149 0.107 0.128 1.000000e+00       2
HLA-DPB1.2      5.292834e-02 -0.5777343 0.280 0.215 1.000000e+00       2
PTPRA.2         5.299150e-02  0.3541854 0.440 0.339 1.000000e+00       2
MGP.2           5.311090e-02 -1.8114216 0.173 0.327 1.000000e+00       2
AP003486.1      5.358393e-02  0.5713297 0.253 0.128 1.000000e+00       2
CDK5RAP2.2      5.369031e-02 -0.4676739 0.187 0.246 1.000000e+00       2
NQO2.2          5.463395e-02 -1.1026795 0.187 0.308 1.000000e+00       2
TEDC1.1         5.541602e-02 -0.7408942 0.160 0.179 1.000000e+00       2
CDC7            5.541634e-02  0.3924852 0.133 0.143 1.000000e+00       2
DIO3OS.2        5.658474e-02 -0.9116024 0.133 0.154 1.000000e+00       2
ISYNA1.2        5.685261e-02  0.3686866 0.333 0.251 1.000000e+00       2
SARDH.1         5.709868e-02 -0.5330487 0.173 0.195 1.000000e+00       2
TLE4.1          5.712113e-02  0.9433244 0.427 0.362 1.000000e+00       2
RAD21.2         5.748078e-02  0.3771842 0.493 0.404 1.000000e+00       2
AC009414.2.1    5.784231e-02 -0.5549537 0.187 0.257 1.000000e+00       2
SOBP            5.809196e-02 -0.4064078 0.147 0.202 1.000000e+00       2
LINC01836.1     5.813745e-02  0.5323020 0.120 0.100 1.000000e+00       2
TP53I3.2        5.877736e-02 -0.6038254 0.293 0.434 1.000000e+00       2
MMP7.1          5.882329e-02 -0.6353213 0.080 0.119 1.000000e+00       2
SATB1           5.889227e-02 -0.6715549 0.093 0.198 1.000000e+00       2
PTPRE           5.900723e-02 -0.6233730 0.187 0.133 1.000000e+00       2
AC108062.1      5.916870e-02 -0.8609109 0.133 0.207 1.000000e+00       2
MAPK12.1        6.002806e-02  0.2729776 0.373 0.272 1.000000e+00       2
NECAB3          6.019178e-02  0.3043363 0.293 0.274 1.000000e+00       2
SPACA6          6.035590e-02 -0.4364349 0.213 0.197 1.000000e+00       2
TET1.2          6.049669e-02 -0.7147216 0.120 0.147 1.000000e+00       2
RBP1.2          6.058968e-02 -0.9266948 0.333 0.340 1.000000e+00       2
CDC42EP3.2      6.068516e-02  0.2724664 0.480 0.396 1.000000e+00       2
MANSC1.2        6.073235e-02 -0.4226699 0.107 0.157 1.000000e+00       2
ROGDI.1         6.080318e-02 -0.8055902 0.187 0.296 1.000000e+00       2
CTSZ.2          6.139589e-02 -0.6422838 0.267 0.432 1.000000e+00       2
LAYN            6.161037e-02  0.2745126 0.267 0.186 1.000000e+00       2
PPM1K.1         6.161044e-02  0.4972442 0.360 0.332 1.000000e+00       2
RIOX1           6.172973e-02 -1.0806682 0.053 0.159 1.000000e+00       2
TXNIP.2         6.184951e-02  0.4222992 0.453 0.418 1.000000e+00       2
AC008771.1      6.201738e-02 -0.4233349 0.333 0.435 1.000000e+00       2
LCMT2.1         6.206515e-02  0.5340447 0.160 0.170 1.000000e+00       2
G6PD.2          6.245068e-02  0.3816758 0.453 0.342 1.000000e+00       2
RIC8B           6.274087e-02 -0.7648679 0.160 0.183 1.000000e+00       2
YRDC            6.298366e-02 -0.5000894 0.333 0.311 1.000000e+00       2
AC244090.1      6.298367e-02 -0.4976979 0.240 0.378 1.000000e+00       2
AC092040.1      6.406093e-02 -0.4550063 0.307 0.316 1.000000e+00       2
ATRNL1.2        6.438201e-02 -0.2545948 0.333 0.263 1.000000e+00       2
CLIC1.1         6.475435e-02 -0.3388424 0.347 0.449 1.000000e+00       2
PPIP5K1.1       6.500346e-02  0.4330376 0.133 0.111 1.000000e+00       2
LHPP            6.502844e-02 -0.4580769 0.227 0.215 1.000000e+00       2
AC007878.1.2    6.510332e-02 -0.4953365 0.080 0.127 1.000000e+00       2
ST20-AS1        6.510337e-02 -0.6320171 0.187 0.233 1.000000e+00       2
STX12.2         6.537869e-02 -0.2825833 0.440 0.390 1.000000e+00       2
FBXW8.1         6.669322e-02  0.2514790 0.320 0.251 1.000000e+00       2
PHIP.2          6.707644e-02 -0.3531558 0.267 0.275 1.000000e+00       2
TCN2            6.712764e-02 -0.3497332 0.093 0.127 1.000000e+00       2
KLHDC9          6.738429e-02  0.2688636 0.253 0.137 1.000000e+00       2
DNMT3A          6.771914e-02 -0.4065111 0.200 0.276 1.000000e+00       2
GNPDA1.2        6.818506e-02  0.3735251 0.347 0.324 1.000000e+00       2
CLEC1B.2        6.821084e-02 -0.5704177 0.120 0.132 1.000000e+00       2
PPA1.1          6.847106e-02 -0.6594760 0.333 0.449 1.000000e+00       2
MAGI1.1         6.867968e-02  0.3844849 0.440 0.340 1.000000e+00       2
FAM83G.2        6.881012e-02  0.6662030 0.320 0.151 1.000000e+00       2
LAMC3.2         7.015468e-02 -0.2928113 0.120 0.171 1.000000e+00       2
AC132192.2.2    7.026101e-02 -0.5576841 0.120 0.113 1.000000e+00       2
BCL2L1          7.050079e-02  0.4526192 0.240 0.231 1.000000e+00       2
TRAF4.2         7.068775e-02  0.4782543 0.413 0.345 1.000000e+00       2
ANKRD29.2       7.106282e-02 -0.5492543 0.080 0.122 1.000000e+00       2
C1QTNF1.1       7.208925e-02 -0.3924285 0.133 0.212 1.000000e+00       2
CLEC11A         7.211641e-02 -0.7771569 0.173 0.285 1.000000e+00       2
JAGN1.1         7.230687e-02 -0.8017086 0.280 0.402 1.000000e+00       2
SH3PXD2A        7.249759e-02  0.4437833 0.293 0.225 1.000000e+00       2
MAPKAPK2        7.282588e-02  0.2782164 0.400 0.355 1.000000e+00       2
ODC1.2          7.301786e-02 -0.7773839 0.360 0.448 1.000000e+00       2
GALNT16.1       7.312771e-02  0.2723558 0.307 0.184 1.000000e+00       2
EML4            7.356866e-02  0.3281014 0.480 0.454 1.000000e+00       2
CARD16.2        7.406687e-02  0.3736762 0.173 0.149 1.000000e+00       2
TTC30B.2        7.412283e-02 -0.2507214 0.120 0.096 1.000000e+00       2
PCYOX1L         7.445681e-02 -0.6789214 0.107 0.175 1.000000e+00       2
SEPT9.2         7.445699e-02  0.4728140 0.400 0.320 1.000000e+00       2
MAMDC2          7.459640e-02 -0.5679641 0.173 0.242 1.000000e+00       2
CRK.1           7.526960e-02  0.8784128 0.547 0.424 1.000000e+00       2
SLC19A2.2       7.526961e-02  0.4252872 0.400 0.361 1.000000e+00       2
ZIK1.2          7.631676e-02 -0.3740452 0.147 0.161 1.000000e+00       2
ANKDD1A         7.657338e-02 -0.4868092 0.173 0.171 1.000000e+00       2
ARHGAP29.2      7.763543e-02  0.2877741 0.293 0.284 1.000000e+00       2
NEDD4L.1        7.792450e-02 -0.3064424 0.160 0.158 1.000000e+00       2
CYP27A1.2       7.801106e-02 -0.8157513 0.227 0.139 1.000000e+00       2
RTN3.2          7.812740e-02 -0.3651070 0.387 0.379 1.000000e+00       2
EPHB2.1         7.815637e-02 -0.4166140 0.213 0.147 1.000000e+00       2
RSRC2.1         7.894322e-02 -0.3178089 0.387 0.485 1.000000e+00       2
FHOD1           7.903104e-02 -0.6221705 0.120 0.232 1.000000e+00       2
FIRRE.2         7.911889e-02  0.4523640 0.293 0.179 1.000000e+00       2
IQCK            7.911894e-02 -0.7042774 0.240 0.364 1.000000e+00       2
LITAF.2         7.955958e-02 -0.7858090 0.173 0.246 1.000000e+00       2
SPATA17.1       7.973643e-02 -0.5612899 0.040 0.105 1.000000e+00       2
RHOT2           8.003187e-02  0.5972845 0.293 0.303 1.000000e+00       2
PSME4           8.003189e-02  0.4009136 0.493 0.403 1.000000e+00       2
SH3BGRL         8.029854e-02 -0.2836729 0.333 0.463 1.000000e+00       2
GDNF.1          8.032818e-02  0.5359760 0.240 0.246 1.000000e+00       2
MED12L          8.038757e-02 -0.3641736 0.133 0.112 1.000000e+00       2
FIGNL1.1        8.104286e-02  0.6872180 0.240 0.148 1.000000e+00       2
STXBP5L.1       8.182290e-02 -0.7275241 0.227 0.157 1.000000e+00       2
WDYHV1.1        8.206397e-02 -0.8486082 0.133 0.169 1.000000e+00       2
AC064836.3.2    8.248806e-02 -0.3590242 0.080 0.100 1.000000e+00       2
KCNK15-AS1      8.248810e-02 -0.3635952 0.173 0.125 1.000000e+00       2
SLC30A1.1       8.248810e-02  0.5127881 0.427 0.282 1.000000e+00       2
AC103706.1.2    8.273086e-02 -0.5461554 0.147 0.150 1.000000e+00       2
SOCS1.2         8.279177e-02 -0.6373087 0.120 0.270 1.000000e+00       2
EFEMP2.1        8.315740e-02  0.8892622 0.453 0.404 1.000000e+00       2
ELF1            8.318790e-02 -0.3251344 0.440 0.363 1.000000e+00       2
PTPN1           8.331011e-02  0.2560077 0.467 0.363 1.000000e+00       2
INTS7           8.334017e-02  0.3041252 0.253 0.154 1.000000e+00       2
SUCO            8.334068e-02  0.5377237 0.507 0.424 1.000000e+00       2
BDH2.2          8.352431e-02 -0.4421708 0.240 0.395 1.000000e+00       2
GNAS-AS1.1      8.358547e-02 -0.7543325 0.027 0.139 1.000000e+00       2
PARPBP          8.383082e-02 -0.2573861 0.133 0.170 1.000000e+00       2
ZNRF3.2         8.395391e-02 -0.4099455 0.067 0.127 1.000000e+00       2
PELI2           8.450914e-02 -0.5375938 0.080 0.204 1.000000e+00       2
PALLD           8.460197e-02  0.2780145 0.560 0.449 1.000000e+00       2
PLEKHA5         8.497401e-02 -0.3753750 0.320 0.308 1.000000e+00       2
PPP5D1.2        8.506716e-02  0.6867044 0.147 0.096 1.000000e+00       2
PI4KA           8.544096e-02 -0.2620084 0.240 0.199 1.000000e+00       2
AC009126.1.1    8.625520e-02 -0.4599124 0.093 0.159 1.000000e+00       2
CEBPD.2         8.641246e-02 -1.0131935 0.387 0.444 1.000000e+00       2
FERMT1          8.644401e-02 -0.4630307 0.120 0.211 1.000000e+00       2
RGS11           8.657006e-02 -0.3826459 0.173 0.152 1.000000e+00       2
KLHL42.1        8.660159e-02 -0.7308994 0.253 0.363 1.000000e+00       2
UBE2D1.1        8.679102e-02 -0.4076636 0.240 0.355 1.000000e+00       2
TMEM97          8.729774e-02 -0.3942624 0.200 0.215 1.000000e+00       2
PDLIM4          8.729775e-02 -0.7280272 0.320 0.433 1.000000e+00       2
IGFBP6          8.745649e-02  0.5317514 0.360 0.318 1.000000e+00       2
LUM.2           8.754619e-02 -0.8116271 0.400 0.473 1.000000e+00       2
TMEM243         8.771121e-02 -0.6034217 0.267 0.417 1.000000e+00       2
PYGB.2          8.783870e-02 -0.2793731 0.320 0.223 1.000000e+00       2
FAM69A.2        8.851066e-02  0.2564489 0.200 0.132 1.000000e+00       2
ATP2B4.2        8.896109e-02  0.2539765 0.453 0.338 1.000000e+00       2
AL138724.1.1    8.938088e-02 -0.4571057 0.053 0.123 1.000000e+00       2
SCAMP1-AS1.2    8.957519e-02 -0.9308207 0.133 0.229 1.000000e+00       2
MSX2.2          8.989977e-02 -0.4816245 0.240 0.247 1.000000e+00       2
CTDSPL2         8.993228e-02  0.3133634 0.280 0.270 1.000000e+00       2
NAP1L1.2        8.993229e-02 -0.5971317 0.360 0.425 1.000000e+00       2
NEK5.2          9.064989e-02  0.3234577 0.200 0.215 1.000000e+00       2
KIAA1841.1      9.074757e-02 -0.7672574 0.120 0.165 1.000000e+00       2
GAB2.2          9.114178e-02  0.3096684 0.240 0.168 1.000000e+00       2
LRCH4.2         9.166888e-02 -0.7725736 0.187 0.212 1.000000e+00       2
PDCD4-AS1.1     9.180104e-02 -0.3841846 0.213 0.232 1.000000e+00       2
UCHL1.1         9.256376e-02 -0.9366745 0.307 0.441 1.000000e+00       2
AC010336.1.2    9.326466e-02 -0.6646187 0.080 0.132 1.000000e+00       2
AKAP12.2        9.333159e-02 -1.0114701 0.080 0.201 1.000000e+00       2
HSPB8.1         9.343233e-02 -0.3078480 0.267 0.271 1.000000e+00       2
IER5L.2         9.410481e-02 -0.3872534 0.347 0.299 1.000000e+00       2
FBXO34          9.417228e-02 -0.3386674 0.293 0.333 1.000000e+00       2
HDDC2.1         9.437490e-02 -0.2744198 0.280 0.441 1.000000e+00       2
FABP5.2         9.447627e-02 -0.7397459 0.253 0.295 1.000000e+00       2
CXXC5           9.447634e-02 -0.5175356 0.373 0.454 1.000000e+00       2
C19orf66        9.461126e-02 -0.9577700 0.067 0.177 1.000000e+00       2
CYP3A5          9.481503e-02  0.2598981 0.200 0.158 1.000000e+00       2
C7orf43         9.495033e-02 -0.6197043 0.213 0.158 1.000000e+00       2
KIAA1551        9.522290e-02 -0.4561258 0.133 0.218 1.000000e+00       2
NUB1            9.532505e-02 -0.3938018 0.400 0.355 1.000000e+00       2
BDNF-AS         9.549534e-02 -0.4690320 0.240 0.130 1.000000e+00       2
UBP1.1          9.583723e-02  0.7310544 0.413 0.340 1.000000e+00       2
AC245140.2      9.652337e-02 -0.5834084 0.067 0.110 1.000000e+00       2
DOK1.1          9.697174e-02  0.3260157 0.413 0.254 1.000000e+00       2
ENO1            9.721378e-02 -0.4402860 0.400 0.447 1.000000e+00       2
WNT11.2         9.776879e-02 -0.5119900 0.120 0.176 1.000000e+00       2
PITPNM1.1       9.783834e-02  0.6327609 0.240 0.207 1.000000e+00       2
AC091271.1      9.797718e-02 -1.0196870 0.027 0.188 1.000000e+00       2
PALMD           9.797751e-02 -0.2578583 0.160 0.223 1.000000e+00       2
MVK.2           9.815182e-02 -0.3814072 0.253 0.276 1.000000e+00       2
CBLN3.1         9.825649e-02 -0.3808028 0.213 0.124 1.000000e+00       2
SOCS2.2         9.902678e-02 -0.8883575 0.080 0.149 1.000000e+00       2
SCAF8           9.927297e-02  0.3734842 0.307 0.236 1.000000e+00       2
YBX3.2          1.000497e-01 -0.5071319 0.320 0.465 1.000000e+00       2
SLC39A2.2       1.009377e-01 -0.9118324 0.067 0.113 1.000000e+00       2
HIF1A.2         1.012238e-01 -0.4553403 0.533 0.446 1.000000e+00       2
RERE.2          1.014026e-01  0.3570462 0.427 0.378 1.000000e+00       2
SIN3B           1.017970e-01  0.3486070 0.333 0.296 1.000000e+00       2
YDJC.1          1.018688e-01 -0.8512837 0.213 0.334 1.000000e+00       2
SDC1.2          1.020845e-01  0.3239038 0.387 0.267 1.000000e+00       2
PLN.2           1.022636e-01 -1.3138064 0.080 0.188 1.000000e+00       2
DNAJB4.1        1.022646e-01  0.4831798 0.573 0.451 1.000000e+00       2
C9orf47.2       1.026616e-01 -0.2607816 0.067 0.098 1.000000e+00       2
MGARP           1.028063e-01 -0.5135819 0.227 0.239 1.000000e+00       2
C21orf91.1      1.028785e-01 -0.5257430 0.160 0.204 1.000000e+00       2
GSEC.1          1.034951e-01  0.5084808 0.213 0.161 1.000000e+00       2
ACTA2.2         1.043608e-01 -0.8278776 0.453 0.475 1.000000e+00       2
HSPB3.2         1.044081e-01 -0.9748968 0.040 0.170 1.000000e+00       2
MEGF9           1.052168e-01 -0.7111925 0.040 0.158 1.000000e+00       2
SMOC2           1.052902e-01  0.2763851 0.333 0.247 1.000000e+00       2
RAB13.2         1.053275e-01 -0.2699096 0.320 0.426 1.000000e+00       2
SDHAF3.2        1.053644e-01  0.5343822 0.347 0.326 1.000000e+00       2
AC004803.1.2    1.054381e-01  0.3441719 0.213 0.110 1.000000e+00       2
MAML3.1         1.054382e-01  0.5955618 0.307 0.220 1.000000e+00       2
ZNF35           1.055121e-01 -0.2859363 0.227 0.199 1.000000e+00       2
RNF227.2        1.055860e-01 -0.7875711 0.200 0.190 1.000000e+00       2
ICAM5           1.060303e-01 -0.3929352 0.173 0.113 1.000000e+00       2
SPA17.1         1.061785e-01 -0.9649488 0.040 0.157 1.000000e+00       2
DYNLL2.2        1.064017e-01  0.6799306 0.387 0.304 1.000000e+00       2
IP6K3.2         1.065502e-01 -0.9606466 0.133 0.209 1.000000e+00       2
DMPK            1.068861e-01  0.6817028 0.333 0.258 1.000000e+00       2
IGF1            1.069972e-01 -1.3004901 0.080 0.233 1.000000e+00       2
SQSTM1.2        1.077848e-01  0.2548298 0.520 0.440 1.000000e+00       2
CA11            1.079352e-01  0.3980546 0.227 0.153 1.000000e+00       2
PEA15.1         1.081988e-01  0.2927944 0.493 0.435 1.000000e+00       2
ZEB2.1          1.084628e-01 -0.4629770 0.307 0.458 1.000000e+00       2
AF165147.1      1.087273e-01 -0.2577734 0.120 0.138 1.000000e+00       2
EFNA5.1         1.089546e-01 -0.2588879 0.173 0.117 1.000000e+00       2
C16orf74        1.089924e-01 -0.4227342 0.200 0.227 1.000000e+00       2
FAM171B         1.091060e-01  0.2908768 0.227 0.181 1.000000e+00       2
PHLDB1.2        1.094481e-01  0.3419442 0.427 0.378 1.000000e+00       2
TNK2.1          1.097139e-01  0.9511833 0.293 0.118 1.000000e+00       2
AL356056.1.2    1.100193e-01 -1.0349963 0.173 0.282 1.000000e+00       2
BCO2.2          1.104402e-01 -0.7551676 0.147 0.190 1.000000e+00       2
SLC8A1          1.104786e-01 -0.4996731 0.173 0.197 1.000000e+00       2
LAGE3.1         1.105170e-01 -0.2939433 0.307 0.435 1.000000e+00       2
SKP2.2          1.107088e-01  0.2961129 0.373 0.340 1.000000e+00       2
CEBPZOS.1       1.108240e-01 -0.4672820 0.267 0.409 1.000000e+00       2
RFWD3.1         1.112469e-01  0.4200046 0.240 0.147 1.000000e+00       2
WFDC1.2         1.115169e-01 -0.2981152 0.267 0.167 1.000000e+00       2
MICALL2.2       1.116324e-01  0.5662839 0.267 0.175 1.000000e+00       2
AC097478.1.2    1.120196e-01 -1.0235952 0.133 0.157 1.000000e+00       2
PHPT1           1.125630e-01 -0.4379157 0.373 0.422 1.000000e+00       2
CCNA2.2         1.128740e-01 -0.6695118 0.107 0.151 1.000000e+00       2
SKIDA1          1.130302e-01 -0.5581775 0.120 0.140 1.000000e+00       2
CENPK.1         1.134988e-01 -0.8096381 0.093 0.144 1.000000e+00       2
ZC3H7B.1        1.140089e-01  0.9786570 0.360 0.230 1.000000e+00       2
HAND2-AS1.2     1.145597e-01 -0.3072790 0.507 0.434 1.000000e+00       2
ABHD14A         1.160657e-01 -0.3422134 0.253 0.431 1.000000e+00       2
SETD4.1         1.163048e-01  0.2824331 0.213 0.192 1.000000e+00       2
AL356599.1.1    1.164245e-01 -0.7774309 0.040 0.129 1.000000e+00       2
BBC3            1.169045e-01 -0.3370948 0.267 0.246 1.000000e+00       2
ECI1.2          1.173056e-01  0.3171952 0.413 0.386 1.000000e+00       2
PCDH9           1.174664e-01 -0.5902079 0.200 0.222 1.000000e+00       2
SPECC1L.2       1.193681e-01  0.5132468 0.267 0.190 1.000000e+00       2
KPNA5.1         1.194088e-01  0.5392007 0.253 0.191 1.000000e+00       2
MAP1A           1.200619e-01 -0.3017496 0.387 0.379 1.000000e+00       2
ZFP36L2.2       1.203895e-01 -0.4666984 0.333 0.318 1.000000e+00       2
PAFAH1B3        1.215416e-01 -0.7852438 0.120 0.293 1.000000e+00       2
RAB9A.2         1.217896e-01 -0.6928847 0.173 0.340 1.000000e+00       2
MAPK8IP3.2      1.218308e-01  0.3012085 0.280 0.193 1.000000e+00       2
SH3GL3          1.227854e-01 -0.5648458 0.093 0.118 1.000000e+00       2
FAM133A.2       1.228687e-01 -0.6363943 0.093 0.189 1.000000e+00       2
SYT8.2          1.236195e-01 -0.6506959 0.160 0.154 1.000000e+00       2
IL27RA.1        1.240392e-01 -0.3833717 0.107 0.112 1.000000e+00       2
ODF3L1          1.242911e-01 -0.5965221 0.120 0.129 1.000000e+00       2
PTPRM           1.247963e-01  0.3484694 0.307 0.200 1.000000e+00       2
MGA             1.253030e-01 -0.3334667 0.387 0.301 1.000000e+00       2
VCL             1.256417e-01  0.4171395 0.573 0.453 1.000000e+00       2
FBXO8           1.256840e-01 -0.4894965 0.360 0.337 1.000000e+00       2
PPP1R3B.1       1.258536e-01 -0.8722250 0.160 0.191 1.000000e+00       2
ZNF320.2        1.258962e-01 -0.3928054 0.093 0.107 1.000000e+00       2
PRAC1.2         1.259811e-01 -0.4352717 0.253 0.230 1.000000e+00       2
TXNDC16.2       1.264064e-01 -0.3108588 0.453 0.359 1.000000e+00       2
ODF3B           1.265767e-01 -0.6127571 0.093 0.164 1.000000e+00       2
FIBIN.2         1.268314e-01  0.2518488 0.493 0.427 1.000000e+00       2
GINS4.2         1.274313e-01  0.3502072 0.240 0.159 1.000000e+00       2
USP12.1         1.276460e-01  0.3249992 0.267 0.258 1.000000e+00       2
PCBD1.1         1.278606e-01 -0.5105271 0.347 0.482 1.000000e+00       2
ZFAND2A.1       1.282477e-01 -0.4084380 0.347 0.304 1.000000e+00       2
RAP2C           1.285926e-01  0.3321036 0.427 0.381 1.000000e+00       2
ADAMTS1.2       1.286354e-01 -0.5504488 0.227 0.228 1.000000e+00       2
DSEL.2          1.288085e-01  0.4570851 0.320 0.334 1.000000e+00       2
SAT1.2          1.291966e-01 -1.0903591 0.347 0.419 1.000000e+00       2
CEP44.2         1.293279e-01 -0.3360316 0.173 0.248 1.000000e+00       2
FDFT1.2         1.293712e-01 -0.7388195 0.360 0.441 1.000000e+00       2
ANXA1           1.295013e-01 -0.4132501 0.320 0.464 1.000000e+00       2
TVP23B.1        1.296316e-01 -0.4963449 0.253 0.332 1.000000e+00       2
AURKA.2         1.300659e-01  0.4518632 0.160 0.160 1.000000e+00       2
CAMK2N1         1.301099e-01 -0.8929109 0.333 0.440 1.000000e+00       2
PIDD1.2         1.307642e-01 -0.4419090 0.147 0.133 1.000000e+00       2
CDKN1B          1.309832e-01 -0.5050429 0.173 0.255 1.000000e+00       2
ARMCX4          1.317728e-01 -0.3375313 0.067 0.135 1.000000e+00       2
IL17RE.2        1.318607e-01 -0.7391648 0.080 0.129 1.000000e+00       2
TMEM176A        1.320368e-01 -0.7245894 0.133 0.182 1.000000e+00       2
SYNM.1          1.324781e-01 -0.4759846 0.347 0.428 1.000000e+00       2
FGFRL1.2        1.337635e-01  0.4287869 0.360 0.262 1.000000e+00       2
CLSTN3          1.338971e-01 -0.3006650 0.133 0.150 1.000000e+00       2
COTL1.1         1.338971e-01 -0.5137226 0.333 0.435 1.000000e+00       2
ST6GAL1.2       1.339861e-01 -0.8738085 0.107 0.163 1.000000e+00       2
AXIN1.1         1.340747e-01 -0.9594177 0.093 0.138 1.000000e+00       2
MAD2L2          1.344769e-01 -0.4535115 0.227 0.277 1.000000e+00       2
MBP.2           1.350581e-01 -0.8257475 0.200 0.205 1.000000e+00       2
MYBL1.1         1.355974e-01 -0.5288515 0.107 0.143 1.000000e+00       2
FBXO5.1         1.358673e-01 -0.7750112 0.053 0.127 1.000000e+00       2
CITED2.2        1.362732e-01 -0.3018881 0.427 0.415 1.000000e+00       2
ZNF22           1.367704e-01 -0.3839759 0.200 0.314 1.000000e+00       2
SRSF7           1.370422e-01 -0.3783171 0.347 0.443 1.000000e+00       2
UGDH.1          1.371783e-01 -0.4520975 0.373 0.448 1.000000e+00       2
IKZF2.2         1.391856e-01  0.3625582 0.213 0.110 1.000000e+00       2
RPGR            1.401514e-01 -0.7058091 0.227 0.176 1.000000e+00       2
STAMBP          1.404287e-01 -0.3138162 0.267 0.340 1.000000e+00       2
FAM13B          1.406598e-01 -0.3376981 0.293 0.258 1.000000e+00       2
TBPL1           1.407986e-01 -0.6786734 0.173 0.363 1.000000e+00       2
RAPGEF2.1       1.409838e-01 -0.2960204 0.333 0.254 1.000000e+00       2
RASSF1-AS1      1.410765e-01 -0.4271723 0.107 0.136 1.000000e+00       2
SNX18.1         1.411693e-01 -0.4074643 0.333 0.307 1.000000e+00       2
BTBD3.2         1.415407e-01 -0.5081521 0.213 0.234 1.000000e+00       2
AL135999.1      1.416799e-01  0.6089531 0.253 0.160 1.000000e+00       2
POPDC3          1.420060e-01 -0.4877084 0.080 0.114 1.000000e+00       2
KIAA0040.2      1.425659e-01 -0.3459865 0.187 0.146 1.000000e+00       2
MMADHC.2        1.428934e-01 -0.3854211 0.307 0.465 1.000000e+00       2
SNRPN.2         1.429403e-01 -0.3410787 0.347 0.453 1.000000e+00       2
KLRG1.1         1.429871e-01  0.4411941 0.133 0.124 1.000000e+00       2
GOPC.1          1.433622e-01  0.2549630 0.493 0.450 1.000000e+00       2
PRLR.2          1.437374e-01 -0.7419244 0.227 0.224 1.000000e+00       2
PLEKHA4.2       1.438791e-01 -0.4683876 0.013 0.107 1.000000e+00       2
CADPS2.2        1.443976e-01 -0.4577227 0.280 0.281 1.000000e+00       2
ZNF460          1.447278e-01 -0.7468959 0.120 0.143 1.000000e+00       2
HDGFL2          1.447282e-01 -0.3389185 0.400 0.327 1.000000e+00       2
RTL8B           1.450594e-01 -0.4949907 0.307 0.429 1.000000e+00       2
ANO4.2          1.453908e-01 -0.6676641 0.133 0.122 1.000000e+00       2
ANGPT1          1.460566e-01 -0.8556341 0.133 0.174 1.000000e+00       2
NAAA            1.462948e-01 -0.3154303 0.267 0.300 1.000000e+00       2
RTN2.1          1.465332e-01 -0.6349823 0.187 0.260 1.000000e+00       2
CDKN1C.2        1.467715e-01 -0.6342408 0.253 0.426 1.000000e+00       2
SCN1B.1         1.469154e-01 -0.6649589 0.227 0.230 1.000000e+00       2
TAF1A.1         1.470111e-01 -0.5023289 0.067 0.117 1.000000e+00       2
ALKAL2.2        1.472025e-01 -1.0319423 0.080 0.201 1.000000e+00       2
VEGFB           1.472026e-01 -0.5710140 0.360 0.455 1.000000e+00       2
RGCC            1.477263e-01 -0.5990452 0.307 0.331 1.000000e+00       2
REXO5.2         1.479704e-01 -0.2637137 0.080 0.103 1.000000e+00       2
RIN2            1.483555e-01 -0.4132198 0.413 0.354 1.000000e+00       2
FLOT2           1.484519e-01 -0.2830278 0.187 0.224 1.000000e+00       2
APOC1.2         1.486443e-01 -0.9591388 0.147 0.221 1.000000e+00       2
LDLR.2          1.486930e-01 -0.2722942 0.373 0.365 1.000000e+00       2
ERMARD.1        1.491278e-01 -0.8860286 0.093 0.185 1.000000e+00       2
MAP3K7CL.2      1.492234e-01 -0.7785606 0.307 0.274 1.000000e+00       2
GSTO1.2         1.492730e-01 -0.3778167 0.387 0.446 1.000000e+00       2
MIR22HG.2       1.498548e-01 -0.2724015 0.453 0.395 1.000000e+00       2
TSPAN13.1       1.507794e-01  0.5618696 0.480 0.409 1.000000e+00       2
MYCBP           1.510723e-01 -0.4807735 0.133 0.232 1.000000e+00       2
NOP56           1.511700e-01  0.3943933 0.413 0.419 1.000000e+00       2
SC5D            1.529374e-01 -0.6864366 0.173 0.338 1.000000e+00       2
ZDHHC14.2       1.529865e-01  0.6919599 0.253 0.160 1.000000e+00       2
CCDC171.2       1.532331e-01 -0.8665153 0.080 0.151 1.000000e+00       2
CTBP2           1.532335e-01 -0.4582361 0.347 0.322 1.000000e+00       2
ICE1.2          1.533816e-01 -0.6421883 0.373 0.361 1.000000e+00       2
UBE2W           1.535300e-01 -0.3471954 0.400 0.365 1.000000e+00       2
FKBP5.2         1.540743e-01 -1.0382538 0.187 0.363 1.000000e+00       2
SERTAD3         1.542732e-01 -0.4862324 0.173 0.325 1.000000e+00       2
CNN1.2          1.543228e-01 -0.8779693 0.133 0.206 1.000000e+00       2
RALGAPA2        1.544220e-01 -0.4263525 0.080 0.157 1.000000e+00       2
PDCD4.2         1.548697e-01 -0.5817543 0.360 0.424 1.000000e+00       2
MEX3A.1         1.549195e-01  0.4738398 0.213 0.175 1.000000e+00       2
ALOX5AP.2       1.555179e-01 -0.9916007 0.053 0.194 1.000000e+00       2
SPSB1.2         1.576262e-01 -0.2831092 0.333 0.444 1.000000e+00       2
BATF3.2         1.577269e-01 -0.2971655 0.147 0.148 1.000000e+00       2
CARMIL1         1.579795e-01 -0.2604923 0.240 0.242 1.000000e+00       2
NUDT6.1         1.582320e-01 -0.7922290 0.160 0.189 1.000000e+00       2
MMS22L.1        1.596032e-01  0.3050880 0.213 0.199 1.000000e+00       2
ZCCHC2          1.599089e-01 -0.9725186 0.147 0.200 1.000000e+00       2
KIF22.2         1.605731e-01 -0.4124444 0.107 0.208 1.000000e+00       2
LRIF1.2         1.608803e-01  0.3241836 0.320 0.340 1.000000e+00       2
IL11.1          1.610340e-01 -0.5206215 0.307 0.271 1.000000e+00       2
YWHAH.2         1.613933e-01 -0.3298475 0.360 0.441 1.000000e+00       2
GYPC.2          1.617017e-01 -1.2754305 0.080 0.277 1.000000e+00       2
C1orf112.2      1.626296e-01 -0.9814184 0.120 0.243 1.000000e+00       2
CITED1          1.628358e-01 -1.1402189 0.067 0.179 1.000000e+00       2
NSG1.2          1.631466e-01 -0.5046616 0.187 0.246 1.000000e+00       2
FASN.1          1.641329e-01  0.4558609 0.387 0.342 1.000000e+00       2
CAST.2          1.646537e-01 -0.4339867 0.347 0.476 1.000000e+00       2
PSAT1.1         1.649667e-01  0.5266302 0.387 0.369 1.000000e+00       2
PDLIM1          1.655943e-01 -0.5740348 0.333 0.452 1.000000e+00       2
WNT2.2          1.658560e-01 -0.4164536 0.347 0.418 1.000000e+00       2
S1PR5           1.663810e-01 -0.5138665 0.147 0.136 1.000000e+00       2
SENP5           1.663812e-01 -0.5254933 0.267 0.404 1.000000e+00       2
FAXDC2.1        1.666967e-01 -0.6502628 0.027 0.120 1.000000e+00       2
TRNAU1AP        1.667494e-01 -0.3867764 0.200 0.310 1.000000e+00       2
PLEKHG2         1.673820e-01  0.5081662 0.333 0.208 1.000000e+00       2
CDKN2B.1        1.679635e-01  0.3275291 0.547 0.441 1.000000e+00       2
AP1S2           1.684934e-01 -0.5492335 0.373 0.445 1.000000e+00       2
AC069544.1.2    1.690777e-01  0.3069606 0.227 0.125 1.000000e+00       2
ANKRD36         1.691309e-01 -0.3842600 0.240 0.111 1.000000e+00       2
LLGL1           1.697699e-01 -0.6840291 0.133 0.168 1.000000e+00       2
MYOCD.2         1.705186e-01 -0.3173835 0.427 0.338 1.000000e+00       2
FHL3.1          1.717532e-01 -0.4646773 0.280 0.315 1.000000e+00       2
SEMA3D.2        1.719145e-01 -0.5904480 0.133 0.132 1.000000e+00       2
DTX3L.2         1.739700e-01 -0.7028574 0.200 0.196 1.000000e+00       2
PSMB8.2         1.740792e-01 -0.9536215 0.053 0.174 1.000000e+00       2
MTSS1.1         1.746234e-01 -0.6975645 0.147 0.171 1.000000e+00       2
BNIP3.2         1.750053e-01 -0.5318932 0.307 0.441 1.000000e+00       2
FBXW7           1.751145e-01 -0.5988989 0.227 0.355 1.000000e+00       2
GLIS1           1.752780e-01 -0.5789725 0.133 0.161 1.000000e+00       2
DAAM2.2         1.753330e-01  0.3383505 0.280 0.219 1.000000e+00       2
LIMS2.2         1.758802e-01  0.4612853 0.507 0.413 1.000000e+00       2
TNFRSF12A.2     1.762092e-01 -0.5236869 0.400 0.463 1.000000e+00       2
CRYGS.2         1.762639e-01 -0.9255249 0.053 0.148 1.000000e+00       2
TSPO.2          1.764276e-01  0.3472324 0.307 0.238 1.000000e+00       2
GAS1.2          1.766485e-01  0.2607327 0.467 0.387 1.000000e+00       2
NPEPL1.1        1.773640e-01  0.6717166 0.213 0.173 1.000000e+00       2
DUSP16.2        1.774192e-01  0.5629297 0.160 0.091 1.000000e+00       2
DCAF16.2        1.775295e-01 -0.2956539 0.187 0.241 1.000000e+00       2
TRIM16.2        1.776399e-01  0.4459549 0.213 0.264 1.000000e+00       2
CCNG2           1.776951e-01  0.5098869 0.400 0.372 1.000000e+00       2
MT1E.2          1.777493e-01 -0.3763753 0.267 0.325 1.000000e+00       2
ATF7IP2         1.778055e-01 -0.4904614 0.133 0.146 1.000000e+00       2
AC073611.1.2    1.778608e-01 -0.5517403 0.240 0.301 1.000000e+00       2
HACD3           1.779713e-01 -0.3656614 0.360 0.394 1.000000e+00       2
AL031714.1.1    1.782478e-01 -0.3515005 0.053 0.171 1.000000e+00       2
KLF7            1.784139e-01 -0.4329720 0.240 0.244 1.000000e+00       2
MAMSTR.1        1.786902e-01 -0.8606025 0.133 0.160 1.000000e+00       2
ANP32B.2        1.786909e-01 -0.3912938 0.400 0.480 1.000000e+00       2
HOXA10-AS.1     1.789127e-01 -0.8927803 0.120 0.216 1.000000e+00       2
INSIG1          1.796908e-01  0.2566898 0.400 0.307 1.000000e+00       2
CKS2.2          1.798022e-01 -0.8082351 0.173 0.352 1.000000e+00       2
CALD1.2         1.805263e-01 -1.0702357 0.373 0.432 1.000000e+00       2
RBPMS2          1.807507e-01 -0.4958332 0.280 0.262 1.000000e+00       2
EPHX2.2         1.819274e-01 -0.8520933 0.053 0.102 1.000000e+00       2
FLVCR1.2        1.819840e-01 -0.4567299 0.213 0.208 1.000000e+00       2
LINC00545.2     1.821524e-01 -0.6268364 0.067 0.133 1.000000e+00       2
ARID5B.2        1.828283e-01  0.4195367 0.400 0.378 1.000000e+00       2
AC012645.1.2    1.833928e-01 -0.4844651 0.160 0.116 1.000000e+00       2
MRPS26          1.836756e-01 -0.5132658 0.213 0.402 1.000000e+00       2
DNAJC12.1       1.839019e-01 -0.5007080 0.147 0.229 1.000000e+00       2
MNDA.2          1.842417e-01  0.6155562 0.227 0.105 1.000000e+00       2
DNMBP.1         1.844122e-01 -0.7677271 0.173 0.199 1.000000e+00       2
SSPN.2          1.846393e-01 -0.4498603 0.227 0.374 1.000000e+00       2
AP002884.1.2    1.848665e-01 -0.8310601 0.147 0.254 1.000000e+00       2
TTLL11.2        1.856636e-01 -0.6220797 0.133 0.114 1.000000e+00       2
DIRAS3          1.871508e-01 -0.6463440 0.213 0.170 1.000000e+00       2
ITGA10.2        1.871510e-01 -0.3938088 0.120 0.153 1.000000e+00       2
OTUD3.2         1.879555e-01 -0.5070917 0.200 0.245 1.000000e+00       2
FLOT1.2         1.883587e-01 -0.5030389 0.307 0.436 1.000000e+00       2
GATA6           1.885317e-01  0.8195745 0.467 0.423 1.000000e+00       2
NCS1.2          1.889357e-01 -0.3775138 0.280 0.310 1.000000e+00       2
PKIB.1          1.893979e-01 -0.2547545 0.133 0.196 1.000000e+00       2
AC092279.1      1.895721e-01 -0.3037798 0.173 0.231 1.000000e+00       2
PPIB.1          1.896879e-01  0.7997352 0.440 0.422 1.000000e+00       2
INAFM1          1.898038e-01  0.2921272 0.400 0.324 1.000000e+00       2
MAGI3           1.899198e-01 -1.0013977 0.093 0.215 1.000000e+00       2
PITPNM3.1       1.907911e-01  0.6320598 0.293 0.190 1.000000e+00       2
PHKG1.1         1.907911e-01 -0.6384285 0.147 0.273 1.000000e+00       2
CDKN3.1         1.916653e-01 -1.0454433 0.067 0.177 1.000000e+00       2
ATP10A.1        1.916655e-01  0.4256149 0.333 0.301 1.000000e+00       2
DAB2IP          1.928945e-01  0.2943757 0.293 0.196 1.000000e+00       2
FAM213A         1.929532e-01 -0.5240826 0.320 0.342 1.000000e+00       2
SRR             1.931876e-01 -0.8948877 0.147 0.194 1.000000e+00       2
GIPC1           1.935406e-01 -0.4212624 0.333 0.415 1.000000e+00       2
PNMT.2          1.946599e-01 -0.4607404 0.227 0.241 1.000000e+00       2
MLPH.2          1.947781e-01  0.2566051 0.227 0.153 1.000000e+00       2
TMEM154.2       1.952514e-01  0.2781421 0.187 0.145 1.000000e+00       2
TRIM16L.2       1.953699e-01 -0.4051922 0.200 0.244 1.000000e+00       2
ADSL            1.957847e-01 -0.6441992 0.173 0.253 1.000000e+00       2
IDI1            1.962002e-01 -0.6551131 0.347 0.447 1.000000e+00       2
MCRIP2          1.962002e-01 -0.8183389 0.187 0.371 1.000000e+00       2
AL365203.2.1    1.962592e-01  1.0691502 0.213 0.124 1.000000e+00       2
BMPER.2         1.963783e-01  0.4715904 0.453 0.362 1.000000e+00       2
PRDM1           1.973311e-01 -0.3981485 0.080 0.135 1.000000e+00       2
KCNAB3.1        1.981076e-01  0.7780852 0.253 0.153 1.000000e+00       2
BTBD9           1.983467e-01  0.3458523 0.253 0.153 1.000000e+00       2
ID3.2           1.984660e-01 -0.4681090 0.347 0.397 1.000000e+00       2
PROS1.1         1.987057e-01  0.2984174 0.280 0.197 1.000000e+00       2
AHCTF1.2        1.987665e-01 -0.2510505 0.320 0.258 1.000000e+00       2
OCIAD2.2        1.988264e-01 -0.6715815 0.120 0.223 1.000000e+00       2
SH3PXD2B.1      1.991866e-01 -0.4609401 0.347 0.294 1.000000e+00       2
FAM217B         1.993668e-01 -0.2665617 0.480 0.419 1.000000e+00       2
CRACR2B.1       1.996068e-01 -0.8595938 0.053 0.139 1.000000e+00       2
PCAT1.2         1.999684e-01 -0.6573464 0.133 0.214 1.000000e+00       2
KALRN           2.004502e-01 -0.8819266 0.093 0.147 1.000000e+00       2
AKIRIN2.1       2.004507e-01 -0.5244475 0.227 0.393 1.000000e+00       2
SRRM3.2         2.012362e-01 -0.6559042 0.147 0.210 1.000000e+00       2
KLF6.1          2.014783e-01  0.3368662 0.480 0.385 1.000000e+00       2
PHGDH           2.022061e-01 -0.4364465 0.307 0.392 1.000000e+00       2
GUCY1A2.1       2.039726e-01 -0.3983180 0.160 0.169 1.000000e+00       2
RTF1            2.040337e-01  0.3718602 0.520 0.474 1.000000e+00       2
DIP2C.1         2.045843e-01  0.4406563 0.453 0.316 1.000000e+00       2
LHFPL2.2        2.047065e-01 -1.0430835 0.160 0.168 1.000000e+00       2
CCDC97.2        2.049516e-01  0.3494127 0.227 0.191 1.000000e+00       2
SCG2.2          2.051198e-01 -0.8498392 0.267 0.321 1.000000e+00       2
NPTX2           2.053201e-01 -0.5541284 0.080 0.104 1.000000e+00       2
PNPLA3.2        2.053816e-01 -0.7837866 0.200 0.212 1.000000e+00       2
SMAGP.1         2.055654e-01  0.3031851 0.107 0.137 1.000000e+00       2
STXBP1          2.056273e-01  0.4537233 0.227 0.183 1.000000e+00       2
PMEPA1.2        2.072918e-01 -0.4954150 0.427 0.443 1.000000e+00       2
SLC16A4.1       2.074155e-01 -0.3790134 0.227 0.298 1.000000e+00       2
NUDT8.2         2.075392e-01 -0.5839632 0.307 0.432 1.000000e+00       2
DCTN2           2.087796e-01 -0.4833747 0.187 0.412 1.000000e+00       2
USP40           2.090283e-01 -0.3385870 0.333 0.324 1.000000e+00       2
C16orf46        2.095263e-01 -0.4729678 0.093 0.108 1.000000e+00       2
KIAA1217.2      2.103374e-01 -0.3500595 0.280 0.354 1.000000e+00       2
AC073111.5.1    2.103999e-01 -0.4715775 0.120 0.149 1.000000e+00       2
ERICH5.1        2.111509e-01 -1.0445432 0.107 0.205 1.000000e+00       2
AC007014.1.2    2.121551e-01 -0.5493278 0.080 0.126 1.000000e+00       2
CITED4          2.124068e-01 -0.5372942 0.053 0.095 1.000000e+00       2
FOXO3.2         2.125327e-01 -0.4283354 0.360 0.318 1.000000e+00       2
RTN4.1          2.127216e-01 -0.4847551 0.373 0.464 1.000000e+00       2
MCTP1.2         2.127841e-01 -0.7615682 0.187 0.156 1.000000e+00       2
EPOR.1          2.134786e-01 -0.4769793 0.267 0.308 1.000000e+00       2
NR2F2           2.136681e-01 -0.3710115 0.360 0.450 1.000000e+00       2
CGREF1          2.143640e-01  0.3051238 0.373 0.300 1.000000e+00       2
HLTF            2.144274e-01 -0.4096487 0.253 0.284 1.000000e+00       2
NDC1            2.146810e-01  0.2602105 0.200 0.202 1.000000e+00       2
TMEM64.1        2.155065e-01 -0.2885231 0.413 0.368 1.000000e+00       2
PLA2G5.2        2.167171e-01 -0.5093653 0.213 0.215 1.000000e+00       2
SNHG7.2         2.189601e-01 -0.2623572 0.533 0.448 1.000000e+00       2
RTL9.2          2.190244e-01 -0.7701394 0.227 0.204 1.000000e+00       2
TSC22D3.2       2.192171e-01 -0.5570060 0.373 0.397 1.000000e+00       2
CKB.2           2.202488e-01 -0.9089411 0.293 0.424 1.000000e+00       2
SLC12A7.1       2.202492e-01  1.0321229 0.253 0.118 1.000000e+00       2
LRRC73.1        2.204427e-01 -0.8583303 0.107 0.170 1.000000e+00       2
SLC7A11.2       2.213491e-01  0.2995246 0.280 0.247 1.000000e+00       2
PPP1R12A        2.213493e-01 -0.3342916 0.400 0.467 1.000000e+00       2
SHPK.1          2.225833e-01  0.7059735 0.147 0.105 1.000000e+00       2
UFM1            2.226485e-01  0.5035924 0.587 0.475 1.000000e+00       2
OBSCN.1         2.231697e-01  0.9468127 0.320 0.163 1.000000e+00       2
SNHG25          2.233001e-01  0.4312035 0.347 0.287 1.000000e+00       2
USP54           2.236917e-01 -0.5279369 0.133 0.155 1.000000e+00       2
KIF9-AS1.2      2.243456e-01 -0.4273297 0.107 0.133 1.000000e+00       2
DDIT3.2         2.244764e-01 -1.0028234 0.227 0.340 1.000000e+00       2
ZNF503          2.247385e-01 -0.6793245 0.253 0.303 1.000000e+00       2
ID1.2           2.250662e-01  0.3769648 0.267 0.266 1.000000e+00       2
CLEC3A.2        2.252632e-01 -0.4411936 0.253 0.342 1.000000e+00       2
SRSF3.2         2.259204e-01 -0.6251508 0.307 0.435 1.000000e+00       2
DUSP8.1         2.261832e-01  0.5537828 0.187 0.137 1.000000e+00       2
TMEM44.1        2.263151e-01 -0.7082520 0.053 0.142 1.000000e+00       2
MCM3            2.271726e-01  0.3755730 0.227 0.196 1.000000e+00       2
MGAT4B          2.273047e-01 -0.5139237 0.373 0.457 1.000000e+00       2
PITX2.1         2.275691e-01  0.4628244 0.507 0.455 1.000000e+00       2
GPX3            2.276343e-01 -0.5546328 0.293 0.267 1.000000e+00       2
KCNK12.2        2.276349e-01 -0.6279268 0.267 0.246 1.000000e+00       2
FLRT3.2         2.278997e-01 -0.3406371 0.173 0.196 1.000000e+00       2
HSBP1L1.2       2.279661e-01 -0.5474986 0.160 0.234 1.000000e+00       2
CSRNP3.2        2.282307e-01 -0.4097418 0.267 0.170 1.000000e+00       2
MMP24OS.1       2.282973e-01 -0.4789005 0.320 0.434 1.000000e+00       2
SSH3.1          2.289600e-01 -0.5634059 0.213 0.181 1.000000e+00       2
GPR143          2.291594e-01 -0.2748904 0.213 0.190 1.000000e+00       2
ANGPTL2.1       2.293592e-01 -0.6391738 0.307 0.274 1.000000e+00       2
KDM7A           2.296251e-01 -0.7132293 0.240 0.159 1.000000e+00       2
SOX11.2         2.298899e-01 -0.2944180 0.347 0.315 1.000000e+00       2
IGF2BP3.2       2.310259e-01 -0.5429397 0.253 0.301 1.000000e+00       2
SAT2.2          2.310259e-01 -0.4392423 0.373 0.457 1.000000e+00       2
PCDHGB6.1       2.316273e-01 -1.0233989 0.053 0.160 1.000000e+00       2
KTN1.1          2.316280e-01  0.3256976 0.480 0.448 1.000000e+00       2
SERPING1.2      2.324996e-01 -0.4623801 0.120 0.223 1.000000e+00       2
MIAT            2.325667e-01  0.3156009 0.067 0.115 1.000000e+00       2
GUSB.2          2.333735e-01 -0.8819418 0.147 0.251 1.000000e+00       2
LRP2BP          2.341823e-01 -0.4153358 0.160 0.150 1.000000e+00       2
DNAJC14.1       2.343170e-01  0.5382570 0.307 0.212 1.000000e+00       2
DNAAF4          2.343173e-01 -0.3501156 0.200 0.129 1.000000e+00       2
SAMD14          2.345873e-01 -0.4074959 0.107 0.081 1.000000e+00       2
MCAM.2          2.354668e-01 -0.9050228 0.227 0.172 1.000000e+00       2
H1FX            2.356024e-01 -0.7841339 0.227 0.336 1.000000e+00       2
FAM53B.1        2.368244e-01  0.4501161 0.187 0.124 1.000000e+00       2
AREG            2.368922e-01 -0.5282429 0.200 0.190 1.000000e+00       2
SMCO4.2         2.371648e-01 -0.2668042 0.347 0.355 1.000000e+00       2
FDXR            2.373009e-01 -0.5734599 0.267 0.297 1.000000e+00       2
LINC01534.1     2.375734e-01 -0.7579668 0.080 0.158 1.000000e+00       2
ABLIM1.2        2.376417e-01  0.2575220 0.267 0.202 1.000000e+00       2
PANK1.2         2.381194e-01  0.5973360 0.280 0.208 1.000000e+00       2
MYLK.2          2.384517e-01 -1.0869689 0.213 0.328 1.000000e+00       2
TBC1D2          2.395563e-01 -0.3920732 0.213 0.261 1.000000e+00       2
DMRTA1.2        2.397621e-01 -0.3089280 0.267 0.228 1.000000e+00       2
UBE2E2.2        2.398994e-01 -0.6646339 0.293 0.426 1.000000e+00       2
NTM.2           2.402425e-01  0.2502543 0.280 0.210 1.000000e+00       2
AC109322.1.1    2.403795e-01 -0.6410624 0.107 0.136 1.000000e+00       2
SLC38A1         2.403802e-01  0.4748697 0.520 0.458 1.000000e+00       2
ULK4.2          2.407239e-01 -0.8203459 0.133 0.186 1.000000e+00       2
LINC00623       2.408614e-01 -0.8856498 0.160 0.169 1.000000e+00       2
ATP6V1A         2.411371e-01 -0.3554188 0.480 0.446 1.000000e+00       2
MFSD14A         2.421029e-01  0.3496273 0.267 0.156 1.000000e+00       2
PRNCR1          2.425174e-01 -0.7017189 0.093 0.117 1.000000e+00       2
AL451050.2.1    2.432100e-01 -0.4535309 0.120 0.166 1.000000e+00       2
DNAJA1.1        2.432101e-01 -0.4750235 0.360 0.460 1.000000e+00       2
SGK1.1          2.436262e-01 -0.5658280 0.400 0.422 1.000000e+00       2
ZNF781.1        2.441811e-01 -0.5979488 0.093 0.136 1.000000e+00       2
VAMP8.1         2.441815e-01 -0.5828608 0.187 0.239 1.000000e+00       2
SLC26A11.2      2.444598e-01 -0.9000598 0.160 0.206 1.000000e+00       2
TMLHE.1         2.445986e-01 -0.7049701 0.107 0.136 1.000000e+00       2
CASTOR1.2       2.446686e-01 -0.6047288 0.187 0.207 1.000000e+00       2
SDCCAG8         2.466927e-01  0.6730889 0.440 0.383 1.000000e+00       2
FKRP.2          2.468328e-01 -0.3934551 0.147 0.197 1.000000e+00       2
ITGB1.1         2.468328e-01  0.6351723 0.520 0.472 1.000000e+00       2
COPG2.1         2.470429e-01 -0.3843955 0.213 0.215 1.000000e+00       2
TGFBR3L.1       2.476039e-01 -0.8766592 0.120 0.283 1.000000e+00       2
FAM229B.1       2.480956e-01 -0.3112486 0.347 0.438 1.000000e+00       2
ZNF280C         2.483059e-01 -0.8087772 0.107 0.131 1.000000e+00       2
SBNO2           2.491509e-01  0.4325578 0.293 0.166 1.000000e+00       2
GHR             2.492218e-01  0.4614951 0.253 0.222 1.000000e+00       2
PCDHB14.1       2.499275e-01  0.4623148 0.147 0.145 1.000000e+00       2
EXOG            2.509179e-01  0.3533424 0.293 0.303 1.000000e+00       2
C19orf73.1      2.513429e-01 -0.5136339 0.120 0.134 1.000000e+00       2
CLK1.2          2.514850e-01 -0.5287134 0.240 0.321 1.000000e+00       2
DPH6.2          2.529065e-01 -0.9775812 0.067 0.176 1.000000e+00       2
MSRA.2          2.541197e-01  0.3407212 0.213 0.256 1.000000e+00       2
AC147067.1.1    2.543340e-01 -1.0546737 0.133 0.217 1.000000e+00       2
GPC3.2          2.549772e-01 -0.2707556 0.293 0.284 1.000000e+00       2
NKD1            2.550499e-01 -0.5306922 0.147 0.212 1.000000e+00       2
AC090409.1      2.563414e-01 -0.8735058 0.040 0.133 1.000000e+00       2
PHF20.1         2.575662e-01 -0.2937282 0.307 0.346 1.000000e+00       2
ISPD.2          2.579269e-01 -0.6331533 0.067 0.173 1.000000e+00       2
HOXB3.1         2.587945e-01  0.4289712 0.333 0.254 1.000000e+00       2
TMEM131L.1      2.591564e-01  0.3063196 0.160 0.134 1.000000e+00       2
MIOS.2          2.595190e-01 -0.7188458 0.120 0.241 1.000000e+00       2
LYSMD3          2.605356e-01 -0.5143380 0.293 0.393 1.000000e+00       2
PRNP.1          2.605357e-01  0.4068148 0.387 0.371 1.000000e+00       2
LAMA2.1         2.607538e-01  0.7692615 0.347 0.223 1.000000e+00       2
DUSP2.1         2.609720e-01 -0.7446685 0.040 0.109 1.000000e+00       2
KCNK1.2         2.613363e-01 -0.6226143 0.307 0.300 1.000000e+00       2
DUSP23.1        2.630155e-01 -0.8909426 0.107 0.170 1.000000e+00       2
RPP14.1         2.653653e-01 -0.4966850 0.227 0.223 1.000000e+00       2
ZNF211          2.655120e-01 -0.8677264 0.053 0.160 1.000000e+00       2
SQLE.1          2.663239e-01  0.6226461 0.480 0.419 1.000000e+00       2
CAVIN4.2        2.677288e-01 -0.8818948 0.160 0.218 1.000000e+00       2
AC104986.2.2    2.684704e-01 -0.6784347 0.133 0.143 1.000000e+00       2
BCL2L11.1       2.693623e-01 -0.5379019 0.107 0.157 1.000000e+00       2
PFKFB4          2.702563e-01 -0.3502344 0.240 0.130 1.000000e+00       2
CLASRP.1        2.708533e-01 -0.7414771 0.200 0.174 1.000000e+00       2
TIPARP          2.715261e-01  0.4276058 0.293 0.219 1.000000e+00       2
CCDC18.2        2.720499e-01 -0.7680168 0.053 0.104 1.000000e+00       2
PSME1.2         2.720501e-01 -0.6123776 0.227 0.316 1.000000e+00       2
AP1G2.1         2.724243e-01  0.3563625 0.213 0.143 1.000000e+00       2
TMEM88.2        2.726497e-01 -1.0144386 0.333 0.397 1.000000e+00       2
CXADR.2         2.727991e-01 -1.0621587 0.040 0.175 1.000000e+00       2
TLCD2.2         2.734008e-01 -0.4901450 0.347 0.362 1.000000e+00       2
NT5DC2          2.744545e-01 -0.2874401 0.427 0.425 1.000000e+00       2
ZC3H12C.2       2.752087e-01  0.3079677 0.187 0.126 1.000000e+00       2
DET1.2          2.753590e-01 -0.9686993 0.080 0.173 1.000000e+00       2
PCDH7           2.756621e-01  0.3643641 0.413 0.386 1.000000e+00       2
MSC-AS1         2.759645e-01 -0.3438461 0.227 0.225 1.000000e+00       2
TTLL12.1        2.761158e-01  0.4817981 0.280 0.237 1.000000e+00       2
TYMS.2          2.766459e-01 -0.5556240 0.093 0.200 1.000000e+00       2
MKNK2.1         2.774041e-01 -0.8054030 0.173 0.194 1.000000e+00       2
FRMD6-AS1.2     2.780881e-01 -0.8523828 0.133 0.232 1.000000e+00       2
CALM3.1         2.780881e-01 -0.5976400 0.280 0.395 1.000000e+00       2
MMP23B.2        2.783161e-01 -0.3350724 0.360 0.410 1.000000e+00       2
PSEN2           2.783924e-01 -0.7251465 0.133 0.206 1.000000e+00       2
MATN3.2         2.791536e-01 -0.3986251 0.240 0.349 1.000000e+00       2
CNST            2.798408e-01  0.5718612 0.293 0.227 1.000000e+00       2
POPDC2.2        2.800697e-01 -0.9639957 0.213 0.175 1.000000e+00       2
SMPDL3A.2       2.816010e-01 -0.4961126 0.173 0.270 1.000000e+00       2
ACAT2.1         2.827530e-01 -0.7203693 0.160 0.345 1.000000e+00       2
TOM1.2          2.832916e-01  0.5642273 0.373 0.316 1.000000e+00       2
PHLDB2          2.835998e-01  0.2522083 0.333 0.308 1.000000e+00       2
LARP6           2.836769e-01 -0.3266828 0.347 0.467 1.000000e+00       2
DRAM1           2.841396e-01  0.2790452 0.373 0.295 1.000000e+00       2
DUSP14.1        2.845256e-01 -0.6571814 0.280 0.387 1.000000e+00       2
GBP1.2          2.848346e-01 -0.6207142 0.227 0.267 1.000000e+00       2
IMPA2.2         2.856080e-01 -1.1581211 0.093 0.216 1.000000e+00       2
KRTCAP2.1       2.862281e-01  0.3573352 0.373 0.446 1.000000e+00       2
COBLL1          2.863832e-01  0.3983791 0.413 0.405 1.000000e+00       2
SLC25A33        2.864608e-01 -0.7367840 0.133 0.309 1.000000e+00       2
URAD.1          2.868487e-01 -0.6851686 0.160 0.166 1.000000e+00       2
ST6GAL2.2       2.870042e-01 -0.7135110 0.107 0.153 1.000000e+00       2
YTHDF3-AS1.2    2.874706e-01 -0.5710696 0.147 0.197 1.000000e+00       2
GCH1.2          2.877810e-01 -0.8392199 0.200 0.222 1.000000e+00       2
TSPYL2          2.880153e-01  0.3196145 0.307 0.304 1.000000e+00       2
NR2F1-AS1.2     2.884048e-01 -0.5817603 0.293 0.318 1.000000e+00       2
NHSL1.1         2.884828e-01 -0.7799081 0.173 0.270 1.000000e+00       2
NLRP1.2         2.899665e-01 -0.5228912 0.253 0.329 1.000000e+00       2
DMXL2           2.904357e-01 -0.5832618 0.147 0.134 1.000000e+00       2
TUB             2.915336e-01 -0.7160877 0.040 0.150 1.000000e+00       2
YPEL2           2.915339e-01 -0.3485105 0.320 0.324 1.000000e+00       2
SRRD.2          2.928706e-01 -0.5444821 0.267 0.356 1.000000e+00       2
EPB41L4A-AS1.2  2.933433e-01 -0.6218101 0.400 0.441 1.000000e+00       2
GRIA3.2         2.934219e-01 -0.3859976 0.187 0.153 1.000000e+00       2
ZNF550          2.934222e-01 -0.4701810 0.107 0.212 1.000000e+00       2
MSH3            2.943694e-01  0.3646850 0.280 0.245 1.000000e+00       2
RAB5IF          2.944484e-01 -0.5928588 0.227 0.383 1.000000e+00       2
AC078850.1.2    2.948437e-01 -0.2685459 0.213 0.178 1.000000e+00       2
BCAT2           2.953187e-01 -0.2874173 0.213 0.255 1.000000e+00       2
DPYSL2.2        2.957941e-01 -0.3476923 0.373 0.345 1.000000e+00       2
PHAX            2.959527e-01 -0.4968591 0.320 0.353 1.000000e+00       2
ZNF772.1        2.977802e-01  1.0263429 0.227 0.134 1.000000e+00       2
MSC.2           2.977804e-01 -0.4141304 0.173 0.237 1.000000e+00       2
ATP6V0A1.1      2.986573e-01  0.4212323 0.387 0.305 1.000000e+00       2
RNF115.2        2.988169e-01 -0.4329385 0.413 0.402 1.000000e+00       2
SHLD3           2.989762e-01 -0.8302293 0.147 0.149 1.000000e+00       2
AUTS2           2.989765e-01 -0.3576286 0.200 0.130 1.000000e+00       2
ZNF891          2.991359e-01 -0.5917176 0.173 0.133 1.000000e+00       2
UQCC3           2.992961e-01 -0.2510360 0.320 0.442 1.000000e+00       2
UNC119B         2.994559e-01  0.6165022 0.293 0.211 1.000000e+00       2
RBBP5           3.013783e-01 -0.4280932 0.307 0.296 1.000000e+00       2
YPEL3.1         3.013783e-01 -0.8398710 0.267 0.345 1.000000e+00       2
SLC22A15.2      3.018593e-01 -0.2744119 0.173 0.187 1.000000e+00       2
DDR2.1          3.026644e-01 -0.4315935 0.267 0.277 1.000000e+00       2
SNX14.1         3.037122e-01  0.3396949 0.360 0.346 1.000000e+00       2
ARHGAP5-AS1.1   3.039544e-01 -0.3893853 0.173 0.200 1.000000e+00       2
HSPA1B.1        3.041159e-01 -0.4290668 0.213 0.233 1.000000e+00       2
ZNF358.2        3.050860e-01 -0.5770132 0.293 0.358 1.000000e+00       2
DUSP1.2         3.058960e-01 -0.3507153 0.320 0.326 1.000000e+00       2
HIST1H2AC.1     3.111957e-01 -0.3569004 0.400 0.396 1.000000e+00       2
GGCT            3.120985e-01  0.4092057 0.347 0.376 1.000000e+00       2
ZNF416          3.137442e-01 -0.6213952 0.080 0.150 1.000000e+00       2
ZC3H12A         3.147346e-01  0.3173672 0.213 0.141 1.000000e+00       2
HPS4            3.154787e-01  0.4050167 0.267 0.236 1.000000e+00       2
IDE.1           3.156443e-01 -0.3440154 0.293 0.270 1.000000e+00       2
ZNF516          3.159754e-01  0.4762322 0.293 0.208 1.000000e+00       2
HNRNPH1.1       3.160583e-01 -0.6804951 0.360 0.425 1.000000e+00       2
ATP6V0E2.2      3.163897e-01 -0.9732367 0.160 0.190 1.000000e+00       2
REL.2           3.168874e-01 -0.4352474 0.147 0.211 1.000000e+00       2
LTBR.2          3.170534e-01 -0.6302079 0.200 0.186 1.000000e+00       2
LINC00607       3.172194e-01 -0.3695245 0.253 0.184 1.000000e+00       2
LINC00672       3.178841e-01 -0.4742332 0.107 0.124 1.000000e+00       2
SLC6A17.1       3.189663e-01  0.5128618 0.387 0.266 1.000000e+00       2
RBM43.1         3.195499e-01 -0.5154158 0.280 0.178 1.000000e+00       2
KLHL5.2         3.195500e-01  0.4621346 0.360 0.277 1.000000e+00       2
NFYA.2          3.200506e-01 -0.9412104 0.160 0.167 1.000000e+00       2
GFOD1.2         3.202177e-01 -0.2753010 0.133 0.150 1.000000e+00       2
SHARPIN         3.203015e-01  0.3356844 0.413 0.401 1.000000e+00       2
HES6.1          3.203847e-01 -0.7185865 0.107 0.148 1.000000e+00       2
METRNL          3.203851e-01 -0.2752150 0.440 0.454 1.000000e+00       2
COPZ2.1         3.205523e-01 -0.7326975 0.320 0.445 1.000000e+00       2
PHLDA2.1        3.217239e-01 -0.8933062 0.373 0.400 1.000000e+00       2
HMGB2.2         3.217241e-01 -0.7991145 0.173 0.249 1.000000e+00       2
LTC4S.1         3.218914e-01 -0.8656196 0.120 0.160 1.000000e+00       2
RHOXF1-AS1      3.232349e-01 -0.5176499 0.107 0.125 1.000000e+00       2
RGS4.2          3.234856e-01 -0.7045103 0.253 0.225 1.000000e+00       2
FNIP2           3.236555e-01 -0.4137094 0.267 0.283 1.000000e+00       2
FAM162A         3.239080e-01 -0.3937060 0.347 0.449 1.000000e+00       2
TSHZ1.1         3.249191e-01 -0.3256623 0.400 0.402 1.000000e+00       2
AL139082.1      3.250878e-01 -0.5911449 0.147 0.176 1.000000e+00       2
GAMT.1          3.261860e-01  0.3171855 0.280 0.344 1.000000e+00       2
LRRC45.2        3.279647e-01 -0.6050203 0.160 0.116 1.000000e+00       2
RPS4Y1.2        3.282189e-01 -1.1048826 0.227 0.274 1.000000e+00       2
STAG1.2         3.283045e-01 -0.5676294 0.200 0.312 1.000000e+00       2
CORO2A          3.285593e-01 -0.6836853 0.267 0.284 1.000000e+00       2
AC073323.1.2    3.294947e-01 -0.2575139 0.093 0.226 1.000000e+00       2
MAK16.1         3.294947e-01 -0.5111262 0.267 0.260 1.000000e+00       2
FAM133B.2       3.295799e-01 -0.5242208 0.253 0.326 1.000000e+00       2
H2AFJ.2         3.297502e-01 -0.5910900 0.347 0.427 1.000000e+00       2
SNTA1.2         3.298353e-01 -0.9437634 0.160 0.217 1.000000e+00       2
TBX3.2          3.300909e-01  0.2954151 0.373 0.366 1.000000e+00       2
IL10RB          3.307729e-01 -0.7210548 0.093 0.125 1.000000e+00       2
TSC22D1-AS1.1   3.323966e-01 -0.9097445 0.093 0.182 1.000000e+00       2
TSPAN5          3.323970e-01 -0.4247476 0.213 0.276 1.000000e+00       2
LAMC2.2         3.324825e-01 -0.3687810 0.200 0.184 1.000000e+00       2
AL441883.1.2    3.327393e-01 -0.6555681 0.040 0.123 1.000000e+00       2
PSME2.1         3.327395e-01 -0.3052322 0.240 0.369 1.000000e+00       2
MRPL35.2        3.334252e-01 -0.5129894 0.240 0.300 1.000000e+00       2
NEIL1           3.342829e-01 -0.8558400 0.120 0.139 1.000000e+00       2
NPNT.2          3.342836e-01 -0.8412431 0.173 0.193 1.000000e+00       2
PER1.2          3.342837e-01 -0.3385295 0.213 0.202 1.000000e+00       2
MED28.2         3.346275e-01 -0.6420464 0.307 0.410 1.000000e+00       2
KATNBL1.2       3.354018e-01 -0.3059460 0.347 0.347 1.000000e+00       2
ASTE1.1         3.361763e-01 -1.0281903 0.067 0.145 1.000000e+00       2
HMGCL           3.372130e-01 -0.3905542 0.227 0.297 1.000000e+00       2
AK4             3.375587e-01 -0.4925374 0.107 0.170 1.000000e+00       2
AC092807.3.2    3.375587e-01 -0.4133066 0.307 0.347 1.000000e+00       2
USP13.2         3.384231e-01 -0.7430005 0.240 0.170 1.000000e+00       2
HIST1H2BG.2     3.387702e-01 -0.5632314 0.173 0.145 1.000000e+00       2
TIGAR           3.388572e-01 -0.3897772 0.387 0.395 1.000000e+00       2
PTGR1.1         3.392906e-01 -0.8037350 0.213 0.305 1.000000e+00       2
GCHFR.1         3.400719e-01 -0.7169186 0.133 0.311 1.000000e+00       2
ARL4D.2         3.401588e-01 -0.5679766 0.173 0.261 1.000000e+00       2
AC097376.2      3.403326e-01 -0.8009548 0.133 0.199 1.000000e+00       2
ZNF232          3.413765e-01 -0.2888187 0.120 0.137 1.000000e+00       2
TENT4B          3.413766e-01 -0.3234175 0.187 0.244 1.000000e+00       2
CCDC183.2       3.427717e-01 -0.5906257 0.093 0.163 1.000000e+00       2
PGM1            3.435581e-01 -0.3404932 0.360 0.421 1.000000e+00       2
LIMCH1.1        3.443456e-01 -0.4954379 0.173 0.206 1.000000e+00       2
SCYL3           3.445201e-01 -0.7136158 0.120 0.175 1.000000e+00       2
CRISPLD2.2      3.458362e-01 -0.3416027 0.440 0.390 1.000000e+00       2
AL050341.2.2    3.460112e-01 -0.8516408 0.120 0.213 1.000000e+00       2
APTR            3.475072e-01 -0.5130822 0.160 0.226 1.000000e+00       2
AEBP1.2         3.475948e-01 -0.3918599 0.320 0.357 1.000000e+00       2
DCUN1D2.1       3.480351e-01  0.3649122 0.213 0.134 1.000000e+00       2
NOP58.1         3.489182e-01 -0.2915545 0.347 0.299 1.000000e+00       2
AC096733.2.1    3.494480e-01 -0.6642708 0.067 0.118 1.000000e+00       2
SLC25A25-AS1.1  3.495364e-01 -0.4764030 0.187 0.117 1.000000e+00       2
PLEKHG3.2       3.496246e-01 -0.4479685 0.253 0.145 1.000000e+00       2
ZNF28           3.515734e-01  0.3346120 0.213 0.146 1.000000e+00       2
AIMP2           3.520175e-01 -0.6023116 0.320 0.341 1.000000e+00       2
ARMC7           3.524611e-01 -0.7761874 0.147 0.201 1.000000e+00       2
DDAH2.1         3.525506e-01 -0.3938513 0.333 0.355 1.000000e+00       2
DIXDC1.2        3.529952e-01  0.3308626 0.387 0.465 1.000000e+00       2
ZNF519          3.535289e-01 -0.5843536 0.160 0.149 1.000000e+00       2
DPT.1           3.540632e-01 -1.0450501 0.107 0.192 1.000000e+00       2
SMIM10.2        3.545984e-01 -0.8766486 0.067 0.158 1.000000e+00       2
BOD1L1.2        3.547772e-01 -0.3040762 0.453 0.434 1.000000e+00       2
H2AFY.2         3.548664e-01 -0.3876705 0.413 0.449 1.000000e+00       2
STK17A          3.554915e-01 -0.4377365 0.293 0.280 1.000000e+00       2
MYOZ1.2         3.562963e-01 -0.2705329 0.160 0.147 1.000000e+00       2
IGF2.1.2        3.564703e-01 -1.2843760 0.253 0.339 1.000000e+00       2
TMEM143.1       3.574603e-01 -0.8087417 0.067 0.141 1.000000e+00       2
PEX3.2          3.574607e-01 -0.6958917 0.173 0.216 1.000000e+00       2
DUS3L           3.576396e-01 -0.7769173 0.107 0.133 1.000000e+00       2
MCFD2           3.586276e-01 -0.2603011 0.467 0.444 1.000000e+00       2
ALDH3A2.2       3.593468e-01 -0.9403220 0.107 0.206 1.000000e+00       2
C4orf47         3.596167e-01 -0.4374771 0.093 0.122 1.000000e+00       2
CALB2.2         3.601570e-01 -0.6858004 0.120 0.207 1.000000e+00       2
AP000894.4      3.601570e-01 -0.4287764 0.027 0.116 1.000000e+00       2
CARD19.1        3.617810e-01 -0.3910193 0.253 0.323 1.000000e+00       2
STRADB.2        3.619614e-01 -1.0539900 0.080 0.232 1.000000e+00       2
TRAPPC6A.1      3.624137e-01 -0.5616447 0.360 0.360 1.000000e+00       2
RNF207.1        3.640439e-01  0.7350143 0.133 0.141 1.000000e+00       2
RPRML.2         3.657697e-01 -0.7178320 0.107 0.193 1.000000e+00       2
MC1R.2          3.659512e-01 -0.6380672 0.160 0.158 1.000000e+00       2
KCNG1.2         3.672268e-01 -0.6746662 0.187 0.203 1.000000e+00       2
TRAF6           3.675006e-01 -0.6641033 0.147 0.176 1.000000e+00       2
KCTD20          3.678657e-01 -0.3746615 0.360 0.419 1.000000e+00       2
RASSF8.2        3.691451e-01  0.3813179 0.480 0.429 1.000000e+00       2
SP110.1         3.707021e-01 -0.6530032 0.093 0.157 1.000000e+00       2
WDR90.2         3.708859e-01 -0.5208803 0.187 0.163 1.000000e+00       2
SGCD.2          3.710692e-01 -0.5575892 0.093 0.150 1.000000e+00       2
NOTCH3.2        3.725395e-01 -0.5994175 0.200 0.163 1.000000e+00       2
NAPA            3.728160e-01 -0.4762619 0.347 0.344 1.000000e+00       2
ATAD3B          3.745675e-01  0.5736555 0.293 0.203 1.000000e+00       2
CYP2U1.1        3.749369e-01  0.3538279 0.200 0.180 1.000000e+00       2
PRUNE2.2        3.750287e-01 -0.4950221 0.173 0.171 1.000000e+00       2
SMCO3           3.751216e-01 -0.6359917 0.160 0.207 1.000000e+00       2
FAM111A-DT.2    3.760459e-01 -0.9580019 0.080 0.166 1.000000e+00       2
AC007938.3.2    3.762315e-01 -0.2742484 0.173 0.200 1.000000e+00       2
CYTOR.2         3.762315e-01 -0.6624703 0.373 0.431 1.000000e+00       2
ZNF185.1        3.767872e-01 -0.4296091 0.200 0.198 1.000000e+00       2
AP001462.1      3.769724e-01 -0.4444144 0.133 0.111 1.000000e+00       2
MLLT11          3.773426e-01 -0.9413431 0.280 0.340 1.000000e+00       2
TERF1           3.782715e-01 -0.3603435 0.427 0.450 1.000000e+00       2
CENPH           3.786429e-01 -1.1331730 0.093 0.163 1.000000e+00       2
GMPR.2          3.786431e-01 -1.0412771 0.067 0.210 1.000000e+00       2
RCN3.2          3.788283e-01  0.2674556 0.293 0.230 1.000000e+00       2
ZFHX2.1         3.816243e-01  0.3821800 0.213 0.176 1.000000e+00       2
ALDOC.1         3.841505e-01  0.8944367 0.240 0.145 1.000000e+00       2
SCML1.1         3.848076e-01 -0.2534493 0.200 0.267 1.000000e+00       2
B4GALNT4        3.855584e-01 -0.6283563 0.067 0.148 1.000000e+00       2
IFI35.1         3.857461e-01 -0.9045604 0.120 0.176 1.000000e+00       2
ZNF239          3.858408e-01 -0.5076413 0.093 0.105 1.000000e+00       2
ZNF8.1          3.860283e-01 -0.6883868 0.120 0.164 1.000000e+00       2
TPM1.2          3.865905e-01 -1.1129639 0.360 0.462 1.000000e+00       2
L3HYPDH.1       3.869701e-01 -0.4036172 0.347 0.338 1.000000e+00       2
VMO1            3.912224e-01 -0.4659829 0.173 0.189 1.000000e+00       2
CDC42EP4.2      3.912225e-01 -0.5732827 0.107 0.127 1.000000e+00       2
ITPKA.2         3.912226e-01 -0.5025380 0.120 0.121 1.000000e+00       2
ZBTB46          3.914122e-01 -0.2736181 0.093 0.136 1.000000e+00       2
TSPAN12.2       3.940732e-01 -0.6618587 0.027 0.123 1.000000e+00       2
TENT5C.1        3.943590e-01  0.6114496 0.240 0.188 1.000000e+00       2
FAM19A2.2       3.952166e-01 -0.7132981 0.160 0.140 1.000000e+00       2
GLUL.2          3.952170e-01 -0.4510473 0.240 0.262 1.000000e+00       2
LIPE-AS1.1      3.959804e-01 -0.6743982 0.173 0.139 1.000000e+00       2
KAT6B.2         3.964584e-01  0.2543550 0.320 0.302 1.000000e+00       2
TRIM5.1         3.964584e-01 -0.6156982 0.213 0.261 1.000000e+00       2
PRICKLE1.2      3.968408e-01 -0.3668309 0.467 0.408 1.000000e+00       2
ZNF385A.2       3.971274e-01  0.5922559 0.253 0.162 1.000000e+00       2
SAMD9           3.978934e-01 -0.9396735 0.093 0.137 1.000000e+00       2
MFGE8.2         3.981811e-01 -0.3702047 0.467 0.442 1.000000e+00       2
DKK1.2          3.994276e-01 -0.6674277 0.360 0.364 1.000000e+00       2
ATP6V0C.1       4.013509e-01 -0.5294049 0.107 0.149 1.000000e+00       2
SNCA            4.021217e-01 -0.4472878 0.160 0.181 1.000000e+00       2
PRAG1.2         4.029899e-01 -0.2903801 0.333 0.285 1.000000e+00       2
HGD.2           4.032795e-01 -0.5074563 0.160 0.182 1.000000e+00       2
BHLHE40.1       4.044393e-01  0.4350123 0.400 0.329 1.000000e+00       2
EPB41L2.2       4.048264e-01  0.2976199 0.280 0.301 1.000000e+00       2
MYO18A.1        4.052134e-01 -0.6810165 0.147 0.214 1.000000e+00       2
ALDH1L2.2       4.052136e-01  0.3790753 0.400 0.369 1.000000e+00       2
PFN2.1          4.052136e-01  0.4185670 0.507 0.465 1.000000e+00       2
BLOC1S2         4.056980e-01  0.2646208 0.427 0.484 1.000000e+00       2
TMEM229B.2      4.085136e-01 -0.8031890 0.147 0.207 1.000000e+00       2
GGT1            4.085141e-01 -0.4157297 0.147 0.147 1.000000e+00       2
TBC1D23         4.093906e-01  0.4648305 0.373 0.366 1.000000e+00       2
ZNF350.2        4.096824e-01 -0.7996300 0.133 0.156 1.000000e+00       2
PLEKHA7.2       4.098776e-01 -0.6193861 0.133 0.123 1.000000e+00       2
LINC01252.2     4.106578e-01 -0.8259563 0.133 0.191 1.000000e+00       2
CHAC2           4.116353e-01 -0.8634127 0.080 0.178 1.000000e+00       2
CENPU.2         4.122220e-01 -0.4091706 0.107 0.122 1.000000e+00       2
AC068473.5      4.130051e-01 -0.3200202 0.067 0.093 1.000000e+00       2
BICC1.2         4.137886e-01 -0.7798699 0.120 0.146 1.000000e+00       2
ZNF746          4.138872e-01  0.3349642 0.267 0.237 1.000000e+00       2
CCDC50          4.147704e-01 -0.4622158 0.333 0.456 1.000000e+00       2
HOMER3          4.150650e-01 -0.3943212 0.360 0.431 1.000000e+00       2
PEMT.1          4.151633e-01 -0.5084140 0.333 0.311 1.000000e+00       2
DNAJC1.2        4.155564e-01  0.2854676 0.373 0.430 1.000000e+00       2
DCLRE1B         4.163429e-01 -0.6814971 0.067 0.149 1.000000e+00       2
CLSPN.2         4.181168e-01 -0.5515261 0.107 0.144 1.000000e+00       2
CARD8-AS1.2     4.183137e-01 -0.8813102 0.133 0.176 1.000000e+00       2
UBE2F           4.184129e-01  0.2901702 0.253 0.353 1.000000e+00       2
CELF2.1         4.193009e-01 -0.9409652 0.147 0.173 1.000000e+00       2
GALNT1.1        4.194995e-01 -0.6582004 0.427 0.426 1.000000e+00       2
CD63            4.195983e-01  0.5894863 0.320 0.441 1.000000e+00       2
CDC14B.1        4.204887e-01 -0.4064307 0.200 0.207 1.000000e+00       2
TESK1           4.208847e-01 -0.2541631 0.227 0.280 1.000000e+00       2
PON2            4.216775e-01 -0.4370535 0.360 0.410 1.000000e+00       2
CAMTA2.2        4.217766e-01 -0.2939440 0.267 0.277 1.000000e+00       2
ABRACL          4.220742e-01 -0.4973915 0.267 0.327 1.000000e+00       2
G6PC3.1         4.222727e-01 -0.4969893 0.360 0.411 1.000000e+00       2
PLEKHO1.2       4.228683e-01 -0.3624612 0.360 0.389 1.000000e+00       2
LRRN1           4.232657e-01 -0.4004104 0.133 0.135 1.000000e+00       2
RGS3.1          4.237627e-01  0.5388512 0.240 0.199 1.000000e+00       2
HENMT1.1        4.238621e-01 -0.5772938 0.067 0.095 1.000000e+00       2
UNC5B-AS1.1     4.240606e-01 -0.8366251 0.187 0.135 1.000000e+00       2
LEFTY2.2        4.248573e-01 -0.6391921 0.160 0.235 1.000000e+00       2
C2CD3.2         4.256546e-01 -0.2943927 0.267 0.215 1.000000e+00       2
FNBP1L.1        4.267520e-01 -0.6595470 0.240 0.326 1.000000e+00       2
LLGL2           4.277512e-01 -0.4396379 0.200 0.235 1.000000e+00       2
CD68.2          4.280512e-01 -0.8546365 0.187 0.216 1.000000e+00       2
ZNF34.1         4.302545e-01 -0.7889303 0.107 0.135 1.000000e+00       2
PIK3IP1         4.311586e-01 -0.4676375 0.187 0.229 1.000000e+00       2
PCDHA7          4.312590e-01 -0.3700123 0.093 0.172 1.000000e+00       2
C8orf88.2       4.314598e-01 -0.6699530 0.080 0.140 1.000000e+00       2
SCARA3          4.318620e-01 -0.2703955 0.067 0.075 1.000000e+00       2
RIMS1.2         4.320630e-01 -1.0235599 0.107 0.176 1.000000e+00       2
LETM2.2         4.320631e-01 -0.2700318 0.120 0.086 1.000000e+00       2
TRA2A           4.322643e-01 -0.3623702 0.373 0.465 1.000000e+00       2
GINS2.1         4.336733e-01 -0.9774100 0.040 0.144 1.000000e+00       2
TES.1           4.338756e-01 -0.3853356 0.400 0.441 1.000000e+00       2
SNX16.1         4.342789e-01 -0.4636989 0.160 0.225 1.000000e+00       2
HCCS            4.348843e-01 -0.5861239 0.253 0.309 1.000000e+00       2
CMTM3.2         4.354903e-01 -0.3737095 0.320 0.343 1.000000e+00       2
CLDN11.2        4.360964e-01 -0.5194931 0.067 0.131 1.000000e+00       2
RGS10           4.360967e-01 -0.2593741 0.360 0.412 1.000000e+00       2
CPEB3.2         4.369060e-01 -0.2958719 0.133 0.168 1.000000e+00       2
ZNF628.2        4.371084e-01 -0.5756934 0.067 0.117 1.000000e+00       2
FABP3.2         4.385272e-01 -0.3363684 0.147 0.137 1.000000e+00       2
MAP3K20.2       4.385272e-01 -0.5531630 0.293 0.359 1.000000e+00       2
SPRYD3.2        4.387301e-01 -0.8767334 0.213 0.262 1.000000e+00       2
CPEB4           4.394407e-01 -0.2585090 0.467 0.433 1.000000e+00       2
OLFML2A.1       4.398470e-01  0.4960027 0.173 0.126 1.000000e+00       2
DMD             4.399487e-01 -0.4870864 0.187 0.220 1.000000e+00       2
HBEGF           4.401519e-01 -0.4115635 0.253 0.276 1.000000e+00       2
CENPE.2         4.407620e-01 -0.3895000 0.173 0.108 1.000000e+00       2
BCAP31          4.414745e-01  0.5098340 0.480 0.453 1.000000e+00       2
BACH1.2         4.417801e-01 -0.3672071 0.280 0.302 1.000000e+00       2
SYNE2.2         4.423911e-01 -0.6130254 0.160 0.153 1.000000e+00       2
HMGB1.1         4.429014e-01 -0.5380143 0.413 0.441 1.000000e+00       2
PECR.2          4.432074e-01 -0.6024248 0.093 0.139 1.000000e+00       2
CD151.1         4.439222e-01  0.6122260 0.480 0.438 1.000000e+00       2
PRIM2           4.444331e-01 -0.3488929 0.187 0.200 1.000000e+00       2
ASPHD1.1        4.446376e-01 -0.6006438 0.293 0.363 1.000000e+00       2
RALGPS2.2       4.459678e-01 -0.3297583 0.387 0.368 1.000000e+00       2
NFIA.2          4.464801e-01 -0.6940260 0.213 0.223 1.000000e+00       2
DDX24.1         4.467876e-01  0.4541005 0.453 0.430 1.000000e+00       2
FOXF2.2         4.481215e-01 -0.3679824 0.187 0.198 1.000000e+00       2
ZNF653.2        4.487379e-01 -0.3914246 0.080 0.090 1.000000e+00       2
CENPF.2         4.496629e-01  0.3056986 0.187 0.138 1.000000e+00       2
GTF2F1.2        4.496635e-01 -0.4829244 0.307 0.365 1.000000e+00       2
AC124016.1.2    4.499721e-01 -0.3392102 0.120 0.139 1.000000e+00       2
ABHD6           4.502809e-01  0.7557273 0.173 0.117 1.000000e+00       2
NQO1.2          4.505898e-01  0.2640079 0.267 0.309 1.000000e+00       2
CTPS1           4.506931e-01 -0.3816900 0.360 0.340 1.000000e+00       2
ETNK2           4.518272e-01 -0.5750351 0.107 0.156 1.000000e+00       2
SORBS1          4.522401e-01 -0.3720411 0.160 0.197 1.000000e+00       2
ACBD6           4.528597e-01 -0.4847966 0.347 0.366 1.000000e+00       2
GFPT2           4.535832e-01 -0.2829340 0.440 0.418 1.000000e+00       2
SBF1.1          4.539969e-01  0.3018020 0.293 0.211 1.000000e+00       2
FILIP1L.2       4.541003e-01 -0.5529993 0.413 0.418 1.000000e+00       2
CWC25.2         4.543073e-01 -0.3912205 0.213 0.268 1.000000e+00       2
SLC38A5         4.555502e-01 -0.5689195 0.080 0.132 1.000000e+00       2
TRIM46          4.572103e-01 -0.4519303 0.160 0.114 1.000000e+00       2
TFPT.2          4.579378e-01 -0.7179223 0.200 0.357 1.000000e+00       2
SEPT1           4.594987e-01 -0.9702914 0.067 0.202 1.000000e+00       2
AKNA            4.613755e-01 -0.5683414 0.147 0.156 1.000000e+00       2
WISP1.1         4.616889e-01  0.3200411 0.347 0.246 1.000000e+00       2
NETO2           4.623158e-01 -0.3521791 0.453 0.407 1.000000e+00       2
GPI             4.626295e-01 -0.2748120 0.387 0.380 1.000000e+00       2
SLFN5           4.636756e-01 -0.2818419 0.307 0.238 1.000000e+00       2
RNF114          4.638851e-01 -0.3766983 0.240 0.333 1.000000e+00       2
ZNF461          4.640944e-01 -0.5112553 0.067 0.113 1.000000e+00       2
AC005253.1      4.647231e-01 -0.3994871 0.187 0.164 1.000000e+00       2
SAMD15.1        4.652474e-01  0.3195545 0.173 0.109 1.000000e+00       2
NOS1AP.2        4.653521e-01 -0.2553479 0.200 0.236 1.000000e+00       2
FHL2.1          4.662969e-01 -0.8477247 0.373 0.424 1.000000e+00       2
PPP2R3C.1       4.669273e-01 -0.5023786 0.373 0.336 1.000000e+00       2
AL080317.1.1    4.675581e-01  0.4259936 0.240 0.183 1.000000e+00       2
AC084036.1      4.680839e-01 -0.7667565 0.013 0.163 1.000000e+00       2
NUDT16.1        4.682946e-01 -0.5625853 0.267 0.321 1.000000e+00       2
MVD.1           4.691372e-01  0.3435985 0.333 0.286 1.000000e+00       2
AC016831.1.2    4.701915e-01 -0.8842418 0.120 0.221 1.000000e+00       2
BCL10.2         4.707192e-01 -0.5676841 0.267 0.345 1.000000e+00       2
IDH1            4.709303e-01 -0.3397072 0.387 0.401 1.000000e+00       2
CAMK4.2         4.718806e-01 -0.6831433 0.093 0.159 1.000000e+00       2
SH3BP5.2        4.720926e-01 -0.6792675 0.293 0.338 1.000000e+00       2
AC010491.1      4.723041e-01 -0.3265884 0.133 0.127 1.000000e+00       2
PQLC2L          4.733622e-01 -0.7325339 0.093 0.161 1.000000e+00       2
ZNF324.2        4.737858e-01 -0.6262612 0.133 0.145 1.000000e+00       2
Z95115.1        4.737860e-01 -0.4935822 0.040 0.119 1.000000e+00       2
ANKRD1.2        4.747355e-01 -0.5590297 0.253 0.280 1.000000e+00       2
ZNF462          4.753767e-01 -0.2753274 0.213 0.299 1.000000e+00       2
LINC01705.2     4.754827e-01 -0.3179289 0.147 0.152 1.000000e+00       2
HIST1H2AE       4.792060e-01 -0.6686677 0.067 0.159 1.000000e+00       2
LINC00702       4.795260e-01 -0.3303543 0.107 0.129 1.000000e+00       2
MBNL1-AS1.2     4.809135e-01 -0.3755032 0.120 0.201 1.000000e+00       2
HOXC9.1         4.813408e-01 -0.4987450 0.320 0.322 1.000000e+00       2
RCHY1.1         4.820892e-01 -0.8305630 0.173 0.356 1.000000e+00       2
PLPP3.2         4.832665e-01 -0.2861087 0.173 0.207 1.000000e+00       2
GDF7            4.838016e-01 -0.7274102 0.133 0.124 1.000000e+00       2
DDX58           4.838021e-01  0.4143158 0.213 0.147 1.000000e+00       2
RAC3            4.839093e-01 -0.5375377 0.240 0.306 1.000000e+00       2
C5orf38.2       4.840165e-01 -0.4110666 0.267 0.296 1.000000e+00       2
PYGO1.1         4.841237e-01 -0.5570527 0.240 0.287 1.000000e+00       2
ASB6            4.848740e-01 -0.9349076 0.160 0.188 1.000000e+00       2
CISD1.2         4.849817e-01 -0.5137745 0.333 0.450 1.000000e+00       2
SETD6           4.865921e-01 -0.8003301 0.133 0.164 1.000000e+00       2
SLPI.2          4.866997e-01 -0.5111697 0.200 0.238 1.000000e+00       2
SAMD11          4.869152e-01 -0.3276469 0.187 0.249 1.000000e+00       2
RDH14           4.869152e-01 -0.7293338 0.213 0.338 1.000000e+00       2
DNAJB6          4.873455e-01 -0.4255509 0.507 0.496 1.000000e+00       2
AL096865.1      4.876679e-01 -0.6751118 0.147 0.149 1.000000e+00       2
AL355338.1.2    4.889607e-01 -0.6184678 0.240 0.293 1.000000e+00       2
CD55.2          4.892837e-01 -0.3188887 0.373 0.398 1.000000e+00       2
RALY            4.915511e-01 -0.4859856 0.427 0.369 1.000000e+00       2
PRR15           4.924161e-01 -0.4676065 0.147 0.121 1.000000e+00       2
TMED9.2         4.926325e-01  0.3819884 0.400 0.435 1.000000e+00       2
RPL22L1         4.931738e-01 -0.3994009 0.320 0.418 1.000000e+00       2
ATN1            4.953418e-01 -0.3203503 0.373 0.342 1.000000e+00       2
RTTN            4.956674e-01 -0.8539140 0.187 0.222 1.000000e+00       2
PHYHD1.2        4.956674e-01 -0.7327441 0.200 0.272 1.000000e+00       2
INPP1           4.958841e-01 -0.8855786 0.053 0.160 1.000000e+00       2
JDP2            4.959932e-01 -0.4072953 0.333 0.346 1.000000e+00       2
TPM2.2          4.962078e-01 -0.9179253 0.387 0.441 1.000000e+00       2
GSTM3.2         4.962096e-01 -0.5510276 0.307 0.401 1.000000e+00       2
ZSCAN18.1       4.978413e-01  0.2736070 0.267 0.253 1.000000e+00       2
ZXDB            4.987121e-01  0.3717615 0.173 0.142 1.000000e+00       2
CYB561D1.2      4.989301e-01 -0.4894997 0.120 0.127 1.000000e+00       2
SYN1.2          4.991479e-01 -0.7141820 0.053 0.176 1.000000e+00       2
RUNX1           4.995841e-01  0.5100223 0.493 0.440 1.000000e+00       2
SHF             5.013293e-01 -1.0146679 0.107 0.137 1.000000e+00       2
SMIM14          5.015486e-01 -0.3871285 0.413 0.486 1.000000e+00       2
HIST1H4C.2      5.020950e-01 -0.7145252 0.240 0.358 1.000000e+00       2
APOD.2          5.022036e-01 -0.7475678 0.107 0.191 1.000000e+00       2
SMN1            5.028602e-01 -0.5403495 0.187 0.159 1.000000e+00       2
SNAPC1.1        5.030794e-01 -0.7364064 0.160 0.279 1.000000e+00       2
OSGIN1.2        5.037361e-01 -0.5949457 0.067 0.108 1.000000e+00       2
ADAT2.2         5.046124e-01 -0.6631030 0.040 0.106 1.000000e+00       2
CLUAP1.1        5.058189e-01 -0.5065041 0.213 0.252 1.000000e+00       2
VEZT            5.066973e-01 -0.3970827 0.320 0.386 1.000000e+00       2
ACTG2.2         5.077811e-01 -0.8975743 0.440 0.389 1.000000e+00       2
PKIG            5.077963e-01 -0.4778624 0.293 0.411 1.000000e+00       2
ERO1B           5.090066e-01 -0.5350921 0.080 0.124 1.000000e+00       2
SRFBP1.2        5.091168e-01 -0.7868554 0.200 0.285 1.000000e+00       2
CRTAP.2         5.091168e-01 -0.2986347 0.440 0.436 1.000000e+00       2
ACOT2           5.094472e-01 -0.4039835 0.147 0.145 1.000000e+00       2
TM7SF2.2        5.099980e-01 -0.5791528 0.293 0.296 1.000000e+00       2
GDF6.2          5.105493e-01  0.3548913 0.333 0.332 1.000000e+00       2
USP1.2          5.106596e-01 -0.5502142 0.227 0.278 1.000000e+00       2
CTNS            5.107700e-01 -0.8792770 0.120 0.203 1.000000e+00       2
CHORDC1.2       5.118736e-01 -0.2662219 0.333 0.363 1.000000e+00       2
CAB39L          5.125363e-01 -0.5977466 0.080 0.167 1.000000e+00       2
SMIM4           5.128680e-01 -0.3921881 0.360 0.376 1.000000e+00       2
CDKN1A.2        5.130891e-01 -0.6207105 0.453 0.458 1.000000e+00       2
PDGFC.1         5.136421e-01 -0.3553808 0.347 0.415 1.000000e+00       2
CENPX           5.147489e-01 -0.4891254 0.320 0.384 1.000000e+00       2
FANCB.2         5.149701e-01 -0.8755456 0.093 0.157 1.000000e+00       2
BTG1.2          5.153028e-01 -0.8333851 0.253 0.357 1.000000e+00       2
FBXL22.2        5.174100e-01 -0.8294821 0.093 0.123 1.000000e+00       2
FMO5.2          5.180767e-01 -0.9585197 0.120 0.169 1.000000e+00       2
TMEM116.1       5.196336e-01 -0.6421726 0.120 0.153 1.000000e+00       2
NKX3-1.2        5.207473e-01 -0.3780450 0.200 0.199 1.000000e+00       2
ZGLP1.2         5.225316e-01 -0.5722560 0.107 0.136 1.000000e+00       2
KLF4.2          5.234250e-01 -0.2722580 0.133 0.131 1.000000e+00       2
MPDZ            5.238719e-01 -0.3976493 0.333 0.323 1.000000e+00       2
BAG2.1          5.244309e-01 -0.3789354 0.360 0.395 1.000000e+00       2
BAG1            5.244309e-01 -0.4141125 0.373 0.456 1.000000e+00       2
DNALI1          5.249898e-01 -0.8103830 0.173 0.191 1.000000e+00       2
BIN1.2          5.254377e-01 -0.8383771 0.147 0.238 1.000000e+00       2
AFAP1.2         5.258855e-01 -0.3239069 0.267 0.223 1.000000e+00       2
MIS18BP1.2      5.263334e-01 -0.9272279 0.133 0.159 1.000000e+00       2
NT5C.2          5.266697e-01 -0.4697386 0.347 0.382 1.000000e+00       2
ZKSCAN4.1       5.268938e-01  0.2658942 0.160 0.147 1.000000e+00       2
FAM214A         5.271181e-01 -0.6161101 0.267 0.328 1.000000e+00       2
AL121658.1      5.279031e-01 -0.5150462 0.107 0.151 1.000000e+00       2
IER3-AS1.1      5.284640e-01 -0.6445974 0.173 0.164 1.000000e+00       2
MLEC.2          5.299247e-01 -0.2788495 0.387 0.438 1.000000e+00       2
CRYAB.2         5.300365e-01 -0.9997828 0.253 0.346 1.000000e+00       2
AL390957.1      5.300371e-01 -0.7084198 0.227 0.237 1.000000e+00       2
TMEM39B.1       5.300371e-01 -0.3599447 0.160 0.191 1.000000e+00       2
TMEM120A.2      5.308244e-01 -0.5246765 0.307 0.393 1.000000e+00       2
CILP2.2         5.316118e-01 -0.7982649 0.240 0.221 1.000000e+00       2
DGKE            5.322877e-01 -0.2557063 0.133 0.202 1.000000e+00       2
AMMECR1.2       5.324008e-01 -0.4794550 0.400 0.449 1.000000e+00       2
HSD17B7.2       5.325125e-01  0.2624844 0.187 0.200 1.000000e+00       2
GTF2F2          5.338667e-01 -0.3664493 0.400 0.426 1.000000e+00       2
MBIP            5.339795e-01 -0.4112400 0.293 0.322 1.000000e+00       2
LURAP1L         5.343181e-01 -0.5728594 0.373 0.421 1.000000e+00       2
TEX10           5.346568e-01  0.2999122 0.320 0.235 1.000000e+00       2
LPP-AS2         5.348827e-01 -0.3910206 0.133 0.120 1.000000e+00       2
DCBLD1.2        5.357867e-01 -0.3906468 0.307 0.375 1.000000e+00       2
HOXC6.2         5.358997e-01 -0.4198620 0.280 0.379 1.000000e+00       2
CMIP.2          5.366914e-01 -0.3551425 0.560 0.468 1.000000e+00       2
ALG2            5.372573e-01 -0.5683713 0.373 0.395 1.000000e+00       2
SGSM2           5.375968e-01  0.2712599 0.213 0.172 1.000000e+00       2
ZYX             5.375969e-01 -0.2696133 0.427 0.431 1.000000e+00       2
OTULINL.2       5.416774e-01 -1.2563435 0.293 0.325 1.000000e+00       2
GSDMD.2         5.423630e-01 -0.7850216 0.093 0.149 1.000000e+00       2
BTBD19          5.423634e-01 -0.3723531 0.240 0.270 1.000000e+00       2
ATP7B           5.442984e-01  0.3711642 0.293 0.241 1.000000e+00       2
AMIGO2          5.442986e-01 -0.2983929 0.440 0.416 1.000000e+00       2
JUP.2           5.448684e-01 -0.2786214 0.280 0.225 1.000000e+00       2
PKIA.2          5.450965e-01 -0.3398430 0.080 0.124 1.000000e+00       2
IQCH-AS1.1      5.462369e-01 -0.8558457 0.107 0.137 1.000000e+00       2
NCAM2.2         5.466939e-01 -0.3338512 0.067 0.104 1.000000e+00       2
HMOX1.2         5.469223e-01 -0.9949605 0.133 0.220 1.000000e+00       2
NINJ1.1         5.484080e-01 -0.9146828 0.160 0.285 1.000000e+00       2
WDSUB1.1        5.485223e-01 -0.6683907 0.080 0.131 1.000000e+00       2
TRAF2.2         5.496668e-01 -0.3549904 0.120 0.124 1.000000e+00       2
EHD1            5.508123e-01 -0.4997956 0.293 0.269 1.000000e+00       2
SFT2D3          5.544859e-01 -0.4422133 0.200 0.241 1.000000e+00       2
ARAP2.2         5.548309e-01  0.3410348 0.240 0.179 1.000000e+00       2
TMEM59L         5.549458e-01 -0.3934500 0.093 0.107 1.000000e+00       2
SLC1A1.2        5.556362e-01 -0.2662732 0.387 0.352 1.000000e+00       2
CCDC85B         5.558666e-01 -0.2697623 0.400 0.448 1.000000e+00       2
SERP1.1         5.559817e-01 -0.3449569 0.480 0.451 1.000000e+00       2
C2orf69.1       5.563271e-01 -0.4190598 0.160 0.219 1.000000e+00       2
AC090204.1.2    5.564423e-01 -0.3369820 0.253 0.397 1.000000e+00       2
TPCN2.2         5.565569e-01 -0.7420993 0.133 0.184 1.000000e+00       2
MELTF-AS1       5.567878e-01 -0.4940561 0.160 0.135 1.000000e+00       2
AL135905.1.2    5.579402e-01 -0.5513245 0.173 0.115 1.000000e+00       2
GPSM2.2         5.586325e-01 -0.2838866 0.133 0.171 1.000000e+00       2
LINC01611       5.590945e-01 -0.9555652 0.093 0.168 1.000000e+00       2
SRGAP2C.1       5.595565e-01 -0.3052294 0.347 0.298 1.000000e+00       2
ZNF608.1        5.600185e-01 -0.8004609 0.093 0.192 1.000000e+00       2
IFFO2.1         5.609433e-01  0.4367695 0.240 0.215 1.000000e+00       2
EPOP            5.627949e-01 -0.5604522 0.200 0.126 1.000000e+00       2
CHST7.1         5.658101e-01 -0.7811340 0.213 0.262 1.000000e+00       2
AC087190.1      5.681348e-01 -0.9973056 0.093 0.212 1.000000e+00       2
MRPS18B.2       5.689495e-01 -0.4970341 0.213 0.306 1.000000e+00       2
UST             5.702309e-01 -0.3778837 0.253 0.262 1.000000e+00       2
MIDN            5.703474e-01 -0.3238107 0.427 0.453 1.000000e+00       2
SNAI3-AS1       5.706969e-01  0.3503798 0.253 0.182 1.000000e+00       2
TCEANC          5.713969e-01 -0.4395027 0.067 0.137 1.000000e+00       2
ITGA3.1         5.720971e-01  0.6284749 0.293 0.324 1.000000e+00       2
BCL7A.2         5.724473e-01 -0.4651878 0.173 0.216 1.000000e+00       2
RPAP1.1         5.745508e-01 -0.6542440 0.240 0.208 1.000000e+00       2
APOBEC3C.2      5.761897e-01 -0.5392003 0.187 0.305 1.000000e+00       2
INPP5A          5.775960e-01 -0.3747829 0.267 0.253 1.000000e+00       2
TNFRSF21.2      5.784171e-01 -0.4471188 0.280 0.262 1.000000e+00       2
JPT1.2          5.788865e-01 -0.7439991 0.440 0.441 1.000000e+00       2
CTSL            5.801778e-01 -0.4470119 0.427 0.399 1.000000e+00       2
MOV10L1.1       5.802958e-01 -0.7334122 0.040 0.163 1.000000e+00       2
METRN           5.802960e-01 -0.5937031 0.240 0.398 1.000000e+00       2
GNA13.1         5.817070e-01 -0.3330158 0.320 0.330 1.000000e+00       2
AC069185.1.2    5.819423e-01 -0.2622404 0.200 0.140 1.000000e+00       2
MTHFD1L         5.824132e-01 -0.3953332 0.307 0.326 1.000000e+00       2
AP002761.3.2    5.828841e-01  0.4340754 0.213 0.174 1.000000e+00       2
PDGFRB.2        5.830019e-01  0.2629657 0.387 0.315 1.000000e+00       2
KATNA1          5.846518e-01 -0.6557401 0.120 0.262 1.000000e+00       2
CCDC51.1        5.851236e-01 -0.6126232 0.200 0.301 1.000000e+00       2
KCNMB4.1        5.854776e-01 -0.4837665 0.253 0.283 1.000000e+00       2
TLL2.2          5.858316e-01  0.4013556 0.213 0.159 1.000000e+00       2
SEC23B          5.892595e-01  0.2862948 0.413 0.340 1.000000e+00       2
SPINK2.1        5.893777e-01 -0.9170736 0.160 0.199 1.000000e+00       2
TNFSF4.2        5.899697e-01 -1.0111584 0.067 0.187 1.000000e+00       2
B3GALT4         5.907991e-01 -0.3712264 0.173 0.241 1.000000e+00       2
JUN.1           5.923405e-01 -0.6171134 0.387 0.437 1.000000e+00       2
GOLGA7.2        5.923405e-01  0.4113996 0.453 0.438 1.000000e+00       2
ITM2C           5.945967e-01 -0.4490859 0.360 0.441 1.000000e+00       2
AKR1B1          5.948343e-01 -0.4219882 0.280 0.365 1.000000e+00       2
CYFIP2          5.959047e-01 -0.4143917 0.227 0.202 1.000000e+00       2
SHISA4          5.970950e-01 -0.5542030 0.293 0.307 1.000000e+00       2
PFDN1.1         5.973332e-01 -0.2994827 0.400 0.457 1.000000e+00       2
SLC3A2.2        5.999559e-01 -0.4719982 0.427 0.436 1.000000e+00       2
IQGAP2.2        6.000749e-01 -0.7955402 0.253 0.268 1.000000e+00       2
ZC4H2.1         6.016275e-01  0.3421277 0.213 0.124 1.000000e+00       2
FLNB.2          6.016276e-01 -0.3133573 0.333 0.301 1.000000e+00       2
CLDND1          6.016277e-01 -0.5553524 0.320 0.380 1.000000e+00       2
FXYD5.2         6.019862e-01 -0.6703882 0.413 0.449 1.000000e+00       2
HIF1AN          6.042590e-01 -0.3744731 0.280 0.260 1.000000e+00       2
DDR1.2          6.046182e-01  0.2653754 0.160 0.209 1.000000e+00       2
TYW5.1          6.052171e-01 -0.5630770 0.133 0.229 1.000000e+00       2
PRKAG2          6.055766e-01 -0.2871499 0.213 0.309 1.000000e+00       2
ATP1B1.1        6.066551e-01 -0.6024422 0.440 0.430 1.000000e+00       2
MINCR.2         6.076150e-01 -0.7347998 0.147 0.200 1.000000e+00       2
RECK.1          6.082156e-01  0.6964569 0.387 0.391 1.000000e+00       2
FBLIM1          6.088161e-01 -0.5601700 0.400 0.387 1.000000e+00       2
AL031058.1      6.091765e-01 -0.6984434 0.187 0.300 1.000000e+00       2
CENPQ.1         6.098971e-01 -0.8765141 0.107 0.180 1.000000e+00       2
URB2.2          6.100174e-01 -0.7582648 0.187 0.162 1.000000e+00       2
RACGAP1.2       6.102578e-01 -0.8771926 0.133 0.148 1.000000e+00       2
GCNA.1          6.107393e-01 -0.7006649 0.040 0.103 1.000000e+00       2
CFDP1           6.107394e-01 -0.3471746 0.440 0.443 1.000000e+00       2
SPATA7.2        6.112203e-01 -0.6017567 0.160 0.176 1.000000e+00       2
C1S.2           6.119429e-01 -0.2949095 0.280 0.281 1.000000e+00       2
TMEM176B.2      6.136294e-01 -0.3405149 0.067 0.113 1.000000e+00       2
ATP23.1         6.145942e-01 -0.4993131 0.067 0.188 1.000000e+00       2
AGAP9.2         6.148354e-01  0.4557466 0.200 0.174 1.000000e+00       2
PLA2G16         6.151973e-01 -0.5607227 0.227 0.215 1.000000e+00       2
PRIM1.1         6.155595e-01 -0.3439445 0.120 0.120 1.000000e+00       2
BAIAP2-DT.2     6.166462e-01 -0.2931876 0.147 0.121 1.000000e+00       2
PPARGC1B        6.179757e-01 -0.3770334 0.093 0.168 1.000000e+00       2
MARCKS.2        6.190645e-01  0.4194345 0.400 0.442 1.000000e+00       2
ZNF436          6.191851e-01 -0.8037731 0.147 0.142 1.000000e+00       2
TMC6.2          6.203960e-01  0.6440750 0.253 0.111 1.000000e+00       2
GLCCI1          6.206385e-01  0.4619611 0.373 0.360 1.000000e+00       2
TMEM92          6.216074e-01 -0.9707974 0.093 0.170 1.000000e+00       2
RABL2A.1        6.218502e-01 -0.8362342 0.147 0.156 1.000000e+00       2
FYN.1           6.220930e-01 -0.8098479 0.173 0.219 1.000000e+00       2
SLC27A4.1       6.228205e-01 -0.8671251 0.173 0.168 1.000000e+00       2
NOG.2           6.234275e-01 -0.6378060 0.200 0.325 1.000000e+00       2
ANKRD37.2       6.237913e-01 -1.0047910 0.107 0.195 1.000000e+00       2
TRIM58.2        6.242769e-01 -0.9472456 0.053 0.187 1.000000e+00       2
USP28           6.247634e-01  0.3444474 0.227 0.200 1.000000e+00       2
GADD45A.2       6.251278e-01 -0.7004334 0.453 0.426 1.000000e+00       2
CHN1            6.252495e-01 -0.5190112 0.400 0.397 1.000000e+00       2
YOD1            6.254925e-01  0.4367262 0.240 0.216 1.000000e+00       2
SLC25A4         6.267086e-01 -0.6508221 0.333 0.426 1.000000e+00       2
SLC46A3.2       6.275604e-01 -0.5387496 0.400 0.449 1.000000e+00       2
CHAC1.1         6.281692e-01 -0.8182034 0.173 0.208 1.000000e+00       2
ADAM19.2        6.292656e-01 -0.5283264 0.307 0.348 1.000000e+00       2
RBMS1.1         6.292656e-01 -0.2828065 0.520 0.454 1.000000e+00       2
AASS            6.295094e-01  0.3033422 0.293 0.234 1.000000e+00       2
MARCH9          6.301187e-01 -0.9945172 0.080 0.171 1.000000e+00       2
OTUD1.2         6.301190e-01  0.4384769 0.213 0.185 1.000000e+00       2
TMEM100         6.307288e-01 -0.8675304 0.080 0.206 1.000000e+00       2
IFNAR2.2        6.312168e-01 -0.4525729 0.240 0.240 1.000000e+00       2
SLC12A5         6.320713e-01 -0.6880420 0.107 0.222 1.000000e+00       2
FAM43A.2        6.336594e-01 -0.5715612 0.240 0.275 1.000000e+00       2
POLE.2          6.342706e-01 -0.5776833 0.120 0.156 1.000000e+00       2
TWNK.1          6.347599e-01  0.3794658 0.173 0.129 1.000000e+00       2
TRHDE.2         6.353716e-01 -0.3588319 0.200 0.241 1.000000e+00       2
ZNF223.2        6.374534e-01 -0.5228492 0.080 0.137 1.000000e+00       2
LINC02001.2     6.374534e-01 -0.4365407 0.107 0.241 1.000000e+00       2
AIFM2.2         6.376984e-01 -0.5693652 0.173 0.258 1.000000e+00       2
DPF2.1          6.381888e-01 -0.4542971 0.320 0.363 1.000000e+00       2
SOCS3.2         6.390472e-01 -0.3063018 0.187 0.174 1.000000e+00       2
TP53TG1.1       6.391699e-01 -0.6338358 0.213 0.309 1.000000e+00       2
AC016831.7      6.399059e-01 -0.7736136 0.027 0.126 1.000000e+00       2
FBXO32.2        6.401512e-01 -0.7595736 0.160 0.189 1.000000e+00       2
AP001160.3      6.423629e-01 -0.4980473 0.093 0.111 1.000000e+00       2
TARSL2.2        6.423629e-01 -0.4764469 0.427 0.434 1.000000e+00       2
AHNAK.2         6.435927e-01 -0.4076588 0.440 0.435 1.000000e+00       2
MANCR.2         6.438385e-01 -1.0563669 0.133 0.241 1.000000e+00       2
HAAO            6.442080e-01 -0.3405564 0.160 0.213 1.000000e+00       2
SKIL            6.443311e-01 -0.2659602 0.453 0.444 1.000000e+00       2
ZNF117.2        6.447004e-01 -0.7734039 0.187 0.191 1.000000e+00       2
ZCWPW1.2        6.448235e-01 -0.7433343 0.120 0.166 1.000000e+00       2
HOXB5           6.453161e-01 -0.4553742 0.227 0.281 1.000000e+00       2
NDN.2           6.459320e-01 -0.4186748 0.333 0.391 1.000000e+00       2
HSPB6.1         6.463014e-01 -0.4391236 0.200 0.185 1.000000e+00       2
ACVR2B          6.463014e-01 -0.7050568 0.133 0.176 1.000000e+00       2
NPIPB2          6.475347e-01 -0.4613207 0.067 0.130 1.000000e+00       2
RFC1            6.507449e-01 -0.3375596 0.373 0.389 1.000000e+00       2
KRCC1           6.507449e-01 -0.5845782 0.293 0.406 1.000000e+00       2
MIR503HG.1      6.524761e-01 -0.8512374 0.160 0.193 1.000000e+00       2
UCK2            6.534663e-01 -0.5669677 0.307 0.306 1.000000e+00       2
RPP38.2         6.544570e-01 -0.4315913 0.280 0.340 1.000000e+00       2
PKDCC.2         6.550765e-01 -0.6577174 0.280 0.284 1.000000e+00       2
SLC29A4.2       6.552004e-01 -0.3645846 0.133 0.117 1.000000e+00       2
AL606491.1.1    6.559442e-01 -0.6144602 0.013 0.111 1.000000e+00       2
CENPS.2         6.559442e-01 -0.3774606 0.120 0.221 1.000000e+00       2
LRPAP1          6.568124e-01  0.4585249 0.493 0.442 1.000000e+00       2
HIST1H3H        6.586744e-01 -0.4582526 0.093 0.153 1.000000e+00       2
PSD.1           6.589227e-01 -0.8197844 0.040 0.131 1.000000e+00       2
GATM.2          6.590469e-01 -0.8275663 0.227 0.233 1.000000e+00       2
DSP.1           6.597926e-01 -0.3317123 0.480 0.434 1.000000e+00       2
CHCHD7          6.599169e-01 -0.2670654 0.387 0.410 1.000000e+00       2
SGK494.2        6.614090e-01 -0.4413746 0.133 0.142 1.000000e+00       2
ABCA1.2         6.624045e-01 -0.6015858 0.253 0.253 1.000000e+00       2
NDRG2.2         6.636496e-01  0.4252165 0.160 0.140 1.000000e+00       2
LETMD1          6.655197e-01 -0.5048628 0.187 0.278 1.000000e+00       2
TXNRD2.2        6.665175e-01  0.4443733 0.320 0.358 1.000000e+00       2
VASN            6.667673e-01  0.2856526 0.293 0.272 1.000000e+00       2
MFAP5.2         6.675087e-01 -0.4060019 0.413 0.369 1.000000e+00       2
JAK1.2          6.676412e-01 -0.2689068 0.440 0.421 1.000000e+00       2
S1PR1.2         6.693898e-01 -0.6373703 0.120 0.165 1.000000e+00       2
SALL3           6.706402e-01 -0.6678999 0.160 0.186 1.000000e+00       2
PERP            6.707658e-01 -0.4402025 0.400 0.451 1.000000e+00       2
SPEF2           6.716418e-01 -0.2946988 0.107 0.100 1.000000e+00       2
TUBA1B.1        6.720173e-01 -0.3057548 0.453 0.420 1.000000e+00       2
MCM5.1          6.731443e-01 -0.3014036 0.107 0.140 1.000000e+00       2
RGS2.2          6.731444e-01 -0.2599928 0.333 0.338 1.000000e+00       2
PCDHA10.2       6.733949e-01 -0.4734941 0.147 0.155 1.000000e+00       2
PHACTR2.2       6.733950e-01 -0.2533544 0.493 0.452 1.000000e+00       2
TIMP4.2         6.751497e-01 -0.8385308 0.093 0.184 1.000000e+00       2
RELA            6.752754e-01  0.3106278 0.293 0.300 1.000000e+00       2
FUOM            6.755263e-01 -0.4730535 0.347 0.392 1.000000e+00       2
APOE.2          6.767810e-01 -0.4432578 0.213 0.247 1.000000e+00       2
CERS4           6.770323e-01 -0.4206498 0.120 0.224 1.000000e+00       2
DCTN4.2         6.813063e-01  0.6892273 0.360 0.350 1.000000e+00       2
PDZRN4.2        6.814321e-01 -0.2568126 0.107 0.173 1.000000e+00       2
AL136084.3.2    6.821871e-01 -0.9532681 0.040 0.152 1.000000e+00       2
LINC00342.1     6.824389e-01 -0.7180853 0.080 0.152 1.000000e+00       2
SLC9A3R2.1      6.826912e-01 -0.5359507 0.333 0.345 1.000000e+00       2
EID2B.1         6.839512e-01 -0.7450015 0.120 0.238 1.000000e+00       2
ZNF784          6.854643e-01 -0.4038910 0.093 0.137 1.000000e+00       2
TEX29.2         6.862214e-01 -0.3858227 0.160 0.191 1.000000e+00       2
VPS13A          6.869788e-01  0.2802540 0.293 0.262 1.000000e+00       2
PLOD3           6.882417e-01  0.4115807 0.373 0.398 1.000000e+00       2
PACS1           6.892527e-01  0.6030112 0.507 0.485 1.000000e+00       2
CEBPB.2         6.896320e-01 -0.5303569 0.440 0.426 1.000000e+00       2
EMC9            6.896320e-01 -0.5773471 0.200 0.334 1.000000e+00       2
HSD17B12        6.900114e-01 -0.3870562 0.320 0.415 1.000000e+00       2
CHERP.1         6.905173e-01  0.3429701 0.240 0.243 1.000000e+00       2
KRT18.2         6.919021e-01 -0.4337146 0.373 0.387 1.000000e+00       2
RAB7B.2         6.921625e-01  0.2784321 0.213 0.160 1.000000e+00       2
TUBA1A.2        6.930482e-01 -0.4346208 0.320 0.400 1.000000e+00       2
EPSTI1          6.930488e-01 -1.1144583 0.027 0.192 1.000000e+00       2
STMN1.1         6.930489e-01 -0.5086044 0.333 0.408 1.000000e+00       2
TUBB            6.933023e-01 -0.5754547 0.427 0.457 1.000000e+00       2
CAPN10.1        6.933023e-01  0.4030071 0.240 0.217 1.000000e+00       2
TRAPPC10.1      6.935557e-01 -0.6589023 0.227 0.260 1.000000e+00       2
PRR5            6.957109e-01 -0.4630175 0.333 0.344 1.000000e+00       2
ARNTL.1         6.958372e-01 -0.8034393 0.093 0.154 1.000000e+00       2
WNT4.2          6.959631e-01 -0.8278103 0.173 0.230 1.000000e+00       2
MICAL2          6.976145e-01 -0.3761931 0.400 0.377 1.000000e+00       2
UBE2S.2         6.987577e-01 -0.4389224 0.307 0.396 1.000000e+00       2
TYMP            6.993930e-01 -0.3773891 0.280 0.340 1.000000e+00       2
TRIM39          7.004097e-01 -0.7414381 0.213 0.149 1.000000e+00       2
SVIL.2          7.005372e-01 -0.3758460 0.267 0.272 1.000000e+00       2
RTL8A           7.034643e-01 -0.4958692 0.360 0.425 1.000000e+00       2
LIMS1           7.038465e-01 -0.3544497 0.467 0.445 1.000000e+00       2
RWDD3.1         7.042284e-01 -0.4228682 0.093 0.124 1.000000e+00       2
MDK             7.063658e-01 -0.4282574 0.400 0.450 1.000000e+00       2
ELF4            7.065232e-01  0.2883453 0.133 0.160 1.000000e+00       2
ISOC1           7.095872e-01 -0.4475515 0.200 0.238 1.000000e+00       2
SLC1A3.2        7.116323e-01 -0.3189572 0.080 0.137 1.000000e+00       2
C18orf25.2      7.116324e-01  0.3645295 0.280 0.253 1.000000e+00       2
HMGN2.2         7.117603e-01 -0.5437552 0.333 0.390 1.000000e+00       2
ZBTB8A          7.118881e-01 -0.5124403 0.213 0.239 1.000000e+00       2
RHOU.1          7.120161e-01 -0.3995378 0.267 0.275 1.000000e+00       2
NAPRT           7.139354e-01 -0.7296745 0.187 0.184 1.000000e+00       2
TRIM56.1        7.164979e-01 -0.6419067 0.267 0.312 1.000000e+00       2
GAS6-DT.2       7.176520e-01 -0.5271746 0.093 0.144 1.000000e+00       2
MGST2           7.176520e-01 -0.4079818 0.307 0.385 1.000000e+00       2
HOXA10.1        7.186784e-01 -0.4066843 0.440 0.472 1.000000e+00       2
MRPL58          7.188067e-01 -0.6544395 0.200 0.301 1.000000e+00       2
RSPO2.2         7.194485e-01 -0.2672306 0.360 0.397 1.000000e+00       2
ZCCHC10.2       7.204758e-01 -0.6094725 0.227 0.328 1.000000e+00       2
PHKG2           7.211181e-01 -0.5073926 0.320 0.360 1.000000e+00       2
PDGFA.1         7.240751e-01 -0.4944626 0.267 0.347 1.000000e+00       2
COQ2            7.242038e-01 -0.4142496 0.067 0.186 1.000000e+00       2
ENAH.2          7.254908e-01 -0.5770843 0.387 0.428 1.000000e+00       2
MICA.2          7.262635e-01  0.2614522 0.413 0.453 1.000000e+00       2
ANAPC7          7.265211e-01  0.3772534 0.307 0.266 1.000000e+00       2
MAP3K8.2        7.296142e-01 -1.1134956 0.067 0.173 1.000000e+00       2
LGALS8          7.305174e-01  0.3056896 0.240 0.183 1.000000e+00       2
MRPL46          7.307759e-01 -0.4828316 0.253 0.327 1.000000e+00       2
ZNF106          7.314213e-01 -0.2710896 0.400 0.434 1.000000e+00       2
OSER1           7.324543e-01  0.5046458 0.213 0.311 1.000000e+00       2
NEGR1.1         7.332293e-01 -0.4699407 0.160 0.117 1.000000e+00       2
SPDL1           7.334878e-01 -0.5518419 0.227 0.302 1.000000e+00       2
RMI1            7.345217e-01 -0.5552977 0.133 0.130 1.000000e+00       2
FAM174A         7.347804e-01 -0.3313699 0.467 0.449 1.000000e+00       2
ZWILCH.1        7.355563e-01 -0.4012999 0.280 0.230 1.000000e+00       2
AL031777.3.1    7.363322e-01  0.3653509 0.173 0.156 1.000000e+00       2
SEPT10.2        7.369795e-01 -0.4545081 0.360 0.347 1.000000e+00       2
TMTC1.2         7.394398e-01 -0.6091445 0.213 0.215 1.000000e+00       2
GPAM.2          7.404766e-01 -0.4821659 0.187 0.220 1.000000e+00       2
PCGF5.2         7.426812e-01 -0.4259260 0.200 0.232 1.000000e+00       2
RAD54B.1        7.428106e-01 -0.6536745 0.147 0.148 1.000000e+00       2
OSBPL3.1        7.430702e-01 -0.8978905 0.120 0.163 1.000000e+00       2
VEGFC.2         7.451476e-01 -0.8168668 0.160 0.178 1.000000e+00       2
NOTCH1          7.467068e-01 -0.5264330 0.080 0.111 1.000000e+00       2
SLC31A2         7.469668e-01 -0.2902781 0.147 0.219 1.000000e+00       2
HOXB8.2         7.482671e-01 -0.3564041 0.253 0.270 1.000000e+00       2
CD44            7.485272e-01 -0.5111781 0.200 0.236 1.000000e+00       2
CCDC28B.2       7.495680e-01 -0.4861287 0.307 0.361 1.000000e+00       2
SHROOM3.2       7.500886e-01 -0.2597594 0.240 0.223 1.000000e+00       2
HIST1H2BJ.1     7.516510e-01 -1.0331581 0.093 0.234 1.000000e+00       2
LRRCC1          7.539965e-01 -0.2749604 0.067 0.105 1.000000e+00       2
HUNK.2          7.558223e-01 -0.3504808 0.160 0.165 1.000000e+00       2
ZNF391          7.594781e-01 -0.5721820 0.133 0.161 1.000000e+00       2
TENT4A          7.594782e-01 -0.8682091 0.187 0.173 1.000000e+00       2
KLHL4.2         7.601316e-01 -0.5977203 0.200 0.309 1.000000e+00       2
PPP1R15A.1      7.605237e-01 -0.2992953 0.387 0.370 1.000000e+00       2
EFHD2           7.622236e-01 -0.2505145 0.227 0.274 1.000000e+00       2
TRAPPC5.2       7.626159e-01 -0.5873856 0.093 0.152 1.000000e+00       2
SH3BGRL3.2      7.637937e-01 -0.4741042 0.373 0.409 1.000000e+00       2
C1orf226.2      7.652338e-01 -0.5814217 0.080 0.115 1.000000e+00       2
FGF7            7.668048e-01 -0.5864006 0.293 0.303 1.000000e+00       2
BRSK2.1         7.678541e-01 -0.9452713 0.133 0.193 1.000000e+00       2
MASP1           7.689032e-01 -0.5322446 0.147 0.171 1.000000e+00       2
HMGB3.2         7.700840e-01 -0.6887594 0.213 0.309 1.000000e+00       2
SDSL            7.704776e-01 -0.4925242 0.227 0.247 1.000000e+00       2
ATP8B4.1        7.717899e-01  0.4516871 0.240 0.156 1.000000e+00       2
AVIL.1          7.728407e-01 -0.7892646 0.147 0.184 1.000000e+00       2
SMYD3           7.731035e-01 -0.3183170 0.360 0.441 1.000000e+00       2
KIAA1755.2      7.733662e-01 -0.5111967 0.067 0.128 1.000000e+00       2
EZR             7.734975e-01 -0.3070034 0.387 0.402 1.000000e+00       2
AC062029.1.2    7.738917e-01 -0.2730188 0.080 0.160 1.000000e+00       2
C21orf2.2       7.742859e-01 -0.8197470 0.187 0.285 1.000000e+00       2
ITPR2.2         7.763894e-01 -0.4082509 0.413 0.441 1.000000e+00       2
FAM129A.2       7.775733e-01 -0.8148600 0.200 0.206 1.000000e+00       2
GBP2.2          7.778363e-01 -0.8159316 0.053 0.152 1.000000e+00       2
SPON2           7.782292e-01 -1.1379179 0.387 0.385 1.000000e+00       2
SLAIN2          7.790210e-01 -0.4422989 0.320 0.367 1.000000e+00       2
COMT.2          7.792839e-01 -0.2674203 0.173 0.272 1.000000e+00       2
GSTZ1.2         7.792843e-01 -0.6510727 0.240 0.309 1.000000e+00       2
THY1.1          7.800742e-01 -0.5749768 0.373 0.434 1.000000e+00       2
ZBTB39          7.802061e-01 -0.4727841 0.067 0.125 1.000000e+00       2
KIF23.2         7.804695e-01 -0.9268584 0.147 0.182 1.000000e+00       2
S100A16.2       7.813916e-01 -0.3748115 0.320 0.399 1.000000e+00       2
FBXO2           7.817870e-01 -0.7866060 0.147 0.214 1.000000e+00       2
SGK3            7.833687e-01 -0.3133216 0.160 0.177 1.000000e+00       2
NOP14.2         7.835006e-01 -0.5075371 0.293 0.305 1.000000e+00       2
KLF2.2          7.838962e-01 -0.5109164 0.200 0.222 1.000000e+00       2
DDX10.2         7.846876e-01 -0.2773090 0.253 0.268 1.000000e+00       2
NMI.1           7.849512e-01 -0.9609345 0.067 0.187 1.000000e+00       2
AC020928.1.1    7.852152e-01 -0.6664575 0.107 0.146 1.000000e+00       2
TANGO6.1        7.862710e-01  0.3583075 0.200 0.143 1.000000e+00       2
ARHGDIB         7.865348e-01 -0.8777044 0.027 0.149 1.000000e+00       2
IGFL3           7.867990e-01 -0.4793922 0.227 0.193 1.000000e+00       2
SLC48A1.1       7.869310e-01  0.4416214 0.253 0.250 1.000000e+00       2
SNTB2           7.869310e-01 -0.3314480 0.347 0.418 1.000000e+00       2
BLOC1S5.2       7.870631e-01 -0.4105544 0.173 0.215 1.000000e+00       2
DGKZ            7.873271e-01 -0.5310959 0.240 0.265 1.000000e+00       2
S100A4          7.894399e-01 -0.3307618 0.333 0.353 1.000000e+00       2
LMOD1           7.894405e-01 -0.6025693 0.227 0.252 1.000000e+00       2
TONSL           7.907621e-01 -0.4884748 0.107 0.184 1.000000e+00       2
LIG3.1          7.931426e-01 -0.3407516 0.160 0.249 1.000000e+00       2
LRRFIP1         7.934073e-01 -0.4361173 0.413 0.447 1.000000e+00       2
AP000866.2.2    7.942012e-01 -0.3277627 0.133 0.139 1.000000e+00       2
DBN1.2          7.947307e-01 -0.4547519 0.480 0.486 1.000000e+00       2
CRNDE           7.956575e-01 -0.6105124 0.227 0.388 1.000000e+00       2
CCDC167         7.957899e-01 -0.5148794 0.280 0.328 1.000000e+00       2
KIFC2.2         7.965844e-01 -0.5862956 0.120 0.127 1.000000e+00       2
SSBP2           7.979092e-01 -0.7707659 0.053 0.154 1.000000e+00       2
PRKAA1          7.989697e-01  0.3966851 0.293 0.313 1.000000e+00       2
USPL1           8.002956e-01 -0.9413214 0.147 0.218 1.000000e+00       2
DCN             8.010355e-01 -0.3170915 0.453 0.421 1.000000e+00       2
MAFF.1          8.013567e-01  0.4631124 0.133 0.128 1.000000e+00       2
SYNGR1.2        8.024182e-01 -0.5832629 0.187 0.258 1.000000e+00       2
ZNF816.2        8.025509e-01  0.2527851 0.120 0.126 1.000000e+00       2
FH              8.028164e-01 -0.4525833 0.280 0.323 1.000000e+00       2
PPME1.2         8.042764e-01 -0.5384230 0.360 0.367 1.000000e+00       2
OSBPL10.2       8.056049e-01 -0.3328660 0.400 0.425 1.000000e+00       2
SNX25.1         8.069337e-01 -0.6666290 0.173 0.218 1.000000e+00       2
C1orf56.2       8.089278e-01 -0.6132463 0.320 0.377 1.000000e+00       2
HTATIP2         8.093268e-01 -0.9841905 0.173 0.226 1.000000e+00       2
HECTD3.2        8.095927e-01  0.3143347 0.227 0.214 1.000000e+00       2
DPPA4           8.095928e-01 -0.3783787 0.133 0.147 1.000000e+00       2
SYNC            8.102580e-01 -0.4587335 0.333 0.377 1.000000e+00       2
DNM3OS          8.105240e-01 -0.6486694 0.467 0.376 1.000000e+00       2
PSPH.2          8.106571e-01 -0.7632149 0.133 0.228 1.000000e+00       2
AC021016.3.1    8.119880e-01 -0.3512951 0.080 0.125 1.000000e+00       2
PFKFB3          8.130529e-01  0.3808978 0.213 0.170 1.000000e+00       2
EPHA4.1         8.138518e-01 -0.6659373 0.053 0.229 1.000000e+00       2
AC097534.2      8.143847e-01 -0.7735115 0.053 0.160 1.000000e+00       2
SLC49A3         8.154507e-01 -0.5308282 0.133 0.155 1.000000e+00       2
FAM122C.1       8.154507e-01 -0.3662836 0.080 0.135 1.000000e+00       2
TNNT2           8.158495e-01 -0.4357058 0.440 0.400 1.000000e+00       2
PHC1            8.159837e-01 -0.5105964 0.080 0.109 1.000000e+00       2
DHRS2.2         8.189170e-01 -0.5737039 0.200 0.230 1.000000e+00       2
AKAP17A.2       8.190504e-01 -0.4266998 0.213 0.262 1.000000e+00       2
PKN1.2          8.193172e-01 -0.5590142 0.227 0.287 1.000000e+00       2
ATL1.1          8.197175e-01 -0.3819639 0.093 0.123 1.000000e+00       2
TRAF3IP1.2      8.201177e-01  0.2900393 0.240 0.251 1.000000e+00       2
GPX7.2          8.214523e-01 -0.2665059 0.160 0.273 1.000000e+00       2
LY96            8.226538e-01 -0.7900664 0.120 0.188 1.000000e+00       2
UBE2V1          8.226540e-01 -0.4098204 0.227 0.263 1.000000e+00       2
PHF13.1         8.246576e-01 -0.3358941 0.267 0.203 1.000000e+00       2
NEDD1           8.253257e-01 -0.4710869 0.213 0.219 1.000000e+00       2
TBX4            8.266621e-01  0.4195143 0.280 0.164 1.000000e+00       2
RNF19A          8.266623e-01 -0.3021902 0.427 0.427 1.000000e+00       2
BMP7            8.275982e-01 -0.6677425 0.120 0.213 1.000000e+00       2
SMAP1           8.277320e-01  0.3090416 0.467 0.474 1.000000e+00       2
EFNB1           8.304073e-01 -0.6436723 0.187 0.174 1.000000e+00       2
CDKN2C          8.328171e-01 -0.5280860 0.120 0.154 1.000000e+00       2
DNAJB1.2        8.329511e-01 -0.4928898 0.333 0.362 1.000000e+00       2
AF213884.3      8.330849e-01  0.4305864 0.227 0.168 1.000000e+00       2
CCDC14          8.330850e-01 -0.2949317 0.347 0.336 1.000000e+00       2
SNAI1.2         8.344244e-01 -0.8600140 0.107 0.172 1.000000e+00       2
DEK.2           8.350944e-01 -0.5228119 0.320 0.383 1.000000e+00       2
SLC22A2.2       8.365686e-01 -0.6480406 0.160 0.176 1.000000e+00       2
DUSP28.1        8.373730e-01 -0.8310086 0.133 0.213 1.000000e+00       2
SMURF2.2        8.375071e-01 -0.4603622 0.387 0.379 1.000000e+00       2
AC105942.1.1    8.379094e-01 -0.8031324 0.147 0.207 1.000000e+00       2
RFXANK.1        8.380435e-01 -0.4032570 0.333 0.340 1.000000e+00       2
CENPW.1         8.395188e-01 -1.0122914 0.240 0.327 1.000000e+00       2
HOXB-AS1.2      8.419343e-01 -0.6257957 0.080 0.156 1.000000e+00       2
RHPN2.1         8.419343e-01 -0.4282372 0.133 0.152 1.000000e+00       2
PHTF2.2         8.419344e-01 -0.3220356 0.440 0.434 1.000000e+00       2
PHF1            8.422028e-01 -0.3046530 0.240 0.258 1.000000e+00       2
ST6GALNAC4      8.427398e-01 -0.6633737 0.080 0.117 1.000000e+00       2
ANKZF1.1        8.432769e-01 -0.3240711 0.213 0.210 1.000000e+00       2
KCTD2.2         8.435454e-01 -0.6968316 0.200 0.176 1.000000e+00       2
AC103702.2      8.435455e-01 -0.7748792 0.267 0.308 1.000000e+00       2
NENF.2          8.460978e-01 -0.6703209 0.253 0.398 1.000000e+00       2
PITHD1.2        8.477107e-01 -0.6555083 0.333 0.396 1.000000e+00       2
BDNF            8.486518e-01 -0.6919079 0.173 0.256 1.000000e+00       2
ZNF700.1        8.489207e-01 -0.4134034 0.067 0.155 1.000000e+00       2
CYB5A.1         8.489207e-01 -0.6295118 0.307 0.371 1.000000e+00       2
HEBP2.2         8.491897e-01 -0.6650678 0.160 0.332 1.000000e+00       2
EYS             8.494586e-01 -0.3404277 0.160 0.095 1.000000e+00       2
PTGFR.2         8.499966e-01 -0.3368007 0.173 0.175 1.000000e+00       2
CMTM7.2         8.499966e-01 -0.3944523 0.173 0.238 1.000000e+00       2
ZNF343          8.502656e-01  0.2577262 0.173 0.129 1.000000e+00       2
SPATA13.1       8.505346e-01 -0.5210253 0.253 0.278 1.000000e+00       2
CFH             8.513419e-01 -0.2894634 0.120 0.156 1.000000e+00       2
THAP9-AS1       8.517455e-01  0.2998198 0.200 0.267 1.000000e+00       2
AL451085.2.1    8.521492e-01 -0.4367400 0.067 0.130 1.000000e+00       2
AP000487.1.2    8.537644e-01 -0.5873531 0.107 0.136 1.000000e+00       2
C12orf73        8.551109e-01 -0.4001871 0.280 0.333 1.000000e+00       2
ZNF345.2        8.557841e-01 -0.8362308 0.093 0.134 1.000000e+00       2
PIR.2           8.559189e-01 -0.9990353 0.107 0.149 1.000000e+00       2
DACT3.2         8.579397e-01 -0.7528355 0.067 0.127 1.000000e+00       2
RCAN2.1         8.610402e-01 -0.9877821 0.133 0.179 1.000000e+00       2
SNHG15          8.621192e-01  0.2806987 0.240 0.326 1.000000e+00       2
CRIP2.1         8.629285e-01 -0.7082862 0.013 0.093 1.000000e+00       2
FOXQ1.2         8.629285e-01 -0.5159809 0.160 0.195 1.000000e+00       2
KIF21A          8.640079e-01 -0.5393658 0.347 0.298 1.000000e+00       2
WDR25.1         8.640079e-01 -0.3283476 0.200 0.232 1.000000e+00       2
UBE2G1          8.644127e-01 -0.4097256 0.267 0.281 1.000000e+00       2
KLF10           8.644127e-01 -0.2891914 0.267 0.310 1.000000e+00       2
MGST1.2         8.654922e-01 -1.0516100 0.267 0.348 1.000000e+00       2
PROCR.1         8.654923e-01 -0.7463801 0.293 0.380 1.000000e+00       2
THAP2.1         8.656274e-01 -0.4804776 0.200 0.236 1.000000e+00       2
FSIP1           8.658974e-01 -0.4209879 0.080 0.121 1.000000e+00       2
CYP1B1.1        8.664357e-01 -0.6283006 0.427 0.349 1.000000e+00       2
SOWAHC          8.669775e-01 -0.6307860 0.173 0.190 1.000000e+00       2
FHOD3.2         8.675176e-01 -0.4189882 0.187 0.199 1.000000e+00       2
SLMAP.1         8.680578e-01  0.5174502 0.320 0.301 1.000000e+00       2
ZNF678.2        8.688680e-01 -0.4896181 0.133 0.174 1.000000e+00       2
SERTAD1         8.690033e-01 -0.6108722 0.267 0.349 1.000000e+00       2
LINC00862.2     8.699489e-01 -0.8110021 0.147 0.222 1.000000e+00       2
GPBAR1.2        8.699489e-01 -0.5213183 0.107 0.176 1.000000e+00       2
HS3ST3A1.2      8.704894e-01 -0.6582477 0.133 0.141 1.000000e+00       2
SIRT5           8.707595e-01 -0.7632804 0.107 0.146 1.000000e+00       2
CREBRF.2        8.713002e-01 -0.3729107 0.347 0.348 1.000000e+00       2
SMYD4.1         8.718408e-01 -0.3605339 0.200 0.231 1.000000e+00       2
BZW2.1          8.719760e-01 -0.4493083 0.333 0.405 1.000000e+00       2
TRMT61A.1       8.721112e-01 -0.8709401 0.173 0.243 1.000000e+00       2
GASAL1.2        8.726518e-01 -0.7442064 0.107 0.171 1.000000e+00       2
VLDLR.1         8.729221e-01 -0.8028999 0.173 0.198 1.000000e+00       2
NEXN            8.733278e-01 -0.3843924 0.480 0.437 1.000000e+00       2
NTRK2.2         8.763030e-01 -1.1240875 0.067 0.216 1.000000e+00       2
ICAM3.2         8.764384e-01 -0.3603941 0.333 0.387 1.000000e+00       2
ZNF441          8.765737e-01  0.3576059 0.147 0.152 1.000000e+00       2
ZNF432          8.775208e-01 -0.7843911 0.227 0.235 1.000000e+00       2
ZSWIM6.2        8.780621e-01 -0.2696898 0.413 0.392 1.000000e+00       2
UBE3D.1         8.783327e-01 -1.0436263 0.133 0.179 1.000000e+00       2
ST20.1          8.786035e-01 -0.3456878 0.320 0.312 1.000000e+00       2
CCAR2           8.787388e-01  0.3188518 0.240 0.199 1.000000e+00       2
FAM122A.2       8.791449e-01 -0.4307850 0.227 0.295 1.000000e+00       2
LTB4R           8.800923e-01 -0.5027204 0.133 0.116 1.000000e+00       2
TNRC6C.1        8.800925e-01 -0.3799439 0.213 0.262 1.000000e+00       2
ARL6IP1.2       8.807694e-01 -0.3396038 0.453 0.440 1.000000e+00       2
PEX5            8.813109e-01 -0.6370814 0.147 0.152 1.000000e+00       2
TUBB2A          8.817173e-01 -0.5058753 0.480 0.437 1.000000e+00       2
FENDRR.2        8.845619e-01  0.3103924 0.240 0.184 1.000000e+00       2
TXNRD1.2        8.875437e-01 -0.3115788 0.453 0.451 1.000000e+00       2
AC002310.1      8.883571e-01 -0.4642851 0.053 0.143 1.000000e+00       2
TAS2R14.2       8.886282e-01 -0.7026766 0.093 0.148 1.000000e+00       2
TMCO6.2         8.886283e-01 -0.3533035 0.160 0.144 1.000000e+00       2
SRSF4.1         8.894419e-01 -0.4049619 0.347 0.418 1.000000e+00       2
FILIP1          8.895776e-01 -0.4187777 0.347 0.313 1.000000e+00       2
CFLAR           8.899844e-01 -0.3014502 0.187 0.233 1.000000e+00       2
ZFP36.2         8.899844e-01 -0.3421118 0.200 0.248 1.000000e+00       2
CXCL14.2        8.903759e-01 -0.6281339 0.280 0.330 1.000000e+00       2
KCTD1.2         8.907982e-01 -0.4740577 0.200 0.205 1.000000e+00       2
TAF1B           8.921548e-01 -0.6498281 0.160 0.160 1.000000e+00       2
HIST1H4H        8.922906e-01 -0.4512139 0.213 0.273 1.000000e+00       2
TMEM45A.1       8.924262e-01 -0.3048707 0.480 0.448 1.000000e+00       2
FAM111A.2       8.929690e-01 -0.3781981 0.173 0.234 1.000000e+00       2
PRPF39          8.932404e-01 -0.7453006 0.147 0.229 1.000000e+00       2
TLE2            8.948691e-01 -0.7185197 0.213 0.199 1.000000e+00       2
DUSP6.2         8.951405e-01 -0.4515747 0.227 0.228 1.000000e+00       2
SAMM50          8.964982e-01 -0.5342728 0.320 0.336 1.000000e+00       2
MACROD2.1       8.970413e-01 -0.5704123 0.080 0.164 1.000000e+00       2
FIP1L1.2        8.986710e-01 -0.5691579 0.293 0.327 1.000000e+00       2
NBL1.2          8.986710e-01 -0.3989822 0.280 0.392 1.000000e+00       2
ERG28.2         8.990785e-01 -0.4034982 0.467 0.439 1.000000e+00       2
CCNO.2          9.003011e-01 -0.4533954 0.067 0.145 1.000000e+00       2
FAP             9.007087e-01 -0.3275343 0.320 0.338 1.000000e+00       2
ABHD5           9.008446e-01 -0.3063850 0.253 0.277 1.000000e+00       2
ADGRL4.2        9.016599e-01 -0.4575605 0.160 0.150 1.000000e+00       2
GALM.1          9.019317e-01 -0.6000352 0.133 0.229 1.000000e+00       2
GTF2I           9.019317e-01 -0.5729680 0.347 0.358 1.000000e+00       2
SHB.1           9.022035e-01 -0.3016369 0.293 0.283 1.000000e+00       2
AC026979.2      9.035626e-01 -0.4480884 0.133 0.168 1.000000e+00       2
PDCL3.2         9.036985e-01 -0.6192085 0.213 0.271 1.000000e+00       2
ZC2HC1A         9.038343e-01 -0.9007572 0.067 0.143 1.000000e+00       2
UGP2.2          9.056019e-01 -0.2921221 0.440 0.422 1.000000e+00       2
FST.2           9.104981e-01 -0.3020861 0.427 0.425 1.000000e+00       2
EXPH5.2         9.111790e-01 -0.5346966 0.160 0.172 1.000000e+00       2
CEP78           9.114512e-01 -0.2801689 0.267 0.294 1.000000e+00       2
BARD1.2         9.122676e-01 -0.8910997 0.120 0.168 1.000000e+00       2
CRY1.1          9.144457e-01 -0.2551434 0.240 0.270 1.000000e+00       2
EXO5            9.152625e-01 -0.8023411 0.093 0.159 1.000000e+00       2
BTG2            9.159434e-01  0.3369622 0.227 0.255 1.000000e+00       2
ZNF701          9.189396e-01 -0.8836668 0.120 0.181 1.000000e+00       2
MFSD4B          9.193485e-01 -0.3571455 0.293 0.311 1.000000e+00       2
KLHDC8B         9.198934e-01 -0.9184152 0.120 0.163 1.000000e+00       2
DDAH1.2         9.219373e-01 -0.2794799 0.360 0.374 1.000000e+00       2
EIF4A3.1        9.226187e-01 -0.2587717 0.387 0.421 1.000000e+00       2
AKR1C1.2        9.231638e-01 -0.2673281 0.173 0.145 1.000000e+00       2
PCDHA9          9.231638e-01 -0.3540093 0.067 0.142 1.000000e+00       2
IDH3B.2         9.245269e-01 -0.5220805 0.293 0.326 1.000000e+00       2
TMEM91.2        9.265719e-01 -0.4988569 0.240 0.248 1.000000e+00       2
ODAM            9.269810e-01 -0.5472846 0.160 0.168 1.000000e+00       2
PODXL2.1        9.269810e-01 -0.5051941 0.093 0.105 1.000000e+00       2
TNFAIP6.2       9.280719e-01 -0.3571860 0.120 0.133 1.000000e+00       2
KIZ             9.288902e-01 -0.4895630 0.213 0.225 1.000000e+00       2
PLAGL1.2        9.290266e-01 -0.5458645 0.440 0.416 1.000000e+00       2
TKT.2           9.301178e-01 -0.2781119 0.453 0.460 1.000000e+00       2
SIRT7           9.306634e-01 -0.3467314 0.267 0.284 1.000000e+00       2
ZNF639.2        9.313455e-01  0.2959691 0.307 0.334 1.000000e+00       2
C8orf33.2       9.325734e-01 -0.2562649 0.413 0.418 1.000000e+00       2
TRIP13          9.343473e-01 -0.7463671 0.080 0.132 1.000000e+00       2
PCYT1B          9.346202e-01 -0.2706842 0.147 0.155 1.000000e+00       2
SAXO2.2         9.357121e-01 -0.4750754 0.080 0.119 1.000000e+00       2
XPO6.1          9.358486e-01 -0.4042813 0.307 0.330 1.000000e+00       2
FDPS.1          9.396709e-01 -0.6235307 0.440 0.424 1.000000e+00       2
ST3GAL5         9.398075e-01 -0.4904474 0.187 0.160 1.000000e+00       2
ITGA4.2         9.414461e-01 -0.3529638 0.160 0.139 1.000000e+00       2
MALT1           9.415827e-01 -0.7091499 0.200 0.218 1.000000e+00       2
CCDC81.2        9.418558e-01 -0.9830545 0.093 0.155 1.000000e+00       2
HLA-A.1         9.426753e-01 -0.4716544 0.427 0.440 1.000000e+00       2
LANCL2.1        9.440411e-01  0.3343248 0.307 0.316 1.000000e+00       2
CALCOCO2.2      9.441777e-01 -0.2569618 0.320 0.401 1.000000e+00       2
PIM1.2          9.445875e-01  0.6819672 0.240 0.255 1.000000e+00       2
Z97832.2.1      9.452705e-01  0.4897281 0.173 0.099 1.000000e+00       2
MARCH3.1        9.455437e-01 -1.0125092 0.200 0.251 1.000000e+00       2
GPR153          9.458170e-01 -0.3595393 0.120 0.131 1.000000e+00       2
NNMT.1          9.473198e-01 -0.2701224 0.467 0.473 1.000000e+00       2
SYP             9.488229e-01 -0.2506613 0.067 0.098 1.000000e+00       2
CTNNB1.2        9.489595e-01 -0.8571066 0.360 0.428 1.000000e+00       2
PANCR           9.495062e-01 -0.6639568 0.147 0.190 1.000000e+00       2
TRAF3IP2-AS1.2  9.515561e-01 -0.8289232 0.120 0.168 1.000000e+00       2
KNTC1           9.515561e-01 -0.3257095 0.120 0.172 1.000000e+00       2
CD320.2         9.518295e-01 -0.2677030 0.413 0.442 1.000000e+00       2
XKR8            9.523762e-01 -0.7091567 0.120 0.140 1.000000e+00       2
NRP1            9.538799e-01 -0.5894093 0.333 0.339 1.000000e+00       2
SBSPON          9.540166e-01 -0.7728178 0.080 0.123 1.000000e+00       2
MT2A            9.543770e-01 -0.4167675 0.493 0.475 1.000000e+00       2
DDIT4.2         9.577081e-01 -0.3376357 0.213 0.318 1.000000e+00       2
SIGIRR.1        9.578448e-01 -0.9566931 0.067 0.175 1.000000e+00       2
TGM2.2          9.579813e-01 -0.4444668 0.333 0.348 1.000000e+00       2
LNCSRLR         9.582550e-01  0.3255456 0.147 0.164 1.000000e+00       2
FOXP1.1         9.582551e-01 -0.4901665 0.307 0.375 1.000000e+00       2
CYSTM1.1        9.586653e-01 -0.5465015 0.253 0.373 1.000000e+00       2
UAP1.2          9.590755e-01 -0.3587195 0.360 0.421 1.000000e+00       2
WRNIP1          9.594858e-01 -0.3502908 0.240 0.266 1.000000e+00       2
SOX4.2          9.608534e-01 -0.3184997 0.453 0.426 1.000000e+00       2
ZNF329.1        9.609901e-01  0.7216095 0.160 0.151 1.000000e+00       2
DYRK4           9.619476e-01 -0.3996118 0.280 0.429 1.000000e+00       2
TUSC1.1         9.622211e-01 -1.0219356 0.107 0.187 1.000000e+00       2
C1orf53         9.641361e-01 -0.6812342 0.227 0.306 1.000000e+00       2
KAT8.2          9.641361e-01 -0.3837599 0.200 0.202 1.000000e+00       2
VARS2.1         9.642729e-01  0.3389566 0.187 0.178 1.000000e+00       2
SSPO.1          9.656409e-01  0.2945262 0.147 0.136 1.000000e+00       2
ARHGEF9         9.657777e-01 -0.6368989 0.253 0.299 1.000000e+00       2
PARP9.2         9.681035e-01 -1.0158844 0.160 0.186 1.000000e+00       2
IRF2BPL.1       9.685140e-01 -0.4964118 0.293 0.294 1.000000e+00       2
PMP22.2         9.691981e-01 -0.4077702 0.480 0.427 1.000000e+00       2
ATF5            9.707033e-01 -0.5237346 0.107 0.217 1.000000e+00       2
FAM219A         9.739877e-01 -0.3977684 0.213 0.205 1.000000e+00       2
SLC27A5.1       9.746720e-01 -0.4883799 0.227 0.293 1.000000e+00       2
MARCKSL1.2      9.764512e-01 -0.7538613 0.320 0.399 1.000000e+00       2
ANKRD44.1       9.772725e-01  0.2864621 0.293 0.230 1.000000e+00       2
HMMR.2          9.780937e-01 -0.8368647 0.133 0.164 1.000000e+00       2
SIRPA           9.782306e-01 -0.4372354 0.253 0.239 1.000000e+00       2
CCDC144NL-AS1.2 9.783675e-01 -0.2624176 0.147 0.223 1.000000e+00       2
PHF3.2          9.797364e-01 -0.4702529 0.280 0.382 1.000000e+00       2
ZFX             9.801470e-01 -0.5327501 0.360 0.329 1.000000e+00       2
PXMP4           9.802839e-01 -0.6926740 0.120 0.164 1.000000e+00       2
ADPGK           9.802839e-01 -0.5677626 0.347 0.343 1.000000e+00       2
VIM.2           9.804207e-01 -0.4983777 0.373 0.434 1.000000e+00       2
PPIE            9.809684e-01 -0.3996403 0.333 0.389 1.000000e+00       2
PLIN2.2         9.811053e-01 -0.6554166 0.120 0.225 1.000000e+00       2
SUGCT.2         9.823373e-01 -0.6857920 0.320 0.381 1.000000e+00       2
CFAP97.2        9.832956e-01 -0.3810015 0.320 0.361 1.000000e+00       2
FAM107B.2       9.835694e-01 -0.3701800 0.200 0.270 1.000000e+00       2
ZNF711.2        9.841170e-01 -0.6064011 0.107 0.184 1.000000e+00       2
DOHH.2          9.843908e-01 -0.4456919 0.253 0.230 1.000000e+00       2
SLC23A2.2       9.846647e-01 -0.9155569 0.133 0.181 1.000000e+00       2
ACTL10          9.852123e-01 -0.4697139 0.147 0.208 1.000000e+00       2
CD47.1          9.857599e-01  0.3816998 0.387 0.429 1.000000e+00       2
AC009005.1      9.860337e-01 -0.3932204 0.133 0.158 1.000000e+00       2
CENPV.2         9.865814e-01 -0.7877176 0.187 0.332 1.000000e+00       2
MBNL3.1         9.879505e-01  0.3575869 0.200 0.120 1.000000e+00       2
AL138885.3      9.893197e-01 -0.4452044 0.213 0.173 1.000000e+00       2
PLPP1           9.910997e-01 -0.3979561 0.307 0.345 1.000000e+00       2
PIAS2           9.916474e-01 -0.4614242 0.320 0.324 1.000000e+00       2
GPATCH4         9.928797e-01 -0.6258258 0.253 0.313 1.000000e+00       2
RASGRF2.2       9.953444e-01 -0.4323917 0.320 0.305 1.000000e+00       2
PTGDS           9.957551e-01 -0.7830058 0.267 0.340 1.000000e+00       2
VASP.2          9.957552e-01 -0.5105770 0.307 0.352 1.000000e+00       2
VAMP3.2         9.973983e-01  0.2685073 0.360 0.384 1.000000e+00       2
LINC01943.1     9.978091e-01 -0.4750904 0.053 0.108 1.000000e+00       2
CYC1            9.980830e-01 -0.4383445 0.387 0.425 1.000000e+00       2
FAM171A2.1      9.983568e-01 -0.7349882 0.120 0.148 1.000000e+00       2
                         gene
TPM1                     TPM1
TAGLN                   TAGLN
CALD1                   CALD1
TPM2                     TPM2
IGFBP7                 IGFBP7
TSC22D3               TSC22D3
C12orf75             C12orf75
CSRP2                   CSRP2
VIM                       VIM
ACTA2                   ACTA2
GLIPR1                 GLIPR1
OTULINL               OTULINL
FTH1                     FTH1
MYL9                     MYL9
FKBP5                   FKBP5
SPINT2                 SPINT2
MFGE8                   MFGE8
CEBPD                   CEBPD
C9orf16               C9orf16
SAA1                     SAA1
MYL12A                 MYL12A
ANXA2                   ANXA2
CNN3                     CNN3
CRISPLD2             CRISPLD2
IGF2.1                 IGF2.1
XRCC4                   XRCC4
PDPN                     PDPN
IL1R1                   IL1R1
PLS3                     PLS3
CCND2                   CCND2
HOXD11                 HOXD11
ACTB                     ACTB
L2HGDH                 L2HGDH
LMO4                     LMO4
GYPC                     GYPC
PLAC9                   PLAC9
MYL6                     MYL6
CSF1                     CSF1
CEP128                 CEP128
CDKN2A                 CDKN2A
RAB11FIP1           RAB11FIP1
FALEC                   FALEC
PRLR                     PRLR
KCTD16                 KCTD16
S100A10               S100A10
ACSL1                   ACSL1
ACTN1                   ACTN1
MARCKSL1             MARCKSL1
DPYSL2                 DPYSL2
GGT5                     GGT5
IRS2                     IRS2
FOXO1                   FOXO1
SOX4                     SOX4
PDCD4                   PDCD4
CDCA8                   CDCA8
MARCKS                 MARCKS
TPM4                     TPM4
INHBA                   INHBA
TXNDC16               TXNDC16
PHF19                   PHF19
ZIK1                     ZIK1
TUBA1A                 TUBA1A
CCDC69                 CCDC69
NFIL3                   NFIL3
BTG1                     BTG1
TIMP3                   TIMP3
AC242426.2         AC242426.2
MFAP5                   MFAP5
PRDX2                   PRDX2
CEBPB                   CEBPB
AC067838.1         AC067838.1
SLC7A8                 SLC7A8
CALR                     CALR
ATF4                     ATF4
CDKN1C                 CDKN1C
ZNF573                 ZNF573
ITGA1                   ITGA1
CCND1                   CCND1
VKORC1                 VKORC1
CCNO                     CCNO
MME                       MME
LIMA1                   LIMA1
COL12A1               COL12A1
PLA2G5                 PLA2G5
AL357055.3         AL357055.3
GNL1                     GNL1
IQGAP2                 IQGAP2
RPS6KA1               RPS6KA1
BOC                       BOC
PMEPA1                 PMEPA1
MSANTD2               MSANTD2
PCSK5                   PCSK5
TPRG1                   TPRG1
CLDN11                 CLDN11
RAB31                   RAB31
DDIT4                   DDIT4
DUSP10                 DUSP10
PPP1CB                 PPP1CB
TXNRD2                 TXNRD2
NR3C1                   NR3C1
ABCA1                   ABCA1
FBXO32                 FBXO32
COL3A1                 COL3A1
TMEM141               TMEM141
ABCC4                   ABCC4
TYMS                     TYMS
PDGFRA                 PDGFRA
THBS1                   THBS1
S100A16               S100A16
MIGA2                   MIGA2
POLE                     POLE
NAV2                     NAV2
SNHG7                   SNHG7
KDR                       KDR
SH3BGRL3             SH3BGRL3
NFKBIA                 NFKBIA
PYGB                     PYGB
LRRC2                   LRRC2
ITPR2                   ITPR2
FILNC1                 FILNC1
CADM1                   CADM1
CLTB                     CLTB
TMEM120A             TMEM120A
PRKAR2B               PRKAR2B
TTF2                     TTF2
RTL9                     RTL9
CALCOCO2             CALCOCO2
C1R                       C1R
SAT1                     SAT1
AC012467.2         AC012467.2
CD99                     CD99
C1QL1                   C1QL1
S1PR1                   S1PR1
EBI3                     EBI3
UACA                     UACA
DYNLL1                 DYNLL1
ZFP36L2               ZFP36L2
MKS1                     MKS1
SFMBT2                 SFMBT2
AKAP12                 AKAP12
HACD1                   HACD1
ACTBL2                 ACTBL2
TFRC                     TFRC
TGFBI                   TGFBI
CCDC171               CCDC171
ITM2B                   ITM2B
MERTK                   MERTK
TIMP2                   TIMP2
ADGRG6                 ADGRG6
YBX3                     YBX3
OSR1                     OSR1
LIG1                     LIG1
RAB40A                 RAB40A
EPS8                     EPS8
HEBP2                   HEBP2
PTGER2                 PTGER2
OGN                       OGN
TGFBR1                 TGFBR1
KIAA1024             KIAA1024
LRRC38                 LRRC38
PPME1                   PPME1
B2M                       B2M
NOS1AP                 NOS1AP
CST3                     CST3
ADRA2C                 ADRA2C
PLAGL1                 PLAGL1
FAM111A               FAM111A
ATP1A1                 ATP1A1
NREP                     NREP
ACTG2                   ACTG2
Z97832.2             Z97832.2
PPFIA3                 PPFIA3
PNRC1                   PNRC1
HOOK3                   HOOK3
HMGN3-AS1           HMGN3-AS1
SELENOW               SELENOW
NAV1                     NAV1
WDR90                   WDR90
JPT1                     JPT1
CALB2                   CALB2
PNPLA3                 PNPLA3
C1GALT1               C1GALT1
LAMC1                   LAMC1
NCK1                     NCK1
ADM                       ADM
SERPINF1             SERPINF1
WASF3                   WASF3
MSX1                     MSX1
TNFAIP8               TNFAIP8
CENPU                   CENPU
KLF9                     KLF9
SCAMP1-AS1         SCAMP1-AS1
AC098487.1         AC098487.1
TOX2                     TOX2
PDGFRB                 PDGFRB
GPX4                     GPX4
DKK2                     DKK2
SPATS2L               SPATS2L
ARL4D                   ARL4D
PRSS23                 PRSS23
COL6A2                 COL6A2
DACH1                   DACH1
FLNB                     FLNB
EIF4EBP1             EIF4EBP1
AL031714.1         AL031714.1
AQP1                     AQP1
LIX1                     LIX1
MMP2                     MMP2
DDIT3                   DDIT3
ARID5B                 ARID5B
SESN2                   SESN2
LAMTOR5               LAMTOR5
TMEM190               TMEM190
EMP3                     EMP3
CES4A                   CES4A
HLA-C                   HLA-C
TESK2                   TESK2
MYL12B                 MYL12B
ZNF320                 ZNF320
AC124016.1         AC124016.1
BRI3                     BRI3
ABCD3                   ABCD3
SLIT2                   SLIT2
ANOS1                   ANOS1
SUGCT                   SUGCT
ACTG1                   ACTG1
AL669970.3         AL669970.3
SCNN1A                 SCNN1A
LMNB1                   LMNB1
CPM                       CPM
CDC42EP1             CDC42EP1
PDLIM7                 PDLIM7
LY6E                     LY6E
MBLAC1                 MBLAC1
JAK1                     JAK1
NFIA                     NFIA
DPP4                     DPP4
MT-CO1                 MT-CO1
LGALS1                 LGALS1
EBF1                     EBF1
LMO7                     LMO7
NAMPT                   NAMPT
MNDA                     MNDA
FLRT3                   FLRT3
FGFR1                   FGFR1
MGST1                   MGST1
STC2                     STC2
AL928654.1         AL928654.1
GAS1                     GAS1
CTSK                     CTSK
PNMT                     PNMT
IFITM1                 IFITM1
UBAP1L                 UBAP1L
MRPS2                   MRPS2
BMPER                   BMPER
SMPDL3A               SMPDL3A
CNTROB                 CNTROB
HAPLN3                 HAPLN3
C1orf112             C1orf112
CD68                     CD68
CPZ                       CPZ
FRY-AS1               FRY-AS1
PCDHB14               PCDHB14
PLIN2                   PLIN2
ACKR2                   ACKR2
MYLIP                   MYLIP
PRPS2                   PRPS2
ANGPT2                 ANGPT2
MCM3AP-AS1         MCM3AP-AS1
DGCR8                   DGCR8
PTMA                     PTMA
SELENBP1             SELENBP1
ANGPTL4               ANGPTL4
LAMB1                   LAMB1
P3H2                     P3H2
CUBN                     CUBN
CERS6                   CERS6
RAD51AP1             RAD51AP1
PDLIM3                 PDLIM3
AL138966.2         AL138966.2
EIF1                     EIF1
SEPT9                   SEPT9
RNF207                 RNF207
YTHDF3-AS1         YTHDF3-AS1
KIF26B                 KIF26B
PKP2                     PKP2
ABHD14A-ACY1     ABHD14A-ACY1
UBASH3B               UBASH3B
PHACTR2               PHACTR2
P4HA2                   P4HA2
PIM1                     PIM1
GADD45A               GADD45A
RASSF4                 RASSF4
PLPP3                   PLPP3
MMP11                   MMP11
DUSP1                   DUSP1
PRDX3                   PRDX3
TCEAL4                 TCEAL4
RGS20                   RGS20
FAH                       FAH
NEK5                     NEK5
DIO3OS                 DIO3OS
SPECC1L               SPECC1L
APOC1                   APOC1
HOXB-AS3             HOXB-AS3
AC097359.2         AC097359.2
UGP2                     UGP2
IFITM3                 IFITM3
TMEM178A             TMEM178A
GNAI2                   GNAI2
ZNF653                 ZNF653
MANSC1                 MANSC1
AL117379.1         AL117379.1
LENG8                   LENG8
FADS3                   FADS3
AL135905.1         AL135905.1
HGH1                     HGH1
AC011978.2         AC011978.2
ODAPH                   ODAPH
RBMS3                   RBMS3
SLC3A2                 SLC3A2
CAV1                     CAV1
SLPI                     SLPI
BATF3                   BATF3
CD55                     CD55
SDHAF3                 SDHAF3
BCO2                     BCO2
ANKRD28               ANKRD28
C9orf47               C9orf47
TM6SF2                 TM6SF2
MATN2                   MATN2
KCNC4                   KCNC4
FAM117B               FAM117B
SOCS1                   SOCS1
NEDD4                   NEDD4
FAM171A2             FAM171A2
SLC22A3               SLC22A3
CTSZ                     CTSZ
CCDC183               CCDC183
MAN1A1                 MAN1A1
PRDX6                   PRDX6
TANGO6                 TANGO6
ASNS                     ASNS
CENPE                   CENPE
FEZ1                     FEZ1
MGP                       MGP
CCNA2                   CCNA2
BOD1L1                 BOD1L1
HLA-B                   HLA-B
RGS2                     RGS2
PPP3CA                 PPP3CA
CTGF                     CTGF
TRIM65                 TRIM65
TTLL7                   TTLL7
SERINC5               SERINC5
SRPX2                   SRPX2
ZC3H12C               ZC3H12C
ZFAS1                   ZFAS1
NAP1L1                 NAP1L1
ZNF136                 ZNF136
CANX                     CANX
TMED9                   TMED9
CSPG4                   CSPG4
HAUS4                   HAUS4
UBC                       UBC
F3                         F3
NKAIN4                 NKAIN4
XPOT                     XPOT
SPSB1                   SPSB1
PRAC1                   PRAC1
MSX2                     MSX2
ABTB2                   ABTB2
REXO5                   REXO5
TNFAIP1               TNFAIP1
MAML3                   MAML3
BLOC1S5               BLOC1S5
MBNL2                   MBNL2
NDRG2                   NDRG2
UBOX5                   UBOX5
TMED3                   TMED3
HAND2-AS1           HAND2-AS1
RPP38                   RPP38
BEX3                     BEX3
ISPD                     ISPD
ECE1                     ECE1
FILIP1L               FILIP1L
ISYNA1                 ISYNA1
ADAM23                 ADAM23
BTBD3                   BTBD3
PTPRJ                   PTPRJ
CPEB3                   CPEB3
AL031777.3         AL031777.3
SLC26A2               SLC26A2
DERL3                   DERL3
ARSE                     ARSE
ZNF610                 ZNF610
GAS7                     GAS7
CHD1L                   CHD1L
BRCC3                   BRCC3
HIF3A                   HIF3A
EGFR                     EGFR
LRRC49                 LRRC49
FZD2                     FZD2
OCIAD2                 OCIAD2
TUT4                     TUT4
NUP133                 NUP133
C5orf38               C5orf38
TIMP4                   TIMP4
CBR3                     CBR3
FGD4                     FGD4
GNPDA1                 GNPDA1
ZNF345                 ZNF345
PPL                       PPL
RPS27                   RPS27
AP000695.2         AP000695.2
GPBAR1                 GPBAR1
SPOUT1                 SPOUT1
PRICKLE1             PRICKLE1
METTL7A               METTL7A
SNHG12                 SNHG12
UTP23                   UTP23
PTMS                     PTMS
SACS                     SACS
PMP22                   PMP22
KCTD2                   KCTD2
C1QTNF6               C1QTNF6
AMMECR1               AMMECR1
DEPP1                   DEPP1
IFFO1                   IFFO1
GPC4                     GPC4
PBX1                     PBX1
FST                       FST
TSLP                     TSLP
SLC37A3               SLC37A3
APBB3                   APBB3
SARDH                   SARDH
ZNF496                 ZNF496
ZDHHC14               ZDHHC14
ZWINT                   ZWINT
TRIB3                   TRIB3
FOXF1                   FOXF1
MYO10                   MYO10
ZNF43                   ZNF43
ITPKA                   ITPKA
RNF6                     RNF6
NORAD                   NORAD
FAM129A               FAM129A
PER1                     PER1
ALKAL2                 ALKAL2
GPSM2                   GPSM2
ERRFI1                 ERRFI1
ZBTB16                 ZBTB16
S1PR3                   S1PR3
HSPB1                   HSPB1
ID3                       ID3
ADAMTS9               ADAMTS9
SLC24A1               SLC24A1
TMEM91                 TMEM91
HIF1A                   HIF1A
AHCTF1                 AHCTF1
CPD                       CPD
DSC3                     DSC3
LHFPL6                 LHFPL6
CRYAB                   CRYAB
KIF22                   KIF22
TTC21B                 TTC21B
PGAP1                   PGAP1
SLC46A3               SLC46A3
PRNP                     PRNP
TMEM98                 TMEM98
GNG11                   GNG11
LZTS1                   LZTS1
PLEKHG3               PLEKHG3
SEPSECS               SEPSECS
FOXO3                   FOXO3
SOX11                   SOX11
C15orf39             C15orf39
PKIA                     PKIA
HAND2                   HAND2
STAG1                   STAG1
KAT6B                   KAT6B
B3GNT5                 B3GNT5
IKZF2                   IKZF2
ALG9                     ALG9
YARS                     YARS
FGFRL1                 FGFRL1
GCH1                     GCH1
HIGD1A                 HIGD1A
FAM89A                 FAM89A
KIF11                   KIF11
PHTF2                   PHTF2
HS6ST2                 HS6ST2
GJA1                     GJA1
ANAPC16               ANAPC16
IFNGR1                 IFNGR1
PCK2                     PCK2
BASP1                   BASP1
CLSPN                   CLSPN
RASAL2-AS1         RASAL2-AS1
UCHL1                   UCHL1
CILP2                   CILP2
AC132192.2         AC132192.2
OSGIN1                 OSGIN1
EXOSC9                 EXOSC9
GFPT1                   GFPT1
TMEM160               TMEM160
ZNF772                 ZNF772
AC069544.1         AC069544.1
CNOT6L                 CNOT6L
GATAD2B               GATAD2B
ZNF816                 ZNF816
TUBB6                   TUBB6
CARD8-AS1           CARD8-AS1
CYCS                     CYCS
MRAS                     MRAS
FIBIN                   FIBIN
MT1X                     MT1X
DNAJB2                 DNAJB2
ZFPM2                   ZFPM2
RNF112                 RNF112
PHF3                     PHF3
AC012615.1         AC012615.1
ODC1                     ODC1
SPOCK1                 SPOCK1
CLU                       CLU
KIAA0232             KIAA0232
APOE                     APOE
PTPRA                   PTPRA
CDC42EP3             CDC42EP3
MAN2B1                 MAN2B1
SLC5A6                 SLC5A6
ICE1                     ICE1
SPINT1                 SPINT1
CLK1                     CLK1
SCRG1                   SCRG1
KREMEN1               KREMEN1
KCNG1                   KCNG1
DYNLL2                 DYNLL2
PDLIM2                 PDLIM2
DCBLD1                 DCBLD1
KLHDC10               KLHDC10
ARHGAP26             ARHGAP26
FLCN                     FLCN
FRZB                     FRZB
ADARB1                 ADARB1
MAP3K7CL             MAP3K7CL
LAMC3                   LAMC3
DEK                       DEK
RASA3                   RASA3
AC103740.2         AC103740.2
CTNNAL1               CTNNAL1
ARPC5                   ARPC5
PYCARD                 PYCARD
UBXN4                   UBXN4
DACT3                   DACT3
POLR2J3.1           POLR2J3.1
THRA                     THRA
C1RL                     C1RL
MLEC                     MLEC
NAA25                   NAA25
SURF4                   SURF4
AC104986.2         AC104986.2
SLC19A2               SLC19A2
DDR1                     DDR1
MT-CYB                 MT-CYB
NQO2                     NQO2
TNFRSF1A             TNFRSF1A
PRKD1                   PRKD1
ENDOD1                 ENDOD1
SIX4                     SIX4
AC096733.2         AC096733.2
CDC37L1               CDC37L1
METTL27               METTL27
PAPPA                   PAPPA
KCNS3                   KCNS3
TFPI                     TFPI
FHL1                     FHL1
DDAH1                   DDAH1
SOCS2                   SOCS2
RTN3                     RTN3
ALDH3A2               ALDH3A2
SPTBN1                 SPTBN1
XPC                       XPC
PLEKHA4               PLEKHA4
MYDGF                   MYDGF
EIF1B                   EIF1B
SSR4                     SSR4
TNKS                     TNKS
COTL1                   COTL1
C1S                       C1S
ANK3                     ANK3
SIPA1L1               SIPA1L1
AP002884.1         AP002884.1
SLCO4A1               SLCO4A1
FAM53B                 FAM53B
UHMK1                   UHMK1
TMEM158               TMEM158
PIK3R1                 PIK3R1
SNAI2                   SNAI2
NPTXR                   NPTXR
GDF6                     GDF6
TPGS2                   TPGS2
ANXA5                   ANXA5
HLA-DPB1             HLA-DPB1
HSPB7                   HSPB7
STXBP5L               STXBP5L
MT-ATP6               MT-ATP6
ALCAM                   ALCAM
FMO5                     FMO5
CNPY2                   CNPY2
PTPN11                 PTPN11
CYB561D1             CYB561D1
SCG2                     SCG2
ARHGAP29             ARHGAP29
TMX4                     TMX4
KIF9                     KIF9
SDCBP                   SDCBP
DDX60                   DDX60
KCNK6                   KCNK6
ADGRL4                 ADGRL4
CHCHD10               CHCHD10
SMARCE1               SMARCE1
COL14A1               COL14A1
MATN3                   MATN3
ATP2B4                 ATP2B4
OSER1-DT             OSER1-DT
CDCA3                   CDCA3
TACC1                   TACC1
TBC1D12               TBC1D12
OCA2                     OCA2
INMT                     INMT
SLC38A4               SLC38A4
MICALL2               MICALL2
GPC6                     GPC6
FHL2                     FHL2
ATOX1                   ATOX1
CLEC2D                 CLEC2D
AC090204.1         AC090204.1
CRIP2                   CRIP2
NT5E                     NT5E
UBE2B                   UBE2B
DCAF16                 DCAF16
CEP295                 CEP295
KCTD1                   KCTD1
LIPE-AS1             LIPE-AS1
KRT10                   KRT10
NT5DC1                 NT5DC1
CENPS                   CENPS
TWSG1                   TWSG1
GSDMD                   GSDMD
CDKN1A                 CDKN1A
SLC9A7                 SLC9A7
NID1                     NID1
NR2F1                   NR2F1
KIF20B                 KIF20B
SLC26A11             SLC26A11
RNF128                 RNF128
ANP32B                 ANP32B
TRAPPC5               TRAPPC5
C19orf33             C19orf33
HSPA1A                 HSPA1A
HIST1H4C             HIST1H4C
TRHDE                   TRHDE
AGL                       AGL
NFKBIZ                 NFKBIZ
MIATNB                 MIATNB
NEAT1                   NEAT1
CYTOR                   CYTOR
CARHSP1               CARHSP1
C6orf48               C6orf48
LGALS3                 LGALS3
CD82                     CD82
H2AFJ                   H2AFJ
CYP27A1               CYP27A1
DUSP16                 DUSP16
SLC7A11               SLC7A11
FGF1                     FGF1
PSME1                   PSME1
TMEM229B             TMEM229B
BLVRB                   BLVRB
ZNF565                 ZNF565
SMIM3                   SMIM3
TEX2                     TEX2
CFAP97                 CFAP97
ADGRV1                 ADGRV1
HIST3H2A             HIST3H2A
BNIP3                   BNIP3
GMPR                     GMPR
KCND3                   KCND3
SLC26A6               SLC26A6
POPDC2                 POPDC2
ALDH2                   ALDH2
AC005476.2         AC005476.2
ANKRD1                 ANKRD1
BLM                       BLM
HMMR                     HMMR
HIP1                     HIP1
PLD3                     PLD3
RNF115                 RNF115
ISG15                   ISG15
LINC01503           LINC01503
FBLN5                   FBLN5
TGIF1                   TGIF1
CLIC6                   CLIC6
CTH                       CTH
KLHL5                   KLHL5
C3orf58               C3orf58
AVPI1                   AVPI1
COMMD7                 COMMD7
KRT18                   KRT18
RAB29                   RAB29
PLEKHA2               PLEKHA2
NME4                     NME4
GTF2B                   GTF2B
AC016831.1         AC016831.1
SOAT1                   SOAT1
TNKS1BP1             TNKS1BP1
MT1E                     MT1E
STX12                   STX12
FMO4                     FMO4
PPIP5K1               PPIP5K1
EGLN1                   EGLN1
AC007014.1         AC007014.1
IGF2BP3               IGF2BP3
PTPN9                   PTPN9
SLC38A10             SLC38A10
SNCAIP                 SNCAIP
HAS2                     HAS2
HEYL                     HEYL
SLC25A53             SLC25A53
SLC39A2               SLC39A2
TBX3                     TBX3
C8orf33               C8orf33
RBPMS                   RBPMS
FAM111A-DT         FAM111A-DT
Z93930.2             Z93930.2
LNX2                     LNX2
WDR81                   WDR81
DSEL                     DSEL
ZSWIM3                 ZSWIM3
CCDC97                 CCDC97
CCDC9B                 CCDC9B
CENPW                   CENPW
ITGB6                   ITGB6
SEPT4                   SEPT4
CAP2                     CAP2
SLC5A12               SLC5A12
AC005332.6         AC005332.6
RASSF2                 RASSF2
ABCD1                   ABCD1
GLB1L                   GLB1L
TES                       TES
AC124798.1         AC124798.1
WARS                     WARS
STRIP1                 STRIP1
FBN1                     FBN1
CYFIP1                 CYFIP1
CLDN12                 CLDN12
MAP3K4                 MAP3K4
DOK5                     DOK5
NUAK1                   NUAK1
SAXO2                   SAXO2
WDR24                   WDR24
SERPINE1             SERPINE1
MT-ND5                 MT-ND5
BTG3                     BTG3
BAG3                     BAG3
CDKL5                   CDKL5
YWHAG                   YWHAG
PGRMC1                 PGRMC1
CXCL14                 CXCL14
STMN1                   STMN1
SELENOP               SELENOP
TTC21A                 TTC21A
OSBPL10               OSBPL10
FIGNL1                 FIGNL1
HSD17B6               HSD17B6
SQSTM1                 SQSTM1
MIR4435-2HG       MIR4435-2HG
COL5A1                 COL5A1
ATL3                     ATL3
AC110597.1         AC110597.1
MINCR                   MINCR
BCAR3                   BCAR3
TRIM16L               TRIM16L
TKT                       TKT
MIR22HG               MIR22HG
BIN3                     BIN3
FAM177A1             FAM177A1
LTBR                     LTBR
TLCD2                   TLCD2
HECTD3                 HECTD3
RAD21                   RAD21
IQGAP1                 IQGAP1
GSTZ1                   GSTZ1
CEP126                 CEP126
MSC                       MSC
AIMP1                   AIMP1
ERICH5                 ERICH5
GOLGA7                 GOLGA7
TTLL12                 TTLL12
AATF                     AATF
DET1                     DET1
RARB                     RARB
CXCL2                   CXCL2
BIN1                     BIN1
TBCE.1                 TBCE.1
SMAD6                   SMAD6
TNFRSF12A           TNFRSF12A
MAF                       MAF
FADS1                   FADS1
PRSS35                 PRSS35
ANKRD6                 ANKRD6
FAAH                     FAAH
PHLDB1                 PHLDB1
BDH2                     BDH2
MANF                     MANF
MBOAT2                 MBOAT2
MON2                     MON2
SBNO1                   SBNO1
CCL20                   CCL20
ECI1                     ECI1
MAFG                     MAFG
E2F6                     E2F6
PDS5B                   PDS5B
AC009159.3         AC009159.3
CTNNB1                 CTNNB1
SNRPN                   SNRPN
MAP3K7                 MAP3K7
CD320                   CD320
EDN1                     EDN1
IGDCC4                 IGDCC4
ZNF468                 ZNF468
TAF1D                   TAF1D
FOXF2                   FOXF2
CEP290                 CEP290
PODXL2                 PODXL2
COL6A1                 COL6A1
RPL23                   RPL23
TMCO4                   TMCO4
GK5                       GK5
SLC35A3               SLC35A3
UBXN2A                 UBXN2A
SAT2                     SAT2
INA                       INA
AC105942.1         AC105942.1
EIF5A                   EIF5A
PWAR6                   PWAR6
DNAJC1                 DNAJC1
ZSWIM6                 ZSWIM6
USP1                     USP1
DBN1                     DBN1
AC064836.3         AC064836.3
GSTP1                   GSTP1
SPTBN2                 SPTBN2
AL355338.1         AL355338.1
GALNT18               GALNT18
ENC1                     ENC1
DDX17                   DDX17
DDX5                     DDX5
SBF2-AS1             SBF2-AS1
LDLR                     LDLR
AL590705.1         AL590705.1
STK35                   STK35
TXNRD1                 TXNRD1
AP002761.3         AP002761.3
PLAU                     PLAU
PELI3                   PELI3
NRG1                     NRG1
CDK5RAP2             CDK5RAP2
AHNAK                   AHNAK
CMSS1                   CMSS1
MSI2                     MSI2
ADAMTS1               ADAMTS1
HMOX1                   HMOX1
LINC01504           LINC01504
MC1R                     MC1R
MIOS                     MIOS
CHEK1                   CHEK1
XPO1                     XPO1
MCUB                     MCUB
HTRA1                   HTRA1
TTC39B                 TTC39B
RHOB                     RHOB
AC128688.2         AC128688.2
CPNE7                   CPNE7
SEPT11                 SEPT11
NOL3                     NOL3
FANCF                   FANCF
ZNRF3                   ZNRF3
STRADB                 STRADB
HECW1                   HECW1
IMPA2                   IMPA2
GTF2A2                 GTF2A2
TMEM92-AS1         TMEM92-AS1
FAM227A               FAM227A
EPRS                     EPRS
PCNX1                   PCNX1
FAM129B               FAM129B
SEMA5A                 SEMA5A
AC116667.1         AC116667.1
PDGFRL                 PDGFRL
VAMP3                   VAMP3
AC004803.1         AC004803.1
FAM83G                 FAM83G
FGF2                     FGF2
MINDY2                 MINDY2
SMOX                     SMOX
TBCK                     TBCK
TARSL2                 TARSL2
AHCYL1                 AHCYL1
MAP2K3                 MAP2K3
CAPG                     CAPG
ARHGAP10             ARHGAP10
ARSI                     ARSI
SGPP1                   SGPP1
MRPS30                 MRPS30
ADAM19                 ADAM19
RBP1                     RBP1
ZBTB1                   ZBTB1
SMIM15                 SMIM15
CLIP1                   CLIP1
MXRA5                   MXRA5
UTP6                     UTP6
ARHGEF26             ARHGEF26
C1orf216             C1orf216
NOP53                   NOP53
RIOK2                   RIOK2
SLC23A2               SLC23A2
TRIM16                 TRIM16
ARID3A                 ARID3A
RING1                   RING1
IER5L                   IER5L
SGTA                     SGTA
ZNF385A               ZNF385A
CACNA2D1             CACNA2D1
LARP7                   LARP7
PSIP1                   PSIP1
SKP2                     SKP2
THBD                     THBD
LDLRAD4               LDLRAD4
CDCA7L                 CDCA7L
IGFBP5                 IGFBP5
FMOD                     FMOD
WIPF1                   WIPF1
FAM133A               FAM133A
LRIG3                   LRIG3
ZFYVE27               ZFYVE27
RPP25                   RPP25
KCNK12                 KCNK12
SMC4                     SMC4
COL11A1               COL11A1
KAT8                     KAT8
GTF2E1                 GTF2E1
PRPS1                   PRPS1
FOXN2                   FOXN2
RNF227                 RNF227
CBX4                     CBX4
NUDT8                   NUDT8
ALDH1L2               ALDH1L2
EGLN3                   EGLN3
AL356056.1         AL356056.1
OR8S1                   OR8S1
PITX1                   PITX1
SPPL2B                 SPPL2B
ARHGEF11             ARHGEF11
GADD45B               GADD45B
LRRC73                 LRRC73
HSBP1L1               HSBP1L1
AL606491.1         AL606491.1
KCNK1                   KCNK1
DNAJB1                 DNAJB1
SP110                   SP110
WNT2                     WNT2
DOCK5                   DOCK5
PIM3                     PIM3
FERMT2                 FERMT2
MMP23B                 MMP23B
SPATA7                 SPATA7
SERPINB1             SERPINB1
C18orf25             C18orf25
MBNL1-AS1           MBNL1-AS1
BMP4                     BMP4
GALNT6                 GALNT6
ETV5                     ETV5
CCDC28B               CCDC28B
RSPO2                   RSPO2
PPM1F                   PPM1F
OTUB1                   OTUB1
NDFIP2                 NDFIP2
PLXNA1                 PLXNA1
RGS4                     RGS4
WWC2                     WWC2
ANTXR1                 ANTXR1
CAST                     CAST
IRAK1                   IRAK1
HSPA9                   HSPA9
FBLN1                   FBLN1
GREB1                   GREB1
MORC3                   MORC3
TCEAL9                 TCEAL9
SLC12A7               SLC12A7
CITED2                 CITED2
ASB13                   ASB13
AMACR                   AMACR
FAM229B               FAM229B
VPS9D1                 VPS9D1
PDIA4                   PDIA4
BACE1-AS             BACE1-AS
PCDHGB6               PCDHGB6
NT5C                     NT5C
NENF                     NENF
AGPAT2                 AGPAT2
ADAMTS2               ADAMTS2
BBX                       BBX
SLC39A14             SLC39A14
TCEA1                   TCEA1
AXL                       AXL
EPB41L4A-AS1     EPB41L4A-AS1
VGLL3                   VGLL3
WDR4                     WDR4
LIFR                     LIFR
CAVIN3                 CAVIN3
SOD2                     SOD2
FSTL3                   FSTL3
FOXQ1                   FOXQ1
ACOT7                   ACOT7
MYC                       MYC
VAMP5                   VAMP5
TOM1                     TOM1
PRSS12                 PRSS12
KRT8                     KRT8
SLC1A1                 SLC1A1
SH3BP5                 SH3BP5
WWC3                     WWC3
PISD                     PISD
PLA2G4C               PLA2G4C
PLEKHA7               PLEKHA7
ABCA5                   ABCA5
MEF2A                   MEF2A
CKB                       CKB
HDAC4                   HDAC4
AL731569.1         AL731569.1
TPT1                     TPT1
FBLN2                   FBLN2
ATAD2B                 ATAD2B
CAMTA2                 CAMTA2
CMTR2                   CMTR2
IRF1                     IRF1
KANK4                   KANK4
PITPNM1               PITPNM1
WDR26                   WDR26
SLC4A8                 SLC4A8
RAB13                   RAB13
ENPP2                   ENPP2
LDHB                     LDHB
TXNIP                   TXNIP
GAS6-DT               GAS6-DT
GARS                     GARS
SMIM10                 SMIM10
STK26                   STK26
EXPH5                   EXPH5
RAB11FIP5           RAB11FIP5
AC073389.1         AC073389.1
UBE2E2                 UBE2E2
EFEMP2                 EFEMP2
LTBP3                   LTBP3
AGPAT4                 AGPAT4
GRAMD2B               GRAMD2B
SPATA13               SPATA13
AC092807.3         AC092807.3
FENDRR                 FENDRR
ZNF251                 ZNF251
H1F0                     H1F0
TMEM88                 TMEM88
UBE3D                   UBE3D
AKR1C1                 AKR1C1
LRRC37A3             LRRC37A3
MELTF                   MELTF
SERPING1             SERPING1
WHAMM                   WHAMM
OLFM2                   OLFM2
SESN1                   SESN1
FLVCR1                 FLVCR1
CISD1                   CISD1
DUSP6                   DUSP6
CPNE2                   CPNE2
TMBIM1                 TMBIM1
SEC16A                 SEC16A
SMURF2                 SMURF2
CADPS2                 CADPS2
OTUD1                   OTUD1
ZADH2                   ZADH2
MT-ND4                 MT-ND4
PPP6R1                 PPP6R1
BOD1                     BOD1
LPCAT2                 LPCAT2
PEG10                   PEG10
LINC00862           LINC00862
PSMC6                   PSMC6
SCD5                     SCD5
TET1                     TET1
STK24                   STK24
PROSER2               PROSER2
RSPO3                   RSPO3
TTLL11                 TTLL11
MYH10                   MYH10
LHFPL2                 LHFPL2
MICAL3                 MICAL3
TMEM106A             TMEM106A
LMO7-AS1             LMO7-AS1
STK38L                 STK38L
CENPV                   CENPV
TCEAL1                 TCEAL1
KIAA0040             KIAA0040
CPE                       CPE
SPTAN1                 SPTAN1
ZNF711                 ZNF711
SYT8                     SYT8
EGR1                     EGR1
DYM                       DYM
AZI2                     AZI2
C16orf86             C16orf86
LCOR                     LCOR
CEP44                   CEP44
NDRG1                   NDRG1
NQO1                     NQO1
CNTRL                   CNTRL
DNAJC15               DNAJC15
REC8                     REC8
RIMS1                   RIMS1
GSTO1                   GSTO1
AC010255.1         AC010255.1
MAK16                   MAK16
FIP1L1                 FIP1L1
RAPGEF2               RAPGEF2
SVEP1                   SVEP1
APH1B                   APH1B
TFPT                     TFPT
NTM                       NTM
TNFRSF10B           TNFRSF10B
VTI1A                   VTI1A
ANKFN1                 ANKFN1
MORC4                   MORC4
KIT                       KIT
CARD16                 CARD16
SART1                   SART1
PARP9                   PARP9
IGFBP3                 IGFBP3
PCMTD2                 PCMTD2
MVD                       MVD
MMADHC                 MMADHC
ID4                       ID4
BARD1                   BARD1
TSPAN6                 TSPAN6
ARIH2                   ARIH2
ZNF358                 ZNF358
MBNL1                   MBNL1
AKAP17A               AKAP17A
VSIG10L               VSIG10L
ZNF324                 ZNF324
PCSK1N                 PCSK1N
KIAA0355             KIAA0355
PBLD                     PBLD
PGD                       PGD
GBP1                     GBP1
CLEC1B                 CLEC1B
CCDC144NL-AS1   CCDC144NL-AS1
TRAF2                   TRAF2
CMIP                     CMIP
HAND1                   HAND1
ASS1                     ASS1
SEMA3D                 SEMA3D
AL035413.1         AL035413.1
LINC01311           LINC01311
SMC6                     SMC6
TYRP1                   TYRP1
NEK8                     NEK8
FXYD5                   FXYD5
HOXA7                   HOXA7
IFITM2                 IFITM2
HMGB3                   HMGB3
TRAF4                   TRAF4
OTUD3                   OTUD3
VASP                     VASP
CRTAP                   CRTAP
SYNGR2                 SYNGR2
FAM133B               FAM133B
HSPB3                   HSPB3
VARS2                   VARS2
SCUBE2                 SCUBE2
TOB1                     TOB1
OLFML2B               OLFML2B
GPR68                   GPR68
KIF23                   KIF23
YWHAH                   YWHAH
CNTLN                   CNTLN
LETM2                   LETM2
AC103706.1         AC103706.1
ZNF394                 ZNF394
RNF152                 RNF152
CNKSR3                 CNKSR3
PARM1                   PARM1
REL                       REL
TMPO                     TMPO
CAP1                     CAP1
MRC2                     MRC2
WEE1                     WEE1
PLXNB2                 PLXNB2
SDC1                     SDC1
ING2                     ING2
C1QTNF3               C1QTNF3
SPTLC3                 SPTLC3
GBE1                     GBE1
ZKSCAN4               ZKSCAN4
IER3                     IER3
JUP                       JUP
SRRD                     SRRD
PPFIBP1               PPFIBP1
AC139795.3         AC139795.3
HSD17B7               HSD17B7
LAMB2                   LAMB2
LINC01759           LINC01759
TCTN3                   TCTN3
PIDD1                   PIDD1
GLUL                     GLUL
PHYHD1                 PHYHD1
CPQ                       CPQ
LSAMP                   LSAMP
ANKAR                   ANKAR
HUNK                     HUNK
MTSS1                   MTSS1
CNN1                     CNN1
CELF2                   CELF2
FLOT1                   FLOT1
CTSV                     CTSV
PHIP                     PHIP
MAMSTR                 MAMSTR
SLC2A5                 SLC2A5
ETHE1                   ETHE1
C6orf62               C6orf62
HADH                     HADH
GRIP2                   GRIP2
IGFBP2                 IGFBP2
TAB3                     TAB3
ME1                       ME1
GPR87                   GPR87
ZBTB38                 ZBTB38
ST6GAL1               ST6GAL1
SZT2                     SZT2
LIMK2                   LIMK2
PPM1H                   PPM1H
RBMX                     RBMX
EPHB6                   EPHB6
NUDCD1                 NUDCD1
ZBTB7B                 ZBTB7B
ESYT2                   ESYT2
PACSIN3               PACSIN3
ST7-AS1               ST7-AS1
HIST1H2BJ           HIST1H2BJ
NEBL                     NEBL
EFNA1                   EFNA1
GNAS-AS1             GNAS-AS1
GSTM3                   GSTM3
AL121827.2         AL121827.2
ASAP1                   ASAP1
NRTN                     NRTN
ANTXR2                 ANTXR2
COL18A1               COL18A1
GAB2                     GAB2
SH3D19                 SH3D19
CDYL2                   CDYL2
TACC2                   TACC2
PHLDA3                 PHLDA3
BACH1                   BACH1
TMEM8B                 TMEM8B
MID1IP1               MID1IP1
WLS                       WLS
NKTR                     NKTR
TLL2                     TLL2
DYRK1B                 DYRK1B
SLC29A4               SLC29A4
ZNF678                 ZNF678
USP13                   USP13
MRPS6                   MRPS6
CCDC18                 CCDC18
ARHGAP5-AS1       ARHGAP5-AS1
ST3GAL1               ST3GAL1
LRIF1                   LRIF1
KATNBL1               KATNBL1
CYB5D1                 CYB5D1
TNFAIP6               TNFAIP6
KTN1                     KTN1
AURKA                   AURKA
CASP6                   CASP6
H3F3B                   H3F3B
PECR                     PECR
BRIX1                   BRIX1
KLHL28                 KLHL28
G6PC3                   G6PC3
SPINK2                 SPINK2
ST3GAL6               ST3GAL6
CMTM7                   CMTM7
SHROOM3               SHROOM3
GALNT5                 GALNT5
TIGD1                   TIGD1
HOXB3                   HOXB3
NCAM2                   NCAM2
MED28                   MED28
LINC01750           LINC01750
SPRYD3                 SPRYD3
BSDC1                   BSDC1
TP53I3                 TP53I3
DUSP14                 DUSP14
DDX10                   DDX10
HELLS                   HELLS
LINC00342           LINC00342
SNAI1                   SNAI1
TNNC1                   TNNC1
TP53INP1             TP53INP1
CFAP20                 CFAP20
PCGF5                   PCGF5
DHCR7                   DHCR7
KANSL1L               KANSL1L
JPH1                     JPH1
PER2                     PER2
GSTA4                   GSTA4
HIST1H2BG           HIST1H2BG
RAD50                   RAD50
PSMB8                   PSMB8
ICAM3                   ICAM3
GRIA3                   GRIA3
AL662884.4         AL662884.4
RCAN1                   RCAN1
MAP3K2                 MAP3K2
FRMD6-AS1           FRMD6-AS1
DAAM1                   DAAM1
MYLK                     MYLK
BICC1                   BICC1
HMGN3                   HMGN3
SSTR1                   SSTR1
DSE                       DSE
GREM1                   GREM1
NCS1                     NCS1
CYP26A1               CYP26A1
TANGO2                 TANGO2
HMGN2                   HMGN2
RBM6                     RBM6
SEPHS2                 SEPHS2
ZC3H7B                 ZC3H7B
LINC01355           LINC01355
GINS1                   GINS1
LEF1                     LEF1
ASAP2                   ASAP2
ZFP36                   ZFP36
CRYGS                   CRYGS
CYP1B1                 CYP1B1
CASTOR1               CASTOR1
SHKBP1                 SHKBP1
RTN2                     RTN2
RUNX2                   RUNX2
ANO4                     ANO4
B4GALT1               B4GALT1
TRPM7                   TRPM7
PTTG1IP               PTTG1IP
TEX22                   TEX22
TRAF3IP1             TRAF3IP1
SORT1                   SORT1
LITAF                   LITAF
TGM2                     TGM2
CREBRF                 CREBRF
ACAA1                   ACAA1
MYOCD                   MYOCD
ADCY7                   ADCY7
G6PD                     G6PD
MAPRE2                 MAPRE2
NCOA1                   NCOA1
INAFM2                 INAFM2
CTSO                     CTSO
PLSCR1                 PLSCR1
NSFL1C                 NSFL1C
TSTD1                   TSTD1
HIPK2                   HIPK2
TMEM11                 TMEM11
TIFA                     TIFA
PLAA                     PLAA
ITGB1BP2             ITGB1BP2
THBS3                   THBS3
RNASET2               RNASET2
GALNT2                 GALNT2
ZNF883                 ZNF883
HSPH1                   HSPH1
COL1A1                 COL1A1
CSGALNACT2         CSGALNACT2
KLF4                     KLF4
KIAA0825             KIAA0825
NASP                     NASP
NSRP1                   NSRP1
CLIC1                   CLIC1
SYPL1                   SYPL1
C2CD3                   C2CD3
MFAP4                   MFAP4
COL25A1               COL25A1
AEBP1                   AEBP1
ENOSF1                 ENOSF1
ZNF350                 ZNF350
BEX1                     BEX1
AC010336.1         AC010336.1
MBP                       MBP
INTS13                 INTS13
CENPF                   CENPF
ID1                       ID1
VLDLR                   VLDLR
CNRIP1                 CNRIP1
ATP2A2                 ATP2A2
AIFM2                   AIFM2
CALM1                   CALM1
HSPA12A               HSPA12A
AC112220.3         AC112220.3
PKDCC                   PKDCC
IFNGR2                 IFNGR2
ZNF613                 ZNF613
RABL2A                 RABL2A
FAM69B                 FAM69B
TSHZ1                   TSHZ1
DOCK1                   DOCK1
ACTC1                   ACTC1
TXLNA                   TXLNA
DUSP2                   DUSP2
PRKG1                   PRKG1
GATM                     GATM
RFWD3                   RFWD3
FBXO4                   FBXO4
CARMN                   CARMN
SESN3                   SESN3
YAP1                     YAP1
MANCR                   MANCR
AFAP1                   AFAP1
GUSB                     GUSB
ARID4A                 ARID4A
VWA8                     VWA8
NDN                       NDN
LY6G5C                 LY6G5C
MTHFD2                 MTHFD2
PPP1R14A             PPP1R14A
SPATA5L1             SPATA5L1
SSPN                     SSPN
SLC40A1               SLC40A1
C4BPB                   C4BPB
FSTL1                   FSTL1
NRBP2                   NRBP2
HAPLN4                 HAPLN4
PTHLH                   PTHLH
RBM47                   RBM47
ZCCHC10               ZCCHC10
RACGAP1               RACGAP1
BCL7A                   BCL7A
NPNT                     NPNT
PIGQ                     PIGQ
SLC22A15             SLC22A15
ATP10D                 ATP10D
SORBS3                 SORBS3
UNKL                     UNKL
HNRNPUL2             HNRNPUL2
SEPT10                 SEPT10
MSMO1                   MSMO1
KIAA0895L           KIAA0895L
ALDH1B1               ALDH1B1
TMEM176B             TMEM176B
ENAH                     ENAH
FDFT1                   FDFT1
LINC00545           LINC00545
ODF2L                   ODF2L
LINC02001           LINC02001
EPHB2                   EPHB2
TXNDC5                 TXNDC5
ELL2                     ELL2
MPLKIP                 MPLKIP
WNT4                     WNT4
PROS1                   PROS1
TSC22D1-AS1       TSC22D1-AS1
ZNF438                 ZNF438
RASSF7                 RASSF7
HMGA2                   HMGA2
FOXP1                   FOXP1
EMILIN1               EMILIN1
CLIC4                   CLIC4
KANK2                   KANK2
COQ8A                   COQ8A
DNAJC12               DNAJC12
ASPN                     ASPN
DCTN4                   DCTN4
PCAT1                   PCAT1
RPRML                   RPRML
PATJ                     PATJ
FLYWCH1               FLYWCH1
PLXDC1                 PLXDC1
IGF2-AS               IGF2-AS
LYPD1                   LYPD1
TRAF3IP2-AS1     TRAF3IP2-AS1
INTS6                   INTS6
PHLDA2                 PHLDA2
MSRA                     MSRA
ADGRF4                 ADGRF4
FAM96B                 FAM96B
HNRNPH1               HNRNPH1
ADGRA2                 ADGRA2
MYO9B                   MYO9B
MRPL52                 MRPL52
GXYLT2                 GXYLT2
MET                       MET
LUM                       LUM
ECM1                     ECM1
PSAT1                   PSAT1
PPM1K                   PPM1K
HOXC8                   HOXC8
CEP170                 CEP170
PTPN13                 PTPN13
HSPB11                 HSPB11
GCLM                     GCLM
NOP14                   NOP14
PLP2                     PLP2
AC025171.4         AC025171.4
SOCS3                   SOCS3
ANGPTL7               ANGPTL7
CHST3                   CHST3
AP002387.2         AP002387.2
GPC3                     GPC3
RERE                     RERE
NR2F1-AS1           NR2F1-AS1
CIRBP                   CIRBP
EPHA4                   EPHA4
TPCN2                   TPCN2
DPP10-AS1           DPP10-AS1
DTX3L                   DTX3L
ATP6V0A2             ATP6V0A2
MAPK8IP3             MAPK8IP3
MANBA                   MANBA
MFSD1                   MFSD1
LINC01082           LINC01082
TEDC1                   TEDC1
ST6GAL2               ST6GAL2
MYOZ1                   MYOZ1
TPP2                     TPP2
ULK4                     ULK4
C1orf226             C1orf226
ISCU                     ISCU
CAPN10                 CAPN10
SCN1B                   SCN1B
KLHL4                   KLHL4
PDE4B                   PDE4B
STK17B                 STK17B
RAD54B                 RAD54B
C2orf49               C2orf49
PLEKHO1               PLEKHO1
AP1G1                   AP1G1
DHRS3                   DHRS3
AC124283.1         AC124283.1
ATF3                     ATF3
TMEM50B               TMEM50B
SNTA1                   SNTA1
DKK1                     DKK1
FIG4                     FIG4
NCL                       NCL
RPL10                   RPL10
EIF5                     EIF5
TNFRSF21             TNFRSF21
EPHX2                   EPHX2
LRRC75A               LRRC75A
LRCH4                   LRCH4
MAP1B                   MAP1B
AC021242.3         AC021242.3
AL080317.1         AL080317.1
HMGB2                   HMGB2
C1orf56               C1orf56
DNAJC28               DNAJC28
TMTC1                   TMTC1
ARHGAP20             ARHGAP20
ARG2                     ARG2
WFDC3                   WFDC3
DIXDC1                 DIXDC1
ADAT2                   ADAT2
BAIAP2-DT           BAIAP2-DT
FOS                       FOS
FAM84A                 FAM84A
EXTL1                   EXTL1
SLC4A2                 SLC4A2
KIF9-AS1             KIF9-AS1
CALY                     CALY
CRACR2B               CRACR2B
TSPO                     TSPO
TNFSF15               TNFSF15
ZHX1                     ZHX1
GNG2                     GNG2
TM4SF1                 TM4SF1
PSPN                     PSPN
TPD52                   TPD52
HOXB4                   HOXB4
CBR1                     CBR1
AC137767.1         AC137767.1
HELZ2                   HELZ2
RAB3B                   RAB3B
AC073611.1         AC073611.1
TMEM86B               TMEM86B
MRPS18B               MRPS18B
WDR1                     WDR1
TMLHE                   TMLHE
LINC02334           LINC02334
CDC37L1-DT         CDC37L1-DT
DYNC1I1               DYNC1I1
TRIM47                 TRIM47
FN1                       FN1
CDC42EP2             CDC42EP2
KIAA1328             KIAA1328
MDFI                     MDFI
PIR                       PIR
PSD                       PSD
NTF3                     NTF3
NSMF                     NSMF
LINC01638           LINC01638
AL050341.2         AL050341.2
SFRP1                   SFRP1
AMMECR1L             AMMECR1L
SRRM3                   SRRM3
ACAT2                   ACAT2
SPARC                   SPARC
BBOX1-AS1           BBOX1-AS1
SLC4A1AP             SLC4A1AP
BICRA                   BICRA
VDR                       VDR
KRT19                   KRT19
SNX25                   SNX25
SLC9A3R2             SLC9A3R2
CORO2B                 CORO2B
DMRTA1                 DMRTA1
HOXB8                   HOXB8
MIR503HG             MIR503HG
KLF2                     KLF2
FRMD4A                 FRMD4A
GTF2F1                 GTF2F1
MEST                     MEST
KMT5C                   KMT5C
FASTKD3               FASTKD3
ZNF563                 ZNF563
MAP3K8                 MAP3K8
SMCO4                   SMCO4
LIN54                   LIN54
KLHL42                 KLHL42
AC073323.1         AC073323.1
FKBP10                 FKBP10
THBS2                   THBS2
TGFBR3                 TGFBR3
CMTM3                   CMTM3
TMCO6                   TMCO6
MICA                     MICA
PDGFC                   PDGFC
GULP1                   GULP1
GLA                       GLA
CDC42EP4             CDC42EP4
MPC2                     MPC2
MPRIP                   MPRIP
WNT5B                   WNT5B
NDUFB2-AS1         NDUFB2-AS1
NBL1                     NBL1
P4HB                     P4HB
VEGFC                   VEGFC
SLC2A8                 SLC2A8
PRKAB2                 PRKAB2
GPS1                     GPS1
SGK494                 SGK494
LINC01013           LINC01013
CD248                   CD248
ALOX5AP               ALOX5AP
SULF1                   SULF1
FABP3                   FABP3
NKX3-1                 NKX3-1
SELENOK               SELENOK
AC009414.2         AC009414.2
AC112721.1         AC112721.1
TMEM116               TMEM116
ZNF850                 ZNF850
GALNT10               GALNT10
ZNF561                 ZNF561
PROCR                   PROCR
LINC02482           LINC02482
PUS7L                   PUS7L
AL133338.1         AL133338.1
ZNF223                 ZNF223
AC110285.6         AC110285.6
ADAMTS12             ADAMTS12
PCBD1                   PCBD1
COPZ2                   COPZ2
FJX1                     FJX1
ULBP2                   ULBP2
TMEM30A               TMEM30A
LIMS2                   LIMS2
MYL4                     MYL4
AC012645.1         AC012645.1
ZSCAN31               ZSCAN31
PTGFR                   PTGFR
HS3ST3A1             HS3ST3A1
ANKRD37               ANKRD37
TRIM24                 TRIM24
AL590426.2         AL590426.2
RAB9A                   RAB9A
MCTP1                   MCTP1
ERG28                   ERG28
ZCCHC14               ZCCHC14
EPB41L2               EPB41L2
TEX29                   TEX29
SCUBE3                 SCUBE3
LINC01252           LINC01252
SLC6A8                 SLC6A8
AC062029.1         AC062029.1
H2AFY                   H2AFY
PNPLA2                 PNPLA2
KLRG1                   KLRG1
ZNF8                     ZNF8
PCSK6                   PCSK6
TNFSF13B             TNFSF13B
OPTN                     OPTN
TM7SF3                 TM7SF3
HGD                       HGD
NSG1                     NSG1
ZNF781                 ZNF781
ARAP2                   ARAP2
AGAP9                   AGAP9
MAP3K20               MAP3K20
C19orf73             C19orf73
CHORDC1               CHORDC1
TNRC6C                 TNRC6C
IL27RA                 IL27RA
BZW2                     BZW2
FKRP                     FKRP
KLF6                     KLF6
DPP10                   DPP10
MYADM                   MYADM
LEFTY2                 LEFTY2
AL136084.3         AL136084.3
TSPAN18               TSPAN18
UGDH                     UGDH
BTN3A2                 BTN3A2
RDH10                   RDH10
GPR161                 GPR161
KLF5                     KLF5
ATP6V0E2             ATP6V0E2
CCDC81                 CCDC81
SUPV3L1               SUPV3L1
GASAL1                 GASAL1
LPAR6                   LPAR6
CTXN1                   CTXN1
AL132656.2         AL132656.2
TMEM154               TMEM154
FLRT2                   FLRT2
AFAP1L2               AFAP1L2
UBE2L6                 UBE2L6
HOXB-AS1             HOXB-AS1
LINC01705           LINC01705
HOPX                     HOPX
AC013652.1         AC013652.1
IP6K3                   IP6K3
SQLE                     SQLE
SYTL2                   SYTL2
TTC7A                   TTC7A
AC007938.3         AC007938.3
PRAG1                   PRAG1
CPPED1                 CPPED1
LINC00626           LINC00626
PLPPR3                 PLPPR3
AC007878.1         AC007878.1
SLC1A3                 SLC1A3
TNK2                     TNK2
GPT2                     GPT2
APOBEC3C             APOBEC3C
ST5                       ST5
WNT11                   WNT11
DAAM2                   DAAM2
PLN                       PLN
GLIPR2                 GLIPR2
MIS18BP1             MIS18BP1
CWC25                   CWC25
PIGU                     PIGU
DNASE2                 DNASE2
ATL1                     ATL1
SH3BP4                 SH3BP4
UAP1                     UAP1
IQCH-AS1             IQCH-AS1
SLC22A2               SLC22A2
SSH3                     SSH3
ASMTL                   ASMTL
SDC4                     SDC4
FNDC3B                 FNDC3B
TMEM56                 TMEM56
HES6                     HES6
ADAM10                 ADAM10
ZNF839                 ZNF839
RB1CC1                 RB1CC1
PLCB4                   PLCB4
SSC5D                   SSC5D
UBE2S                   UBE2S
RNF217                 RNF217
ATP6V0C               ATP6V0C
TM7SF2                 TM7SF2
MYL7                     MYL7
SLC10A3               SLC10A3
IL11                     IL11
AAED1                   AAED1
AC010976.2         AC010976.2
AC034236.2         AC034236.2
KIAA1217             KIAA1217
NKRF                     NKRF
GPAM                     GPAM
AC112721.2         AC112721.2
NHSL1                   NHSL1
FHOD3                   FHOD3
FAM43A                 FAM43A
AC092069.1         AC092069.1
WFDC1                   WFDC1
SRFBP1                 SRFBP1
MMP24OS               MMP24OS
RASSF1                 RASSF1
RAB7B                   RAB7B
SPOCD1                 SPOCD1
RCN3                     RCN3
LAMC2                   LAMC2
SGCD                     SGCD
APOD                     APOD
RPP14                   RPP14
TGFBR3L               TGFBR3L
SLC2A1-AS1         SLC2A1-AS1
BLMH                     BLMH
SERPINI1             SERPINI1
NOTCH3                 NOTCH3
MYBL1                   MYBL1
DCUN1D2               DCUN1D2
RPN1                     RPN1
B4GALT5               B4GALT5
FBXL22                 FBXL22
SHB                       SHB
AC020571.1         AC020571.1
RNASEH2C             RNASEH2C
PCM1                     PCM1
TMC6                     TMC6
JUNB                     JUNB
EEA1                     EEA1
COL4A4                 COL4A4
LOXL1-AS1           LOXL1-AS1
AQP3                     AQP3
PHKG1                   PHKG1
STARD4-AS1         STARD4-AS1
KCNAB3                 KCNAB3
PLAUR                   PLAUR
PCNA                     PCNA
DNMBP                   DNMBP
LIPG                     LIPG
AL451050.2         AL451050.2
FABP5                   FABP5
PANK1                   PANK1
CEP350                 CEP350
ZFYVE21               ZFYVE21
SYNE2                   SYNE2
KIFC2                   KIFC2
COMT                     COMT
LINC01534           LINC01534
TRIM58                 TRIM58
ITGA10                 ITGA10
NUDT6                   NUDT6
AC069185.1         AC069185.1
RBM24                   RBM24
C8orf88               C8orf88
PPP5D1                 PPP5D1
RASGRF2               RASGRF2
CDC42SE1             CDC42SE1
KIAA1755             KIAA1755
PSPH                     PSPH
PTGFRN                 PTGFRN
MMP24                   MMP24
DMKN                     DMKN
PLK2                     PLK2
MYL10                   MYL10
PLEKHH2               PLEKHH2
KCNJ2                   KCNJ2
LTBP1                   LTBP1
AC132872.1         AC132872.1
SVIP                     SVIP
RAP1B                   RAP1B
AC044849.1         AC044849.1
PNP                       PNP
ZGLP1                   ZGLP1
TUSC3                   TUSC3
APBA2                   APBA2
TTC27                   TTC27
TMEM200A             TMEM200A
RASSF8                 RASSF8
PCDHA10               PCDHA10
THAP7-AS1           THAP7-AS1
AP000487.1         AP000487.1
ID2                       ID2
SHMT2                   SHMT2
AL441883.1         AL441883.1
ARHGEF3               ARHGEF3
GFOD1                   GFOD1
SSC4D                   SSC4D
NTRK2                   NTRK2
CTSC                     CTSC
ZNF117                 ZNF117
FAM122A               FAM122A
ANXA3                   ANXA3
ZSCAN25               ZSCAN25
KCNQ5                   KCNQ5
TRABD2B               TRABD2B
AXIN1                   AXIN1
FAM69A                 FAM69A
ZNF121                 ZNF121
LRRC45                 LRRC45
GBP2                     GBP2
SPA17                   SPA17
ZNF329                 ZNF329
HES1                     HES1
SEMA4F                 SEMA4F
FGD6                     FGD6
HOXC6                   HOXC6
FANCB                   FANCB
HINT1                   HINT1
AC078850.1         AC078850.1
GPX7                     GPX7
RCAN2                   RCAN2
AP000866.2         AP000866.2
FAM19A2               FAM19A2
RALGPS2               RALGPS2
MOB3B                   MOB3B
JAG1                     JAG1
MRPL35                 MRPL35
AHNAK2                 AHNAK2
RASL11B               RASL11B
RTL5                     RTL5
CSNK1G1               CSNK1G1
SLC38A2               SLC38A2
CPA4                     CPA4
DDAH2                   DDAH2
AC097478.1         AC097478.1
POLE4                   POLE4
ACOX1                   ACOX1
TGIF2                   TGIF2
ITGA4                   ITGA4
TTC30B                 TTC30B
ZCWPW1                 ZCWPW1
ARID1B                 ARID1B
PLD5                     PLD5
ZNF699                 ZNF699
BCL10                   BCL10
NFYA                     NFYA
BAG2                     BAG2
SRSF3                   SRSF3
TMTC2                   TMTC2
RNF150                 RNF150
PDCL3                   PDCL3
SLC25A25-AS1     SLC25A25-AS1
ZNF639                 ZNF639
JOSD2                   JOSD2
TUBB4B                 TUBB4B
NCAPD3                 NCAPD3
GINS4                   GINS4
TGFBR2                 TGFBR2
C2orf69               C2orf69
SPRY2                   SPRY2
GPNMB                   GPNMB
ARL6IP1               ARL6IP1
PEX3                     PEX3
LTC4S                   LTC4S
HMCN1                   HMCN1
CCDC138               CCDC138
RAD51B                 RAD51B
URAD                     URAD
HES4                     HES4
MVK                       MVK
KPNA2                   KPNA2
EPS8L2                 EPS8L2
NOG                       NOG
AL078639.1         AL078639.1
SLC27A5               SLC27A5
PCDH18                 PCDH18
VPS33B                 VPS33B
FDPS                     FDPS
NET1                     NET1
PLCE1                   PLCE1
IFNAR2                 IFNAR2
LIF                       LIF
TFPI2                   TFPI2
ANO7                     ANO7
AL163051.1         AL163051.1
C5orf30               C5orf30
DOHH                     DOHH
ABL2                     ABL2
B3GALNT2             B3GALNT2
HIST1H4E             HIST1H4E
MRAP2                   MRAP2
NAV3                     NAV3
SLC39A6               SLC39A6
GPRC5A                 GPRC5A
KLHL21                 KLHL21
POLR1B                 POLR1B
PDGFA                   PDGFA
PITHD1                 PITHD1
ABLIM1                 ABLIM1
FAM107B               FAM107B
RAB3IP                 RAB3IP
SYN1                     SYN1
BMPR2                   BMPR2
THAP2                   THAP2
IDH3B                   IDH3B
MRS2                     MRS2
IPMK                     IPMK
ARNTL                   ARNTL
ZNF628                 ZNF628
MLPH                     MLPH
RECK                     RECK
TRIM22                 TRIM22
TMEM147-AS1       TMEM147-AS1
SRGAP2C               SRGAP2C
SVIL                     SVIL
ATRNL1                 ATRNL1
AL118516.1         AL118516.1
EGR2                     EGR2
HSPB8                   HSPB8
AADAT                   AADAT
NCKAP1                 NCKAP1
ADAM33                 ADAM33
EHD3                     EHD3
NEGR1                   NEGR1
DYNC1H1               DYNC1H1
ZEB2                     ZEB2
FIRRE                   FIRRE
SUMF1                   SUMF1
PDZRN4                 PDZRN4
LONRF1                 LONRF1
TNFSF4                 TNFSF4
KCNMB4                 KCNMB4
CAVIN4                 CAVIN4
DHRS2                   DHRS2
TRIO                     TRIO
RPS4Y1                 RPS4Y1
AC018647.1         AC018647.1
HMCES                   HMCES
KCNN2                   KCNN2
HIST1H2BD           HIST1H2BD
TUBA4A                 TUBA4A
CSRNP3                 CSRNP3
TAS2R14               TAS2R14
MCAM                     MCAM
BX255923.1         BX255923.1
PRRC2A                 PRRC2A
CKS2                     CKS2
LRP1                     LRP1
NLRP1                   NLRP1
CLEC3A                 CLEC3A
SEC14L2               SEC14L2
ANKRD29               ANKRD29
OLR1                     OLR1
CPVL                     CPVL
CCDC51                 CCDC51
FLT1                     FLT1
TGFB2                   TGFB2
FYN                       FYN
TRAPPC10             TRAPPC10
ZNF574                 ZNF574
OPHN1                   OPHN1
CXADR                   CXADR
LYSMD2                 LYSMD2
NXPH4                   NXPH4
PRUNE2                 PRUNE2
MPP5                     MPP5
TMEM65                 TMEM65
RTKN2                   RTKN2
MDM2                     MDM2
IRF2BP2               IRF2BP2
RAB40C                 RAB40C
KMO                       KMO
HIST1H1E             HIST1H1E
HOXA10                 HOXA10
CAMK4                   CAMK4
SRPX                     SRPX
TSPAN12               TSPAN12
PKN1                     PKN1
SPCS3                   SPCS3
CCDC74A               CCDC74A
SUSD6                   SUSD6
CYSTM1                 CYSTM1
C21orf2               C21orf2
TALDO1                 TALDO1
SYNGR1                 SYNGR1
FANCI                   FANCI
AC006480.2         AC006480.2
NKX2-5                 NKX2-5
CDC6                     CDC6
CDH6                     CDH6
ATP6AP1L             ATP6AP1L
AC008467.1         AC008467.1
MACROD2               MACROD2
ZFHX4                   ZFHX4
ANKRD34A             ANKRD34A
IL17RE                 IL17RE
ZNF689                 ZNF689
DGKI                     DGKI
IGFL2                   IGFL2
MBNL3                   MBNL3
SMCR8                   SMCR8
BMP6                     BMP6
URB2                     URB2
EZH2                     EZH2
CRK                       CRK
CTSD                     CTSD
KRTCAP2               KRTCAP2
GGH                       GGH
ZNF473                 ZNF473
DPH6                     DPH6
TPM1.1                   TPM1
FTH1.1                   FTH1
CALD1.1                 CALD1
IGFBP7.1               IGFBP7
TAGLN.1                 TAGLN
CCND2.1                 CCND2
TSC22D3.1             TSC22D3
GLIPR1.1               GLIPR1
TPM2.1                   TPM2
PLS3.1                   PLS3
VIM.1                     VIM
ITGA11                 ITGA11
SAA1.1                   SAA1
FKBP5.1                 FKBP5
CEBPD.1                 CEBPD
SPINT2.1               SPINT2
TIMP3.1                 TIMP3
INHBA.1                 INHBA
GLS                       GLS
ACTN1.1                 ACTN1
PDPN.1                   PDPN
TIMP2.1                 TIMP2
ACTA2.1                 ACTA2
OTULINL.1             OTULINL
COL12A1.1             COL12A1
ACTB.1                   ACTB
COL5A2                 COL5A2
C12orf75.1           C12orf75
XRCC4.1                 XRCC4
IGF2.1.1               IGF2.1
GYPC.1                   GYPC
MFGE8.1                 MFGE8
KCTD16.1               KCTD16
PLAC9.1                 PLAC9
GPX4.1                   GPX4
CRIM1                   CRIM1
MYL9.1                   MYL9
COL4A1                 COL4A1
RPS27.1                 RPS27
CRISPLD2.1           CRISPLD2
IVNS1ABP             IVNS1ABP
CSRP2.1                 CSRP2
BRI3.1                   BRI3
PDCD4.1                 PDCD4
CCND1.1                 CCND1
EIF1.1                   EIF1
CSF1.1                   CSF1
CADM1.1                 CADM1
TMEM141.1             TMEM141
IL1R1.1                 IL1R1
THBS1.1                 THBS1
FTL                       FTL
MARCKSL1.1           MARCKSL1
CNN3.1                   CNN3
RAB11FIP1.1         RAB11FIP1
SOX4.1                   SOX4
B2M.1                     B2M
BTG1.1                   BTG1
CDKN1C.1               CDKN1C
EIF4EBP1.1           EIF4EBP1
GPC4.1                   GPC4
PPP1CB.1               PPP1CB
AC092171.3         AC092171.3
NFIL3.1                 NFIL3
PRLR.1                   PRLR
MARCKS.1               MARCKS
UACA.1                   UACA
ANOS1.1                 ANOS1
MFAP5.1                 MFAP5
DPYSL2.1               DPYSL2
ANXA2.1                 ANXA2
OGN.1                     OGN
TUBA1A.1               TUBA1A
CDKN2A.1               CDKN2A
MYL12A.1               MYL12A
CEBPB.1                 CEBPB
LMO4.1                   LMO4
LIMA1.1                 LIMA1
AQP1.1                   AQP1
CST3.1                   CST3
MMP2.1                   MMP2
HOXD11.1               HOXD11
COL4A2                 COL4A2
ADGRG6.1               ADGRG6
HLA-C.1                 HLA-C
IQGAP2.1               IQGAP2
FALEC.1                 FALEC
SERPINF1.1           SERPINF1
MYL6.1                   MYL6
C9orf16.1             C9orf16
CCNO.1                   CCNO
CEP128.1               CEP128
ODAPH.1                 ODAPH
TXNDC16.1             TXNDC16
SAT1.1                   SAT1
ADAM12                 ADAM12
LZTS1.1                 LZTS1
MKS1.1                   MKS1
ZFAS1.1                 ZFAS1
AKAP12.1               AKAP12
L2HGDH.1               L2HGDH
YBX3.1                   YBX3
MSX1.1                   MSX1
PPIB                     PPIB
CDCA8.1                 CDCA8
FN1.1                     FN1
CDH2                     CDH2
PTGER2.1               PTGER2
ZIK1.1                   ZIK1
CCDC171.1             CCDC171
DDIT4.1                 DDIT4
FOXO1.1                 FOXO1
CAVIN1                 CAVIN1
FLNA                     FLNA
CALR.1                   CALR
CKAP4                   CKAP4
LMO7.1                   LMO7
IRS2.1                   IRS2
SRPX2.1                 SRPX2
ITPR2.1                 ITPR2
TMEM120A.1           TMEM120A
GGT5.1                   GGT5
LAMTOR5.1             LAMTOR5
PDGFRA.1               PDGFRA
TXNRD2.1               TXNRD2
KRT10.1                 KRT10
PRSS23.1               PRSS23
AC007920.2         AC007920.2
ABCA1.1                 ABCA1
S100A16.1             S100A16
BOC.1                     BOC
FBXO32.1               FBXO32
SCAMP1-AS1.1       SCAMP1-AS1
CLDN11.1               CLDN11
COL1A1.1               COL1A1
RPL10.1                 RPL10
S1PR1.1                 S1PR1
SH3BGRL3.1           SH3BGRL3
P4HA2.1                 P4HA2
IFITM3.1               IFITM3
FAH.1                     FAH
PRDX3.1                 PRDX3
POLE.1                   POLE
PLA2G5.1               PLA2G5
PRSS35.1               PRSS35
CTGF.1                   CTGF
S100A10.1             S100A10
LOX                       LOX
NR3C1.1                 NR3C1
ACTG1.1                 ACTG1
PMEPA1.1               PMEPA1
ANAPC16.1             ANAPC16
COL18A1.1             COL18A1
CALCOCO2.1           CALCOCO2
EMP3.1                   EMP3
AC073111.5         AC073111.5
ACTG2.1                 ACTG2
CALB2.1                 CALB2
SEMA3C                 SEMA3C
DKK2.1                   DKK2
SPATS2L.1             SPATS2L
SLC7A8.1               SLC7A8
CEMIP2                 CEMIP2
DDIT3.1                 DDIT3
PAPSS1                 PAPSS1
TPT1.1                   TPT1
FSCN1                   FSCN1
TYMS.1                   TYMS
PPME1.1                 PPME1
ANKH                     ANKH
TNFAIP1.1             TNFAIP1
HSPB1.1                 HSPB1
TPM4.1                   TPM4
LY6E.1                   LY6E
HLA-B.1                 HLA-B
CHPF                     CHPF
SNHG7.1                 SNHG7
TCEAL4.1               TCEAL4
ITM2B.1                 ITM2B
APP                       APP
ARSE.1                   ARSE
NUAK1.1                 NUAK1
SPOCK1.1               SPOCK1
ABCC4.1                 ABCC4
GAS1.1                   GAS1
PHF19.1                 PHF19
HEBP2.1                 HEBP2
ACTR3                   ACTR3
BCL2L11               BCL2L11
PLAGL1.1               PLAGL1
FAM111A.1             FAM111A
MME.1                     MME
SLC25A6               SLC25A6
FAM177A1.1           FAM177A1
NAV2.1                   NAV2
KIF11.1                 KIF11
MGP.1                     MGP
PPP3CA.1               PPP3CA
ITGB5                   ITGB5
SOCS1.1                 SOCS1
AL357055.3.1       AL357055.3
APOC1.1                 APOC1
C9orf47.1             C9orf47
CLTB.1                   CLTB
KCTD12                 KCTD12
RTL9.1                   RTL9
PYGB.1                   PYGB
SIPA1L1.1             SIPA1L1
DPP4.1                   DPP4
DUSP10.1               DUSP10
C6orf48.1             C6orf48
YTHDF3-AS1.1       YTHDF3-AS1
NOS1AP.1               NOS1AP
COL1A2                 COL1A2
NREP.1                   NREP
NME4.1                   NME4
NAP1L1.1               NAP1L1
APBB3.1                 APBB3
PFN2                     PFN2
NCK1.1                   NCK1
JAK1.1                   JAK1
MSANTD2.1             MSANTD2
HS6ST2.1               HS6ST2
FAM89A.1               FAM89A
SEMA5A.1               SEMA5A
TGFBR1.1               TGFBR1
ACSL1.1                 ACSL1
JPT1.1                   JPT1
TTF2.1                   TTF2
TCAF1                   TCAF1
CCDC69.1               CCDC69
C1GALT1.1             C1GALT1
ZFP36L2.1             ZFP36L2
RAB31.1                 RAB31
EIF1B.1                 EIF1B
LOXL2                   LOXL2
SUGCT.1                 SUGCT
RPS6KA1.1             RPS6KA1
HIGD1A.1               HIGD1A
PCSK5.1                 PCSK5
CBLN3                   CBLN3
SPSB1.1                 SPSB1
ANGPT2.1               ANGPT2
CD59                     CD59
MRAS.1                   MRAS
BTBD3.1                 BTBD3
WDR90.1                 WDR90
PDGFRB.1               PDGFRB
KDR.1                     KDR
FLNB.1                   FLNB
ZNF573.1               ZNF573
ZNF653.1               ZNF653
CES4A.1                 CES4A
PRPS2.1                 PRPS2
ITGAV                   ITGAV
TFRC.1                   TFRC
CCDC183.1             CCDC183
GADD45A.1             GADD45A
NDST1                   NDST1
PDLIM5                 PDLIM5
CENPU.1                 CENPU
AC242426.2.1       AC242426.2
TPRG1.1                 TPRG1
CD99.1                   CD99
CD55.1                   CD55
MXRA5.1                 MXRA5
TGFBI.1                 TGFBI
TMED3.1                 TMED3
ISPD.1                   ISPD
MGST1.1                 MGST1
PLIN2.1                 PLIN2
OCIAD2.1               OCIAD2
NFKBIA.1               NFKBIA
STC2.1                   STC2
SPARC.1                 SPARC
PIK3R1.1               PIK3R1
HACD1.1                 HACD1
PNPLA3.1               PNPLA3
LGALS3.1               LGALS3
DPP7                     DPP7
TNFAIP8.1             TNFAIP8
ARL4D.1                 ARL4D
AC124016.1.1       AC124016.1
CD47                     CD47
MBOAT2.1               MBOAT2
P3H1                     P3H1
PRKAR2B.1             PRKAR2B
MERTK.1                 MERTK
AMOTL2                 AMOTL2
SPINT1.1               SPINT1
PIM1.1                   PIM1
MYH9                     MYH9
FAM84B                 FAM84B
OGFRL1                 OGFRL1
UGP2.1                   UGP2
AC012615.1.1       AC012615.1
PDLIM7.1               PDLIM7
CDC42EP3.1           CDC42EP3
ATP1B1                 ATP1B1
UBXN4.1                 UBXN4
CDKN2B                 CDKN2B
CDC42EP1.1           CDC42EP1
KIF22.1                 KIF22
CNTROB.1               CNTROB
FGFRL1.1               FGFRL1
EDIL3                   EDIL3
LRRC32                 LRRC32
NFIA.1                   NFIA
HAPLN3.1               HAPLN3
CDC37L1.1             CDC37L1
CDH11                   CDH11
IGFBP3.1               IGFBP3
ITGB1                   ITGB1
NQO2.1                   NQO2
LAMC3.1                 LAMC3
PLOD2                   PLOD2
PRICKLE1.1           PRICKLE1
ARID5B.1               ARID5B
PKM                       PKM
FIBIN.1                 FIBIN
NDRG2.1                 NDRG2
HSPB7.1                 HSPB7
C5orf38.1             C5orf38
SOX11.1                 SOX11
ADAMTS3               ADAMTS3
PNRC1.1                 PNRC1
PHACTR2.1             PHACTR2
PLEC                     PLEC
CD68.1                   CD68
BLOC1S5.1             BLOC1S5
ITPKA.1                 ITPKA
ICE1.1                   ICE1
DACH1.1                 DACH1
AMMECR1.1             AMMECR1
ATF4.1                   ATF4
IER3-AS1             IER3-AS1
IGFBP5.1               IGFBP5
MAP3K7CL.1           MAP3K7CL
STAG1.1                 STAG1
ZNF320.1               ZNF320
TIMP4.1                 TIMP4
PDS5B.1                 PDS5B
AC012467.2.1       AC012467.2
FBN2                     FBN2
AC067750.1         AC067750.1
FLRT3.1                 FLRT3
PLPP3.1                 PLPP3
SMPDL3A.1             SMPDL3A
TMEM143               TMEM143
SLC3A2.1               SLC3A2
GNL1.1                   GNL1
HOOK3.1                 HOOK3
EXOSC9.1               EXOSC9
KLF9.1                   KLF9
PLEKHA4.1             PLEKHA4
GALNT10.1             GALNT10
ISCU.1                   ISCU
MNDA.1                   MNDA
AL135905.1.1       AL135905.1
GPSM2.1                 GPSM2
CLK1.1                   CLK1
TOX2.1                   TOX2
PITX2                   PITX2
CRYAB.1                 CRYAB
EPOR                     EPOR
PNMT.1                   PNMT
ZNF345.1               ZNF345
NAV1.1                   NAV1
CHAC1                   CHAC1
LRRC38.1               LRRC38
SLPI.1                   SLPI
BMPER.1                 BMPER
RGS2.1                   RGS2
LMNB1.1                 LMNB1
TMEM178A.1           TMEM178A
GNAI2.1                 GNAI2
ITGA1.1                 ITGA1
SLC46A3.1             SLC46A3
WASF3.1                 WASF3
LRP1.1                   LRP1
SLC9A9                 SLC9A9
TMEM98.1               TMEM98
DUSP1.1                 DUSP1
C1orf112.1           C1orf112
FST.1                     FST
FAM129A.1             FAM129A
ENC1.1                   ENC1
LRRC2.1                 LRRC2
PMP22.1                 PMP22
HECW1.1                 HECW1
C1R.1                     C1R
CHD9                     CHD9
NKAIN4.1               NKAIN4
RPP38.1                 RPP38
CLASRP                 CLASRP
GPBAR1.1               GPBAR1
ZNF10                   ZNF10
CENPE.1                 CENPE
TMED9.1                 TMED9
NAMPT.1                 NAMPT
KCTD2.1                 KCTD2
PLOD1                   PLOD1
TMEM91.1               TMEM91
ANP32B.1               ANP32B
DCBLD1.1               DCBLD1
KIAA1024.1           KIAA1024
EPS8.1                   EPS8
AC245060.5         AC245060.5
ATP1A1.1               ATP1A1
MATN3.1                 MATN3
RAD9A                   RAD9A
HIF3A.1                 HIF3A
PHTF2.1                 PHTF2
BOD1L1.1               BOD1L1
AVIL                     AVIL
GADD45B.1             GADD45B
ACTC1.1                 ACTC1
KIRREL1               KIRREL1
YARS.1                   YARS
SLC26A11.1           SLC26A11
PGM2L1                 PGM2L1
BNIP3.1                 BNIP3
EBI3.1                   EBI3
HAND2.1                 HAND2
FILIP1L.1             FILIP1L
GATAD2B.1             GATAD2B
ATXN1                   ATXN1
FHL1.1                   FHL1
MDH1                     MDH1
EGLN3.1                 EGLN3
P3H2.1                   P3H2
ACLY                     ACLY
ANXA5.1                 ANXA5
TPX2                     TPX2
DDR1.1                   DDR1
CPEB3.1                 CPEB3
BATF3.1                 BATF3
SPECC1L.1             SPECC1L
ZC3H12C.1             ZC3H12C
GDF6.1                   GDF6
PRDX2.1                 PRDX2
TTLL7.1                 TTLL7
SEPT9.1                 SEPT9
EBF1.1                   EBF1
CD151                   CD151
MT1E.1                   MT1E
SBF2-AS1.1           SBF2-AS1
CYB561D1.1           CYB561D1
ACKR2.1                 ACKR2
ARHGEF26.1           ARHGEF26
RAI14                   RAI14
DAPK1                   DAPK1
PRDX6.1                 PRDX6
NEK5.1                   NEK5
BLVRB.1                 BLVRB
MYLIP.1                 MYLIP
ATOX1.1                 ATOX1
DYNLL1.1               DYNLL1
PGAP1.1                 PGAP1
RHOU                     RHOU
PHF3.1                   PHF3
SULF2                   SULF2
OSBPL3                 OSBPL3
GNA13                   GNA13
H2AFJ.1                 H2AFJ
AC007014.1.1       AC007014.1
RPP25.1                 RPP25
DEK.1                     DEK
FGFR1.1                 FGFR1
ADM.1                     ADM
ANTXR1.1               ANTXR1
BCO2.1                   BCO2
SELENOW.1             SELENOW
MAP2K3.1               MAP2K3
SDHAF3.1               SDHAF3
FSTL3.1                 FSTL3
BACE1-AS.1           BACE1-AS
MCUB.1                   MCUB
PDIA4.1                 PDIA4
GUCY1A2               GUCY1A2
MYDGF.1                 MYDGF
ZMAT3                   ZMAT3
DIO3OS.1               DIO3OS
PDGFRL.1               PDGFRL
ANKRD28.1             ANKRD28
CDKN1A.1               CDKN1A
ROGDI                   ROGDI
GMPR.1                   GMPR
FLCN.1                   FLCN
TMEM160.1             TMEM160
NT5E.1                   NT5E
FSTL1.1                 FSTL1
PCK2.1                   PCK2
ZNF816.1               ZNF816
CDK5RAP2.1           CDK5RAP2
PHLDB1.1               PHLDB1
MANSC1.1               MANSC1
HSD17B6.1             HSD17B6
UGCG                     UGCG
SGK1                     SGK1
ZNF23                   ZNF23
CCNA2.1                 CCNA2
RASAL2-AS1.1       RASAL2-AS1
SCNN1A.1               SCNN1A
ADAM10.1               ADAM10
EEF1A1                 EEF1A1
REXO5.1                 REXO5
PYCARD.1               PYCARD
PKIA.1                   PKIA
EPG5                     EPG5
LIG1.1                   LIG1
ANKRD1.1               ANKRD1
PRAC1.1                 PRAC1
UBASH3B.1             UBASH3B
DACT3.1                 DACT3
MT1X.1                   MT1X
CTSZ.1                   CTSZ
GNG11.1                 GNG11
CAP2.1                   CAP2
ARHGAP26.1           ARHGAP26
S100A13               S100A13
FAM111A-DT.1       FAM111A-DT
BEX1.1                   BEX1
FAXDC2                 FAXDC2
HSD17B4               HSD17B4
AL138966.2.1       AL138966.2
UBE2B.1                 UBE2B
APOE.1                   APOE
DDAH1.1                 DDAH1
CCNI                     CCNI
POLR2J3.1.1         POLR2J3.1
PER1.1                   PER1
CILP2.1                 CILP2
HAND2-AS1.1         HAND2-AS1
CPZ.1                     CPZ
FKBP10.1               FKBP10
ADGRL4.1               ADGRL4
ABHD14A-ACY1.1   ABHD14A-ACY1
IGFBP2.1               IGFBP2
OSGIN1.1               OSGIN1
DSE.1                     DSE
FLT1.1                   FLT1
MSX2.1                   MSX2
CPE.1                     CPE
TMEM39B               TMEM39B
KAT6B.1                 KAT6B
DSC3.1                   DSC3
CDKN3                   CDKN3
FILNC1.1               FILNC1
AC011978.2.1       AC011978.2
SLC4A7                 SLC4A7
PLEKHG3.1             PLEKHG3
FOXF1.1                 FOXF1
SLC22A3.1             SLC22A3
COL8A2                 COL8A2
RPN1.1                   RPN1
TMEM38B               TMEM38B
CLSPN.1                 CLSPN
PABPC1L               PABPC1L
UBC.1                     UBC
LTBR.1                   LTBR
NRG1.1                   NRG1
TMEM190.1             TMEM190
RAB40A.1               RAB40A
TSPAN13               TSPAN13
NDFIP2.1               NDFIP2
CARD8-AS1.1         CARD8-AS1
BOD1.1                   BOD1
LINC00513           LINC00513
DGCR8.1                 DGCR8
AL121827.2.1       AL121827.2
MSRB3                   MSRB3
SAT2.1                   SAT2
UBE2D1                 UBE2D1
TLE4                     TLE4
GCH1.1                   GCH1
C15orf39.1           C15orf39
MCM3AP-AS1.1       MCM3AP-AS1
MSN                       MSN
CLU.1                     CLU
LRP5                     LRP5
ZDHHC14.1             ZDHHC14
ID3.1                     ID3
CENPL                   CENPL
ASNS.1                   ASNS
TUFT1                   TUFT1
MACF1                   MACF1
ISYNA1.1               ISYNA1
SKP2.1                   SKP2
MLEC.1                   MLEC
AHCTF1.1               AHCTF1
SLC26A2.1             SLC26A2
KCTD1.1                 KCTD1
LINC01355.1         LINC01355
RGS20.1                 RGS20
TBC1D9                 TBC1D9
C1S.1                     C1S
FOXO3.1                 FOXO3
C3orf58.1             C3orf58
ALKAL2.1               ALKAL2
EDN1.1                   EDN1
LBH                       LBH
PTPRJ.1                 PTPRJ
CHIC2                   CHIC2
CAVIN3.1               CAVIN3
CFAP97.1               CFAP97
LGALS1.1               LGALS1
IFITM2.1               IFITM2
MIOS.1                   MIOS
GNPDA1.1               GNPDA1
KCNG1.1                 KCNG1
MGAT5                   MGAT5
MANF.1                   MANF
CAV1.1                   CAV1
HMMR.1                   HMMR
EGFR.1                   EGFR
RND3                     RND3
CSNK1E                 CSNK1E
ALDH1B1.1             ALDH1B1
FRY-AS1.1             FRY-AS1
BEX3.1                   BEX3
SLC27A4               SLC27A4
FGD4.1                   FGD4
C1QTNF6.1             C1QTNF6
DOCK5.1                 DOCK5
FEZ1.1                   FEZ1
RDH10.1                 RDH10
TRAPPC5.1             TRAPPC5
FMO5.1                   FMO5
DNAJC1.1               DNAJC1
MYH10.1                 MYH10
COL6A2.1               COL6A2
HOXB-AS3.1           HOXB-AS3
CHCHD10.1             CHCHD10
SPOUT1.1               SPOUT1
CNOT6L.1               CNOT6L
TCEAL1.1               TCEAL1
SLC5A6.1               SLC5A6
C21orf91             C21orf91
IKZF2.1                 IKZF2
SLC25A53.1           SLC25A53
CENPS.1                 CENPS
TRMT61A               TRMT61A
HIF1A.1                 HIF1A
MC1R.1                   MC1R
F3.1                       F3
MAN1A1.1               MAN1A1
P4HA3                   P4HA3
AC069544.1.1       AC069544.1
CERCAM                 CERCAM
ALDOC                   ALDOC
ADRA2C.1               ADRA2C
STRADB.1               STRADB
ALDH3A2.1             ALDH3A2
HLA-A                   HLA-A
DDX60.1                 DDX60
AC090204.1.1       AC090204.1
TNNC1.1                 TNNC1
NUDT8.1                 NUDT8
EGLN1.1                 EGLN1
OSBPL10.1             OSBPL10
TRHDE.1                 TRHDE
USP1.1                   USP1
MYC.1                     MYC
TCF4                     TCF4
C8orf33.1             C8orf33
TUT4.1                   TUT4
SOAT1.1                 SOAT1
AC104986.2.1       AC104986.2
FAT1                     FAT1
ALDH3B1               ALDH3B1
GSDMD.1                 GSDMD
CXCL14.1               CXCL14
NEAT1.1                 NEAT1
MAF.1                     MAF
HACD2                   HACD2
CAST.1                   CAST
ANGPTL1               ANGPTL1
GSTZ1.1                 GSTZ1
PAPPA.1                 PAPPA
HSBP1L1.1             HSBP1L1
PDLIM3.1               PDLIM3
RBMS3.1                 RBMS3
OSR1.1                   OSR1
GOPC                     GOPC
PTTG1IP.1             PTTG1IP
HECTD3.1               HECTD3
PPA1                     PPA1
LY6G5C.1               LY6G5C
ERRFI1.1               ERRFI1
LINC01503.1         LINC01503
MBNL1.1                 MBNL1
SMIM3.1                 SMIM3
RAB13.1                 RAB13
HGH1.1                   HGH1
HSPG2                   HSPG2
MBLAC1.1               MBLAC1
LPCAT2.1               LPCAT2
ACVR2A                 ACVR2A
CEP295.1               CEP295
ZNF565.1               ZNF565
PRSS12.1               PRSS12
SLC24A1.1             SLC24A1
ARSJ                     ARSJ
ZMIZ1                   ZMIZ1
AC013652.1.1       AC013652.1
HIST1H4C.1           HIST1H4C
CYR61                   CYR61
PPFIA3.1               PPFIA3
MRPS2.1                 MRPS2
COL7A1                 COL7A1
KRT18.1                 KRT18
SAXO2.1                 SAXO2
PPFIBP1.1             PPFIBP1
CNN2                     CNN2
XPC.1                     XPC
ANKFN1.1               ANKFN1
AC025171.4.1       AC025171.4
TKT.1                     TKT
JAGN1                   JAGN1
TSLP.1                   TSLP
VGLL3.1                 VGLL3
MMP23B.1               MMP23B
SELENBP1.1           SELENBP1
MSI2.1                   MSI2
REC8.1                   REC8
WWC2.1                   WWC2
COL15A1               COL15A1
DPT                       DPT
HSPB6                   HSPB6
DLC1                     DLC1
CORO2B.1               CORO2B
SPTAN1.1               SPTAN1
SIX4.1                   SIX4
HMGN3.1                 HMGN3
AC132192.2.1       AC132192.2
TXN                       TXN
TACC1.1                 TACC1
RNF152.1               RNF152
PRIM1                   PRIM1
DBN1.1                   DBN1
DYNLL2.1               DYNLL2
TWSG1.1                 TWSG1
SCD5.1                   SCD5
AP002884.1.1       AP002884.1
PGRMC1.1               PGRMC1
PWAR6.1                 PWAR6
XPOT.1                   XPOT
AGRN                     AGRN
LHFPL6.1               LHFPL6
CYTOR.1                 CYTOR
RGS4.1                   RGS4
CTNNB1.1               CTNNB1
ODC1.1                   ODC1
CENPK                   CENPK
SCG2.1                   SCG2
CD320.1                 CD320
TMEM229B.1           TMEM229B
S1PR3.1                 S1PR3
BMP4.1                   BMP4
ZSWIM6.1               ZSWIM6
SYT8.1                   SYT8
TARSL2.1               TARSL2
TRIB3.1                 TRIB3
RAB11FIP5.1         RAB11FIP5
EPB41L4A-AS1.1   EPB41L4A-AS1
SLC35A3.1             SLC35A3
IFITM1.1               IFITM1
CEP44.1                 CEP44
IFNGR1.1               IFNGR1
SDCBP.1                 SDCBP
AC016831.1.1       AC016831.1
TSPAN6.1               TSPAN6
TRAF4.1                 TRAF4
PLEKHA2.1             PLEKHA2
ECM1.1                   ECM1
MOV10L1               MOV10L1
KCNC4.1                 KCNC4
RAD51AP1.1           RAD51AP1
AL451085.2         AL451085.2
ORAI3                   ORAI3
PAM                       PAM
AVPI1.1                 AVPI1
FADS1.1                 FADS1
SFMBT2.1               SFMBT2
ENTPD4                 ENTPD4
ZBTB16.1               ZBTB16
SP4                       SP4
PTPRA.1                 PTPRA
FAM96B.1               FAM96B
PAPSS2                 PAPSS2
OCA2.1                   OCA2
SSR4.1                   SSR4
PLPPR3.1               PLPPR3
FZD2.1                   FZD2
PKP2.1                   PKP2
SOCS2.1                 SOCS2
SERPING1.1           SERPING1
GALNT18.1             GALNT18
TCTN3.1                 TCTN3
SLC19A2.1             SLC19A2
GTF2B.1                 GTF2B
BEND6                   BEND6
HINT1.1                 HINT1
TMEM87B               TMEM87B
DYNLT3                 DYNLT3
BRCC3.1                 BRCC3
TMEM88.1               TMEM88
PSME1.1                 PSME1
DCAF16.1               DCAF16
THY1                     THY1
RASSF4.1               RASSF4
PDCD4-AS1           PDCD4-AS1
CTNNAL1.1             CTNNAL1
KLHL5.1                 KLHL5
TTC21B.1               TTC21B
SPRED1                 SPRED1
MCM5                     MCM5
HSPB3.1                 HSPB3
RTN3.1                   RTN3
RNF115.1               RNF115
POPDC2.1               POPDC2
RBPMS.1                 RBPMS
MINCR.1                 MINCR
SEPSECS.1             SEPSECS
PTHLH.1                 PTHLH
SPATA7.1               SPATA7
LASP1                   LASP1
EPHB3                   EPHB3
HEYL.1                   HEYL
YWHAG.1                 YWHAG
SLC7A11.1             SLC7A11
ADARB1.1               ADARB1
SMIM10.1               SMIM10
PLEKHA7.1             PLEKHA7
C19orf48             C19orf48
DHCR7.1                 DHCR7
AC092807.3.1       AC092807.3
CAV2                     CAV2
MYL12B.1               MYL12B
TXNRD1.1               TXNRD1
SNHG12.1               SNHG12
NEDD4.1                 NEDD4
NOP58                   NOP58
MAGI1                   MAGI1
THRA.1                   THRA
GOLGA7.1               GOLGA7
TBX3.1                   TBX3
EMILIN1.1             EMILIN1
SDC1.1                   SDC1
VAMP3.1                 VAMP3
RNF112.1               RNF112
SEMA3D.1               SEMA3D
AATF.1                   AATF
SLC2A1                 SLC2A1
HMG20A                 HMG20A
RASSF7.1               RASSF7
MICALL2.1             MICALL2
RBMS1                   RBMS1
ANGPTL4.1             ANGPTL4
DUSP16.1               DUSP16
NTM.1                     NTM
DDR2                     DDR2
SYNPO                   SYNPO
FARP1                   FARP1
RALBP1                 RALBP1
BCAT1                   BCAT1
CSGALNACT2.1       CSGALNACT2
ARHGAP29.1           ARHGAP29
ACOX1.1                 ACOX1
SEPT11.1               SEPT11
HOXA10-AS           HOXA10-AS
ATP2B4.1               ATP2B4
FADS3.1                 FADS3
BIN1.1                   BIN1
SPATA5L1.1           SPATA5L1
CCDC28B.1             CCDC28B
OTUD3.1                 OTUD3
MINDY2.1               MINDY2
CALM1.1                 CALM1
TFPI.1                   TFPI
COL6A3                 COL6A3
RTN4                     RTN4
AK6                       AK6
TEX22.1                 TEX22
IGF2BP3.1             IGF2BP3
MICAL3.1               MICAL3
AC097103.2         AC097103.2
HLA-DPB1.1           HLA-DPB1
SLC23A2.1             SLC23A2
CNPY2.1                 CNPY2
PATJ.1                   PATJ
TAF1D.1                 TAF1D
CEP290.1               CEP290
BRSK2                   BRSK2
CDK6                     CDK6
KAT8.1                   KAT8
HIP1.1                   HIP1
HEPH                     HEPH
COL5A1.1               COL5A1
BIN3.1                   BIN3
RASA3.1                 RASA3
FJX1.1                   FJX1
TARBP1                 TARBP1
RAB29.1                 RAB29
IFNGR2.1               IFNGR2
GBP1.1                   GBP1
ZNF358.1               ZNF358
SEPT4.1                 SEPT4
ZFYVE27.1             ZFYVE27
GPC6.1                   GPC6
HEATR5A               HEATR5A
TM4SF1.1               TM4SF1
FERMT2.1               FERMT2
PITPNM3               PITPNM3
CNTLN.1                 CNTLN
AHNAK.1                 AHNAK
SESN2.1                 SESN2
RARB.1                   RARB
SHPK                     SHPK
DNAJB2.1               DNAJB2
ZNF563.1               ZNF563
CCDC97.1               CCDC97
BNC1                     BNC1
AP002761.3.1       AP002761.3
CTSV.1                   CTSV
FOXF2.1                 FOXF2
GUCY1A1               GUCY1A1
TNFRSF1A.1           TNFRSF1A
EPRS.1                   EPRS
NQO1.1                   NQO1
NENF.1                   NENF
CRY1                     CRY1
CYP27A1.1             CYP27A1
AC020928.1         AC020928.1
AC098487.1.1       AC098487.1
WNT5B.1                 WNT5B
FLOT1.1                 FLOT1
AL355338.1.1       AL355338.1
C1QL1.1                 C1QL1
ASS1.1                   ASS1
UTP23.1                 UTP23
HMOX1.1                 HMOX1
MSC.1                     MSC
TLCD2.1                 TLCD2
PLXNA3                 PLXNA3
TMEM44                 TMEM44
TMEM11.1               TMEM11
CBR3.1                   CBR3
FAM133B.1             FAM133B
CDCA7L.1               CDCA7L
BBX.1                     BBX
PIM2                     PIM2
AP000695.2.1       AP000695.2
FRZB.1                   FRZB
DET1.1                   DET1
COMMD7.1               COMMD7
DUSP28                 DUSP28
DNAJB1.1               DNAJB1
ADAM19.1               ADAM19
DSEL.1                   DSEL
NINJ1                   NINJ1
IRF2BPL               IRF2BPL
MPLKIP.1               MPLKIP
ZNF324.1               ZNF324
SNAI2.1                 SNAI2
EIF4A3                 EIF4A3
HMGA2.1                 HMGA2
REL.1                     REL
USP20                   USP20
SPATA6                 SPATA6
KREMEN1.1             KREMEN1
SLC25A45             SLC25A45
SLC39A2.1             SLC39A2
TPGS2.1                 TPGS2
KIF20B.1               KIF20B
NUPR1                   NUPR1
TRIM16L.1             TRIM16L
IFI16                   IFI16
EFNB2                   EFNB2
LITAF.1                 LITAF
TNFAIP3               TNFAIP3
GALNT7                 GALNT7
SBNO1.1                 SBNO1
RCAN1.1                 RCAN1
FAM117B.1             FAM117B
TRAF2.1                 TRAF2
YDJC                     YDJC
STX12.1                 STX12
PBX1.1                   PBX1
EIF5.1                   EIF5
TESK2.1                 TESK2
MIATNB.1               MIATNB
WWC3.1                   WWC3
DDX5.1                   DDX5
C18orf25.1           C18orf25
MSMO1.1                 MSMO1
SELENOP.1             SELENOP
NOP53.1                 NOP53
MT-CO1.1               MT-CO1
SALL4                   SALL4
PLAU.1                   PLAU
CRTAP.1                 CRTAP
RSPO2.1                 RSPO2
SLC37A3.1             SLC37A3
PPP1R14A.1           PPP1R14A
TUSC1                   TUSC1
MIR22HG.1             MIR22HG
TNKS1BP1.1           TNKS1BP1
FOXN2.1                 FOXN2
AC103740.2.1       AC103740.2
ABCD3.1                 ABCD3
P4HA1                   P4HA1
VTI1A.1                 VTI1A
TNFRSF12A.1         TNFRSF12A
FAM84A.1               FAM84A
PCCB                     PCCB
CPA4.1                   CPA4
DUSP6.1                 DUSP6
C4BPB.1                 C4BPB
CDCA3.1                 CDCA3
HSPA12A.1             HSPA12A
FOXQ1.1                 FOXQ1
GLUL.1                   GLUL
NPW                       NPW
NR2F1.1                 NR2F1
PLXNA4                 PLXNA4
DUSP8                   DUSP8
GLIS3                   GLIS3
WDR1.1                   WDR1
FAM122C               FAM122C
SYNM                     SYNM
INTS13.1               INTS13
MDFI.1                   MDFI
PSIP1.1                 PSIP1
GCNT1                   GCNT1
TYRP1.1                 TYRP1
ASPHD1                 ASPHD1
HSPB11.1               HSPB11
CMSS1.1                 CMSS1
TRIM65.1               TRIM65
SQSTM1.1               SQSTM1
FBXO5                   FBXO5
AC010336.1.1       AC010336.1
NPTXR.1                 NPTXR
EXPH5.1                 EXPH5
PRKD1.1                 PRKD1
PPP2R3C               PPP2R3C
LAGE3                   LAGE3
SLC16A4               SLC16A4
PTPN9.1                 PTPN9
SNAPC1                 SNAPC1
NCS1.1                   NCS1
AKR1C1.1               AKR1C1
FRMD6                   FRMD6
IMPA2.1                 IMPA2
ZNF385A.1             ZNF385A
AL590705.1.1       AL590705.1
ZSWIM3.1               ZSWIM3
LENG8.1                 LENG8
JUP.1                     JUP
FENDRR.1               FENDRR
PSMC6.1                 PSMC6
SNRPN.1                 SNRPN
YPEL3                   YPEL3
HSD17B7.1             HSD17B7
KLHDC10.1             KLHDC10
IRS1                     IRS1
VAMP8                   VAMP8
AKAP17A.1             AKAP17A
SYNGR2.1               SYNGR2
C1orf216.1           C1orf216
ISG15.1                 ISG15
CARHSP1.1             CARHSP1
SNCAIP.1               SNCAIP
SELENOK.1             SELENOK
COL14A1.1             COL14A1
METTL27.1             METTL27
KATNBL1.1             KATNBL1
CENPV.1                 CENPV
SMARCE1.1             SMARCE1
GPRC5B                 GPRC5B
ZNF700                 ZNF700
MBNL1-AS1.1         MBNL1-AS1
ZNF468.1               ZNF468
BAG3.1                   BAG3
PLP2.1                   PLP2
ECI1.1                   ECI1
HSP90B1               HSP90B1
GLB1L.1                 GLB1L
ZNF711.1               ZNF711
MBP.1                     MBP
HMGB3.1                 HMGB3
ALDH1L2.1             ALDH1L2
ACAA1.1                 ACAA1
LINC01504.1         LINC01504
MED28.1                 MED28
ARID3A.1               ARID3A
IGDCC4.1               IGDCC4
SLC26A6.1             SLC26A6
LINC00862.1         LINC00862
ZBTB1.1                 ZBTB1
MFSD1.1                 MFSD1
PPL.1                     PPL
SMURF2.1               SMURF2
GAS6-DT.1             GAS6-DT
SLC2A13               SLC2A13
ALDH4A1               ALDH4A1
LDLR.1                   LDLR
BACH1.1                 BACH1
LRRC49.1               LRRC49
HAUS4.1                 HAUS4
POLR1B.1               POLR1B
NT5C.1                   NT5C
BDH2.1                   BDH2
LDLRAD4.1             LDLRAD4
COPG2                   COPG2
GXYLT2.1               GXYLT2
AC245041.2         AC245041.2
DNAJA1                 DNAJA1
RAD21.1                 RAD21
PARP9.1                 PARP9
SLC1A1.1               SLC1A1
C5orf30.1             C5orf30
AGL.1                     AGL
CCDC144NL-AS1.1 CCDC144NL-AS1
CUBN.1                   CUBN
COL4A5                 COL4A5
LUM.1                     LUM
PECR.1                   PECR
FIP1L1.1               FIP1L1
MORC3.1                 MORC3
GALNT1                 GALNT1
RSRC2                   RSRC2
RFXANK                 RFXANK
NID2                     NID2
SPRY2.1                 SPRY2
AC092069.1.1       AC092069.1
TNKS.1                   TNKS
STK38L.1               STK38L
SNX16                   SNX16
MYO10.1                 MYO10
ST13                     ST13
ADAMTS1.1             ADAMTS1
PARD6B                 PARD6B
BARD1.1                 BARD1
STK17B.1               STK17B
THBS2.1                 THBS2
CEP135                 CEP135
SCRG1.1                 SCRG1
MPC2.1                   MPC2
NOL3.1                   NOL3
ID1.1                     ID1
EID2B                   EID2B
SH3BP5.1               SH3BP5
SLCO4A1.1             SLCO4A1
ODF2L.1                 ODF2L
CYP2U1                 CYP2U1
VASP.1                   VASP
KCNK12.1               KCNK12
AC064836.3.1       AC064836.3
WNT5A                   WNT5A
RERE.1                   RERE
KCNK1.1                 KCNK1
LARP7.1                 LARP7
OTUD1.1                 OTUD1
TGIF1.1                 TGIF1
USP15                   USP15
TCEA1.1                 TCEA1
HIST1H4E.1           HIST1H4E
SMOX.1                   SMOX
TNC                       TNC
PISD.1                   PISD
TUBB6.1                 TUBB6
MBNL2.1                 MBNL2
NCAM2.1                 NCAM2
CHEK1.1                 CHEK1
TRIM16.1               TRIM16
ITGB6.1                 ITGB6
PHF20                   PHF20
HIST3H2A.1           HIST3H2A
BRD1                     BRD1
AADAT.1                 AADAT
PTGR1                   PTGR1
GUSB.1                   GUSB
MPRIP.1                 MPRIP
DYM.1                     DYM
POLE4.1                 POLE4
ADGRV1.1               ADGRV1
AZI2.1                   AZI2
KCNS3.1                 KCNS3
CAMTA2.1               CAMTA2
PTPN11.1               PTPN11
CLIP1.1                 CLIP1
AC004803.1.1       AC004803.1
IFFO2                   IFFO2
SMIM15.1               SMIM15
TMEM158.1             TMEM158
NSMF.1                   NSMF
GPR161.1               GPR161
CISD1.1                 CISD1
ZNRF3.1                 ZNRF3
FGD6.1                   FGD6
LPAR6.1                 LPAR6
B4GALT1.1             B4GALT1
KRT19.1                 KRT19
TRIM22.1               TRIM22
KRT8.1                   KRT8
BBOX1-AS1.1         BBOX1-AS1
RASSF2.1               RASSF2
SMC4.1                   SMC4
RASSF8-AS1         RASSF8-AS1
GOLIM4                 GOLIM4
TBC1D12.1             TBC1D12
ZSCAN25.1             ZSCAN25
JUND                     JUND
WNT2.1                   WNT2
AC020571.1.1       AC020571.1
PSPN.1                   PSPN
LYSMD2.1               LYSMD2
CARMN.1                 CARMN
ATP13A3               ATP13A3
ADNP-AS1             ADNP-AS1
HENMT1                 HENMT1
ENDOD1.1               ENDOD1
FAM133A.1             FAM133A
DTX3L.1                 DTX3L
C2CD3.1                 C2CD3
GRAMD2B.1             GRAMD2B
CXCL2.1                 CXCL2
TOM1.1                   TOM1
LINC00842           LINC00842
B3GNT5.1               B3GNT5
DNASE2.1               DNASE2
KIF23.1                 KIF23
MMP14                   MMP14
ATP8B4                 ATP8B4
SLC7A5                 SLC7A5
RIOK2.1                 RIOK2
CLTCL1                 CLTCL1
SLC22A15.1           SLC22A15
CFAP20.1               CFAP20
FGFR2                   FGFR2
FAM83G.1               FAM83G
UTP6.1                   UTP6
PEG10.1                 PEG10
IER5L.1                 IER5L
ZWINT.1                 ZWINT
ZNF136.1               ZNF136
INMT.1                   INMT
PCGF5.1                 PCGF5
SORBS2                 SORBS2
MUM1L1                 MUM1L1
TXNDC5.1               TXNDC5
CYCS.1                   CYCS
CAP1.1                   CAP1
OPTN.1                   OPTN
MAFF                     MAFF
ASTE1                   ASTE1
MRPS30.1               MRPS30
WARS.1                   WARS
DAAM1.1                 DAAM1
RNF227.1               RNF227
MEX3A                   MEX3A
DDX24                   DDX24
AL356056.1.1       AL356056.1
TMEM51                 TMEM51
FGF1.1                   FGF1
UBXN2A.1               UBXN2A
KLF4.1                   KLF4
CLTA                     CLTA
FXYD5.1                 FXYD5
PHLDA3.1               PHLDA3
TUBA4A.1               TUBA4A
ADAMTS9.1             ADAMTS9
BCAR3.1                 BCAR3
GPC3.1                   GPC3
TFPT.1                   TFPT
RBP1.1                   RBP1
TRIM24.1               TRIM24
CMIP.1                   CMIP
CEBPZOS               CEBPZOS
RIMS1.1                 RIMS1
GNAI1                   GNAI1
ATP9A                   ATP9A
CKB.1                     CKB
XYLT1                   XYLT1
DNAJC15.1             DNAJC15
CTSO.1                   CTSO
ULK4.1                   ULK4
ASAP1.1                 ASAP1
UHMK1.1                 UHMK1
TSPO.1                   TSPO
ECHDC2                 ECHDC2
GSDME                   GSDME
BLM.1                     BLM
UBE2E2.1               UBE2E2
CPPED1.1               CPPED1
CITED2.1               CITED2
PUS7L.1                 PUS7L
HUNK.1                   HUNK
KLF5.1                   KLF5
MMS22L                 MMS22L
AC124798.1.1       AC124798.1
TRAPPC6A             TRAPPC6A
AC018647.1.1       AC018647.1
APBA2.1                 APBA2
BTG3.1                   BTG3
MAFG.1                   MAFG
C1QTNF1               C1QTNF1
LYPD1.1                 LYPD1
C3AR1                   C3AR1
ST20                     ST20
ZNF608                 ZNF608
ICAM3.1                 ICAM3
PRKCI                   PRKCI
AFAP1.1                 AFAP1
PELI3.1                 PELI3
KCNE4                   KCNE4
ADAMTS12.1           ADAMTS12
SYPL1.1                 SYPL1
SLC39A10             SLC39A10
GPR68.1                 GPR68
AGPAT2.1               AGPAT2
NT5DC1.1               NT5DC1
TNFAIP6.1             TNFAIP6
ZER1                     ZER1
IL6ST                   IL6ST
AKAP13                 AKAP13
SSC4D.1                 SSC4D
MORC4.1                 MORC4
SMIM1                   SMIM1
ZNF678.1               ZNF678
TRAF3IP1.1           TRAF3IP1
AL731563.3         AL731563.3
PKDCC.1                 PKDCC
SRRM3.1                 SRRM3
SLC5A12.1             SLC5A12
TMEM131L             TMEM131L
NORAD.1                 NORAD
ZFP36.1                 ZFP36
HIST1H2BD.1         HIST1H2BD
LANCL2                 LANCL2
AL035413.1.1       AL035413.1
CTSK.1                   CTSK
IRF1.1                   IRF1
ST6GALNAC5         ST6GALNAC5
FOS.1                     FOS
JPH2                     JPH2
BCL7A.1                 BCL7A
MANCR.1                 MANCR
AC009126.1         AC009126.1
GPN3                     GPN3
PBLD.1                   PBLD
FLVCR1.1               FLVCR1
CPD.1                     CPD
NCSTN                   NCSTN
SMYD4                   SMYD4
CMTM7.1                 CMTM7
JUN                       JUN
ADAM23.1               ADAM23
ADAMTS2.1             ADAMTS2
LHFPL2.1               LHFPL2
SLC9A8                 SLC9A8
FASN                     FASN
CHST7                   CHST7
HADH.1                   HADH
HMGB2.1                 HMGB2
LETM2.1                 LETM2
GREM1.1                 GREM1
HAS2.1                   HAS2
SYTL2.1                 SYTL2
WNK1                     WNK1
VWA8.1                   VWA8
COL11A1.1             COL11A1
CTH.1                     CTH
TRIL                     TRIL
TTLL11.1               TTLL11
CCND3                   CCND3
SNAI1.1                 SNAI1
ALOX5AP.1             ALOX5AP
DMRTA1.1               DMRTA1
TALDO1.1               TALDO1
KIAA0825.1           KIAA0825
ING2.1                   ING2
PHYHD1.1               PHYHD1
MKNK2                   MKNK2
UNC5B-AS1           UNC5B-AS1
TM7SF3.1               TM7SF3
LINC02001.1         LINC02001
DIXDC1.1               DIXDC1
PLEKHO1.1             PLEKHO1
TMEM8B.1               TMEM8B
DDX10.1                 DDX10
CEP126.1               CEP126
SHROOM3.1             SHROOM3
SGTA.1                   SGTA
RAB9A.1                 RAB9A
CADPS2.1               CADPS2
ZNF496.1               ZNF496
PTPA                     PTPA
MYADM.1                 MYADM
RHOB.1                   RHOB
ZCCHC10.1             ZCCHC10
CBR1.1                   CBR1
SLC29A4.1             SLC29A4
ZWILCH                 ZWILCH
KIT.1                     KIT
SLC10A3.1             SLC10A3
SERPINE1.1           SERPINE1
ASMTL.1                 ASMTL
AC044849.1.1       AC044849.1
NCOA1.1                 NCOA1
TENT5C                 TENT5C
TGM2.1                   TGM2
CDKL5.1                 CDKL5
WNT4.1                   WNT4
NID1.1                   NID1
LINC00626.1         LINC00626
IL13RA1               IL13RA1
TSTD1.1                 TSTD1
TPCN2.1                 TPCN2
SPTLC3.1               SPTLC3
TLL2.1                   TLL2
KIAA0040.1           KIAA0040
HSPA9.1                 HSPA9
SPTBN1.1               SPTBN1
LINC01750.1         LINC01750
GSTM3.1                 GSTM3
MIR4435-2HG.1     MIR4435-2HG
HMGB1                   HMGB1
SOCS3.1                 SOCS3
APH1B.1                 APH1B
MRPL52.1               MRPL52
TTC39B.1               TTC39B
NCAPD3.1               NCAPD3
CREBRF.1               CREBRF
HMCN1.1                 HMCN1
GLA.1                     GLA
GSTO1.1                 GSTO1
PGD.1                     PGD
TTYH3                   TTYH3
FASTKD3.1             FASTKD3
LCMT2                   LCMT2
ATP6AP1L.1           ATP6AP1L
SH3BP4.1               SH3BP4
LMBRD2                 LMBRD2
MT-ATP6.1             MT-ATP6
ZNF431                 ZNF431
TCEAL3                 TCEAL3
SEMA4F.1               SEMA4F
MT-CYB.1               MT-CYB
LGR4                     LGR4
PSME2                   PSME2
DEPP1.1                 DEPP1
TXNIP.1                 TXNIP
CLDN12.1               CLDN12
NET1.1                   NET1
LDHA                     LDHA
HMGN2.1                 HMGN2
SRRD.1                   SRRD
TNFRSF21.1           TNFRSF21
IP6K3.1                 IP6K3
BRIX1.1                 BRIX1
KCNQ5.1                 KCNQ5
WNT11.1                 WNT11
SH3PXD2B             SH3PXD2B
FGF2.1                   FGF2
MMADHC.1               MMADHC
TMEM106A.1           TMEM106A
SLC16A10             SLC16A10
SOX6                     SOX6
FBLN2.1                 FBLN2
PEA15                   PEA15
PDXK                     PDXK
TANGO2.1               TANGO2
SERP1                   SERP1
RPS4X                   RPS4X
TWIST1                 TWIST1
SOD2.1                   SOD2
TRIO.1                   TRIO
DNAJC28.1             DNAJC28
NAV3.1                   NAV3
MYOF                     MYOF
MON2.1                   MON2
ZNF394.1               ZNF394
C1orf198             C1orf198
H1F0.1                   H1F0
ABCA5.1                 ABCA5
ZBED3-AS1           ZBED3-AS1
COL4A4.1               COL4A4
TRIM56                 TRIM56
GTF2F1.1               GTF2F1
DLG1                     DLG1
SLC2A1-AS1.1       SLC2A1-AS1
CYB5A                   CYB5A
AMMECR1L.1           AMMECR1L
CDC42SE1.1           CDC42SE1
ZNF561.1               ZNF561
GDNF                     GDNF
GCNA                     GCNA
C1orf226.1           C1orf226
MRPS18B.1             MRPS18B
TTC27.1                 TTC27
PITX1.1                 PITX1
SVIP.1                   SVIP
PCDH19                 PCDH19
WEE1.1                   WEE1
TET1.1                   TET1
DGKI.1                   DGKI
TRAF3IP2-AS1.1   TRAF3IP2-AS1
SPRYD3.1               SPRYD3
CMTM3.1                 CMTM3
SCML1                   SCML1
YWHAH.1                 YWHAH
RNF150.1               RNF150
CNTN5                   CNTN5
HDDC2                   HDDC2
ZADH2.1                 ZADH2
KANK4.1                 KANK4
USP13.1                 USP13
WDR25                   WDR25
AC128688.2.1       AC128688.2
TMEM154.1             TMEM154
EGR1.1                   EGR1
TRIM59                 TRIM59
CARD16.1               CARD16
PTBP3                   PTBP3
ANXA3.1                 ANXA3
FABP3.1                 FABP3
CPQ.1                     CPQ
C1orf56.1             C1orf56
ST5.1                     ST5
LINC01836           LINC01836
WISP1                   WISP1
SRPX.1                   SRPX
ZNF121.1               ZNF121
CENPN                   CENPN
PLAA.1                   PLAA
STOM                     STOM
FABP5.1                 FABP5
THBS3.1                 THBS3
BICC1.1                 BICC1
RBM43                   RBM43
SCUBE2.1               SCUBE2
PCDHA4                 PCDHA4
MYL4.1                   MYL4
DPP10.1                 DPP10
AIFM2.1                 AIFM2
CCDC18.1               CCDC18
SUPV3L1.1             SUPV3L1
H2AFY.1                 H2AFY
TRIM5                   TRIM5
GATM.1                   GATM
AC132872.1.1       AC132872.1
CDC6.1                   CDC6
CLEC1B.1               CLEC1B
SPECC1                 SPECC1
CNN1.1                   CNN1
AKIRIN2               AKIRIN2
ARIH2.1                 ARIH2
ANKRD6.1               ANKRD6
TUSC3.1                 TUSC3
TCEAL9.1               TCEAL9
SCUBE3.1               SCUBE3
NKX3-1.1               NKX3-1
TFPI2.1                 TFPI2
HIST1H2BG.1         HIST1H2BG
SEPT10.1               SEPT10
PLXNA2                 PLXNA2
RAB3B.1                 RAB3B
IER3.1                   IER3
PIDD1.1                 PIDD1
RAB33A                 RAB33A
NDN.1                     NDN
CDC37L1-DT.1       CDC37L1-DT
ENAH.1                   ENAH
PYGO1                   PYGO1
THAP1                   THAP1
DENND1B               DENND1B
OSER1-DT.1           OSER1-DT
HOXB8.1                 HOXB8
LIF.1                     LIF
CIRBP.1                 CIRBP
PYGL                     PYGL
AC008467.1.1       AC008467.1
GK5.1                     GK5
BMPR2.1                 BMPR2
HGD.1                     HGD
AC103706.1.1       AC103706.1
PIM3.1                   PIM3
MEF2A.1                 MEF2A
BHLHE40               BHLHE40
CHERP                   CHERP
RAD51B.1               RAD51B
KLF2.1                   KLF2
NPEPL1                 NPEPL1
RAB15                   RAB15
ST6GAL1.1             ST6GAL1
GRIA3.1                 GRIA3
OLFM2.1                 OLFM2
CEP170.1               CEP170
LIPG.1                   LIPG
NOP14.1                 NOP14
RACGAP1.1             RACGAP1
ADRA2A                 ADRA2A
DCTN4.1                 DCTN4
CCDC125               CCDC125
PLA2G4C.1             PLA2G4C
SORBS3.1               SORBS3
STK26.1                 STK26
FRMD6-AS1.1         FRMD6-AS1
AC005476.2.1       AC005476.2
FAM69A.1               FAM69A
TMEM176B.1           TMEM176B
ME1.1                     ME1
CENPF.1                 CENPF
LINC01013.1         LINC01013
SAMD15                 SAMD15
MAP3K20.1             MAP3K20
SEMA7A                 SEMA7A
PKIB                     PKIB
CCL20.1                 CCL20
CCDC138.1             CCDC138
DBI                       DBI
COL3A1.1               COL3A1
PPP6R1.1               PPP6R1
ANKZF1                 ANKZF1
ZNF350.1               ZNF350
TTC30B.1               TTC30B
LEFTY2.1               LEFTY2
WDYHV1                 WDYHV1
ZNF223.1               ZNF223
SIPA1L2               SIPA1L2
LRIF1.1                 LRIF1
KIF13B                 KIF13B
AC010976.2.1       AC010976.2
CMTR2.1                 CMTR2
PRRC2A.1               PRRC2A
MYLK.1                   MYLK
GALNT5.1               GALNT5
ITGA10.1               ITGA10
KANK2.1                 KANK2
THBD.1                   THBD
KLHL4.1                 KLHL4
SLC2A8.1               SLC2A8
NCLN                     NCLN
SUSD6.1                 SUSD6
CHORDC1.1             CHORDC1
MED31                   MED31
PRAG1.1                 PRAG1
TSPAN2                 TSPAN2
WDR77                   WDR77
SEC14L2.1             SEC14L2
ZC4H2                   ZC4H2
TMEM50B.1             TMEM50B
ADAMTS6               ADAMTS6
ENTPD1                 ENTPD1
PRKAB2.1               PRKAB2
AEBP1.1                 AEBP1
CLIC6.1                 CLIC6
SERPINI1.1           SERPINI1
AC007114.1         AC007114.1
ANKRD11               ANKRD11
KCNN2.1                 KCNN2
CASTOR1.1             CASTOR1
PAMR1                   PAMR1
DYRK1B.1               DYRK1B
ZBTB38.1               ZBTB38
GUCY1B1               GUCY1B1
AURKA.1                 AURKA
TNFRSF10B.1         TNFRSF10B
PLSCR1.1               PLSCR1
KMO.1                     KMO
ARMCX1                 ARMCX1
TFAP2A                 TFAP2A
TUBB4B.1               TUBB4B
SLC5A3                 SLC5A3
CPNE7.1                 CPNE7
FMO4.1                   FMO4
PHIP.1                   PHIP
FNBP1L                 FNBP1L
DYNC1H1.1             DYNC1H1
PSMB8.1                 PSMB8
SESN3.1                 SESN3
ZFHX2                   ZFHX2
FZD3                     FZD3
ZNF34                   ZNF34
HMCES.1                 HMCES
GAB2.1                   GAB2
CWC25.1                 CWC25
TMTC1.1                 TMTC1
B3GNT2                 B3GNT2
TEX29.1                 TEX29
HOXC8.1                 HOXC8
ABAT                     ABAT
GTF2E1.1               GTF2E1
RBM24.1                 RBM24
PIR.1                     PIR
PTPRU                   PTPRU
RSPO3.1                 RSPO3
GPS1.1                   GPS1
CTSH                     CTSH
TMBIM1.1               TMBIM1
ZNF251.1               ZNF251
SMCHD1                 SMCHD1
NR1H3                   NR1H3
RCHY1                   RCHY1
UNKL.1                   UNKL
RCN3.1                   RCN3
COL25A1.1             COL25A1
MRAP2.1                 MRAP2
SGCD.1                   SGCD
L3HYPDH               L3HYPDH
CRYGS.1                 CRYGS
FANCI.1                 FANCI
RAP1B.1                 RAP1B
KLHL24                 KLHL24
TMEM241               TMEM241
MIS18BP1.1           MIS18BP1
CDH6.1                   CDH6
GINS4.1                 GINS4
HS3ST3A1.1           HS3ST3A1
EXTL1.1                 EXTL1
NPNT.1                   NPNT
ZNF486                 ZNF486
Z93930.2.1           Z93930.2
ARHGEF11.1           ARHGEF11
HOXA7.1                 HOXA7
FCGRT                   FCGRT
TM7SF2.1               TM7SF2
TMCO6.1                 TMCO6
SERPINB1.1           SERPINB1
AC112721.1.1       AC112721.1
FANCC                   FANCC
RPRML.1                 RPRML
MYOCD.1                 MYOCD
RNF217.1               RNF217
CTXN1.1                 CTXN1
BAIAP2-DT.1         BAIAP2-DT
SIGIRR                 SIGIRR
GAN                       GAN
GLP2R                   GLP2R
HAND1.1                 HAND1
SGCA                     SGCA
ATP23                   ATP23
GARS.1                   GARS
ANO4.1                   ANO4
PARM1.1                 PARM1
LIMCH1                 LIMCH1
GLIPR2.1               GLIPR2
DUSP23                 DUSP23
MT-ND5.1               MT-ND5
ERG28.1                 ERG28
NBL1.1                   NBL1
C1RL.1                   C1RL
CCDC74A.1             CCDC74A
AC009113.1         AC009113.1
RALA                     RALA
SRFBP1.1               SRFBP1
ZFHX4.1                 ZFHX4
AC021016.3         AC021016.3
SLC4A8.1               SLC4A8
KCND3.1                 KCND3
TRPC1                   TRPC1
KPNA5                   KPNA5
FIRRE.1                 FIRRE
MELTF.1                 MELTF
NFKBID                 NFKBID
EVA1A                   EVA1A
MAP3K8.1               MAP3K8
DHRS3.1                 DHRS3
DPP9                     DPP9
SBF1                     SBF1
B3GALNT2.1           B3GALNT2
KIAA1841             KIAA1841
ZNF469                 ZNF469
KMT5C.1                 KMT5C
VSIG10L.1             VSIG10L
TUBA1B                 TUBA1B
STK24.1                 STK24
EFNA1.1                 EFNA1
ANKRD34A.1           ANKRD34A
ADAT2.1                 ADAT2
SYNE2.1                 SYNE2
DGKH                     DGKH
IRAK1.1                 IRAK1
ZNF689.1               ZNF689
AL441883.1.1       AL441883.1
TP53I3.1               TP53I3
MMP16                   MMP16
SRSF3.1                 SRSF3
FBXL22.1               FBXL22
ANKRD29.1             ANKRD29
GOLGA8A               GOLGA8A
IFT88                   IFT88
AL356599.1         AL356599.1
C8orf88.1             C8orf88
ALCAM.1                 ALCAM
MICA.1                   MICA
AC078850.1.1       AC078850.1
RRM1                     RRM1
GK                         GK
DKK1.1                   DKK1
MYOZ1.1                 MYOZ1
ASB13.1                 ASB13
FRMD4A.1               FRMD4A
SPATA17               SPATA17
AC073389.1.1       AC073389.1
PPP1R3B               PPP1R3B
MOB3B.1                 MOB3B
MMP24.1                 MMP24
SSPN.1                   SSPN
ZNF692                 ZNF692
VEGFC.1                 VEGFC
GFPT1.1                 GFPT1
TYW5                     TYW5
MSRA.1                   MSRA
PKD1                     PKD1
FBLN1.1                 FBLN1
RNF128.1               RNF128
PTGFR.1                 PTGFR
PLCE1.1                 PLCE1
BICRA.1                 BICRA
C3orf52               C3orf52
RHPN2                   RHPN2
ZBTB7B.1               ZBTB7B
ATP10A                 ATP10A
KCTD5                   KCTD5
OR8S1.1                 OR8S1
NR2F1-AS1.1         NR2F1-AS1
PER2.1                   PER2
PTTG1                   PTTG1
INA.1                     INA
ZNF185                 ZNF185
PNPLA2.1               PNPLA2
GGACT                   GGACT
ZSWIM9                 ZSWIM9
AL118516.1.1       AL118516.1
GPAM.1                   GPAM
SNTA1.1                 SNTA1
CCDC81.1               CCDC81
LINC00545.1         LINC00545
PLD5.1                   PLD5
PCAT1.1                 PCAT1
MARCH3                 MARCH3
SLC22A2.1             SLC22A2
ST6GAL2.1             ST6GAL2
FMOD.1                   FMOD
HES1.1                   HES1
GALNT16               GALNT16
PPOX                     PPOX
GINS2                   GINS2
C3orf70               C3orf70
SLC40A1.1             SLC40A1
ZNF669                 ZNF669
BX255923.1.1       BX255923.1
SGK494.1               SGK494
NFYA.1                   NFYA
SSC5D.1                 SSC5D
FHL3                     FHL3
SORT1.1                 SORT1
AP1G2                   AP1G2
HOXC9                   HOXC9
CHST6                   CHST6
TAF1A                   TAF1A
CBL                       CBL
MAP1LC3B             MAP1LC3B
FAM19A2.1             FAM19A2
GAMT                     GAMT
ZGLP1.1                 ZGLP1
RBM3                     RBM3
ADD3                     ADD3
AC110285.6.1       AC110285.6
TOB1.1                   TOB1
CSNK1G1.1             CSNK1G1
NRTN.1                   NRTN
SNHG8                   SNHG8
HIST1H2BN           HIST1H2BN
JPH1.1                   JPH1
G6PD.1                   G6PD
RGS3                     RGS3
F2RL1                   F2RL1
PGM5                     PGM5
CLUAP1                 CLUAP1
HSPA1B                 HSPA1B
PTAR1                   PTAR1
AC062029.1.1       AC062029.1
PLEKHH2.1             PLEKHH2
TMEM64                 TMEM64
UAP1.1                   UAP1
FZD8                     FZD8
MPP5.1                   MPP5
AGPAT4.1               AGPAT4
ABLIM1.1               ABLIM1
RAB3IP.1               RAB3IP
RALGPS2.1             RALGPS2
EGR2.1                   EGR2
AC116667.1.1       AC116667.1
FDFT1.1                 FDFT1
TGIF2.1                 TGIF2
DDX17.1                 DDX17
CD1D                     CD1D
PBXIP1                 PBXIP1
H2AFZ                   H2AFZ
ERMARD                 ERMARD
PEX3.1                   PEX3
CDC14B                 CDC14B
OGA                       OGA
LSAMP.1                 LSAMP
GPRC5A.1               GPRC5A
LINC01082.1         LINC01082
S100A11               S100A11
LPP                       LPP
SHKBP1.1               SHKBP1
LTBP1.1                 LTBP1
LINC01311.1         LINC01311
LRRC37A3.1           LRRC37A3
AL050341.2.1       AL050341.2
ANKRD37.1             ANKRD37
ZNF574.1               ZNF574
MARC2                   MARC2
GASAL1.1               GASAL1
PPIC                     PPIC
TMEM45A               TMEM45A
AL365203.2         AL365203.2
HIST1H2AC           HIST1H2AC
AC009159.3.1       AC009159.3
ANGPTL2               ANGPTL2
JUNB.1                   JUNB
PCDH18.1               PCDH18
NEDD4L                 NEDD4L
GSEC                     GSEC
UBP1                     UBP1
BTN3A2.1               BTN3A2
IL17RE.1               IL17RE
LINC01759.1         LINC01759
HOXB-AS1.1           HOXB-AS1
MAD2L1                 MAD2L1
SMAGP                   SMAGP
KIFC2.1                 KIFC2
AL163051.1.1       AL163051.1
ZNF107                 ZNF107
ARSI.1                   ARSI
PCDHA10.1             PCDHA10
NUDT16                 NUDT16
MDM2.1                   MDM2
IER2                     IER2
NNMT                     NNMT
ZNF117.1               ZNF117
KIAA1755.1           KIAA1755
GSTP1.1                 GSTP1
NKTR.1                   NKTR
KIF9-AS1.1           KIF9-AS1
USP12                   USP12
ELL2.1                   ELL2
SLIT3                   SLIT3
PTPRF                   PTPRF
PCMTD2.1               PCMTD2
NDUFA12               NDUFA12
CDC42EP4.1           CDC42EP4
PSPH.1                   PSPH
EPHX2.1                 EPHX2
OLFML2B.1             OLFML2B
ITGA3                   ITGA3
CLEC3A.1               CLEC3A
AC073323.1.1       AC073323.1
NSFL1C.1               NSFL1C
ANKAR.1                 ANKAR
LPAR2                   LPAR2
XPR1                     XPR1
AGAP9.1                 AGAP9
AL136084.3.1       AL136084.3
TPD52.1                 TPD52
AC109322.1         AC109322.1
LINC02482.1         LINC02482
PCM1.1                   PCM1
CD248.1                 CD248
PLAUR.1                 PLAUR
CNIH2                   CNIH2
UBE2L6.1               UBE2L6
SLC2A5.1               SLC2A5
MAPK8IP3.1           MAPK8IP3
AC145285.6         AC145285.6
ATRNL1.1               ATRNL1
HIPK2.1                 HIPK2
TGFBR3.1               TGFBR3
ALG10B                 ALG10B
MTCH1                   MTCH1
OLR1.1                   OLR1
LAMA5                   LAMA5
FAAH.1                   FAAH
FBXW8                   FBXW8
C16orf86.1           C16orf86
TRIM47.1               TRIM47
AC137767.1.1       AC137767.1
TGFB2.1                 TGFB2
ETV5.1                   ETV5
GBE1.1                   GBE1
EZH2.1                   EZH2
FHOD3.1                 FHOD3
SLC1A3.1               SLC1A3
ARAP2.1                 ARAP2
PCNA.1                   PCNA
CPNE2.1                 CPNE2
ID2.1                     ID2
KIAA0895L.1         KIAA0895L
CEP350.1               CEP350
ZCWPW1.1               ZCWPW1
BTN3A3                 BTN3A3
RAB40C.1               RAB40C
ANKRD36C             ANKRD36C
LIMK2.1                 LIMK2
NLRP1.1                 NLRP1
LAMA1                   LAMA1
RPS4Y1.1               RPS4Y1
TRABD2B.1             TRABD2B
TMEM56.1               TMEM56
AC073611.1.1       AC073611.1
H3F3B.1                 H3F3B
AP1G1.1                 AP1G1
SMCO4.1                 SMCO4
TTC7A.1                 TTC7A
GBP2.1                   GBP2
AC147067.1         AC147067.1
SESN1.1                 SESN1
CASP6.1                 CASP6
ANO8                     ANO8
HAPLN4.1               HAPLN4
ZNF628.1               ZNF628
SLC30A1               SLC30A1
HEG1                     HEG1
CNTRL.1                 CNTRL
DIP2C                   DIP2C
B4GALT5.1             B4GALT5
SEPHS2.1               SEPHS2
ZNF883.1               ZNF883
PPM1H.1                 PPM1H
RASGRF2.1             RASGRF2
CARD19                 CARD19
BMP6.1                   BMP6
WHAMM.1                 WHAMM
UBE2S.1                 UBE2S
ZNF839.1               ZNF839
ID4.1                     ID4
GINS1.1                 GINS1
EPB41L2.1             EPB41L2
FAM122A.1             FAM122A
SLC39A14.1           SLC39A14
ZSCAN18               ZSCAN18
LINC01252.1         LINC01252
VAMP5.1                 VAMP5
TGFBR2.1               TGFBR2
UNC5B                   UNC5B
AHR                       AHR
ADGRF4.1               ADGRF4
KLHL28.1               KLHL28
AL132656.2.1       AL132656.2
ADAM15                 ADAM15
RAMP1                   RAMP1
ASAP2.1                 ASAP2
OBSCN                   OBSCN
APOBEC3C.1           APOBEC3C
MRS2.1                   MRS2
DPH6.1                   DPH6
STARD10               STARD10
AL590426.2.1       AL590426.2
CENPQ                   CENPQ
NRP2                     NRP2
DNAJC14               DNAJC14
ASPN.1                   ASPN
ENTPD6                 ENTPD6
RASL11B.1             RASL11B
CNKSR3.1               CNKSR3
KCNJ2.1                 KCNJ2
LRCH4.1                 LRCH4
RAB7B.1                 RAB7B
GPSM1                   GPSM1
ZNF577                 ZNF577
PRUNE2.1               PRUNE2
FAF2                     FAF2
LINC01806           LINC01806
GNG2.1                   GNG2
CKS2.1                   CKS2
NSRP1.1                 NSRP1
PLXDC1.1               PLXDC1
SOD1                     SOD1
LINC01705.1         LINC01705
ADCY7.1                 ADCY7
NASP.1                   NASP
FAM43A.1               FAM43A
PPP1R13L             PPP1R13L
NUDCD1.1               NUDCD1
PSMA3                   PSMA3
THAP7-AS1.1         THAP7-AS1
MYL7.1                   MYL7
TC2N                     TC2N
NMB                       NMB
SLC6A17               SLC6A17
GPR87.1                 GPR87
PNP.1                     PNP
NDUFB2-AS1.1       NDUFB2-AS1
RPAP1                   RPAP1
IGF2-AS.1             IGF2-AS
SLC6A8.1               SLC6A8
ZCCHC14.1             ZCCHC14
CEP131                 CEP131
MID1IP1.1             MID1IP1
COMT.1                   COMT
XPO6                     XPO6
PIGU.1                   PIGU
FBXO4.1                 FBXO4
FBLN5.1                 FBLN5
LAMB2.1                 LAMB2
TADA1                   TADA1
TMEM200A.1           TMEM200A
GRIP2.1                 GRIP2
SULF1.1                 SULF1
DOCK1.1                 DOCK1
RNASEH2C.1           RNASEH2C
MST1                     MST1
GREB1.1                 GREB1
ZFAND2A               ZFAND2A
GALM                     GALM
MVK.1                     MVK
RAD50.1                 RAD50
LOXL3                   LOXL3
AP000487.1.1       AP000487.1
PDCL3.1                 PDCL3
F2R                       F2R
CSRNP3.1               CSRNP3
FBXO16                 FBXO16
BLMH.1                   BLMH
NTRK2.1                 NTRK2
ASZ1                     ASZ1
MCAM.1                   MCAM
ARL6IP1.1             ARL6IP1
RNASET2.1             RNASET2
TNFSF4.1               TNFSF4
TTC21A.1               TTC21A
RASAL2                 RASAL2
ZNF639.1               ZNF639
SDC4.1                   SDC4
CXADR.1                 CXADR
TIFA.1                   TIFA
NKX2-5.1               NKX2-5
SNX18                   SNX18
RBMX.1                   RBMX
ULBP2.1                 ULBP2
TMC6.1                   TMC6
IDE                       IDE
MIR193BHG           MIR193BHG
KANSL1L.1             KANSL1L
AC007938.3.1       AC007938.3
MCTP1.1                 MCTP1
ABL2.1                   ABL2
SVIL.1                   SVIL
ZNF850.1               ZNF850
DOK1                     DOK1
PPP1R15A             PPP1R15A
ETHE1.1                 ETHE1
ANGPTL7.1             ANGPTL7
EFCAB7                 EFCAB7
TRPM7.1                 TRPM7
TNFSF13B.1           TNFSF13B
AP002387.2.1       AP002387.2
TP53TG1               TP53TG1
MLPH.1                   MLPH
AC012645.1.1       AC012645.1
DPP10-AS1.1         DPP10-AS1
TRIM58.1               TRIM58
ATP6V0E2.1           ATP6V0E2
LIMS2.1                 LIMS2
GULP1.1                 GULP1
WFS1                     WFS1
HOXB7                   HOXB7
ITGA4.1                 ITGA4
COQ8A.1                 COQ8A
APOD.1                   APOD
IFI35                   IFI35
NEO1                     NEO1
STARD4-AS1.1       STARD4-AS1
TPP2.1                   TPP2
TBC1D1                 TBC1D1
CALY.1                   CALY
CAVIN4.1               CAVIN4
NMI                       NMI
ZNF331                 ZNF331
TIGD1.1                 TIGD1
DTNB                     DTNB
WFDC3.1                 WFDC3
GPX7.1                   GPX7
ANKRD44               ANKRD44
AC069185.1.1       AC069185.1
TUBB2B                 TUBB2B
FKRP.1                   FKRP
PPM1F.1                 PPM1F
ITGB1BP2.1           ITGB1BP2
PTPRK                   PTPRK
SZT2.1                   SZT2
EHD3.1                   EHD3
TSPAN18.1             TSPAN18
AC010255.1.1       AC010255.1
AC097478.1.1       AC097478.1
SMCR8.1                 SMCR8
TSPAN12.1             TSPAN12
PLD3.1                   PLD3
AP000866.2.1       AP000866.2
AL138724.1         AL138724.1
YJU2                     YJU2
TOR1B                   TOR1B
AFAP1L2.1             AFAP1L2
AL133338.1.1       AL133338.1
MAPK12                 MAPK12
NEK8.1                   NEK8
ITGA2                   ITGA2
AL731569.1.1       AL731569.1
FAM196A               FAM196A
RWDD3                   RWDD3
LAMA2                   LAMA2
DNAJB4                 DNAJB4
LIG3                     LIG3
CHST3.1                 CHST3
PANX2                   PANX2
HDAC4.1                 HDAC4
MET.1                     MET
DOHH.1                   DOHH
ENPP2.1                 ENPP2
WDR4.1                   WDR4
IPMK.1                   IPMK
C6orf62.1             C6orf62
IGFL2.1                 IGFL2
ARG2.1                   ARG2
RBM47.1                 RBM47
KPNA2.1                 KPNA2
SLC48A1               SLC48A1
NSG1.1                   NSG1
AC112220.3.1       AC112220.3
PEMT                     PEMT
URB2.1                   URB2
CAMK4.1                 CAMK4
ANKRD33B             ANKRD33B
APOBEC3F             APOBEC3F
MTHFD2.1               MTHFD2
HELLS.1                 HELLS
C2orf49.1             C2orf49
TDO2                     TDO2
NOTCH3.1               NOTCH3
EPS8L2.1               EPS8L2
PTPRN                   PTPRN
SFRP1.1                 SFRP1
SNX14                   SNX14
PRKG1.1                 PRKG1
TWNK                     TWNK
DMKN.1                   DMKN
HSPA8                   HSPA8
FAM107B.1             FAM107B
SLMAP                   SLMAP
LAMC2.1                 LAMC2
ARHGEF3.1             ARHGEF3
MYO18A                 MYO18A
ATXN1L                 ATXN1L
PPP5D1.1               PPP5D1
KDM1B                   KDM1B
TMSB10                 TMSB10
GALNT12               GALNT12
KIAA1328.1           KIAA1328
SLC22A23             SLC22A23
DYNC1I1.1             DYNC1I1
ZNF703                 ZNF703
AHNAK2.1               AHNAK2
CCNB2                   CCNB2
FLYWCH1.1             FLYWCH1
MOB1B                   MOB1B
SSTR1.1                 SSTR1
TNFSF15.1             TNFSF15
ATAD2B.1               ATAD2B
JAZF1                   JAZF1
TAS2R14.1             TAS2R14
DPF2                     DPF2
SRSF4                   SRSF4
IDH3B.1                 IDH3B
ATF3.1                   ATF3
LONRF1.1               LONRF1
LEF1.1                   LEF1
WDSUB1                 WDSUB1
ABHD15                 ABHD15
PLK2.1                   PLK2
AC124283.1.1       AC124283.1
PKN1.1                   PKN1
TACC2.1                 TACC2
PM20D2                 PM20D2
GPNMB.1                 GPNMB
SAMD4A                 SAMD4A
HOXC6.1                 HOXC6
ATP6V0A1             ATP6V0A1
DAAM2.1                 DAAM2
NTF3.1                   NTF3
SYNGR1.1               SYNGR1
PITHD1.1               PITHD1
PHF13                   PHF13
GCLM.1                   GCLM
ACSS2                   ACSS2
LINC01638.1         LINC01638
SETD4                   SETD4
FANCB.1                 FANCB
SHMT2.1                 SHMT2
FIG4.1                   FIG4
KIAA1217.1           KIAA1217
PLCB4.1                 PLCB4
FANCF.1                 FANCF
MMP7                     MMP7
EFNA5                   EFNA5
NXPH4.1                 NXPH4
MYO9B.1                 MYO9B
MTMR3                   MTMR3
FAM69B.1               FAM69B
TINAGL1               TINAGL1
ST3GAL6.1             ST3GAL6
CYP26A1.1             CYP26A1
AC021242.3.1       AC021242.3
TAOK1                   TAOK1
EPHA2                   EPHA2
VPS33B.1               VPS33B
AL358472.4         AL358472.4
LINC01943           LINC01943
ARHGEF2               ARHGEF2
TCTA                     TCTA
RASSF8.1               RASSF8
SGF29                   SGF29
SOD3                     SOD3
PCSK6.1                 PCSK6
CALM3                   CALM3
TVP23B                 TVP23B
JAG1.1                   JAG1
RPL39L                 RPL39L
PLXDC2                 PLXDC2
GCHFR                   GCHFR
IL32                     IL32
NOG.1                     NOG
SNX30                   SNX30
OLFML2A               OLFML2A
C21orf2.1             C21orf2
MT-ND4.1               MT-ND4
MYL10.1                 MYL10
SYN1.1                   SYN1
IFNAR2.1               IFNAR2
AC112721.2.1       AC112721.2
CCNDBP1               CCNDBP1
SLC4A1AP.1           SLC4A1AP
PANK1.1                 PANK1
BCL10.1                 BCL10
NKRF.1                   NKRF
NPIPA1                 NPIPA1
SPOCD1.1               SPOCD1
AC007878.1.1       AC007878.1
SSPO                     SSPO
DHRS2.1                 DHRS2
C1QTNF3.1             C1QTNF3
AQP3.1                   AQP3
INTS6.1                 INTS6
CNRIP1.1               CNRIP1
PROSER2.1             PROSER2
RTKN2.1                 RTKN2
GFOD1.1                 GFOD1
HOXA3                   HOXA3
WFDC1.1                 WFDC1
GGH.1                     GGH
LPXN                     LPXN
PLN.1                     PLN
PDZRN4.1               PDZRN4
FAM160B2             FAM160B2
CDYL2.1                 CDYL2
MRPL35.1               MRPL35
PFDN1                   PFDN1
PTGFRN.1               PTGFRN
AAED1.1                 AAED1
ESD                       ESD
LRRC45.1               LRRC45
RFC3                     RFC3
CMBL                     CMBL
FAM227A.1             FAM227A
DSP                       DSP
CYBRD1                 CYBRD1
SPARC.2                 SPARC
COL5A1.2               COL5A1
FSTL1.2                 FSTL1
FN1.2                     FN1
COL3A1.2               COL3A1
COL5A2.1               COL5A2
RPL10.2                 RPL10
RPL7A                   RPL7A
RPS4X.1                 RPS4X
COL4A1.1               COL4A1
FBN1.1                   FBN1
CANX.1                   CANX
RPS3A                   RPS3A
COL1A1.2               COL1A1
COL11A1.2             COL11A1
RPS8                     RPS8
FTL.1                     FTL
RPL10A                 RPL10A
RPN1.2                   RPN1
RPS27.2                 RPS27
LRP1.2                   LRP1
SURF4.1                 SURF4
COL18A1.2             COL18A1
FTH1.2                   FTH1
GLS.1                     GLS
TIMP2.2                 TIMP2
FBLN5.2                 FBLN5
SH2B2                   SH2B2
IL6ST.1                 IL6ST
MT-ND3                 MT-ND3
AC133550.2         AC133550.2
RPS3                     RPS3
PLOD2.1                 PLOD2
RPL3                     RPL3
IVNS1ABP.1           IVNS1ABP
RPL23.1                 RPL23
MRC2.1                   MRC2
PLXNA1.1               PLXNA1
ITGA11.1               ITGA11
LOX.1                     LOX
FLT1.2                   FLT1
AL358472.4.1       AL358472.4
MATN2.1                 MATN2
ITGAV.1                 ITGAV
CCND2.2                 CCND2
GPX4.2                   GPX4
LAMC1.1                 LAMC1
RPL18A                 RPL18A
CKAP4.1                 CKAP4
GALNT10.2             GALNT10
TPT1.2                   TPT1
RPL13A                 RPL13A
ATP6V0E1             ATP6V0E1
C1QL1.2                 C1QL1
FGFR2.1                 FGFR2
YAP1.1                   YAP1
SEC16A.1               SEC16A
VSIG10L.2             VSIG10L
GALNT2.1               GALNT2
HACD2.1                 HACD2
CTSK.2                   CTSK
UBOX5.1                 UBOX5
SLC38A10.1           SLC38A10
GPC4.2                   GPC4
EIF1.2                   EIF1
MTBP                     MTBP
COL6A3.1               COL6A3
TNFRSF19             TNFRSF19
CRIM1.1                 CRIM1
FLNA.1                   FLNA
GALNT5.2               GALNT5
PLD3.2                   PLD3
COL1A2.1               COL1A2
FBN2.1                   FBN2
BCAT1.1                 BCAT1
ISCU.2                   ISCU
GFPT1.2                 GFPT1
AL731563.3.1       AL731563.3
HSF4                     HSF4
EDIL3.1                 EDIL3
BRI3.2                   BRI3
ADAM10.2               ADAM10
RPS23                   RPS23
XYLT1.1                 XYLT1
WDR26.1                 WDR26
ADGRA2.1               ADGRA2
PRSS35.2               PRSS35
VCAN                     VCAN
MAP3K7.1               MAP3K7
P4HB.1                   P4HB
SERPINB2             SERPINB2
PPFIBP2               PPFIBP2
SCAMP5                 SCAMP5
PRDX2.2                 PRDX2
IRAK4                   IRAK4
ZNF473.1               ZNF473
GTF2A2.1               GTF2A2
DSTN                     DSTN
ALG10B.1               ALG10B
SLC39A6.1             SLC39A6
FAM227A.2             FAM227A
APP.1                     APP
PLOD1.1                 PLOD1
SFMBT2.2               SFMBT2
CDH11.1                 CDH11
SLC7A7                 SLC7A7
COL12A1.2             COL12A1
MAP3K4.1               MAP3K4
IQGAP1.1               IQGAP1
TIMP3.2                 TIMP3
GRIK2                   GRIK2
ZNF692.1               ZNF692
SERPINH1             SERPINH1
MAN2B1.1               MAN2B1
HDAC4.2                 HDAC4
C1RL.2                   C1RL
LMAN1                   LMAN1
EFCAB7.1               EFCAB7
SVEP1.1                 SVEP1
NRTN.2                   NRTN
TEAD1                   TEAD1
AL133215.2         AL133215.2
TTC21A.2               TTC21A
TMSB4X                 TMSB4X
TESK2.2                 TESK2
FANCF.2                 FANCF
SLC9A7.1               SLC9A7
ALCAM.2                 ALCAM
EFNB2.1                 EFNB2
ABCD3.2                 ABCD3
COL4A2.1               COL4A2
AXIN2                   AXIN2
PLEC.1                   PLEC
FLNC                     FLNC
TTYH3.1                 TTYH3
PER2.2                   PER2
AC098487.1.2       AC098487.1
PLS3.2                   PLS3
LEPR                     LEPR
EME2                     EME2
SMC6.1                   SMC6
AC020571.1.2       AC020571.1
RUNX2.1                 RUNX2
PTMA.1                   PTMA
FURIN                   FURIN
AC005332.6.1       AC005332.6
DEPP1.2                 DEPP1
KRT10.2                 KRT10
AL662884.4.1       AL662884.4
TRIM59.1               TRIM59
LRRC32.1               LRRC32
WNT3                     WNT3
CPM.1                     CPM
INTS6-AS1           INTS6-AS1
H2AFZ.1                 H2AFZ
EIF3E                   EIF3E
CHCHD2                 CHCHD2
ADAM23.2               ADAM23
AC010255.1.2       AC010255.1
UNKL.2                   UNKL
UBAP1L.1               UBAP1L
RNF128.2               RNF128
CLIC4.1                 CLIC4
ADNP-AS1.1           ADNP-AS1
GGACT.1                 GGACT
ADA                       ADA
CERCAM.1               CERCAM
MCC                       MCC
ACSL1.2                 ACSL1
LOXL3.1                 LOXL3
ITGB5.1                 ITGB5
NDRG1.1                 NDRG1
NDST1.1                 NDST1
SESN2.2                 SESN2
PCCB.1                   PCCB
LZTS1.2                 LZTS1
MT-CO1.2               MT-CO1
SEMA3C.1               SEMA3C
WNT5A.1                 WNT5A
FKBP10.2               FKBP10
CLCN5                   CLCN5
OSR1.2                   OSR1
INHBA.2                 INHBA
PIGQ.1                   PIGQ
DOK5.1                   DOK5
RDH10.2                 RDH10
AC015813.1         AC015813.1
CHIC2.1                 CHIC2
CEP131.1               CEP131
ZNF136.2               ZNF136
TTC3                     TTC3
FAM129B.1             FAM129B
ANKRD33B.1           ANKRD33B
HINT1.2                 HINT1
ACTC1.2                 ACTC1
ZNF496.2               ZNF496
LAMB1.1                 LAMB1
CDH2.1                   CDH2
LINC00626.2         LINC00626
H3F3B.2                 H3F3B
USP20.1                 USP20
PCNX4                   PCNX4
ECE1.1                   ECE1
ZFPM2.1                 ZFPM2
MT-ND4.2               MT-ND4
AL117379.1.1       AL117379.1
NUAK2                   NUAK2
PIK3R3                 PIK3R3
DIS3L                   DIS3L
PLXNB2.1               PLXNB2
NPC2                     NPC2
STIL                     STIL
ITGA1.2                 ITGA1
RAN                       RAN
OGA.1                     OGA
ARMCX6                 ARMCX6
LOXL1                   LOXL1
HSPA5                   HSPA5
CEMIP                   CEMIP
IGFBP3.2               IGFBP3
PPP1R13L.1           PPP1R13L
SLC2A1.1               SLC2A1
NEBL.1                   NEBL
ATP2A2.1               ATP2A2
PLA2G6                 PLA2G6
LRRN2                   LRRN2
NUAK1.2                 NUAK1
AGRN.1                   AGRN
CEMIP2.1               CEMIP2
KCND3.2                 KCND3
PDLIM2.1               PDLIM2
MMP14.1                 MMP14
ADAM12.1               ADAM12
PTPA.1                   PTPA
HEG1.1                   HEG1
KIRREL1.1             KIRREL1
C9orf16.2             C9orf16
KCTD9                   KCTD9
RING1.1                 RING1
FNDC3B.1               FNDC3B
NCKAP1.1               NCKAP1
FAM78A                 FAM78A
ZNF23.1                 ZNF23
MT-CO3                 MT-CO3
CSRP2.2                 CSRP2
AL683813.1         AL683813.1
SEMA5A.2               SEMA5A
SMAD9                   SMAD9
BSDC1.1                 BSDC1
EEF1G                   EEF1G
NAA25.1                 NAA25
MYL12B.2               MYL12B
AC067750.1.1       AC067750.1
FAM177A1.2           FAM177A1
TDO2.1                   TDO2
NORAD.2                 NORAD
EXTL1.2                 EXTL1
LTN1                     LTN1
LTBP2                   LTBP2
SLC38A2.1             SLC38A2
TMX4.1                   TMX4
PPP2R1B               PPP2R1B
ACAP3                   ACAP3
ZKSCAN7               ZKSCAN7
SLC4A2.1               SLC4A2
LENG8.2                 LENG8
CPNE8                   CPNE8
FAM84B.1               FAM84B
SMAD6.1                 SMAD6
CPD.2                     CPD
TMEM144               TMEM144
PTPN13.1               PTPN13
MIGA2.1                 MIGA2
AC002456.1         AC002456.1
RFC3.1                   RFC3
C6orf62.2             C6orf62
HNRNPU                 HNRNPU
PPM1F.2                 PPM1F
TUFT1.1                 TUFT1
SPTBN2.1               SPTBN2
MEST.1                   MEST
MBNL2.2                 MBNL2
SPPL2B.1               SPPL2B
ATAD2B.2               ATAD2B
CYBRD1.1               CYBRD1
OSER1-DT.2           OSER1-DT
TEX2.1                   TEX2
GSTP1.2                 GSTP1
DPP10.2                 DPP10
ADI1                     ADI1
FSCN1.1                 FSCN1
WDR4.2                   WDR4
TMSB10.1               TMSB10
PDZRN3                 PDZRN3
PTK7                     PTK7
ATP6V1B2             ATP6V1B2
RAD50.2                 RAD50
IL13RA1.1             IL13RA1
B3GNT2.1               B3GNT2
ATF4.2                   ATF4
E2F6.1                   E2F6
L1CAM                   L1CAM
MICAL1                 MICAL1
MAMDC2-AS1         MAMDC2-AS1
SYNPO.1                 SYNPO
MPRIP-AS1           MPRIP-AS1
Z93930.2.2           Z93930.2
ZBED3-AS1.1         ZBED3-AS1
RAB33A.1               RAB33A
MST1.1                   MST1
SERPINE2             SERPINE2
ENTPD1.1               ENTPD1
LNX2.1                   LNX2
ZNF254                 ZNF254
FAM196A.1             FAM196A
HAUS4.2                 HAUS4
ZMYND8                 ZMYND8
GRK5                     GRK5
CNIH2.1                 CNIH2
PAPSS2.1               PAPSS2
UBA6                     UBA6
AC137630.3         AC137630.3
ARFGAP1               ARFGAP1
SLC2A1-AS1.2       SLC2A1-AS1
GNL1.2                   GNL1
TMEM30A.1             TMEM30A
PTPRK.1                 PTPRK
GALNT13               GALNT13
MT-ND5.2               MT-ND5
TNFRSF25             TNFRSF25
FMO4.2                   FMO4
AC105760.2         AC105760.2
AHR.1                     AHR
AC009133.1         AC009133.1
ALDH4A1.1             ALDH4A1
MT-CYB.2               MT-CYB
AL731569.1.2       AL731569.1
ADAMTS9.2             ADAMTS9
FAM234B               FAM234B
ENPP2.2                 ENPP2
TBCK.1                   TBCK
DDX17.2                 DDX17
TBC1D9.1               TBC1D9
SLC2A10               SLC2A10
MXRA5.2                 MXRA5
PLPPR3.2               PLPPR3
GOLIM4.1               GOLIM4
ABCD1.1                 ABCD1
JAM2                     JAM2
CSPG4.1                 CSPG4
CYFIP1.1               CYFIP1
PSMA7                   PSMA7
KIF9.1                   KIF9
MMP16.1                 MMP16
KIAA0930             KIAA0930
RAB40A.2               RAB40A
HEPH.1                   HEPH
GREB1.2                 GREB1
NCSTN.1                 NCSTN
FLVCR2                 FLVCR2
C19orf18             C19orf18
SEMA7A.1               SEMA7A
SH3D19.1               SH3D19
PCNX1.1                 PCNX1
HSP90B1.1             HSP90B1
EIF4EBP1.2           EIF4EBP1
AC005476.2.2       AC005476.2
CA2                       CA2
CDIP1                   CDIP1
ACOT7.1                 ACOT7
TM6SF2.1               TM6SF2
LINC00960           LINC00960
LYSMD2.2               LYSMD2
GAS7.1                   GAS7
HMGN3-AS1.1         HMGN3-AS1
XPR1.1                   XPR1
EVA1A.1                 EVA1A
KCNK6.1                 KCNK6
AC007114.1.1       AC007114.1
RAB15.1                 RAB15
SULF2.1                 SULF2
HNRNPUL2.1           HNRNPUL2
WDR81.1                 WDR81
TRIM65.2               TRIM65
GNG5                     GNG5
SLC35G1               SLC35G1
ARAP3                   ARAP3
ABAT.1                   ABAT
CDK6.1                   CDK6
ARSJ.1                   ARSJ
TBC1D10A             TBC1D10A
HELLS.2                 HELLS
ARSI.2                   ARSI
TMCO4.1                 TMCO4
IGF2R                   IGF2R
LDHB.1                   LDHB
CCDC69.2               CCDC69
BEX1.2                   BEX1
SPTBN1.2               SPTBN1
PTPRF.1                 PTPRF
ACVR2A.1               ACVR2A
DLG1.1                   DLG1
AC242426.2.2       AC242426.2
MT-ND2                 MT-ND2
TUBGCP6               TUBGCP6
PAPSS1.1               PAPSS1
AC116667.1.2       AC116667.1
C8orf49               C8orf49
PLXNA2.1               PLXNA2
SEMA4F.2               SEMA4F
QSOX1                   QSOX1
LRRC49.2               LRRC49
ATP2C1                 ATP2C1
ANOS1.2                 ANOS1
AL049597.2         AL049597.2
CLTA.1                   CLTA
XPO1.1                   XPO1
ADCK1                   ADCK1
ECM2                     ECM2
DTNB.1                   DTNB
CPA4.2                   CPA4
ZNF703.1               ZNF703
STARD9                 STARD9
DDB2                     DDB2
ACTN1.2                 ACTN1
LIG1.2                   LIG1
UNC5B.1                 UNC5B
CYB5D1.1               CYB5D1
ADAMTSL4             ADAMTSL4
SEC61B                 SEC61B
ZNF510                 ZNF510
PSMD8                   PSMD8
ETV5.2                   ETV5
ZNF486.1               ZNF486
MGAT2                   MGAT2
CCDC71L               CCDC71L
AL035563.1         AL035563.1
ODAPH.2                 ODAPH
HSPBAP1               HSPBAP1
CHSY3                   CHSY3
ZNF610.1               ZNF610
NIN                       NIN
CTSD.1                   CTSD
SLC39A14.2           SLC39A14
EIF5A.1                 EIF5A
COL6A1.1               COL6A1
GLP2R.1                 GLP2R
PCDH10                 PCDH10
CHPF.1                   CHPF
CYCS.2                   CYCS
TMPO.1                   TMPO
TAGLN.2                 TAGLN
HMCN1.2                 HMCN1
PKM.1                     PKM
FADS2                   FADS2
PLXDC1.2               PLXDC1
NID2.1                   NID2
DNAJC6                 DNAJC6
CORO2B.2               CORO2B
RNF6.1                   RNF6
SPHK1                   SPHK1
EEF1A1.1               EEF1A1
RNF32                   RNF32
SART1.1                 SART1
DYNLL1.2               DYNLL1
C5orf30.2             C5orf30
MYO10.2                 MYO10
COL8A1                 COL8A1
IFITM1.2               IFITM1
DTX4                     DTX4
COL6A2.2               COL6A2
ATXN1.1                 ATXN1
CCDC71                 CCDC71
CLEC2D.1               CLEC2D
NME4.2                   NME4
PRDX1                   PRDX1
ATP6V0A2.1           ATP6V0A2
LINC01806.1         LINC01806
SRGAP2                 SRGAP2
INAFM2.1               INAFM2
OGFRL1.1               OGFRL1
ANAPC16.2             ANAPC16
SEC11C                 SEC11C
PLCE1-AS1           PLCE1-AS1
LINC01004           LINC01004
FOSB                     FOSB
SLC25A21-AS1     SLC25A21-AS1
AC132872.1.2       AC132872.1
ZNF76                   ZNF76
ATP13A3.1             ATP13A3
SLC39A10.1           SLC39A10
LRRC2.2                 LRRC2
FLRT2.1                 FLRT2
ACTBL2.1               ACTBL2
PAM.1                     PAM
ALDH1B1.2             ALDH1B1
AMOTL2.1               AMOTL2
AL669970.3.1       AL669970.3
AC006213.1         AC006213.1
IFNAR1                 IFNAR1
ALDH2.1                 ALDH2
TRAM2                   TRAM2
AL162258.2         AL162258.2
LPAR2.1                 LPAR2
FMOD.2                   FMOD
MBLAC1.2               MBLAC1
NOTCH2                 NOTCH2
ARRDC4                 ARRDC4
RAB2B                   RAB2B
RNF150.2               RNF150
SOD3.1                   SOD3
MAST3                   MAST3
KIF26B.1               KIF26B
B2M.2                     B2M
MYL12A.2               MYL12A
SPRY2.2                 SPRY2
LDLRAD2               LDLRAD2
CMTM6                   CMTM6
TRIM22.2               TRIM22
ZNF799                 ZNF799
IFFO1.1                 IFFO1
DBI.1                     DBI
HEATR5A.1             HEATR5A
GALNT7.1               GALNT7
PKP2.2                   PKP2
AC110285.6.2       AC110285.6
GJA1.1                   GJA1
ZNF438.1               ZNF438
NISCH                   NISCH
WIPF1.1                 WIPF1
C6orf48.2             C6orf48
JCAD                     JCAD
ZNF107.1               ZNF107
SLC7A5.1               SLC7A5
SLC7A1                 SLC7A1
RAD51AP1.2           RAD51AP1
S100A10.2             S100A10
DLC1.1                   DLC1
AQP10                   AQP10
TCAF1.1                 TCAF1
IRF7                     IRF7
MICB                     MICB
TUBA4A.2               TUBA4A
NR1H3.1                 NR1H3
RAD9A.1                 RAD9A
SGCE                     SGCE
TUBB6.2                 TUBB6
UBALD2                 UBALD2
RBM47.2                 RBM47
ANGPTL7.2             ANGPTL7
UHMK1.2                 UHMK1
LOXL2.1                 LOXL2
PDE4B.1                 PDE4B
LPP.1                     LPP
GSDME.1                 GSDME
PROSER1               PROSER1
ARRDC3                 ARRDC3
TSPAN2.1               TSPAN2
CACNA2D1.1           CACNA2D1
PGRMC2                 PGRMC2
PPM1E                   PPM1E
TALDO1.2               TALDO1
LBR                       LBR
SNX30.1                 SNX30
GOLGB1                 GOLGB1
CALM2                   CALM2
ARHGAP23             ARHGAP23
IGFBP5.2               IGFBP5
AC025171.4.2       AC025171.4
GNG2.2                   GNG2
SRPX2.2                 SRPX2
CPNE2.2                 CPNE2
STAT2                   STAT2
JADE3                   JADE3
DPP10-AS1.2         DPP10-AS1
NFKBIZ.1               NFKBIZ
ZNF10.1                 ZNF10
MOB1B.1                 MOB1B
TACC2.2                 TACC2
FEM1C                   FEM1C
CEP85                   CEP85
FCMR                     FCMR
TMEM87B.1             TMEM87B
LINC01355.2         LINC01355
COL8A2.1               COL8A2
ENDOD1.2               ENDOD1
AQP1.2                   AQP1
COL15A1.1             COL15A1
ANXA2.2                 ANXA2
SLIT3.1                 SLIT3
EDEM3                   EDEM3
AC018647.1.2       AC018647.1
CBX4.1                   CBX4
ST3GAL6.2             ST3GAL6
SPOCK1.2               SPOCK1
ARHGAP5               ARHGAP5
SLC9A8.1               SLC9A8
ZNF691                 ZNF691
DUSP19                 DUSP19
NDUFA12.1             NDUFA12
SBF2                     SBF2
CPNE7.2                 CPNE7
MET.2                     MET
SLC37A3.2             SLC37A3
SPON1                   SPON1
FAM69B.2               FAM69B
SOX6.1                   SOX6
TAF15                   TAF15
MRPS6.1                 MRPS6
KLHL29                 KLHL29
EVA1B                   EVA1B
CNTRL.2                 CNTRL
AC092171.3.1       AC092171.3
CDV3                     CDV3
ANKRD6.2               ANKRD6
CHST3.2                 CHST3
SGPP1.1                 SGPP1
ADAM15.1               ADAM15
REPS1                   REPS1
TXN.1                     TXN
ARHGAP12             ARHGAP12
ZNF573.2               ZNF573
MAD2L1.1               MAD2L1
EPAS1                   EPAS1
CUL2                     CUL2
S100A11.1             S100A11
GUCD1                   GUCD1
JAZF1.1                 JAZF1
SACS.1                   SACS
POLE4.2                 POLE4
MESP1                   MESP1
LINC02482.2         LINC02482
SELENBP1.2           SELENBP1
TRIM21                 TRIM21
PCDHA4.1               PCDHA4
SLC20A1               SLC20A1
TMEM241.1             TMEM241
JMY                       JMY
ATP6V1G2             ATP6V1G2
ANKH.1                   ANKH
ID4.2                     ID4
SCRG1.2                 SCRG1
FBLN1.2                 FBLN1
C16orf86.2           C16orf86
FLYWCH1.2             FLYWCH1
LY6G5C.2               LY6G5C
CBL.1                     CBL
AGMAT                   AGMAT
LRIG3.1                 LRIG3
KIAA0232.1           KIAA0232
TMEM190.2             TMEM190
SMAD1                   SMAD1
MMP2.2                   MMP2
GDNF-AS1             GDNF-AS1
APLP2                   APLP2
AL078639.1.1       AL078639.1
KCTD12.1               KCTD12
MAP4K3                 MAP4K3
NCOA7                   NCOA7
ATP6AP1L.2           ATP6AP1L
NCLN.1                   NCLN
C3orf52.1             C3orf52
KCNS3.2                 KCNS3
SCOC-AS1             SCOC-AS1
CLDN12.2               CLDN12
LASP1.1                 LASP1
ASB13.2                 ASB13
LRRC37A3.2           LRRC37A3
AMOTL1                 AMOTL1
LGR4.1                   LGR4
SERINC5.1             SERINC5
AL162231.2         AL162231.2
TPM4.2                   TPM4
LAMP2                   LAMP2
ARHGAP20.1           ARHGAP20
METTL7A.1             METTL7A
LGMN                     LGMN
SLC38A4.1             SLC38A4
HSPB1.2                 HSPB1
GLIPR1.2               GLIPR1
DYNLT3.1               DYNLT3
TBC1D1.1               TBC1D1
LAMA1.1                 LAMA1
PCOLCE                 PCOLCE
ENTPD4.1               ENTPD4
BAX                       BAX
LIF.2                     LIF
ZSCAN5A               ZSCAN5A
ETHE1.2                 ETHE1
SSC5D.2                 SSC5D
PXDN                     PXDN
AEN                       AEN
FGD6.2                   FGD6
TRIL.1                   TRIL
ALDH3B1.1             ALDH3B1
FRMD6.1                 FRMD6
MDH1.1                   MDH1
INMT.2                   INMT
VKORC1.1               VKORC1
ABTB2.1                 ABTB2
HOXB4.1                 HOXB4
LGALS3BP             LGALS3BP
PPIC.1                   PPIC
TACC3                   TACC3
MAP3K9                 MAP3K9
AL121827.2.2       AL121827.2
KCNC4.2                 KCNC4
FRY                       FRY
LGALS1.2               LGALS1
LAMB2.2                 LAMB2
IBTK                     IBTK
HLA-C.2                 HLA-C
AC067838.1.1       AC067838.1
CHD1L.1                 CHD1L
NCOA4                   NCOA4
CNKSR3.2               CNKSR3
C2orf49.2             C2orf49
CCDC138.2             CCDC138
STXBP6                 STXBP6
ECM1.2                   ECM1
CD59.1                   CD59
SELENOW.2             SELENOW
GTF2E1.2               GTF2E1
PDLIM3.2               PDLIM3
IRF2BP2.1             IRF2BP2
GSN                       GSN
SLC25A25             SLC25A25
NUP133.1               NUP133
MAP1B.1                 MAP1B
STK35.1                 STK35
LCOR.1                   LCOR
TDG                       TDG
TP53INP1.1           TP53INP1
FAM234A               FAM234A
HAS2.2                   HAS2
ITGA2.1                 ITGA2
ATL3.1                   ATL3
CNRIP1.2               CNRIP1
PTPN11.2               PTPN11
SLC4A7.1               SLC4A7
PCMTD2.2               PCMTD2
MACF1.1                 MACF1
CLIP4                   CLIP4
SSTR1.2                 SSTR1
WNT5B.2                 WNT5B
WDFY3                   WDFY3
LEF1.2                   LEF1
FRA10AC1             FRA10AC1
SH3RF3                 SH3RF3
GRIP2.2                 GRIP2
C12orf75.2           C12orf75
TMEM86B.1             TMEM86B
STRIP1.1               STRIP1
TMEM141.2             TMEM141
FAM168A               FAM168A
IL17B                   IL17B
NUBP1                   NUBP1
KIAA1024.2           KIAA1024
TUBB4B.2               TUBB4B
PAMR1.1                 PAMR1
FAM96B.2               FAM96B
THBD.2                   THBD
NPIPA1.1               NPIPA1
CDKL5.2                 CDKL5
GPSM1.1                 GPSM1
LIX1.1                   LIX1
GALNT12.1             GALNT12
NDUFB6                 NDUFB6
DPYSL3                 DPYSL3
CD82.1                   CD82
MON2.2                   MON2
PTAR1.1                 PTAR1
ERCC3                   ERCC3
PIAS4                   PIAS4
MIR4435-2HG.2     MIR4435-2HG
SP4.1                     SP4
P4HA3.1                 P4HA3
PLXNA4.1               PLXNA4
SULF1.2                 SULF1
ADRA2A.1               ADRA2A
AL590426.2.2       AL590426.2
TESC                     TESC
TNKS.2                   TNKS
CTGF.2                   CTGF
FGF1.2                   FGF1
BLM.2                     BLM
CSNK1E.1               CSNK1E
NDUFAF8               NDUFAF8
TNC.1                     TNC
ISLR                     ISLR
ADAMTS10             ADAMTS10
ARPC5.1                 ARPC5
MYOF.1                   MYOF
KLHDC10.2             KLHDC10
EPG5.1                   EPG5
GPRC5B.1               GPRC5B
ZNF577.1               ZNF577
DGKA                     DGKA
AL928654.1.1       AL928654.1
CDK2AP2               CDK2AP2
ATE1                     ATE1
LRIG1                   LRIG1
PAG1                     PAG1
ZNF14                   ZNF14
CERS6.1                 CERS6
HSP90AB1             HSP90AB1
NCL.1                     NCL
SLC29A3               SLC29A3
WFS1.1                   WFS1
KLF16                   KLF16
MYL6.2                   MYL6
FXYD6                   FXYD6
AC004816.2         AC004816.2
LRRC75A.1             LRRC75A
HIST1H1E.1           HIST1H1E
TMEM51.1               TMEM51
PSMA3.1                 PSMA3
SAMD4A.1               SAMD4A
TNNC1.2                 TNNC1
F11R                     F11R
CDC42EP2.1           CDC42EP2
SLC5A12.2             SLC5A12
CKAP2                   CKAP2
KIF11.2                 KIF11
BASP1.1                 BASP1
DGKI.2                   DGKI
VDR.1                     VDR
OLFML2B.2             OLFML2B
SLC4A8.2               SLC4A8
SFRP1.2                 SFRP1
MT-ND4L               MT-ND4L
ZER1.1                   ZER1
CDKN2B-AS1         CDKN2B-AS1
AC128688.2.2       AC128688.2
GUCY1A1.1             GUCY1A1
STK10                   STK10
CCNI.1                   CCNI
TENT2                   TENT2
C3orf70.1             C3orf70
COL7A1.1               COL7A1
GUCY1B1.1             GUCY1B1
CEP128.2               CEP128
P3H1.1                   P3H1
PDS5B.2                 PDS5B
PPIL4                   PPIL4
TBC1D17               TBC1D17
RPS27L                 RPS27L
IL17RA                 IL17RA
ANTXR2.1               ANTXR2
HDAC6                   HDAC6
RBMS3.2                 RBMS3
MIR222HG             MIR222HG
ZNF850.2               ZNF850
DNAH14                 DNAH14
HELZ2.1                 HELZ2
TMEM147-AS1.1     TMEM147-AS1
ZNF469.1               ZNF469
NPW.1                     NPW
RBM6.1                   RBM6
ANO8.1                   ANO8
RPS6KA1.2             RPS6KA1
KCTD5.1                 KCTD5
AL161669.3         AL161669.3
NFAT5                   NFAT5
METTL27.2             METTL27
LIN54.1                 LIN54
EIF1B.2                 EIF1B
SERPINF1.2           SERPINF1
UBE2G2                 UBE2G2
KANSL1L.2             KANSL1L
TTC23                   TTC23
TENM3                   TENM3
MELTF.2                 MELTF
CDC6.2                   CDC6
PDLIM5.1               PDLIM5
RAB6B                   RAB6B
PSMD3                   PSMD3
RELT                     RELT
ESYT2.1                 ESYT2
FZD6                     FZD6
SLC25A6.1             SLC25A6
C7orf50               C7orf50
BRCC3.2                 BRCC3
SLC22A23.1           SLC22A23
RND3.1                   RND3
ABHD3                   ABHD3
HSPH1.1                 HSPH1
SCRN2                   SCRN2
HMGA1                   HMGA1
ORC6                     ORC6
CRTC1                   CRTC1
GADD45B.2             GADD45B
TRIO.2                   TRIO
LGALS3.2               LGALS3
ZSCAN31.1             ZSCAN31
SLC2A13.1             SLC2A13
SPRED1.1               SPRED1
SNHG12.2               SNHG12
ACTR3.1                 ACTR3
ZMAT3.1                 ZMAT3
RASAL2.1               RASAL2
CST3.2                   CST3
HECW1.2                 HECW1
SLC2A3                 SLC2A3
TRNP1                   TRNP1
IFITM3.2               IFITM3
TSPAN15               TSPAN15
GKAP1                   GKAP1
HSPG2.1                 HSPG2
LIMK2.2                 LIMK2
AC124798.1.2       AC124798.1
CCND1.2                 CCND1
UGCG.1                   UGCG
MAP3K2.1               MAP3K2
EMILIN1.2             EMILIN1
HOXB-AS3.2           HOXB-AS3
PBXIP1.1               PBXIP1
NUFIP2                 NUFIP2
GPT2.1                   GPT2
ARL4A                   ARL4A
AC245041.2.1       AC245041.2
ACSL3                   ACSL3
TMEM132A             TMEM132A
PLEKHH2.2             PLEKHH2
SLC2A5.2               SLC2A5
HMGN3.2                 HMGN3
ATOH8                   ATOH8
AC103740.2.2       AC103740.2
PRRC2A.2               PRRC2A
AREL1                   AREL1
MCM3AP-AS1.2       MCM3AP-AS1
NOD1                     NOD1
PIFO                     PIFO
CSNK1G2               CSNK1G2
LMO4.2                   LMO4
RASSF1.1               RASSF1
MYH9.1                   MYH9
ZBTB20                 ZBTB20
CHST6.1                 CHST6
MTMR3.1                 MTMR3
FAH.2                     FAH
GGT5.2                   GGT5
SAA1.2                   SAA1
LIFR.1                   LIFR
BTG3.2                   BTG3
APLP1                   APLP1
ABCC5                   ABCC5
LMCD1                   LMCD1
NFKBID.1               NFKBID
HAGH                     HAGH
C19orf48.1           C19orf48
SRPX.2                   SRPX
CRAMP1                 CRAMP1
PRKG1.2                 PRKG1
AC112220.3.2       AC112220.3
FTX                       FTX
WLS.1                     WLS
TRPM7.2                 TRPM7
UACA.2                   UACA
PRDX3.2                 PRDX3
AFP                       AFP
EPHB3.1                 EPHB3
ADGRV1.2               ADGRV1
TNFAIP1.2             TNFAIP1
DYRK1A                 DYRK1A
TOB1.2                   TOB1
PILRB                   PILRB
CCDC9B.1               CCDC9B
P4HA2.2                 P4HA2
DNA2                     DNA2
CD1D.1                   CD1D
ZFP36L1               ZFP36L1
ENC1.2                   ENC1
TBC1D31               TBC1D31
CAMSAP1               CAMSAP1
OPHN1.1                 OPHN1
BACE1-AS.2           BACE1-AS
SULT1B1               SULT1B1
LRRC38.2               LRRC38
ZNF251.2               ZNF251
KIAA1324L           KIAA1324L
ARID5A                 ARID5A
IRS2.2                   IRS2
CCDC151               CCDC151
PRKRIP1               PRKRIP1
KREMEN1.2             KREMEN1
CFL2                     CFL2
ATP10D.1               ATP10D
P4HA1.1                 P4HA1
IGFBP7.2               IGFBP7
AC048341.1         AC048341.1
WARS.2                   WARS
WDR24.1                 WDR24
NPTXR.2                 NPTXR
NRBP2.1                 NRBP2
FN3K                     FN3K
AC034236.2.1       AC034236.2
HSPD1                   HSPD1
CASP6.2                 CASP6
PATJ.2                   PATJ
C3AR1.1                 C3AR1
AC008467.1.2       AC008467.1
SMARCE1.2             SMARCE1
SLC25A17             SLC25A17
S1PR2                   S1PR2
NAV3.2                   NAV3
TCF4.1                   TCF4
COL14A1.2             COL14A1
KLHL21.1               KLHL21
SV2A                     SV2A
SEL1L3                 SEL1L3
FGF2.2                   FGF2
ARHGAP44             ARHGAP44
PRPF8                   PRPF8
LINC02086           LINC02086
RGS20.2                 RGS20
ORAI3.1                 ORAI3
TMBIM1.2               TMBIM1
AC097103.2.1       AC097103.2
MSANTD2.2             MSANTD2
GAN.1                     GAN
ZNF804A               ZNF804A
CNN3.2                   CNN3
GLIS3.1                 GLIS3
TCEAL3.1               TCEAL3
CCDC80                 CCDC80
GCLM.2                   GCLM
CLCN3                   CLCN3
LSAMP.2                 LSAMP
IFITM2.2               IFITM2
LMO7.2                   LMO7
TIFA.2                   TIFA
SPATA5L1.2           SPATA5L1
FZD3.1                   FZD3
ASIC1                   ASIC1
CAV1.2                   CAV1
KMO.2                     KMO
CPZ.2                     CPZ
YJU2.1                   YJU2
BOLA2B                 BOLA2B
FOSL2                   FOSL2
LAMA5.1                 LAMA5
SARS                     SARS
WBP1L                   WBP1L
MGAT5.1                 MGAT5
HS6ST2.2               HS6ST2
SCD                       SCD
WNK1.1                   WNK1
TCEAL9.2               TCEAL9
RSPO3.2                 RSPO3
SLIT2.1                 SLIT2
ZNF431.1               ZNF431
MYL7.2                   MYL7
GPR161.2               GPR161
FUCA1                   FUCA1
MED13L                 MED13L
IER2.1                   IER2
HYI                       HYI
S100A3                 S100A3
FBXO6                   FBXO6
ATP6V0B               ATP6V0B
PLEKHF1               PLEKHF1
ZNF430                 ZNF430
FAM89A.2               FAM89A
SORBS2.1               SORBS2
HOPX.1                   HOPX
PLCE1.2                 PLCE1
SLC25A45.1           SLC25A45
ANO7.1                   ANO7
PLA2G4C.2             PLA2G4C
NEDD9                   NEDD9
AGAP3                   AGAP3
AL121603.2         AL121603.2
LAD1                     LAD1
RASSF2.2               RASSF2
BRD1.1                   BRD1
LINC02334.1         LINC02334
KIAA0895             KIAA0895
IFI16.1                 IFI16
TC2N.1                   TC2N
BCAR3.2                 BCAR3
PCDH19.1               PCDH19
NKTR.2                   NKTR
KDR.2                     KDR
STON1                   STON1
HSPA13                 HSPA13
MDFI.2                   MDFI
DYRK3                   DYRK3
HLA-B.2                 HLA-B
MINPP1                 MINPP1
AL357055.3.2       AL357055.3
RHOB.2                   RHOB
AC139795.3.1       AC139795.3
PDIA6                   PDIA6
FZD8.1                   FZD8
HES4.1                   HES4
PAN2                     PAN2
CNTN5.1                 CNTN5
LRP5.1                   LRP5
PITX1.2                 PITX1
ZNF670                 ZNF670
RPP25.2                 RPP25
PIP5K1A               PIP5K1A
RALBP1.1               RALBP1
F2R.1                     F2R
UBR7                     UBR7
CALM1.2                 CALM1
BOD1.2                   BOD1
AC021242.3.2       AC021242.3
TBX18                   TBX18
ZNF93                   ZNF93
FAM219B               FAM219B
FAM84A.2               FAM84A
RRAS                     RRAS
BNC2                     BNC2
PRELP                   PRELP
GINS1.2                 GINS1
FAM241A               FAM241A
HOXB9                   HOXB9
ADAMTS3.1             ADAMTS3
PIK3R1.2               PIK3R1
PCF11                   PCF11
ST7-AS1.1             ST7-AS1
HAND1.2                 HAND1
HSD17B14             HSD17B14
EIF5.2                   EIF5
FNDC5                   FNDC5
PTPN7                   PTPN7
AMPD2                   AMPD2
SAMD5                   SAMD5
AC124283.1.2       AC124283.1
ARHGEF26.2           ARHGEF26
TTC21B.2               TTC21B
ANKRD36C.1           ANKRD36C
SEPSECS.2             SEPSECS
PRRT2                   PRRT2
NRM                       NRM
RAB32                   RAB32
TAB3.1                   TAB3
PURPL                   PURPL
PRPS1.1                 PRPS1
NASP.2                   NASP
HRAS                     HRAS
SRM                       SRM
VASH2                   VASH2
USP51                   USP51
RCL1                     RCL1
NUPR1.1                 NUPR1
DENND1B.1             DENND1B
NFATC2                 NFATC2
C9orf3                 C9orf3
CYP4V2                 CYP4V2
CDCA3.2                 CDCA3
TPGS2.2                 TPGS2
ERBIN                   ERBIN
ID2.2                     ID2
TAOK1.1                 TAOK1
C19orf33.1           C19orf33
PRKAR2B.2             PRKAR2B
AC010531.6         AC010531.6
MANBA.1                 MANBA
ZBTB16.2               ZBTB16
PLXNB1                 PLXNB1
UCP2                     UCP2
HMG20A.1               HMG20A
CRABP2                 CRABP2
SELENOM               SELENOM
PTBP3.1                 PTBP3
TRIB3.2                 TRIB3
DOCK1.2                 DOCK1
PRKAR1B               PRKAR1B
SIPA1L2.1             SIPA1L2
CCT6B                   CCT6B
ZNF669.1               ZNF669
SLC4A1AP.2           SLC4A1AP
SPCS3.1                 SPCS3
ADAM33.1               ADAM33
PLXDC2.1               PLXDC2
AL138966.2.2       AL138966.2
PRTFDC1               PRTFDC1
HOXA7.2                 HOXA7
SRI                       SRI
PPFIBP1.2             PPFIBP1
RAI14.1                 RAI14
ASRGL1                 ASRGL1
TBCE.1.1               TBCE.1
F2RL1.1                 F2RL1
AC097359.2.1       AC097359.2
ARMCX2                 ARMCX2
MRPL44                 MRPL44
ENTPD6.1               ENTPD6
SLC26A6.2             SLC26A6
PACSIN3.1             PACSIN3
GREB1L                 GREB1L
CAVIN1.1               CAVIN1
TIA1                     TIA1
LRRC8A                 LRRC8A
COQ8A.2                 COQ8A
HTRA1.1                 HTRA1
CCPG1                   CCPG1
WASF3.2                 WASF3
ERAP2                   ERAP2
GRPEL2                 GRPEL2
BLMH.2                   BLMH
ZCRB1                   ZCRB1
FEZ2                     FEZ2
ZBED9                   ZBED9
TPX2.1                   TPX2
PRKCI.1                 PRKCI
HIGD1A.2               HIGD1A
MPC2.2                   MPC2
SOD1.1                   SOD1
APBB3.2                 APBB3
CIP2A                   CIP2A
PIK3C3                 PIK3C3
CXCL2.2                 CXCL2
MT-ND6                 MT-ND6
RNASEH2C.2           RNASEH2C
DST                       DST
PHKA2                   PHKA2
CPPED1.2               CPPED1
ZNF101                 ZNF101
RBM3.1                   RBM3
GM2A                     GM2A
SSX2IP                 SSX2IP
ZNF43.1                 ZNF43
FAM160B2.1           FAM160B2
LDHA.1                   LDHA
CCDC74A.2             CCDC74A
TMEM65.1               TMEM65
ZBTB7B.2               ZBTB7B
ERV3-1                 ERV3-1
MED26                   MED26
HMGA2.2                 HMGA2
RNF112.2               RNF112
CBX2                     CBX2
TADA1.1                 TADA1
CHSY1                   CHSY1
PIM2.1                   PIM2
LMO7-AS1.1           LMO7-AS1
KCNJ2.2                 KCNJ2
STRBP                   STRBP
ADAM28                 ADAM28
PDHA1                   PDHA1
PELI3.2                 PELI3
HCFC1R1               HCFC1R1
TEX22.2                 TEX22
THBS2.2                 THBS2
AIMP1.1                 AIMP1
TNFSF15.2             TNFSF15
DHCR7.2                 DHCR7
ECHDC2.1               ECHDC2
KRBA2                   KRBA2
COL4A5.1               COL4A5
ZFYVE21.1             ZFYVE21
UBASH3B.2             UBASH3B
DKK3                     DKK3
USP15.1                 USP15
ADGRG6.2               ADGRG6
SGTA.2                   SGTA
TFPI2.2                 TFPI2
TCF21                   TCF21
AC011978.2.2       AC011978.2
CBFB                     CBFB
QPRT                     QPRT
GLRX                     GLRX
HEXIM1                 HEXIM1
SNHG8.1                 SNHG8
ST3GAL1.1             ST3GAL1
ISG15.2                 ISG15
CCDC107               CCDC107
UBXN2A.2               UBXN2A
EBF1.2                   EBF1
ZNF791                 ZNF791
RCN1                     RCN1
SEPT6                   SEPT6
CDH6.2                   CDH6
PCSK6.2                 PCSK6
HOXC5                   HOXC5
PDPN.2                   PDPN
AVEN                     AVEN
CDKN2A.2               CDKN2A
PPP1R3C               PPP1R3C
HIST1H2BK           HIST1H2BK
ZFP62                   ZFP62
PLPP4                   PLPP4
ANK3.1                   ANK3
TSLP.2                   TSLP
TPCN1                   TPCN1
ARHGEF2.1             ARHGEF2
CNN2.1                   CNN2
CHML                     CHML
ST13.1                   ST13
PGM5-AS1             PGM5-AS1
SDC2                     SDC2
LTBP3.1                 LTBP3
SBF2-AS1.2           SBF2-AS1
SEC61G                 SEC61G
CYR61.1                 CYR61
MPRIP.2                 MPRIP
CREB3L2               CREB3L2
LMBRD2.1               LMBRD2
AKAP11                 AKAP11
KSR1                     KSR1
PRPF38B               PRPF38B
ARHGAP10.1           ARHGAP10
ZNF793                 ZNF793
SGF29.1                 SGF29
SRP68                   SRP68
BEX3.2                   BEX3
TMEM102               TMEM102
TMEM80                 TMEM80
CERS1                   CERS1
ENOSF1.1               ENOSF1
AC110597.1.1       AC110597.1
ADAMTS19             ADAMTS19
AL163051.1.2       AL163051.1
TMEM92-AS1.1       TMEM92-AS1
DYNC1I1.2             DYNC1I1
ARG2.2                   ARG2
TCEAL1.2               TCEAL1
KANK4.2                 KANK4
TPD52L1               TPD52L1
TMEM222               TMEM222
MSN.1                     MSN
ASZ1.1                   ASZ1
MIATNB.2               MIATNB
MYH10.2                 MYH10
ANKRD45               ANKRD45
JMJD6                   JMJD6
TPP2.2                   TPP2
KIAA1328.2           KIAA1328
NEO1.1                   NEO1
FRS2                     FRS2
CCDC184               CCDC184
BX255923.1.2       BX255923.1
EGR2.2                   EGR2
HOXA3.1                 HOXA3
RTL5.1                   RTL5
APLF                     APLF
ACLY.1                   ACLY
HSPB11.2               HSPB11
SRSF8                   SRSF8
KIF20B.2               KIF20B
EVC                       EVC
RIMS2                   RIMS2
TUBB2B.1               TUBB2B
EFNA1.2                 EFNA1
IGF2-AS.2             IGF2-AS
WSB1                     WSB1
TEDC2                   TEDC2
LOXL1-AS1.1         LOXL1-AS1
HSP90AA1             HSP90AA1
PAWR                     PAWR
FCGRT.1                 FCGRT
C1orf198.1           C1orf198
LAMTOR5.2             LAMTOR5
OTUB1.1                 OTUB1
SOD2.2                   SOD2
RGS1                     RGS1
CLSTN2                 CLSTN2
MSMO1.2                 MSMO1
PPP1CB.2               PPP1CB
IFNGR1.2               IFNGR1
PCSK1N.1               PCSK1N
MBNL1.2                 MBNL1
AP003068.2         AP003068.2
EGFL7                   EGFL7
MCUB.2                   MCUB
MYL9.2                   MYL9
AK6.1                     AK6
MAPRE2.1               MAPRE2
TWIST1.1               TWIST1
TK1                       TK1
ABCA5.2                 ABCA5
CMTM8                   CMTM8
TFPI.2                   TFPI
HSD17B4.1             HSD17B4
RAD51B.2               RAD51B
DTWD1                   DTWD1
CDCA8.2                 CDCA8
TRIAP1                 TRIAP1
FRMD8                   FRMD8
PTPN9.2                 PTPN9
CCNDBP1.1             CCNDBP1
RRM1.1                   RRM1
RBM12                   RBM12
TOX2.2                   TOX2
DNAJC25               DNAJC25
TGFB1I1               TGFB1I1
SVIP.2                   SVIP
ZNF699.1               ZNF699
CMBL.1                   CMBL
PRAC2                   PRAC2
PPM1N                   PPM1N
TENT5A                 TENT5A
HACD1.2                 HACD1
HNRNPDL               HNRNPDL
CCDC125.1             CCDC125
TPM3                     TPM3
HSPA1A.1               HSPA1A
RASL11B.2             RASL11B
FAT1.1                   FAT1
CD99.2                   CD99
CCNT2-AS1           CCNT2-AS1
SPINT1.2               SPINT1
TMEM11.2               TMEM11
IAH1                     IAH1
KLF9.2                   KLF9
UBXN4.2                 UBXN4
EIF2AK3               EIF2AK3
GALNT6.1               GALNT6
AMACR.1                 AMACR
ZNF160                 ZNF160
SLC5A3.1               SLC5A3
BCLAF1                 BCLAF1
MAN1B1-DT           MAN1B1-DT
PM20D2.1               PM20D2
CDC37L1.2             CDC37L1
SYTL2.2                 SYTL2
EEA1.1                   EEA1
HAPLN4.2               HAPLN4
GLB1L.2                 GLB1L
MYLIP.2                 MYLIP
TRIM24.2               TRIM24
BNC1.1                   BNC1
BEX4                     BEX4
SMIM1.1                 SMIM1
PANX2.1                 PANX2
ANKRD34A.2           ANKRD34A
AL121944.1         AL121944.1
TSPAN4                 TSPAN4
CTSC.1                   CTSC
ZNF121.2               ZNF121
MED31.1                 MED31
WFDC3.2                 WFDC3
BEND6.1                 BEND6
ZSWIM3.2               ZSWIM3
DERL3.1                 DERL3
BCL9                     BCL9
MCTP2                   MCTP2
AC006480.2.1       AC006480.2
ABHD15.1               ABHD15
ACKR2.2                 ACKR2
PTPRJ.2                 PTPRJ
UCN                       UCN
AC144831.1         AC144831.1
ZADH2.2                 ZADH2
NEK7                     NEK7
JOSD2.1                 JOSD2
TCEAL4.2               TCEAL4
SLTM                     SLTM
GSTA4.1                 GSTA4
AXL.1                     AXL
ANKFN1.2               ANKFN1
LYRM1                   LYRM1
FBLN2.2                 FBLN2
RAB42                   RAB42
ABHD4                   ABHD4
VPS9D1.1               VPS9D1
PLP2.2                   PLP2
EPHB6.1                 EPHB6
CALY.2                   CALY
CAVIN3.2               CAVIN3
KDM1B.1                 KDM1B
STS                       STS
EGFR.2                   EGFR
IL32.1                   IL32
FGD4.2                   FGD4
SLC9A9.1               SLC9A9
TMTC2.1                 TMTC2
PLCB4.2                 PLCB4
SLC24A1.2             SLC24A1
HLA-DPB1.2           HLA-DPB1
PTPRA.2                 PTPRA
MGP.2                     MGP
AP003486.1         AP003486.1
CDK5RAP2.2           CDK5RAP2
NQO2.2                   NQO2
TEDC1.1                 TEDC1
CDC7                     CDC7
DIO3OS.2               DIO3OS
ISYNA1.2               ISYNA1
SARDH.1                 SARDH
TLE4.1                   TLE4
RAD21.2                 RAD21
AC009414.2.1       AC009414.2
SOBP                     SOBP
LINC01836.1         LINC01836
TP53I3.2               TP53I3
MMP7.1                   MMP7
SATB1                   SATB1
PTPRE                   PTPRE
AC108062.1         AC108062.1
MAPK12.1               MAPK12
NECAB3                 NECAB3
SPACA6                 SPACA6
TET1.2                   TET1
RBP1.2                   RBP1
CDC42EP3.2           CDC42EP3
MANSC1.2               MANSC1
ROGDI.1                 ROGDI
CTSZ.2                   CTSZ
LAYN                     LAYN
PPM1K.1                 PPM1K
RIOX1                   RIOX1
TXNIP.2                 TXNIP
AC008771.1         AC008771.1
LCMT2.1                 LCMT2
G6PD.2                   G6PD
RIC8B                   RIC8B
YRDC                     YRDC
AC244090.1         AC244090.1
AC092040.1         AC092040.1
ATRNL1.2               ATRNL1
CLIC1.1                 CLIC1
PPIP5K1.1             PPIP5K1
LHPP                     LHPP
AC007878.1.2       AC007878.1
ST20-AS1             ST20-AS1
STX12.2                 STX12
FBXW8.1                 FBXW8
PHIP.2                   PHIP
TCN2                     TCN2
KLHDC9                 KLHDC9
DNMT3A                 DNMT3A
GNPDA1.2               GNPDA1
CLEC1B.2               CLEC1B
PPA1.1                   PPA1
MAGI1.1                 MAGI1
FAM83G.2               FAM83G
LAMC3.2                 LAMC3
AC132192.2.2       AC132192.2
BCL2L1                 BCL2L1
TRAF4.2                 TRAF4
ANKRD29.2             ANKRD29
C1QTNF1.1             C1QTNF1
CLEC11A               CLEC11A
JAGN1.1                 JAGN1
SH3PXD2A             SH3PXD2A
MAPKAPK2             MAPKAPK2
ODC1.2                   ODC1
GALNT16.1             GALNT16
EML4                     EML4
CARD16.2               CARD16
TTC30B.2               TTC30B
PCYOX1L               PCYOX1L
SEPT9.2                 SEPT9
MAMDC2                 MAMDC2
CRK.1                     CRK
SLC19A2.2             SLC19A2
ZIK1.2                   ZIK1
ANKDD1A               ANKDD1A
ARHGAP29.2           ARHGAP29
NEDD4L.1               NEDD4L
CYP27A1.2             CYP27A1
RTN3.2                   RTN3
EPHB2.1                 EPHB2
RSRC2.1                 RSRC2
FHOD1                   FHOD1
FIRRE.2                 FIRRE
IQCK                     IQCK
LITAF.2                 LITAF
SPATA17.1             SPATA17
RHOT2                   RHOT2
PSME4                   PSME4
SH3BGRL               SH3BGRL
GDNF.1                   GDNF
MED12L                 MED12L
FIGNL1.1               FIGNL1
STXBP5L.1             STXBP5L
WDYHV1.1               WDYHV1
AC064836.3.2       AC064836.3
KCNK15-AS1         KCNK15-AS1
SLC30A1.1             SLC30A1
AC103706.1.2       AC103706.1
SOCS1.2                 SOCS1
EFEMP2.1               EFEMP2
ELF1                     ELF1
PTPN1                   PTPN1
INTS7                   INTS7
SUCO                     SUCO
BDH2.2                   BDH2
GNAS-AS1.1           GNAS-AS1
PARPBP                 PARPBP
ZNRF3.2                 ZNRF3
PELI2                   PELI2
PALLD                   PALLD
PLEKHA5               PLEKHA5
PPP5D1.2               PPP5D1
PI4KA                   PI4KA
AC009126.1.1       AC009126.1
CEBPD.2                 CEBPD
FERMT1                 FERMT1
RGS11                   RGS11
KLHL42.1               KLHL42
UBE2D1.1               UBE2D1
TMEM97                 TMEM97
PDLIM4                 PDLIM4
IGFBP6                 IGFBP6
LUM.2                     LUM
TMEM243               TMEM243
PYGB.2                   PYGB
FAM69A.2               FAM69A
ATP2B4.2               ATP2B4
AL138724.1.1       AL138724.1
SCAMP1-AS1.2       SCAMP1-AS1
MSX2.2                   MSX2
CTDSPL2               CTDSPL2
NAP1L1.2               NAP1L1
NEK5.2                   NEK5
KIAA1841.1           KIAA1841
GAB2.2                   GAB2
LRCH4.2                 LRCH4
PDCD4-AS1.1         PDCD4-AS1
UCHL1.1                 UCHL1
AC010336.1.2       AC010336.1
AKAP12.2               AKAP12
HSPB8.1                 HSPB8
IER5L.2                 IER5L
FBXO34                 FBXO34
HDDC2.1                 HDDC2
FABP5.2                 FABP5
CXXC5                   CXXC5
C19orf66             C19orf66
CYP3A5                 CYP3A5
C7orf43               C7orf43
KIAA1551             KIAA1551
NUB1                     NUB1
BDNF-AS               BDNF-AS
UBP1.1                   UBP1
AC245140.2         AC245140.2
DOK1.1                   DOK1
ENO1                     ENO1
WNT11.2                 WNT11
PITPNM1.1             PITPNM1
AC091271.1         AC091271.1
PALMD                   PALMD
MVK.2                     MVK
CBLN3.1                 CBLN3
SOCS2.2                 SOCS2
SCAF8                   SCAF8
YBX3.2                   YBX3
SLC39A2.2             SLC39A2
HIF1A.2                 HIF1A
RERE.2                   RERE
SIN3B                   SIN3B
YDJC.1                   YDJC
SDC1.2                   SDC1
PLN.2                     PLN
DNAJB4.1               DNAJB4
C9orf47.2             C9orf47
MGARP                   MGARP
C21orf91.1           C21orf91
GSEC.1                   GSEC
ACTA2.2                 ACTA2
HSPB3.2                 HSPB3
MEGF9                   MEGF9
SMOC2                   SMOC2
RAB13.2                 RAB13
SDHAF3.2               SDHAF3
AC004803.1.2       AC004803.1
MAML3.1                 MAML3
ZNF35                   ZNF35
RNF227.2               RNF227
ICAM5                   ICAM5
SPA17.1                 SPA17
DYNLL2.2               DYNLL2
IP6K3.2                 IP6K3
DMPK                     DMPK
IGF1                     IGF1
SQSTM1.2               SQSTM1
CA11                     CA11
PEA15.1                 PEA15
ZEB2.1                   ZEB2
AF165147.1         AF165147.1
EFNA5.1                 EFNA5
C16orf74             C16orf74
FAM171B               FAM171B
PHLDB1.2               PHLDB1
TNK2.1                   TNK2
AL356056.1.2       AL356056.1
BCO2.2                   BCO2
SLC8A1                 SLC8A1
LAGE3.1                 LAGE3
SKP2.2                   SKP2
CEBPZOS.1             CEBPZOS
RFWD3.1                 RFWD3
WFDC1.2                 WFDC1
MICALL2.2             MICALL2
AC097478.1.2       AC097478.1
PHPT1                   PHPT1
CCNA2.2                 CCNA2
SKIDA1                 SKIDA1
CENPK.1                 CENPK
ZC3H7B.1               ZC3H7B
HAND2-AS1.2         HAND2-AS1
ABHD14A               ABHD14A
SETD4.1                 SETD4
AL356599.1.1       AL356599.1
BBC3                     BBC3
ECI1.2                   ECI1
PCDH9                   PCDH9
SPECC1L.2             SPECC1L
KPNA5.1                 KPNA5
MAP1A                   MAP1A
ZFP36L2.2             ZFP36L2
PAFAH1B3             PAFAH1B3
RAB9A.2                 RAB9A
MAPK8IP3.2           MAPK8IP3
SH3GL3                 SH3GL3
FAM133A.2             FAM133A
SYT8.2                   SYT8
IL27RA.1               IL27RA
ODF3L1                 ODF3L1
PTPRM                   PTPRM
MGA                       MGA
VCL                       VCL
FBXO8                   FBXO8
PPP1R3B.1             PPP1R3B
ZNF320.2               ZNF320
PRAC1.2                 PRAC1
TXNDC16.2             TXNDC16
ODF3B                   ODF3B
FIBIN.2                 FIBIN
GINS4.2                 GINS4
USP12.1                 USP12
PCBD1.1                 PCBD1
ZFAND2A.1             ZFAND2A
RAP2C                   RAP2C
ADAMTS1.2             ADAMTS1
DSEL.2                   DSEL
SAT1.2                   SAT1
CEP44.2                 CEP44
FDFT1.2                 FDFT1
ANXA1                   ANXA1
TVP23B.1               TVP23B
AURKA.2                 AURKA
CAMK2N1               CAMK2N1
PIDD1.2                 PIDD1
CDKN1B                 CDKN1B
ARMCX4                 ARMCX4
IL17RE.2               IL17RE
TMEM176A             TMEM176A
SYNM.1                   SYNM
FGFRL1.2               FGFRL1
CLSTN3                 CLSTN3
COTL1.1                 COTL1
ST6GAL1.2             ST6GAL1
AXIN1.1                 AXIN1
MAD2L2                 MAD2L2
MBP.2                     MBP
MYBL1.1                 MYBL1
FBXO5.1                 FBXO5
CITED2.2               CITED2
ZNF22                   ZNF22
SRSF7                   SRSF7
UGDH.1                   UGDH
IKZF2.2                 IKZF2
RPGR                     RPGR
STAMBP                 STAMBP
FAM13B                 FAM13B
TBPL1                   TBPL1
RAPGEF2.1             RAPGEF2
RASSF1-AS1         RASSF1-AS1
SNX18.1                 SNX18
BTBD3.2                 BTBD3
AL135999.1         AL135999.1
POPDC3                 POPDC3
KIAA0040.2           KIAA0040
MMADHC.2               MMADHC
SNRPN.2                 SNRPN
KLRG1.1                 KLRG1
GOPC.1                   GOPC
PRLR.2                   PRLR
PLEKHA4.2             PLEKHA4
CADPS2.2               CADPS2
ZNF460                 ZNF460
HDGFL2                 HDGFL2
RTL8B                   RTL8B
ANO4.2                   ANO4
ANGPT1                 ANGPT1
NAAA                     NAAA
RTN2.1                   RTN2
CDKN1C.2               CDKN1C
SCN1B.1                 SCN1B
TAF1A.1                 TAF1A
ALKAL2.2               ALKAL2
VEGFB                   VEGFB
RGCC                     RGCC
REXO5.2                 REXO5
RIN2                     RIN2
FLOT2                   FLOT2
APOC1.2                 APOC1
LDLR.2                   LDLR
ERMARD.1               ERMARD
MAP3K7CL.2           MAP3K7CL
GSTO1.2                 GSTO1
MIR22HG.2             MIR22HG
TSPAN13.1             TSPAN13
MYCBP                   MYCBP
NOP56                   NOP56
SC5D                     SC5D
ZDHHC14.2             ZDHHC14
CCDC171.2             CCDC171
CTBP2                   CTBP2
ICE1.2                   ICE1
UBE2W                   UBE2W
FKBP5.2                 FKBP5
SERTAD3               SERTAD3
CNN1.2                   CNN1
RALGAPA2             RALGAPA2
PDCD4.2                 PDCD4
MEX3A.1                 MEX3A
ALOX5AP.2             ALOX5AP
SPSB1.2                 SPSB1
BATF3.2                 BATF3
CARMIL1               CARMIL1
NUDT6.1                 NUDT6
MMS22L.1               MMS22L
ZCCHC2                 ZCCHC2
KIF22.2                 KIF22
LRIF1.2                 LRIF1
IL11.1                   IL11
YWHAH.2                 YWHAH
GYPC.2                   GYPC
C1orf112.2           C1orf112
CITED1                 CITED1
NSG1.2                   NSG1
FASN.1                   FASN
CAST.2                   CAST
PSAT1.1                 PSAT1
PDLIM1                 PDLIM1
WNT2.2                   WNT2
S1PR5                   S1PR5
SENP5                   SENP5
FAXDC2.1               FAXDC2
TRNAU1AP             TRNAU1AP
PLEKHG2               PLEKHG2
CDKN2B.1               CDKN2B
AP1S2                   AP1S2
AC069544.1.2       AC069544.1
ANKRD36               ANKRD36
LLGL1                   LLGL1
MYOCD.2                 MYOCD
FHL3.1                   FHL3
SEMA3D.2               SEMA3D
DTX3L.2                 DTX3L
PSMB8.2                 PSMB8
MTSS1.1                 MTSS1
BNIP3.2                 BNIP3
FBXW7                   FBXW7
GLIS1                   GLIS1
DAAM2.2                 DAAM2
LIMS2.2                 LIMS2
TNFRSF12A.2         TNFRSF12A
CRYGS.2                 CRYGS
TSPO.2                   TSPO
GAS1.2                   GAS1
NPEPL1.1               NPEPL1
DUSP16.2               DUSP16
DCAF16.2               DCAF16
TRIM16.2               TRIM16
CCNG2                   CCNG2
MT1E.2                   MT1E
ATF7IP2               ATF7IP2
AC073611.1.2       AC073611.1
HACD3                   HACD3
AL031714.1.1       AL031714.1
KLF7                     KLF7
MAMSTR.1               MAMSTR
ANP32B.2               ANP32B
HOXA10-AS.1         HOXA10-AS
INSIG1                 INSIG1
CKS2.2                   CKS2
CALD1.2                 CALD1
RBPMS2                 RBPMS2
EPHX2.2                 EPHX2
FLVCR1.2               FLVCR1
LINC00545.2         LINC00545
ARID5B.2               ARID5B
AC012645.1.2       AC012645.1
MRPS26                 MRPS26
DNAJC12.1             DNAJC12
MNDA.2                   MNDA
DNMBP.1                 DNMBP
SSPN.2                   SSPN
AP002884.1.2       AP002884.1
TTLL11.2               TTLL11
DIRAS3                 DIRAS3
ITGA10.2               ITGA10
OTUD3.2                 OTUD3
FLOT1.2                 FLOT1
GATA6                   GATA6
NCS1.2                   NCS1
PKIB.1                   PKIB
AC092279.1         AC092279.1
PPIB.1                   PPIB
INAFM1                 INAFM1
MAGI3                   MAGI3
PITPNM3.1             PITPNM3
PHKG1.1                 PHKG1
CDKN3.1                 CDKN3
ATP10A.1               ATP10A
DAB2IP                 DAB2IP
FAM213A               FAM213A
SRR                       SRR
GIPC1                   GIPC1
PNMT.2                   PNMT
MLPH.2                   MLPH
TMEM154.2             TMEM154
TRIM16L.2             TRIM16L
ADSL                     ADSL
IDI1                     IDI1
MCRIP2                 MCRIP2
AL365203.2.1       AL365203.2
BMPER.2                 BMPER
PRDM1                   PRDM1
KCNAB3.1               KCNAB3
BTBD9                   BTBD9
ID3.2                     ID3
PROS1.1                 PROS1
AHCTF1.2               AHCTF1
OCIAD2.2               OCIAD2
SH3PXD2B.1           SH3PXD2B
FAM217B               FAM217B
CRACR2B.1             CRACR2B
PCAT1.2                 PCAT1
KALRN                   KALRN
AKIRIN2.1             AKIRIN2
SRRM3.2                 SRRM3
KLF6.1                   KLF6
PHGDH                   PHGDH
GUCY1A2.1             GUCY1A2
RTF1                     RTF1
DIP2C.1                 DIP2C
LHFPL2.2               LHFPL2
CCDC97.2               CCDC97
SCG2.2                   SCG2
NPTX2                   NPTX2
PNPLA3.2               PNPLA3
SMAGP.1                 SMAGP
STXBP1                 STXBP1
PMEPA1.2               PMEPA1
SLC16A4.1             SLC16A4
NUDT8.2                 NUDT8
DCTN2                   DCTN2
USP40                   USP40
C16orf46             C16orf46
KIAA1217.2           KIAA1217
AC073111.5.1       AC073111.5
ERICH5.1               ERICH5
AC007014.1.2       AC007014.1
CITED4                 CITED4
FOXO3.2                 FOXO3
RTN4.1                   RTN4
MCTP1.2                 MCTP1
EPOR.1                   EPOR
NR2F2                   NR2F2
CGREF1                 CGREF1
HLTF                     HLTF
NDC1                     NDC1
TMEM64.1               TMEM64
PLA2G5.2               PLA2G5
SNHG7.2                 SNHG7
RTL9.2                   RTL9
TSC22D3.2             TSC22D3
CKB.2                     CKB
SLC12A7.1             SLC12A7
LRRC73.1               LRRC73
SLC7A11.2             SLC7A11
PPP1R12A             PPP1R12A
SHPK.1                   SHPK
UFM1                     UFM1
OBSCN.1                 OBSCN
SNHG25                 SNHG25
USP54                   USP54
KIF9-AS1.2           KIF9-AS1
DDIT3.2                 DDIT3
ZNF503                 ZNF503
ID1.2                     ID1
CLEC3A.2               CLEC3A
SRSF3.2                 SRSF3
DUSP8.1                 DUSP8
TMEM44.1               TMEM44
MCM3                     MCM3
MGAT4B                 MGAT4B
PITX2.1                 PITX2
GPX3                     GPX3
KCNK12.2               KCNK12
FLRT3.2                 FLRT3
HSBP1L1.2             HSBP1L1
CSRNP3.2               CSRNP3
MMP24OS.1             MMP24OS
SSH3.1                   SSH3
GPR143                 GPR143
ANGPTL2.1             ANGPTL2
KDM7A                   KDM7A
SOX11.2                 SOX11
IGF2BP3.2             IGF2BP3
SAT2.2                   SAT2
PCDHGB6.1             PCDHGB6
KTN1.1                   KTN1
SERPING1.2           SERPING1
MIAT                     MIAT
GUSB.2                   GUSB
LRP2BP                 LRP2BP
DNAJC14.1             DNAJC14
DNAAF4                 DNAAF4
SAMD14                 SAMD14
MCAM.2                   MCAM
H1FX                     H1FX
FAM53B.1               FAM53B
AREG                     AREG
SMCO4.2                 SMCO4
FDXR                     FDXR
LINC01534.1         LINC01534
ABLIM1.2               ABLIM1
PANK1.2                 PANK1
MYLK.2                   MYLK
TBC1D2                 TBC1D2
DMRTA1.2               DMRTA1
UBE2E2.2               UBE2E2
NTM.2                     NTM
AC109322.1.1       AC109322.1
SLC38A1               SLC38A1
ULK4.2                   ULK4
LINC00623           LINC00623
ATP6V1A               ATP6V1A
MFSD14A               MFSD14A
PRNCR1                 PRNCR1
AL451050.2.1       AL451050.2
DNAJA1.1               DNAJA1
SGK1.1                   SGK1
ZNF781.1               ZNF781
VAMP8.1                 VAMP8
SLC26A11.2           SLC26A11
TMLHE.1                 TMLHE
CASTOR1.2             CASTOR1
SDCCAG8               SDCCAG8
FKRP.2                   FKRP
ITGB1.1                 ITGB1
COPG2.1                 COPG2
TGFBR3L.1             TGFBR3L
FAM229B.1             FAM229B
ZNF280C               ZNF280C
SBNO2                   SBNO2
GHR                       GHR
PCDHB14.1             PCDHB14
EXOG                     EXOG
C19orf73.1           C19orf73
CLK1.2                   CLK1
DPH6.2                   DPH6
MSRA.2                   MSRA
AC147067.1.1       AC147067.1
GPC3.2                   GPC3
NKD1                     NKD1
AC090409.1         AC090409.1
PHF20.1                 PHF20
ISPD.2                   ISPD
HOXB3.1                 HOXB3
TMEM131L.1           TMEM131L
MIOS.2                   MIOS
LYSMD3                 LYSMD3
PRNP.1                   PRNP
LAMA2.1                 LAMA2
DUSP2.1                 DUSP2
KCNK1.2                 KCNK1
DUSP23.1               DUSP23
RPP14.1                 RPP14
ZNF211                 ZNF211
SQLE.1                   SQLE
CAVIN4.2               CAVIN4
AC104986.2.2       AC104986.2
BCL2L11.1             BCL2L11
PFKFB4                 PFKFB4
CLASRP.1               CLASRP
TIPARP                 TIPARP
CCDC18.2               CCDC18
PSME1.2                 PSME1
AP1G2.1                 AP1G2
TMEM88.2               TMEM88
CXADR.2                 CXADR
TLCD2.2                 TLCD2
NT5DC2                 NT5DC2
ZC3H12C.2             ZC3H12C
DET1.2                   DET1
PCDH7                   PCDH7
MSC-AS1               MSC-AS1
TTLL12.1               TTLL12
TYMS.2                   TYMS
MKNK2.1                 MKNK2
FRMD6-AS1.2         FRMD6-AS1
CALM3.1                 CALM3
MMP23B.2               MMP23B
PSEN2                   PSEN2
MATN3.2                 MATN3
CNST                     CNST
POPDC2.2               POPDC2
SMPDL3A.2             SMPDL3A
ACAT2.1                 ACAT2
TOM1.2                   TOM1
PHLDB2                 PHLDB2
LARP6                   LARP6
DRAM1                   DRAM1
DUSP14.1               DUSP14
GBP1.2                   GBP1
IMPA2.2                 IMPA2
KRTCAP2.1             KRTCAP2
COBLL1                 COBLL1
SLC25A33             SLC25A33
URAD.1                   URAD
ST6GAL2.2             ST6GAL2
YTHDF3-AS1.2       YTHDF3-AS1
GCH1.2                   GCH1
TSPYL2                 TSPYL2
NR2F1-AS1.2         NR2F1-AS1
NHSL1.1                 NHSL1
NLRP1.2                 NLRP1
DMXL2                   DMXL2
TUB                       TUB
YPEL2                   YPEL2
SRRD.2                   SRRD
EPB41L4A-AS1.2   EPB41L4A-AS1
GRIA3.2                 GRIA3
ZNF550                 ZNF550
MSH3                     MSH3
RAB5IF                 RAB5IF
AC078850.1.2       AC078850.1
BCAT2                   BCAT2
DPYSL2.2               DPYSL2
PHAX                     PHAX
ZNF772.1               ZNF772
MSC.2                     MSC
ATP6V0A1.1           ATP6V0A1
RNF115.2               RNF115
SHLD3                   SHLD3
AUTS2                   AUTS2
ZNF891                 ZNF891
UQCC3                   UQCC3
UNC119B               UNC119B
RBBP5                   RBBP5
YPEL3.1                 YPEL3
SLC22A15.2           SLC22A15
DDR2.1                   DDR2
SNX14.1                 SNX14
ARHGAP5-AS1.1     ARHGAP5-AS1
HSPA1B.1               HSPA1B
ZNF358.2               ZNF358
DUSP1.2                 DUSP1
HIST1H2AC.1         HIST1H2AC
GGCT                     GGCT
ZNF416                 ZNF416
ZC3H12A               ZC3H12A
HPS4                     HPS4
IDE.1                     IDE
ZNF516                 ZNF516
HNRNPH1.1             HNRNPH1
ATP6V0E2.2           ATP6V0E2
REL.2                     REL
LTBR.2                   LTBR
LINC00607           LINC00607
LINC00672           LINC00672
SLC6A17.1             SLC6A17
RBM43.1                 RBM43
KLHL5.2                 KLHL5
NFYA.2                   NFYA
GFOD1.2                 GFOD1
SHARPIN               SHARPIN
HES6.1                   HES6
METRNL                 METRNL
COPZ2.1                 COPZ2
PHLDA2.1               PHLDA2
HMGB2.2                 HMGB2
LTC4S.1                 LTC4S
RHOXF1-AS1         RHOXF1-AS1
RGS4.2                   RGS4
FNIP2                   FNIP2
FAM162A               FAM162A
TSHZ1.1                 TSHZ1
AL139082.1         AL139082.1
GAMT.1                   GAMT
LRRC45.2               LRRC45
RPS4Y1.2               RPS4Y1
STAG1.2                 STAG1
CORO2A                 CORO2A
AC073323.1.2       AC073323.1
MAK16.1                 MAK16
FAM133B.2             FAM133B
H2AFJ.2                 H2AFJ
SNTA1.2                 SNTA1
TBX3.2                   TBX3
IL10RB                 IL10RB
TSC22D1-AS1.1     TSC22D1-AS1
TSPAN5                 TSPAN5
LAMC2.2                 LAMC2
AL441883.1.2       AL441883.1
PSME2.1                 PSME2
MRPL35.2               MRPL35
NEIL1                   NEIL1
NPNT.2                   NPNT
PER1.2                   PER1
MED28.2                 MED28
KATNBL1.2             KATNBL1
ASTE1.1                 ASTE1
HMGCL                   HMGCL
AK4                       AK4
AC092807.3.2       AC092807.3
USP13.2                 USP13
HIST1H2BG.2         HIST1H2BG
TIGAR                   TIGAR
PTGR1.1                 PTGR1
GCHFR.1                 GCHFR
ARL4D.2                 ARL4D
AC097376.2         AC097376.2
ZNF232                 ZNF232
TENT4B                 TENT4B
CCDC183.2             CCDC183
PGM1                     PGM1
LIMCH1.1               LIMCH1
SCYL3                   SCYL3
CRISPLD2.2           CRISPLD2
AL050341.2.2       AL050341.2
APTR                     APTR
AEBP1.2                 AEBP1
DCUN1D2.1             DCUN1D2
NOP58.1                 NOP58
AC096733.2.1       AC096733.2
SLC25A25-AS1.1   SLC25A25-AS1
PLEKHG3.2             PLEKHG3
ZNF28                   ZNF28
AIMP2                   AIMP2
ARMC7                   ARMC7
DDAH2.1                 DDAH2
DIXDC1.2               DIXDC1
ZNF519                 ZNF519
DPT.1                     DPT
SMIM10.2               SMIM10
BOD1L1.2               BOD1L1
H2AFY.2                 H2AFY
STK17A                 STK17A
MYOZ1.2                 MYOZ1
IGF2.1.2               IGF2.1
TMEM143.1             TMEM143
PEX3.2                   PEX3
DUS3L                   DUS3L
MCFD2                   MCFD2
ALDH3A2.2             ALDH3A2
C4orf47               C4orf47
CALB2.2                 CALB2
AP000894.4         AP000894.4
CARD19.1               CARD19
STRADB.2               STRADB
TRAPPC6A.1           TRAPPC6A
RNF207.1               RNF207
RPRML.2                 RPRML
MC1R.2                   MC1R
KCNG1.2                 KCNG1
TRAF6                   TRAF6
KCTD20                 KCTD20
RASSF8.2               RASSF8
SP110.1                 SP110
WDR90.2                 WDR90
SGCD.2                   SGCD
NOTCH3.2               NOTCH3
NAPA                     NAPA
ATAD3B                 ATAD3B
CYP2U1.1               CYP2U1
PRUNE2.2               PRUNE2
SMCO3                   SMCO3
FAM111A-DT.2       FAM111A-DT
AC007938.3.2       AC007938.3
CYTOR.2                 CYTOR
ZNF185.1               ZNF185
AP001462.1         AP001462.1
MLLT11                 MLLT11
TERF1                   TERF1
CENPH                   CENPH
GMPR.2                   GMPR
RCN3.2                   RCN3
ZFHX2.1                 ZFHX2
ALDOC.1                 ALDOC
SCML1.1                 SCML1
B4GALNT4             B4GALNT4
IFI35.1                 IFI35
ZNF239                 ZNF239
ZNF8.1                   ZNF8
TPM1.2                   TPM1
L3HYPDH.1             L3HYPDH
VMO1                     VMO1
CDC42EP4.2           CDC42EP4
ITPKA.2                 ITPKA
ZBTB46                 ZBTB46
TSPAN12.2             TSPAN12
TENT5C.1               TENT5C
FAM19A2.2             FAM19A2
GLUL.2                   GLUL
LIPE-AS1.1           LIPE-AS1
KAT6B.2                 KAT6B
TRIM5.1                 TRIM5
PRICKLE1.2           PRICKLE1
ZNF385A.2             ZNF385A
SAMD9                   SAMD9
MFGE8.2                 MFGE8
DKK1.2                   DKK1
ATP6V0C.1             ATP6V0C
SNCA                     SNCA
PRAG1.2                 PRAG1
HGD.2                     HGD
BHLHE40.1             BHLHE40
EPB41L2.2             EPB41L2
MYO18A.1               MYO18A
ALDH1L2.2             ALDH1L2
PFN2.1                   PFN2
BLOC1S2               BLOC1S2
TMEM229B.2           TMEM229B
GGT1                     GGT1
TBC1D23               TBC1D23
ZNF350.2               ZNF350
PLEKHA7.2             PLEKHA7
LINC01252.2         LINC01252
CHAC2                   CHAC2
CENPU.2                 CENPU
AC068473.5         AC068473.5
BICC1.2                 BICC1
ZNF746                 ZNF746
CCDC50                 CCDC50
HOMER3                 HOMER3
PEMT.1                   PEMT
DNAJC1.2               DNAJC1
DCLRE1B               DCLRE1B
CLSPN.2                 CLSPN
CARD8-AS1.2         CARD8-AS1
UBE2F                   UBE2F
CELF2.1                 CELF2
GALNT1.1               GALNT1
CD63                     CD63
CDC14B.1               CDC14B
TESK1                   TESK1
PON2                     PON2
CAMTA2.2               CAMTA2
ABRACL                 ABRACL
G6PC3.1                 G6PC3
PLEKHO1.2             PLEKHO1
LRRN1                   LRRN1
RGS3.1                   RGS3
HENMT1.1               HENMT1
UNC5B-AS1.1         UNC5B-AS1
LEFTY2.2               LEFTY2
C2CD3.2                 C2CD3
FNBP1L.1               FNBP1L
LLGL2                   LLGL2
CD68.2                   CD68
ZNF34.1                 ZNF34
PIK3IP1               PIK3IP1
PCDHA7                 PCDHA7
C8orf88.2             C8orf88
SCARA3                 SCARA3
RIMS1.2                 RIMS1
LETM2.2                 LETM2
TRA2A                   TRA2A
GINS2.1                 GINS2
TES.1                     TES
SNX16.1                 SNX16
HCCS                     HCCS
CMTM3.2                 CMTM3
CLDN11.2               CLDN11
RGS10                   RGS10
CPEB3.2                 CPEB3
ZNF628.2               ZNF628
FABP3.2                 FABP3
MAP3K20.2             MAP3K20
SPRYD3.2               SPRYD3
CPEB4                   CPEB4
OLFML2A.1             OLFML2A
DMD                       DMD
HBEGF                   HBEGF
CENPE.2                 CENPE
BCAP31                 BCAP31
BACH1.2                 BACH1
SYNE2.2                 SYNE2
HMGB1.1                 HMGB1
PECR.2                   PECR
CD151.1                 CD151
PRIM2                   PRIM2
ASPHD1.1               ASPHD1
RALGPS2.2             RALGPS2
NFIA.2                   NFIA
DDX24.1                 DDX24
FOXF2.2                 FOXF2
ZNF653.2               ZNF653
CENPF.2                 CENPF
GTF2F1.2               GTF2F1
AC124016.1.2       AC124016.1
ABHD6                   ABHD6
NQO1.2                   NQO1
CTPS1                   CTPS1
ETNK2                   ETNK2
SORBS1                 SORBS1
ACBD6                   ACBD6
GFPT2                   GFPT2
SBF1.1                   SBF1
FILIP1L.2             FILIP1L
CWC25.2                 CWC25
SLC38A5               SLC38A5
TRIM46                 TRIM46
TFPT.2                   TFPT
SEPT1                   SEPT1
AKNA                     AKNA
WISP1.1                 WISP1
NETO2                   NETO2
GPI                       GPI
SLFN5                   SLFN5
RNF114                 RNF114
ZNF461                 ZNF461
AC005253.1         AC005253.1
SAMD15.1               SAMD15
NOS1AP.2               NOS1AP
FHL2.1                   FHL2
PPP2R3C.1             PPP2R3C
AL080317.1.1       AL080317.1
AC084036.1         AC084036.1
NUDT16.1               NUDT16
MVD.1                     MVD
AC016831.1.2       AC016831.1
BCL10.2                 BCL10
IDH1                     IDH1
CAMK4.2                 CAMK4
SH3BP5.2               SH3BP5
AC010491.1         AC010491.1
PQLC2L                 PQLC2L
ZNF324.2               ZNF324
Z95115.1             Z95115.1
ANKRD1.2               ANKRD1
ZNF462                 ZNF462
LINC01705.2         LINC01705
HIST1H2AE           HIST1H2AE
LINC00702           LINC00702
MBNL1-AS1.2         MBNL1-AS1
HOXC9.1                 HOXC9
RCHY1.1                 RCHY1
PLPP3.2                 PLPP3
GDF7                     GDF7
DDX58                   DDX58
RAC3                     RAC3
C5orf38.2             C5orf38
PYGO1.1                 PYGO1
ASB6                     ASB6
CISD1.2                 CISD1
SETD6                   SETD6
SLPI.2                   SLPI
SAMD11                 SAMD11
RDH14                   RDH14
DNAJB6                 DNAJB6
AL096865.1         AL096865.1
AL355338.1.2       AL355338.1
CD55.2                   CD55
RALY                     RALY
PRR15                   PRR15
TMED9.2                 TMED9
RPL22L1               RPL22L1
ATN1                     ATN1
RTTN                     RTTN
PHYHD1.2               PHYHD1
INPP1                   INPP1
JDP2                     JDP2
TPM2.2                   TPM2
GSTM3.2                 GSTM3
ZSCAN18.1             ZSCAN18
ZXDB                     ZXDB
CYB561D1.2           CYB561D1
SYN1.2                   SYN1
RUNX1                   RUNX1
SHF                       SHF
SMIM14                 SMIM14
HIST1H4C.2           HIST1H4C
APOD.2                   APOD
SMN1                     SMN1
SNAPC1.1               SNAPC1
OSGIN1.2               OSGIN1
ADAT2.2                 ADAT2
CLUAP1.1               CLUAP1
VEZT                     VEZT
ACTG2.2                 ACTG2
PKIG                     PKIG
ERO1B                   ERO1B
SRFBP1.2               SRFBP1
CRTAP.2                 CRTAP
ACOT2                   ACOT2
TM7SF2.2               TM7SF2
GDF6.2                   GDF6
USP1.2                   USP1
CTNS                     CTNS
CHORDC1.2             CHORDC1
CAB39L                 CAB39L
SMIM4                   SMIM4
CDKN1A.2               CDKN1A
PDGFC.1                 PDGFC
CENPX                   CENPX
FANCB.2                 FANCB
BTG1.2                   BTG1
FBXL22.2               FBXL22
FMO5.2                   FMO5
TMEM116.1             TMEM116
NKX3-1.2               NKX3-1
ZGLP1.2                 ZGLP1
KLF4.2                   KLF4
MPDZ                     MPDZ
BAG2.1                   BAG2
BAG1                     BAG1
DNALI1                 DNALI1
BIN1.2                   BIN1
AFAP1.2                 AFAP1
MIS18BP1.2           MIS18BP1
NT5C.2                   NT5C
ZKSCAN4.1             ZKSCAN4
FAM214A               FAM214A
AL121658.1         AL121658.1
IER3-AS1.1           IER3-AS1
MLEC.2                   MLEC
CRYAB.2                 CRYAB
AL390957.1         AL390957.1
TMEM39B.1             TMEM39B
TMEM120A.2           TMEM120A
CILP2.2                 CILP2
DGKE                     DGKE
AMMECR1.2             AMMECR1
HSD17B7.2             HSD17B7
GTF2F2                 GTF2F2
MBIP                     MBIP
LURAP1L               LURAP1L
TEX10                   TEX10
LPP-AS2               LPP-AS2
DCBLD1.2               DCBLD1
HOXC6.2                 HOXC6
CMIP.2                   CMIP
ALG2                     ALG2
SGSM2                   SGSM2
ZYX                       ZYX
OTULINL.2             OTULINL
GSDMD.2                 GSDMD
BTBD19                 BTBD19
ATP7B                   ATP7B
AMIGO2                 AMIGO2
JUP.2                     JUP
PKIA.2                   PKIA
IQCH-AS1.1           IQCH-AS1
NCAM2.2                 NCAM2
HMOX1.2                 HMOX1
NINJ1.1                 NINJ1
WDSUB1.1               WDSUB1
TRAF2.2                 TRAF2
EHD1                     EHD1
SFT2D3                 SFT2D3
ARAP2.2                 ARAP2
TMEM59L               TMEM59L
SLC1A1.2               SLC1A1
CCDC85B               CCDC85B
SERP1.1                 SERP1
C2orf69.1             C2orf69
AC090204.1.2       AC090204.1
TPCN2.2                 TPCN2
MELTF-AS1           MELTF-AS1
AL135905.1.2       AL135905.1
GPSM2.2                 GPSM2
LINC01611           LINC01611
SRGAP2C.1             SRGAP2C
ZNF608.1               ZNF608
IFFO2.1                 IFFO2
EPOP                     EPOP
CHST7.1                 CHST7
AC087190.1         AC087190.1
MRPS18B.2             MRPS18B
UST                       UST
MIDN                     MIDN
SNAI3-AS1           SNAI3-AS1
TCEANC                 TCEANC
ITGA3.1                 ITGA3
BCL7A.2                 BCL7A
RPAP1.1                 RPAP1
APOBEC3C.2           APOBEC3C
INPP5A                 INPP5A
TNFRSF21.2           TNFRSF21
JPT1.2                   JPT1
CTSL                     CTSL
MOV10L1.1             MOV10L1
METRN                   METRN
GNA13.1                 GNA13
AC069185.1.2       AC069185.1
MTHFD1L               MTHFD1L
AP002761.3.2       AP002761.3
PDGFRB.2               PDGFRB
KATNA1                 KATNA1
CCDC51.1               CCDC51
KCNMB4.1               KCNMB4
TLL2.2                   TLL2
SEC23B                 SEC23B
SPINK2.1               SPINK2
TNFSF4.2               TNFSF4
B3GALT4               B3GALT4
JUN.1                     JUN
GOLGA7.2               GOLGA7
ITM2C                   ITM2C
AKR1B1                 AKR1B1
CYFIP2                 CYFIP2
SHISA4                 SHISA4
PFDN1.1                 PFDN1
SLC3A2.2               SLC3A2
IQGAP2.2               IQGAP2
ZC4H2.1                 ZC4H2
FLNB.2                   FLNB
CLDND1                 CLDND1
FXYD5.2                 FXYD5
HIF1AN                 HIF1AN
DDR1.2                   DDR1
TYW5.1                   TYW5
PRKAG2                 PRKAG2
ATP1B1.1               ATP1B1
MINCR.2                 MINCR
RECK.1                   RECK
FBLIM1                 FBLIM1
AL031058.1         AL031058.1
CENPQ.1                 CENPQ
URB2.2                   URB2
RACGAP1.2             RACGAP1
GCNA.1                   GCNA
CFDP1                   CFDP1
SPATA7.2               SPATA7
C1S.2                     C1S
TMEM176B.2           TMEM176B
ATP23.1                 ATP23
AGAP9.2                 AGAP9
PLA2G16               PLA2G16
PRIM1.1                 PRIM1
BAIAP2-DT.2         BAIAP2-DT
PPARGC1B             PPARGC1B
MARCKS.2               MARCKS
ZNF436                 ZNF436
TMC6.2                   TMC6
GLCCI1                 GLCCI1
TMEM92                 TMEM92
RABL2A.1               RABL2A
FYN.1                     FYN
SLC27A4.1             SLC27A4
NOG.2                     NOG
ANKRD37.2             ANKRD37
TRIM58.2               TRIM58
USP28                   USP28
GADD45A.2             GADD45A
CHN1                     CHN1
YOD1                     YOD1
SLC25A4               SLC25A4
SLC46A3.2             SLC46A3
CHAC1.1                 CHAC1
ADAM19.2               ADAM19
RBMS1.1                 RBMS1
AASS                     AASS
MARCH9                 MARCH9
OTUD1.2                 OTUD1
TMEM100               TMEM100
IFNAR2.2               IFNAR2
SLC12A5               SLC12A5
FAM43A.2               FAM43A
POLE.2                   POLE
TWNK.1                   TWNK
TRHDE.2                 TRHDE
ZNF223.2               ZNF223
LINC02001.2         LINC02001
AIFM2.2                 AIFM2
DPF2.1                   DPF2
SOCS3.2                 SOCS3
TP53TG1.1             TP53TG1
AC016831.7         AC016831.7
FBXO32.2               FBXO32
AP001160.3         AP001160.3
TARSL2.2               TARSL2
AHNAK.2                 AHNAK
MANCR.2                 MANCR
HAAO                     HAAO
SKIL                     SKIL
ZNF117.2               ZNF117
ZCWPW1.2               ZCWPW1
HOXB5                   HOXB5
NDN.2                     NDN
HSPB6.1                 HSPB6
ACVR2B                 ACVR2B
NPIPB2                 NPIPB2
RFC1                     RFC1
KRCC1                   KRCC1
MIR503HG.1           MIR503HG
UCK2                     UCK2
RPP38.2                 RPP38
PKDCC.2                 PKDCC
SLC29A4.2             SLC29A4
AL606491.1.1       AL606491.1
CENPS.2                 CENPS
LRPAP1                 LRPAP1
HIST1H3H             HIST1H3H
PSD.1                     PSD
GATM.2                   GATM
DSP.1                     DSP
CHCHD7                 CHCHD7
SGK494.2               SGK494
ABCA1.2                 ABCA1
NDRG2.2                 NDRG2
LETMD1                 LETMD1
TXNRD2.2               TXNRD2
VASN                     VASN
MFAP5.2                 MFAP5
JAK1.2                   JAK1
S1PR1.2                 S1PR1
SALL3                   SALL3
PERP                     PERP
SPEF2                   SPEF2
TUBA1B.1               TUBA1B
MCM5.1                   MCM5
RGS2.2                   RGS2
PCDHA10.2             PCDHA10
PHACTR2.2             PHACTR2
TIMP4.2                 TIMP4
RELA                     RELA
FUOM                     FUOM
APOE.2                   APOE
CERS4                   CERS4
DCTN4.2                 DCTN4
PDZRN4.2               PDZRN4
AL136084.3.2       AL136084.3
LINC00342.1         LINC00342
SLC9A3R2.1           SLC9A3R2
EID2B.1                 EID2B
ZNF784                 ZNF784
TEX29.2                 TEX29
VPS13A                 VPS13A
PLOD3                   PLOD3
PACS1                   PACS1
CEBPB.2                 CEBPB
EMC9                     EMC9
HSD17B12             HSD17B12
CHERP.1                 CHERP
KRT18.2                 KRT18
RAB7B.2                 RAB7B
TUBA1A.2               TUBA1A
EPSTI1                 EPSTI1
STMN1.1                 STMN1
TUBB                     TUBB
CAPN10.1               CAPN10
TRAPPC10.1           TRAPPC10
PRR5                     PRR5
ARNTL.1                 ARNTL
WNT4.2                   WNT4
MICAL2                 MICAL2
UBE2S.2                 UBE2S
TYMP                     TYMP
TRIM39                 TRIM39
SVIL.2                   SVIL
RTL8A                   RTL8A
LIMS1                   LIMS1
RWDD3.1                 RWDD3
MDK                       MDK
ELF4                     ELF4
ISOC1                   ISOC1
SLC1A3.2               SLC1A3
C18orf25.2           C18orf25
HMGN2.2                 HMGN2
ZBTB8A                 ZBTB8A
RHOU.1                   RHOU
NAPRT                   NAPRT
TRIM56.1               TRIM56
GAS6-DT.2             GAS6-DT
MGST2                   MGST2
HOXA10.1               HOXA10
MRPL58                 MRPL58
RSPO2.2                 RSPO2
ZCCHC10.2             ZCCHC10
PHKG2                   PHKG2
PDGFA.1                 PDGFA
COQ2                     COQ2
ENAH.2                   ENAH
MICA.2                   MICA
ANAPC7                 ANAPC7
MAP3K8.2               MAP3K8
LGALS8                 LGALS8
MRPL46                 MRPL46
ZNF106                 ZNF106
OSER1                   OSER1
NEGR1.1                 NEGR1
SPDL1                   SPDL1
RMI1                     RMI1
FAM174A               FAM174A
ZWILCH.1               ZWILCH
AL031777.3.1       AL031777.3
SEPT10.2               SEPT10
TMTC1.2                 TMTC1
GPAM.2                   GPAM
PCGF5.2                 PCGF5
RAD54B.1               RAD54B
OSBPL3.1               OSBPL3
VEGFC.2                 VEGFC
NOTCH1                 NOTCH1
SLC31A2               SLC31A2
HOXB8.2                 HOXB8
CD44                     CD44
CCDC28B.2             CCDC28B
SHROOM3.2             SHROOM3
HIST1H2BJ.1         HIST1H2BJ
LRRCC1                 LRRCC1
HUNK.2                   HUNK
ZNF391                 ZNF391
TENT4A                 TENT4A
KLHL4.2                 KLHL4
PPP1R15A.1           PPP1R15A
EFHD2                   EFHD2
TRAPPC5.2             TRAPPC5
SH3BGRL3.2           SH3BGRL3
C1orf226.2           C1orf226
FGF7                     FGF7
BRSK2.1                 BRSK2
MASP1                   MASP1
HMGB3.2                 HMGB3
SDSL                     SDSL
ATP8B4.1               ATP8B4
AVIL.1                   AVIL
SMYD3                   SMYD3
KIAA1755.2           KIAA1755
EZR                       EZR
AC062029.1.2       AC062029.1
C21orf2.2             C21orf2
ITPR2.2                 ITPR2
FAM129A.2             FAM129A
GBP2.2                   GBP2
SPON2                   SPON2
SLAIN2                 SLAIN2
COMT.2                   COMT
GSTZ1.2                 GSTZ1
THY1.1                   THY1
ZBTB39                 ZBTB39
KIF23.2                 KIF23
S100A16.2             S100A16
FBXO2                   FBXO2
SGK3                     SGK3
NOP14.2                 NOP14
KLF2.2                   KLF2
DDX10.2                 DDX10
NMI.1                     NMI
AC020928.1.1       AC020928.1
TANGO6.1               TANGO6
ARHGDIB               ARHGDIB
IGFL3                   IGFL3
SLC48A1.1             SLC48A1
SNTB2                   SNTB2
BLOC1S5.2             BLOC1S5
DGKZ                     DGKZ
S100A4                 S100A4
LMOD1                   LMOD1
TONSL                   TONSL
LIG3.1                   LIG3
LRRFIP1               LRRFIP1
AP000866.2.2       AP000866.2
DBN1.2                   DBN1
CRNDE                   CRNDE
CCDC167               CCDC167
KIFC2.2                 KIFC2
SSBP2                   SSBP2
PRKAA1                 PRKAA1
USPL1                   USPL1
DCN                       DCN
MAFF.1                   MAFF
SYNGR1.2               SYNGR1
ZNF816.2               ZNF816
FH                         FH
PPME1.2                 PPME1
OSBPL10.2             OSBPL10
SNX25.1                 SNX25
C1orf56.2             C1orf56
HTATIP2               HTATIP2
HECTD3.2               HECTD3
DPPA4                   DPPA4
SYNC                     SYNC
DNM3OS                 DNM3OS
PSPH.2                   PSPH
AC021016.3.1       AC021016.3
PFKFB3                 PFKFB3
EPHA4.1                 EPHA4
AC097534.2         AC097534.2
SLC49A3               SLC49A3
FAM122C.1             FAM122C
TNNT2                   TNNT2
PHC1                     PHC1
DHRS2.2                 DHRS2
AKAP17A.2             AKAP17A
PKN1.2                   PKN1
ATL1.1                   ATL1
TRAF3IP1.2           TRAF3IP1
GPX7.2                   GPX7
LY96                     LY96
UBE2V1                 UBE2V1
PHF13.1                 PHF13
NEDD1                   NEDD1
TBX4                     TBX4
RNF19A                 RNF19A
BMP7                     BMP7
SMAP1                   SMAP1
EFNB1                   EFNB1
CDKN2C                 CDKN2C
DNAJB1.2               DNAJB1
AF213884.3         AF213884.3
CCDC14                 CCDC14
SNAI1.2                 SNAI1
DEK.2                     DEK
SLC22A2.2             SLC22A2
DUSP28.1               DUSP28
SMURF2.2               SMURF2
AC105942.1.1       AC105942.1
RFXANK.1               RFXANK
CENPW.1                 CENPW
HOXB-AS1.2           HOXB-AS1
RHPN2.1                 RHPN2
PHTF2.2                 PHTF2
PHF1                     PHF1
ST6GALNAC4         ST6GALNAC4
ANKZF1.1               ANKZF1
KCTD2.2                 KCTD2
AC103702.2         AC103702.2
NENF.2                   NENF
PITHD1.2               PITHD1
BDNF                     BDNF
ZNF700.1               ZNF700
CYB5A.1                 CYB5A
HEBP2.2                 HEBP2
EYS                       EYS
PTGFR.2                 PTGFR
CMTM7.2                 CMTM7
ZNF343                 ZNF343
SPATA13.1             SPATA13
CFH                       CFH
THAP9-AS1           THAP9-AS1
AL451085.2.1       AL451085.2
AP000487.1.2       AP000487.1
C12orf73             C12orf73
ZNF345.2               ZNF345
PIR.2                     PIR
DACT3.2                 DACT3
RCAN2.1                 RCAN2
SNHG15                 SNHG15
CRIP2.1                 CRIP2
FOXQ1.2                 FOXQ1
KIF21A                 KIF21A
WDR25.1                 WDR25
UBE2G1                 UBE2G1
KLF10                   KLF10
MGST1.2                 MGST1
PROCR.1                 PROCR
THAP2.1                 THAP2
FSIP1                   FSIP1
CYP1B1.1               CYP1B1
SOWAHC                 SOWAHC
FHOD3.2                 FHOD3
SLMAP.1                 SLMAP
ZNF678.2               ZNF678
SERTAD1               SERTAD1
LINC00862.2         LINC00862
GPBAR1.2               GPBAR1
HS3ST3A1.2           HS3ST3A1
SIRT5                   SIRT5
CREBRF.2               CREBRF
SMYD4.1                 SMYD4
BZW2.1                   BZW2
TRMT61A.1             TRMT61A
GASAL1.2               GASAL1
VLDLR.1                 VLDLR
NEXN                     NEXN
NTRK2.2                 NTRK2
ICAM3.2                 ICAM3
ZNF441                 ZNF441
ZNF432                 ZNF432
ZSWIM6.2               ZSWIM6
UBE3D.1                 UBE3D
ST20.1                   ST20
CCAR2                   CCAR2
FAM122A.2             FAM122A
LTB4R                   LTB4R
TNRC6C.1               TNRC6C
ARL6IP1.2             ARL6IP1
PEX5                     PEX5
TUBB2A                 TUBB2A
FENDRR.2               FENDRR
TXNRD1.2               TXNRD1
AC002310.1         AC002310.1
TAS2R14.2             TAS2R14
TMCO6.2                 TMCO6
SRSF4.1                 SRSF4
FILIP1                 FILIP1
CFLAR                   CFLAR
ZFP36.2                 ZFP36
CXCL14.2               CXCL14
KCTD1.2                 KCTD1
TAF1B                   TAF1B
HIST1H4H             HIST1H4H
TMEM45A.1             TMEM45A
FAM111A.2             FAM111A
PRPF39                 PRPF39
TLE2                     TLE2
DUSP6.2                 DUSP6
SAMM50                 SAMM50
MACROD2.1             MACROD2
FIP1L1.2               FIP1L1
NBL1.2                   NBL1
ERG28.2                 ERG28
CCNO.2                   CCNO
FAP                       FAP
ABHD5                   ABHD5
ADGRL4.2               ADGRL4
GALM.1                   GALM
GTF2I                   GTF2I
SHB.1                     SHB
AC026979.2         AC026979.2
PDCL3.2                 PDCL3
ZC2HC1A               ZC2HC1A
UGP2.2                   UGP2
FST.2                     FST
EXPH5.2                 EXPH5
CEP78                   CEP78
BARD1.2                 BARD1
CRY1.1                   CRY1
EXO5                     EXO5
BTG2                     BTG2
ZNF701                 ZNF701
MFSD4B                 MFSD4B
KLHDC8B               KLHDC8B
DDAH1.2                 DDAH1
EIF4A3.1               EIF4A3
AKR1C1.2               AKR1C1
PCDHA9                 PCDHA9
IDH3B.2                 IDH3B
TMEM91.2               TMEM91
ODAM                     ODAM
PODXL2.1               PODXL2
TNFAIP6.2             TNFAIP6
KIZ                       KIZ
PLAGL1.2               PLAGL1
TKT.2                     TKT
SIRT7                   SIRT7
ZNF639.2               ZNF639
C8orf33.2             C8orf33
TRIP13                 TRIP13
PCYT1B                 PCYT1B
SAXO2.2                 SAXO2
XPO6.1                   XPO6
FDPS.1                   FDPS
ST3GAL5               ST3GAL5
ITGA4.2                 ITGA4
MALT1                   MALT1
CCDC81.2               CCDC81
HLA-A.1                 HLA-A
LANCL2.1               LANCL2
CALCOCO2.2           CALCOCO2
PIM1.2                   PIM1
Z97832.2.1           Z97832.2
MARCH3.1               MARCH3
GPR153                 GPR153
NNMT.1                   NNMT
SYP                       SYP
CTNNB1.2               CTNNB1
PANCR                   PANCR
TRAF3IP2-AS1.2   TRAF3IP2-AS1
KNTC1                   KNTC1
CD320.2                 CD320
XKR8                     XKR8
NRP1                     NRP1
SBSPON                 SBSPON
MT2A                     MT2A
DDIT4.2                 DDIT4
SIGIRR.1               SIGIRR
TGM2.2                   TGM2
LNCSRLR               LNCSRLR
FOXP1.1                 FOXP1
CYSTM1.1               CYSTM1
UAP1.2                   UAP1
WRNIP1                 WRNIP1
SOX4.2                   SOX4
ZNF329.1               ZNF329
DYRK4                   DYRK4
TUSC1.1                 TUSC1
C1orf53               C1orf53
KAT8.2                   KAT8
VARS2.1                 VARS2
SSPO.1                   SSPO
ARHGEF9               ARHGEF9
PARP9.2                 PARP9
IRF2BPL.1             IRF2BPL
PMP22.2                 PMP22
ATF5                     ATF5
FAM219A               FAM219A
SLC27A5.1             SLC27A5
MARCKSL1.2           MARCKSL1
ANKRD44.1             ANKRD44
HMMR.2                   HMMR
SIRPA                   SIRPA
CCDC144NL-AS1.2 CCDC144NL-AS1
PHF3.2                   PHF3
ZFX                       ZFX
PXMP4                   PXMP4
ADPGK                   ADPGK
VIM.2                     VIM
PPIE                     PPIE
PLIN2.2                 PLIN2
SUGCT.2                 SUGCT
CFAP97.2               CFAP97
FAM107B.2             FAM107B
ZNF711.2               ZNF711
DOHH.2                   DOHH
SLC23A2.2             SLC23A2
ACTL10                 ACTL10
CD47.1                   CD47
AC009005.1         AC009005.1
CENPV.2                 CENPV
MBNL3.1                 MBNL3
AL138885.3         AL138885.3
PLPP1                   PLPP1
PIAS2                   PIAS2
GPATCH4               GPATCH4
RASGRF2.2             RASGRF2
PTGDS                   PTGDS
VASP.2                   VASP
VAMP3.2                 VAMP3
LINC01943.1         LINC01943
CYC1                     CYC1
FAM171A2.1           FAM171A2</code></pre>
<pre class="r"><code>features.plot &lt;- c(&quot;TIMP3&quot;, &quot;TIMP2&quot;, &quot;COL11A1&quot;, 
                   &quot;SOX5&quot;, &quot;SOX6&quot;,
                   &quot;TGFBI&quot;)
RidgePlot(object = SCT.integrated, features = features.plot, ncol = 3)</code></pre>
<pre><code>Picking joint bandwidth of 0.431</code></pre>
<pre><code>Picking joint bandwidth of 0.26</code></pre>
<pre><code>Picking joint bandwidth of 0.316</code></pre>
<pre><code>Picking joint bandwidth of 0.228</code></pre>
<pre><code>Picking joint bandwidth of 0.176</code></pre>
<pre><code>Picking joint bandwidth of 0.394</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/markers%20seurat%20DE-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>FeaturePlot(object = SCT.integrated, features = features.plot)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/markers%20seurat%20DE-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>DE using limma pseudobulk (from raw counts)</p>
<pre class="r"><code>library(SingleCellExperiment)</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following object is masked from &#39;package:gridExtra&#39;:

    combine</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, basename, cbind,
    colnames, dirname, do.call, duplicated, eval, evalq, Filter,
    Find, get, grep, grepl, intersect, is.unsorted, lapply, Map,
    mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, Position, rank, rbind, Reduce, rownames, sapply,
    setdiff, sort, table, tapply, union, unique, unsplit, which,
    which.max, which.min</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>Loading required package: BiocParallel</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    aperm, apply, rowsum</code></pre>
<pre><code>
Attaching package: &#39;SummarizedExperiment&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:Seurat&#39;:

    Assays</code></pre>
<pre class="r"><code>library(Matrix.utils)</code></pre>
<pre><code>Loading required package: Matrix</code></pre>
<pre><code>
Attaching package: &#39;Matrix&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:S4Vectors&#39;:

    expand</code></pre>
<pre class="r"><code>library(variancePartition)</code></pre>
<pre><code>Loading required package: ggplot2</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre><code>
Attaching package: &#39;limma&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:BiocGenerics&#39;:

    plotMA</code></pre>
<pre><code>Loading required package: foreach</code></pre>
<pre><code>Loading required package: scales</code></pre>
<pre><code>
Attaching package: &#39;variancePartition&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:limma&#39;:

    classifyTestsF</code></pre>
<pre class="r"><code>library(tidyr)</code></pre>
<pre><code>
Attaching package: &#39;tidyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Matrix&#39;:

    expand, pack, unpack</code></pre>
<pre><code>The following object is masked from &#39;package:S4Vectors&#39;:

    expand</code></pre>
<pre class="r"><code>library(ashr)

#psuedobulk
data &lt;- SCT.integrated@assays$SCT@counts
metadata &lt;- SCT.integrated@meta.data

# Set up metadata as desired for aggregation and DE analysis
metadata$seurat_clusters &lt;- factor(metadata$seurat_clusters)
metadata$labels &lt;- factor(metadata$labels)

# Create single cell experiment object
sce &lt;- SingleCellExperiment(assays = list(counts = data), 
                           colData = metadata)

# Identify groups for aggregation of counts
groups &lt;- colData(sce)[, c(&quot;labels&quot;, &quot;seurat_clusters&quot;)]

#pseudobulk
pb &lt;- aggregate.Matrix(t(counts(sce)), 
                       groupings = groups, fun = &quot;sum&quot;) 
pb_matrix &lt;- as.matrix(pb)</code></pre>
<p>Use modification of limma workflow implemented in variancePartition::dream</p>
<pre class="r"><code># process raw counts (pseudobulked)
library(edgeR)</code></pre>
<pre><code>
Attaching package: &#39;edgeR&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:SingleCellExperiment&#39;:

    cpm</code></pre>
<pre class="r"><code># filter genes by number of counts
isexpr &lt;- rowSums(cpm(t(pb_matrix))&gt;0.1) &gt;= 5

# Standard usage of limma/voom (normalize)
geneExpr &lt;- DGEList( t(pb_matrix)[isexpr,] )
geneExpr &lt;- calcNormFactors( geneExpr , method = &quot;TMM&quot;)</code></pre>
<pre class="r"><code>form &lt;- ~ 0 + seurat_clusters + (1|individual) + n_cells

pb_data &lt;- as.data.frame(rownames(pb_matrix))
pb_data &lt;- pb_data %&gt;% 
     tidyr::separate(`rownames(pb_matrix)`, c(&quot;individual&quot;, &quot;condition&quot;, &quot;seurat_clusters&quot;), sep = &quot;_&quot;) 
rownames(pb_data) &lt;- rownames(pb_matrix)
pb_data$n_cells &lt;- c(cluster_memberships[1,], cluster_memberships[2,], cluster_memberships[3,])
pb_data$lib.size &lt;- geneExpr$samples$lib.size

# estimate weights using linear mixed model of dream
vobjDream &lt;- voomWithDreamWeights( geneExpr, form, pb_data )</code></pre>
<pre><code>Memory usage to store result: &gt; 19.3 Mb 
Dividing work into 100 chunks...

Total: 17 s</code></pre>
<pre><code>Warning in regularize.values(x, y, ties, missing(ties)): collapsing to
unique &#39;x&#39; values</code></pre>
<pre class="r"><code># variance partition plot to see contribution of each variable to overall variance
form_vp &lt;- ~ (1|individual) + (1|seurat_clusters)
vp &lt;- variancePartition::fitExtractVarPartModel(vobjDream, form_vp, pb_data)</code></pre>
<pre><code>Dividing work into 100 chunks...

Total: 27 s</code></pre>
<pre class="r"><code>plotVarPart( sortCols(vp))</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/run%20lmm-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># Fit the dream model on each gene
# By default, uses the Satterthwaite approximation for the hypothesis test
fitmm &lt;- dream( vobjDream, form, pb_data )</code></pre>
<pre><code>Dividing work into 100 chunks...

Total: 43 s</code></pre>
<pre class="r"><code># Examine design matrix
fitmm$design</code></pre>
<pre><code>                   seurat_clusters0 seurat_clusters1 seurat_clusters2
NA18855_Unstrain_0                1                0                0
NA18856_Unstrain_0                1                0                0
NA19160_Unstrain_0                1                0                0
NA18855_Unstrain_1                0                1                0
NA18856_Unstrain_1                0                1                0
NA19160_Unstrain_1                0                1                0
NA18855_Unstrain_2                0                0                1
NA18856_Unstrain_2                0                0                1
NA19160_Unstrain_2                0                0                1
                   n_cells
NA18855_Unstrain_0     416
NA18856_Unstrain_0     279
NA19160_Unstrain_0     148
NA18855_Unstrain_1     141
NA18856_Unstrain_1      85
NA19160_Unstrain_1      59
NA18855_Unstrain_2      29
NA18856_Unstrain_2      31
NA19160_Unstrain_2      15
attr(,&quot;assign&quot;)
[1] 1 1 1 2
attr(,&quot;contrasts&quot;)
attr(,&quot;contrasts&quot;)$seurat_clusters
[1] &quot;contr.treatment&quot;

attr(,&quot;msgScaleX&quot;)
character(0)</code></pre>
<pre class="r"><code># Get results of hypothesis test on coefficients of interest
topTable( fitmm, coef=&#39;seurat_clusters1&#39;, number=3 )</code></pre>
<pre><code>           logFC   AveExpr        t      P.Value    adj.P.Val    z.std
SERBP1  7.936863  7.869018 305.2001 7.166887e-12 9.059661e-08 6.854287
RPL8   11.330302 11.059113 235.6734 2.610159e-11 1.649751e-07 6.667039
FAM50A  7.733634  7.589375 206.8750 5.007889e-11 1.966341e-07 6.570701</code></pre>
<pre class="r"><code>#next, figure out what the genes that separate each cluster mean...</code></pre>
<pre class="r"><code>#functions
get_results &lt;- function(x, number = nrow(x$coefficients), sort.by = &quot;none&quot;,
                        ...) {
  # x - object MArrayLM2 from dream output
  # ... - additional arguments passed to topTable
  stopifnot(class(x) == &quot;MArrayLM2&quot;)
  results &lt;- topTable(x, number = number, sort.by = sort.by, ...)
  return(results)
}

run_ash &lt;- function(x, coef) {
  # Perform multiple testing correction with adaptive shrinkage (ASH)
  #
  # x - object MArrayLM from eBayes output
  # coef - coefficient tested by eBayes
  stopifnot(class(x) == &quot;MArrayLM2&quot;, coef %in% colnames(x$coefficients))
  result &lt;- ash(betahat = x$coefficients[, coef],
                sebetahat = x$stdev.unscaled[, coef] * sqrt(x$sigma^2), #changed s2.post to sigma^2 since didn&#39;t run eBayes
                df = x$df.total[1])
  return(result)
}

plot_volcano &lt;- function(x, threshold) {
  # Create volcano plot.
  #
  # x - data frame with topTable and ASH output
  #     (columns logFC, P.Value, and adj.pval)
  # threshold - adj.pval cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;logFC&quot;, &quot;P.Value&quot;, &quot;adj.P.Val&quot;) %in% colnames(x),
            is.numeric(threshold), threshold &lt;= 1, threshold &gt;= 0)
  x$highlight &lt;- ifelse(x$adj.P.Val &lt; threshold, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = logFC, y = -log10(P.Value), color = highlight)) +
    geom_point(shape = 1) +
    labs(x = &quot;Log fold change&quot;,
         y = expression(-log[10] * &quot; p-value&quot;)) +
    scale_color_identity(drop = FALSE) +
    theme(legend.position = &quot;none&quot;)
}

plot_pval_hist &lt;- function(x, threshold) {
  # Create histogram of p-values.
  #
  # x - data frame with topTable and ash output (columns P.Value and adj.p.val)
  # threshold -  cutoff for calling a gene DE
  #
  stopifnot(is.data.frame(x), c(&quot;P.Value&quot;, &quot;adj.P.Val&quot;) %in% colnames(x))
  x$highlight &lt;- ifelse(x$adj.P.Val &lt; threshold, &quot;darkred&quot;, &quot;gray75&quot;)
  x$highlight &lt;- factor(x$highlight, levels = c(&quot;darkred&quot;, &quot;gray75&quot;))
  ggplot(x, aes(x = P.Value, fill = highlight)) +
    geom_histogram(position = &quot;stack&quot;, binwidth = 0.01) +
    scale_fill_identity(drop = FALSE) +
    labs(x = &quot;p-value&quot;, y = &quot;Number of genes&quot;)
}

#use functions to extract results
tests &lt;- colnames(fitmm$coefficients)
results &lt;- vector(length = length(tests), mode = &quot;list&quot;)
names(results) &lt;- tests

for (test in tests) {
  # Extract dream results
  results[[test]] &lt;- get_results(fitmm, coef = test)
  # Add mutliple testing correction with ASH
  output_ash &lt;- run_ash(fitmm, coef = test)$result
  results[[test]] &lt;- cbind(results[[test]], lfsr = output_ash$lfsr,
                           lfdr = output_ash$lfdr, qvalue = output_ash$qvalue,
                           svalue = output_ash$svalue)
}</code></pre>
<p>GO enrichments in DE genes Background set is ~12k genes in pb_matrix after filtering out lowly expressed genes</p>
<pre class="r"><code>library(&quot;topGO&quot;)</code></pre>
<pre><code>Loading required package: graph</code></pre>
<pre><code>Loading required package: GO.db</code></pre>
<pre><code>Loading required package: AnnotationDbi</code></pre>
<pre><code></code></pre>
<pre><code>Loading required package: SparseM</code></pre>
<pre><code>
Attaching package: &#39;SparseM&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    backsolve</code></pre>
<pre><code>
groupGOTerms:   GOBPTerm, GOMFTerm, GOCCTerm environments built.</code></pre>
<pre><code>
Attaching package: &#39;topGO&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:IRanges&#39;:

    members</code></pre>
<pre class="r"><code>threshold &lt;- 0.01 #adjusted p value threshold

#cluster 0 vs rest
gene_universe_0 &lt;- as.numeric(results[[&quot;seurat_clusters0&quot;]]$adj.P.Val &lt; threshold)
gene_universe_0 &lt;- factor(gene_universe_0)
names(gene_universe_0) &lt;- rownames(results[[&quot;seurat_clusters0&quot;]])
head(gene_universe_0)</code></pre>
<pre><code>AL669831.5 AL645608.1     SAMD11      NOC2L     KLHL17       HES4 
         0          0          0          1          0          1 
Levels: 0 1</code></pre>
<pre class="r"><code>plot_volcano(results[[&quot;seurat_clusters0&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot_pval_hist(results[[&quot;seurat_clusters0&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>go_data_0 &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_0,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;symbol&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>Loading required package: org.Hs.eg.db</code></pre>
<pre><code></code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>go_test_0 &lt;- runTest(go_data_0, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 7472 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  4 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  14 nodes to be scored   (13 eliminated genes)</code></pre>
<pre><code>
     Level 16:  32 nodes to be scored   (25 eliminated genes)</code></pre>
<pre><code>
     Level 15:  74 nodes to be scored   (83 eliminated genes)</code></pre>
<pre><code>
     Level 14:  156 nodes to be scored  (269 eliminated genes)</code></pre>
<pre><code>
     Level 13:  264 nodes to be scored  (701 eliminated genes)</code></pre>
<pre><code>
     Level 12:  405 nodes to be scored  (1616 eliminated genes)</code></pre>
<pre><code>
     Level 11:  677 nodes to be scored  (3289 eliminated genes)</code></pre>
<pre><code>
     Level 10:  886 nodes to be scored  (4917 eliminated genes)</code></pre>
<pre><code>
     Level 9:   1020 nodes to be scored (6147 eliminated genes)</code></pre>
<pre><code>
     Level 8:   1042 nodes to be scored (7502 eliminated genes)</code></pre>
<pre><code>
     Level 7:   1059 nodes to be scored (8366 eliminated genes)</code></pre>
<pre><code>
     Level 6:   884 nodes to be scored  (9031 eliminated genes)</code></pre>
<pre><code>
     Level 5:   537 nodes to be scored  (9444 eliminated genes)</code></pre>
<pre><code>
     Level 4:   274 nodes to be scored  (9713 eliminated genes)</code></pre>
<pre><code>
     Level 3:   120 nodes to be scored  (9883 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9930 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (10006 eliminated genes)</code></pre>
<pre class="r"><code>go_table_0 &lt;- GenTable(go_data_0, weightFisher = go_test_0,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test_0) &lt; .01))
go_table_0</code></pre>
<pre><code>         GO.ID                                        Term Annotated
1   GO:0006614 SRP-dependent cotranslational protein ta...        93
2   GO:0019083                         viral transcription       168
3   GO:0000184 nuclear-transcribed mRNA catabolic proce...       117
4   GO:0006413                    translational initiation       170
5   GO:0000398              mRNA splicing, via spliceosome       284
6   GO:0070125      mitochondrial translational elongation        85
7   GO:0032981 mitochondrial respiratory chain complex ...        58
8   GO:0070126     mitochondrial translational termination        87
9   GO:0006120 mitochondrial electron transport, NADH t...        46
10  GO:0043687     post-translational protein modification       273
11  GO:0043312                    neutrophil degranulation       316
12  GO:0016032                               viral process       672
13  GO:0006406                    mRNA export from nucleus        99
14  GO:0043488                regulation of mRNA stability       145
15  GO:1900034     regulation of cellular response to heat        72
16  GO:0006364                             rRNA processing       150
17  GO:1902036 regulation of hematopoietic stem cell di...        60
18  GO:0061418 regulation of transcription from RNA pol...        70
19  GO:0010972 negative regulation of G2/M transition o...        74
20  GO:0006457                             protein folding       155
21  GO:0031146 SCF-dependent proteasomal ubiquitin-depe...        65
22  GO:0060071 Wnt signaling pathway, planar cell polar...        92
23  GO:0002479 antigen processing and presentation of e...        60
24  GO:0000413       protein peptidyl-prolyl isomerization        26
25  GO:0006888      ER to Golgi vesicle-mediated transport       148
26  GO:0042147     retrograde transport, endosome to Golgi        64
27  GO:0000381 regulation of alternative mRNA splicing,...        32
28  GO:0038061                     NIK/NF-kappaB signaling       120
29  GO:0000209                  protein polyubiquitination       253
30  GO:0042795    snRNA transcription by RNA polymerase II        68
31  GO:0015031                           protein transport      1276
32  GO:0000715 nucleotide-excision repair, DNA damage r...        22
33  GO:0030433            ubiquitin-dependent ERAD pathway        57
34  GO:0002181                     cytoplasmic translation        63
35  GO:0006368 transcription elongation from RNA polyme...        72
36  GO:0006521 regulation of cellular amino acid metabo...        52
37  GO:0008380                                RNA splicing       368
38  GO:0006417                   regulation of translation       273
39  GO:0006890 retrograde vesicle-mediated transport, G...        68
40  GO:0045948 positive regulation of translational ini...        20
41  GO:0006099                    tricarboxylic acid cycle        26
42  GO:0032481 positive regulation of type I interferon...        59
43  GO:0048208                       COPII vesicle coating        57
44  GO:0031124                      mRNA 3&#39;-end processing        89
45  GO:0006886             intracellular protein transport       716
46  GO:0045727          positive regulation of translation        99
47  GO:0050434 positive regulation of viral transcripti...        40
48  GO:0016579                    protein deubiquitination       232
49  GO:0006110            regulation of glycolytic process        55
50  GO:0070936           protein K48-linked ubiquitination        51
51  GO:0008543 fibroblast growth factor receptor signal...        69
52  GO:0048025 negative regulation of mRNA splicing, vi...        17
53  GO:0045070 positive regulation of viral genome repl...        30
54  GO:0042771 intrinsic apoptotic signaling pathway in...        37
55  GO:0031145 anaphase-promoting complex-dependent cat...        74
56  GO:0033209 tumor necrosis factor-mediated signaling...       115
57  GO:0043928 exonucleolytic nuclear-transcribed mRNA ...        32
58  GO:0045815 positive regulation of gene expression, ...        44
59  GO:0071480        cellular response to gamma radiation        27
60  GO:0006370              7-methylguanosine mRNA capping        31
61  GO:0061077          chaperone-mediated protein folding        33
62  GO:0043044          ATP-dependent chromatin remodeling        56
63  GO:1904874 positive regulation of telomerase RNA lo...        15
64  GO:0006397                             mRNA processing       415
65  GO:0016241                regulation of macroautophagy       151
66  GO:0007030                          Golgi organization        87
67  GO:0060964       regulation of gene silencing by miRNA        82
68  GO:0032436 positive regulation of proteasomal ubiqu...        62
69  GO:0006363 termination of RNA polymerase I transcri...        30
70  GO:0045454                      cell redox homeostasis        47
71  GO:0075733            intracellular transport of virus        51
72  GO:1900026 positive regulation of substrate adhesio...        25
73  GO:1903508 positive regulation of nucleic acid-temp...      1037
74  GO:0009048 dosage compensation by inactivation of X...        14
75  GO:1902176 negative regulation of oxidative stress-...        14
76  GO:0061024                       membrane organization       540
77  GO:0034605                   cellular response to heat        96
78  GO:0006283 transcription-coupled nucleotide-excisio...        72
79  GO:0090263 positive regulation of canonical Wnt sig...       111
80  GO:0002223 stimulatory C-type lectin receptor signa...        75
81  GO:0018279 protein N-linked glycosylation via aspar...        28
82  GO:0006362 transcription elongation from RNA polyme...        28
83  GO:0016567                      protein ubiquitination       665
84  GO:0051028                              mRNA transport       131
85  GO:0030512 negative regulation of transforming grow...        56
86  GO:0006396                              RNA processing       738
87  GO:0031333 negative regulation of protein complex a...        88
88  GO:0035722 interleukin-12-mediated signaling pathwa...        36
89  GO:0007584                        response to nutrient       127
90  GO:0090090 negative regulation of canonical Wnt sig...       136
91  GO:0000045                      autophagosome assembly        64
92  GO:0016925                         protein sumoylation        68
93  GO:0006986                response to unfolded protein       147
94  GO:0000380 alternative mRNA splicing, via spliceoso...        44
95  GO:0006369 termination of RNA polymerase II transcr...        31
96  GO:0006409                    tRNA export from nucleus        31
97  GO:0001732 formation of cytoplasmic translation ini...        12
98  GO:0019827            stem cell population maintenance       118
99  GO:0007032                       endosome organization        66
100 GO:0016197                         endosomal transport       170
101 GO:0006997                        nucleus organization       101
102 GO:0000278                          mitotic cell cycle       743
103 GO:0042776 mitochondrial ATP synthesis coupled prot...        17
104 GO:0046827 positive regulation of protein export fr...        17
105 GO:0040014 regulation of multicellular organism gro...        38
106 GO:0006914                                   autophagy       405
107 GO:0061025                             membrane fusion        79
108 GO:0070979           protein K11-linked ubiquitination        26
109 GO:0042407                           cristae formation        26
110 GO:0016055                       Wnt signaling pathway       385
111 GO:0031397 negative regulation of protein ubiquitin...        64
112 GO:0006879               cellular iron ion homeostasis        42
113 GO:0036258                multivesicular body assembly        30
114 GO:0002576                      platelet degranulation        75
115 GO:0038095       Fc-epsilon receptor signaling pathway        89
116 GO:0006367 transcription initiation from RNA polyme...       141
117 GO:0042274          ribosomal small subunit biogenesis        24
118 GO:0039694                viral RNA genome replication        21
119 GO:0071044              histone mRNA catabolic process        11
120 GO:0043248                         proteasome assembly        11
121 GO:0006122 mitochondrial electron transport, ubiqui...        11
122 GO:0031065 positive regulation of histone deacetyla...        16
123 GO:0032922     circadian regulation of gene expression        49
124 GO:0034976    response to endoplasmic reticulum stress       222
125 GO:0007050                           cell cycle arrest       190
126 GO:0019886 antigen processing and presentation of e...        67
    Significant Expected weightFisher
1            91    60.65      1.8e-15
2           154   109.57      1.0e-14
3           111    76.31      1.2e-14
4           162   110.87      1.3e-13
5           258   185.22      4.9e-13
6            80    55.44      2.3e-10
7            57    37.83      5.1e-10
8            81    56.74      8.4e-10
9            45    30.00      2.3e-08
10          219   178.05      2.4e-08
11          250   206.10      3.1e-08
12          551   438.28      1.2e-07
13           87    64.57      1.3e-07
14          121    94.57      6.9e-07
15           65    46.96      9.0e-07
16          127    97.83      1.5e-06
17           55    39.13      1.9e-06
18           62    45.65      2.6e-06
19           63    48.26      3.8e-06
20          134   101.09      8.1e-06
21           58    42.39      8.9e-06
22           75    60.00      9.0e-06
23           54    39.13      1.0e-05
24           26    16.96      1.5e-05
25          128    96.53      1.7e-05
26           57    41.74      1.9e-05
27           31    20.87      2.0e-05
28           93    78.26      2.9e-05
29          197   165.01      4.7e-05
30           59    44.35      5.4e-05
31          950   832.21      7.6e-05
32           22    14.35      8.1e-05
33           51    37.18      0.00011
34           56    41.09      0.00012
35           63    46.96      0.00013
36           46    33.91      0.00013
37          331   240.01      0.00014
38          229   178.05      0.00014
39           58    44.35      0.00018
40           20    13.04      0.00019
41           25    16.96      0.00022
42           44    38.48      0.00038
43           49    37.18      0.00039
44           72    58.05      0.00039
45          576   466.97      0.00039
46           87    64.57      0.00046
47           35    26.09      0.00048
48          176   151.31      0.00064
49           45    35.87      0.00065
50           44    33.26      0.00065
51           54    45.00      0.00066
52           17    11.09      0.00069
53           28    19.57      0.00070
54           32    24.13      0.00070
55           61    48.26      0.00084
56           89    75.00      0.00086
57           29    20.87      0.00103
58           36    28.70      0.00104
59           25    17.61      0.00111
60           28    20.22      0.00145
61           30    21.52      0.00152
62           43    36.52      0.00153
63           15     9.78      0.00163
64          359   270.66      0.00183
65          116    98.48      0.00190
66           73    56.74      0.00197
67           69    53.48      0.00200
68           53    40.44      0.00200
69           27    19.57      0.00202
70           40    30.65      0.00204
71           43    33.26      0.00205
72           23    16.30      0.00226
73          704   676.33      0.00235
74           14     9.13      0.00251
75           14     9.13      0.00251
76          412   352.19      0.00274
77           87    62.61      0.00305
78           58    46.96      0.00326
79           86    72.39      0.00343
80           60    48.91      0.00377
81           25    18.26      0.00390
82           25    18.26      0.00390
83          509   433.71      0.00401
84          115    85.44      0.00403
85           46    36.52      0.00418
86          614   481.32      0.00452
87           65    57.39      0.00458
88           31    23.48      0.00458
89           88    82.83      0.00462
90          103    88.70      0.00522
91           50    41.74      0.00536
92           54    44.35      0.00560
93          122    95.87      0.00574
94           43    28.70      0.00579
95           27    20.22      0.00589
96           27    20.22      0.00589
97           12     7.83      0.00590
98           88    76.96      0.00599
99           56    43.05      0.00600
100         136   110.87      0.00673
101          82    65.87      0.00683
102         525   484.58      0.00695
103          16    11.09      0.00700
104          16    11.09      0.00700
105          27    24.78      0.00704
106         310   264.14      0.00717
107          63    51.52      0.00732
108          23    16.96      0.00743
109          23    16.96      0.00743
110         276   251.10      0.00747
111          54    41.74      0.00776
112          35    27.39      0.00788
113          26    19.57      0.00796
114          59    48.91      0.00807
115          69    58.05      0.00814
116         107    91.96      0.00826
117          23    15.65      0.00900
118          18    13.70      0.00905
119          11     7.17      0.00906
120          11     7.17      0.00906
121          11     7.17      0.00906
122          13    10.44      0.00907
123          40    31.96      0.00917
124         188   144.79      0.00922
125         140   123.92      0.00938
126          53    43.70      0.00969</code></pre>
<pre class="r"><code># k-s test making use of ranked based on p-values
genes_0 &lt;- results[[&quot;seurat_clusters0&quot;]]$adj.P.Val
names(genes_0) &lt;- rownames(results[[&quot;seurat_clusters0&quot;]])

selection &lt;- function(allScore){ return(allScore &lt; 0.05)} # function that returns TRUE/FALSE for p-values&lt;0.05
allGO2genes &lt;- annFUN.org(whichOnto=&quot;BP&quot;, feasibleGenes=NULL, mapping=&quot;org.Hs.eg.db&quot;, ID=&quot;symbol&quot;)
GOdata &lt;- new(&quot;topGOdata&quot;,
  ontology=&quot;BP&quot;,
  allGenes=genes_0,
  annot=annFUN.GO2genes,
  GO2genes=allGO2genes,
  geneSel=selection,
  nodeSize=10)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>results.ks &lt;- runTest(GOdata, algorithm=&quot;classic&quot;, statistic=&quot;ks&quot;)</code></pre>
<pre><code>
             -- Classic Algorithm -- 

         the algorithm is scoring 5371 nontrivial nodes
         parameters: 
             test statistic: ks
             score order: increasing</code></pre>
<pre class="r"><code>goEnrichment &lt;- GenTable(GOdata, KS=results.ks, orderBy=&quot;KS&quot;, topNodes=20)
goEnrichment &lt;- goEnrichment %&gt;% 
     dplyr::mutate(KS = ifelse(grepl(&quot;&lt;&quot;, KS), 1e-30, KS))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)
goEnrichment &lt;- goEnrichment[goEnrichment$KS&lt;0.05,]
goEnrichment &lt;- goEnrichment[,c(&quot;GO.ID&quot;,&quot;Term&quot;,&quot;KS&quot;)]
goEnrichment$Term &lt;- gsub(&quot; [a-z]*\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- gsub(&quot;\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- paste(goEnrichment$GO.ID, goEnrichment$Term, sep=&quot;, &quot;)
goEnrichment$Term &lt;- factor(goEnrichment$Term, levels=rev(goEnrichment$Term))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)

require(ggplot2)
ggplot(goEnrichment, aes(x=Term, y=-log10(KS))) +
    stat_summary(geom = &quot;bar&quot;, fun = mean, position = &quot;dodge&quot;) +
    xlab(&quot;Biological process&quot;) +
    ylab(&quot;Enrichment&quot;) +
    ggtitle(&quot;Title&quot;) +
    scale_y_continuous(breaks = round(seq(0, max(-log10(goEnrichment$KS)), by = 2), 1)) +
    theme_bw(base_size=24) +
    theme(
        legend.position=&#39;none&#39;,
        legend.background=element_rect(),
        plot.title=element_text(angle=0, size=24, face=&quot;bold&quot;, vjust=1),
        axis.text.x=element_text(angle=0, size=18, face=&quot;bold&quot;, hjust=1.10),
        axis.text.y=element_text(angle=0, size=18, face=&quot;bold&quot;, vjust=0.5),
        axis.title=element_text(size=24, face=&quot;bold&quot;),
        legend.key=element_blank(),     #removes the border
        legend.key.size=unit(1, &quot;cm&quot;),      #Sets overall area/size of the legend
        legend.text=element_text(size=18),  #Text size
        title=element_text(size=18)) +
    guides(colour=guide_legend(override.aes=list(size=2.5))) +
    coord_flip()</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#cluster 1 vs rest
gene_universe_1 &lt;- as.numeric(results[[&quot;seurat_clusters1&quot;]]$adj.P.Val &lt; threshold)
gene_universe_1 &lt;- factor(gene_universe_1)
names(gene_universe_1) &lt;- rownames(results[[&quot;seurat_clusters1&quot;]])
head(gene_universe_1)</code></pre>
<pre><code>AL669831.5 AL645608.1     SAMD11      NOC2L     KLHL17       HES4 
         0          0          1          1          1          1 
Levels: 0 1</code></pre>
<pre class="r"><code>plot_volcano(results[[&quot;seurat_clusters1&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot_pval_hist(results[[&quot;seurat_clusters1&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>go_data_1 &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_1,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;symbol&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>go_test_1 &lt;- runTest(go_data_1, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 7484 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  4 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  14 nodes to be scored   (13 eliminated genes)</code></pre>
<pre><code>
     Level 16:  32 nodes to be scored   (25 eliminated genes)</code></pre>
<pre><code>
     Level 15:  74 nodes to be scored   (83 eliminated genes)</code></pre>
<pre><code>
     Level 14:  157 nodes to be scored  (269 eliminated genes)</code></pre>
<pre><code>
     Level 13:  263 nodes to be scored  (701 eliminated genes)</code></pre>
<pre><code>
     Level 12:  407 nodes to be scored  (1617 eliminated genes)</code></pre>
<pre><code>
     Level 11:  679 nodes to be scored  (3287 eliminated genes)</code></pre>
<pre><code>
     Level 10:  889 nodes to be scored  (4915 eliminated genes)</code></pre>
<pre><code>
     Level 9:   1021 nodes to be scored (6148 eliminated genes)</code></pre>
<pre><code>
     Level 8:   1043 nodes to be scored (7502 eliminated genes)</code></pre>
<pre><code>
     Level 7:   1059 nodes to be scored (8366 eliminated genes)</code></pre>
<pre><code>
     Level 6:   886 nodes to be scored  (9031 eliminated genes)</code></pre>
<pre><code>
     Level 5:   537 nodes to be scored  (9444 eliminated genes)</code></pre>
<pre><code>
     Level 4:   275 nodes to be scored  (9713 eliminated genes)</code></pre>
<pre><code>
     Level 3:   120 nodes to be scored  (9883 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9930 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (10006 eliminated genes)</code></pre>
<pre class="r"><code>go_table_1 &lt;- GenTable(go_data_1, weightFisher = go_test_1,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test_1) &lt; .01))
go_table_1</code></pre>
<pre><code>         GO.ID                                        Term Annotated
1   GO:0000398              mRNA splicing, via spliceosome       284
2   GO:0000184 nuclear-transcribed mRNA catabolic proce...       117
3   GO:0019083                         viral transcription       168
4   GO:0006614 SRP-dependent cotranslational protein ta...        93
5   GO:0006413                    translational initiation       170
6   GO:0070126     mitochondrial translational termination        87
7   GO:0032981 mitochondrial respiratory chain complex ...        58
8   GO:0070125      mitochondrial translational elongation        85
9   GO:0043687     post-translational protein modification       273
10  GO:0043488                regulation of mRNA stability       145
11  GO:1900034     regulation of cellular response to heat        72
12  GO:0042795    snRNA transcription by RNA polymerase II        68
13  GO:0043312                    neutrophil degranulation       316
14  GO:0006888      ER to Golgi vesicle-mediated transport       148
15  GO:0006406                    mRNA export from nucleus        99
16  GO:0006368 transcription elongation from RNA polyme...        72
17  GO:0006521 regulation of cellular amino acid metabo...        52
18  GO:0002479 antigen processing and presentation of e...        60
19  GO:0006886             intracellular protein transport       716
20  GO:0006120 mitochondrial electron transport, NADH t...        46
21  GO:0042147     retrograde transport, endosome to Golgi        64
22  GO:0061418 regulation of transcription from RNA pol...        70
23  GO:0006364                             rRNA processing       150
24  GO:0016567                      protein ubiquitination       665
25  GO:0006890 retrograde vesicle-mediated transport, G...        68
26  GO:0010972 negative regulation of G2/M transition o...        74
27  GO:1902036 regulation of hematopoietic stem cell di...        60
28  GO:0006457                             protein folding       155
29  GO:0006283 transcription-coupled nucleotide-excisio...        72
30  GO:0016032                               viral process       672
31  GO:0043928 exonucleolytic nuclear-transcribed mRNA ...        32
32  GO:0006370              7-methylguanosine mRNA capping        31
33  GO:0007030                          Golgi organization        87
34  GO:0000209                  protein polyubiquitination       253
35  GO:1903508 positive regulation of nucleic acid-temp...      1037
36  GO:0006110            regulation of glycolytic process        55
37  GO:0038061                     NIK/NF-kappaB signaling       120
38  GO:0075733            intracellular transport of virus        51
39  GO:0015031                           protein transport      1276
40  GO:0048208                       COPII vesicle coating        57
41  GO:0002223 stimulatory C-type lectin receptor signa...        75
42  GO:0006417                   regulation of translation       273
43  GO:0008380                                RNA splicing       368
44  GO:0006397                             mRNA processing       415
45  GO:0050434 positive regulation of viral transcripti...        40
46  GO:0051028                              mRNA transport       131
47  GO:0000381 regulation of alternative mRNA splicing,...        32
48  GO:0000245               spliceosomal complex assembly        43
49  GO:0000387                 spliceosomal snRNP assembly        33
50  GO:0006409                    tRNA export from nucleus        31
51  GO:0016925                         protein sumoylation        68
52  GO:0070936           protein K48-linked ubiquitination        51
53  GO:0031145 anaphase-promoting complex-dependent cat...        74
54  GO:0060071 Wnt signaling pathway, planar cell polar...        92
55  GO:0090263 positive regulation of canonical Wnt sig...       111
56  GO:0000715 nucleotide-excision repair, DNA damage r...        22
57  GO:0032481 positive regulation of type I interferon...        59
58  GO:0006418 tRNA aminoacylation for protein translat...        42
59  GO:0002181                     cytoplasmic translation        63
60  GO:0000278                          mitotic cell cycle       743
61  GO:1901998                             toxin transport        29
62  GO:0007584                        response to nutrient       127
63  GO:0006367 transcription initiation from RNA polyme...       141
64  GO:0030433            ubiquitin-dependent ERAD pathway        57
65  GO:0006897                                 endocytosis       387
66  GO:0031146 SCF-dependent proteasomal ubiquitin-depe...        65
67  GO:0016241                regulation of macroautophagy       151
68  GO:0032008        positive regulation of TOR signaling        31
69  GO:0045948 positive regulation of translational ini...        20
70  GO:0033209 tumor necrosis factor-mediated signaling...       115
71  GO:0090502 RNA phosphodiester bond hydrolysis, endo...        52
72  GO:0002576                      platelet degranulation        75
73  GO:0007050                           cell cycle arrest       190
74  GO:0071480        cellular response to gamma radiation        27
75  GO:0006383         transcription by RNA polymerase III        49
76  GO:0099132 ATP hydrolysis coupled cation transmembr...        39
77  GO:0034067     protein localization to Golgi apparatus        19
78  GO:0038095       Fc-epsilon receptor signaling pathway        89
79  GO:0008543 fibroblast growth factor receptor signal...        69
80  GO:0006513                  protein monoubiquitination        60
81  GO:0001558                   regulation of cell growth       268
82  GO:0008542                             visual learning        26
83  GO:0000413       protein peptidyl-prolyl isomerization        26
84  GO:0042407                           cristae formation        26
85  GO:0006622               protein targeting to lysosome        18
86  GO:0016197                         endosomal transport       170
87  GO:0006361 transcription initiation from RNA polyme...        34
88  GO:0070911    global genome nucleotide-excision repair        25
89  GO:0042776 mitochondrial ATP synthesis coupled prot...        17
90  GO:0048025 negative regulation of mRNA splicing, vi...        17
91  GO:0071243 cellular response to arsenic-containing ...        17
92  GO:0032922     circadian regulation of gene expression        49
93  GO:0070534           protein K63-linked ubiquitination        39
94  GO:0050852           T cell receptor signaling pathway       119
95  GO:0016579                    protein deubiquitination       232
96  GO:0006363 termination of RNA polymerase I transcri...        30
97  GO:1904380      endoplasmic reticulum mannose trimming        16
98  GO:0031333 negative regulation of protein complex a...        88
99  GO:0006396                              RNA processing       738
100 GO:0045070 positive regulation of viral genome repl...        30
    Significant Expected weightFisher
1           273   211.33      3.0e-12
2           114    87.06      9.1e-12
3           159   125.01      4.2e-11
4            91    69.20      1.4e-10
5           165   126.50      1.1e-08
6            84    64.74      3.0e-08
7            58    43.16      3.4e-08
8            82    63.25      5.0e-08
9           239   203.14      6.5e-08
10          133   107.90      7.6e-08
11           70    53.58      1.8e-07
12           66    50.60      5.2e-07
13          270   235.14      1.0e-06
14          139   110.13      1.2e-06
15           92    73.67      2.1e-06
16           69    53.58      3.8e-06
17           51    38.69      3.8e-06
18           58    44.65      4.4e-06
19          626   532.79      4.8e-06
20           45    34.23      5.3e-06
21           61    47.62      7.4e-06
22           66    52.09      1.1e-05
23          136   111.62      1.2e-05
24          570   494.84      2.4e-05
25           64    50.60      2.4e-05
26           65    55.06      2.8e-05
27           57    44.65      3.1e-05
28          141   115.34      4.1e-05
29           67    53.58      4.6e-05
30          580   500.05      6.1e-05
31           32    23.81      7.7e-05
32           31    23.07      0.00010
33           80    64.74      0.00010
34          219   188.26      0.00011
35          791   771.65      0.00014
36           49    40.93      0.00018
37           98    89.29      0.00022
38           48    37.95      0.00028
39         1043   949.50      0.00032
40           53    42.41      0.00032
41           68    55.81      0.00036
42          245   203.14      0.00040
43          350   273.84      0.00067
44          388   308.81      0.00079
45           38    29.76      0.00082
46          123    97.48      0.00086
47           31    23.81      0.00093
48           42    32.00      0.00109
49           33    24.56      0.00110
50           30    23.07      0.00121
51           63    50.60      0.00124
52           47    37.95      0.00126
53           66    55.06      0.00134
54           78    68.46      0.00136
55           96    82.60      0.00142
56           22    16.37      0.00149
57           49    43.90      0.00159
58           39    31.25      0.00162
59           58    46.88      0.00197
60          592   552.88      0.00203
61           28    21.58      0.00206
62          100    94.50      0.00209
63          121   104.92      0.00227
64           53    42.41      0.00240
65          304   287.97      0.00249
66           58    48.37      0.00258
67          130   112.36      0.00267
68           30    23.07      0.00268
69           20    14.88      0.00269
70           99    85.57      0.00279
71           47    38.69      0.00300
72           66    55.81      0.00306
73          156   141.38      0.00347
74           26    20.09      0.00349
75           45    36.46      0.00359
76           35    29.02      0.00361
77           19    14.14      0.00362
78           77    66.23      0.00409
79           59    51.34      0.00443
80           55    44.65      0.00447
81          213   199.42      0.00449
82           25    19.35      0.00454
83           25    19.35      0.00454
84           25    19.35      0.00454
85           18    13.39      0.00487
86          151   126.50      0.00503
87           32    25.30      0.00587
88           24    18.60      0.00589
89           17    12.65      0.00655
90           17    12.65      0.00655
91           17    12.65      0.00655
92           44    36.46      0.00659
93           36    29.02      0.00874
94          101    88.55      0.00877
95          191   172.64      0.00877
96           28    22.32      0.00878
97           16    11.91      0.00880
98           69    65.48      0.00905
99          674   549.16      0.00926
100          29    22.32      0.00984</code></pre>
<pre class="r"><code># k-s test making use of ranked based on p-values
genes_1 &lt;- results[[&quot;seurat_clusters1&quot;]]$adj.P.Val
names(genes_1) &lt;- rownames(results[[&quot;seurat_clusters0&quot;]])

selection &lt;- function(allScore){ return(allScore &lt; 0.05)} # function that returns TRUE/FALSE for p-values&lt;0.05
allGO2genes &lt;- annFUN.org(whichOnto=&quot;BP&quot;, feasibleGenes=NULL, mapping=&quot;org.Hs.eg.db&quot;, ID=&quot;symbol&quot;)
GOdata &lt;- new(&quot;topGOdata&quot;,
  ontology=&quot;BP&quot;,
  allGenes=genes_1,
  annot=annFUN.GO2genes,
  GO2genes=allGO2genes,
  geneSel=selection,
  nodeSize=10)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>results.ks &lt;- runTest(GOdata, algorithm=&quot;classic&quot;, statistic=&quot;ks&quot;)</code></pre>
<pre><code>
             -- Classic Algorithm -- 

         the algorithm is scoring 5371 nontrivial nodes
         parameters: 
             test statistic: ks
             score order: increasing</code></pre>
<pre class="r"><code>goEnrichment &lt;- GenTable(GOdata, KS=results.ks, orderBy=&quot;KS&quot;, topNodes=20)
goEnrichment &lt;- goEnrichment %&gt;% 
     dplyr::mutate(KS = ifelse(grepl(&quot;&lt;&quot;, KS), 1e-30, KS))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)
goEnrichment &lt;- goEnrichment[goEnrichment$KS&lt;0.05,]
goEnrichment &lt;- goEnrichment[,c(&quot;GO.ID&quot;,&quot;Term&quot;,&quot;KS&quot;)]
goEnrichment$Term &lt;- gsub(&quot; [a-z]*\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- gsub(&quot;\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- paste(goEnrichment$GO.ID, goEnrichment$Term, sep=&quot;, &quot;)
goEnrichment$Term &lt;- factor(goEnrichment$Term, levels=rev(goEnrichment$Term))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)

require(ggplot2)
ggplot(goEnrichment, aes(x=Term, y=-log10(KS))) +
    stat_summary(geom = &quot;bar&quot;, fun = mean, position = &quot;dodge&quot;) +
    xlab(&quot;Biological process&quot;) +
    ylab(&quot;Enrichment&quot;) +
    ggtitle(&quot;Title&quot;) +
    scale_y_continuous(breaks = round(seq(0, max(-log10(goEnrichment$KS)), by = 2), 1)) +
    theme_bw(base_size=24) +
    theme(
        legend.position=&#39;none&#39;,
        legend.background=element_rect(),
        plot.title=element_text(angle=0, size=24, face=&quot;bold&quot;, vjust=1),
        axis.text.x=element_text(angle=0, size=18, face=&quot;bold&quot;, hjust=1.10),
        axis.text.y=element_text(angle=0, size=18, face=&quot;bold&quot;, vjust=0.5),
        axis.title=element_text(size=24, face=&quot;bold&quot;),
        legend.key=element_blank(),     #removes the border
        legend.key.size=unit(1, &quot;cm&quot;),      #Sets overall area/size of the legend
        legend.text=element_text(size=18),  #Text size
        title=element_text(size=18)) +
    guides(colour=guide_legend(override.aes=list(size=2.5))) +
    coord_flip()</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#cluster 2 vs rest
gene_universe_2 &lt;- as.numeric(results[[&quot;seurat_clusters2&quot;]]$adj.P.Val &lt; threshold)
gene_universe_2 &lt;- factor(gene_universe_2)
names(gene_universe_2) &lt;- rownames(results[[&quot;seurat_clusters2&quot;]])
head(gene_universe_2)</code></pre>
<pre><code>AL669831.5 AL645608.1     SAMD11      NOC2L     KLHL17       HES4 
         0          0          1          1          1          1 
Levels: 0 1</code></pre>
<pre class="r"><code>plot_volcano(results[[&quot;seurat_clusters2&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot_pval_hist(results[[&quot;seurat_clusters2&quot;]], threshold)</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>go_data_2 &lt;- new(&quot;topGOdata&quot;,
               ontology = &quot;BP&quot;,
               allGenes = gene_universe_2,
               nodeSize = 5,
               annotationFun = annFUN.org,
               mapping = &quot;org.Hs.eg.db&quot;,
               ID = &quot;symbol&quot;)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>go_test_2 &lt;- runTest(go_data_2, algorithm = &quot;weight01&quot;, statistic = &quot;fisher&quot;)</code></pre>
<pre><code>
             -- Weight01 Algorithm -- 

         the algorithm is scoring 7485 nontrivial nodes
         parameters: 
             test statistic: fisher</code></pre>
<pre><code>
     Level 19:  1 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 18:  4 nodes to be scored    (0 eliminated genes)</code></pre>
<pre><code>
     Level 17:  14 nodes to be scored   (13 eliminated genes)</code></pre>
<pre><code>
     Level 16:  32 nodes to be scored   (25 eliminated genes)</code></pre>
<pre><code>
     Level 15:  74 nodes to be scored   (83 eliminated genes)</code></pre>
<pre><code>
     Level 14:  157 nodes to be scored  (269 eliminated genes)</code></pre>
<pre><code>
     Level 13:  263 nodes to be scored  (701 eliminated genes)</code></pre>
<pre><code>
     Level 12:  407 nodes to be scored  (1617 eliminated genes)</code></pre>
<pre><code>
     Level 11:  680 nodes to be scored  (3287 eliminated genes)</code></pre>
<pre><code>
     Level 10:  889 nodes to be scored  (4915 eliminated genes)</code></pre>
<pre><code>
     Level 9:   1021 nodes to be scored (6148 eliminated genes)</code></pre>
<pre><code>
     Level 8:   1043 nodes to be scored (7502 eliminated genes)</code></pre>
<pre><code>
     Level 7:   1059 nodes to be scored (8366 eliminated genes)</code></pre>
<pre><code>
     Level 6:   886 nodes to be scored  (9031 eliminated genes)</code></pre>
<pre><code>
     Level 5:   537 nodes to be scored  (9444 eliminated genes)</code></pre>
<pre><code>
     Level 4:   275 nodes to be scored  (9713 eliminated genes)</code></pre>
<pre><code>
     Level 3:   120 nodes to be scored  (9883 eliminated genes)</code></pre>
<pre><code>
     Level 2:   22 nodes to be scored   (9930 eliminated genes)</code></pre>
<pre><code>
     Level 1:   1 nodes to be scored    (10006 eliminated genes)</code></pre>
<pre class="r"><code>go_table_2 &lt;- GenTable(go_data_2, weightFisher = go_test_2,
                     orderBy = &quot;weightFisher&quot;, ranksOf = &quot;weightFisher&quot;,
                     topNodes = sum(score(go_test_2) &lt; .01))
go_table_2</code></pre>
<pre><code>         GO.ID                                        Term Annotated
1   GO:0000184 nuclear-transcribed mRNA catabolic proce...       117
2   GO:0000398              mRNA splicing, via spliceosome       284
3   GO:0019083                         viral transcription       168
4   GO:0006614 SRP-dependent cotranslational protein ta...        93
5   GO:0006413                    translational initiation       170
6   GO:0006406                    mRNA export from nucleus        99
7   GO:0042147     retrograde transport, endosome to Golgi        64
8   GO:0070126     mitochondrial translational termination        87
9   GO:0070125      mitochondrial translational elongation        85
10  GO:0032981 mitochondrial respiratory chain complex ...        58
11  GO:0043687     post-translational protein modification       273
12  GO:1900034     regulation of cellular response to heat        72
13  GO:0006364                             rRNA processing       150
14  GO:0006368 transcription elongation from RNA polyme...        72
15  GO:0016032                               viral process       672
16  GO:0006120 mitochondrial electron transport, NADH t...        46
17  GO:0043312                    neutrophil degranulation       316
18  GO:0006888      ER to Golgi vesicle-mediated transport       148
19  GO:0043488                regulation of mRNA stability       145
20  GO:0075733            intracellular transport of virus        51
21  GO:0042795    snRNA transcription by RNA polymerase II        68
22  GO:0006370              7-methylguanosine mRNA capping        31
23  GO:0008380                                RNA splicing       368
24  GO:0006886             intracellular protein transport       716
25  GO:0006363 termination of RNA polymerase I transcri...        30
26  GO:0002479 antigen processing and presentation of e...        60
27  GO:0006110            regulation of glycolytic process        55
28  GO:0015031                           protein transport      1276
29  GO:0061418 regulation of transcription from RNA pol...        70
30  GO:0006362 transcription elongation from RNA polyme...        28
31  GO:0016197                         endosomal transport       170
32  GO:0031333 negative regulation of protein complex a...        88
33  GO:0007030                          Golgi organization        87
34  GO:0042407                           cristae formation        26
35  GO:1902036 regulation of hematopoietic stem cell di...        60
36  GO:0006361 transcription initiation from RNA polyme...        34
37  GO:0050434 positive regulation of viral transcripti...        40
38  GO:0000381 regulation of alternative mRNA splicing,...        32
39  GO:0043928 exonucleolytic nuclear-transcribed mRNA ...        32
40  GO:0000245               spliceosomal complex assembly        43
41  GO:0070936           protein K48-linked ubiquitination        51
42  GO:0008543 fibroblast growth factor receptor signal...        69
43  GO:0006409                    tRNA export from nucleus        31
44  GO:0016567                      protein ubiquitination       665
45  GO:0006890 retrograde vesicle-mediated transport, G...        68
46  GO:0031124                      mRNA 3&#39;-end processing        89
47  GO:0030512 negative regulation of transforming grow...        56
48  GO:0000715 nucleotide-excision repair, DNA damage r...        22
49  GO:0032481 positive regulation of type I interferon...        59
50  GO:0006418 tRNA aminoacylation for protein translat...        42
51  GO:0036498     IRE1-mediated unfolded protein response        60
52  GO:0006417                   regulation of translation       273
53  GO:0002181                     cytoplasmic translation        63
54  GO:0006397                             mRNA processing       415
55  GO:0030433            ubiquitin-dependent ERAD pathway        57
56  GO:0006897                                 endocytosis       387
57  GO:0006457                             protein folding       155
58  GO:0018279 protein N-linked glycosylation via aspar...        28
59  GO:0010972 negative regulation of G2/M transition o...        74
60  GO:0045948 positive regulation of translational ini...        20
61  GO:0045454                      cell redox homeostasis        47
62  GO:0000209                  protein polyubiquitination       253
63  GO:0045727          positive regulation of translation        99
64  GO:0042149     cellular response to glucose starvation        34
65  GO:1903508 positive regulation of nucleic acid-temp...      1037
66  GO:0006521 regulation of cellular amino acid metabo...        52
67  GO:0038061                     NIK/NF-kappaB signaling       120
68  GO:0071480        cellular response to gamma radiation        27
69  GO:0006383         transcription by RNA polymerase III        49
70  GO:0099132 ATP hydrolysis coupled cation transmembr...        39
71  GO:0034067     protein localization to Golgi apparatus        19
72  GO:0016925                         protein sumoylation        68
73  GO:0060964       regulation of gene silencing by miRNA        82
74  GO:0061024                       membrane organization       540
75  GO:0000413       protein peptidyl-prolyl isomerization        26
76  GO:0006099                    tricarboxylic acid cycle        26
77  GO:0006283 transcription-coupled nucleotide-excisio...        72
78  GO:0090263 positive regulation of canonical Wnt sig...       111
79  GO:0045815 positive regulation of gene expression, ...        44
80  GO:0050821                       protein stabilization       144
81  GO:0035735 intraciliary transport involved in ciliu...        38
82  GO:0034614 cellular response to reactive oxygen spe...       115
83  GO:0070911    global genome nucleotide-excision repair        25
84  GO:0070527                        platelet aggregation        36
85  GO:0031146 SCF-dependent proteasomal ubiquitin-depe...        65
86  GO:0042776 mitochondrial ATP synthesis coupled prot...        17
87  GO:0048025 negative regulation of mRNA splicing, vi...        17
88  GO:0016241                regulation of macroautophagy       151
89  GO:0036258                multivesicular body assembly        30
90  GO:0007584                        response to nutrient       127
91  GO:0006509     membrane protein ectodomain proteolysis        29
92  GO:1902236 negative regulation of endoplasmic retic...        16
93  GO:1904380      endoplasmic reticulum mannose trimming        16
94  GO:0090502 RNA phosphodiester bond hydrolysis, endo...        52
95  GO:0006396                              RNA processing       738
96  GO:1990090 cellular response to nerve growth factor...        35
97  GO:0000387                 spliceosomal snRNP assembly        33
98  GO:0045070 positive regulation of viral genome repl...        30
99  GO:0048208                       COPII vesicle coating        57
100 GO:0031334 positive regulation of protein complex a...       172
    Significant Expected weightFisher
1           115    86.42      2.9e-13
2           271   209.76      1.2e-12
3           158   124.09      2.0e-11
4            91    68.69      7.5e-11
5           166   125.56      6.7e-10
6            95    73.12      4.2e-09
7            63    47.27      2.2e-08
8            83    64.26      1.3e-07
9            81    62.78      2.2e-07
10           57    42.84      4.8e-07
11          235   201.64      7.5e-07
12           69    53.18      9.4e-07
13          138   110.79      1.6e-06
14           68    53.18      2.6e-06
15          586   496.34      2.9e-06
16           45    33.98      3.9e-06
17          265   233.40      1.2e-05
18          135   109.31      1.9e-05
19          130   107.10      2.6e-05
20           49    37.67      3.4e-05
21           63    50.23      7.9e-05
22           31    22.90      8.2e-05
23          350   271.81      8.8e-05
24          619   528.84      0.00010
25           30    22.16      0.00011
26           56    44.32      0.00012
27           49    40.62      0.00014
28         1036   942.46      0.00016
29           64    51.70      0.00020
30           28    20.68      0.00020
31          156   125.56      0.00024
32           70    65.00      0.00025
33           79    64.26      0.00037
34           26    19.20      0.00037
35           55    44.32      0.00049
36           34    25.11      0.00050
37           38    29.54      0.00068
38           31    23.64      0.00075
39           31    23.64      0.00075
40           41    31.76      0.00092
41           47    37.67      0.00096
42           58    50.96      0.00097
43           30    22.90      0.00099
44          555   491.17      0.00104
45           61    50.23      0.00105
46           81    65.74      0.00112
47           51    41.36      0.00119
48           22    16.25      0.00126
49           50    43.58      0.00130
50           39    31.02      0.00130
51           54    44.32      0.00150
52          245   201.64      0.00159
53           59    46.53      0.00168
54          386   306.52      0.00171
55           53    42.10      0.00187
56          299   285.84      0.00189
57          138   114.48      0.00192
58           27    20.68      0.00223
59           62    54.66      0.00228
60           20    14.77      0.00232
61           43    34.71      0.00236
62          211   186.87      0.00261
63           91    73.12      0.00268
64           32    25.11      0.00276
65          799   765.94      0.00279
66           47    38.41      0.00280
67           95    88.63      0.00283
68           26    19.94      0.00293
69           43    36.19      0.00313
70           34    28.81      0.00314
71           19    14.03      0.00314
72           62    50.23      0.00340
73           72    60.57      0.00340
74          447   398.85      0.00382
75           25    19.20      0.00384
76           25    19.20      0.00384
77           63    53.18      0.00386
78           94    81.99      0.00445
79           39    32.50      0.00453
80          120   106.36      0.00456
81           35    28.07      0.00470
82           95    84.94      0.00495
83           24    18.47      0.00502
84           30    26.59      0.00505
85           57    48.01      0.00533
86           17    12.56      0.00577
87           17    12.56      0.00577
88          130   111.53      0.00585
89           28    22.16      0.00739
90           95    93.80      0.00755
91           26    21.42      0.00781
92           16    11.82      0.00781
93           16    11.82      0.00781
94           46    38.41      0.00785
95          675   545.09      0.00793
96           34    25.85      0.00848
97           32    24.37      0.00849
98           29    22.16      0.00851
99           50    42.10      0.00889
100         145   127.04      0.00907</code></pre>
<pre class="r"><code># k-s test making use of ranked based on p-values
genes_2 &lt;- results[[&quot;seurat_clusters0&quot;]]$adj.P.Val
names(genes_2) &lt;- rownames(results[[&quot;seurat_clusters2&quot;]])

selection &lt;- function(allScore){ return(allScore &lt; 0.05)} # function that returns TRUE/FALSE for p-values&lt;0.05
allGO2genes &lt;- annFUN.org(whichOnto=&quot;BP&quot;, feasibleGenes=NULL, mapping=&quot;org.Hs.eg.db&quot;, ID=&quot;symbol&quot;)
GOdata &lt;- new(&quot;topGOdata&quot;,
  ontology=&quot;BP&quot;,
  allGenes=genes_2,
  annot=annFUN.GO2genes,
  GO2genes=allGO2genes,
  geneSel=selection,
  nodeSize=10)</code></pre>
<pre><code>
Building most specific GOs .....</code></pre>
<pre><code>    ( 10086 GO terms found. )</code></pre>
<pre><code>
Build GO DAG topology ..........</code></pre>
<pre><code>    ( 14189 GO terms and 33511 relations. )</code></pre>
<pre><code>
Annotating nodes ...............</code></pre>
<pre><code>    ( 10161 genes annotated to the GO terms. )</code></pre>
<pre class="r"><code>results.ks &lt;- runTest(GOdata, algorithm=&quot;classic&quot;, statistic=&quot;ks&quot;)</code></pre>
<pre><code>
             -- Classic Algorithm -- 

         the algorithm is scoring 5371 nontrivial nodes
         parameters: 
             test statistic: ks
             score order: increasing</code></pre>
<pre class="r"><code>goEnrichment &lt;- GenTable(GOdata, KS=results.ks, orderBy=&quot;KS&quot;, topNodes=20)
goEnrichment &lt;- goEnrichment %&gt;% 
     dplyr::mutate(KS = ifelse(grepl(&quot;&lt;&quot;, KS), 1e-30, KS))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)
goEnrichment &lt;- goEnrichment[goEnrichment$KS&lt;0.05,]
goEnrichment &lt;- goEnrichment[,c(&quot;GO.ID&quot;,&quot;Term&quot;,&quot;KS&quot;)]
goEnrichment$Term &lt;- gsub(&quot; [a-z]*\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- gsub(&quot;\\.\\.\\.$&quot;, &quot;&quot;, goEnrichment$Term)
goEnrichment$Term &lt;- paste(goEnrichment$GO.ID, goEnrichment$Term, sep=&quot;, &quot;)
goEnrichment$Term &lt;- factor(goEnrichment$Term, levels=rev(goEnrichment$Term))
goEnrichment$KS &lt;- as.numeric(goEnrichment$KS)

require(ggplot2)
ggplot(goEnrichment, aes(x=Term, y=-log10(KS))) +
    stat_summary(geom = &quot;bar&quot;, fun = mean, position = &quot;dodge&quot;) +
    xlab(&quot;Biological process&quot;) +
    ylab(&quot;Enrichment&quot;) +
    ggtitle(&quot;Title&quot;) +
    scale_y_continuous(breaks = round(seq(0, max(-log10(goEnrichment$KS)), by = 2), 1)) +
    theme_bw(base_size=24) +
    theme(
        legend.position=&#39;none&#39;,
        legend.background=element_rect(),
        plot.title=element_text(angle=0, size=24, face=&quot;bold&quot;, vjust=1),
        axis.text.x=element_text(angle=0, size=18, face=&quot;bold&quot;, hjust=1.10),
        axis.text.y=element_text(angle=0, size=18, face=&quot;bold&quot;, vjust=0.5),
        axis.title=element_text(size=24, face=&quot;bold&quot;),
        legend.key=element_blank(),     #removes the border
        legend.key.size=unit(1, &quot;cm&quot;),      #Sets overall area/size of the legend
        legend.text=element_text(size=18),  #Text size
        title=element_text(size=18)) +
    guides(colour=guide_legend(override.aes=list(size=2.5))) +
    coord_flip()</code></pre>
<p><img src="figure/integration_across_individuals.Rmd/GO-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>I’m having an issue where the vast majority of genes come up as DE between clusters. This may be caused by different individuals contributing different numbers of cells to each cluster. A work-around from Joyce (via Abishek) is to: 1. sctransform all the cells 2. perform a two sample t-test for each gene comparing the sctransformed values between clusters (ignoring individual)</p>
<pre class="r"><code>SCTransformed_data &lt;- SCT.integrated@assays$SCT@data
cluster_membership &lt;- SCT.integrated@meta.data$integrated_snn_res.0.4

clust0 &lt;- SCTransformed_data[,cluster_membership == 0]
clust1 &lt;- SCTransformed_data[,cluster_membership == 1]
clust2 &lt;- SCTransformed_data[,cluster_membership == 2]

t.test.results &lt;- matrix(NA, nrow = nrow(SCTransformed_data), ncol = 6)
colnames(t.test.results) &lt;- c(&quot;0vs1 p&quot;, &quot;0vs1 T.stat&quot;, &quot;0vs2 p&quot;, &quot;0vs2 T.stat&quot;, &quot;1vs2 p&quot;, &quot;1vs2 T.stat&quot;)
rownames(t.test.results) &lt;- rownames(SCTransformed_data)

for(gene in 1:nrow(SCTransformed_data)){
     t.test.results[gene,1] &lt;- t.test(x = clust0[gene,], y = clust1[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$p.value
     t.test.results[gene,2] &lt;- t.test(x = clust0[gene,], y = clust1[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$statistic
     t.test.results[gene,3] &lt;- t.test(x = clust0[gene,], y = clust2[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$p.value
     t.test.results[gene,4] &lt;- t.test(x = clust0[gene,], y = clust2[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$statistic
     t.test.results[gene,5] &lt;- t.test(x = clust1[gene,], y = clust2[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$p.value
     t.test.results[gene,6] &lt;- t.test(x = clust1[gene,], y = clust2[gene,], alternative = &quot;two.sided&quot;, var.equal = FALSE)$statistic
}</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] org.Hs.eg.db_3.8.2          topGO_2.36.0               
 [3] SparseM_1.78                GO.db_3.8.2                
 [5] AnnotationDbi_1.46.0        graph_1.62.0               
 [7] edgeR_3.26.5                ashr_2.2-47                
 [9] tidyr_1.1.2                 variancePartition_1.14.1   
[11] scales_1.1.1                foreach_1.4.4              
[13] limma_3.40.6                ggplot2_3.3.2              
[15] Matrix.utils_0.9.7          Matrix_1.2-18              
[17] SingleCellExperiment_1.6.0  SummarizedExperiment_1.14.1
[19] DelayedArray_0.10.0         BiocParallel_1.18.1        
[21] matrixStats_0.56.0          Biobase_2.44.0             
[23] GenomicRanges_1.36.0        GenomeInfoDb_1.20.0        
[25] IRanges_2.18.3              S4Vectors_0.22.1           
[27] BiocGenerics_0.30.0         gridExtra_2.3              
[29] Seurat_3.1.3               

loaded via a namespace (and not attached):
  [1] backports_1.1.9        workflowr_1.6.2        plyr_1.8.6            
  [4] igraph_1.2.4.1         lazyeval_0.2.2         splines_3.6.1         
  [7] listenv_0.8.0          digest_0.6.25          invgamma_1.1          
 [10] htmltools_0.5.0        lmerTest_3.1-0         SQUAREM_2020.4        
 [13] gdata_2.18.0           memoise_1.1.0          magrittr_1.5          
 [16] cluster_2.1.0          doParallel_1.0.14      ROCR_1.0-7            
 [19] globals_0.12.5         RcppParallel_5.0.2     prettyunits_1.1.1     
 [22] colorspace_1.4-1       blob_1.2.0             rappdirs_0.3.1        
 [25] ggrepel_0.8.2          xfun_0.8               dplyr_1.0.2           
 [28] crayon_1.3.4           RCurl_1.98-1.1         jsonlite_1.7.0        
 [31] lme4_1.1-21            survival_2.44-1.1      zoo_1.8-8             
 [34] iterators_1.0.12       ape_5.3                glue_1.4.2            
 [37] gtable_0.3.0           zlibbioc_1.30.0        XVector_0.24.0        
 [40] leiden_0.3.1           future.apply_1.3.0     DBI_1.1.0             
 [43] bibtex_0.4.2           Rcpp_1.0.5             metap_1.1             
 [46] viridisLite_0.3.0      progress_1.2.2         reticulate_1.16       
 [49] bit_4.0.4              rsvd_1.0.1             tsne_0.1-3            
 [52] truncnorm_1.0-8        htmlwidgets_1.5.1      httr_1.4.2            
 [55] gplots_3.0.1.1         RColorBrewer_1.1-2     ellipsis_0.3.1        
 [58] ica_1.0-2              pkgconfig_2.0.3        farver_2.0.3          
 [61] uwot_0.1.5             locfit_1.5-9.1         tidyselect_1.1.0      
 [64] labeling_0.3           rlang_0.4.7            reshape2_1.4.3        
 [67] later_1.1.0.1          munsell_0.5.0          tools_3.6.1           
 [70] RSQLite_2.1.1          generics_0.0.2         ggridges_0.5.1        
 [73] evaluate_0.14          stringr_1.4.0          yaml_2.2.1            
 [76] npsurv_0.4-0           bit64_0.9-7            knitr_1.23            
 [79] fs_1.3.1               fitdistrplus_1.0-14    caTools_1.17.1.2      
 [82] purrr_0.3.4            RANN_2.6.1             pbapply_1.4-0         
 [85] future_1.18.0          nlme_3.1-140           whisker_0.3-2         
 [88] grr_0.9.5              compiler_3.6.1         pbkrtest_0.4-7        
 [91] plotly_4.9.2.1         png_0.1-7              lsei_1.2-0            
 [94] tibble_3.0.3           stringi_1.4.6          RSpectra_0.15-0       
 [97] lattice_0.20-41        nloptr_1.2.2.1         vctrs_0.3.4           
[100] pillar_1.4.6           lifecycle_0.2.0        Rdpack_0.11-0         
[103] lmtest_0.9-37          RcppAnnoy_0.0.12       data.table_1.13.0     
[106] cowplot_1.0.0          bitops_1.0-6           irlba_2.3.3           
[109] gbRd_0.4-11            httpuv_1.5.1           colorRamps_2.3        
[112] R6_2.4.1               promises_1.1.1         KernSmooth_2.23-15    
[115] codetools_0.2-16       boot_1.3-23            MASS_7.3-52           
[118] gtools_3.8.1           rprojroot_1.3-2        withr_2.2.0           
[121] sctransform_0.2.0      GenomeInfoDbData_1.2.1 hms_0.5.3             
[124] grid_3.6.1             minqa_1.2.4            rmarkdown_1.13        
[127] Rtsne_0.15             git2r_0.26.1           mixsqp_0.3-43         
[130] numDeriv_2016.8-1.1   </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
