<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2020-01-29" />

<title>Pilot_GoMAnalysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Pilot_GoMAnalysis</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2020-01-29</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-09-08
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/project2/gilad/anthonyhung/Projects/OAStrain_project/OAStrain/data/Combined_singlecell_data_noliver.rds
</td>
<td style="text-align:left;">
data/Combined_singlecell_data_noliver.rds
</td>
</tr>
<tr>
<td style="text-align:left;">
/project2/gilad/anthonyhung/Projects/OAStrain_project/OAStrain/data/gom_sup_
</td>
<td style="text-align:left;">
data/gom_sup_
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree366731caae6f0a2d626c47c895cb376da92a4de2targetblank366731ca"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/366731caae6f0a2d626c47c895cb376da92a4de2" target="_blank">366731c</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree366731caae6f0a2d626c47c895cb376da92a4de2targetblank366731ca" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/366731caae6f0a2d626c47c895cb376da92a4de2" target="_blank">366731c</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/alasoo_etal/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds
    Ignored:    output/GO_terms_enriched.csv
    Ignored:    output/SNPs.csv
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/fasttopics/
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash/
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/Pilot_GoMAnalysis.Rmd
    Modified:   analysis/powerAnalysis.Rmd
    Modified:   output/structureplot.png

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Pilot_GoMAnalysis.Rmd</code>) and HTML (<code>docs/Pilot_GoMAnalysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/366731caae6f0a2d626c47c895cb376da92a4de2/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">366731c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-05
</td>
<td>
run gom separately on each sample
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/04257e77b0763649e596a35db22689ffda42b4fd/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">04257e7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-05
</td>
<td>
add fasttopics on all cells at once in pilot
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5189cd4c20cebd4952d7850c386d90356a280959/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">5189cd4</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-03
</td>
<td>
add scHCL code
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/51cceef198e21d0b03dbcb95ee86fd6a6d88b406/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">51cceef</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-12
</td>
<td>
box plot of pairwise correlations
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5b1c7c6ff5eec4ebeb600f8cabda6eb39926a36/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">a5b1c7c</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-04-14
</td>
<td>
separate into up/downregulated in DE analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6baf979f9ff594f844a14171435152ff5b0e7081/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">6baf979</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-10
</td>
<td>
Pseudobulk correlation heatmap with bulk samples
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c9015d5f25785b2ffd97f0119b94015c2cc84498/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">c9015d5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Revise pseudobulk to use raw counts not normalized
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a46d6f37799b4f9c74c12be5c41c412f95f2fa3e/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">a46d6f3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-09
</td>
<td>
Add gene analysis to semi-supervised topic model
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/ad1e27bd3b051a731c2357fb8d8f981baa35f6af/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">ad1e27b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-03-02
</td>
<td>
separate up/down in de analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/59b30d0b555983d416401d53765f1863f550e7ab/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">59b30d0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-29
</td>
<td>
semi-supervised topic modeling
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f9ae4fada5a4e60b5fc5d1945a443fbc9ce0b200/docs/Pilot_GoMAnalysis.html" target="_blank">f9ae4fa</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-27
</td>
<td>
correlation pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e0cb97297da19032063e3627b0cb97dd39a82c15/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">e0cb972</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-24
</td>
<td>
Separate out sample based on origin; also add pseudobulk
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
add additional chondro markers
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/Pilot_GoMAnalysis.html" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
<td>
add additional chondro markers
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/605da875aae021b28ea700a7c7af372b50a19dc6/docs/Pilot_GoMAnalysis.html" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
<td>
Check expression of chondro markers in outside datasets
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/2bbbdad15237735154e8db0270272fefd55760a9/docs/Pilot_GoMAnalysis.html" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
<td>
Finish GoM analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/Pilot_GoMAnalysis.html" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
<td>
Finish GoM analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e2253361f00148a0c0eaa938f38c35e398fb9fa6/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">e225336</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-03
</td>
<td>
merge sc data and set up functions to run Gom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e2253361f00148a0c0eaa938f38c35e398fb9fa6/docs/Pilot_GoMAnalysis.html" target="_blank">e225336</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-03
</td>
<td>
merge sc data and set up functions to run Gom
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/analysis/Pilot_GoMAnalysis.Rmd" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-01
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="applying-topic-modeling-approaches-on-sc-pilot-data-first-analysis-with-my-data-and-outside-data-from-other-cell-types" class="section level1">
<h1>Applying topic modeling approaches on sc pilot data (first analysis with my data and outside data from other cell types)</h1>
<p>Include data from: Pilot (iPSC-derived chondrocytes from 3 individuals x 2 treatment conditions; 10x) MSC-Pilot (Genevieve Housman’s human data of iPSCs and iPSC-derived MSCs from human-chimp study; 10x) Also one dataset from chondrocytes (ipsc-derived) Hepatocytes (from liver homogenate; 10x) Chondrocytes (from OA patients; STRT protocol)</p>
<div id="load-libraries" class="section level2">
<h2>Load Libraries</h2>
<pre class="r"><code>library(Seurat)
library(tidyverse)</code></pre>
<pre><code>── Attaching packages ─────────────────────────────────────────────────────────────────────────────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>✓ ggplot2 3.3.2     ✓ purrr   0.3.4
✓ tibble  3.0.3     ✓ dplyr   1.0.2
✓ tidyr   1.1.2     ✓ stringr 1.4.0
✓ readr   1.3.1     ✓ forcats 0.4.0</code></pre>
<pre><code>── Conflicts ────────────────────────────────────────────────────────────────────────────────────────────────────────────── tidyverse_conflicts() ──
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(CountClust)
library(maptpx)</code></pre>
<pre><code>Loading required package: slam</code></pre>
<pre class="r"><code>library(ggplot2)
library(MAST)</code></pre>
<pre><code>Loading required package: SingleCellExperiment</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following object is masked from &#39;package:maptpx&#39;:

    normalize</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    combine, intersect, setdiff, union</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, basename, cbind,
    colnames, dirname, do.call, duplicated, eval, evalq, Filter,
    Find, get, grep, grepl, intersect, is.unsorted, lapply, Map,
    mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, Position, rank, rbind, Reduce, rownames, sapply,
    setdiff, sort, table, tapply, union, unique, unsplit, which,
    which.max, which.min</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    first, rename</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    expand</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>
Attaching package: &#39;IRanges&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:dplyr&#39;:

    collapse, desc, slice</code></pre>
<pre><code>The following object is masked from &#39;package:purrr&#39;:

    reduce</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    count</code></pre>
<pre><code>Loading required package: BiocParallel</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following object is masked from &#39;package:purrr&#39;:

    simplify</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    aperm, apply, rowsum</code></pre>
<pre><code>
Attaching package: &#39;SummarizedExperiment&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:Seurat&#39;:

    Assays</code></pre>
<pre><code>
Attaching package: &#39;MAST&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    filter</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    filter</code></pre>
<pre class="r"><code>library(RColorBrewer)
library(ggfortify)
library(stringi)
library(stringr)
library(colorspace)
library(dplyr)</code></pre>
</div>
<div id="load-in-datasets-and-merge-them" class="section level2">
<h2>Load in datasets and merge them</h2>
<p>Load in datasets and create Seurat objects for each of them. Add metadata about the cell type they were assigned in their original contexts. Merge the Seurat object into one large object.</p>
<pre class="r"><code>if (file.exists(&quot;data/Combined_singlecell_data_first.rds&quot;) &amp; file.exists(&quot;data/Combined_singlecell_data_allGenes.rds&quot;)){
Combined.common &lt;- readRDS(&quot;data/Combined_singlecell_data_first.rds&quot;)
Combined.common.allGenes &lt;- readRDS(&quot;data/Combined_singlecell_data_allGenes.rds&quot;)
} else {

#Pilot data
ANT1.2 &lt;- readRDS(&quot;data/ANT1_2.rds&quot;)
ANT1.2 &lt;- AddMetaData(ANT1.2, &quot;iPSC-Chondrocyte&quot;, col.name = &quot;Cell.Type&quot;)
dim(ANT1.2)

#Chondrocytes from OA patients (https://www.ncbi.nlm.nih.gov/pubmed/30026257) run through STRT protocol
chondro &lt;- read.table(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/outside_scData/GSE104782_allcells_UMI_count.txt.gz&quot;, header = T)
chondro_genes &lt;- chondro[,1]
chondro_data &lt;- chondro[,-1]
chondro &lt;- chondro_data
rownames(chondro) &lt;- chondro_genes
#load chondro cluster data from paper
chondro_metadata &lt;- read_csv(&quot;data/GSE104782_Table_Cell_quality_information_and_clustering_information.csv&quot;)
#subset data to include only those that were assigned to a chondrocyte cluster in the paper
chondro.filtered &lt;- chondro[,!is.na(chondro_metadata$Cluster)]
cluster.ident &lt;- paste0(&quot;Chondro &quot;, chondro_metadata$Cluster[!is.na(chondro_metadata$Cluster)])
#Create seurat object from chondrocytes
chondro.seurat &lt;- CreateSeuratObject(chondro.filtered, project=&quot;Chondrocytes&quot;)
chondro.seurat &lt;- AddMetaData(chondro.seurat, cluster.ident, col.name = &quot;Cell.Type&quot;)
dim(chondro.seurat)

#iPSCs/MSCs/osteoblasts from Human/chimp panel. Combine all the datasets and subset for humans
ghousman_data_dir &lt;- (&quot;/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/cellranger-data-full/&quot;)
gh_ipsc_1.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.GHO-1.rds&quot;))
gh_msc_1.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.GHO-2.rds&quot;))
gh_ipsc_1.2 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-10.rds&quot;))
gh_msc_1.2 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-12.rds&quot;))
gh_ipsc_2.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-1.rds&quot;))
gh_msc_2.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-2.rds&quot;))
gh_ipsc_3.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-9.rds&quot;))
gh_msc_3.1 &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.HOU-11.rds&quot;))
gh_chondro &lt;- readRDS(paste0(ghousman_data_dir, &quot;h.c.dat.spp.GHO-4.rds&quot;)) #need to verify the sample name for the chondro/if these were processed yet
gh_big &lt;- merge(gh_ipsc_1.1, y = c(gh_msc_1.1, gh_ipsc_1.2, gh_msc_1.2, gh_ipsc_2.1, gh_msc_2.1, gh_ipsc_3.1, gh_msc_3.1), 
                add.cell.ids = c(&quot;ipsc1.1&quot;, &quot;msc1.1&quot;, &quot;ipsc1.2&quot;, &quot;msc1.2&quot;, &quot;ipsc2.1&quot;, &quot;msc2.1&quot;, &quot;ipsc3.1&quot;, &quot;msc3.1&quot;), project = &quot;ghousman&quot;)
gh_big_humans &lt;- subset(gh_big, species == &quot;Human&quot;)
gh_big_humans &lt;- AddMetaData(gh_big_humans, &quot;Ghousman&quot;, col.name = &quot;orig.ident&quot;)
dim(gh_big_humans)

#Human Liver samples run through 10x (https://doi.org/10.1038/s41467-018-06318-7). This is a subset of the entire dataset, which is from whole liver homogenate. I used scViz to subset the data for clusters that were called hepatocytes in the original study (3490 cells)
liver &lt;- readRDS(&quot;data/humanLiverSubset_hepatocytes.rds&quot;)
liver &lt;- AddMetaData(liver, &quot;Hepatocyte&quot;, col.name = &quot;Cell.Type&quot;)
dim(liver)



# Merge Datasets

# Merge the seurat objects into one large one, keeping metadata about which study they originated in as well as their assigned cell type.

# merge based on common genes between ALL datasets
common.features &lt;- Reduce(intersect, list(rownames(ANT1.2), rownames(chondro.seurat), rownames(gh_big_humans), rownames(liver)))
length(x = common.features)
Combined.common &lt;- merge(ANT1.2[common.features, ], y = c(chondro.seurat[common.features, ], gh_big_humans[common.features, ], liver[common.features, ]), project = &quot;Combined.common&quot;)

#merge keeping all genes
Combined.common.allGenes &lt;- Combined.common &lt;- merge(ANT1.2, y = c(chondro.seurat, gh_big_humans, liver), project = &quot;Combined.common&quot;)
Combined.common.allGenes_noliver &lt;- Combined.common &lt;- merge(ANT1.2, y = c(chondro.seurat, gh_big_humans), project = &quot;Combined.common&quot;)


saveRDS(Combined.common, &quot;data/Combined_singlecell_data_first.rds&quot;)
saveRDS(Combined.common.allGenes, &quot;data/Combined_singlecell_data_allGenes.rds&quot;)
saveRDS(Combined.common.allGenes_noliver, &quot;data/Combined_singlecell_data_allGenes_noliver.rds&quot;)

}</code></pre>
</div>
<div id="check-expression-of-chondrocyte-markers-in-chondrocyte-data-and-pilot-data" class="section level2">
<h2>Check Expression of chondrocyte markers in chondrocyte data and pilot data</h2>
<pre class="r"><code>#Chondrocytes from OA patients (https://www.ncbi.nlm.nih.gov/pubmed/30026257) run through STRT protocol
chondro &lt;- read.table(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/outside_scData/GSE104782_allcells_UMI_count.txt.gz&quot;, header = T)
chondro_genes &lt;- chondro[,1]
chondro_data &lt;- chondro[,-1]
chondro &lt;- chondro_data
rownames(chondro) &lt;- chondro_genes
#load chondro cluster data from paper
chondro_metadata &lt;- read_csv(&quot;data/GSE104782_Table_Cell_quality_information_and_clustering_information.csv&quot;)</code></pre>
<pre><code>Parsed with column specification:
cols(
  Cell = col_character(),
  `Raw reads` = col_double(),
  `Clean reads` = col_double(),
  `Aligned reads` = col_double(),
  `Mapping rate` = col_character(),
  `Gene number` = col_double(),
  `Transcript number` = col_double(),
  Cluster = col_character()
)</code></pre>
<pre class="r"><code>#subset data to include only those that were assigned to a chondrocyte cluster in the paper
chondro.filtered &lt;- chondro[,!is.na(chondro_metadata$Cluster)]
cluster.ident &lt;- paste0(&quot;Chondro &quot;, chondro_metadata$Cluster[!is.na(chondro_metadata$Cluster)])
#Create seurat object from chondrocytes
chondro.seurat &lt;- CreateSeuratObject(chondro.filtered, project=&quot;Chondrocytes&quot;)</code></pre>
<pre><code>Warning: Feature names cannot have underscores (&#39;_&#39;), replacing with dashes
(&#39;-&#39;)</code></pre>
<pre class="r"><code>chondro.seurat &lt;- AddMetaData(chondro.seurat, cluster.ident, col.name = &quot;Cell.Type&quot;)
dim(chondro.seurat)</code></pre>
<pre><code>[1] 24153  1464</code></pre>
<pre class="r"><code>VlnPlot(object=chondro.seurat, features = c(&quot;SOX9&quot;, &quot;ACAN&quot;, &quot;COL2A1&quot;, &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;))</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-1">
Past versions of unnamed-chunk-1-1.png
</button>
</p>
<div id="fig-unnamed-chunk-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-1.png" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-1.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=chondro.seurat, features = c(&quot;TGFB3&quot;))</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-2">
Past versions of unnamed-chunk-1-2.png
</button>
</p>
<div id="fig-unnamed-chunk-1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-2.png" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-2.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Pilot data
ANT1.2 &lt;- readRDS(&quot;data/ANT1_2.rds&quot;)
ANT1.2 &lt;- AddMetaData(ANT1.2, &quot;iPSC-Chondrocyte&quot;, col.name = &quot;Cell.Type&quot;)
dim(ANT1.2)</code></pre>
<pre><code>[1] 33538  1815</code></pre>
<pre class="r"><code>VlnPlot(object=ANT1.2, features = c(&quot;SOX9&quot;, &quot;ACAN&quot;, &quot;COL2A1&quot;, &quot;COL11A1&quot;, &quot;SOX5&quot;, &quot;SOX6&quot;))</code></pre>
<pre><code>Warning in SingleExIPlot(type = type, data = data[, x, drop = FALSE],
idents = idents, : All cells have the same value of ACAN.</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-3">
Past versions of unnamed-chunk-1-3.png
</button>
</p>
<div id="fig-unnamed-chunk-1-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-3.png" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=ANT1.2, features = c(&quot;TGFB3&quot;))</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-4">
Past versions of unnamed-chunk-1-4.png
</button>
</p>
<div id="fig-unnamed-chunk-1-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/figure/Pilot_GoMAnalysis.Rmd/unnamed-chunk-1-4.png" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="../analysis/figures/scviz1.png" alt="Cell Type Clusters from original paper" /> <img src="../analysis/figures/scviz2.png" alt="Subsetting gate drawn in scViz" /></p>
</div>
<div id="analysis-of-combined-dataset-in-seurat" class="section level2">
<h2>Analysis of combined dataset in Seurat</h2>
<p>Now we can look at the cells in Seurat, including PCA and UMAP</p>
<pre class="r"><code>VlnPlot(Combined.common, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;), ncol = 2, group.by = &quot;orig.ident&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-1">
Past versions of “Seurat analysis-1.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-1.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(Combined.common, features = c(&quot;nFeature_RNA&quot;, &quot;nCount_RNA&quot;), ncol = 2, group.by = &quot;Cell.Type&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-2">
Past versions of “Seurat analysis-2.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-2.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># FeatureScatter is typically used to visualize feature-feature relationships, but can be used
# for anything calculated by the object, i.e. columns in object metadata, PC scores etc.

FeatureScatter(Combined.common, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;nFeature_RNA&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-3">
Past versions of “Seurat analysis-3.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-3.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>if (file.exists(&quot;data/Combined_singlecell_data.rds&quot;) &amp; file.exists(&quot;data/Combined_singlecell_data_allGenes_normalized.rds&quot;)){
Combined.common &lt;- readRDS(&quot;data/Combined_singlecell_data.rds&quot;)
Combined.common.allGenes &lt;- readRDS(&quot;data/Combined_singlecell_data_allGenes_normalized.rds&quot;)
} else {

#Normalize Data
Combined.common &lt;- NormalizeData(Combined.common, normalization.method = &quot;LogNormalize&quot;, scale.factor = 10000)

# Feature selection
Combined.common &lt;- FindVariableFeatures(Combined.common, selection.method = &quot;vst&quot;, nfeatures = 2000)

# Scaling Data
scale_data &lt;- function(seurat.obj){
  all.genes &lt;- rownames(seurat.obj)
  return(ScaleData(seurat.obj, features = all.genes))
}

Combined.common &lt;- scale_data(Combined.common)

# Dimensionality Reduction
#Using the highly variable features selected above, perform PCA
nPC &lt;- 100
Combined.common &lt;- RunPCA(Combined.common, features = VariableFeatures(object = Combined.common), npcs = nPC)


#UMAP
num_PCs &lt;- 50

Combined.common &lt;- FindNeighbors(Combined.common, dims = 1:num_PCs)
Combined.common &lt;- FindClusters(Combined.common, resolution = 0.5)

#Run UMAP
Combined.common &lt;- RunUMAP(Combined.common, dims = 1:num_PCs)
saveRDS(Combined.common, &quot;data/Combined_singlecell_data.rds&quot;)






#Normalize Data
Combined.common.allGenes &lt;- NormalizeData(Combined.common.allGenes, normalization.method = &quot;LogNormalize&quot;, scale.factor = 10000)

# Feature selection

Combined.common.allGenes &lt;- FindVariableFeatures(Combined.common.allGenes, selection.method = &quot;vst&quot;, nfeatures = 2000)

# Scaling Data

scale_data &lt;- function(seurat.obj){
  all.genes &lt;- rownames(seurat.obj)
  return(ScaleData(seurat.obj, features = all.genes))
}

Combined.common.allGenes &lt;- scale_data(Combined.common.allGenes)

# Dimensionality Reduction

#Using the highly variable features selected above, perform PCA

nPC &lt;- 100
Combined.common.allGenes &lt;- RunPCA(Combined.common.allGenes, features = VariableFeatures(object = Combined.common.allGenes), npcs = nPC)


#UMAP
num_PCs &lt;- 50

Combined.common.allGenes &lt;- FindNeighbors(Combined.common.allGenes, dims = 1:num_PCs)
Combined.common.allGenes &lt;- FindClusters(Combined.common.allGenes, resolution = 0.5)

#Run UMAP
Combined.common.allGenes &lt;- RunUMAP(Combined.common.allGenes, dims = 1:num_PCs)
saveRDS(Combined.common.allGenes, &quot;data/Combined_singlecell_data_allGenes_normalized.rds&quot;)
}



#Plots 

#Var features
plot_variable_features &lt;- function(seurat.obj){
  # Identify the 10 most highly variable genes
  top10 &lt;- head(VariableFeatures(seurat.obj), 10)
  
  # plot variable features with and without labels
  plot1 &lt;- VariableFeaturePlot(seurat.obj) + 
       theme(legend.position=&quot;bottom&quot;)
  plot2 &lt;- LabelPoints(plot = plot1, points = top10, repel = TRUE) +
       theme(legend.position=&quot;bottom&quot;)
  return(CombinePlots(plots = list(plot1, plot2)))
}

plot_variable_features(Combined.common)</code></pre>
<pre><code>Warning: Using `as.character()` on a quosure is deprecated as of rlang 0.3.0.
Please use `as_label()` or `as_name()` instead.
This warning is displayed once per session.</code></pre>
<pre><code>When using repel, set xnudge and ynudge to 0 for optimal results</code></pre>
<pre><code>Warning: Transformation introduced infinite values in continuous x-axis</code></pre>
<pre><code>Warning: Transformation introduced infinite values in continuous x-axis</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-4">
Past versions of “Seurat analysis-4.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/88682cc9e2c11f7ae7ba25b9959dfde687206e29/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-4.png" target="_blank">88682cc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/605da875aae021b28ea700a7c7af372b50a19dc6/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-4.png" target="_blank">605da87</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-4.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-4.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Examine and visualize PCA results a few different ways
print(Combined.common[[&quot;pca&quot;]], dims = 1:5, nfeatures = 5)</code></pre>
<pre><code>PC_ 1 
Positive:  PMAIP1, CRABP2, ALPL, ID1, PTN 
Negative:  C2orf40, FMOD, PRELP, BGN, SCRG1 
PC_ 2 
Positive:  SPARC, COL1A1, COL5A1, COL1A2, TIMP1 
Negative:  RBP4, MT1G, TF, SAA2, SAA1 
PC_ 3 
Positive:  COL3A1, CST3, SERPINE2, COL5A1, NEAT1 
Negative:  PMAIP1, COL9A3, FABP5, ID1, IER2 
PC_ 4 
Positive:  UGP2, PDPN, SOX4, COL9A3, ALPL 
Negative:  ANXA1, PHLDA2, CD44, PLAUR, DUSP1 
PC_ 5 
Positive:  DNAJB1, FOS, NR4A2, RGS16, MTRNR2L2 
Negative:  BMP2, FAM101B, LRRC8C, CLCF1, INHBA-AS1 </code></pre>
<pre class="r"><code>VizDimLoadings(Combined.common, dims = 1:2, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-5">
Past versions of “Seurat analysis-5.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-5.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(Combined.common, reduction = &quot;pca&quot;, group.by = &quot;Cell.Type&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-6">
Past versions of “Seurat analysis-6.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-6.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimHeatmap(Combined.common, dims = 1:15, cells = 500, balanced = TRUE)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-7">
Past versions of “Seurat analysis-7.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-7.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#extract scaled data for highlyvargenes to a matrix outside of seurat to perform PCA and examine varexplained
scaled &lt;- Combined.common@assays$RNA@scale.data[VariableFeatures(Combined.common),]
dim(scaled)</code></pre>
<pre><code>[1]  2000 30716</code></pre>
<pre class="r"><code>pca_genes &lt;- prcomp(t(scaled), scale = F)
scores &lt;- pca_genes$x

autoplot(pca_genes, label = T)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-8">
Past versions of “Seurat analysis-8.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-8.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-8.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#% Var explained by top 10 pcs
for(n in 1:10){
     print(n)
     print(summary(pca_genes)$importance[2,n]*100)
}</code></pre>
<pre><code>[1] 1
[1] 14.291
[1] 2
[1] 3.607
[1] 3
[1] 2.41
[1] 4
[1] 1.719
[1] 5
[1] 1.059
[1] 6
[1] 0.925
[1] 7
[1] 0.673
[1] 8
[1] 0.602
[1] 9
[1] 0.421
[1] 10
[1] 0.389</code></pre>
<pre class="r"><code>nPC &lt;- 100
#elbow plot
ElbowPlot(Combined.common, ndims = nPC) #50 PCs?</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-9">
Past versions of “Seurat analysis-9.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-9.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-9.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#UMAP
DimPlot(Combined.common, reduction = &quot;umap&quot;, group.by = &quot;Cell.Type&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/Seurat%20analysis-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Seurat-analysis-10">
Past versions of “Seurat analysis-10.png”
</button>
</p>
<div id="fig-no-spaces-Seurat-analysis-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-10.png" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/e1229a05fec914deb0447add85f8f485f5d99f7f/docs/figure/Pilot_GoMAnalysis.Rmd/Seurat analysis-10.png" target="_blank">e1229a0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="semi-supervised-clustering-with-classtpx" class="section level2">
<h2>Semi-supervised clustering with classtpx</h2>
<pre class="r"><code># Grades of Membership analysis with classtpx

library(classtpx)</code></pre>
<pre><code>
Attaching package: &#39;classtpx&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:maptpx&#39;:

    expit, logit, rdir, stm_tfidf</code></pre>
<pre class="r"><code>library(Seurat)
library(ggplot2)
library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)</code></pre>
<pre><code>
Attaching package: &#39;scales&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:purrr&#39;:

    discard</code></pre>
<pre><code>The following object is masked from &#39;package:readr&#39;:

    col_factor</code></pre>
<pre class="r"><code>library(&quot;cowplot&quot;)</code></pre>
<pre><code>
********************************************************</code></pre>
<pre><code>Note: As of version 1.0.0, cowplot does not change the</code></pre>
<pre><code>  default ggplot2 theme anymore. To recover the previous</code></pre>
<pre><code>  behavior, execute:
  theme_set(theme_cowplot())</code></pre>
<pre><code>********************************************************</code></pre>
<pre class="r"><code>library(&quot;CountClust&quot;)
library(&quot;maptpx&quot;)
library(&quot;MAST&quot;)
library(&quot;ggplot2&quot;)
library(&quot;ggfortify&quot;)
library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:IRanges&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:S4Vectors&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>tpx.clust &lt;- readRDS(file=&quot;data/gom_sup_3.rda&quot;)

# if (file.exists(&quot;data/gom_sup_3.rda&quot;)){
#      tpx.clust &lt;- readRDS(file=&quot;data/gom_sup_3.rda&quot;)
# } else {
# 
# Combined.common_noliver &lt;- readRDS(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/OAStrain/data/Combined_singlecell_data_noliver.rds&quot;)
# dim(Combined.common_noliver)
# 
# # specify the fixed loadings for certain cells (1 = iPSC, 2 = MSC, 3 = chondrocyte)
# counts_allgenes &lt;- as.matrix(Combined.common_noliver@assays$RNA@counts)
# labels &lt;- Combined.common_noliver@meta.data$Cell.Type
# 
# #labels for &quot;known&quot; clusters
# idx.ips &lt;- which(labels==&quot;iPSC&quot;)
# idx.msc &lt;- which(labels==&quot;MSC&quot;)
# idx.cho &lt;- which(grepl(&quot;Chondro &quot;, labels))
# kwn.smp &lt;- c(idx.ips,idx.msc,idx.cho)
# 
# class.labs &lt;- c(rep(&quot;iPSC&quot;, length(idx.ips)),
#                 rep(&quot;MSC&quot;, length(idx.msc)),
#                 rep(&quot;Chondrocyte&quot;, length(idx.cho)))
# 
# #Perform topic modeling k=3
# #Save GoM data
# k=3
# print(k)
# tpx.clust &lt;- classtpx::class_topics(
#      t(counts_allgenes),
#      K=k,
#      known_samples=kwn.smp,
#      class_labs=class.labs,
#      method=&quot;omega.fix&quot;,
#      tol=1)
# saveRDS(tpx.clust, file=paste0(&quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/OAStrain/data/gom_sup_&quot;,k,&quot;.rda&quot;))
# }
# 
# omega &lt;- tpx.clust$omega
# annotation &lt;- data.frame(
#      sample_id = paste0(&quot;X&quot;, c(1:NROW(omega))),
#      tissue_label = as.factor(labels))
# rownames(omega) &lt;- annotation$sample_id
# CountClust::StructureGGplot(omega = omega,
#                             annotation = annotation,
#                             palette = RColorBrewer::brewer.pal(8, &quot;Accent&quot;),
#                             yaxis_label = &quot;Tissue/Cell Type&quot;,
#                             order_sample = TRUE,
#                             axis_tick = list(axis_ticks_length = .1,
#                                              axis_ticks_lwd_y = .1,
#                                              axis_ticks_lwd_x = .1,
#                                              axis_label_size = 7,
#                                              axis_label_face = &quot;bold&quot;)
# )
# 
# ggsave(&quot;output/structureplot.png&quot;)</code></pre>
</div>
<div id="analysis-of-the-topics-within-the-topic-models-generated-for-each-dataset" class="section level2">
<h2>Analysis of the Topics within the Topic models generated for each dataset</h2>
<pre class="r"><code>visualizeGoM &lt;- function(omega, seuratobject, data){
  omega &lt;- omega[match(colnames(seuratobject@assays$RNA), colnames(data)),]
  labs_clus &lt;- colnames(omega)[apply(omega,1,which.max)]
  data_ggplot &lt;- data.frame(labels = labs_clus,
                          umap1 = seuratobject@reductions$umap@cell.embeddings[, 1],
                          umap2 = seuratobject@reductions$umap@cell.embeddings[, 2])
  plot1 &lt;- ggplot(data_ggplot) + theme(panel.background = element_rect(colour = &quot;black&quot;, size = 1),
                                     panel.grid.major = element_blank(),
                                     panel.grid.minor = element_blank(),
                                     axis.text.y = element_blank(),
                                     axis.text.x = element_blank(),
                                     axis.ticks = element_blank(),
                                     legend.text = element_text(size = 15)) + geom_point(aes(x = umap1, y = umap2, color = labels), size = 0.5) + labs(x = &#39;tSNE 1&#39;, y = &#39;tSNE 2&#39;, title = &#39;&#39;) + scale_color_discrete(&#39;&#39;)
  return(plot1)
}

visualizeGoM_topic_gradients &lt;- function(omega, seuratobject, data){
  omega &lt;- omega[match(colnames(seuratobject@assays$RNA), colnames(data)),]
  ntopics &lt;- ncol(omega)
  for (n in 1:ntopics){
  data_ggplot &lt;- data.frame(proportion = omega[,n],
                          umap1 = seuratobject@reductions$umap@cell.embeddings[, 1],
                          umap2 = seuratobject@reductions$umap@cell.embeddings[, 2])
  plot.gradient &lt;- ggplot(data_ggplot) + theme(panel.background = element_rect(colour = &quot;black&quot;, size = 1),
                                     panel.grid.major = element_blank(),
                                     panel.grid.minor = element_blank(),
                                     axis.text.y = element_blank(),
                                     axis.text.x = element_blank(),
                                     axis.ticks = element_blank(),
                                     legend.text = element_text(size = 15)) + geom_point(aes(x = umap1, y = umap2, color = proportion), size = 0.5) + labs(x = &#39;tSNE 1&#39;, y = &#39;tSNE 2&#39;, title = &#39;&#39;) + scale_colour_gradient(low=&quot;grey&quot;, high=&quot;red&quot;)
  plot(plot.gradient)
  }
}



plot_structure_plot &lt;- function(omega, seuratobject){
     labels &lt;- sapply(strsplit(seuratobject@meta.data$Cell.Type,&quot; &quot;), `[`, 1)
     annotation &lt;- data.frame(
          sample_id = rownames(omega),
          tissue_label = labels
          )
     StructureGGplot(omega = omega,
                     annotation = annotation,
                     #palette = my_color_palette1,
                     order_sample = TRUE,
                     axis_tick = list(axis_ticks_length = .1,
                                      axis_ticks_lwd_y = .1,
                                      axis_ticks_lwd_x = .1,
                                      axis_label_size = 7,
                                      axis_label_face = &quot;bold&quot;))
     }


make_topic_plots &lt;- function(omega, celltypes){
  omega &lt;- as.data.frame(omega)
  omega &lt;- cbind(omega,celltypes)
  for (k in 1:(ncol(omega)-1)) {
    base_plot &lt;- ggplot(data = omega, mapping = aes(x = celltypes, y = omega[,k])) + labs(y = paste0(&quot;Topic &quot;, k, &quot; value&quot;), x = paste0(&quot;Cell Type&quot;)) + theme(axis.text.x = element_text(angle = -20, hjust = 0, vjust = 0))
    plot(base_plot + geom_violin(width = 1.5, fill = &quot;skyblue&quot;,color = &quot;skyblue&quot;) +
      geom_boxplot(width = 0.1,outlier.shape = NA, alpha = 0.2))
   }
}</code></pre>
<p>Use the CountClust::ExtractTopFeatures function to obtain KL measurement scores for the top 100 genes per cluster that best distinguish that cluster from the rest.</p>
<pre class="r"><code>annotate_clusters &lt;- function(theta_mat, countmatrix){
  top_features &lt;- ExtractTopFeatures(theta_mat,
                                     top_features=100,
                                     method=&quot;poisson&quot;,
                                     options=&quot;min&quot;)
  gene_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) rownames(countmatrix)[top_features[[1]][x,]]))
  scores_vector &lt;- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) top_features[[2]][x,]))
  print(gene_vector)
  print(scores_vector)
}</code></pre>
</div>
<div id="apply-visualizationannotation-functions-to-gom-results" class="section level2">
<h2>Apply Visualization/annotation functions to GoM results</h2>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM3[[1]]$clust_3$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM3[[1]]$clust_3$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM3[[1]]$clust_3$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM3[[1]]$clust_3$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM3[[1]]$clust_3$theta, Combined_counts)</code></pre>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM4[[1]]$clust_4$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM4[[1]]$clust_4$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM4[[1]]$clust_4$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM4[[1]]$clust_4$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM4[[1]]$clust_4$theta, Combined_counts)</code></pre>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM5[[1]]$clust_5$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM5[[1]]$clust_5$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM5[[1]]$clust_5$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM5[[1]]$clust_5$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM5[[1]]$clust_5$theta, Combined_counts)</code></pre>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM6[[1]]$clust_6$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM6[[1]]$clust_6$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM6[[1]]$clust_6$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM6[[1]]$clust_6$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM6[[1]]$clust_6$theta, Combined_counts)</code></pre>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM7[[1]]$clust_7$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM7[[1]]$clust_7$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM7[[1]]$clust_7$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM7[[1]]$clust_7$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM7[[1]]$clust_7$theta, Combined_counts)</code></pre>
<pre class="r"><code># visualizeGoM(omega=Combined_GoM8[[1]]$clust_8$omega, seuratobject=Combined.common, data=Combined_counts)
# visualizeGoM_topic_gradients(omega=Combined_GoM8[[1]]$clust_8$omega, seuratobject=Combined.common, data=Combined_counts)
# #plot_structure_plot(omega=Combined_GoM8[[1]]$clust_8$omega, seuratobject=Combined.common)
# make_topic_plots(Combined_GoM8[[1]]$clust_8$omega, Combined_counts_celltypes)
# annotate_clusters(Combined_GoM8[[1]]$clust_8$theta, Combined_counts)</code></pre>
</div>
<div id="code-for-loading-in-cell-atlas-samples" class="section level2">
<h2>Code for loading in cell atlas samples</h2>
<p>First compute correlations between each individual cell in ANT1.2 and each of the primary cell lines in the cell atlas dataset to assign labels. Then use those primary cell lines to compute memberships in clusters</p>
<pre class="r"><code>###SCRIPTS GOTTEN FROM BINGQING XIE###

###compute correlation matrix between sample and cell lines in cellAtlas
###object: Seurat object
###cell.line: cell Atlas expression matrix
###CorMethod: correlation method to use
ComputeCorMat &lt;- function(object, cell.line, CorMethod=&quot;pearson&quot;){

    data=GetAssayData(object, slot=&quot;data&quot;, assay=&quot;RNA&quot;) #additions - added slot and assay info because SCT assay 
    data_norm &lt;- apply(data, 2, function(x) x/sum(x))
    data_libsize &lt;- apply(data_norm, 2, function(x) x * 1e+06)

    Data.use &lt;- data_libsize

    gene.id &lt;- rownames(Data.use)
    gene.ref &lt;- rownames(cell.line)
    common &lt;- intersect(gene.id, gene.ref)
    logxx &lt;- apply(Data.use[common, ], 2, function(x) {
        log(x + 0.1)
    })
    selected.cell.line &lt;- apply(cell.line[common, ], 2, function(x) {
        x - mean(x)
    })
    n &lt;- ncol(logxx)
    m &lt;- ncol(cell.line)
    cor.mat &lt;- matrix(0, nrow = n, ncol = m)
    for (j in 1:m) {
        for (i in 1:n) {
            cor.mat[i,j] &lt;- cor(logxx[,i], selected.cell.line[,j], method=CorMethod)
        }
    }
    rownames(cor.mat) &lt;- colnames(Data.use)
    colnames(cor.mat) &lt;- colnames(cell.line)
    return(cor.mat)
}

###Assign clustered cell atlas as cell type to all cells in the sample
###object: seurat object
###cor.mat: correlation matrix
###K: number of clusters generated from cell atlas
###topCorrelation: number of highest correlated cell lines in cell atlas to be assigned, set to 1 to extract only the top correlated cell line
AssignClusteredLabel &lt;- function (object, cor.mat, K, topCorrelation, dist.method=&quot;euclidean&quot;, hclust.method=&quot;ward.D&quot;){
    
    dist_mat &lt;- dist(t(cor.mat), method=dist.method)
    hclust_avg_cellline &lt;- hclust(dist_mat, method=hclust.method)
    #plot(hclust_avg_cellline)

    cut_cellline &lt;- cutree(hclust_avg_cellline, k = K)
    cellline_abstract=sapply(strsplit(as.character(colnames(cor.mat)), &quot;\\_&quot;), &quot;[[&quot;, 2)
    clusteredCellline=rbind(cut_cellline,cellline_abstract)
    nametable=rep(&#39;n&#39;,K)
    for(i in 1:K){
        indexy=as.character(i)
        nametable[i]=paste0(names(sort(table(clusteredCellline[2,clusteredCellline[1,]==indexy]),decreasing=TRUE)),collapse=&#39;_&#39; )
    }
    assignm=apply(cor.mat,1,function(x){
        indexy=names(which.max(table(clusteredCellline[1,names(sort(x,decreasing=T)[1:topCorrelation])])))
        y=paste0(names( sort(table(clusteredCellline[2,clusteredCellline[1,]==indexy]),decreasing=TRUE)),collapse=&#39;_&#39; )
        return(y)
    })
    object$clusteredCellAtlas=assignm
    return(object)
}

#Load cell atlas data matrix (currated by GAH)
#references: Human Primary Cell Atlas (Mabbott et al., 2013)
#description: 713 microarray samples processed and normalized as described in Aran et al. (2019); each sample assigned to one of 37 main cell types and 157 subtypes
#library(SingleR, lib.loc=&quot;&quot;) #only available on R/3.6.0??? ...BUT able to install on local computer
#if(!requireNamespace(&quot;BiocManager&quot;, quietly = TRUE)) install.packages(&quot;BiocManager&quot;)
#BiocManager::install(&quot;SingleR&quot;)
#library(SingleR)
#hpca.se &lt;- HumanPrimaryCellAtlasData()
#write.table(hpca.se@assays@data$logcounts, file=&quot;G:/My Drive/Documents/University of Chicago/Gilad Lab/project notes/study-human-chimp-bone-cartilage-rna/study-collections/sc-rna/HumanPrimaryCellAtlasData-SingleR&quot;)
#write.csv(as.data.frame(hpca.se@colData),file=&quot;G:/My Drive/Documents/University of Chicago/Gilad Lab/project notes/study-human-chimp-bone-cartilage-rna/study-collections/sc-rna/HumanPrimaryCellAtlasLabels-SingleR.csv&quot;, row.names=TRUE)
#manually currated HumanPrimaryCellAtlasLabels-SingleR.csv
#transferred files to cluster
hpca &lt;- read.table(&quot;/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/HumanPrimaryCellAtlasData-SingleR&quot;)
hpca.names &lt;- read.csv(&quot;/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/HumanPrimaryCellAtlasLabels-SingleR.csv&quot;, row.names=1)
hpca.names$label &lt;- paste0(hpca.names$geo.curated,&quot;_&quot;,hpca.names$label.curated)
i=1
while(i &lt;= length(colnames(hpca))) {
  colnames(hpca)[i] &lt;- hpca.names$label[rownames(hpca.names)==colnames(hpca)[i]]
  i=i+1
}

hpca_subset &lt;- hpca[, grepl(c(&quot;iPSC|MSC|Chondrocytes|Osteoblasts|Adipocyte&quot;), colnames(hpca))]

#Compute the correlation matrix use a seurat object and human primary cell atlas matrix as input
cor.mat.hpca &lt;- ComputeCorMat(ANT1.2,hpca)

cor.mat.hpca_subset &lt;- ComputeCorMat(ANT1.2,hpca_subset)


#Add a metadata column &quot;clusteredCellAtlas&quot; to the object
ANT1.2 &lt;- AssignClusteredLabel(ANT1.2, cor.mat.hpca, K=100, topCorrelation=5)

#Visualize assigned clusters
DimPlot(ANT1.2, reduction=&quot;umap&quot;, group.by=&quot;clusteredCellAtlas&quot;, pt.size=0.5)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/cell%20atlas%20assignment-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>as.data.frame(table(ANT1.2$clusteredCellAtlas))</code></pre>
<pre><code>                                                                                  Var1
1                                 Chondrocytes_MSC-Chondrocyte_MSC-Osteoblast_iPSC_MSC
2                                                Endothelial.cells_Dental.Pulp_Neurons
3                                             iPSC_Fibroblasts_MSC_Smooth.muscle.cells
4                                                                                  MSC
5                                              MSC_MSC-Osteoblast.D3_MSC-Osteoblast.D7
6 Smooth.muscle.cells_Fibroblasts_iPSC_Chondrocytes_MSC_MSC-Chondrocyte_MSC-Osteoblast
7                                                              Smooth.muscle.cells_MSC
  Freq
1    9
2    4
3 1382
4  227
5  155
6   37
7    1</code></pre>
<pre class="r"><code>df_clusteredCellAtlas &lt;- as.data.frame(cbind(ANT1.2$clusteredCellAtlas, ANT1.2$labels)) 
df_clusteredCellAtlas &lt;- dplyr::rename(dplyr::count(df_clusteredCellAtlas, `V1`, `V2`), Freq = n) %&gt;% arrange(V2)
table(df_clusteredCellAtlas$V2)</code></pre>
<pre><code>
  NA18855_Strain NA18855_Unstrain NA18856_Unstrain   NA19160_Strain 
               5                6                5                5 
NA19160_Unstrain 
               3 </code></pre>
<pre class="r"><code>DimPlot(ANT1.2, reduction=&quot;umap&quot;, group.by=&quot;labels&quot;, split.by=&quot;clusteredCellAtlas&quot;, pt.size=0.5, ncol = 3) +
  theme(plot.title = element_text(size=1)) +
  labs(x=&quot;UMAP1&quot;,y=&quot;UMAP2&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/cell%20atlas%20assignment-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#subset data
#Add a metadata column &quot;clusteredCellAtlas&quot; to the object
ANT1.2 &lt;- AssignClusteredLabel(ANT1.2, cor.mat.hpca_subset, K=10, topCorrelation=5)

#Visualize assigned clusters
DimPlot(ANT1.2, reduction=&quot;umap&quot;, group.by=&quot;clusteredCellAtlas&quot;, pt.size=0.5)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/cell%20atlas%20assignment-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>as.data.frame(table(ANT1.2$clusteredCellAtlas))</code></pre>
<pre><code>                                                       Var1 Freq
1      iPSC_MSC_Chondrocytes_MSC-Chondrocyte_MSC-Osteoblast   65
2                                                  MSC_iPSC 1512
3 MSC_MSC-Osteoblast.D3_MSC-Osteoblast.D7_MSC-Osteoblast.D1  237
4       MSC-Chondrocyte_Chondrocytes_iPSC_MSC_MSC-Adipocyte    1</code></pre>
<pre class="r"><code>df_clusteredCellAtlas &lt;- as.data.frame(cbind(ANT1.2$clusteredCellAtlas, ANT1.2$labels)) 
df_clusteredCellAtlas &lt;- dplyr::rename(dplyr::count(df_clusteredCellAtlas, `V1`, `V2`), Freq = n) %&gt;% arrange(V2)
table(df_clusteredCellAtlas$V2)</code></pre>
<pre><code>
  NA18855_Strain NA18855_Unstrain NA18856_Unstrain   NA19160_Strain 
               4                3                3                3 
NA19160_Unstrain 
               2 </code></pre>
<pre class="r"><code>DimPlot(ANT1.2, reduction=&quot;umap&quot;, group.by=&quot;labels&quot;, split.by=&quot;clusteredCellAtlas&quot;, pt.size=0.5, ncol = 3) +
  theme(plot.title = element_text(size=1)) +
  labs(x=&quot;UMAP1&quot;,y=&quot;UMAP2&quot;)</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/cell%20atlas%20assignment-4.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="run-fasttopics-on-only-my-pilot-data-ipsc-chondrocytes" class="section level1">
<h1>Run FastTopics on only my pilot data (iPSC-chondrocytes)</h1>
<pre class="r"><code>library(fastTopics)
SCT_integrated &lt;- readRDS(&quot;data/SCT_ANT12_integrated.rds&quot;)
counts &lt;- SCT_integrated@assays$RNA@counts
counts &lt;- counts[,colSums(counts &gt; 0) &gt; 0]
counts &lt;- t(counts)

# fitall_2 &lt;- fit_poisson_nmf(counts,k = 2,numiter = 200)
# fitall.multinom_2 &lt;- poisson2multinom(fitall_2)
# saveRDS(fitall.multinom_2, &quot;output/fasttopics/fasttopics_k=2.rds&quot;)
# saveRDS(fitall_2, &quot;output/fasttopics/fasttopics_k=2_nmf.rds&quot;)
# 
# fitall_3 &lt;- fit_poisson_nmf(counts,k = 3,numiter = 200)
# fitall.multinom_3 &lt;- poisson2multinom(fitall_3)
# saveRDS(fitall.multinom_3, &quot;output/fasttopics/fasttopics_k=3.rds&quot;)
# saveRDS(fitall_3, &quot;output/fasttopics/fasttopics_k=3_nmf.rds&quot;)
# 
# fitall_4 &lt;- fit_poisson_nmf(counts,k = 4,numiter = 200)
# fitall.multinom_4 &lt;- poisson2multinom(fitall_4)
# saveRDS(fitall.multinom_4, &quot;output/fasttopics/fasttopics_k=4.rds&quot;)
# saveRDS(fitall_4, &quot;output/fasttopics/fasttopics_k=4_nmf.rds&quot;)
# 
# fitall_5 &lt;- fit_poisson_nmf(counts,k = 5,numiter = 200)
# fitall.multinom_5 &lt;- poisson2multinom(fitall_5)
# saveRDS(fitall.multinom_5, &quot;output/fasttopics/fasttopics_k=5.rds&quot;)
# saveRDS(fitall_5, &quot;output/fasttopics/fasttopics_k=5_nmf.rds&quot;)
# 
# fitall_6 &lt;- fit_poisson_nmf(counts,k = 6,numiter = 200)
# fitall.multinom_6 &lt;- poisson2multinom(fitall_6)
# saveRDS(fitall.multinom_6, &quot;output/fasttopics/fasttopics_k=6.rds&quot;)
# saveRDS(fitall_6, &quot;output/fasttopics/fasttopics_k=6_nmf.rds&quot;)
# 
# fitall_7 &lt;- fit_poisson_nmf(counts,k = 7,numiter = 200)
# fitall.multinom_7 &lt;- poisson2multinom(fitall_7)
# saveRDS(fitall.multinom_7, &quot;output/fasttopics/fasttopics_k=7.rds&quot;)
# saveRDS(fitall_7, &quot;output/fasttopics/fasttopics_k=7_nmf.rds&quot;)
# 
# fitall_8 &lt;- fit_poisson_nmf(counts,k = 8,numiter = 200)
# fitall.multinom_8 &lt;- poisson2multinom(fitall_8)
# saveRDS(fitall.multinom_8, &quot;output/fasttopics/fasttopics_k=8.rds&quot;)
# saveRDS(fitall_8, &quot;output/fasttopics/fasttopics_k=8_nmf.rds&quot;)

fitall_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=2_nmf.rds&quot;)
fitall_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=3_nmf.rds&quot;)
fitall_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=4_nmf.rds&quot;)
fitall_5 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=5_nmf.rds&quot;)
fitall_6 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=6_nmf.rds&quot;)
fitall_7 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=7_nmf.rds&quot;)
fitall_8 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=8_nmf.rds&quot;)

fitall.multinom_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=2.rds&quot;)
fitall.multinom_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=3.rds&quot;)
fitall.multinom_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=4.rds&quot;)
fitall.multinom_5 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=5.rds&quot;)
fitall.multinom_6 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=6.rds&quot;)
fitall.multinom_7 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=7.rds&quot;)
fitall.multinom_8 &lt;- readRDS(&quot;output/fasttopics/fasttopics_k=8.rds&quot;)</code></pre>
<p>Visualize</p>
<pre class="r"><code>plot_UMAP_GoM &lt;- function(seurat.obj, fit){
     k &lt;- ncol(fit$L)
     seurat.obj@meta.data &lt;- cbind(seurat.obj@meta.data, fit$L)
     for(topic in 1:k){
          plot(FeaturePlot(seurat.obj, paste0(&quot;k&quot;, topic)))
          plot(VlnPlot(seurat.obj, features = paste0(&quot;k&quot;, topic), ncol = 1, group.by = &quot;labels&quot;)
)
     }
}

for(fit in list(fitall.multinom_2, fitall.multinom_3, fitall.multinom_4, fitall.multinom_5, fitall.multinom_6, fitall.multinom_7, fitall.multinom_8)){
     print(ncol(fit$L))
     plot(DimPlot(SCT_integrated, reduction = &quot;umap&quot;, group.by = &quot;labels&quot;))
     plot(DimPlot(SCT_integrated, reduction = &quot;umap&quot;, group.by = &quot;seurat_clusters&quot;))
     plot_UMAP_GoM(SCT_integrated, fit)
}</code></pre>
<pre><code>[1] 2</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-2.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-3.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-4.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-5.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 3</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-7.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-8.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-9.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-10.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-11.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-12.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-13.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-14.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 4</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-15.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-16.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-17.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-18.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-19.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-20.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-21.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-22.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-23.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-24.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 5</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-25.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-26.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-27.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-28.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-29.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-30.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-31.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-32.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-33.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-34.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-35.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-36.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 6</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-37.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-38.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-39.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-40.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-41.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-42.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-43.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-44.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-45.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-46.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-47.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-48.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-49.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-50.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 7</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-51.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-52.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-53.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-54.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-55.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-56.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-57.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-58.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-59.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-60.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-61.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-62.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-63.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-64.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-65.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-66.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 8</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-67.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-68.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-69.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-70.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-71.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-72.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-73.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-74.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-75.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-76.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-77.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-78.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-79.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-80.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-81.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-82.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-83.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/visualize%20gom%20All-84.png" width="672" style="display: block; margin: auto;" /></p>
<p>Exploratory: Fit fastTopics GoM model to a single sample, then use the semi-supervised version of fastTopics to fix these loadings and fit the rest of the cells</p>
<pre class="r"><code>#subset
NA18855_unstrain_only &lt;- counts[SCT_integrated@meta.data$labels == &quot;NA18855_Unstrain&quot;,]
NA18855_unstrain_only &lt;- NA18855_unstrain_only[,colSums(NA18855_unstrain_only &gt; 0) &gt; 0]
NA18856_unstrain_only &lt;- counts[SCT_integrated@meta.data$labels == &quot;NA18856_Unstrain&quot;,]
NA18856_unstrain_only &lt;- NA18856_unstrain_only[,colSums(NA18856_unstrain_only &gt; 0) &gt; 0]
NA19160_unstrain_only &lt;- counts[SCT_integrated@meta.data$labels == &quot;NA19160_Unstrain&quot;,]
NA19160_unstrain_only &lt;- NA19160_unstrain_only[,colSums(NA19160_unstrain_only &gt; 0) &gt; 0]

#only keep genes with non zero counts in each of the three individuals
gene_overlap &lt;- intersect(intersect(colnames(NA18855_unstrain_only), colnames(NA18856_unstrain_only)), colnames(NA19160_unstrain_only))
NA18855_unstrain_only &lt;- NA18855_unstrain_only[,gene_overlap]
NA18856_unstrain_only &lt;- NA18856_unstrain_only[,gene_overlap]
NA19160_unstrain_only &lt;- NA19160_unstrain_only[,gene_overlap]
counts_gene_overlap &lt;- counts[,gene_overlap]

#Fit GoM with k=2 (and k=3,4 in case that is more interpretable)
# fit18855_2 &lt;- fit_poisson_nmf(NA18855_unstrain_only, k = 2, numiter = 200)
# fit18855.multinom_2 &lt;- poisson2multinom(fit18855_2)
# saveRDS(fit18855.multinom_2, &quot;output/fasttopics/fasttopics_NA18855_only_k=2.rds&quot;)
# saveRDS(fit18855_2, &quot;output/fasttopics/fasttopics_NA18855_only_k=2_nmf.rds&quot;)
# fit18855_3 &lt;- fit_poisson_nmf(NA18855_unstrain_only, k = 3, numiter = 200)
# fit18855.multinom_3 &lt;- poisson2multinom(fit18855_3)
# saveRDS(fit18855.multinom_3, &quot;output/fasttopics/fasttopics_NA18855_only_k=3.rds&quot;)
# saveRDS(fit18855_3, &quot;output/fasttopics/fasttopics_NA18855_only_k=3_nmf.rds&quot;)
# fit18855_4 &lt;- fit_poisson_nmf(NA18855_unstrain_only, k = 4, numiter = 200)
# fit18855.multinom_4 &lt;- poisson2multinom(fit18855_4)
# saveRDS(fit18855.multinom_4, &quot;output/fasttopics/fasttopics_NA18855_only_k=4.rds&quot;)
# saveRDS(fit18855_4, &quot;output/fasttopics/fasttopics_NA18855_only_k=4_nmf.rds&quot;)
# 
# fit18856_2 &lt;- fit_poisson_nmf(NA18856_unstrain_only, k = 2, numiter = 200)
# fit18856.multinom_2 &lt;- poisson2multinom(fit18856_2)
# saveRDS(fit18856.multinom_2, &quot;output/fasttopics/fasttopics_NA18856_only_k=2.rds&quot;)
# saveRDS(fit18856_2, &quot;output/fasttopics/fasttopics_NA18856_only_k=2_nmf.rds&quot;)
# fit18856_3 &lt;- fit_poisson_nmf(NA18856_unstrain_only, k = 3, numiter = 200)
# fit18856.multinom_3 &lt;- poisson2multinom(fit18856_3)
# saveRDS(fit18856.multinom_3, &quot;output/fasttopics/fasttopics_NA18856_only_k=3.rds&quot;)
# saveRDS(fit18856_3, &quot;output/fasttopics/fasttopics_NA18856_only_k=3_nmf.rds&quot;)
# fit18856_4 &lt;- fit_poisson_nmf(NA18856_unstrain_only, k = 4, numiter = 200)
# fit18856.multinom_4 &lt;- poisson2multinom(fit18856_4)
# saveRDS(fit18856.multinom_4, &quot;output/fasttopics/fasttopics_NA18856_only_k=4.rds&quot;)
# saveRDS(fit18856_4, &quot;output/fasttopics/fasttopics_NA18856_only_k=4_nmf.rds&quot;)
# 
# #
# fit19160_2 &lt;- fit_poisson_nmf(NA19160_unstrain_only, k = 2, numiter = 200)
# fit19160.multinom_2 &lt;- poisson2multinom(fit19160_2)
# saveRDS(fit19160.multinom_2, &quot;output/fasttopics/fasttopics_NA19160_only_k=2.rds&quot;)
# saveRDS(fit19160_2, &quot;output/fasttopics/fasttopics_NA19160_only_k=2_nmf.rds&quot;)
# fit19160_3 &lt;- fit_poisson_nmf(NA19160_unstrain_only, k = 3, numiter = 200)
# fit19160.multinom_3 &lt;- poisson2multinom(fit19160_3)
# saveRDS(fit19160.multinom_3, &quot;output/fasttopics/fasttopics_NA19160_only_k=3.rds&quot;)
# saveRDS(fit19160_3, &quot;output/fasttopics/fasttopics_NA19160_only_k=3_nmf.rds&quot;)
# fit19160_4 &lt;- fit_poisson_nmf(NA19160_unstrain_only, k = 4, numiter = 200)
# fit19160.multinom_4 &lt;- poisson2multinom(fit19160_4)
# saveRDS(fit19160.multinom_4, &quot;output/fasttopics/fasttopics_NA19160_only_k=4.rds&quot;)
# saveRDS(fit19160_4, &quot;output/fasttopics/fasttopics_NA19160_only_k=4_nmf.rds&quot;)


fit18855_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=2_nmf.rds&quot;)
fit18855_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=3_nmf.rds&quot;)
fit18855_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=4_nmf.rds&quot;)
fit18856_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=2_nmf.rds&quot;)
fit18856_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=3_nmf.rds&quot;)
fit18856_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=4_nmf.rds&quot;)
fit19160_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=2_nmf.rds&quot;)
fit19160_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=3_nmf.rds&quot;)
fit19160_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=4_nmf.rds&quot;)

fit18855.multinom_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=2.rds&quot;)
fit18855.multinom_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=3.rds&quot;)
fit18855.multinom_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18855_only_k=4.rds&quot;)
fit18856.multinom_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=2.rds&quot;)
fit18856.multinom_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=3.rds&quot;)
fit18856.multinom_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA18856_only_k=4.rds&quot;)
fit19160.multinom_2 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=2.rds&quot;)
fit19160.multinom_3 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=3.rds&quot;)
fit19160.multinom_4 &lt;- readRDS(&quot;output/fasttopics/fasttopics_NA19160_only_k=4.rds&quot;)</code></pre>
<p>Now, use the models that have been fit to fit the remaining samples</p>
<pre class="r"><code>init.fit_18855_2 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18855_2$F,
                 init.method = &quot;random&quot;)
refit_18855_2 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18855_2,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-2 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.438590831e+07 +2.91075641e+07 3.73e+03 1.3e+02 4.7e+00 0.911 0.998 0.00
   2 -2.431182724e+07 +2.89594020e+07 7.80e+02 5.1e+00 1.2e+00 0.911 0.995 0.00
   3 -2.430872899e+07 +2.89532055e+07 1.74e+02 1.1e+00 2.9e-01 0.911 0.993 0.00
   4 -2.430858601e+07 +2.89529195e+07 4.13e+01 2.7e-01 6.4e-02 0.911 0.993 0.00
   5 -2.430857843e+07 +2.89529044e+07 9.59e+00 6.6e-02 1.6e-02 0.911 0.993 0.00
   6 -2.430857799e+07 +2.89529035e+07 2.22e+00 1.7e-02 4.1e-03 0.911 0.993 0.00
   7 -2.430857796e+07 +2.89529034e+07 6.74e-01 4.3e-03 1.1e-03 0.911 0.993 0.00
   8 -2.430857796e+07 +2.89529034e+07 2.10e-01 1.1e-03 3.5e-04 0.911 0.993 0.00
   9 -2.430857796e+07 +2.89529034e+07 6.57e-02 2.9e-04 1.1e-04 0.911 0.993 0.00
  10 -2.430857796e+07 +2.89529034e+07 2.06e-02 7.5e-05 3.6e-05 0.911 0.993 0.00
  11 -2.430857796e+07 +2.89529034e+07 6.71e-03 2.0e-05 1.2e-05 0.911 0.993 0.00
  12 -2.430857796e+07 +2.89529034e+07 2.19e-03 5.3e-06 3.8e-06 0.911 0.993 0.00
  13 -2.430857796e+07 +2.89529034e+07 7.11e-04 1.4e-06 1.2e-06 0.911 0.993 0.00
  14 -2.430857796e+07 +2.89529034e+07 2.29e-04 3.8e-07 4.1e-07 0.911 0.993 0.00
  15 -2.430857796e+07 +2.89529034e+07 7.14e-05 1.1e-07 1.3e-07 0.911 0.993 0.00
  16 -2.430857796e+07 +2.89529034e+07 2.00e-05 2.9e-08 4.3e-08 0.911 0.993 0.00
  17 -2.430857796e+07 +2.89529034e+07 1.65e-05 8.2e-09 1.4e-08 0.911 0.993 0.00
  18 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-09 4.6e-09 0.911 0.993 0.00
  19 -2.430857796e+07 +2.89529034e+07 1.65e-05 6.7e-10 1.5e-09 0.911 0.993 0.00
  20 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.9e-10 4.9e-10 0.911 0.993 0.00
  21 -2.430857796e+07 +2.89529034e+07 1.65e-05 5.6e-11 1.6e-10 0.911 0.993 0.00
  22 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.7e-11 5.2e-11 0.911 0.993 0.00
  23 -2.430857796e+07 +2.89529034e+07 1.65e-05 5.8e-12 1.7e-11 0.911 0.993 0.00
  24 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.4e-12 5.6e-12 0.911 0.993 0.00
  25 -2.430857796e+07 +2.89529034e+07 1.65e-05 9.1e-13 1.8e-12 0.911 0.993 0.00
  26 -2.430857796e+07 +2.89529034e+07 1.65e-05 4.0e-13 6.0e-13 0.911 0.993 0.00
  27 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 1.9e-13 0.911 0.993 0.00
  28 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.1e-13 6.3e-14 0.911 0.993 0.00
  29 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.7e-13 1.9e-14 0.911 0.993 0.00
  30 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.1e-13 7.8e-15 0.911 0.993 0.00
  31 -2.430857796e+07 +2.89529034e+07 1.65e-05 5.7e-14 3.6e-15 0.911 0.993 0.00
  32 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.7e-13 4.0e-15 0.911 0.993 0.00
  33 -2.430857796e+07 +2.89529034e+07 1.65e-05 1.7e-13 2.9e-15 0.911 0.993 0.00
  34 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 4.0e-15 0.911 0.993 0.00
  35 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 4.4e-15 0.911 0.993 0.00
  36 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 4.4e-15 0.911 0.993 0.00
  37 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 6.2e-15 0.911 0.993 0.00
  38 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.8e-13 5.3e-15 0.911 0.993 0.00
  39 -2.430857796e+07 +2.89529034e+07 1.65e-05 2.3e-13 4.4e-15 0.911 0.993 0.00
  40 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  41 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  42 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  43 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  44 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  45 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  46 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  47 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  48 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  49 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  50 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  51 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  52 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  53 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  54 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  55 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  56 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  57 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  58 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  59 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  60 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  61 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  62 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  63 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  64 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  65 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  66 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  67 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  68 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  69 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  70 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  71 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  72 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  73 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  74 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  75 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  76 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  77 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  78 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  79 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  80 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  81 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  82 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  83 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  84 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  85 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  86 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  87 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  88 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  89 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  90 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  91 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  92 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  93 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  94 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  95 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  96 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
  97 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
  98 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
  99 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 100 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 101 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 102 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 103 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 104 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 105 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 106 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 107 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 108 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 109 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 110 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 111 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 112 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 113 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 114 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 115 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 116 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 117 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 118 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 119 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 120 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 121 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 122 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 123 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 124 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 125 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 126 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 127 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 128 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 129 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 130 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 131 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 132 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 133 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 134 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 135 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 136 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 137 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 138 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 139 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 140 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 141 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 142 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 143 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 144 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 145 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 146 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 147 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 148 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 149 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 150 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 151 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 152 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 153 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 154 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 155 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 156 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 157 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 158 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 159 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 160 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 161 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 162 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 163 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 164 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 165 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 166 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 167 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 168 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 169 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 170 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 171 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 172 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 173 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 174 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 175 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 176 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 177 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 178 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 179 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 180 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 181 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 182 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 183 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 184 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 185 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 186 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 187 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 188 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 189 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 190 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 191 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 192 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 193 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 194 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 195 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 196 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.4e-15 0.911 0.993 0.00
 197 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00
 198 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 199 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.2e-15 0.911 0.993 0.00
 200 -2.430857796e+07 +2.89529034e+07 1.65e-05 0.0e+00 2.7e-15 0.911 0.993 0.00</code></pre>
<pre class="r"><code>init.fit_18855_3 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18855_3$F,
                 init.method = &quot;random&quot;)
refit_18855_3 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18855_3,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-3 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.398096255e+07 +2.82976726e+07 6.03e+03 1.3e+02 4.9e+00 0.840 0.986 0.00
   2 -2.390870279e+07 +2.81531531e+07 1.19e+03 4.4e+00 1.6e+00 0.840 0.984 0.00
   3 -2.390701562e+07 +2.81497787e+07 2.38e+02 1.1e+00 3.3e-01 0.840 0.984 0.00
   4 -2.390698104e+07 +2.81497096e+07 4.60e+01 1.7e-01 6.4e-02 0.840 0.984 0.00
   5 -2.390698027e+07 +2.81497081e+07 8.85e+00 2.4e-02 1.2e-02 0.840 0.984 0.00
   6 -2.390698025e+07 +2.81497080e+07 1.70e+00 2.8e-03 2.4e-03 0.840 0.984 0.00
   7 -2.390698025e+07 +2.81497080e+07 3.27e-01 3.0e-04 4.7e-04 0.840 0.984 0.00
   8 -2.390698025e+07 +2.81497080e+07 6.27e-02 3.4e-05 9.0e-05 0.840 0.984 0.00
   9 -2.390698025e+07 +2.81497080e+07 1.21e-02 4.2e-06 1.7e-05 0.840 0.984 0.00
  10 -2.390698025e+07 +2.81497080e+07 2.33e-03 5.7e-07 3.3e-06 0.840 0.984 0.00
  11 -2.390698025e+07 +2.81497080e+07 4.71e-04 6.8e-08 7.4e-07 0.840 0.984 0.00
  12 -2.390698025e+07 +2.81497080e+07 1.29e-04 2.8e-09 1.7e-07 0.840 0.984 0.00
  13 -2.390698025e+07 +2.81497080e+07 1.12e-04 2.1e-09 4.1e-08 0.840 0.984 0.00
  14 -2.390698025e+07 +2.81497080e+07 1.09e-04 1.0e-09 9.6e-09 0.840 0.984 0.00
  15 -2.390698025e+07 +2.81497080e+07 1.08e-04 3.4e-10 2.3e-09 0.840 0.984 0.00
  16 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.0e-10 5.3e-10 0.840 0.984 0.00
  17 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.6e-11 1.3e-10 0.840 0.984 0.00
  18 -2.390698025e+07 +2.81497080e+07 1.08e-04 7.0e-12 3.0e-11 0.840 0.984 0.00
  19 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.6e-12 7.0e-12 0.840 0.984 0.00
  20 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.1e-12 1.7e-12 0.840 0.984 0.00
  21 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 3.9e-13 0.840 0.984 0.00
  22 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 9.0e-14 0.840 0.984 0.00
  23 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 2.2e-14 0.840 0.984 0.00
  24 -2.390698025e+07 +2.81497080e+07 1.08e-04 4.0e-13 5.4e-15 0.840 0.984 0.00
  25 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 4.2e-15 0.840 0.984 0.00
  26 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.1e-13 9.5e-15 0.840 0.984 0.00
  27 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 5.8e-15 0.840 0.984 0.00
  28 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.3e-13 4.4e-15 0.840 0.984 0.00
  29 -2.390698025e+07 +2.81497080e+07 1.08e-04 4.0e-13 7.5e-15 0.840 0.984 0.00
  30 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.1e-15 0.840 0.984 0.00
  31 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
  32 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.3e-13 4.9e-15 0.840 0.984 0.00
  33 -2.390698025e+07 +2.81497080e+07 1.08e-04 3.4e-13 8.9e-15 0.840 0.984 0.00
  34 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
  35 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.3e-13 4.0e-15 0.840 0.984 0.00
  36 -2.390698025e+07 +2.81497080e+07 1.08e-04 3.4e-13 6.9e-15 0.840 0.984 0.00
  37 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.3e-13 4.2e-15 0.840 0.984 0.00
  38 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  39 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
  40 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 4.4e-15 0.840 0.984 0.00
  41 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.1e-13 4.0e-15 0.840 0.984 0.00
  42 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 4.4e-15 0.840 0.984 0.00
  43 -2.390698025e+07 +2.81497080e+07 1.08e-04 4.0e-13 7.5e-15 0.840 0.984 0.00
  44 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 6.2e-15 0.840 0.984 0.00
  45 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 3.8e-15 0.840 0.984 0.00
  46 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 3.6e-15 0.840 0.984 0.00
  47 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 4.0e-15 0.840 0.984 0.00
  48 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.8e-14 4.4e-15 0.840 0.984 0.00
  49 -2.390698025e+07 +2.81497080e+07 1.08e-04 8.5e-14 2.9e-15 0.840 0.984 0.00
  50 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.1e-13 3.6e-15 0.840 0.984 0.00
  51 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.7e-13 3.8e-15 0.840 0.984 0.00
  52 -2.390698025e+07 +2.81497080e+07 1.08e-04 2.3e-13 4.4e-15 0.840 0.984 0.00
  53 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 4.4e-15 0.840 0.984 0.00
  54 -2.390698025e+07 +2.81497080e+07 1.08e-04 1.1e-13 3.1e-15 0.840 0.984 0.00
  55 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  56 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  57 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  58 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  59 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  60 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.4e-15 0.840 0.984 0.00
  61 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  62 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  63 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  64 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  65 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  66 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  67 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  68 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  69 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  70 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  71 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  72 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  73 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  74 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  75 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  76 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  77 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
  78 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  79 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  80 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  81 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  82 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  83 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  84 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  85 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  86 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  87 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  88 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  89 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  90 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  91 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  92 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  93 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  94 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  95 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  96 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
  97 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  98 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
  99 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 100 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 101 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
 102 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 103 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 104 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 105 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 106 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 107 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 108 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 109 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 110 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 111 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 112 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 113 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 114 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 115 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 116 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 117 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 118 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 119 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 120 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 121 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 122 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 123 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 124 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 125 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
 126 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 127 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 128 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 129 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 130 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 131 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 132 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 133 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 134 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 135 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 136 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 137 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 138 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 139 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 140 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 141 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 142 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 143 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 144 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 145 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 146 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 147 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 148 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 149 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
 150 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 151 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 152 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 153 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 154 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 155 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 156 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 157 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 158 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 159 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 160 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 161 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 162 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 163 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 164 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 165 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 166 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 167 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 168 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 169 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 170 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 171 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 172 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 173 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
 174 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 175 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 176 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 177 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 178 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 179 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 180 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 181 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 182 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 183 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 184 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 185 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 186 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 187 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 188 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 189 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 190 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 191 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 192 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 193 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 194 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 195 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 196 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00
 197 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 3.6e-15 0.840 0.984 0.00
 198 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 199 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.7e-15 0.840 0.984 0.00
 200 -2.390698025e+07 +2.81497080e+07 1.08e-04 5.7e-14 2.2e-15 0.840 0.984 0.00</code></pre>
<pre class="r"><code>init.fit_18855_4 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18855_4$F,
                 init.method = &quot;random&quot;)
refit_18855_4 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18855_4,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-4 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.352524840e+07 +2.73862443e+07 2.04e+03 1.0e+02 5.6e+00 0.780 0.964 0.00
   2 -2.346161263e+07 +2.72589728e+07 3.87e+02 6.5e+00 1.2e+00 0.780 0.974 0.00
   3 -2.345951576e+07 +2.72547790e+07 1.32e+02 6.4e-01 4.5e-01 0.780 0.973 0.00
   4 -2.345949725e+07 +2.72547420e+07 8.49e+00 4.1e-02 3.4e-02 0.780 0.973 0.00
   5 -2.345949700e+07 +2.72547415e+07 1.04e+00 3.2e-03 3.6e-03 0.780 0.973 0.00
   6 -2.345949700e+07 +2.72547415e+07 1.35e-01 5.4e-04 4.5e-04 0.780 0.973 0.00
   7 -2.345949700e+07 +2.72547415e+07 1.40e-02 3.9e-05 5.9e-05 0.780 0.973 0.00
   8 -2.345949700e+07 +2.72547415e+07 2.74e-03 1.8e-05 1.3e-05 0.780 0.973 0.00
   9 -2.345949700e+07 +2.72547415e+07 3.84e-04 2.5e-06 2.1e-06 0.780 0.973 0.00
  10 -2.345949700e+07 +2.72547415e+07 7.54e-05 1.6e-07 3.4e-07 0.780 0.973 0.00
  11 -2.345949700e+07 +2.72547415e+07 6.35e-05 8.4e-08 6.9e-08 0.780 0.973 0.00
  12 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-08 1.2e-08 0.780 0.973 0.00
  13 -2.345949700e+07 +2.72547415e+07 6.36e-05 8.1e-10 1.9e-09 0.780 0.973 0.00
  14 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.9e-10 3.5e-10 0.780 0.973 0.00
  15 -2.345949700e+07 +2.72547415e+07 6.36e-05 6.1e-11 6.6e-11 0.780 0.973 0.00
  16 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.5e-12 9.9e-12 0.780 0.973 0.00
  17 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-12 1.6e-12 0.780 0.973 0.00
  18 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.0e-13 0.780 0.973 0.00
  19 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.0e-13 8.0e-14 0.780 0.973 0.00
  20 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 1.2e-14 0.780 0.973 0.00
  21 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-13 1.1e-14 0.780 0.973 0.00
  22 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.0e-13 6.4e-15 0.780 0.973 0.00
  23 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 5.6e-15 0.780 0.973 0.00
  24 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.1e-13 8.9e-15 0.780 0.973 0.00
  25 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 6.2e-15 0.780 0.973 0.00
  26 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 5.8e-15 0.780 0.973 0.00
  27 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.5e-13 6.9e-15 0.780 0.973 0.00
  28 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 4.9e-15 0.780 0.973 0.00
  29 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 6.2e-15 0.780 0.973 0.00
  30 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.0e-13 6.2e-15 0.780 0.973 0.00
  31 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.5e-13 7.5e-15 0.780 0.973 0.00
  32 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.5e-13 8.4e-15 0.780 0.973 0.00
  33 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 5.6e-15 0.780 0.973 0.00
  34 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.6e-15 0.780 0.973 0.00
  35 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.3e-13 4.4e-15 0.780 0.973 0.00
  36 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.3e-13 5.8e-15 0.780 0.973 0.00
  37 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 6.7e-15 0.780 0.973 0.00
  38 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 7.1e-15 0.780 0.973 0.00
  39 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 6.2e-15 0.780 0.973 0.00
  40 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.4e-15 0.780 0.973 0.00
  41 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 3.6e-15 0.780 0.973 0.00
  42 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.0e-15 0.780 0.973 0.00
  43 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 6.2e-15 0.780 0.973 0.00
  44 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 4.9e-15 0.780 0.973 0.00
  45 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 6.2e-15 0.780 0.973 0.00
  46 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 6.2e-15 0.780 0.973 0.00
  47 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.4e-15 0.780 0.973 0.00
  48 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.0e-15 0.780 0.973 0.00
  49 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.6e-15 0.780 0.973 0.00
  50 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 3.8e-15 0.780 0.973 0.00
  51 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.3e-13 4.4e-15 0.780 0.973 0.00
  52 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.1e-13 7.5e-15 0.780 0.973 0.00
  53 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.0e-13 4.7e-15 0.780 0.973 0.00
  54 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.8e-13 5.6e-15 0.780 0.973 0.00
  55 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 5.3e-15 0.780 0.973 0.00
  56 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.1e-15 0.780 0.973 0.00
  57 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.0e-15 0.780 0.973 0.00
  58 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 8.0e-15 0.780 0.973 0.00
  59 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.0e-15 0.780 0.973 0.00
  60 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 6.0e-15 0.780 0.973 0.00
  61 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.3e-13 3.8e-15 0.780 0.973 0.00
  62 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 3.6e-15 0.780 0.973 0.00
  63 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.9e-15 0.780 0.973 0.00
  64 -2.345949700e+07 +2.72547415e+07 6.36e-05 3.4e-13 5.3e-15 0.780 0.973 0.00
  65 -2.345949700e+07 +2.72547415e+07 6.36e-05 2.3e-13 4.7e-15 0.780 0.973 0.00
  66 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.5e-13 6.4e-15 0.780 0.973 0.00
  67 -2.345949700e+07 +2.72547415e+07 6.36e-05 4.5e-13 6.7e-15 0.780 0.973 0.00
  68 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.2e-15 0.780 0.973 0.00
  69 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.6e-15 0.780 0.973 0.00
  70 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.9e-15 0.780 0.973 0.00
  71 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.4e-15 0.780 0.973 0.00
  72 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 3.6e-15 0.780 0.973 0.00
  73 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.3e-15 0.780 0.973 0.00
  74 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 4.0e-15 0.780 0.973 0.00
  75 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.0e-15 0.780 0.973 0.00
  76 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.7e-13 4.4e-15 0.780 0.973 0.00
  77 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 2.7e-15 0.780 0.973 0.00
  78 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 3.6e-15 0.780 0.973 0.00
  79 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 2.9e-15 0.780 0.973 0.00
  80 -2.345949700e+07 +2.72547415e+07 6.36e-05 5.7e-14 3.6e-15 0.780 0.973 0.00
  81 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 3.1e-15 0.780 0.973 0.00
  82 -2.345949700e+07 +2.72547415e+07 6.36e-05 1.1e-13 2.9e-15 0.780 0.973 0.00
  83 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
  84 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
  85 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 4.0e-15 0.780 0.973 0.00
  86 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
  87 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
  88 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.4e-15 0.780 0.973 0.00
  89 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.8e-15 0.780 0.973 0.00
  90 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
  91 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
  92 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
  93 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
  94 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
  95 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
  96 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
  97 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
  98 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
  99 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 100 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 101 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 102 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 103 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 104 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 105 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 106 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 107 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 108 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 109 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 110 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 111 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 112 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 113 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 114 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 115 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 116 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 117 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 118 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 119 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 120 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 121 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 122 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 123 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.2e-15 0.780 0.973 0.00
 124 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 125 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 126 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 127 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 128 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.4e-15 0.780 0.973 0.00
 129 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 130 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 131 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 132 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 133 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 134 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 135 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 136 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 137 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 138 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 139 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 140 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 4.2e-15 0.780 0.973 0.00
 141 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.8e-15 0.780 0.973 0.00
 142 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 143 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 144 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 145 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 146 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 147 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 148 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 149 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 150 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 151 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 152 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 153 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 154 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 155 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 156 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 157 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 158 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 159 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 160 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 161 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 162 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.2e-15 0.780 0.973 0.00
 163 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 164 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 165 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 166 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 167 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 168 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 169 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 170 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 171 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 172 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 173 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 174 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 175 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 4.2e-15 0.780 0.973 0.00
 176 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.8e-15 0.780 0.973 0.00
 177 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 178 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 179 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 180 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 181 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 182 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 183 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 184 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 185 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 186 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 187 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 188 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 189 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 190 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 191 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 192 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 193 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 194 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 195 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.2e-15 0.780 0.973 0.00
 196 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.6e-15 0.780 0.973 0.00
 197 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 3.1e-15 0.780 0.973 0.00
 198 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00
 199 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.9e-15 0.780 0.973 0.00
 200 -2.345949700e+07 +2.72547415e+07 6.36e-05 0.0e+00 2.7e-15 0.780 0.973 0.00</code></pre>
<pre class="r"><code>init.fit_18856_2 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18856_2$F,
                 init.method = &quot;random&quot;)
refit_18856_2 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18856_2,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-2 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.436531817e+07 +2.90663839e+07 4.51e+03 7.4e+01 5.6e+00 0.948 0.997 0.00
   2 -2.432452867e+07 +2.89848049e+07 5.88e+02 1.3e+00 1.1e+00 0.948 0.997 0.00
   3 -2.432385510e+07 +2.89834577e+07 1.05e+02 7.2e-02 2.6e-01 0.948 0.997 0.00
   4 -2.432384204e+07 +2.89834316e+07 1.71e+01 2.1e-02 4.3e-02 0.948 0.997 0.00
   5 -2.432384176e+07 +2.89834310e+07 2.74e+00 4.9e-03 6.8e-03 0.948 0.997 0.00
   6 -2.432384175e+07 +2.89834310e+07 4.38e-01 9.5e-04 1.1e-03 0.948 0.997 0.00
   7 -2.432384175e+07 +2.89834310e+07 7.00e-02 1.7e-04 1.7e-04 0.948 0.997 0.00
   8 -2.432384175e+07 +2.89834310e+07 1.12e-02 2.9e-05 2.8e-05 0.948 0.997 0.00
   9 -2.432384175e+07 +2.89834310e+07 1.79e-03 4.9e-06 4.5e-06 0.948 0.997 0.00
  10 -2.432384175e+07 +2.89834310e+07 2.89e-04 8.4e-07 7.1e-07 0.948 0.997 0.00
  11 -2.432384175e+07 +2.89834310e+07 4.92e-05 1.4e-07 1.1e-07 0.948 0.997 0.00
  12 -2.432384175e+07 +2.89834310e+07 2.30e-05 2.5e-08 1.8e-08 0.948 0.997 0.00
  13 -2.432384175e+07 +2.89834310e+07 2.30e-05 4.3e-09 2.9e-09 0.948 0.997 0.00
  14 -2.432384175e+07 +2.89834310e+07 2.30e-05 7.5e-10 4.6e-10 0.948 0.997 0.00
  15 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.3e-10 7.4e-11 0.948 0.997 0.00
  16 -2.432384175e+07 +2.89834310e+07 2.30e-05 2.1e-11 1.3e-11 0.948 0.997 0.00
  17 -2.432384175e+07 +2.89834310e+07 2.30e-05 4.8e-12 2.7e-12 0.948 0.997 0.00
  18 -2.432384175e+07 +2.89834310e+07 2.30e-05 6.8e-13 5.4e-13 0.948 0.997 0.00
  19 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.1e-13 1.1e-13 0.948 0.997 0.00
  20 -2.432384175e+07 +2.89834310e+07 2.30e-05 2.3e-13 2.2e-14 0.948 0.997 0.00
  21 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.1e-13 4.1e-15 0.948 0.997 0.00
  22 -2.432384175e+07 +2.89834310e+07 2.30e-05 2.3e-13 3.8e-15 0.948 0.997 0.00
  23 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 3.1e-15 0.948 0.997 0.00
  24 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 1.8e-15 0.948 0.997 0.00
  25 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 1.8e-15 0.948 0.997 0.00
  26 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.1e-13 2.0e-15 0.948 0.997 0.00
  27 -2.432384175e+07 +2.89834310e+07 2.30e-05 5.7e-14 2.7e-15 0.948 0.997 0.00
  28 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.1e-13 3.6e-15 0.948 0.997 0.00
  29 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  30 -2.432384175e+07 +2.89834310e+07 2.30e-05 1.1e-13 2.2e-15 0.948 0.997 0.00
  31 -2.432384175e+07 +2.89834310e+07 2.30e-05 5.7e-14 2.7e-15 0.948 0.997 0.00
  32 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.2e-15 0.948 0.997 0.00
  33 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  34 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  35 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  36 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  37 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  38 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  39 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  40 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  41 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  42 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  43 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  44 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  45 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  46 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  47 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  48 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  49 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  50 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  51 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  52 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  53 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  54 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  55 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  56 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  57 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  58 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  59 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  60 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  61 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  62 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  63 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  64 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  65 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  66 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  67 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  68 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  69 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  70 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  71 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  72 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  73 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  74 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  75 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  76 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  77 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  78 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  79 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  80 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  81 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  82 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  83 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  84 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  85 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  86 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  87 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  88 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  89 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  90 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  91 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  92 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  93 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  94 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  95 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  96 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  97 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  98 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
  99 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 100 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 101 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 102 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 103 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 104 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 105 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 106 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 107 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 108 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 109 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 110 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 111 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 112 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 113 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 114 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 115 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 116 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 117 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 118 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 119 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 120 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 121 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 122 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 123 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 124 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 125 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 126 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 127 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 128 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 129 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 130 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 131 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 132 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 133 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 134 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 135 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 136 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 137 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 138 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 139 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 140 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 141 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 142 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 143 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 144 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 145 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 146 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 147 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 148 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 149 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 150 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 151 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 152 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 153 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 154 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 155 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 156 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 157 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 158 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 159 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 160 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 161 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 162 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 163 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 164 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 165 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 166 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 167 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 168 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 169 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 170 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 171 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 172 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 173 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 174 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 175 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 176 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 177 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 178 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 179 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 180 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 181 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 182 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 183 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 184 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 185 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 186 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 187 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 188 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 189 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 190 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 191 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 192 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 193 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 194 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 195 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 196 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 197 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 198 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 199 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00
 200 -2.432384175e+07 +2.89834310e+07 2.30e-05 0.0e+00 2.7e-15 0.948 0.997 0.00</code></pre>
<pre class="r"><code>init.fit_18856_3 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18856_3$F,
                 init.method = &quot;random&quot;)
refit_18856_3 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18856_3,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-3 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.397701280e+07 +2.82897731e+07 3.76e+03 6.0e+01 5.9e+00 0.903 0.989 0.00
   2 -2.391712698e+07 +2.81700015e+07 6.25e+02 8.1e+00 1.3e+00 0.903 0.988 0.00
   3 -2.391376895e+07 +2.81632854e+07 1.43e+02 1.8e+00 9.2e-01 0.903 0.987 0.00
   4 -2.391371607e+07 +2.81631797e+07 3.02e+01 2.1e-01 4.8e-02 0.903 0.987 0.00
   5 -2.391371402e+07 +2.81631756e+07 7.10e+00 3.4e-02 1.1e-02 0.903 0.987 0.00
   6 -2.391371392e+07 +2.81631753e+07 1.67e+00 6.1e-03 2.7e-03 0.903 0.987 0.00
   7 -2.391371391e+07 +2.81631753e+07 4.24e-01 1.2e-03 6.5e-04 0.903 0.987 0.00
   8 -2.391371391e+07 +2.81631753e+07 1.25e-01 2.3e-04 1.9e-04 0.903 0.987 0.00
   9 -2.391371391e+07 +2.81631753e+07 3.66e-02 4.9e-05 5.6e-05 0.903 0.987 0.00
  10 -2.391371391e+07 +2.81631753e+07 1.08e-02 1.0e-05 1.7e-05 0.903 0.987 0.00
  11 -2.391371391e+07 +2.81631753e+07 3.17e-03 2.3e-06 4.9e-06 0.903 0.987 0.00
  12 -2.391371391e+07 +2.81631753e+07 9.35e-04 5.2e-07 1.4e-06 0.903 0.987 0.00
  13 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.2e-07 4.2e-07 0.903 0.987 0.00
  14 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-08 1.4e-07 0.903 0.987 0.00
  15 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.6e-09 4.7e-08 0.903 0.987 0.00
  16 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.6e-09 1.6e-08 0.903 0.987 0.00
  17 -2.391371391e+07 +2.81631753e+07 6.19e-04 3.9e-10 5.3e-09 0.903 0.987 0.00
  18 -2.391371391e+07 +2.81631753e+07 6.19e-04 9.2e-11 1.8e-09 0.903 0.987 0.00
  19 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-11 6.1e-10 0.903 0.987 0.00
  20 -2.391371391e+07 +2.81631753e+07 6.19e-04 8.5e-12 2.0e-10 0.903 0.987 0.00
  21 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.2e-12 6.9e-11 0.903 0.987 0.00
  22 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.8e-13 2.3e-11 0.903 0.987 0.00
  23 -2.391371391e+07 +2.81631753e+07 6.19e-04 8.0e-13 7.8e-12 0.903 0.987 0.00
  24 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.3e-13 2.6e-12 0.903 0.987 0.00
  25 -2.391371391e+07 +2.81631753e+07 6.19e-04 3.4e-13 8.9e-13 0.903 0.987 0.00
  26 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.3e-13 3.0e-13 0.903 0.987 0.00
  27 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 1.0e-13 0.903 0.987 0.00
  28 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 3.4e-14 0.903 0.987 0.00
  29 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.7e-13 1.2e-14 0.903 0.987 0.00
  30 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.0e-13 6.7e-15 0.903 0.987 0.00
  31 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 4.4e-15 0.903 0.987 0.00
  32 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.1e-13 5.1e-15 0.903 0.987 0.00
  33 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.8e-13 8.9e-15 0.903 0.987 0.00
  34 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.1e-13 7.1e-15 0.903 0.987 0.00
  35 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 6.2e-15 0.903 0.987 0.00
  36 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 6.3e-15 0.903 0.987 0.00
  37 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.7e-15 0.903 0.987 0.00
  38 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 5.3e-15 0.903 0.987 0.00
  39 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.0e-13 7.1e-15 0.903 0.987 0.00
  40 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.3e-13 8.0e-15 0.903 0.987 0.00
  41 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.0e-13 5.3e-15 0.903 0.987 0.00
  42 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 3.6e-15 0.903 0.987 0.00
  43 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 4.4e-15 0.903 0.987 0.00
  44 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 6.2e-15 0.903 0.987 0.00
  45 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.1e-13 4.9e-15 0.903 0.987 0.00
  46 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.0e-13 4.7e-15 0.903 0.987 0.00
  47 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 5.3e-15 0.903 0.987 0.00
  48 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 4.0e-15 0.903 0.987 0.00
  49 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.1e-13 2.7e-15 0.903 0.987 0.00
  50 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.7e-15 0.903 0.987 0.00
  51 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.1e-13 6.2e-15 0.903 0.987 0.00
  52 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 6.2e-15 0.903 0.987 0.00
  53 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 3.6e-15 0.903 0.987 0.00
  54 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.1e-13 4.0e-15 0.903 0.987 0.00
  55 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.1e-13 3.1e-15 0.903 0.987 0.00
  56 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.9e-15 0.903 0.987 0.00
  57 -2.391371391e+07 +2.81631753e+07 6.19e-04 5.1e-13 6.4e-15 0.903 0.987 0.00
  58 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 3.6e-15 0.903 0.987 0.00
  59 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.9e-15 0.903 0.987 0.00
  60 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 8.0e-15 0.903 0.987 0.00
  61 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.3e-13 8.0e-15 0.903 0.987 0.00
  62 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.5e-13 7.5e-15 0.903 0.987 0.00
  63 -2.391371391e+07 +2.81631753e+07 6.19e-04 4.0e-13 8.0e-15 0.903 0.987 0.00
  64 -2.391371391e+07 +2.81631753e+07 6.19e-04 6.3e-13 6.7e-15 0.903 0.987 0.00
  65 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 4.4e-15 0.903 0.987 0.00
  66 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.4e-15 0.903 0.987 0.00
  67 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.3e-13 4.4e-15 0.903 0.987 0.00
  68 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.7e-13 3.6e-15 0.903 0.987 0.00
  69 -2.391371391e+07 +2.81631753e+07 6.19e-04 2.8e-13 4.7e-15 0.903 0.987 0.00
  70 -2.391371391e+07 +2.81631753e+07 6.19e-04 1.7e-13 3.6e-15 0.903 0.987 0.00
  71 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
  72 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.9e-15 0.903 0.987 0.00
  73 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
  74 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  75 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  76 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  77 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.2e-15 0.903 0.987 0.00
  78 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
  79 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
  80 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
  81 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
  82 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
  83 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
  84 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
  85 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  86 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  87 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
  88 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
  89 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  90 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
  91 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
  92 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
  93 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
  94 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  95 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  96 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
  97 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
  98 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
  99 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 100 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 101 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 102 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 103 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 104 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 105 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 106 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 107 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 108 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 109 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 110 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 111 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 112 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 113 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 114 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 115 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 116 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 117 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 118 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 119 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 120 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 121 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 122 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 123 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 124 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 125 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 126 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 127 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 128 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 129 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 130 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 131 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 132 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 133 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 134 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 135 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 136 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 137 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 138 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 139 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 140 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 141 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 142 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 143 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 144 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 145 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 146 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 147 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 148 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 149 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 150 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 151 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 152 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 153 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 154 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 155 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 156 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 157 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 158 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 159 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 160 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 161 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 162 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 163 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 164 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 165 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 166 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 167 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 168 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 169 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 170 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 171 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 172 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 173 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 174 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 175 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 176 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 177 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 178 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 179 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 180 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 181 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 182 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 183 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 184 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 185 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 186 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 187 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 188 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 189 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 190 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 191 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 192 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.4e-15 0.903 0.987 0.00
 193 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.0e-15 0.903 0.987 0.00
 194 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 195 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 4.4e-15 0.903 0.987 0.00
 196 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 5.3e-15 0.903 0.987 0.00
 197 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 198 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 2.7e-15 0.903 0.987 0.00
 199 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00
 200 -2.391371391e+07 +2.81631753e+07 6.19e-04 0.0e+00 3.6e-15 0.903 0.987 0.00</code></pre>
<pre class="r"><code>init.fit_18856_4 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit18856_4$F,
                 init.method = &quot;random&quot;)
refit_18856_4 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_18856_4,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-4 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.348353296e+07 +2.73028134e+07 3.84e+03 1.6e+02 5.6e+00 0.856 0.961 0.00
   2 -2.341919415e+07 +2.71741358e+07 6.47e+02 5.0e+00 1.6e+00 0.856 0.964 0.00
   3 -2.341711953e+07 +2.71699866e+07 9.10e+01 3.5e-01 2.8e-01 0.856 0.963 0.00
   4 -2.341707567e+07 +2.71698988e+07 2.11e+01 6.0e-02 6.0e-02 0.856 0.963 0.00
   5 -2.341707470e+07 +2.71698969e+07 4.53e+00 1.3e-02 9.5e-03 0.856 0.963 0.00
   6 -2.341707467e+07 +2.71698968e+07 9.76e-01 2.6e-03 2.1e-03 0.856 0.963 0.00
   7 -2.341707467e+07 +2.71698968e+07 3.57e-01 4.7e-04 5.2e-04 0.856 0.963 0.00
   8 -2.341707467e+07 +2.71698968e+07 4.51e-02 8.8e-05 1.2e-04 0.856 0.963 0.00
   9 -2.341707467e+07 +2.71698968e+07 9.70e-03 2.2e-05 3.0e-05 0.856 0.963 0.00
  10 -2.341707467e+07 +2.71698968e+07 2.09e-03 4.7e-06 7.3e-06 0.856 0.963 0.00
  11 -2.341707467e+07 +2.71698968e+07 4.73e-04 1.0e-06 1.8e-06 0.856 0.963 0.00
  12 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-07 4.3e-07 0.856 0.963 0.00
  13 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.3e-08 1.0e-07 0.856 0.963 0.00
  14 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.2e-08 2.5e-08 0.856 0.963 0.00
  15 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-09 6.0e-09 0.856 0.963 0.00
  16 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.5e-10 1.5e-09 0.856 0.963 0.00
  17 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.5e-10 3.5e-10 0.856 0.963 0.00
  18 -2.341707467e+07 +2.71698968e+07 1.40e-04 3.6e-11 8.5e-11 0.856 0.963 0.00
  19 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.7e-12 2.1e-11 0.856 0.963 0.00
  20 -2.341707467e+07 +2.71698968e+07 1.40e-04 3.4e-12 5.0e-12 0.856 0.963 0.00
  21 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.1e-13 1.2e-12 0.856 0.963 0.00
  22 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.1e-13 2.9e-13 0.856 0.963 0.00
  23 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.0e-14 0.856 0.963 0.00
  24 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 1.7e-14 0.856 0.963 0.00
  25 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
  26 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
  27 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
  28 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.8e-15 0.856 0.963 0.00
  29 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.8e-15 0.856 0.963 0.00
  30 -2.341707467e+07 +2.71698968e+07 1.40e-04 3.4e-13 5.3e-15 0.856 0.963 0.00
  31 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 9.8e-15 0.856 0.963 0.00
  32 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.1e-14 0.856 0.963 0.00
  33 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 9.8e-15 0.856 0.963 0.00
  34 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 6.2e-15 0.856 0.963 0.00
  35 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 5.3e-15 0.856 0.963 0.00
  36 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 6.2e-15 0.856 0.963 0.00
  37 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.9e-15 0.856 0.963 0.00
  38 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 1.1e-14 0.856 0.963 0.00
  39 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.0e-13 9.5e-15 0.856 0.963 0.00
  40 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.3e-15 0.856 0.963 0.00
  41 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.3e-15 0.856 0.963 0.00
  42 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
  43 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 4.9e-15 0.856 0.963 0.00
  44 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.0e-15 0.856 0.963 0.00
  45 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 4.4e-15 0.856 0.963 0.00
  46 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 5.3e-15 0.856 0.963 0.00
  47 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 3.6e-15 0.856 0.963 0.00
  48 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.4e-15 0.856 0.963 0.00
  49 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 7.1e-15 0.856 0.963 0.00
  50 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.6e-15 0.856 0.963 0.00
  51 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.1e-15 0.856 0.963 0.00
  52 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.2e-15 0.856 0.963 0.00
  53 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.1e-15 0.856 0.963 0.00
  54 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 4.4e-15 0.856 0.963 0.00
  55 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.0e-15 0.856 0.963 0.00
  56 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.9e-15 0.856 0.963 0.00
  57 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 9.7e-15 0.856 0.963 0.00
  58 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 9.5e-15 0.856 0.963 0.00
  59 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.0e-15 0.856 0.963 0.00
  60 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 1.0e-14 0.856 0.963 0.00
  61 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.2e-14 0.856 0.963 0.00
  62 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 8.4e-15 0.856 0.963 0.00
  63 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 9.3e-15 0.856 0.963 0.00
  64 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.2e-14 0.856 0.963 0.00
  65 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.4e-14 0.856 0.963 0.00
  66 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.0e-14 0.856 0.963 0.00
  67 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.4e-12 1.2e-14 0.856 0.963 0.00
  68 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 9.8e-15 0.856 0.963 0.00
  69 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
  70 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 4.4e-15 0.856 0.963 0.00
  71 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 6.7e-15 0.856 0.963 0.00
  72 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.9e-15 0.856 0.963 0.00
  73 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.7e-15 0.856 0.963 0.00
  74 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 5.3e-15 0.856 0.963 0.00
  75 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.0e-15 0.856 0.963 0.00
  76 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 4.7e-15 0.856 0.963 0.00
  77 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.3e-13 8.0e-15 0.856 0.963 0.00
  78 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 8.1e-15 0.856 0.963 0.00
  79 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 9.3e-15 0.856 0.963 0.00
  80 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 9.5e-15 0.856 0.963 0.00
  81 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 9.4e-15 0.856 0.963 0.00
  82 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.3e-12 1.2e-14 0.856 0.963 0.00
  83 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.5e-12 1.7e-14 0.856 0.963 0.00
  84 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 1.2e-14 0.856 0.963 0.00
  85 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 5.3e-15 0.856 0.963 0.00
  86 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.0e-15 0.856 0.963 0.00
  87 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.3e-13 6.7e-15 0.856 0.963 0.00
  88 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 8.9e-15 0.856 0.963 0.00
  89 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 5.7e-15 0.856 0.963 0.00
  90 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.2e-15 0.856 0.963 0.00
  91 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.1e-13 6.2e-15 0.856 0.963 0.00
  92 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.7e-15 0.856 0.963 0.00
  93 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 8.0e-15 0.856 0.963 0.00
  94 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 1.2e-14 0.856 0.963 0.00
  95 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 7.1e-15 0.856 0.963 0.00
  96 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.8e-15 0.856 0.963 0.00
  97 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 7.1e-15 0.856 0.963 0.00
  98 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.8e-15 0.856 0.963 0.00
  99 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.2e-15 0.856 0.963 0.00
 100 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 9.3e-15 0.856 0.963 0.00
 101 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.7e-15 0.856 0.963 0.00
 102 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.3e-15 0.856 0.963 0.00
 103 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.3e-15 0.856 0.963 0.00
 104 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 7.5e-15 0.856 0.963 0.00
 105 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 8.4e-15 0.856 0.963 0.00
 106 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 7.8e-15 0.856 0.963 0.00
 107 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.9e-15 0.856 0.963 0.00
 108 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 4.8e-15 0.856 0.963 0.00
 109 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.1e-13 6.2e-15 0.856 0.963 0.00
 110 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 1.1e-14 0.856 0.963 0.00
 111 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.4e-12 1.5e-14 0.856 0.963 0.00
 112 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.1e-14 0.856 0.963 0.00
 113 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 6.2e-15 0.856 0.963 0.00
 114 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 7.2e-15 0.856 0.963 0.00
 115 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.4e-15 0.856 0.963 0.00
 116 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 6.6e-15 0.856 0.963 0.00
 117 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.5e-12 1.7e-14 0.856 0.963 0.00
 118 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.3e-12 1.5e-14 0.856 0.963 0.00
 119 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.9e-15 0.856 0.963 0.00
 120 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 8.9e-15 0.856 0.963 0.00
 121 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.0e-12 9.8e-15 0.856 0.963 0.00
 122 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 7.3e-15 0.856 0.963 0.00
 123 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.3e-13 7.1e-15 0.856 0.963 0.00
 124 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 7.1e-15 0.856 0.963 0.00
 125 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.0e-12 1.2e-14 0.856 0.963 0.00
 126 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 1.1e-14 0.856 0.963 0.00
 127 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.0e-13 8.9e-15 0.856 0.963 0.00
 128 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 1.1e-14 0.856 0.963 0.00
 129 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.4e-15 0.856 0.963 0.00
 130 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.9e-15 0.856 0.963 0.00
 131 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 7.1e-15 0.856 0.963 0.00
 132 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 4.4e-15 0.856 0.963 0.00
 133 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 4.4e-15 0.856 0.963 0.00
 134 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 5.3e-15 0.856 0.963 0.00
 135 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 6.8e-15 0.856 0.963 0.00
 136 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 6.4e-15 0.856 0.963 0.00
 137 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 7.5e-15 0.856 0.963 0.00
 138 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.3e-13 7.1e-15 0.856 0.963 0.00
 139 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.0e-15 0.856 0.963 0.00
 140 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.1e-13 9.2e-15 0.856 0.963 0.00
 141 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 8.9e-15 0.856 0.963 0.00
 142 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-12 1.1e-14 0.856 0.963 0.00
 143 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.7e-13 9.8e-15 0.856 0.963 0.00
 144 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.3e-13 1.1e-14 0.856 0.963 0.00
 145 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.7e-15 0.856 0.963 0.00
 146 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.1e-13 9.8e-15 0.856 0.963 0.00
 147 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 9.3e-15 0.856 0.963 0.00
 148 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.1e-15 0.856 0.963 0.00
 149 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.3e-15 0.856 0.963 0.00
 150 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.7e-15 0.856 0.963 0.00
 151 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 7.5e-15 0.856 0.963 0.00
 152 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.9e-15 0.856 0.963 0.00
 153 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 7.1e-15 0.856 0.963 0.00
 154 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 4.4e-15 0.856 0.963 0.00
 155 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 9.2e-15 0.856 0.963 0.00
 156 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.0e-13 8.2e-15 0.856 0.963 0.00
 157 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.4e-15 0.856 0.963 0.00
 158 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 6.2e-15 0.856 0.963 0.00
 159 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 8.0e-15 0.856 0.963 0.00
 160 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 7.2e-15 0.856 0.963 0.00
 161 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 8.7e-15 0.856 0.963 0.00
 162 -2.341707467e+07 +2.71698968e+07 1.40e-04 9.1e-13 9.8e-15 0.856 0.963 0.00
 163 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.0e-15 0.856 0.963 0.00
 164 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.5e-13 8.9e-15 0.856 0.963 0.00
 165 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 4.9e-15 0.856 0.963 0.00
 166 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 6.2e-15 0.856 0.963 0.00
 167 -2.341707467e+07 +2.71698968e+07 1.40e-04 8.0e-13 1.0e-14 0.856 0.963 0.00
 168 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 7.1e-15 0.856 0.963 0.00
 169 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 5.3e-15 0.856 0.963 0.00
 170 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 4.4e-15 0.856 0.963 0.00
 171 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 3.9e-15 0.856 0.963 0.00
 172 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 4.2e-15 0.856 0.963 0.00
 173 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.2e-15 0.856 0.963 0.00
 174 -2.341707467e+07 +2.71698968e+07 1.40e-04 7.4e-13 8.9e-15 0.856 0.963 0.00
 175 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 6.4e-15 0.856 0.963 0.00
 176 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 3.6e-15 0.856 0.963 0.00
 177 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.3e-13 5.6e-15 0.856 0.963 0.00
 178 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 5.4e-15 0.856 0.963 0.00
 179 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-14 4.0e-15 0.856 0.963 0.00
 180 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 4.0e-15 0.856 0.963 0.00
 181 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 5.4e-15 0.856 0.963 0.00
 182 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-14 4.4e-15 0.856 0.963 0.00
 183 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 5.6e-15 0.856 0.963 0.00
 184 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 5.3e-15 0.856 0.963 0.00
 185 -2.341707467e+07 +2.71698968e+07 1.40e-04 6.8e-13 9.8e-15 0.856 0.963 0.00
 186 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.4e-12 1.6e-14 0.856 0.963 0.00
 187 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 8.7e-15 0.856 0.963 0.00
 188 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
 189 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-14 3.1e-15 0.856 0.963 0.00
 190 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.1e-13 3.1e-15 0.856 0.963 0.00
 191 -2.341707467e+07 +2.71698968e+07 1.40e-04 2.8e-13 5.3e-15 0.856 0.963 0.00
 192 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.0e-13 6.7e-15 0.856 0.963 0.00
 193 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 6.7e-15 0.856 0.963 0.00
 194 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 6.6e-15 0.856 0.963 0.00
 195 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 8.2e-15 0.856 0.963 0.00
 196 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 6.2e-15 0.856 0.963 0.00
 197 -2.341707467e+07 +2.71698968e+07 1.40e-04 5.7e-13 6.2e-15 0.856 0.963 0.00
 198 -2.341707467e+07 +2.71698968e+07 1.40e-04 4.5e-13 5.4e-15 0.856 0.963 0.00
 199 -2.341707467e+07 +2.71698968e+07 1.40e-04 1.7e-13 4.4e-15 0.856 0.963 0.00
 200 -2.341707467e+07 +2.71698968e+07 1.40e-04 3.4e-13 5.3e-15 0.856 0.963 0.00</code></pre>
<pre class="r"><code>init.fit_19160_2 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit19160_2$F,
                 init.method = &quot;random&quot;)
refit_19160_2 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_19160_2,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-2 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.547018705e+07 +3.12761216e+07 4.35e+03 1.6e+02 4.6e+00 0.923 0.993 0.00
   2 -2.537720019e+07 +3.10901479e+07 9.28e+02 2.1e+01 2.5e+00 0.923 0.993 0.00
   3 -2.537169821e+07 +3.10791439e+07 1.76e+02 4.0e+00 1.2e+00 0.923 0.994 0.00
   4 -2.537134900e+07 +3.10784455e+07 5.51e+01 8.4e-01 4.4e-01 0.923 0.994 0.00
   5 -2.537132010e+07 +3.10783877e+07 1.99e+01 2.0e-01 1.3e-01 0.923 0.994 0.00
   6 -2.537131727e+07 +3.10783821e+07 7.07e+00 5.3e-02 3.4e-02 0.923 0.994 0.00
   7 -2.537131696e+07 +3.10783814e+07 2.50e+00 1.5e-02 8.5e-03 0.923 0.994 0.00
   8 -2.537131692e+07 +3.10783814e+07 8.82e-01 4.5e-03 2.3e-03 0.923 0.994 0.00
   9 -2.537131691e+07 +3.10783813e+07 3.14e-01 1.4e-03 9.0e-04 0.923 0.994 0.00
  10 -2.537131691e+07 +3.10783813e+07 1.26e-01 4.5e-04 3.8e-04 0.923 0.994 0.00
  11 -2.537131691e+07 +3.10783813e+07 5.05e-02 1.5e-04 1.6e-04 0.923 0.994 0.00
  12 -2.537131691e+07 +3.10783813e+07 2.02e-02 4.9e-05 7.1e-05 0.923 0.994 0.00
  13 -2.537131691e+07 +3.10783813e+07 8.35e-03 1.7e-05 3.0e-05 0.923 0.994 0.00
  14 -2.537131691e+07 +3.10783813e+07 3.81e-03 5.7e-06 1.3e-05 0.923 0.994 0.00
  15 -2.537131691e+07 +3.10783813e+07 1.74e-03 2.0e-06 5.6e-06 0.923 0.994 0.00
  16 -2.537131691e+07 +3.10783813e+07 8.00e-04 6.7e-07 2.4e-06 0.923 0.994 0.00
  17 -2.537131691e+07 +3.10783813e+07 3.69e-04 2.3e-07 1.0e-06 0.923 0.994 0.00
  18 -2.537131691e+07 +3.10783813e+07 1.73e-04 8.0e-08 4.4e-07 0.923 0.994 0.00
  19 -2.537131691e+07 +3.10783813e+07 8.39e-05 2.7e-08 1.9e-07 0.923 0.994 0.00
  20 -2.537131691e+07 +3.10783813e+07 4.32e-05 8.9e-09 8.2e-08 0.923 0.994 0.00
  21 -2.537131691e+07 +3.10783813e+07 3.46e-05 2.8e-09 3.5e-08 0.923 0.994 0.00
  22 -2.537131691e+07 +3.10783813e+07 3.62e-05 8.5e-10 1.6e-08 0.923 0.994 0.00
  23 -2.537131691e+07 +3.10783813e+07 3.69e-05 2.3e-10 7.2e-09 0.923 0.994 0.00
  24 -2.537131691e+07 +3.10783813e+07 3.71e-05 4.4e-11 3.3e-09 0.923 0.994 0.00
  25 -2.537131691e+07 +3.10783813e+07 3.72e-05 2.8e-12 1.5e-09 0.923 0.994 0.00
  26 -2.537131691e+07 +3.10783813e+07 3.73e-05 8.5e-12 6.8e-10 0.923 0.994 0.00
  27 -2.537131691e+07 +3.10783813e+07 3.73e-05 5.8e-12 3.1e-10 0.923 0.994 0.00
  28 -2.537131691e+07 +3.10783813e+07 3.73e-05 4.3e-12 1.4e-10 0.923 0.994 0.00
  29 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.3e-12 6.4e-11 0.923 0.994 0.00
  30 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-12 2.9e-11 0.923 0.994 0.00
  31 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.2e-12 1.3e-11 0.923 0.994 0.00
  32 -2.537131691e+07 +3.10783813e+07 3.73e-05 8.2e-13 6.1e-12 0.923 0.994 0.00
  33 -2.537131691e+07 +3.10783813e+07 3.73e-05 2.8e-13 2.8e-12 0.923 0.994 0.00
  34 -2.537131691e+07 +3.10783813e+07 3.73e-05 2.3e-13 1.3e-12 0.923 0.994 0.00
  35 -2.537131691e+07 +3.10783813e+07 3.73e-05 2.3e-13 5.8e-13 0.923 0.994 0.00
  36 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-13 2.6e-13 0.923 0.994 0.00
  37 -2.537131691e+07 +3.10783813e+07 3.73e-05 5.7e-14 1.2e-13 0.923 0.994 0.00
  38 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.7e-13 5.5e-14 0.923 0.994 0.00
  39 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-13 2.3e-14 0.923 0.994 0.00
  40 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.4e-13 1.0e-14 0.923 0.994 0.00
  41 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-13 6.2e-15 0.923 0.994 0.00
  42 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-13 3.9e-15 0.923 0.994 0.00
  43 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.1e-13 4.9e-15 0.923 0.994 0.00
  44 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.4e-13 6.3e-15 0.923 0.994 0.00
  45 -2.537131691e+07 +3.10783813e+07 3.73e-05 1.7e-13 5.2e-15 0.923 0.994 0.00
  46 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 4.0e-15 0.923 0.994 0.00
  47 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  48 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  49 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  50 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  51 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  52 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  53 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  54 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  55 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  56 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  57 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  58 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  59 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  60 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  61 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  62 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  63 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  64 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  65 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  66 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  67 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  68 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  69 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  70 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  71 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  72 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  73 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  74 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  75 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  76 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  77 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  78 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  79 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  80 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  81 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  82 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  83 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  84 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  85 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  86 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  87 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  88 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  89 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  90 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  91 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  92 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  93 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  94 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  95 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  96 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  97 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  98 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
  99 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 100 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 101 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 102 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 103 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 104 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 105 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 106 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 107 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 108 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 109 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 110 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 111 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 112 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 113 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 114 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 115 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 116 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 117 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 118 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 119 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 120 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 121 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 122 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 123 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 124 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 125 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 126 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 127 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 128 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 129 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 130 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 131 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 132 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 133 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 134 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 135 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 136 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 137 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 138 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 139 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 140 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 141 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 142 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 143 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 144 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 145 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 146 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 147 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 148 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 149 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 150 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 151 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 152 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 153 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 154 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 155 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 156 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 157 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 158 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 159 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 160 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 161 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 162 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 163 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 164 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 165 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 166 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 167 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 168 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 169 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 170 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 171 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 172 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 173 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 174 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 175 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 176 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 177 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 178 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 179 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 180 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 181 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 182 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 183 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 184 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 185 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 186 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 187 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 188 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 189 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 190 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 191 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 192 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 193 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 194 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 195 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 196 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 197 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 198 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 199 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00
 200 -2.537131691e+07 +3.10783813e+07 3.73e-05 0.0e+00 3.9e-15 0.923 0.994 0.00</code></pre>
<pre class="r"><code>init.fit_19160_3 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit19160_3$F,
                 init.method = &quot;random&quot;)
refit_19160_3 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_19160_3,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-3 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.459246466e+07 +2.95206768e+07 3.01e+03 1.0e+02 4.1e+00 0.872 0.978 0.00
   2 -2.447095379e+07 +2.92776551e+07 6.58e+02 2.0e+01 1.9e+00 0.872 0.979 0.00
   3 -2.446548360e+07 +2.92667147e+07 2.84e+02 4.1e+00 6.5e-01 0.872 0.980 0.00
   4 -2.446516655e+07 +2.92660806e+07 1.26e+02 1.2e+00 2.0e-01 0.872 0.980 0.00
   5 -2.446514710e+07 +2.92660417e+07 4.68e+01 2.8e-01 6.1e-02 0.872 0.980 0.00
   6 -2.446514576e+07 +2.92660390e+07 1.48e+01 5.6e-02 1.6e-02 0.872 0.980 0.00
   7 -2.446514567e+07 +2.92660388e+07 4.17e+00 9.2e-03 4.0e-03 0.872 0.980 0.00
   8 -2.446514566e+07 +2.92660388e+07 1.00e+00 1.3e-03 1.0e-03 0.872 0.980 0.00
   9 -2.446514566e+07 +2.92660388e+07 2.77e-01 3.0e-04 3.6e-04 0.872 0.980 0.00
  10 -2.446514566e+07 +2.92660388e+07 9.79e-02 7.9e-05 1.3e-04 0.872 0.980 0.00
  11 -2.446514566e+07 +2.92660388e+07 3.43e-02 2.8e-05 4.4e-05 0.872 0.980 0.00
  12 -2.446514566e+07 +2.92660388e+07 1.20e-02 6.9e-06 1.5e-05 0.872 0.980 0.00
  13 -2.446514566e+07 +2.92660388e+07 4.15e-03 1.1e-06 5.3e-06 0.872 0.980 0.00
  14 -2.446514566e+07 +2.92660388e+07 1.43e-03 3.9e-07 1.8e-06 0.872 0.980 0.00
  15 -2.446514566e+07 +2.92660388e+07 5.47e-04 1.7e-07 6.8e-07 0.872 0.980 0.00
  16 -2.446514566e+07 +2.92660388e+07 2.32e-04 7.9e-08 2.9e-07 0.872 0.980 0.00
  17 -2.446514566e+07 +2.92660388e+07 9.61e-05 3.4e-08 1.3e-07 0.872 0.980 0.00
  18 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.4e-08 5.5e-08 0.872 0.980 0.00
  19 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.2e-09 2.4e-08 0.872 0.980 0.00
  20 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.0e-09 1.0e-08 0.872 0.980 0.00
  21 -2.446514566e+07 +2.92660388e+07 5.40e-05 7.7e-10 4.5e-09 0.872 0.980 0.00
  22 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.0e-10 2.0e-09 0.872 0.980 0.00
  23 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.2e-10 8.5e-10 0.872 0.980 0.00
  24 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.1e-11 3.7e-10 0.872 0.980 0.00
  25 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.0e-11 1.6e-10 0.872 0.980 0.00
  26 -2.446514566e+07 +2.92660388e+07 5.40e-05 9.8e-12 6.9e-11 0.872 0.980 0.00
  27 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.5e-12 3.0e-11 0.872 0.980 0.00
  28 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.4e-12 1.3e-11 0.872 0.980 0.00
  29 -2.446514566e+07 +2.92660388e+07 5.40e-05 7.4e-13 5.6e-12 0.872 0.980 0.00
  30 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 2.4e-12 0.872 0.980 0.00
  31 -2.446514566e+07 +2.92660388e+07 5.40e-05 9.9e-14 1.0e-12 0.872 0.980 0.00
  32 -2.446514566e+07 +2.92660388e+07 5.40e-05 8.5e-14 4.6e-13 0.872 0.980 0.00
  33 -2.446514566e+07 +2.92660388e+07 5.40e-05 9.9e-14 2.0e-13 0.872 0.980 0.00
  34 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 8.4e-14 0.872 0.980 0.00
  35 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 3.7e-14 0.872 0.980 0.00
  36 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 1.7e-14 0.872 0.980 0.00
  37 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.3e-13 6.6e-15 0.872 0.980 0.00
  38 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.7e-13 4.4e-15 0.872 0.980 0.00
  39 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.3e-13 7.9e-15 0.872 0.980 0.00
  40 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.4e-15 0.872 0.980 0.00
  41 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.4e-15 0.872 0.980 0.00
  42 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.9e-15 0.872 0.980 0.00
  43 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 7.7e-15 0.872 0.980 0.00
  44 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 7.2e-15 0.872 0.980 0.00
  45 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 8.5e-15 0.872 0.980 0.00
  46 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.5e-15 0.872 0.980 0.00
  47 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 7.5e-15 0.872 0.980 0.00
  48 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 7.3e-15 0.872 0.980 0.00
  49 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 1.1e-14 0.872 0.980 0.00
  50 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.3e-15 0.872 0.980 0.00
  51 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 1.1e-14 0.872 0.980 0.00
  52 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 1.1e-14 0.872 0.980 0.00
  53 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.5e-15 0.872 0.980 0.00
  54 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 1.1e-14 0.872 0.980 0.00
  55 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.1e-13 1.1e-14 0.872 0.980 0.00
  56 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 6.4e-15 0.872 0.980 0.00
  57 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 7.2e-15 0.872 0.980 0.00
  58 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 4.7e-15 0.872 0.980 0.00
  59 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 7.1e-15 0.872 0.980 0.00
  60 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 5.8e-15 0.872 0.980 0.00
  61 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 5.8e-15 0.872 0.980 0.00
  62 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 6.9e-15 0.872 0.980 0.00
  63 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 6.4e-15 0.872 0.980 0.00
  64 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.1e-13 6.3e-15 0.872 0.980 0.00
  65 -2.446514566e+07 +2.92660388e+07 5.40e-05 6.3e-13 9.0e-15 0.872 0.980 0.00
  66 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.3e-13 7.1e-15 0.872 0.980 0.00
  67 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.7e-13 9.3e-15 0.872 0.980 0.00
  68 -2.446514566e+07 +2.92660388e+07 5.40e-05 3.4e-13 6.3e-15 0.872 0.980 0.00
  69 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.7e-13 6.2e-15 0.872 0.980 0.00
  70 -2.446514566e+07 +2.92660388e+07 5.40e-05 9.9e-14 6.2e-15 0.872 0.980 0.00
  71 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 6.2e-15 0.872 0.980 0.00
  72 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 5.3e-15 0.872 0.980 0.00
  73 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.5e-13 9.3e-15 0.872 0.980 0.00
  74 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.0e-13 8.3e-15 0.872 0.980 0.00
  75 -2.446514566e+07 +2.92660388e+07 5.40e-05 7.1e-14 6.3e-15 0.872 0.980 0.00
  76 -2.446514566e+07 +2.92660388e+07 5.40e-05 4.3e-14 5.4e-15 0.872 0.980 0.00
  77 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 5.6e-15 0.872 0.980 0.00
  78 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.1e-13 6.9e-15 0.872 0.980 0.00
  79 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.7e-15 0.872 0.980 0.00
  80 -2.446514566e+07 +2.92660388e+07 5.40e-05 2.8e-13 4.4e-15 0.872 0.980 0.00
  81 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.7e-13 7.1e-15 0.872 0.980 0.00
  82 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.6e-15 0.872 0.980 0.00
  83 -2.446514566e+07 +2.92660388e+07 5.40e-05 1.4e-14 4.4e-15 0.872 0.980 0.00
  84 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
  85 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.9e-15 0.872 0.980 0.00
  86 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 4.9e-15 0.872 0.980 0.00
  87 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
  88 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
  89 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.9e-15 0.872 0.980 0.00
  90 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
  91 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.8e-15 0.872 0.980 0.00
  92 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
  93 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
  94 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.9e-15 0.872 0.980 0.00
  95 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
  96 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.0e-15 0.872 0.980 0.00
  97 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
  98 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
  99 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 100 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
 101 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 5.1e-15 0.872 0.980 0.00
 102 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 103 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 104 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
 105 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 106 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.4e-15 0.872 0.980 0.00
 107 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 108 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 109 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.9e-15 0.872 0.980 0.00
 110 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 111 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 112 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 5.6e-15 0.872 0.980 0.00
 113 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.9e-15 0.872 0.980 0.00
 114 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 115 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 2.9e-15 0.872 0.980 0.00
 116 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 117 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 118 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
 119 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 120 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 5.8e-15 0.872 0.980 0.00
 121 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 122 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
 123 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.8e-15 0.872 0.980 0.00
 124 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.1e-15 0.872 0.980 0.00
 125 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 126 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.6e-15 0.872 0.980 0.00
 127 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 128 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 129 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.9e-15 0.872 0.980 0.00
 130 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 131 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
 132 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 133 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 134 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.9e-15 0.872 0.980 0.00
 135 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 136 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 137 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 5.6e-15 0.872 0.980 0.00
 138 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 139 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 140 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 141 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 142 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 143 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
 144 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.2e-15 0.872 0.980 0.00
 145 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 146 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 147 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 148 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.7e-15 0.872 0.980 0.00
 149 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 150 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 151 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.1e-15 0.872 0.980 0.00
 152 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 153 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.1e-15 0.872 0.980 0.00
 154 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 155 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 156 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 157 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 158 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 5.8e-15 0.872 0.980 0.00
 159 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 160 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 161 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 162 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
 163 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 164 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.1e-15 0.872 0.980 0.00
 165 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 2.9e-15 0.872 0.980 0.00
 166 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 167 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.8e-15 0.872 0.980 0.00
 168 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 169 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 170 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.1e-15 0.872 0.980 0.00
 171 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 172 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 173 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.6e-15 0.872 0.980 0.00
 174 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 175 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 176 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.8e-15 0.872 0.980 0.00
 177 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.0e-15 0.872 0.980 0.00
 178 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
 179 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.8e-15 0.872 0.980 0.00
 180 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 181 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 182 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 4.2e-15 0.872 0.980 0.00
 183 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 184 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.1e-15 0.872 0.980 0.00
 185 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.3e-15 0.872 0.980 0.00
 186 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 187 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.7e-15 0.872 0.980 0.00
 188 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 189 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 3.3e-15 0.872 0.980 0.00
 190 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 191 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.8e-15 0.872 0.980 0.00
 192 -2.446514566e+07 +2.92660388e+07 5.40e-05 0.0e+00 4.0e-15 0.872 0.980 0.00
 193 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.1e-15 0.872 0.980 0.00
 194 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.2e-15 0.872 0.980 0.00
 195 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.9e-15 0.872 0.980 0.00
 196 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.6e-15 0.872 0.980 0.00
 197 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.2e-15 0.872 0.980 0.00
 198 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 4.4e-15 0.872 0.980 0.00
 199 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 2.7e-15 0.872 0.980 0.00
 200 -2.446514566e+07 +2.92660388e+07 5.40e-05 5.7e-14 3.1e-15 0.872 0.980 0.00</code></pre>
<pre class="r"><code>init.fit_19160_4 &lt;- init_poisson_nmf(X = counts_gene_overlap,
                 F = fit19160_4$F,
                 init.method = &quot;random&quot;)
refit_19160_4 &lt;- fit_poisson_nmf(X = counts_gene_overlap,
                fit0 = init.fit_19160_4,
                numiter = 200,
                update.factors = NULL)</code></pre>
<pre><code>Using 40 RcppParallel threads.</code></pre>
<pre><code>Fitting rank-4 Poisson NMF to 1815 x 14647 sparse matrix.
Running 200 SCD updates, without extrapolation (fastTopics 0.3-174).
iter   log-likelihood        deviance res(KKT)  |F-F&#39;|  |L-L&#39;| nz(F) nz(L) beta
   1 -2.421905935e+07 +2.87738662e+07 4.57e+03 1.4e+02 4.9e+00 0.814 0.965 0.00
   2 -2.412014622e+07 +2.85760399e+07 8.86e+02 1.5e+01 2.1e+00 0.814 0.978 0.00
   3 -2.411492441e+07 +2.85655963e+07 1.44e+02 4.7e+00 5.2e-01 0.814 0.977 0.00
   4 -2.411473472e+07 +2.85652170e+07 7.22e+01 5.9e-01 8.0e-02 0.814 0.977 0.00
   5 -2.411473055e+07 +2.85652086e+07 8.23e+00 4.4e-02 2.3e-02 0.814 0.977 0.00
   6 -2.411473042e+07 +2.85652083e+07 3.32e+00 3.2e-03 9.3e-03 0.814 0.977 0.00
   7 -2.411473041e+07 +2.85652083e+07 1.34e+00 4.9e-04 3.7e-03 0.814 0.977 0.00
   8 -2.411473041e+07 +2.85652083e+07 5.44e-01 1.3e-04 1.5e-03 0.814 0.977 0.00
   9 -2.411473041e+07 +2.85652083e+07 2.20e-01 5.9e-05 6.2e-04 0.814 0.977 0.00
  10 -2.411473041e+07 +2.85652083e+07 8.92e-02 2.8e-05 2.5e-04 0.814 0.977 0.00
  11 -2.411473041e+07 +2.85652083e+07 3.62e-02 1.1e-05 1.0e-04 0.814 0.977 0.00
  12 -2.411473041e+07 +2.85652083e+07 1.47e-02 4.6e-06 4.1e-05 0.814 0.977 0.00
  13 -2.411473041e+07 +2.85652083e+07 5.94e-03 1.8e-06 1.7e-05 0.814 0.977 0.00
  14 -2.411473041e+07 +2.85652083e+07 2.40e-03 7.4e-07 6.7e-06 0.814 0.977 0.00
  15 -2.411473041e+07 +2.85652083e+07 9.71e-04 3.0e-07 2.7e-06 0.814 0.977 0.00
  16 -2.411473041e+07 +2.85652083e+07 3.91e-04 1.2e-07 1.1e-06 0.814 0.977 0.00
  17 -2.411473041e+07 +2.85652083e+07 1.55e-04 4.9e-08 4.5e-07 0.814 0.977 0.00
  18 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.0e-08 1.8e-07 0.814 0.977 0.00
  19 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.1e-09 7.4e-08 0.814 0.977 0.00
  20 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.3e-09 3.0e-08 0.814 0.977 0.00
  21 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.3e-09 1.2e-08 0.814 0.977 0.00
  22 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.4e-10 4.9e-09 0.814 0.977 0.00
  23 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.2e-10 2.0e-09 0.814 0.977 0.00
  24 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.8e-11 8.0e-10 0.814 0.977 0.00
  25 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.6e-11 3.3e-10 0.814 0.977 0.00
  26 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.6e-11 1.3e-10 0.814 0.977 0.00
  27 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.5e-12 5.4e-11 0.814 0.977 0.00
  28 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.1e-12 2.2e-11 0.814 0.977 0.00
  29 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 8.8e-12 0.814 0.977 0.00
  30 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-12 3.6e-12 0.814 0.977 0.00
  31 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.5e-13 1.4e-12 0.814 0.977 0.00
  32 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.0e-13 5.9e-13 0.814 0.977 0.00
  33 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 2.3e-13 0.814 0.977 0.00
  34 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-14 9.8e-14 0.814 0.977 0.00
  35 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 4.0e-14 0.814 0.977 0.00
  36 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 1.7e-14 0.814 0.977 0.00
  37 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.0e-13 8.0e-15 0.814 0.977 0.00
  38 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 1.9e-14 0.814 0.977 0.00
  39 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.8e-15 0.814 0.977 0.00
  40 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 6.2e-15 0.814 0.977 0.00
  41 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 4.4e-15 0.814 0.977 0.00
  42 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 6.2e-15 0.814 0.977 0.00
  43 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 6.0e-15 0.814 0.977 0.00
  44 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 1.0e-14 0.814 0.977 0.00
  45 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.6e-15 0.814 0.977 0.00
  46 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 1.3e-14 0.814 0.977 0.00
  47 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.1e-13 1.1e-14 0.814 0.977 0.00
  48 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 6.2e-15 0.814 0.977 0.00
  49 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.3e-15 0.814 0.977 0.00
  50 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 4.7e-15 0.814 0.977 0.00
  51 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 7.1e-15 0.814 0.977 0.00
  52 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.6e-13 1.5e-14 0.814 0.977 0.00
  53 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 7.4e-15 0.814 0.977 0.00
  54 -2.411473041e+07 +2.85652083e+07 1.41e-04 0.0e+00 3.6e-15 0.814 0.977 0.00
  55 -2.411473041e+07 +2.85652083e+07 1.41e-04 0.0e+00 4.6e-15 0.814 0.977 0.00
  56 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.4e-14 3.6e-15 0.814 0.977 0.00
  57 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.3e-13 6.2e-15 0.814 0.977 0.00
  58 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.3e-15 0.814 0.977 0.00
  59 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.1e-15 0.814 0.977 0.00
  60 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 7.1e-15 0.814 0.977 0.00
  61 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 6.2e-15 0.814 0.977 0.00
  62 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.3e-15 0.814 0.977 0.00
  63 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 5.8e-15 0.814 0.977 0.00
  64 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.5e-14 5.9e-15 0.814 0.977 0.00
  65 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.3e-13 8.5e-15 0.814 0.977 0.00
  66 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.4e-15 0.814 0.977 0.00
  67 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 4.4e-15 0.814 0.977 0.00
  68 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 4.7e-15 0.814 0.977 0.00
  69 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.8e-15 0.814 0.977 0.00
  70 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.7e-15 0.814 0.977 0.00
  71 -2.411473041e+07 +2.85652083e+07 1.41e-04 0.0e+00 4.6e-15 0.814 0.977 0.00
  72 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.4e-14 4.7e-15 0.814 0.977 0.00
  73 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.9e-15 0.814 0.977 0.00
  74 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 1.2e-14 0.814 0.977 0.00
  75 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 1.6e-14 0.814 0.977 0.00
  76 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 9.3e-15 0.814 0.977 0.00
  77 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 7.1e-15 0.814 0.977 0.00
  78 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 7.9e-15 0.814 0.977 0.00
  79 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 9.0e-15 0.814 0.977 0.00
  80 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.3e-15 0.814 0.977 0.00
  81 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 8.4e-15 0.814 0.977 0.00
  82 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 9.8e-15 0.814 0.977 0.00
  83 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.0e-13 4.9e-15 0.814 0.977 0.00
  84 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 8.0e-15 0.814 0.977 0.00
  85 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 1.8e-14 0.814 0.977 0.00
  86 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.3e-13 1.0e-14 0.814 0.977 0.00
  87 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 6.2e-15 0.814 0.977 0.00
  88 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.6e-15 0.814 0.977 0.00
  89 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 1.1e-14 0.814 0.977 0.00
  90 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 1.3e-14 0.814 0.977 0.00
  91 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 1.3e-14 0.814 0.977 0.00
  92 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 7.1e-15 0.814 0.977 0.00
  93 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 6.2e-15 0.814 0.977 0.00
  94 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.3e-13 9.3e-15 0.814 0.977 0.00
  95 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 5.3e-15 0.814 0.977 0.00
  96 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 4.9e-15 0.814 0.977 0.00
  97 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 9.8e-15 0.814 0.977 0.00
  98 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 8.2e-15 0.814 0.977 0.00
  99 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 1.1e-14 0.814 0.977 0.00
 100 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 3.0e-14 0.814 0.977 0.00
 101 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 2.4e-14 0.814 0.977 0.00
 102 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 8.0e-15 0.814 0.977 0.00
 103 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.1e-15 0.814 0.977 0.00
 104 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 8.2e-15 0.814 0.977 0.00
 105 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 8.9e-15 0.814 0.977 0.00
 106 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 5.7e-15 0.814 0.977 0.00
 107 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.2e-15 0.814 0.977 0.00
 108 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.1e-13 5.8e-15 0.814 0.977 0.00
 109 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 6.2e-15 0.814 0.977 0.00
 110 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 5.3e-15 0.814 0.977 0.00
 111 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 9.1e-15 0.814 0.977 0.00
 112 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.8e-13 9.8e-15 0.814 0.977 0.00
 113 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 4.4e-15 0.814 0.977 0.00
 114 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 8.4e-15 0.814 0.977 0.00
 115 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 8.9e-15 0.814 0.977 0.00
 116 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 7.3e-15 0.814 0.977 0.00
 117 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 7.1e-15 0.814 0.977 0.00
 118 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 1.2e-14 0.814 0.977 0.00
 119 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.6e-15 0.814 0.977 0.00
 120 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 8.9e-15 0.814 0.977 0.00
 121 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 8.0e-15 0.814 0.977 0.00
 122 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 6.7e-15 0.814 0.977 0.00
 123 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 5.3e-15 0.814 0.977 0.00
 124 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 4.2e-15 0.814 0.977 0.00
 125 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 4.9e-15 0.814 0.977 0.00
 126 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.0e-13 7.1e-15 0.814 0.977 0.00
 127 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.2e-15 0.814 0.977 0.00
 128 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.3e-15 0.814 0.977 0.00
 129 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 5.6e-15 0.814 0.977 0.00
 130 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-14 5.6e-15 0.814 0.977 0.00
 131 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-14 4.6e-15 0.814 0.977 0.00
 132 -2.411473041e+07 +2.85652083e+07 1.41e-04 9.9e-14 7.5e-15 0.814 0.977 0.00
 133 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 7.1e-15 0.814 0.977 0.00
 134 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.8e-13 7.1e-15 0.814 0.977 0.00
 135 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 5.6e-15 0.814 0.977 0.00
 136 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.3e-13 6.2e-15 0.814 0.977 0.00
 137 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.3e-15 0.814 0.977 0.00
 138 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.3e-13 7.5e-15 0.814 0.977 0.00
 139 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 4.9e-15 0.814 0.977 0.00
 140 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.2e-15 0.814 0.977 0.00
 141 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.9e-15 0.814 0.977 0.00
 142 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 1.5e-14 0.814 0.977 0.00
 143 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 8.9e-15 0.814 0.977 0.00
 144 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 4.4e-15 0.814 0.977 0.00
 145 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 6.0e-15 0.814 0.977 0.00
 146 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 4.5e-15 0.814 0.977 0.00
 147 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 4.4e-15 0.814 0.977 0.00
 148 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 4.4e-15 0.814 0.977 0.00
 149 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 4.4e-15 0.814 0.977 0.00
 150 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 8.0e-15 0.814 0.977 0.00
 151 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 1.2e-14 0.814 0.977 0.00
 152 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 5.3e-15 0.814 0.977 0.00
 153 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 5.8e-15 0.814 0.977 0.00
 154 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 8.9e-15 0.814 0.977 0.00
 155 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 5.8e-15 0.814 0.977 0.00
 156 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 5.3e-15 0.814 0.977 0.00
 157 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.3e-15 0.814 0.977 0.00
 158 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 6.2e-15 0.814 0.977 0.00
 159 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 7.5e-15 0.814 0.977 0.00
 160 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 6.2e-15 0.814 0.977 0.00
 161 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 4.4e-15 0.814 0.977 0.00
 162 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.1e-15 0.814 0.977 0.00
 163 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.1e-15 0.814 0.977 0.00
 164 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 4.9e-15 0.814 0.977 0.00
 165 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 8.9e-15 0.814 0.977 0.00
 166 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 4.0e-15 0.814 0.977 0.00
 167 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 6.2e-15 0.814 0.977 0.00
 168 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 7.5e-15 0.814 0.977 0.00
 169 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 6.2e-15 0.814 0.977 0.00
 170 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 8.0e-15 0.814 0.977 0.00
 171 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.1e-15 0.814 0.977 0.00
 172 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 5.3e-15 0.814 0.977 0.00
 173 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 6.2e-15 0.814 0.977 0.00
 174 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.1e-15 0.814 0.977 0.00
 175 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 6.0e-15 0.814 0.977 0.00
 176 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 4.4e-15 0.814 0.977 0.00
 177 -2.411473041e+07 +2.85652083e+07 1.41e-04 9.1e-13 7.5e-15 0.814 0.977 0.00
 178 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.0e-12 8.9e-15 0.814 0.977 0.00
 179 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 6.7e-15 0.814 0.977 0.00
 180 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 5.3e-15 0.814 0.977 0.00
 181 -2.411473041e+07 +2.85652083e+07 1.41e-04 5.7e-13 6.3e-15 0.814 0.977 0.00
 182 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 5.1e-15 0.814 0.977 0.00
 183 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 8.9e-15 0.814 0.977 0.00
 184 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 1.1e-14 0.814 0.977 0.00
 185 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 5.3e-15 0.814 0.977 0.00
 186 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 5.3e-15 0.814 0.977 0.00
 187 -2.411473041e+07 +2.85652083e+07 1.41e-04 9.1e-13 6.7e-15 0.814 0.977 0.00
 188 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.8e-15 0.814 0.977 0.00
 189 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 4.0e-15 0.814 0.977 0.00
 190 -2.411473041e+07 +2.85652083e+07 1.41e-04 6.8e-13 5.6e-15 0.814 0.977 0.00
 191 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 7.5e-15 0.814 0.977 0.00
 192 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 7.1e-15 0.814 0.977 0.00
 193 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 5.3e-15 0.814 0.977 0.00
 194 -2.411473041e+07 +2.85652083e+07 1.41e-04 8.0e-13 8.0e-15 0.814 0.977 0.00
 195 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.5e-13 5.1e-15 0.814 0.977 0.00
 196 -2.411473041e+07 +2.85652083e+07 1.41e-04 3.4e-13 1.0e-14 0.814 0.977 0.00
 197 -2.411473041e+07 +2.85652083e+07 1.41e-04 4.0e-13 1.2e-14 0.814 0.977 0.00
 198 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.1e-13 4.6e-15 0.814 0.977 0.00
 199 -2.411473041e+07 +2.85652083e+07 1.41e-04 1.7e-13 5.3e-15 0.814 0.977 0.00
 200 -2.411473041e+07 +2.85652083e+07 1.41e-04 2.3e-13 8.9e-15 0.814 0.977 0.00</code></pre>
<p>Visualize</p>
<pre class="r"><code>#visualize
for(fit in list(refit_18855_2, refit_18855_3, refit_18855_4, refit_18856_2, refit_18856_3, refit_18856_4, refit_19160_2, refit_19160_3, refit_19160_4)){
     print(ncol(fit$L))
     plot(DimPlot(SCT_integrated, reduction = &quot;umap&quot;, group.by = &quot;labels&quot;))
     plot(DimPlot(SCT_integrated, reduction = &quot;umap&quot;, group.by = &quot;seurat_clusters&quot;))
     plot_UMAP_GoM(SCT_integrated, fit)
}</code></pre>
<pre><code>[1] 2</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-2.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-3.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-4.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-5.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 3</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-7.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-8.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-9.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-10.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-11.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-12.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-13.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-14.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 4</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-15.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-16.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-17.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-18.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-19.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-20.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-21.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-22.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-23.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-24.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 2</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-25.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-26.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-27.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-28.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-29.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-30.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 3</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-31.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-32.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-33.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-34.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-35.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-36.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-37.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-38.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 4</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-39.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-40.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-41.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-42.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-43.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-44.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-45.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-46.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-47.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-48.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 2</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-49.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-50.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-51.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-52.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-53.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-54.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 3</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-55.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-56.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-57.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-58.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-59.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-60.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-61.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-62.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>[1] 4</code></pre>
<p><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-63.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-64.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-65.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-66.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-67.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-68.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-69.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-70.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-71.png" width="672" style="display: block; margin: auto;" /><img src="figure/Pilot_GoMAnalysis.Rmd/viz_refit%20semisupervised-72.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] fastTopics_0.3-174          gplots_3.0.1.1             
 [3] cowplot_1.0.0               scales_1.1.1               
 [5] classtpx_0.99.0             colorspace_1.4-1           
 [7] stringi_1.4.6               ggfortify_0.4.10           
 [9] RColorBrewer_1.1-2          MAST_1.10.0                
[11] SingleCellExperiment_1.6.0  SummarizedExperiment_1.14.1
[13] DelayedArray_0.10.0         BiocParallel_1.18.1        
[15] matrixStats_0.56.0          Biobase_2.44.0             
[17] GenomicRanges_1.36.0        GenomeInfoDb_1.20.0        
[19] IRanges_2.18.3              S4Vectors_0.22.1           
[21] BiocGenerics_0.30.0         maptpx_1.9-7               
[23] slam_0.1-47                 CountClust_1.12.0          
[25] forcats_0.4.0               stringr_1.4.0              
[27] dplyr_1.0.2                 purrr_0.3.4                
[29] readr_1.3.1                 tidyr_1.1.2                
[31] tibble_3.0.3                ggplot2_3.3.2              
[33] tidyverse_1.3.0             Seurat_3.1.3               

loaded via a namespace (and not attached):
  [1] reticulate_1.16        tidyselect_1.1.0       lme4_1.1-21           
  [4] htmlwidgets_1.5.1      grid_3.6.1             Rtsne_0.15            
  [7] munsell_0.5.0          codetools_0.2-16       ica_1.0-2             
 [10] future_1.18.0          withr_2.2.0            highr_0.8             
 [13] knitr_1.23             rstudioapi_0.11        ROCR_1.0-7            
 [16] gbRd_0.4-11            listenv_0.8.0          Rdpack_0.11-0         
 [19] labeling_0.3           git2r_0.26.1           GenomeInfoDbData_1.2.1
 [22] MCMCpack_1.4-9         farver_2.0.3           rprojroot_1.3-2       
 [25] coda_0.19-3            vctrs_0.3.4            generics_0.0.2        
 [28] xfun_0.8               R6_2.4.1               rsvd_1.0.1            
 [31] flexmix_2.3-15         bitops_1.0-6           assertthat_0.2.1      
 [34] promises_1.1.1         nnet_7.3-12            gtable_0.3.0          
 [37] npsurv_0.4-0           globals_0.12.5         conquer_1.0.2         
 [40] mcmc_0.9-7             workflowr_1.6.2        MatrixModels_0.4-1    
 [43] rlang_0.4.7            splines_3.6.1          lazyeval_0.2.2        
 [46] broom_0.7.0            yaml_2.2.1             reshape2_1.4.3        
 [49] abind_1.4-5            modelr_0.1.8           backports_1.1.9       
 [52] httpuv_1.5.1           tools_3.6.1            ellipsis_0.3.1        
 [55] ggridges_0.5.1         Rcpp_1.0.5             plyr_1.8.6            
 [58] progress_1.2.2         zlibbioc_1.30.0        RCurl_1.98-1.1        
 [61] prettyunits_1.1.1      pbapply_1.4-0          zoo_1.8-8             
 [64] haven_2.3.1            ggrepel_0.8.2          cluster_2.1.0         
 [67] fs_1.3.1               magrittr_1.5           data.table_1.13.0     
 [70] SparseM_1.78           lmtest_0.9-37          reprex_0.3.0          
 [73] RANN_2.6.1             SQUAREM_2020.4         whisker_0.3-2         
 [76] fitdistrplus_1.0-14    hms_0.5.3              lsei_1.2-0            
 [79] evaluate_0.14          readxl_1.3.1           gridExtra_2.3         
 [82] compiler_3.6.1         KernSmooth_2.23-15     crayon_1.3.4          
 [85] minqa_1.2.4            htmltools_0.5.0        mgcv_1.8-28           
 [88] later_1.1.0.1          RcppParallel_5.0.2     lubridate_1.7.9       
 [91] DBI_1.1.0              dbplyr_1.4.2           MASS_7.3-52           
 [94] rappdirs_0.3.1         boot_1.3-23            Matrix_1.2-18         
 [97] permute_0.9-5          cli_2.0.2              quadprog_1.5-8        
[100] gdata_2.18.0           metap_1.1              igraph_1.2.4.1        
[103] pkgconfig_2.0.3        plotly_4.9.2.1         xml2_1.3.2            
[106] picante_1.8            blme_1.0-4             XVector_0.24.0        
[109] bibtex_0.4.2           rvest_0.3.6            digest_0.6.25         
[112] sctransform_0.2.0      RcppAnnoy_0.0.12       tsne_0.1-3            
[115] vegan_2.5-5            rmarkdown_1.13         cellranger_1.1.0      
[118] leiden_0.3.1           uwot_0.1.5             quantreg_5.61         
[121] gtools_3.8.1           modeltools_0.2-22      nloptr_1.2.2.1        
[124] lifecycle_0.2.0        nlme_3.1-140           jsonlite_1.7.0        
[127] viridisLite_0.3.0      limma_3.40.6           fansi_0.4.1           
[130] pillar_1.4.6           lattice_0.20-41        httr_1.4.2            
[133] survival_2.44-1.1      glue_1.4.2             png_0.1-7             
[136] caTools_1.17.1.2       irlba_2.3.3            future.apply_1.3.0    
[139] ape_5.3               </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
