<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>Analysis of Raw Data</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Analysis of Raw Data</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-11-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreeccaa3e4b0f52e1d768a74045f9cb0efdb6dd0875targetblankccaa3e4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/ccaa3e4b0f52e1d768a74045f9cb0efdb6dd0875" target="_blank">ccaa3e4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreeccaa3e4b0f52e1d768a74045f9cb0efdb6dd0875targetblankccaa3e4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/ccaa3e4b0f52e1d768a74045f9cb0efdb6dd0875" target="_blank">ccaa3e4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/SCT_ANT12_integrated.rds
    Ignored:    data/SCT_integrated_by_individual.rds
    Ignored:    data/Sample.info.full.RNAseq.reordered.csv
    Ignored:    data/Tachmazidou_30664745_ALLOA.txt.gz
    Ignored:    data/alasoo_etal/
    Ignored:    data/caliskan_etal/
    Ignored:    data/external_scRNA/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/filtered_counts_full.rds
    Ignored:    data/gene-annotation_hg37.txt
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/log_normalized_ANT12_integrated.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/norm_filtered_counts_full.rds
    Ignored:    data/raw_counts_full_relabeled.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    output/ANT1.2.rds
    Ignored:    output/GO_terms_enriched.csv
    Ignored:    output/LDSC-SEG/
    Ignored:    output/SNPs.csv
    Ignored:    output/ash_power_analysis/
    Ignored:    output/classtpx/
    Ignored:    output/eQTL_output_strain.csv
    Ignored:    output/eQTL_output_strain_INT.csv
    Ignored:    output/eQTL_output_unstrain.csv
    Ignored:    output/eQTL_output_unstrain_INT.csv
    Ignored:    output/fasttopics/
    Ignored:    output/interaction_genes.rds
    Ignored:    output/least_var_ctl_most_var_treat.rds
    Ignored:    output/mash/
    Ignored:    output/overlap_crossomic.rds
    Ignored:    output/overlap_fungenomic.rds
    Ignored:    output/pairwise_ttest_results_SCTdata.rds
    Ignored:    output/pairwise_ttest_results_integrateddata.rds
    Ignored:    output/permuted_results_DE.rds
    Ignored:    output/strain_top.rds
    Ignored:    output/strong_subset.rds
    Ignored:    output/strong_subset_INT.rds
    Ignored:    output/unstrain_top.rds
    Ignored:    output/voom_results.rds

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd
    Modified:   analysis/Norm_filter.Rmd
    Modified:   analysis/fasttopics_on_unintegrated_data.Rmd
    Modified:   analysis/raw_counts_analysis.Rmd
    Modified:   data/DEgenes_FDR05.rda
    Modified:   data/Sample.info.RNAseq.reordered.csv
    Modified:   fig-limma-treatment-hit-1.eps

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/raw_counts_analysis.Rmd</code>) and HTML (<code>docs/raw_counts_analysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/09ab774d153d2b21b6cc4c63627ed0b448acda1f/docs/raw_counts_analysis.html" target="_blank">09ab774</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-27
</td>
<td>
add html files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/e855314ea98fd1fef1af178e7dc9ca1541f8deec/docs/raw_counts_analysis.html" target="_blank">e855314</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-09-27
</td>
<td>
delete html files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/022f905186e6ab8f784b7266671f2729d9200c6c/docs/raw_counts_analysis.html" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
<td>
change RUVg to use least variable genes found through raw unnormalized data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/26132157c6605eb09d8628dff14a004a6a374809/analysis/raw_counts_analysis.Rmd" target="_blank">2613215</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-01
</td>
<td>
outside bulk datasets comparison to my pilot bulk data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a3565148512e79e1482c774b845a50b8c9c06e74/docs/raw_counts_analysis.html" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/629408f29313ac35a6601656ac80732937de98e1/docs/raw_counts_analysis.html" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Getting to the bottom of the mystery of the changing correlation heatmaps – conclusion: filtering out lowly expressed genes is responsible
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/analysis/raw_counts_analysis.Rmd" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/158fdf3e906a4a488e486cb96684b4e23b567895/docs/raw_counts_analysis.html" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/277772fa43e48353985e2c0360dae84b9286a2b3/docs/raw_counts_analysis.html" target="_blank">277772f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Run raw analysis with RIN scores
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/414262e83ffc17132ef3688335857954f181c8be/docs/raw_counts_analysis.html" target="_blank">414262e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2ebb87bdc9f4434543540478cb23f0d4ec466b2a/analysis/raw_counts_analysis.Rmd" target="_blank">2ebb87b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Add to Raw analysis -&gt; library size
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/2ebb87bdc9f4434543540478cb23f0d4ec466b2a/docs/raw_counts_analysis.html" target="_blank">2ebb87b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Add to Raw analysis -&gt; library size
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/71b21dc32a48e1198c8180d3608b050ed1a2e44a/analysis/raw_counts_analysis.Rmd" target="_blank">71b21dc</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Add lib size plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/4922bc1430dee5ae27ce4f6bc97b51393d1ed0f3/docs/raw_counts_analysis.html" target="_blank">4922bc1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-14
</td>
<td>
Knit analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/raw_counts_analysis.html" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5eb29e88b6057837b2c5c0719c225364eb484690/docs/raw_counts_analysis.html" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/analysis/raw_counts_analysis.Rmd" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/raw_counts_analysis.html" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/analysis/raw_counts_analysis.Rmd" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/raw_counts_analysis.html" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/raw_counts_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/b7948f028fdd5633f9fb0645fee59f12457d68e0/docs/raw_counts_analysis.html" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/11d20cf3138fe01e8eeb5b599396ce1a925881ab/docs/raw_counts_analysis.html" target="_blank">11d20cf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/analysis/raw_counts_analysis.Rmd" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
run through with dummy data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/raw_counts_analysis.html" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
run through with dummy data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/raw_counts_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a8cef1a7772685bd04a25c93dd5d7920caa1e492/analysis/raw_counts_analysis.Rmd" target="_blank">a8cef1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Create analysis files for analysis of BulkRNA pilot data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>#configuration
proj_dir &lt;- &quot;/project2/gilad/anthonyhung/Projects/OAStrain_project/&quot;

#order of samples to relabel count matrix (sample ID in format 18855_1_S)
#NOTE: NEED TO CHANGE THIS TO MATCH ACTUAL SAMPLE ORDERS
samplenames &lt;- c(&quot;18855_3_S&quot;,&quot;19160_3_S&quot;, &quot;18856_3_U&quot;,
                 &quot;18856_1_U&quot;,&quot;18855_2_S&quot;, &quot;18856_2_S&quot;,
                 &quot;19160_3_U&quot;,&quot;18855_2_U&quot;, &quot;19160_2_S&quot;,
                 &quot;18855_1_S&quot;,&quot;18856_1_S&quot;, &quot;19160_1_S&quot;,
                 &quot;19160_2_U&quot;,&quot;19160_1_U&quot;, &quot;18855_1_U&quot;,
                 &quot;18856_3_S&quot;,&quot;18856_2_U&quot;, &quot;18855_3_U&quot;
                 )</code></pre>
<div id="overview" class="section level1">
<h1>Overview:</h1>
<p>Here, I load in the raw count matrix that is the output of the Snakemake pipeline and perform PCA and hierarchical clustering to see what patterns can be observed. I then visualize the pairwise correlations between replicates of the same sample, between samples in the same treatment, and samples from the same individual. Based on QC metrics (only 38.4% aligned reads for sample 02), I removed this sample from further analyses.</p>
</div>
<div id="load-in-bulk-rna-sequencing-data-from-pilot-experiment-and-visualize-correlations-between-samples" class="section level1">
<h1>Load in bulk RNA sequencing data from pilot experiment and visualize correlations between samples</h1>
<pre class="r"><code>library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)
library(&quot;edgeR&quot;)</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre class="r"><code># Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))

# load in counts
raw_counts &lt;- read.table(paste0(proj_dir, &quot;bulkRNAseq/out/counts/counts.txt&quot;), header = T)

# assign row.names
row.names(raw_counts) &lt;- raw_counts$Geneid

# exclude extra data columns
raw_counts &lt;- raw_counts[, -c(1:6)] 

# assign col.names
names(raw_counts) &lt;- samplenames

# remove 02 sample that does not pass QC
#raw_counts &lt;- raw_counts[, -14] 

# save relabeled raw count data matrix
saveRDS(raw_counts, &quot;data/raw_counts_relabeled.rds&quot;)


# Check that there&#39;s no &quot;NAs&quot; in the data
summary(apply(raw_counts, 1, var) == 0) </code></pre>
<pre><code>   Mode   FALSE    TRUE 
logical   33237   25158 </code></pre>
<pre class="r"><code>#Sample information
sampleinfo &lt;- read.csv(&quot;data/Sample.info.RNAseq.csv&quot;)
head(sampleinfo)</code></pre>
<pre><code>  Sample_ID Individual Sex Replicate treatment  RIN LibraryPrepBatch
1 18855_1_S   NA18855    F         1    Strain  9.9                1
2 18856_1_S   NA18856    M         1    Strain  8.5                1
3 19160_1_S   NA19160    M         1    Strain  9.9                1
4 18855_2_S   NA18855    F         2    Strain 10.0                1
5 18856_2_S   NA18856    M         2    Strain  9.6                1
6 19160_2_S   NA19160    M         2    Strain 10.0                1
   LibSize
1 19712744
2 13571439
3 20979958
4 18256163
5 16782764
6 16622407</code></pre>
<pre class="r"><code>#Re-order sample information to match the columns of count matrix
sampleinfo &lt;- sampleinfo[match(samplenames, sampleinfo$Sample_ID),]
#remove 02 sample that does not pass QC
#sampleinfo &lt;- sampleinfo[-14,]
# save reorganized sample information
saveRDS(sampleinfo, &quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="perform-pca-and-hierarchical-clustering-on-raw-data" class="section level1">
<h1>Perform PCA and hierarchical clustering on raw data</h1>
<pre class="r"><code>labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)

#load PCA function
source(&quot;code/PCA_fn.R&quot;)

#spearman
cors &lt;- cor(raw_counts, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

heatmap.2(cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/raw_counts_analysis.Rmd/corr%20heatmap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-1">
Past versions of “corr heatmap-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/022f905186e6ab8f784b7266671f2729d9200c6c/docs/figure/raw_counts_analysis.Rmd/corr heatmap-1.png" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/raw_counts_analysis.Rmd/corr heatmap-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/raw_counts_analysis.Rmd/corr heatmap-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/figure/raw_counts_analysis.Rmd/corr heatmap-1.png" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Pearson
cors &lt;- cor(raw_counts, method=&quot;pearson&quot;, use=&quot;pairwise.complete.obs&quot;)


heatmap.2(cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/raw_counts_analysis.Rmd/corr%20heatmap-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmap-2">
Past versions of “corr heatmap-2.png”
</button>
</p>
<div id="fig-no-spaces-corr-heatmap-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/022f905186e6ab8f784b7266671f2729d9200c6c/docs/figure/raw_counts_analysis.Rmd/corr heatmap-2.png" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/414262e83ffc17132ef3688335857954f181c8be/docs/figure/raw_counts_analysis.Rmd/corr heatmap-2.png" target="_blank">414262e</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="perform-pca-and-color-by-factors-to-look-for-any-potential-correlations" class="section level1">
<h1>Perform PCA and color by factors to look for any potential correlations</h1>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(raw_counts), scale = F)
scores &lt;- pca_genes$x


#Make PCA plots with the factors colored by individual

### PCA Raw Data
for (n in 1:3){
  col.v &lt;- pal[as.integer(sampleinfo$Individual)]
  plot_scores(pca_genes, scores, n, n+1, col.v)
}</code></pre>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-1">
Past versions of “PCA raw-1.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/raw_counts_analysis.Rmd/PCA raw-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/raw_counts_analysis.Rmd/PCA raw-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/figure/raw_counts_analysis.Rmd/PCA raw-1.png" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-2">
Past versions of “PCA raw-2.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/raw_counts_analysis.Rmd/PCA raw-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/raw_counts_analysis.Rmd/PCA raw-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/raw_counts_analysis.Rmd/PCA raw-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/figure/raw_counts_analysis.Rmd/PCA raw-2.png" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-3">
Past versions of “PCA raw-3.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/raw_counts_analysis.Rmd/PCA raw-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/raw_counts_analysis.Rmd/PCA raw-3.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/raw_counts_analysis.Rmd/PCA raw-3.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/figure/raw_counts_analysis.Rmd/PCA raw-3.png" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Make PCA plots with the factors colored by treatment

for (n in 1:3){
  col.v &lt;- pal[as.integer(sampleinfo$treatment)]
  plot_scores(pca_genes, scores, n, n+1, col.v)
}</code></pre>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-4">
Past versions of “PCA raw-4.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/raw_counts_analysis.Rmd/PCA raw-4.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/raw_counts_analysis.Rmd/PCA raw-4.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/raw_counts_analysis.Rmd/PCA raw-4.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/docs/figure/raw_counts_analysis.Rmd/PCA raw-4.png" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-5">
Past versions of “PCA raw-5.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/raw_counts_analysis.Rmd/PCA raw-5.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/raw_counts_analysis.Rmd/PCA%20raw-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCA-raw-6">
Past versions of “PCA raw-6.png”
</button>
</p>
<div id="fig-no-spaces-PCA-raw-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/raw_counts_analysis.Rmd/PCA raw-6.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="library-size-by-individual-x-treatment" class="section level1">
<h1>Library size by individual x treatment</h1>
<pre class="r"><code># Plot library size
raw_counts &lt;- DGEList(raw_counts, group = colnames(raw_counts))

boxplot_library_size &lt;- ggplot(raw_counts$samples, aes(x = sampleinfo$Individual, y = raw_counts$samples$lib.size, fill = sampleinfo$treatment)) + geom_boxplot()
 
boxplot_library_size + labs(title = &quot;Library size by Individual&quot;) + labs(y = &quot;Library size&quot;) + labs(x = &quot;Individual&quot;) + guides(fill=guide_legend(title=&quot;Treatment&quot;))</code></pre>
<p><img src="figure/raw_counts_analysis.Rmd/lib%20size-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-lib-size-1">
Past versions of “lib size-1.png”
</button>
</p>
<div id="fig-no-spaces-lib-size-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/022f905186e6ab8f784b7266671f2729d9200c6c/docs/figure/raw_counts_analysis.Rmd/lib size-1.png" target="_blank">022f905</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2ebb87bdc9f4434543540478cb23f0d4ec466b2a/docs/figure/raw_counts_analysis.Rmd/lib size-1.png" target="_blank">2ebb87b</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] edgeR_3.26.5       limma_3.40.6       scales_1.1.1      
[4] RColorBrewer_1.1-2 ggplot2_3.3.2      gplots_3.0.1.1    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.5         compiler_3.6.1     pillar_1.4.6      
 [4] later_1.1.0.1      git2r_0.26.1       workflowr_1.6.2   
 [7] bitops_1.0-6       tools_3.6.1        digest_0.6.25     
[10] lattice_0.20-41    evaluate_0.14      lifecycle_0.2.0   
[13] tibble_3.0.4       gtable_0.3.0       pkgconfig_2.0.3   
[16] rlang_0.4.8        yaml_2.2.1         xfun_0.8          
[19] withr_2.3.0        stringr_1.4.0      dplyr_1.0.2       
[22] knitr_1.23         generics_0.0.2     fs_1.3.1          
[25] vctrs_0.3.4        gtools_3.8.1       caTools_1.17.1.2  
[28] locfit_1.5-9.1     tidyselect_1.1.0   rprojroot_1.3-2   
[31] grid_3.6.1         glue_1.4.2         R6_2.4.1          
[34] rmarkdown_1.13     gdata_2.18.0       farver_2.0.3      
[37] purrr_0.3.4        magrittr_1.5       whisker_0.3-2     
[40] backports_1.1.10   promises_1.1.1     htmltools_0.5.0   
[43] ellipsis_0.3.1     colorspace_1.4-1   httpuv_1.5.1      
[46] labeling_0.3       KernSmooth_2.23-15 stringi_1.4.6     
[49] munsell_0.5.0      crayon_1.3.4      </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
