<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>Analysis of Technical factors</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Analysis of Technical factors</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-12-16
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree8a223922484f03dd37f02804f03666bf26bbe1d2targetblank8a22392a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/8a223922484f03dd37f02804f03666bf26bbe1d2" target="_blank">8a22392</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintree8a223922484f03dd37f02804f03666bf26bbe1d2targetblank8a22392a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/filtered_counts.rds
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/raw_counts_relabeled.rds

Unstaged changes:
    Modified:   analysis/DE_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8a223922484f03dd37f02804f03666bf26bbe1d2/analysis/technical_factors_analysis.Rmd" target="_blank">8a22392</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-17
</td>
<td>
wflow_publish(files = c(“analysis/preprocessing.Rmd”,
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/analysis/technical_factors_analysis.Rmd" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
run through with dummy data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/technical_factors_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/technical_factors_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a8cef1a7772685bd04a25c93dd5d7920caa1e492/analysis/technical_factors_analysis.Rmd" target="_blank">a8cef1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Create analysis files for analysis of BulkRNA pilot data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="examine-normalizedfiltered-data-and-see-if-any-pcs-correlate-with-technical-factors" class="section level1">
<h1>Examine normalized/filtered data and see if any PCs correlate with technical factors</h1>
<pre class="r"><code>#library(&quot;polycor&quot;)
library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;reshape&quot;)
library(&quot;edgeR&quot;)</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre class="r"><code>library(&quot;RColorBrewer&quot;)
library(&quot;scales&quot;)
library(&quot;cowplot&quot;)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>library(&quot;dplyr&quot;)</code></pre>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:reshape&#39;:

    rename</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code># Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))

#load in normalized/filtered data
filt_cpm &lt;- readRDS(&quot;data/norm_filtered_counts.rds&quot;)
# load in reordered sample information
sampleinfo &lt;- readRDS(&quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="pca-and-hierarchical-clustering-of-normalized-and-filtered-data" class="section level1">
<h1>PCA and hierarchical clustering of normalized and filtered data</h1>
<pre class="r"><code>#Load PCA plotting Function
source(&quot;code/PCA_fn.R&quot;)


# Clustering (original code from Julien Roux)
cors &lt;- cor(filt_cpm, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/clustering-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="perform-pca-and-color-by-factors-to-look-for-any-potential-correlations" class="section level1">
<h1>Perform PCA and color by factors to look for any potential correlations</h1>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(filt_cpm), scale = T)
scores &lt;- pca_genes$x

variances &lt;- pca_genes$sdev^2
explained &lt;- variances / sum(variances)
plot(pca_genes, main = &quot;Variance per PC&quot;)</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Make PCA plots with the factors colored by Individual

### PCs 1 and 2 norm+filt Data
for (n in 1:1){
  col.v &lt;- pal[as.integer(sampleinfo$Individual)]
  plot_scores(pca_genes, scores, n, n+1, col.v)
}</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>### PCs 3 and 4 norm+filt Data

for (n in 3:3){
  col.v &lt;- pal[as.integer(sampleinfo$Individual)]
  plot_scores(pca_genes, scores, n, n+1, col.v)
}</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-3.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="check-that-technical-factors-do-not-correlate-with-pcs-using-polycor" class="section level1">
<h1>Check that technical factors do not correlate with PCs using polycor</h1>
</div>
<div id="remove-unwanted-variation" class="section level1">
<h1>Remove unwanted variation</h1>
<pre class="r"><code>#Use RUV for replicates (adjust k until corr heatmap looks right)</code></pre>
</div>
<div id="clusteringpca-after-removing-unwanted-variation" class="section level1">
<h1>Clustering/PCA after removing unwanted variation</h1>
<pre class="r"><code># # Clustering (original code from Julien Roux)
# cors &lt;- cor(filt_cpm, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)
# 
# 
# heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(samples$Species))], RowSideColors=pal[as.integer(as.factor(samples$Tissue))+9], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))
# 
# # Perform PCA
# pca_genes &lt;- prcomp(t(filt_cpm), scale = F)
# scores &lt;- pca_genes$x
# 
# 
# #Make PCA plots with the factors colored by tissue
# 
# ### PCs 1 and 2 Raw Data
# for (n in 1:1){
#   col.v &lt;- pal[as.integer(samples$Tissue)]
#   plot_scores(pca_genes, scores, n, n+1, col.v)
# }
# 
# ### PCs 3 and 4 Raw Data
# 
# for (n in 3:3){
#   col.v &lt;- pal[as.integer(samples$Tissue)]
#   plot_scores(pca_genes, scores, n, n+1, col.v)
# }</code></pre>
</div>
<div id="save-data-adjusted-for-unwanted-variation" class="section level1">
<h1>Save data adjusted for unwanted variation</h1>
<!-- Calculate the relationship between each recorded covariate and the top 6 PCs. -->
<!-- ```{r pc-covariate-correlation} -->
<!-- p_comps <- 1:6 -->
<!-- pc_cov_cor <- matrix(nrow = ncol(info), ncol = length(p_comps), -->
<!--                      dimnames = list(colnames(info), colnames(pca$x)[p_comps])) -->
<!-- for (pc in p_comps) { -->
<!--   for (covariate in 1:ncol(info)) { -->
<!--     lm_result <- lm(pca$x[, pc] ~ info[, covariate]) -->
<!--     r2 <- summary(lm_result)$r.squared -->
<!--     pc_cov_cor[covariate, pc] <- r2 -->
<!--   } -->
<!-- } -->
<!-- datatable(pc_cov_cor) -->
<!-- ``` -->
<!-- ```{r pc-covariate-correlation-heatmap, fig.width=8} -->
<!-- heatmap.2(pc_cov_cor, trace = "none", margins = c(4, 8)) -->
<!-- ``` -->
<!-- ```{r long} -->
<!-- pc_cov_cor_2 <- as.data.frame(pc_cov_cor) -->
<!-- pc_cov_cor_2$variable <- rownames(pc_cov_cor) -->
<!-- pc_cov_cor_2 <- gather(pc_cov_cor_2, key = "pc", value = "cor", -variable) -->
<!-- head(pc_cov_cor_2) -->
<!-- ``` -->
<!-- ```{r pca-heatmap} -->
<!-- d_heatmap <- pc_cov_cor_2 -->
<!-- d_heatmap$variable <- factor(d_heatmap$variable, -->
<!--                              levels = c("rin", "master_mix", -->
<!--                                         "extraction", "arrival", "infection", -->
<!--                                         "individual", "status", "treatment"), -->
<!--                              labels = c("RNA quality", "Library prep batch", -->
<!--                                         "RNA extraction batch", "Arrival batch", -->
<!--                                         "Infection batch", "Individual", -->
<!--                                         "Susceptibility status", "Treatment")) -->
<!-- pca_heat <- ggplot(d_heatmap, aes(x = pc, y = variable)) + -->
<!--   geom_tile(aes(fill = cor), colour = "white") + -->
<!--   scale_fill_gradient(low = "white", high = "red", limits = c(0, 1)) + -->
<!--   labs(x = "Principal Component", y = "", -->
<!--        title = "Correlation between principal components and experimental variables") -->
<!-- pca_heat -->
<!-- ``` -->
<!-- ```{r pca-data} -->
<!-- pca_data <- cbind(info, pca$x[, p_comps]) -->
<!-- pca_data$status <- factor(pca_data$status, levels = c("contact", "tb"), -->
<!--                           labels = c("resistant", "susceptible")) -->
<!-- pca_data$treatment <- factor(pca_data$treatment, levels = c("none", "infected")) -->
<!-- ``` -->
<!-- PC1 versus PC2. -->
<!-- ```{r pc1-pc2} -->
<!-- pc1v2 <- ggplot(pca_data, aes(x = PC1, y = PC2, color = treatment)) + -->
<!--   geom_text(aes(label = individual)) + -->
<!--   labs(x = sprintf("PC%d (%.2f%%)", 1, round(explained[1] * 100, 2)), -->
<!--        y = sprintf("PC%d (%.2f%%)", 2, round(explained[2] * 100, 2))) -->
<!-- pc1v2 -->
<!-- ``` -->
<!-- PC3 versus PC4. -->
<!-- ```{r pc3-pc4} -->
<!-- pc3v4 <- ggplot(pca_data, aes(x = PC3, y = PC4, color = infection)) + -->
<!--   geom_text(aes(label = individual)) + -->
<!--   labs(x = sprintf("PC%d (%.2f%%)", 3, round(explained[3] * 100, 2)), -->
<!--        y = sprintf("PC%d (%.2f%%)", 4, round(explained[4] * 100, 2))) -->
<!-- pc3v4 -->
<!-- ``` -->
<!-- PC5 versus PC6. -->
<!-- ```{r pc5-pc6} -->
<!-- pc5v6 <- ggplot(pca_data, aes(x = PC5, y = PC6, color = infection)) + -->
<!--   geom_text(aes(label = individual)) + -->
<!--   labs(x = sprintf("PC%d (%.2f%%)", 5, round(explained[5] * 100, 2)), -->
<!--        y = sprintf("PC%d (%.2f%%)", 6, round(explained[6] * 100, 2))) -->
<!-- pc5v6 -->
<!-- ``` -->
<!-- ## Effect of treatment -->
<!-- PC1 is correlated with treatment. -->
<!-- ```{r} -->
<!-- pc_cov_cor_2 %>% filter(variable == "treatment", cor > 0.5) -->
<!-- pc1_treatment <- ggplot(pca_data, aes(x = treatment, y = PC1)) + -->
<!--   geom_boxplot() + -->
<!--   labs(title = "PC1 captures the effect of treatment") -->
<!-- pc1_treatment -->
<!-- ``` -->
<!-- ## Any technical batch effects? -->
<!-- There are multiple factors with a strong correlation with at least one of the first 6 PCs. -->
<!-- ```{r} -->
<!-- pc_cov_cor_2 %>% filter(cor > 0.5) -->
<!-- ``` -->
<!-- Most of the major correlations are with biological variables: treatment or individual. -->
<!-- It is expected that the largest source of variation in gene expression is due to the transcriptional changes induced by the immune response to MTB. -->
<!-- Also, we expect there to be differences between individuals, and this will be explicitly modeled for in the differential expression analysis. -->
<!-- Thus the main concern is the variable infection. -->
<!-- The infection experiments were done as the whole blood samples were obtained, so they are the most confounded with susceptibility status. -->
<!-- ```{r} -->
<!-- pc_cov_cor_2 %>% filter(variable == "infection", cor > 0.5) -->
<!-- table(info$infection[info$treatment == "none"], -->
<!--       info$status[info$treatment == "none"]) -->
<!-- ``` -->
<!-- The infection dates with susceptible samples are 2013-04-05, 2014-06-10, and 2014-11-24. -->
<!-- We clearly see there is variation in PC3 by the date of the infection experiment. -->
<!-- ```{r pc3-infection} -->
<!-- pc3_infection <- ggplot(pca_data, aes(x = infection, y = PC3, fill = treatment)) + -->
<!--   geom_boxplot() + -->
<!--   theme(axis.text.x = element_text(angle = 90)) + -->
<!--   labs(title = "PC3 varies by date of infection") -->
<!-- pc3_infection -->
<!-- ``` -->
<!-- Reassuringly, however, this variation does not correlate strongly with susceptibilty status. -->
<!-- ```{r pc3-status} -->
<!-- pc_cov_cor_2 %>% filter(variable == "status", pc == "PC3") -->
<!-- pc3_status <- ggplot(pca_data, aes(x = reorder(individual, PC3), y = PC3, color = status)) + -->
<!--   geom_point() + -->
<!--   facet_wrap(~treatment) + -->
<!--   labs(title = "PC3 does not correlate with susceptibility status", -->
<!--        x = "Individual") + -->
<!--   theme(axis.text.x = element_text(angle = 90)) -->
<!-- pc3_status -->
<!-- ``` -->
<!-- The situation is the same with PC5. -->
<!-- It varies with infection date. -->
<!-- ```{r pc5-infection} -->
<!-- pc5_infection <- pc3_infection %+% aes(y = PC5) + -->
<!--   labs(title = "PC5 varies by date of infection") -->
<!-- pc5_infection -->
<!-- ``` -->
<!-- But does not correlate strongly with susceptibilty status. -->
<!-- ```{r pc5-status} -->
<!-- pc_cov_cor_2 %>% filter(variable == "status", pc == "PC5") -->
<!-- pc5_status <- pc3_status %+% aes(x = reorder(individual, PC5), y = PC5) + -->
<!--   labs(title = "PC5 does not correlate with susceptibility status", -->
<!--        x = "Individual") -->
<!-- pc5_status -->
<!-- ``` -->
<!-- ## Multipanel figures -->
<!-- PCA -->
<!-- ```{r pca-mulit, fig.width=14, fig.height=14} -->
<!-- plot_grid(pc1v2 + theme(legend.position = "none"), -->
<!--           pc3v4 + theme(legend.position = "none"), -->
<!--           pc5v6 + theme(legend.position = "none"), -->
<!--           pca_heat, -->
<!--           labels = LETTERS[1:4]) -->
<!-- ``` -->
<!-- Batch effects -->
<!-- ```{r effect-of-infection, fig.width=14, fig.height=14} -->
<!-- plot_grid(pc3_infection + theme(legend.position = "bottom"), -->
<!--           pc5_infection  + theme(legend.position = "bottom"), -->
<!--           pc3_status + theme(legend.position = "bottom"), -->
<!--           pc5_status + theme(legend.position = "bottom"), -->
<!--           labels = LETTERS[1:4]) -->
<!-- ``` -->
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_0.8.3        cowplot_0.9.3      scales_1.1.0       RColorBrewer_1.1-2
[5] edgeR_3.20.9       limma_3.34.9       reshape_0.8.8      ggplot2_3.2.1     
[9] gplots_3.0.1.1    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3         plyr_1.8.4         compiler_3.4.3     pillar_1.4.2      
 [5] later_1.0.0        git2r_0.26.1       workflowr_1.5.0    bitops_1.0-6      
 [9] tools_3.4.3        digest_0.6.23      lattice_0.20-35    evaluate_0.14     
[13] lifecycle_0.1.0    tibble_2.1.3       gtable_0.3.0       pkgconfig_2.0.3   
[17] rlang_0.4.2        yaml_2.2.0         xfun_0.11          withr_2.1.2       
[21] stringr_1.4.0      knitr_1.26         fs_1.3.1           gtools_3.8.1      
[25] caTools_1.17.1.3   locfit_1.5-9.1     tidyselect_0.2.5   rprojroot_1.3-2   
[29] grid_3.4.3         glue_1.3.1         R6_2.4.1           rmarkdown_1.18    
[33] gdata_2.18.0       purrr_0.3.3        magrittr_1.5       whisker_0.4       
[37] backports_1.1.5    promises_1.1.0     htmltools_0.4.0    assertthat_0.2.1  
[41] colorspace_1.4-1   httpuv_1.5.2       KernSmooth_2.23-15 stringi_1.4.3     
[45] lazyeval_0.2.2     munsell_0.5.0      crayon_1.3.4      </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
