<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Anthony Hung" />

<meta name="date" content="2019-12-16" />

<title>Analysis of Technical factors</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.13/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">OAStrain</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Analysis of Technical factors</h1>
<h4 class="author">Anthony Hung</h4>
<h4 class="date">2019-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-05-27
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>OAStrain/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191127code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191127)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191127code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191127)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreef67c530e28c716d21836cc7ec59639bf42505212targetblankf67c530a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/anthonyhung/OAStrain/tree/f67c530e28c716d21836cc7ec59639bf42505212" target="_blank">f67c530</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomanthonyhungOAStraintreef67c530e28c716d21836cc7ec59639bf42505212targetblankf67c530a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/anthonyhung/OAStrain/tree/f67c530e28c716d21836cc7ec59639bf42505212" target="_blank">f67c530</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ANT1_2.rds
    Ignored:    data/Combined.common.allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data.rds
    Ignored:    data/Combined_singlecell_data_allGenes.rds
    Ignored:    data/Combined_singlecell_data_allGenes_noliver.rds
    Ignored:    data/Combined_singlecell_data_allGenes_normalized.rds
    Ignored:    data/Combined_singlecell_data_first.rds
    Ignored:    data/Combined_singlecell_data_noliver.rds
    Ignored:    data/DE_results.rds
    Ignored:    data/GoMresults.rds
    Ignored:    data/GoMresultsk3.rds
    Ignored:    data/GoMresultsk4.rds
    Ignored:    data/GoMresultsk5.rds
    Ignored:    data/GoMresultsk6.rds
    Ignored:    data/GoMresultsk7.rds
    Ignored:    data/GoMresultsk8.rds
    Ignored:    data/HumanLiverSeurat.rds
    Ignored:    data/RUVsCPM.rds
    Ignored:    data/RUVsOut.rds
    Ignored:    data/filtered_counts.rds
    Ignored:    data/gom_sup3.rds
    Ignored:    data/gom_sup_3.rda
    Ignored:    data/gom_sup_noiPSCChond_3.rda
    Ignored:    data/humanLiverSubset_hepatocytes.rds
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_7_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_adMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_allMSC_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC1_ENCODEChond.rda
    Ignored:    data/merged_topic_fit_8_classtpx_omega_fix_bmMSC2_ENCODEChond.rda
    Ignored:    data/norm_filtered_counts.rds
    Ignored:    data/norm_filtered_counts_RLE.rds
    Ignored:    data/raw_counts_relabeled.rds
    Ignored:    data/sclabels_noliver.rda
    Ignored:    output/ANT1.2.rds

Untracked files:
    Untracked:  .RDataTmp

Unstaged changes:
    Modified:   analysis/enrichment_DE.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/technical_factors_analysis.Rmd</code>) and HTML (<code>docs/technical_factors_analysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/51cceef198e21d0b03dbcb95ee86fd6a6d88b406/analysis/technical_factors_analysis.Rmd" target="_blank">51cceef</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-05-12
</td>
<td>
box plot of pairwise correlations
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/2bbbdad15237735154e8db0270272fefd55760a9/analysis/technical_factors_analysis.Rmd" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/2bbbdad15237735154e8db0270272fefd55760a9/docs/technical_factors_analysis.html" target="_blank">2bbbdad</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-02-11
</td>
<td>
Update html
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/797305a2d5605d72ff27136f96afcfe42a089f7f/docs/technical_factors_analysis.html" target="_blank">797305a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
include Individual as a FE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/a3565148512e79e1482c774b845a50b8c9c06e74/docs/technical_factors_analysis.html" target="_blank">a356514</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
knit analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/analysis/technical_factors_analysis.Rmd" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/technical_factors_analysis.html" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
<td>
edgeR DE analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/analysis/technical_factors_analysis.Rmd" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Getting to the bottom of the mystery of the changing correlation heatmaps – conclusion: filtering out lowly expressed genes is responsible
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/629408f29313ac35a6601656ac80732937de98e1/docs/technical_factors_analysis.html" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
Getting to the bottom of the mystery of the changing correlation heatmaps – conclusion: filtering out lowly expressed genes is responsible
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/dacaeafc28ab1298a80fdd4dd762933bdfd550b0/docs/technical_factors_analysis.html" target="_blank">dacaeaf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
<td>
RUVseq with replicates
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/analysis/technical_factors_analysis.Rmd" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/158fdf3e906a4a488e486cb96684b4e23b567895/docs/technical_factors_analysis.html" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Use quant_normalization
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/fd9ab65abed3896df96a2bfcc38ddcfe09cb1d1a/analysis/technical_factors_analysis.Rmd" target="_blank">fd9ab65</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Remove MSC from sample info
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/analysis/technical_factors_analysis.Rmd" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Add RIN Scores
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/technical_factors_analysis.html" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
<td>
Add RIN Scores
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/00a66fbc7e4e6e654586196c8171bdd4bfbf7682/docs/technical_factors_analysis.html" target="_blank">00a66fb</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
run GO enrichment analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/5eb29e88b6057837b2c5c0719c225364eb484690/docs/technical_factors_analysis.html" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Knit analysis files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/technical_factors_analysis.html" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
<td>
Correct CPM Cutoff strategy
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/analysis/technical_factors_analysis.Rmd" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/technical_factors_analysis.html" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
<td>
Re-run analyses with real data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/b7948f028fdd5633f9fb0645fee59f12457d68e0/analysis/technical_factors_analysis.Rmd" target="_blank">b7948f0</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-06
</td>
<td>
DE analysis design matrix
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/anthonyhung/OAStrain/11d20cf3138fe01e8eeb5b599396ce1a925881ab/docs/technical_factors_analysis.html" target="_blank">11d20cf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/8a223922484f03dd37f02804f03666bf26bbe1d2/analysis/technical_factors_analysis.Rmd" target="_blank">8a22392</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
wflow_publish(files = c(“analysis/preprocessing.Rmd”,
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a5c9482ccd1a7acb2e0ee078e460cb62420e1bed/analysis/technical_factors_analysis.Rmd" target="_blank">a5c9482</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
run through with dummy data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bb98d141e6a1ad34c642aa8583b371a0f3dc000e/analysis/technical_factors_analysis.Rmd" target="_blank">bb98d14</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Add sampleinfo file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/c1f7cf7e67e7a3372e359d52a2cfa24d1a2672a6/analysis/technical_factors_analysis.Rmd" target="_blank">c1f7cf7</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Start up additional analysis files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/a8cef1a7772685bd04a25c93dd5d7920caa1e492/analysis/technical_factors_analysis.Rmd" target="_blank">a8cef1a</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
<td>
Create analysis files for analysis of BulkRNA pilot data
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="examine-normalizedfiltered-data-and-see-if-any-pcs-correlate-with-technical-factors" class="section level1">
<h1>Examine normalized/filtered data and see if any PCs correlate with technical factors</h1>
<pre class="r"><code>library(&quot;gplots&quot;)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>library(&quot;ggplot2&quot;)
library(&quot;reshape&quot;)
library(&quot;edgeR&quot;)</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre class="r"><code>library(&quot;RColorBrewer&quot;)  
library(&quot;scales&quot;)
library(&quot;cowplot&quot;)</code></pre>
<pre><code>
********************************************************</code></pre>
<pre><code>Note: As of version 1.0.0, cowplot does not change the</code></pre>
<pre><code>  default ggplot2 theme anymore. To recover the previous</code></pre>
<pre><code>  behavior, execute:
  theme_set(theme_cowplot())</code></pre>
<pre><code>********************************************************</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:reshape&#39;:

    stamp</code></pre>
<pre class="r"><code>library(&quot;DT&quot;)
library(&quot;tidyr&quot;)</code></pre>
<pre><code>
Attaching package: &#39;tidyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:reshape&#39;:

    expand, smiths</code></pre>
<pre class="r"><code>library(&quot;RUVSeq&quot;)</code></pre>
<pre><code>Loading required package: Biobase</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>Loading required package: parallel</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following object is masked from &#39;package:limma&#39;:

    plotMA</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, basename, cbind,
    colnames, dirname, do.call, duplicated, eval, evalq, Filter,
    Find, get, grep, grepl, intersect, is.unsorted, lapply, Map,
    mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, Position, rank, rbind, Reduce, rownames, sapply,
    setdiff, sort, table, tapply, union, unique, unsplit, which,
    which.max, which.min</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>Loading required package: EDASeq</code></pre>
<pre><code>Loading required package: ShortRead</code></pre>
<pre><code>Loading required package: BiocParallel</code></pre>
<pre><code>Loading required package: Biostrings</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    expand</code></pre>
<pre><code>The following objects are masked from &#39;package:reshape&#39;:

    expand, rename</code></pre>
<pre><code>The following object is masked from &#39;package:gplots&#39;:

    space</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: XVector</code></pre>
<pre><code>
Attaching package: &#39;Biostrings&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    strsplit</code></pre>
<pre><code>Loading required package: Rsamtools</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<pre><code>Loading required package: GenomicRanges</code></pre>
<pre><code>Loading required package: GenomicAlignments</code></pre>
<pre><code>Loading required package: SummarizedExperiment</code></pre>
<pre><code>Loading required package: DelayedArray</code></pre>
<pre><code>Loading required package: matrixStats</code></pre>
<pre><code>
Attaching package: &#39;matrixStats&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:Biobase&#39;:

    anyMissing, rowMedians</code></pre>
<pre><code>
Attaching package: &#39;DelayedArray&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges</code></pre>
<pre><code>The following object is masked from &#39;package:Biostrings&#39;:

    type</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    aperm, apply, rowsum</code></pre>
<pre class="r"><code>library(&quot;dplyr&quot;)</code></pre>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ShortRead&#39;:

    id</code></pre>
<pre><code>The following objects are masked from &#39;package:GenomicAlignments&#39;:

    first, last</code></pre>
<pre><code>The following object is masked from &#39;package:matrixStats&#39;:

    count</code></pre>
<pre><code>The following objects are masked from &#39;package:GenomicRanges&#39;:

    intersect, setdiff, union</code></pre>
<pre><code>The following object is masked from &#39;package:GenomeInfoDb&#39;:

    intersect</code></pre>
<pre><code>The following objects are masked from &#39;package:Biostrings&#39;:

    collapse, intersect, setdiff, setequal, union</code></pre>
<pre><code>The following object is masked from &#39;package:XVector&#39;:

    slice</code></pre>
<pre><code>The following objects are masked from &#39;package:IRanges&#39;:

    collapse, desc, intersect, setdiff, slice, union</code></pre>
<pre><code>The following objects are masked from &#39;package:S4Vectors&#39;:

    first, intersect, rename, setdiff, setequal, union</code></pre>
<pre><code>The following object is masked from &#39;package:Biobase&#39;:

    combine</code></pre>
<pre><code>The following objects are masked from &#39;package:BiocGenerics&#39;:

    combine, intersect, setdiff, union</code></pre>
<pre><code>The following object is masked from &#39;package:reshape&#39;:

    rename</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code># Load colors 
colors &lt;- colorRampPalette(c(brewer.pal(9, &quot;Blues&quot;)[1],brewer.pal(9, &quot;Blues&quot;)[9]))(100)
pal &lt;- c(brewer.pal(9, &quot;Set1&quot;), brewer.pal(8, &quot;Set2&quot;), brewer.pal(12, &quot;Set3&quot;))

#load in normalized/filtered data
filt_norm_counts &lt;- readRDS(&quot;data/norm_filtered_counts.rds&quot;)
#load in filtered counts
filt_counts &lt;- readRDS(&quot;data/filtered_counts.rds&quot;)

# load in reordered sample information
sampleinfo &lt;- readRDS(&quot;data/Sample.info.RNAseq.reordered.csv&quot;)</code></pre>
</div>
<div id="pca-and-hierarchical-clustering-of-normalized-and-filtered-data" class="section level1">
<h1>PCA and hierarchical clustering of normalized and filtered data</h1>
<pre class="r"><code>#Load PCA plotting Function
source(&quot;code/PCA_fn.R&quot;)


#spearman
cors &lt;- cor(filt_norm_counts, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/clustering%20norm_filt_data-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-norm_filt_data-1">
Past versions of “clustering norm_filt_data-1.png”
</button>
</p>
<div id="fig-no-spaces-clustering-norm_filt_data-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/clustering norm_filt_data-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/clustering norm_filt_data-1.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#extract correlations between technical replicates
coords &lt;- cbind(rbind(c(1,5),c(1,10), c(2,9), c(2,12), c(3,4), c(3,16), c(4,16), c(5,10), c(6,11), c(6,15), c(7,13), c(8,14), c(8,17), c(9,12), c(11,15), c(14,17)))
colnames(coords) &lt;- c(&quot;index1&quot;,&quot;index2&quot;)
coords</code></pre>
<pre><code>      index1 index2
 [1,]      1      5
 [2,]      1     10
 [3,]      2      9
 [4,]      2     12
 [5,]      3      4
 [6,]      3     16
 [7,]      4     16
 [8,]      5     10
 [9,]      6     11
[10,]      6     15
[11,]      7     13
[12,]      8     14
[13,]      8     17
[14,]      9     12
[15,]     11     15
[16,]     14     17</code></pre>
<pre class="r"><code>cors_technical &lt;- cors[coords]
median(cors_technical)</code></pre>
<pre><code>[1] 0.9870723</code></pre>
<pre class="r"><code>min(cors_technical)</code></pre>
<pre><code>[1] 0.9717589</code></pre>
<pre class="r"><code>#pearson
cors &lt;- cor(filt_norm_counts, method=&quot;pearson&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/clustering%20norm_filt_data-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-clustering-norm_filt_data-2">
Past versions of “clustering norm_filt_data-2.png”
</button>
</p>
<div id="fig-no-spaces-clustering-norm_filt_data-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/clustering norm_filt_data-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/clustering norm_filt_data-2.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="perform-pca-and-color-by-factors-to-look-for-any-potential-correlations" class="section level1">
<h1>Perform PCA and color by factors to look for any potential correlations</h1>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(filt_norm_counts), scale = T)
scores &lt;- pca_genes$x

variances &lt;- pca_genes$sdev^2
explained &lt;- variances / sum(variances)
plot(pca_genes, main = &quot;Variance per PC&quot;)</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-1">
Past versions of PCA-1.png
</button>
</p>
<div id="fig-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/11d20cf3138fe01e8eeb5b599396ce1a925881ab/docs/figure/technical_factors_analysis.Rmd/PCA-1.png" target="_blank">11d20cf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Make PCA plots with the factors colored by Individual

### PCA norm+filt Data
for (n in 1:3){
  col.v &lt;- pal[as.integer(sampleinfo$Individual)]
  plot_scores(pca_genes, scores, n, n+1, col.v)
}</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-2">
Past versions of PCA-2.png
</button>
</p>
<div id="fig-PCA-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/11d20cf3138fe01e8eeb5b599396ce1a925881ab/docs/figure/technical_factors_analysis.Rmd/PCA-2.png" target="_blank">11d20cf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-3">
Past versions of PCA-3.png
</button>
</p>
<div id="fig-PCA-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/11d20cf3138fe01e8eeb5b599396ce1a925881ab/docs/figure/technical_factors_analysis.Rmd/PCA-3.png" target="_blank">11d20cf</a>
</td>
<td>
Anthony Hung
</td>
<td>
2019-12-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/technical_factors_analysis.Rmd/PCA-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PCA-4">
Past versions of PCA-4.png
</button>
</p>
<div id="fig-PCA-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/PCA-4.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/PCA-4.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="check-that-technical-factors-do-not-correlate-with-pcs" class="section level1">
<h1>Check that technical factors do not correlate with PCs</h1>
<pre class="r"><code># Calculate the relationship between each recorded covariate and the top 6 PCs. 
p_comps &lt;- 1:6
info &lt;- sampleinfo %&gt;% 
     dplyr::select(c(Individual, Sex, Replicate, treatment, RIN, LibraryPrepBatch, LibSize)) #subset sample info for technical/biological variables

#Calculate correlations
pc_cov_cor &lt;- matrix(nrow = ncol(info), ncol = length(p_comps), 
                      dimnames = list(colnames(info), colnames(pca_genes$x)[p_comps])) 
for (pc in p_comps) { 
  for (covariate in 1:ncol(info)) { 
        lm_result &lt;- lm(pca_genes$x[, pc] ~ info[, covariate]) 
    r2 &lt;- summary(lm_result)$r.squared 
    pc_cov_cor[covariate, pc] &lt;- r2 
  } 
} 
datatable(pc_cov_cor) </code></pre>
<div id="htmlwidget-0c9a476e309710ae11e9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0c9a476e309710ae11e9">{"x":{"filter":"none","data":[["Individual","Sex","Replicate","treatment","RIN","LibraryPrepBatch","LibSize"],[0.937228077639086,0.0128997480184668,0.0058540386525592,0.0131477134223356,0.379545181902797,0.00163891093223698,0.342995060146532],[0.76507519687444,0.72314930628686,0.00282821963541056,0.14254767046418,0.141599583602183,0.000466214578389926,0.0202194451692656],[0.121357303753474,0.113417972850562,0.298593975451342,0.133677231500241,0.00284173534282695,0.0843207067025867,0.0556430900947727],[0.0949190484431928,0.0848709094819573,0.0836910330415088,0.31151383396958,0.0252154836152721,0.00152507083815934,0.00248155794863571],[0.0519940357997539,0.0514511583390726,0.169158613857536,2.18350931994213e-05,0.00138233077074414,0.165035084411512,0.299547291113072],[0.00628575388746607,0.00246335789934117,0.0416949054631132,0.288018144066067,0.0563919132547233,0.0530300238400473,0.0154618430618555]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>PC1<\/th>\n      <th>PC2<\/th>\n      <th>PC3<\/th>\n      <th>PC4<\/th>\n      <th>PC5<\/th>\n      <th>PC6<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>#Convert to long format to plot in ggplot2
pc_cov_cor_2 &lt;- as.data.frame(pc_cov_cor) 
pc_cov_cor_2$variable &lt;- rownames(pc_cov_cor) 
pc_cov_cor_2 &lt;- gather(pc_cov_cor_2, key = &quot;pc&quot;, value = &quot;cor&quot;, -variable) 
head(pc_cov_cor_2) </code></pre>
<pre><code>          variable  pc         cor
1       Individual PC1 0.937228078
2              Sex PC1 0.012899748
3        Replicate PC1 0.005854039
4        treatment PC1 0.013147713
5              RIN PC1 0.379545182
6 LibraryPrepBatch PC1 0.001638911</code></pre>
<pre class="r"><code>#Plot heatmap
d_heatmap &lt;- pc_cov_cor_2 
d_heatmap$variable &lt;- factor(d_heatmap$variable, 
                              levels = c(&quot;Individual&quot;, &quot;Sex&quot;, &quot;Replicate&quot;, 
                                         &quot;treatment&quot;, &quot;RIN&quot;, &quot;LibraryPrepBatch&quot;, &quot;LibSize&quot;), 
                              labels = c(&quot;Individual&quot;, &quot;Sex&quot;, &quot;Replicate&quot;, 
                                         &quot;treatment&quot;, &quot;RIN&quot;, &quot;LibraryPrepBatch&quot;, &quot;LibSize&quot;)) 
pca_heat &lt;- ggplot(d_heatmap, aes(x = pc, y = variable)) + 
  geom_tile(aes(fill = cor), colour = &quot;white&quot;) + 
  scale_fill_gradient(low = &quot;white&quot;, high = &quot;red&quot;, limits = c(0, 1)) + 
  labs(x = &quot;Principal Component&quot;, y = &quot;&quot;, 
       title = &quot;Correlation between principal components and experimental variables&quot;) 
pca_heat</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/corr%20PC-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-PC-2">
Past versions of “corr PC-2.png”
</button>
</p>
<div id="fig-no-spaces-corr-PC-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/corr PC-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="remove-unwanted-variation-using-ruvseq" class="section level1">
<h1>Remove unwanted variation using RUVSeq</h1>
<pre class="r"><code>#The RUVSeq vignette loads raw counts and uses the RUVSeq package to filter and normalize data (upper quantile normalization) before performing RUVs. 

#Use RUVs (replicates) and adjust k until corr heatmap looks right
replicates &lt;- makeGroups(paste0(sampleinfo$Individual, sampleinfo$treatment))
x &lt;- paste0(sampleinfo$Individual, sampleinfo$treatment)

#load data into expressionset
set &lt;- newSeqExpressionSet(as.matrix(filt_counts$counts),
                           phenoData = data.frame(sampleinfo, row.names=colnames(filt_norm_counts)))
set</code></pre>
<pre><code>SeqExpressionSet (storageMode: lockedEnvironment)
assayData: 10447 features, 17 samples 
  element names: counts, normalizedCounts, offset 
protocolData: none
phenoData
  sampleNames: 18855_3_S 19160_3_S ... 18855_3_U (17 total)
  varLabels: Sample_ID Individual ... LibSize (8 total)
  varMetadata: labelDescription
featureData: none
experimentData: use &#39;experimentData(object)&#39;
Annotation:  </code></pre>
<pre class="r"><code>#normalization
set &lt;- betweenLaneNormalization(x = set, which = &quot;upper&quot;, round = T)

#Spearman
cors &lt;- cor(set@assayData$normalizedCounts, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-1">
Past versions of RUV-1.png
</button>
</p>
<div id="fig-RUV-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/RUV-1.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Pearson
cors &lt;- cor(set@assayData$normalizedCounts, method=&quot;pearson&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-2">
Past versions of RUV-2.png
</button>
</p>
<div id="fig-RUV-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/RUV-2.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># 
set1 &lt;- RUVs(x=set, cIdx = rownames(filt_counts), k=2, scIdx = replicates, round = F)
pData(set1)</code></pre>
<pre><code>          Sample_ID Individual Sex Replicate treatment  RIN
18855_3_S 18855_3_S   NA18855    F         3    Strain 10.0
19160_3_S 19160_3_S   NA19160    M         3    Strain 10.0
18856_3_U 18856_3_U   NA18856    M         3  Unstrain 10.0
18856_1_U 18856_1_U   NA18856    M         1  Unstrain  9.9
18855_2_S 18855_2_S   NA18855    F         2    Strain 10.0
18856_2_S 18856_2_S   NA18856    M         2    Strain  9.6
19160_3_U 19160_3_U   NA19160    M         3  Unstrain 10.0
18855_2_U 18855_2_U   NA18855    F         2  Unstrain 10.0
19160_2_S 19160_2_S   NA19160    M         2    Strain 10.0
18855_1_S 18855_1_S   NA18855    F         1    Strain  9.9
18856_1_S 18856_1_S   NA18856    M         1    Strain  8.5
19160_1_S 19160_1_S   NA19160    M         1    Strain  9.9
19160_2_U 19160_2_U   NA19160    M         2  Unstrain 10.0
18855_1_U 18855_1_U   NA18855    F         1  Unstrain 10.0
18856_3_S 18856_3_S   NA18856    M         3    Strain  9.6
18856_2_U 18856_2_U   NA18856    M         2  Unstrain  9.5
18855_3_U 18855_3_U   NA18855    F         3  Unstrain 10.0
          LibraryPrepBatch  LibSize       W_1        W_2
18855_3_S                2 15419569 0.6403178 -1.0334196
19160_3_S                2 15177653 0.4887528 -1.3355831
18856_3_U                2 13722909 0.9502388 -0.7892404
18856_1_U                1 18647599 1.2228123 -1.4496449
18855_2_S                1 18256163 0.8233400 -0.6748150
18856_2_S                1 16782764 0.0515988 -1.2784836
19160_3_U                2 19839604 0.3406102 -1.4859733
18855_2_U                1 18618691 0.7609177 -0.8596946
19160_2_S                1 16622407 0.4247172 -1.4414851
18855_1_S                1 19712744 0.7356719 -1.4499412
18856_1_S                1 13571439 1.1409773 -1.1686123
19160_1_S                1 20979958 0.5909682 -1.9538068
19160_2_U                1 20924281 0.1295380 -1.8537818
18855_1_U                1 20146697 0.7569979 -1.3228202
18856_3_S                2 15807863 0.8205467 -0.8981659
18856_2_U                1 15142152 0.4572116 -1.0041098
18855_3_U                2 18157918 0.5909283 -1.1455295</code></pre>
<pre class="r"><code>#Plots after correcting RUV
plotRLE(set1, outline=FALSE, ylim=c(-4, 4), col=colors[x])</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-3">
Past versions of RUV-3.png
</button>
</p>
<div id="fig-RUV-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/5eb29e88b6057837b2c5c0719c225364eb484690/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">5eb29e8</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/f1425df1cce9caed0803bd9f25483dc1795ec89e/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">f1425df</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/bd0e9ca9fce8fcacec0258136d57c630ecf50508/docs/figure/technical_factors_analysis.Rmd/RUV-3.png" target="_blank">bd0e9ca</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotPCA(set1, col=colors[x], cex=1.2)</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-4">
Past versions of RUV-4.png
</button>
</p>
<div id="fig-RUV-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-4.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/RUV-4.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-4.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/6c3b7d1459fe987e580b53ce89a1581434ee1041/docs/figure/technical_factors_analysis.Rmd/RUV-4.png" target="_blank">6c3b7d1</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#Spearman
cors &lt;- cor(set1@assayData$normalizedCounts, method=&quot;spearman&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-5">
Past versions of RUV-5.png
</button>
</p>
<div id="fig-RUV-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-5.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/RUV-5.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-5.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#extract correlations between technical replicates
coords &lt;- cbind(rbind(c(1,5),c(1,10), c(2,9), c(2,12), c(3,4), c(3,16), c(4,16), c(5,10), c(6,11), c(6,15), c(7,13), c(8,14), c(8,17), c(9,12), c(11,15), c(14,17)))
colnames(coords) &lt;- c(&quot;index1&quot;,&quot;index2&quot;)
coords</code></pre>
<pre><code>      index1 index2
 [1,]      1      5
 [2,]      1     10
 [3,]      2      9
 [4,]      2     12
 [5,]      3      4
 [6,]      3     16
 [7,]      4     16
 [8,]      5     10
 [9,]      6     11
[10,]      6     15
[11,]      7     13
[12,]      8     14
[13,]      8     17
[14,]      9     12
[15,]     11     15
[16,]     14     17</code></pre>
<pre class="r"><code>cors_technical &lt;- cors[coords]
median(cors_technical)</code></pre>
<pre><code>[1] 0.9941572</code></pre>
<pre class="r"><code>min(cors_technical)</code></pre>
<pre><code>[1] 0.9908676</code></pre>
<pre class="r"><code>#Pearson
cors &lt;- cor(set1@assayData$normalizedCounts, method=&quot;pearson&quot;, use=&quot;pairwise.complete.obs&quot;)

labels &lt;- paste(sampleinfo$Individual, sampleinfo$treatment, sep=&quot; &quot;)
heatmap.2( cors, scale=&quot;none&quot;, col = colors, margins = c(12, 12), trace=&#39;none&#39;, denscol=&quot;white&quot;, labCol=labels, ColSideColors=pal[as.integer(as.factor(sampleinfo$Individual))], RowSideColors=pal[as.integer(as.factor(sampleinfo$treatment))+3], cexCol = 0.2 + 1/log10(15), cexRow = 0.2 + 1/log10(15))</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/RUV-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-RUV-6">
Past versions of RUV-6.png
</button>
</p>
<div id="fig-RUV-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/RUV-6.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/RUV-6.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/RUV-6.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>“The RUVg [RUVs] function returns two pieces of information: the estimated factors of unwanted variation (added as columns to the phenoData slot of set) and the normalized counts obtained by regressing the original counts on the unwanted factors. The normalized values are stored in the normalizedCounts slot of set and can be accessed with the normCounts method. These counts should be used only for exploration. It is important that subsequent DE analysis be done on the original counts (accessible through the counts method), as removing the unwanted factors from the counts can also remove part of a factor of interest [6].”</p>
</div>
<div id="check-that-technical-factors-do-not-correlate-with-pcs-after-ruvs" class="section level1">
<h1>Check that technical factors do not correlate with PCs after RUVs</h1>
<pre class="r"><code># Perform PCA
pca_genes &lt;- prcomp(t(set1@assayData$normalizedCounts), scale = T)
scores &lt;- pca_genes$x

variances &lt;- pca_genes$sdev^2
explained &lt;- variances / sum(variances)
plot(pca_genes, main = &quot;Variance per PC&quot;)</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/corr%20PC%20after%20RUVs-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-PC-after-RUVs-1">
Past versions of “corr PC after RUVs-1.png”
</button>
</p>
<div id="fig-no-spaces-corr-PC-after-RUVs-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-1.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-1.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-1.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Calculate the relationship between each recorded covariate and the top 6 PCs. 
p_comps &lt;- 1:6
info &lt;- sampleinfo %&gt;% 
     dplyr::select(c(Individual, Sex, Replicate, treatment, RIN, LibraryPrepBatch, LibSize)) #subset sample info for technical/biological variables

#Calculate correlations
pc_cov_cor &lt;- matrix(nrow = ncol(info), ncol = length(p_comps), 
                      dimnames = list(colnames(info), colnames(pca_genes$x)[p_comps])) 
for (pc in p_comps) { 
  for (covariate in 1:ncol(info)) { 
        lm_result &lt;- lm(pca_genes$x[, pc] ~ info[, covariate]) 
    r2 &lt;- summary(lm_result)$r.squared 
    pc_cov_cor[covariate, pc] &lt;- r2 
  } 
} 
datatable(pc_cov_cor) </code></pre>
<div id="htmlwidget-69e05b0782e67294644f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-69e05b0782e67294644f">{"x":{"filter":"none","data":[["Individual","Sex","Replicate","treatment","RIN","LibraryPrepBatch","LibSize"],[0.942423350263046,0.00668853490335315,0.013138424505935,0.00991989728634239,0.36015243807383,0.00652946944654191,0.287523347602406],[0.769917707954356,0.737388446965363,0.00343860005650645,0.152831524598534,0.160631002359224,0.00170955367296791,0.0353045119613469],[0.249299897087026,0.232409132934962,0.00117716205454386,0.426508309999628,0.0112572091229938,0.0154782383649311,2.69693770910216e-05],[0.0228456288560085,0.0182119551249912,0.00225753035572013,0.0155684514488493,0.00125319628274225,0.0132824492887433,0.114059598899647],[0.00105169957782929,1.53715013657476e-05,0.0374723286708525,0.328271897748881,0.0493089596231799,0.051752050913322,0.0531747215441057],[0.00282652703469331,0.00050644763439204,0.0416013868698152,0.0426959451070654,0.147399837512511,0.116786929159432,0.0645922262080093]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>PC1<\/th>\n      <th>PC2<\/th>\n      <th>PC3<\/th>\n      <th>PC4<\/th>\n      <th>PC5<\/th>\n      <th>PC6<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>#Convert to long format to plot in ggplot2
pc_cov_cor_2 &lt;- as.data.frame(pc_cov_cor) 
pc_cov_cor_2$variable &lt;- rownames(pc_cov_cor) 
pc_cov_cor_2 &lt;- gather(pc_cov_cor_2, key = &quot;pc&quot;, value = &quot;cor&quot;, -variable) 
head(pc_cov_cor_2) </code></pre>
<pre><code>          variable  pc         cor
1       Individual PC1 0.942423350
2              Sex PC1 0.006688535
3        Replicate PC1 0.013138425
4        treatment PC1 0.009919897
5              RIN PC1 0.360152438
6 LibraryPrepBatch PC1 0.006529469</code></pre>
<pre class="r"><code>#Plot heatmap
d_heatmap &lt;- pc_cov_cor_2 
d_heatmap$variable &lt;- factor(d_heatmap$variable, 
                              levels = c(&quot;Individual&quot;, &quot;Sex&quot;, &quot;Replicate&quot;, 
                                         &quot;treatment&quot;, &quot;RIN&quot;, &quot;LibraryPrepBatch&quot;, &quot;LibSize&quot;), 
                              labels = c(&quot;Individual&quot;, &quot;Sex&quot;, &quot;Replicate&quot;, 
                                         &quot;treatment&quot;, &quot;RIN&quot;, &quot;LibraryPrepBatch&quot;, &quot;LibSize&quot;)) 
pca_heat &lt;- ggplot(d_heatmap, aes(x = pc, y = variable)) + 
  geom_tile(aes(fill = cor), colour = &quot;white&quot;) + 
  scale_fill_gradient(low = &quot;white&quot;, high = &quot;red&quot;, limits = c(0, 1)) + 
  labs(x = &quot;Principal Component&quot;, y = &quot;&quot;, 
       title = &quot;Correlation between principal components and experimental variables&quot;) 
pca_heat</code></pre>
<p><img src="figure/technical_factors_analysis.Rmd/corr%20PC%20after%20RUVs-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-PC-after-RUVs-3">
Past versions of “corr PC after RUVs-3.png”
</button>
</p>
<div id="fig-no-spaces-corr-PC-after-RUVs-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/cb6a0a50529469a1c598d448a06e6ee146719d27/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-3.png" target="_blank">cb6a0a5</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/629408f29313ac35a6601656ac80732937de98e1/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-3.png" target="_blank">629408f</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/anthonyhung/OAStrain/blob/158fdf3e906a4a488e486cb96684b4e23b567895/docs/figure/technical_factors_analysis.Rmd/corr PC after RUVs-3.png" target="_blank">158fdf3</a>
</td>
<td>
Anthony Hung
</td>
<td>
2020-01-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="save-data-and-ruvs-output-for-unwanted-variation-contains-the-same-cpm-data-as-before-as-well-as-pheno-data-w-values-from-ruvs" class="section level1">
<h1>Save data and RUVs output for unwanted variation (contains the same cpm data as before as well as pheno data, W values from RUVs)</h1>
<pre class="r"><code>saveRDS(set1, &quot;data/RUVsOut.rds&quot;)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] dplyr_0.8.5                 RUVSeq_1.18.0              
 [3] EDASeq_2.18.0               ShortRead_1.42.0           
 [5] GenomicAlignments_1.20.1    SummarizedExperiment_1.14.1
 [7] DelayedArray_0.10.0         matrixStats_0.55.0         
 [9] Rsamtools_2.0.0             GenomicRanges_1.36.1       
[11] GenomeInfoDb_1.20.0         Biostrings_2.52.0          
[13] XVector_0.24.0              IRanges_2.18.3             
[15] S4Vectors_0.22.1            BiocParallel_1.18.1        
[17] Biobase_2.44.0              BiocGenerics_0.30.0        
[19] tidyr_1.0.3                 DT_0.13                    
[21] cowplot_1.0.0               scales_1.1.1               
[23] RColorBrewer_1.1-2          edgeR_3.26.8               
[25] limma_3.40.6                reshape_0.8.8              
[27] ggplot2_3.3.0               gplots_3.0.1.1             

loaded via a namespace (and not attached):
 [1] colorspace_1.4-1       hwriter_1.3.2          ellipsis_0.3.1        
 [4] rprojroot_1.3-2        fs_1.3.1               farver_2.0.3          
 [7] bit64_0.9-7            AnnotationDbi_1.46.0   splines_3.6.1         
[10] R.methodsS3_1.7.1      DESeq_1.36.0           geneplotter_1.62.0    
[13] knitr_1.23             jsonlite_1.6.1         workflowr_1.6.2       
[16] annotate_1.62.0        R.oo_1.23.0            compiler_3.6.1        
[19] httr_1.4.1             backports_1.1.7        assertthat_0.2.1      
[22] Matrix_1.2-18          later_1.0.0            htmltools_0.4.0       
[25] prettyunits_1.1.1      tools_3.6.1            gtable_0.3.0          
[28] glue_1.4.1             GenomeInfoDbData_1.2.1 Rcpp_1.0.4.6          
[31] vctrs_0.3.0            gdata_2.18.0           rtracklayer_1.44.0    
[34] crosstalk_1.1.0.1      xfun_0.8               stringr_1.4.0         
[37] lifecycle_0.2.0        gtools_3.8.1           XML_3.98-1.20         
[40] zlibbioc_1.30.0        MASS_7.3-51.6          aroma.light_3.14.0    
[43] hms_0.5.3              promises_1.1.0         yaml_2.2.1            
[46] memoise_1.1.0          biomaRt_2.40.1         latticeExtra_0.6-28   
[49] stringi_1.4.6          RSQLite_2.1.1          genefilter_1.66.0     
[52] GenomicFeatures_1.36.3 caTools_1.17.1.2       rlang_0.4.6           
[55] pkgconfig_2.0.3        bitops_1.0-6           evaluate_0.14         
[58] lattice_0.20-41        purrr_0.3.4            htmlwidgets_1.5.1     
[61] labeling_0.3           bit_1.1-14             tidyselect_1.1.0      
[64] plyr_1.8.6             magrittr_1.5           R6_2.4.1              
[67] DBI_1.0.0              pillar_1.4.4           whisker_0.3-2         
[70] withr_2.2.0            survival_2.44-1.1      RCurl_1.98-1.1        
[73] tibble_3.0.1           crayon_1.3.4           KernSmooth_2.23-15    
[76] rmarkdown_1.13         progress_1.2.2         locfit_1.5-9.1        
[79] grid_3.6.1             blob_1.2.0             git2r_0.26.1          
[82] digest_0.6.25          xtable_1.8-4           httpuv_1.5.1          
[85] R.utils_2.9.2          munsell_0.5.0         </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
