---
title: "ANT1/ANT2_Pilot"
author: "Anthony Hung"
date: "2019-12-02"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Analysis of processed sc pilot data ANT1/ANT2

ANT1 contains multiplexed samples from replicate of the pilot. Equal numbers of cells from 18855 (OA strained), 19160 (unstrained), and 18856 (unstrained) were pooled into one collection and sequenced using the 10x chromium technology.

Raw FASTQ files were processed through Kenneth Barr's pipeline, which aligns reads to hg38 using STAR solo and runs demuxlet on the output. 

## Load libraries

```{r}
library(data.table)
library(tidyverse)
library(dplyr)
library(stringr)
library(readr)
library(Matrix)
library(DropletUtils)
```

## Load in data

```{r}
## read in the output from demuxlet and the count matrices
proj_dir <- "/project2/gilad/anthonyhung/Projects/OAStrain_project/"
ANT1_dir <- paste0("YriMultiplex/YG-AH-2S-ANT-1_S1_L008/Gene/raw/")
ANT1_dir <- paste0("YriMultiplex/YG-AH-2S-ANT-2_S2_L008/Gene/raw/")

#read in data



#ANT1
demuxlet1 <- fread(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-1_S1_L008/demuxlet.best", sep = ""))
count_data1 <- readMM(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-1_S1_L008/Gene/raw/matrix.mtx"))
genes1 <- read_tsv(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-1_S1_L008/Gene/raw/genes.tsv"), col_names = F)
barcodes1 <- as.data.frame(read_tsv(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-1_S1_L008/Gene/raw/barcodes.tsv"), col_names = F))
#ANT2
demuxlet2 <- fread(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-2_S2_L008/demuxlet.best", sep = ""))
count_data2 <- readMM(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-2_S2_L008/Gene/raw/matrix.mtx"))
genes2 <- read_tsv(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-2_S2_L008/Gene/raw/genes.tsv"), col_names = F)
barcodes2 <- as.data.frame(read_tsv(paste0(proj_dir, "YriMultiplex/YG-AH-2S-ANT-2_S2_L008/Gene/raw/barcodes.tsv"), col_names = F))

## filter for droplets that are singlets
#ANT1
demuxlet_single1 <- demuxlet1 %>% 
     dplyr::filter(grepl("SNG-", BEST))
singlets_index1 <- unlist(lapply(barcodes1,"%in%", table = demuxlet_single1$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets1 <- barcodes1[singlets_index1,] #use index to subset matrix + barcode names
count_data_singlets1 <- count_data1[,singlets_index1]

#ANT2
demuxlet_single2 <- demuxlet2 %>% 
     dplyr::filter(grepl("SNG-", BEST))
singlets_index2 <- unlist(lapply(barcodes2,"%in%", table = demuxlet_single2$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets2 <- barcodes2[singlets_index2,] #use index to subset matrix + barcode names
count_data_singlets2 <- count_data2[,singlets_index2]
```
