---
title: "ANT1/ANT2_Pilot"
author: "Anthony Hung"
date: "2019-12-02"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Analysis of processed sc pilot data ANT1/ANT2

ANT1_ANT2 contain multiplexed samples from replicate of the pilot. Equal numbers of cells from 18855 (OA strained), 19160 (unstrained), and 18856 (unstrained) were pooled into one collection and sequenced using the 10x chromium technology.

ANT1: (19160 unstrained; 18856 unstrained; 18855 strained)

ANT2: (19160 strained; 18856 strained; 18855 unstrained)

Raw FASTQ files were processed through Kenneth Barr's pipeline, which aligns reads to hg38 using STAR solo and runs demuxlet on the output.

## Load libraries

```{r load libraries}
library(data.table)
library(tidyverse)
library(dplyr)
library(stringr)
library(readr)
library(Matrix)
library(DropletUtils)
library(biomaRt)
library(scater)
```

## Load in data

```{r specify directory structure}
## link to directories containing data files (count matrices)
proj_dir <- "~/Desktop/scPilot/"
ANT1_dir <- paste0(proj_dir, "YG-AH-2S-ANT-1_S1_L008/")
ANT2_dir <- paste0(proj_dir, "YG-AH-2S-ANT-2_S2_L008/")

# #read in data
# 
# #ANT1
# demuxlet1 <- fread(paste0(ANT1_dir, "demuxlet.best", sep = ""))
# count_data1 <- readMM(paste0(ANT1_dir, "Gene/raw/matrix.mtx"))
# genes1 <- read_tsv(paste0(ANT1_dir, "Gene/raw/genes.tsv"), col_names = F)
# barcodes1 <- as.data.frame(read_tsv(paste0(ANT1_dir, "Gene/raw/barcodes.tsv"), col_names = F))
# #ANT2
# demuxlet2 <- fread(paste0(ANT2_dir, "demuxlet.best", sep = ""))
# count_data2 <- readMM(paste0(ANT2_dir, "Gene/raw/matrix.mtx"))
# genes2 <- read_tsv(paste0(ANT2_dir, "Gene/raw/genes.tsv"), col_names = F)
# barcodes2 <- as.data.frame(read_tsv(paste0(ANT2_dir, "Gene/raw/barcodes.tsv"), col_names = F))
# 
# ######################################3
# 
# ## filter for droplets that are singlets
# #ANT1
# demuxlet_single1 <- demuxlet1 %>% 
#      dplyr::filter(grepl("SNG-", BEST))
# singlets_index1 <- unlist(lapply(barcodes1,"%in%", table = demuxlet_single1$BARCODE), use.names = F) #get index of barcodes that are singlets
# barcodes_singlets1 <- barcodes1[singlets_index1,] #use index to subset matrix + barcode names
# count_data_singlets1 <- count_data1[,singlets_index1]
# 
# #ANT2
# demuxlet_single2 <- demuxlet2 %>% 
#      dplyr::filter(grepl("SNG-", BEST))
# singlets_index2 <- unlist(lapply(barcodes2,"%in%", table = demuxlet_single2$BARCODE), use.names = F) #get index of barcodes that are singlets
# barcodes_singlets2 <- barcodes2[singlets_index2,] #use index to subset matrix + barcode names
# count_data_singlets2 <- count_data2[,singlets_index2]
```


# Run emptyDrops to infer empty droplets and filter them

## Load in data

```{r load data}
ANT1 <- read10xCounts(paste0(ANT1_dir, "Gene/raw/"), col.names=F)
ANT2 <- read10xCounts(paste0(ANT2_dir, "Gene/raw/"), col.names=F)

```

## Assign gene annotations

```{r gene annotate}
#obtain gene annotations using biomaRt
ensembl <- useMart("ensembl",dataset="hsapiens_gene_ensembl", host = "www.ensembl.org", ensemblRedirect = FALSE)
attr.string <- c('ensembl_gene_id', 'hgnc_symbol', 'chromosome_name', 'start_position', 'end_position', 'strand', 'description', 'percentage_gene_gc_content', 'gene_biotype')

     #function to annotate genes using biomaRt
AnnotateGenes <- function(obj_10xCounts){
     return(getBM(attributes=attr.string, 
                          filters =  'ensembl_gene_id', 
                          values = rowData(obj_10xCounts)$ID, 
                          mart = ensembl))
}

# ANT1/2 Annotate genes
ANT1_gene.annotation <- AnnotateGenes(ANT1)
ANT2_gene.annotation <- AnnotateGenes(ANT2)
dim(ANT1_gene.annotation)
dim(ANT2_gene.annotation)



#collapse duplicate annotations
collapse_duplicate_anno <- function(gene.annotation){
  t1 <- table(gene.annotation$ensembl_gene_id)
  t2 <- t1[t1 > 1]
  t2 
  
  gene.annotation <- distinct(gene.annotation, ensembl_gene_id, 
                           .keep_all = TRUE)
}

ANT1_gene.annotation <- collapse_duplicate_anno(ANT1_gene.annotation)
ANT2_gene.annotation <- collapse_duplicate_anno(ANT2_gene.annotation)

dim(ANT1_gene.annotation)
dim(ANT2_gene.annotation)



# add gene.annotations to 10xCounts object
add_annotations <- function(obj_10xCounts, gene.annotation){
  w2kp <- match(gene.annotation$ensembl_gene_id, rowData(obj_10xCounts)$ID)
  obj_10xCounts <- obj_10xCounts[w2kp,]
  table(gene.annotation$ensembl_gene_id == rowData(obj_10xCounts)$ID)
  print(table(gene.annotation$ensembl_gene_id == rowData(obj_10xCounts)$ID)) #check that the entries in the annotation match the rowData in our count object (prints out when you run this UDF)
  rowData(obj_10xCounts)  <- gene.annotation
  rownames(obj_10xCounts) <- uniquifyFeatureNames(rowData(obj_10xCounts)$ensembl_gene_id, 
                                     rowData(obj_10xCounts)$hgnc_symbol)
  return(obj_10xCounts)
}

ANT1 <- add_annotations(ANT1, ANT1_gene.annotation)
ANT2 <- add_annotations(ANT2, ANT2_gene.annotation)
dim(ANT1)
dim(ANT2)
```

## Identify low quality cells using emptyDrops

```{r plot barcode ranks}
# calculate barcode ranks
bcrank_ANT1 <- barcodeRanks(counts(ANT1))
bcrank_ANT2 <- barcodeRanks(counts(ANT2))

plot_cells <- function(bcrank){
  # Only showing unique points for plotting speed.
  uniq <- !duplicated(bcrank$rank)
  print(metadata(bcrank)$inflection) #inflection
  print(metadata(bcrank)$knee) #knee
  
  par(mar=c(5,4,2,1), bty="n")
  plot(bcrank$rank[uniq], bcrank$total[uniq], log="xy", 
     xlab="Rank", ylab="Total UMI count", cex=0.5, cex.lab=1.2)
  abline(h=metadata(bcrank)$inflection, col="darkgreen", lty=2)
  abline(h=metadata(bcrank)$knee, col="dodgerblue", lty=2)
  
  legend("left", legend=c("Inflection", "Knee"), bty="n", 
       col=c("darkgreen", "dodgerblue"), lty=2, cex=1.2)
}

rank_data <- function(bcrank){
  print("inflection")
  print(metadata(bcrank)$inflection)
  print("knee")
  print(metadata(bcrank)$knee)
  print("summary of total UMI counts")
  print(summary(bcrank$total))
  print("UMIs above knee")
  print(table(bcrank$total >= bcrank$knee))
  print("UMIs above inflection")
  print(table(bcrank$total >= bcrank$inflection))
}


plot_cells(bcrank_ANT1) + title("ANT1")
rank_data(bcrank_ANT1)
plot_cells(bcrank_ANT2) + title("ANT2")
rank_data(bcrank_ANT2)
```

## Run emptyDrops

```{r emptydrops}
emptyDrops_out_ANT1 <- emptyDrops(counts(ANT1))
emptyDrops_out_ANT1

emptyDrops_out_ANT2 <- emptyDrops(counts(ANT2))
emptyDrops_out_ANT2

#function to plot emptyDrops output
plot_e.drops <- function(bcrank, e.out){
  par(mar=c(5,4,1,1), mfrow=c(1,2), bty="n")
  plot(e.out$Total, -e.out$LogProb, col=ifelse(is.cell, "red", "black"),
    xlab="Total UMI count", ylab="-Log Probability", cex=0.2)
  abline(v = metadata(bcrank)$inflection, col="darkgreen")
  abline(v = metadata(bcrank)$knee, col="dodgerblue")
  legend("bottomright", legend=c("Inflection", "Knee"), bty="n", 
       col=c("darkgreen", "dodgerblue"), lty=1, cex=1.2)

  plot(e.out$Total, -e.out$LogProb, col=ifelse(is.cell, "red", "black"),
    xlab="Total UMI count", ylab="-Log Probability", cex=0.2, xlim=c(0,2000), ylim=c(0,2000))
  abline(v = metadata(bcrank)$inflection, col="darkgreen")
  abline(v = metadata(bcrank)$knee, col="dodgerblue")
}

plot_e.drops(bcrank_ANT1, emptyDrops_out_ANT1)
plot_e.drops(bcrank_ANT2, emptyDrops_out_ANT2)
```

Now, we filter barcodes based on the emptyDrops output. 

```{r filter barcodes}

```


