---
title: "ANT1/ANT2_Pilot"
author: "Anthony Hung"
date: "2019-12-02"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Analysis of processed sc pilot data ANT1/ANT2

ANT1_ANT2 contain multiplexed samples from replicate of the pilot. Equal numbers of cells from 18855 (OA strained), 19160 (unstrained), and 18856 (unstrained) were pooled into one collection for ANT1, and equal numbers of cells from 18855 (Unstrained), 19160 (OA strained) were pooled into one collection for ANT2. The collections were sequenced using the 10x chromium technology.

ANT1: (19160 unstrained; 18856 unstrained; 18855 strained)
  estimated saturation statistics from Kenneth's Pipeline:
n.reads n.unique        total.size      saturation
192546655       101667280       2.15403e+08     0.471986

ANT2: (19160 strained; 18856 strained; 18855 unstrained)
  estimated saturation statistics from Kenneth's Pipeline:
n.reads n.unique        total.size      saturation
180694884       120674466       3.63297e+08     0.332164
  
Raw FASTQ files were processed through Kenneth Barr's pipeline, which aligns reads to hg38 using STAR solo and runs demuxlet on the output.

## Load libraries

```{r load libraries}
library(data.table)
library(tidyverse)
library(plyr)
library(dplyr)
library(stringr)
library(readr)
library(Matrix)
library(Seurat)
library(gridExtra)
library(heplots)
library(DT)
library(gplots)
library(cowplot)
library("ggpubr")

```

## Load in data

```{r specify directory structure}
## link to directories containing data files (count matrices)
proj_dir <- "/project2/gilad/anthonyhung/Projects/OAStrain_project/YriMultiplex/"
ANT1_dir <- paste0(proj_dir, "YG-AH-2S-ANT-1_S1_L008/")
ANT2_dir <- paste0(proj_dir, "YG-AH-2S-ANT-2_S2_L008/")

#read in data

# ##Gene output from STARSOLO
# #ANT1
# demuxlet1 <- fread(paste0(ANT1_dir, "demuxlet.best", sep = ""))
# count_data1 <- readMM(paste0(ANT1_dir, "Gene/filtered/matrix.mtx"))
# genes1 <- read_tsv(paste0(ANT1_dir, "Gene/filtered/genes.tsv"), col_names = F)
# barcodes1 <- as.data.frame(read_tsv(paste0(ANT1_dir, "Gene/filtered/barcodes.tsv"), col_names = F))
# #ANT2
# demuxlet2 <- fread(paste0(ANT2_dir, "demuxlet.best", sep = ""))
# count_data2 <- readMM(paste0(ANT2_dir, "Gene/filtered/matrix.mtx"))
# genes2 <- read_tsv(paste0(ANT2_dir, "Gene/filtered/genes.tsv"), col_names = F)
# barcodes2 <- as.data.frame(read_tsv(paste0(ANT2_dir, "Gene/filtered/barcodes.tsv"), col_names = F))


##Gene Output from STARSOLO
#ANT1
demuxlet1 <- fread(paste0(ANT1_dir, "demuxlet.best", sep = ""))
count_data1 <- readMM(paste0(ANT1_dir, "Gene/filtered/matrix.mtx"))
genes1 <- read_tsv(paste0(ANT1_dir, "Gene/filtered/genes.tsv"), col_names = F)
barcodes1 <- as.data.frame(read_tsv(paste0(ANT1_dir, "Gene/filtered/barcodes.tsv"), col_names = F))
UMIperCell1 <- read.table(paste0(ANT1_dir, "Gene/UMIperCellSorted.txt"), header = F)
#ANT2
demuxlet2 <- fread(paste0(ANT2_dir, "demuxlet.best", sep = ""))
count_data2 <- readMM(paste0(ANT2_dir, "Gene/filtered/matrix.mtx"))
genes2 <- read_tsv(paste0(ANT2_dir, "Gene/filtered/genes.tsv"), col_names = F)
barcodes2 <- as.data.frame(read_tsv(paste0(ANT2_dir, "Gene/filtered/barcodes.tsv"), col_names = F))
UMIperCell2 <- read.table(paste0(ANT2_dir, "Gene/UMIperCellSorted.txt"), header = F)
```

# number of umis per cell for each sample

```{r hist of umipercell}
table(UMIperCell1$V1 > 10000)
table(UMIperCell2$V1 > 10000)
```


# Based on the demuxlet output, assign label for barcodes based on "BEST" output and filter for "SNG-" barcodes

```{r Filter droplets for singlets}
#returns a dataframe with two columns, one corresponding to the barcodes and one corresponding to the label given by demuxlet
return_singlet_label <- function(barcodes, demuxlet.out){
  labels <- demuxlet.out$BEST[match(unlist(barcodes), demuxlet.out$BARCODE)]
  return(cbind(barcodes, labels))
}

barcodes1_labeled <- return_singlet_label(barcodes1, demuxlet1)
barcodes2_labeled <- return_singlet_label(barcodes2, demuxlet2)

#table of singlets/multiplets in the filtered data based on demuxlet
table(barcodes1_labeled$labels)
table(barcodes2_labeled$labels)

## filter for droplets that are singlets
#ANT1
demuxlet_single1 <- demuxlet1 %>%
     dplyr::filter(grepl("SNG-", BEST))
singlets_index1 <- unlist(lapply(barcodes1_labeled$X1,"%in%", table = demuxlet_single1$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets1 <- barcodes1_labeled[singlets_index1,] #use index to subset matrix + barcode names
count_data_singlets1 <- count_data1[,singlets_index1]

#ANT2
demuxlet_single2 <- demuxlet2 %>%
     dplyr::filter(grepl("SNG-", BEST))
singlets_index2 <- unlist(lapply(barcodes2_labeled$X1,"%in%", table = demuxlet_single2$BARCODE), use.names = F) #get index of barcodes that are singlets
barcodes_singlets2 <- barcodes2_labeled[singlets_index2,] #use index to subset matrix + barcode names
count_data_singlets2 <- count_data2[,singlets_index2]
```



# Create Seurat object for each dataset (for singlet barcodes) and add metadata in the form of singlet identity for each barcode. Rest of analysis adapted from: https://satijalab.org/seurat/v3.0/pbmc3k_tutorial.html

```{r seurat}
#Change labels to reflect strain/unstrain

strainIndlabels1 <- revalue(barcodes_singlets1$labels, 
                            c("SNG-NA18856"= "NA18856_Unstrain", 
                              "SNG-NA18855" = "NA18855_Strain", 
                              "SNG-NA19160" = "NA19160_Unstrain"))

strainIndlabels2 <- revalue(barcodes_singlets2$labels, 
                            c("SNG-NA18855" = "NA18855_Unstrain", 
                              "SNG-NA19160" = "NA19160_Strain"))


rownames(count_data_singlets1) <- genes1$X2
colnames(count_data_singlets1) <- barcodes_singlets1$X1

ANT1_seurat <- CreateSeuratObject(counts = count_data_singlets1, project = "ANT1") %>% 
  AddMetaData(strainIndlabels1, col.name = "labels")


rownames(count_data_singlets2) <- genes2$X2
colnames(count_data_singlets2) <- barcodes_singlets2$X1

ANT2_seurat <- CreateSeuratObject(counts = count_data_singlets2, project = "ANT2") %>% 
  AddMetaData(strainIndlabels2, col.name = "labels")

```

# Data Integration

```{r log_transform_integrate}
# Merge into one dataset (https://satijalab.org/seurat/v3.2/integration.html)

n_dim <- 10
#filter (use same criteria for both), normalize and find variable features
#filter
ANT1_seurat[["percent.mt"]] <- PercentageFeatureSet(ANT1_seurat, pattern = "^MT-")
VlnPlot(ANT1_seurat, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")
ANT1_seurat <- subset(ANT1_seurat, subset = nFeature_RNA > 2000 & percent.mt < 10)
VlnPlot(ANT1_seurat, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")
#normalize
ANT1_seurat <- NormalizeData(ANT1_seurat, normalization.method = "LogNormalize", scale.factor = 10000)
#find var features
ANT1_seurat <- FindVariableFeatures(ANT1_seurat, selection.method = "vst", nfeatures = 5000)

#filter
ANT2_seurat[["percent.mt"]] <- PercentageFeatureSet(ANT2_seurat, pattern = "^MT-")
VlnPlot(ANT2_seurat, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")
ANT2_seurat <- subset(ANT2_seurat, subset = nFeature_RNA > 2000 & percent.mt < 10)
VlnPlot(ANT2_seurat, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")
#normalize
ANT2_seurat <- NormalizeData(ANT2_seurat, normalization.method = "LogNormalize", scale.factor = 10000)
#find var features
ANT2_seurat <- FindVariableFeatures(ANT2_seurat, selection.method = "vst", nfeatures = 5000)

#Integrate using FindIntegrationAnchors
reference.list <- list(ANT1_seurat, ANT2_seurat)
anchors <- FindIntegrationAnchors(object.list = reference.list, dims = 1:n_dim)
integrated <- IntegrateData(anchorset = anchors, dims = 1:n_dim)
```

Visualize integrated data

```{r viz_log_integrate}
DefaultAssay(integrated) <- "integrated"
integrated <- ScaleData(integrated, verbose = FALSE)
integrated <- RunPCA(integrated, npcs = 100, verbose = FALSE)
DimPlot(integrated, reduction = "pca", group.by = "orig.ident")
DimPlot(integrated, reduction = "pca", group.by = "labels")
ElbowPlot(integrated, ndims = 100) #34 PCs?
integrated <- FindNeighbors(integrated, dims = 1:34)
integrated <- FindClusters(integrated, resolution = 0.5)
integrated <- RunUMAP(integrated, reduction = "pca", dims = 1:34)
p1 <- DimPlot(integrated, reduction = "umap", group.by = "orig.ident")
p2 <- DimPlot(integrated, reduction = "umap", group.by = "labels", label = TRUE, 
    repel = TRUE) + NoLegend()
p3 <- DimPlot(integrated, reduction = "umap", group.by = "seurat_clusters")

grid.arrange(p1, p2, p3, nrow = 3)

saveRDS(integrated, "data/log_normalized_ANT12_integrated.rds")
```

```{r viz_log_integrated_markers}
#Chondrocyte markers
VlnPlot(integrated, features = c("COL2A1", "SOX9", "ACAN"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(integrated, features = c("COL10A1", "COL11A1", "SOX5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(integrated, features = c("TGFB3", "SOX6"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")

#Strain markers
VlnPlot(integrated, features = c("MMP1", "MMP13", "IL1B", "MMP3"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(integrated, features = c("MMP2", "COMP", "TNF"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(integrated, features = c("IBSP", "TIMP1",  "MMP9"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(integrated, features = c("ADAMTS4", "ADAMTS5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")

```




# Integrate using SCTransform

```{r integrate_SCT}
seurat.list <- list(ANT1_seurat, ANT2_seurat)
for (i in 1:length(seurat.list)) {
    seurat.list[[i]] <- SCTransform(seurat.list[[i]], verbose = FALSE)
}

SCT.features <- SelectIntegrationFeatures(object.list = seurat.list, nfeatures = 3000)
seurat.list <- PrepSCTIntegration(object.list = seurat.list, anchor.features = SCT.features, 
    verbose = FALSE)

#find anchors
seurat.anchors <- FindIntegrationAnchors(object.list = seurat.list, normalization.method = "SCT", 
    anchor.features = SCT.features, verbose = FALSE)
SCT.integrated <- IntegrateData(anchorset = seurat.anchors, normalization.method = "SCT", 
    verbose = FALSE)
```

# Visualize SCT Transform Integration

```{r visualize_SCT_integration}
SCT.integrated <- RunPCA(SCT.integrated, verbose = FALSE, npcs = 100)
ElbowPlot(SCT.integrated, ndims = 100) #38 PCs?
SCT.integrated <- FindNeighbors(SCT.integrated, dims = 1:38)
SCT.integrated <- FindClusters(SCT.integrated, resolution = 0.5)
SCT.integrated <- RunUMAP(SCT.integrated, dims = 1:38)
p1_SCT <- DimPlot(SCT.integrated, group.by = c("orig.ident"))
p2_SCT <- DimPlot(SCT.integrated, group.by = c("labels"))
p3_SCT <- DimPlot(SCT.integrated, group.by = c("seurat_clusters"))
grid.arrange(p1_SCT, p2_SCT, p3_SCT, nrow = 3)

saveRDS(SCT.integrated, "data/SCT_ANT12_integrated.rds")
```

```{r visualize_SCT_markers}
#Chondrocyte markers
VlnPlot(SCT.integrated, features = c("COL2A1", "SOX9", "ACAN"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(SCT.integrated, features = c("COL10A1", "COL11A1", "SOX5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(SCT.integrated, features = c("TGFB3", "SOX6"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")

#Strain markers
VlnPlot(SCT.integrated, features = c("MMP1", "MMP13", "IL1B", "MMP3"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(SCT.integrated, features = c("MMP2", "COMP", "TNF"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(SCT.integrated, features = c("IBSP", "TIMP1",  "MMP9"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(SCT.integrated, features = c("ADAMTS4", "ADAMTS5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")


```








# No anchoring

Now we can perform some QC on cells in Seurat

```{r QC}
ANT1.2 <- merge(x = ANT1_seurat,
                   y = ANT2_seurat,
                   add.cell.ids = c("ANT1", "ANT2"),
                   merge.data = F,
                   project = "OAStrain")
ANT1.2[["percent.mt"]] <- PercentageFeatureSet(ANT1.2, pattern = "^MT-")

#visualize QC metrics as violin plot
VlnPlot(ANT1.2, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3)
VlnPlot(ANT1.2, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")

# FeatureScatter is typically used to visualize feature-feature relationships, but can be used
# for anything calculated by the object, i.e. columns in object metadata, PC scores etc.

plot1 <- FeatureScatter(ANT1.2, feature1 = "nCount_RNA", feature2 = "percent.mt")
plot2 <- FeatureScatter(ANT1.2, feature1 = "nCount_RNA", feature2 = "nFeature_RNA")
CombinePlots(plots = list(plot1, plot2))




#Filter barcodes based on nFeatures and %MT 
ANT1.2 <- subset(ANT1.2, subset = nFeature_RNA > 2000 & percent.mt < 10)
table(ANT1.2$labels)

#Look at QC metrics after filtering
VlnPlot(ANT1.2, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3, group.by = "labels")
```


# Normalizing Data

```{r Norm}
ANT1.2 <- NormalizeData(ANT1.2, normalization.method = "LogNormalize", scale.factor = 10000)
```


# Dimensionality Reduction

Using the highly variable features selected above, perform PCA

```{r PCA}
ANT1.2 <- FindVariableFeatures(ANT1.2, selection.method = "vst", nfeatures = 2000)
ANT1.2 <- ScaleData(ANT1.2, verbose = FALSE)
nPC <- 100
ANT1.2 <- RunPCA(ANT1.2, features = VariableFeatures(object = ANT1.2), npcs = nPC)


# Examine and visualize PCA results a few different ways
print(ANT1.2[["pca"]], dims = 1:5, nfeatures = 5)
VizDimLoadings(ANT1.2, dims = 1:2, reduction = "pca")

DimPlot(ANT1.2, reduction = "pca", group.by = "labels")
DimPlot(ANT1.2, reduction = "pca", group.by = "orig.ident")

# var explained by top 10 PCs
mat <- Seurat::GetAssayData(ANT1.2, assay = "RNA", slot = "scale.data")
pca <- ANT1.2[["pca"]]

# Get the total variance:
total_variance <- sum(matrixStats::rowVars(mat))

eigValues = (pca@stdev)^2  ## EigenValues
varExplained = eigValues / total_variance

DimHeatmap(ANT1.2, dims = 1:15, cells = 500, balanced = TRUE)






#extract scaled data for highlyvargenes to a matrix outside of seurat to perform PCA and examine varexplained
source("code/PCA_fn.R")
library("RColorBrewer")
library("scales")

# Load colors 
colors <- colorRampPalette(c(brewer.pal(9, "Blues")[1],brewer.pal(9, "Blues")[9]))(100)
pal <- c(brewer.pal(9, "Set1"), brewer.pal(8, "Set2"), brewer.pal(12, "Set3"))

scaled <- ANT1.2@assays$RNA@scale.data[VariableFeatures(ANT1.2),]
dim(scaled)

pca_genes <- prcomp(t(scaled), scale = F)
scores <- pca_genes$x


### PCA Raw Data
for (n in 1:3){
     col.v <- pal[as.integer(factor(ANT1.2@meta.data$labels))]
     plot_scores(pca_genes, scores, n, n+1, cols = col.v, legend = F, points = T)
}

#% Var explained by top 10 pcs
for(n in 1:10){
     print(n)
     print(summary(pca_genes)$importance[2,n]*100)
}


ElbowPlot(ANT1.2, ndims = nPC) #50 PCs?
```

# Correlation between PC loadings and variables

```{r PC corr}
#get PC embeddings for each barcode
PC1_embeddings <- ANT1.2@reductions$pca@cell.embeddings[,1]
PC2_embeddings <- ANT1.2@reductions$pca@cell.embeddings[,2]

#get information about individual, collection, and treatment
barcode_labels <- ANT1.2$labels
individual <- substr(barcode_labels, 1, 7)
collection <- ANT1.2@meta.data$orig.ident
treatment <- substr(barcode_labels, 9, nchar(barcode_labels))

#combine all into a dataframe
PC1_dataframe <- data.frame(individual = individual, sc_collection = collection, treatment = treatment, PC1_embeddings = PC1_embeddings)
PC2_dataframe <- data.frame(individual = individual, sc_collection = collection, treatment = treatment, PC2_embeddings = PC2_embeddings)
info <- data.frame(individual = individual, sc_collection = collection, treatment = treatment)

#plot box plots for PC embeddings
PC1_dataframe %>% 
  ggplot(aes(x=individual, y=PC1_embeddings)) +
  geom_boxplot()

PC1_dataframe %>% 
  ggplot(aes(x=sc_collection, y=PC1_embeddings)) +
  geom_boxplot()

PC1_dataframe %>% 
  ggplot(aes(x=treatment, y=PC1_embeddings)) +
  geom_boxplot()



PC2_dataframe %>% 
  ggplot(aes(x=individual, y=PC1_embeddings)) +
  geom_boxplot()

PC2_dataframe %>% 
  ggplot(aes(x=sc_collection, y=PC1_embeddings)) +
  geom_boxplot()

PC2_dataframe %>% 
  ggplot(aes(x=treatment, y=PC1_embeddings)) +
  geom_boxplot()




#one way ANOVAs
#PC1
model.aov <- aov(PC1_embeddings ~ individual, data = PC1_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)

model.aov <- aov(PC1_embeddings ~ sc_collection, data = PC1_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)

model.aov <- aov(PC1_embeddings ~ treatment, data = PC1_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)

#PC2
model.aov <- aov(PC2_embeddings ~ individual, data = PC2_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)

model.aov <- aov(PC2_embeddings ~ sc_collection, data = PC2_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)

model.aov <- aov(PC2_embeddings ~ treatment, data = PC2_dataframe)
summary(model.aov)
etasq(model.aov, partial = FALSE)



#correlation between variables and first 6 PCs
p_comps <- 1:6
pc_cov_cor <- matrix(nrow = ncol(info), ncol = length(p_comps),
                     dimnames = list(colnames(info), colnames(ANT1.2@reductions$pca@cell.embeddings)[p_comps]))
for (pc in p_comps) {
  for (covariate in 1:ncol(info)) {
    lm_result <- lm(ANT1.2@reductions$pca@cell.embeddings[, pc] ~ info[, covariate])
    r2 <- summary(lm_result)$r.squared
    pc_cov_cor[covariate, pc] <- r2
  }
}
datatable(pc_cov_cor)

#convert to long format
pc_cov_cor_2 <- as.data.frame(pc_cov_cor)
pc_cov_cor_2$variable <- rownames(pc_cov_cor)
pc_cov_cor_2 <- gather(pc_cov_cor_2, key = "pc", value = "cor", -variable)
head(pc_cov_cor_2)

#plot correlation heat map for PC1-6 vs variables
d_heatmap <- pc_cov_cor_2
d_heatmap$variable <- factor(d_heatmap$variable,
                             levels = c("individual", "sc_collection", "treatment"),
                             labels = c("individual", "collection batch", "treatment status"))
pca_heat <- ggplot(d_heatmap, aes(x = pc, y = variable)) +
  geom_tile(aes(fill = cor), colour = "white") +
  scale_fill_gradient(low = "white", high = "red", limits = c(0, 1)) +
  labs(x = "Principal Component", y = "",
       title = "Correlation between principal components and experimental variables")
pca_heat
```





# Clustering

```{r clustering}
num_PCs <- 50

ANT1.2 <- FindNeighbors(ANT1.2, dims = 1:num_PCs)
ANT1.2 <- FindClusters(ANT1.2, resolution = 0.5)

#Run UMAP
ANT1.2 <- RunUMAP(ANT1.2, dims = 1:num_PCs)
p1 <- DimPlot(ANT1.2, reduction = "umap")
p2 <- DimPlot(ANT1.2, reduction = "umap", group.by = "orig.ident")
p3 <- DimPlot(ANT1.2, reduction = "umap", group.by = "labels")
grid.arrange(p1, p2, p3, nrow = 2)
```

```{r expression by individual}
#Chondrocyte markers
VlnPlot(ANT1.2, features = c("COL2A1", "SOX9", "ACAN"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("COL10A1", "COL11A1", "SOX5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("TGFB3", "SOX6"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")

#Strain markers
VlnPlot(ANT1.2, features = c("MMP1", "MMP13", "IL1B", "MMP3"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("MMP2", "COMP", "TNF"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("IBSP", "TIMP1",  "MMP9"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("ADAMTS4", "ADAMTS5"), 
    pt.size = 0.2, ncol = 3, group.by = "labels")


#MSC Flow markers
VlnPlot(ANT1.2, features = c("ENG", "THY1",  "NT5E"), 
     pt.size = 0.2, ncol = 3, group.by = "labels")
VlnPlot(ANT1.2, features = c("PTPRC", "CD34", "CD14", "ITGAM", "CD79A", "CD19"), 
     pt.size = 0.2, ncol = 3, group.by = "labels")

```

# Number of cells that have non-zero expression of SOX9, ACAN, or COL2A1

```{r SOX9 ACAN CoL2A1}
counts <- ANT1.2@assays$RNA@counts
#155 cells express SOX9
table(counts["SOX9", ])
#3 cells express ACAN
table(counts["ACAN", ])
#4 cells express COL2A1
table(counts["COL2A1", ])

cell_labels <- ANT1.2@meta.data$labels
#SOX9 broken down by sample
table(counts["SOX9", ][cell_labels == "NA18855_Strain"]) #27/376 cells
table(counts["SOX9", ][cell_labels == "NA18855_Unstrain"]) #38/996 cells
table(counts["SOX9", ][cell_labels == "NA18856_Unstrain"]) #84/428 cells
table(counts["SOX9", ][cell_labels == "NA19160_Strain"]) #5/426 cells
table(counts["SOX9", ][cell_labels == "NA19160_Unstrain"]) #1/253 cells

sox9_expression <- data.frame("counts" = c(27, 38, 84, 5, 1, 349, 958, 344, 421, 252),
                              "type" = c(rep("SOX9 positive", 5), rep("SOX9 negative", 5)),
                              "sample" = rep(c("NA18855_Strain", "NA18855_Unstrain", "NA18856_Unstrain", "NA19160_Strain", "NA19160_Unstrain"),2))


ggplot(sox9_expression, aes(fill=type, y=counts, x=sample)) + 
  geom_bar(position="stack", stat="identity") +
  theme_cowplot(12) +
  theme(legend.position = "top", legend.title = element_blank()) + 
  labs(title = "SOX9 expression by sample", x = "sample", y = "Number of cells")

ggplot(sox9_expression, aes(fill=type, y=counts, x=sample)) + 
  geom_bar(position="fill", stat="identity") +
  theme_cowplot(12) +
  theme(legend.position = "top", legend.title = element_blank()) + 
  labs(title = "SOX9 expression by sample", x = "sample", y = "Proportion of cells")
```


# Save seurat object

```{r save}
# save reorganized sample information
saveRDS(ANT1.2, "data/ANT1_2.rds")
```

# Correlations with cell atlas samples

```{r cell atlas}
###compute correlation matrix between sample and cell lines in cellAtlas
###object: Seurat object
###cell.line: cell Atlas expression matrix
###CorMethod: correlation method to use
ComputeCorMat <- function(object, refs, CorMethod="pearson"){
	data <- GetAssayData(object, slot="data", assay="RNA") #additions - added slot and assay info because SCT assay can cause issues
  #data <- apply(data, 2, function(x) x/sum(x)) #normalize
  #data <- apply(data, 2, function(x) x * 1e+06) #library size
  gene.id <- rownames(data)
  gene.ref <- rownames(refs)
  common <- intersect(gene.id, gene.ref)
  #data <- apply(data[common, ], 2, function(x) { log(x + 0.1) }) #log-scale
  #refs <- apply(refs[common, ], 2, function(x) { x - mean(x) }) #adjust
  data <- data[common, ]
  refs <- refs[common, ]
  n <- ncol(data)
  m <- ncol(refs)
  cor.mat <- matrix(0, nrow=n, ncol=m)
  for (j in 1:m) {
    for (i in 1:n) {
      cor.mat[i,j] <- cor(data[,i], refs[,j], method=CorMethod)
      }
    }
  rownames(cor.mat) <- colnames(data)
  colnames(cor.mat) <- colnames(refs)
	return(cor.mat)
}

###Assign clustered cell atlas as cell type to all cells in the sample
###object: seurat object
###cor.mat: correlation matrix
###K: number of clusters generated from cell atlas
###topCorrelation: number of highest correlated cell lines in cell atlas to be assigned, set to 1 to extract only the top correlated cell line
AssignClusteredLabel <- function (object, cor.mat, K, topCorrelation, dist.method="euclidean", hclust.method="ward.D"){
  dist_mat <- dist(t(cor.mat), method=dist.method)
	hclust_avg_cellline <- hclust(dist_mat, method=hclust.method)
	#plot(hclust_avg_cellline)
	cut_cellline <- cutree(hclust_avg_cellline, k = K)
	cellline_abstract=sapply(strsplit(as.character(colnames(cor.mat)), "\\_"), "[[", 2)
	clusteredCellline=rbind(cut_cellline,cellline_abstract)
	nametable=rep('n',K)
	for(i in 1:K){
		indexy=as.character(i)
		nametable[i]=paste0(names(sort(table(clusteredCellline[2,clusteredCellline[1,]==indexy]),decreasing=TRUE)),collapse='_' )
	}
	assignm=apply(cor.mat,1,function(x){
		indexy=names(which.max(table(clusteredCellline[1,names(sort(x,decreasing=T)[1:topCorrelation])])))
		y=paste0(names( sort(table(clusteredCellline[2,clusteredCellline[1,]==indexy]),decreasing=TRUE)),collapse='_' )
		return(y)
	})
	object$clusteredCellAtlas=assignm
	return(object)
}

#Load cell atlas data matrix (curated in Mengjie's lab)
#references: Mabbott et al. BMC Genomics 2013; Wu C et al. Nucleic acids research 2016
load("/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/cell_atlas_ref_panel")

#Load cell atlas data matrix (curated by GAH)
#references: Human Primary Cell Atlas (Mabbott et al., 2013)
#description: 713 microarray samples processed and normalized as described in Aran et al. (2019); each sample assigned to one of 37 main cell types and 157 subtypes
hpca <- read.table("/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/HumanPrimaryCellAtlasData-SingleR")
hpca.names <- read.csv("/project2/gilad/ghousman/skeletal-human-chimp/human-chimp-skeletal-scRNA/data/HumanPrimaryCellAtlasLabels-SingleR.csv", row.names=1)
hpca.names$label <- paste0(hpca.names$geo.curated,"_",hpca.names$label.curated)
i=1
while(i <= length(colnames(hpca))) {
  colnames(hpca)[i] <- hpca.names$label[rownames(hpca.names)==colnames(hpca)[i]]
  i=i+1
}


#Compute the correlation matrix use a seurat object and cell line matrix as input
cor.mat=ComputeCorMat(ANT1.2, refs=cell.line)
cor.mat.hpca=ComputeCorMat(ANT1.2, refs=hpca)

#Add a metadata column "clusteredCellAtlas" to the object
ANT1.2 <- AssignClusteredLabel(ANT1.2, cor.mat, K=50, topCorrelation=10)
ANT1.2 <- AssignClusteredLabel(ANT1.2, cor.mat.hpca, K=50, topCorrelation=5)


#Visualize assigned clusters
DimPlot(ANT1.2, reduction="umap", group.by="clusteredCellAtlas", pt.size=0.5)
as.data.frame(table(ANT1.2$clusteredCellAtlas))

DimPlot(ANT1.2, reduction="umap", group.by="labels", split.by="clusteredCellAtlas", pt.size=0.5) +
  theme(plot.title = element_text(size=1)) +
  labs(x="UMAP1",y="UMAP2")


saveRDS(ANT1.2, "data/ANT1_2.rds")
```

