---
title: "Check expression of genes of interest"
author: "Anthony Hung"
date: "2019-12-16"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# 

Check chondro/msc markers for absolute expression
strain markers boxplot comparisons

```{r load}
#load in normalized/filtered data
filt_cpm <- readRDS("data/norm_filtered_counts.rds")

# load in reordered sample information
sampleinfo <- readRDS("data/Sample.info.RNAseq.reordered.csv")

# load gene annotations
gene_anno <- read.delim("data/gene-annotation.txt",
                        sep = "\t")
```

# Define functions to plot boxplot of results

```{r boxplot functions}
#Inputs: count matrix, sample information data frame, and gene of interest in the ENSG format
get_gene_long <- function(counts, info, gene){
     gene_counts <- as.data.frame(counts[gene,])
     names(gene_counts) <- "CPM"
     gene_counts_info <- cbind(gene_counts, info)
     return(gene_counts_info)
}

boxplot_gene_treatment <- function(counts_, info_, gene_){
     df <- get_gene_long(counts = counts_, info = info_, gene = gene_)
     
     ggplot(df, aes(x = treatment, y = CPM)) +
          geom_boxplot() + 
          labs(title=paste0("Expression of ", as.character(gene_), " by treatment"), x= "Treatment", y = "Counts per million reads")
}
```

