---
title: "fasttopics_on_unintegrated_data"
author: "Anthony Hung"
date: "2020-09-30"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

#load data (merged but unintegrated data from single cell pilot experiment)

```{r load}
library(Seurat)
library(dplyr)
ANT1.2 <- readRDS("data/ANT1_2.rds")
```

# Fasttopics on unintegrated data

##separately on each 10x lane

### ANT1

```{r fasttopics ANT1}
library(fastTopics)
#need to fit the model to the count data (unintegrated)
raw_counts <- ANT1.2@assays$RNA@counts[ANT1.2$orig.ident == "ANT1",]
raw_counts <- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

#fit model
# fit <- fit_poisson_nmf(raw_counts,k = 7,numiter = 200)
# saveRDS(fit, "output/fasttopics/ANT1_k=7.rds")
fit <- readRDS("output/fasttopics/ANT1_k=7.rds")

#compute weights and topics (need to add up to 1)
l <- fit$L
f <- fit$F
weights <- sweep(l, MARGIN = 2, colSums(f), `*`)
scale <- rowSums(weights)
weights <- weights / scale
topics <- f / colSums(f) # add up to 1
```

```{r markers}
markers <- c("THY1", "NT5E", "ENG", "CD44",
             "COL2A1", "SOX9", "COL10A1",
             "COL11A1", "SOX5", "SOX6", "TIMP2", "TIMP3"
             )

markers_description <- data.frame(marker_type = c(rep("MSC", 4), rep("Chondrocyte", 8)))
rownames(markers_description) <- markers
```

```{r plot correlations ANT1}
library(pheatmap)
library(RColorBrewer)
library(dummies)
library(tidyr)
library(ggplot2)
library(cowplot)
theme_set(theme_cowplot())

#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT1"])
rownames(sample) <- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == "ANT1",])
sample_labels <- sample %>% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT1"]`) 
sample <- sample_labels %>% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT1"]`, c("Individual", "condition"))
#reorder weights df to match sample order
weights <- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = "Heatmap of topic weights grouped by cell labels")
#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT1"])
rownames(sample_dummy) <- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == "ANT1",])
sample_dummies <- dummy.data.frame(sample_dummy)
names(sample_dummies) <- c("NA18855_Strain", "NA18856_Unstrain", "NA19160_Unstrain")
cor_by_sample <- cor(sample_dummies, weights, method = "pearson")
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by sample label")
#same heatmap but group all the cells that are of the same treatment together
sample_ind_treat_dummies <- dummy.data.frame(sample)
cor_by_treat <- cor(sample_ind_treat_dummies %>% dplyr::select(contains("condition")), weights, method = "pearson")
pheatmap(cor_by_treat, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by Treatment")

#plot heatmap of relative expression of marker genes in each topic
topics_markers <- topics[markers[-c(4,8,10:12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
topics_markers <- topics[markers[-c(1,2,4,8,10:12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
```

### ANT2

```{r fasttopics ANT2}
#need to fit the model to the count data (unintegrated)
raw_counts <- ANT1.2@assays$RNA@counts[ANT1.2$orig.ident == "ANT2",]
raw_counts <- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

#fit model
# fit <- fit_poisson_nmf(raw_counts,k = 7,numiter = 200)
# saveRDS(fit, "output/fasttopics/ANT2_k=7.rds")
fit <- readRDS("output/fasttopics/ANT2_k=7.rds")

#compute weights and topics (need to add up to 1)
l <- fit$L
f <- fit$F
weights <- sweep(l, MARGIN = 2, colSums(f), `*`)
scale <- rowSums(weights)
weights <- weights / scale
topics <- f / colSums(f) # add up to 1
```

```{r plot correlations ANT2}
library(pheatmap)
library(RColorBrewer)
library(dummies)
library(tidyr)

#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT2"])
rownames(sample) <- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == "ANT2",])
sample_labels <- sample %>% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == \"ANT2\"]`)
sample <- sample_labels %>% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$orig.ident == \"ANT2\"]`, c("Individual", "condition"))
#reorder weights df to match sample order
weights <- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = "Heatmap of topic weights grouped by cell labels")
#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$orig.ident == "ANT2"])
rownames(sample_dummy) <- rownames(ANT1.2@meta.data[ANT1.2$orig.ident == "ANT2",])
sample_dummies <- dummy.data.frame(sample_dummy)
names(sample_dummies) <- c("NA18855_Untrain", "NA19160_Strain")
cor_by_sample <- cor(sample_dummies, weights, method = "pearson")
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by sample label")

#plot heatmap of relative expression of marker genes in each topic
topics_markers <- topics[markers[c(4,8,10,11,12)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
topics_markers <- topics[markers[c(4,8,10,11)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
```



##jointly on both 10x lanes

```{r fasttopics joint}
#need to fit the model to the count data (unintegrated)
raw_counts <- ANT1.2@assays$RNA@counts
raw_counts <- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

#fit model
# fit <- fit_poisson_nmf(raw_counts,k = 11,numiter = 200)
# saveRDS(fit, "output/fasttopics/unintegrated_k=11.rds")
fit <- readRDS("output/fasttopics/unintegrated_k=11.rds")


#compute weights and topics (need to add up to 1)
l <- fit$L
f <- fit$F
weights <- sweep(l, MARGIN = 2, colSums(f), `*`)
scale <- rowSums(weights)
weights <- weights / scale #rescaled l
topics <- f / colSums(f) # add up to 1; rescaled f
```

```{r plot correlations joint}
#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample <- as.data.frame(ANT1.2@meta.data$labels)
rownames(sample) <- rownames(ANT1.2@meta.data)
sample_labels <- sample %>% 
     arrange(`ANT1.2@meta.data$labels`) 
sample <- sample_labels %>% 
     separate(`ANT1.2@meta.data$labels`, c("Individual", "condition"))
#reorder weights df to match sample order
weights <- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = "Heatmap of topic weights grouped by cell labels")
#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy <- as.data.frame(ANT1.2@meta.data$labels)
rownames(sample_dummy) <- rownames(ANT1.2@meta.data)
sample_dummies <- dummy.data.frame(sample_dummy)
cor_by_sample <- cor(sample_dummies, weights, method = "pearson")
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by sample label")
#same heatmap but group all the cells that are of the same individual or treatment together
sample_ind_treat_dummies <- dummy.data.frame(sample)
cor_by_ind <- cor(sample_ind_treat_dummies %>% dplyr::select(contains("Individual")), weights, method = "pearson")
pheatmap(cor_by_ind, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by Individual")
cor_by_treat <- cor(sample_ind_treat_dummies %>% dplyr::select(contains("condition")), weights, method = "pearson")
pheatmap(cor_by_treat, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by Treatment")
#same heatmap but group all the cells that are of the same 10x lane together 
collection_dummy <- as.data.frame(ANT1.2@meta.data$orig.ident)
rownames(collection_dummy) <- rownames(ANT1.2@meta.data)
collection_dummies <- dummy.data.frame(collection_dummy)
cor_by_collection <- cor(collection_dummies, weights, method = "pearson")
pheatmap(cor_by_collection, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by collection")

#plot heatmap of relative expression of marker genes in each topic
topics_markers <- topics[markers,]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
topics_markers <- topics[markers[-12],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
```

## Identify clusters from PCA of topic proportions

```{r PCA topics}
#k=11, 10 pcs
p1 <- pca_plot(poisson2multinom(fit),pcs = 1:2,fill = "none")
p2 <- pca_plot(poisson2multinom(fit),pcs = 3:4,fill = "none")
p3 <- pca_plot(poisson2multinom(fit),pcs = 5:6,fill = "none")
p4 <- pca_plot(poisson2multinom(fit),pcs = 7:8,fill = "none")
p5 <- pca_plot(poisson2multinom(fit),pcs = 9:10,fill = "none")
plot_grid(p1,p2,p3,p4,p5,nrow = 2,ncol = 3)
#k=11, 10 pcs
p1 <- pca_plot(poisson2multinom(fit),pcs = 1:2,fill = "none") + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p2 <- pca_plot(poisson2multinom(fit),pcs = 3:4,fill = "none") + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p3 <- pca_plot(poisson2multinom(fit),pcs = 5:6,fill = "none") + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels))) 
p4 <- pca_plot(poisson2multinom(fit),pcs = 7:8,fill = "none") + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
p5 <- pca_plot(poisson2multinom(fit),pcs = 9:10,fill = "none") + geom_point(aes(color = as.factor(ANT1.2@meta.data$labels)))
plot_grid(p1,p2,p3,p4,p5,nrow = 2,ncol = 3)


p6  <- pca_hexbin_plot(poisson2multinom(fit),pcs = 1:2) + guides(fill = "none")
p7  <- pca_hexbin_plot(poisson2multinom(fit),pcs = 3:4) + guides(fill = "none")
p8  <- pca_hexbin_plot(poisson2multinom(fit),pcs = 5:6) + guides(fill = "none")
p9  <- pca_hexbin_plot(poisson2multinom(fit),pcs = 7:8) + guides(fill = "none")
p10 <- pca_hexbin_plot(poisson2multinom(fit),pcs = 9:10)+ guides(fill = "none")
plot_grid(p6,p7,p8,p9,p10,nrow = 2,ncol = 3)
```

# Look at expression of specific markers between samples

```{r plot markers}
#by individual
for (marker in markers){
     query <- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot <- cbind(query, sample)
     plot <- ggplot(query_plot, aes(x=query, color = as.factor(Individual), group=as.factor(Individual))) +
             stat_density(geom='line',kernel='gaussian',bw='nrd0' 
                                                            #nrd0='Silverman'
                                                            ,size=1,position='identity') +
                          labs(title = paste0(marker), x = "Latent gene expression", color = "Individual")
     print(plot)
}

#by condition
for (marker in markers){
     query <- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot <- cbind(query, sample)
     plot <- ggplot(query_plot, aes(x=query, color = as.factor(condition), group=as.factor(condition))) +
             stat_density(geom='line',kernel='gaussian',bw='nrd0' 
                                                            #nrd0='Silverman'
                                                            ,size=1,position='identity') +
                          labs(title = paste0(marker), x = "Latent gene expression", color = "Condition")
     print(plot)
}

# by sample
for (marker in markers){
     query <- weights %*% topics[marker, ] #represents latent gene expression; has same dim as num of barcodes (can subset by the labels or whatever category of interest and use to compute the gaussian curves to plot)
     query_plot <- cbind(query, sample_labels)
     plot <- ggplot(query_plot, aes(x=query, color = as.factor(ANT1.2@meta.data$labels), group=as.factor(ANT1.2@meta.data$labels))) +
             stat_density(geom='line',kernel='gaussian',bw='nrd0' 
                                                            #nrd0='Silverman'
                                                            ,size=1,position='identity') +
                          labs(title = paste0(marker), x = "Latent gene expression", color = "Label")
     print(plot)
}
```

##jointly on both 10x lanes (only unstrain samples)

```{r fasttopics joint unstrain}
#need to fit the model to the count data (unintegrated)
raw_counts <- ANT1.2@assays$RNA@counts
raw_counts <- as.matrix(raw_counts)
raw_counts <- raw_counts[,ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain")]
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

#fit model
# fit <- fit_poisson_nmf(raw_counts,k = 11,numiter = 200)
# saveRDS(fit, "output/fasttopics/unintegrated_unstrain_k=11.rds")
fit <- readRDS("output/fasttopics/unintegrated_unstrain_k=11.rds")


#compute weights and topics (need to add up to 1)
l <- fit$L
f <- fit$F
weights <- sweep(l, MARGIN = 2, colSums(f), `*`)
scale <- rowSums(weights)
weights <- weights / scale
topics <- f / colSums(f) # add up to 1
```

```{r plot correlations joint only unstrain}
#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain")])
rownames(sample) <- rownames(ANT1.2@meta.data)[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain")]
sample_labels <- sample %>% 
     arrange(`ANT1.2@meta.data$labels[ANT1.2$labels %in% c(\"NA19160_Unstrain\", \"NA18855_Unstrain\", \"NA18856_Unstrain\")]`) 
sample <- sample_labels %>% 
     separate(`ANT1.2@meta.data$labels[ANT1.2$labels %in% c(\"NA19160_Unstrain\", \"NA18855_Unstrain\", \"NA18856_Unstrain\")]`, c("Individual", "condition"))
#reorder weights df to match sample order
weights <- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = "Heatmap of topic weights grouped by cell labels")
#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy <- as.data.frame(ANT1.2@meta.data$labels[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain")])
rownames(sample_dummy) <- rownames(ANT1.2@meta.data[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain"),])
sample_dummies <- dummy.data.frame(sample_dummy)
cor_by_sample <- cor(sample_dummies, weights, method = "pearson")
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by sample label")
#same heatmap but group all the cells that are of the same individual or treatment together
sample_ind_treat_dummies <- dummy.data.frame(sample)
cor_by_ind <- cor(sample_ind_treat_dummies %>% dplyr::select(contains("Individual")), weights, method = "pearson")
pheatmap(cor_by_ind, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by Individual")
cor_by_treat <- cor(sample_ind_treat_dummies %>% dplyr::select(contains("condition")), weights, method = "pearson")
#same heatmap but group all the cells that are of the same 10x lane together 
collection_dummy <- as.data.frame(ANT1.2@meta.data$orig.ident[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain")])
rownames(collection_dummy) <- rownames(ANT1.2@meta.data[ANT1.2$labels %in% c("NA19160_Unstrain", "NA18855_Unstrain", "NA18856_Unstrain"),])
collection_dummies <- dummy.data.frame(collection_dummy)
cor_by_collection <- cor(collection_dummies, weights, method = "pearson")
pheatmap(cor_by_collection, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by collection")

#plot heatmap of relative expression of marker genes in each topic
topics_markers <- topics[markers,]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
topics_markers <- topics[markers[-12],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
```


# fasttopics with external single cell datasets as well

```{r load external data}
Combined.common <- readRDS("data//external_scRNA/Combined_singlecell_data_first.rds")
#Combine.common.noliver <- readRDS("data/Combined_singlecell_data_allGenes_noliver.rds")

#need to fit the model to the count data (unintegrated)
raw_counts <- Combined.common@assays$RNA@counts
raw_counts <- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

#fit model
# fit <- fit_poisson_nmf(raw_counts,k = 4,numiter = 200)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=4.rds")
# fit <- fit_poisson_nmf(raw_counts,k = 5,numiter = 200)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=5.rds")
# fit <- fit_poisson_nmf(raw_counts,k = 6,numiter = 200)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=6.rds")
# fit <- fit_poisson_nmf(raw_counts,k = 7,numiter = 200)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=7.rds")
# fit <- fit_poisson_nmf(raw_counts,k = 8,numiter = 150)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=8.rds")
# fit <- fit_poisson_nmf(raw_counts,k = 11,numiter = 200)
# saveRDS(fit, "output/fasttopics/Combined_external_data_k=11.rds")
fit <- readRDS("output/fasttopics/Combined_external_data_k=5.rds") #200 iterations

#compute weights and topics (need to add up to 1)
l <- fit$L
f <- fit$F
weights <- sweep(l, MARGIN = 2, colSums(f), `*`)
scale <- rowSums(weights)
weights <- weights / scale
topics <- f / colSums(f) # add up to 1
```


```{r plot correlations external}
library(stringr)
library(pheatmap)
library(dummies)
#plot heatmap of topic weights for samples
#get labels of cells and reorder to group them
sample <- as.data.frame(Combined.common@meta.data$Cell.Type)
sample_labels <- sample %>% 
     dplyr::transmute(Cell.Type = stringr::word(`Combined.common@meta.data$Cell.Type`, start = 1))
rownames(sample_labels) <- rownames(Combined.common@meta.data)
sample <- sample_labels %>% 
     dplyr::arrange(Cell.Type)
#reorder weights df to match sample order
weights_reordered <- weights[match(rownames(sample), rownames(weights)),]
#make the plot (color strip on the left is by indivdual)
pheatmap(weights_reordered, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = sample,
         show_rownames = FALSE, main = "Heatmap of topic weights grouped by cell labels")
#same heatmap but group all the cells that are of the same label together (more interpretable??)
sample_dummy <- as.data.frame(Combined.common@meta.data$Cell.Type)
sample_dummy <- sample_dummy %>% 
     dplyr::transmute(Cell.Type = stringr::word(`Combined.common@meta.data$Cell.Type`, start = 1))
rownames(sample_dummy) <- rownames(Combined.common@meta.data)
sample_dummies <- dummy.data.frame(sample_dummy)
cor_by_sample <- cor(sample_dummies, weights, method = "pearson")
pheatmap(cor_by_sample, cluster_cols = TRUE, cluster_rows = TRUE, main = "Correlation by sample label")


markers <- c("THY1", "NT5E", "ENG", "CD44",
             "COL2A1", "SOX9", "COL10A1", "ACAN",
             "COL11A1", "SOX5", "SOX6", "TIMP2", "TIMP3"
             )

markers_description <- data.frame(marker_type = c(rep("MSC", 4), rep("Chondrocyte", 9)))
rownames(markers_description) <- markers

#plot heatmap of relative expression of marker genes in each topic
topics_markers <- topics[markers[-c(5,7,8,9)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
topics_markers <- topics[markers[-c(4,5,7,8,9,13)],]
pheatmap(topics_markers, cluster_cols = FALSE, cluster_rows = FALSE, annotation_row = markers_description)
```

# Semisupervised fasttopics

Constrain it so that k = 4, and chondrocyte, iPSC, iPSC-MSC, and hepatocytes are the four existing topics/clusters. Then find out where iPSC-Chondrocytes fall in terms of membership in these four clusters.

```{r semisupervised}
library(fastTopics)
library(Seurat)
library(Matrix)
Combined.common <- readRDS("data//external_scRNA/Combined_singlecell_data_first.rds")
#Combine.common.noliver <- readRDS("data/Combined_singlecell_data_allGenes_noliver.rds")

#randomly subset 1/4 of the cells to hopefully avoid segfault
# Get the number of cells from the original Seurat object
n.cells <- length(rownames(Combined.common@meta.data))

# Define number of cells for the first object (subset 1)
# 'ceiling' is used to get integer values if n.cells is odd  
n.cells.subset.1 <- ceiling(n.cells/10)

# Set a seed for reproducible subsampling of cells
set.seed(seed = 1)

# Sample randomly half (or half + 1 if n.cells is odd) of the cells 
# for the first object (subset 1)
cells.subset.1 <- base::sample(x = rownames(Combined.common@meta.data), size = n.cells.subset.1, replace = F)


# Create 2 subsetted objects, each formed by the corresponding cells
# Set 'do.clean = T' if you want to keep only the 'data' and 'raw.data' slots
object.subset.1 <- subset(x = Combined.common, cells = cells.subset.1)



#need to fit the model to the count data (unintegrated)
raw_counts <- object.subset.1@assays$RNA@counts
raw_counts <- as.matrix(raw_counts)
#remove genes without any counts in droplets
raw_counts <- raw_counts[rowSums(raw_counts > 0) > 0,] 
#get into correct orientation (barcodes x features)
raw_counts <- t(as.matrix(raw_counts))

n_topics <- 4
loadings <- matrix(0.01, ncol = n_topics, nrow = nrow(raw_counts))
rownames(loadings) <- rownames(raw_counts)
colnames(loadings) <- c("k1", "k2", "k3", "k4")
celltypes <- object.subset.1@meta.data$Cell.Type
#chondrocyte: k1
loadings[celltypes == "Chondrocyte_", ] <- c(1, 0, 0, 0)
#iPSC: k2
loadings[celltypes == "iPSC", ] <- c(0, 1, 0, 0)
#hepatocyte: k3
loadings[celltypes == "Hepatocyte", ] <- c(0, 0, 1, 0)
#iPSC-MSC: k4
loadings[celltypes == "iPSC-MSC", ] <- c(0, 0, 0, 1)
update_loadings <- 1 - rowSums(loadings)

initial_fit <- init_poisson_nmf(X = raw_counts,
                 F = matrix(0.1, ncol = n_topics, nrow = ncol(raw_counts)),
                 L = loadings)

refit_semisupervised_k4 <- fit_poisson_nmf(X = raw_counts,
                                           fit0 = initial_fit,
                                           update.loadings = update_loadings,
                                           method = "scd",
                                           numiter = 200)
saveRDS(refit_semisupervised_k4, "output/fasttopics/Combined_external_data_semisupervised_k=4.rds")
```

