---
title: "Normalization and filtering of bulkRNA Count data"
author: "Anthony Hung"
date: "2019-12-16"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Perform cpm normalization of raw count data and filtering of lowly expressed genes

```{r load libraries and data}
library("gplots")
library("ggplot2")
library("reshape")
library("edgeR")
library("RColorBrewer")
library("scales")
library("cowplot")
library("dplyr")

# Load colors 
colors <- colorRampPalette(c(brewer.pal(9, "Blues")[1],brewer.pal(9, "Blues")[9]))(100)
pal <- c(brewer.pal(9, "Set1"), brewer.pal(8, "Set2"), brewer.pal(12, "Set3"))


# load in relabeled counts
raw_counts <- readRDS("data/raw_counts_relabeled.rds")

# load in reordered sample information
sampleinfo <- readRDS("data/Sample.info.RNAseq.reordered.csv")
```

# Perform cpm normalization on all counts and plot density plots

```{r cpm normalization}
cpm <- cpm(raw_counts, log=TRUE)
head(cpm)

strained <- sampleinfo$treatment == "Strain"
unstrained <- sampleinfo$treatment == "Unstrain"
ind_1 <- sampleinfo$Individual == "NA18855 "
ind_2 <- sampleinfo$Individual == "NA18856 "
ind_3 <- sampleinfo$Individual == "NA19160 "


# Look at density plots for all individuals broken down by each treatment type
col = as.data.frame(pal[as.numeric(sampleinfo$Individual)])

plotDensities(cpm[,strained], col=col[strained, ], legend="topright")
plotDensities(cpm[,unstrained], col=col[unstrained, ], legend="topright")

# Look at density plots broken down by individual
col = as.data.frame(pal[as.numeric(sampleinfo$treatment)])

plotDensities(cpm[,ind_1], col=col[ind_1, ], legend="topright")
plotDensities(cpm[,ind_2], col=col[ind_2, ], legend="topright")
plotDensities(cpm[,ind_3], col=col[ind_3, ], legend="topright")
```

# Boxplots of cpm across samples

```{r boxplot cpm}
meltcpm <- melt(cpm)
names(meltcpm) <- c("gene", "sampleID", "cpm")
p <- ggplot(meltcpm, aes(factor(sampleID), cpm)) 
p + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))
```

# Filtering for lowly expressed genes (avg CPM > 0.5 in at least 2 samples)

```{r filter}
cutoff <- 0

#plot densities to determine cutoff
cpm_median <- apply(cpm, 1, median)
cpm_median_density <- density(cpm_median)
cpm_density <- apply(cpm, 2, density)
plot(cpm_density[[1]], lty = "dotted", col = gray(0.05, alpha = 0.5),
     main = "Gene expression distribution",
     xlab = "log2 cpm", sub = "red = median; blue = cutoff")
suppress_output <- lapply(cpm_density, lines, lty = "dotted",
                          col = gray(0.05, alpha = 0.5))
lines(cpm_median_density, col = "red", lwd = 3)
abline(v = cutoff, col = "blue")

#apply filter (Keeping the `r sum(cpm_median > cutoff)` genes with a median log~2~ cpm greater than `r cutoff`.)

counts <- raw_counts %>% 
     filter(cpm_median > cutoff)
rownames(counts) <- rownames(raw_counts)[cpm_median > cutoff]
filtered_cpm <- cpm(counts, log = T)
```

# Boxplots of normalized+filtered counts across samples
```{r boxplot filt cpm}
melt_filt_cpm <- melt(filtered_cpm)
names(melt_filt_cpm) <- c("gene", "sampleID", "cpm")
p1 <- ggplot(melt_filt_cpm, aes(factor(sampleID), cpm)) 
p1 + geom_boxplot() + theme(axis.text.x = element_text(angle = 90))
```

# Save normalized/filtered count matrix and filtered count matrix

```{r save}
saveRDS(counts, "data/filtered_counts.rds")
saveRDS(filtered_cpm, "data/norm_filtered_counts.rds")
```
