---
title: "Normalization and filtering of bulkRNA Count data"
author: "Anthony Hung"
date: "2019-12-16"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Perform cpm normalization of raw count data and filtering of lowly expressed genes

```{r config}
#configuration
proj_dir <- "/project2/gilad/anthonyhung/Projects/OAStrain_project/"
sample_file_info <- "bulkPilot_Sample_info_RNAseq.csv" # located within the proj_dir
```

```{r load libraries and data}
library()



# load in counts
raw_counts <- readRDS("data/raw_counts_relabeled.rds")

#Sample information
samples <- read.csv(paste0(proj_dir, sample_file_info))
head(samples)
```

# Perform cpm normalization on all counts

```{r cpm normalization}
cpm <- cpm(raw_counts, log=TRUE)
head(cpm)

strained <- c() #indices for strained samples
unstrained <- c()
ind_1 <- c()
ind_2 <- c()
ind_3 <- c()


# Look at all individuals for each treatment
col = as.data.frame(pal[as.numeric(samples$Individual)])

plotDensities(cpm[,strained], col=col[strained, ], legend="topright")
plotDensities(cpm[,unstrained], col=col[unstrained, ], legend="topright")

# Look at all individuals for each treatment
col = as.data.frame(pal[as.numeric(samples$treatment)])

plotDensities(cpm[,ind_1], col=col[ind_1, ], legend="topright")
plotDensities(cpm[,ind_2], col=col[ind_2, ], legend="topright")
plotDensities(cpm[,ind_3], col=col[ind_3, ], legend="topright")
```

# Boxplots of normalized counts across samples
```{r}

```

# Filtering for lowly expressed genes (avg CPM > 0.5 in at least 2 samples)

```{r filter}
cpm_cutoff <- 0.5
n_samples <- 2


```

# Boxplots of normalized/filtered counts across samples
```{r}

```

# Save normalized/filtered count matrix

```{r save}

```






## Setup

```{r packages, message=FALSE}
library("data.table")
library("dplyr")
library("edgeR")
```

Input raw counts file.

```{r input-counts}
data_raw <- fread("../data/subread-counts-per-sample.txt", data.table = FALSE)
data_raw[1:5, 1:5]
```

Input experimental information file.

```{r experimental-info}
info <- read.delim("../data/experiment-info.txt", stringsAsFactors = FALSE)
head(info)
```

Sort raw data file so that it is in the same order as the information file.

```{r sort-data-raw}
data_raw <- data_raw %>% arrange(desc(status), individual, desc(treatment))
stopifnot(data_raw$inindividual == info$individual,
          data_raw$status == info$status,
          data_raw$treatment == info$treatment)
```  

Create a sample-x-gene counts matrix.

```{r transpose-counts}
counts_raw <- data_raw %>% select(-(individual:treatment)) %>% t()
colnames(counts_raw) <- info$id
stopifnot(ncol(counts_raw) == 50)
```

## Remove lowly expressed genes

```{r median-density}
cpm_raw <- cpm(counts_raw, log = TRUE)
cpm_raw_median <- apply(cpm_raw, 1, median)
cpm_raw_median_density <- density(cpm_raw_median)
cpm_raw_density <- apply(cpm_raw, 2, density)
plot(cpm_raw_density[[1]], lty = "dotted", col = gray(0.05, alpha = 0.5),
     main = "Gene expression distribution",
     xlab = "log2 cpm", sub = "red = median; blue = cutoff")
suppress_output <- lapply(cpm_raw_density, lines, lty = "dotted",
                          col = gray(0.05, alpha = 0.5))
lines(cpm_raw_median_density, col = "red", lwd = 3)
cutoff <- 0
abline(v = cutoff, col = "blue")
```

Keeping the `r sum(cpm_raw_median > cutoff)` genes with a median log~2~ cpm greater than `r cutoff`.

```{r filter-genes}
counts <- as.data.frame(counts_raw) %>% filter(cpm_raw_median > cutoff)
rownames(counts) <- rownames(counts_raw)[cpm_raw_median > cutoff]
```

Save the count matrix.

```{r}
write.table(counts, "../data/counts.txt", quote = FALSE, sep = "\t",
            col.names = NA)
```
