---
title: "Pilot_GoMAnalysis"
author: "Anthony Hung"
date: "2020-01-29"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Applying topic modeling approaches on sc pilot data

Include data from:
     Pilot (iPSC-derived chondrocytes from 3 individuals x 2 treatment conditions; 10x)
     MSC-Pilot (Genevieve Housman's Pilot data of iPSCs and iPSC-derived MSCs after 9 days of differentiation; dropseq)
     Cardiomyocytes (Reem's cardiomyocyte data for ____ day chondrocytes; dropseq)
     

# Load in datasets
```{r Loading in data matrices}
#Pilot data
ANT1.2 <- readRDS("data/ANT1_2.rds")

#Chondrocytes from OA patients (https://www.ncbi.nlm.nih.gov/pubmed/30026257)
chondro <- read.table("/project2/gilad/anthonyhung/Projects/OAStrain_project/outside_scData/GSE104782_allcells_UMI_count.txt.gz", header = T)
chondro_genes <- chondro[,1]
chondro_data <- chondro[,-1]
chondro <- chondro_data
rownames(chondro) <- chondro_genes

#iPSCs from Human/chimp panel

#MSCs from Human/chimp panel d9

#Cardiomyocytes differentiated from iPSCs d

```

## Subset Pulse-seq data for finer-scale analysis
In the interest of exploring finer structure within the Pulse-seq data, here I partition these data into groups of cell types as identified by the authors. These smaller datasets will then be analyzed separately.
```{r subset pulseseq}
pulseseq_counts_Tuft_Neuro <- pulseseq_counts[,grepl("Tuft|Neuroendocrine", colnames(pulseseq_counts))]
dim(pulseseq_counts_Tuft_Neuro)
pulseseq_counts_Goblet <- pulseseq_counts[,grepl("Goblet", colnames(pulseseq_counts))]
dim(pulseseq_counts_Goblet)
pulseseq_counts_Club <- pulseseq_counts[,grepl("Club", colnames(pulseseq_counts))]
dim(pulseseq_counts_Club)
pulseseq_counts_Goblet_Club <- pulseseq_counts[,grepl("Club|Goblet", colnames(pulseseq_counts))]
dim(pulseseq_counts_Goblet_Club)
```


# Data preprocessing in Seurat (code adapted from code supplied by Kushal Dey)
```{r load_packages, echo=FALSE, eval=TRUE, message=FALSE, warning=FALSE}
library(CountClust)
library(maptpx)
library(ggplot2)
library(MAST)
library(Seurat)
library(RColorBrewer)
```
The initial preprocessing steps take place in Seurat.

## Make a function to convert datasets into a seurat object, filter/normalize/scale, find variable genes, and Run PCA and tSNE; run the function on the three datasets and save the output as rda files. Also label each cell with its identified cell type.
The code below represents preprocessing of the three datasets performed using Seurat. In brief, variable genes are selected and cells are selected that pass the mitochondrial gene content filter.

In each of these datasets, cell type for each cell was parsed from the cell labels. These cell types were determined by the authors through k nearest neighbour-graph based clustering followed by labeling of each cluster using known markers for tracheal epithelial subsets.

Details from the paper: "The number k of nearest neighbours was chosen in a manner roughly consistent with the size of the dataset, and set to 25 and 150 for plate- and droplet-based data, respectively. For sub-clustering of rare cell subsets, we used k = 100, 50, 50 and 20 for tuft cells, neuroendocrine cells, ionocytes and goblet cells, respectiveThe k nearest neighbour graph was computed using the function ‘nng’ from the R package ‘cccd’ and was then used as the input to Infomap62, implemented using the ‘infomap.community’ function from the ‘igraph’ R package"

In cases where multiple clusters showed marker gene expression for a particular cell type, these clusters were combined to represent that one cell type. This is relevant because of the large number of clusters generated by the authors in order to better separate out relatively rare cell types.

```{r create function to makeSeurat object and filter}
makeSeurat <- function(dataset, filter = TRUE, cutoffs){
  seurat_object <- CreateSeuratObject(raw.data = dataset, min.cells = 3, min.genes = 200)
  
  # calculate percent mitochondrial genes
  mito.genes <- grep(pattern = "^MT-", x = rownames(x = seurat_object@data), value = TRUE)
  percent.mito <- Matrix::colSums(seurat_object@raw.data[mito.genes, ])/Matrix::colSums(seurat_object@raw.data)
  
  # AddMetaData adds columns to object@meta.data, and is a great place to stash QC stats
  seurat_object <- AddMetaData(object = seurat_object, metadata = percent.mito, col.name = "percent.mito")
  
  # Filter cells and normalize
  if(filter == TRUE){
  seurat_object <- FilterCells(object = seurat_object, subset.names = c("nGene", "percent.mito"), 
    low.thresholds = c(200, -Inf), high.thresholds = c(2500, 0.05))
  
  seurat_object <- NormalizeData(object = seurat_object, normalization.method = "LogNormalize", scale.factor = 10000)
  }
  
  # find variable genes
  seurat_object <- FindVariableGenes(object = seurat_object, mean.function = ExpMean, dispersion.function = LogVMR, do.plot = TRUE, x.low.cutoff = cutoffs[1], x.high.cutoff = cutoffs[2], y.cutoff = cutoffs[3])

  # scale object
  seurat_object <- ScaleData(object = seurat_object, vars.to.regress = c("nUMI", "percent.mito"), check.for.norm = FALSE)
  
  # Run PCA and tSNE
  seurat_object <- RunPCA(object = seurat_object, pc.genes = seurat_object@var.genes, do.print = TRUE, pcs.print = 1:5, 
    genes.print = 5)
  seurat_object <- RunTSNE(object = seurat_object, dims.use = 1:10, do.fast = TRUE)
  
  # return seurat object
  return(seurat_object)
}

# make seurat objects for all 3 datasets
if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_seurat.rda")
} else {
  threeprime_seurat <- makeSeurat(threeprime_counts, cutoffs = c(0.0125, 3, 2))
  ident.names <- names(threeprime_seurat@ident)
  threeprime_seurat@ident <- factor(sapply(strsplit(ident.names,split = "_"),function (x) x[3]))
  names(threeprime_seurat@ident) <- ident.names
  save(threeprime_seurat, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_seurat.rda")
}
dim(threeprime_seurat@raw.data)
dim(threeprime_seurat@data)

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_seurat.rda")
} else {
  pulseseq_seurat <- makeSeurat(pulseseq_counts, cutoffs = c(0.0125, 5, 0.5))
  ident.names0 <- names(pulseseq_seurat@ident)
  pulseseq_seurat@ident <- factor(sapply(strsplit(pulseseq_seurat@cell.names,split = "_"),function (x) x[5]))
  names(pulseseq_seurat@ident) <- ident.names0
  save(pulseseq_seurat, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_seurat.rda")
}
dim(pulseseq_seurat@raw.data)
dim(pulseseq_seurat@data)

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_seurat.rda")
} else {
  fullLength_seurat <- makeSeurat(fullLength_logTPM, filter = FALSE, cutoffs = c(0.0125, 5, 0.5))
  ident.names1 <- names(fullLength_seurat@ident)
  fullLength_seurat@ident <- factor(sapply(strsplit(fullLength_seurat@cell.names,split = "_"),function (x) x[4]))
  names(fullLength_seurat@ident) <- ident.names1
  save(fullLength_seurat, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_seurat.rda")
}

# make seurat objects for Pulseseq subsetted data

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_tuft_neuro_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_tuft_neuro_seurat.rda")
} else {
  pulseseq_seurat_Tuft_Neuro <- makeSeurat(pulseseq_counts_Tuft_Neuro, cutoffs = c(0.0125, 5, 0.5))
  ident.names0 <- names(pulseseq_seurat_Tuft_Neuro@ident)
  pulseseq_seurat_Tuft_Neuro@ident <- factor(sapply(strsplit(pulseseq_seurat_Tuft_Neuro@cell.names,split = "_"),function (x) x[5]))
  names(pulseseq_seurat_Tuft_Neuro@ident) <- ident.names0
  save(pulseseq_seurat_Tuft_Neuro, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_tuft_neuro_seurat.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_seurat.rda")
} else {
  pulseseq_seurat_Goblet <- makeSeurat(pulseseq_counts_Goblet, cutoffs = c(0.0125, 5, 0.5))
  ident.names0 <- names(pulseseq_seurat_Goblet@ident)
  pulseseq_seurat_Goblet@ident <- factor(sapply(strsplit(pulseseq_seurat_Goblet@cell.names,split = "_"),function (x) x[5]))
  names(pulseseq_seurat_Goblet@ident) <- ident.names0
  save(pulseseq_seurat_Goblet, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_seurat.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Club_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Club_seurat.rda")
} else {
  pulseseq_seurat_Club <- makeSeurat(pulseseq_counts_Club, cutoffs = c(0.0125, 5, 0.5))
  ident.names0 <- names(pulseseq_seurat_Club@ident)
  pulseseq_seurat_Club@ident <- factor(sapply(strsplit(pulseseq_seurat_Club@cell.names,split = "_"),function (x) x[5]))
  names(pulseseq_seurat_Club@ident) <- ident.names0
  save(pulseseq_seurat_Club, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Club_seurat.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_Club_seurat.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_Club_seurat.rda")
} else {
  pulseseq_seurat_Goblet_Club <- makeSeurat(pulseseq_counts_Goblet_Club, cutoffs = c(0.0125, 5, 0.5))
  ident.names0 <- names(pulseseq_seurat_Goblet_Club@ident)
  pulseseq_seurat_Goblet_Club@ident <- factor(sapply(strsplit(pulseseq_seurat_Goblet_Club@cell.names,split = "_"),function (x) x[5]))
  names(pulseseq_seurat_Goblet_Club@ident) <- ident.names0
  save(pulseseq_seurat_Goblet_Club, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/Pulseseq_Goblet_Club_seurat.rda")
}
```

## Plot PCA and tSNE from Seurat objects
Visualize the datasets after performing the preprocessing.
```{r Plot PCA and tSNE}
#3prime
threeprime_identities <- levels(threeprime_seurat@ident)
my_color_palette <- brewer.pal(length(threeprime_identities), name = "Set2")
DimPlot(object = threeprime_seurat, cols.use = my_color_palette, group.by = "ident") 
DimPlot(threeprime_seurat, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE,  cols.use = my_color_palette, group.by = "ident") 
DimPlot(threeprime_seurat, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette, group.by = "ident") 
threeprime_seurat <- ProjectPCA(object = threeprime_seurat, do.print = FALSE) 
TSNEPlot(object = threeprime_seurat, do.label = TRUE, colors.use = my_color_palette, group.by = "ident") 

#pulseseq
pulseseq_identities <- levels(pulseseq_seurat@ident)
my_color_palette1.1 <- brewer.pal(8, name = "Set2")
my_color_palette1.2 <- brewer.pal(5, name = "Set3")
my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
DimPlot(object = pulseseq_seurat, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
pulseseq_seurat <- ProjectPCA(object = pulseseq_seurat, do.print = FALSE)
TSNEPlot(object = pulseseq_seurat, do.label = TRUE, colors.use = my_color_palette1, group.by = "ident") 

#fullLength
fullLength_identities <- levels(fullLength_seurat@ident)
my_color_palette2 <- brewer.pal(length(fullLength_identities), name = "Set2")
DimPlot(object = fullLength_seurat, cols.use = my_color_palette2, group.by = "ident", pt.shape = "ident")
DimPlot(fullLength_seurat, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette2, group.by = "ident", pt.shape = "ident")
DimPlot(fullLength_seurat, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette2, group.by = "ident", pt.shape = "ident")
fullLength_seurat <- ProjectPCA(object = fullLength_seurat, do.print = FALSE)
TSNEPlot(object = fullLength_seurat, do.label = TRUE, colors.use = my_color_palette2, group.by = "ident", pt.shape = "ident")

#Pulseseq Tuft and neuroendocrine
pulseseq_identities_Tuft_Neuro <- levels(pulseseq_seurat_Tuft_Neuro@ident)
my_color_palette1.1 <- brewer.pal(8, name = "Set2")
my_color_palette1.2 <- brewer.pal(5, name = "Set3")
my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
DimPlot(object = pulseseq_seurat_Tuft_Neuro, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Tuft_Neuro, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Tuft_Neuro, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
pulseseq_seurat_Tuft_Neuro <- ProjectPCA(object = pulseseq_seurat_Tuft_Neuro, do.print = FALSE)
TSNEPlot(object = pulseseq_seurat_Tuft_Neuro, do.label = TRUE, colors.use = my_color_palette1, group.by = "ident") 

#Pulseseq Goblet
pulseseq_identities_Goblet <- levels(pulseseq_seurat_Goblet@ident)
my_color_palette1.1 <- brewer.pal(8, name = "Set2")
my_color_palette1.2 <- brewer.pal(5, name = "Set3")
my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
DimPlot(object = pulseseq_seurat_Goblet, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Goblet, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Goblet, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
pulseseq_seurat_Goblet <- ProjectPCA(object = pulseseq_seurat_Goblet, do.print = FALSE)
TSNEPlot(object = pulseseq_seurat_Goblet, do.label = TRUE, colors.use = my_color_palette1, group.by = "ident") 

#Pulseseq Club
pulseseq_identities_Club <- levels(pulseseq_seurat_Club@ident)
my_color_palette1.1 <- brewer.pal(8, name = "Set2")
my_color_palette1.2 <- brewer.pal(5, name = "Set3")
my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
DimPlot(object = pulseseq_seurat_Club, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Club, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Club, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
pulseseq_seurat_Club <- ProjectPCA(object = pulseseq_seurat_Club, do.print = FALSE)
TSNEPlot(object = pulseseq_seurat_Club, do.label = TRUE, colors.use = my_color_palette1, group.by = "ident") 

#Pulseseq Goblet and Club
pulseseq_identities_Goblet_Club <- levels(pulseseq_seurat_Goblet_Club@ident)
my_color_palette1.1 <- brewer.pal(8, name = "Set2")
my_color_palette1.2 <- brewer.pal(5, name = "Set3")
my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
DimPlot(object = pulseseq_seurat_Goblet_Club, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Goblet_Club, reduction.use = "pca", 3, 4, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
DimPlot(pulseseq_seurat_Goblet_Club, reduction.use = "pca", 5, 6, cells.use = NULL, pt.size = 1, do.return = FALSE, do.bare = FALSE, cols.use = my_color_palette1, group.by = "ident") 
pulseseq_seurat_Goblet_Club <- ProjectPCA(object = pulseseq_seurat_Goblet_Club, do.print = FALSE)
TSNEPlot(object = pulseseq_seurat_Goblet_Club, do.label = TRUE, colors.use = my_color_palette1, group.by = "ident") 

```

# Run CountClust::FitGoM on the data for k = 2:10
Make a function to run the CountClust::FitGoM function on the datasets. 
```{r fitgom function}
runfitGoM <- function(datamatrix, tol = 0.1){
  listofks <- list()
  counter <- 1
  for(i in 2:10){
    listofks[[counter]] <- CountClust::FitGoM(datamatrix, K=i, tol = tol, control=list(tmax=100))
    counter <- counter + 1
  }
  return(listofks)
}
```

## Run the function on the 3 datasets. 
First create a data matrix for each dataset that only includes counts for highly variable genes and cells that passed the filter as determined by the Seurat functions above. Also create a vector containing the cell type identities for the cells that pass the filter by parsing the cell labels. For both the threeprime and Pulseseq datasets, this represents the raw count data subsetted to include only variable genes and cells that passed the filter. For the FullLength dataset, this represents the TPM data subsetted to include only variable genes and cells that passed the filter.
```{r run the function on the datasets}
threeprime_counts_filtered <- threeprime_seurat@raw.data[threeprime_seurat@var.genes, colnames(threeprime_seurat@data)]
dim(threeprime_counts_filtered)
threeprime_counts_filtered_celltypes <- factor(sapply(strsplit(colnames(threeprime_counts_filtered),split = "_"),function (x) x[3]))
length(threeprime_counts_filtered_celltypes)

pulseseq_counts_filtered <- pulseseq_seurat@raw.data[pulseseq_seurat@var.genes, colnames(pulseseq_seurat@data)]
dim(pulseseq_counts_filtered)
pulseseq_counts_filtered_celltypes <- factor(sapply(strsplit(colnames(pulseseq_counts_filtered),split = "_"),function (x) x[5]))
length(pulseseq_counts_filtered_celltypes)

fullLengthTPM_filtered <- fullLength_seurat@raw.data[fullLength_seurat@var.genes, ]
dim(fullLengthTPM_filtered)
fullLengthTPM_filtered_celltypes <- factor(sapply(strsplit(colnames(fullLengthTPM_filtered),split = "_"),function (x) x[4]))
length(fullLengthTPM_filtered_celltypes)



pulseseq_counts_filtered_Tuft_Neuro <- pulseseq_seurat_Tuft_Neuro@raw.data[pulseseq_seurat_Tuft_Neuro@var.genes, colnames(pulseseq_seurat_Tuft_Neuro@data)]
dim(pulseseq_counts_filtered_Tuft_Neuro)
pulseseq_counts_filtered_celltypes_Tuft_Neuro <- factor(sapply(strsplit(colnames(pulseseq_counts_filtered_Tuft_Neuro),split = "_"),function (x) x[5]))
length(pulseseq_counts_filtered_celltypes_Tuft_Neuro)

pulseseq_counts_filtered_Goblet <- pulseseq_seurat_Goblet@raw.data[pulseseq_seurat_Goblet@var.genes, colnames(pulseseq_seurat_Goblet@data)]
dim(pulseseq_counts_filtered_Goblet)
pulseseq_counts_filtered_celltypes_Goblet <- factor(sapply(strsplit(colnames(pulseseq_counts_filtered_Goblet),split = "_"),function (x) x[5]))
length(pulseseq_counts_filtered_celltypes_Goblet)

pulseseq_counts_filtered_Club <- pulseseq_seurat_Club@raw.data[pulseseq_seurat_Club@var.genes, colnames(pulseseq_seurat_Club@data)]
dim(pulseseq_counts_filtered_Club)
pulseseq_counts_filtered_celltypes_Club <- factor(sapply(strsplit(colnames(pulseseq_counts_filtered_Club),split = "_"),function (x) x[5]))
length(pulseseq_counts_filtered_celltypes_Club)

pulseseq_counts_filtered_Goblet_Club <- pulseseq_seurat_Goblet_Club@raw.data[pulseseq_seurat_Goblet_Club@var.genes, colnames(pulseseq_seurat_Goblet_Club@data)]
dim(pulseseq_counts_filtered_Goblet_Club)
pulseseq_counts_filtered_celltypes_Goblet_Club <- factor(sapply(strsplit(colnames(pulseseq_counts_filtered_Goblet_Club),split = "_"),function (x) x[5]))
length(pulseseq_counts_filtered_celltypes_Goblet_Club)

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_gom.rda")
} else {
  threeprimeGoM <- runfitGoM(t(as.matrix(threeprime_counts_filtered)))
  save(threeprimeGoM, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/threeprime_gom.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_gom.rda")
} else {
  pulseseqGoM <- runfitGoM(t(as.matrix(pulseseq_counts_filtered)))
  save(pulseseqGoM, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_gom.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_gom.rda")
} else {
  fullLengthGoM <- runfitGoM(t(as.matrix(fullLengthTPM_filtered)))
  save(fullLengthGoM, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/fullLength_gom.rda")
}




if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Tuft_Neuro_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Tuft_Neuro_gom.rda")
} else {
  pulseseqGoM_Tuft_Neuro <- runfitGoM(t(as.matrix(pulseseq_counts_filtered_Tuft_Neuro)))
  save(pulseseqGoM_Tuft_Neuro, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Tuft_Neuro_gom.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_gom.rda")
} else {
  pulseseqGoM_Goblet <- runfitGoM(t(as.matrix(pulseseq_counts_filtered_Goblet)))
  save(pulseseqGoM_Goblet, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_gom.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Club_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Club_gom.rda")
} else {
  pulseseqGoM_Club <- runfitGoM(t(as.matrix(pulseseq_counts_filtered_Club)))
  save(pulseseqGoM_Club, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Club_gom.rda")
}

if (file.exists("/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_Club_gom.rda")){
  load(file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_Club_gom.rda")
} else {
  pulseseqGoM_Goblet_Club <- runfitGoM(t(as.matrix(pulseseq_counts_filtered_Goblet_Club)))
  save(pulseseqGoM_Goblet_Club, file = "/project2/mstephens/topic-modeling-airway-epithelium/GoM_rda_files/pulseseq_Goblet_Club_gom.rda")
}
```

# Analysis of the Topics within the Topic models generated for each dataset

See the bottom of this document for visualizations for all topic models generated for each dataset as well as the top 100 genes for each respective topic model.
Visualizations:
  
  visualizeGoM plots each cell based on the tSNE1 and tSNE2 embeddings. Colors are determmined based on assignment of each cell to a particular topic based on plurality membership (cells are colored by the topic that constitutes the largest proportion out of all the topics contributing to that cell).
  
  visualizeGoM_topic_gradients plots each cell based on the tSNE1 and tSNE2 embeddings. Separate plots are created for each topic. In each plot, cells are colored on a gradient according to the proportion of each cell contributed to by the topic.
  
  plot_structure_plot plots a stacked bar plot for each cell (separated into cell type categories determined by the authors) for the topics that contribute to each cell.
  
  plot_structure_plot_rare_only plots the same as above but removes the basal and club cell bar plots to allow for better visualization of the rare cell types.
  
  make_topic_plots plots separate boxplots for each topic. The y axis in each case is proportion contributed by the topic in question, and the x axis is faceted based on the cell types assigned to single cells by the authors.
```{r functions to visualize GoM}
visualizeGoM <- function(omega, seuratobject, data){
  omega <- omega[match(seuratobject@cell.names, colnames(data)),]
  labs_clus <- colnames(omega)[apply(omega,1,which.max)]
  data_ggplot <- data.frame(labels = labs_clus,
                          tSNE_1 = seuratobject@dr$tsne@cell.embeddings[, 1],
                          tSNE_2 = seuratobject@dr$tsne@cell.embeddings[, 2])
  plot1 <- ggplot(data_ggplot) + theme(panel.background = element_rect(colour = "black", size = 1), 
                                     panel.grid.major = element_blank(), 
                                     panel.grid.minor = element_blank(), 
                                     axis.text.y = element_blank(),
                                     axis.text.x = element_blank(),
                                     axis.ticks = element_blank(),
                                     legend.text = element_text(size = 15)) + geom_point(aes(x = tSNE_1, y = tSNE_2, color = labels), size = 0.5) + labs(x = 'tSNE 1', y = 'tSNE 2', title = '') + scale_color_discrete('')
  return(plot1)
}

visualizeGoM_topic_gradients <- function(omega, seuratobject, data){
  omega <- omega[match(seuratobject@cell.names, colnames(data)),]
  ntopics <- ncol(omega)
  for (n in 1:ntopics){
  data_ggplot <- data.frame(proportion = omega[,n],
                          tSNE_1 = seuratobject@dr$tsne@cell.embeddings[, 1],
                          tSNE_2 = seuratobject@dr$tsne@cell.embeddings[, 2])
  plot.gradient <- ggplot(data_ggplot) + theme(panel.background = element_rect(colour = "black", size = 1), 
                                     panel.grid.major = element_blank(), 
                                     panel.grid.minor = element_blank(), 
                                     axis.text.y = element_blank(),
                                     axis.text.x = element_blank(),
                                     axis.ticks = element_blank(),
                                     legend.text = element_text(size = 15)) + geom_point(aes(x = tSNE_1, y = tSNE_2, color = proportion), size = 0.5) + labs(x = 'tSNE 1', y = 'tSNE 2', title = '') + scale_colour_gradient(low="grey", high="red")
  plot(plot.gradient)
  }
}

plot_structure_plot <- function(omega, seuratobject){
  annotation <- data.frame(
    sample_id = rownames(omega),
    tissue_label = seuratobject@ident
    )
  my_color_palette1.1 <- brewer.pal(8, name = "Set2")
  my_color_palette1.2 <- brewer.pal(3, name = "Set3")
  my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
  StructureGGplot(omega = omega,
                annotation = annotation,
                palette = my_color_palette1,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = "bold"))
}

plot_structure_plot_rare_only <- function(omega, seuratobject){
  annotation <- data.frame(
    sample_id = rownames(omega)[!(seuratobject@ident %in% c("Basal", "Club"))],
    tissue_label = seuratobject@ident[!(seuratobject@ident %in% c("Basal", "Club"))]
    )
  omega_rare <- omega[!(seuratobject@ident %in% c("Basal", "Club")),]
  my_color_palette1.1 <- brewer.pal(8, name = "Set2")
  my_color_palette1.2 <- brewer.pal(3, name = "Set3")
  my_color_palette1 <- c(my_color_palette1.1, my_color_palette1.2)
  StructureGGplot(omega = omega_rare,
                annotation = annotation,
                palette = my_color_palette1,
                order_sample = TRUE,
                axis_tick = list(axis_ticks_length = .1,
                                 axis_ticks_lwd_y = .1,
                                 axis_ticks_lwd_x = .1,
                                 axis_label_size = 7,
                                 axis_label_face = "bold"))
}

make_topic_plots <- function(omega, celltypes){
  omega <- as.data.frame(omega)
  omega <- cbind(omega,celltypes)
  for (k in 1:(ncol(omega)-1)) {
    base_plot <- ggplot(data = omega, mapping = aes(x = celltypes, y = omega[,k])) + labs(y = paste0("Topic ", k, " value"), x = paste0("Cell Type")) + theme(axis.text.x = element_text(angle = -20, hjust = 0, vjust = 0))
    plot(base_plot + geom_violin(width = 1.5, fill = "skyblue",color = "skyblue") + 
      geom_boxplot(width = 0.1,outlier.shape = NA, alpha = 0.2))
   }
}
```

Use the CountClust::ExtractTopFeatures function to obtain KL measurement scores for the top 100 genes per cluster that best distinguish that cluster from the rest.
```{r cluster annotations function}
annotate_clusters <- function(theta_mat, countmatrix){
  top_features <- ExtractTopFeatures(theta_mat, 
                                     top_features=100, 
                                     method="poisson", 
                                     options="min")
  gene_vector <- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) rownames(countmatrix)[top_features[[1]][x,]]))
  scores_vector <- do.call(rbind, lapply(1:dim(top_features[[1]])[1], function(x) top_features[[2]][x,]))
  print(gene_vector)
  print(scores_vector)
}
```

For the topic model fit to the threeprime dataset containing 7 topics, 

  topic 1 seems to correspond to Basal cells, 
  
  topic 2 seems to correspond to Club cells,
  
  topic 3 seems to correspond to Ciliated cells, 
  
  topic 4 seems to correspond to Goblet cells,
  
  topic 5 seems to correspond to Tuft cells, 
  
  topic 6 seems to correspond to Neuroendocrine cells,
  
  topic 7 does not seem to correspond strongly to any cell type.
```{r threeprime k of 7}
visualizeGoM(omega=threeprimeGoM[[6]]$fit$omega, seuratobject=threeprime_seurat, data=threeprime_counts_filtered)
visualizeGoM_topic_gradients(omega=threeprimeGoM[[6]]$fit$omega, seuratobject=threeprime_seurat, data=threeprime_counts_filtered)
plot_structure_plot(omega=threeprimeGoM[[6]]$fit$omega, seuratobject=threeprime_seurat)
plot_structure_plot_rare_only(omega=threeprimeGoM[[6]]$fit$omega, seuratobject=threeprime_seurat)
make_topic_plots(threeprimeGoM[[6]]$fit$omega, threeprime_counts_filtered_celltypes)
annotate_clusters(threeprimeGoM[[6]]$fit$theta, threeprime_counts_filtered)
```

For the topic model fit to the Pulseseq dataset containing 8 topics, 

  topic 2 seems to correspond to Basal and Proliferating cells,
  
  topic 5 seems to correspond to Ciliated cells.
```{r pulseseq k of 8}
visualizeGoM(omega=pulseseqGoM[[7]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered)
visualizeGoM_topic_gradients(omega=pulseseqGoM[[7]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered)
plot_structure_plot(omega=pulseseqGoM[[7]]$fit$omega, seuratobject=pulseseq_seurat)
plot_structure_plot_rare_only(omega=pulseseqGoM[[7]]$fit$omega, seuratobject=pulseseq_seurat)
make_topic_plots(pulseseqGoM[[7]]$fit$omega, pulseseq_counts_filtered_celltypes) 
annotate_clusters(pulseseqGoM[[7]]$fit$theta, pulseseq_counts_filtered)
```

For the topic model fit to the Pulseseq dataset containing 10 topics, 

  topic 2 seems to correspond to a subset of proliferating and basal cells,
  
  topic 3 seems to correspond to a subset of proliferating and basal cells,
  
  topic 9 seems to correspond to Tuft, Ionocyte, and Neuroendocrine cells,
  
  topic 10 seems to correspond to a subset of Ionocytes.
```{r pulseseq k of 10}
visualizeGoM(omega=pulseseqGoM[[9]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered)
visualizeGoM_topic_gradients(omega=pulseseqGoM[[9]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered)
plot_structure_plot(omega=pulseseqGoM[[9]]$fit$omega, seuratobject=pulseseq_seurat)
plot_structure_plot_rare_only(omega=pulseseqGoM[[9]]$fit$omega, seuratobject=pulseseq_seurat)
make_topic_plots(pulseseqGoM[[9]]$fit$omega, pulseseq_counts_filtered_celltypes)
annotate_clusters(pulseseqGoM[[9]]$fit$theta, pulseseq_counts_filtered)
```

For the topic model fit to the FullLength dataset containing 7 topics, 

  topic 1 seems to correspond to Ciliated cells, 
  
  topic 4 seems to correspond to a subset of Basal cells,
  
  topic 5 seems to correspond to a subset of Basal cells,
  
  topic 7 seems to correspond to Tuft, Neuroendocrine cells, and a subset of Ionocytes. (Note: Since the data used to do the GoM analysis was TPM data, this is not very meaningful.)
```{r FullLength k of 7}
visualizeGoM(omega=fullLengthGoM[[6]]$fit$omega, seuratobject=fullLength_seurat, data=fullLengthTPM_filtered)
visualizeGoM_topic_gradients(omega=fullLengthGoM[[6]]$fit$omega, seuratobject=fullLength_seurat, data=fullLengthTPM_filtered)
plot_structure_plot(omega=fullLengthGoM[[6]]$fit$omega, seuratobject=fullLength_seurat)
plot_structure_plot_rare_only(omega=fullLengthGoM[[6]]$fit$omega, seuratobject=fullLength_seurat)
make_topic_plots(fullLengthGoM[[6]]$fit$omega, fullLengthTPM_filtered_celltypes)
annotate_clusters(fullLengthGoM[[6]]$fit$theta, fullLengthTPM_filtered)
```

# Visualizations for All values of k for each dataset
## Visualize GoM clustering output
### Visualize threeprime Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize threeprime}
for (i in 3:length(threeprimeGoM)){
  plot(visualizeGoM(omega=threeprimeGoM[[i]]$fit$omega, seuratobject=threeprime_seurat, data=threeprime_counts_filtered))
}
for (i in 3:length(threeprimeGoM)){
  visualizeGoM_topic_gradients(omega=threeprimeGoM[[i]]$fit$omega, seuratobject=threeprime_seurat, data=threeprime_counts_filtered)
}
for (i in 3:length(threeprimeGoM)){
  plot(plot_structure_plot(omega=threeprimeGoM[[i]]$fit$omega, seuratobject=threeprime_seurat))
}
for (i in 3:length(threeprimeGoM)){
  plot(plot_structure_plot_rare_only(omega=threeprimeGoM[[i]]$fit$omega, seuratobject=threeprime_seurat))
}
for (i in 3:length(threeprimeGoM)){
  j <- i + 1
  print(paste0("Plots for ThreePrime k = ", j))
  make_topic_plots(threeprimeGoM[[i]]$fit$omega, threeprime_counts_filtered_celltypes)
}
```

### Visualize Pulseseq Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize pulseseq}
for (i in 3:length(pulseseqGoM)){
  plot(visualizeGoM(omega=pulseseqGoM[[i]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered))
}
for (i in 3:length(pulseseqGoM)){
  visualizeGoM_topic_gradients(omega=pulseseqGoM[[i]]$fit$omega, seuratobject=pulseseq_seurat, data=pulseseq_counts_filtered)
}
for (i in 3:length(pulseseqGoM)){
  plot(plot_structure_plot(omega=pulseseqGoM[[i]]$fit$omega, seuratobject=pulseseq_seurat))
}
for (i in 3:length(pulseseqGoM)){
  plot(plot_structure_plot_rare_only(omega=pulseseqGoM[[i]]$fit$omega, seuratobject=pulseseq_seurat))
}
for (i in 3:length(pulseseqGoM)){
  j <- i + 1
  print(paste0("Plots for Pulseseq k = ", j))
  make_topic_plots(pulseseqGoM[[i]]$fit$omega, pulseseq_counts_filtered_celltypes)
}
```

### Visualize Full Length Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize fullLength}
for (i in 3:length(fullLengthGoM)){
  plot(visualizeGoM(omega=fullLengthGoM[[i]]$fit$omega, seuratobject=fullLength_seurat, data=fullLengthTPM_filtered))
}
for (i in 3:length(fullLengthGoM)){
  visualizeGoM_topic_gradients(omega=fullLengthGoM[[i]]$fit$omega, seuratobject=fullLength_seurat, data=fullLengthTPM_filtered)
}
for (i in 3:length(fullLengthGoM)){
  plot(plot_structure_plot(omega=fullLengthGoM[[i]]$fit$omega, seuratobject=fullLength_seurat))
}
for (i in 3:length(fullLengthGoM)){
  plot(plot_structure_plot_rare_only(omega=fullLengthGoM[[i]]$fit$omega, seuratobject=fullLength_seurat))
}
for (i in 3:length(fullLengthGoM)){
  j <- i + 1
  print(paste0("Plots for FullLength k = ", j))
  make_topic_plots(fullLengthGoM[[i]]$fit$omega, fullLengthTPM_filtered_celltypes)
}
```

### Visualize Pulseseq (Tuft and Neuroendocrine) Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize pulseseq tuftneuro}
for (i in 1:length(pulseseqGoM_Tuft_Neuro)){
  plot(visualizeGoM(omega=pulseseqGoM_Tuft_Neuro[[i]]$fit$omega, seuratobject=pulseseq_seurat_Tuft_Neuro, data=pulseseq_counts_filtered_Tuft_Neuro))
}
for (i in 1:length(pulseseqGoM_Tuft_Neuro)){
  visualizeGoM_topic_gradients(omega=pulseseqGoM_Tuft_Neuro[[i]]$fit$omega, seuratobject=pulseseq_seurat_Tuft_Neuro, data=pulseseq_counts_filtered_Tuft_Neuro)
}
for (i in 1:length(pulseseqGoM_Tuft_Neuro)){
  plot(plot_structure_plot(omega=pulseseqGoM_Tuft_Neuro[[i]]$fit$omega, seuratobject=pulseseq_seurat_Tuft_Neuro))
}
for (i in 1:length(pulseseqGoM_Tuft_Neuro)){
  plot(plot_structure_plot_rare_only(omega=pulseseqGoM_Tuft_Neuro[[i]]$fit$omega, seuratobject=pulseseq_seurat_Tuft_Neuro))
}
for (i in 1:length(pulseseqGoM_Tuft_Neuro)){
  j <- i + 1
  print(paste0("Plots for Pulseseq Tuft_Neuro k = ", j))
  make_topic_plots(pulseseqGoM_Tuft_Neuro[[i]]$fit$omega, pulseseq_counts_filtered_celltypes_Tuft_Neuro)
}
```

### Visualize Pulseseq (Goblet) Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize pulseseq Goblet}
for (i in 1:5){
  plot(visualizeGoM(omega=pulseseqGoM_Goblet[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet, data=pulseseq_counts_filtered_Goblet))
}
for (i in 1:5){
  visualizeGoM_topic_gradients(omega=pulseseqGoM_Goblet[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet, data=pulseseq_counts_filtered_Goblet)
}
for (i in 1:5){
  plot(plot_structure_plot(omega=pulseseqGoM_Goblet[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet))
}
for (i in 1:5){
  plot(plot_structure_plot_rare_only(omega=pulseseqGoM_Goblet[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet))
}
for (i in 1:5){
  j <- i + 1
  print(paste0("Plots for Pulseseq Goblet k = ", j))
  make_topic_plots(pulseseqGoM_Goblet[[i]]$fit$omega, pulseseq_counts_filtered_celltypes_Goblet)
}
```

### Visualize Pulseseq (Club) Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize pulseseq club}
for (i in 1:5){
  plot(visualizeGoM(omega=pulseseqGoM_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Club, data=pulseseq_counts_filtered_Club))
}
for (i in 1:5){
  visualizeGoM_topic_gradients(omega=pulseseqGoM_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Club, data=pulseseq_counts_filtered_Club)
}
for (i in 1:5){
  plot(plot_structure_plot(omega=pulseseqGoM_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Club))
}
for (i in 1:5){
  plot(plot_structure_plot_rare_only(omega=pulseseqGoM_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Club))
}
for (i in 1:5){
  j <- i + 1
  print(paste0("Plots for Pulseseq Club k = ", j))
  make_topic_plots(pulseseqGoM_Club[[i]]$fit$omega, pulseseq_counts_filtered_celltypes_Club)
}
```

### Visualize Pulseseq (Goblet and Club) Dataset in tSNE plots, STRUCTURE plots, boxplots, and violin plots
```{r visualize pulseseq goblet club}
for (i in 1:length(pulseseqGoM_Goblet_Club)){
  plot(visualizeGoM(omega=pulseseqGoM_Goblet_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet_Club, data=pulseseq_counts_filtered_Goblet_Club))
}
for (i in 1:length(pulseseqGoM_Goblet_Club)){
  visualizeGoM_topic_gradients(omega=pulseseqGoM_Goblet_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet_Club, data=pulseseq_counts_filtered_Goblet_Club)
}
for (i in 1:length(pulseseqGoM_Goblet_Club)){
  plot(plot_structure_plot(omega=pulseseqGoM_Goblet_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet_Club))
}
for (i in 1:length(pulseseqGoM_Goblet_Club)){
  plot(plot_structure_plot_rare_only(omega=pulseseqGoM_Goblet_Club[[i]]$fit$omega, seuratobject=pulseseq_seurat_Goblet_Club))
}
for (i in 1:length(pulseseqGoM_Goblet_Club)){
  j <- i + 1
  print(paste0("Plots for Pulseseq Goblet_Club k = ", j))
  make_topic_plots(pulseseqGoM_Goblet_Club[[i]]$fit$omega, pulseseq_counts_filtered_celltypes_Goblet_Club)
}
```

## Cluster annotations
### Top 100 Genes Driving Clusters in the threeprime Dataset 
```{r visualize threeprime genes}
annotate_clusters(threeprimeGoM[[3]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[4]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[5]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[6]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[7]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[8]]$fit$theta, threeprime_counts_filtered)
annotate_clusters(threeprimeGoM[[9]]$fit$theta, threeprime_counts_filtered)
```

### Top 100 Genes Driving Clusters in the Pulseseq Dataset 
```{r visualize pulseseq genes}
annotate_clusters(pulseseqGoM[[3]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[4]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[5]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[6]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[7]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[8]]$fit$theta, pulseseq_counts_filtered)
annotate_clusters(pulseseqGoM[[9]]$fit$theta, pulseseq_counts_filtered)
```

### Top 100 Genes Driving Clusters in the FullLength Dataset 
```{r visualize fullLength genes}
annotate_clusters(fullLengthGoM[[3]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[4]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[5]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[6]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[7]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[8]]$fit$theta, fullLengthTPM_filtered)
annotate_clusters(fullLengthGoM[[9]]$fit$theta, fullLengthTPM_filtered)
```

### Top 100 Genes Driving Clusters in the Pulseseq (Tuft and Neuroendocrine) Dataset 
```{r visualize pulseseq genes tuft neuro}
annotate_clusters(pulseseqGoM_Tuft_Neuro[[1]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[2]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[3]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[4]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[5]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[6]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[7]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[8]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
annotate_clusters(pulseseqGoM_Tuft_Neuro[[9]]$fit$theta, pulseseq_counts_filtered_Tuft_Neuro)
```

### Top 100 Genes Driving Clusters in the Pulseseq (Goblet) Dataset 
```{r visualize pulseseq genes goblet}
#annotate_clusters(pulseseqGoM_Goblet[[1]]$fit$theta, pulseseq_counts_filtered_Goblet)
annotate_clusters(pulseseqGoM_Goblet[[2]]$fit$theta, pulseseq_counts_filtered_Goblet)
annotate_clusters(pulseseqGoM_Goblet[[3]]$fit$theta, pulseseq_counts_filtered_Goblet)
annotate_clusters(pulseseqGoM_Goblet[[4]]$fit$theta, pulseseq_counts_filtered_Goblet)
annotate_clusters(pulseseqGoM_Goblet[[5]]$fit$theta, pulseseq_counts_filtered_Goblet)
```

### Top 100 Genes Driving Clusters in the Pulseseq (Club) Dataset 
```{r visualize pulseseq genes club}
annotate_clusters(pulseseqGoM_Club[[1]]$fit$theta, pulseseq_counts_filtered_Club)
annotate_clusters(pulseseqGoM_Club[[2]]$fit$theta, pulseseq_counts_filtered_Club)
annotate_clusters(pulseseqGoM_Club[[3]]$fit$theta, pulseseq_counts_filtered_Club)
annotate_clusters(pulseseqGoM_Club[[4]]$fit$theta, pulseseq_counts_filtered_Club)
annotate_clusters(pulseseqGoM_Club[[5]]$fit$theta, pulseseq_counts_filtered_Club)
```

### Top 100 Genes Driving Clusters in the Pulseseq (Goblet and Club) Dataset 
```{r visualize pulseseq genes goblet club}
annotate_clusters(pulseseqGoM_Goblet_Club[[1]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[2]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[3]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[4]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[5]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[6]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[7]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[8]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
annotate_clusters(pulseseqGoM_Goblet_Club[[9]]$fit$theta, pulseseq_counts_filtered_Goblet_Club)
```