---
title: "Preprocessing bulk RNA data"
author: "Anthony Hung"
date: "2019-12-16"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

# Visualizing metrics for bulk data

library size
rin scores
duplication rate
total reads vs mapped reads box plot
gc content





<!-- ## Setup -->

<!-- ```{r packages, message=FALSE} -->
<!-- library("data.table") -->
<!-- library("dplyr") -->
<!-- library("tidyr") -->
<!-- library("ggplot2") -->
<!-- library("cowplot") -->
<!-- theme_set(theme_bw(base_size = 12)) -->
<!-- theme_update(panel.grid.minor.x = element_blank(), -->
<!--              panel.grid.minor.y = element_blank(), -->
<!--              panel.grid.major.x = element_blank(), -->
<!--              panel.grid.major.y = element_blank()) -->
<!-- ``` -->

<!-- ```{r input-total-counts} -->
<!-- total <- fread("../data/total-counts.txt", data.table = FALSE) -->
<!-- total$stage <- factor(total$stage, levels = c("raw", "mapped to genome", "mapped to exons")) -->
<!-- total$status <- factor(total$status, levels = c("contact", "tb")) -->
<!-- total$treatment <- factor(total$treatment, levels = c("none", "infected")) -->
<!-- total$counts_mil <- total$counts / 10^6 -->
<!-- total$flow_cell <- factor(total$flow_cell, levels = c("HCMF5BBXX", "HCN77BBXX", "H7LVWBBXX")) -->
<!-- head(total) -->
<!-- ``` -->

<!-- ## Sequencing output per lane -->

<!-- ```{r sequencing-output, fig.width=6, fig.height=4} -->
<!-- ggplot(total[total$stage == "raw", ], aes(x = lane, y = counts_mil)) + -->
<!--   geom_bar(stat = "identity") + -->
<!--   facet_wrap(~flow_cell) + -->
<!--   labs(x = "Sequencing lane", y = expression("Number of raw reads (" * 10^6 * ")"), -->
<!--        title = "Sequencing output from HiSeq 4000") -->
<!-- ``` -->

<!-- ## Number of reads across processing stages -->

<!-- Then summarize and show per sample: -->

<!-- ```{r} -->
<!-- total_per_sample <- total %>% -->
<!--   group_by(stage, id, status, treatment) %>% -->
<!--   summarize(counts = sum(counts), -->
<!--             counts_mil = counts / 10^6) %>% -->
<!--   group_by(stage, status, treatment) %>% -->
<!--   mutate(counts_mil_median = round(median(counts_mil), digits = 2)) -->
<!-- ``` -->

<!-- ```{r reads-per-processing-stage, fig.width=6, fig.height=6} -->
<!-- ggplot(total_per_sample, aes(x = stage, y = counts_mil)) + -->
<!--   geom_boxplot() + -->
<!--   facet_grid(status ~ treatment) + -->
<!--   geom_text(aes(label = counts_mil_median, y = 0), size = 5) + -->
<!--   scale_x_discrete(labels = c("Raw\nreads", "Mapped\nto\ngenome\n(uniquely)", "Assigned\nto\nexons\n(protein-coding)")) + -->
<!--   labs(x = "Processing stage", y = expression("Number of reads (" * 10^6 * ")"), -->
<!--        title = "Number of reads across processing stages\n(median displayed below)") -->
<!-- ``` -->

<!-- ## Distribution of final exonic reads across samples -->

<!-- ```{r exonic-reads-per-sample, fig.width=6, fig.height=4} -->
<!-- ggplot(total_per_sample[total_per_sample$stage == "mapped to exons", ], -->
<!--        aes(x = reorder(id, counts_mil), y = counts_mil, color = status, shape = treatment)) + -->
<!--   geom_point() + -->
<!--    labs(x = "Sample", y = expression("Number of exonic reads (" * 10^6 * ")"), -->
<!--        title = "Distribution of exonic reads per sample") + -->
<!--   theme(legend.position = c(0.25, 0.5)) + -->
<!--   theme(axis.ticks = element_blank(), axis.text.x = element_blank()) -->
<!-- ``` -->